{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "# import cell\n",
    "import nltk\n",
    "from nltk.corpus import stopwords\n",
    "import re\n",
    "import pandas as pd\n",
    "import pickle\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# conda environments:\n",
      "#\n",
      "base                  *  C:\\Users\\paten\\anaconda3\n",
      "my-rdkit-env             C:\\Users\\paten\\anaconda3\\envs\\my-rdkit-env\n",
      "py37                     C:\\Users\\paten\\anaconda3\\envs\\py37\n",
      "\n",
      "\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "# conda info --env"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 유사도 확인 함수\n",
    "import numpy as np\n",
    "\n",
    "def most_similar(doc_id,similarity_matrix,matrix):\n",
    "    print (f'Document: {documents_df.iloc[doc_id][\"documents\"]}')\n",
    "    print ('\\n')\n",
    "    print ('Similar Documents:')\n",
    "    if matrix=='Cosine Similarity':\n",
    "        similar_ix=np.argsort(similarity_matrix[doc_id])[::-1]\n",
    "    elif matrix=='Euclidean Distance':\n",
    "        similar_ix=np.argsort(similarity_matrix[doc_id])\n",
    "    for ix in similar_ix:\n",
    "        if ix==doc_id:\n",
    "            continue\n",
    "        print('\\n')\n",
    "        print (f'Document: {documents_df.iloc[ix][\"documents\"]}')\n",
    "        print (f'{matrix} : {similarity_matrix[doc_id][ix]}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load documents \n",
    "\n",
    "# Drug bank complete slim data\n",
    "drugbank_approved_df = pd.read_csv('D:/WorkSpace/master_thesis/Drug_Bank/result/drugbank_approved.tsv', sep='\\t')\n",
    "drugName = drugbank_approved_df['name']\n",
    "\n",
    "# Drugbank vocabulary data\n",
    "drugbank_voca_df = pd.read_csv('D:/WorkSpace/master_thesis/Drug_Bank/drugbank_vocabulary.csv')\n",
    "\n",
    "# documents data (title, abstracts)\n",
    "with open('D:/WorkSpace/master_thesis/data_pubmed/documents/approved_documents.pkl', 'rb') as f:\n",
    "    documents = pickle.load(f)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approved Drug: 4270\n",
      "number of scraped Documents: 4027\n"
     ]
    }
   ],
   "source": [
    "print('Approved Drug: {}'.format(len(drugbank_approved_df)))\n",
    "# len(drugbank_voca_df)\n",
    "print('number of scraped Documents: {}'.format(len(documents)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [],
   "source": [
    "# convert from dictionary to data_frame \n",
    "documents_df = pd.DataFrame(list(documents.items()), columns=['drugName', 'documents'])\n",
    "\n",
    "\n",
    "# Documents 소문자 변환\n",
    "for i in range(len(documents_df)):\n",
    "    documents_df.documents[i] = df_documents.documents[i].lower()\n",
    "\n",
    "\n",
    "# Remove stopwords from nltk\n",
    "stop_words_l = stopwords.words('english')\n",
    "documents_df['documents_cleaned'] = documents_df.documents.apply(lambda x: \" \".join(re.sub(r'[^a-zA-Z]',' ',w).lower() for w in x.split() if re.sub(r'[^a-zA-Z]',' ',w).lower() not in stop_words_l) )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [],
   "source": [
    "# using base BERT\n",
    "from sentence_transformers import SentenceTransformer\n",
    "sbert_model = SentenceTransformer('bert-base-nli-mean-tokens')\n",
    "\n",
    "document_embeddings = sbert_model.encode(documents_df['documents_cleaned'])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "biobert_model = SentenceTransformer('')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n",
      "Document:   \n",
      "Cosine Similarity : -0.021146124228835106\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document: changing wheat bran structural properties by extrusion-cooking on a pilot and industrial scale: a comparative study. combination of extrusion and fermentation with  influence of durum wheat bran particle size on phytochemical content and on leavened bread baking quality. immunomodulatory effects of wheat bran arabinoxylan on raw264.7 macrophages via the nf-κb signaling pathway using rna-seq analysis. fortification of yellow alkaline noodles with wheat bran and the impact on physical and sensorial properties. a wheat bran inducible expression system for the efficient production of α-l-arabinofuranosidase in bacillus subtilis. conversion of wheat bran to xylanases and dye adsorbent by  efficiency of twin-screw extrusion of biodegradable poly (butylene succinate)-wheat bran blend. effects of different solid-state fermentation ratios of s. cerevisiae and l. plantarum on physico-chemical properties of wheat bran and the quality of whole wheat bread. digestibility, lactation performance, plasma metabolites, ruminal fermentation, and bacterial communities in holstein cows fed a fermented corn gluten-wheat bran mixture as a substitute for soybean meal. the effect of xylooligosaccharide, xylan, and whole wheat bran on the human gut bacteria. preparative separation of high-purity trans- and cis-ferulic acid from wheat bran by ph-zone-refining counter-current chromatography. effects of wheat bran and  the effect of wet milling and cryogenic milling on the structure and physicochemical properties of wheat bran. pilot-scale ohmic heating-assisted extraction of wheat bran bioactive compounds: effects of the extract on corn oil stability. preparation and characterization of ph-responsive microgel using arabinoxylan from wheat bran for bsa delivery. regression-derived ileal endogenous amino acid losses in broiler chickens and cannulated pigs fed corn fiber, wheat bran, and pectin. effects of wheat bran applied to maternal diet on the intestinal architecture and immune gene expression in suckling piglets. saccharomyces cerevisiae and phytase co-fermentation wheat bran on growth, antioxidation, immunity and intestinal morphology in broilers. a double-blind, placebo-controlled randomized trial of skin-lightening cream containing lycopene and wheat bran extract on melasma. extrusion-cooking can be used to change the techno-functional and nutrition-related properties of wheat bran. in this study, pilot-scale (bc21) and industrial-scale (bc45) twin-screw extrusion-cooking using different types of extrusion (single-pass, double-pass and acid extrusion-cooking) and process parameters (temperature, moisture) were compared for their impact on wheat bran. when applying the same process settings, the higher strong water-binding capacity, extract viscosity and extractability displayed by bran extruded using the industrial set-up reflected a more considerable wheat bran structure degradation compared to pilot-scale extrusion-cooking. this was attributed to the overall higher specific mechanical energy (sme), pressure and product temperature that were reached inside the industrial extruder. when changing the type of extrusion-cooking from single-pass to double-pass and acid extrusion-cooking, wheat bran physicochemical characteristics evolved in the same direction, irrespective of extruder scale. the differences in bran characteristics were, however, smaller on industrial-scale. results show that the differentiating power of the latter can be increased by decreasing the moisture content and increasing product temperature, beyond what is possible in the pilot-scale extruder. this was confirmed by using a bc72 industrial-scale extruder at low moisture content. in conclusion, the extruder scale mainly determines the sme that can be reached and determines the potential to modify wheat bran. processed wheat bran (w) is of great importance for food and feed. consequently, the biosafety of w should be evaluated and improved with valorisation strategies. this study tested a design combining extrusion (at temperature of 115 and 130 °c; screw speeds of 16, 20, and 25 rpm) and fermentation with lactobacillus plantarum and l. uvarum strains for the valorisation of w to provide safer food and feed stock. the influence of different treatments on biogenic amine formation, mycotoxin content, and free amino acids, as well as acidity, microbiological parameters, and sugar concentration, were analysed. this research showed that a combination of extrusion and fermentation with selected strains can change several aspects of w characteristics. there was a significant effect of applied treatments on acidity and the microbiological parameters of w, as well as biogenic amines content. the lowest total mycotoxin concentration (29.8 µg/kg) was found in extruded (130 °c; 25 rpm) and fermented with l. uvarum sample. finally, the combination of the abovementioned treatments can be confirmed as a prospective innovative pre-treatment for w, capable of potentially enhancing their safety characteristics and composition. wheat bran is a conventional by-product of the wheat milling industry mainly used for animal feed. it is a rich and inexpensive source of phytonutrients, so is in demand for fibre-rich food products but creates quality issues when incorporated into bread. the purpose of this study was to characterize the physicochemical properties and phytochemical composition of different size durum bran fractions and show how they impact bread quality. durum wheat (triticum durum desf.) was milled to create a coarse bran fraction (cb), which was further ground into a finer fraction (fb) which was sieved using four screens with apertures 425, 315, 250, 180, and <180 µm to create a particle size range of 1497 to 115 µm. all fractions contained phytosterol with highest in the 180 and fb, while total phenolic acids and antioxidant capacity was highest in cb and 425. use of the fractions in a leavened common wheat (t. aestivum l.) bread formula at 10% incorporation negatively impacted bread loaf volume, colour, and texture compared to standard loaves, with cb having the least impact. results suggest that to combine the highest phytochemical content with minimal impact on bread quality, bran particle size should be considered, with cb being the best choice. arabinoxylan (ax) extracted from wheat bran has attracted much attention due to its immunomodulatory activity. however, the molecular mechanisms underlying this activity remain unclear. in this study, we conducted a comprehensive transcriptional study to investigate genetic changes related to ax and identified 2325 differentially expressed genes (degs). gene ontology classification revealed that the degs were mainly enriched in a series of immune-related processes. kyoto encyclopedia of genes and genomes pathway analysis revealed that immune-related pathways were significantly enriched in top 20 pathways, including the nuclear factor-kappa b (nf-κb) signaling pathway and the tnf signaling pathway. validation using quantitative polymerase chain reaction analysis revealed dynamic changes in the mrna levels of immune-related cd40, csf1, csf2, fas, il-1β, il-6, il-5, irf1, and tnfaip3, which were significantly up-regulated in the ax-treated group. moreover, ax treatment led to the up-regulation of the nuclear translocation of nf-κb and its upstream target proteins such as pdk1, akt, iκb-α, and gsk-3β. the dataset compiled from this study provides valuable information for further research on the complex molecular mechanisms associated with ax and the identification of target genes. food fortification is a commonly-used method to increase nutritional value of food products in order to reduce the risks of non-communicable diseases (ncds). noodles are a versatile and popular staple food, especially in asia, that can be nutritionally enriched using various types of ingredients and one of them is wheat bran that is rich in dietary fiber. this study aimed to determine effect of fortification of fresh yellow alkaline noodles with wheat bran on its physical and sensorial properties. addition of wheat bran was varied in concentration (0-20%) and physical properties of fortified yellow alkaline noodles were assessed. additionally, 9-point hedonic scale and 'just about right' scale were employed to assess sensory properties of the fortified noodles, using 40 non-trained panelists. noodles fortified with the highest amount of wheat bran exhibited the highest cooking loss (4.61%) compared to the non-fortified one (2.78%), indicating weak structural integrity, presumably due to the impaired gluten network. on contrary, noodles fortified with various amount of wheat bran did not demonstrate significant alteration of textural properties in terms of springiness, cohesiveness, and resilience. sensorially, noodles fortified with 10% of wheat bran had the highest preference among the panelist based on general appearance, overall acceptability, color suitability, hardness suitability, and smoothness. proximate analysis showed noodles supplemented with 10% of wheat bran met regulatory criteria for claim as a high-fiber food, with its dietary fiber content of 5.40 g/100 g of noodles. conclusively, fortification of yellow alkaline noodles with wheat bran could improve its product properties. α-l-arabinofuranosidases (ec 3.2.1.55; afs) cause the release of arabinosyl residues from hemicellulose polymers such as xylans, and are receiving increased levels of research attention as they could be applied in a range of processes that involve the enzymatic degradation of xylans. the secretory production of bacterial afs has not been attempted previously. in this study, we designed a unique induction system for the production of a recombinant af in bacillus subtilis in order to exploit its enzymic degradation of wheat bran. we found that non-starch phytochemicals were more efficient than d-xylose when inducing the expression of t7 rna polymerase and driving the transcription of af by the t7 promoter. the host cell, b. subtilis (atcc 6051a-derived strain 164t7p) was engineered to incorporate a dna cassette that expressed t7 rna polymerase under the control of a d-xylose inducible promoter (pxyla). the t7 promoter engineered into 164t7p was initially tested and compared with p43 in terms of gfp expression; we found that the expression level of gfp by the t7 promoter was ten-fold higher than that achieved by p43. when cultured in a flask with gentle shaking, and with d-xylose as an inducer, the recombinant strain successfully expressed arbf, a family 51 (gh 51) glycoside hydrolase from bacillus licheniformis, and secreted 141.4 ± 4.8 u/ml of enzyme, with a km of 1.4 ± 0.1 mm and a kcat of 139.4 s-1. however, the protein was devoid of a secretary signal peptide. when cultures were supplemented with wheat bran, the maximal yield of the secreted af reached 194.8 ± 4.1 u/ml. the results provide a foundation for the high level production of heterologous proteins using wheat bran as the inducer in b. subtilis. agro-byproducts can be utilized as effective and low-cost nutrient sources for microbial fermentation to produce a variety of usable products. in this study, wheat bran powder (wbp) was found to be the most effective carbon source for xylanase production by streptomyces thermocarboxydus tku045. the optimal media for xylanase production was 2% (w/v) wbp, 1.50% (w/v) kno3, 0.05% (w/v) mgso4, and 0.10% (w/v) k2hpo4, and the optimal culture conditions were 50 ml (in a 250 ml-volume erlenmeyer flask), initial ph 9.0, 37 °c, 125 rpm, and 48 h. accordingly, the highest xylanase activity was 6.393 ± 0.130 u/ml, 6.9-fold higher than that from un-optimized conditions. s. thermocarboxydus tku045 secreted at least four xylanases with the molecular weights of >180, 36, 29, and 27 kda when cultured on the wbp-containing medium. the enzyme cocktail produced by s. thermocarboxydus tku045 was optimally active over a broad range of temperature and ph (40-70 °c and ph 5-8, respectively) and could hydrolyze birchwood xylan to produce xylobiose as the major product. the obtained xylose oligosaccharide (xos) were investigated for 2,2-diphenyl-1-picrylhydrazyl radical scavenging activity and the growth effect of lactic acid bacteria. finally, the solid waste from the wbp fermentation using s. thermocarboxydus tku045 revealed the high adsorption of congo red, red 7, and methyl blue. thus, s. thermocarboxydus tku045 could be a potential strain to utilize wheat bran to produce xylanases for xos preparation and dye adsorbent. unmodified poly (butylene succinate) (pbs) is characterized by very good processability; however, after the incorporation of various fillers of plant origin, its processing becomes much more complicated and its properties are significantly affected. detailed studies of the processing aspects of pbs/wheat bran (wb) biocomposition are lacking, despite the addition of wb having a significant impact on both the production efficiency and the properties of end products. this research paper presents test results of the co-rotating twin-screw extrusion processing of a biodegradable polymer blend, the matrix of which was pbs, with wb as the filler. in undertaking this task, we examined the impact of extruder screw rotational speed and wb content on the characteristics of extrusion processing, as well as on certain thermal, physical, structural and processing properties of the obtained blend. the wb introduced to the blend was in the form of a selected fraction with particles smaller than 0.2 mm. the measurements were conducted using the design of experiment (doe) methods, which enabled establishing the studied relationships in the form of polynomials and response surfaces. the determined extrusion process characteristics covered the impact of screw rotational speed and wb content on the mass flow rate of the processed blend and its pressure, the screw drive torque and specific energy consumption. the studies of the obtained polymer blend included determining the impact of the aforementioned variable factors on the melt flow rate (mfr) index, chemical structure (ftir), thermal properties (differential scanning calorimetry (dsc), thermogravimetry (tg), derivative thermogravimetry (dtg)), p-v-t relationships, microstructure, density and moisture absorbance. analysis of variance (anova) was used to assess the effect of individual variable factors. the results of this work are presented, inter alia, using pareto charts of standardized effects, which illustrate the influence of individual terms of the determined regression equations on the studied quantity. the addition of wheat bran (wb) could make full use of the nutritional activity; however, it also caused many negative effects on the quality of whole wheat bread (wwb). to improve the physico-chemical properties of wb and the quality of wwb, wb was solid-state fermented with different ratios of commercially available s. cerevisiae and l. plantarum, and utilized to prepare wwb. the physico-chemical properties of wb including dietary fiber content and its components, amino acid composition, and antioxidant activities were determined. after solid-state fermentation, the physico-chemical properties of wb were improved. wbsac:lac=2:1 showed higher antioxidant activities (only the total antioxidant activity was slightly lower than wbsac:lac=1:1 ), greater concentration of soluble dietary fiber (9.22%) and essential amino acids/total amino acids (42.04) than the other wb samples. furthermore, the wwb quality was investigated by measuring specific volume, porosity, texture, aroma, and volatile compounds. wwb made with wbsac:lac=2:1 showed a higher specific volume, more uniform porosity structure, better texture, and more volatile compounds than the other samples. using a ratio of yeast and lactobacilli of 2:1, the solid-state fermentation maximally improves the processing properties of wb, and prepares wwb with the best quality. this article is protected by copyright. all rights reserved. the purpose of this research was to investigate the effects of replacing soybean meal (sbm) with a fermented corn gluten-wheat bran mixture (fcwm) on nutrient digestibility, lactation performance, plasma metabolites, ruminal fermentation, and bacterial communities in holstein cows. nine healthy multiparous (parity = 3) holstein cows with similar body weights (624 ± 14.4 kg), days in milk (112 ± 4.2), and milk yields (31.8 ± 1.73 kg; all mean ± standard deviation) were used in a replicated 3 × 3 latin square design with 3 periods of 28 d. cows were fed 1 of 3 dietary treatments in which fcwm replaced sbm as follows: basal diet with no replacement (0fcwm); 50% replacement of sbm with fcwm (50%fcwm); and 100% replacement of sbm with fcwm (100%fcwm). the diets were formulated to be isocaloric and isonitrogenous. the results showed that the total-tract digestibility of dry matter and crude protein increased linearly with increased dietary fcwm, and we found a trend for increased total-tract neutral detergent fiber and potentially digestible ndf digestibility. milk yield tended to increase in a linear manner as more fcwm was consumed, and energy-corrected milk production was significantly increased with fcwm supplementation as a result of increased milk protein and lactose yields. plasma glucose and igg concentrations increased linearly with increasing fcwm supplementation, but plasma malondialdehyde concentration decreased linearly. concentrations of total volatile fatty acids and propionate showed a linear increase with increasing fcwm supplementation, leading to a linear decrease in ph. the relative abundance of ruminal prevotellaceae, veillonellaceae, and prevotella 1 increased linearly with increasing fcwm supplementation, and the relative abundance of ruminal succinivibrionaceae and muribaculaceae decreased linearly. the relative abundance of fecal ruminococcaceae, prevotellaceae, and ruminococcaceae ucg-005 increased linearly with increasing fcwm supplementation, but the relative abundance of fecal peptostreptococcaceae decreased linearly. overall, the replacement of sbm with fcwm altered the composition of the ruminal bacterial community and improved nutrient digestibility, lactation performance, and ruminal fermentation in cows, providing a data reference for the use of fcwm in dairy production. wheat bran is a cereal rich in dietary fibers that have high levels of ferulic acid, which has prebiotic effects on the intestinal microbiota and the host. herein we explored the effect of xylooligosaccharide, xylan, and whole wheat bran on the human gut bacteria and screened for potential ferulic acid esterase genes. using in vitro fermentation, we analyzed the air pressure, ph-value, and short-chain fatty acid levels. we also performed 16s rrna gene and metagenomic sequencing. a venn diagram analysis revealed that 80% of the core operational taxonomic units (otus) were shared among the samples, and most of the xylooligosaccharide treatment core otus (319/333 otus) were shared with the other two treatments' core otus. a significant difference analysis revealed that the relative abundance of dorea, bilophila, and sulfurovum in wheat bran treatment was higher than that in xylan and xylooligosaccharide treatments. the clusters of orthologous groups of proteins functional composition of all samples was similar to the microbiota composition of the control. using metagenomic sequencing, we revealed seven genes containing the conserved residues, gly-x-ser-x-gly, and the catalytic triad, ser-his-asp, which are thus potential ferulic acid esterase genes. all the results indicate that xylan and/or xylooligosaccharide, the main dietary fibers in wheat bran, plays a major role in in vitro fermentation by the human gut microbiota. ferulic acid stereoisomers are the most abundant phenolic acids in cereal bran. however, it is challenging to separate them because of the similar structures and properties. in this study, a preparative separation method of ferulic acid stereoisomers from the crude extract of wheat bran was successfully developed. the method contained a two-step separation, the traditional counter-current chromatography (ccc, hexane: ethyl acetate: methanol: water = 2:5:2:4) was followed with a ph-zone-refining ccc (hexane: ethyl acetate: acetonitrile: water = 2:5:2:2, 10 mmol l-1 trifluoroacetic acid in organic stationary phase and 10 mmol l-1 ammonia in aqueous mobile phase). trans-ferulic acid and cis-ferulic acid with hplc high purity over than 99% and 98% can be yielded in large-scale separation. moreover, it is found that different proton affinity, deprotonation ability and interaction site of hydrogen bond result in distinct partition behavior of stereoisomers, which is illustrated by quantitative analysis of molecular surface. this contributes to our in-depth understanding of the separation mechanism toward ph-zone refining ccc. the developed method can be applied in the exploitation of ferulic acids and related phenolic acids from other resources. feed additives that can improve intestinal health and maintain a diverse and resilient intestinal microbiota of poultry are of great importance. thus, the current study investigated the effects of a single strain butyric acid-producing clostridium (c. butyricum) with (symbiotic) or without wheat bran supplementation on cecal microbiota composition and gut health characteristics of broiler chickens. in total, 384 male ross 308 day-old chickens were divided into four dietary treatment groups and fed ad libitum until day 37 of life. cecal samples were taken for illumina sequencing and ph and short-chain fatty acid analyses, as well as for histological analysis at the end of the experimental period. neither of the supplemented diets improved chicken growth performance. caecum was dominated by the members of bacteroidetes phyla followed by firmicutes in each dietary group. at the genus level, bacteroides, oscillospira, akkermansia, faecalibacterium, ruminococcus and streptococcus genera exceeded 1% relative abundance. dietary treatment influenced the relative abundance of the akkermansia genus, which had a lower relative abundance in the c. butyricum group than in the other groups and in the symbiotic group compared to the wheat bran supplemented group. dietary treatment also altered cecal crypt depth and had a trend to modify the cecal fermentation profile. additive effects of wheat bran and c. butyricum supplementation were not detected. our results suggest that akkermansia muciniphila colonization in chicken can be influenced by diet composition. wheat bran consumption is associated with several health benefits, but its incorporation into food products remains low because of sensory and technofunctional issues. besides, its full beneficial potential is probably not achieved because of its recalcitrant nature and inaccessible structure. particle size reduction can affect both technofunctional and nutrition-related properties. therefore, in this study, wet milling and cryogenic milling, two techniques that showed potential for extreme particle size reduction, were used. the effect of the milling techniques, performed on laboratory and large scale, was evaluated on the structure and physicochemical properties of wheat bran. with a median particle size (d50) of 6 µm, the smallest particle size was achieved with cryogenic milling on a laboratory scale. cryogenic milling on a large scale and wet milling on laboratory and large scale resulted in a particle size reduction to a d50 of 28-38 µm. in the milled samples, the wheat bran structure was broken down, and almost all cells were opened. wet milling on laboratory and large scale resulted in bran with a more porous structure, a larger surface area and a higher capacity for binding water compared to cryogenic milling on a large scale. the extensive particle size reduction by cryogenic milling on a laboratory scale resulted in wheat bran with the highest surface area and strong water retention capacity. endogenous enzyme activity and mechanical breakdown during the different milling procedures resulted in different extents of breakdown of starch, sucrose, β-glucan, arabinoxylan and phytate. therefore, the diverse impact of the milling techniques on the physicochemical properties of wheat bran could be used to target different technofunctional and health-related properties. recent studies introduced ohmic heating-assisted extraction (ohae) as a promising emerging technology at laboratory-scales. the objectives of the present study were, first, to investigate the applicability of ohae at pilot-scale for extraction of bioactive compounds from wheat bran immersed in a polar solvent (salted water containing 0.1% nacl) at the electric field strengths (efs) of 4.28, 7.90, and 15.71 v/cm and, second, to evaluate the effects of the wheat extracts on the corn oil stability during 30 days of storage at 45 °c. the results showed that ohae saved 63% of energy consumption compared with the conventional extraction method. also, the scaled-up ohae unit yielded extracts with high quantities of bioactive compounds (110-460 ppm total phenolics) and higher antioxidant activities (antioxidant effectiveness of 56-84%) than those of the extract obtained through the conventional extraction method, i.e., 95 ppm total phenolics with antioxidant effectiveness of 51%. increasing the efs increased total phenolics and antioxidant effectiveness of extracts. the incorporation of 250 ppm of the extract obtained at the highest efs effectively postponed the oxidation of corn oil during one month of storage (peroxide value of 7 vs. 19 meq/kg compared with the control sample) and extended the half-life of oil from 11 to 26 days. besides, mathematical models proposed in this study well-predicted the oxidation stability of the oil samples mixed with the extract. wheat bran arabinoxylan (ax) discard from wheat production was utilized to form ph-responsive microgels. ax was modified by carboxymethylation, and the carboxymethylated arabinoxylans (cmax) were characterized by ft-ir, nmr, gel permeation chromatography (gpc), and rheological analysis. the cmax microgel was cross-linked by fe3+ using an inverse emulsification polymerization. the morphology, particle size, ph sensitivity, and mechanism of cross-linking between coo- and fe3+ of the cmax microgel was investigated. the cmax microgel was used to be an oral protein drug carrier. the cmax microgel particles exhibited a stable spherical structure. ft-ir spectral analysis of the cmax microgel indicated that the microgel was crosslinked by bridging fe3+ and coo- with unidentate binding. the cmax microgel exhibited good ph sensitivity and high stability in acid condition. additionally, bsa was used as the embedding protein, and the controlled release effect of cmax microgel was explored in gastrointestinal tract simulation. the objective of these studies was to determine the effect corn fiber (cf), wheat bran (wb) and pectin (pec) on basal ileal endogenous amino acid (eaa) losses in broiler chickens (exp. 1) and cannulated pigs (exp. 2) using the regression method. semi-purified diets containing 100 g/kg of cf, wb, or pec (broiler chickens) and cf or pec (pigs) were fed to replicate cages consisting of eight birds per cage of 21-d-old broiler chickens and six replicates of pigs consisting of one pig per pen. ileal endogenous his, glu, and pro losses were higher (p < 0.05) for cf and wb compared with birds fed diets containing pec. contrasts between cf and wb showed higher (p < 0.05) ileal endogenous nitrogen, total amino acid, his, ile, met, glu, pro, and tyr losses in birds fed the cf diets (exp. 1). contrasts of eaa losses between birds fed the wb and pec diets showed higher (p < 0.05) losses for his, glu, and pro. in the cannulated pigs, cf resulted in higher (p < 0.05) ileal endogenous his, leu, and tyr losses. in summary, cf induced higher ileal eaa losses in broiler chickens and cannulated pigs. the strategy of improving the growth and health of piglets through maternal fiber diet intervention has attracted increasing attention. therefore, 15 sows were conducted to a wheat bran (wb) group, in which the sows' diets included 25% of wb in gestation and 14% in lactation, and a control (con) group, in which the sows' diets at all stages of reproduction did not contain wb. the results show that maternal high wb intervention seems not to have an impact on the growth of the offspring or the villus height of the duodenum, and the ratio of villi/crypts in the duodenum and jejunum were all higher in piglets born from wb sows, which may indicate that wb piglets had a larger absorption area and capacity for nutrients. the peroxisome proliferator-activated receptor gamma (pparγ) and interleukin 6 (il6) expression levels were notably upregulated in the ileal mucosa of wb piglets, while no immune-related genes in the colonic mucosa were affected by the maternal wb supplementation. in conclusion, adding a high proportion of wheat bran to the sow's gestation and lactation diet can affect the intestinal architecture and the expression of some inflammation genes, to some extent, in the ileal mucosa in the progeny. the aim of this study was to investigate the effects of different amounts of wheat bran (wb) inclusion and postbiotics form by saccharomyces cerevisiae and phytase co-fermented wheat bran (fwb) on the growth performance and health status of broilers. study randomly allocated a total of 300 male broilers to a control and 4 treatment groups (5% wb, 5% fwb, 10% wb, and 10% fwb inclusion, respectively) with each pen having 20 broilers and 3 pens per treatment. wheat bran does not contain enzymes, but there are 152.8, 549.2, 289.5, and 147.1 u/g dry matter (dm) xylanase, protease, cellulase and ß-glucanase in fwb, respectively. furthermore, fwb can decrease nitric oxide release of lipopolysaccharide (lps) stimulated chicken peripheral blood mononuclear cells by about two times. results show that 10% fwb inclusion had significantly the highest weight gain (wg) at 1-21d; 5% fwb had the lowest feed conversion rate at 22-35d; 10% wb and 10% fwb inclusion have the highest villus height and lactobacillus spp. number in caecum; and both 5% and 10% fwb can increase ash content in femurs. compared to control group, all treatments increase mucin 2 (muc2), and tight junction (tj), such as occludin, claudin-1, zonula occludens-1 (zo-1), and mrna expression in ileum by at least 5 folds. in chpbmcs, nadph oxidase 1 (nox-1) mrna expression decreases from 2 to 5 times, and glutamate-cysteine ligase catalytic subunit (gclc) mrna expression also increases in all treatment groups compared to control group. the mrna expression of pro-inflammatory cytokines, including interleukin-6 (il-6), nuclear factor-κb (nf-κb), and interleukin-1ß (il-1ß), decreases in 5% and 10% fwb groups compared to control group. to summarize, both wb and fwb inclusion in broilers diets increase tj mrna expression and anti-oxidation and anti-inflammation, but up to 10% fwb groups have better wg in different stages of broiler development. melasma is an acquired hyperpigmentation disorder, and reactive oxygen species play important role in regulating melanin synthesis. lycopene is one of the most effective oxygen neutralizers among tomato-derived carotenoids. also, hydroquinone is a compound that has been used for the treatment of hyperpigmentation by mechanism of tyrosinase inhibition and can be found in wheat. an appropriate cream formulation containing 0.05% tomato lycopene and 3.45% wheat bran extract was prepared, and physicochemical characterization was performed. the prepared formulations were applied twice a day for three months in combination with spf = 30 sunscreen by 22 patients diagnosed with melasma. masi score in two groups was evaluated at weeks 0, 3, 6, 9, and 12 and 1 month after the treatment. the prepared formulation shows smooth and homogeneous appearance with suitable spreadability and viscosity. the masi score of intervention group from the sixth week until the end of the treatment was significantly decreased compared to the onset of the treatment (p < .05), and the mean difference of the masi score and the rate of skin discoloration in intervention group (0.53 ± 0.47 and 3.73 ± 1.90, respectively) were significantly higher than in placebo group (0.14 ± 0.20 and 0.91 ± 0.07, respectively; p < .05). size of melasma during the study was decreased significantly from 6.59 ± 3.47 to 5.97 ± 3.83 (p < .05). the data of mean difference of the masi score indicated skin improvement in intervention group. meanwhile, no recurrence was observed one month after the end of the treatment. these data suggest that the prepared formulation containing lycopene and wheat bran extract is safe and could be promising as an efficacious cosmetic treatment. \n",
      "Euclidean Distance : 15.224271774291992\n",
      "\n",
      "\n",
      "Document: selenic acid etching assisted vacancy engineering for designing highly active electrocatalysts toward the oxygen evolution reaction. taming the oxidative power of seo(3) in 1,4-dioxane, isolation of two new isomers of mixed-valence selenium oxides, and two unprecedented cyclic esters of selenic acid. rapid fabrication of self-ordered porous alumina with 10-/sub-10-nm-scale nanostructures by selenic acid anodizing. oxygen evolution electrocatalysts are central to overall water splitting, and they should meet the requirements of low cost, high activity, high conductivity, and stable performance. herein, a general, selenic-acid-assisted etching strategy is designed from a metal-organic framework as a precursor to realize carbon-coated 3d metal selenides mm sen (co0.85 se1-x , nise2-x , fese2-x ) with rich se vacancies as high-performance precious metal-free oxygen evolution reaction (oer) electrocatalysts. specifically, the as-prepared co0.85 se1-x @c nanocages deliver an overpotential of only 231 mv at a current density of 10 ma cm-2 for the oer and the corresponding full water-splitting electrolyzer requires only a cell voltage of 1.49 v at 10 ma cm-2 in alkaline media. density functional theory calculation reveals the important role of abundant se vacancies for improving the catalytic activity through improving the conductivity and reducing reaction barriers for the formation of intermediates. although phase change after long-term operation is observed with the formation of metal hydroxides, catalytic activity is not obviously affected, which strengthens the important role of the carbon network in the operating stability. this study provides a new opportunity to realize high-performance oer electrocatalysts by a general strategy on selenic acid etching assisted vacancy engineering. the reaction of (seo3)4 with 1,4-dioxane (diox, dioxane) with or without diluting solvent led to the isolation of the unprecedented esters of selenic acid-1,2-ethyl selenate (ch2o)2seo2 and the glyoxal diselenate o2se[(ocho)2]seo2. it was possible to isolate an unknown dimeric form of se2o5 (se4o10·(diox)2) and a geometrical isomer of the mixed-valence oxide trans-se3o7, both stabilized by dioxane. the dioxane adduct of monomeric selenium trioxide seo3·diox was obtained from the reaction of (seo3)4 with dioxane in liquid so2. the reaction mechanism for the formation of these compounds was elucidated, and the molecular structure of the unstable form of the selenium trioxide was determined, consisting in a trimeric arrangement (seo3)3. anodic porous alumina has been widely investigated and used as a nanostructure template in various nanoapplications. the porous structure consists of numerous hexagonal cells perpendicular to the aluminum substrate and each cell has several tens or hundreds of nanoscale pores at its center. because the nanomorphology of anodic porous alumina is limited by the electrolyte during anodizing, the discovery of additional electrolytes would expand the applicability of porous alumina. in this study, we report a new self-ordered nanoporous alumina formed by selenic acid (h2seo4) anodizing. by optimizing the anodizing conditions, anodic alumina possessing 10-nm-scale pores was rapidly assembled (within 1 h) during selenic acid anodizing without any special electrochemical equipment. novel sub-10-nm-scale spacing can also be achieved by selenic acid anodizing and metal sputter deposition. our new nanoporous alumina can be used as a nanotemplate for various nanostructures in 10-/sub-10-nm-scale manufacturing. \n",
      "Euclidean Distance : 15.22529411315918\n",
      "\n",
      "\n",
      "Document: genetic analysis of wheat (triticum aestivum) adaptation to heat stress. proteomic analysis reveals how pairing of a mycorrhizal fungus with plant growth-promoting bacteria modulates growth and defense in wheat. impact of grape pomace powder on the phenolic bioaccessibility and on in vitro starch digestibility of wheat based bread. heteroalleles in common wheat: multiple differences between allelic variants of the  exploitation of tolerance of wheat kernel weight and shape-related traits from  genome-wide identification and characterization of wheat 14-3-3 genes unravels the role of tagrf6-a in salt stress tolerance by binding myb transcription factor. incentives to stimulate european wheat farmers to adapt their  co-cultivation of  reactivity and pozzolanic properties of biomass ashes generated by wheat and soybean straw combustion. induction of low temperature tolerance in wheat by pre-soaking and parental treatment with melatonin. wheat grain protein content under mediterranean conditions measured with chlorophyll meter. evolutionary divergence and biased expression of nac transcription factors in hexaploid bread wheat ( optimisation of the extrusion process through a response surface methodology for improvement of the physical properties and nutritional components of whole black-grained wheat flour. resistant starch type 2 from wheat reduces postprandial glycemic response with concurrent alterations in gut microbiota composition. coccidia vaccine challenge and exogenous enzyme supplementation in broiler chicken 1. effect on digesta viscosity, diet energy utilization, and apparent metabolizable energy value of wheat. glutacetine evaluation of physiological and morphological traits for improving spring wheat adaptation to terminal heat stress. comparative proteomic analysis of developmental changes in p-type cytoplasmic male sterile and maintainer anthers in wheat. transformation of selected trichothecenes during the wheat malting production. changing wheat bran structural properties by extrusion-cooking on a pilot and industrial scale: a comparative study. adaptation to abiotic stresses such as high-temperature conditions should be considered as its independent components of total performance and responsiveness. understanding and identifying improved adaptation to abiotic stresses such as heat stress has been the focus of a number of studies in recent decades. however, confusing and potentially misleading terminology has made progress difficult and hard to apply within breeding programs selecting for improved adaption to heat stress conditions. this study proposes that adaption to heat stress (and other abiotic stresses) be considered as the combination of total performance and responsiveness to heat stress. in this study, 1413 doubled haploid lines from seven populations were screened through a controlled environment assay, subjecting plants to three consecutive eight hour days of an air temperature of 36 °c and a wind speed of 40 km h-1, 10 days after the end of anthesis. qtl mapping identified a total of 96 qtl for grain yield determining traits and anthesis date with nine correlating to responsiveness, 72 for total performance and 15 for anthesis date. responsiveness qtl were found both collocated with other performance qtl as well as independently. a sound understanding of genomic regions associated with total performance and responsiveness will be important for breeders. genomic regions of total performance, those that show higher performance that is stable under both stressed and non-stressed conditions, potentially offer significant opportunities to breeders. we propose this as a definition and selection target that has not previously been defined for heat stress adaptation. plants rely on their microbiota for improving the nutritional status and environmental stress tolerance. previous studies mainly focused on bipartite interactions (a plant challenged by a single microbe), while plant responses to multiple microbes have received limited attention. here, we investigated local and systemic changes induced in wheat by two plant growth-promoting bacteria (pgpb), azospirillum brasilense and paraburkholderia graminis, either alone or together with an arbuscular mycorrhizal fungus (amf). we conducted phenotypic, proteomic, and biochemical analyses to investigate bipartite (wheat-pgpb) and tripartite (wheat-pgpb-amf) interactions, also upon a leaf pathogen infection. results revealed that only amf and a. brasilense promoted plant growth by activating photosynthesis and n assimilation which led to increased glucose and amino acid content. the bioprotective effect of the pgpb-amf interactions on infected wheat plants depended on the pgpb-amf combinations, which caused specific phenotypic and proteomic responses (elicitation of defense related proteins, immune response, and jasmonic acid biosynthesis). in the whole, wheat responses strongly depended on the inoculum composition (single vs. multiple microbes) and the investigated organs (roots vs. leaf). our findings showed that amf is the best-performing microbe, suggesting its presence as the crucial one for synthetic microbial community development. this article is protected by copyright. all rights reserved. breads were prepared by substituting common wheat flour with 0 (gp0), 5 (gp5) and 10 (gp10) g/100 g (w/w) of grape pomace powder (gpp) and were analyzed for the phenolic profile bioaccessibility as well as the in vitro starch digestion during simulated digestion. the free and bound phenolic composition of native gpp and resulting breads were profiled using ultra-high-performance chromatography-quadrupole-time-of-flight (uhplc-qtof). the raw gpp was characterized by 190 polyphenols with the anthocyanins representing the most abundant class, accounting for 11.60 mg/g of cyanidin equivalents. regarding the fortified bread, the greatest (p < 0.05) content in phenolic compounds was recorded for the gp10 sample (considering both bound and free fractions) being 127.76 mg/100 g dry matter (dm), followed by the gp5 (106.96 mg/100 g dm), and gp0 (63.76 mg/100 g dm). the use of gpp determined an increase of anthocyanins (considered the markers of the gpp inclusion), recording 20.98 mg/100 g dm in gp5 and 35.82 mg/100 g dm in gp10. the bioaccessibility of anthocyanins increased in both gp5 and gp10 breads when moving from the gastric to the small intestine in vitro digestion phase with an average value of 24%. both the starch hydrolysis and the predicted glycemic index decreased with the progressive inclusion of gpp in bread. present findings showed that gpp in bread could promote an antioxidant environment in the digestive tract and influence the in vitro starch digestion. the gli-b1-encoded γ-gliadins and non-coding γ-gliadin dna sequences for 15 different alleles of common wheat have been compared using seven tests: electrophoretic mobility (em) and molecular weight (mw) of the encoded major γ-gliadin, restriction fragment length polymorphism patterns (rflps) (three different markers), gli-b1-γ-gliadin-pseudogene known snp markers (single nucleotide polymorphisms) and sequencing the pseudogene gag56b. it was discovered that encoded γ-gliadins, with contrasting em, had similar mws. however, seven allelic variants (designated from i to vii) differed among them in the other six tests: i (alleles gli-b1i, k, m, o), ii (gli-b1n, q, s), iii (gli-b1b), iv (gli-b1e, f, g), v (gli-b1h), vi (gli-b1d) and vii (gli-b1a). allele gli-b1c (variant viii) was identical to the alleles from group iv in four of the tests. some tests might show a fine difference between alleles belonging to the same variant. our results attest in favor of the independent origin of at least seven variants at the gli-b1 locus that might originate from deeply diverged genotypes of the donor(s) of the b genome in hexaploid wheat and therefore might be called \"heteroallelic\". the donor's particularities at the gli-b1 locus might be conserved since that time and decisively contribute to the current high genetic diversity of common wheat. kernel weight and shape-related traits are inherited stably and increase wheat yield. narrow genetic diversity limits the progress of wheat breeding. here, we evaluated kernel weight and shape-related traits and applied genome-wide association analysis to a panel of wheat multiple synthetic derivative (msd) lines. the msd lines harbored genomic fragments from aegilops tauschii. these materials were grown under optimum conditions in japan, as well as under heat and combined heat-drought conditions in sudan. we aimed to explore useful qtls for kernel weight and shape-related traits under stress conditions. these can be useful for enhancing yield under stress conditions. msd lines possessed remarkable genetic variation for all traits under all conditions, and some lines showed better performance than the background parent norin 61. we identified 82 marker trait associations (mtas) under the three conditions; most of them originated from the d genome. all of the favorable alleles originated from ae. tauschii. for the first time, we identified markers on chromosome 5d associated with a candidate gene encoding a ring-type e3 ubiquitin-protein ligase and expected to have a role in regulating wheat seed size. our study provides important knowledge for the improvement of wheat yield under optimum and stress conditions. the results emphasize the importance of ae. tauschii as a gene reservoir for wheat breeding. 14-3-3 proteins are a large multigenic family of general regulatory factors (grf) ubiquitously found in eukaryotes and play vital roles in the regulation of plant growth, development, and response to stress stimuli. however, so far, no comprehensive investigation has been performed in the hexaploid wheat. in the present study, a total of 17 potential 14-3-3 gene family members were identified from the chinese spring whole-genome sequencing database. the phylogenetic comparison with six 14-3-3 families revealed that the majority of wheat 14-3-3 genes might have evolved as an independent branch and grouped into ε and non-ε group using the phylogenetic comparison. analysis of gene structure and motif indicated that 14-3-3 protein family members have relatively conserved exon/intron arrangement and motif composition. physical mapping showed that wheat 14-3-3 genes are mainly distributed on chromosomes 2, 3, 4, and 7. moreover, most 14-3-3 members in wheat exhibited significantly down-regulated expression in response to alkaline stress. vigs assay and protein-protein interaction analysis further confirmed that tagrf6-a positively regulated slat stress tolerance by interacting with a myb transcription factor, tamyb64. taken together, our findings provide fundamental information on the involvement of the wheat 14-3-3 family in salt stress and further investigating their molecular mechanism. fusarium species infection in wheat can lead to fusarium head blight (fhb) and contamination with mycotoxins. to fully exploit more recent insights into fhb and mycotoxin management, farmers might need to adapt their agronomic management, which can be stimulated through incentives. this study aimed to identify incentives to stimulate european farmers to adapt their agronomic management to reduce fhb and related mycotoxins in wheat. a questionnaire was distributed among 224 wheat farmers from italy, the netherlands, serbia, and the united kingdom. using the respondents' data, bayesian network modelling was applied to estimate the probability that farmers would adapt their current agronomic management under eight different incentives given the conditions set by their farm and farmer characteristics. results show that most farmers would adapt their current agronomic management under the incentives \"paid extra when wheat contains low levels of mycotoxins\" and \"wheat is tested for the presence of mycotoxins for free\". the most effective incentive depended on farm and farmer characteristics, such as country, crop type, size of arable land, soil type, education, and mycotoxin knowledge. insights into the farmer characteristics related to incentives can help stakeholders in the wheat supply chain, such as farmer cooperatives and the government, to design tailor-made incentive plans. mycotoxigenic fungal pathogens fusarium and alternaria are a leading cause of loss in cereal production. on wheat-ears, they are confronted by bacterial antagonists such as pseudomonads. studies on these groups' interactions often neglect the infection process's temporal aspects and the associated priority effects. in the present study, the focus was on how the first colonizer affects the subsequent ones. in a climate chamber experiment, wheat-ears were successively inoculated with two different strains (alternaria tenuissima at625, fusarium graminearum fg23, or pseudomonas simiae ps9). over three weeks, microbial abundances and mycotoxin concentrations were analyzed and visualized via self organizing maps with sammon mapping (som-sm). all three strains revealed different characteristics and strategies to deal with co-inoculation: fg23, as the first colonizer, suppressed the establishment of at625 and ps9. nevertheless, primary inoculation of at625 reduced all of the fusarium toxins and stopped ps9 from establishing. ps9 showed priority effects in delaying and blocking the production of the fungal mycotoxins. the som-sm analysis visualized the competitive strengths: fg23 ranked first, at625 second, ps9 third. our findings of species-specific priority effects in a natural environment and the role of the mycotoxins involved are relevant for developing biocontrol strategies. a sustainable use of locally available wastes from agriculture as supplementary cementitious materials (scms) is an alternative solution for the prevention of excessive raw material usage, reduction of co2 emission and cost-effective concrete production. this paper studies the reactivity of non-traditional waste scms: wheat straw ash (wsa), mixture of wheat and soybean straw ash (wssa) and soybean straw ash (ssa), which are abundant as agricultural by-products in serbia. the chemical evaluation using xrf technique, thermal analysis (tga/dsc), xrd and ftir methods were performed along with physical properties tests to investigate the feasibility of utilizing biomass ashes as cement substitutes. the obtained results demonstrate a high pozzolanic activity of wsa, which is attributed to a high reactive silica content of the ash and its satisfactory level of fineness. a wider hump in xrd pattern of wsa compared to wssa and ssa confirmed that it abounds in amorphous (reactive) phase. the insufficient activity index of soybean-based biomass ashes, characterized with a low silica content, was improved by additional grinding and/or blending with amorphous silica-rich material. this points out the mechanical activation, i.e., grinding procedure, and chemical activation, i.e., modification of the chemical composition, as techniques efficient at producing pozzolanic materials from biomass wastes. tested biomass ashes are characterized with negligible leaching values of heavy metals, thereby satisfying eco-friendly principles of scm utilization. the application of biomass ashes as scms leads to substantial cost savings, as well as benefits to the environment, such as lower consumption of cement, reduction of co2 emissions during the production of cement and sustainable waste management. low temperatures seriously depress germination and seedling establishment in wheat and it is of great significance to explore approaches to improve wheat tolerance to low temperatures. in this study, the effects of seed pre-soaking and parental treatment with melatonin on seed germination and low temperature tolerance during the early growing stage in wheat were studied. the results showed that pre-soaking with melatonin increased the germination rate, improved antioxidant capacity and accelerated starch degradation under low temperature, which alleviated low temperature-induced damage to the chloroplasts in coleoptiles of wheat seedlings. parental melatonin treatment during grain filling stage significantly decreased the grain weight. seeds from parental melatonin-treated plants showed higher germination rates and higher antioxidant enzyme activity than the control seeds under low temperature. in addition, parental treatment with melatonin modulated the activities of carbohydrate metabolism enzymes, which contributes to enhanced low temperature tolerance in wheat offspring. it was suggested that both seed pre-soaking and parental treatment with melatonin could be the effective approaches for low temperature tolerance induction in wheat. adequate n fertilisation is crucial to increase the grain protein content (gpc) values in wheat. the recommended level of gpc needed to achieve high-quality bread-making flour should be higher than 12.5%. however, it is difficult to ensure the gpc values that the crop will achieve because n in grain is derived from two different sources: n remobilized into the grain from n accumulated in the pre-anthesis period, and n absorbed from the soil in the post-anthesis period. this study aimed to (i) evaluate the effect of the application of n on the rate of stem elongation (gs30) when farmyard manures are applied as initial fertilisers on gpc and on the chlorophyll meter (cm) values at mid-anthesis (gs65), (ii) establish a relationship between the cm values at gs65 and gpc, and (iii) determine a minimum cm value at gs65 to obtain gpc values above 12.5%. three field trials were performed in three consecutive growing seasons, and different n fertilisation doses were applied. readings using the cm yara n-testertm were taken at gs65. the type of initial fertiliser did not affect the gpc and cm values. generally, the greater the n application at gs30 is, the higher the gpc and cm values are. cm values can help to estimate gpc values only when yields are below 8000 kg ha-1. additionally, cm values at gs65 should be higher than 700 to achieve high-quality bread-making flour (12.5%) at such yield levels. these results will allow farmers and cooperatives to make better decisions regarding late-nitrogen fertilisation and wheat sales. the nac genes, a large plant-specific family of transcription factors, regulate a wide range of pathways involved in development and response to biotic and abiotic stress. in this study, the nac transcription factors were identified in 27 green plants, and the results showed that nac transcription factors in plants undergo an appearance stage from water to land and a number expansion stage from gymnosperm to angiosperm. investigating the evolutionary process of the nac transcription factors from diploid species to hexaploid wheat revealed that tandem replications during the polyploidization process is an important event for increasing the number of nac transcription factors in wheat. then, the molecular characteristics, phylogenetic relationships, and expression patterns of 462 nac transcription factors of hexaploid wheat (tanacs) were analyzed. the protein structure results showed that tanac was relatively conservative at the n-terminal that contains five subdomains. all these tanacs were divided into group i and group ii by phylogenetic analysis, and the tanacs in group i should undergo strong artificial selection based on single nucleotide polymorphism (snp) analysis. through genome synteny and phylogenetic analysis, these tanacs were classified into 88 groups and 9 clusters. the biased expression results of these tanacs showed that there are 24 groups and 67 groups of neofunctionalization genes under biotic and abiotic stress, respectively, and 16 groups and 59 groups of subfunctionalization genes. this shows that neofunctionalization plays an important role in coping with different stresses. our study provides new insights into the evolution of nac transcription factors in hexaploid wheat. chronic undernourishment affects billions of people. the development of whole-grain food with high nutritional quality may provide a valuable solution to nutritional security. black-grained wheat (bgw), as a rich source of protein and micronutrients, is a good raw material for value-added products. the objectives of this study were to investigate the effects of barrel temperature, feed moisture content, and feed rate on the physical properties and nutritional components of whole bgw flour extrudates and to optimise their processing conditions by using the response surface methodology. the increasing barrel temperature, feed moisture content, and feed rate affected the specific volume, expansion ratio, hardness, fracturability, water absorption index (wai), water solubility index (wsi), and total starch content of the extrudates, but did not significantly affect the content of protein, ash, iron (fe), zinc (zn), copper (cu), and manganese (mn). the extruded wheat flour had a significantly higher content of fe and cu, and a lower total starch content than the unextruded flour under extrusion conditions. a significantly higher content of protein, ash, zn, cu, and mn, and a significantly lower total starch content were found in the extruded and unextruded flours made of whole bgw than in those made of whole white-grained wheat. according to the significance of the regression coefficients of the quadratic polynomial model, the optimum extrusion parameters were as follows: a barrel temperature of 145.63 °c, feed moisture content of 19.56%, and feed rate of 40.64 g·min-1 in terms of the maximum specific volume, expansion ratio, fracturability, wai and wsi, and the minimum hardness. these results may be used by food manufacturers to successfully develop extruded products from whole bgw flour, meeting consumer demands and needs. the majority of research on the physiological effects of dietary resistant starch type 2 (rs2) has focused on sources derived from high-amylose maize. in this study, we conduct a double-blind, randomized, placebo-controlled, crossover trial investigating the effects of rs2 from wheat on glycemic response, an important indicator of metabolic health, and the gut microbiota. overall, consumption of rs2-enriched wheat rolls for one week resulted in reduced postprandial glucose and insulin responses relative to conventional wheat when participants were provided with a standard breakfast meal containing the respective treatment rolls (rs2-enriched or conventional wheat). this was accompanied by an increase in the proportions of bacterial taxa ruminococcus and gemmiger in the fecal contents, reflecting the composition in the distal intestine. additionally, fasting breath hydrogen and methane were increased during rs2-enriched wheat consumption. however, although changes in fecal short-chain fatty acid (scfa) concentrations were not significant between control and rs-enriched wheat roll consumption, butyrate and total scfas were positively correlated with relative abundance of faecalibacterium, ruminoccocus, roseburia, and barnesiellaceae. these effects show that rs2-enriched wheat consumption results in a reduction in postprandial glycemia, altered gut microbial composition, and increased fermentation activity relative to wild-type wheat. this study examined the effect of exogenous mixed-enzyme supplementation (xylanase, β-glucanase, and pectinase) to a corn-sbm (cs) and a wheat-cs-based (wcs) diet in birds challenged with coccidia vaccine (coccivac b-52™). the wcs-based diet was produced by replacing 30% of the energy-yielding portions of the cs-based diet with wheat. on day 14, 448 (n = 7) cobb by-product breeder male broilers were assigned to a 2 (diet types) × 2 (with or without enzyme supplementation) × 2 (0 or 20 × coccidia vaccine challenge; cvc) factorial arrangement of treatments in a completely randomized design for the determination of the apparent metabolizable energy (ame) value of wheat. treatment effects on jejunum digesta viscosity and ame corrected for nitrogen (amen) of the diets were evaluated within each diet type as a 2 × 2 factorial arrangement of treatments, 7- and 14-day post-challenge. seven-day post-challenge (day 21), dry matter (dm) and energy utilization, ame, and amen of the cs- and wcs-based diets decreased (p < 0.05) with cvc. both ame and amen of wheat decreased (p < 0.05) by about a 20% in cvc-birds 7-day post-challenge. enzyme and cvc resulted in a decrease (p < 0.05) in jejunal digesta viscosity in birds fed the cs-based diets, while there was an interaction (p < 0.05) between cvc and enzyme, with enzyme lowering (p < 0.05) the viscosity of digesta 7-day post-challenge. results from this study showed that cvc resulted in a 20% decrease in amen 7-day post-challenge, while the interaction between exogenous enzyme supplementation and cvc resulted in an improvement in nitrogen utilization (~6%) in cvc birds fed the corn-sbm-based diet 7-days post challenge. wheat is one of the most important cereals for human nutrition, but nitrogen (n) losses during its cultivation cause economic problems and environmental risks. in order to improve n use efficiency (nue), biostimulants are increasingly used. the present study aimed to evaluate the effects of glutacetine®, a biostimulant sprayed at 5 l ha-1 in combination with fertilizers (urea or urea ammonium nitrate (uan)), on n-related traits, grain yield components, and the grain quality of winter bread wheat grown at three field sites in normandy (france). glutacetine® improved grain yield via a significant increase in the grain number per spike and per m2, which also enhanced the thousand grain weight, especially with urea. the total n in grains and the nue tended to increase in response to glutacetine®, irrespective of the site or the form of n fertilizer. depending on the site, spraying glutacetine® can also induce changes in the grain ionome (analyzed by x-ray fluorescence), with a reduction in p content observed (site 2 under urea nutrition) or an increase in mn content (site 3 under uan nutrition). these results provide a roadmap for utilizing glutacetine® biostimulant to enhance wheat production and flour quality in a temperate climate. wheat crop experiences high temperature stress during flowering and grain-filling stages, which is termed as \"terminal heat stress\". characterizing genotypes for adaptive traits could increase their selection for better performance under terminal heat stress. the present study evaluated the morpho-physiological traits of two spring wheat cultivars (millet-11, punjab-11) and two advanced lines (v-07096, v-10110) exposed to terminal heat stress under late sowing. early maturing millet-11 was used as heat-tolerant control. late sowing reduced spike length (13%), number of grains per spike (10%), 1000-grain weight (13%) and biological yield (15-20%) compared to timely sowing. nonetheless, higher number of productive tillers per plant (19-20%) and grain yield (9%) were recorded under late sowing. advanced lines and genotype punjab-11 had delayed maturity and better agronomic performance than early maturing heat-tolerant millet-11. advanced lines expressed reduced canopy temperature during grain filling and high leaf chlorophyll a (20%) and b (71-125%) contents during anthesis under late sowing. all wheat genotypes expressed improved stem water-soluble carbohydrates under terminal heat stress that were highest for heat-tolerant millet-11 genotype during anthesis. improved grain yield was associated with the highest chlorophyll contents showing stay green characteristics with maintenance of high photosynthetic rates and cooler canopies under late sowing. the results revealed that advanced lines and punjab-11 with heat adaptive traits could be promising source for further use in the selection of heat-tolerant wheat genotypes. cytoplasmic male sterility (cms) plays an important role in the application of heterosis in wheat (triticum aestivum l.). however, the molecular mechanism underlying cms remains unknown. this study provides a comprehensive morphological and proteomic analysis of the anthers of a p-type cms wheat line (p) and its maintainer line, yanshi 9 hao (y). cytological observations indicated that the p-type cms line shows binucleate microspore abortion. in this line, the tapetum degraded early, leading to anther cuticle defects, which could not provide the nutrition needed for microspore development in a timely manner, thus preventing the development of the microspore to the normal binucleate stage. proteomic analysis revealed novel proteins involved in p-type cms. up to 2576 differentially expressed proteins (deps) were quantified in all anthers, and these proteins were significantly enriched in oxidative phosphorylation, glycolysis/gluconeogenesis, citrate cycle (tca cycle), starch and sucrose metabolism, phenylpropanoid biosynthesis, and pyruvate metabolism pathways. these proteins may comprise a network that regulates male sterility in wheat. based on the function analysis of deps involved in the complex network, we concluded that the p-type cms line may be due to cellular dysfunction caused by disturbed carbohydrate metabolism, inadequate energy supply, and disturbed protein synthesis. these results provide insights into the molecular mechanism underlying male sterility and serve as a valuable resource for researchers in plant biology, in general, and plant sexual reproduction, in particular. the transformation of deoxynivalenol (don), nivalenol (niv), and their glucosides (don3g and niv3g) during the malting of grains of two wheat varieties was studied. the concentration of don3g and niv3g started to increase significantly before the concentration of don and niv increased. this may reflect the transformation of the parent mycotoxin forms into their glucosides due to xenobiotic detoxification reactions. after a sharp rise during the last 2 days of the process (day 6 and 7), the don concentration reached 3010 ± 338 µg/kg in the legenda wheat-based malt and 4678 ± 963 µg/kg in the pokusa wheat-based malt. the niv concentration, at 691 ± 65 µg/kg, remained the same as that in the dry grain. the concentration of don3g in the legenda and pokusa wheat-based malt was five and three times higher, respectively, than that in the steeped grain. the concentration of niv3g in the legenda wheat-based malt was more than twice as high as that in the steeped grain. the sharp increase in the concentration of don at the end of the malting process reflected the high pathogen activity. we set aside some samples to study a batch that was left undisturbed without turning and aeration, for the entire period of malting. the concentration of don in the malt produced from the latter batch was 135% and 337% higher, for legenda and pokusa, respectively, than that in the malt produced from the batch that was turned and aerated. the niv concentration was 22% higher in the latter batch. extrusion-cooking can be used to change the techno-functional and nutrition-related properties of wheat bran. in this study, pilot-scale (bc21) and industrial-scale (bc45) twin-screw extrusion-cooking using different types of extrusion (single-pass, double-pass and acid extrusion-cooking) and process parameters (temperature, moisture) were compared for their impact on wheat bran. when applying the same process settings, the higher strong water-binding capacity, extract viscosity and extractability displayed by bran extruded using the industrial set-up reflected a more considerable wheat bran structure degradation compared to pilot-scale extrusion-cooking. this was attributed to the overall higher specific mechanical energy (sme), pressure and product temperature that were reached inside the industrial extruder. when changing the type of extrusion-cooking from single-pass to double-pass and acid extrusion-cooking, wheat bran physicochemical characteristics evolved in the same direction, irrespective of extruder scale. the differences in bran characteristics were, however, smaller on industrial-scale. results show that the differentiating power of the latter can be increased by decreasing the moisture content and increasing product temperature, beyond what is possible in the pilot-scale extruder. this was confirmed by using a bc72 industrial-scale extruder at low moisture content. in conclusion, the extruder scale mainly determines the sme that can be reached and determines the potential to modify wheat bran. \n",
      "Euclidean Distance : 15.234861373901367\n",
      "\n",
      "\n",
      "Document: nutrient digestibility of a vegetarian diet with or without the supplementation of feather meal and either corn meal, fermented rye or rye and its effect on fecal quality in dogs. assessment of corn starch as substitute for agarose in dna gel electrophoresis. high-amylose corn starch/konjac glucomannan composite film: reinforced by incorporating β-cyclodextrin. beet pulp substituted for corn silage and barley grain in diets fed to dairy cows in the summer months: feed intake, total-tract digestibility, and milk production. corn starch/chitosan nanoparticles/thymol bio-nanocomposite films for potential food packaging applications. abomasal infusion of ground corn and ammonium chloride in early-lactating holstein-friesian dairy cows to induce hindgut and metabolic acidosis. combined effects of hydroxypropylation and alcoholic alkaline treatment on structural, functional and rheological characteristics of sorghum and corn starches. synthetic mechanism of octenyl succinic anhydride modified corn starch based on shells separation pretreatment. effect of ionic liquid 1-buyl-3-methylimidazolium halide on the structure and tensile property of pbs/corn starch blends. corn starch ( modification of corn starch via innovative contactless thermal effect from induced electric field. effects on the structure and properties of native corn starch modified by enzymatic debranching (ed), microwave assisted esterification with citric acid (mcae) and by the dual ed/mcae treatment. an overview on plasticized biodegradable corn starch-based films: the physicochemical properties and gelatinization process. expression, purification and characterization of car/ncoa-1 tethered protein in e. coli using maltose-binding protein fusion tag and gelatinized corn starch. low starch/corn silk/zno as environmentally friendly nanocomposites assembling on pet fabrics. physicochemical properties and antibacterial activity of corn starch-based films incorporated with zanthoxylum bungeanum essential oil. aerogels based on corn starch as carriers for pinhão coat extract (araucaria angustifolia) rich in phenolic compounds for active packaging. [effects of different corn varieties on development and reproduction of  effect of kernel processing and particle size of whole-plant corn silage with vitreous endosperm on dairy cow performance. structure and in vitro digestibility on complex of corn starch with soy isoflavone. cereals with low environmental input like rye and animal by-products which cannot be used for human food like feather meal are receiving growing interest as sustainable feed sources. thus, eight beagle dogs were included in a 4 × 4 latin square design and received a vegetarian basic diet or the same diets supplemented with hydrolyzed feather meal (2.7%) and either 20.1% of corn meal, 60.4% of fermented rye or 20.1% of rye as is basis (moisture content of the diets about 42%). compared to other groups the dry matter (dm) content of feces from dogs fed the basic diet was higher (30.0%, p < 0.05), while dogs fed the basic diet + rye had the lowest dm-content (26.5%, p < 0.05). however, the fecal scores were considered to be within an acceptable range (well-formed and firm). starch digestibility was lower (p < 0.05) for dogs fed the basic diet + corn meal. the dogs showed a high and identical acceptance (scoring of food intake) of the experimental diets. as a comparable quality of feces and a high nutrient digestibility were observed when rye was used in the experimental diets-it can be considered an alternative carbohydrate source in dog foods. the use of agarose in nucleic acid electrophoresis is the gold standard. however, agarose is very expensive and not readily available in resource limited developing countries like ghana. hence, finding a more affordable and readily available alternative to agarose will be a major boost to molecular research in developing countries. this study was aimed at investigating the use of corn starch as a potential substitute for agarose in dna gel electrophoresis. genomic deoxyribonucleic acid (dna) extracted from plasmodium falciparum and primers were obtained from the west african centre for cell biology of infectious pathogens and amplified using polymerase chain reaction. the amplicon was run on agarose gel to ascertain the molecular weight (as a positive control). when visualized under both blue light and ultraviolet light, the dna and ladder showed clear and clean bands with the expected molecular weight. corn starch was then modified with sodium borate buffer, casted into a gel and used to run the same dna sample. our findings indicated that similar to agarose, the dna sample and ladder migrated successfully through the modified starch gel but no bands were visible when visualized under blue and ultra-violet light. glycerol-plasticized high-amylose corn starch/konjac glucomannan (hcs/kgm) composite films incorporated with various concentrations of β-cyclodextrin (β-cd) were prepared and investigated for structural, mechanical, and physical properties. the results of x-ray diffraction, attenuated total reflectance fourier transform infrared spectroscopy, thermogravimetric analyses, and scanning electron microscopy indicated that β-cd excluded from the polymer chains and aggregated to form crystals during film formation, which drove hcs to interact with kgm more compactly. the thickness and transparency of the films increased after β-cd was incorporated. more associations of hcs/kgm enhanced the mechanical properties and reduced the moisture content of the films. the water vapor permeability of the hcs/kgm composite film was also improved significantly with the incorporation of β-cd. the enhanced association between biopolymers in the presence of β-cd will advance the development of a degradable active composite packaging film. the responses of dairy cows to the substitution of beet pulp (bp) for grain or forage are not consistent, and heat stress may affect the response of dairy cows to this substitution. the effects of substituted bp for corn silage and barley grain on feed intake, performance, and ruminal parameters were evaluated using eight multiparous holstein cows in a duplicated 4 × 4 latin square design with 21-day periods. cows were in mid-lactation (45.4 ± 3.6 kg/day milk production and 116 ± 10 days in milk) with an average bw of 664 ± 41.2 kg. dietary treatments were as follows: 1) 0% bp (0bp, control, 38.5% barley grain, and 20.3% corn silage); 2) 12% bp (12bp, 32.5% barley grain, and 14.3% corn silage); 3) 18% bp (18bp, 29.5% barley grain, and 11.3% corn silage); and 4) 24% bp (24bp, 26.5% barley grain, and 8.3% corn silage). cows were under mild heat stress and the average temperature-humidity index was 70.5; increasing bp caused a linear decrease in respiration rate (p < 0.01). higher bp in the diet caused a linear increase in dm intake (p = 0.01) and ndf digestibility (p = 0.03). dry and organic matter (om) digestibilities tended to increase linearly with higher bp (p < 0.10). milk yield, energy-corrected milk, protein, lactose, and fat production and content were not affected by the treatments. increasing bp in the diet caused a linear decrease in feed efficiency and rumen ammonia (p < 0.05) and a tendency to a linear decrease in milk urea nitrogen (p < 0.10). rumen ph and acetate to propionate ratio were not affected by the replacement. total volatile fatty acid concentration in the rumen increased linearly with increasing the bp inclusion (p = 0.04). acetate and butyrate (p = 0.07) proportion tended to increase, whereas propionate (p = 0.06) and isovalerate (p = 0.08) proportion tended to decrease linearly as bp was substituted for corn silage and barley grain. the results indicated that under mild heat stress condition, bp can be successfully substituted for barley grain and corn silage up to 24% of the diet without any negative effect on production and ruminal ph. this work aims to develop corn starch/chitosan nanoparticles/thymol (cs/cnp/thy) bio-nanocomposite films as potential food packaging materials that can enhance the shelf life of food. cs/cnp/thy bio-nanocomposite films were prepared by the addition of different concentrations of thymol (0, 1.5, 3.0, 4.5 w/w%) using a solvent casting method. the resulting films were characterized in terms of optical, mechanical, and water vapor permeability (wvp) properties. the addition of thymol was found to reduce the tensile strength (ts), elongation at break (eab), and young's modulus (ym) of the films. generally, the increment in the concentration of thymol did not significantly affect the ts, eab, and ym values. the addition of 1.5 w/w% thymol increased the wvp of the films but the wvp reduced with the increase in thymol concentrations. cs/cnp/thy-3% bio-nanocomposite films demonstrated the potential to lengthen the shelf life of cherry tomatoes packed with the films, whereby the cherry tomatoes exhibited no significant changes in firmness and the lowest weight loss. in addition, no mold growth was observed on the sliced cherry tomatoes that were in direct contact with the films during 7 days of storage, proving the promising application of the films as active food packaging materials. next to rumen acidosis, other forms of acidosis may also affect lactational performance of cows. therefore, the effects of hindgut acidosis, induced via abomasal infusion of ground corn, and metabolic acidosis, induced via abomasal infusion of nh4cl, were studied in cows in early lactation. observations were made on intake and digestibility of nutrients, lactation performance, energy and n partitioning, blood acid-base status, and rumen and hindgut fermentation characteristics. in a 6 × 6 latin square design, 6 rumen-fistulated, second-lactation holstein-friesian dairy cows (48 ± 17 d in milk) were subjected to 5 d of continuous abomasal infusions of water as control, or solutions of 2.5 mol of nh4cl/d, 5.0 mol of nh4cl/d, 3.0 kg of ground corn/d, or the combination of ground corn with either of the 2 nh4cl levels, followed by 2 d of rest. treatment solutions were administered via peristaltic pumps through infusion lines attached to the rumen cannula plug and an abomasal infusion line with a flexible disk (equipped with holes to allow digesta passage) to secure its placement through the sulcus omasi. a total mixed ration consisting of 70% grass silage and 30% concentrate (on dry matter basis) was fed at 95% of ad libitum intake of individual cows. the experiment was conducted in climate respiration chambers to determine feed intake, lactation performance, and energy and n balance. abomasal infusion of nh4cl affected the acid-base status of the cows, but more strongly when in combination with abomasal infusion of ground corn. metabolic acidosis (defined as a blood ph < 7.40, blood hco3 concentration < 25.0 mmol/l, and a negative base excess) was observed with 5.0 mol of nh4cl/d, 3.0 kg of ground corn/d + 2.5 mol of nh4cl/d, and 3.0 kg of ground corn/d + 5.0 mol of nh4cl/d. metabolic acidosis was associated with decreased milk lactose content, metabolic body weight, energy retained as protein, and fecal n excretion, and increased urine n excretion, and tended to decrease intake of nutrients. digestibility of several nutrients increased with 5.0 mol of nh4cl/d, likely as a result of decreased intake. abomasal ground corn infusion resulted in hindgut acidosis, where fecal ph decreased from 6.86 without ground corn to 6.00 with ground corn, regardless of nh4cl level. the decrease in fecal ph was likely the result of increased hindgut fermentation, evidenced by increased fecal volatile fatty acid concentrations. hindgut acidosis was associated with decreased digestibility of nutrients, except for starch, which increased, and crude fat, which was not affected. no systemic inflammatory response was observed, suggesting that the hindgut epithelium was not severely affected by the more acidic conditions or barrier damage. abomasal infusion of ground corn increased milk yield, milk protein and lactose yield, fecal n excretion, n use efficiency, and total energy retained as well as energy retained in fat, and reduced milk fat content and urine n excretion. this study describes the effects of hydroxypropylation (hp) on sorghum and corn cold water soluble (cws) starches prepared via alcoholic alkaline treatment (aat). propylene oxide (5% and 12% on starch weight basis) was used to modify both sorghum and corn starches. sem analysis revealed that hp modification prior to aat altered the granular morphology of native cws starches. the characteristic peaks at 2980.28 cm-1 and 2979.87 cm-1 indicated the presence of hydroxypropyl groups and the complete loss of the granular order for hs12-cws (hydroxypropylated sorghum cws starch treated with 12% propylene oxide based on dry starch weight) and hc12-cws (hydroxypropylated corn cws starch treated with 12% propylene oxide based on dry starch weight) starches. increase in swelling power and water binding capacity of hp modified cws starches was observed. however, percent transmittance was significantly reduced due to fragmented water-soluble granules. hp-modified cws starch gels exhibited a more rigid gel network. broader linear viscoelastic range suggesting greater stability and well dispersed behavior of hp-cws starches. high g' values of hp-cws starches were due to the ordered and elastic gel network that resisted deformation. furthermore, all hp-cws starches exhibited higher shear and thermal resistance compared to unmodified cws starches. herein, the synthetic mechanism of octenyl succinic anhydride (osa) modified corn starch (oscs) and granule shells (oscs) based on shells separation pretreatment (ssp) was investigated. high intensity peaks around 1720 and 1570 cm-1 were observed for oscs in fourier transform infrared (ftir) spectra after ssp. oscs showed higher degree of substitution (ds) values (ranging from 0.128 to 0.170) than oscs (0.121) determined by 1h nmr. the average molecular weight (mw) of osa modified cs decreased, due to the introduction of os groups. x-ray diffraction (xrd) indicated that esterification mainly took place in the amorphous regions of starch granules. x-ray photoelectron spectroscopy (xps) revealed that a new peak corresponding to 1s orbital electrons of na was obtained due to the introduction of osa molecules. meanwhile, lower surface ds and higher fluorescence intensity were noticed for oscs. conclusively, ssp would significantly increase the reaction efficiency of osa modification process of cs. as a promising biodegradable resin, poly (butylene succinate) (pbs) is often blended with starch to reduce the cost. in this paper, 1-buyl-3-methylimidazolium halide pre-plasticized corn starch (cs) was blended with pbs to prepare pbs/corn starch blend material modified by ionic liquid (pbs/cs-il). ionic liquid (il) acted as plasticizer and compatibilizer, and the effects of 1-butyl-3-methylimidazolium halide with different halogen anion on pbs/starch blends were explored. the effects of il on the structure and tensile property of pbs/starch blends were evaluated by ftir, sem, dsc, tga and xrd, respectively. test results showed that the addition of il significantly reduced the crystallinity of pbs/starch blends, and the size of starch particles in the pbs matrix was also effectively reduced. il also acted as a compatibilizer of starch and pbs, and induced the morphology of the blends to change from \"sea-island\" structure to homogeneous phase. the results of the tensile test showed that compared with the pbs/starch blend without il, the elongation at break of pbs/cs-il increased from 22% to 93%. this study provided a simple and feasible method for the preparation of low-cost pbs bio-composite materials, and provided theoretical support for future industrial production. the research included corn starch (cs) films using sorbitol (s), glycerol (g), and their combination (sg) as plasticizers at 30, 45, and 60 wt %, with a traditional solution casting technique. the introduction of plasticizer to cs film-forming solutions led to solving the fragility and brittleness of cs films. the increased concentration of plasticizers contributed to an improvement in film thickness, weight, and humidity. conversely, plasticized films reduced their density and water absorption, with increasing plasticizer concentrations. the increase in the amount of the plasticizer from 30 to 60% showed a lower impact on the moisture content and water absorption of s-plasticized films. the s30-plasticized films also showed outstanding mechanical properties with 13.62 mpa and 495.97 mpa, for tensile stress and tensile modulus, respectively. glycerol and-sorbitol/glycerol plasticizer (g and sg) films showed higher moisture content and water absorption relative to s-plasticized films. this study has shown that the amount and type of plasticizers significantly affect the appearances, physical, morphological, and mechanical properties of the corn starch biopolymer plastic. growing attention has been focused on modifications of starch using electric field, but electrode corrosion and metal contamination remain unavoidable during the process. to solve these problems, the magneto-induced electric field was used to assist corn starch hydrolysis due to its thermal effect. results indicated that the method accelerated corn starch acid hydrolysis and decreased the treatment time. the reducing sugar content increased to 0.59 g/l after a 60 s treatment, which was 353.44 % higher than the 20 s treatment, while the average degree of polymerization reached a minimum. the treated starch showed increased solubility and swelling power, as well as decreased freeze-thaw stability. x-ray diffraction, fourier transform infrared spectroscopy, and scanning electron microscopy results suggested that the physicochemical changes of corn starch were due to the thermal effect of the induced electric field. this study is expected to provide an important basis for applying new electric field hydrolysis technology to starch modification. native corn starch was modified by enzymatic debranching (ed), microwave assisted citric acid esterification (mcae), and by dual ed/mcae. the structure and properties of native starch (ns), and the resulting debranched starch (ds), microwave assisted citric acid esterified starch (mcs), and microwave assisted citric acid debranched starch (mcds) were determined and compared. both the morphology and crystalline regions of the modified starches were changed by ed and mcae. ed increased significantly the amylose content and transparency, but decreased the in vitro enzymatic digestibility, freeze thaw stability and relative crystallinity of ds compared to those of ns. mcae produced a decrease in amylose content, transparency, in vitro enzymatic digestibility, and relative crystallinity, but increased the freeze-thaw stability of mcs compared to ns, and of mcds compared to ds. the a-type crystalline structure of ns and ds was changed to b-type crystalline structure after mcae treatment, and a new ftir characteristic band at 1735 cm-1 was observed for mcs and mcds. this work provides insights for producing esterified corn starches by a combined enzyme, microwave and organic acid novel technology. with increasing awareness of environmental protection, petroleum-based raw materials are continuously decreasing, which in turn necessitated the development of eco-friendly sustainable biomaterials, as alternative strategy. starch could be an ideal substitute. corn starch has been used as a renewable material for development of biodegradable packaging, owing to great varieties, low cost, large-scale industrial production, and good films forming properties. unfortunately, its poor mechanical and barrier properties have limited the application of starch-based films. thence, plasticizers were added to overcome the aforementioned pitfalls and improve the films elongation, distribution, flexibility, elasticity, and rigidity. addition of plasticizers can change the continuity and therefore would enhance the properties of corn starch-based films. while plasticization can improve the tensile strength and percent elongation, it can reduce the water resistance in prepared films. herein, we focused on changes of starch granules during gelatinization process, types of biodegradable films, as well as the types of modified starch with plasticizers. furthermore, the influence of plasticizers on corn starch-based films and the physicochemical properties of various types of corn starch-based films were also addressed. the constitutive active/androstane receptor (car) is a nuclear receptor that functions as a xenobiotic sensor, which regulates the expression of enzymes involved in drug metabolism and of efflux transporters. evaluation of the binding properties between car and a drug was assumed to facilitate the prediction of drug-drug interaction, thereby contributing to drug discovery. the purpose of this study is to construct a system for the rapid evaluation of interactions between car and drugs. we prepared recombinant car protein using the escherichia coli expression system. since isolated car protein is known to be unstable, we designed a fusion protein with the car binding sequence of the nuclear receptor coactivator 1 (ncoa1), which was expressed as a fusion protein with maltose binding protein (mbp), and purified it by several chromatography steps. the thus-obtained car/ncoa1 tethered protein (car-ncoa1) was used to evaluate the interactions of car with agonists and inverse agonists by a thermal denaturation experiment using differential scanning fluorometry (dsf) in the presence and absence of drugs. an increase in the melting temperature was observed with the addition of the drugs, confirming the direct interaction between them and car. dsf is easy to set up and compatible with multiwell plate devices (such as 96-well plates). the use of dsf and the car-ncoa1 fusion protein together allows for the rapid evaluation of the interaction between a drug and car, and is thereby considered to be useful in drug discovery. starch is a benign bio-polymeric material with a diversity of desirable functionalities namely biocompatibility and hydrophilicity features. besides, corn silk with cellulose-protein structure can be used as an available and clean compound for medical applications. hence, the advantages of both mentioned biocompatible compounds with potentiality to form hydrogel are considered via their combination. up to now, there is no report on dealing with starch beside corn silk on polyester fabric in the literatures. herein, low starch/corn silk dual hydrogel was incorporated into nano zno functionalized polyester fabric via a one-step simple method. imparting flame retardant feature with no dripping, antibacterial/antifungal and self-cleaning activities with the enhanced mechanical characteristics are the advantages of the stated approach in this paper. presence of dual hydrogel on nano zno treated polyester fabric helps to significantly improve the cell viability to 129% because of hydrogel feature. finally, this paper renders a feasible and clean approach for textile functionalization with respect to the both human health issues and environmental observations. herein, corn starch-based films were prepared by casting method and different concentrations of zanthoxylum bungeanum essential oil (zyo) were added to evaluate the morphological, optical, mechanical, and barrier properties of the resultant films. additionally, structural analysis was carried out via atomic force microscopy and the antibacterial activities against staphylococcus aureus, escherichia coli, and listeria monocytogenes were assessed. we found that the elongation at break was significantly increased (p <  0.05), whereas tensile strength, moisture content, solubility in water, and water vapor permeability rate were significantly decreased (p <  0.05) in films incorporated with zyo compared with oil-free films. furthermore, incorporation of zyo increased the opacity and decreased the gloss of films. incorporation of zyo appears to increase the surface roughness and the antibacterial activity of the films. in sum, zyo can potentially be used in food packaging, particularly food intended to be protected from light and susceptible to spoilage by microorganisms. the objective of this study was to produce renewable aerogels from native and anionic corn starches loaded with pinhão coat extract (pce) with water absorbent capacity, antioxidant activity and controlled release of phenolic compounds in a hydrophilic food simulant media. starch aerogels were produced with different concentrations of pce, 5 and 10%, and evaluated for ft-ir spectra, relative crystallinity, thermal properties, water absorption capacity (wac), density, antioxidant activity and in vitro release. thermal stability of the compounds was improved by the incorporation of pce. the aerogels presented high wac of 541 to 731% and low-density values of 0.03 g.cm-1. the highest inhibition of dpph and abts radicals was presented to anionic starch aerogels with 10% pce rendering 26% of inhibition of abts and 24% of dpph. the maximum in vitro releases for native and anionic starch aerogels with 5% of pce were 28.70 and 29.44%, respectively, and for aerogels with 10% of pce they were 34.27 and 35.94%, respectively. the anionic starch aerogels had the highest amount of phenolic compounds released when compared to the native starch aerogels. the starch-based bioactive aerogels showed potential to be applied in food packaging as water absorbent and as a carrier of phenolic compounds. to understand the damage capability of the invasive pest spodoptera frugiperda on diffe-rent corn varieties in china, we evaluated survival fitness of s. frugiperda on three varieties of sweet corn (i.e., suitian no.1, zhengtian 68, and huajintian no.1) and three varieties of waxy corn (i.e., jingkenuo 2000, guangheitiannuo, and guangnuo no.1) by constructing life table for all populations. the correlation of biological parameters of s. frugiperda to the main nutrients and crude fiber content in corn leaves were analyzed. results showed that the larva survival rate, pupal weight, and production of s. frugiperda on sweet corns were significantly higher than those on waxy corns. the intrinsic rate of increase (rm) of s. frugiperda on the sweet corns ranged from 0.1566 to 0.1843, and the net reproductive rate (r0) ranged from 187.97 to 353.35, being higher than those reared on the waxy corns (rm was from 0.0998 to 0.1465, and r0 was from 25.89 to 95.34). s. frugiperda reared on the sweet corns had higher population growth ability than those reared on the waxy corns. the content of main nutrients such as vitamin c, starch, soluble sugar, protein, fat, total amino acids, and crude fiber in the sweet corn leaves were higher than those in the waxy corn leaves. there were positive correlations between r0 of s. frugiperda to the contenst of vitamin c, starch, soluble sugar, protein, and crude fiber in corn leaves. the results indicated that s. frugiperda reared on the sweet corn leaves had higher survival fitness than those reared on the waxy corn leaves. kernel processing and theoretical length of cut (tloc) of whole-plant corn silage (wpcs) can affect feed intake, digestibility, and performance of dairy cows. the objective of this study was to evaluate for lactating dairy cows the effects of kernel processing and tloc of wpcs with vitreous endosperm. the treatments were a pull-type forage harvester without kernel processor set for a 6-mm tloc (pt6) and a self-propelled forage harvester with kernel processor set for a 6-mm tloc (sp6), 12-mm tloc (sp12), and 18-mm tloc (sp18). processing scores of the wpcs were 32.1% (pt6), 53.9% (sp6), 49.0% (sp12), and 40.1% (sp18). twenty-four holstein cows (139 ± 63 d in milk) were blocked and assigned to six 4 × 4 latin squares with 24-d periods (18 d of adaptation). diets were formulated to contain 48.5% wpcs, 15.5% citrus pulp, 15.0% dry ground corn, 9.5% soybean meal, 6.8% low rumen degradability soybean meal, 1.8% calcium soap of palm fatty acids (fa), 1.7% mineral and vitamin mix, and 1% urea (dry matter basis). nutrient composition of the diets (% of dry matter) was 16.5% crude protein, 28.9% neutral detergent fiber, and 25.4% starch. three orthogonal contrasts were used to compare treatments: effect of kernel processing (pt6 vs. sp6) and effect of tloc (particle size; sp6 vs. sp12 and sp12 vs. sp18). cows fed sp6 produced 1.2 kg/d greater milk yield with no changes in dry matter intake, resulting in greater feed efficiency compared with pt6. cows fed sp6 also produced more milk protein (+36 g/d), lactose (+61 g/d), and total solids (+94 g/d) than cows fed pt6. the mechanism for increased yield of milk and milk components involved greater kernel fragmentation, starch digestibility, and glucose availability for lactose synthesis by the mammary gland. however, cows fed sp6 had lower chewing time and tended to have greater levels of serum amyloid a compared with pt6. milk yield was similar for sp6 and sp12, but sp12 cows tended to have less serum amyloid a with greater chewing time. cows fed sp18 had lower total-tract starch digestibility and tended to have lower plasma glucose and produce less milk compared with cows fed sp12. compared with pt6, feeding sp6 raised linear odd-chain fa concentration in milk. similarly, a reduction of these same fa occurred for sp12 compared with sp6. cows fed sp6 had greater proportion of milk c14:1 and c16:1 compared with pt6 and sp12. lesser trans c18:1 followed by greater c18:0 concentrations were observed for sp12 and pt6 compared with sp6, which is an indication of more complete biohydrogenation in the rumen. under the conditions of this study, the use of a self-propelled forage harvester with kernel processing set for a 12-mm tloc is recommended for wpcs from hybrids with vitreous endosperm. rapid starch digestion rate is negative for the normal level of human blood glucose. this study investigated the protective effects of corn starch (cs) complexed with soy isoflavone (si) on the control of starch digestibility and glycemic index (gi). the structure of the corn starch-soy isoflavone (cs-si) complexes was characterized by fourier transform infrared (ft-ir), x-ray diffraction (xrd), thermogravimetric analysis (tga), and differential scanning calorimetry (dsc), and the complexes digestibility was evaluated using in vitro digestion model. the results of ft-ir spectrum showed that, compared with corn starch, new characteristic peaks were not occurred in cs-si complexes, and the value of r1047/1022 was decreased, which indicated the short-range structure of cs-si complexes had been reduced. the v-shaped structure characteristic peaks occurred obviously in cs-si complexes detected by xrd patterns, which formed a new crystalline structure. the thermal stability was improved in cs-si complexes revealed by tga and dtg curves that the thermal cracking temperature increased from 315°c to 320°c. the enthalpy (δh) of cs-si complexes decreased from 2.34 j/g to 1.75 j/g showed by dsc data, which indicated that the ordered structure of starch was destroyed. furthermore, the content of resistant starch increased from 10.53% to 21.78% and predicted glycemic index (pgi) reduced in cs-si complexes. in conclusion, the digestibility and pgi of starch can be improved by complexed with soy isoflavone. \n",
      "Euclidean Distance : 15.247357368469238\n",
      "\n",
      "\n",
      "Document: pollen-mediated gene flow from transgenic perennial creeping bentgrass and hybridization at the landscape level. the planting of 162 ha of transgenic glyphosate-resistant creeping bentgrass (agrostis stolonifera) near madras, or, usa, allowed a unique opportunity to study gene flow over time from a perennial outcrossing species at the landscape level. while conducting a four year in situ survey, we collected panicles and leaf tissue samples from creeping bentgrass and its sexually compatible species. seeds from the panicles were planted, and seedlings were tested in the greenhouse for expression of the transgene. gene flow via pollen was found in all four years, at frequencies of 0.004 to 2.805%. chloroplast markers, in combination with internal transcribed spacer nuclear sequence analysis, were used to aid in identification of transgenic interspecific and intergeneric hybrid seedlings found during the testing and of established plants that could not be positively identified in the field. interspecific transgenic hybrids produced on redtop (agrostis gigantea) plants in situ were identified three of the four years and one intergeneric transgenic creeping bentgrass x rabbitfoot grass (polypogon monspeliensis) hybrid was identified in 2005. in addition, we confirmed a non-transgenic creeping bentgrass x redtop hybrid in situ, demonstrating that interspecific hybrids have established in the environment outside production fields. results of this study should be considered for deregulation of transgenic events, studies of population dynamics, and prediction of gene flow in the environment. \n",
      "Euclidean Distance : 15.24769115447998\n",
      "\n",
      "\n",
      "Document: pollen-mediated gene flow from transgenic perennial creeping bentgrass and hybridization at the landscape level. the planting of 162 ha of transgenic glyphosate-resistant creeping bentgrass (agrostis stolonifera) near madras, or, usa, allowed a unique opportunity to study gene flow over time from a perennial outcrossing species at the landscape level. while conducting a four year in situ survey, we collected panicles and leaf tissue samples from creeping bentgrass and its sexually compatible species. seeds from the panicles were planted, and seedlings were tested in the greenhouse for expression of the transgene. gene flow via pollen was found in all four years, at frequencies of 0.004 to 2.805%. chloroplast markers, in combination with internal transcribed spacer nuclear sequence analysis, were used to aid in identification of transgenic interspecific and intergeneric hybrid seedlings found during the testing and of established plants that could not be positively identified in the field. interspecific transgenic hybrids produced on redtop (agrostis gigantea) plants in situ were identified three of the four years and one intergeneric transgenic creeping bentgrass x rabbitfoot grass (polypogon monspeliensis) hybrid was identified in 2005. in addition, we confirmed a non-transgenic creeping bentgrass x redtop hybrid in situ, demonstrating that interspecific hybrids have established in the environment outside production fields. results of this study should be considered for deregulation of transgenic events, studies of population dynamics, and prediction of gene flow in the environment. \n",
      "Euclidean Distance : 15.24769115447998\n",
      "\n",
      "\n",
      "Document: ovary signals for pollen tube guidance in chalazogamous  pollen allergen skin test and specific ige reactivity among filipinos: a community-based study. pollen wall development in mango (mangifera indica l., anacardiaceae). most flowering plants show porogamy in which the pollen tubes reach the egg apparatus through the micropyle. however, several species show chalazogamy, an unusual pollen tube growth, in which the pollen tubes reach the embryo sac through the chalaza. while ovary signals for pollen tube growth and guidance have been extensively studied in porogamous species, few studies have addressed the process in chalazogamous species such as mango (mangifera indica l.), one of the five most important fruit crops worldwide in terms of production. in this study, we characterize pollen-pistil interaction in mango, paying special attention to three key players known to be involved in the directional pollen tube growth of porogamous species such as starch, arabinogalactan proteins (agps), and γ-aminobutyric acid (gaba). starch grains were observed in the style and in the ponticulus at anthesis, but their number decreased 1 day after anthesis. agps, revealed by jim8 and jim13 antibodies, were homogenously observed in the style and ovary, but were more conspicuous in the nucellus around the egg apparatus. gaba, revealed by anti-gaba antibodies, was specifically observed in the transmitting tissue, including the ponticulus. moreover, gaba was shown to stimulate in vitro mango pollen tube elongation. the results support the heterotrophic growth of mango pollen tubes in the style at the expense of starch, similarly to the observations in porogamous species. however, unlike porogamous species, the micropyle of mango does not show high levels of gaba and starch, although they were observed in the ponticulus and could play a role in supporting the unusual pollen tube growth in chalazogamous species. despite the clinical importance of pollen allergens among filipinos, few studies delve into the sensitization profiles of filipinos against pollen allergens. this study determined the sensitization profile of filipinos to pollen using skin prick test (spt) and pollen-specific elisa. pollen from fifteen selected plant sources was collected and extracted for use in sensitization tests. volunteers were interviewed for their clinical history prior to blood sampling and spt. the blood samples collected were assessed using enzyme-linked immunosorbent assay (elisa). the best panel of pollen allergens for the skin prick test was mangifera indica (64%), acacia auriculiformis (28%), mimosa spp. (25%) amaranthus spinosus (22%), lantana camara (20%), pilea microphylla (16%) and dichanthium aristatum (15%). young adults had more sensitizations to pollen than among early childhood and elderly. there were more allergic subjects that have rhinitis (61%) than asthma (42%) and atopic dermatitis (35%). pollen-specific ige levels show low percent reactivity as compared to the skin test with cocos nucifera obtaining the highest ige reactivity (21%). pollen allergens from both arboreal and herbaceous plants used in this study yielded positive reactivities for both skin tests and specific ige tests. the mango (mangifera indica) is a woody perennial crop currently cultivated worldwide in regions with tropical and subtropical climates. despite its importance, an essential process such as pollen development, and, specifically, cell wall composition that influences crosstalk between somatic cells and the male germline, is still poorly understood in this species and in the anacardiaceae as a whole. a detailed understanding of this process is particularly important to know the effect of low temperatures during flowering on pollen development that can be a limiting factor for fertilization and fruit set. to fill this gap, we performed a thorough study on the cell wall composition during pollen development in mango. the results obtained reveal a clear differentiation of the cell wall composition of the male germline by pectins, agps and extensins from the early developmental stages during microsporogenesis and microgametogenesis reflecting a restricted communication between the male germline and the surrounding somatic cells that is very sensitive to low temperatures. the combination of the results obtained provides an integrated study on cell wall composition of the male germline in mango that reveals the crucial role of the sporophyte and the gametophyte and the vulnerability of the process to low temperatures. \n",
      "Euclidean Distance : 15.250540733337402\n",
      "\n",
      "\n",
      "Document: electronic properties of triangle molybdenum disulfide (mos dna cleavage by chemically exfoliated molybdenum disulfide nanosheets. a long-term stable aqueous aluminum battery electrode based on one-dimensional molybdenum-tantalum oxide nanotube arrays. photodriven transient picosecond top-layer semiconductor to metal phase-transition in p-doped molybdenum disulfide. efficient dispersive micro solid-phase extraction of antidepressant drugs by a robust molybdenum-based coordination polymer. effect of side chain modifications in imidazolium ionic liquids on the properties of the electrical double layer at a molybdenum disulfide electrode. nanocomposite of ultra-small moo spatial-temporal analysis and background value determination of molybdenum concentration in basins with high molybdenum geochemical background - a case study of the upper yi river basin. elucidation of molybdenum trioxide sulfurization: mechanistic insights into two-dimensional molybdenum disulfide growth. engineering heterologous molybdenum-cofactor-biosynthesis and nitrate-assimilation pathways enables nitrate utilization by saccharomyces cerevisiae. conversion of dinitrogen into nitrile: cross metathesis of n2-derived molybdenum nitride with alkynes. molybdenum-containing polypyrrole self-supporting hollow flexible electrode for hydrogen peroxide detection in living cells. molybdenum sulfide-modified metal-free graphitic carbon nitride/black phosphorus photocatalyst synthesized via high-energy ball-milling for efficient hydrogen evolution and hexavalent chromium reduction. a hybrid polyvinyl alcohol/molybdenum disulfide nanosheet hydrogel with light-triggered rapid self-healing capability. molybdenum derived from nanomaterials incorporates into molybdenum enzymes and affects their activities in vivo. allergic connubial contact dermatitis caused by molybdenum from fiberglass. highly efficient antimicrobial agents based on sulfur-enriched, hydrophilic molybdenum disulfide nano/microparticles and coatings functionalized with palladium nanoparticles. 2d-molybdenum disulfide-derived ion source for mass spectrometry. removal of cadmium from wastewater by magnetic zeolite synthesized from natural, low-grade molybdenum. zeolite stabilized di- and tetranuclear molybdenum sulfide clusters form stable catalytic hydrogenation sites. the electronic structures and transition properties of three types of triangle mos2 clusters, a (mo edge passivated with two s atoms), b (mo edge passivated with one s atom), and c (s edge) have been explored using quantum chemistry methods. the highest occupied molecular orbital (homo)-lowest unoccupied molecular orbital (lumo) gap of b and c is larger than that of a, due to the absence of the dangling of edge s atoms. the frontier orbitals (fmos) of a can be divided into two categories, edge states from s3 at the edge and hybrid states of mo4 and s3 covering the whole cluster. due to edge/corner states appearing in the fmos of triangle mos2 clusters, their absorption spectra show unique characteristics along with the edge structure and size. chemically exfoliated mos2 (ce-mos2) nanosheets have been widely used in biomedical and environmental fields. some in vitro studies demonstrated that ce-mos2 might induce toxicity. however, the understanding of the mechanism of potential toxicity is lacking. in this study, we found that ce-mos2 could directly induce breakage of double-stranded dna with or without an external energy input, making it different from other two-dimensional nanomaterials. in a dark environment, the dna cleavage exhibited a ph-dependent trend due to reactive oxygen species generation under different ph values. under photoirradiation, dna cleavage could be enhanced. this study provides insights into the potential environmental risk and toxicity of ce-mos2 in the aquatic environment. aluminum ion aqueous batteries (aibs) are among the most promising candidates for high energy density devices due to the multivalent redox processes associated with al3+ ion intercalation. however, only a few stable aib electrode materials have been reported so far. moo3 is a very promising electrode material due to its octahedral layered crystal structure which can accommodate multivalent cation by intercalation. however, the poor electrochemical stability of moo3 and the sluggish intercalation kinetics of al3+ ion in mo oxides electrodes limit its practical application. in this work, we propose a strategy to overcome such shortcomings of moo3 by fabricating electrodes composed of self-ordered one-dimensional (1d) motaox nanotubes synthesized via electrochemical anodization of mo-ta alloy substrates. we show that this approach allows for direct incorporation of ta in the mo oxide nanotubes. the resulting motaox nanotubes, composed of octahedral moo3 and rhombohedral mo2ta2o11 phases, exhibit remarkable electrochemical stability and al-ion storage properties in aqueous electrolytes that are superior to that of pristine mo oxide or other most efficient electrode materials reported to date. such motaox nanotube-based electrodes can achieve a specific capacity of 1180 ma h cm-3 (337 ma h g-1, 141 μa h cm-2) at 1.25 a cm-3 (∼0.35 a g-1, 0.15 ma cm-2). more importantly, the capacity retention of such nanotube array electrodes remains above 83% of the initial capacity after 3000 cycles. visible light is shown to create a transient metallic s-mo-s surface layer on bulk semiconducting p-doped indirect-bandgap 2h-mos2 . optically created electron-hole pairs separate in the surface band bending region of the p-doped semiconducting crystal causing a transient accumulation of electrons in the surface region. this triggers a reversible 2h-semiconductor to 1t-metal phase-transition of the surface layer. electron-phonon coupling of the indirect-bandgap p-doped 2h-mos2 enables this efficient pathway even at a low density of excited electrons with a distinct optical excitation threshold and saturation behavior. this mechanism needs to be taken into consideration when describing the surface properties of illuminated p-doped 2h-mos2 . in particular, light-induced increased charge mobility and surface activation can cause and enhance the photocatalytic and photoassisted electrochemical hydrogen evolution reaction of water on 2h-mos2 . generally, it opens up for a way to control not only the surface of p-doped 2h-mos2 but also related dichalcogenides and layered systems. the findings are based on the sensitivity of time-resolved electron spectroscopy for chemical analysis with photon-energy-tuneable synchrotron radiation. a molybdenum-based coordination polymer {[mo(pda)(no)(μ-o)moo3]·1.42h2o·0.58c2h5oh}n (1) (pda is 1,10-phenanthroline-2,9-dicarboxylate) was synthesized using solvothermal reaction conditions and characterized using a suite of analytical techniques. single-crystal x-ray diffraction studies reveal a 1d chain structure, with close contacts expanding the structure into 3d including π-interactions and hydrogen bonding. the utility of 1 as a sorbent for dispersive micro solid-phase extraction (d-μspe) of basic organic compounds such as antidepressants is supported by the presence of many functional groups on the surface of 1 (such as pendant carboxylates, mo=o, mo-no, and ch groups) as well as extensive electrostatic interactions. therefore, 1 can be a suitable choice as sorbent in the d-μspe of antidepressant drugs from human plasma samples via appreciable adsorbate-adsorbent interactions. determination of the extracted antidepressant drugs was conducted using high-performance liquid chromatography-ultraviolet (hplc-uv), with calibration plots being linear in the concentration range 0.1-500 ng ml-1 for amitriptyline and nortriptyline, 0.2-500 ng ml-1 for imipramine, and 0.5-300 ng ml-1 for sertraline. the relative standard deviation (rsd) values were calculated for both intra-day and inter-day precision, and the rsd% values were in the range 3.9 to 5.2% and 4.6-5.4%, respectively. the limits of detection (lods) was determined as 0.03-0.2 ng ml-1. due to the good stability and reusability of the sorbent, the adsorption capacity had no obvious decrease after being used 20 times. finally, the d-μspe-hplc-uv method was applied for the determination of antidepressant drugs in human plasma samples with recoveries of the analytes in the range 94.9 to 102%. the article describes the synthesis of a robust molybdenum-based coordination polymer, and its application as sorbent for dispersive micro solid-phase extraction of antidepressant drugs from human plasma samples. knowledge of how the molecular structures of ionic liquids (ils) affect their properties at electrified interfaces is key to the rational design of ils for electric applications. polarizable molecular dynamics simulations were performed to investigate the structural, electrical, and dynamic properties of electric double layers (edls) formed by imidazolium dicyanamide ([imx1][dca]) at the interface with the molybdenum disulfide electrode. the effect of side chain of imidazolium on the properties of edls was analyzed by using 1-ethyl-3-methylimidazolium ([im21]), 1-octyl-3-methylimidazolium ([im81]), 1-benzyl-3-methylimidazolium ([imb1]), and 1-(2-hydroxyethyl)-3-methylimidazolium ([imo1]) as cations. using [im21] as reference, we find that the introduction of octyl or benzyl groups significantly alters the interfacial structures near the cathode because of the reorientation of cations. for [im81], the positive charge on the cathode induces pronounced polar and non-polar domain separation. in contrast, the hydroxyl group has a minor effect on the interfacial structures. [imb1] is shown to deliver slightly larger capacitance than other ils even though it has larger molecular volume than [im21]. this is attributed to the limiting factor for capacitance being the strong association between counter-ions, instead of the free space available to ions at the interface. for [im81], the charging mechanism is mainly the exchange between anions and octyl tails, while for the other ils, the mechanism is mainly the exchange of counter-ions. analysis on the charging process shows that the charging speed does not correlate strongly with macroscopic bulk dynamics like viscosity. instead, it is dominated by local displacement and reorientation of ions. moo2 is a promising anode material for lithium-ion batteries, however, the lithiation of bulk moo2 is usually limited to addition-type reaction at room temperature, and the conversion reaction is hindered because of the sluggish kinetics. herein, a nanocomposite of moo2 embedded in nitrogen-doped carbon (moo2/nc) is synthesized through the in situ thermolysis of an organic molybdenum complex moo2(acac)(phen) (acac = acetylacetone, phen = 1,10-phenanthroline). owing to the fact that [moo2]2+ can be strongly chelated by phen, the molybdenum source in the moo2(acac)(phen) precursor is highly dispersed, leading to the formation of ultra-small moo2 nanoparticles in the nanocomposite, which can facilitate the conversion reaction. moreover, the nc matrix can guarantee a high electrical conductivity and effectively accommodate the volume changes triggered by the conversion reaction. consequently, the moo2/nc nanocomposite exhibits outstanding electrochemical properties, including large reversible capacity of 950 ma h g-1 at 0.1 a g-1, high-rate capability of 605 ma h g-1 at 2 a g-1, and excellent cycling stability over 500 cycles as an anode material for lithium-ion batteries. the environmental background value of the river section is important. it can be used to evaluate the effect of pollution control of the upstream of that river section, analyze the trend of environmental pollution, and assist the government to make decisions. yi river is the main tributary of the yellow river. in the headwaters of the yi river, there are two very large molybdenum mines with a history of mining and smelting of many years. this area is also a region with a high molybdenum geochemical background. using the collected regional molybdenum geochemical map, historical monitoring data, sampling data, remote sensing image, and spatial information of mineral enterprises, we found two reasons of why the molybdenum concentration is unusual in the basin. the first reason is the area is a high molybdenum region. the second reason is that the inherent solubility of molybdenum in the soil is changed due to human engineering activities. in this paper, we did a linear fitting on the soil samples and water samples collected from the natural areas and areas affected by human mining activities, and established a leaching model. by comparing the leaching capability of molybdenum in the soil of different areas, we found that the molybdenum release capability in areas affected by human mining was much higher than that in natural areas. finally, this paper proposed a method to analyze the contribution rate of molybdenum concentration of this river section, using a combination of the leaching model and the d8 algorithm. the experimental results showed that the contribution rate of natural factors and human influence factors at the exit section of yi river was 81.38% and 18.62% respectively. the background molybdenum concentration in this section was 0.16 mg/l. powder vaporization is a common method for the generation of large-area, single-crystal, two-dimensional molybdenum disulfide. while commonly employed as a growth method, the fundamental molecular mechanisms are not well understood. recent ab initio analyses have shown that molybdenum oxysulfide rings play a key role in the sulfurization of molybdenum trioxide from elemental sulfur. in this study, we utilize molecular dynamics simulations with a reactive force field and ab initio calculations to elucidate the reaction pathway of sulfur with molybdenum trioxide. the molecular dynamics simulations demonstrated that for all sulfur allotropes the reaction pathway could be reduced to that of disulfur, trisulfur, or a combination of the two and that molybdenum trioxide can catalyze the decomposition of larger sulfur allotropes. ab initio calculations were used to illuminate the intermediates and transition states in the reaction pathways for disulfur and trisulfur. analysis of the temperature dependence of the transition state energies shows that the maximum reaction rates occur between 1000 and 1100 k, which corresponds with commonly reported experimental growth temperatures for molybdenum disulfide. metabolic capabilities of cells are not only defined by their repertoire of enzymes and metabolites, but also by availability of enzyme cofactors. the molybdenum cofactor (moco) is widespread among eukaryotes but absent from the industrial yeast saccharomyces cerevisiae. no less than 50 moco-dependent enzymes covering over 30 catalytic activities have been described to date, introduction of a functional moco synthesis pathway offers interesting options to further broaden the biocatalytic repertoire of s. cerevisiae. in this study, we identified seven moco biosynthesis genes in the non-conventional yeast ogataea parapolymorpha by spycas9-mediated mutational analysis and expressed them in s. cerevisiae. functionality of the heterologously expressed moco biosynthesis pathway in s. cerevisiae was assessed by co-expressing o. parapolymorpha nitrate-assimilation enzymes, including the moco-dependent nitrate reductase. following two-weeks of incubation, growth of the engineered s. cerevisiae strain was observed on nitrate as sole nitrogen source. relative to the rationally engineered strain, the evolved derivatives showed increased copy numbers of the heterologous genes, increased levels of the encoded proteins and a 5-fold higher nitrate-reductase activity in cell extracts. growth at nm molybdate concentrations was enabled by co-expression of a chlamydomonas reinhardtii high-affinity molybdate transporter. in serial batch cultures on nitrate-containing medium, a non-engineered s. cerevisiae strain was rapidly outcompeted by the spoilage yeast brettanomyces bruxellensis. in contrast, an engineered and evolved nitrate-assimilating s. cerevisiae strain persisted during 35 generations of co-cultivation. this result indicates that the ability of engineered strains to use nitrate may be applicable to improve competitiveness of baker's yeast in industrial processes upon contamination with spoilage yeasts. direct synthesis of nitrile from n2 under mild conditions is of great importance and has attracted much interest. herein, we report a direct conversion of n2 into nitrile via nitrile alkyne cross metathesis (nacm) process involving a n2-derived mo nitride. treatment of the mo nitride with alkyne in the presence of kotf afforded an alkyne coordinated nitride, which was then transformed into mo(v) carbyne and corresponding nitrile upon 1e- oxidation. both aryl and alkyl substituted alkyne underwent this process smoothly. experiments and dft calculations have proved that the oxidation state of mo center played a crucial role. this method does not rely on the nucleophilicity of the n2-derived metal nitride, offering a novel strategy to the n2 fixation chemistry. a flexible electrode based on polypyrrole-supported free-standing molybdenum oxide-molybdenum disulfide/polypyrrole nanostructure (moo3-mos2/ppy) was synthesized. the petal-like moo3-mos2 sheets grown on ppy were prepared step by step through simple electrodeposition and hydrothermal methods. the corresponding surface morphological and structural characterizations were characterized by field emission scanning electron microscopy (fesem), x-ray diffraction (xrd) and x-ray photoelectron spectroscopy (xps). the results showed that the prepared petal moo3-mos2 hybrid nanomaterials were uniformly distributed on the ppy skeleton and exhibited a three-dimensional porous network structure. the flexible electrode was used for non-enzymatic detection of hydrogen peroxide (h2o2), and the developed moo3-mos2/ppy nanomaterials exhibited high electrochemical sensing performance in the range of 0.3-150 μm, with the detection limit of 0.18 μm (s/n = 3). the excellent detection properties enabled the moo3-mos2/ppy flexible electrode to detect h2o2 released by living cells. the resulting moo3-mos2/ppy flexible electrode also has the advantages of customizable shape and adjustability, which provides a potential platform for constructing clinically diagnosed in vivo portable instruments and real-time environmental monitoring. improving the photocatalytic property of metal-free photocatalyst is still a challenging work. herein, a novel high-efficiency molybdenum sulfide (mos2)-modified metal-free graphitic carbon nitride (g-c3n4)/black phosphorus (bp) photocatalyst (mcn/bp/ms) was synthesized on a large scale via high-energy ball milling process. the optimized mcn/bp/ms exhibits the excellent hydrogen evolution rate of 2146.8 µmol·g-1·h-1, and hexavalent chromium (cr(ⅵ)) reduction activity with an apparent rate constant of 0.1464 min-1 and a degradation rate of 100% in 25 min. detailed characterizations and mechanism research verified that the significantly improved photocatalytic activity of mcn/bp/ms mainly profited from the matched band structure, enhanced light absorption, intense interface contact, as well as the type-ⅰ/z hybrid charge transfer mechanism, which gave rise to a consecutive multistep charge migration, thus the photocarriers transfer and separation can be greatly promoted, and the photogenerated electrons with high reducing capacity can be preserved. this work not only provides a high-efficiency g-c3n4-based noble-metal-free photocatalyst, but also affords a beneficial inspiration for improving the photocatalytic property of the metal free photocatalyst. a hybrid self-healing hydrogel (pm hydrogel), based on polyvinyl alcohol (pva) and chemically exfoliated molybdenum disulfide (ce-mos2) nanosheets, was prepared by a simple freeze-thaw method. due to the excellent photothermal conversion properties of ce-mos2 nanosheets, the pm hydrogel self-repaired rapidly under near infrared (nir) light irradiation for only 3 min with a high healing efficiency of 91.8 ± 3.3%. the pva content, ce-mos2 nanosheet loading and light irradiation time played important roles in the self-healing performance. additionally, the pm hydrogel also revealed good self-healing properties with a healing efficiency of 60.6 ± 3.6% after the cut surfaces were separated for 24 h. the present approach provides an effective strategy for fabricating fast light-triggered hydrogen-bond based self-healing systems. the as-prepared hybrid pm hydrogel has great potential as a soft biomaterial for long-term applications due to its biocompatibility and self-healing capability. many nanoscale biomaterials fail to reach the clinical trial stage due to a poor understanding of the fundamental principles of their in vivo behaviour. here we describe the transport, transformation and bioavailability of mos2 nanomaterials through a combination of in vivo experiments and molecular dynamics simulations. we show that after intravenous injection molybdenum is significantly enriched in liver sinusoid and splenic red pulp. this biodistribution is mediated by protein coronas that spontaneously form in the blood, principally with apolipoprotein e. the biotransformation of mos2 leads to incorporation of molybdenum into molybdenum enzymes, which increases their specific activities in the liver, affecting its metabolism. our findings reveal that nanomaterials undergo a protein corona-bridged transport-transformation-bioavailability chain in vivo, and suggest that nanomaterials consisting of essential trace elements may be converted into active biological molecules that organisms can exploit. our results also indicate that the long-term biotransformation of nanomaterials may have an impact on liver metabolism. in this research the molybdenum disulfide (mos2)-based nano/microparticles and coatings were synthesized through a simple, one-step hydrothermal approach without any other additives. composition, structure, and morphology of the synthesized mos2-based materials were investigated using ultraviolet-visible spectroscopy (uv-vis), inductively coupled plasma optical emission spectrometry (icp-oes), scanning electron microscopy (sem), transmission electron microscopy (tem), atomic force microscopy (afm), x-ray diffraction (xrd), x-ray photoelectron spectroscopy (xps) and energy-dispersive x-ray spectroscopy (edx) techniques. the fabricated materials exhibited relatively small (δθ = 18.7 ± 2.5⁰) contact angle and prominent hydrophilic properties, which are attributable to sulfur-enriched mos2 composite as evidenced by simultaneous thermal analysis (sta) coupled with mass spectrometric (ms) analysis of evolving gaseous species (tg/dta-ms) analysis. such nanostructures exhibit a better adhesion of biomolecules, thus facilitating the interaction between them, as confirmed by highly effective antimicrobial action. the present study examines antimicrobial properties of hydrophilic, sulfur-enriched mos2 nano/microparticles as well as mos2-based coatings against various humans' pathogenic bacteria such as salmonella enterica, pseudomonas aeruginosa, escherichia coli, methicillin-resistant staphylococcus aureus (mrsa), micrococcus luteus, and two candida yeast strains (c. parapsilosis, c. krusei). the mos2-ns (40 μg ml-1) showed over 90% killing efficiency against s. aureus mrsa bacteria and both candida yeast when exposed for 24 h. petal-like mos2 microstructures and heterostructured mos2/ti and pd/mos2/ti coatings also possessed high antimicrobial potential and are considered as a promising antimicrobial agent. the mos2-induced production of intracellular reactive oxygen species (ros) was evidenced by measuring the standard dcf dye fluorescence. generation of current or potential at nanostructures using appropriate stimuli is one of the futuristic methods of energy generation. we developed an ambient soft ionization method for mass spectrometry using 2d-mos2, termed streaming ionization, which eliminates the use of traditional energy sources needed for ion formation. the ionic dissociation-induced electrokinetic effect at the liquid-solid interface is the reason for energy generation. we report the highest figure of merit of current generation of 1.3 a/m2 by flowing protic solvents at 22 μl/min over a 1 × 1 mm2 surface coated with 2d-mos2, which is adequate to produce continuous ionization of an array of analytes, making mass spectrometry possible. weakly bound ion clusters and uric acid in urine have been detected. further, the methodology was used as a self-energized breath alcohol sensor capable of detecting 3% alcohol in the breath. zeolite has a high adsorption capacity for heavy metals, but it is difficult to separate from the medium because of its small particle size. in this study, magnetic zeolite was synthesized from natural, low-grade molybdenum ore by adding nano ferroferric oxide (saturation magnetization 83.43 emu/g) directly in the hydrothermal synthesis process, which was used to adsorb cadmium from wastewater. the results of scanning electron microscopy showed that the nano ferroferric oxide was adhered to the surface of the zeolite to make it magnetic. the vibrating sample magnetometer showed that the larger the amount of nano ferroferric oxide added, the higher the saturation magnetization of the magnetic zeolite. the saturation magnetization of the magnetic zeolite with a loading proportion of 25% was 18.18 emu/g with a specific surface area of 459.8 m2/g. the adsorption experiments showed that when the ph value is greater than 4, the adsorption capacity of magnetic zeolite is high and stable, and the theoretical maximum adsorption capacity is 204.2 mg cd/g. na+ and ca2+ have different inhibitory functions on the adsorption capacity. the mapping graphs showed that cadmium is captured by the magnetic zeolite after contact with cadmium, and xrd confirmed the presence of cadmium oxide in the magnetic zeolite after adsorption, xps and eds results indicated that ion exchange is one of the main mechanisms of cadmium adsorption by magnetic zeolites, and electrostatic adsorption may also have a contribution. supercages of faujasite (fau)-type zeolites serve as a robust scaffold for stabilizing dinuclear (mo 2 s 4 ) and tetranuclear (mo 4 s 4 ) molybdenum sulfide clusters. the fau-encaged mo 4 s 4 clusters have a distorted cubane structure similar to the femo-cofactor in nitrogenase. both clusters possess one unpaired electron per mo atom. additionally, they show identical catalytic activity per sulfide cluster. their catalytic activity is stable (> 150 h) for ethene hydrogenation, while layered mos 2 structures deactivate significantly under the same reaction conditions. \n",
      "Euclidean Distance : 15.253223419189453\n",
      "\n",
      "\n",
      "Document: mof-on-mof membrane with cascading functionality for capturing dichromate ions and  capillary zone electrophoresis as a simple approach for the study of p-arsanilic acid transformation in the process of photolytic degradation. lignin-based magnetic activated carbon for p-arsanilic acid removal: applications and absorption mechanisms. simultaneous degradation of p-arsanilic acid and inorganic arsenic removal using m-rgo/ps fenton-like system under neutral conditions. photodegradation of para-arsanilic acid mediated by photolysis of iron(iii) oxalate complexes. simultaneous removal of para-arsanilic acid and the released inorganic arsenic species by cufe2o4 activated peroxymonosulfate process. highly efficient removal of p-arsanilic acid with fe(ii)/peroxydisulfate under near-neutral conditions. vestibular compensation after vestibular dysfunction induced by arsanilic acid in mice. arsanilic acid contributes more to total arsenic than roxarsone in chicken meat from chinese markets. new insights into mechanism of direct uv photolysis of p-arsanilic acid. protective effects of anthocyanin against apoptosis and oxidative stress induced by arsanilic acid in df-1 cells. phototransformation of p-arsanilic acid in aqueous media containing nitrogen species. silicon improves growth and alleviates oxidative stress in rice seedlings (oryza sativa l.) by strengthening antioxidant defense and enhancing protein metabolism under arsanilic acid exposure. enhanced adsorption of p-arsanilic acid from water by amine-modified uio-67 as examined using extended x-ray absorption fine structure, x-ray photoelectron spectroscopy, and density functional theory calculations. arsanilic acid modified superparamagnetic iron oxide nanoparticles for purification of alkaline phosphatase from hen's egg yolk. the removal efficiency and insight into the mechanism of para arsanilic acid adsorption on fe-mn framework. transformation of para arsanilic acid by manganese oxide: adsorption, oxidation, and influencing factors. mechanisms and effects of arsanilic acid on antibiotic resistance genes and microbial communities during pig manure digestion. arsenic accumulation and speciation in rice grown in arsanilic acid-elevated paddy soil. multiple transformation pathways of p-arsanilic acid to inorganic arsenic species in water during uv disinfection. it is very significant that functional porous metal-organic frameworks are used to manufacture hierarchical components to achieve cascading functions that cannot be achieved by a single-layer metal-organic framework (mof). here, we report two cases of novel mofs constructed by the same ligand, cu(i)-tpt and cu(ii)-tpt (htpt = 5-[4(1h-1,2,4-triazol-1-yl)]phenyl-2h-tetrazole), and prepared a cu(ii)-tpt-on-cu(i)-tpt membrane by a layer-by-layer approach ignoring the lattice mismatch problem. the first cu(i)-tpt layer is grown on an oriented cu2o nanostructured array by a \"one-pot\" approach. the aligned second cu(ii)-tpt layer can be deposited using liquid-phase epitaxy. notably, the prepared cu(ii)-tpt-on-cu(i)-tpt membrane combines adsorption and fluorescence sensing, which exhibited significant adsorption for cr2o72- (203.25 mg g-1) as typical highly poisonous ions with a fluorescence quenching response. hence, based on the oxidation-reduction between cr2o72- and p-arsanilic acid (p-asa), the cu(ii)-tpt-on-cu(i)-tpt membrane's ability to adsorb cr2o72- could be used to design \"on-off-on\" mode fluorescence probes to detect p-asa with high sensitivity (limit of detection (lod) = 0.0556 μg l-1). p-asa can be degraded into highly toxic inorganic arsenic compounds in the natural environment and has received widespread attention. therefore, the integration of adsorption and fluorescence properties makes the cu(ii)-tpt-on-cu(i)-tpt membrane a feasible multifunctional material for pollution control and detection. arsenic aromatic compounds including p-arsanylic acid (pasa) are still widely used in a number of countries as the feed additives in animal breeding resulting in its entering the environment. under the influence of oxidizing agents or uv radiation, pasa undergoes transformations leading to generation of inorganic arsenic species that are more mobile and toxic than organic ones. on the one hand, an approach based on the treatment of contaminated waters by uv irradiation seems perspective for their detoxification, but the feasibility of this approach depends on the composition of the products forming as a result of photodegradation. in the present work, a cze was applied for the study of the pasa degradation process during stationary (308 nm) photolysis in the presence of fe(iii)-oxalate complex. a developed assay allowed controlling the parent compounds and also as-containing products of pasa degradation, presented mainly by arsenate and arsenite ions. it was found that the main inorganic derivatives of the pasa photolytic conversions are presented by arsenate and arsenite ions whose ratio depends on the initial amount of pasa and reaction conditions. it is crucial for water environment security to remove its p-arsanilic acid (p-asa) efficiently. namely, removing p-arsanilic acid from aqueous media through magnetic separation, has become a novel method of removing toxic pollutants from water. batch adsorption experiments demonstrated a higher adsorption of lignin-based magnetic activated carbon (201.64 mg g-1) toward p-asa. in addition, lmac nanoparticles exhibited typical magnetism (35.63 emu g-1 of saturation magnetization) and could be easily separated from the aqueous solution. meanwhile, the endothermic adsorption of p-asa over lmac could spontaneously proceed and be well described by the pseudo-first-order and pseudo-second-order model as well as the intra-particle diffusion model. moreover, the mechanisms during p-asa adsorption over lmac included the electrostatic attraction, surface complexation, π-π stacking and hydrogen bonding interaction. importantly, lignin-based magnetic activated carbon has high absorbability and preferable reusability in real water samples. consequently, this paper provides insights into preparation of the lignin-based magnetic activated carbon may be potential adsorbents for the remediation of organoarsenic compounds. in this study, magnetic material based reduced graphene oxide (m-rgo) was prepared through co-precipitation and displayed high catalytic efficiency together with persulfate (ps) for simultaneous p-arsanilic acid (p-asa) decomposition and arsenic removal. linear sweep voltammetry and chronoamperometric measurements with m-rgo revealed that ps was effectively bound to m-rgo surface and probably formed charge transfer complex, in which m-rgo was pivotal in mediating facile electron transfer. the effects of ph, temperatures, anions, p-asa concentration, ps, and m-rgo dosages on p-asa decomposition were studied in the system. excellent degradation of p-asa was carried out at a wide range of ph values, which was unattainable by other fenton-like processes. under optimal conditions, m-rgo exhibited prominent removal of both p-asa (98.8 %) and inorganic arsenic (89.8 %). m-rgo had reasonably excellent repeatability and stability, and 77.7 % p-asa degraded in the third recovered catalyst. the advantages of environmental friendliness, short reaction time, and straightforward synthesis of m-rgo will facilitate the development of heterogeneous fenton-like catalysts under neutral conditions. organic arsenicals are important environment pollutants due to wide use in livestock and toxicity of degradation products. in this work we report about the efficient photodegradation of the p-arsanilic acid (p-asa) and its decomposition products in the fe(iii)-oxalate assisted approach under nature-relevant conditions. at neutral ph under near-visible uv irradiation the fe(iii) oxalate complexes generate the primary oxidizing intermediate, oh radical (the quantum yield of ϕoh ∼ 0.06), which rapidly reacts with p-asa with high rate constant, (8.6 ± 0.5) × 109 m-1s-1. subsequent radical reactions result in the complete photooxidation of both p-asa and basic aromatic photoproducts with the predominant formation of inorganic arsenic species, mainly as(v), under optimal conditions. comparing with the direct uv photolysis, the presented fe(iii)-oxalate mediated degradation of p-asa has several advantages: higher efficiency at low p-asa concentration and complete degradation of organic arsenic by-products without use of short-wavelength uv radiation. the obtained results illustrate that the fe(iii)-oxalate complexes are promising natural photosensitizers for the removal of arsenic pollutants from contaminated waters. para-arsanilic acid (p-asa), as a major phenylarsonic feed additive, was used annually in many countries. once it enters the water environment, p-asa would be transformed into hypertoxic inorganic arsenic species, causing severe arsenic pollution. in this study, magnetic copper ferrite (cufe2o4) was applied to activate peroxymonosulfate (pms) for p-asa removal and synchronous control of the released inorganic arsenic species. results showed that cufe2o4/pms system presented favorable oxidation ability and close to 85% of 10 mg/l p-asa was eliminated under the condition of simultaneous dosing 0.2 g/l cufe2o4 and 1 mm pms. the rapid decomposition of p-asa resulted from homogeneous pms oxidation and the attack of reactive oxygen species (i.e., so4-, ho and o2-), which was involved the heterogeneous pms activation through the cycles between fe(ii)/fe(iii) and cu(ii)/cu(i). meanwhile, the released inorganic arsenic species during p-asa degradation were found to be controllable via the adsorption on cufe2o4 surface and metal hydroxyl groups played the crucial role. cufe2o4/pms system exhibited the stable and efficient performance within the broad range of ph 3.0-11.0. the existence of common anions (cl-, no3-, hco3-, so42-) and humic acid presented the slight inhibition for p-asa degradation. the reduction of initial p-asa concentration favored the p-asa removal. besides, the catalyst retained a favorable reactivity and stability even after four successive cycles and almost no metal leaching was observed. the rational degradation pathway was mainly involved in the cleavage of asc bond, oxidation of amino group, substitution and oxidation of hydroxyl group. the transformation of arsenic species could be divided into the release of inorganic arsenic species, the oxidation of as(iii) into as(v) and the adsorption of as(v) by cufe2o4. as a common animal feed additive, p-arsanilic acid (p-asa) is thought to be excreted with little uptake and unchanged chemical structure, threatening the environment by potentially releasing more toxic inorganic arsenic. we herein investigated the removal of arsenic by in situ formed ferric (oxyhydr)oxides with the promotion of p-asa degradation in fe(ii)/peroxydisulfate (pds) system. results showed that under acid conditions, p-asa degraded very quickly and over 99% of p-asa (5 μm) was degraded within 10 min at the optimal dosage of fe(ii) (100 μm) and pds (150 μm) at ph 3, while less than 66.4% of arsenic was removed at ph 3-5. higher ph (3-7) would inhibit the degradation of p-asa but promote the arsenic removal. at ph 6-7, over 98.5% of total arsenic was removed, while the degradation efficiency of p-asa was lower than 52.4%. hplc-icp-ms results indicated that the arsenic group was cleaved from p-asa in the form of as(iii) and then rapidly oxidized to as(v). ftir and xps analysis indicated that both as(v) products and residual p-asa were bonded to ferric (oxyhydr)oxides via hydroxyl groups. common cations (e.g., na+, ca2+, mg2+) and anions such as cl-, so42-, co32- had no significant influence on arsenic removal, while sio32-, po43- and ha inhibited the removal of total arsenic, mainly by affecting the zeta potential of iron particles. in summary, the fe(ii)/pds process, as an efficient method for partial oxidation and simultaneous adsorption of p-asa under near-neutral conditions, is expected to control the potential environmental risks of p-asa. when vestibular function is lost, vestibular compensation works for the reacquisition of body balance. for the study of vestibular dysfunction and vestibular compensation, surgical or chemical labyrinthectomy has been performed in various animal species. in the present study, we performed chemical labyrinthectomy using arsanilic acid in mice and investigated the time course of vestibular compensation through behavioral observations and histological studies. the surgical procedures required only paracentesis and storage of 50 µl of p-arsanilic acid sodium salt solution in the tympanic cavity for 5 min. from behavioral observations, vestibular functions were worst at 2 days and recovered by 7 days after surgery. spontaneous nystagmus appeared at 1 day after surgery with arsanilic acid and disappeared by 2 days. histological studies revealed specific damage to the vestibular endorgans. in the ipsilateral spinal vestibular nucleus, the medial vestibular nucleus, and the contralateral prepositus hypoglossal nucleus, a substantial number of c-fos-immunoreactive cells appeared by 1 day after surgery with arsanilic acid, with a maximum increase in number by 2 days and complete disappearance by 7 days. taken together, these findings indicate that chemical labyrinthectomy with arsanilic acid and the subsequent observation of vestibular compensation is a useful strategy for elucidation of the molecular mechanisms underlying vestibular pathophysiologies. organoarsenicals have been used in poultry production for years, however, studies focused on roxarsone (rox), with little attention to p-arsanilic acid (asa). we assessed arsenic (as) concentration and speciation in chicken meat collected from 10 cities in china. the geometric mean for total as in 249 paired raw and cooked samples was 4.85 and 7.27 μg kg-1 fw, respectively. among 81 paired raw and cooked samples, asa and rox were detected in >90% samples, suggesting the prevalence of organoarsenical use in china. asa contributed the most (45% on average) to total as in cooked samples, followed by as(v), dma, as(iii), and rox (7.2-22%). asa was found to contribute more to total as in chicken meat compared to rox for the first time. arsenic in chicken meat showed considerable geographic variation, with higher inorganic arsenic (ias) being detected from cities with higher rox and asa, indicating that organoarsenical use increased ias concentration in chicken meat. when health risk was estimated, dietary exposure to ias would result in an increase of 3.2 bladder and lung cancer cases per 100,000 adults. the result supports the removal of organoarsenicals in poultry production from chinese market and further supports its removal from the global markets. the mechanism of direct uv photolysis of p-arsanilic acid (p-asa), a widely used veterinary drug, was revised by means of laser flash photolysis coupled with high resolution liquid chromatography - mass spectrometry (lc-ms). none of p-asa triplet state or singlet oxygen was found to directly participate in the photodegradation of p-asa as it was assumed in previous works. here we demonstrate that the main primary photoprocess is a monophotonic ionization (ϕion266nm = 0.032) leading to the formation of hydrated electron and corresponding anilinyl cation radical. these primary species react with dissolved oxygen yielding secondary reactive oxygen species. the final organic photoproducts, such as aminophenol and different dimeric products, originate from various reactions between these secondary species. the generation of inorganic arsenic, both as(v) and as(iii), was also observed in agreement with previous works. for the first time we report the quantum yield of p-asa photodegradation, which decreases from 0.058 to 0.035 with the excitation wavelength from 222 to 308 nm. anthocyanin is a natural plant pigment that acts as an antioxidant and scavenges free radicals. this study aimed to investigate the potential protective role of nightshade anthocyanin (na), a natural flavonoid compound, against the arsanilic acid (asa)-induced cell death of df-1 cells. df-1 cells were initially exposed to asa, and then na was applied to the treated cells. cell viability, intracellular reactive oxygen species (ros), mitochondrial membrane potential (mmp), and apoptosis were examined. results showed that na inhibited the asa-induced decrease in cell viability, increase in ros, and loss of mmp in df-1 cells. moreover, caspase-3 activation was inhibited by asa supplementation and na attenuated the asa-induced increase in the percentage of apoptotic cells. in summary, our study suggested that na can enhance asa-induced cytotoxicity and apoptosis, thereby providing a basis for the molecular mechanisms of na-mediated protection. the effects of co-existing nitrogen species in surface water on the phototransformation of organoarsenical p-arsanilic acid (p-asa) have been investigated using a xenon lamp as a simulated solar light source. significant enhancements of p-asa phototransformation efficiency were observed in the presence of nitrate and nitrite, increasing with the concentration of these species and ph, whereas ammonia showed no obvious effect. the products, including inorganic arsenic species and organic derivatives, have been analyzed in order to reveal the phototransformation pathways. in the nitrate and nitrite systems, only small proportions of inorganic arsenic species were generated, with the majority of p-asa being converted into other organoarsenical derivatives through hydroxylation, nitration, and nitrosation. phototransformation of p-asa in collected natural surface water was also observed. this work has implications for the phototransformation of p-asa in nitrogen-contaminated surface water. organoarsenic arsanilic acid (asa) contamination of paddy soil is a serious but less concerned hazard to agriculture and health of people consuming rice as staple food, for rice is one major pathway of arsenic (as) exposure to human food. to date little research has studied the effect of asa on biochemical process of rice. silicon (si) application is able to reduce the toxicities of heavy metals in numerous plants, but little information about asa. this work investigated whether and how si influenced alleviation of asa toxicity in rice at biochemical level to have a better understanding of defense mechanism by si against asa stress. results showed that asa reduced rice growth, disturbed protein metabolism, increased lipid peroxidation but decreased the efficiencies of antioxidant activities compared to control plants, more severe in roots than in shoots. the addition of si in asa-stressed rice plants noticeably increased growth and development as well as soluble protein contents, but decreased malondialdehyde (mda) contents in asa-stressed rice plants, suggesting that si did have critical roles in asa detoxification in rice. furthermore, increased superoxide dismutase (sod), catalase (cat) and peroxidase (pod) activities along with elevated glutathione (gsh) and ascorbic acid (asa) contents implied the active involvement of ros scavenging and played, at least in part, to si-mediated alleviation of asa toxicity in rice, and these changes were related to rice genotypes and tissues. the study provided physio-chemical mechanistic evidence on the beneficial effect of si on organoarsenic asa toxicity in rice seedlings. p-arsanilic acid ( p-asa) is an emerging organoarsenic pollutant comprising both inorganic and organic moieties. for the efficient removal of p-asa, adsorbents with high adsorption affinity are urgently needed. herein, amine-modified uio-67 (uio-67-nh2) metal-organic frameworks (mofs) were synthesized, and their adsorption affinities toward p-asa were 2 times higher than that of the pristine uio-67. extended x-ray absorption fine structure (exafs), x-ray photoelectron spectroscopy (xps), and density functional theory (dft) calculation results revealed adsorption through a combination of as-o-zr coordination, hydrogen bonding, and π-π stacking, among which as-o-zr coordination was the dominant force. amine groups played a significant role in enhancing the adsorption affinity through strengthening the as-o-zr coordination and π-π stacking, as well as forming new adsorption sites via hydrogen bonding. uio-67-nh2s could remove p-asa at low concentrations (<5 mg l-1) in simulated natural and wastewaters to an arsenic level lower than that of the drinking water standard of world health organization (who) and the surface water standard of china, respectively. this work provided an emerging and promising method to increase the adsorption affinity of mofs toward pollutants containing both organic and inorganic moieties, via modifying functional groups based on the pollutant structure to achieve synergistic adsorption effect. recent studies of magnetic carrier technology have focused on its applications in separation and purification technologies, due to easy separation of the target from the reaction medium by applying an external magnetic field. in the present study, fe3o4 superparamagnetic nanoparticles were prepared to utilize a chemical co-precipitation method, then the surfaces of the nanoparticles were modified with arsanilic acid derivatives which were used as the specific nanocarriers for the affinity purification of alkaline phosphatase from the hen's egg yolk. the six different types of magnetic nanocarriers with varied lengths of the linkers were obtained. all samples were characterized step by step and validated using ftir, sem, edx, vsm and xrd analysis methods as the results were shown, the use of inflexible tags with long linkers on the surface of the nanocarrier could lead to better results for separation of alkaline phosphatase from the hen's egg yolk with 76.2% recovery and 1361.7-fold purification. the molecular weight of the purified alkaline phosphatase was estimated to be 68kda by sds-page. the results of this study showed that the novel magnetic nanocarriers were capable of purifying alkaline phosphatase in a practically time and cost effective way. para arsanilic acid (p-asa) is extensively used as feed additives in poultry industry, resulting contaminates soil and natural water sources through the use of poultry litter as a fertilizer in croplands. thus, removal of p-asa prior to its entering environments is significant to control their environmental risk. herein, we studied fe-mn framework and cubic fe(oh)3 as promising novel adsorbents for the removal of p-asa from aqueous solution. the chemical and micro-structural properties of fe-mn framework and cubic fe(oh)3 materials were characterized by x-ray diffraction patterns (xrd), nitrogen adsorption (sbet), zeta (ζ-) potential, scanning electron microscopy (sem), fourier transform infrared spectroscopy (ftir), and x-ray photoelectron spectra (xps). the maximum adsorption capacity for p-asa on fe-mn framework and cubic fe(oh)3 was determined to be 1.3mmolg-1 and 0.72mmolg-1 at ph4.0, respectively. adsorption of p-asa decreased gradually with increasing ph indicated that adsorption was strongly ph dependent. azophenylarsonic acid was identified as an oxidation intermediate product of p-asa after adsorption on fe-mn framework. plausible removal mechanism for p-asa by fe-mn framework was proposed. the obtained results gain insight into the potential applicability of fe-mn framework, which can be potentially important for the removal of p-asa from water. aromatic organoarsenic compounds tend to transform into more mobile toxic inorganic arsenic via several processes, and can inadvertently spread toxic inorganic arsenic through the environment to water sources. to gain insight into the transformation mechanisms, we herein investigated how the process of para arsanilic acid (p-asa) transformation works in detail on the surface of adsorbents by comparing it with phenylarsonic acid (pa) and aniline, which have similar chemical structures. in contrast to the values of 0.23 mmol g-1 and 0.68 mmol g-1 for pa and aniline, the maximum adsorption capacity was determined to be 0.40 mmol g-1 for p-asa at ph 4.0. the results of ftir and xps spectra supported the presence of a protonated amine, resulting in a suitable condition for the oxidation of p-asa. based on the combined results of uv-spectra and uplc-q-tof-ms, we confirmed that the adsorbed p-asa was first oxidized through the transfer of one electron from p-asa on mno2 surface to form a radical intermediate, which through further hydrolysis and coupling led to formation of benzoquinone and azophenylarsonic acid, which was identified as a major intermediate. after that, p-asa radical intermediate was cleaved to form arsenite (iii), and then further oxidized into arsenate (v) with the release of manganese (mn) into solution, indicating a heterogeneous oxidation process. high concentrations of residual arsanilic acid occur in pig manure due to its use in feed to promote growth and control diseases. this study compared the effects of arsanilic acid at three concentrations (0, 325, and 650mg/kg dry pig manure) on the abundance of antibiotic resistance genes (args) and the microbial community during anaerobic digestion. addition of 650mg/kg arsanilic acid enhanced the absolute abundances of tetc, sul2, ermb, and gyra more than twofold in the digestion product. redundancy analysis indicated that the change in the microbial community structure was the main driver of variation in the args profile. the as resistance gene arsc co-occurred with four args and inti1, possibly causing the increase in args under pressure by arsanilic acid. high arsanilic acid concentrations can increase the risk of args occurring in anaerobic digestion products. the amount of arsanilic acid used as a feed additive should be controlled. p-arsanilic acid (asa) is a emerging but less concerned contaminant used in animal feeding operations, for it can be degraded to more toxic metabolites after being excreted by animals. rice is the staple food in many parts of the world, and also more efficient in accumulating arsenic (as) compared to other cereals. however, the uptake and transformation of asa by rice is unclear. this study aimed to evaluate the potential risk of using asa as a feed additive and using the asa contaminated animal manure as a fertilizer. five rice cultivars were grown in soil containing 100mg asa/kg soil, after harvest, as species and their concentrations in different tissues were determined. total as concentration of the hybrid rice cultivar was more than conventional rice cultivars for whole rice plant. for rice organs, the highest as concentration was found in roots. asa could be absorbed by rice, partly degraded and converted to arsenite, monomethylarsonic acid, dimethylarsinic acid, arsenate. the number of as species and their concentrations in each cultivar were related to their genotypes. the soil containing 100mg asa/kg or more is unsuitable for growing rice. the use of asa and the disposal of animal manure requires detailed attention. p-arsanilic acid (p-asa) is widely used in china as livestock and poultry feed additive for promoting animal growth. the use of organoarsenics poses a potential threat to the environment because it is mostly excreted by animals in its original form and can be transformed by uv-vis light excitation. this work examined the initial rate and efficiency of p-asa phototransformation under uv-c disinfection lamp. several factors influencing p-asa phototransformation, namely, ph, initial concentration, temperature, as well as the presence of nacl, nh4(+), and humic acid, were investigated. quenching experiments and lc-ms were performed to investigate the mechanism of p-asa phototransformation. results show that p-asa was decomposed to inorganic arsenic (including as(iii) and as(v)) and aromatic products by uv-c light through direct photolysis and indirect oxidation. the oxidation efficency of p-asa by direct photosis was about 32%, and those by ho and (1)o2 were 19% and 49%, respectively. cleavage of the arsenic-benzene bond through direct photolysis, ho oxidation or (1)o2 oxidation results in simultaneous formation of inorganic as(iii), as(iv), and as(v). inorganic as(iii) is oxidized to as(iv) and then to as(v) by (1)o2 or ho. as(iv) can undergo dismutation or simply react with oxygen to produce as(v) as well. reactions of the organic moieties of p-asa produce aniline, aminophenol and azobenzene derivatives as main products. the photoconvertible property of p-asa implies that uv disinfection of wastewaters from poultry and swine farms containing p-asa poses a potential threat to the ecosystem, especially agricultural environments. \n",
      "Euclidean Distance : 15.260066986083984\n",
      "\n",
      "\n",
      "Document: corn silage-based diet supplemented with increasing amounts of linseed oil: effects on methane production, rumen fermentation, nutrient digestibility, n utilization, and milk production of dairy cows. peanut and linseed oil emulsion gels as potential fat replacer in emulsified sausages. dynamic-mechanical and decomposition properties of flax/basalt hybrid laminates based on an epoxidized linseed oil polymer. improvement of impact strength of polylactide blends with a thermoplastic elastomer compatibilized with biobased maleinized linseed oil for applications in rigid packaging. influence of linseed oil varnish admixture on glauconite clay mortar properties. influence of reaction parameters on the gelation of silanised linseed oil. antioxidant effects of hemp ( calcium salts of long-chain fatty acids from linseed oil decrease methane production by altering the rumen microbiome in vitro. effect of emulsifiers on linseed oil emulsion structure, lipolysis and oxidation during  silane-functionalized sheep wool fibers from dairy industry waste for the development of plasticized pla composites with maleinized linseed oil for injection-molded parts. in vitro and in vivo studies of rumen-protected microencapsulated supplement comprising linseed oil, vitamin e, rosemary extract, and hydrogenated palm oil on rumen fermentation, physiological profile, milk yield, and milk composition in dairy cows. linseed oil supplementation affects fatty acid desaturase 2, peroxisome proliferator activated receptor gamma, and insulin-like growth factor 1 gene expression in turkeys (meleagris gallopavo). effect of adding curcumin on the properties of linseed oil organogels used as fat replacers in pâtés. itraq-based quantitative phosphoproteomics provides insights into the metabolic and physiological responses of a carnivorous marine fish (nibea albiflora) fed a linseed oil-rich diet. structural insights of humins/epoxidized linseed oil/ hardener terpolymerization. biopolymer films based on chitosan/potato protein/linseed oil/zno nps to maintain the storage quality of raw meat. effects of dietary incorporation of linseed oil with soybean isoflavone on fatty acid profiles and lipid metabolism-related gene expression in breast muscle of chickens. effects of supplemental pine needles powder (pinus brutia) on growth performance, breast meat composition, and antioxidant status in broilers fed linseed oil-based diets. synthesis and antimicrobial activity of metal-containing linseed oil-based waterborne urethane oil wood coatings. tea catechin role in decreasing the oxidation of dairy beverages containing linseed oil. in this study, we assessed the effects of increasing amounts of linseed oil (lso) in corn silage-based diets on enteric ch4 production, rumen fermentation characteristics, protozoal population, nutrient digestibility, n utilization, and milk production. for this purpose, 12 multiparous lactating holstein cows (84 ± 28 d in milk; mean ± sd) fitted with ruminal cannula were used in a replicated 4 × 4 latin square design (35-d period). the cows were fed ad libitum a total mixed ration without supplementation (control) or supplemented [on a dry matter (dm) basis] with lso at 2% (lso2), 3% (lso3) or 4% (lso4). the forage:concentrate ratio was 61:39 (on dm basis) and was similar among the experimental diets. the forage portion consisted of corn silage (58% diet dm) and timothy hay (3% diet dm). the proportions of soybean meal, corn grain and soybean hulls decreased as the amount of lso in the diet increased. daily methane production (g/d) decreased quadratically as the amount of lso increased in the diet. increasing lso dietary supplementation caused a linear decrease in ch4 emissions expressed on either dm intake (dmi) basis (-9, -20, and -28%, for lso2, lso3, and lso4, respectively) or gross energy intake basis (-12, -22, and -31%, for lso2, lso3, and lso4, respectively). at 2 and 3% lso, the decrease in enteric ch4 emissions occurred without negatively affecting dmi or apparent total-tract digestibility of fiber and without changing protozoa numbers. however, these 2 diets caused a shift in volatile fatty acids pattern toward less acetate and more propionate. the effect of the lso4 diet on enteric ch4 emissions was associated with a decrease in dmi, fiber apparent-total-tract digestibility, protozoa numbers (total and genera), and an increase in propionate proportion at the expense of acetate and butyrate proportions. methane emission intensity [g of ch4/kg of energy-corrected milk (ecm)] decreased linearly (up to 28% decrease) with increasing lso level in the diet. milk fat yield decreased linearly (up to 19% decrease) with increasing inclusion of lso in the diet. milk protein yield increased at 2% or 3% lso and decreased to the same level as that of the nonsupplemented diet at 4% lso (quadratic effect). yield of ecm was unchanged by lso2 and lso3 treatments but decreased (-2.8 kg/d) upon supplementation with 4% lso (quadratic effect). efficiency of milk production (kg ecm/kg dmi) was unaffected by the 3 levels of lso. ruminal nh3 concentration was quadratically affected by lso supplementation; decreasing only at the highest level of lso supplementation. the amount (g/d) of n excreted in feces and urine decreased linearly and quadratically, respectively, as the amount of lso increased in the diet, mainly because of the reduction in n intake. efficiency of dietary n used for milk n secretion increased linearly with increasing lso supplementation in the diet. we conclude that supplementing corn silage-based diets with 2 or 3% of lso can reduce enteric ch4 emissions up by to 20% without impairing animal productivity (i.e., ecm yield and feed efficiency). the study aimed to highlight the utilization of gelled emulsion (ge) systems containing peanut and linseed oils to replace beef fat partially or completely in emulsified sausages. total fat content was reduced by up to 40% and energy content was lowered by up to 27% in reformulated products. saturated fatty acids and cholesterol were successfully decreased while noticeable increments were provided in mono and poly-unsaturated fatty acids in sausages containing ge. moreover, the reformulation procedure presented a good potential for increasing n-3 content, while lowering atherogenicity index, thrombogenicity index, and n-6/n-3 ratios. although the incorporated ge resulted in color and texture alterations, it was effective to improve the technological attributes in terms of emulsion stability and cooking behaviors. in ge added samples, oxidative stability of final products decreased; however sensory features were acceptable. overall results pointed out that ge systems could be successfully conveyed to emulsified sausage formulations to ensure a healthier lipid profile with good technological and sensory quality. this contribution focuses on the development of flax and flax/basalt hybrid reinforced composites based on epoxidized linseed oil (elo) resin, exploiting the feasibility of different ratios of glutaric anhydride (ga) to maleinized linseed oil (mlo) in the hardener system (50:0, 40:10 and 30:20 wt.%) to provide crosslinked thermosets with balanced properties. the hybrid laminates have been manufactured by resin transfer molding (rtm) and subjected to dynamic-mechanical (dma) and thermal gravimetry (tga) analysis. the presence of glutaric anhydride (ga) resulted in hard and relatively brittle flax and flax/basalt laminates, whose loss moduli decreased as the number of basalt plies diminished. furthermore, the increase in mlo content in the ga:mlo hardener system shifted the glass transition temperatures (tg) from 70 °c to 59 and 56 °c, which is representative of a decrease in brittleness of the crosslinked resin. all samples exhibited two stages of their decomposition process irrespective of the mlo content. the latter influenced the residual mass content that increased with the increase of the mlo wt.% from 10 to 30 wt.%, with shifts of the final degradation temperatures from 410 °c to 425 °c and 445 °c, respectively. this research work reports the potential of maleinized linseed oil (mlo) as biobased compatibilizer in polylactide (pla) and a thermoplastic elastomer, namely, polystyrene-b-(ethylene-ran-butylene)-b-styrene (sebs) blends (pla/sebs), with improved impact strength for the packaging industry. the effects of mlo are compared with a conventional polystyrene-b-poly(ethylene-ran-butylene)-b-polystyrene-graft-maleic anhydride terpolymer (sebs-g-ma) since it is widely used in these blends. uncompatibilized and compatibilized pla/sebs blends can be manufactured by extrusion and then shaped into standard samples for further characterization by mechanical, thermal, morphological, dynamical-mechanical, wetting and colour standard tests. the obtained results indicate that the uncompatibilized pla/sebs blend containing 20 wt.% sebs gives improved toughness (4.8 kj/m2) compared to neat pla (1.3 kj/m2). nevertheless, the same blend compatibilized with mlo leads to an increase in impact strength up to 6.1 kj/m2, thus giving evidence of the potential of mlo to compete with other petroleum-derived compatibilizers to obtain tough pla formulations. mlo also provides increased ductile properties, since neat pla is a brittle polymer with an elongation at break of 7.4%, while its blend with 20 wt.% sebs and mlo as compatibilizer offers an elongation at break of 50.2%, much higher than that provided by typical sebs-g-ma compatibilizer (10.1%). mlo provides a slight decrease (about 3 °c lower) in the glass transition temperature (t) of the pla-rich phase, thus showing some plasticization effects. although mlo addition leads to some yellowing due to its intrinsic yellow colour, this can contribute to serving as a uv light barrier with interesting applications in the packaging industry. therefore, mlo represents a cost-effective and sustainable solution to the use of conventional petroleum-derived compatibilizers. raw clay is used nowadays in construction as a component of mortars and plasters and as a binder in composites based on straw or shives. it is a material with good sorption properties and vapor permeability, but it is susceptible to shrinkage, is not resistant to water, and also is characterized by low mechanical strength, which makes it impossible to be used, for example, in external plasters. various additives and admixtures are used to improve selected properties of clay mortars. the article presents the research results and assessment of the effect of glauconite clay mortar modification with an admixture of linseed oil varnish on selected properties. admixtures in the amounts of 1%, 2%, and 3% in relation to clay weight were used. flexural and compressive strength, water resistance, shrinkage, drying capacity, density, and porosity of mortar, were tested. the admixture of linseed oil varnish in the amounts used in the investigation had a positive effect on some of the tested properties; regardless of the quantity of the admixture, the modified mortars had better parameters concerning flexural strength, shrinkage reduction, and water resistance than the reference mortar, without admixture. the subject of this work was to characterize the catalytic course of the linseed oil silylation reaction with vinyltrimethoxysilane (vtmos), carried out under elevated pressure and temperature conditions, and an explanation of the reasons for rapid gelation of the reaction product. to explain and describe the process, analytical methods were used, i.e., 1h and 13c nmr (nuclear magnetic resonance), gc-fid (gas chromatography coupled with flame ionisation detection), and gpc (gel permeation chromatography). reaction products were monitored after 3, 6 and 12 h. the molar mass of the vtmos-modified oil in only 3 h was comparable with the molar mass of the product obtained by conventional polymerisation. an increase in the reaction time resulted in further transformations resulting from the hydrolysis and condensation reactions taking place. in contrast to reactivity of soybean oil, the silanisation of linseed oil occurred much faster and without the need for cross-linking catalysts. the reason for the high reactivity of linseed oil to vtmos and rapid gelation of the resulting product was primarily the amount of double bonds present in linseed oil and their high availability, in particular the double bond in the acid linolenic acid located at the c16 carbon. the ability of hemp (cannabis sativa l.) inflorescence extract to counteract lipid oxidation was studied in stripped linseed oil. the ethanolic extract was characterized in terms of terpenes (6.00 mg/ml), cannabidiol (4.99% w/w), phenolic compounds (1.80 mg gallic acid equivalents (gae)/ml), antiradical, and metal ion-chelating activities (50% effective concentration (ec50) of 2.47 mg/ml and 0.39 mg/ml, respectively). the stripped linseed oil, used as control (co), was mixed with hemp extract (ho) or α-tocopherol (eo) at a ratio of 0.6% (w/w) and stored for 7 days in darkness at 40 °c. hemp extract reduced the oxidation and lipolysis processes. at the end of the storage, ho showed a significantly higher level of α-linolenic acid (ala; 26.64 g/100 g), lower peroxide value (pv) (21.19 meq o2/kg oil), and lower hexanal content (7.67 mmol/kg oil) than those found in the control. in contrast, eo showed a marked lipolysis (the free fatty acids increased by 42.57%) and a noticeable oxidation, since the ala content decreased by 2.10% and a pv of 50 meq o2/kg oil was observed. this study demonstrates that hemp inflorescences can be used as a source of natural antioxidants in vegetable oils and lipid products to retard their oxidation, especially those characterized by a high degree of unsaturation. calcium salts of long-chain fatty acids (csfa) from linseed oil have the potential to reduce methane (ch4) production from ruminants; however, there is little information on the effect of supplementary csfa on rumen microbiome as well as ch4 production. the aim of the present study was to evaluate the effects of supplementary csfa on ruminal fermentation, digestibility, ch4 production, and rumen microbiome in vitro. we compared five treatments: three csfa concentrations-0% (con), 2.25% (fal) and 4.50% (fah) on a dry matter (dm) basis-15 mm of fumarate (fum), and 20 mg/kg dm of monensin (mon). the results showed that the proportions of propionate in fal, fah, fum, and mon were increased, compared with con (p < 0.05). although dm and neutral detergent fiber expressed exclusive of residual ash (ndfom) digestibility decreased in fal and fah compared to those in con (p < 0.05), dm digestibility-adjusted ch4 production in fal and fah was reduced by 38.2% and 63.0%, respectively, compared with that in con (p < 0.05). the genera ruminobacter, succinivibrio, succiniclasticum, streptococcus, selenomonas.1, and megasphaera, which are related to propionate production, were increased (p < 0.05), while methanobrevibacter and protozoa counts, which are associated with ch4 production, were decreased in fah, compared with con (p < 0.05). the results suggested that the inclusion of csfa significantly changed the rumen microbiome, leading to the acceleration of propionate production and the reduction of ch4 production. in conclusion, although further in vivo study is needed to evaluate the reduction effect on rumen ch4 production, csfa may be a promising candidate for reduction of ch4 emission from ruminants. health benefits have been associated with the consumption of omega-3 polyunsaturated fatty acids (pufa). linseed oil is rich in long chain omega-3 pufa, but can generate toxic compounds due to its high susceptibility to oxidation. the nature of the emulsifier can affect both lipolysis and oxidation during digestion since these phenomena occur at the oil-water interface. the objective of this study was to compare the effect of low-molecular weight surfactants (cetyltrimethylammonium bromide (ctab), citrem), protein (sodium caseinate, fish gelatin) and polysaccharides (gum arabic, modified starch) on the structure of linseed oil emulsions, lipolysis and formation of reactive oxidation species during in vitro digestion. the emulsion stabilized with citrem underwent extensive coalescence in the gastric phase, which strongly decreased the extent of lipid digestion and reduced the formation of oxidation markers relative to other emulsions. emulsions stabilized by proteins and modified starch showed aggregation with partial coalescence in the gastric phase, but protein-stabilized emulsions showed better resistance to oxidation. this study shows that emulsifier properties affect the susceptibility of the emulsion to aggregation and coalescence in the gastrointestinal environment, and strongly influence the extent of lipid digestion and the formation of reactive oxidation products. these findings point out the importance of the choice of the emulsifier to control the lipid digestibility and the protection of sensible lipids thus promoting optimal nutritional properties in omega-3-enriched foods. poly(lactic acid) (pla) was plasticized with maleinized linseed oil (mlo) and further reinforced with sheep wool fibers recovered from the dairy industry. the wool fibers were firstly functionalized with 1 and 2.5 phr of tris(2-methoxyethoxy)(vinyl) (tvs) silane coupling agent and were further used in 1, 5, and 10 phr to reinforce the pla/mlo matrix. then, the composite materials were processed by extrusion, followed by injection-molding processes. the mechanical, thermal, microstructural, and surface properties were assessed. while the addition of untreated wool fibers to the plasticized pla/mlo matrix caused a general decrease in the mechanical properties, the tvs treatment was able to slightly compensate for such mechanical losses. additionally, a shift in cold crystallization and a decrease in the degree of crystallization were observed due to the fiber silane modification. the microstructural analysis confirmed enhanced interaction between silane-modified fibers and the polymeric matrix. the inclusion of the fiber into the pla/mlo matrix made the obtained material more hydrophobic, while the yellowish color of the material increased with the fiber content. the aim of the present study was to evaluate the effects of adding dietary rumen-protected microencapsulated supplements into the ruminal fluid on the milk fat compositions of dairy cows. these supplements comprised linseed oil, vitamin e, rosemary extract, and hydrogenated palm oil (mo; microtinic® omega, vetagro s.p.a, reggio emilia, italy). for in vitro ruminal fermentation, holstein-friesian dairy cows each equipped with a rumen cannula were used to collect ruminal fluid. different amounts (0%, 1%, 2%, 3%, 4%, and 5%) of mo were added to the diets to collect ruminal fluids. for the in vivo study, 36 holstein-friesian dairy cows grouped by milk yield (32.1 ± 6.05 kg/d/head), days in milk (124 ± 84 d), and parity (2 ± 1.35) were randomly and evenly assigned to 0.7% linseed oil (lo; as dry matter (dm) basis) and 2% mo (as dm basis) groups. these two groups were fed only a basal diet (total mixed ration (tmr), silage, and concentrate for 4 weeks) (period 1). they were then fed with the basal diet supplemented with oil (0.7 lo and 2% mo of dm) for 4 weeks (period 2). in the in vitro experiment, the total gas production was found to be numerically decreased in the group supplemented with 3% mo at 48 h post in vitro fermentation. a reduction of total gas production (at 48 h) and increase in ammonia concentration (24 h) were also observed in the group supplemented with 4% to 5% mo (p < 0.05). there were no differences in the in vitro fermentation results, including ph, volatile fatty acids, or ch4 among groups supplemented with 0%, 1%, and 2% mo. the results of the in vitro study suggest that 2% mo is an optimal dosage of mo supplementation in cows' diets. in the in vivo experiment, the mo supplement more significantly (p < 0.01) increased the yield of total w3 fatty acids than lo (9.24 vs. 17.77 mg/100 g milk). as a result, the ratio of total omega-6 to omega-3 fatty acids was decreased (p < 0.001) in the mo group compared to that in the lo group (6.99 vs. 3.48). however, the milk yield and other milk compositions, except for milk urea nitrogen, were similar between the two groups (p > 0.05). collectively, these results suggest that the dietary supplementation of 2% mo is beneficial for increasing omega-3 fatty acids without any negative effects on the milk yield of dairy cows. effects of linseed oil (lo) supplementation on the fat content and fatty acid profile of breast meat, and the expression of three genes in the liver, breast muscle and fat tissues of commercial 154-day-old hybrid male turkeys were investigated. the animals in the control group were fed a commercially available feed and received no lo supplementation (n=70), whereas animals in the lo group (n=70) were fed the same basic diet supplemented with lo (day 15-21: 0.5%; day 22-112: 1%). the effect of dietary lo supplementation on fatty acid composition of breast muscle was examined by gas chromatography, and the expression of fatty acid desaturase 2 (fads2), peroxisome proliferator activated receptor gamma (pparγ), and insulin-like growth factor 1 (igf1) genes was analysed by means of quantitative reverse transcription pcr. the lo supplementation affected the fatty acid composition of breast muscle. hepatic fads2 levels were considerably lower (p<0.001), while adipose tissue expression was higher (p<0.05) in the control compared to the lo group. the pparγ expression was lower (p<0.05), whereas igf1 was higher (p<0.05) in the fat of control animals. there were no significant (p>0.05) differences in fads2, pparγ and igf1 gene expressions of breast muscle; however, omega-6/omega-3 ratio of breast muscle substantially decreased (p<0.001) in the lo group compared to control. fatty acid composition of breast meat was positively influenced by lo supplementation without deterioration of fattening parameters. remarkably, increased fads2 expression in the liver of lo supplemented animals was associated with a significantly decreased omega-6/omega-3 ratio, providing a potentially healthier meat product for human consumption. increased pparγ expression in fat tissue of the lo group was not associated with fat content of muscle, whereas a decreased igf1 expression in fat tissue was associated with a trend of decreasing fat content in muscle of the experimental lo group. beeswax-based organogels were formulated with linseed oil and curcumin according to a statistical design to increase the oxidative stability of spreadable meat products (pâté) where these organogels (ogcur) were incorporated as fat substitutes. the organogels obtained under optimal conditions (9.12% beeswax, 0.54% curcumin) showed a mechanical strength similar to pork backfat determined by back extrusion and high oil binding capacity (obc; over 90%). the incorporation of curcumin at this concentration did not lead to any change in the arrangement of the crystal network, obc, and mechanical, thermal, or rheological properties of the organogels. beeswax organogels with and without curcumin, with a β' orthorhombic subcell structure, showed a predominant elastic behavior and a melting event wider and shifted to lower temperatures than pure beeswax, suggesting a plasticizer effect of the oil in the wax crystals. the oxidative stability of the organogels under accelerated oxidation conditions increased due to the incorporation of curcumin. a decrease in the curcumin content was found from day 4 at 60 °c, together with a significantly lower formation of both peroxides and malonaldehyde. when pork backfat was partially or totally replaced by ogcur in pâtés, a noticeable protective effect of curcumin against lipid oxidation was found during chilled storage. the limited production of fish oil (fo) cannot meet the increasing demand of the aqua-feed industry. accordingly, the replacement of fo with sustainable alternatives such as vegetable oils has become imperative. linseed oil (lo), which has an abundance of α-linolenic acid (ala), is a promising fo alternative. nevertheless, high dietary lo inclusion generally causes abnormal hepatic lipid accumulation and growth retardation across carnivorous marine fish species. previous studies have attempted to elucidate the mechanisms with regard to gene and protein levels; however, little is known about posttranslational modifications. in the present study, itraq phosphoproteomics was conducted to investigate the metabolic and physiological responses of the yellow drum (nibea albiflora) fed an lo-rich or fo-rich diet. identification and analyses of differentially abundant phosphorylated proteins in hepatic tissue were conducted with parallel reaction monitoring (prm) verification in reference to the genome sequences of the yellow drum. among the specific peptides that passed the threshold, 203 hyperphosphorylated and 182 hypophosphorylated peptides were identified. several functional categories and pathways were associated with these proteins that were mostly related to lipid, amino acid, and carbohydrate metabolism. in particular, the amp-activated protein kinase (ampk) pathway, mechanistic target of rapamycin (mtor) pathway, and citrate (tca) cycle are discussed. statement of significance: dietary lo caused abnormal hepatic lipid accumulation and retarded growth performance. the abnormal hepatic lipid accumulation in the fish fed the lo-rich diet might have arisen from ampk signaling pathway-mediated de novo synthesis of fatty acids and the synthesis of phospholipids. an mtor signaling pathway-mediated reduction in ribosome protein synthesis could have been a factor in the growth retardation. changes in ups and autophagy proteins phosphorylation and aminotransferase activity levels were related to the flux of amino acids into the tca cycle for atp production. the results of prm were highly correlated with the phosphoproteomics results. these findings will contribute to a better understanding of the mechanisms of abnormal hepatic lipid accumulation and retarded growth performance in carnivorous marine fish fed linseed oil-rich diets. bio-based thermosetting resins were synthesized from a ternary composition: humins; epoxidized linseed oil (elo); and an industrial hardener, capcure3-800 (cap). humins are in a focused attention in the last years, as biorefinery by-product, therefore its valorization through materials design is very important. here we present a structural study of terpolymerization of humins/ elo/cap. the reactivity of these systems was highlighted by in situ ft-ir and 1h and 13c nmr. the integration of humins in thermosetting resins gives alternatives to new feedstocks for future bio-based materials. biopolymer films based on chitosan/potato protein/linseed oil/zno nps were developed to maintain the storage quality of raw meat. results indicated that the incorporation of zno nps could effectively improve the transparency and tensile strength of the films, while addition of linseed oil could make the composite film maintain good elastic property. films blended with chitosan/potato protein/linseed oil/zno nps (fcpzl) exhibited an excellent moisture barrier capability. sem showed that zno nps could harmoniously exist in various polymers matrix. ftir analysis demonstrated that different components were bound together by intramolecular and intermolecular interactions, among which hydrogen bonds were the main force. raw meat samples were wrapped with different films to evaluate the preservative effect during 4 °c storage. results indicated that fcpzl possessed best protective effect of raw meat with excellent acceptable sensory properties during 7 days storage, which could reduce the speed of increasing ph and total bacterial counts. the meat quality of chicken is an important factor affecting the consumer's health. it was hypothesized that n-3 polyunsaturated fatty acid (n-3 pufa) could be effectively deposited in chicken, by incorporating antioxidation of soybean isoflavone (si), which led to improved quality of chicken meat for good health of human beings. effects of partial or complete dietary substitution of lard (la) with linseed oil (lo), with or without si on growth performance, biochemical indicators, meat quality, fatty acid profiles, lipid-related health indicators and gene expression of breast muscle were examined in chickens. a total of 900 males were fed a corn-soybean meal diet supplemented with 4% la, 2% la + 2% lo and 4% lo and the latter two including 30 mg si/kg (2% la + 2% lo + si and 4% lo + si) from 29 to 66 days of age; each of the five dietary treatments included six replicates of 30 birds. compared with the 4% la diet, dietary 4% lo significantly increased the feed efficiency and had no negative effect on objective indices related to meat quality; lo significantly decreased plasma triglycerides and total cholesterol (tch); abdominal fat percentage was significantly decreased in birds fed the 4% lo and 4% lo + si diets. chickens with lo diets resulted in higher contents of α-linolenic acid (c18:3n-3), epa (c20:5n-3) and total n-3 pufa, together with a lower content of palmitic acid (c16:0), lignoceric acid (c24:0), saturated fatty acids and n-6:n-3 ratio in breast muscle compared to 4% la diet (p < 0.05); they also significantly decreased atherogenic index, thrombogenic index and increased the hypocholesterolemic to hypercholesterolemic ratio. adding si to the lo diets enhanced the contents of epa and dha (c22:6n-3), plasma total superoxide dismutase, reduced glutathione (gsh)/oxidized glutathione and muscle gsh content, while decreased plasma total triglyceride and tch and malondialdehyde content in plasma and breast muscle compared to its absence (p < 0.05). expression in breast muscle of fatty acid desaturase 1 (fads1), fads2, elongase 2 (elovl2) and elovl5 genes were significantly higher with the lo diets including si than with the 4% la diet. significant interactions existed between lo level and inclusion of si on epa and tch contents. these findings indicate that diet supplemented with lo combined with si is an effective alternative when optimizing the nutritional value of chicken meat for human consumers. this study was proposed to examine the effects of pine needles powder (pinus brutia) supplementation on growth performance, breast meat composition, and antioxidant status in broilers fed linseed oil-based diets. for this purpose, a total of 210, ross-308 1-day-old male broiler chicks were allocated to 5 experimental groups each containing 42 birds. broilers were fed a linseed oil-based basal diet supplemented with 0% (control), 0.25% (p1), 0.50% (p2), 0.75% (p3), and 1% (p4) pine needles powder. during the 42-d feeding period, no significant differences were observed between experimental groups for body weight gain, feed intake, and feed conversion ratio; however, carcass yield was increased linearly with pine needles powder supplementation. no marked changes in the breast meat chemical composition were observed among experimental groups. supplemental pine needles powder linearly decreased the malondialdehyde concentration in breast meat and liver tissues; however, 2,2-diphenyl-1-picrylhydrazyl radical scavenging activity of breast meat samples remained unaffected. no significant variation was observed among experimental groups for superoxide dismutase enzyme activity in blood erythrocyte lysates, but blood serum total oxidation status tended to decrease with pine needles powder supplementation. in conclusion, results suggested that pine needles powder supplementation to broiler diets could be a viable option to improve the animal antioxidant status and meat oxidative stability; however, supplementation of pinus brutia needles powder up to 1% into broiler diets was not sufficient to efficiently curb the fat-induced oxidation in meat. further investigation is needed to determine the full antioxidant potential of pine needles powder supplementation in poultry by comparing different pine species, evaluating the bioavailability of their active compounds and determining most effective dietary concentration for broiler meat production without any adverse effects. in this study, the antimicrobial agents of mono(hydroxyethoxyethyl)phthalate (m(heep)2) with different metal of m = zn, mn, pb, and ca were synthesized from diethylene glycol (deg), phthalic anhydride (pa), and divalent metal acetates including calcium acetate, zinc acetate, manganese acetate, and lead acetate, respectively. the waterborne urethane oil (wuo) dispersions synthesized from linseed oil, diisocyanates (hexamethylene diisocyanate (hdi) and isophorone diisocyanate (ipdi)), dimethylolpropionic acid at nco/oh molars of 0.9, by acetone processing method were described as in our previous report. the m(heep)2 antimicrobial agents as well as the commercial nanosilver powder were added into wuo dispersions as the antimicrobial coatings. the effects of various antimicrobial agents and dosages (0.0, 0.2, 0.6, 0.8, 1.0, 2.0, and 4.0 phr) on antimicrobial activity of wuo films against gram-negative bacterium of escherichia coli, gram-positive bacterium of staphylococcus aureus, brown-rot fungus of gloeophyllum trabeum, and white-rot fungus of lenzites betulina were assessed. in addition, the film properties of the best antimicrobial wuo coatings were also examined. the results showed that the antimicrobial agents of mono(hydroxyethoxyethyl) phthalate m(heep)2 (m = zn, mn, pb, and ca) powders should certainly be synthesized by ftir, 1h-nmr, 13c-nmr, and energy-dispersive x-ray spectroscopy (eds) identifications and the yields of them were 43-55%. the results also revealed that the wuo film synthesizing with hdi films containing zn(heep)2 of 2.0 phr and pb(heep)2 of 0.4 phr had the best antibacterial activity for e. coli and s. aureus, respectively. the ipdi films containing zn(heep)2 of 1.0 phr had the best antibacterial activity for both e. coli and s. aureus. for antifungal activity, the wuo film synthesizing with hdi films containing pb(heep)2 of 0.8 phr and zn(heep)2 of 2.0 phr as well as ipdi films containing mn(heep)2 of 0.2 phr and zn(heep)2 of 4.0 phr had the best performances against g. trabeum and l. betulina, respectively. comparing with commercial nanoag powder, the zn(heep)2 and pb(heep)2 had a superior antifungal efficiency for g. trabeum and l. betulina, while it had a slightly inferior efficiency in the antibacterial activity for e. coli and s. aureus. on the properties of wuo films, adding metal-containing antimicrobial agents could slightly enhance the thermal stability, but lowered the gloss of all films, however, the tg value increased for hdi film and decreased for ipdi film. in addition to this, they had no significant difference in the film properties including hardness, impact resistance, bending resistance, adhesion, mass retention, and light-fastness between the wuo films with and without adding antimicrobial agents. dairy beverages containing emulsified linseed oil is a suitable vehicle for delivering polyunsaturated fatty acids to consumers. however, these beverages are prone to oxidation. the purpose of this study was to evaluate the effect of adding various concentrations (0, 0.001, 0.01 and 0.1% (w/w)) of green tea extract (gte) to dairy beverages (db) containing linseed oil (2.0%, w/w), in order to inhibit lipid oxidation during storage at high temperature (50 °c) or under fluorescent light exposure. during storage, the concentration of catechin (c), epicatechin (ec) and epicatechin gallate (ecg) were significantly reduced (p ≤ 0.05) and degradation rate was greater when the db were exposed to light (c 35%, ec 74% and ecg 68%) as compared to high temperature (c 34%, ec 45% and ecg 49%). in db without gte, the conjugated dienes (cd) hydroperoxides concentration increased significantly (p ≤ 0.05) from 23 mmol kg-1 fat to 243 mmol kg-1 fat under 6-day-light exposition, and to 83 mmol kg-1 fat under 6-day-heat temperature. the addition of gte significantly increased the antioxidant capacity of db and reduced the formation of cd, propanal and hexanal, induced by light exposure or high temperature. gte at 0.10% completely inhibited cd formation during the storage period and reduced propanal and hexanal concentrations below the threshold. \n",
      "Euclidean Distance : 15.265612602233887\n",
      "\n",
      "\n",
      "Document: antioxidant activity and bio-accessibility of polyphenols in black carrot ( resistance and tolerance of ten carrot cultivars to the hawthorn-carrot aphid, dysaphis crataegi kalt., in poland. nanoencapsulation in low molecular weight chitosan improves in vivo antioxidant potential of black carrot anthocyanin. a multi-omics approach sheds light on the higher yield phenotype and enhanced abiotic stress tolerance in tobacco lines expressing the carrot  soil-aged nano titanium dioxide effects on full-grown carrot: dose and surface-coating dependent improvements on growth and nutrient quality. discoloration investigations of freeze-dried carrot cylinders from physical structure and color-related chemical compositions. an ems mutant library for carrot and genetic analysis of some mutants. insights into long non-coding rna regulation of anthocyanin carrot root pigmentation. oxidative stress and antioxidative activity in leaves and roots of carrot plants induced by  a novel isoallergen dau c 1.0401 in carrot: stability, allergenicity, and comparison with other isoallergens. 'candidatus phytoplasma asteris' subgroups display distinct disease progression dynamics during the carrot growing season. mango and carrot mixed juice: a new matrix for the vehicle of probiotic lactobacilli. entomopathogenic nematodes for control of carrot weevil: efficacy and longevity in muck and mineral soils. complete genome sequence of clostridium botulinum cj0611a1, a type a(b) isolate associated with an international outbreak of botulism from commercial carrot juice. integrating optical imaging tools for rapid and non-invasive characterization of seed quality: tomato ( carrot hairy roots: factories for secondary metabolite production. metabolism of black carrot polyphenols during in vitro fermentation is not affected by cellulose or cell wall association. biofortification of potato and carrot with iodine by applying different soils and irrigation with iodine-containing water. three way anova for emulsion of carotenoids extracted in flaxseed oil from carrot bio-waste. effects of auxin (indole-3-butyric acid) on growth characteristics, lignification, and expression profiles of genes involved in lignin biosynthesis in carrot taproot. black carrot has been attracting increasing thanks to its high bioactive compound content. this study presents the polyphenol bio-accessibility of black carrot and two derived products (black carrot snack (bc snack) and black carrot seasoning (bc seasoning)) after in vitro gastrointestinal digestion and colonic fermentation. additionally, antioxidant activity was measured by 2,2'-azinobis-(3-ethylbenzothiazoline-6-sulphonic acid) diammonium salt (abts), 1,1-diphenyl-2-picryl-hydrazyl (dpph) and oxygen radical absorbance capacity (orac) assays. nine flavonoids and eight anthocyanins were determined by ultra high-performance liquid chromatography high resolution mass spectrometry (uhplc-hrms) analysis, the predominant compounds being the hydroxycinnamic acids 3-o-feruloylquinic acid, 4-o-feruloylquinic acid and chlorogenic acid. the bc snack (108 µmol/g dw) presented the highest total polyphenol content, followed by bc seasoning (53 µmol/g dw) and black carrot (11.4 µmol/g dw). the main polyphenols still bio-accessible after in vitro digestion were the hydroxycinnamic acids, with mean recovery rates of 113 % for black carrot, 69% for bc snack and 81% for bc seasoning. the incubation of black carrot and its derived products with human faecal bacterial resulted in the complete degradation of anthocyanins and in the formation of mainly 3-(4'-hydroxyphenyl)propanoic acid as the major catabolic event. in conclusion, our results suggest that the black carrot matrix impacts significantly affects the bio-accessibility of polyphenols and, therefore, their potential health benefits. damage caused to cultivated carrots by the hawthorn-carrot aphid, dysaphis crataegi kalt. (hemiptera: aphididae) is one of the factors limiting carrot production in poland. planting resistant and tolerant cultivars could reduce yield losses due to the damage caused by this pest. this study was conducted to evaluate the resistance and/or tolerance of 10 carrot genotypes to hawthorn-carrot aphid. their field resistance was determined under field conditions based on five indicators, namely, mean number of alates (migrants) per plant and mean percentage of plants colonized by them, mean seasonal number of aphids per plant, mean number of aphids per plant and mean percentage of infested plants at peak abundance. antibiosis experiments were conducted under laboratory conditions and pre-reproductive, reproductive time, fertility, and demographic parameters, represented by the net reproduction rate (ro), intrinsic rate of increase (rm) and mean generation time (t), were calculated. five cultivars, afro f1, nipomo f1, samba f1, white satin f1, and yellowstone showed field resistance. antibiosis experiments revealed significant differences among the carrot cultivars in the length of the reproductive period, female fecundity in the time equal to the pre-reproduction time, and total progeny of hawthorn-carrot aphid. the intrinsic rate of natural increase (rm) for apterous aphids varied significantly, ranging between 0.181 (nipomo f1) and 0.343 females/female/day (white satin f1). additionally, the estimated net reproductive rate (r0) was the lowest on nipomo f1, and this genotype was determined to be resistant. our results suggest that a very high density of trichomes on the leaf petioles (71.94 trichomes/cm2) could adversely affect the feeding, bionomy, and demographic parameters of hawthorn-carrot aphid on the cultivar nipomo f1. in addition, napa f1 and kongo f1 demonstrated high tolerance. considering all the results collectively, four genotypes, afro f1, kongo f1, napa f1 and nipomo f1, were relatively resistant/tolerant to the hawthorn-carrot aphid. anthocyanins are flavonoids that are potential antioxidant, anti-inflammatory, anti-obesity, and anti-carcinogenic nutraceutical ingredient. however, low chemical stability and low bioavailability limit the use of anthocyanin in food. nanoencapsulation using biopolymers is a recent successful strategy for stabilization of anthocyanins. the study reports development, characterization, and antioxidant activity of black carrot anthocyanin loaded chitosan nanoparticles. ionic gelation technique yielded the anthocyanin loaded chitosan nanoparticles (acnps). the mean hydrodynamic diameter (d) and polydispersity index (pdi) of chitosan nanoparticles (cnps) and acnps were found to be d = 455 nm; pdi = 0.542 and d = 274 nm; pdi = 0.376 respectively. the size distribution was bimodal. the surface topography revealed that the acnps are spherical and display coacervate structure. ftir analysis revealed physicochemical interactions of anthocyanins with chitosan. the loading process could achieve an encapsulation efficiency of 70 %. the flow behavior index (ƞ) of encapsulated acnps samples revealed newtonian and shear thickening characteristics. there was a marginal reduction in the in vitro antioxidant potential of anthocyanins after nanoencapsulation as evidenced from dpph, frap and abts assays. interestingly, the in vivo antioxidant potential of anthocyanins improved following nanoencapsulation as observed in the serum antioxidant assays. the optimized nano-encapsulation process resulted in spherical nanoparticles with appreciable encapsulation efficiency. the nanoencapsulation process improved the in vivo antioxidant activity of anthocyanins indicating enhanced stability and bioavailability. the promising antioxidant activity of the acnps suggests a potential for utilization as a nutraceutical supplement. this article is protected by copyright. all rights reserved. recently, we published a set of tobacco lines expressing the daucus carota (carrot) dclcyb1 gene with accelerated development, increased carotenoid content, photosynthetic efficiency, and yield. because of this development, dclcyb1 expression might be of general interest in crop species as a strategy to accelerate development and increase biomass production under field conditions. however, to follow this path, a better understanding of the molecular basis of this phenotype is essential. here, we combine omics (rnaseq, proteomics, and metabolomics) approaches to advance our understanding of the broader effect of lcyb expression on the tobacco transcriptome and metabolism. upon dclcyb1 expression, the tobacco transcriptome (~2,000 genes), proteome (~700 proteins), and metabolome (26 metabolites) showed a high number of changes in the genes involved in metabolic processes related to cell wall, lipids, glycolysis, and secondary metabolism. gene and protein networks revealed clusters of interacting genes and proteins mainly involved in ribosome and rna metabolism and translation. in addition, abiotic stress-related genes and proteins were mainly upregulated in the transgenic lines. this was well in line with an enhanced stress (high light, salt, and h2o2) tolerance response in all the transgenic lines compared with the wild type. altogether, our results show an extended and coordinated response beyond the chloroplast (nucleus and cytosol) at the transcriptome, proteome, and metabolome levels, supporting enhanced plant growth under normal and stress conditions. this final evidence completes the set of benefits conferred by the expression of the dclcyb1 gene, making it a very promising bioengineering tool to generate super crops. rutile titanium dioxide nanoparticles (ntio2) were weathered in field soil at 0, 100, 200, and 400 mg ti/kg soil for four months. two types of ntio2 with different surface coatings (hydrophilic and hydrophobic), uncoated ntio2 (pristine), and the untreated control were included. thereafter, carrot seeds (daucus carota l.) were sown in those soils and grown in a growth chamber for 115 days until full maturity. a comparison was made between this and our previous unaged study, where carrots were treated in the same way in soil with freshly amended ntio2. the responses of plants depended on the ntio2 surface coating and concentration. the aged hydrophobic and hydrophilic-coated ntio2 induced more positive effects on plant development at 400 and 100 mg ti/kg soil, respectively, compared with control and pristine treatments. taproot and leaf fresh biomass and plant height were improved by up to 64%, 40%, and 40% compared with control, respectively. meanwhile, nutrient elements such as fe in leaves, mg in taproots, and ca, zn, and k in roots were enhanced by up to 66%, 64%, 41, 143% and 46%, respectively. however, the contents of sugar, starch, and some other metal elements in taproots were negatively affected, which may compromise their nutritional quality. taken together, the overall growth of carrots was benefited by the aged ntio2 depending on coating and concentration. the aging process served as a potential sustainable strategy to alleviate the phytotoxicity of unweathered nanoparticles. high carotenoids contents always led to a yellower/redder color of carrots, while a puzzling phenomenon still existed that freeze-dried carrots (fdc) had higher carotenoids contents but a lighter color compared with thermal-dried carrots. it seemed that carotenoid was not the only main factor that affected sample color. hence, the discoloration characteristics of freeze-dried carrots were comprehensively analyzed from physical structure and color-related chemical compositions profiles. outcomes of low field nuclear magnetic resonance and scanning electron microscope showed that sublimation of immobilized water preserved the intact porous structure of fdc, which kept the volume shrinkage less than 30% and led to less accumulations of color-related compositions. besides, results of correlation and pca-x model analysis proved that lutein and caffeic acid mainly affected a* value (r = 0.917) and b* value (r = 0.836) of fdc, respectively. moreover, lipoxygenase indirectly affected sample color by degrading carotenoids, the lutein contents loss for fresh and blanching fdcs were 41.56% and 47.14%, respectively. the discoloration of fdc was significantly affected by both of physical structure and color-related chemical compositions. this article is protected by copyright. all rights reserved. carrot is among the top-ten most economically important vegetable crops in the world. however, the gene pool of cultivated carrot is narrow and the collection and preservation of carrot germplasm are very limited in china. in this study, seeds of carrot inbred line \"17005\" were treated by ethyl methyl sulfone (ems) to construct a mutant library. the conditions of ems mutagenesis on inbred line \"17005\" were optimized, including treatment of seeds at 0.5% ems for 6 h. we obtained a number of mutant lines showing inheritable morphological mutations in cotyledon, growth point, leaf, taproot in the vegetative phase and mutations in bolting time, primary stem height and color, secondary branch, flower and seed in the reproductive phase through m2 and m3. the f2 segregating population from the cross of yellow taproot and purple-red epidermis taproot with wild type showed that these two mutants were controlled by single recessive gene respectively. the inheritable mutants can not only enrich the carrot germplasm, but also provide resources for the function research of carrot gene. carrot (daucus carota l.) is one of the most cultivated vegetable in the world and of great importance in the human diet. its storage organs can accumulate large quantities of anthocyanins, metabolites that confer the purple pigmentation to carrot tissues and whose biosynthesis is well characterized. long non-coding rnas (lncrnas) play critical roles in regulating gene expression of various biological processes in plants. in this study, we used a high throughput stranded rna-seq to identify and analyze the expression profiles of lncrnas in phloem and xylem root samples using two genotypes with a strong difference in anthocyanin production. we discovered and annotated 8484 new genes, including 2095 new protein-coding and 6373 non-coding transcripts. moreover, we identified 639 differentially expressed lncrnas between the phenotypically contrasted genotypes, including certain only detected in a particular tissue. we then established correlations between lncrnas and anthocyanin biosynthesis genes in order to identify a molecular framework for the differential expression of the pathway between genotypes. a specific natural antisense transcript linked to the dcmyb7 key anthocyanin biosynthetic transcription factor suggested how the regulation of this pathway may have evolved between genotypes. the present study examined the effects of candidatus phytoplasma solani infection on antioxidative metabolism in leaves and roots of carrot (daucus carota l.). disease symptoms appeared at the end of june in the form of the chlorosis on some of the leaves, which became intensely red one week later, while the previously healthy leaves from the same branch becme chlorotic. a few days later, all leaves from the infected leaf branch were intensely red. infected plants also had slower growth compared to the healthy ones with fewer leaf branches developed. the roots of infected plants were less developed, seared, or gummy with or without brown-colored root hair. the presence of the pathogen was detected by sequencing the 16s rrna. national center for biotechnology information (ncbi) blast analyses of the obtained sequence revealed 100% identity of tested strain with deposited ca. phytoplasma solani strains from various countries and hosts, all belonging to the \"stolbur\" group (16srxii-a). identity of 99.74% was found when the tested serbian strain (mf503627) was compared with the reference stolbur strain stol11 (af248959). the oxidative damage of membranes in carrot cells was accompanied by a decrease in the content of photosynthetic pigments. furthermore, for the determination of specific scavenging properties of the extracts, in vitro antioxidant assay was performed. in phytoplasma-infected carrot leaves, there was a greater reduction in the level of glutathione content (gsh); however; flavonoids and anthocyanidins seem to be responsible for the accompanied increased antioxidative capacity against hydroxyl radical and hydrogen peroxide. around 25% of food allergic persons in central europe suffer from carrot allergy caused by the major carrot allergen dau c 1. three different isoallergens, dau c 1.01, dau c 1.02 and dau c 1.03 are identified. however, information about the qualitative and quantitative composition of natural (n)dau c 1 is scarce. the new carrot allergen dau c 1.0401 is identified on the mrna and protein level by rt-pcr and mass spectrometry. it displays only around 60% sequence identity to the other known dau c 1 isoallergens. nmr and cd-spectra are typical for a well-folded protein containing both α-helices and β-strands. it showed a poor refolding capacity after incubation at 95 °c. ige-binding is impaired in immunoblots, whereas in inhibition assays ige binding to soluble dau c 1.0401 is detected and it clearly provoked a response in mediator release assays. dau c 1.0401 is a new isoallergen which contributes to the allergenicity of carrots. the absence of immunoreactivity in immobilized assays indicates that ige binding is impaired when the protein is blotted on a solid phase. altogether, the results point out that its allergenicity can be reduced upon carrot processing. aster yellows phytoplasma (ayp; 'candidatus phytoplasma asteris') is an obligate bacterial pathogen that is the causative agent of multiple diseases in herbaceous plants. while this phytoplasma has been examined in depth for its disease characteristics, knowledge about the spatial and temporal dynamics of pathogen spread is lacking. the phytoplasma is found in plant's phloem and is vectored by leafhoppers (cicadellidae: hemiptera), including the aster leafhopper, macrosteles quadrilineatus forbes. the aster leafhopper is a migratory insect pest that overwinters in the southern united states, and historical data suggest these insects migrate from southern overwintering locations to northern latitudes annually, transmitting and driving phytoplasma infection rates as they migrate. a more in-depth understanding of the spatial, temporal and genetic determinants of aster yellows disease progress will lead to better integrated pest management strategies for aster yellows disease control. carrot, daucus carota l., plots were established at two planting densities in central wisconsin and monitored during the 2018 growing season for aster yellows disease progression. symptomatic carrots were sampled and assayed for the presence of the aster yellows phytoplasma. aster yellows disease progression was determined to be significantly associated with calendar date, crop density, location within the field, and phytoplasma subgroup. this study evaluated the ph, acidity, soluble solids, color, dietary fiber, sensory acceptance and the viability of lactobacillus rhamnosus, lactobacillus plantarum and lactobacillus acidophilus in mango and carrot mixed juices. in addition, this study verified the resistance of l. plantarum that presented greater viability to the gastrointestinal tract simulated in vitro. three formulations were elaborated (varying the pulps concentration) and the products were stored at 8 °c for 35 days. no difference was found in the total soluble solids and color of the products during storage time at 8 °c. a reduction in ph and an increase in acidity were observed in all samples during storage, probably due to the fermentative action of probiotics, which negatively influenced acceptance after 35 days of storage. on the other hand, juices with a higher concentration of mango pulp were more accepted and may be a strategy to improve the acceptance of fermented juices. microorganisms showed greater viability in juices that had higher amount of carrot pulp, probably due to the higher fiber content in these samples. during the 35-day shelf life, all juices with l. plantarum maintained counts above 7 log cfu ml-1 after gastrointestinal conditions simulation. therefore, mango and carrot mixed juice showed to be as a good vehicle for probiotic bacteria and meets the needs of consumers looking for functional, healthy, non-dairy and low-sugar foods. carrot weevil is an important pest throughout carrot-producing regions in canada. pesticides to control carrot weevil adults require application when the majority of adults have emerged and often this occurs after oviposition has already commenced and damage will be realized. one alternative to conventional pesticides are entomopathogenic nematodes. we studied four commercially available entomopathogenic nematode products (steinernema feltiae, s. carpocapsae, s. kraussei and heterorhabditis bacteriophora) for efficacy against carrot weevil in nova scotia and ontario carrot fields in 2017 and 2018. the longevity and infectivity of the products in fine sandy loam soil (ontario) and sandy loam soil (nova scotia) were evaluated using galleria mellonella larvae. in nova scotia soils, only s. kraussei when applied twice, showed some efficacy to reduce damage from carrot weevil in 2017. in ontario, an early application of h. bacteriophora and s. feltiae significantly reduced the percentage of carrots with weevil damage in 2018. longevity and infectivity of s. carpocapsae and s. feltiae (against g. mellonella) was obtained up to 6 weeks post application in nova scotia in 2017. similarly, s. feltiae showed infectivity up to 9 weeks post application in ontario and nova scotia in 2018. entomopathogenic products showed an ability to survive and remain infective for up to 9 weeks in soils without irrigation. timing of application to effect control of carrot weevil requires further study. the influence of soil moisture on the longevity and infectivity of these products is discussed. we report the complete genome (3.9-mb chromosome, 5.9-kb plasmid) of clostridium botulinum cj0611a1, a type a(b) strain isolated from carrot juice distributed in canada and linked to an international 2006 foodborne botulism outbreak. this strain encodes a full-length bont/a1 gene and a truncated bont/b gene. light-based methods are being further developed to meet the growing demands for food in the agricultural industry. optical imaging is a rapid, non-destructive, and accurate technology that can produce consistent measurements of product quality compared to conventional techniques. in this research, a novel approach for seed quality prediction is presented. in the proposed approach two advanced optical imaging techniques based on chlorophyll fluorescence and chemometric-based multispectral imaging were employed. the chemometrics encompassed principal component analysis (pca) and quadratic discrimination analysis (qda). among plants that are relevant as both crops and scientific models, tomato, and carrot were selected for the experiment. we compared the optical imaging techniques to the traditional analytical methods used for quality characterization of commercial seedlots. results showed that chlorophyll fluorescence-based technology is feasible to discriminate cultivars and to identify seedlots with lower physiological potential. the exploratory analysis of multispectral imaging data using a non-supervised approach (two-component pca) allowed the characterization of differences between carrot cultivars, but not for tomato cultivars. a random forest (rf) classifier based on gini importance was applied to multispectral data and it revealed the most meaningful bandwidths from 19 wavelengths for seed quality characterization. in order to validate the rf model, we selected the five most important wavelengths to be applied in a qda-based model, and the model reached high accuracy to classify lots with high-and low-vigor seeds, with a correct classification from 86 to 95% in tomato and from 88 to 97% in carrot for validation set. further analysis showed that low quality seeds resulted in seedlings with altered photosynthetic capacity and chlorophyll content. in conclusion, both chlorophyll fluorescence and chemometrics-based multispectral imaging can be applied as reliable proxies of the physiological potential in tomato and carrot seeds. from the practical point of view, such techniques/methodologies can be potentially used for screening low quality seeds in food and agricultural industries. fruit and vegetable polyphenols are associated with health benefits, and those not absorbed could be fermented by the gastro-intestinal tract microbiota. many fermentation studies focus on \"pure\" polyphenols, rather than those associated with plant cell walls (pcw). black carrots (blkc), are an ideal model plant food as their polyphenols bind to pcw with minimal release after gastro-intestinal digestion. blkc were fractionated into three components-supernatant, pellet after centrifugation, and whole puree. bacterial cellulose (bcell) was soaked in supernatant (bcell&s) as a model substrate. all substrates were fermented in vitro with a pig faecal inoculum. gas kinetics, short chain fatty acids, and ammonium production, and changes in anthocyanins and phenolic acids were compared. this study showed that metabolism of blkc polyphenols during in vitro fermentation was not affected by cellulose/cell wall association. in addition, bcell&s is an appropriate model to represent blkc fermentation, suggesting the potential to examine fermentability of pcw-associated polyphenols in other fruits/vegetables. accumulation of iodine by potato (solanum tuberosum l.) and carrot (daucus carota l. var. sativus) plants cultivated on different soils (sand, sandy silt, and silt) using irrigation water containing iodine at concentrations of 0.1 and 0.5 mg/l was investigated. in the edible organs of potato and carrot control plants grown on sand, sandy silt, and silt soils, the iodine concentrations were 0.15, 0.17, and 0.20 mg/kg (potato) and 0.012, 0.012, and 0.013 mg/kg (carrot); after the treatment by applying 0.5 mg/l iodine dosage, the iodine concentrations were 0.21, 0.19, 0.27 mg/kg (potato) and 3.5, 3.7, 3.0 mg/kg (carrot), respectively. although the iodine treatment had no significant effect on the biomass production of these plants, in potato tubers, it resulted in higher fe and lower mg and p concentrations, whereas no similar trend was observable in carrot roots. the accumulation of mn, cu, zn, and b in the edible part of both plants was not influenced by the iodine treatment. the soil properties did not have a significant impact on biomass production under the same environmental conditions. the concentration and the distribution of iodine in both plants were slightly modified by the growing medium; however, the photosynthetic efficiency and the chlorophyll content index of potato plants cultivated in silt soil increased significantly. potato plant was not suitable for biofortification with iodine, while considering the iodine concentration and the moisture content of carrot roots, it can be calculated that consuming 100 g fresh carrot would cover about 38% of the daily iodine intake requirement for an average adult person. the juice expelled from carrot, a globally produced root vegetable, leavesbehind carrot pomace (a bio- and horticultural waste) which is potentially rich source of micro-nutrients and carotenoids.however, it is discarded as waste or used as animal feed. it holds potential to be channelized to food chain by a couple of technological interventions. in this regard, present work was aimed at preparing stable emulsion based delivery system for 'green' carotenoids extracted from carrot-pomace in flaxseed oil (a green solvent), and at maximizing the amount of core material so that the resultant emulsion can potentially be used as a source of both carotenoids and omega-3 fatty acid of flaxseed oil origin. the study used natural emulsifier. preparation of oil-in-water emulsion was optimized using 33 factorial experiment by varying levels of extract containing carotenoid (30-40%), whey protein concentrates (wpc-80) and lactose. the optimized emulsion (crem) was selected on the basis of particle size, zeta potential, color values (l*, a*, b*) and viscosity statistically analyzed via three-way anova using proc glm of sas 9.3 (described in detail in this paper); the respective values of these parameters being 120.03 ± 8.20 nm, -16.57 ± 0.49 mv, 75.11 ± 0.04, 9.66 ± 0.32, 50.29 ± 0.62, and 0.124 ± 0.0115 pa.s for crem. crem contained 35% flaxseed oil, 10% wpc-80 and 5% lactose and showed good centrifugal and gravitational stability (15 days). it was analyzed for total carotenoid content, antioxidant activities (abts (2,2-azinobis-(3-ethylbenzthiazoline-6sulfonic acid), dpph (2,2-diphenyl-1-picrylhydrazyl) and frap (ferric reducing antioxidant power assay)) and microstructure. carrot is an important root vegetable crop abundant in bioactive compounds including carotenoids, vitamins, and dietary fibers. carrot intake and its products are gradually growing owing to its high antioxidant activity. auxins are a class of plant hormones that control many processes of plant growth and development. yet, the effects of exogenous application of auxin on lignin biosynthesis and gene expression profiles of lignin-related genes in carrot taproot are still unclear. in order to investigate the effect of exogenous indole-3-butyric acid (iba) on lignin-related gene profiles, lignin accumulation, anatomical structures and morphological characteristics in carrot taproots, carrots were treated with different concentrations of iba (0, 50, 100, and 150 µm). the results showed that iba application significantly improved the growth parameters of carrot. the 100 or 150 µm iba treatment increased the number and area of xylem vessels, whereas transcript levels of lignin-related genes were restricted, resulting in a decline in lignin content in carrot taproots. the results indicate that taproot development and lignin accumulation may be influenced by the auxin levels within carrot plants. \n",
      "Euclidean Distance : 15.353768348693848\n",
      "\n",
      "\n",
      "Document: links between synergistic effects and microbial community characteristics of anaerobic co-digestion of food waste, cattle manure and corn straw. effects of corn silage inclusion level and type of anabolic implant on animal growth performance, apparent total tract digestibility, beef production per hectare, and carcass characteristics of finishing steers. western corn rootworm, plant and microbe interactions: a review and prospects for new management tools. influence of enzyme supplementation in the diets of broiler chickens formulated with different corn hybrids dried at various temperatures. western corn rootworm ( nutrient digestibility of a vegetarian diet with or without the supplementation of feather meal and either corn meal, fermented rye or rye and its effect on fecal quality in dogs. influence of shrub willow buffers strategically integrated in an illinois corn-soybean field on soil health and microbial community composition. corn silage-based diet supplemented with increasing amounts of linseed oil: effects on methane production, rumen fermentation, nutrient digestibility, n utilization, and milk production of dairy cows. effect of stevia, xylitol, and corn syrup in the development of velvet tamarind (dialium indum l.) chewy candy. life cycle analysis of electrofuels: fischer-tropsch fuel production from hydrogen and corn ethanol byproduct co evaluating the efficacy of alfaxalone in corn snakes (pantherophis guttatus). biotechnological production of the european corn borer sex pheromone in the yeast yarrowia lipolytica. detection of acacia honey adulteration with high fructose corn syrup through determination of targeted α‑dicarbonyl compound using ion mobility-mass spectrometry coupled with uhplc-ms/ms. phenolics and related in vitro functional activities of different varieties of fresh waxy corn: a whole grain. location and variety but not phosphate starter fertilization influence the profiles of fatty acids, carotenoids, and tocochromanols in kernels of modern corn ( differential insecticidal properties of spodoptera frugiperda multiple nucleopolyhedrovirus isolates against corn-strain and rice-strain fall armyworm, and genomic analysis of three isolates. assessment of corn starch as substitute for agarose in dna gel electrophoresis. extended investigation of field-evolved resistance of the corn earworm helicoverpa zea(lepidoptera: noctuidae) to bacillus thuringiensis cry1a.105 and cry2ab2 proteins in thesoutheastern united states. widely targeted metabolomics analysis reveals key quality-related metabolites in kernels of sweet corn. genome assembly and population genomic analysis provide insights into the evolution of modern sweet corn. links between synergy and microbial community characteristics in co-digestion of food waste (fw), cattle manure (cm) and corn straw (cs) were investigated. mono-digestion of fw and cs were inhibited by organic acids. co-digestion of fw with cm achieved greater synergistic rates (18.5% and 22.3%) than cm with cs (14.8% and 12.3%). synergy resulted from coupling effects of improving nutrient balance, dilution of toxic compounds, higher buffering capacity, detoxification based on co-metabolism, which ultimately reflected in microbial community functions. although co-digestion of fw with cs exhibited lowest synergistic rates (7.9% and 4.9%), detoxification based on co-metabolism of syntrophic communities of syntrophomonadaceae with hydrogenotrophic methanogens accelerated system recovery. digester with the greatest synergy (65% fw + 35% cm) maintained dominant growth of hydrogenotrophic methanogens (68.9%), highest methanogenic community diversity and relative abundance of methanosarcina (14.6%), which sustained more diverse and switchable methanogenic pathways therefore ensured powerful methanogenic functions and vigorous methanogenic capability. maine-anjou × angus cross-bred steers (n = 156 steers; initial body weight (bw) 366 ± 37.2 kg) were used in a 132 d finishing study conducted at the ruminant nutrition center (rnc) in brookings, sd. steers were blocked by weight (n = 5 bw blocks) and randomly assigned to an implant and dietary treatment of a randomized complete block design with each pen containing seven to eight steers (n = 20 pens). dietary treatments consisted of (1) 15% (cs15) or (2) 30% corn silage (cs30) where corn silage displaced corn grain in the diet. steers received one of two implants (both from zoetis, parsippany, nj) containing equal doses of trenbolone acetate (tba) and estradiol benzoate (eb): (1) synovex plus (non-coated implant; 200 mg tba and 28 mg eb; plus) or (2) synovex one feedlot (coated implant; 200 mg tba and 28 mg eb; one-f). bunks were managed using a slick bunk approach, and all diets contained dry matter (dm) basis 33 mg/kg monensin sodium. all steers were offered ad libitum access to feed, and feeding occurred twice daily in equal portions. there was no interaction between the implant and dietary treatment for any variables measured (p ≥ 0.08). carcass-adjusted basis final bw, average daily gain (adg), and grain to feed (g:f) were increased (p ≤ 0.02) by 2.2%, 6.5%, and 7.2%, respectively, for cs15. observed net energy (ne) and the ratio of observed-to-expected ne for maintenance and gain was not influenced (p ≥ 0.15) by silage inclusion treatment. beef production per hectare was not impacted (p ≥ 0.13) by corn silage inclusion level. fecal output was increased, and digestibility coefficients for dry matter, organic matter, and crude protein were decreased in cs30 (p ≤ 0.03). dressing percent and hot carcass weight (hcw) were greater (p ≤ 0.02) in cs15. implant type did not influence any traits measured (p ≥ 0.14) except for marbling. marbling was decreased for plus (433 vs. 466 ± 17.5; p = 0.02) compared to one-f steers. similar beef produced per hectare of crop land-based upon silage feeding level means producers can feed greater inclusions of corn silage to finishing cattle without impacting carcass quality or beef production; implanting with a coated implant had no detrimental effects to growth performance but increases marbling scores. the western corn rootworm, diabrotica virgifera virgifera leconte, is resistant to four separate classes of traditional insecticides, all bacillius thuringiensis (bt) toxins currently registered for commercial use, crop rotation, innate plant resistance factors, and even double-stranded rna (dsrna) targeting essential genes via environmental rna interference (rnai), which has not been sold commercially to date. clearly, additional tools are needed as management options. in this review, we discuss the state-of-the-art knowledge about biotic factors influencing herbivore success, including host location and recognition, plant defensive traits, plant-microbe interactions, and herbivore-pathogens/predator interactions. we then translate this knowledge into potential new management tools and improved biological control. we evaluated the influence of enzymatic supplementation on the growth performance and cecal microbiota of broilers. a total of 2160 1-day-old male chicks were used in a 3 × 2 × 2 factorial arrangement (three corn hybrids, two drying temperatures -80 and 110 °c, with or without the inclusion of an enzymatic blend (amylase, xylanase, and protease) (20 birds/pen, n = 9). for all performance and digestibility parameters, we observed, in general, isolated effects of the corn hybrids and drying temperature. birds that received the enzymatic blend in the diet showed better weight gain from 1 to 21 days (d) and better digestibility coefficients of nutrients at 42 d. birds fed diets with corn dried at 80 °c showed a better feed conversion ratio from 1 to 42 d. at 21 d of age, enzymatic supplementation had positive effects on jejunum morphology. enzyme supplementation increased the abundance of the phylum tenericutes, class bacilli and mollicutes, reduced clostridia, and increased the abundances of the families lactobacillaceae, anaeroplasmataceae, and o_rf39;f. in conclusion, the addition of amylase, xylanase, and protease led to a better nutrient digestibility, performance, and intestinal morphology. in addition, enzyme supplementation changed the diversity, composition, and predicted function of the cecal microbiota at d 21. western corn rootworm (wcr), or diabrotica virgifera virgifera leconte, became a very serious quarantine maize pest in europe in the mid-1990s. between 1995 and 2010, european countries were involved in international projects to share information and plan common research for integrated pest management (ipm) implementation. since 2011, however, common efforts have declined, and an overview of wcr population spread, density, and research is in serious need of update. therefore, we retained that it was necessary to (1) summarize the research activities carried out in the last 12 years in various countries and the research topics addressed, and analyze how these activities have contributed to ipm for wcr and (2) present the current distribution of wcr in the eu and analyze the current population levels in different european countries, focusing on different management strategies. a review of scientific papers published from 2008 to 2020, in addition to direct interviews with experts in charge of wcr management in a range of european countries, was conducted. over the past 12 years, scientists in europe have continued their research activities to investigate various aspects of wcr management by implementing several approaches to wcr control. a considerable amount of new knowledge has been produced, contributing to the development of pest management strategies applicable in eu farming systems. among the 10 eu countries analyzed, there is no country reporting economic damage on a large scale. thanks to intensive research leading to specific agricultural practices and the eu common agricultural policy, there are crop-rotation-based solutions that can adequately control this pest avoiding insecticide use. cereals with low environmental input like rye and animal by-products which cannot be used for human food like feather meal are receiving growing interest as sustainable feed sources. thus, eight beagle dogs were included in a 4 × 4 latin square design and received a vegetarian basic diet or the same diets supplemented with hydrolyzed feather meal (2.7%) and either 20.1% of corn meal, 60.4% of fermented rye or 20.1% of rye as is basis (moisture content of the diets about 42%). compared to other groups the dry matter (dm) content of feces from dogs fed the basic diet was higher (30.0%, p < 0.05), while dogs fed the basic diet + rye had the lowest dm-content (26.5%, p < 0.05). however, the fecal scores were considered to be within an acceptable range (well-formed and firm). starch digestibility was lower (p < 0.05) for dogs fed the basic diet + corn meal. the dogs showed a high and identical acceptance (scoring of food intake) of the experimental diets. as a comparable quality of feces and a high nutrient digestibility were observed when rye was used in the experimental diets-it can be considered an alternative carbohydrate source in dog foods. soil serves many important ecological functions and is an integral part of our existence as a society. however, concerns for soil health are growing globally, in part due to the negative impacts of agricultural management on soil resources. the production of perennial bioenergy crops on marginal land in row-crop production systems is one solution that could improve land-use efficiency and address the sustainability of cropland management. because the relationship between crop management and the environment is complex, more research is needed to evaluate the potential benefits perennial bioenergy crop production has on soil health, as well as other ecosystem services. in this study, shrub willow buffers were strategically integrated into a corn-soybean cropping system with the main objective of reducing nitrate-n leaching from grain crop production while producing biomass for bioenergy. two buffer systems (defined by landscape positions) were included for comparison, one on marginal land with exposure to nitrate-n leaching from upslope grain (southern plots) and one on fertile soils with less nitrate-n leaching potential (northern plots). evaluation of soil (chemistry, bulk density, microbial community) and shrub willow vegetation properties (fine roots, leaf litter decomposition, and nutrient uptake dynamics), showed that landscape position plays an important role in (1) the dynamics of soil chemical properties, (2) shrub willow's influence and productivity, and (3) the provision of additional ecosystem services such as reductions in nitrous oxide emissions and nitrate-n leaching. in addition, the combination of crop type and landscape position (n-grain, n-willow, s-grain, and s-willow) influenced the species composition of the soil microbial community, resulting in unique and identifiable communities. these results highlight the potential application of shrub willow buffers for ecosystem service provision and support of ecosystem processes; however, understanding the relationship between the microbial community, crop type, and landscape is important for understanding the sustainability of the design. in this study, we assessed the effects of increasing amounts of linseed oil (lso) in corn silage-based diets on enteric ch4 production, rumen fermentation characteristics, protozoal population, nutrient digestibility, n utilization, and milk production. for this purpose, 12 multiparous lactating holstein cows (84 ± 28 d in milk; mean ± sd) fitted with ruminal cannula were used in a replicated 4 × 4 latin square design (35-d period). the cows were fed ad libitum a total mixed ration without supplementation (control) or supplemented [on a dry matter (dm) basis] with lso at 2% (lso2), 3% (lso3) or 4% (lso4). the forage:concentrate ratio was 61:39 (on dm basis) and was similar among the experimental diets. the forage portion consisted of corn silage (58% diet dm) and timothy hay (3% diet dm). the proportions of soybean meal, corn grain and soybean hulls decreased as the amount of lso in the diet increased. daily methane production (g/d) decreased quadratically as the amount of lso increased in the diet. increasing lso dietary supplementation caused a linear decrease in ch4 emissions expressed on either dm intake (dmi) basis (-9, -20, and -28%, for lso2, lso3, and lso4, respectively) or gross energy intake basis (-12, -22, and -31%, for lso2, lso3, and lso4, respectively). at 2 and 3% lso, the decrease in enteric ch4 emissions occurred without negatively affecting dmi or apparent total-tract digestibility of fiber and without changing protozoa numbers. however, these 2 diets caused a shift in volatile fatty acids pattern toward less acetate and more propionate. the effect of the lso4 diet on enteric ch4 emissions was associated with a decrease in dmi, fiber apparent-total-tract digestibility, protozoa numbers (total and genera), and an increase in propionate proportion at the expense of acetate and butyrate proportions. methane emission intensity [g of ch4/kg of energy-corrected milk (ecm)] decreased linearly (up to 28% decrease) with increasing lso level in the diet. milk fat yield decreased linearly (up to 19% decrease) with increasing inclusion of lso in the diet. milk protein yield increased at 2% or 3% lso and decreased to the same level as that of the nonsupplemented diet at 4% lso (quadratic effect). yield of ecm was unchanged by lso2 and lso3 treatments but decreased (-2.8 kg/d) upon supplementation with 4% lso (quadratic effect). efficiency of milk production (kg ecm/kg dmi) was unaffected by the 3 levels of lso. ruminal nh3 concentration was quadratically affected by lso supplementation; decreasing only at the highest level of lso supplementation. the amount (g/d) of n excreted in feces and urine decreased linearly and quadratically, respectively, as the amount of lso increased in the diet, mainly because of the reduction in n intake. efficiency of dietary n used for milk n secretion increased linearly with increasing lso supplementation in the diet. we conclude that supplementing corn silage-based diets with 2 or 3% of lso can reduce enteric ch4 emissions up by to 20% without impairing animal productivity (i.e., ecm yield and feed efficiency). the research aimed to study the effect of stevia, xylitol, and corn syrup on the physical, physicochemical, and sensory properties of velvet tamarind chewy candy (vtcc). the content of sweeteners was optimized using stevia (5.5-6%), xylitol (5.5-6%), and corn syrup (7.5-8.5%) by mixture design (d-optimal) with 3 centerpoints and response surface methodology (rsm). the sweeteners in optimized vtcc consisted of velvet tamarind powder (40.5%), water (40%), stevia (6%), xylitol (6%), and corn syrup (7.5%) which provided the approximation error between prediction and observation values: of color (a* and b*), hardness, adhesiveness, gumminess, bioactive activities, and sensory properties within the range of 0.07-9.69%. the optimized vtcc using stevia and xylitol can reduce the sugar content by up to 60%. the sensory preference scores of vtcc from consumer acceptance were slightly like (6.1-6.9) which indicated that the vtcc with stevia and xylitol can provide satisfaction in all evaluated attributes and can be applied to this concept to create fruit chewy candy using stevia and xylitol. electrofuels from renewable h2 and waste co2 streams are of increasing interest because of their co2 emissions reduction potentials compared to fossil counterparts. this study evaluated the well-to-wheel (wtw) greenhouse gas (ghg) emissions of fischer-tropsch (ft) fuels from various electrolytic h2 pathways and co2 sources, using various process designs (i.e., with and without h2 recycle) and system boundaries. two systems with different boundaries were considered: a stand-alone plant (with co2 from any source) and an integrated plant with corn ethanol production (supplying co2). the ft fuel synthesis process was modeled using aspen plus, which showed that 45% of the carbon in co2 can be fixed in the ft fuel, with a fuel production energy efficiency of 58%. using nuclear or solar/wind electricity, the stand-alone ft fuel production from various plant designs can reduce wtw ghg emissions by 90-108%, relative to petroleum fuels. when integrating the ft fuel production process with corn ethanol production, the wtw ghg emissions of ft fuels are 57-65% lower compared to petroleum counterparts. this study highlights the sensitivity of the carbon intensity of ft fuels to the system boundary selection (i.e., stand-alone vs integrated), which has different implications under various ghg emission credit frameworks. alfaxalone is a popular veterinary anesthetic; however, research on this anesthetic in snakes has been limited to ball pythons, garter snakes and several australian species. the objective was to evaluate the anesthetic effects of alfaxalone in corn snakes (pantherophis guttatus), a popular pet snake. prospective, randomized crossover study. a total of eight corn snakes. in phase i, snakes were subcutaneously administered three doses of alfaxalone (5, 10 and 15 mg kg-1) in the cranial third of the body to determine the most effective dose. in phase ii, a dose of 15 mg kg-1 was administered in the cranial and caudal thirds of the snakes to determine if injection site affected anesthesia duration. heart rate (hr), respiratory rate (fr), righting reflex, escape response, tail pinch, needle prick and tongue flick were monitored at baseline and 5 minute intervals until the snakes fully recovered. duration of anesthesia differed significantly, with higher doses lasting longer than lower doses: 5 mg kg-1 [23.8 ± 4.4 (15-30) minutes]; 10 mg kg-1 [40.6 ± 9.4 (25-55) minutes]; and 15 mg kg-1 [56.9 ± 8.4 (50-70) minutes], mean ± standard deviation (range). the tail pinch reflex was not completely lost in phase 1. there was a significant change in fr over time, but this was not related to dose. hr was not different by time or dose. duration of anesthesia was not different after administration of alfaxalone (15 mg kg-1) in the cranial third versus the caudal third of the body; however, there was a significant decrease in hr and fr at this dose, regardless of injection site. based on these results, alfaxalone (15 mg kg-1) provides adequate anesthesia for brief procedures or intubation; however, additional analgesia is required for painful procedures. the european corn borer (ecb) ostrinia nubilalis is a widespread pest of cereals, particularly maize. mating disruption with the sex pheromone is a potentially attractive method for managing this pest; however, chemical synthesis of pheromones requires expensive starting materials and catalysts and generates hazardous waste. the goal of this study was to develop a biotechnological method for the production of ecb sex pheromone. our approach was to engineer the oleaginous yeast yarrowia lipolytica to produce (z)-11-tetradecenol (z11-14:oh), which can then be chemically acetylated to (z)-11-tetradecenyl acetate (z11-14:oac), the main pheromone component of the z-race of o. nubilalis. first, a c14 platform strain with increased biosynthesis of myristoyl-coa was obtained by introducing a point mutation into the α-subunit of fatty acid synthase, replacing isoleucine 1220 with phenylalanine (fas2pi1220f ). the intracellular accumulation of myristic acid increased 8.4-fold. next, fatty acyl-coa desaturases (fad) and fatty acyl-coa reductases (far) from nine different species of lepidoptera were screened in the c14 platform strain, individually and in combinations. a titer of 29.2±1.6 mg/l z11-14:oh was reached in small-scale cultivation with an optimal combination of a fad (lbo_pptq) from lobesia botrana and far (harfar) from helicoverpa armigera. when the second copies of fad and far genes were introduced, the titer improved 2.1-fold. the native fas1 gene's overexpression led to a further 1.5-fold titer increase, reaching 93.9±11.7 mg/l in small-scale cultivation. when the same engineered strain was cultivated in controlled 1 l bioreactors in fed-batch mode, 188.1±13.4 mg/l of z11-14:oh was obtained. fatty alcohols were extracted from the biomass and chemically acetylated to obtain z11-14:oac. electroantennogram experiments showed that males of the z-race of o. nubilalis were responsive to biologically-derived pheromone blend. behavioral bioassays in a wind tunnel revealed attraction of male o. nubilalis at a level similar to that of the chemically synthesized pheromone used as a control, although full precopulatory behavior was observed less often. the study paves the way for the production of ecb pheromone by fermentation. this article is protected by copyright. all rights reserved. high-value acacia honey is often adulterated with inexpensive high fructose corn syrup (hfcs), due to their similar color and sugar composition. α‑dicarbonyl compounds formed by maillard reaction or caramelization during heat treatment or storage, differ between hfcs and honey due to differences in starting materials and processing methods. in this study, we compared α-dicarbonyl compounds in acacia honey and hfcs by ion mobility-mass spectrometry and multivariate statistical analysis. through α-dicarbonyl compound derivatization with o-phenylenediamine, we screened a marker with 189.1023 m/z and 139.3 å2 collision cross-section that can distinguish hfcs from acacia honey. nuclear magnetic resonance spectra identified this marker compound as 3,4-dideoxypentosulose. we then used chromatography-coupled tandem mass spectrometry to quantitate 3,4-dideoxypentosulose in market samples of honey and hfcs and found that 3,4-dideoxypentosulose was negligible (<0.098 mg/kg) in honey, but prevalent in hfcs (≧1.174 mg/kg), indicating 3,4-dideoxypentosulose can serve as an alternative indicator of hfcs adulteration of acacia honey. the polyphenol distribution in fresh waxy corns of different color varieties and their functional activities, which may be useful for treating various chronic diseases, were investigated. the in vitro antioxidant activity, and hypoglycemic and hypocholesterolemic effects of the free and bound corn phenolics were determined to evaluate the edible value of fresh waxy corn. the colored varieties contained more phenols than the common varieties (white and/or yellow). the total free phenolic acid content of the six varieties was 6637.73 µg/g dw (dry weight), which was slightly higher (p > 0.05) than that of the total bound form (6276.65 µg/g dw). the total free flavonoid content was 5850.09 µg/g dw, which was higher (p < 0.05) than that of the total bound form (4874.51µg/g dw). no bound anthocyanin was detected in the methanol extracts of the tested varieties. for all the varieties, free polyphenols contributed 86-100 % and 70-78 % of the 2, 2-diphenyl-1-picrylhydrazyl (dpph) and hydroxyl radical scavenging abilities, respectively, and 100 % of the ferric reducing capacity. the free phenolics in fresh waxy corn showed better (p < 0.05) hypoglycemic effect than the bound form in terms of inhibition of α-amylase activity, whereas the bound phenolics of most varieties showed higher α-glucosidase inhibitory activity than the free forms. the free phenolics showed better (p < 0.05) glycocholesterol binding than the bound form for all varieties. the bound polyphenols showed better sodium cholate and taurocholate binding than the free form for most varieties. in conclusion, the difference between free and bound polyphenol content and functional activities indicates that fresh waxy corn can be potentially used for the development of functional food. phosphate is a limiting plant nutrient and essential for corn growth and development. thus, the impact of phosphate fertilization, location, and the variety of modern corn (zea mays l.) hybrids on the profiles of fatty acids, carotenoids, and tocochromanols (vitamin e) was assessed in corn grains. eight different corn hybrids were grown with (52.9 kg of phosphorus per ha) or without starter fertilizer at three experimental sites in germany. location (p < 0.05) and genetics (p < 0.001) but not phosphate fertilization significantly altered the concentrations of individual saturated and unsaturated fatty acids, carotenoids, and tocochromanols. significant (p < 0.05) interaction effects on the concentrations were mainly observed between the variety and the location. in conclusion, the choice of the corn variety had a more significant impact on the biosynthesis of fatty acids, carotenoids, and tocochromanols than the location or phosphate application on phosphate-sufficient soils. the fall armyworm, spodoptera frugiperda (j.e. smith) (lepidoptera: noctuidae) is a destructive crop pest native to north and south america that recently has spread to africa and asia. isolates of spodoptera frugiperda multiple nucleopolyhedrovirus (sfmnpv) have the potential to be developed as low-risk biopesticides for management of fall armyworm, and a commercially available formulation has been developed for control of fall armyworm in north and south america. in this study, the virulence (lc50 and lt50) of several sfmnpv isolates towards larvae of both corn-strain and rice-strain fall armyworm was assessed. bioassays with corn-strain larvae revealed that the isolates could be organized into fast-killing (lt50 < 56 h post-infection) and slow-killing (lt50 > 68 h post-infection) groups. rice-strain larvae exhibited narrower ranges of susceptibility to baculovirus infection and of survival times in bioassays with different isolates. two sfmnpv isolates with rapid speeds of kill (sfmnpv-459 from colombia and sfmnpv-1197 from georgia, usa) along with an isolate that killed corn-strain at relatively low concentrations (sfmnpv-281 from georgia) were selected for the complete determination of their genome sequences. the sfmnpv-1197 genome sequence shared high sequence identity with genomes of a nicaraguan isolate, while sfmnpv-281 formed a separate clade with a usa and a brazilian isolate in phylogenetic trees. the sfmnpv-459 sequence was more divergent with the lowest genome sequence identities in pairwise alignments with other sequenced sfmnpv genomes, and was not grouped reliably with either the 1197 clade or the 281 clade. sfmnpv-459 contained homologs of two orfs that were unique to another colombian isolate, but these isolates were not placed in the same clade in phylogenetic trees. this study identifies isolates with superior properties for control of fall armyworm and adds to our knowledge of the genetics of sfmnpv. the use of agarose in nucleic acid electrophoresis is the gold standard. however, agarose is very expensive and not readily available in resource limited developing countries like ghana. hence, finding a more affordable and readily available alternative to agarose will be a major boost to molecular research in developing countries. this study was aimed at investigating the use of corn starch as a potential substitute for agarose in dna gel electrophoresis. genomic deoxyribonucleic acid (dna) extracted from plasmodium falciparum and primers were obtained from the west african centre for cell biology of infectious pathogens and amplified using polymerase chain reaction. the amplicon was run on agarose gel to ascertain the molecular weight (as a positive control). when visualized under both blue light and ultraviolet light, the dna and ladder showed clear and clean bands with the expected molecular weight. corn starch was then modified with sodium borate buffer, casted into a gel and used to run the same dna sample. our findings indicated that similar to agarose, the dna sample and ladder migrated successfully through the modified starch gel but no bands were visible when visualized under blue and ultra-violet light. previous studies have reported that the corn earworm/bollworm, helicoverpa zea (boddie), has developed field resistance to pyramided bacillus thuringiensis (bt) cry1a/cry2a maize and cotton in certain areas of the southeastern united states. the objective of the current study was to determine the current status and distribution of the resistance to cry1a.105 and cry2ab2 in h. zea. in the study, 31 h. zea populations were collected from major maize planting areas across seven southeastern states of the united states during 2018 and 2019 and assayed against the two bt proteins. diet over-lay bioassays showed that most of the populations collected during the two years were significantly resistant to the cry1a.105 protein. most of the populations collected during 2019 were also resistant to cry2ab2, while significant variances were observed in the susceptibility of the populations collected during 2018 to cry2ab2. the results showed that cry1a.105 and cry2ab2 resistance in h. zea is widely distributed in the regions sampled. the resistance to cry1a.105 appeared to have plateaued, while selection for cry2ab2 resistance is likely still occurring. thus, effective measures for mitigating the cry1a/cry2a resistance need to be developed and implemented to ensure the sustainable use of the bt crop biotechnology. sweet corn (zea mays convar. saccharata var. rugosa) is a major economic vegetable crop. different sweet corn cultivars vary largely in flavor, texture, and nutrition. the present study performed widely targeted metabolomics analysis based on the hplc-ms/ms technology to analyze the metabolic profiles in three sweet corn cultivars widely grown in china. a total of 568 metabolites in the three sweet corn cultivars were detected, of which 262 differential metabolites significantly changed among cultivars. carbohydrates, organic acids, and amino acids were the majority detected primary metabolites. organic acids were mainly concentrated on shikimate, benzoic acids, and quinic acid with aromatic groups. and the essential amino acids for the human body, methionine and threonine, were highly accumulated in the high-quality cultivar. in addition, phenylpropanoids and alkaloids were the most enriched secondary metabolites while terpenes were low-detected in sweet corn kernels. we found that the flavonoids exist in both free form and glycosylated form in sweet corn kernels. pca and hca revealed clear separations among the three sweet corn cultivars, suggesting distinctive metabolome profiles among three cultivars. the differential metabolites were mapped into flavonoid biosynthesis, phenylpropanoid biosynthesis, biosynthesis of amino acids, and other pathways according to the kegg classification. furthermore, we identified skimmin, n',n″-diferuloylspermidine, and 3-hydroxyanthranilic acid as the key quality-related metabolites related to grain quality traits in sweet corn. the results suggested variations of metabolic composition among the three cultivars, providing the reference quality-related metabolites for sweet corn breeding. sweet corn is one of the most important vegetables in the united states and canada. here, we present a de novo assembly of a sweet corn inbred line ia453 with the mutated shrunken2-reference allele (ia453-sh2). this mutation accumulates more sugar and is present in most commercial hybrids developed for the processing and fresh markets. the ten pseudochromosomes cover 92% of the total assembly and 99% of the estimated genome size, with a scaffold n50 of 222.2 mb. this reference genome completely assembles the large structural variation that created the mutant sh2-r allele. furthermore, comparative genomics analysis with six field corn genomes highlights differences in single-nucleotide polymorphisms, structural variations, and transposon composition. phylogenetic analysis of 5,381 diverse maize and teosinte accessions reveals genetic relationships between sweet corn and other types of maize. our results show evidence for a common origin in northern mexico for modern sweet corn in the u.s. finally, population genomic analysis identifies regions of the genome under selection and candidate genes associated with sweet corn traits, such as early flowering, endosperm composition, plant and tassel architecture, and kernel row number. our study provides a high-quality reference-genome sequence to facilitate comparative genomics, functional studies, and genomic-assisted breeding for sweet corn. \n",
      "Euclidean Distance : 15.383827209472656\n",
      "\n",
      "\n",
      "Document: identification of acremonium acutatum and trichothecium roseum isolated from grape with white stain symptom in korea. during 2010 and 2012 grape harvest seasons in gyeonggi-do, korea, a white stain symptom was observed on the harvested grape fruits in 'campbell-early' and 'kyoho' varieties. in samples collected from the infected vine, two different strains of pathogenic fungi have been found and identified as acremonium acutatum and trichothecium roseum based on fungal morphology and nucleotide sequence of internal transcribed spacer (its) and supported by the phylogenetic analysis of the rdna-its region. the dna homologies of the isolated strains were 99.8% and 99.6% identical with t. roseum (ifb-22133) and a. acutatum (cbs682.71), respectively. in the pathogenicity test, the spores of a. acutatum and t. roseum sprayed on the grapes caused white stain symptoms on the fruits in two weeks after the artificial inoculation, which is similar to observations in the field. to our knowledge, this is the first report of white stain symptoms caused by a. acutatum and t. roseum on the grapes in korea. \n",
      "Euclidean Distance : 15.385601043701172\n",
      "\n",
      "\n",
      "Document: millet-seed conjunctival nodules. quality characteristics of biscuits prepared from finger millet seed coat based composite flour. finger millet seed coat is an edible material and contains good proportion of dietary fibre, minerals and phytochemicals. the seed coat matter (scm) forms a by-product of millet milling, malting and decortication industries and can be utilised as composite flour in biscuit preparation. the scm from native, malted and hydrothermally treated millet contained 9.5-12% protein, 2.6-3.7% fat and 40-48% dietary fibre, besides 3-5% polyphenols and 700-860mg/100g of calcium. the biscuits prepared using the composite flour were of crisp texture and exhibited breaking strength of 1480-1690g compared to control biscuits (1560g). the biscuits were of mild grey colour (δe=40-50) and exhibited higher protein, dietary fibre and calcium contents. the sensory evaluation of the biscuits indicated that 10% of scm from native and hydrothermally processed millet and 20% from malted millet could be used in composite biscuit flour. \n",
      "Euclidean Distance : 15.390721321105957\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. inter- and intra-tree variability of carbon and oxygen stable isotope ratios of modern pollen from nine european tree species. phenological match drives pollen-mediated gene flow in a temporally dimorphic tree. fagaceae pollen from the early cenozoic of west greenland: revisiting engler's and chaney's arcto-tertiary hypotheses. preparation and characterisation of biodegradable pollen-chitosan microcapsules and its application in heavy metal removal. effects of co₂ on acer negundo pollen fertility, protein content, allergenic properties, and carbohydrates. models to predict the start of the airborne pollen season. pollen loads and specificity of native pollinators of lowbush blueberry. changes in the ige-reacting protein profiles of acer negundo, platanus x acerifolia and quercus robur pollen in response to ozone treatment. first annual register of allergenic pollen in talca, chile. in vitro exposure of acer negundo pollen to atmospheric levels of so₂ and no₂: effects on allergenicity and germination. spatial and temporal modeling of daily pollen concentrations. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. stable carbon and oxygen isotope ratios of raw pollen sampled from nine abundant tree species growing in natural habitats of central and northern europe were investigated to understand the intra- and inter-specific variability of pollen-isotope values. all species yielded specific δ13cpollen and δ18opollen values and patterns, which can be ascribed to their physiology and habitat preferences. broad-leaved trees flowering early in the year before leaf proliferation (alnus glutinosa and corylus avellana) exhibited on average 2.6‰ lower δ13cpollen and 3.1‰ lower δ18opollen values than broad-leaved and coniferous trees flowering during mid and late spring (acer pseudoplatanus, betula pendula, carpinus betulus, fagus sylvatica, picea abies, pinus sylvestris and quercus robur). mean species-specific δ13cpollen values did not change markedly over time, whereas δ18opollen values of two consecutive years were often statistically distinct. an intra-annual analysis of b. pendula and p. sylvestris pollen revealed increasing δ18opollen values during the final weeks of pollen development. however, the δ13cpollen values remained consistent throughout the pollen-maturation process. detailed intra-individual analysis yielded circumferential and height-dependent variations within carbon and oxygen pollen-isotopes and the sampling position on a tree accounted for differences of up to 3.5‰ for δ13cpollen and 2.1‰ for δ18opollen. a comparison of isotope ranges from different geographic settings revealed gradients between maritime and continental as well as between high and low altitudinal study sites. the results of stepwise regression analysis demonstrated, that carbon and oxygen pollen-isotopes also reflect local non-climate environmental conditions. a detailed understanding of isotope patterns and ranges in modern pollen is necessary to enhance the accuracy of palaeoclimate investigations on δ13c and δ18o of fossil pollen. furthermore, pollen-isotope values are species-specific and the analysis of species growing during different phenophases may be valuable for palaeoweather reconstructions of different seasons. variation in flowering phenology is common in natural populations, and is expected to be, together with inter-mate distance, an important driver of effective pollen dispersal. in populations composed of plants with temporally separated sexual phases (i.e. dichogamous or heterodichogamous populations), pollen-mediated gene flow is assumed to reflect phenological overlap between complementary sexual phases. in this study, we conducted paternity analyses to test this hypothesis in the temporally dimorphic tree acer opalus. we performed spatially explicit analyses based on categorical and fractional paternity assignment, and included tree size, pair-wise genetic relatedness and morph type as additional predictors. because differences between morphs in flowering phenology may also influence pollination distances, we modelled separate pollen dispersal kernels for the two morphs. extended phenological overlap between male and female phases (mainly associated with inter-morph crosses) resulted in higher siring success after accounting for the effects of genetic relatedness, morph type and tree size, while reduced phenological overlap (mainly associated with intra-morph crosses) resulted in longer pollination distances achieved. siring success also increased in larger trees. mating patterns could not be predicted by phenology alone. however, as heterogeneity in flowering phenology was the single morph-specific predictor of siring success, it is expected to be key in maintaining the temporal dimorphism in a. opalus, by promoting not only a prevalent pattern of inter-morph mating, but also long-distance pollination resulting from intra-morph mating events. in this paper we document fagaceae pollen from the eocene of western greenland. the pollen record suggests a remarkable diversity of the family in the early cenozoic of greenland. extinct fagaceae pollen types include eotrigonobalanus, which extends at least back to the paleocene, and two ancestral pollen types with affinities to the eurasian quercus group ilex and the western north american quercus group protobalanus. in addition, modern lineages of fagaceae are unambiguously represented by pollen of fagus, quercus group lobatae/quercus, and three castaneoideae pollen types. these findings corroborate earlier findings from axel heiberg island that fagaceae were a dominant element at high latitudes during the early cenozoic. comparison with coeval or older mid-latitude records of modern lineages of fagaceae shows that modern lineages found in western greenland and axel heiberg likely originated at lower latitudes. further examples comprise (possibly) acer, aesculus, alnus, ulmus, and others. thus, before fossils belonging to modern northern temperate lineages will have been recovered from older (early eocene, paleocene) strata from high latitudes, engler's hypothesis of an arctic origin of the modern temperate woody flora of eurasia, termed 'arcto-tertiary element', and later modification by r. w. chaney and h. d. mai ('arcto-tertiary geoflora') needs to be modified. biosorbents have been widely used in heavy metal removal. new resources should be exploited to develop more efficient biosorbents. this study reports the preparation of three novel chitosan microcapsules from pollens of three common, wind-pollinated plants (acer negundo, cupressus sempervirens and populus nigra). the microcapsules were characterized (fourier transform infrared spectroscopy, thermogravimetric analysis, scanning electron microscopy and elemental analysis) and used in removal of heavy metal ions: cd(ii), cr(iii), cu(ii), ni(ii) and zn(ii). their sorption capacities were compared to those of cross-linked chitosan beads without pollen grains. c. sempervirens-chitosan microcapsules exhibited better performance (cd(ii): 65.98; cu(ii): 67.10 and zn(ii): 49.55 mg g(-1)) than the other microcapsules and the cross-linked beads. a. negundo-chitosan microcapsules were more efficient in cr(iii) (70.40 mg g(-1)) removal. p. nigra-chitosan microcapsules were found to be less efficient. chitosan-pollen microcapsules (except p. nigra-chitosan microcapsules) can be used in heavy metal removal. atmospheric gaseous pollutants can induce qualitative and quantitative changes in airborne pollen characteristics. in this work, it was investigated the effects of carbon dioxide (co2) on acer negundo pollen fertility, protein content, allergenic properties, and carbohydrates. pollen was collected directly from the anthers and in vitro exposed to three co2 levels (500, 1000, and 3000 ppm) for 6 and 24 h in an environmental chamber. pollen fertility was determined using viability and germination assays, total soluble protein was determined with coomassie protein assay reagent, and the antigenic and allergenic properties were investigated by sodium dodecyl sulfate polyacrylamide gel electrophoresis (sds-page) and immunological techniques using patients' sera. also, pollen fructose, sucrose, and glucose values were determined. carbon dioxide exposure affected negatively pollen fertility, total soluble protein content, and fructose content. the patient sera revealed increased ige reactivity to proteins of a. negundo pollen exposed to increasing levels of the pollutant. no changes were detected in the sds-page protein profiles and in sucrose and glucose levels. our results indicate that increase in atmospheric co2 concentrations can have a negative influence of some features of a. negundo airborne pollen that can influence the reproductive processes as well as respiratory pollen allergies in the future. aerobiological data can be used as indirect but reliable measures of flowering phenology to analyze the response of plant species to ongoing climate changes. the aims of this study are to evaluate the performance of several phenological models for predicting the pollen start of season (pss) in seven spring-flowering trees (alnus glutinosa, acer negundo, carpinus betulus, platanus occidentalis, juglans nigra, alnus viridis, and castanea sativa) and in two summer-flowering herbaceous species (artemisia vulgaris and ambrosia artemisiifolia) by using a 26-year aerobiological data set collected in turin (northern italy). data showed a reduced interannual variability of the pss in the summer-flowering species compared to the spring-flowering ones. spring warming models with photoperiod limitation performed best for the greater majority of the studied species, while chilling class models were selected only for the early spring flowering species. for ambrosia and artemisia, spring warming models were also selected as the best models, indicating that temperature sums are positively related to flowering. however, the poor variance explained by the models suggests that further analyses have to be carried out in order to develop better models for predicting the pss in these two species. modeling the pollen season start on a very wide data set provided a new opportunity to highlight the limits of models in elucidating the environmental factors driving the pollen season start when some factors are always fulfilled, as chilling or photoperiod or when the variance is very poor and is not explained by the models. the reproduction of lowbush blueberry (vaccinium angustifolium aiton) is closely tied to insect pollination, owing to self-incompatibility. many species are known to have greater pollination efficiency than the introduced apis mellifera l., commonly used for commercial purposes. in this study, we measured the pollen loads of several antophilous insect species, mostly apoidea and syrphidae, present in four lowbush blueberry fields in lac-st-jean, québec. to measure pollen loads and species specificity toward v. angustifolium, we net-collected 627 specimens of pollinators, retrieved their pollen loads, identified pollen taxa, and counted pollen grains. we found that the sizes of pollen loads were highly variable among species, ranging from a few hundred to more than 118,000 pollen grains per individual. bombus and andrena species in particular carried large amounts of vaccinium pollen and thus may have greater pollination efficiency. also, two species (andrena bradleyi viereck and andrena carolina viereck) showed nearly monolectic behavior toward lowbush blueberry. finally, we identified alternative forage plants visited by native pollinators, notably species of acer, rubus, ilex mucronata, ledum groenlandicum, and taraxacum. protecting these flowering plants should be part of management practices to maintain healthy pollinator communities in a lowbush blueberry agroecosystem. this study aims to investigate the effects of o3 in protein content and immunoglobulin e (ige)-binding profiles of acer negundo, platanus x acerifolia and quercus robur pollen. pollen was exposed to o3 in an environmental chamber, at half, equal and four times the limit value for the human health protection in europe. pollen total soluble protein was determined with coomassie protein assay reagent, and the antigenic and allergenic properties were investigated by sds-page and immunological techniques using patients' sera. o3 exposure affected total soluble protein content and some protein species within the sds-page protein profiles. most of the sera revealed increased ige reactivity to proteins of a. negundo and q. robur pollen exposed to the pollutant compared with the non-exposed one, while the opposite was observed in p. x acerifolia pollen. so, the modifications seem to be species dependent, but do not necessarily imply that increase allergenicity would occur in atopic individuals. there are no data on atmospheric pollen in talca. in the present work, our aim is to describe the amount of pollen grain in the atmosphere of the city of talca likely to cause pollinosis of its inhabitants. a volumetric hirst sampler (burkard seven-day recording device) was used to study pollen levels. it was placed in the centre of talca from may 2007 to april 2008. the highest airborne presence of pollen, as measured in weekly averages, was platanus acerifolia with a maximum weekly daily average of 203 grains/m³ registered during september and october. the second highest was acer pseudoplatanus with a maximum weekly daily average of 116 grains/m³. populus spp. had a maximum weekly daily average 103 grains/m³. olea europaea reached 19 grains/m³ in november. grasses presented high levels of pollen counts with a maximum weekly daily average of 27 grains/m³ from the end of august until the end of january. pollens of plantago spp. rumex acetosella and chenopodium spp. had a similar distribution and were present from october to april with maximum weekly daily average of 7 grains/m³, 7 grains/m³ and 3 grains/m³ respectively. significant concentrations of ambrosia artemisiifolia were detected from february until april. the population of talca was exposed to high concentrations of allergenic pollen, such as p. acerifolia, a. pseudoplatanus, and grasses in the months of august through november. the detection of o. europaea and a. artemisiifolia is important as these are emergent pollens in the city of talca. aerobiological monitoring will provide the community with reliable information about the level of allergenic pollens, improving treatment and quality of life of patients with respiratory allergy. in the last years, a rising trend of pollen allergies in urban areas has been attributed to atmospheric pollution. in this work, we investigated the effects of so(2) and no(2) on the protein content, allergenicity, and germination rate of acer negundo pollen. a novel environmental chamber was assembled to exposure pollen samples with so(2) or no(2) at two different levels: just below and two times the atmospheric hour-limit value acceptable for human health protection in europe. results showed that protein content was lower in so(2)-exposed pollen samples and slightly higher in no(2)-exposed pollen compared to the control sample. no different polypeptide profiles were revealed by sds-page between exposed and nonexposed pollen, but the immunodetection assays indicated higher ige recognition by all sera of sensitized patients to acer negundo pollen extracts in all exposed samples in comparison to the nonexposed samples. a decrease in the germination rate of exposed in contrast to nonexposed pollen was verified, which was more pronounced for no(2)-exposed samples. our results indicated that in urban areas, concentrations of so(2) and no(2) below the limits established for human protection can indirectly aggravate pollen allergy on predisposed individuals and affect plant reproduction. accurate assessments of pollen counts are valuable to allergy sufferers, the medical industry, and health researchers; however, monitoring stations do not exist in most areas. in addition, the degree of spatial reliability provided by the limited number of monitoring stations is poorly understood. we developed and compared spatial models to estimate pollen concentrations in locations without monitoring stations. daily acer, quercus, and overall tree, grass, and weed pollen counts, in grains/m(3), were obtained from 14 aeroallergen monitoring stations located in the northeastern and mid-atlantic region of the united states from 2003 to 2006. pollen counts were spatially interpolated using ordinary kriging. mixed effects and generalized estimating equations incorporating daily and seasonal weather characteristics, pollen season characteristics and land-cover information were also developed to estimate daily pollen concentrations. we then compared observed values from a monitoring station to model estimates for that location. observed counts and kriging estimates for tree pollen differed (p = 0.04), but not when peak periods were removed (p = 0.29). no differences between observed and kriging estimates of acer (p = 0.46), quercus (p = 0.24), grass (p = 0.31) or weed pollen (p = 0.29) were found. estimates from longitudinal models also demonstrated good agreement with observed counts, except for the extremes of pollen distributions. our results demonstrate that spatial interpolation techniques as well as regression methods incorporating both weather and land-cover characteristics can provide reliable estimates of daily pollen concentrations in areas where monitors do not exist for all but periods of extremely high pollen. \n",
      "Euclidean Distance : 15.399052619934082\n",
      "\n",
      "\n",
      "Document: feather mites (acari, astigmata) from azorean passerines (aves, passeriformes): lower species richness compared to european mainland. ten passerine species were examined on three islands of the azores (north atlantic) during 2013 and 2014 in order to identify their feather mite assemblages. we recorded 19 feather mite species belonging to four families of the superfamily analgoidea (analgidae, proctophyllodidae, psoroptoididae and trouessartiidae). a high prevalence of feather mite species was recorded on the majority of the examined host species. only three passerine species (sylvia atricapilla, regulus regulus and serinus canaria) presented the same full complex of mite species as commonly occurs in the plumage of their closest relatives in continental europe. passer domesticus presented the same limited fauna of feather mites living in the plumage as do its co-specifics in continental europe. carduelis carduelis bears the same feather mite species as do most of its continental populations in europe, but it lacks one mite species occurring on this host in egypt. turdus merula, pyrrhula murina and fringilla coelebs are missing several mite species common to their continental relatives. this diminution could be explained by the founder effect, whereby a limited number of colonizing individuals did not transport the full set of feather mite species, or by the extinction of some mite species after initially having reached the azores. the only individual of motacilla cinerea sampled in this study presented a new host record for the mite species trouessartia jedliczkai. \n",
      "Euclidean Distance : 15.435935020446777\n",
      "\n",
      "\n",
      "Document: impact of elevated co2 and heat stress on wheat pollen viability and grain production. light promoted brown staining of protoplasm by ag+ is ideal to test wheat pollen viability rapidly. genetic mapping of male sterility and pollen fertility qtls in triticale with sterilizing triticum timopheevii cytoplasm. a novel reversibly glycosylated polypeptide-2 of bee pollen from rape (brassica napus l.): purification and characterization. comprehensive analysis of polygalacturonase gene family highlights candidate genes related to pollen development and male fertility in wheat (triticum aestivum l.). pollen grain counting using a cell counter. blocked synthesis of sporopollenin and jasmonic acid leads to pollen wall defects and anther indehiscence in genic male sterile wheat line 4110s at high temperatures. cytological and proteomic analysis of wheat pollen abortion induced by chemical hybridization agent. bermuda grass pollen allergen implicated in clinically relevant cross-reactivity to multiple grains: a case report. wheat tams1 is a glycosylphosphatidylinositol-anchored lipid transfer protein necessary for pollen development. oxidative stress and aberrant programmed cell death are associated with pollen abortion in isonuclear alloplasmic male-sterile wheat. alterations in wheat pollen lipidome during high day and night temperature stress. ms26/cyp704b is required for anther and pollen wall development in bread wheat (triticum aestivum l.) and combining mutations in all three homeologs causes male sterility. a proteomics approach to discover drought tolerance proteins in wheat pollen grain at meiosis stage. investigating pollen and gene flow of wymv-resistant transgenic wheat n12-1 using a dwarf male-sterile line as the pollen receptor. visualisation of plastid degradation in sperm cells of wheat pollen. can  addition of rye chromosome 4r to wheat increases anther length and pollen grain number. pollen flow of wheat under natural conditions in the huanghuai river wheat region, china. chilling to zero degrees disrupts pollen formation but not meiotic microtubule arrays in triticum aestivum l. periods of high temperature and an expected increase in atmospheric co2 concentration as a result of global climate change are major threats to wheat (triticum aestivum l.) production. developing heat-tolerant wheat cultivars demands improved understanding of the impacts of high temperature and elevated co2 on plant growth and development. this research investigated the interactive effects of heat stress and co2 concentration on pollen viability and its relationship to grain formation and yield of wheat in greenhouse conditions. nineteen wheat genotypes and a current cultivar, suntop, were heat stressed at either meiosis or anthesis at ambient (400 µl l-1) or elevated (800 µl l-1) co2. elevated co2 and heat stress at meiosis reduced pollen viability, spikelet number and grain yield per spike; however, increased tillering at the elevated co2 level helped to minimise yield loss. both heat-tolerant genotypes (e.g. genotype 1, 2, 10 or 12) and heat-sensitive genotypes (e.g. genotype 6 or 9) were identified and response related to pollen sensitivity and subsequent impacts on grain yield and yield components were characterised. a high-throughput protocol for screening wheat for heat stress response at elevated co2 was established and meiosis was the most sensitive stage, affecting pollen viability, grain formation and yield. pollen viability is crucial for wheat breeding programs. the unique potential of the protoplasm of live cells to turn brown due to the synthesis of silver nanoparticles (agnps) through rapid photoreduction of ag+, was exploited for testing wheat pollen viability. ag+-viability test medium (consisting of 0.5 mm agno3 and 300 mm kno3) incubated with wheat pollen turned brown within 2 min under intense light (~600 μmol photon flux density m-2s-1), but not in dark. the brown medium displayed agnps-specific surface plasmon resonance band in its absorption spectrum. light microscopic studies showed the presence of uniformly stained brown protoplasm in viable pollen incubated with ag+-viability medium in the presence of light. investigations with transmission electron microscope coupled with energy dispersive x-ray established the presence of distinct 5-35 nm nps composed of ag. powder x-ray diffraction analysis revealed that agnps were crystalline and biphasic composed of ag0 and ag2o. conversely, non-viable pollen and heat-killed pollen did not turn brown on incubation with ag+-medium in light. we believe that the viable wheat pollen turn brown rapidly by bio-transforming ag+ to agnps through photoreduction. our findings furnish a novel simplest and rapid method for testing wheat pollen viability. cytoplasmic male sterility (cms) phenomenon is widely exploited in commercial hybrid seed production in economically important crop species, including rye, wheat, maize, rice, sorghum, cotton, sugar beets, and many vegetables. although some commercial successes, little is known about qtls responsible for the trait in case of triticale with sterilizing triticum timopheevii (tt) cytoplasm. recombinant inbred line (ril) f6 mapping population encompassing 182 individuals derived from the cross of individual plants representing the ht352 line and cv borwo was employed for genetic map construction using snp markers and identification of qtls conferring pollen sterility in triticale with cms tt. the phenotypes of the f1 lines resulting from crossing of the ht352 (tt) with ht352 (maintainer) × borwo were determined by assessing the number of the f2 seeds per spike. a genetic map with 21 linkage groups encompasses 29,737 markers and spanned over the distance of 2549 cm. composite (cim) and multiple (mim) interval mappings delivered comparable results. single qtls mapped to the 1a, 1b, 2a, 2r, 3b, 3r, 4b, and 5b chromosomes, whereas the 5r and 6b chromosomes shared 3 and 2 qtls, respectively. the qtls with the highest lod score mapped to the 5r, 3r, 1b, and 4b chromosomes; however, the qrft-5r.3 has the highest explained variance of the trait. reversibly glycosylated polypeptide (rgp), a kind of hydrosoluble and plasmodesmal-associated protein found in plants, plays a crucial role in the development of pollen. a novel rgp 2 was isolated and identified from rape (brassica napus l.) bee pollen. rgp2 was isolated and purified by ion-exchange column and gel filtration chromatography, and characterized by maldi-tof-ms, lc-ms, immunological histological chemistry, and transmission electron microscope. our results indicated that the rgp2 is an acidic protein (pi=5.46) with the molecular weight 42388 da. it contained 17 kinds of amino acids, among which aspartic acid had the highest amount (71.56 mg/g). homologous alignment of amino acid sequence results showed that rgp2 was 80.33%, 85.02%, 86.06%, and 88.93% identical to arabidopsis thaliana rgp2 (atrgp2), oryza sativa rgp (osrgp), triticum aestivum rgp (targp), and zea maize rgp (zmrgp), respectively. the localization results showed that rgp2 in rape anther existed in exine and intine of anther cells of rape flower by immunological histological chemistry and the subcellular localization identified that rgp2 appeared around the golgi apparatus in cytoplasm by transmission electron microscope. rgp2 has a highly conserved sequence of amino acid residues and potential glycosylation sites. four polygalacturonase gene family members were highlighted that contribute to elucidate the roles of polygalacturonase during the fertility conversion process in male-sterile wheat. polygalacturonase (pg) belongs to a large family of hydrolases with important functions in cell separation during plant growth and development via the degradation of pectin. specific expressed pgs in anthers may be significant for male sterility research and hybrid wheat breeding, but they have not been characterized in wheat (triticum aestivum l.). in this study, we systematically studied the pg gene family using the latest published wheat reference genomic information. in total, 113 wheat pg genes were identified, which could be classified into six categories a-f according to their structure characteristics and phylogenetic comparisons with arabidopsis and rice. polyploidy and segmental duplications in wheat were proved to be mainly responsible for the expansion of the wheat pg gene family. rna-seq showed that tapgs have specific temporal and spatial expression characteristics, in which 12 tapgs with spike-specific expression patterns were detected by qrt-pcr in different fertility anthers of ktm3315a, a thermo-sensitive cytoplasmic male-sterile wheat. four of them specific upregulated (tapg09, tapg95, and tapg93) or downregulated (tapg87) at trinucleate stage of fertile anthers, and further aligning with the homologous in arabidopsis revealed that they may undertake functions such as anther dehiscence, separation of pollen, pollen development, and pollen tube elongation, thereby inducing male fertility conversion in ktm3315a. these findings facilitate function investigations of the wheat pg gene family and provide new insights into the fertility conversion mechanism in male-sterile wheat. the number of pollen grains is a critical part of the reproductive strategies in plants and varies greatly between and within species. in agriculture, pollen viability is important for crop breeding. it is a laborious work to count pollen tubes using a counting chamber under a microscope. here, we present a method of counting the number of pollen grains using a cell counter. in this method, the counting step is shortened to 3 min per flower, which, in our setting, is more than five times faster than the counting chamber method. this technique is applicable to species with a lower and higher number of pollen grains, as it can count particles in a wide range, from 0 to 20,000 particles, in one measurement. the cell counter also estimates the size of the particles together with the number. because aborted pollen shows abnormal membrane characteristics and/or a distorted or smaller shape, a cell counter can quantify the number of normal and aborted pollen separately. we explain how to count the number of pollen grains and measure pollen size in arabidopsis thaliana, arabidopsis kamchatica, and wheat (triticum aestivum). environment-sensitive genic male sterility is a valid tool for hybrid production and hybrid breeding, but there are no previous reports of the molecular mechanism of fertility conversion. in this study, rna-seq, phenotypic and cytological observations, and physiological indexes were applied to analyze thermo-sensitive genic male sterility line 4110s under different temperature conditions to explore the fertility transformation mechanism. in total, 3420 differentially expressed genes (degs) were identified comprising 2331 upregulated genes and 1089 downregulated genes. the degs were apparently distributed among 54 gene ontology functional groups. the phenylpropanoid, long-chain fatty acid, and jasmonic acid (ja) biosynthesis pathways were related to male sterility, where their downregulation blocked the synthesis of sporopollenin and ja. phenotypic and cytological analyses showed that pollen wall defects and anther indehiscence at high temperatures induced sterility. moreover, enzyme-linked immunosorbent assay results indicated that the abundance of ja was lower in 4110s under restrictive conditions (high temperature) than permissive conditions (low temperature). a possible regulated network of pathways associated with male sterility was suggested. these results provided insights into the molecular mechanism of fertility conversion in the thermosensitive male sterility system. in plants, pollen grain transfers the haploid male genetic material from anther to stigma, both between flowers (cross-pollination) and within the same flower (self-pollination). in order to better understand chemical hybridizing agent (cha) sq-1-induced pollen abortion in wheat, comparative cytological and proteomic analyses were conducted. results indicated that pollen grains underwent serious structural injury, including cell division abnormality, nutritional deficiencies, pollen wall defect and pollen grain malformations in the cha-sq-1-treated plants, resulting in pollen abortion and male sterility. a total of 61 proteins showed statistically significant differences in abundance, among which 18 proteins were highly abundant and 43 proteins were less abundant in cha-sq-1 treated plants. 60 proteins were successfully identified using maldi-tof/tof mass spectrometry. these proteins were found to be involved in pollen maturation and showed a change in the abundance of a battery of proteins involved in multiple biological processes, including pollen development, carbohydrate and energy metabolism, stress response, protein metabolism. interactions between these proteins were predicted using bioinformatics analysis. gene ontology and pathway analyses revealed that the majority of the identified proteins were involved in carbohydrate and energy metabolism. accordingly, a protein-protein interaction network involving in pollen abortion was proposed. these results provide information for the molecular events underlying cha-sq-1-induced pollen abortion and may serve as an additional guide for practical hybrid breeding. anaphylaxis is a severe and potentially fatal type of allergic reaction and is characterized by the rapid development of symptoms in the respiratory and circulatory systems, possibly leading to death if not treated properly. occupational anaphylaxis, which does not exhibit significant differences in pathogenesis from the nonoccupational form, develops in response to work-related triggers. however, the onset of occupational anaphylaxis can also be triggered by other factors. therefore, an unexpected episode may occur due to exposure to a previously sensitized antigen or cross-reaction in the occupational environment, even if the direct trigger has been removed. accordingly, it is difficult to diagnosis and treat such cases and ensure avoidance of potential triggers. an adult male patient developed anaphylaxis following exposure to grass antigens while replacing and burying sewer pipes at a theme park. he later developed cross-reactivity to other grains. despite symptomatic treatment, his total serum level of allergen-specific immunoglobulin e (ig e) antibodies continuously increased, and thus, he was admitted with severe hypersensitivity, at which time his serum levels of ig e antibodies specific for bermuda grass, wheat, and rice had also increased. in korea, bermuda grass is rarely seen and is generally found in athletic fields or theme parks. following exposure to this relatively rare grass, our patient exhibited new anaphylactic responses to various external antigens. therefore, we attribute his severe anaphylaxis to sensitization caused by bermuda grass exposure and cross-reactive hypersensitivity to other grains. in flowering plants, lipid biosynthesis and transport within anthers is essential for male reproductive success. tams1, a dominant wheat fertility gene located on chromosome 4bs, has been previously fine mapped and identified to encode a glycosylphosphatidylinositol (gpi)-anchored non-specific lipid transfer protein (nsltp). although this gene is critical for pollen exine development, details of its function remains poorly understood. in this study, we report that tams1 is only expressed from the b sub-genome, with highest transcript abundance detected in anthers containing microspores undergoing pre-meiosis through to meiosis. β-glucuronidase transcriptional fusions further revealed that tams1 is expressed throughout all anther cell-types. tams1 was identified to be expressed at an earlier stage of anther development relative to genes reported to be necessary for sporopollenin precursor biosynthesis. in anthers missing a functional tams1 (ms1c deletion mutant), these same genes were not observed to be mis-regulated, indicating an independent function for tams1 in pollen development. exogenous hormone treatments on gus reporter lines suggest that tams1 expression is increased by both indole-3-acetic acid (iaa) and abscisic acid (aba). translational fusion constructs showed that tams1 is targeted to the plasma membrane. in summary, tams1 is a wheat fertility gene, expressed early in anther development and encodes a gpi-ltp targeted to the plasma membrane. the work presented provides a new insight into the process of wheat pollen development. cytoplasmic male sterility is crucial for the utilization of hybrid heterosis and it possibly occurs in parallel with tapetal programmed cell death (pcd) and oxidative metabolism responses. however, little is known about the mechanisms that underlie pollen abortion in wheat. therefore, we obtained two isonuclear alloplasmic male sterile lines (iamsls) with aegilops kotschyi and ae. juvenalis cytoplasm. compared with the maintainer line, cytochemical analyses of the anthers demonstrated that the iamsls exhibited anomalous tapetal pcd and organelles, with premature pcd in k87b1-706a and delayed pcd in ju87b1-706a. we also found that the dynamic trends in reactive oxygen species (ros) were consistent in these two iamsls during anther development and they were potentially associated with the initiation of tapetal pcd. in addition, the activities of ros-scavenging enzymes increased rapidly, whereas non-enzymatic antioxidants were downregulated together with excess ros production in iamsls. real-time pcr analysis showed that the expression levels of superoxide dismutase, catalase, and ascorbate peroxidase genes, which encode important antioxidant enzymes, were significantly upregulated during early pollen development. thus, we inferred that excessive ros and the abnormal transcript levels of antioxidant enzyme genes disrupted the balance of the antioxidant system and the presence of excess ros may have been related to aberrant tapetal pcd progression, thereby affecting the development of microspores and ultimately causing male sterility. these relationships between the mechanism of pcd and ros metabolism provide new insights into the mechanisms responsible for abortive pollen in wheat. understanding the adaptive changes in wheat pollen lipidome under high temperature (ht) stress is critical to improving seed set and developing ht tolerant wheat varieties. we measured 89 pollen lipid species under optimum and high day and/or night temperatures using electrospray ionization-tandem mass spectrometry in wheat plants. the pollen lipidome had a distinct composition compared with that of leaves. unlike in leaves, 34:3 and 36:6 species dominated the composition of extraplastidic phospholipids in pollen under optimum and ht conditions. the most ht-responsive lipids were extraplastidic phospholipids, phosphatidylcholine (pc), phosphatidylethanolamine (pe), phosphatidylinositol, phosphatidic acid, and phosphatidylserine. the unsaturation levels of the extraplastidic phospholipids decreased through the decreases in the levels of 18:3 and increases in the levels of 16:0, 18:0, 18:1, and 18:2 acyl chains. pc and pe were negatively correlated. higher pc:pe at ht indicated possible pe-to-pc conversion, lower pe formation, or increased pe degradation, relative to pc. correlation analysis revealed lipids experiencing coordinated metabolism under ht and confirmed the ht responsiveness of extraplastidic phospholipids. comparison of the present results on wheat pollen with results of our previous research on wheat leaves suggests that similar lipid changes contribute to ht adaptation in both leaves and pollen, though the lipidomes have inherently distinct compositions. development of anthers and pollen represents an important aspect of the life cycle in flowering plants. genes contributing to anther and pollen development have been widely studied in many plant species. ms26/cyp704b genes play an important role in pollen development through biosynthesis of sporopollenin for pollen exine formation. to investigate the role of ms26/cyp704b genes in anther and pollen development of bread wheat, mutations in the a-, b-, and d-homeologs of the putative ms26/cyp704b gene were analyzed. single and double homozygous mutants in any of the homeologs did not affect pollen development and male fertility. triple homozygous mutants resulted in completely male sterile plants that were defective in pollen and anther development. additionally, double homozygous-single heterozygous mutants were also male sterile although with varying levels of residual fertility. the fertility of these triple mutants was dependent upon the homeolog contributing the wild-type allele. two heterologous ms26/cyp704b genes, when transformed into a triple homozygous mutant background, completely restored male fertility, whereas a single gene was unable to restore fertility. functional analysis of ms26/cyp704b furthers the understanding of male fertility genes which can be utilized for the development of novel hybrid seed production systems in wheat. plants reproductive phase, when grain yield and consequently farmers' investment is most in jeopardy, is considered as the most sensitive stage to drought stress. in this study, we aimed to explore the proteomic response of wheat anther at meiosis stage in a drought tolerant, darab, and susceptible, shiraz, wheat genotypes. wheat plants were exposed to drought stress at meiosis stage for four days under controlled environmental conditions. then, anthers from both genotypes were sampled, and their proteomes were examined via quantitative proteomics analysis. our results demonstrated that short-term stress at meiosis stage reduced plant seed-setting compared to well-watered plants. this reduction was more pronounced in the susceptible genotype, shiraz, by 51%, compared to the drought tolerant darab by 14.3%. proteome analysis revealed that 60 protein spots were drought responsive, out of which 44 were identified using a mass spectrometer. we observed a dramatic up-regulation of several heat shock proteins, as well as induction of bet v i allergen family proteins, peroxiredoxin-5, and glutathione transferase with similar abundance in both genotypes. however, the abundance of proteins such as several stress response related proteins, including glutaredoxin, proteasome subunit alpha type 5, and ribosomal proteins showed a different response to drought stress in two genotypes. the differential abundance of proteins in two genotypes may suggest mechanisms by which tolerant genotype cope with drought stress. to the best of our knowledge, this is the first proteome analysis of plant reproductive tissue response to drought stress in wheat and could broaden our insight into plant adaptation to drought stress. pollen-mediated gene flow (pmgf) is the main mode of transgene flow in flowering plants. the study of pollen and gene flow of transgenic wheat can help to establish the corresponding strategy for preventing transgene escape and contamination between compatible genotypes in wheat. to investigate the pollen dispersal and gene flow frequency in various directions and distances around the pollen source and detect the association between frequency of transgene flow and pollen density from transgenic wheat, a concentric circle design was adopted to conduct a field experiment using transgenic wheat with resistance to wheat yellow mosaic virus (wymv) as the pollen donor and dwarf male-sterile wheat as the pollen receptor. the results showed that the pollen and gene flow of transgenic wheat varied significantly among the different compass sectors. a higher pollen density and gene flow frequency was observed in the downwind sw and w sectors, with average frequencies of transgene flow of 26.37 and 23.69% respectively. the pollen and gene flow of transgenic wheat declined dramatically with increasing distance from its source. most of the pollen grains concentrated within 5 m and only a few pollen grains were detected beyond 30 m. the percentage of transgene flow was the highest where adjacent to the pollen source, with an average of 48.24% for all eight compass directions at 0 m distance. transgene flow was reduced to 50% and 95% between 1.61 to 3.15 m, and 10.71 to 20.93 m, respectively. our results suggest that climate conditions, especially wind direction, may significantly affect pollen dispersal and gene flow of wheat. the isolation-by-distance model is one of the most effective methods for achieving stringent transgene confinement in wheat. the frequency of transgene flow is directly correlated with the relative density of gm pollen grains in air currents, and pollen competition may be a major factor influencing transgene flow. like most angiosperms, wheat (triticum aestivum) shows maternal inheritance of plastids. it is thought that this takes place by cytoplasmic stripping at fertilisation rather than the absence of plastids in sperm cells. to determine the fate of plastids during sperm cell development, plastid-targeted green fluorescent protein was used to visualise these organelles in nuclear transgenic wheat lines. fewer than thirty small 1-2-μm plastids were visible in early uninucleate pollen cells. these dramatically increased to several hundred larger (4 μm) plastids during pollen maturation and went through distinct morphological changes. only small plastids were visible in generative cells (n = 25) and young sperm cells (n = 9). in mature sperm cells, these green fluorescent protein (gfp)-tagged plastids were absent. this is consistent with maternal inheritance of plastids resulting from their degradation in mature sperm cells in wheat. the domestication of the one-grained einkorn (triticum monococcum) in the near east is relatively well known. however, an independent two-grained einkorn-like domestication has been archaeobotanically detected and scarce information is available. triticum urartu, a wild wheat, was not fully described until the 1970s because the phenology does not allow it to be distinguished easily from wild einkorn (triticum boeoticum subsp. thaoudar), although a genetic separation exists. both species are mostly two grained and could potentially be the relatives of the extinct two-grained form. pollen grains of several genetically well-identified wheat species, including t. urartu and t. boeoticum subsp. thaoudar, were studied by measuring the grain diameter and examining the exine sculpturing with phase-contrast microscopy and scanning electron microscopy to gain an insight into differences enabling taxonomic identification. this work showed that, although t. urartu pollen is smaller on average, grain diameter is not sufficient because of the size overlap between the species, but t. urartu presents a different exine sculpturing (scabrate) from other triticum spp. (aerolate). this outcome is useful for taxonomists and archaeobotanists. first, it will allow a simple re-classification of herbarium materials. second, further research could establish whether t. urartu was cultivated. the research identified rye chromosome 4r arms associated with good pollinator traits, and demonstrated possible use of rye genetic resources to develop elite pollinators for hybrid wheat breeding. bread wheat (triticum aestivum) is a predominantly self-pollinating plant which has relatively small-sized anthers and produces a low number of pollen grains. these features limit the suitability of most wheat lines as pollinators for hybrid seed production. one strategy for improving the pollination ability of wheat is to introgress cross-pollination traits from related species. one such species is rye (secale cereale l.), which has suitable traits such as high anther extrusion, long anthers containing large amounts of pollen and long pollen viability. therefore, introducing these traits into wheat is of great interest in hybrid wheat breeding. here, we investigated wheat-rye chromosome addition lines for the effects of rye chromosomes on anther and pollen development in wheat. using a single nucleotide polymorphism genotyping array, we detected 984 polymorphic markers that showed expected syntenic relationships between wheat and rye. our results revealed that the addition of rye chromosomes 1r or 2r reduced pollen fertility, while addition of rye chromosome 4r increased anther size by 16% and pollen grain number by 33%. the effect on anther length was associated with increases in both cell size and the number of endothecium cells and was attributed to the long arm of chromosome 4r. in contrast, the effect on pollen grain number was attributed to the short arm of chromosome 4r. these results indicate that rye chromosome 4r contains at least two genetic factors associated with increased anther size and pollen grain number that can favourably affect pollination traits in wheat. the transgenic pollen spread is the main pathway of transgenic plant gene flow. the maximum distance of pollen dispersal (horizontal), the spatial dynamics of pollen movement (vertical), and the patterns of pollen dispersal are important considerations in biosafety assessments of genetically modified crops. to evaluate wheat (triticum aestivum) pollen dispersal, we measured the pollen suspension velocity and analyzed pollen dispersal patterns under natural conditions in the huanghuai river wheat-growing region in 2009. the pollen suspension velocity was 0.3-0.4 m/s. the highest pollen densities were detected in the north, northwest, and south of the pollen source. pollen was dispersed over distances greater than 245 m in the northwest and northeast directions. at the pollen source center, pollen density decreased with increasing vertical height. in the north of the pollen source, the pollen density from 65 m to 225 m showed a wave-mode decrease with increasing height. the horizontal transport of pollen over longer distances fitted polynomial equations. in the north, the pollen density was highest at 45 m from the pollen source, and decreased with increasing distance. in the northwest, the pollen density showed a double-peak trend. in the northeast, pollen density was highest from 45 m to 125 m from the source. wind speeds greater than the pollen suspension velocity and the duration of continuous gusts were the main factors affecting pollen dispersal. this information will be useful for determining the spatial isolation distances for hybrid seed production and for the commercial production of transgenic wheat. throughout the wheat-growing regions of australia, chilling temperatures below 2 °c occur periodically on consecutive nights during the period of floral development in spring wheat (triticum aestivum l.). in this study, wheat plants showed significant reductions in fertility when exposed to prolonged chilling temperatures in controlled environment experiments. among the cultivars tested, the australian cultivars kite and hartog had among the lowest levels of seed set due to chilling and their responses were investigated further. the developmental stage at exposure, the chilling temperature and length of exposure all influenced the level of sterility. the early period of booting, and specifically the +4 cm auricle distance class, was the most sensitive and corresponded to meiosis within the anthers. the response of microtubules to chilling during meiosis in hartog was monitored, but there was little difference between chilled and control plants. other abnormalities, such as plasmolysis and cytomixis increased in frequency, were associated with death of developing pollen cells, and could contribute to loss of fertility. the potential for an above-zero chilling sensitivity in australian spring wheat varieties could have implications for exploring the tolerance of wheat flower development to chilling and freezing conditions in the field. \n",
      "Euclidean Distance : 15.439106941223145\n",
      "\n",
      "\n",
      "Document: identification of a lipid transfer protein as a new allergen from morus alba pollen.  \n",
      "Euclidean Distance : 15.483986854553223\n",
      "\n",
      "\n",
      "Document: antipodal views on branding amylocaine as stovaine: modesty or political correctness?  \n",
      "Euclidean Distance : 15.588306427001953\n",
      "\n",
      "\n",
      "Document: whole genome sequence insight of two plant growth-promoting bacteria (b. subtilis bs87 and b. megaterium bm89) isolated and characterized from sugarcane rhizosphere depicting better crop yield potentiality. co-zeolitic imidazolate framework@cellulose aerogels from sugarcane bagasse for activating peroxymonosulfate to degrade p-nitrophenol. plant-growth endophytic bacteria improve nutrient use efficiency and modulate foliar n-metabolites in sugarcane seedling. natural occurrence of entomopathogenic fungi as endophytes of sugarcane ( bacterial communities associated with sugarcane under different agricultural management exhibit a diversity of plant growth-promoting traits and evidence of synergistic effect. near-infrared spectroscopy outperforms genomics for predicting sugarcane feedstock quality traits. knock-down of glucose transporter and sucrose non-fermenting gene in the hemibiotrophic fungus colletotrichum falcatum causing sugarcane red rot. dynamics of co agro-industrial residues for the production of red biopigment by monascus ruber: rice flour and sugarcane molasses. xylo-oligosaccharide microparticles with synbiotic potential obtained from enzymatic hydrolysis of sugarcane straw. regulation of photosystem ii heterogeneity and photochemistry in two cultivars of c three founding ancestral genomes involved in the origin of sugarcane. selection and validation of reference genes by rt-qpcr under photoperiodic induction of flowering in sugarcane (saccharum spp.). sugarcane/peanut intercropping system improves physicochemical properties by changing n and p cycling and organic matter turnover in root zone soil. eight species of  whole genome analysis of sugarcane root-associated endophyte  upgrading from batch to continuous flow process for the pyrolysis of sugarcane bagasse: structural characterization of the biochars produced. characterization of bio-fabricated silver nanoparticles for distinct anti-fungal activity against sugarcane phytopathogens. measuring convergence in the sugarcane industry in china's guangxi province. environmental metal exposures and kidney function of guatemalan sugarcane workers. since sugarcane is a ratoon crop, genome analysis of plant growth-promoting bacteria that exist in its soil rhizosphere, can provide opportunity to better understand their characteristics and use of such bacteria in turn, may especially improve perennial crop productivity. in the present study, genome of two bacterial strains, one each of b. megaterium (bm89) and b. subtilis (bs87), isolated and reported earlier (chandra et al., 2018), were sequenced and characterized. though both strains have demonstrated plant growth promoting properties and enhanced in-vitro plant growth responses, functional annotation and analysis of genes indicated superiority of bs87 as it possessed more plant growth promotion attributable genes over bm89. apart from some common genes, trehalose metabolism, glycine betaine production, peroxidases, super oxide dismutase, cold shock proteins and phenazine production associated genes were selectively identified in bs87 genome indicating better plant growth performances and survival potential under harsh environmental conditions. genes for chitinase, d-cysteine desulfhydrase and γ-aminobutyric acid (gaba), as found in bm89, propose its selective utilization in defense and bio-control measures. concomitant with better settlings' growth, scanning electron micrographs indicated these isolated and characterized bacteria exhibiting healthy colonization within root of sugarcane crop. kegg pathways' assignment also revealed added pathways namely carbohydrate and amino acid metabolism attached to b. subtilis strain bs87, a preferable candidate for bio-fertilizer and its utilization to promote growth of both plant and ratoon crops of sugarcane usually experiencing harsh environmental conditions. an efficient, green and reusable catalyst for organic pollutant wastewater treatment has been a subject of intense research in recent decades due to the limitation of current technologies. cellulose based aerogel composites are considered to be an especially promising candidate for next-generation catalytic material. this project was conducted in order to evaluate the behavior and ability of green and reusable sugarcane bagasse aerogels to remove p-nitrophesnol from waste-water aqueous. co-zeolitic imidazolate framework@ sugarcane bagasse aerogels composite catalysts were successfully prepared via simple in situ synthesis. the structure of hybrid aerogels and their efficient catalyst in peroxymonosulfate (pms) activation for the degradation of p-nitrophenol (pnp) was investigated. as a result, the hybrid aerogels/pms system removed 98.5% of pnp (10mg/l) within 60~70 min, while the traditional water treatment technology could not achieve this. in addition, through a free radical capture experiment and electron paramagnetic resonance (epr), the degradation mechanism of pnp was investigated. further research found that the hybrid aerogels can effectively activate pms to produce sulfate (so4•-) and hydroxyl (oh• ). both of them contributed to the degradation of pnp, and so4•- plays a crucial role in the degradative process. the most important feature of hybrid aerogels can be easily separated from the solution. the obtained results showed that the outer coating structure of cellulose can stabilize co-zif and reduce the dissolution of cobalt ions under complex reaction conditions. moreover, the prepared hybrid aerogels exhibit excellent reusability and are environmentally friendly with efficient catalytic efficiency. this work provides a new strategy for bagasse applications and material reusability. beneficial plant-microbe interactions lead to physiological and biochemical changes that may result in plant-growth promotion. this study evaluated the effect of the interaction between sugarcane and endophytic bacterial strains on plant physiological and biochemical responses under two levels of nitrogen (n) fertilization. six strains of endophytic bacteria, previously selected as plant growth-promoting bacteria (pgpb), were used to inoculate sugarcane mini stalks, with and without n fertilization. after 45 days, biomass production; shoot nutrient concentrations; foliar polyamine and free amino acid profiles; activities of nitrate reductase and glutamine synthase; and the relative transcript levels of the gs1, gs2, and shr5 genes in sugarcane leaves were determined. all six endophytic strains promoted sugarcane growth, increasing shoot and root biomass, plant nutritional status, and the use efficiency of most nutrients. the inoculation-induced changes at the biochemical level altered the foliar free amino acid and polyamine profiles, mainly regarding the relative concentrations of citrulline, putrescine, glycine, alanine, glutamate, glutamine, proline, and aspartate. the transcription of gs1, gs2, and shr5 was higher in the n fertilized seedlings, and almost not altered by endophytic bacterial strains. the endophytic strains promoted sugarcane seedlings growth mainly by improving nutrient efficiency. this improvement could not be explained by their ability to induce the production of amino acid and polyamine composts, or gs1, gs2, and shr5, showing that complex interactions may be associated with enhancement of the sugarcane seedlings' performance by endophytic bacteria. the strains demonstrated biotechnological potential for sugarcane seedling production. the natural occurrence of entomopathogenic fungal endophytes in sugarcane (saccharum officinarum) and in soil samples from sugarcane fields was evaluated in chikwawa district, southern malawi. fungi from soil were isolated by baiting using galleria mellonella larva. fungal endophytes were isolated from surface-sterilized plant tissue sections. forty-seven isolates resembled the genus beauveria, 9 isolates were metarhizium, and 20 isolates were isaria. there was no significant difference in the number and type of fungal isolates collected from soil and from plant tissue. there was, however, a significant difference in the part of the plant where fungal species were isolated, which fungal species were isolated, and the number of fungal species isolated at each location. phylogenetic analysis of 47 beauveria isolates based on dna sequencing of the bloc intergenic region indicated that these isolates all belonged to b. bassiana and aligned with sequences of b. bassiana isolates of african and neotropical origin. the malawian b. bassiana isolates formed a distinct clade. no larvae died from infestation by multiple fungi. to the best of our knowledge, this is the first report of b. bassiana and isaria spp. occurring naturally as endophytes in sugarcane. further, it is the first report of b. bassiana, isaria spp., and metarhizium spp. in the soil of sugarcane fields in africa. plant-associated microbiomes have been a target of interest for the prospection of microorganisms, which may be acting as effectors to increase agricultural productivity. for years, the search for beneficial microorganisms has been carried out from the characterization of functional traits of growth-promotion using tests with a few isolates. however, eventually, the expectations with positive results may not be realized when the evaluation is performed in association with plants. in our study, we accessed the cultivable sugarcane microbiome under two conditions of agronomic management: organic and conventional. from the use of a new customized culture medium, we recovered 944 endophytic and epiphytic bacterial communities derived from plant roots, stalks, leaves, and rhizospheric soil. this could be accomplished by using a large-scale approach, initially performing an in planta (cynodon dactylon) screening process of inoculation to avoid early incompatibility. the inoculation was performed using the bacterial communities, considering that in this way, they could act synergistically. this process resulted in 38 candidate communities, 17 of which had higher indole-3-acetic acid (iaa) production and phosphate solubilization activity and, were submitted to a new in planta test using brachiaria ruziziensis and quantification of functional traits for growth-promotion and physiological tests. enrichment analysis of selected communities has shown that they derived mainly from epiphytic populations of sugarcane stalks under conventional management. the sequencing of the v3-v4 region of the 16s rrna gene revealed 34 genera and 24 species distributed among the phylum proteobacteria, bacteroidetes, firmicutes, and actinobacteria. we also observed a network of genera in these communities where the genus chryseobacterium stands out with a greater degree of interaction, indicating a possible direct or indirect role as a keystone taxon in communities with plant-growth promotion capacities. from the results achieved, we can conclude that the approach is useful in the recovery of a set of sugarcane bacterial communities and that there is, evidence of synergistic action providing benefits to plants, and that they are compatible with plants of the same family (poaceae). thus, we are reporting the beneficial bacterial communities identified as suitable candidates with rated potential to be exploited as bioinoculants for crops. the main objectives of this study were to evaluate the prediction performance of genomic and near-infrared spectroscopy (nir) data and whether the integration of genomic and nir predictor variables can increase the prediction accuracy of two feedstock quality traits (fiber and sucrose content) in a sugarcane population (saccharum spp.). the following three modeling strategies were compared: m1 (genome-based prediction), m2 (nir-based prediction), and m3 (integration of genomics and nir wavenumbers). data were collected from a commercial population comprised of three hundred and eighty-five individuals, genotyped for single nucleotide polymorphisms and screened using nir spectroscopy. we compared partial least squares (pls) and bayesb regression methods to estimate marker and wavenumber effects. in order to assess model performance, we employed random sub-sampling cross-validation to calculate the mean pearson correlation coefficient between observed and predicted values. our results showed that models fitted using bayesb were more predictive than pls models. we found that nir (m2) provided the highest prediction accuracy, whereas genomics (m1) presented the lowest predictive ability, regardless of the measured traits and regression methods used. the integration of predictors derived from nir spectroscopy and genomics into a single model (m3) did not significantly improve the prediction accuracy for the two traits evaluated. these findings suggest that nir-based prediction can be an effective strategy for predicting the genetic merit of sugarcane clones. red rot caused by colletotrichum falcatum, is one of the economically important disease of sugarcane and breeding for resistant varieties is considered to be the major solution to manage the disease. however, breakdown of red rot resistance become usual phenomenon due to development of newer races by culture adaptation on newly released varieties. hence it is needed to characterize the genes responsible for pathogen virulence in order to take care of host resistance or to manage the disease by other methods. the transcript studies gave foundation to characterize the huge number of pathogenicity determinants and their role in pathogenesis. here we studied role of two important genes viz., glucose transporter (gt) and sucrose non-fermenting1 (snf1) during pathogenesis of c. falcatum, which said to be involved in carbon source metabolism. sugar metabolism has a vital role in disease progression of c. falcatum by regulating their cell growth, metabolism and development of the pathogen during various stages of infection. the present study was aimed to find out the role of gt and snf1 genes in response to pathogenicity by rna silencing (rnai) approach. knock-down of the target pathogenicity gene homologs in standard c. falcatum isolate cf671 was carried out by amplifying sense and antisense fragments of targets individually using psilent-1 vector. the expression cassette was cloned into the binary vector pcambia1300 followed by fungal transformation through agarobacterium mediated transformation. resulted mutants of both the genes showed less virulence compared to wild type isolate. simultaneously, both the mutants did not produce spores. moreover, the molecular confirmation of the mutants displayed the expression of hygromycin gene with reduced expression of the target gene during host-pathogen interaction. knockdown of the pathogenicity related genes (gt and snf1) by rnai approach corroborate the possible role of the genes in causing the disease. in this study, co2 exchange over sugarcane and wheat growing season was quantified by continuous measurement of co2 fluxes using eddy covariance (ec) system from january 2014 to june 2015. we also elaborated on the response of co2 fluxes to environmental variables. the results show that the ecosystem has seasonal and diurnal dynamics of co2 with a distinctive u-shaped curve in both growing seasons with maximal co2 absorption reaching up to -8.94 g c m-2 day-1 and -6.08 g c m-2 day-1 over sugarcane and wheat crop, respectively. the ecosystem as a whole acted as a carbon sink during the active growing season while it exhibits a carbon source prior to sowing and post-harvesting of crops. the cumulative net ecosystem exchange (nee), gross primary productivity (gpp), and ecosystem respiration (reco) were -923.04, 3316.65, and 2433.18 g c m-2 over the sugarcane growing season while the values were -192.30, 621.47, and 488.34 g c m-2 over the wheat growing season. the sesbania (green manure) appeared to be a carbon source once it is incorporated into soil. the response of day-time nee to photosynthetically active radiation (par) under two vapor pressure deficit (vpd) sections (0-20 h pa and 20-40 h pa) seems more effective over sugarcane (r2 = 0.41-0.61) as compared to the wheat crop (r2 = 0.25-0.40). a decrease in net co2 uptake was observed under higher vpd conditions. similarly, night-time nee was exponentially related to temperature at different soil moisture conditions and showed higher response to optimum soil moisture conditions for sugarcane (r2 = 0.87, 0.33 ≤ swc < 0.42 m3 m-3) and wheat (r2 = 0.75, 0.31 ≤ swc < 0.37 m3 m-3) crop seasons. the response of daily averaged nee to environmental variables through path analysis indicates that par was the dominant predictor with the direct path coefficient of -0.65 and -0.74 over sugarcane and wheat growing season, respectively. satellite-based gpp products from moderate resolution imaging spectroradiometer (gppmod) and vegetation photosynthetic model (gppvpm) were also compared with the gpp obtained from ec (gppec) technique. the seasonal dynamics of gppec and gppvpm agreed well with each other. this study covers the broad aspects ranging from micro-meteorology to remote sensing over c4-c3 cropping system. three culture media were studied for red pigment production by monascus ruber in submerged cultivation: rice flour (20 g l-1), sugarcane molasses (30 g l-1), and, finally, molasses + rice flour (10 g l-1+10 g l-1); all culture media were added of 5 g l-1 glycine as nitrogen source. rice flour showed pigment production of 7.05 ua510nm and molasses 5.08 ua510nm, and the mixture of rice flour and molasses showed the best result of 16.38 ua510nm. molasses culture presented good results for cell biomass production of 11.09 g l-1. with these results, it was observed that one substrate presented good pigment production (rice flour) and another attained better results for cell biomass growth (molasses), and a third medium containing 10 g l-1 of rice flour + 10 g l-1 of molasses was formulated. the results for this mixture showed satisfactory results, with global pigment productivity of 0.097 ua510nm h-1 and maximum productivity rate of 0.17 ua510nm h-1. the high production and productivity obtained for the mixture of rice flour and molasses indicated that the production of red pigment by submerged fermentation, using the mixture of these low-cost culture media, may be promising in terms of commercial production. synbiotic formulations and microencapsulation techniques have been explored in food industries to guarantee the viability of probiotic organisms; playing an important role in microbiota balance. microparticles of alginate, gelatin and xylo-oligosaccharides (xos) were produced by external gelation with the purpose of enhancing the survival rate of the probiotic l. acidophilus. xos was obtained through enzymatic hydrolysis of xylan extracted from sugarcane straw, achieving more than 70% conversion and used for microparticle preparation. microparticles containing 3% xos provided greater cell protection during exposure to the gastrointestinal tract and during refrigerated storage; keeping 97.86 ± 0.44% of viability during 28 days of storage and enabling 87.50 ± 0.02% survival after digestive simulation. however, particles without xos showed 84.49 ± 0.59% of viability after storage and 68.45 ± 0.03% after digestion assay. these results lead to promising applications in synbiotic and functional food formulations comprised of components requiring extended shelf-life, protection from gastrointestinal conditions and gradual bioactive delivery. in subtropical regions, chilling stress is one of the major constraints for sugarcane cultivation, which hampers yield and sugar production. two recently released sugarcane cultivars, moderately chilling tolerant guitang 49 and chilling tolerant guitang 28, were selected. the experiments were conducted in the controlled environment, and seedlings were exposed to optimum (25°c/15°c), chilling (10°c/5°c), and recovery (25°c/15°c) temperature conditions. psii heterogeneity was studied in terms of reducing side and antenna size heterogeneity. under chilling, reducing side heterogeneity resulted in increased number of qb non-reducing centers, whereas antenna side heterogeneity resulted in enhanced number of inactive β centers in both cultivars, but the magnitude of change was higher in guitang 49 than guitang 28. furthermore, in both cultivars, quantum efficiency of psii, status of water splitting complex, and performance index were adversely affected by chilling, along with reduction in net photosynthesis rate and nighttime respiration and alterations in leaf optical properties. the extents of negative effect on these parameters were larger in guitang 49 than in guitang 28. these results reveal a clear differentiation in psii heterogeneity between differentially chilling tolerant cultivars. based on our studies, it is concluded that psii heterogeneity can be used as an additional non-invasive and novel technique for evaluating any type of environmental stress in plants. modern sugarcane cultivars (saccharum spp.) are high polyploids, aneuploids (2n = ~12x = ~120) derived from interspecific hybridizations between the domesticated sweet species saccharum officinarum and the wild species s. spontaneum. to analyse the architecture and origin of such a complex genome, we analysed the sequences of all 12 hom(oe)ologous haplotypes (bac clones) from two distinct genomic regions of a typical modern cultivar, as well as the corresponding sequence in miscanthus sinense and sorghum bicolor, and monitored their distribution among representatives of the saccharum genus. the diversity observed among haplotypes suggested the existence of three founding genomes (a, b, c) in modern cultivars, which diverged between 0.8 and 1.3 mya. two genomes (a, b) were contributed by s. officinarum; these were also found in its wild presumed ancestor s. robustum, and one genome (c) was contributed by s. spontaneum. these results suggest that s. officinarum and s. robustum are derived from interspecific hybridization between two unknown ancestors (a and b genomes). the a genome contributed most haplotypes (nine or ten) while the b and c genomes contributed one or two haplotypes in the regions analysed of this typical modern cultivar. interspecific hybridizations likely involved accessions or gametes with distinct ploidy levels and/or were followed by a series of backcrosses with the a genome. the three founding genomes were found in all s. barberi, s. sinense and modern cultivars analysed. none of the analysed accessions contained only the a genome or the b genome, suggesting that representatives of these founding genomes remain to be discovered. this evolutionary model, which combines interspecificity and high polyploidy, can explain the variable chromosome pairing affinity observed in saccharum. it represents a major revision of the understanding of saccharum diversity. although reference genes have previously been used in the expression analysis of genes involved in sugarcane flowering they had not been experimentally validated for stability and consistency of expression between different samples over a wide range of experimental conditions. here we report the analysis of candidate reference genes in different tissue types, at different temporal time-points, in both short and long day photoperiodic treatments. the stability of the candidate reference genes in all conditions was evaluated with normfinder, bestkeeper, and reffinder algorithms that complement each other for a more robust analysis. as the normfinder algorithm was more appropriate for our experimental conditions, greater emphasis was placed on normfinder when choosing the most stable genes. ubq1 and tub were shown to be the most stable reference genes to use for normalizing rt-qpcr gene expression data during floral induction, whilst 25srrna1 and gapdh were the least stable. their use as a reference gene pair was validated by analyzing the expression of two differentially expressed target genes (pil5 and lhp1). the ubq1/tub reference genes combination was able to reveal small significant differences in gene expression of the two target genes that were not detectable when using the least stable reference gene combination. these results can be used to inform the choice of reference genes to use in the study of the sugarcane floral induction pathway. our work also demonstrates that both pil5 and lhp1 are significantly up-regulated in the initial stages of photoperiodic induction of flowering in sugarcane. the sugarcane/peanut intercropping system is a specific and efficient cropping pattern in south china. intercropping systems change the bacterial diversity of soils and decrease disease rates. it can not only utilized light, heat, water and land resources efficiently, but also increased yield and economic benefits of farmers. we determined soil nutrients, enzymes and microbes in sugarcane/peanut intercropping system, and analyzed relevance of the soil physicochemical properties and the genes involved in n and p cycling and organic matter turnover by metagenome sequencing. the results showed that sugarcane/peanut intercropping significantly boosted the content of total nitrogen, available phosphorus, total potassium, organic matter, ph value and bacteria and enhanced the activity of acid phosphatase compared to monocropping. especially the content of available nitrogen, available phosphorus and organic matter increased significantly by 20.1%, 65.3% and 56.0% in root zone soil of ip2 treatment than monocropping treatment. the content of available potassium and microbial biomass carbon, as well as the activity of catalase, sucrase and protease, significantly decreased in intercropping root zone soil. intercropping resulted in a significant increase by 7.8%, 16.2% and 23.0% in is, ip1 and ip2, respectively, of the acid phosphatase content relative to ms. metagenomic analysis showed that the pathways involved in carbohydrate and amino acid metabolism were dominant and more abundant in intercropping than in monocropping. moreover, the relative abundances of genes related to n cycling (glna, glud1_2, nirk), p cycling (phor, phob) and organic matter turnover (prdx2_4) were higher in the intercropping soil than in the monocropping soil. the relative abundance of glud1_2 and phor were 25.5% and 13.8% higher in the ip2 treatment respectively,and bgix was higher in is treatment compared to the monocropping treatment. genes that were significantly related to phosphorus metabolism and nitrogen metabolism (treh, kate, gudb) were more abundant in intercropping than in monocropping. the results of this study indicate that the intercropping system changed the numbers of microbes as well as enzymes activities, and subsequently regulate genes involved in n cycling, p cycling and organic matter turnover. finally, it leads to the increase of nutrients in root zone soil and improved the soil environment. sugarcane mosaic virus (scmv) was detected by reverse-transcription polymerase chain reaction in eight different species of the poaceae family in the everglades agricultural area (eaa) of south florida: broadleaf signalgrass (urochloa platyphylla), columbus grass (sorghum almum), goosegrass (eleusine indica), maize (zea mays), sorghum (sorghum bicolor), st. augustine grass (stenotaphrum secundatum), southern crabgrass (digitaria ciliaris), and sugarcane (saccharum interspecific hybrids). based on their coat protein (cp) gene sequence, 62 isolates of scmv from florida and 29 worldwide isolates representing the known genetic diversity of this virus were distributed into eight major phylogenetic groups. scmv isolates infecting columbus grass, maize, and sorghum in florida formed a unique group whereas virus isolates infecting sugarcane in the usa (florida and louisiana) clustered with isolates from other countries. based on the entire genome coding region, scmv isolates infecting sugarcane in florida were closest to virus isolates infecting sorghum species or st. augustine grass. virus isolates from columbus grass, st. augustine grass, and sugarcane showed different virulence patterns after mechanical inoculation of columbus grass, st. augustine grass, and sugarcane plants, thus proving that these isolates were different pathogenic strains. sugarcane was symptomless and tested negative for scmv by tissue blot immunoassay after inoculation with crude sap from scmv-infected columbus grass, indicating that columbus grass was not a reservoir for scmv infecting sugarcane in the eaa. close cp sequence identity between isolates of scmv from columbus grass, maize, and sorghum suggested that the same virus strain was naturally spreading among these three plants in south florida. sugarcane smut is a significant fungal disease that causes a major loss in sugar yield and quality. in this study, we isolated an endophytic strain b18 from a sugarcane root, which showed plant growth-promotion, hydrolytic enzyme production, antifungal activity against sugarcane pathogens (sporisorium scitamineum, ceratocystis paradoxa, fusarium verticillioides), and the presence of nifh, acds, and antibiotic genes (hcn, prn, and phca) under in vitro conditions. biolog(r) phenotypic profiling of b18 established its ability to use various carbon and nitrogen sources and tolerate a range of ph and osmotic and temperature stresses. whole-genome analysis of b18, identified as pseudomonas aeruginosa, showed that it consists of a single circular chromosome of 6,490,014 bp with 66.33% gc content. genome annotation has identified 5,919 protein-coding genes, and 65 trna, and 12 rrna genes. the p. aeruginosa b18 genome encodes genes related to ethylene, nitrogen (nifu, norbcderq, gltbdps, and aatjmpq), and phosphate (pstabcs and phobdhru) metabolism and produce indole-3-acetic acid and siderophores. this also includes genes encoding hydrolases and oxidoreductases, those associated with biocontrol mechanisms (hcnabc, phza_b, phzdefgms, and pcha), colonization (mincde and lysc), and biofilm formation (efp, hfq, flgbcdefghi, and motab), and those associated with metabolism of secondary metabolites. collectively, these results suggest a role for p. aeruginosa b18 in plant growth enhancement and biocontrol mechanisms. the p. aeruginosa b18 strain was found to be an efficient colonizer in sugarcane; it can improve growth through modulation of plant hormone production and enhanced host-plant resistance to smut pathogen s. scitamineum in a smut-susceptible sugarcane variety (yacheng71-374). these biocontrol and plant growth promotion properties of p. aeruginosa b18 area are discussed in this report. the waste from agriculture can be used for biochar production by the pyrolysis process. the present work aimed was to produce sugarcane bagasse biochars using different temperatures and processes (batch and pilot-scale continuous flow). the samples were characterized by ftir, functional group pka, elemental analysis, zeta potential, raman spectroscopy, epr, and sem. the ftir spectra showed bands around 1400-1650 cm-1 corresponded to vibrations of cc bonds and pka revealed the presence of carboxylic acids (pka ≤5) and lactones (pka ~5-9). the elemental analyses (h/c ~ 0.31) and raman spectra (id/ig ~ 0.55) confirmed greater carbonization and less structural disorder of the material produced using the continuous flow process. sem images showed that the biochar morphologies were similar to that of the precursor biomass, with the formation of pores. the continuous flow process is a promising technique for the production of biochars with high carbon contents and aromatic structures, as well as lower defect degrees, compared to biochars produced using a batch process. advanced research, development, and application of silver nanoparticles is proceeding in recent times due to their incredible utilization in various fields. present study was focused on the production, characterization, and antifungal activities of silver nanoparticles (agnps). an environment friendly extracellular biosynthetic approach was adopted to produce the agnps by using bacteria, fungi, and sugarcane husk. agents used for reduction of silver to nanoparticles were taken from culture filtrate of plant growth promoting bacteria, fusarium oxysporum and supernatant extract of sugarcane husk. nanoparticles were also characterized by scanning electron microscopy (sem). synthesis of colloidal agnps was observed by uv-visible diffused reflectance spectroscopy (uv-vis drs). primary peak of surface plasmon resonance band was noticed around 339.782, 336.735, and 338.258 nm for bacterial, fungal, and sugarcane husk produced agnps. structure of all biologically produced nanoparticles were crystalline cubic with nano size of 45.41, 49.06, and 42.75 nm for bacterial, fungal, and sugarcane husk-based nanoparticles, respectively as calculated by debye-scherrer equation using xrd. fourier transform infrared spectroscopy (ftir) analysis revealed the presence of various compounds that aid in the reduction, capping, and stability of agnps. the antifungal activity of agnps was also investigated for sugarcane fungal pathogens colletotricum falcatum and fusarium moniliforme. all nanoparticles exhibit prominent antifungal activities. maximum zone of fungal inhibition was noticed about 18, 19, and 21 mm for c. falcatum while 21, 20, and 24 mm for f. moniliforme in case of bacterial, fungal, and plant-based nanoparticles (15 ppm), respectively. best fungal inhibition was observed under application of sugarcane husk based agnps. moreover, biologically produced agnps responded better towards the suppression of f. moniliforme in comparison to c. falcatum. mentioned sources in present study can be ecofriendly nano-factories for biosynthesis of agnps and mankind should benefit from their commercial application. using the agricultural industry convergence framework, we analyzed the main sugarcane producing areas in china's guangxi province from 2010 to 2017. we selected the key elements on sugarcane industry convergence and on the benefit derived from industry convergence and development. we selected six secondary indexes and 18 tertiary indexes and used them to construct an overarching index system for measuring the convergence degree of sugarcane in the primary, secondary, and tertiary industries. we calculated the index's system weight using the ahp and entropy method, and then the integrated development indicator of the sugarcane industry. finally, we analyzed the convergence the development stage, achievements, and benefits of the sugarcane industry, as well as the shortage of convergence development. our findings can provide guidance for the future development of the sugarcane industry in china from the perspective of industry convergence. exposure to environmental metals can cause nephrotoxicity. there is an international epidemic of chronic kidney disease of unknown cause (ckdu). whether metal exposures contribute to kidney dysfunction in populations at risk for ckdu remains unresolved. urinary metals (arsenic, cadmium, nickel, and uranium) were examined in 222 sugarcane cutters in guatemala at three time points over 1 year. we explored the relationships between metal concentrations and markers of kidney function using multivariable linear mixed-effect models. arsenic, cadmium, and nickel were detected in the majority of the 340 urine samples and were generally within limits previously considered to be nonnephrotoxic. nevertheless, higher urine cadmium was inversely associated with estimated glomerular filtration rate (egfr) (β: -4.23, 95% confidence interval [ci]: -6.92, -1.54) and positively associated with neutrophil gelatinase-associated lipocalin (ngal) (β: 2.92, 95% ci: 1.20, 4.64). higher urine arsenic was also inversely associated with egfr (β: -4.36, 95% ci: -7.07, -1.64). our findings suggest that exposures to metals, including cadmium and arsenic, might contribute to kidney toxicity seen in workers at risk for ckdu. these findings are consistent with the potential for metal nephrotoxicity at lower than expected levels in the setting of manual work in a very hot environment. \n",
      "Euclidean Distance : 15.602226257324219\n",
      "\n",
      "\n",
      "Document: [determination of chlormezanone in serum by gas chromatography].  \n",
      "Euclidean Distance : 15.626596450805664\n",
      "\n",
      "\n",
      "Document: [effects of lime and ammonium carbonate fumigation coupled with bio-organic fertilizer application on banana fusarium wilt and bacterial community.] natural inspired carboxymethyl cellulose (cmc) doped with ammonium carbonate (ac) as biopolymer electrolyte. lime and ammonium carbonate fumigation coupled with bio-organic fertilizer application steered banana rhizosphere to assemble a unique microbiome against panama disease. lactulose production from cheese whey using recyclable catalyst ammonium carbonate. structure, hydrogen bonding and thermal expansion of ammonium carbonate monohydrate. effects of ammonium carbonate pretreatment on the enzymatic digestibility and structural features of rice straw. ammonium carbonate is more attractive than apple and hawthorn fruit volatile lures to rhagoletis pomonella (diptera: tephritidae) in washington state. kinetic study of uranium residue dissolution in ammonium carbonate media. preparation and characterisation of novel spray-dried nano-structured para-aminosalicylic acid particulates for pulmonary delivery: impact of ammonium carbonate on morphology, chemical composition and solid state. taking banana continuous planting soil with high banana fusarium wilt disease incidence as a test site, we examined the effect of lime and ammonium carbonate fumigation coupled with bio-organic fertilizer on the suppression of banana fusarium wilt disease and the structure and composition of bacterial community, using real-time quantitative pcr and high-throughput sequencing. the results showed that the disease incidence was reduced by 13.3% and 21.7% in the treatments of laof (lime and ammonium carbonate fumigation coupled with organic fertilizer) and labf (lime and ammonium carbonate fumigation coupled with bio-organic fertilizer), respectively, compared with of (application of organic fertilizer without fumigation), while the copy number of fusarium was decreased by 22.4% and 33.0%, respectively. compared with non-fumigation treatment, lime and ammonium fumigation coupled with different fertilizer applications significantly reduced bacteria richness and diversity, with different community structure, while fumigation had a decisive effect on bacterial community composition. bacterial richness and diversity of labf were lower than those of other treatments, while microbial community structure was clearly disparate from other treatments. compared with non-fumigation treatment, the relative abundance of mizugakiibacter, brucella, and rhodanobacter were significantly improved in the fumigation coupled with different fertilization treatments. those three genera in labf were higher than those in laof, with significant differences for the relative abundances of mizugakiibacter and brucella. therefore, fumigation combined with bio-organic fertilizer application could reduce the copy number of pathogen, alter soil bacterial community structure and stimulate beneficial bacteria in the resident soil, and thus reduce the occurrence of banana fusarium wilt. green and safer materials in energy storage technology are important right now due to increased consumption. in this study, a biopolymer electrolyte inspired from natural materials was developed by using carboxymethyl cellulose (cmc) as the core material and doped with varied ammonium carbonate (ac) composition. x-ray diffraction (xrd) shows the prepared cmc-ac electrolyte films exhibited low crystallinity content, x (~30%) for sample ac7. a specific wavenumber range between 900-1200 cm-1 and 1500-1800 cm-1 was emphasized in fourier transform infrared (ftir) testing, as this is the most probable interaction to occur. the highest ionic conductivity, σ of the electrolyte system achieved was 7.71 × 10-6 scm-1 and appeared greatly dependent on ionic mobility, µ and diffusion coefficient, d. the number of mobile ions, η, increased up to the highest conducting sample (ac7) but it became less prominent at higher ac composition. the transference measurement, t showed that the electrolyte system was predominantly ionic with sample ac7 having the highest value (t = 0.98). further assessment also proved that the h+ ion was the main conducting species in the cmc-ac electrolyte system, which presumably was due to protonation of ammonium salt onto the complexes site and contributed to the overall ionic conductivity enhancement. microbiome plays a key role in determining soil suppressiveness against invading pathogens. our previous study revealed that microbial community of bulk soil could be manipulated by lime and ammonium bicarbonate fumigation followed by biofertilizer application. however, the assembly of microbial community suppressive to banana panama disease in the rhizosphere is still unclear. in this study, we used high-throughput sequencing and quantitative pcr to explore the assembly of rhizosphere microbiome associated with banana panama disease suppression in a two-seasonal pot experiment. we found biofertilizer applied to lime and ammonium bicarbonate fumigated soil significantly (p < 0.05) reduced the abundance of rhizosphere fusarium oxysporum compared to biofertilizer applied to non-fumigated soil. principal coordinate analysis revealed that biofertilizer applied to lime and ammonium bicarbonate fumigated soil re-shaped the rhizosphere bacterial community composition by increasing the phylogenetic relatedness, and stimulating indigenous microbes, for example, gemmatimonas, sphingomonas, pseudomonas, lysobacter and bacillus. co-occurrence analysis revealed that potential species involved in disease suppression were more interrelated in disease-suppressive soils. taken together, lime and ammonium bicarbonate fumigation followed by biofertilizer application could induce banana rhizosphere to assemble beneficial microbes dominated consortia to suppress banana panama disease. ammonium carbonate ((nh4)2co3) was used as an alkaline catalyst of lactulose production from cheese whey. maximum yield of 29.6% was obtained at reaction time of 28.44 min, (nh4)2co3 of 0.76% at 97°c. during reaction, (nh4)2co3 was fully decomposed to nh3 and co2, and these gases were recovered. to boost up nh3 recovery, various methods such as heating, aeration, and ph adjustment were applied. the optimal condition for the purpose of nh3 retrieval was temperature of up to 60°c alongside aeration. easy separation and recovery make (nh4)2co3 a catalyst alternative to common alkaline chemicals especially for the weak alkaline reaction. we have determined the crystal structure of ammonium carbonate monohydrate, (nh4)2co3·h2o, using laue single-crystal diffraction methods with pulsed neutron radiation. the crystal is orthorhombic, space group pnma (z = 4), with unit-cell dimensions a = 12.047 (3), b = 4.453 (1), c = 11.023 (3) å and v = 591.3 (3) å(3) [ρcalc = 1281.8 (7) kg m(-3)] at 10 k. the single-crystal data collected at 10 and 100 k are complemented by x-ray powder diffraction data measured from 245 to 273 k, raman spectra measured from 80 to 263 k and an athermal zero-pressure calculation of the electronic structure and phonon spectrum carried out using density functional theory (dft). we find no evidence of a phase transition between 10 and 273 k; above 273 k, however, the title compound transforms first to ammonium sesquicarbonate monohydrate and subsequently to ammonium bicarbonate. the crystallographic and spectroscopic data and the calculations reveal a quite strongly hydrogen-bonded structure (ehb ≃ 30-40 kj mol(-1)), on the basis of h...o bond lengths and the topology of the electron density at the bond critical points, in which there is no free rotation of the ammonium cation at any temperature. the barrier to free rotation of the ammonium ions is estimated from the observed librational frequency to be ∼ 36 kj mol(-1). the c-axis exhibits negative thermal expansion, but the thermal expansion behaviour of the a and b axes is ormal. rice straw was pretreated with ammonium carbonate ((nh₄)₂co₃), a major intermediate of ammonia-based carbon capture process, and evaluated for the effects of critical pretreatment parameters including (nh₄)₂co₃ concentration (5-25%), temperature (60-90°c), and reaction time (4-24 h) on enzymatic digestibility. pretreatment of rice straw at 80°c for 12 h using 20% (nh₄)₂co₃ and 1:10 solid to liquid ratio resulted in enzymatic digestibility of 72.2%, which was higher than that pretreated with the same moles of aqueous ammonia. we also investigated physical characteristics of pretreated rice straw, including surface area, pore volume and size, crystallinity, and scanning electron microscopy image. the ammonium carbonate pretreatment process, as a novel pretreatment technique, enhanced enzymatic hydrolysis of lignocellulose by altering structural features. the apple maggot fly, rhagoletis pomonella (walsh), is an introduced, quarantine pest of apple (malus domestica borkhausen) in the pacific northwest of the united states. in the eastern united states where the fly is native, fruit volatiles have been reported to be more attractive than ammonia compounds to r. pomonella. however, the opposite may be true in the western united states. here, we determined whether newly identified western apple and western hawthorn fruit volatiles are more attractive than ammonium carbonate (ac) to r. pomonella in apple, black hawthorn, and ornamental hawthorn trees in western washington state. in all three host trees, sticky red sphere or yellow panel traps baited with ac generally caught more flies than traps baited with lures containing the four newly developed fruit blends (modified eastern apple, western apple, western ornamental hawthorn, and western black hawthorn) or two older blends (eastern apple and eastern downy hawthorn). fruit volatiles also displayed more variation among trapping studies conducted at different sites, in different host trees, and across years than ac. the results imply that traps baited with ac represent the best approach to monitoring r. pomonella in washington state. the purpose of this study was to determine the kinetics of the dissolution of a uranium residue in ammonium carbonate media. the residue is generated in the production of medical isotopes. the effects of parameters, such as varying peroxide and carbonate concentrations, dissolution time as well as temperature on the extraction rate have been separately studied. results indicate complete dissolution of the residue at 60 °c, after 30 min, in ammonium carbonate solution enriched with hydrogen peroxide. the yield and rate of uranium extraction were found to increase as a function of both temperature, in the range of 25-60 °c, and hydrogen peroxide concentration. the extraction process was governed by chemical reaction as the activation energy was found to be 45.5 kj/mol. the order of reaction with respect to uranium concentration was found to be approximately first order. the objective of this work was to spray dry p-aminosalicylic acid (pas) and its ammonium salt and to investigate the impact of the pore-forming agent, ammonium carbonate (ac), on the morphological, aerodynamic and physicochemical properties of the resulting powders. microparticles were prepared by spray drying from ethanol/water solvent systems. their solid-state properties were evaluated by scanning electron microscopy, powder x-ray diffraction, differential scanning calorimetry, thermogravimetric analysis and in-vitro deposition, using the twin impinger. the physicochemical properties of pas were altered on spray drying with ac and a new solid state was produced. the solution composition impacted on the morphology of the resulting powders, which ranged from irregular crystal agglomerates to spherical crystal clusters and porous microparticles. the chemical composition, structure and morphology were dependent on process inlet temperature, low inlet temperatures resulting in a novel solid of stoichiometry; pas : ammonia : water, 2 : 1 : 0.5. at higher temperatures pure pas was obtained. in-vitro deposition studies showed an increase in emitted dose from spray dried drug, relative to the micronised pas. under appropriate process conditions ac interacts with the acidic pas, resulting in the formation of a novel solid-state drug phase. spray-dried pas powders have potential for pulmonary delivery. \n",
      "Euclidean Distance : 15.629270553588867\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. \n",
      "Euclidean Distance : 15.636975288391113\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. \n",
      "Euclidean Distance : 15.636975288391113\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. \n",
      "Euclidean Distance : 15.636975288391113\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. \n",
      "Euclidean Distance : 15.636975288391113\n",
      "\n",
      "\n",
      "Document: nutrient digestibility of a vegetarian diet with or without the supplementation of feather meal and either corn meal, fermented rye or rye and its effect on fecal quality in dogs. [molecular markers in the genetic analysis of crossability of bread wheat with rye]. the consensus rye microsatellite map with est-ssrs transferred from wheat. isolation of saturated alkylresorcinols from rye grains by countercurrent chromatography. early prediction of biomass in hybrid rye based on hyperspectral data surpasses genomic predictability in less-related breeding material. bioaccessibility of folate in faba bean, oat, rye and wheat matrices. impact of a fermented high-fiber rye diet on  the polyphenol profile and antioxidant potential of irradiated rye grains. impact of ultra-fine milling and air classification on biochemical and techno-functional characteristics of wheat and rye bran. identification of rye b chromosome-associated peptides by mass spectrometry. effect of crude oil on growth, oxidative stress and response of antioxidative system of two rye ( structural rearrangements in wheat (1bs)-rye (1rs) recombinant chromosomes affect gene dosage and root length. wheat, rye, and barley genomes can associate during meiosis in newly synthesized trigeneric hybrids. effect of complete replacement of dry-rolled corn with unprocessed rye on growth performance, efficiency of dietary net energy use, and carcass traits of finishing heifers. influence of rye cover cropping on denitrification potential and year-round field n genetic mapping of pollen fertility restoration qtls in rye (secale cereale l.) with cms pampa. extensive evaluation via blinded testing of an uhplc-ms/ms method for quantitation of ten ergot alkaloids in rye and wheat grains. characterization of a new gene for resistance to wheat powdery mildew on chromosome 1rl of wild rye secale sylvestre. rye bread defects: analysis of composition and further influence factors as determinants of dry-baking. some characteristics of crossing over in induced recombination between chromosomes of wheat and rye. cereals with low environmental input like rye and animal by-products which cannot be used for human food like feather meal are receiving growing interest as sustainable feed sources. thus, eight beagle dogs were included in a 4 × 4 latin square design and received a vegetarian basic diet or the same diets supplemented with hydrolyzed feather meal (2.7%) and either 20.1% of corn meal, 60.4% of fermented rye or 20.1% of rye as is basis (moisture content of the diets about 42%). compared to other groups the dry matter (dm) content of feces from dogs fed the basic diet was higher (30.0%, p < 0.05), while dogs fed the basic diet + rye had the lowest dm-content (26.5%, p < 0.05). however, the fecal scores were considered to be within an acceptable range (well-formed and firm). starch digestibility was lower (p < 0.05) for dogs fed the basic diet + corn meal. the dogs showed a high and identical acceptance (scoring of food intake) of the experimental diets. as a comparable quality of feces and a high nutrient digestibility were observed when rye was used in the experimental diets-it can be considered an alternative carbohydrate source in dog foods. bread wheat (triticum aestivum l.), the varieties of which are widely used for the grain production, is difficultly crossable with related species of triticeae dum. this factor limits the chance of introduction of alien genetic material into the wheat gene pool and the possibility of new varieties breeding with good adaptation to adverse environmental factors. the crossability between wheat and related species is controlled by kr1-kr4 genes (crossability with rye, hordeum and aegilops spp.) and the skr gene (suppressor of crossability). skr and kr1 have the largest influence on the trait. in the case of the recessive alleles, these genes do not function and the quantity of hybrid seeds after pollination with alien species can achieve more than 50 %. skr is located on 5bs between the gbr0233 and xgwm234 markers, closely linked with the markers xcfb341, tglc2 and gene12. kr1 was mapped on 5bl, proximally to the ph1 gene, between the est-ssr markers xw5145 and xw9340. the markers of skr were used to control the transfer of its recessive allele into other wheat genotypes, which made it possible to obtain highly crossable forms. however, the advantages of using the skr and kr1 markers in marker-assisted selection and in the screening of ex situ collections are not sufficiently studied. the published kr1 sequence for varieties with different crossability offers great prospects, because it will be possible to create allele-specific markers. in this review, the following issues are considered: genetic resources created by wheat and rye hybridization, the geographical distribution of easy-to-cross forms of wheat, genetic control of the wheat and rye compatibility, advances of the use of molecular markers in the mapping of kr-genes and their transmission control. microsatellite (ssr) markers with known precise intrachromosomal locations are widely used for mapping genes in rye and for the investigation of wheat-rye translocation lines and triticale highly demanded for mapping economically important genes and qtl-analysis. one of the sources of novel ssr markers in rye are microsatellites transferable from the wheat genome. broadening the list of available ssrs in rye mapped to chromosomes is still needed, since some rye chromosome maps still have just a few microsatellite loci mapped. the goal of the current study was to integrate wheat est-ssrs into the existing rye genetic maps and to construct a consensus rye microsatellite map. four rye mapping populations (p87/p105, n6/n2, n7/n2 and n7/n6) were tested with cfe (est-ssrs) primers. a total of 23 xcfe loci were mapped on rye chromosomes: xcfe023, -136 and -266 on chromosome 1r, xcfe006, -067, -175 and -187 on 2r, xcfe029 and -282 on 3r, xcfe004, -100, -152, -224 and -260 on 4r, xcfe037, -208 and -270 on 5r, xcfe124, -159 and -277 on 6r, xcfe010, -143 and -228 on 7r. with the exception of xcfe159 and xcfe224, all the xcfe loci mapped were found in orthologous positions considering multiple evolutionary translocations in the rye genome relative to those of common wheat. the consensus map was constructed using mapping data from the four bi-parental populations. it contains a total of 123 microsatellites, 12 snps, 118 rflps and 2 isozyme loci. alkylresorcinols (5-alkyl-1,3-dihydroxybenzenes) are amphiphilic phenolic lipid compounds which are abundant in cereals, with highest contents in rye. alkylresorcinols are suspected to show a wide range of favourable biological activities. for such and further testing, highly pure alkylresorcinol standards are required. especially, purities > 98% were partly difficult to obtain in the past. here, we aimed to isolate the most abundant (saturated) alkylresorcinols from rye using countercurrent chromatography. to achieve very high purity, alkylresorcinol-containing extract (∼7.14 g) of rye grains (cold extracts with cyclohexane/ethyl acetate (46/54, w/w)) were preparatively transesterified followed by a preparative hydrogenation. countercurrent chromatography separation of ∼1 g hydrogenated and transesterified rye grain extract using the solvent system n-hexane - ethyl acetate - methanol - water (9:1:9:1, v/v/v/v) yielded 51.8 mg ar17:0, 77.4 mg ar19:0, 57.2 mg ar21:0, 28.8 mg ar23:0 and 11.5 mg ar25:0 with purities > 99% in either case. the isolated alkylresorcinol homologues can be used for subsequent bioassays. this article is protected by copyright. all rights reserved. hyperspectral data is a promising complement to genomic data to predict biomass under scenarios of low genetic relatedness. sufficient environmental connectivity between data used for model training and validation is required. the demand for sustainable sources of biomass is increasing worldwide. the early prediction of biomass via indirect selection of dry matter yield (dmy) based on hyperspectral and/or genomic prediction is crucial to affordably untap the potential of winter rye (secale cereale l.) as a dual-purpose crop. however, this estimation involves multiple genetic backgrounds and genetic relatedness is a crucial factor in genomic selection (gs). to assess the prospect of prediction using reflectance data as a suitable complement to gs for biomass breeding, the influence of trait heritability ([formula: see text]) and genetic relatedness were compared. models were based on genomic (gblup) and hyperspectral reflectance-derived (hblup) relationship matrices to predict dmy and other biomass-related traits such as dry matter content (dmc) and fresh matter yield (fmy). for this, 270 elite rye lines from nine interconnected bi-parental families were genotyped using a 10 k-snp array and phenotyped as testcrosses at four locations in two years (eight environments). from 400 discrete narrow bands (410 nm-993 nm) collected by an uncrewed aerial vehicle (uav) on two dates in each environment, 32 hyperspectral bands previously selected by lasso were incorporated into a prediction model. hblup showed higher prediction abilities (0.41 - 0.61) than gblup (0.14 - 0.28) under a decreased genetic relationship, especially for mid-heritable traits (fmy and dmy), suggesting that hblup is much less affected by relatedness and [formula: see text]. however, the predictive power of both models was largely affected by environmental variances. prediction abilities for dmy were further enhanced (up to 20%) by integrating both matrices and plant height into a bivariate model. thus, data derived from high-throughput phenotyping emerges as a suitable strategy to efficiently leverage selection gains in biomass rye breeding; however, sufficient environmental connectivity is needed. cereals and legumes are rich in folate. however, due to the instability of folate, processing and digestion can induce significant folate loss. in this paper, folate bioaccessibility of faba bean, oat, rye and wheat flours and pastes was studied using a static in vitro digestion model. folate bioaccessibility depended on food matrices, varying from 42% to 67% in flours and from 40% to 123% in pastes. digestion was associated with the interconversion of formyl folates, as well as the increase of oxidised vitamers and decrease of reduced vitamers. especially in faba bean, 5-methyltetrahydrofolate showed surprisingly good stability both in digestion and heat treatment, resulting in high bioaccessibility. the physiological concentration of ascorbic acid did not stabilise folate in digestion; however, a higher level helped to maintain reduced vitamers. heat treatment (10-min paste making) could improve folate bioaccessibility by liberating folate from the food matrices and by altering folate vitamer distribution. background: high dietary fiber intake has been associated with reduced risk of helicobacter pylori infection and co-morbidities such as gastric cancer but also with reduced risk of cardiovascular disease. it has been suggested that fermented rye could affect helicobacter pylori bacterial load and that high- fiber rye may be superior to wheat for improvement of several cardiometabolic risk factors, but few long-term interventions with high fiber rye foods have been conducted. objective: to examine the effect of high-fiber wholegrain rye foods with added fermented rye bran vs. refined wheat on helicobacter pylori infection and cardiometabolic risk markers in a chinese population with a low habitual consumption of high fiber cereal foods. design: a parallel dietary intervention was set up and 182 normal- or overweight men and women were randomized to consume wholegrain rye products containing fermented rye bran (frb) or refined wheat (rw) for 12 weeks. anthropometric measurements, fasting blood sample collection and 13c-urea breath test (13c-ubt) were performed at baseline and after 6 and 12 weeks of intervention as well as 12 weeks after the end of the intervention. results: no difference between diets on helicobacter pylori bacterial load measured by 13c-ubt breath test or in virulence factors of helicobacter pylori in blood samples were found. low density lipoprotein cholesterol (ldl-c) and high sensitivity c-reactive protein (hs-crp) were significantly lower in the frb group, compared to the rw group after 12 weeks of intervention. the intervention diets did not affect markers of glucose metabolism or insulin sensitivity. conclusions: while the results of the present study did not support any effect of frb on helicobacter pylori bacterial load, beneficial effects on ldl-c and hs-crp were clearly shown. this suggest that consumption of high fiber rye foods instead of refined wheat could be one strategy for primary prevention of cardiovascular disease. clinical trial registration: the trial was registered at www.clinicaltrials.gov, identifier: nct03103386. the irradiation process extends the cereal grain storage period, but also affects their chemical composition and antioxidants properties. the aim of this study was to analyze the effect of gamma irradiation on the content of total polyphenols, flavonoids, and tannins as well as the quantitative and qualitative profile of polyphenols in rye grains. the potential antioxidant capacity was also evaluated. the irradiation process resulted in an average increase of 10% of the total phenolic content as compared to the raw material, with each of the analyzed varieties reacting in different manners. the amount of tannins increased after irradiation at a constant level regardless of the applied gamma ray doses in the all analyzed rye grain varieties. the antiradical and antioxidant activity of rye grains after the irradiation process did not change or was reduced. dry milling and air classification were applied to produce three different ingredients from wheat and rye brans. dried and pin disc-milled brans having particle size medians of 89-131 µm were air classified to produce protein- and soluble dietary fibre-enriched hybrid ingredients (median particle size 7-9 µm) and additionally brans were ultra-finely milled (median particle size 17-19 µm). the samples were characterised in regard to their composition and techno-functional properties. in air classification, protein content increased from 16.4 and 14.7% to 30.9 and 30.7% for wheat and rye brans, which corresponded to protein separation efficiencies of 18.0 and 26.9%, respectively. concurrently, the ratio between soluble and insoluble dietary fibre increased from 0.22 to 0.85 for wheat and from 0.56 to 1.75 for rye bran. the protein- and soluble dietary fibre-enriched wheat bran fraction showed improved protein solubility at alkaline ph when compared to pin disc- and ultra-finely-milled wheat bran, whereas less difference between the wheat ingredients was observed at native and acidic ph. the protein- and soluble dietary fibre-enriched rye bran fraction exhibited lower solubility than the pin disc- or ultra-finely-milled rye brans at all the studied ph-values. ultra-fine milling alone decreased protein solubility and increased damaged starch content when compared to the pin disc-milled brans. both protein enrichment and ultra-fine milling improved colloidal stability in comparison to the pin disc-milled raw materials. the lowest water and oil binding capacities were obtained for the protein-enriched fractions. ultrasound-assisted emulsification of the protein- and soluble dietary fibre-enriched fractions and the ultra-finely-milled brans revealed no major differences in the visual quality or stability of the emulsions. the results suggest that modification of the techno-functional properties of cereal brans may be acquired via both air classification and ultra-fine milling. lb chromosomes (bs) are supernumerary dispensable components of the standard genome (a chromosomes, as) that have been found in many eukaryotes. so far, it is unkown whether the b-derived transcripts translate to proteins or if the host proteome is changed due to the presence of bs. la comparative mass spectrometry was performed using the protein samples isolated from shoots of rye plants with and without bs. we aimed to identify b-associated peptides and analyzed the effects of bs on the total proteome. lour comparative proteome analysis demonstrates that the presence of rye bs affects the total proteome including different biological function processes. 319 out of 16776 quantified features were found in at least three out of five +b plants but not in 0b plants. 31 out of 319 features were identified as b-associated peptide features. according to our data mining, one b-specific protein fragment showed similarity to a glycine-rich rna binding protein which differed from its a-paralog by two amino acid insertions. lour result represents a milestone in b chromosome research, since this is the first report to demonstrate the existence of bs changing the proteome of the host. rye (secale cereale l.) is one of the most important cereal crops in eastern and northern europe, showing better tolerance to environmental stress factors compared to wheat and triticale. plant response to the crude oil-polluted soil depends on plant species, oil concentration, time of exposure, etc. the current study is aimed at investigating the growth, oxidative stress and the response of antioxidative system of two rye varieties (krona and valdai) cultivated on crude oil-contaminated soils at different concentrations (1.5, 3.0, 6.0, and 12.0%). inhibition of rye growth was observed at crude oil concentrations of above 3% for above-ground plant parts and of above 1.5% for roots. a decrease in content of chlorophyll a and total chlorophylls in krona variety was detected at 1.5% oil concentration in soil and in valdai variety at 3% oil concentration. compared with the control, the content of malondialdehyde was significantly increased in the krona variety at 3% oil concentration and in valdai variety at 6% oil concentration. the crude oil-induced oxidative stress was minimized in rye plants by the enhanced contents of low-molecular antioxidants (proline, non-protein thiols, ascorbic acid, phenolic compounds) and activities of superoxide dismutase, catalase, ascorbate peroxidase, and glutathione peroxidase. the strongest positive correlation was detected between the content of malondialdehyde and contents of proline (r = 0.89-0.95, p ≤ 0.05) and phenolic compounds (r = 0.90-0.94, p ≤ 0.05) as well as superoxide dismutase activity (r = 0.81-0.90, p ≤ 0.05). based on the results of a comprehensive analysis of growth and biochemical parameters and of the cluster analysis, valdai variety proved to be more resistant to oil pollution. due to this, valdai variety is considered to be a promising rye variety for cultivation on moderately oil-polluted soils in order to decontaminate them. at the same time, it is necessary to conduct further studies aimed at investigating oil transformation processes in the soil-rye system, which would make it possible to determine the efficiency of using this cereal for soil remediation. good understanding of the genes controlling root development is required to engineer root systems better adapted to different soil types. in wheat (triticum aestivum l.), the 1rs.1bl wheat-rye (secale cereale l.) translocation has been associated with improved drought tolerance and a large root system. however, an isogenic line carrying an interstitial segment from wheat chromosome arm 1bs in the distal region of the 1rs arm (1rsrw ) showed reduced grain yield and shorter roots both in the field and in hydroponic cultures relative to isogenic lines with the complete 1rs arm. in this study, we used exome capture to characterize 1rsrw and its parental lines t-9 and 1b+40. we show that 1rsrw has a 7.0 mb duplicated 1rs region and a 4.8 mb 1bs insertion colinear with the 1rs duplication, resulting in triplicated genes. lines homozygous for 1rsrw have short seminal roots, while lines heterozygous for this chromosome have roots of intermediate length. by contrast, near-isogenic lines carrying only the 1bs distal region or the 1rs-1bs duplication have long seminal roots similar to 1rs, suggesting a limited effect of the 1bs genes. these results suggest that the dosage of duplicated 1rs genes is critical for seminal root length. an induced deletion encompassing 38 orthologous wheat and rye duplicated genes restored root length and confirmed the importance of gene dosage in the short-root phenotype. we explored the expression profiles and functional annotation of these genes and discuss their potential as candidate genes for the regulation of seminal root length in wheat. polyploidization, or whole genome duplication (wgd), has an important role in evolution and speciation. one of the biggest challenges faced by a new polyploid is meiosis, in particular, discriminating between multiple related chromosomes so that only homologs recombine to ensure regular chromosome segregation and fertility. here, we report the production of two new hybrids formed by the genomes of species from three different genera: a hybrid between aegilops tauschii (dd), hordeum chilense (hchhch), and secale cereale (rr) with the haploid genomic constitution hchdr (n = 7× = 21); and a hybrid between triticum turgidum spp. durum (aabb), h. chilense, and s. cereale with the constitution abhchr (n = 7× = 28). we used genomic in situ hybridization and immunolocalization of key meiotic proteins to establish the chromosome composition of the new hybrids and to study their meiotic behavior. interestingly, there were multiple chromosome associations at metaphase i in both hybrids. a high level of crossover (co) formation was observed in hchdr, which shows the possibility of meiotic recombination between the different genomes. we succeeded in the duplication of the abhchr genome, and several amphiploids, aabbhchhchrr, were obtained and characterized. these results indicate that recombination between the genera of three economically important crops is possible. continental crossbred beef heifers were used in a randomized complete block design experiment to evaluate the effects of replacement of dry-rolled corn with unprocessed rye on the finishing-phase growth performance and efficiency of dietary net energy (ne) use. fifty-six heifers (433 ± 34.0 kg) were transported 241 km from a sale barn in north central south dakota to the ruminant nutrition center in brookings, sd. heifers were blocked by weight grouping and allotted to treatment pens (n = 7 heifers/pen and 4 pens/treatment). treatments included a finishing diet that contained 60% grain (diet dry matter basis) as dry-rolled corn (drc) or unprocessed rye grain (rye). on study day 14, all heifers were consuming the final diet and were implanted with 200 mg of trenbolone acetate and 28 mg of estradiol benzoate (synovex-plus, zoetis, parsippany, nj, usa). the rye heifers had decreased (p ≤ 0.01) final body weight, average daily gain, and gain efficiency; however, they tended (p = 0.08) to have a greater dry matter intake compared to drc heifers. rye heifers had decreased (p ≤ 0.01) observed dietary ne and decreased (p ≤ 0.01) observed-to-expected dietary ne ratio for maintenance and gain compared to drc heifers. the dressing percentage, 12th rib fat thickness, ribeye area, and the distribution of yield and quality grades were not altered (p ≥ 0.12) by dietary treatment. the hot carcass weight, calculated yield grade, estimated empty body fat (ebf), and body weight at 28% ebf decreased (p ≤ 0.02) and retail yield increased (p = 0.01) in rye compared to drc heifers. these data indicate that unprocessed rye is a palatable feed ingredient for inclusion in finishing diets for beef cattle and that rye inclusion only minimally influences the carcass quality grade. the feeding value of unprocessed rye is considerably less (21.4%) than that of dry-rolled corn using current standards and approximately 91% of the ne value of processed rye (processing index = 78.8%). rye grain fed as processed or unprocessed grain has an ne value that is less than 90% of that of dry-rolled corn. cover cropping is beneficial for reducing soil erosion and nutrient losses, but there are conflicting reports on how cover cropping affects emissions of nitrous oxide (n2o), a potent greenhouse gas. in this study, we measured n2o fluxes over a full year in illinois corn plots with and without rye cover crop. we compared these year-round measurements to n2o emissions predicted by the intergovernmental panel on climate change (ipcc) tier 1 equation and the denitrification-decomposition (dndc) model. in addition, we measured potential denitrification and n2o production rates. the field measurements showed typical n2o peaks shortly after fertilizer application, as well as a significant late-winter peak. cover cropping significantly reduced all peak n2o fluxes, with decreases ranging from 39 to 95%. neither model was able to accurately predict annual n2o fluxes or the decrease in n2o emissions from cover-cropped fields. in contrast to field measurements, lab assays found that cover cropping significantly increased potential denitrification by 90-127% and potential n2o production by 54-106%. the rye cover-cropped plots had lower soil nitrate and higher soil carbon. when limiting nitrate and excess carbon were provided in lab assays, the proportion of n2o resulting from denitrification decreased. these results suggest that the discrepancy between the observed decrease in field n2o emissions and the increase in denitrification potential may be due to the difference in available nutrients between the field and laboratory measurements. overall, these results suggest the importance of late-winter peaks in n2o emissions and the potential of rye cover cropping to reduce n2o emissions from agricultural fields. cytoplasmic male sterility (cms) is a widely applied plant breeding tool for hybrid seed production. the phenomenon is often caused by chimeric genes with altered open reading frames (orfs) located in the mitochondrial genomes and expressed as novel genotoxic products that induce pollen abortion. the fertility of cms plants can be restored by nuclear-encoded genes that inhibit the action of orfs responsible for pollen sterility. a recombinant inbred line (ril) mapping population s64/04/01, encompassing 175 individuals, was used for genetic map construction and identification of quantitative trait loci (qtls) responsible for fertility restoration in rye (secale cereale l.) with cms pampa. the genetic map of all seven rye chromosomes included 15,516 snp and silicodart markers and covered 1070.5 cm. individual qtls explaining 60% and 5.5% of the fertility trait's phenotypic variance were mapped to chromosomes 4r (qrft-4r) and 5r (qrft-5r), respectively. association mapping identified markers with the highest r2 value of 0.58 (p value = 2.21e-28). markers showing the highest associations with the trait were also mapped to the 4r chromosome within the qrft-4r region. based on marker sequence homology, putative genes involved in pollen fertility restoration were suggested. five silicodarts were converted into pcr-based markers for further breeding purposes. ergot alkaloids are mycotoxins produced by the fungus claviceps, which can contaminate grains and pose a health risk to humans and animals. validation of an ergot alkaloid method in collaborative projects can be challenging due to instability of analytes, and lack of reliable reference materials and a fully validated reference methods. to extensively evaluate performance of a quantitative uhplc-ms/ms method to detect ten ergot alkaloids at concentrations between 16-500 ng/g in grains. the method performance was evaluated in the blinded method test (bmt) exercise, which allowed organizers to successfully address the challenges. forty completely blinded test samples were prepared in an independent laboratory and shipped to participating laboratory to analyze on two separate days. precision, accuracy and horratr values met or exceeded the food and drug administration (fda) recommendations. the design of the bmt exercise provided a high degree of confidence in data and conclusions drawn. the method performed in a manner as expected and the method can be used by the laboratory for routine testing of wheat and rye grains. blinded method tests of laboratory methods facilitate validation of tests by evaluating performance in an unbiased manner. pmsesy, a new wheat powdery mildew resistance gene was characterized and genetically mapped to the terminal region of chromosome 1rl of wild rye secale sylvestre. the genus secale is an important resource for wheat improvement. the secale species are usually considered as non-adapted hosts of blumeria graminis f. sp. tritici (bgt) that causes wheat powdery mildew. however, as a wild species of cultivated rye, s. sylvestre is rarely studied. here, we reported that 25 s. sylvestre accessions were susceptible to isolate bgtyz01, whereas the other five confer effective resistance to all the tested isolates of bgt. a population was then constructed by crossing the resistant accession sesy-01 with the susceptible accession sesy-11. genetic analysis showed that the resistance in sesy-01 was controlled by a single dominant gene, temporarily designated as pmsesy. subsequently, combining bulked segregant rna-seq (bsr-seq) analysis with molecular analysis, pmsesy was mapped into a 1.88 cm genetic interval in the terminus of the long arm of 1r, which was closely flanked by markers xss06 and xss09 with genetic distances of 0.87 cm and 1.01 cm, respectively. comparative mapping demonstrated that the corresponding physical region of the pmsesy locus was about 3.81 mb in rye cv. lo7 genome, where 30 disease resistance-related genes were annotated, including five nlr-type disease resistance genes, three kinase family protein genes, three leucine-rich repeat receptor-like protein kinase genes and so on. this study gives a new insight into s. sylvestre that shows divergence in response to bgt and reports a new powdery mildew resistance gene that has potential to be used for resistance improvement in wheat. for decades, the evaluation of rye milling products have been aimed at detecting raw material defects that are linked to excessive enzyme activity. those defects were indirectly characterized by the rheological methods of the dough or the final products. however, such methods do not sufficiently reflect the baking properties of all rye flours present on the market. a further problem is the continuing climate change, which affects compound composition in rye. so far, these bread defects can only be corrected by process engineering (e.g., extended dough resting). therefore, it is necessary to characterize the main determinants of the quality defects prior to the baking process in order to predict baking quality and not waste raw material, energy, and time. in this study, it was found that the water accessibility of starch for gelatinization and its partial inhibition by certain components play a major role in baking quality. specifically, high amounts of insoluble nonstarch-polysaccharides (nspss) and a hindered denaturation of proteins seem to be an indication and reason for poor baking quality. however, traditional quantitative analysis of the ingredients and properties of the rye milling products (e.g., falling number, protein content, amylographic data) does not allow any reliable conclusions about rye flour suitability for use as bread rye. it can be concluded that more complex compositional aspects (e.g., complexation of compounds) need to be characterized for future quality control of rye. allopolyploid wheat (triticum aestivum l.) carries three pairs of homoeologous genomes but its meiotic pairing is diploid-like. this is the effect of the ph (pairing homoeologous) system which restricts chromosome pairing to strictly homologous. ph1 is the locus with the strongest effect. disabling ph1 permits pairing between homoeologues and is routinely used in chromosome engineering to introgress alien variation into breeding stocks. whereas the efficiency of ph1 and the general pattern of homoeologous crossovers in its absence are quite well known from numerous studies, other characteristics of such crossovers remain unknown. this study analyzed the crossover points in four sets of the ph1b-induced recombinants between wheat homologues as well as between three wheat and rye (secale cereale) homoeologous chromosome arms, and compared them to crossovers between homologues in a reference wheat population. the results show the ph1 locus also controls crossing over of homologues, and the general patterns of homologous (with ph1) and homoeologous (with ph1b) crossing over are the same. in all intervals analyzed, homoeologous crossovers fell within the range of frequency distribution of homologous crossovers among individual families of the reference population. no specific dna sequence characteristics were identified that could be recognized by the ph1 locus; the only difference between homologous and homoeologous crossing over appears to be in frequency. it is concluded that the ph1 locus likely recognizes dna sequence similarity; crossing over is permitted between very similar sequences. in the absence of ph1 dissimilarities are ignored, in proportion to the level of the sequence divergence. \n",
      "Euclidean Distance : 15.667235374450684\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. comparative proteomic analysis of mature pollen in triploid and diploid populus deltoides. proteomic analysis and candidate allergenic proteins in populus deltoides cl. \"2ken8\" mature pollen. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. ploidy affects plant growth vigor and cell size, but the relative effects of pollen fertility and allergenicity between triploid and diploid have not been systematically examined. here we performed comparative analyses of fertility, proteome, and abundances of putative allergenic proteins of pollen in triploid poplar 'zhonghuai1' ('zh1', triploid) and 'zhonghuai2' ('zh2', diploid) generated from the same parents. the mature pollen was sterile in triploid poplar 'zh1'. by applying two-dimensional gel electrophoresis (2-de), a total of 72 differentially expressed protein spots (deps) were detected in triploid poplar pollen. among them, 24 upregulated and 43 downregulated proteins were identified in triploid poplar pollen using matrix-assisted laser desorption/ionisation coupled with time of-flight tandem mass spectrometer analysis (maldi-tof/tof ms/ms). the main functions of these deps were related with \"s-adenosylmethionine metabolism\", \"actin cytoskeleton organization\", or \"translational elongation\". the infertility of triploid poplar pollen might be related to its abnormal cytoskeletal system. in addition, the abundances of previously identified 28 putative allergenic proteins were compared among three poplar varieties ('zh1', 'zh2', and '2ken8'). most putative allergenic proteins were downregulated in triploid poplar pollen. this work provides an insight into understanding the protein regulation mechanism of pollen infertility and low allergenicity in triploid poplar, and gives a clue to improving poplar polyploidy breeding and decreasing the pollen allergenicity. proteomic analysis was used to generate a map of populus deltoides cl. \"2ken8\" mature pollen proteins. by applying 2-d electrophoresis, we resolved 403 protein spots from mature pollen. using the matrix-assisted laser desorption/ionization time time-of-flight/time-of-flight tandem mass spectrometry method, we identified 178 distinct proteins from 218 protein spots expressed in mature pollen. moreover, out of these, 28 proteins were identified as putative allergens. the expression patterns of these putative allergen genes indicate that several of these genes are highly expressed in pollen. in addition, the members of profilin allergen family were analyzed and their expression patterns were compared with their homologous genes in arabidopsis and rice. knowledge of these identified allergens has the potential to improve specific diagnosis and allergen immunotherapy treatment for patients with poplar pollen allergy. \n",
      "Euclidean Distance : 15.739869117736816\n",
      "\n",
      "\n",
      "Document: the thermal behavior of lyocell fibers containing bis(trimethylsilyl)acetylene. one-step ethylene purification from an acetylene/ethylene/ ethane ternary mixture by cyclopentadiene cobalt-functionalized metal-organic frameworks. microporous metal-organic framework with dual functionalities for efficient separation of acetylene from light hydrocarbon mixtures. sulfur promotion in au/c catalyzed acetylene hydrochlorination. isomeric scandium-organic frameworks with high hydrolytic stability and selective adsorption of acetylene. robust heterometallic co re-evaluation of ortho-para-dependence of self pressure-broadening in the ν copper-based catalysts for selective hydrogenation of acetylene derived from cu(oh) melt-diffusion and ion-exchange methods: self-triggered o-rich groups on the surface of acetylene black using s-eda solution to boost its sulfur content for lithium-sulfur batteries. precise pore space partition combined with high-density h-bonding acceptor within a metal-organic framework for highly-efficient acetylene storage and separation. acetylene-fueled trichloroethene reductive dechlorination in a groundwater enrichment culture. contaminant-free end-capped and single-mode acetylene photonic microcell for sub-doppler spectroscopy. electrostatically-driven selective adsorption of carbon dioxide over acetylene in an ultramicroporous material. effect of various g-c reactivity trends in the gas-phase addition of acetylene to the  an unprecedented pillar-cage fluorinated hybrid porous framework with highly efficient acetylene storage and separation. sustainable synthesis of bimetallic single atom gold-based catalysts with enhanced durability in acetylene hydrochlorination. a mof-based ultra-strong acetylene nano-trap for highly efficient c infrared comb spectroscopy of buffer-gas-cooled molecules: toward absolute frequency metrology of cold acetylene. efficient separation of acetylene and carbon dioxide in a decorated zeolite. this study focuses on the preparation of carbon fiber precursors from solutions of cellulose in n-methylmorpholine-n-oxide with the addition of bis(trimethylsilyl)acetylene, studying their structural features and evaluating thermal behavior. the introduction of a silicon-containing additive into cellulose leads to an increase in the carbon yield during carbonization of composite precursors. the type of the observed peaks on the differential scanning calorimetry (dsc) curves cardinally changes from endo peaks intrinsic for cellulose fibers to the combination of endo and exo peaks for composite fibers. for the first time, coefficient of thermal expansion (cte) values were obtained for lyocell fibers and composite fibers with bis(trimethylsilyl)acetylene (btmsa). the study of the dependence of linear dimensions of the heat treatment fibers on temperature made it possible to determine the relation between thermal expansion coefficients of carbonized fibers and thermogravimetric curves, as well as to reveal the relationship between fiber shrinkage and btmsa bis(trimethylsilyl)acetylene content. carbon fibers from composite precursors are obtained at a processing temperature of 1200 °c. a study of the structure of carbon fibers by x-ray diffraction, raman spectroscopy, and transmission electron microscopy made it possible to determine the amorphous structure of the fibers obtained. the separation of ethylene (c 2 h 4 ) from a mixture of ethane (c 2 h 6 ), ethylene (c 2 h 4 ), and acetylene (c 2 h 2 ) at normal temperature and pressure is a significant challenge. the sieving effect of pores is powerless due to the similar molecular size and kinetic diameter of these molecules. we report a new modification method based on a stable ftw topological zr-mof platform ( mof-525 ). introduction of a cyclopentadiene cobalt functional group led to new ftw-type mofs materials ( upc-612 and upc-613 ), which increase the host-guest interaction and achieve efficient ethylene purification from the mixture of hydrocarbon gases. the high performance of upc-612 and upc-613 for c 2 h 2 /c 2 h 4 /c 2 h 6 separation has been verified by gas sorption isotherms, density functional theory (dft), and experimentally determined breakthrough curves. this work provides a one-step separation of the ternary gas mixture and can further serve as a blueprint for the design and construction of function-oriented porous structures for such applications. separating acetylene from light hydrocarbon mixtures like ethylene is a very important process for downstream industrial applications. herein, we report a new mof [cul2(sif6)] (utsa-220, l = (1e,2e)-1,2-bis(pyridin-4-ylmethylene)hydrazine) with dual functionalities featuring optimal pore size with strong binding sites for acetylene. utsa-220 exhibits apparently higher uptake capacity for c2h2 than those for other light hydrocarbons. the potential of this material for trace c2h2 removal from c2h4 has also been demonstrated by a dynamic breakthrough experiment performed with c2h2/c2h4 (1/99 v/v) under simulated industrial conditions. according to the dispersion-corrected density functional theory (dft-d) simulation, sif62- and azine moieties serve as preferential binding sites for c2h2, indicating the feasibility of the dual functionalities incorporated in utsa-220 for adsorbent-based c2h2 separations. the formation of highly active and stable acetylene hydrochlorination catalysts is of great industrial importance. the successful replacement of the highly toxic mercuric chloride catalyst with gold has led to a flurry of research in this area. one key aspect, which led to the commercialization of the gold catalyst is the use of thiosulphate as a stabilizing ligand. this study investigates the use of a range of sulfur containing compounds as promoters for production of highly active au/c catalysts. promotion is observed across a range of metal sulfates, non-metal sulfates, and sulfuric acid treatments. this observed enhancement can be optimized by careful consideration of either pre- or post-treatments, concentration of dopants used, and modification of washing steps. pre-treatment of the carbon support with sulfuric acid (0.76 m) resulted in the most active au/c in this series with an acetylene conversion of ≈70% at 200 °c. two solvent-controlled topological isomers of scandium-organic frameworks [sc(hpzc)(pzc)]·dmf·2h2o (1·dmf·2h2o) and [sc(hpzc)(pzc)]·dma·4h2o (2·dma·4h2o) were synthesized using 2,5-pyrazinedicarboxylate (pzc2-) (dmf = dimethylformamide; dma = dimethylacetamide). despite the isomeric nature of the obtained metal-organic frameworks (mofs), they possess different structural features and unique adsorption properties toward gases and iodine. the compound 1 has widely spread among mof structures a dia topology with ultranarrow channels, which together with inner surface functionalization leads to enhanced co2 adsorption and high selectivity factors in co2/ch4 and co2/n2 gas mixtures (25.9 and 35.6, respectively, 1/1 v/v). moreover, a rare preferable adsorption of co2 over c2h2 was demonstrated. the biporous isomeric framework 2 has a crb topology inherent in zeolites. a remarkable adsorption affinity to c2h2 with the ideal adsorbed solution theory selectivity factor of 127.1 for a c2h2/c2h4 mixture (1/99 v/v) was achieved for 2. both compounds have exceptional chemical stability in a wide range of ph from acidic to basic media. a highly robust heterometallic coiilaiii2-organic framework (nuc-19) is synthesized and features 1d rectangle-like channels with a window size of 3.4 × 6.3 å2 and a carboxyl-group-modified inner surface. nuc-19 exhibits significantly differential uptake for c2h2, c2h4, and ch4, and the ideal adsorbed solution theory selectivities of c2h2/c2h4 (1:99, v/v) and c2h2/ch4 (50:50, v/v) at room temperature can reach up to 4.0 and 79.7, respectively. fixed-bed dynamic breakthrough experiments under simulated industrial conditions indicate that nuc-19 could efficiently trap c2h2 from mixed gases of c2h2/ch4 and c2h2/c2h4. optical frequency comb-referenced measurements of self pressure-broadened line profiles of the r(8) to r(13) lines in the ν1 + ν3 combination band of acetylene near 1.52 µm are reported. the analysis of the data found no evidence for a previously reported [iwakuni et al., phys. rev. lett. 117(14), 143902 (2016)] systematic alternation in self pressure-broadened line widths with the nuclear spin state of the molecule. the present work brought out the need for the use of an accurate line profile model and careful accounting for weak background absorptions due to hot band and lower abundance isotopomer lines. the data were adequately fit using the quadratic speed-dependent voigt profile model, neglecting the small speed-dependent shift. parameters describing the most probable and speed-dependent pressure-broadening, most probable shift, and the line strength were determined for each line. detailed modeling of the results of iwakuni et al. showed that their neglect of collisional narrowing due to the speed-dependent broadening term combined with the strongly absorbing data recorded and analyzed in transmission mode were the reasons for their results. replacing precious metals with cheap metals in catalysts is a topic of interest in both industry and academia but challenging. here, a selective hydrogenation catalyst was prepared by thermal treatment of cu(oh)2 nanowires with acetylene-containing gas at 120 °c followed by hydrogen reduction at 150 °c. the characterization by means of transmission electron microscopy observation, x-ray diffraction, and x-ray photoelectron spectroscopy revealed that two crystallites were present in the resultant catalyst. one of the crystal phases was metal cu, whereas the other crystal phase was ascribed to an interstitial copper carbide (cu \n",
      "                         c) phase. the reduction of freshly prepared copper (ii) acetylide (cuc2) at 150 °c also afforded the formation of cu and cu \n",
      "                         c crystallites, indicating that cuc2 was the precursor or an intermediate in the formation of cu \n",
      "                         c. the prepared catalysts consisting of cu and cu \n",
      "                         c exhibited a considerably high hydrogenation activity at low temperatures in the selective hydrogenation of acetylene in the ethylene stream. in the presence of a large excess of ethylene, acetylene was completely converted at 110 °c and atmospheric pressure with an ethane selectivity of <15%, and the conversion and selectivity were constant in a 260 h run. only a few studies have described the use of h+-attacking s-eda in nucleophilic substitution reactions to bind frameworks and sulfur in cathode materials, which is also known as the ion-exchange method. the pros and cons of this method are still unclear in relation to lithium-sulfur battery applications. here, the influences of two synthetic routes, a melt-diffusion method and h+ reacting with s-eda via nucleophilic substitution, on the morphologies and electrochemical properties of cathode materials are discussed in detail based on in situ xrd and other advanced technologies. accordingly, high s-loading is achieved when h+ reacts with s-eda via ion exchange on the surface of acetylene black, and capacities of 693.8, 644.5, and 638.9 ma h g-1 are obtained over the first three cycles when the c/s composite is used as a cathode in coin cells without the conductive additive super p. in situ xrd data confirm that poor electrochemical properties can mainly be attributed to the conversion rate of s species being too rapid to thoroughly utilize the s molecules that are immobilized, which means that more fixed sulfur can form during the charge/discharge process when using the ion-exchange method to make the c/s composite. in addition, a long-chain polysulfide shuttling effect is directly noticed via afm in tapping-kpfm mode in the c/s composite that was synthesized via the melt-diffusion method, even though polar s-o bonds exist in the composite. the increase in the cathodic surface potential from 102.8 to 141.1 mv and the increase in the morphological height from 547.7 to 829.7 nm during the discharge/charge process can be attributed to the process of s loss. the high storage capacity versus high selectivity trade-off barrier keeps a daunting challenge for addressing practical acetylene (c 2 h 2 ) adsorbent. a structure-performance relationship screening for sixty-two high-performance metal-organic framework adsorbents reveals that a moderate pore size distribution around 5.0-7.5 å is critical to fulfill this task. motivated by this inspiration, a precise pore space partition approach was involved to partition 1d hexagonal channels of typical mil-88 architecture into finite segments with pore sizes varying from 4.5 å (snnu-26) to 6.4 å (snnu-27), 7.1 å (snnu-28), and 8.1 å (snnu-29). coupled with bare tetrazole n sites (6 or 12 bare n sties within one cage) as high-density h-bonding acceptors for c 2 h 2 , the target mofs offer a good combination of high c 2 h 2 /co 2 adsorption selectivity and high c 2 h 2 uptake capacity in addition to good stability. especially, the optimized snnu-27-fe material demonstrates super c 2 h 2 uptake of 182.4 cm 3 g -1 and extraordinary c 2 h 2 /co 2 dynamic breakthrough time up to 91 min g -1 under ambient conditions, thus outperforms all benchmark mof adsorbents and represents a promising material for practical c 2 h 2 /co 2 separation. in aquifers, acetylene (c2h2) is a product of abiotic degradation of trichloroethene (tce) catalyzed by in situ minerals. c2h2 can, in turn, inhibit multiple microbial processes including tce dechlorination and metabolisms that commonly support dechlorination, in addition to supporting the growth of acetylenotrophic microorganisms. previously, c2h2 was shown to support tce reductive dechlorination in synthetic, laboratory-constructed cocultures containing the acetylenotroph pelobacter sp. strain sfb93 and dehalococcoides mccartyi strain 195 or strain bav1. in this study, we demonstrate tce and perchloroethene (pce) reductive dechlorination by a microbial community enriched from contaminated groundwater and amended with c2h2 as the sole electron donor and organic carbon source. the metagenome of the stable, enriched community was analyzed to elucidate putative community functions. a novel anaerobic acetylenotroph in the phylum actinobacteria was identified using metagenomic analysis. these results demonstrate that the coupling of acetylenotrophy and reductive dechlorination can occur in the environment with native bacteria and broaden our understanding of biotransformation at contaminated sites containing both tce and c2h2importance understanding the complex metabolisms of microbial communities in contaminated groundwaters is a challenge. pce and tce are among the most common groundwater contaminants in the united states that, when exposed to certain minerals, exhibit a unique abiotic degradation pathway in which c2h2 is a product. c2h2 can act as both an inhibitor of tce dechlorination and of supporting metabolisms and an energy source for acetylenotrophic bacteria. here, we combine laboratory microcosm studies with computational approaches to enrich and characterize an environmental microbial community that couples two uncommon metabolisms, demonstrating unique metabolic interactions only yet reported in synthetic, laboratory-constructed settings. using this comprehensive approach, we have identified the first reported anaerobic acetylenotroph in the phylum actinobacteria, demonstrating the yet-undescribed diversity of this metabolism that is widely considered to be uncommon. we report on the development of an acetylene-filled photonic microcell based on an assembly process that is contaminant free and requires no helium buffer gas nor gluing procedure. the microcell consists of a 7-m-long and 30 µm core-diameter inhibited-coupling guiding hollow-core photonic crystal fiber filled with acetylene gas at a pressure in the range of 80 µbar, sealed by capping its ends with fusion-collapsing a glass-tube sleeve, and mounted on fc connectors for integration. the microcell shows a robust single-mode behavior and a total insertion loss of ∼1.5db. the spectroscopic merit of the formed microcell is tested by generating electromagnetic induced transparency and saturated absorption on r13 and p9 absorption lines, respectively. the sub-doppler transparencies show a close to transit time limited linewidth of 17±3mhz. the latter was monitored for over 3 months. as a demonstration, the microcell was used to frequency stabilize a laser with fractional frequency instability improvement by a factor 50 at 100 s integration time compared to free running laser operation. separating acetylene from carbon dioxide is important but highly challenging due to their similar physical properties and molecular dimensions. adsorptive capture of carbon dioxide from acetylene can directly produce pure acetylene but rarely realized. here, we report highly efficient electrostatically-driven co 2 /c 2 h 2 separation in an ultramicroporous cadmium-nitroprusside ( cd-np ) with compact pore space and complementary electrostatic potential well fitting for co 2 , thus enabling molecular quadrupole moment recognition towards co 2 over c 2 h 2 with significant sieving effect. this material shows high co 2 /c 2 h 2 uptake ratio of 6.0 as well as remarkable co 2 /c 2 h 2 selectivity of 85 under ambient condition with modest co 2 adsorption heat, which is superior to most other co 2 -selective absorbents. neutron powder diffraction experiments and molecular simulations reveal that the electrostatic potential compatibility between pore structure and co 2 allows it to be trapped in a head-on orientation towards the cd center, whereas the diffusion of c 2 h 2 is electrostatically forbidden. dynamic breakthrough experiments have validated the separation performance of this compound for co 2 /c 2 h 2 separation. a series of alkylorganotin-based catalysts (sn-g-c3n4 /ac) was prepared by wet impregnation in ethanol using different g-c3n4 precursors and alkylorganotin compounds. the structure, texture, surface composition, and adsorption properties of the as-prepared catalysts were extensively characterized. then, the obtained samples were evaluated for their catalytic performance in hydrochlorination of acetylene. the results provided by the x-ray photoelectron spectroscopy, acetylene temperature-programmed desorption, and hcl adsorption confirmed the nature of the active sites (i.e. sn-nx) involved in the reactant adsorption, and hence in the improved catalytic performance. these active sites were also related to the improved lifetime of alkylorganotin-based catalysts in the hydrochlorination of acetylene. at a constant reaction temperature of 200 °c with an acetylene gas hourly space velocity (c2h2 -ghsv) of 30 h-1 , sn-g1 -c3n4 /ac-550 exhibited the highest acetylene conversion (~98.0%) and selectivity toward the vinyl chloride monomer (>98.0%). from the catalytic test results, it was reasonably concluded that the hexamethylenetetramine is the most suitable n precursor, as compared to the dicyandiamide and urea, to prepare high-performance catalysts. from the bet specific surface area of fresh and used catalysts, it was suggested that, in contrast to dicyandiamide and urea, hexamethylenetetramine could delay the deposition of coke on alkylorganotin-based catalysts, which is reflected by the extended lifetime. a key step in gas-phase polycyclic aromatic hydrocarbon (pah) formation involves the addition of acetylene (or other alkyne) to σ-type aromatic radicals, with successive additions yielding more complex pahs. a similar process can happen for n-containing aromatics. in cold diffuse environments, such as the interstellar medium, rates of radical addition may be enhanced when the σ-type radical is charged. this paper investigates the gas-phase ion-molecule reactions of acetylene with nine aromatic distonic σ-type radical cations derived from pyridinium (pyr), anilinium (anl), and benzonitrilium (bzn) ions. three isomers are studied in each case (radical sites at the ortho, meta, and para positions). using a room temperature ion trap, second-order rate coefficients, product branching ratios, and reaction efficiencies are measured. the rate coefficients increase from para to ortho positions. the second-order rate coefficients can be sorted into three groups: low, between 1 and 3 × 10-12 cm3 molecule-1 s-1 (3anl and 4anl); intermediate, between 5 and 15 × 10-12 cm3 molecule-1 s-1 (2bzn, 3bzn, and 4bzn); and high, between 8 and 31 × 10-11 cm3 molecule-1 s-1 (2anl, 2pyr, 3pyr, and 4pyr); and 2anl is the only radical cation with a rate coefficient distinctly different from its isomers. quantum chemical calculations, using m06-2x-d3(0)/6-31++g(2df,p) geometries and dsd-pbep86-nl/aug-cc-pvqz energies, are deployed to rationalize reactivity trends based on the stability of prereactive complexes. the g3x-k method guides the assignment of product ions following adduct formation. the rate coefficient trend can be rationalized by a simple model based on the prereactive complex forward barrier height. despite much intense investigation on the c2 h2 /co2 separation, the trade-off between the adsorption capacity and separation selectivity is still tricky. to overcome the dilemma, we have rationally synthesized an ultra-stable fluorinated hybrid porous material sifsix-cu-tpa with the ith-d topology. completely differing from the famous pillar-layer fluorinated materials, sifsix-cu-tpa possesses a unique pillar-cage structure, in which the sif62- anions cross-link two adjacent metal nodes as pillars to stabilize the three-dimensional framework constructed by icosahedral and tetrahedral cages. as anticipated, sifsix-cu-tpa has high bet surface area (1330 m2  g-1 ) as well as high c2 h2 uptake (185 cm3  g-1 at 298 k and 1 bar). at the same time, due to the obvious difference in the adsorption performance of co2 and c2 h2 especially in the low pressure area, sifsix-cu-tpa also exhibits an excellent c2 h2 /co2 separation performance (breakthrough time up to 68 min g-1 at 298 k and 1 bar). gold single-atom catalysts (sacs) exhibit outstanding reactivity in acetylene hydrochlorination to vinyl chloride, but their practical applicability is compromised by current synthesis protocols, using aqua regia as chlorine-based dispersing agent, and their high susceptibility to sintering on non-functionalized carbon supports at >500 k and/or under reaction conditions. herein, a sustainable synthesis route to carbon-supported gold nanostructures in bimetallic catalysts is developed by employing salts as alternative chlorine source, allowing for tailored gold dispersion, ultimately reaching atomic level when using h2 ptcl6 . to rationalize these observations, several synthesis parameters (i.e., ph, cl-content) as well as the choice of metal chlorides are evaluated, hinting at the key role of platinum in promoting a chlorine-mediated dispersion mechanism. this can be further extrapolated to redisperse large gold agglomerates (>70 nm) on carbon carriers into isolated atoms, which has important implications for catalyst regeneration. another key role of platinum single atoms is to inhibit the sintering of their spatially isolated gold-based analogs up to 800 k and during acetylene hydrochlorination, without compromising the intrinsic activity of au(i)-cl active sites. accordingly, exploiting cooperativity effects of a second metal is a promising strategy towards practical applicability of gold sacs, opening up exciting opportunities for multifunctional single-atom catalysis. porous materials with open metal sites have been investigated to separate various gas mixtures. however, open metal sites show the limitation in the separation of some challenging gas mixtures, such as c2 h2 /co2 . herein, we propose a new type of ultra-strong c2 h2 nano-trap based on multiple binding interactions to efficiently capture c2 h2 molecules and separate c2 h2 /co2 mixture. the ultra-strong acetylene nano-trap shows a benchmark qst of 79.1 kj mol-1 for c2 h2 , a record high pure c2 h2 uptake of 2.54 mmol g-1 at 1×10-2  bar, and the highest c2 h2 /co2 selectivity (53.6), making it as a new benchmark material for the capture of c2 h2 and the separation of c2 h2 /co2 . the locations of c2 h2 molecules within the mof-based nanotrap have been visualized by the in situ single-crystal x-ray diffraction studies, which also identify the multiple binding sites accountable for the strong interactions with c2 h2 . we review the recent developments in precision ro-vibrational spectroscopy of buffer-gas-cooled neutral molecules, obtained using infrared frequency combs either as direct probe sources or as ultra-accurate optical rulers. in particular, we show how coherent broadband spectroscopy of complex molecules especially benefits from drastic simplification of the spectra brought about by cooling of internal temperatures. moreover, cooling the translational motion allows longer light-molecule interaction times and hence reduced transit-time broadening effects, crucial for high-precision spectroscopy on simple molecules. in this respect, we report on the progress of absolute frequency metrology experiments with buffer-gas-cooled molecules, focusing on the advanced technologies that led to record measurements with acetylene. finally, we briefly discuss the prospects for further improving the ultimate accuracy of the spectroscopic frequency measurement. the almost identical molecular sizes and volatilities of acetylene and carbon dioxide make their separation extremely challenging in industry. reported here is the efficient separation of acetylene and carbon dioxide (v/v=2/1, which is relevant to that in the industrial cracking stream) in faujasite zeolites decorated with atomically-dispersed copper(ii) sites under ambient conditions. in situ neutron powder diffraction and inelastic neutron scattering confirm that the confined copper(ii) site displays chemoselective yet reversible binding to acetylene, whereas adsorbed carbon dioxide molecules are stabilized by weak host-guest supramolecular interactions with the framework oxygen centers, thus resulting in the efficient separation of these two gases under flow conditions. a designed adsorption-purging-desorption system based upon cu@fau is established for the recovery of high purity acetylene (98-99 %) from the mixture of acetylene and carbon dioxide, offering an unprecedented separation factor of 22.2 with an effective dynamic uptake of acetylene of 1.51 mmol g-1 at 298 k. \n",
      "Euclidean Distance : 15.760201454162598\n",
      "\n",
      "\n",
      "Document: climatic factors directly impact the volatile organic compound fingerprint in green arabica coffee bean as well as coffee beverage quality. coffee grown at high elevations fetches a better price than that grown in lowland regions. this study was aimed at determining whether climatic conditions during bean development affected sensory perception of the coffee beverage and combinations of volatile compounds in green coffee. green coffee samples from 16 plots representative of the broad range of climatic variations in réunion island were compared by sensory analysis. volatiles were extracted by solid phase micro-extraction and the volatile compounds were analysed by gc-ms. the results revealed that, among the climatic factors, the mean air temperature during seed development greatly influenced the sensory profile. positive quality attributes such as acidity, fruity character and flavour quality were correlated and typical of coffees produced at cool climates. two volatile compounds (ethanal and acetone) were identified as indicators of these cool temperatures. among detected volatiles, most of the alcohols, aldehydes, hydrocarbons and ketones appeared to be positively linked to elevated temperatures and high solar radiation, while the sensory profiles displayed major defects (i.e. green, earthy flavour). two alcohols (butan-1,3-diol and butan-2,3-diol) were closely correlated with a reduction in aromatic quality, acidity and an increase in earthy and green flavours. we assumed that high temperatures induce accumulation of these compounds in green coffee, and would be detected as off-flavours, even after roasting. climate change, which generally involves a substantial increase in average temperatures in mountainous tropical regions, could be expected to have a negative impact on coffee quality. \n",
      "Euclidean Distance : 15.760354042053223\n",
      "\n",
      "\n",
      "Document: synthesis and application of eco-friendly superabsorbent composites based on xanthan gum and semi-coke. improved color stability of anthocyanins in the presence of ascorbic acid with the combination of rosmarinic acid and xanthan gum. hydratability and improved fermentability in vitro of guar gum by combination of xanthan gum. tea polyphenols encapsulated in w/o/w emulsions with xanthan gum-locust bean gum mixture: evaluation of their stability and protection. the quality of emulsions with new synthetized lipids stabilized by xanthan gum. modified xanthan gum for methyl orange uptake: kinetic, isotherm, and thermodynamic behaviors. preparation and evaluation of a xanthan gum-containing linezolid ophthalmic solution for topical treatment of experimental bacterial keratitis. elucidation of the controlled-release behavior of metoprolol succinate from directly compressed xanthan gum/chitosan polymers: computational and experimental studies. an investigation into the relationship between xanthan gum film coating materials and predicted oro-esophageal gliding performance for solid oral dosage forms. effect of thermal processing on flow properties and stability of thickened fluid matrices formulated by tapioca starch, hydroxyl distarch phosphate (e-1442), and xanthan gum associating dysphagia-friendly potential. tragacanth gum/chitosan polyelectrolyte complexes-based hydrogels enriched with xanthan gum as promising materials for buccal application. effect of xanthan gum-based food thickeners on the dissolution profile of fluoroquinolones oral formulations. effect of ibuprofen entrapment procedure on physicochemical and controlled drug release performances of chitosan/xanthan gum polyelectrolyte complexes. xanthan gum as a carrier for bacterial cell entrapment: developing a novel immobilised biocatalyst. spray technology applications of xanthan gum-based edible coatings for fresh-cut lotus root (nelumbo nucifera). self-crosslinked chitosan/dialdehyde xanthan gum blended hypromellose hydrogel for the controlled delivery of ampicillin, minocycline and rifampicin. critical overlap concentration and intrinsic viscosity data of xanthan gum aqueous solutions in dimethyl sulfoxide. innovative bactericidal adsorbents containing modified xanthan gum/montmorillonite nanocomposites for wastewater treatment. physicochemical characterization of protein isolates of amaranth and common bean and a study of their compatibility with xanthan gum. chemical modification of xanthan gum through graft copolymerization: tailored properties and potential applications in drug delivery and wastewater treatment. an eco-friendly superabsorbent composites of xanthan gum-g-polyacrylic acid/semi-coke (xg-g-paa/sc) were fabricated via grafting of polyacrylic acid onto the xg in the presence of sc. the obtained products were characterized in combination with fourier transform infrared spectroscopy (ftir), thermogravimetric analysis (tga) and scanning electron microscopy (sem). the result indicated that the sc interacted with the polymeric network by hydrogen bond or electrostatic interaction. the swelling ratio of the best sample xg-g-paa/sc (15 wt%) in distilled water and 0.9 wt% nacl solution was 410.8 and 61.5 g/g by optimizing the polymerization conditions. in addition, compared with the blank sample (only containing soil), it can be found that adding a certain amount of xg-g-paa/sc can significantly improve the soil water retention efficiency, which can be further proved by the results of plant pot experiment. based on the above excellent swelling capacity, water holding capacity and plant growth promoting performance, it can be inferred that the xg-g-paa/sc is expected to become a water retaining agent or soil regulator for plant growth. this study investigated the protective effect and mechanism of action of combined use of rosmarinic acid (ra) and xanthan gum (xg) on the stability of anthocyanins (acns) in the presence of l-ascorbic acid (ph 3.0). the addition of ra and xg, alone and in combination, significantly enhanced the color stability of acns, and the combined use of ra and xg showed the best effect. ftir, 1h nmr, afm and computational molecular simulation analyses revealed that the improvement in acn stability following the combined addition of ra and xg was due to intermolecular interactions such as hydrogen bonding and van der waals forces. in the acn-ra-xg ternary complexes, xg had stronger binding interactions with acns than ra. our findings provide a valuable potential to enhance the stability of acns in the presence of ascorbic acid with the combined use of ra and xg. dietary fibers with high water-binding capacity (wbc), swelling capacity (sc) and fermentability regulate food intake and intestinal microbiota. however, dietary fibers with such properties are generally rare or expensive. we evaluated sc, wbc, fermentability and bacterial shifts during in vitro fermentations of guar gum (gg), xanthan gum (xg) and the combined gum (cg) of xg and gg. sc and wbc were enhanced by the combination of gg and xg. fermentation of cg showed similar short chain fatty acids production and lower molecular weight compared with gg. analyzing of fermentation kinetics by logistic-exponential model, initial fractional rate of degradation of cg were remarkable higher than gg. microbiota analysis revealed that gg enriched fusobacterium, bacteriodes and prevotella_9, and cg lead to promotion of sphaerochaeta, prevotella_9, bacteroides and christensenellaceae_r-7_group. these data suggest that combination of xg to gg changed hydration and fermentation characteristics of gg, and cg resulted in promotion of beneficial microbiota. herein, improvement of the stability of the water-in-oil-in-water (w/o/w) emulsions by addition of xanthan gum (xg)/locust bean gum (lbg) mixture in the inner water phase was aimed. the impact of xg/lbg mixture on the physical stability, microstructure and rheological properties of w/o/w emulsions was investigated. it was found that, compared with the control emulsions, the presence of xg/lbg mixture could improve the stability of w/o/w emulsions against coalescence. the tea polyphenols (tpps) and xg/lbg mixture were simultaneously included in the internal aqueous phase of the double emulsion and stored at 25 and 40 °c in the dark for 28 d. the results showed that xg/lbg mixture not only had a protective role for tpps encapsulated in the internal water phase, but also maintained more than 50% of the antioxidant capacity of tpps. the study investigated the quality of emulsions containing rabbit fat modified with vegetable oil. the modification of the fat and introducing it as a fatty base into the emulsion was dictated by consumer preferences. emulsion systems containing various fatty bases and viscosity modifier contents were evaluated in the terms of their stability (by means of turbiscan test), texture properties, color, and viscosity. moreover, the emulsions were assessed by a sensory panel in the context of the intensity of the following parameters: color, fragrance, consistency, greasiness, and hydration. the same characteristics were also subject to consumer evaluation. the results of the sensory assessment showed the sensory panel attributed higher scores to consistency and skin hydration to the emulsions formed with modified fats; these systems were more appreciated by consumers as well. the results confirmed a major role of sensory determinations in the development of new emulsion products. they also provide knowledge on modifications to product characteristics that would lead to the best possible quality and consumer acceptance. this research has also reaffirmed that looking for new fats among waste fats is becoming a solution to finding new fatty bases for emulsions. the natural origin of these components, and thus their agreeability with the human body, appear noteworthy as well. enrichment with unsaturated fatty acids is an added advantage of the enzymatic modification of rabbit fat with pumpkin seed oil and can be applied not only for food but also for skin applications. a novel hydrogel, named xgtte, was prepared by modification of acrylamide and trimethylolpropane triglycidyl ether (tte) on xanthan gum (xg). xgtte was utilized as an adsorbent to remove methyl orange (mo) from an aqueous solution. functional groups present in the external surface of xgtte, such as carboxyl, hydroxyl, and amino groups, were identified, and these functional groups are responsible for the occurrence of the mechanism for mo adsorption. the structure of xgtte after mo adsorption became more disordered. the adsorption dynamics and isotherms of mo onto xgtte conformed well to the pseudo-second-order kinetics and the freundlich equation, indicated that the adsorption included physical and chemical adsorption, with electrostatic interaction and hydrogen-bonding interaction. the calculated values of the thermodynamic parameters, such as changes in enthalpy (δh) and entropy (δs) were 27.30 kj/mol and 125.63 j/mol/k, respectively. the changes in free energy (δg) were - 9.53 kj/mol (293.15 k), -10.16 kj/mol (298.15 k), -10.78 kj/mol (303.15 k), -11.41 kj/mol (308.15 k), and - 12.04 kj/mol (313.15 k), respectively, indicating the adsorption process is endothermic, spontaneous, and entropic. the maximum adsorption capacities of xgtte2, xgtte3, xgtte4, xgtte5, and xgtte6 for mo were 18.62 ± 0.99, 21.92 ± 0.84, 28.60 ± 0.84, 29.56 ± 0.99, and 12.38 ± 0.84 mg/g, respectively. to formulate a xanthan gum-containing linezolid ophthalmic solution (lzd-xg) as a new antibiotic treatment against ocular bacterial infection. lzd-xg was prepared and evaluated for its in vitro/in vivo ocular tolerance, in vitro/in vivo antibacterial activity, and in vivo ocular penetration. the optimized lzd-xg exhibited good in vitro/in vivo eye tolerance. a prolonged ocular surface residence time of lzd-xg was observed after topical instillation, and the ocular permeation was significantly better for lzd-xg than fora linezolid (lzd) ophthalmic solution. the in vitro antimicrobial activity was significantly better with lzd-xg than with lzd. in vivo evaluation also confirmed a strong therapeutic treatment effect of lzd-xg, as it significantly improved the clinical symptoms, ameliorated the damage of staphylococcus aureus to ocular tissues, lowered the colony forming unit counts in the cornea, and decreased the myeloperoxidase activity in the cornea. lzd-xg was deemed a viable ophthalmic solution against ocular bacterial infection due to its excellent in vitro and in vivo characterizations. the development and evaluation of a controlled-release (cr) pharmaceutical solid dosage form comprising xanthan gum (xg), low molecular weight chitosan (lcs), and metoprolol succinate (ms) are reported. the research is, partly, based upon the utilization of computational tools: in this case, molecular dynamics simulations (mds) and the response surface method (rsm) in order to underpin the design/prediction and to minimize the experimental work required to achieve the desired pharmaceutical outcomes. the capability of the system to control the release of ms was studied as a function of lcs (% w/w) and total polymer (lcs and xanthan gum (xg)) to drug ratio (p/d) at different tablet tensile strengths. mds trajectories, obtained by using different ratios of xg/lcs as well as xg and high molecular weight chitosan (hcs), showed that the driving force for the interaction between xg and lcs is electrostatic in nature, the most favorable complex is formed when lcs is used at 15% (w/w) and, importantly, the interaction between xg and lcs is more favorable than that between xg and hcs. rsm outputs revealed that the release of the drug from the lcs/xg matrix is highly dependent on both the % lcs and the p/d ratio and that the required cr effect can be achieved when using weight fractions of lcs ≤ 20% and p/d ratios ≥2.6:1. results obtained from in vitro drug release and swelling studies on the prepared tablets showed that using lcs at the weight fractions suggested by mds and rsm data plays a major role in overcoming the high sensitivity of the controlled drug release effect of xg on ionic strength and ph changes of the dissolution media. in addition, it was found that polymer relaxation is the major contributor to the release of ms from lcs/xg tablets. using raman spectroscopy, ms was shown to be localized more in the core of the tablets at the initial stages of dissolution due to film formation between lcs and xg on the tablet surface, which prevents excess water penetration into the matrix. in the later stages of the dissolution process, the film starts to dissolve/erode, allowing full tablet hydration and a uniform drug distribution in the swollen tablet. oral drug therapy is generally provided in the form of solid oral dosage forms (sodf) that have to be swallowed and move throughout the oro-esophageal system. previous studies have provided evidence that the oro-esophageal transit of sodf depends on their shape, size, density, and surface characteristics. to estimate the impact of sodf surface coatings during esophageal transit, an in vitro system was implemented to investigate the gliding performance across an artificial mucous layer. in this work, formulations comprised of different slippery-inducing agents combined with a common film forming agent were evaluated using the artificial mucous layer system. xanthan gum (xg) and polyethylene glycol 1500 (peg) were applied as film-forming agents, while carnauba wax (cw), lecithin (le), carrageenan (ca), gellan gum (gg) and sodium alginate (sa), and their combination with sodium lauryl sulfate (sls), were applied as slippery-inducing components. all tested formulations presented lower static friction (sf) as compared to the negative control (uncoated disc, c, f0), whereas only cw/sls-based formulations showed similar performance to f0 regarding dynamic friction (df). the applied multivariate analysis approach allowed a higher level of detail to the evaluation and supported a better identification of excipients and respective concentrations that are predicted to improve in vivo swallowing safety. the flow behavior of the administrated fluid matrices demands careful assessments for stability when consumed by individuals with dysphagia. in the present study, we incorporated tapioca starch (ts), hydroxypropyl distarch phosphate (hdp), and xanthan gum (xg) as thickeners into different nectars (300 ± 20 mpa.s) undergoing thermal processing and evaluated their stability. the thickened nectars presented better water holding and oil binding capacities at 25 °c than 4 °c, and the nectars with ts provided the best results for both capacities as well as the highest solubility index and swelling power (p < 0.05). all prepared nectars appeared to be shear-thinning fluids with yield stress closely fitting the power law and casson models. xg-containing nectars presented a higher yield stress and consistency index. matrices thickened by hdp exhibited a higher viscoelastic property compared to those thickened by ts during thermal processing. ts nectars presented viscous behavior, whereas hdp and xg nectars presented elastic behavior at 80 °c processing. the 3 min thermal processing hdp nectars remained stable and met dysphagia-friendly requirements under 4 °c storage for 28 days, regardless of the type of fluid base (distilled water, sport drink, or orange juice). the employed thickeners present adequate physicochemical properties to be potentially utilized for producing dysphagia-friendly formulations. polyelectrolyte complexes based on the electrostatic interactions between the polymers mixed are of increasing importance, therefore, the aim of this study was to develop hydrogels composed of anionic tragacanth gum and cationic chitosan with or without the addition of anionic xanthan gum as carriers for buccal drug delivery. besides the routine quality tests evaluating the hydrogel's applicability on the buccal mucosa, different methods directed toward the assessment of the interpolymer complexation process (e.g., turbidity or zeta potential analysis, scanning electron microscopy and fourier-transform infrared spectroscopy) were employed. the addition of xanthan gum resulted in stronger complexation of chitosan that affected the hydrogel's characteristics. the formation of a more viscous pec hydrogel with improved mucoadhesiveness and mechanical strength points out the potential of such polymer combination in the development of buccal drug dosage forms. xanthan gum-based food thickeners (xg-fts) are often ingested by patients with dysphagia to prevent aspiration during drug treatment. reportedly, xg-fts affect tablet disintegration, drug dissolution rates, and reduce the efficacy of postprandial antihyperglycemic agents. the absorption rate and quantity of fluoroquinolone antimicrobial agents correlate with drug efficacy, raising concern about the impact of xg-fts. previously, we reported that film-coated tablets were less susceptible to the effects of xg-ft than conventional and orally disintegrating tablets. here, we compare the effect of xg-fts on dissolution profiles of three oral fluoroquinolone-based film-coated tablets by evaluating the dissolution of crushed products, fine granules, and film-coated fine granules. we examined formulations of tosufloxacin tosylate monohydrate (tflx), levofloxacin hemihydrate (lvfx), and ciprofloxacin hydrochloride hydrate (cpfx). the formulations were immersed in 20 ml of 1.5% (w/v) xg-ft aqueous solution for 2.5 min followed by a dissolution test using the paddle method according to the japanese pharmacopoeia (dissolution test solution ph 1.2; volume 900 ml; temperature 37 ± 0.5 °c). the dissolution profile was evaluated according to the dissolution quantity indicated in product specifications and guidelines for bioequivalence testing of generic drugs. the 15-min mean dissolution rate was determined for a formulation immersed in 1.5% (w/v) xg-ft aqueous solution and compared with that for a non-immersed formulation (control). fluoroquinolone film-coated tablets were mixed with starch-based fts, guar gum-based fts, or xg-fts to observe their appearances. the dissolution profile of lvfx film-coated tablets was not affected by xg-fts, but the dissolution of tflx and cpfx was delayed. for crushed film-coated tablets, the 15-min mean dissolution rate was significantly delayed for all three fluoroquinolones when compared with that of uncrushed products. the dissolution profile of tflx film-coated fine granules was unchanged by xg-fts. cpfx film-coated tablets and crushed products produced a gel-like precipitate when mixed with xg-fts and failed to meet product-dissolution specifications. a gel-like precipitate was also observed with guar gum-based fts. the effect of xg-fts on the dissolution profile of film-coated fluoroquinolone formulations varied depending on the formulation. the cpfx formulation formed a gel-like precipitate when immersed in xg-fts resulting in a significantly delayed dissolution. the effect of the entrapment procedure of a poorly water soluble drug (ibuprofen) on physicochemical and drug release performances of chitosan/xanthan polyelectrolyte complexes (pecs) was investigated to achieve controlled drug release as the ultimate goal. the formation of pecs for two drug entrapment procedures (before or after the mixing of polymers) at ph 4.6 and 5.6 and three chitosan-to-xanthan mass ratios (1:1, 1:2 and 1:3) was observed by continuous decrease in conductivity during the pecs formation and increased apparent viscosity and hysteresis values. the most extensive crosslinking was observed with ibuprofen added before the pecs formation at ph 4.6 and chitosan-to-xanthan mass ratio 1:1. the pecs prepared at polymers' mass ratios 1:2 and 1:3 had higher yield and drug entrapment efficiency. dsc and ft-ir analysis confirmed ibuprofen entrapment in pecs and the partial disruption of its crystallinity. all ibuprofen release profiles were similar, with 60-70% of drug released after 12 h, mainly by diffusion, but erosion and polymer chain relaxation were also included. potentially optimal can be considered the pec prepared at ph 4.6, ibuprofen entrapped before the mixing of polymers at chitosan-to-xanthan mass ratio 1:2, which provided controlled drug release by zero-order kinetics, high yield, and drug entrapment efficiency. xanthan gum (xan) is a widely used polysaccharide in various industries. because of its unique properties, in this study, an attempt was made to adopt the procedure of xanthan gum cross-linking for the entrapment of bacterial cells that are able to biodegrade naproxen. the developed procedure proved to be completely neutral for bacillus thuringiensis b1(2015b) cells, which demonstrated a survival rate of 99%. a negative impact of entrapment was noted for strain planococcus sp. s5, which showed a survival rate in the 93-51% range. to achieve good mechanical properties of the composites, they were additionally hardened using polydopamine (pda). xan/pda composites revealed a high stability in a wide range of ph, and their sorption capacity included both cationic and anionic molecules. analysis of the survival rate during storage at 4 °c in 0.9% nacl showed that, after 35 days, 98-99% of b1(2015b) and 47% of s5 cells entrapped in xan/pda remained alive. this study also presents the results of naproxen biodegradation conducted using xan/pda/b1(2015b) in a trickling filter with autochthonous microflora. hence, owing to the significant acceleration of drug biodegradation (1 mg/l in 14 days) and the chemical oxygen demand removal, the entrapped b1(2015b) cells in xan/pda composites showed a promising potential in bioremediation studies and industrial applications. in this study, the effect of spraying method as an application technique for xanthan gum-based edible coatings was investigated, based on its barrier and microbial properties on fresh-cut lotus root. xanthan gum solutions (0.1%, 0.3%, and 0.5%) were prepared and incorporated with 2% (w/w) citric acid as an anti-browning agent and 1% (w/w) glycerol as plasticizer. the coatings were then sprayed using a pilot spray system to 5 mm-thick slices of fresh-cut lotus root for 20 s, packed in polyethylene bags, stored for 16 d at 5 °c and analyzed for color, ph, morphology and microbial counts. it was found that spray-coated fresh-cut lotus root samples had significant reduction in the total color changes as compared to non-coated samples. the experimental results suggested that the spray coating treatments were effective in decreasing the enzymatic browning of fresh-cut lotus root during storage which could potentially increase its shelf-life in the market. in addition, we have also found that the xanthan gum-based spray coated treatments were also effective against inhibiting the growth of bacillus subtilis during 24 h of incubation which were indicated by the lower microbial counts recorded as compared to non-coated fresh-cut lotus root samples. in this part of the work, the author highlighted the spray coating technique of xanthan gum-based edible coatings as a promising strategy in improving the storage stability of fresh-cut lotus root during post-harvest storage. overall, the application of edible coatings is a promising strategy in extending the shelf life of fresh-cut lotus root. in the future, the author aims to widen the scope of the application of these coatings to other agricultural products which are prone to degradation during storage in the market. the design of improved biopolymeric based hydrogel materials with high load-capacity to serve as biocompatible drug carriers is a challenging task with vital implications in health sciences. in this work, chitosan crosslinked dialdehyde xanthan gum interpenetrated hydroxypropyl methylcellulose gels were developed for the controlled delivery of different antibiotic drugs including ampicillin, minocycline and rifampicin. the prepared hydrogel scaffolds were characterized by rheology method, ftir, sem, tga and compression analysis. in addition, gelation kinetics, swelling, in vitro degradation and drug release rate were studied under simulated gastrointestinal fluid conditions of ph 2.0 and 7.4 at 37 °c. results demonstrated the gel composition and structure affected drug release kinetics. the release study showed more than 50% cumulative release within 24 h for all investigated antibiotic drugs. in vitro cell cytocompatibility using mouse embryonic fibroblast cell lines depicted ≥80% cell viability, indicating the gels are non-toxic. finally, the antibacterial activity of loaded gels was evaluated against gram-negative and positive bacteria (escherichia coli, staphylococcus aureus and klebsiella pneumonia), which correlated well with swelling and drug release results. overall, the present study demonstrated that the produced hydrogel scaffolds serves as promising material for controlled antibiotic delivery towards microbial growth inhibition. this is the supplementary information of the research paper \"haemodynamics around confined microscopic cylinders\" by rodrigues et al. [1]. the critical overlap concentration of entanglement of polymer coils indicates whether a polymer solution is dilute or semidilute. here, the reader will find the determination of \n",
      "                         for xanthan gum aqueous solutions in 52 wt.% of dimethyl sulfoxide, often used as non-particulate blood analogues. from the shear flow curves of a dilution series of the polymer the zero-shear viscosities \n",
      "                         were obtained, allowing us to estimate the intrinsic viscosity \n",
      "                         based on the xanthan gum concentration of the fluids. two methodologies for doing so are described: using information from multi-concentration measurements and from a single polymer solution (rough estimate). with the intrinsic viscosity the determination of \n",
      "                         is straightforward. herein, we reported the preparation of novel antibacterial nanocomposites based on biodegradable polymers. the nanocomposites were applied as capable adsorbent for removing of malachite green (mg) dye, as well as inhibiting of e. coli and s. aureus growth as the most common pollutants for water. the grafted xanthan gum with poly(vinylimidazole) (xg-g-pvi) nanocomposites were synthesized in the presence of different montmorillonite (mmt) nanoclays concentrations (1%, 3% and 5%). the prepared modified xg nanocomposites were detected through xrd, sem-edx, ftir and tem. the maximum adsorption mg capacity was determined as 99.99% (909.1 mg/g) in basic medium at 30 °c for 90 min. the adsorption isotherm for removal of mg dye was studied against different models like langmuir, freundlich, temkin, floryhuggins isotherm models, however, the adsorption results were good fitted with langmuir isotherm model (r2 = 0.9942). additionally, various adsorption kinetic models: pseudo-first order, second order, pseudo-second order, and intra-particle diffusion models were studied for adsorption mechanism of mg dye on top of prepared nanocomposite surface. finally, the antibacterial activity outcomes displayed that the prepared xg-g-pvi/mmt nanocomposites had excellent inhibition growth for bacteria and the antibacterial activity increased abruptly with the increased of mmt nanoclay concentrations. vegetables are considered to be a sustainable source of promising biomaterials such as proteins and polysaccharides. in this study, four protein isolates (amaranth protein isolate api, amaranth globulin-rich protein isolate agr, bean protein isolate bpi, and bean phaseolin-rich protein isolate bpr) were structurally characterized under different ph conditions (2-12) and their compatibility behavior with xanthan gum (xg) in aqueous medium was described. all protein isolates showed β turn and β sheet (78.24-81.11%), as the major secondary structures without statistically significant difference under the ph conditions surveyed. protein isolates show solubility at ph ≤ 3 (40.4-85.1%) and ph ≥ 8 (57.6-99.9%) and surface hydrophobicity results suggest protein denaturation at ph ≤ 3. in the compatibility study, api/xg ratios between 1:1 and 5:1 at ph from 7 to 9 and the bpi/xg ratios from 1:1 to 20:1 at ph 7 form gels that do not require heating nor crosslinking agent addition. zeta potential results, on the other hand, evidenced that formation of gels is driven by attractive electrostatic interaction of the charged regions of both biopolymers and intermolecular interactions such as hydrogen bonds. xanthan gum (xg) possesses numerous hydroxyl groups, which are the focal points for graft copolymerization of synthetic monomers. a detailed mechanism of graft copolymerization is of utmost importance in obtaining new materials with desirable attributes. the physicochemical, thermal, and morphological changes gained after graft copolymerization is also described. the graft copolymerization can further improve adsorption efficiency of toxic heavy metals and synthetic dyes from wastewater or industrial effluents. the swelling and ph-sensitivity of graft copolymer are attractive features for the purpose of controlled drug delivery. despite a plethora of reports, comprehensive reviews on xg-based graft copolymers and their potential applications are scarce. hence, this review undertakes detailed discussion on the synthesis of xg-based graft copolymer, their properties and potential application in drug delivery and wastewater treatment, which would be interesting for the readers and budding scientists to progress further with polysaccharide research and explore new materials for the intended purposes. \n",
      "Euclidean Distance : 15.84416389465332\n",
      "\n",
      "\n",
      "Document: kinetic of no2 uptake by phleum pratense pollen: chemical and allergenic implications. phleum pratense pollen was exposed to no(2) in a reactor allowing a continuous analysis of no(2) concentration by ftir. the uptake coefficient of no(2) on pollen was calculated postulating a first order kinetic reaction and a value of (1.1 ± 0.1) x 10(-7) was determined. no(2) uptake was faster when the pollen water content was increased and when the pollen was pre-treated with ozone. the effect of no(2) exposure on pollen allergic properties was investigated by quantifying th2- and th1-associated chemokines in a model of human dendritic cells. cellular analysis clearly showed that cells exposed to fumigated pollen favored the production of chemokines known to promote th2-cell responses. altogether these data demonstrate that no(2) uptake by pollen directly correlates with increased th2 response in human cells,and are in favor of the involvement of no(2) pollution in the increase of allergic diseases. \n",
      "Euclidean Distance : 15.880023956298828\n",
      "\n",
      "\n",
      "Document: pollen production for 13 urban north american tree species: allometric equations for tree trunk diameter and crown area. proteomic identification of allergenic proteins in red oak ( pollen gene flow, male reproductive success, and genetic correlations among offspring in a northern red oak (quercus rubra l.) seed orchard. estimates of airborne pollen concentrations at the urban scale would be useful for epidemiologists, land managers, and allergy sufferers. mechanistic models could be well suited for this task, but their development will require data on pollen production across cities, including estimates of pollen production by individual trees. in this study, we developed predictive models for pollen production as a function of trunk size, canopy area, and height, which are commonly recorded in tree surveys or readily extracted from remote sensing data. pollen production was estimated by measuring the number of flowers per tree, the number of anthers per flower, and the number of pollen grains per anther. variability at each morphological scale was assessed using bootstrapping. pollen production was estimated for the following species: acer negundo, acer platanoides, acer rubrum, acer saccharinum, betula papyrifera, gleditsia triacanthos, juglans nigra, morus alba, platanus x acerfolia, populus deltoides, quercus palustris, quercus rubra, and ulmus americana. basal area predicted pollen production with a mean r2 of 0.72 (range: 0.41 - 0.99), whereas canopy area predicted pollen production with a mean r2 of 0.76 (range: 0.50 - 0.99). these equations are applied to two tree datasets to estimate total municipal pollen production and the spatial distribution of street tree pollen production for the focal species. we present some of the first individual-tree based estimates of pollen production at the municipal scale; the observed spatial heterogeneity in pollen production is substantial and can feasibly be included in mechanistic models of airborne pollen at fine spatial scales. red oak pollen is an important cause of allergic respiratory disease and it is widely distributed in north america and central europe. to date, however, red oak pollen allergens have not been identified. here, we describe the allergenic protein profile from red oak pollen. total proteins were extracted from red oak pollen using a modified phenolic extraction method, and, subsequently, proteins were separated by two-dimensional gel electrophoresis (2de) for both total protein stain (coomassie blue) and immunoblotting. a pool of 8 sera from red oak sensitive patients was used to analyze blotted proteins. protein spots were analyzed by mass spectrometry. electrophoretic pattern of total soluble proteins showed higher intensity bands in the regions of 26-40 and 47-52 kda. two dimensional immunoblots using pool sera from patients revealed four allergenic proteins spots with molecular masses in the range from 50 to 55 kda. mass spectrometry analysis identified 8 proteins including enolase 1 and enolase 1 chloroplastic, xylose isomerase (x1 isoform), mitochondrial aldehyde dehydrogenase, utp-glusose-1-phosphate uridylyltransferase, betaxylosidase/alpha-l-arabinofuranosidase and alpha- and beta subunits of atp synthase. this study has identified for first time 8 ige binding proteins from red oak pollen. these findings will pave the way towards the development of new diagnostic and therapeutic modalities for red oak allergy. northern red oak is a high-value hardwood used for lumber, furniture and veneer. intensively managed northern red oak seed orchards are required to obtain genetic gain for trait improvement. data from conifer seed orchards and natural and managed stands of hardwood trees have shed light on the distance over which pollen can move, and underscore the need for managerial attention to seed orchard design, placement, and maintenance. we used eleven microsatellite markers to investigate pollen gene flow, female mate choice, and male reproductive success in a clonal seed orchard of northern red oak based on paternity analysis of seed orchard offspring in progeny tests. nearly all (93%) offspring were sired by a male parent within the seed orchard. the mean number of male parents per year was 69.5, or 47.6% of all clones in the seed orchard. female clones in the early phenology group had more offspring sired from extra-orchard pollen (13%) than clones in the intermediate (5%) and late (1%) phenology groups. distance was the largest influence on pollination success, and pollination occurred most often by male trees in the same subline as the maternal tree. males in the early phenology group sired more offspring overall in the progeny pool and more offspring per mother tree than males in the intermediate or late phenology groups. average genetic correlations among all op progeny ranged between 0.2557 and 0.3529 with a mean of 0.28±0.01. the importance of progeny test genotyping for northern red oak improvement likely is increasing with the demand for improved varieties. the current study demonstrated the feasibility of post hoc assembly of full-sib families for genetic analysis. \n",
      "Euclidean Distance : 15.98211669921875\n",
      "\n",
      "\n",
      "Document: [effect of water substrate activity and relative air humidity on penicillium chrysogenum thom, aspergillus repens (corda) sacc., trichoderma viride pers., isolated from dwelling premises].  \n",
      "Euclidean Distance : 16.004043579101562\n",
      "\n",
      "\n",
      "Document: impact of zno addition on er three-electron reversible redox for a high-energy fluorophosphate cathode: na a bifunctional fluorophosphate electrolyte for safer sodium-ion batteries. cost-effective synthesis and superior electrochemical performance of sodium vanadium fluorophosphate nanoparticles encapsulated in conductive graphene network as high-voltage cathode for sodium-ion batteries. electrochemical and diffusional investigation of na ruthenium-oxide-coated sodium vanadium fluorophosphate nanowires as high-power cathode materials for sodium-ion batteries. synthesis and characterization of the crystal structure, the magnetic and the electrochemical properties of the new fluorophosphate linafe[po4]f. synthesis and characterization of the crystal structure and magnetic properties of the new fluorophosphate linaco[po4]f. the impact of the progressive addition of zno up to 5 mol% on the thermal, structural, and optical properties of er3+-doped phosphate glasses within the system napo3-naf-zno-ag2o is discussed. the glass network was found to depolymerize upon the addition of zno. this promotes a slight increase in the intensity of the emission at 1.5 µm as well as enhances the silver ions clustering ability under the heat treating. the ag-nanoparticles formed after moderate heat-treatment can further enhance the emission at 1.5 µm, whereas an excessive amount of the clusters leads to the opposite effect. the addition of zno helps to slightly increase the glass ability of the system. the crystallization behavior study revealed that surface crystallization is observed for all the glasses. it is found that even a small zno addition changes the crystalline phases formed after devitrification. moreover, the addition of zno decreases the crystallization tendency of the glass. a three-electron structural reaction for na3v2o2(po4)2f (na3vopf) in space group i4/mmm shows a priori stabilisation in terms of long-life in the voltage range of 2.0-4.5 v, with embedding of more than one sodium ion to generate na5vopf upon discharge to 1.0 v in the first cycle, thereby increasing the specific capacity from ∼170 ma h g-1. the capacity of the crystals was 180 ma h g-1 after 20 cycles. the results provide a probable route to improve fluorophosphate cathode performance. most of the currently developed sodium-ion batteries (sibs) have potential safety hazards due to the use of highly volatile and flammable alkyl carbonate electrolytes. to overcome this challenge, we report an electrochemically compatible and nonflammable electrolyte, tris(2,2,2-trifluoroethyl) phosphate (tfep) with low-concentration sodium bis(fluorosulfonyl)imide (0.9 m), which is designed not only to match perfectly with the hard carbon (hc) anode but also to enhance the thermal stability of sibs. experimental results and theoretical calculations reveal that tfep molecules have a significantly low barrier to decompose before na+ inserts into hc, forming a stable inorganic solid-electrolyte interface layer, thus improving the electrochemical and structural stabilities of hc anodes. an hc/na3v2(po4)3 full cell using tfep electrolyte shows a high capacity retention of 89.2% after 300 cycles and a dramatically reduced exothermic heat at elevated temperature, implying its potential application for safe and low-cost larger-scale energy storage. sodium vanadium fluorophosphate (na3(vo)2(po4)2f, denoted as nvpf) has attracted particular interests as cathode for high-energy sodium-ion batteries (sibs) owing to the high working potential, high specific capacity, and robust structural framework. however, it is challenged by the low electron conductivity and lack of available facile synthesis method. herein, an environmentally benign, cost-effective synthesis route is reported to produce nvpf nanoparticles encapsulated in conductive graphene network (nvpf/c), involving low-temperature synthesis of nvpf nanoparticles in absolute aqueous solvents and subsequent construction of conductive network through thermally induced transformation of graphene-oxide nanosheets. the resultant product is structurally and electrochemically investigated by combining x-ray diffraction, fourier transform infrared spectroscopy, scanning electron microscope, transition electron microscope, and electrochemical analysis. experimental results show that the optimized nvpf/c product possesses a three-dimensional graphene-encapsulation nanostructure composed of ∼100 nm nvpf nanoparticles and ∼4 nm carbon-coating layer. the unique hierarchical structure enables it cycling with excellent electrochemical performance in terms of a high reversible capacity (116.4 ma h g-1 at 0.2 c), excellent high-rate capability (87.4 ma h g-1 at 10 c) and long-term lifetime (82.1% capacity retention after 1200 cycles). it is indicated that the facile synthesis route can achieve high-performance nvpf/c material for sibs. sodium iron fluorophosphate (na2feiipo4f) was synthesized by economic solvothermal combustion technique using feiii precursors, developing one-step carbon-coated homogeneous product. synchrotron diffraction and mössbauer spectroscopy revealed the formation of single-phase product assuming an orthorhombic structure (s.g. pbcn) with feii species. this feiii precursor derived na2feiipo4f exhibited reversible na+ (de)intercalation with discharge capacity of 100 mah/g at a rate of c/10 involving flat feiii/feii redox plateaus located at 2.92 and 3.05 v (vs na/na+). it delivered good cycling stability and rate kinetics at room temperature. the stability of na2fepo4f cathode was further verified by electrochemical impedance spectroscopy at different stages of galvanostatic analysis. bond valence site energy (bvse) calculations revealed the existence of 2-dimensional na+ percolation pathways in the a-c plane with a moderate migration barrier of 0.6 ev. combustion synthesized na2feiipo4f forms an economically viable sodium battery material. although the capacity of this cathode is relatively low, this study continues systematic work, which attempts to broaden the scope of reversible sodium insertion materials. sodium-ion batteries are a very promising alternative to lithium-ion batteries because of their reliance on an abundant supply of sodium salts, environmental benignity, and low cost. however, the low rate capability and poor long-term stability still hinder their practical application. a cathode material, formed of ruo2 -coated na3 v2 o2 (po4 )2 f nanowires, has a 50 nm diameter with the space group of i4/mmm. when used as a cathode material for na-ion batteries, a reversible capacity of 120 mah g(-1) at 1 c and 95 mah g(-1) at 20 c can be achieved after 1000 charge-discharge cycles. the ultrahigh rate capability and enhanced cycling stability are comparable with high performance lithium cathodes. combining first principles computational investigation with experimental observations, the excellent performance can be attributed to the uniform and highly conductive ruo2 coating and the preferred growth of the (002) plane in the na3 v2 o2 (po4 )2 f nanowires. the new compound linafe[po(4)]f was synthesized by a solid state reaction route, and its crystal structure was determined using neutron powder diffraction data. linafe[po(4)]f was characterized by (57)fe mössbauer spectroscopy, magnetic susceptibility, specific heat capacity, and electrochemical measurements. linafe[po(4)]f crystallizes with orthorhombic symmetry, space group pnma, with a = 10.9568(6) å, b = 6.3959(3) å, c = 11.4400(7) å, v = 801.7(1) å(3) and z = 8. the structure consists of edge-sharing feo(4)f(2) octahedra forming fefo(3) chains running along the b axis. these chains are interlinked by po(4) tetrahedra forming a three-dimensional framework with the tunnels and the cavities filled by the well-ordered sodium and lithium atoms, respectively. the specific heat and magnetization measurements show that linafe[po(4)]f undergoes a three-dimensional antiferromagnetic ordering at t(n) = 20 k. the neutron powder diffraction measurements at 3 k show that each fefo(3) chain along the b-direction is ferromagnetic (fm), while these fm chains are antiferromagnetically coupled along the a and c-directions with a non-collinear spin arrangement. the galvanometric cycling showed that without any optimization, one mole of alkali metal is extractable between 1.0 v and 5.0 v vs. li(+)/li with a discharge capacity between 135 and 145 mah g(-1). the new compound linaco[po(4)]f was synthesized by a solid state reaction route, and its crystal structure was determined by single-crystal x-ray diffraction measurements. the magnetic properties of linaco[po(4)]f were characterized by magnetic susceptibility, specific heat, and neutron powder diffraction measurements and also by density functional calculations. linaco[po(4)]f crystallizes with orthorhombic symmetry, space group pnma, with a = 10.9334(6), b = 6.2934(11), c = 11.3556(10) å, and z = 8. the structure consists of edge-sharing coo(4)f(2) octahedra forming cofo(3) chains running along the b axis. these chains are interlinked by po(4) tetrahedra forming a three-dimensional framework with the tunnels and the cavities filled by the well-ordered sodium and lithium atoms, respectively. the magnetic susceptibility follows the curie-weiss behavior above 60 k with θ = -21 k. the specific heat and magnetization measurements show that linaco[po(4)]f undergoes a three-dimensional magnetic ordering at t(mag) = 10.2(5) k. the neutron powder diffraction measurements at 3 k show that the spins in each cofo(3) chain along the b-direction are ferromagnetically coupled, while these fm chains are antiferromagnetically coupled along the a-direction but have a noncollinear arrangement along the c-direction. the noncollinear spin arrangement implies the presence of spin conflict along the c-direction. the observed magnetic structures are well explained by the spin exchange constants determined from density functional calculations. \n",
      "Euclidean Distance : 16.090538024902344\n",
      "\n",
      "\n",
      "Document: modelling strawberry quality in a longitudinal study under the marketing concept of branding. polymorphism of the faomt and fafad1 genes for fruit flavor volatiles in strawberry varieties and wild species from the genetic collection of the michurin federal research center. transient expression assay in strawberry fruits. shelf life extension of strawberry juice by equivalent ultrasound, high pressure, and pulsed electric fields processes. reannotation of the cultivated strawberry genome and establishment of a strawberry genome database. a review of strawberry photobiology and fruit flavonoids in controlled environments. the nac transcription factor farif controls fruit ripening in strawberry. first report of  use of ultraviolet light to suppress powdery mildew in strawberry fruit production fields. development of gum arabic-based nanocomposite films reinforced with cellulose nanocrystals for strawberry preservation. identification of candidate genes influencing anthocyanin biosynthesis during the development and ripening of red and white strawberry fruits via comparative transcriptome analysis. improved vision-based detection of strawberry diseases using a deep neural network. comprehensive characterization of bile acids in human biological samples and effect of 4-week strawberry intake on bile acid composition in human plasma. preparation of modified polyvinyl formal vibration-damping material and its application in strawberry. the n-glycan processing enzymes β-d-n-acetylhexosaminidase are involved in ripening-associated softening in strawberry fruit. discovery of three loci increasing resistance to charcoal rot caused by macrophomina phaseolina in octoploid strawberry. contribution of pathogenic fungi to n independent validation of genomic prediction in strawberry over multiple cycles. chemical fungicides and  strawberry (fragaria ananassa duch.) alba extract attenuates dna damage in lymphocytes of patients with alzheimer's disease. marketing strategies, such as branding, redefine how consumers perceive quality and create new requirements related to season length and quality homogeneity, among others. for short-day (sd) strawberry cultivar brands, the commercial season is short due to a dependency on temperature and photoperiod. a plausible strategy to extend the commercialization period is to use different varieties within a single brand; however, this has led to inconsistent quality in other fruit crops. a form of quality assessment to evaluate the impact of a multi-varietal brand on sensory quality is a critical longitudinal study with several sources of variability, such as the inherent variation among assessors and fruit replicates that can affect the reliability of the results. therefore, this study aimed to develop a methodology to assess the sensorial and physicochemical quality of strawberry brands in two contexts: a short-term season composed of two sd cultivars and a long-term season with one sd and one day-neutral (dn) cultivar. new statistical models are proposed in this study. an anova mixed model with assessors and replicates as random terms and a multiple factor analysis highlighted a lack of homogeneity with regard to texture parameters and sourness, while partial least square models identified aroma and sweetness as the best quality indicators. this work has successfully illustrated a methodology that is capable of handling critical aspects of longitudinal studies by using univariate models that account for different sources of variability and constrained multivariate models to relate parameters with overall liking. a long-term brand is a viable solution to valorise strawberries, as parameter heterogeneity did not affect overall quality. fruit aroma is an important consumer attribute of strawberry varieties. the key volatile compounds of the aromatic complex of strawberry fruit are mesifurane (fruity and caramel aromas) and γ-decalactone (fruity, sweet, or peachy aroma). the mesifurane content in strawberry fruit is controlled by the faomt gene, which is mapped to the distal region of the long arm of chromosome vii-f.1. the γ-decalactone content in strawberry fruit is controlled by the fafad1 gene, mapped to the distal region of the long arm of chromosome iii-2. identification of forms carrying genes for fruit flavor volatiles is an important step in breeding varieties with fragrant fruit. the use of molecular markers allows highly reliable detection of target gene alleles in a genome at early developmental stages. this study involves molecular genotyping of fragaria l. varieties for the faomt and fafad1 genes, analysis of polymorphism of the loci in question, and identification of genotypes valuable for breeding. the objects of our study were wild species of the genus fragaria l. and strawberry varieties (fragaria × ananassa duch.) of different ecological and geographic origins. to assess the allelic states of the faomt gene, the codominant marker faomt-si/no was used, and for the fafad1 gene, the dominant marker fafad1. the functional allele of the faomt gene (faomt+) in the heterozygous state (faomt+faomt- genotype) was detected in 34.9 % of the accessions tested. the functional allele of the faomt gene in the homozygous state (faomt+faomt+ genotype) was detected in 51.2 % of the accessions. the homozygous state of the inactive allele (faomt-faomt- genotype) was detected in 13.9 % of the studied strawberry accessions. the fafad1 gene was identified in 25.6 % of the analyzed collection of strawberry genotypes, including the wild species f. orientalis los., f. moschata duch., f. ovalis rydb. the combination of functional alleles of the faomt and fafad1 genes was detected in 16.3 % of the analyzed forms. the wild species f. orientalis los. and f. moschata duch. and strawberry variety red gauntlet combine the functional allele of the fafad1 gene with the homozygous state of the active allele of the faomt gene; therefore, we recommend them as promising sources of high contents of mesifurane and γ-decactone in fruit in breeding programs for fruit aroma. strawberry, including the woodland strawberry fragaria vesca (2x) and the cultivated strawberry (fragaria × ananassa, 8x), has emerged as a model system for studying fruit development and ripening. transient expression provides a quick assay for gene functions or gene interactions. in strawberry, virus-induced gene silencing (vigs) and agrobacterium tumefaciens-mediated transformation in fruit have been widely used as the transient expression approaches. unlike vigs, the latter one can be utilized not only for gene knock-down, but also for overexpression and knock-out. here, we show the procedures of transiently expressing the 35s::fvemyb10 construct into fruit of the white-fruited f. vesca accession yellow wonder. as a master regulator of anthocyanin production, overexpressing fvemyb10 will cause fruit coloration, which was observed at one week post infiltration. we also exhibit the previous results of knocking down reduced anthocyanin in petioles (rap), encoding an anthocyanin transporter, by rnai in fruit of the strawberry cultivar 'sweet charlie'. overall, agrobacterium-mediated transient transformation in strawberry fruit is a quick and versatile approach for studying gene functions in fruit ripening. nonthermal processing technologies have focused on the production of safe, fresh-like and high quality food products very much in line with current consumer demands. it is a high priority to maintain the quality attributes of the food during its shelf life. in this study; microbial stability, physicochemical properties and phytochemical characteristics of strawberry juice (sj) pasteurized by ultrasonication (us) (55 °c, 0.29 w/ml acoustic energy density, 120 μm amplitude, 3 min), high pressure processing (hpp) (300 mpa, 1 min), and pulsed electric fields (pef) (35 kv/cm, 27 μs) were evaluated during 42 days of storage at 4 °c in comparison with conventional thermal pasteurization as a reference treatment (72 °c, 15 s). the nonthermal processes were equivalent in terms of e. coli inactivation since the selected processing conditions previously led to almost identical inactivation level (at least 5-log) of inoculated e. coli. thus, the current study demonstrates how these equivalent us, hpp, and pef treatments differ from each other in terms of their effect on sj natural microbiota and quality characteristics during refrigerated storage. results showed that us, hpp, and heat treatment ensured the microbial stability of sj for at least 42 days while pef extended the shelf life of sj by at least 28 days based on the natural microbiota. no significant difference was found for the total soluble solids of the processed samples (p > 0.05) whereas acidity and ph of the samples varied during the storage period (p < 0.05). immediately after processing, the total phenolic contents and antioxidant activities of sj were better retained by hpp and pef compared to thermal pasteurization. furthermore, hpp and pef significantly increased total anthocyanin content of sj by 15 and 17% with respect to untreated sj (p < 0.05). phytochemical characteristics of processed sj started to decrease after 7 days of storage irrespective of treatment type. hpp treated juices showed significantly higher levels of total anthocyanin and antioxidant activity at the final day of storage. principal component and cluster analysis showed that the processed sj samples had higher similarity to the untreated fresh sj during storage up to 14 days, while the samples beyond this storage period clustered together and discriminated from the rest indicating a decreased similarity to the fresh juice. this study rendered simultaneous evaluation of several quality characteristics during storage of pasteurized strawberry juice based on the equivalent processing approach and multivariate data analysis. under the selected processing conditions, hpp was the best option to extend the shelf life of sj and enhance its phytochemical characteristics. cultivated strawberry (fragaria × ananassa) is an important fruit crop species whose fruits are enjoyed by many worldwide. an octoploid of hybrid origin, the complex genome of this species was recently sequenced, serving as a key reference genome for cultivated strawberry and related species of the rosaceae family. the current annotation of the f. ananassa genome mainly relies on ab initio predictions and, to a lesser extent, transcriptome data. here, we present the structure and functional reannotation of the f. ananassa genome based on one pacbio full-length rna library and ninety-two illumina rna-seq libraries. this improved annotation of the f. ananassa genome, v1.0.a2, comprises a total of 108,447 gene models, with 97.85% complete buscos. the models of 19,174 genes were modified, 360 new genes were identified, and 11,044 genes were found to have alternatively spliced isoforms. additionally, we constructed a strawberry genome database (sgd) for strawberry gene homolog searching and annotation downloading. finally, the transcriptome of the receptacles and achenes of f. ananassa at four developmental stages were reanalyzed and qualified, and the expression profiles of all the genes in this annotation are also provided. together, this study provides an updated annotation of the f. ananassa genome, which will facilitate genomic analyses across the rosaceae family and gene functional studies in cultivated strawberry. rapid technology development in controlled environment (ce) plant production has been applied to a large variety of plants. in recent years, strawberries have become a popular fruit for ce production because of their high economic and nutritional values. with the widespread use of light-emitting diode (led) technology in the produce industry, growers can manipulate strawberry growth and development by providing specific light spectra. manipulating light intensity and spectral composition can modify strawberry secondary metabolism and highly impact fruit quality and antioxidant properties. while the impact of visible light on secondary metabolite profiles for other greenhouse crops is well documented, more insight into the impact of different light spectra, from uv radiation to the visible light spectrum, on strawberry plants is required. this will allow growers to maximize yield and rapidly adapt to consumer preferences. in this review, a compilation of studies investigating the effect of light properties on strawberry fruit flavonoids is provided, and a comparative analysis of how light spectra influences strawberry's photobiology and secondary metabolism is presented. the effects of pre-harvest and post-harvest light treatments with uv radiation and visible light are considered. future studies and implications for led lighting configurations in strawberry fruit production for researchers and growers are discussed. in contrast to climacteric fruits such as tomato, the knowledge on key regulatory genes controlling the ripening of strawberry, a non-climacteric fruit, is still limited. nac transcription factors mediate different developmental processes in plants. here, we identified and characterized farif (ripening inducing factor), a nac transcription factor that is highly expressed and induced in strawberry receptacles during ripening. functional analyses based on stable transgenic lines aimed at silencing farif by rna interference, either from a constitutive promoter or the ripe receptacle-specific exp2 promoter, as well as overexpression lines showed that farif controls critical ripening-related processes such as fruit softening and pigment and sugar accumulation. physiological, metabolome and transcriptome analyses of receptacles of farif-silenced and overexpression lines point to farif as a key regulator of strawberry fruit ripening from early developmental stages, controlling abscisic acid (aba) biosynthesis and signaling, cell wall degradation and modification, the phenylpropanoid pathway, volatiles production, and the balance of the aerobic/anaerobic metabolism. farif is therefore a target to be modified/edited to control the quality of strawberry fruits. strawberry (fragaria × ananassa duch.) is an important fruit crop in china. typical crown infection pockets symptoms were observed on the infected strawberry in liaoning province, china (121°60'e, 38°90'n) in the autumn of 2017. the disease incidence was estimated to be around 5 to 10 %, but could reach 30 to 40 % in some heavily affected plastic tunnels. the infected plants early displayed water-soaked symptoms on the abaxial leaf surface and subsequently developed reddish-brown shaped stripes and coalesced lesions on the adaxial leaf surface around the main veins (fig. 1-a, 1-b). several variable-size (0.3-0.8 mm in diameter) pockets were observed inside the crown tissues after dissection (fig. 1-c). the diseased plants rarely reached fruiting and were easily broken between the crown tissue and the stem, and would eventually die. to identify the causal agent of this disease, the several surface-disinfested infected main veins and crown tissues were individually ground in sterile water and plated on sucrose peptone agar(spa) medium (hayward 1960) with 10-fold serial dilutions and incubated at 25℃. a number of yellow colonies grew on the medium at the 10-4 dilution 7 days after plating (fig. 1-d) in all specimens. the colonies were aerobic, yellow, viscous, smooth, and gram-negative, which is a typical characteristic of xanthomonas. to confirm identity of the causal bacteria, 18 colonies selected randomly were subjected to polymerase chain reactions (pcr) for the amplification of the cpn60 (sahin et al. 2010), gyrb, rpod, and fyua (c manceau et al. 2011), respectively. the results showed that the 18 colonies are identical. the cpn60, gyrb, rpod, and fyua sequences of this isolate were deposited in genbank with accession numbers mt513132.1, mw233896, mw233897, and mw233895, respectively. blast searches with sequences of this isolate cpn60, gyrb, rpod, and fyua revealed 97.7%, 96.4%, 97.8%, and 97.3% similarity with the corresponding sequences of x. fragariae strain nbc2815 (lt853880.1), respectively. the resulting concatenated data set of cpn60-gyrb-rpod-fyua was used to build a multilocus sequence analysis (mlsa) by maximum likelihood criteria (fig. 2). the cpn60-gyrb-rpod-fyua sequences of the isolate from liaoning clustered in the clade containing the type strain of x. fragariae nbc2815, indicating that it belongs to x. fragariae. thus, the bacterial strain from liaoning was designated as x. fragariae strain yl19. to fulfill koch's postulates, the base of leaf petioles of disease-free strawberry plants were syringe-infiltrated inoculated with bacterial suspension (2×108 cfu) prepared from colonies of x. fragariae yl19 washed from spa plates. the inoculated and control (treated with sterile water) were placed in a chamber (25/20℃day/night,≥90% relative humidity(rh), 12/12 h photoperiod) for three months. after one month, water-soaked symptoms were observed in the crown tissues of all x. fragariae yl19-inoculated plants. two months after inoculation, a significant crown pocket similarly to initial symptoms observed in the field was developed on all inoculated plants. no symptoms were observed in the control plants. the bacteria were re-isolated from the symptomatic leaves, petioles and crowns, and confirmed as x. fragariae yl19 by the above mentioned morphological and molecular analyses. pathogenicity tests were conducted three times and the same results were obtained. it was reported that x. fragariae usually causes angular leaf spot, a serious bacterial disease in many strawberry production regions worldwide. the typical symptoms of angular leaf spot caused by x. fragariae include reddish-brown, irregular spots on the upper leaf surface, water-soaked lesions developed along leaf veins. although angular leaf spot caused by x. fragariae has been reported in tianjin and taiwan province, china (wang et al. 2017; wu et al. 2020), there is no report about the symptoms that infection pockets on crowns caused by x. fragariae strain yl19 as described above. this result indicated that yl19 is different from the other two x. fragariae strains reported in china or the disease caused by yl19 could be a severe case of angular leaf spot and vascular decline or collapse in strawberry (bradbury, 1977). . to the best of our knowledge, these results showed a previously unreported new strain yl19 of x. fragariae is the causal agent of crown infecton pocket in strawberry in china, it may lead to serious losses to the local strawberry industry. this report will assist in developing management measures for this disease promptly. we designed and deployed an apparatus to apply uv light for suppression of powdery mildew in open field production of strawberry. the unit was evaluated in a commercial production field for one season, and for two additional seasons in open field research plots at the university of florida gulf coast research and education center. the apparatus contained two 180 cm long hemicylindrical arrays of twenty 55 w low-pressure discharge uv-c lamps (operated at 30 w; peak wavelength 254 nm) backed by polished aluminum reflectors covering two adjacent beds of the strawberry planting. the lamp arrays were suspended within a steel carriage that was tractor-drawn through the planting at 2.3, 4.6 and 5.6 km/h. night-time applications of uv-c at doses ranging from 65 to 170 j.m-2 either once or twice weekly provided suppression of foliar and fruit disease that was consistently equal to or better than that provided by a commercial calendar-based fungicide spray program. the present study aimed to develop a new bio-nanocomposite film based on gum arabic (ga) reinforced with cellulose nanocrystals (cnc). cnc was successfully fabricated and its microstructure was characterized. subsequently, the effects of cnc on the rheological, physicochemical and functional properties of ga-based films were systematically evaluated. results showed that the tensile strength (2.21 mpa) and elongation at break (62.79%) of film incorporated with 4% (w/w) cnc were effectively increased compared with the ga film (1.08 mpa and 42.50%). additionally, 4% cnc reduced the water vapor and oxygen permeability by 10.61% and 25.30% respectively, while improved the ultraviolet light barrier and thermal stability of film. the well dispersion and filling effect of nanofiller contributed to form a compact and homogeneous film structure. furthermore, the film containing 4% cnc decreased the weight loss of strawberries by 23.80% compared with the control group, thus delaying the deterioration of strawberry quality during storage. strawberries are one of the most economically important berry fruits worldwide and exhibit colours ranging from white to dark red, providing a rich genetic resource for strawberry quality improvement. in the present study, we conducted transcriptome analyses of three strawberry cultivars, namely, 'benihoppe', 'xiaobai', and 'snow white', and compared their gene expression profiles. among the high-quality sequences, 5,049 and 53,200 differentially expressed genes (degs) were obtained when comparing the diploid and octoploid strawberry genomes and analysed to identify anthocyanin-related candidate genes. sixty-five degs in the diploid genome (transcriptome data compared to the diploid strawberry genome) and 317 degs in the octoploid genome (transcriptome data compared to the octoploid strawberry genome) were identified among the three cultivars. among these degs, 19 and 70 anthocyanin pathway genes, six and 42 sugar pathway genes, 23 and 101 hormone pathway genes, and 17 and 104 transcription factors in the diploid and octoploid genomes, respectively, correlated positively or negatively with the anthocyanin accumulation observed among the three cultivars. real-time qpcr analysis of nine candidate genes showed a good correlation with the transcriptome data. for example, the expression of pal was higher in 'benihoppe' and 'xiaobai' than in 'snow white', consistent with the rna-seq data. thus, the rna-seq data and candidate degs identified in the present study provide a sound basis for further studies of strawberry fruit colour formation. detecting plant diseases in the earliest stages, when remedial intervention is most effective, is critical if damage crop quality and farm productivity is to be contained. in this paper, we propose an improved vision-based method of detecting strawberry diseases using a deep neural network (dnn) capable of being incorporated into an automated robot system. in the proposed approach, a backbone feature extractor named plantnet, pre-trained on the plantclef plant dataset from the lifeclef 2017 challenge, is installed in a two-stage cascade disease detection model. plantnet captures plant domain knowledge so well that it outperforms a pre-trained backbone using an imagenet-type public dataset by at least 3.2% in mean average precision (map). the cascade detector also improves accuracy by up to 5.25% map. the results indicate that plantnet is one way to overcome the lack-of-annotated-data problem by applying plant domain knowledge, and that the human-like cascade detection strategy effectively improves the accuracy of automated disease detection methods when applied to strawberry plants. primary bile acids (bas) and their gut microbial metabolites have a role in regulating human health. comprehensive characterization of bas species in human biological samples will aid in understanding the interaction between diet, gut microbiota, and bile acid metabolism. therefore, we developed a qualitative method using ultra-high performance liquid chromatography (uhplc) coupled with a quadrupole time-of-flight (q-tof) to identify bas in human plasma, feces, and urine samples. a quantitative method was developed using uhplc coupled with triple quadrupole (qqq) and applied to a previous clinical trial conducted by our group to understand the bile acid metabolism in overweight/obese middle-aged adults (n = 34) after four weeks strawberry vs. control intervention. the qualitative study tentatively identified a total of 81 bas in human biological samples. several ba glucuronide-conjugates were characterized for the first time in human plasma and/or urine samples. the four-week strawberry intervention significantly reduced plasma concentrations of individual secondary bas, deoxycholic acid, lithocholic acid and their glycine conjugates, as well as glycoursodeoxycholic acid compared to control (p < 0.05); total glucuronide-, total oxidized-, total dehydroxyl-, total secondary, and total plasma bas were also lowered compared to control (p < 0.05). the reduced secondary bas concentrations suggest that regular strawberry intake modulates the microbial metabolism of bas. mechanical damage and microbial infestation were critical issues during strawberry transportation. this study prepared a modified polyvinyl formal (pvfm) vibration-damping material and investigated its application in strawberry transportation. this modified pvfm was primarily composed of polyvinyl alcohol, starch, formaldehyde, and allyl isothiocyanate. our results showed that the modified pvfm had good vibration-damping and antifungal properties with 0.137 g/m3 of density, a water absorption ratio value of 6.24, 278 pa of tensile strength, 3.86 pa of elastic modulus, a buffer coefficient of 0.318, and antifungal rates of 68.8% and 80.6% against bacillus anthracis and botrytis cinerea, respectively. moreover, this modified pvfm could enhance the storage life of strawberry by improving pal activity from 376 to 3,125 u/g, inhibiting pod activity with a maximum value of 3.1 u/g, and decreasing cat activity from 324 to 35 u/g. the modified pvfm presented a good vibration-damping capacity and the potential to extend strawberry shelf life. practical applications: berry fruits were vulnerable to mechanical damage caused by vibration during transportation, resulting in quality deterioration, rot, and aging. in this study, modified polyvinyl formal (pvfm) was prepared and was of low density and displayed good mechanical, vibration-damping, and antifungal properties. the modified pvfm could improve the storage life of strawberry. thus, the modified pvfm could have potential application in berry fruits transportation. excessive softening of fruits during the ripening process leads to rapid deterioration. n-glycan processing enzymes are reported to play important roles during fruit ripening associated softening. efforts have been made to identify and purify β-d-n-acetyl hexosaminidase (β-hex) from strawberry fruit and also to investigate its function during ripening. more than that, the postharvest treatment effect of alginate oligosaccharides (aos) at a concentration of 0.1 g l-1 on fruit firmness and the activity of n-glycan processing enzymes were also investigated during the storage of strawberry. results demonstrated that the full-length of β-hex1 and β-hex2 genes were 2186 and 2013 bp, including an orf of 1598 and 1724 bp and encoding 532 and 574 amino acids with a predicted molecular weight of 60 and 71 kda, respectively. moreover, β-hex enzyme activity and the expression of their encoding genes increased during the ripening of strawberry. in addition, postharvest application of aos delayed the loss of firmness and suppressed the activity of n-glycan processing enzymes (α-man and β-hex) along with n-glycan processing enzymes associated genes expression resulting in delayed fruit softening. therefore, our study suggests that n-glycan processing enzymes may play roles in strawberry softening and aos treatment suppressed enzymes activity and preserve firmness of the fruit. charcoal rot caused by macrophomina phaseolina is an increasing economic problem in annualized strawberry production systems around the world. currently there are no effective post-fumigation chemical controls for managing charcoal rot, and no information is available on the genetic architecture of resistance to m. phaseolina in strawberry (fragaria ×ananassa). in this study, three multiparental discovery populations and two validation populations were inoculated at planting and evaluated for mortality in three consecutive growing seasons. genome-wide snp genotyping and pedigree-based analysis with flexqtl™ software were performed. two large-effect quantitative trait loci (qtl) increasing charcoal rot resistance were discovered and validated in cultivated germplasm. farmp1 was located on linkage group 2 a in the interval 20.4 cm to 24.9 cm, while farmp2 was located on linkage group 4b in the interval 41.1 cm to 61.2 cm. together these qtl explained 27% and 17% of the phenotypic variance in two discovery populations consisting of elite breeding germplasm. for both qtl, the resistant allele showed some evidence of partial dominance, but no significant interaction was detected between the two loci. as the dosage of resistant alleles increased from 0-4 across the two qtl, mortality decreased regardless of the combination of alleles. a third locus, farmp3 on 4 d, was discovered in fvc 11-58, a reconstituted f. × ananassa originating from diverse f. virginiana and f. chiloensis accessions. this locus accounted for 44% of phenotypic variation in four segregating crosses. these findings will form the basis for dna-informed breeding for resistance to charcoal rot in cultivated strawberry. intensively managed agriculture land is a significant contributor to nitrous oxide (n2o) emissions, which adds to global warming and the depletion of the ozone layer. recent studies have suggested that fungal dominant n2o production may be promoted by pathogenic fungi under high nitrogen fertilization and continuous cropping. here, we measured the contribution of fungal communities to n2o production under intensively managed strawberry fields of three continuous cropping years (1, 5, and 10 years) and compared this adjacent bare soil. higher n2o emission was observed from the 10-year field, of which fungi and prokaryotes accounted for 79.7% and 21.3%, respectively. fungal population density in the 10-year field soil (4.25 × 105 colony forming units per g (cfu/g) of air-dried soil) was greater than the other cropping years. illumina miseq sequencing of the nirk gene showed that long-term continuous cropping decreased the diversity of the fungal denitrifier community, but increased the abundance of fusarium oxysporum. additionally, f. oxysporum produced large amounts of n2o in culture and in sterile 10-year field soil. a systemic infection displayed by bioassay strawberry plants after inoculation demonstrated that f. oxysporum was a pathogenic fungus. together, results suggest that long-term intensively managed monocropping significantly influenced the denitrifying fungal community and increased their biomass, which increased fungal contribution to n2o emissions and specifically by pathogenic fungi. key points: • distinguishing the role of fungi in long-term continuous cropping field. • identifying the abundant fungal species with denitrifying ability. the university of florida strawberry (fragaria × ananassa) breeding program has implemented genomic prediction (gp) as a tool for choosing outstanding parents for crosses over the last five seasons. this has allowed the use of some parents 1 year earlier than with traditional methods, thus reducing the duration of the breeding cycle. however, as the number of breeding cycles increases over time, greater knowledge is needed on how multiple cycles can be used in the practical implementation of gp in strawberry breeding. advanced selections and cultivars totaling 1,558 unique individuals were tested in field trials for yield and fruit quality traits over five consecutive years and genotyped for 9,908 snp markers. prediction of breeding values was carried out using bayes b models. independent validation was carried out using separate trials/years as training (trn) and testing (tst) populations. single-trial predictive abilities for five polygenic traits averaged 0.35, which was reduced to 0.24 when individuals common across trials were excluded, emphasizing the importance of relatedness among training and testing populations. training populations including up to four previous breeding cycles increased predictive abilities, likely due to increases in both training population size and relatedness. predictive ability was also strongly influenced by heritability, but less so by changes in linkage disequilibrium and effective population size. genotype by year interactions were minimal. a strategy for practical implementation of gp in strawberry breeding is outlined that uses multiple cycles to predict parental performance and accounts for traits not included in gp models when constructing crosses. given the importance of relatedness to the success of gp in strawberry, future work could focus on the optimization of relatedness in the design of trn and tst populations to increase predictive ability in the short-term without compromising long-term genetic gains. chemical and biological agents were evaluated to inhibit colletotrichum fructicola, phytophthora cactorum, and lasiodiplodia theobromae causing strawberry diseases. mycelial growths of c. fructicola were gradually arrested by increasing concentrations of fungicides pyraclostrobin and iminoctadine tris (albesilate). p. cactorum and l. theobromae were more sensitive to pyraclostrobin compared to c. fructicola, but iminoctadine tris (albesilate) was not or less effective to limit p. cactorum or l. theobromae, respectively. bacillus siamensis h30-3 was antagonistic against the three pathogens by diffusible as well as volatile molecules, and evidently reduced aerial mycelial formation of p. cactorum. b. siamensis h30-3 growth was declined by at least 0.025 mg/ml of pyraclostrobin. the two fungicides additively inhibited mycelial growths of c. fructicola, but not of p. cactorum and l. theobromae. b. siamensis h30-3 volatiles led to less growth of c. fructicola than one reduced by the fungicides. taken together, in vitro antimicrobial activities of the two fungicides together with or without b. siamensis h30-3 volatiles may be cautiously incorporated into integrated management of strawberry diseases dependent on causal pathogens. increased levels of oxidative stress and oxidative dna damage are common features in the pathology of alzheimer's disease (ad) found in neurons and peripheral cells like peripheral blood lymphocytes (pbl). natural products such as strawberry cultivar alba are an important source of bioactive nutrients that could help in lowering both the oxidative stress and dna damage levels. the objective was to estimate the effects of alba extract on dna damage in peripheral blood lymphocytes of sporadic ad (aged 60-84 years) patients, and healthy elderly (aged 69-83 years) and young (aged 21-30 years) individuals in in vitro conditions. comet assay was used as a sensitive technique for the evaluation of pbl dna damage levels. reduction of basal dna damage level in pbl was shown in the young group after the incubation with alba extract ranging from 25 to 200 μg/ml, with 100 μg/ml being the most effective concentration. selected alba extract of 100 μg/ml was further used for pbl treatment of ad and healthy elderly age matched group, displaying potential to significantly attenuate dna damage levels in both groups (p < .05). alba extract displayed biological activity against oxidative dna damage, suggesting that its functional ingredients may have beneficial health effects. practical applications: the data obtained in this preliminary study displayed that strawberry alba extract is efficient against dna damage induced by endogenous and exogenous oxidative stress in peripheral blood lymphocytes of alzheimer`s disease in vitro. an active area of future research of alba cultivar should be to determine the trials in in vivo systems. our findings also suggest that alba cultivar's functional ingredients potentially may have beneficial health effects in ad. \n",
      "Euclidean Distance : 16.154281616210938\n",
      "\n",
      "\n",
      "Document: the effect of grazing level and ageing time on the physicochemical and sensory characteristics of beef meat in organic and conventional production. sire effects on carcass of beef-cross-dairy cattle: a case study in new zealand. genomic prediction using alternative strategies of weighted single-step genomic blup for yearling weight and carcass traits in hanwoo beef cattle. influence of supplementing  effects of corn silage inclusion level and type of anabolic implant on animal growth performance, apparent total tract digestibility, beef production per hectare, and carcass characteristics of finishing steers. alternative management systems of beef cattle manure for reducing nitrogen loadings: a case-study approach. an engineered reporter phage for the fluorometric detection of  simulation modelling to estimate the herd-sensitivity of various pool sizes to test beef herds for johne's disease in australia. modeling intervention scenarios during potential foot-and-mouth disease outbreaks within u.s. beef feedlots. identification of predictor genes for feed efficiency in beef cattle by applying machine learning methods to multi-tissue transcriptome data. invited review: beef-on-dairy-the generation of crossbred beef × dairy cattle. vitamin a regulates intramuscular adipose tissue and muscle development: promoting high-quality beef production. genetic changes in beef cow traits following selection for calving ease. reducing climate impacts of beef production: a synthesis of life cycle assessments across management systems and global regions. sunnhemp (crotalaria juncea, l.) silage can enrich rumen fermentation process, microbial protein synthesis, and nitrogen utilization efficiency in beef cattle crossbreds. dry-aged beef manufactured in japan: microbiota identification and their effects on product characteristics. wide-scope multi-residue analysis of pesticides in beef by ultra-high-performance liquid chromatography coupled with quadrupole time-of-flight mass spectrometry. effect of pre-slaughter fasting duration on physiology, carcass and meat quality in beef cattle finished on pastures or feedlot. application of proteomics to understand the molecular mechanisms determining meat quality of beef muscles during postmortem aging. effect of feeding a pellet diet containing high sulphur with fresh cassava root supplementation on feed use efficiency, ruminal characteristics and blood metabolites in thai native beef cattle. this study investigated the influence of the production system (conventional vs. organic), the grass consumption level and the ageing period (7 and 14 days) on beef quality. three groups of samples from limousin × avileña calves were analysed: f100, formed by animals fed 100% on forage; f74, formed by animals fed on an average amount of forage of 74%; and f35, formed by animals fed on straw fodder (35%) and concentrate (65%). the results showed that the higher the grass content, the lower the fat oxidation and the higher the n-3 content, but also the higher the sfa level, the initial warner-bratzler shear force (wbsf), and the more residue it leaves on chewing. as for the effect of production system, organic samples showed higher intramuscular fat content and lower moisture and mufa content. these samples were darker and showed lower values for flavour quality. among the organic samples, f100 had higher n-3 and cla content and higher values for colour, hardness, odour and flavour quality. increased ageing time may improve the sensory characteristics of the meat, especially in the case of the f100 samples. the results pointed out that f100 samples aged at least 14 days showed the best physico-chemical, nutritional and sensory characteristics. there is interest in increasing the carcass value of surplus calves born in the dairy industry that are reared for beef production in new zealand. this experiment evaluated the carcass of angus and hereford sires via progeny testing of beef-cross-dairy offspring grown on hill country pasture. weight and carcass traits were analyzed from 1015 animals and 1000 carcasses of 73 sires. the mean of the progeny group means was 567 kg for live weight at slaughter, 277 kg for carcass weight, 48.9% for dressing-out, 240.3 cm for carcass length, 73.6 cm2 for eye muscle area, 7.4 mm for rib fat depth, 0.91 for marble score, 3.05 for fat color score, 3.01 for meat color score, and 5.62 for ultimate ph. sire affected (p < 0.05) carcass size and fat traits, but not fat color, meat color, or ultimate ph (p > 0.05). there was a 46 kg increase in carcass weight between the best and worst sires tested. carcass fat traits were the most variable among sires. the use of genetically superior beef-breed sires over dairy-breed cows has the potential to increase carcass weights from surplus calves born in the dairy industry, while maintaining adequate fat levels and carcass quality. the weighted single-step genomic best linear unbiased prediction (gblup) method has been proposed to exploit information from genotyped and non-genotyped relatives, allowing the use of weights for single-nucleotide polymorphism in the construction of the genomic relationship matrix. the purpose of this study was to investigate the accuracy of genetic prediction using the following single-trait best linear unbiased prediction methods in hanwoo beef cattle: pedigree-based (pblup), un-weighted (ssgblup), and weighted (wssgblup) single-step genomic methods. we also assessed the impact of alternative single and window weighting methods according to their effects on the traits of interest. the data was comprised of 15,796 phenotypic records for yearling weight (yw) and 5622 records for carcass traits (backfat thickness: bft, carcass weight: cw, eye muscle area: ema, and marbling score: ms). also, the genotypic data included 6616 animals for yw and 5134 for carcass traits on the 43,950 single-nucleotide polymorphisms. the ssgblup showed significant improvement in genomic prediction accuracy for carcass traits (71%) and yearling weight (99%) compared to the pedigree-based method. the window weighting procedures performed better than single snp weighting for cw (11%), ema (11%), ms (3%), and yw (6%), whereas no gain in accuracy was observed for bft. besides, the improvement in accuracy between window wssgblup and the un-weighted method was low for bft and ms, while for cw, ema, and yw resulted in a gain of 22%, 15%, and 20%, respectively, which indicates the presence of relevant quantitative trait loci for these traits. these findings indicate that wssgblup is an appropriate method for traits with a large quantitative trait loci effect. the aim of the study was to evaluate the effect of crude protein (cp) levels in concentrate and sesbania grandiflora pod meal (sg) supplementation on feed intake, rumen fermentation, and methane (ch4) mitigation in thai purebred beef cattle. four cattle with 100 ± 5.0 kg body weight were used in this study. a 2 × 2 factorial experiment in a 4 × 4 latin square design was conducted, in which factor a was the cp levels in concentrate of 14% and 16% of dry matter (dm) and factor b was the supplement levels of sg at 0.4% and 0.6% dm intake, respectively. the results showed that the cp content in concentrate and sg supplementation had no interaction effect on intake, digestibility, ruminal ecologies, ruminal fermentation products, and nitrogen utilization. increasing cp content to 16% significantly (p < 0.05) increased the ruminal ammonia nitrogen (nh3-n), nitrogen (n) intake, n absorption, and n retention. sg supplementation significantly (p < 0.05) decreased cp digestibility, nh3-n, blood urea nitrogen, and protozoa. in addition, sg significantly decreased acetate (c2), acetate to propionate ratio, methane, and fecal n excretion, while it significantly increased total volatile fatty acids (vfas) and propionate (c3) concentration. in conclusion, sg could mitigate methane emission and enhance nitrogen utilization. maine-anjou × angus cross-bred steers (n = 156 steers; initial body weight (bw) 366 ± 37.2 kg) were used in a 132 d finishing study conducted at the ruminant nutrition center (rnc) in brookings, sd. steers were blocked by weight (n = 5 bw blocks) and randomly assigned to an implant and dietary treatment of a randomized complete block design with each pen containing seven to eight steers (n = 20 pens). dietary treatments consisted of (1) 15% (cs15) or (2) 30% corn silage (cs30) where corn silage displaced corn grain in the diet. steers received one of two implants (both from zoetis, parsippany, nj) containing equal doses of trenbolone acetate (tba) and estradiol benzoate (eb): (1) synovex plus (non-coated implant; 200 mg tba and 28 mg eb; plus) or (2) synovex one feedlot (coated implant; 200 mg tba and 28 mg eb; one-f). bunks were managed using a slick bunk approach, and all diets contained dry matter (dm) basis 33 mg/kg monensin sodium. all steers were offered ad libitum access to feed, and feeding occurred twice daily in equal portions. there was no interaction between the implant and dietary treatment for any variables measured (p ≥ 0.08). carcass-adjusted basis final bw, average daily gain (adg), and grain to feed (g:f) were increased (p ≤ 0.02) by 2.2%, 6.5%, and 7.2%, respectively, for cs15. observed net energy (ne) and the ratio of observed-to-expected ne for maintenance and gain was not influenced (p ≥ 0.15) by silage inclusion treatment. beef production per hectare was not impacted (p ≥ 0.13) by corn silage inclusion level. fecal output was increased, and digestibility coefficients for dry matter, organic matter, and crude protein were decreased in cs30 (p ≤ 0.03). dressing percent and hot carcass weight (hcw) were greater (p ≤ 0.02) in cs15. implant type did not influence any traits measured (p ≥ 0.14) except for marbling. marbling was decreased for plus (433 vs. 466 ± 17.5; p = 0.02) compared to one-f steers. similar beef produced per hectare of crop land-based upon silage feeding level means producers can feed greater inclusions of corn silage to finishing cattle without impacting carcass quality or beef production; implanting with a coated implant had no detrimental effects to growth performance but increases marbling scores. livestock manure is one of the main sources of agricultural nonpoint source pollution and poses a great threat to the environment and human health. sustainable management of manure via recycling is an effective means to tackle the problem. based on field interviews in china, multiple case studies were employed to investigate alternative manure management systems. four conclusions arose. first, compost-based systems, product-based systems, substrate-based systems, and biogas-based systems were identified as four main types of manure management systems, with each possessing its success factors and risk factors. the adoption of a system was driven by various factors. second, market-oriented operation was the dominant operation mechanism of all the manure management systems. third, compared to direct application of manure to croplands, all the four manure management systems could reduce nitrogen loadings from livestock farms and lower their environmental effects. among the systems, biogas-based systems could reduce nitrogen loadings to the greatest extent, followed by product-based systems and substrate-based systems, and then by compost-based systems. lastly, integrated management of manure with mixed recycling systems is imperative for reducing its environmental effects, which can benefit from the increasing role of third-party entities in manure recycling. policy implications were also discussed. despite enhanced sanitation implementations, foodborne bacterial pathogens still remain a major threat to public health and generate high costs for the food industry. reporter bacteriophage (phage) systems have been regarded as a powerful technology for diagnostic assays for their extraordinary specificity to target cells and cost-effectiveness. our study introduced an enzyme-based fluorescent assay for detecting the presence of e. coli using the t7 phage engineered with the lacz operon which encodes beta-galactosidase (β-gal). both endogenous and overexpressed β-gal expression was monitored using a fluorescent-based method with 4-methylumbelliferyl β-d-galactopyranoside (mug) as the substrate. the infection of e. coli with engineered phages resulted in a detection limit of 10 cfu/ml in ground beef juice after 7 h of incubation. in this study, we demonstrated that the overexpression of β-gal coupled with a fluorogenic substrate can provide a straightforward and sensitive approach to detect the potential biological contamination in food samples. the results also suggested that this system can be applied to detect e. coli strains isolated from environmental samples, indicating a broader range of bacterial detection. johne's disease is a chronic intestinal disease affecting livestock. it leads to the shedding of mycobacterium avium subspecies paratuberculosis (map) in the faeces, wasting and eventually death, with animal welfare, economic, and trade implications. the johne's beef assurance scheme, used in australia to determine the risk of johne's disease on beef properties and facilitate trade, is based on testing a subset of the herd with pooled faecal quantitative pcr. this study aimed to model the herd-sensitivity of pooled faecal testing under different australian farming scenarios. animals from simulated herds were randomly sampled and allocated into their respective pools. each tested pool was provided a test outcome, with herd-sensitivity estimated as the probability of detecting a truly infected herd. the models simulated the test performance for the 'sample' and 'check' tests used in the assurance schemes (recommended sample sizes of 300 and 50, respectively) for a range of herd sizes, infection prevalence and map faecal shedding levels for the pool sizes of 5, 10, 15 and 20. sensitivity and specificity input values of each pool size were obtained from a previous laboratory investigation. the herd-sensitivity estimate increased with herd size and infection prevalence levels, regardless of the pool size. higher herd-sensitivity was also achieved for testing scenarios involving larger sample sizes. a pool size of 10 achieved similar herd-sensitivity to that of the current pool size for the majority of the sample test and check test scenarios. this was particularly evident when pool-specificity was assumed to be perfect. the overall herd-sensitivity of the check test was very low for all infection prevalence levels and pool sizes, but it more than doubled, when the sample size increased from 50 to 100 animals (11% versus 26% for a herd size of 500 cattle with a 2% infection prevalence). the results show that the majority of beef producers participating in the assurance scheme can benefit from using a larger pool size for the pooled faecal quantitative pcr testing of their herd, in comparison to the pool size currently used. foot-and-mouth disease (fmd) is a highly contagious disease of livestock and has severely affected livestock industries during the past two decades in previously fmd-free countries. the disease was eliminated in north america in 1953 but remains a threat for re-introduction. approximately 44% of the on-feed beef cattle in the u.s. are concentrated in feedlots <32,000 heads, but little information is available on dynamics of fmd in large feedlots. therefore, there is a need to explore possible management and intervention strategies that might be implemented during potential fmd outbreaks on feedlots. we used a within home-pen stochastic susceptible-latent-infectious-recovered (slir) fmd dynamics model nested in a meta-population model of home-pens in a feedlot. the combinatory model was previously developed to simulate foot-and-mouth disease virus (fmdv) transmission within u.s. beef feedlots. we evaluated three intervention strategies initiated on the day of fmd detection: stopping movements of cattle between home-pens and hospital-pen(s) (nh), barrier depopulation combined with nh (nh-bd), and targeted depopulation of at-risk home-pens combined with nh (nh-td). depopulation rates investigated ranged from 500 to 4,000 cattle per day. we evaluated the projected effectiveness of interventions by comparing them with the no-intervention fmd dynamics in the feedlot. we modeled a small-size (4,000 cattle), medium-size (12,000 cattle), and large-size (24,000 cattle) feedlots. implementation of nh delayed the outbreak progression, but it did not prevent infection of the entire feedlot. implementation of nh-bd resulted in depopulation of 50% of cattle in small- and medium-size feedlots, and 25% in large-size feedlots, but the intervention prevented infection of the entire feedlot in 40% of simulated outbreaks in medium-size feedlots, and in 8% in large-size feedlots. implementation of nh-td resulted in depopulation of up to 50% of cattle in small-size feedlots, 75% in medium-size feedlots, and 25% in large-size feedlots, but rarely prevented infection of the entire feedlot. number of hospital-pens in the feedlot was shown to weakly impact the success of nh-td. overall, the results suggest that stopping cattle movements between the home-pens and hospital-pens, without or with barrier or targeted cattle depopulation, would not be highly effective to interrupt fmdv transmission within a feedlot. machine learning (ml) methods have shown promising results in identifying genes when applied to large transcriptome datasets. however, no attempt has been made to compare the performance of combining different ml methods together in the prediction of high feed efficiency (hfe) and low feed efficiency (lfe) animals. in this study, using rna sequencing data of five tissues (adrenal gland, hypothalamus, liver, skeletal muscle, and pituitary) from nine hfe and nine lfe nellore bulls, we evaluated the prediction accuracies of five analytical methods in classifying fe animals. these included two conventional methods for differential gene expression (dge) analysis (t-test and edger) as benchmarks, and three ml methods: random forests (rfs), extreme gradient boosting (xgboost), and combination of both rf and xgboost (rx). utility of a subset of candidate genes selected from each method for classification of fe animals was assessed by support vector machine (svm). among all methods, the smallest subsets of genes (117) identified by rx outperformed those chosen by t-test, edger, rf, or xgboost in classification accuracy of animals. gene co-expression network analysis confirmed the interactivity existing among these genes and their relevance within the network related to their prediction ranking based on ml. the results demonstrate a great potential for applying a combination of ml methods to large transcriptome datasets to identify biologically important genes for accurately classifying fe animals. because a growing proportion of the beef output in many countries originates from dairy herds, the most critical decisions about the genetic merit of most carcasses harvested are being made by dairy producers. interest in the generation of more valuable calves from dairy females is intensifying, and the most likely vehicle is the use of appropriately selected beef bulls for mating to the dairy females. this is especially true given the growing potential to undertake more beef × dairy matings as herd metrics improve (e.g., reproductive performance) and technological advances are more widely adopted (e.g., sexed semen). clear breed differences (among beef breeds but also compared with dairy breeds) exist for a whole plethora of performance traits, but considerable within-breed variability has also been demonstrated. although such variability has implications for the choice of bull to mate to dairy females, the fact that dairy females themselves exhibit such genetic variability implies that \"one size fits all\" may not be appropriate for bull selection. although differences in a whole series of key performance indicators have been documented between beef and beef-on-dairy animals, of particular note is the reported lower environmental hoofprint associated with beef-on-dairy production systems if the environmental overhead of the mature cow is attributed to the milk she eventually produces. despite the known contribution of beef (i.e., both surplus calves and cull cows) to the overall gross output of most dairy herds globally, and the fact that each dairy female contributes half her genetic merit to her progeny, proxies for meat yield (i.e., veal or beef) are not directly considered in the vast majority of dairy cow breeding objectives. breeding objectives to identify beef bulls suitable for dairy production systems are now being developed and validated, demonstrating the financial benefit of using such breeding objectives over and above a focus on dairy bulls or easy-calving, short-gestation beef bulls. when this approach is complemented by management-based decision-support tools, considerable potential exists to improve the profitability and sustainability of modern dairy production systems by exploiting beef-on-dairy breeding strategies using the most appropriate beef bulls. during growth in cattle, the development of intramuscular adipose tissue and muscle is dependent upon cell hyperplasia (increased number of adipocytes) and hypertrophy (increased size of adipocytes). based on the results of previous studies, other adipose tissue depots (e.g., perirenal and subcutaneous) develop from the fetal stage primarily as brown adipose tissue. the hyperplastic stage of intramuscular adipose is considered to develop from late pregnancy, but there is no evidence indicating that intramuscular adipose tissue develops initially as brown adipose tissue. hyperplastic growth of intramuscular adipose continues well into postweaning and is dependent on the timing of the transition to grain-based diets; thereafter, the late-stage development of intramuscular adipose tissue is dominated by hypertrophy. for muscle development, hyperplasia of myoblasts lasts from early (following development of somites in the embryo) to middle pregnancy, after which growth of muscle is the result of hypertrophy of myofibers. vitamin a is a fat-soluble compound that is required for the normal immunologic function, vision, cellular proliferation, and differentiation. here we review the roles of vitamin a in intramuscular adipose tissue and muscle development in cattle. vitamin a regulates both hyperplasia and hypertrophy in in vitro experiments. vitamin a supplementation at the early stage and restriction at fattening stage generate opposite effects in the beef cattle. appropriate vitamin a supplementation and restriction strategy increase intramuscular adipose tissue development (i.e., marbling or intramuscular fat) in some in vivo trials. besides, hyperplasia and hypertrophy of myoblasts/myotubes were affected by vitamin a treatment in in vitro trials. additionally, some studies reported an interaction between the alcohol dehydrogenase-1c (adh1c) genotype and vitamin a feed restriction for the development of marbling and/or intramuscular adipose tissue, which was dependent on the timing and level of vitamin a restriction. therefore, the feed strategy of vitamin a has the visible impact on the marbling and muscle development in the cattle, which will be helpful to promote the quality of the beef. one approach to reducing calving difficulty is to select heifers with higher breeding value for calving ease. calving ease is often associated with lower birth weight and that may result in other possible effects on lifetime productivity. females from experimental select and control calving ease lines within each of the seven populations were compared. random samples of 720 heifers from lines selected for better calving ease breeding values and 190 heifers from control lines selected for average birth weights were followed through four parities. select and control lines within the same population were selected to achieve similar yearling weight breeding values. weights of sampled heifers in select lines were 2.6 kg (p < 0.01) lighter at birth but not different from control lines at weaning. select lines had significantly shorter hip height, lighter mature weight, and greater calving success at second parity. their calves were born significantly earlier with lighter weights and less assistance. significant interactions with parity showed fewer calves assisted and greater calf survival to weaning as heifers but negligible differences with control lines in later parities. steer progeny sampled from these dams in select lines (n = 204) were not different from steers in control lines (n = 91) for hot carcass weight but had significantly greater fat depth. two production systems were compared considering the seven populations as replicates. the systems differed in selection history of females (select and control lines) and the use of bulls within their lines as young cows, but used the same bulls in both lines as older cows. cows were culled after single unsuccessful breeding and kept for up to four parities. select line cows tended (p ≤ 0.10) to wean more calves and stay in the herd longer. they were assisted significantly fewer times at calving and had greater calf weight gain to weaning when evaluated over their herd life. mature weights were lighter in select lines, but marketable cow weight from the systems was nearly identical. control lines did have more marketable young cow weight and select lines older cow weight. weaned calf weight per heifer starting the system was significantly greater for the select heifer system due to greater survival of calves from heifers and greater calving success at second parity. no important unfavorable effects of genetic differences in calving ease were identified in this experiment. the global demand for beef is rapidly increasing (fao, 2019), raising concern about climate change impacts (clark et al., 2020; leip et al., 2015; springmann et al., 2018). beef and dairy contribute over 70% of livestock greenhouse gas emissions (ghg), which collectively contribute ~6.3 gt co2 -eq/year (gerber et al., 2013; herrero et al., 2016) and account for 14%-18% of human ghg emissions (friedlingstein et al., 2019; gerber et al., 2013). the utility of beef ghg mitigation strategies, such as land-based carbon (c) sequestration and increased production efficiency, are actively debated (garnett et al., 2017). we compiled 292 local comparisons of \"improved\" versus \"conventional\" beef production systems across global regions, assessing net ghg emission data from life cycle assessment (lca) studies. our results indicate that net beef ghg emissions could be reduced substantially via changes in management. overall, a 46 % reduction in net ghg emissions per unit of beef was achieved at sites using carbon (c) sequestration management strategies on grazed lands, and an 8% reduction in net ghgs was achieved at sites using growth efficiency strategies. however, net-zero emissions were only achieved in 2% of studies. among regions, studies from brazil had the greatest improvement, with management strategies for c sequestration and efficiency reducing beef ghg emissions by 57%. in the united states, c sequestration strategies reduced beef ghg emissions by over 100% (net-zero emissions) in a few grazing systems, whereas efficiency strategies were not successful at reducing ghgs, possibly because of high baseline efficiency in the region. this meta-analysis offers insight into pathways to substantially reduce beef production's global ghg emissions. nonetheless, even if these improved land-based and efficiency management strategies could be fully applied globally, the trajectory of growth in beef demand will likely more than offset ghg emissions reductions and lead to further warming unless there is also reduced beef consumption. the objective of this experiment was to evaluate the effect of leguminous fodder silage on rumen fermentation, nutrient digestibility, and utilization in beef cattle crossbreds. four cattle, with an average live weight of 280 ± 10 kg, were used in a 4 × 4 latin square design with supplementation of various levels of sunnhemp silage (shs). sunnhemp silage was fed to cattle at 0, 1, 1.5, and 2 kg dm shs/head/day. the dm, om, and cp digestibilities were increased (p < 0.05), and the highest value was found by feeding 2 kg dm shs/head/day. total volatile fatty acids and individual volatile fatty acid (vfa) especially c3 were increased (p < 0.01), while c2 and c2:c3 ratios were decreased (p < 0.01) when shs was supplemented. nitrogen utilization efficiency and urinary purine derivatives were increased (p < 0.01) by the shs supplementation. in conclusion, these data suggest that feeding shs at 1.5 to 2 kg dm/ head/day can significantly increase rumen fermentation end-products, nitrogen utilization efficiency, and microbial protein synthesis. sunnhemp silage can be practically processed and provided as a good roughage source for ruminants. therefore, sunnhemp silage is recommended as a feeding intervention in the sub-tropical and tropical regions to support the sustainable livestock production. we aimed to determine the mold, yeast, and bacterial distributions in dry-aged beef (dab) manufactured in hokkaido, japan, and to study their effects on meat quality compared to wet-aged beef (wab). two rump blocks from holstein steer were dry- and wet-aged for 35 days at 2.9 °c and 90% rh. the psychrophilic molds mucor flavus and helicostylum pulchrum and other fungi (penicillium sp. and debaryomyces sp.) appeared on the crust of dab, while lactic acid bacteria and coliforms were suppressed in the inner part of the meat. the composition of c16:0, c18:0, and c18:1 fatty acids did not differ between dab and wab, while more c17:0 fatty acids were detected in dab. dry aging suppressed acids and increased the production of various aroma compounds with mushroom-like, nutty, and other pleasant flavors. the meat quality and free amino acid (faa) contents of dab and wab did not differ significantly. in this study, we identified major molds on dab, which might contribute to an increase in aroma. keywords: dry-aged beef; mucor flavus; helicostylum pulchrum; psychrophilic mold; meat quality; volatile aroma compounds. increasing pesticide contamination in foods of animal origin has made the wide-scope multi-residue analysis of pesticides an international concern. in this study, a sensitive and reliable multi-residue pesticide analysis method for beef was developed with the use of 129 pesticides to demonstrate the full scope of the method. the pesticides were extracted and purified using a modified quechers technique and analysed using ultra-high-performance liquid chromatography coupled to quadrupole time-of-flight mass spectrometry. an orthogonal array design was employed for optimisation of the method, significantly scientising and simplifying the process. method validation revealed satisfactory sensitivity with limits of quantification values ranging from 0.003 to 11.37 µg∙kg-1. matrix effects ranged from 83.85% to 120.66%. recovery at three spiking levels (20, 50, and 100 µg∙kg-1) were 70.51-128.12% and the intra-day and inter-day precisions were less than 20%, indicating good accuracy and precision. pre-slaughter fasting duration affect blood parameters related to stress and metabolism, decrease live and carcass weights and impact beef quality of cattle. in three experiments, 1100 steers and heifers, finished on feedlot or on pasture, were evaluated to assess the influence of the site and duration of fasting before slaughter on physiological, carcass and meat quality traits. cattle were allocated to one of two fasting duration - long (23-29 h) or short (2-6 h) - and to one of two sites of fasting- farm or abattoir. all animals had access to water ad libitum until slaughter, except during transportation. cattle were assigned to two (long fasting on abattoir/normal lairage time, short fasting on farm/minimal lairage time) or three (long fasting on abattoir/normal lairage time, short fasting on farm/minimal lairage time, long fasting on farm/minimal lairage time) treatments. seventeen slaughters were considered in a randomized complete block design. there was no effect of the site of fasting on any variable. cattle with short fasting duration had higher carcass weights and water consumption than those under long fasting duration. haematocrit, globulin, total protein, and lactate dehydrogenase at the time of slaughter increased with fasting duration. there was no effect of the duration of fasting on blood ions, meat quality traits, urine ph, liver weight and volume, and skin dry matter. a reduction in the fasting duration returned to farmers and abattoirs 1.2% additional kilograms of carcass, suggesting an improvement in animal welfare according to a better hydration level of cattle at the time of slaughter. proteomics profiling disclosed the molecular mechanism underlying beef poor meat quality. this study aimed to identify protein markers indicating the quality of beef during postmortem storage at 4°c. beef longissimus dorsi samples were stored at 4°c. the meat water holding capacity (whc), ph value and moisture content were determined at different time points during the storage period. the itraq ms/ms approach was used to determine the proteomics profiling at 0, 3.5 and 7 d during storage at 4°c. bioinformatics analysis was performed to investigate the potential correlated proteins associated with meat quality. storage at 4°c gradually decreased the ph value, whc, and hence the moisture content. the itraq proteomic analysis revealed that a cluster of glycolytic enzymes including malate dehydrogenase, cytoplasmic, l-lactate dehydrogenase, phosphoglycerate mutase and pyruvate kinase, and another cluster of proteins involved in oxygen transport and binding (myoglobin) and hemoglobin complex (including globin a1 and hemoglobin subunit alpha) were decreased during the postmortem storage. these results suggest that the decreased glycolysis, oxygen, and heme-binding activities might be associated with the beef muscle low quality and the decline of tenderness during postmortem storage at 4°c. the objective of this experiment was to study the effect of feeding pellet containing high sulphur (pelfur) diet and fresh cassava root (fcr) to thai native beef cattle on feed use efficiency, ruminal characteristics and blood metabolites. four male purebred thai native beef cattle (1.5-2.0 years old) with initial body weight (bw) of 150 ± 15.0 kg were allocated with a 2 × 2 factorial arrangement in a 4 × 4 latin square design. factor a was fcr supplementation at 15 and 20 g/kg of bw. factor b was the sulphur level in the pelfur ration at 15 and 30 g/kg of dry matter (dm). no interaction effect was found among fcr supplementation and pelfur in terms of feed intake and nutrient intake (p > 0.05). cyanide intake was significantly increased based on fcr supplementation (p < 0.05), whereas sulphur intake was increased by level addition of pelfur levels (p < 0.05). there were interaction effects among fcr supplementation and pelfur on digestibility coefficients of dm and organic matter (om) (p < 0.05). fcr supplementation at 20 g/kg bw with pelfur 30 g/kg demonstrated the highest digestibility of dm and om. moreover, interactions were observed between fcr and pelfur for bacterial populations (p < 0.01). the populations of bacteria were highest in fcr supplementation at 20 g/kg bw with pelfur 30 g/kg at various feeding times. an interaction effect from among feeding fcr with pelfur was found on blood thiocyanate concentrations at various feeding times (p < 0.01). the highest mean values of blood thiocyanate were observed when feeding fcr at 20 g/kg bw with pelfur at 30 g/kg. no interaction effect was found between fcr and pelfur on total volatile fatty acids (vfa) and their profiles (p > 0.05). however, the proportions of the total vfa at 0 and 4 h post-feeding were increased when fcr at 20 g/kg bw was supplemented (p < 0.01). fcr at 20 g/kg bw could enhance propionate (c3) at 4 h post-feeding when compared with fcr at 15 g/kg bw (p < 0.01). moreover, supplementation of pelfur at 30 g/kg increased the total vfa at 0 and 4 h post-feeding, whereas the concentration of c3 at 4 h post-feeding was enhanced (p < 0.05). however, no significant changes were found for any parameters among treatments and between the main effect of fcr and pelfur supplementation (p > 0.05). in conclusion, feeding of two combinations (fcr 20 g/kg bw with pelfur 30 g/kg) could promote the nutrient digestibility, the bacterial populations and the rate of disappearance of cyanide without having any adverse effect on rumen fermentation. \n",
      "Euclidean Distance : 16.189125061035156\n",
      "\n",
      "\n",
      "Document: cockroach (blatella germanica) bites in amazonian indigenous peoples.  \n",
      "Euclidean Distance : 16.262853622436523\n",
      "\n",
      "\n",
      "Document: effects of floral scent, color and pollen on foraging decisions and oocyte development of common green bottle flies. the common green bottle fly lucilia sericata (meigen) and other filth flies frequently visit pollen-rich composite flowers such as the oxeye daisy, leucanthemum vulgare lam. in laboratory experiments with l. sericata, we investigated the effect of generic floral scent and color cues, and of oxeye daisy-specific cues, on foraging decisions by recently eclosed flies. we also tested the effect of a floral pollen diet with 0-35% moisture content on the ability of females to mature their oocytes. our data indicate that (1) young flies in the presence of generic floral scent respond more strongly to a uniformly yellow cue than to any other uniform color cue (green, white, black, blue, red) except for ultraviolet (uv); (2) the floral scent of oxeye daisies enhances the attractiveness of a yellow cue; and (3) moisture-rich pollen provides nutrients that facilitate ovary maturation of flies. with evidence that l. sericata exploits floral cues during foraging, and that pollen can be an alternate protein source to animal feces and carrion, pollen apparently plays a major role in the foraging ecology of l. sericata and possibly other filth flies. these flies, in turn, may play a significant role as pollinators, as supported by a recently published study. \n",
      "Euclidean Distance : 16.32317543029785\n",
      "\n",
      "\n",
      "Document: comparative evaluation of acrylamide and polycyclic aromatic hydrocarbons contents in robusta coffee beans roasted by hot air and superheated steam. lactam  swelling properties of roasted coffee particles. distinguishing the region-of-origin of roasted coffee beans with trace element ratios. phytochemical profile and antioxidant capacity of coffee plant organs compared to green and roasted coffee beans. effect of uv-c irradiation on inactivation of  development and validation of an analytical method for the analysis of sterigmatocystin in roasted coffee beans and black pepper using liquid chromatography-tandem mass spectrometry. a predictive spatial model for roasted coffee using oxygen isotopes of α-cellulose. analysis of coffee adulterated with roasted corn and roasted soybean using voltammetric electronic tongue. spatial distributions of furan and 5-hydroxymethylfurfural in unroasted and roasted coffea arabica beans. coffea arabica and c. canephora discrimination in roasted and ground coffee from reference material candidates by real-time pcr. furan in roasted, ground and brewed coffee variability of single bean coffee volatile compounds of arabica and robusta roasted coffees analysed by spme-gc-ms. an  quantitative assessment of specific defects in roasted ground coffee via infrared-photoacoustic spectroscopy. roasted and green coffee extracts show antioxidant and cytotoxic activity in myoblast and endothelial cell lines in a cell specific manner. steam pressure treatment of defective coffea canephora beans improves the volatile profile and sensory acceptance of roasted coffee blends. mid infrared spectroscopy and chemometrics as tools for the classification of roasted coffees by cup quality. bioavailability of hydroxycinnamates in an instant green/roasted coffee blend in humans. identification of novel colonic metabolites. time-resolved gravimetric method to assess degassing of roasted coffee. although hot air (ha) is a conventional roasting medium for coffee beans, ha roasting is known to result in possible formation of toxic compounds, including acrolein, acrylamide and polycyclic aromatic hydrocarbons (pahs). superheated steam (shs) roasting is therefore proposed as an alternative means to alleviate the formation of these toxic compounds in roasted coffee beans. robusta coffee beans were roasted either with ha or shs in a fluidized bed roaster at 210-250 °c until the bean color reached the targeted roast levels. the contents of acrolein, acrylamide and 16 pahs in the roasted beans were determined; only acrylamide and 5 pahs were nevertheless found. shs roasting interestingly resulted in lower acrylamide contents in dark-roasted beans; similar trend was noted in the beans medium-roasted at 250 °c. the contents of three-ring pahs, namely fluorene, phenanthrene and anthracene, in dark-roasted beans were significantly lower upon shs roasting at 250 °c. seven new lactam ent-kaurane diterpenoids, cafemides a-g (1-7), were isolated from roasted beans of coffea arabica. their structures were elucidated by extensive spectroscopic analysis including 1d, 2d nmr (heteronuclear single quantum correlation (hsqc), heteronuclear multiple bond correlation (hmbc), 1h-1h correlation spectroscopy (cosy), and rotating frame overhauser effect spectroscopy (roesy)), high-resolution electrospray ionization mass spectrometry (hresims), and ir spectra. they were divided into subtype i-iii according to the structure. further, with the aid of liquid chromatography-tandem mass spectrometry (lc-ms/ms) based molecular network, seven (8-14) subtype ii diterpenoids were successfully identified. in addition, a variety of other subtypes of n-containing diterpenoids have been proven in roasted coffee. compounds 1, 2, 3, 5, and 7 showed a moderate inhibitory effect on α-glucosidase with an ic50 value of 8.28 ± 0.62 μm, 38.23 ± 8.87 μm, 28.94 ± 1.42 μm, 12.44 ± 1.37 μm, and 22.2 ± 5.34 μm, respectively. to the best of our knowledge, this is the first time that n-containing diterpenoids have been reported in coffee. in this study, the swelling behavior of roasted coffee particles in water and particularly its impact on particle diameter is examined by applying laser-diffraction analysis and microscopy. several potential influencing factors are investigated: initial particle size, roasting degree, and temperature. additionally, the time dependency of swelling and particle shape is evaluated at two different temperatures. we verify that particle erosion occurs - as observed by an increase of the fine particle fraction after wetting - and it is revealed that this effect is more pronounced with a rise in temperature. the total relative increase in particle size is determined as approximately 15% based on a broad range of different sized coffee grounds. it is demonstrated that the degree of swelling is independent of both the initial particle diameter and the roasting degree. the particle shape is found to be unaffected by swelling. this research reveals that swelling is initially quick, with 60-80% of the final steady-state diameter being reached after 30 s and completed after 4 min of wetting, i.e. within the timescale of conventional coffee brewing methods. this work provides a better understanding of the impact of wetting as part of the coffee brewing process, thus aiding the design, modeling, and optimization of coffee extraction. it clarifies the strong deviation of previous results on coffee-particle swelling by considering particle erosion and degassing and provides a robust method for quantification. © 2020 the authors. journal of the science of food and agriculture published by john wiley & sons ltd on behalf of society of chemical industry. determining coffee region-of-origin is most appropriately addressed through analyses of the product available to the consumer. we analyzed the concentrations of 44 trace elements in 53 samples of roasted arabica coffee beans (coffea arabica) from 21 different countries. variations in absolute elemental concentrations of coffee beans arise through varying degrees of roasting (from green through dark roasts). since trace elements are not volatilized at roasting temperatures, we conducted analyses of element ratios to evaluate concentration-related differences among beans of different origins. we used kernel density estimates to compare the distributions of 1892 element ratios for each of these countries with the combined distribution of coffee samples from the other countries. using this quantitative approach, we demonstrated that many of the world's coffee-producing regions can be distinguished from other regions of the world on the basis of element ratios. the current study investigates the phytochemical composition of coffee plant organs and their corresponding antioxidant capacities compared to green and roasted coffee beans. hplc analysis indicated that the investigated compounds were present in all organs except mangiferin, which was absent in roots, stems and seeds, and caffeine, which was absent in stems and roots. total phytochemicals were highest in the green beans (gb) at 9.70 mg g-1 dry weight (dw), while roasting caused a 66% decline in the roasted beans (rb). this decline resulted more from 5-cqa and sucrose decomposition by 68% and 97%, respectively, while caffeine and trigonelline were not significantly thermally affected. roasting increased the total phenolic content (tpc) by 20.8% which was associated with an increase of 68.8%, 47.5% and 13.4% in the antioxidant capacity (teac) determined by 2,2-diphenyl-1-picryl hydrazyl radical (dpph), 2,2-azino bis (3-ethyl benzothiazoline-6-sulphonic acid) radical (abts) and ferric ion reducing antioxidant power (frap) assays, respectively. amongst the leaves, the youngest (l1) contained the highest content at 8.23 mg g-1 dw, which gradually reduced with leaf age to 5.57 mg g-1 dw in the oldest (l6). leaves also contained the highest tpc (over 60 mg g-1 gae) and exhibited high teac, the latter being highest in l1 at 328.0, 345.7 and 1097.4, and least in l6 at 304.6, 294.5 and 755.1 µmol trolox g-1 sample for the respective assays. phytochemical accumulation, tpc and teac were least in woody stem (ws) at 1.42 mg g-1 dw; 8.7 mg g-1 gae; 21.9, 24.9 and 110.0 µmol trolox g-1 sample; while herbaceous stem (hs) contained up to 4.37 mg g-1 dw; 27.8 mg g-1 gae; 110.9, 124.8 and 469.7 µmol trolox g-1 sample, respectively. roots contained up to 1.85 mg g-1 dw, 15.8 mg-1 gae and teac of 36.8, 41.5 and 156.7 µmol trolox g-1 sample. amongst the organs, therefore, coffee leaves possessed higher values than roasted beans on the basis of phytochemicals, tpc and teac. leaves also contain carotenoids and chlorophylls pigments with potent health benefits. with appropriate processing methods, a beverage prepared from leaves (coffee leaf tea) could be a rich source of phytochemicals and antioxidants with therapeutic and pharmacological values for human health. this study investigated the antifungal effect of ultraviolet-c (uv-c) against aspergillus flavus and aspergillus parasiticus on roasted coffee beans. also, any changes in the quality of the roasted coffee beans were measured after uv-c irradiation. as uv-c irradiation time increased (0-2 h), the number of surviving a. flavus and a. parasiticus spores significantly (p < .05) decreased. the reduction values of a. flavus in round part, crack part, and whole roasted coffee beans were 2.16, 0.71, and 1.58 log10 cfu g-1, respectively, and the reduction values of a. parasiticus in round part, crack part, and whole roasted coffee beans were 1.03, 0.37, and 0.72 log10 cfu g-1, respectively, after 2 h of uv-c irradiation. field emission scanning electron microscopy showed that the morphology of a. flavus and a. parasiticus spores included expanded wrinkles that were deformed by uv-c irradiation. the hunter colours were significantly reduced (p < .05). there was no significant change (p > .05) in moisture content, but the ph was significantly decreased (p < .05). most of the sensory parameters did not change, but there was a significant difference (p < .05) in flavour. based on this study, 2 h of uv-c irradiation was effective in reducing 90% of a. flavus, but it was not effective against a. parasiticus present on roasted coffee beans. also, hunter colour, ph, and sensory parameters (flavour) were changed by uv-c irradiation. sterigmatocystin (stc) is a toxic and potentially carcinogenic fungal toxin found in a variety of food commodities. this study describes the development of an analytical method to determine stc in roasted coffee beans and black pepper using ultra performance liquid chromatography (uplc) coupled with triple quadrupole tandem mass spectrometry (ms/ms). 13c18-stc was used as internal standard. stc was extracted with a mixture of acetonitrile/water, diluted with a buffer, followed by purification with a solid-phase extraction and an immunoaffinity column prior to the uplc-ms/ms analysis. two multiple reaction monitoring (mrm) transitions were employed, one for quantification and one for confirmation of stc. the uplc-ms/ms analytical method was validated with respect to selectivity, linearity, sensitivity, accuracy, precision, recovery, and stability. calibration curves were linear over a concentration range 25-2,500 pg ml-1 with correlation coefficients (r) > 0.998. the method limit of quantification for stc in roasted coffee beans and black pepper was 0.10 μg kg-1. the accuracy and precision of the analytical method were acceptable within 15% at all quality control levels. this method was suitable to determine stc levels because of its selectivity, precision, and accuracy. the method was successfully applied to roasted coffee beans and black pepper samples. fraudulent region-of-origin labeling is a concern for high-value, globally traded commodities such as coffee. the oxygen isotope ratio of cellulose is a useful geographic tracer, as it integrates climate and source water signals. a predictive spatial model (\"isoscape\") of the δ18 o values of coffee bean cellulose is generated to evaluate coffee region-of-origin claims. the oxygen isotope ratio of α-cellulose extracted from roasted coffee beans was measured via high-temperature conversion elemental analyzer/isotope ratio mass spectrometry (tc-ea/irms) and used to calculate the δ18 o value of coffee bean water. the 18 o enrichment of coffee bean water relative to the δ18 o value of local precipitation was modeled as a function of local temperature and humidity. this function was incorporated into a mechanistic model of cellulose δ18 o values to predict the δ18 o values of coffee bean cellulose across coffee-producing regions globally. the δ18 o values of analyzed coffee bean cellulose ranged from approximately +22‰ to +42‰ (v-smow). as expected, coffees grown in the same region tended to have similar isotope ratios, and the δ18 o value of coffee bean cellulose was generally higher than the δ18 o value of modeled stem cellulose for the region. modeled δ18 o values of coffee cellulose were within ±2.3‰ of the measured δ18 o value of coffee cellulose. the oxygen isotope ratio of coffee bean cellulose is a useful indicator of region-of-origin and varies predictably in response to climatic factors and precipitation isotope ratios. the isoscape of coffee bean cellulose δ18 o values from this study provides a quantitative tool that can be applied to region-of-origin verification of roasted coffee at the point-of-sale. coffee samples adulterated with roasted corn and roasted soybean were analyzed using a voltammetric electronic tongue equipped with a polypyrrole sensor array. coffee samples were adulterated in concentrations of 2%, 5%, 10% and 20% of roasted corn and roasted soybean; 5 replicates of each were used. the discrimination capacity of a voltammetric electronic tongue elaborated with a polypyrrole sensor array, was evaluated by principal component analysis and cluster analysis, while the capacity to perform quantitative determinations was carried out by partial least squares. the results obtained by the application of principal component analysis showed an excellent ability to discriminate adulterated samples. additionally, the classifications obtained by cluster analysis was concordant with those obtained by principal component analysis. on the other hand, the evaluation of the ability to quantitatively analyze the adulterated samples showed that the polypyrrole sensor array provides sufficient information to allow quantitative determinations by partial least squares regression. it could be concluded that the voltammetric electronic tongue used in this work allows the suf- ficient analysis of coffee samples adulterated with roasted corn and roasted soybean. for the first time, the spatial distributions of the highly volatile compounds furan and 5-hydroxymethylfurfural (hmf) have been determined in cross sections of green and roasted coffea arabica beans. the image maps were revealed by laser ablation dart imaging mass spectrometry (ladi-ms). the presence of these compounds was independently confirmed by gc-ms as well as argon dart-ms. quantification of furan by gc-ms was completed with the final concentrations in roasted and unroasted beans determined to be 96.5 and 4.1 ng/g, respectively. furan was observed to be distributed throughout the tissue of both green and roasted beans, while hmf was localized to the silver skin in green beans. following roasting, the appearance of hmf was more diffuse. the implications of the broad distribution of furan on the one hand, and localization of hmf on the other, are discussed. to produce specific desirable coffee blends, coffea arabica and c. canephora are mixed each other, in some cases to suit consumer preference, but in others to reduce production costs. in this scenario, the aim of this work was to evaluate standard candidate reference materials (rmc) for analysis of different blends of roasted and ground coffee. for this purpose, we analyzed different percentages of c. arabica and c. canephora (100:0; 50:50; 25:75; and 0:100, respectively). these rmc samples were developed in a previous study with green coffee beans submitted to medium roasting. in this work, coffee species differentiation (c. arabica and c. canephora) was analyzed by real-time pcr, using specific primers previously developed, called ara primers. the rmc material with 100% c. canephora did not present amplification, in contrast with the samples containing c. arabica, which all presented amplification. these results indicate the specificity of ara primers for c. arabica and that the detection system assay can be used as a promising molecular tool to identify and quantify percentages of c. arabica in different coffee blends. coffee is the most popular hot beverage in the world. the annual coffee production in 2010, 2014 and 2016 was 8.1, 9.0 and 9.3 million tons respectively. there are more than 100 coffee species, but only two of them: arabica (coffea arabica) and robusta (coffea canephora) have gained commercial importance. during roasting of green coffee beans not only desirable compounds are formed, that exert positive influence on the taste and flavour of coffee, but also small quantities of undesirable ones. furan (c4h4o) is one of the latter.\n",
      "furan is a volatile compound (boiling temp. of 31.4 oc) formed during thermal processing of food. the toxicity of furan has been well documented and it is classified as “possible human carcinogen” (group 2b) by the international agency for research on cancer. various pathways have been reported for furan formation during food processing. it can be formed from carbohydrates, amino acids by their thermal degradation or thermal re-arrangement and by oxidation of ascorbic acid and polyunsaturated acids and carotenoids. high concentrations of furan have been reported in coffee, baked and roasted food and in food subjected to preserving in cans and jars. furan levels in brewed coffee are typically near or below 120 μg/l, but it can approach thousands μg/kg in roasted whole beans or ground coffee. the highest concentration of furan in roasted coffee reaches the level of 7000 μg/kg. taking into account that coffee is the most popular hot drink, it becomes the main contributor to furan exposure from dietary sources for adults.\n",
      "in this article the published scientific papers concerned with the presence of furan in roasted non-brewed and brewed coffee have been reviewed. the formation mechanisms and occurrence of furan in coffee and the harmful influence of furan on the consumer health have been discussed. we report on the analysis of volatile compounds by spme-gc-ms for individual roasted coffee beans. the aim was to understand the relative abundance and variability of volatile compounds between individual roasted coffee beans at constant roasting conditions. twenty-five batches of arabica and robusta species were sampled from 13 countries, and 10 single coffee beans randomly selected from each batch were individually roasted in a fluidised-bed roaster at 210 °c for 3 min. high variability (cv = 14.0-53.3%) of 50 volatile compounds in roasted coffee was obtained within batches (10 beans per batch). phenols and heterocyclic nitrogen compounds generally had higher intra-batch variation, while ketones were the most uniform compounds (cv < 20%). the variation between batches was much higher, with the cv ranging from 15.6 to 179.3%. the highest variation was observed for 2,3-butanediol, 3-ethylpyridine and hexanal. it was also possible to build classification models based on geographical origin, obtaining 99.5% and 90.8% accuracy using lda or mlr classifiers respectively, and classification between arabica and robusta beans. these results give further insight into natural variation of coffee aroma and could be used to obtain higher quality and more consistent final products. our results suggest that coffee volatile concentration is also influenced by other factors than simply the roasting degree, especially green coffee composition, which is in turn influenced by the coffee species, geographical origin, ripening stage and pre- and post-harvest processing. the present non-targeted 1h nmr-based fingerprinting approach along with multivariate analysis established differences between representative aqueous extracts of commercial ground roasted coffee (grc) and instant (soluble) coffee (ic) samples. the latter were prepared either by spray drying or freeze drying. when comparing a total of 33 compounds between grc and ic, the latter product contained a remarkable increase in 5-(hydroxymethyl)furfural and carbohydrates, as well as a clear decrease in trigonelline, n-methylpyridinium, caffeine, caffeoylquinic acids and 2-furylmethanol. furthermore, the current protocol was able to detect the subtle chemical differences between spray-dried and freeze-dried ic. the aforementioned metabolites could serve as target molecules in the attempt to preserve, as much as possible, the organoleptic and nutraceutical properties of grc during the industrial drying processes used in the production of the two commercial types of ic. chemical analyses and sensory evaluation are the most applied methods for quality control of roasted and ground coffee (rg). however, faster alternatives would be highly valuable. here, we applied infrared-photoacoustic spectroscopy (ftir-pas) on rg powder. mixtures of specific defective beans were blended with healthy (defect-free) coffea arabica and coffea canephora bases in specific ratios, forming different classes of blends. principal component analysis allowed predicting the amount/fraction and nature of the defects in blends while partial least squares discriminant analysis revealed similarities between blends (=samples). a successful predictive model was obtained using six classes of blends. the model could classify 100% of the samples into four classes. the specificities were higher than 0.9. application of ftir-pas on rg coffee to characterize and classify blends has shown to be an accurate, easy, quick and \"green\" alternative to current methods. coffee is one of the most highly consumed beverages with potential beneficial health implications, however its molecular mechanism of action has not been completely elucidated yet. to that cause, the polyphenolic composition of different coffee extracts (from light, medium and dark roasts as well as green beans) was examined by uhplc-hrms analysis, indicating chlorogenic acids isomers as the main constituents. in the following step, the toxicity of the extracts was tested in myoblasts and endothelial cells and differential toxicity of green and roasted samples was displayed as the myoblasts were more sensitive to green coffee extracts, in contrast to the endothelial cells. subsequently, biologically relevant, non-cytotoxic extract concentrations were administered to explore their potential effect on cell redox status using flow cytometry and spectrophotometric assays. the results indicated that all coffee extracts improved cell redox status, however differences were observed between the two different cell lines tested, implying that coffee compounds display cell- and tissue-specificity. glutathione levels were increased in almost all cases up to 70%, while the roasting degree affected the free radical scavenging potential of the extracts and their ability to protect from macromolecular oxidation as exhibited by the differences in ros, carb and tbars levels, especially in the myoblasts. between 15 and 20% of brazilian coffee production corresponds to defective beans (pva), which decreases the quality of the coffee brew. steam treatment has been reported as an alternative to improve the volatile profile and cup quality of coffee. the aim of this study was to propose a steam treatment of defective coffea canephora beans to improve the volatile profile of the roasted coffee. the sensory impacts of adding steamed coffee (sc) in coffea arabica blends were evaluated. the steam treatments studied modified the volatile profile of roasted scs, increasing the contents of acetoin, benzyl alcohol, maltol, 2,6-dimethylpyrazine, 2-furfurylthiol, and 5-methylfurfural and decreasing the contents of 4-ethylguaiacol, isovaleric acid, methional, 2,3-diethyl-5-methylpyrazine, and 3-methoxy-3-methylpyrazine. among the evaluated parameters, the best condition to maximized the content of the volatiles with a potential positive impact and minimize those with a potential negative impact was 5bar/16min (sc 5). the thresholds of consumer rejection and of detection indicate that up to 30% sc 5 can be added to a high cup quality coffea arabica coffee without perception or rejection of the coffee brew. a blend of 30% of sc 5 and 70% of coffea arabica was well accepted. sensory (cup) analysis is a reliable methodology for green coffee quality evaluation, but faces barriers when applied to commercial roasted coffees due to lack of information on roasting conditions. the aim of this study was to examine the potential of mid-infrared spectroscopy for predicting cup quality of arabica coffees of different roasting degrees. pca analysis showed separation of arabica and robusta. a two-level pls-da hierarchical strategy was employed, with coffee being classified as high or low quality in the first level and then separated according to cup quality in the second level. validation results showed that the second level models exhibited 100% sensitivity and specificity in the training sets. for the test set, sensitivity ranged from 67% (rio zona) to 100% (soft) while specificity ranged from 71% (rio) to 100% (rioysh, hard). thus, the proposed method can be used for the quality evaluation of arabica coffees regardless of roasting conditions. roasting greatly reduces the phenolic content in green coffee beans. considering the beneficial effects of coffee polyphenols, blends containing green coffee beans are being consumed as a healthier alternative to roasted coffee. this study was aimed at assessing the absorption and metabolism of hydroxycinnamates in an instant green/roasted (35/65) coffee blend in healthy humans. twelve fasting men and women consumed a cup of coffee containing 269.5 mg (760.6 μmol) of chlorogenic acids. blood and urine samples were taken before and after coffee consumption at different times and analyzed by lc-ms-qtof. up to 25 and 42 metabolites were identified in plasma and urine, respectively, mainly in the form of sulfate and methyl derivatives, and to a lower extent as glucuronides. un-metabolized hydroxycinnamate esters (caffeoyl-, feruloyl-, and coumaroylquinic acids), hydroxycinnamic acids (caffeic, ferulic and coumaric acids) and their phase ii metabolites, in addition to phase ii derivatives of lactones, represented a minor group of metabolites (16.3% of the metabolites excreted in urine) with kinetics compatible with small intestine absorption. dihydrohydroxycinnamic acids and their phase ii derivatives, in addition to feruloylglycine, showed delayed kinetics due to their colonic origin and represented the most abundant group of metabolites (75.7% of total urinary metabolites). dihydrohydroxycinnamate esters (dihydroferuloyl-, dihydrocaffeoyl- and dihydrocoumaroylquinic acids) have been identified for the first time in both plasma and urine, with microbial origin (excreted 8-12 h after coffee intake) amounting to 8% of total urinary metabolites. in conclusion, coffee polyphenols are partially bioavailable and extensively metabolized, mainly by the colonic microbiota. during the roasting of coffee, thermally driven chemical reactions lead to the formation of gases, of which a large fraction is carbon dioxide (co2). part of these gases is released during roasting while part is retained inside the porous structure of the roasted beans and is steadily released during storage or more abruptly during grinding and extraction. the release of co2 during the various phases from roasting to consumption is linked to many important properties and characteristics of coffee. it is an indicator for freshness, plays an important role in shelf life and in packaging, impacts the extraction process, is involved in crema formation, and may affect the sensory profile in the cup. indeed, and in view of the multiple roles it plays, co2 is a much underappreciated and little examined molecule in coffee. here, we introduce an accurate, quantitative, and time-resolved method to measure the release kinetics of gases from whole beans and ground coffee using a gravimetric approach. samples were placed in a container with a fitted capillary to allow gases to escape. the time-resolved release of gases was measured via the weight loss of the container filled with coffee. long-term stability was achieved using a customized design of a semimicro balance, including periodic and automatic zero value measurements and calibration procedures. the novel gravimetric methodology was applied to a range of coffee samples: (i) whole arabica beans and (ii) ground arabica and robusta, roasted to different roast degrees and at different speeds (roast air temperatures). modeling the degassing rates allowed structural and mechanistic interpretation of the degassing process. \n",
      "Euclidean Distance : 16.364620208740234\n",
      "\n",
      "\n",
      "Document: whole-genome resequencing of  cucurbita pepo contains two cultivated subspecies, each of which encompasses four fruit-shape morphotypes (cultivar groups). the pumpkin, vegetable marrow, cocozelle, and zucchini groups are of subsp. pepo and the acorn, crookneck, scallop, and straightneck groups are of subsp. ovifera. recently, a de novo assembly of the c. pepo subsp. pepo zucchini genome was published, providing insights into its evolution. to expand our knowledge of evolutionary processes within c. pepo and to identify variants associated with particular morphotypes, we performed whole-genome resequencing of seven of these eight c. pepo morphotypes. we report for the first time whole-genome resequencing of the four subsp. pepo (pumpkin, vegetable marrow, cocozelle, green zucchini, and yellow zucchini) morphotypes and three of the subsp. ovifera (acorn, crookneck, and scallop) morphotypes. a high-depth resequencing approach was followed, using the bgiseq-500 platform that enables the identification of rare variants, with an average of 33.5x. approximately 94.5% of the clean reads were mapped against the reference zucchini genome. in total, 3,823,977 high confidence single-nucleotide polymorphisms (snps) were identified. within each accession, snps varied from 636,918 in green zucchini to 2,656,513 in crookneck, and were distributed homogeneously along the chromosomes. clear differences between subspecies pepo and ovifera in genetic variation and linkage disequilibrium are highlighted. in fact, comparison between subspecies pepo and ovifera indicated 5710 genes (22.5%) with fst > 0.80 and 1059 genes (4.1%) with fst = 1.00 as potential candidate genes that were fixed during the independent evolution and domestication of the two subspecies. linkage disequilibrium was greater in subsp. ovifera than in subsp. pepo, perhaps reflective of the earlier differentiation of morphotypes within subsp. ovifera. some morphotype-specific genes have been localized. our results offer new clues that may provide an improved understanding of the underlying genomic regions involved in the independent evolution and domestication of the two subspecies. comparisons among snps unique to particular subspecies or morphotypes may provide candidate genes responsible for traits of high economic importance. \n",
      "Euclidean Distance : 16.44478416442871\n",
      "\n",
      "\n",
      "Document: contact balanitis to tromantadine.  \n",
      "Euclidean Distance : 16.505916595458984\n",
      "\n",
      "\n",
      "Document: atoltivimab, maftivimab, and odesivimab-ebgn.  \n",
      "Euclidean Distance : 17.6210994720459\n",
      "\n",
      "\n",
      "Document: atoltivimab, maftivimab, and odesivimab-ebgn.  \n",
      "Euclidean Distance : 17.6210994720459\n",
      "\n",
      "\n",
      "Document: atoltivimab, maftivimab, and odesivimab-ebgn.  \n",
      "Euclidean Distance : 17.6210994720459\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.670719146728516\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n",
      "\n",
      "\n",
      "Document:   \n",
      "Euclidean Distance : 22.67072105407715\n"
     ]
    }
   ],
   "source": [
    "from sklearn.metrics.pairwise import cosine_similarity\n",
    "from sklearn.metrics.pairwise import euclidean_distances\n",
    "\n",
    "pairwise_similarities=cosine_similarity(document_embeddings)\n",
    "pairwise_differences=euclidean_distances(document_embeddings)\n",
    "\n",
    "most_similar(0,pairwise_similarities,'Cosine Similarity')\n",
    "most_similar(0,pairwise_differences,'Euclidean Distance')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "6"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(documents)\n",
    "# documents_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[1.0000001  0.8365411  0.61181045 0.4764783  0.6589275  0.81594574]\n",
      " [0.8365411  1.         0.67819023 0.5372207  0.7253677  0.78296375]\n",
      " [0.61181045 0.67819023 0.99999994 0.6800262  0.6249758  0.71018046]\n",
      " [0.4764783  0.5372207  0.6800262  1.0000001  0.5375689  0.6229849 ]\n",
      " [0.6589275  0.7253677  0.6249758  0.5375689  1.0000002  0.78826356]\n",
      " [0.81594574 0.78296375 0.71018046 0.6229849  0.78826356 0.99999994]]\n",
      "\n",
      "[[1.0000001  0.8365411  0.61181045 0.4764783  0.6589275  0.81594574]\n",
      " [0.8365411  1.         0.67819023 0.5372207  0.7253677  0.78296375]\n",
      " [0.61181045 0.67819023 0.99999994 0.6800262  0.6249758  0.71018046]\n",
      " [0.4764783  0.5372207  0.6800262  1.0000001  0.5375689  0.6229849 ]\n",
      " [0.6589275  0.7253677  0.6249758  0.5375689  1.0000002  0.78826356]\n",
      " [0.81594574 0.78296375 0.71018046 0.6229849  0.78826356 0.99999994]]\n"
     ]
    }
   ],
   "source": [
    "# pairwise_similarities\n",
    "# document_embeddings\n",
    "pairwise_similarities_test = cosine_similarity(document_embeddings)\n",
    "print(pairwise_similarities_test)\n",
    "print()\n",
    "print(pairwise_similarities)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gensim  version = 3.8.3\n",
      "pytorch version = 1.7.1\n",
      "sklearn version = 0.23.1\n",
      "nltk  version = 3.5\n"
     ]
    }
   ],
   "source": [
    "import gensim\n",
    "import sklearn\n",
    "import nltk # 영어 자연어 처리\n",
    "from nltk import RegexpTokenizer\n",
    "from nltk.corpus import stopwords\n",
    "import numpy as np\n",
    "import torch\n",
    "\n",
    "print('gensim  version = {}'.format(gensim.__version__))\n",
    "print('pytorch version = {}'.format(torch.__version__))\n",
    "print('sklearn version = {}'.format(sklearn.__version__))\n",
    "print('nltk  version = {}'.format(nltk.__version__))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load pickle data\n",
    "import pickle\n",
    "import pandas as pd\n",
    "\n",
    "\n",
    "# Drug bank complete slim data\n",
    "drugbank_approved_df = pd.read_csv('D:/WorkSpace/master_thesis/Drug_Bank/result/drugbank_approved.tsv', sep='\\t')\n",
    "drugName = drugbank_approved_df['name']\n",
    "\n",
    "# Drugbank vocabulary data\n",
    "drugbank_voca_df = pd.read_csv('D:/WorkSpace/master_thesis/Drug_Bank/drugbank_vocabulary.csv')\n",
    "\n",
    "# documents data (title, abstracts)\n",
    "with open('D:/WorkSpace/master_thesis/data_pubmed/documents/approved_documents.pkl', 'rb') as f:\n",
    "    documents = pickle.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [],
   "source": [
    "# # drugbank_approved_df.name # column 명을 속성으로 접근\n",
    "# # drugbank_approved_df.loc[0] # index명으로 접근\n",
    "\n",
    "# # drug name으로 index 지정: set_index /reset_index\n",
    "# test_df = drugbank_approved_df.set_index('name')\n",
    "# # drugbank_approved_df.loc[0]\n",
    "# print(test_df.loc['Lepirudin'])\n",
    "# print()\n",
    "# print(drugbank_approved_df.loc[0])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "파이썬 데이터프레임 접근 방법\n",
    "https://m.blog.naver.com/PostView.nhn?blogId=rian4u&logNo=221399141190&proxyReferer=https:%2F%2Fwww.google.com%2F"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>drugbank_id</th>\n",
       "      <th>name</th>\n",
       "      <th>type</th>\n",
       "      <th>groups</th>\n",
       "      <th>atc_codes</th>\n",
       "      <th>targets</th>\n",
       "      <th>categories</th>\n",
       "      <th>inchikey</th>\n",
       "      <th>inchi</th>\n",
       "      <th>description</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>DB00001</td>\n",
       "      <td>Lepirudin</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved</td>\n",
       "      <td>B01AE02</td>\n",
       "      <td>Prothrombin</td>\n",
       "      <td>Amino Acids, Peptides, and Proteins|Anticoagul...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Lepirudin is identical to natural hirudin exce...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DB00002</td>\n",
       "      <td>Cetuximab</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved</td>\n",
       "      <td>L01XC06</td>\n",
       "      <td>Epidermal growth factor receptor|Low affinity ...</td>\n",
       "      <td>Amino Acids, Peptides, and Proteins|Antibodies...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Cetuximab is an epidermal growth factor recept...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DB00003</td>\n",
       "      <td>Dornase alfa</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved</td>\n",
       "      <td>R05CB13</td>\n",
       "      <td>DNA</td>\n",
       "      <td>Amino Acids, Peptides, and Proteins|Cough and ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Dornase alfa is a biosynthetic form of human d...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DB00004</td>\n",
       "      <td>Denileukin diftitox</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved|investigational</td>\n",
       "      <td>L01XX29</td>\n",
       "      <td>Interleukin-2 receptor subunit alpha|Interleuk...</td>\n",
       "      <td>ADP Ribose Transferases|Amino Acids, Peptides,...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A recombinant DNA-derived cytotoxic protein co...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>DB00005</td>\n",
       "      <td>Etanercept</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved|investigational</td>\n",
       "      <td>L04AB01</td>\n",
       "      <td>Tumor necrosis factor|High affinity immunoglob...</td>\n",
       "      <td>Agents reducing cytokine levels|Amino Acids, P...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Dimeric fusion protein consisting of the extra...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  drugbank_id                 name     type                    groups  \\\n",
       "0     DB00001            Lepirudin  biotech                  approved   \n",
       "1     DB00002            Cetuximab  biotech                  approved   \n",
       "2     DB00003         Dornase alfa  biotech                  approved   \n",
       "3     DB00004  Denileukin diftitox  biotech  approved|investigational   \n",
       "4     DB00005           Etanercept  biotech  approved|investigational   \n",
       "\n",
       "  atc_codes                                            targets  \\\n",
       "0   B01AE02                                        Prothrombin   \n",
       "1   L01XC06  Epidermal growth factor receptor|Low affinity ...   \n",
       "2   R05CB13                                                DNA   \n",
       "3   L01XX29  Interleukin-2 receptor subunit alpha|Interleuk...   \n",
       "4   L04AB01  Tumor necrosis factor|High affinity immunoglob...   \n",
       "\n",
       "                                          categories inchikey inchi  \\\n",
       "0  Amino Acids, Peptides, and Proteins|Anticoagul...      NaN   NaN   \n",
       "1  Amino Acids, Peptides, and Proteins|Antibodies...      NaN   NaN   \n",
       "2  Amino Acids, Peptides, and Proteins|Cough and ...      NaN   NaN   \n",
       "3  ADP Ribose Transferases|Amino Acids, Peptides,...      NaN   NaN   \n",
       "4  Agents reducing cytokine levels|Amino Acids, P...      NaN   NaN   \n",
       "\n",
       "                                         description  \n",
       "0  Lepirudin is identical to natural hirudin exce...  \n",
       "1  Cetuximab is an epidermal growth factor recept...  \n",
       "2  Dornase alfa is a biosynthetic form of human d...  \n",
       "3  A recombinant DNA-derived cytotoxic protein co...  \n",
       "4  Dimeric fusion protein consisting of the extra...  "
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "drugbank_approved_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4270\n",
      "4027\n",
      "dict_keys(['Lepirudin', 'Cetuximab', 'Dornase alfa', 'Denileukin diftitox', 'Etanercept', 'Bivalirudin', 'Leuprolide', 'Peginterferon alfa-2a', 'Alteplase', 'Sermorelin', 'Interferon alfa-n1', 'Darbepoetin alfa', 'Urokinase', 'Goserelin', 'Reteplase', 'Erythropoietin', 'Salmon calcitonin', 'Interferon alfa-n3', 'Pegfilgrastim', 'Sargramostim', 'Peginterferon alfa-2b', 'Asparaginase Escherichia coli', 'Thyrotropin alfa', 'Antihemophilic factor, human recombinant', 'Anakinra', 'Gramicidin D', 'Human immunoglobulin G', 'Anistreplase', 'Insulin human', 'Tenecteplase', 'Menotropins', 'Interferon gamma-1b', 'Interferon alfa-2a, Recombinant', 'Desmopressin', 'Coagulation factor VIIa Recombinant Human', 'Oprelvekin', 'Palifermin', 'Glucagon', 'Aldesleukin', 'Botulinum Toxin Type B', 'Omalizumab', 'Lutropin alfa', 'Lyme disease vaccine (recombinant OspA)', 'Insulin lispro', 'Insulin glargine', 'Collagenase clostridium histolyticum', 'Rasburicase', 'Cetrorelix', 'Adalimumab', 'Somatotropin', 'Imiglucerase', 'Abciximab', 'Drotrecogin alfa', 'Gemtuzumab ozogamicin', 'Alpha-1-proteinase inhibitor', 'Pegaspargase', 'Interferon beta-1a', 'Pegademase', 'Albumin human', 'Eptifibatide', 'Infliximab', 'Follitropin', 'Vasopressin', 'Interferon beta-1b', 'Interferon alfacon-1', 'Hyaluronidase (ovine)', 'Insulin pork', 'Trastuzumab', 'Rituximab', 'Basiliximab', 'Muromonab', 'Digoxin Immune Fab (Ovine)', 'Daptomycin', 'Tositumomab', 'Pegvisomant', 'Botulinum toxin type A', 'Pancrelipase', 'Streptokinase', 'Alemtuzumab', 'Alglucerase', 'Capromab pendetide', 'Laronidase', 'Cyclosporine', 'Urofollitropin', 'Efalizumab', 'Choriogonadotropin alfa', 'Antithymocyte immunoglobulin (rabbit)', 'Filgrastim', 'Coagulation Factor IX (Recombinant)', 'Becaplermin', 'Agalsidase beta', 'Octreotide', 'Interferon alfa-2b', 'Abarelix', 'Oxytocin', 'Natalizumab', 'Enfuvirtide', 'Palivizumab', 'Bevacizumab', 'Cyanocobalamin', 'Ademetionine', 'Pyruvic acid', 'Biotin', 'L-Lysine', 'Aspartic acid', 'Ornithine', 'L-Glutamine', 'Adenosine phosphate', 'alpha-Linolenic acid', 'Methionine', 'Tyrosine', 'Calcitriol', 'Lutein', 'Cystine', 'Riboflavin', 'Glutamic acid', 'Glycine', 'Calcifediol', 'Creatine', 'Tryptophan', 'Cysteine', 'Thiamine', 'Ergocalciferol', 'Threonine', 'NADH', 'Folic acid', 'Icosapent', 'Valine', 'Vitamin A', 'Vitamin E', 'Pyridoxine', 'Lipoic acid', 'Cholecalciferol', 'Menadione', 'Adenine', 'Asparagine', 'Pravastatin', 'Fluvoxamine', 'Valsartan', 'Ramipril', 'Masoprocol', 'Flunisolide', 'Baclofen', 'Amphetamine', 'Pentagastrin', 'Nicotine', 'Cevimeline', 'Lorazepam', 'Esmolol', 'Bortezomib', 'Ethchlorvynol', 'Carbidopa', 'Phentermine', 'Indecainide', 'Tramadol', 'Vidarabine', 'Betaxolol', 'Troglitazone', 'Oseltamivir', 'Erythromycin', 'Hydroxocobalamin', 'Caffeine', 'Succinylcholine', 'Sildenafil', 'Dofetilide', 'Azithromycin', 'Ticlopidine', 'Trospium', 'Adapalene', 'Midodrine', 'Pantoprazole', 'Torasemide', 'Citalopram', 'Eletriptan', 'Bethanidine', 'Moxifloxacin', 'Oxyphenonium', 'Nelfinavir', 'Isoetharine', 'Glimepiride', 'Diflorasone', 'Indinavir', 'Gadodiamide', 'Guanadrel', 'Lovastatin', 'Enflurane', 'Cefotiam', 'Pregabalin', 'Temazepam', 'Aminosalicylic acid', 'Reboxetine', 'Milrinone', 'Pipobroman', 'Butabarbital', 'Nevirapine', 'Oxiconazole', 'Alclometasone', 'Butalbital', 'Cladribine', 'Ranolazine', 'Mesalazine', 'Benzatropine', 'Ziprasidone', 'Methysergide', 'Cabergoline', 'Idoxuridine', 'Dapsone', 'Terconazole', 'Phenytoin', 'Medrysone', 'Doxycycline', 'Diethylstilbestrol', 'Lymecycline', 'Clotrimazole', 'Calcium acetate', 'Sulfanilamide', 'Cycloserine', 'Anagrelide', 'Sulfisoxazole', 'Metoprolol', 'Crotamiton', 'Dicoumarol', 'Cefmenoxime', 'Ropinirole', 'Isradipine', 'Diatrizoate', 'Betazole', 'Topiramate', 'Olmesartan', 'Amsacrine', 'Theophylline', 'Argatroban', 'Liothyronine', 'Disopyramide', 'Lidocaine', 'Pamidronic acid', 'Acarbose', 'Venlafaxine', 'Conjugated estrogens', 'Travoprost', 'Amcinonide', 'Atomoxetine', 'Bleomycin', 'Chlorambucil', 'Etomidate', 'Raltitrexed', 'Etonogestrel', 'Morphine', 'Ropivacaine', 'Bupivacaine', 'Dapiprazole', 'Tenofovir disoproxil', 'Flucloxacillin', 'Tranexamic acid', 'Ertapenem', 'Desogestrel', 'Mitomycin', 'Talbutal', 'Bexarotene', 'Ibutilide', 'Vindesine', 'Chlorthalidone', 'Valproic acid', 'Zolmitriptan', 'Acetaminophen', 'Gefitinib', 'Codeine', 'Piperacillin', 'Amitriptyline', 'Floxuridine', 'Tolcapone', 'Fluorometholone', 'Nitroprusside', 'Calcium glucoheptonate', 'Hydromorphone', 'Indomethacin', 'Ethambutol', 'Metformin', 'Ipratropium', 'Olanzapine', 'Atenolol', 'Nitrofural', 'Pimecrolimus', 'Omeprazole', 'Pyrazinamide', 'Metixene', 'Cetirizine', 'Terfenadine', 'Diltiazem', 'Protriptyline', 'Aminohippuric acid', 'Alfuzosin', 'Trimethadione', 'Nitisinone', 'Clobazam', 'Minoxidil', 'Megestrol acetate', 'Tioguanine', 'Methylergometrine', 'Buclizine', 'Chlorzoxazone', 'Aminoglutethimide', 'Mefloquine', 'Sulfadiazine', 'Sapropterin', 'Vinorelbine', 'Anidulafungin', 'Clozapine', 'Sucralfate', 'Grepafloxacin', 'Doxylamine', 'Levonorgestrel', 'Norepinephrine', 'Cidofovir', 'Mirtazapine', 'Meprobamate', 'Thiethylperazine', 'Timolol', 'Treprostinil', 'Colestipol', 'Trihexyphenidyl', 'Palonosetron', 'Dydrogesterone', 'Mexiletine', 'Dexrazoxane', 'Amlodipine', 'Tacrine', 'Oxyphencyclimine', 'Triamterene', 'Valrubicin', 'Procyclidine', 'Phenylephrine', 'Carbimazole', 'Digoxin', 'Sulpiride', 'Profenamine', 'Nimodipine', 'Beclomethasone dipropionate', 'Carisoprodol', 'Progesterone', 'Phenylpropanolamine', 'Sorafenib', 'Zoledronic acid', 'Griseofulvin', 'Nisoldipine', 'Eszopiclone', 'Ceruletide', 'Alprazolam', 'Dexbrompheniramine', 'Gentian violet cation', 'Ardeparin', 'Loxapine', 'Remoxipride', 'Mupirocin', 'Carbamoylcholine', 'Rosiglitazone', 'Pramipexole', 'Acetohexamide', 'Metocurine iodide', 'Phenoxymethylpenicillin', 'Secobarbital', 'Miglustat', 'Promazine', 'Spironolactone', 'Methylphenidate', 'Methocarbamol', 'Hyoscyamine', 'Zolpidem', 'Famciclovir', 'Triprolidine', 'Carboprost tromethamine', 'Cefpiramide', 'Lindane', 'Trifluridine', 'Prochlorperazine', 'Cyproheptadine', 'Nitric Oxide', 'Bendroflumethiazide', 'Allopurinol', 'Ceftazidime', 'Cerivastatin', 'Gemcitabine', 'Entecavir', 'Betamethasone', 'Teniposide', 'Chloramphenicol', 'Loracarbef', 'Dipivefrin', 'Droperidol', 'Levothyroxine', 'Framycetin', 'Meperidine', 'Loratadine', 'Cefalotin', 'Imipramine', 'Acitretin', 'Verteporfin', 'Nabumetone', 'Methscopolamine bromide', 'Sodium tetradecyl sulfate', 'Ketorolac', 'Enoxacin', 'Quinine', 'Tenoxicam', 'Dronabinol', 'Montelukast', 'Fluoxetine', 'Hexylcaine', 'Methohexital', 'Chlordiazepoxide', 'Duloxetine', 'Chlorpromazine', 'Rimantadine', 'Amikacin', 'Lenalidomide', 'Raloxifene', 'Gallamine triethiodide', 'Brimonidine', 'Dicloxacillin', 'Nabilone', 'Pefloxacin', 'Altretamine', 'Sotalol', 'Buspirone', 'Miglitol', 'Fosinopril', 'Cefotaxime', 'Entacapone', 'Zidovudine', 'Darifenacin', 'Oxycodone', 'Phenindione', 'Flutamide', 'Tolmetin', 'Haloperidol', 'Ritonavir', 'Levallorphan', 'Nitazoxanide', 'Triflupromazine', 'Dextrothyroxine', 'Acetyldigitoxin', 'Vancomycin', 'Aminocaproic acid', 'Cisplatin', 'Bentoquatam', 'Anisotropine methylbromide', 'Albendazole', 'Trandolapril', 'Caspofungin', 'Carteolol', 'Alitretinoin', 'Metolazone', 'Tolnaftate', 'Oxaliplatin', 'Cinchocaine', 'Foscarnet', 'Erlotinib', 'Mephenytoin', 'Rofecoxib', 'Chlormerodrin', 'Cefdinir', 'Guanidine', 'Ciprofloxacin', 'Gadoversetamide', 'Toremifene', 'Nortriptyline', 'Vincristine', 'Benazepril', 'Amoxapine', 'Fluorouracil', 'Pyridostigmine', 'Desoximetasone', 'Azelaic acid', 'Zafirlukast', 'Propylthiouracil', 'Acetohydroxamic acid', 'Pentostatin', 'Methoxsalen', 'Piroxicam', 'Lamotrigine', 'Perflutren', 'Hydroxyzine', 'Zanamivir', 'Doxapram', 'Benzthiazide', 'Methotrexate', 'Carbamazepine', 'Cisatracurium', 'Succimer', 'Cephalexin', 'Cinnarizine', 'Fondaparinux', 'Vinblastine', 'Propranolol', 'Atropine', 'Fenoprofen', 'Fenfluramine', 'Clonidine', 'Sulfamethizole', 'Valaciclovir', 'Carbenicillin', 'Mazindol', 'Valdecoxib', 'Lactulose', 'Levocarnitine', 'Enalapril', 'Nizatidine', 'Fluticasone propionate', 'Lisuride', 'Doxazosin', 'Fluocinolone acetonide', 'Piperazine', 'Ethosuximide', 'Amiloride', 'Oxytetracycline', 'Ulobetasol', 'Gadoteridol', 'Labetalol', 'Thiopental', 'Monobenzone', 'Linezolid', 'Ivermectin', 'Medroxyprogesterone acetate', 'Cisapride', 'Sulindac', 'Cyclothiazide', 'Nafcillin', 'Chloroquine', 'Ethionamide', 'Metaraminol', 'Butorphanol', 'Bisoprolol', 'Amodiaquine', 'Furazolidone', 'Rifabutin', 'Paramethadione', 'Demeclocycline', 'Imatinib', 'Triamcinolone', 'Oxandrolone', 'Nicardipine', 'Fluphenazine', 'Testosterone', 'Efavirenz', 'Bacitracin', 'Niacin', 'Clorazepic acid', 'Guanabenz', 'Alendronic acid', 'Clofarabine', 'Docosanol', 'Dexmedetomidine', 'Sulfacetamide', 'Prednisone', 'Clofibrate', 'Astemizole', 'Inulin', 'Butoconazole', 'Adenosine', 'Simvastatin', 'Pemetrexed', 'Mebendazole', 'Gonadorelin', 'Dyclonine', 'Nystatin', 'Dextropropoxyphene', 'Mitotane', 'Stavudine', 'Leucovorin', 'Dyphylline', 'Pentazocine', 'Magnesium sulfate', 'Latanoprost', 'Estrone', 'Trazodone', 'Mecamylamine', 'Sevelamer', 'Acamprosate', 'Metaxalone', 'Trimethobenzamide', 'Flumethasone', 'Sulfametopyrazine', 'Nafarelin', 'Epinephrine', 'Sumatriptan', 'Pirenzepine', 'Cefixime', 'Chlorpropamide', 'Aprepitant', 'Galantamine', 'Tamoxifen', 'Benzyl benzoate', 'Isoflurophate', 'Losartan', 'Thioridazine', 'Moricizine', 'Warfarin', 'Midazolam', 'Tobramycin', 'Trovafloxacin', 'Pentosan polysulfate', 'Fludrocortisone', 'Mycophenolate mofetil', 'Cephaloglycin', 'Flurazepam', 'Moexipril', 'Phentolamine', 'Daunorubicin', 'Furosemide', 'Tizanidine', 'Nitrofurantoin', 'Nicergoline', 'Eplerenone', 'Amprenavir', 'Icodextrin', 'Methazolamide', 'Naltrexone', 'Delavirdine', 'Tamsulosin', 'Porfimer sodium', 'Lamivudine', 'Ibandronate', 'Diethylcarbamazine', 'Flurbiprofen', 'Oxacillin', 'Apomorphine', 'Paroxetine', 'Nedocromil', 'Norethisterone', 'Adefovir dipivoxil', 'Azatadine', 'Clodronic acid', 'Procaine', 'Lisinopril', 'Methoxamine', 'Imiquimod', 'Homatropine methylbromide', 'Trimipramine', 'Nitroglycerin', 'Rocuronium', 'Thiabendazole', 'Nateglinide', 'Atracurium besylate', 'Pralidoxime', 'Risperidone', 'Esomeprazole', 'Meclizine', 'Pentamidine', 'Hetacillin', 'Riluzole', 'Hydrocortisone', 'Mannitol', 'Gadobenic acid', 'Zileuton', 'Modafinil', 'Deferoxamine', 'Carbinoxamine', 'Etodolac', 'Prilocaine', 'Epinastine', 'Tranylcypromine', 'Isoflurane', 'Ethotoin', 'Tretinoin', 'Hexachlorophene', 'Dolasetron', 'Clopidogrel', 'Tetracycline', 'Meropenem', 'Potassium chloride', 'Methimazole', 'Metyrosine', 'Clavulanic acid', 'Olopatadine', 'Hydrocortamate', 'Alprostadil', 'Clidinium', 'Malathion', 'Etoposide', 'Hydroflumethiazide', 'Tirofiban', 'Oxcarbazepine', 'Propiomazine', 'Roxithromycin', 'Nalidixic acid', 'Phenelzine', 'Propantheline', 'Estradiol', 'Mefenamic acid', 'Cryptenamine', 'Acyclovir', 'Naproxen', 'Gadopentetic acid', 'Perindopril', 'Uracil mustard', 'Tripelennamine', 'Haloprogin', 'Primidone', 'Sulfasalazine', 'Candesartan cilexetil', 'Tolazoline', 'Gentamicin', 'Tazarotene', 'Fenoldopam', 'Halazepam', 'Alfentanil', 'Colistin', 'Dicyclomine', 'Minaprine', 'Pentoxifylline', 'Proparacaine', 'Indapamide', 'Tropicamide', 'Biperiden', 'Ribavirin', 'Phenylbutazone', 'Fentanyl', 'Meloxicam', 'Orciprenaline', 'Rosoxacin', 'Propofol', 'Tadalafil', 'Carprofen', 'Disulfiram', 'Ethynodiol diacetate', 'Enprofylline', 'Levomenthol', 'Natamycin', 'Cinoxacin', 'Diazepam', 'Phenmetrazine', 'Trifluoperazine', 'Phensuximide', 'Cefaclor', 'Mifepristone', 'Brompheniramine', 'Clocortolone', 'Tolazamide', 'Hydroxypropyl cellulose', 'Dobutamine', 'Oxazepam', 'Donepezil', 'Nalbuphine', 'Flurandrenolide', 'Cysteamine', 'Levamisole', 'Methylphenobarbital', 'Perphenazine', 'Dacarbazine', 'Pseudoephedrine', 'Levorphanol', 'Aminolevulinic acid', 'Chlorphenesin', 'Terbinafine', 'Penicillamine', 'Prednisolone', 'Diflunisal', 'Vardenafil', 'Ranitidine', 'Tacrolimus', 'Benzphetamine', 'Ritodrine', 'Benzonatate', 'Dorzolamide', 'Suprofen', 'Terbutaline', 'Conivaptan', 'Loteprednol etabonate', 'Guaifenesin', 'Flupentixol', 'Eprosartan', 'Sirolimus', 'Chlorhexidine', 'Emtricitabine', 'Chlorothiazide', 'Quinapril', 'Clomifene', 'Isosorbide dinitrate', 'Risedronic acid', 'Pemirolast', 'Bumetanide', 'Mechlorethamine', 'Granisetron', 'Sulfapyridine', 'Oxybuprocaine', 'Iron Dextran', 'Testolactone', 'Benzylpenicilloyl polylysine', 'Rimexolone', 'Triazolam', 'Ethanol', 'Remifentanil', 'Methdilazine', 'Etacrynic acid', 'Ondansetron', 'Bimatoprost', 'Tiagabine', 'Cocaine', 'Quinidine', 'Zonisamide', 'Paricalcitol', 'Tinidazole', 'Anileridine', 'Phenformin', 'Amantadine', 'Dinoprostone', 'Almotriptan', 'Spectinomycin', 'Ketotifen', 'Buprenorphine', 'Levosimendan', 'Ceforanide', 'Cyclobenzaprine', 'Phenoxybenzamine', 'Famotidine', 'Azacitidine', 'Misoprostol', 'Colesevelam', 'Metacycline', 'Tipranavir', 'Mesoridazine', 'Maprotiline', 'Oxymetazoline', 'Salicylic acid', 'Diethylpropion', 'Salmeterol', 'Meclofenamic acid', 'Methantheline', 'Hexafluronium', 'Cycrimine', 'Zalcitabine', 'Demecarium', 'Acetylsalicylic acid', 'Phenprocoumon', 'Fulvestrant', 'Mezlocillin', 'Felbamate', 'Fexofenadine', 'Isoniazid', 'Naratriptan', 'Rizatriptan', 'Netilmicin', 'Hydrocodone', 'Norgestimate', 'Methylprednisolone', 'Pindolol', 'Mepivacaine', 'Zaleplon', 'Bromfenac', 'Apraclonidine', 'Ethiodized oil', 'Telmisartan', 'Desloratadine', 'Methyldopa', 'Alosetron', 'Dactinomycin', 'Selenium Sulfide', 'Azelastine', 'Ezetimibe', 'Edetic acid', 'Dipyridamole', 'Ethinylestradiol', 'Lomefloxacin', 'Cyclopentolate', 'Ramelteon', 'Physostigmine', 'Isotretinoin', 'Formoterol', 'Nandrolone phenpropionate', 'Dimenhydrinate', 'Glycopyrronium', 'Cytarabine', 'Rivastigmine', 'Oxaprozin', 'Methyl aminolevulinate', 'Azathioprine', 'Auranofin', 'Gabapentin', 'Doxorubicin', 'Frovatriptan', 'Hydrochlorothiazide', 'Cyclacillin', 'Salbutamol', 'Levobupivacaine', 'Cromoglicic acid', 'Ganciclovir', 'Hydroxyurea', 'Tioconazole', 'Ketoprofen', 'Edrophonium', 'Metyrapone', 'Cinacalcet', 'Clobetasol propionate', 'Balsalazide', 'Glyburide', 'Minocycline', 'Guanfacine', 'Bethanechol', 'Isosorbide mononitrate', 'Trichlormethiazide', 'Phylloquinone', 'Mycophenolic acid', 'Amlexanox', 'Ketoconazole', 'Methoxyflurane', 'Irbesartan', 'Topotecan', 'Ethinamate', 'Tolterodine', 'Selegiline', 'Fenofibrate', 'Thalidomide', 'Melphalan', 'Gatifloxacin', 'Rifampicin', 'Lubiprostone', 'Fluocinonide', 'Ergoloid mesylate', 'Ibuprofen', 'Novobiocin', 'Nitrendipine', 'Tocainide', 'Echothiophate', 'Norfloxacin', 'Amoxicillin', 'Azlocillin', 'Oxybutynin', 'Acetophenazine', 'Isoprenaline', 'Melatonin', 'Cefditoren', 'Clonazepam', 'Promethazine', 'Dihydrotachysterol', 'Atazanavir', 'Fludarabine', 'Perhexiline', 'Diphenhydramine', 'Atorvastatin', 'Etidronic acid', 'Deslanoside', 'Tegaserod', 'Vigabatrin', 'Diphenoxylate', 'Streptomycin', 'Orlistat', 'Emedastine', 'Pilocarpine', 'Benzocaine', 'Primaquine', 'Iloprost', 'Deserpidine', 'Pentolinium', 'Butenafine', 'Ouabain', 'Dimethyl sulfoxide', 'Fluvastatin', 'Oxamniquine', 'Leflunomide', 'Rosuvastatin', 'Pimozide', 'Capecitabine', 'Arbutamine', 'Sertraline', 'Sibutramine', 'Levocabastine', 'Methyprylon', 'Trilostane', 'Heparin', 'Miconazole', 'Colistimethate', 'Cefuroxime', 'Papaverine', 'Chlorpheniramine', 'Nifedipine', 'Trimethaphan', 'Amiodarone', 'Diazoxide', 'Gliclazide', 'Phenacemide', 'Ambenonium', 'Proflavine', 'Tolbutamide', 'Anisindione', 'Econazole', 'Bicalutamide', 'Rabeprazole', 'Prednicarbate', 'Proguanil', 'Pioglitazone', 'Tiludronic acid', 'Desoxycorticosterone pivalate', 'Doxacurium', 'Carvedilol', 'Levofloxacin', 'Sulfinpyrazone', 'Cefapirin', 'Cefadroxil', 'Doxepin', 'Amifostine', 'Diclofenamide', 'Sulfoxone', 'Diphenylpyraline', 'Flavoxate', 'Nefazodone', 'Cefprozil', 'Desipramine', 'Candicidin', 'Sertaconazole', 'Thiamylal', 'Gemifloxacin', 'Bupropion', 'Trimetrexate', 'Bretylium', 'Halothane', 'Dinoprost tromethamine', 'Chloroprocaine', 'Terazosin', 'Calcium chloride', 'Ofloxacin', 'Cilostazol', 'Procarbazine', 'Arsenic trioxide', 'Guanethidine', 'Moclobemide', 'Kanamycin', 'Orphenadrine', 'Phenobarbital', 'Escitalopram', 'Cyclizine', 'Idarubicin', 'Chlormezanone', 'Podofilox', 'Rescinnamine', 'Ifosfamide', 'Propafenone', 'Naloxone', 'Fluoxymesterone', 'Pergolide', 'Iophendylate', 'Ciclopirox', 'Desflurane', 'Clindamycin', 'Dexfenfluramine', 'Oxymorphone', 'Brinzolamide', 'Flecainide', 'Estramustine', 'Captopril', 'Zopiclone', 'Tubocurarine', 'Bromocriptine', 'Rifapentine', 'Levetiracetam', 'Nadolol', 'Mitoxantrone', 'Flumazenil', 'Lomustine', 'Sparfloxacin', 'Dezocine', 'Levobunolol', 'Clarithromycin', 'Ceftriaxone', 'Fomepizole', 'Metipranolol', 'Estazolam', 'Anastrozole', 'Halofantrine', 'Dantrolene', 'Rifaximin', 'Ketamine', 'Budesonide', 'Aminophylline', 'Quetiapine', 'Enoxaparin', 'Mivacurium', 'Levacetylmethadol', 'Encainide', 'Pemoline', 'Diphenidol', 'Saquinavir', 'Metoclopramide', 'Dexamethasone', 'Levodopa', 'Sevoflurane', 'Bromodiphenhydramine', 'Aripiprazole', 'Chlorprothixene', 'Epoprostenol', 'Gemfibrozil', 'Clomipramine', 'Chloroxine', 'Bepridil', 'Decamethonium', 'Alimemazine', 'Isocarboxazid', 'Docetaxel', 'Iodixanol', 'Olsalazine', 'Ergometrine', 'Lisdexamfetamine', 'Eculizumab', 'Lapatinib', 'Desonide', 'Sitagliptin', 'Decitabine', 'Posaconazole', 'Darunavir', 'Telbivudine', 'Sinecatechins', 'Paliperidone', 'Sunitinib', 'Panitumumab', 'Ranibizumab', 'Idursulfase', 'Alglucosidase alfa', 'Varenicline', 'Arformoterol', 'Hydralazine', 'Exenatide', 'Mecasermin', 'Pramlintide', 'Galsulfase', 'Nelarabine', 'Abatacept', 'Carbetocin', 'Lumiracoxib', 'Tetracosactide', 'Corticotropin', 'Fenoterol', 'Pirbuterol', 'Bismuth subsalicylate', 'Practolol', 'Sulfacytine', 'Rolitetracycline', 'Oxtriphylline', 'Insulin aspart', 'Insulin detemir', 'Insulin glulisine', 'Fosamprenavir', 'Fosphenytoin', 'Kava', 'Polythiazide', 'Quinethazone', 'Cefamandole', 'Cefazolin', 'Cefonicid', 'Cefotetan', 'Cefoxitin', 'Cefradine', 'Metocurine', 'Pancuronium', 'Pipecuronium', 'Vecuronium', 'Cilazapril', 'Tolevamer', 'Potassium cation', 'Quinidine barbiturate', 'Spirapril', 'Amobarbital', 'Butobarbital', 'Mestranol', 'Penbutolol', 'Iohexol', 'Ephedrine', 'Mephentermine', 'Procaterol', 'Rasagiline', 'Quinupristin', 'Aluminium', 'Magnesium oxide', 'Magnesium cation', 'Cortisone acetate', 'Ginkgo biloba', 'Glymidine', 'Sodium bicarbonate', 'Yohimbine', 'Bezafibrate', 'Colchicine', 'Drospirenone', 'Digitoxin', 'Salsalate', 'Neostigmine', 'Choline magnesium trisalicylate', 'Methotrimeprazine', 'Danazol', 'Clenbuterol', 'Tiotropium', 'Ciclesonide', 'Cefacetrile', 'Cefpodoxime', 'Acenocoumarol', 'Antrafenine', 'Aminophenazone', 'Ajmaline', 'Amrinone', 'Almitrine', 'Cholestyramine', 'Antipyrine', 'Alfacalcidol', 'Glutethimide', 'Phenazopyridine', 'gamma-Hydroxybutyric acid', 'Diamorphine', 'Dichloralphenazone', 'Etorphine', 'Difenoxin', 'Carfentanil', 'Boldenone', 'Flunitrazepam', 'Diprenorphine', 'Dihydrocodeine', 'Bromazepam', 'Clotiazepam', 'Chloral hydrate', 'Attapulgite', 'Dextroamphetamine', 'Metamfetamine', 'Phendimetrazine', 'Oxprenolol', 'Sulfamerazine', 'Sulfamethazine', 'Liotrix', 'Thyroglobulin', 'Ursodeoxycholic acid', 'Ketazolam', 'Prazepam', 'Quazepam', 'Everolimus', 'Solifenacin', 'Iron', 'Zinc', 'Nitrazepam', 'Cilastatin', 'Imipenem', 'Probucol', 'Tiaprofenic acid', 'Lopinavir', 'Bacampicillin', 'Meticillin', 'Pivampicillin', 'Pivmecillinam', 'Tazobactam', 'Ticarcillin', 'Periciazine', 'Deferasirox', 'Valganciclovir', 'Hydroxychloroquine', 'Amyl Nitrite', 'Erythrityl tetranitrate', 'Acepromazine', 'Alverine', 'Molindone', 'Phenindamine', 'Pheniramine', 'Pipotiazine', 'Thiothixene', 'Zuclopenthixol', 'Isopropamide', 'Pargyline', 'Lincomycin', 'Etoricoxib', '5-O-phosphono-alpha-D-ribofuranosyl diphosphate', 'Sorbitol', 'Roflumilast', 'Rutin', 'Prasterone', 'Cetrimonium', 'Camphor', 'Dalfopristin', 'Pantothenic acid', 'Propylene glycol', 'Taurine', 'Cocarboxylase', 'Flufenamic acid', 'Calcipotriol', 'Isopropyl alcohol', 'Aminobenzoic acid', 'Thymol', 'gamma-Aminobutyric acid', 'Vorinostat', 'Terlipressin', 'Cholic Acid', 'Nicotinamide', 'Fusidic acid', 'Sucrose', 'Pregnenolone', 'D-Methionine', 'Piretanide', 'Oxitriptan', 'Arsanilic acid', 'Glycolic acid', 'Pidolic acid', 'Acetylcholine', 'Flavin adenine dinucleotide', 'Acetic acid', 'Propyl alcohol', 'Stearic acid', 'Oteracil', 'Flavin mononucleotide', 'Phenol', 'Glutathione disulfide', 'Brivudine', 'Hemin', 'Oxyphenbutazone', 'Mecobalamin', 'Ribostamycin', 'Deoxycholic acid', 'Doconexent', 'Propanoic acid', 'Benzoic acid', 'Palmitic Acid', 'Hexamidine', 'Formaldehyde', 'Urea', 'D-Serine', 'Berberine', 'Pyrophosphoric acid', 'Fructose', 'Dequalinium', 'Didecyldimethylammonium', 'Oleic Acid', 'Citric acid', 'Carbocisteine', 'Taurocholic acid', 'Lactic acid', 'Lactose', 'Cholesterol', 'Gluconolactone', 'Latamoxef', 'Trioxsalen', 'Thiotepa', 'Estriol', 'Estrone sulfate', 'Quinestrol', 'Carboxin', 'Hesperidin', 'Iodipamide', 'Nimesulide', 'Bifonazole', 'Benoxaprofen', 'Bithionol', 'Dantron', 'Metamizole', 'Nialamide', 'Nomifensine', 'Oxeladin', 'Phenolphthalein', 'Prenylamine', 'Thenalidine', 'Urethane', 'Zimelidine', 'Rapacuronium', 'Maraviroc', 'Clofedanol', 'Cyclandelate', 'Cyproterone acetate', 'Debrisoquine', 'Flunarizine', 'Fluspirilene', 'Mepenzolate', 'Tetrabenazine', 'Ixabepilone', 'Celiprolol', 'Febuxostat', 'Dronedarone', 'Nebivolol', 'Huperzine A', 'Omacetaxine mepesuccinate', 'Halofuginone', 'Nilotinib', 'Lorcaserin', 'Vildagliptin', 'Enoximone', 'Bepotastine', 'Pegaptanib', 'Milnacipran', 'Lucinactant', 'Ximelagatran', 'Nesiritide', 'Flibanserin', 'Oxibendazole', 'Oritavancin', 'Ceftobiprole', 'Clevidipine', 'rhThrombin', 'Permethrin', 'Afamelanotide', 'Defibrotide', 'Ospemifene', 'Crofelemer', 'Hemoglobin', 'Iloperidone', 'Lofexidine', 'Pirfenidone', 'Ezogabine', 'Ingenol mebutate', 'Belinostat', 'Ataluren', 'Indacaterol', 'Erdosteine', 'Abaloparatide', 'Ancrod', 'Pretomanid', 'Crisaborole', 'Cobimetinib', 'Silver sulfadiazine', 'Methsuximide', 'Glatiramer', 'Gallium nitrate', 'Rotigotine', 'Samarium (153Sm) lexidronam', 'Hepatitis B Immune Globulin', 'Vandetanib', 'keyhole limpet hemocyanin', 'Ecallantide', 'Pimavanserin', 'Romiplostim', 'Obiltoxaximab', 'Dexlansoprazole', 'Ragweed pollen extract', 'Histamine', 'Iodine', 'Tetrachlorodecaoxide', 'Telaprevir', 'Brivaracetam', 'Ramucirumab', 'Sodium stibogluconate', 'Apremilast', 'Ustekinumab', 'Trastuzumab emtansine', 'Thrombomodulin Alfa', 'Abiraterone', 'Parathyroid hormone', 'Inotuzumab ozogamicin', 'Obeticholic acid', 'Eteplirsen', 'Cariprazine', 'Olaratumab', 'Lumateperone', 'Caplacizumab', 'Cenobamate', 'Tirbanibulin', 'Sertindole', 'Spiramycin', 'Sulfathiazole', 'Mianserin', 'Teicoplanin', 'Sulfadimethoxine', 'Acetylcysteine', 'Nylidrin', 'Pizotifen', 'Rimonabant', 'Canakinumab', 'Ipilimumab', 'Pixantrone', 'Icatibant', 'Rufinamide', 'Lasofoxifene', 'Alogliptin', 'Tapentadol', 'Hyaluronidase (human recombinant)', 'Sugammadex', 'Silodosin', 'Prasugrel', 'Eltrombopag', 'Tolvaptan', 'Regadenoson', 'Ferumoxytol', 'Asenapine', 'Vernakalant', 'Lacosamide', 'Dalbavancin', 'Rivaroxaban', 'Nalmefene', 'Avanafil', 'Eflornithine', 'Arzoxifene', 'Hexaminolevulinate', 'Droxidopa', 'Udenafil', 'Sitaxentan', 'Sulodexide', 'Tocilizumab', 'Alvimopan', 'Levocetirizine', 'Ziconotide', 'Teriparatide', 'Temsirolimus', 'Amisulpride', 'Simeprevir', 'Dapagliflozin', 'Elotuzumab', 'Saxagliptin', 'Teprotumumab', 'rsPSMA Vaccine', 'Pertuzumab', 'Rilonacept', 'Bazedoxifene', 'Telavancin', 'Ambrisentan', 'Human C1-esterase inhibitor', 'Doxercalciferol', 'Oxymetholone', 'Armodafinil', 'Etravirine', 'Calfactant', 'Cangrelor', 'Lonafarnib', 'Prucalopride', 'Pazopanib', 'Ceftaroline fosamil', 'Agomelatine', 'Midostaurin', 'Reslizumab', 'Panobinostat', 'Apixaban', 'Catumaxomab', 'Tafenoquine', 'Mepolizumab', 'Peramivir', 'Bosutinib', 'Axitinib', 'Isavuconazonium', 'Denosumab', 'Ofatumumab', 'Safinamide', 'Liraglutide', 'Pasireotide', 'Golimumab', 'Belatacept', 'Vilazodone', 'Sipuleucel-T', 'Ethanolamine oleate', 'Nitrous oxide', 'Mepyramine', 'Xylometazoline', 'Dabigatran etexilate', 'Artemether', 'Betahistine', 'Degarelix', 'Desvenlafaxine', 'Dexmethylphenidate', 'Fesoterodine', 'Gadobutrol', 'Iobenguane', 'Gadofosveset trisodium', 'Isometheptene', 'Levonordefrin', 'Lumefantrine', 'Methacholine', 'Methyltestosterone', 'Naphazoline', 'Nilvadipine', 'Norelgestromin', 'Propylhexedrine', 'Potassium Iodide', 'Fospropofol', 'Fosaprepitant', 'Stanozolol', 'Buserelin', 'Velaglucerase alfa', 'Aluminum hydroxide', 'Calcium carbonate', 'Lornoxicam', 'Sulfaphenazole', 'Aceclofenac', 'Drotaverine', 'Chymopapain', 'Triclofos', 'Danaparoid', 'Beta carotene', 'Glycine betaine', 'Manganese', 'Fomivirsen', 'Sacrosidase', 'Beractant', 'Pinacidil', 'Tetryzoline', 'Alcaftadine', 'Ammonium chloride', 'Ammonium lactate', 'Bendamustine', 'Benzyl alcohol', 'Besifloxacin', 'Cabazitaxel', 'Human calcitonin', 'Capsaicin', 'Carglumic acid', 'Sodium cellulose phosphate', 'Chenodeoxycholic acid', 'Cupric sulfate', 'Dalteparin', 'Desoxycorticosterone acetate', 'Difluprednate', 'Dimercaprol', 'Prussian blue', 'Gallium citrate Ga-67', 'Ganirelix', 'Halcinonide', 'Hexocyclium', 'Histrelin', 'Hydroxyprogesterone caproate', 'Lanreotide', 'Lanthanum carbonate', 'Lodoxamide', 'Mafenide', 'Mangafodipir', 'Mebutamate', 'Methylnaltrexone', 'Nepafenac', 'Niclosamide', 'Nonoxynol-9', 'Phenyl aminosalicylate', 'Plerixafor', 'Plicamycin', 'Polidocanol', 'Povidone-iodine', 'Pralatrexate', 'Protokylol', 'Pyrithione', 'Raltegravir', 'Sulconazole', 'Sulfameter', 'Tinzaparin', 'Tiopronin', 'Triethylenetetramine', 'Triptorelin', 'Unoprostone', 'Viomycin', 'Chloramphenicol succinate', 'Sulthiame', 'Parecoxib', 'Salicylamide', 'Antazoline', 'Dimetindene', 'Nandrolone decanoate', 'Dichloroacetic acid', 'Nadroparin', 'Triflusal', 'Lurasidone', 'Ticagrelor', 'Hyaluronic acid', 'Tafluprost', 'Ivacaftor', 'Azilsartan medoxomil', 'Ioflupane I-123', 'Deferiprone', 'Lomitapide', 'Vismodegib', 'Acetylcarnitine', 'Pitavastatin', 'Cholecystokinin', 'Rilpivirine', 'Crizotinib', 'Ulipristal', 'Fingolimod', 'Tesamorelin', 'Brentuximab vedotin', 'Eribulin', 'Gabapentin enacarbil', 'Boceprevir', 'Fidaxomicin', 'Cabozantinib', 'Taliglucerase alfa', 'Ruxolitinib', 'Belimumab', 'Vemurafenib', 'Linagliptin', 'Perampanel', 'Gadoxetic acid', 'Aflibercept', 'Asparaginase Erwinia chrysanthemi', 'Icosapent ethyl', 'Ocriplasmin', 'Carfilzomib', 'Linaclotide', 'Peginesatide', 'Tofacitinib', 'Regorafenib', 'Aclidinium', 'Glucarpidase', 'Enzalutamide', 'Teduglutide', 'Ponatinib', 'Raxibacumab', 'Bedaquiline', 'Certolizumab pegol', 'Formestane', 'Fluticasone furoate', 'Canagliflozin', 'Dimethyl fumarate', 'Glycerol phenylbutyrate', 'Pomalidomide', 'Trametinib', 'Dabrafenib', 'Radium Ra 223 dichloride', 'Afatinib', 'Ferric carboxymaltose', 'Levomilnacipran', 'Dolutegravir', 'Riociguat', 'Macitentan', 'Luliconazole', 'Sofosbuvir', 'Obinutuzumab', 'Chlorcyclizine', 'Magaldrate', 'Isoxsuprine', 'Isoxicam', 'Isoconazole', 'Isoaminile', 'Iopanoic acid', 'Iopamidol', 'Inositol nicotinate', 'Indoramin', 'Hexetidine', 'Gemeprost', 'Fusafungine', 'Fursultiamine', 'Dimetotiazine', 'Fluocortolone', 'Fluclorolone acetonide', 'Floctafenine', 'Fenbufen', 'Etilefrine', 'Etidocaine', 'Ethoheptazine', 'Diosmin', 'Articaine', 'Canrenoic acid', 'Butriptyline', 'Bromhexine', 'Bisacodyl', 'Aliskiren', 'Ledipasvir', 'Secukinumab', 'Vorapaxar', 'Miltefosine', 'Vedolizumab', 'Suvorexant', 'Nivolumab', 'Siltuximab', 'Pembrolizumab', 'Empagliflozin', 'Eliglustat', 'Efinaconazole', 'Tedizolid phosphate', 'Albiglutide', 'Dulaglutide', 'Metreleptin', 'Finafloxacin', 'Netupitant', 'Naloxegol', 'Ceftolozane', 'Elosulfase alfa', 'Blinatumomab', 'Ibrutinib', 'Idelalisib', 'Acipimox', 'Amorolfine', 'Anthrax immune globulin human', 'Atosiban', 'Avibactam', 'Ceritinib', 'Ciprofibrate', 'Cobicistat', 'Corifollitropin alfa', 'Corticorelin ovine triflutate', 'Vortioxetine', 'Trimetazidine', 'Tibolone', 'Tasimelteon', 'Sodium oxybate', 'Palbociclib', 'Olaparib', 'Edoxaban', 'Umeclidinium', 'Dinutuximab', 'Lenvatinib', 'Nintedanib', 'Olodaterol', 'Idebenone', 'Vilanterol', 'Ivabradine', 'Benzydamine', 'Tetracaine', 'Potassium alum', 'Amylocaine', 'Trimebutine', 'Pinaverium', 'Tixocortol', 'Xanthinol', 'Chlortetracycline', 'Podophyllin', 'Difluocortolone', 'Benzoyl peroxide', 'Quinagolide', 'Somatrem', 'Somatostatin', 'Thyroid, porcine', 'Elvitegravir', 'Daclatasvir', 'Ancestim', 'Asfotase alfa', 'Hydroxyethyl Starch', 'Methoxy polyethylene glycol-epoetin beta', 'Simoctocog alfa', 'Turoctocog alfa', 'Coenzyme M', 'Pentastarch', 'Nitrous acid', 'Colfosceril palmitate', 'Diiodohydroxyquinoline', 'Calcium carbimide', 'Paraldehyde', 'Stiripentol', 'Zucapsaicin', 'Aurothioglucose', 'Peginterferon beta-1a', 'Dienogest', 'Medrogestone', 'Potassium citrate', 'Chorionic Gonadotropin (Human)', 'Chromic chloride', 'Copper', 'Cupric Chloride', 'Gadoteric acid', 'Iothalamic acid', 'Ioversol', 'Ioxilan', 'Isosulfan blue', 'Technetium Tc-99m mebrofenin', 'Technetium Tc-99m medronate', 'Technetium Tc-99m oxidronate', 'Oxygen', 'Protamine sulfate', 'Sincalide', 'Sonidegib', 'Uridine triacetate', 'Water', 'Iron sucrose', 'Florbetaben (18F)', 'Florbetapir (18F)', 'Flutemetamol (18F)', 'Nitrogen', 'Sodium chloride', 'Sodium citrate', 'Helium', 'Iopromide', 'Carbon dioxide', 'Trypan blue free acid', 'Technetium Tc-99m tetrofosmin', 'Technetium Tc-99m sestamibi', 'Technetium Tc-99m exametazime', 'Technetium Tc-99m disofenin', 'Technetium Tc-99m pyrophosphate', 'Dosulepin', 'Dasabuvir', 'Viloxazine', 'Lorpiprazole', 'Moxisylyte', 'Pegloticase', 'Pholcodine', 'Piracetam', 'Loxoprofen', 'Dexibuprofen', 'Dexketoprofen', 'Tolfenamic acid', 'Firocoxib', 'Bisoxatin', 'Nicorandil', 'Fibrinogen human', 'Zotepine', 'Conestat alfa', 'Lacidipine', 'Levamlodipine', 'Manidipine', 'Methylene blue', 'Moxonidine', 'Tegafur', 'Gimeracil', 'Bemiparin', 'Reviparin', 'Parnaparin', 'Certoparin', 'Patiromer', 'Idarucizumab', 'Lixisenatide', 'Technetium Tc-99m tilmanocept', 'Strontium ranelate', 'Picosulfuric acid', 'Phenylacetic acid', 'Ubidecarenone', 'Eluxadoline', 'Artesunate', 'Bismuth subcitrate potassium', 'Sodium aurothiomalate', 'Choline C 11', 'Activated charcoal', 'Lumacaftor', 'Magnesium trisilicate', 'Molsidomine', 'Polyethylene glycol', 'Rolapitant', 'Sacubitril', 'Iodide I-131', 'Ombitasvir', 'Paritaprevir', 'Tenofovir alafenamide', 'Butylscopolamine', 'Chondroitin sulfate', 'Alirocumab', 'Evolocumab', 'Catridecacog', 'Antilymphocyte immunoglobulin (horse)', 'Ioxaglic acid', 'Technetium Tc-99m pertechnetate', 'Xenon-133', 'Thallous Chloride', 'Synthetic Conjugated Estrogens, A', 'Synthetic Conjugated Estrogens, B', 'Carindacillin', 'Procaine benzylpenicillin', 'Zinc oxide', 'Zinc sulfate', 'Ammonia N-13', 'Tegafur-uracil', 'Vayarin', 'Antihemophilic Factor (Recombinant), PEGylated', 'Osimertinib', 'Daratumumab', 'Kappadione', 'Iopodic acid', 'Seractide acetate', 'Alatrofloxacin', 'Technetium Tc-99m nofetumomab merpentan', 'Medical air', 'Tyropanoic acid', 'Dextrose, unspecified form', 'Propoxycaine', 'Tipiracil', 'Pramocaine', 'Metrizoic acid', 'Propiolactone', 'Gastric intrinsic factor', 'Piperonyl butoxide', 'Levobetaxolol', 'Hydroxyamphetamine', 'Octasulfur', 'Sulfabenzamide', 'Dexpanthenol', 'Procaine merethoxylline', 'Propyliodone', 'Polyestradiol phosphate', 'Sodium phosphate P 32', 'Norethynodrel', 'Tannic acid', 'Trolamine polypeptide oleate condensate', 'Indocyanine green acid form', 'Chymotrypsin', 'Lapyrium', 'Undecoylium chloride iodine complex', 'Fluprednisolone', 'Sutilain', 'Sodium chromate Cr-51', 'Esterified estrogens', 'Iodohippurate sodium I-131', 'Meprednisone', 'Cyanocobalamin Co-57', 'Norgestrel', 'Phosphoric acid', 'Sodium acetate', 'Technetium Tc-99m sulfur colloid', 'Fluoride ion F-18', 'Selenomethionine Se-75', 'Isosorbide', 'Iocetamic acid', 'Magnesium chloride', 'Magnesium acetate tetrahydrate', 'Monopotassium phosphate', 'Dipotassium phosphate', 'Meglumine', 'Potassium perchlorate', 'Xylose', 'Iodide I-123', 'Protirelin', 'Soybean oil', 'Technetium Tc-99m polyphosphate', 'Indium In-111 pentetate', 'Albumin iodinated I-125 serum', 'Albumin iodinated I-131 serum', 'Krypton Kr 81m', 'Safflower oil', 'Technetium Tc-99m albumin colloid', 'Iodohippurate sodium I-123', 'Sodium phosphate, monobasic', 'Insulin beef', 'Sodium carbonate', 'Xenon Xe-127', 'Glycerin', 'Indium In-111 oxyquinoline', 'Enalaprilat', 'Rubidium Rb-82', 'Iofetamine I-123', 'Magnesium carbonate', 'Potassium lactate', 'Sodium fluorophosphate', 'Iotrolan', 'Acrivastine', 'Indium In-111 chloride', 'Technetium Tc-99m red blood cells', 'Cetyl alcohol', 'Avobenzone', 'Octinoxate', 'Strontium chloride Sr-89', 'Thiosulfuric acid', 'Ferric ammonium citrate', 'Fludeoxyglucose (18F)', 'Ferumoxsil', 'Ferumoxides', 'Urea C-13', 'Talc', 'Simethicone', 'Urea C-14', 'Mequinol', 'Sodium ferric gluconate complex', 'Hydroquinone', 'Secretin porcine', 'Perflexane', 'Secretin human', 'Ecamsule', 'Octocrylene', 'Titanium dioxide', 'Iodine povacrylex', 'Omega-3-acid ethyl esters', 'Methyl salicylate', 'Iobenguane sulfate I-123', 'Desoxyribonuclease', 'Thonzonium', 'Dexchlorpheniramine maleate', 'Necitumumab', 'Sodium glycerophosphate', 'Choline C-11', 'Insulin degludec', 'Olive oil', 'Omega-3-carboxylic acids', 'Ixazomib', 'Levmetamfetamine', 'Clostridium tetani', 'Corynebacterium diphtheriae', 'Meningococcal (groups A, C, Y and W-135) oligosaccharide diphtheria CRM197 conjugate vaccine', 'Rabies virus inactivated antigen, B', 'Haemophilus influenzae type B strain 1482 capsular polysaccharide tetanus toxoid conjugate antigen', 'Rotavirus vaccine', 'Rabies virus inactivated antigen, A', 'Neisseria meningitidis serogroup B recombinant LP2086 A05 protein variant antigen', 'Neisseria meningitidis serogroup B recombinant LP2086 B01 protein variant antigen', 'Streptococcus pneumoniae type 4 capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 6b capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 9v capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 14 capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 18c capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 19f capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 23f capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 1 capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 3 capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 5 capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 6A capsular polysaccharide diphtheria CRM197 protein conjugate antigen', 'Streptococcus pneumoniae type 7f capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Streptococcus pneumoniae type 19a capsular polysaccharide diphtheria crm197 protein conjugate antigen', 'Human papillomavirus type 6 L1 capsid protein antigen', 'Human papillomavirus type 11 L1 capsid protein antigen', 'Human papillomavirus type 16 L1 capsid protein antigen', 'Human papillomavirus type 18 L1 capsid protein antigen', 'Human papillomavirus type 31 L1 capsid protein antigen', 'Human papillomavirus type 33 L1 capsid protein antigen', 'Human papillomavirus type 45 L1 capsid protein antigen', 'Human papillomavirus type 52 L1 capsid protein antigen', 'Human papillomavirus type 58 L1 capsid protein antigen', 'Measles virus vaccine live attenuated', 'Mumps virus strain B level jeryl lynn live antigen', 'Rubella virus vaccine', 'Varicella Zoster Vaccine (Live/attenuated)', 'Streptococcus pneumoniae type 1 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 2 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 3 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 4 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 5 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 6b capsular polysaccharide antigen', 'Streptococcus pneumoniae type 7f capsular polysaccharide antigen', 'Streptococcus pneumoniae type 8 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 9n capsular polysaccharide antigen', 'Streptococcus pneumoniae type 9v capsular polysaccharide antigen', 'Streptococcus pneumoniae type 10a capsular polysaccharide antigen', 'Streptococcus pneumoniae type 11a capsular polysaccharide antigen', 'Streptococcus pneumoniae type 12f capsular polysaccharide antigen', 'Streptococcus pneumoniae type 14 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 15b capsular polysaccharide antigen', 'Streptococcus pneumoniae type 17f capsular polysaccharide antigen', 'Streptococcus pneumoniae type 18c capsular polysaccharide antigen', 'Streptococcus pneumoniae type 19f capsular polysaccharide antigen', 'Streptococcus pneumoniae type 19a capsular polysaccharide antigen', 'Streptococcus pneumoniae type 20 capsular polysaccharide antigen', 'Streptococcus pneumoniae type 22f capsular polysaccharide antigen', 'Streptococcus pneumoniae type 23f capsular polysaccharide antigen', 'Streptococcus pneumoniae type 33f capsular polysaccharide antigen', 'Haemophilus influenzae type B capsular polysaccharide meningococcal outer membrane protein conjugate antigen', 'Bacillus calmette-guerin substrain tice live antigen', 'Ginger', 'House dust', 'Acacia longifolia pollen', 'Fraxinus americana pollen', 'Paspalum notatum pollen', 'Fagus grandifolia pollen', 'Cynodon dactylon pollen', 'Betula lenta pollen', 'Poa annua pollen', 'Acer negundo pollen', 'Bromus inermis pollen', 'Xanthium strumarium pollen', 'Zea mays pollen', 'Rumex acetosella pollen', 'Rumex crispus pollen', 'Ulmus americana pollen', 'Ulmus pumila pollen', 'Corylus americana pollen', 'Carya ovata pollen', 'Sorghum halepense pollen', 'Poa pratensis pollen', 'Chenopodium album pollen', 'Acer saccharum pollen', 'Festuca pratensis pollen', 'Prosopis juliflora pollen', 'Artemisia vulgaris pollen', 'Morus rubra pollen', 'Quercus agrifolia pollen', 'Quercus virginiana pollen', 'Quercus alba pollen', 'Avena sativa pollen', 'Olea europaea pollen', 'Dactylis glomerata pollen', 'Carya illinoinensis pollen', 'Amaranthus retroflexus pollen', 'Plantago lanceolata pollen', 'Populus alba pollen', 'Ambrosia acanthicarpa pollen', 'Ambrosia artemisiifolia pollen', 'Ambrosia trifida pollen', 'Ambrosia psilostachya pollen', 'Salsola kali pollen', 'Lolium perenne pollen', 'Artemisia tridentata pollen', 'Atriplex wrightii pollen', 'Platanus occidentalis pollen', 'Platanus racemosa pollen', 'Phleum pratense pollen', 'Holcus lanatus pollen', 'Juglans nigra pollen', 'Juglans californica pollen', 'Salix nigra pollen', 'Felis catus skin', 'Dermatophagoides farinae', 'Dermatophagoides pteronyssinus', 'Agrostis gigantea pollen', 'Anthoxanthum odoratum pollen', 'Cotton seed', 'Canis lupus familiaris skin', 'Equus caballus skin', 'Ceiba pentandra fiber', 'Mus musculus skin', 'Orris', 'Rabbit', 'Cavia porcellus skin', 'Blatella germanica', 'Solenopsis invicta', 'Anas platyrhynchos feather', 'Acremonium strictum', 'Aspergillus fumigatus', 'Aspergillus niger var. niger', 'Aureobasidium pullulans var. pullutans', 'Chaetomium globosum', 'Cladosporium cladosporioides', 'Cladosporium sphaerospermum', 'Cochliobolus sativus', 'Epicoccum nigrum', 'Fusarium oxysporum vasinfectum', 'Helminthosporium solani', 'Mucor plumbeus', 'Neurospora intermedia', 'Penicillium chrysogenum var. chrysogenum', 'Phoma exigua var. exigua', 'Rhizopus arrhizus var. arrhizus', 'Rhodotorula rubra', 'Ustilago tritici', 'Stemphylium solani', 'Trichophyton mentagrophytes', 'Aspergillus repens', 'Aspergillus terreus', 'Acacia', 'Ailanthus altissima pollen', 'Alnus incana subsp. rugosa pollen', 'Medicago sativa pollen', 'Fraxinus velutina pollen', 'Populus tremuloides pollen', 'Morella cerifera pollen', 'Betula nigra pollen', 'Amaranthus palmeri pollen', 'Juniperus ashei pollen', 'Juniperus virginiana pollen', 'Xanthium strumarium var. canadense pollen', 'Populus deltoides pollen', 'Populus fremontii pollen', 'Populus deltoides subsp. monilifera pollen', 'Cupressus arizonica pollen', 'Taxodium distichum pollen', 'Eupatorium capillifolium pollen', 'Ulmus crassifolia pollen', 'Eucalyptus globulus pollen', 'Solidago canadensis pollen', 'Celtis occidentalis pollen', 'Juniperus californica pollen', 'Robinia pseudoacacia pollen', 'Acer rubrum pollen', 'Melaleuca quinquenervia pollen', 'Chenopodium ambrosioides pollen', 'Morus alba pollen', 'Quercus rubra pollen', 'Syagrus romanzoffiana pollen', 'Schinus molle pollen', 'Amaranthus spinosus pollen', 'Casuarina equisetifolia pollen', 'Pinus strobus pollen', 'Pinus echinata pollen', 'Ligustrum vulgare pollen', 'Elymus repens pollen', 'Ambrosia tenuifolia pollen', 'Ambrosia bidentata pollen', 'Artemisia frigida pollen', 'Distichlis spicata pollen', 'Liquidambar styraciflua pollen', 'Juglans regia pollen', 'Amaranthus tuberculatus pollen', 'Triticum aestivum pollen', 'Almond', 'Apricot', 'Asparagus', 'Avocado', 'Banana', 'Barley', 'String bean', 'Beef', 'Brazil nut', 'Broccoli', 'Buckwheat', 'Cabbage', 'Carrot', 'Casein', 'Celery', 'Cherry', 'Chicken', 'Coffee bean', 'Red king crab', 'Cucumber', 'Egg white', 'Egg', 'Egg yolk', 'Garlic', 'Grapefruit', 'Karaya gum', 'Honeydew melon', 'Lamb', 'Lemon', 'Lettuce', 'Lima bean', 'Lobster', 'Cow milk', 'Cultivated mushroom', 'Mustard seed', 'Oat', 'Black olive', 'Onion', 'Orange', 'Oyster', 'Pea', 'Peach', 'Peanut', 'Pear', 'Pecan', 'Green bell pepper', 'Pineapple', 'Pistachio', 'Plum', 'Pork', 'Potato', 'Rice', 'Rye', 'Sesame seed', 'Soybean', 'Spinach', 'Squash', 'Strawberry', 'Corn', 'Tomato', 'Tuna', 'Turkey', 'Vanilla', 'English walnut', 'Watermelon', 'Wheat', 'Tea leaf', 'Khuskia oryzae', 'Artemisia annua pollen', 'Clostridium tetani toxoid antigen (formaldehyde inactivated)', 'Corynebacterium diphtheriae toxoid antigen (formaldehyde inactivated)', 'Influenza A virus A/California/7/2009 X-179A (H1N1) antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Brisbane/60/2008 antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Phuket/3073/2013 antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/California/7/2009 X-181 (H1N1) antigen (propiolactone inactivated)', 'Influenza A virus A/South Australia/55/2014 IVR-175 (H3N2) antigen (propiolactone inactivated)', 'Influenza B virus B/Phuket/3073/2013 antigen (propiolactone inactivated)', 'Meriones unguiculatus skin', 'Capra hircus skin', 'Mesocricetus auratus skin', 'Sus scrofa skin', 'Oryctolagus cuniculus skin', 'Bombyx mori fiber', 'Duck', 'Allergenic extract- beef liver', 'Veal', 'Bluefish', 'Common carp', 'Haddock', 'Pacific halibut', 'Herring', 'Mackerel', 'Northern pike', 'Red snapper', 'Atlantic salmon', 'European pilchard', 'Smelt', 'White fish', 'Blackberry', 'Blueberry', 'Sour cherry', 'Cranberry', 'Date', 'Fig', 'Wine grape', 'Lime (citrus)', 'Raspberry', 'Artichoke', 'Kidney bean', 'Beet', 'Kiwi fruit', 'Brussels sprout', 'Cauliflower', 'Eggplant', 'Lentil', 'Green olive', 'Parsley', 'Sweet potato', 'Raphanus sativus', 'Rhubarb', 'Turnip', 'Hazelnut', 'Macadamia nut', 'Nectarine', 'Mango', 'Papaya', 'Leek', 'Okra', 'Parsnip', 'Chickpea', 'Black-eyed pea', 'Watercress', 'Arabica coffee bean', 'Tragacanth', 'Allspice', 'Caraway seed', 'Clove', 'Dill', 'Horseradish', 'Licorice', 'Nutmeg', 'Oregano', 'Paprika', 'White pepper', 'Poppy seed', 'Spearmint', 'Wheat bran', 'American chestnut', 'Hops', 'Musca domestica', 'Ctenocephalides canis', 'Aedes taeniorhynchus', 'Eurotium herbariorum', 'Curvularia inaequalis', 'Fusarium compactum', 'Geotrichum candidum', 'Phoma glomerata', 'Rhodotorula mucilaginosa', 'Stemphylium sarciniforme', 'Ustilago avenae', 'Tilletia caries', 'Puccinia graminis', 'Cotton', 'Flax seed', 'Agave sisalana fiber', 'Poa compressa pollen', 'Phalaris arundinacea pollen', 'Bouteloua gracilis pollen', 'Urochloa mutica pollen', 'Secale cereale pollen', 'Leymus condensatus pollen', 'Pascopyrum smithii pollen', 'Acacia baileyana pollen', 'Alnus rubra pollen', 'Alnus rhombifolia pollen', 'Fraxinus latifolia pollen', 'Betula populifolia pollen', 'Tamarix gallica pollen', 'Juniperus monosperma pollen', 'Juniperus pinchotii pollen', 'Juniperus scopulorum pollen', 'Juniperus osteosperma pollen', 'Juniperus occidentalis pollen', 'Tilia americana pollen', 'Mangifera indica pollen', 'Acer macrophyllum pollen', 'Acer saccharinum pollen', 'Broussonetia papyrifera pollen', 'Quercus velutina pollen', 'Quercus macrocarpa pollen', 'Quercus kelloggii pollen', 'Quercus gambelii pollen', 'Quercus lobata pollen', 'Quercus nigra pollen', 'Citrus sinensis pollen', 'Pinus taeda pollen', 'Pinus palustris pollen', 'Pinus ponderosa pollen', 'Pinus elliottii pollen', 'Pinus virginiana pollen', 'Pinus monticola pollen', 'Populus nigra pollen', 'Elaeagnus angustifolia pollen', 'Salix lasiolepis pollen', 'Salix discolor pollen', 'Populus balsamifera subsp. trichocarpa pollen', 'Taraxacum officinale pollen', 'Atriplex polycarpa pollen', 'Baccharis halimifolia pollen', 'Hymenoclea salsola pollen', 'Allenrolfea occidentalis pollen', 'Atriplex lentiformis pollen', 'Artemisia douglasiana pollen', 'Artemisia ludoviciana pollen', 'Urtica dioica pollen', 'Iva axillaris pollen', 'Ambrosia deltoidea pollen', 'Sarcobatus vermiculatus pollen', 'Atriplex canescens pollen', 'Artemisia absinthium pollen', 'Japanese encephalitis virus strain sa 14-14-2 antigen (formaldehyde inactivated)', 'Foreskin fibroblast (neonatal)', 'Bovine type I collagen', 'Foreskin keratinocyte (neonatal)', 'Influenza A virus A/California/7/2009 (H1N1) recombinant hemagglutinin antigen', 'Influenza A virus A/Victoria/361/2011 (H3N2) recombinant hemagglutinin antigen', 'Influenza B virus B/Wisconsin/1/2010 recombinant hemagglutinin antigen', 'Influenza A virus A/Texas/50/2012 (H3N2) recombinant hemagglutinin antigen', 'Influenza B virus B/Massachusetts/2/2012 recombinant hemagglutinin antigen', 'Influenza A virus A/Switzerland/9715293/2013 (H3N2) recombinant hemagglutinin antigen', 'Influenza B virus B/Phuket/3073/2013 recombinant hemagglutinin antigen', 'Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 antigen (propiolactone inactivated)', 'Influenza A virus A/Christchurch/16/2010 NIB-74 (H1N1) antigen (propiolactone inactivated)', 'Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (propiolactone inactivated)', 'Neisseria meningitidis serogroup b nhba fusion protein antigen', 'Neisseria meningitidis serogroup b fhbp fusion protein antigen', 'Neisseria meningitidis serogroup b nada protein antigen', 'Neisseria meningitidis serogroup B strain NZ98/254 outer membrane vesicle', 'Bordetella pertussis toxoid antigen (glutaraldehyde inactivated)', 'Bordetella pertussis filamentous hemagglutinin antigen (formaldehyde inactivated)', 'Bordetella pertussis pertactin antigen', 'Bordetella pertussis fimbriae 2/3 antigen', 'Influenza A virus A/California/7/2009 X-179A (H1N1) antigen (formaldehyde inactivated)', 'Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) antigen (formaldehyde inactivated)', 'Influenza B virus B/Phuket/3073/2013 antigen (formaldehyde inactivated)', 'Influenza B virus B/Brisbane/60/2008 antigen (formaldehyde inactivated)', 'Poliovirus type 1 antigen (formaldehyde inactivated)', 'Poliovirus type 2 antigen (formaldehyde inactivated)', 'Poliovirus type 3 antigen (formaldehyde inactivated)', 'Neisseria meningitidis group a capsular polysaccharide diphtheria toxoid conjugate antigen', 'Neisseria meningitidis group c capsular polysaccharide diphtheria toxoid conjugate antigen', 'Neisseria meningitidis group y capsular polysaccharide diphtheria toxoid conjugate antigen', 'Neisseria meningitidis group w-135 capsular polysaccharide diphtheria toxoid conjugate antigen', 'Influenza A virus A/California/7/2009(H1N1)-like antigen (formaldehyde inactivated)', 'Typhoid Vi polysaccharide vaccine', 'Bacillus calmette-guerin substrain connaught live antigen', 'Yellow Fever Vaccine', 'Mucor racemosus', 'Penicillium camemberti', 'Penicillium expansum', 'Penicillium roqueforti', 'Phoma destructiva', 'Rhizopus stolonifer', 'Fraxinus pennsylvanica pollen', 'Atriplex confertifolia pollen', 'Passalora fulva', 'Cochliobolus spicifer', 'Myrothecium verrucaria', 'Hypomyces perniciosus', 'Neurospora crassa', 'Microascus brevicaulis', 'Colletotrichum coccodes', 'Pleospora herbarum', 'Streptomyces griseus', 'Trichoderma viride', 'Trichophyton schoenleinii', 'Betula papyrifera pollen', 'Betula occidentalis pollen', 'Betula alleghaniensis pollen', 'Ustilago cynodontis', 'Ustilago nuda hordei', 'Bromus secalinus pollen', 'Beta vulgaris pollen', 'Chenopodium botrys pollen', 'Mycocladus corymbiferus', 'Humicola grisea', 'Microsporum audouinii', 'Apiospora montagnei', 'Phycomyces blakesleeanus', 'Sporotrichum pruinosum', 'Stachybotrys chartarum', 'Syncephalastrum racemosum', 'Tetracoccosporium paxianum', 'Verticillium albo-atrum', 'White catfish', 'Carob', 'Sweet cherry', 'Opilio crab', 'Bos taurus hair', 'Serinus canaria feather', 'Melilotus albus pollen', 'Melilotus officinalis pollen', 'Typha latifolia pollen', 'Atlantic cod', 'Red currant', 'Cumin', 'Odocoileus virginianus hair', 'Grain mill dust', 'Ambrosia dumosa pollen', 'Ambrosia confertiflora pollen', 'Sporisorium cruentum', 'Concord grape', 'Capra hircus hair', 'Cavia porcellus hair', 'Arrhenatherum elatius pollen', 'Equus caballus hair', 'Sus scrofa hair', 'Carya cordiformis pollen', 'Carya glabra pollen', 'Millet seed', 'Mustard greens', 'Quercus muehlenbergii pollen', 'Quercus palustris pollen', 'Quercus stellata pollen', 'Quercus garryana pollen', 'Red bell pepper', 'Perch', 'Pumpkin', 'Melopsittacus undulatus feather', 'Pinus contorta pollen', 'Pinus sylvestris pollen', 'Pinus rigida pollen', 'Capsicum', 'Oryctolagus cuniculus hair', 'Sequoia sempervirens pollen', 'Rumex obtusifolius pollen', 'Rumex altissimus pollen', 'Rumex salicifolius var. mexicanus pollen', 'Sitotroga cerealella', 'English sole', 'Safflower', 'Sugarcane', 'Sunflower seed', 'Platanus hybrida pollen', 'Artemisia californica pollen', 'Helianthus annuus pollen', 'Carpinus caroliniana pollen', 'Picea pungens pollen', 'Starch, tapioca', 'Larix occidentalis pollen', 'Salix lucida ssp. lasiandra pollen', 'Juniperus virginiana', 'Venison', 'Black walnut', 'Artemisia dracunculus pollen', 'Pinus banksiana pollen', 'Pinus resinosa pollen', 'Pinus nigra pollen', 'Pinus thunbergii pollen', 'Carya tomentosa pollen', 'Iva annua pollen', 'Bassia scoparia pollen', 'Cyclachaena xanthifolia pollen', 'Amaranthus retroflexus whole', 'Cephalosporium roseum', 'Cochliobolus lunatus', 'Haematonectria haematococca', 'Chrysonilia sitophila', 'Pleospora betae', 'Puccinia striiformis var. striiformis', 'Pleospora tarda', 'Acacia pollen', 'Bassia hyssopifolia pollen', 'Callistemon citrinus pollen', 'Dicoria canescens pollen', 'Krascheninnikovia lanata pollen', 'Ambrosia ambrosioides pollen', 'Ambrosia chamissonis pollen', 'Koeleria macrantha pollen', 'Phoenix dactylifera pollen', 'Quercus dumosa pollen', 'Suaeda moquinii pollen', 'Tilia cordata pollen', 'Apis mellifera venom', 'Vespula maculifrons venom protein', 'Dolichovespula arenaria venom protein', 'Dolichovespula maculata venom protein', 'Polistes fuscatus venom protein', 'Hordeum vulgare pollen', 'Agrostis stolonifera pollen', 'Phalaris minor pollen', 'Lolium perenne subsp. multiflorum pollen', 'Sorghum bicolor subsp. bicolor pollen', 'Sorghum bicolor subsp. drummondii pollen', 'Amphiachyris dracunculoides pollen', 'Iva angustifolia pollen', 'Neurospora sitophila', 'Trichophyton tonsurans', 'Mycogone nigra', 'Absidia capillata', 'Thermomyces lanuginosus', 'Trichosporon cutaneum', 'Tanacetum cinerariifolium flower', 'Largemouth bass', 'Cocoa', 'Blue crab', 'Barley malt', 'Radish', 'Sage', 'Thyme', 'Peppermint', 'Ulmus rubra pollen', 'Pseudotsuga menziesii pollen', 'Carya laciniosa pollen', 'Maclura pomifera pollen', 'Hepatitis A Vaccine', 'Haemophilus influenzae type B strain 20752 capsular polysaccharide tetanus toxoid conjugate antigen', 'Bordetella pertussis toxoid antigen (formaldehyde, glutaraldehyde inactivated)', 'Bordetella pertussis pertactin antigen (formaldehyde inactivated)', 'Human papillomavirus type 16 L1 capsid protein residues 2-471 antigen', 'Human papillomavirus type 18 l1 capsid protein residues 2-472 antigen', 'Influenza A virus A/Christchurch/16/2010 NIB-74XP (H1N1) antigen (formaldehyde inactivated)', 'Autologous cultured chondrocytes', 'Influenza A virus A/Brisbane/10/2010 (H1N1) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Texas/50/2012 X-223A (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Massachusetts/2/2012 antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/South Australia/55/2014 (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Utah/9/2014 antigen (MDCK cell derived, propiolactone inactivated)', 'Anthrax vaccine', 'Betula pendula pollen', 'Ulmus parvifolia pollen', 'Parkinsonia florida pollen', 'Cytisus scoparius flowering top', 'Poultry', 'Quahog, unspecified', 'Cod, unspecified', 'Crab leg', 'Salmon, unspecified', 'Canis lupus familiaris dander', 'Cavia porcellus dander', 'Bos taurus dander', 'Equus caballus dander', 'Aspergillus nidulans', 'Penicillium digitatum', 'Clonostachys rosea f. rosea', 'Dendryphiella vinosa', 'Solenopsis richteri', 'Polistes annularis venom protein', 'Polistes exclamans venom protein', 'Polistes metricus venom protein', 'Vespula germanica venom protein', 'Vespula pensylvanica venom protein', 'Vespula squamosa venom protein', 'Vespula vulgaris venom protein', 'Influenza A virus A/Bolivia/559/2013 (H1N1) live (attenuated) antigen', 'Influenza A virus A/Switzerland/9715293/2013 (H3N2) live (attenuated) antigen', 'Influenza B virus B/Phuket/3073/2013 live (attenuated) antigen', 'Influenza B virus B/Brisbane/60/2008 live (attenuated) antigen', 'Influenza A virus A/Brisbane/59/2007(H1N1) hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Uruguay/716/2007(H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Influenza B virus B/Brisbane/60/2008 hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/California/7/2009 X-181 (H1N1) hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Victoria/210/2009 X-187 (H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Victoria/210/2009 X-187 (H3N2) antigen (propiolactone inactivated)', 'Influenza B virus B/Brisbane/60/2008 antigen (propiolactone inactivated)', 'Influenza A virus A/Texas/50/2012 X-223 (H3N2) antigen (propiolactone inactivated)', 'Influenza B virus B/Massachusetts/2/2012 BX-51B antigen (propiolactone inactivated)', 'Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) antigen (propiolactone inactivated)', 'Influenza A virus A/California/7/2009(H1N1)-like hemagglutinin antigen (propiolactone inactivated)', 'Typhoid Vaccine Live', 'Azficel-T', 'Human cord blood hematopoietic progenitor cell', 'Mineral oil', 'Carboxymethylcellulose', 'Polyvinyl alcohol', 'Povidone', 'Octisalate', 'Polysorbate 80', 'Homosalate', 'Pancrelipase amylase', 'Pancrelipase protease', 'Selenic acid', 'Phenyl salicylate', 'Hydrogen fluoride', 'Cetylpyridinium', 'Dimethicone', 'Hypromellose', 'Polyethylene glycol 400', 'Trolamine salicylate', 'Silver nitrate', 'Aluminum chloride', 'Coal tar', 'Resorcinol', 'Cocoa butter', 'Pyrethrum extract', 'Docusate', 'Potassium nitrate', 'Stannous fluoride', 'Calcium citrate', 'Vitamin D', 'Desirudin', 'Meradimate', 'Plantago seed', 'Potassium bicarbonate', 'Allantoin', 'N-acetyltyrosine', 'Sulfur hexafluoride', 'Benzalkonium', 'Aluminum sesquichlorohydrate', 'Magnesium citrate', 'Lanolin', 'Castor oil', 'Eucalyptus oil', 'Ensulizole', 'Aluminum zirconium trichlorohydrex gly', 'Undecylenic acid', 'Ammonia', 'Turpentine', 'Chloroxylenol', 'Hypochlorite', 'Racepinephrine', 'Benzethonium', 'Calcium gluconate', 'Selenious acid', 'Carbamide peroxide', 'Capsicum oleoresin', 'Silicon dioxide', 'Omega-3 fatty acids', 'Cupric oxide', 'Selenium', 'Chromium', 'Molybdenum', 'Chromium Cr-51', 'Manganese gluconate', 'Selenomethionine', 'Cresol', 'Oxyquinoline', 'Aluminum zirconium pentachlorohydrate', 'Butamben', 'Aluminum zirconium tetrachlorohydrex gly', 'Barium sulfate', 'Sodium hydroxide', 'Trichloroacetate', 'Potassium hydroxide', 'Triclocarban', 'Pyrantel', 'Anthralin', 'Pectin', 'Sodium sulfide', 'Phenyltoloxamine', 'Polyethylene glycol 300', 'Bicisate', 'Rosin', 'Antithrombin Alfa', 'Calcium threonate', 'Ferrous asparto glycinate', 'Ferric sulfate', 'Zeaxanthin', 'Polysorbate 20', 'Exametazime', 'Homatropine', 'Rose bengal', 'Light green SF yellowish', 'Oftasceine', 'Pentoxyverine', 'Magnesium glycinate', 'Pantethine', 'Cobamamide', 'Ferrous cysteine glycinate', 'Peppermint oil', 'Aluminum zirconium octachlorohydrex gly', 'Menthyl salicylate', 'N-alkyl ethylbenzyl dimethyl ammonium (C12-C14)', 'Panthenol', 'Alcloxa', 'Bemotrizinol', 'Amiloxate', 'Aluminum chlorohydrex propylene glycol', 'Iron protein succinylate', 'Ferrous bisglycinate', 'Enzacamene', 'Dioxybenzone', 'Benzoin resin', 'Peg-100 stearate', 'Ethylhexyl methoxycrylene', 'Methylcellulose', 'Magnesium Aluminum Silicate', 'Lycopene', 'Saccharide isomerate', 'Thonzylamine', 'Trypsin', 'Borage oil', 'Aluminum sulfate', 'Gelatin', 'Ethyl macadamiate', 'Copper gluconate', 'Zinc gluconate', 'Tocopherol', 'Xanthan gum', 'Hexylresorcinol', 'Chromium picolinate', 'Levomefolic acid', 'Fluoride ion', 'Diacetyl benzoyl lathyrol', 'Atractylodes japonica root', 'Bisoctrizole', 'Polydatin', 'Calcium glycerophosphate', 'Sea salt', 'Diethylamino hydroxybenzoyl hexyl benzoate', 'Atractylodes japonica root oil', 'Polysilicone-15', 'Dihydroergocornine', 'Dihydro-alpha-ergocryptine', 'Epicriptine', 'Aluminum zirconium pentachlorohydrex gly', 'DL-Methylephedrine', 'Brilliant green cation', 'Bismuth subcarbonate', 'Diethyltoluamide', 'DL-dimyristoylphosphatidylcholine', 'DL-dimyristoylphosphatidylglycerol', 'Colloidal oatmeal', 'Aluminium tristearate', 'Rice bran oil', 'Coccidioides immitis spherule', 'Trichophyton verrucosum', 'Vanillyl butyl ether', 'Thrombin', 'Curcuma aromatica root oil', 'Phenoxyethanol', 'Prothrombin', 'Protein C', 'Methscopolamine', 'Cod liver oil', 'Glycol salicylate', 'Boric acid', 'Dipyrithione', 'Tetradecyl hydrogen sulfate (ester)', 'C11-12 isoparaffin', 'Factor IX Complex (Human)', '1-Palmitoyl-2-oleoyl-sn-glycero-3-(phospho-rac-(1-glycerol))', 'Sinapultide', 'Poloxamer 188', 'Kinetin', 'Clove oil', 'Ubiquinol', 'Ichthammol', 'Aluminum oxide', 'Silanol', 'Rubidium', 'Calcium Phosphate', 'Ascorbyl phosphate', 'Propolis wax', 'Polyquaternium-10 (400 cps at 2%)', 'Linseed oil', 'Guaiacol', 'Selexipag', 'Afoxolaner', 'Alfaxalone', 'Altrenogest', 'Amitraz', 'Amprolium', 'Avilamycin', 'Azaperone', 'Bambermycins', 'Buquinolate', 'Butonate', 'Cambendazole', 'Carbarsone', 'Carbomycin', 'Cefalonium', 'Chlorobutanol', 'Clorsulon', 'Coumaphos', 'Cythioate', 'Danofloxacin', 'Decoquinate', 'Deracoxib', 'Dichlorophen', 'Diclazuril', 'Dirlotapide', 'Efrotomycin', 'Emodepside', 'Eprinomectin', 'Famphur', 'Febantel', 'Fenbendazole', 'Fenprostalene', 'Fenthion', 'Fluralaner', 'Gamithromycin', 'Gleptoferron', 'Haloxon', 'Squalane', 'Imidacloprid', 'Lufenuron', 'Luprostiol', 'Marbofloxacin', 'Maropitant', 'Medetomidine', 'Mibolerone', 'Monensin', 'Moxidectin', 'Narasin', 'Nequinate', 'Nicarbazin', 'Nifurpirinol', 'Nitarsone', 'Nitenpyram', 'Norgestomet', 'Oleandomycin', 'Orbifloxacin', 'Oxfendazole', 'Phenothiazine', 'Phosmet', 'Phthalofyne', 'Pimobendan', 'Poloxalene', 'Ponazuril', 'Pradofloxacin', 'Prostalene', 'Robenacoxib', 'Ronnel', 'Rotenone', 'Roxarsone', 'Selamectin', 'Squalene', 'Sulfachlorpyridazine', 'Sulfaethoxypyridazine', 'Sulfanitran', 'Sulfaquinoxaline', 'Tepoxalin', 'Thiostrepton', 'Tiamulin', 'Tildipirosin', 'Tilmicosin', 'Tioxidazole', 'Metrifonate', 'Tulathromycin A', 'Tylosin', 'Virginiamycin', 'Xylazine', 'Zeranol', 'Zilpaterol', 'Zoalene', 'Atipamezole', 'Balsam of Peru', 'Ceftiofur', 'Dexamethasone isonicotinate', 'Nalorphine', 'Sarafloxacin', 'Sodium tripolyphosphate', '(S)-methoprene', '2-mercaptobenzothiazole', 'Sulfotep', 'Amphomycin', 'Bunamidine', 'Butacaine', 'Caramiphen', 'Cefovecin', 'Cloprostenol', 'Cupric glycinate', 'Deslorelin', 'Difloxacin', 'Dihydrostreptomycin', 'Diperodon', 'Dithiazanine', 'Etymemazine', 'Flunixin', 'Fluprostenol', 'Imidocarb', 'Isoflupredone', 'Maduramicin', 'Masitinib', 'Melarsomine', 'Melengestrol', 'Metoserpate', 'Milbemycin oxime', 'Butyl chloride', 'Pirlimycin', 'Propiopromazine', 'Ractopamine', 'Romifidine', 'Salinomycin', 'Sulfabromomethazine', 'Tiletamine', 'Trenbolone', 'Tricaine', 'Tylvalosin', 'Zolazepam', 'Detomidine', 'Lesinurad', 'Sebelipase alfa', 'Ixekizumab', 'Padimate O', 'Human thrombin', 'Thrombin alfa', 'Aluminum chlorohydrate', 'Elbasvir', 'Grazoprevir', 'Indigotindisulfonic acid', 'Hard fat', 'Light Mineral Oil', 'Ravulizumab', 'Venetoclax', 'Cetalkonium', 'Pipradrol', 'Drometrizole trisiloxane', 'Asunaprevir', 'Etafedrine', 'Neon', 'Thimerosal', 'Bilastine', 'Domiphen', 'Atezolizumab', 'Levoleucovorin', 'Human Rho(D) immune globulin', 'Antithrombin III human', 'Starch, corn', 'Beeswax', 'Tuberculin purified protein derivative', 'Hydroxyethyl cellulose', 'Human rabies virus immune globulin', 'Tetanus Immune Globulin', 'Myrrh', 'Susoctocog alfa', 'Efmoroctocog alfa', 'Eftrenonacog alfa', 'Normethadone', 'Oxilofrine', 'Lifitegrast', 'Velpatasvir', 'Rupatadine', 'Gestrinone', 'Human Varicella-Zoster Immune Globulin', 'Dehydrocholic acid', 'Epoetin delta', 'Tasonermin', 'Hepatitis B Vaccine (Recombinant)', 'Laropiprant', 'Temoporfin', 'Eptotermin Alfa', 'Opicapone', 'Isavuconazole', 'Colestilan chloride', 'Tocofersolan', 'Nomegestrol', 'Delamanid', 'Dibotermin alfa', 'Amifampridine', 'Vinflunine', 'Pitolisant', 'Recombinant Cholera Toxin B Subunit', 'Tafamidis', 'Tucatinib', 'Bremelanotide', 'Rebamipide', 'Latanoprostene bunod', 'Curcumin', 'Triheptanoin', 'Treosulfan', 'Selumetinib', 'Naldemedine', 'Tropisetron', 'Setmelanotide', 'Acalabrutinib', 'Iomeprol', 'Tezacaftor', 'Encorafenib', 'Ribociclib', 'Lasmiditan', 'Rilmenidine', 'Ebastine', 'Benfotiamine', 'Clobetasol', 'Cabotegravir', 'Rifamycin', 'Istradefylline', 'Tenapanor', 'Sarilumab', 'Zytron', 'Brodalumab', 'Neuropeptide Y', 'Capmatinib', 'Niraparib', 'Fostemsavir', 'Bictegravir', 'Tivozanib', 'Tilarginine', 'Baricitinib', 'Nifurtimox', 'Esketamine', 'Ertugliflozin', 'Neratinib', 'Guselkumab', 'Indium In-111 pentetreotide', 'Osilodrostat', 'Angiotensin II', 'Relugolix', 'Revefenacin', 'Brexanolone', 'Romosozumab', 'Apalutamide', 'Valbenazine', 'Deflazacort', 'Abametapir', 'Bempedoic acid', 'Selinexor', 'Delafloxacin', 'Avelumab', 'Lemborexant', 'Duvelisib', 'Dacomitinib', 'Binimetinib', 'Glasdegib', 'Elagolix', 'Entrectinib', 'Ocrelizumab', 'Diacerein', 'Avatrombopag', 'Abemaciclib', 'Fostamatinib', 'Alpelisib', 'Tecovirimat', 'Benralizumab', 'Sarecycline', 'Letermovir', 'Gadolinium', 'Telotristat ethyl', 'Mannitol busulfan', 'Vaborbactam', 'Sultamicillin', 'Lorlatinib', 'Gilteritinib', 'Erdafitinib', 'Citicoline', 'Dupilumab', 'Deutetrabenazine', 'Iron isomaltoside 1000', 'Polatuzumab vedotin', 'Edaravone', 'Stem bromelain', 'Brigatinib', 'Polihexanide', 'Propiverine', 'Luspatercept', 'Nefopam', 'P-nitrobiphenyl', 'Doravirine', 'Dopexamine', 'Cantharidin', 'Eravacycline', 'Rucaparib', 'Temocillin', 'Diaminopropanol tetraacetic acid', 'Betrixaban', 'Vestronidase alfa', 'Siponimod', 'Velmanase alfa', 'Relebactam', 'Bromperidol', 'Remimazolam', 'Iobitridol', 'Steviolbioside', 'Rimegepant', 'Favipiravir', 'Lynestrenol', 'Copanlisib', 'Piritramide', 'Mogamulizumab', 'Clascoterone', 'Mizolastine', 'Inebilizumab', 'Oxetacaine', 'Benzodiazepine', 'Mebeverine', 'Ozanimod', 'Protionamide', 'Lurbinectedin', 'Moxetumomab Pasudotox', 'Ibalizumab', 'Perazine', 'Sotagliflozin', 'Fenpropidin', 'Iodide', 'Gaxilose', 'Benserazide', 'Dinoprost', 'Boscalid', 'Trifarotene', 'Perflubutane', 'Grapiprant', 'Pegvaliase', 'Etelcalcetide', 'Benperidol', 'Vonicog Alfa', 'Oxatomide', 'Tazemetostat', 'Sacituzumab govitecan', 'Factor XIII (human)', 'Nicoboxil', 'Ozenoxacin', 'Isoxaflutole', 'Darolutamide', 'Lactitol', 'Dihydralazine', 'Methylprednisone', 'Terizidone', 'Pexidartinib', 'Silicon', 'Enfortumab vedotin', 'Parthenolide', 'Macimorelin', 'Yttrium Y-90', 'Riamet', 'Biguanide', 'Amitriptylinoxide', 'Norflurane', 'Pentafluoropropane', 'Lusutrombopag', 'Von Willebrand Factor Human', 'Fluindione', 'Levosalbutamol', 'Bezlotoxumab', 'Calcium glubionate anhydrous', 'Lenograstim', 'Nedaplatin', 'Fluciclovine (18F)', 'Pancrelipase lipase', 'Coagulation factor X human', 'Protein S human', 'Anti-inhibitor coagulant complex', 'Coagulation Factor IX Human', 'Levomenol', 'Parachlorophenol', 'Esculin', 'Inosine pranobex', 'Sodium lauryl sulfoacetate', 'Clobetasone', 'Nusinersen', 'Terpin hydrate', 'Zofenopril', 'Alclofenac', 'Plecanatide', 'Cerliponase alfa', 'Inositol', 'Troleandomycin', 'Gluconic Acid', 'Antihemophilic factor human', 'Valerian', 'Pepsin', 'Lipegfilgrastim', 'Bismuth subnitrate', 'Guanoxan', 'Butaperazine', 'Oxolamine', 'Mandelic acid', 'Medifoxamine', 'Apronalide', 'Tilbroquinol', 'Dibenzepin', 'Flosequinan', 'Calcium lactate', 'Perboric acid', 'Thiram', 'Quinupramine', 'Pramiracetam', 'Phthalylsulfathiazole', 'Magnesium silicate', 'Proxibarbal', 'Clothiapine', 'Ferrous sulfate anhydrous', 'Etofamide', 'Ethyl chloride', 'Aceclidine', 'Hexobendine', 'Ritiometan', 'Acetarsol', 'Dichlorobenzyl alcohol', 'Dibekacin', 'Sultopride', 'Micronomicin', 'Benziodarone', 'Bucetin', 'Carbocromen', 'Benzododecinium', 'Meticrane', 'Bumadizone', 'Miocamycin', 'Tromantadine', 'Pimethixene', 'Ipecac', 'Atracurium', 'Hydrotalcite', 'Trichloroethylene', 'Pheneticillin', 'Bufexamac', 'Calcium alginate', 'Sodium feredetate', 'Edoxudine', 'Ioxitalamic acid', 'Bendazac', 'Tyrothricin', 'Alginic acid', 'Metergoline', 'Alipogene tiparvovec', 'Cyclopenthiazide', 'Mephenesin', 'Almasilate', 'Mifamurtide', 'Potassium gluconate', 'Methoxyphenamine', 'Ethyl hydroxybenzoate', 'Benorilate', 'Naftazone', 'Cefpirome', 'Sodium tartrate', 'Tritoqualine', 'Diphemanil', 'Sodium aurotiosulfate', 'Bioallethrin', 'Trolamine', 'Magnesium gluconate', 'Glycyrrhizic acid', 'Tilactase', 'Lidoflazine', 'Xamoterol', 'Acemetacin', 'Calcium levulinate', 'Muzolimine', 'Iodoform', 'Noxytiolin', 'Fluorodopa (18F)', 'Artemotil', 'Anethole trithione', 'Gamolenic acid', 'Dimazole', 'Fluticasone', 'Lormetazepam', 'Fenofibric acid', 'Enasidenib', 'Pibrentasvir', 'Glecaprevir', 'Tisagenlecleucel', 'Methyl nicotinate', 'Coral snake (micrurus fulvius) immune globulin antivenin (equine)', 'Albutrepenonacog alfa', 'Ferric subsulfate', 'Human cytomegalovirus immune globulin', 'Meningococcal polysaccharide vaccine group W-135', 'Meningococcal polysaccharide vaccine group Y', 'Meningococcal polysaccharide vaccine group A', 'Meningococcal polysaccharide vaccine group C', 'Crotalus scutulatus antivenin', 'Crotalus atrox antivenin', 'Crotalus adamanteus antivenin', 'Agkistrodon piscivorus antivenin', 'Rhus Glabra Pollen', 'Talimogene laherparepvec', 'Equine Botulinum Neurotoxin E Immune FAB2', 'Equine Botulinum Neurotoxin C Immune FAB2', 'Equine Botulinum Neurotoxin G Immune FAB2', 'Equine Botulinum Neurotoxin A Immune FAB2', 'Equine Botulinum Neurotoxin F Immune FAB2', 'Equine Botulinum Neurotoxin D Immune FAB2', 'Equine Botulinum Neurotoxin B Immune FAB2', 'Cat dander extract', 'Scorpion (centruroides) immune Fab2 antivenin (equine)', 'Amylmetacresol', 'Bismuth subgallate', 'Valproate bismuth', 'Phloxine B', 'Belladonna', 'Axicabtagene ciloleucel', 'Emicizumab', 'Varicella Zoster Vaccine (Recombinant)', 'Dotatate gallium Ga-68', 'Cenegermin', 'Semaglutide', 'Netarsudil', 'Voretigene neparvovec', 'Nonacog beta pegol', 'Testosterone cypionate', 'Testosterone enanthate', 'Testosterone undecanoate', 'Testosterone enantate benzilic acid hydrazone', 'Ferric cation', 'Estradiol acetate', 'Estradiol benzoate', 'Estradiol cypionate', 'Estradiol dienanthate', 'Estradiol valerate', 'Medium-chain triglycerides', 'Bronopol', 'Fish oil', 'Calcium saccharate', 'Peanut oil', 'Sarracenia Purpurea', 'Isopropyl myristate', 'Patent Blue', 'Plasma protein fraction (human)', 'Dioctyldimonium', 'Vanadium', 'Racemethionine', 'Remestemcel-L', 'Passiflora incarnata flower', 'Sulesomab', 'Potassium carbonate', 'Copper Cu-64', 'Nomegestrol acetate', 'Lutetium Lu-177', 'Lutetium Lu 177 dotatate', 'Strontium chloride', 'Ferric pyrophosphate citrate', 'Magnesium acetate', 'Baloxavir marboxil', 'Lonoctocog alfa', 'Moroctocog alfa', 'alpha-Tocopherol succinate', 'D-alpha-Tocopherol acetate', 'alpha-Tocopherol acetate', 'Tildrakizumab', 'Dimethicone 410', 'Choline salicylate', 'Pentetic acid', 'Burosumab', 'Sodium bisulfite', 'Ox bile extract', 'Bromotheophylline', 'Fosnetupitant', 'Benzoin', 'Typhoid vaccine', 'Acetyl sulfisoxazole', 'Erenumab', 'Eptinezumab', 'Fremanezumab', 'Galcanezumab', 'Dichlorobenzene', 'Sodium zirconium cyclosilicate', 'Imidurea', 'Medronic acid', 'Betiatide', 'Bisphenol A diglycidyl ether', 'Butylparaben', 'Cianidanol', 'Dimercaptosuccinic acid', '1,2-icosapentoyl-sn-glycero-3-phosphoserine', '1,2-Distearoyllecithin', 'Pork Collagen', 'Soy isoflavones', 'Linoleic acid', \"Sodium 1,2-Dipalmitoyl-sn-glycero-3-phospho-(1'-rac-glycerol)\", 'Tetrakis(2-methoxyisobutylisocyanide)copper(I) tetrafluoroborate', 'alpha-Arbutin', 'Butylene glycol', 'Human vaccinia virus immune globulin', 'Black widow spider antivenin (equine)', 'Viper antivenom', 'Human botulinum neurotoxin A/B immune globulin', 'Silodrate', 'Phenylethyl resorcinol', 'Racementhol', 'Hydroxyethyl ethylcellulose', 'Distearyldimonium', 'Human albumin microspheres', 'Chloric acid', 'Phosphorus', 'Synthetic camphor', 'Microcrystalline cellulose', 'Oxidronic acid', 'Northern bluefin tuna', 'Anthoxanthum odoratum', 'Betula lenta whole', 'Culex pipiens', 'Rumex acetosella whole', 'Rumex crispus top', 'Sarocladium strictum', 'Salix nigra bark', 'Artemisia vulgaris root', 'Penicillium glaucum', 'Evernia prunastri', 'Diazolidinylurea', 'Dipentamethylenethiuram disulfide', 'alpha-Amyl cinnamaldehyde', 'Benzylparaben', 'Propylparaben', 'Tetramethylthiuram monosulfide', 'Phleum pratense top', 'Nickel sulfate', 'Lanolin alcohols', 'Dichromate', 'Geraniol', 'Aripiprazole lauroxil', 'Cinnamyl alcohol', 'Hydroxycitronellal', 'Isoeugenol', 'Ethylenediamine', 'p-tert-Butylphenol-formaldehyde resin (low molecular weight)', 'Diphenylguanidine', \"N,N'-diphenyl-1,4-phenylenediamine\", 'Ditiocarb zinc', 'Zinc dibutyldithiocarbamate', '4-(Isopropylamino)diphenylamine', \"N-Cyclohexyl-N'-phenyl-1,4-phenylenediamine\", 'Methylchloroisothiazolinone', 'Quaternium-15', 'Bromothalonil', 'Thiohexam', \"2,2'-Dibenzothiazyl disulfide\", 'Morpholinylmercaptobenzothiazole', 'Disperse Blue 106', 'Cobalt chloride', 'Methylparaben', 'Monomethyl fumarate', 'Bifidobacterium longum infantis', 'Albumin Aggregated', 'Egg phospholipids', 'Bilberry', 'Echinacea angustifolia root', 'Echinacea purpurea flowering top', 'Caviar, unspecified', 'Honey', 'Snail, unspecified', 'Jojoba oil', 'Avena sativa flowering top', 'Aloe vera flower', 'Aloe ferox leaf', 'Elm', 'Bird pepper', 'Calendula officinalis flower', 'Rumex crispus whole', 'Ocimum tenuiflorum top', 'Salix alba bark', 'Indian frankincense', 'Turmeric', 'Carthamus tinctorius flower bud', 'Corydalis ambigua tuber', 'Lonicera caprifolium flower', 'Angelica archangelica root', 'Kaempferia galanga root', 'Larix sibirica wood', 'Wormwood', 'Eleuthero', 'Glycyrrhiza glabra', 'Eucalyptus globulus leaf', 'Apricot kernel oil', 'Purslane', 'Heterotheca inuloides flower', 'Inonotus obliquus fruiting body', 'Nelumbo nucifera flower', 'Atractylodes lancea root oil', 'Scutellaria lateriflora', 'Arnica montana flower', 'Equisetum arvense top', 'Inula helenium root', 'Usnea barbata', 'Azadirachta indica leaf', 'Zanthoxylum clava-herculis whole', 'Chamomile', 'Saw palmetto', 'Rice bran', 'Milk thistle', 'Influenza A virus A/California/7/2009 (H1N1)-like hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/California/7/2009 X-179A (H1N1) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Victoria/210/2009 X-187 (H3N2) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza B virus B/Brisbane/60/2008 hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza B virus B/Texas/6/2011 hemagglutinin antigen (formaldehyde inactivated)', 'Bordetella pertussis toxoid antigen (formaldehyde inactivated)', 'Influenza A virus A/Texas/50/2012 X-223A (H3N2) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza B virus B/Massachusetts/2/2012 hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Christchurch/16/2010 NIB-74 (H1N1) hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Texas/50/2012 X-223 (H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Influenza B virus B/Massachusetts/2/2012 hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/California/7/2009 (H1N1) live (attenuated) antigen', 'Influenza A virus A/Victoria/361/2011 (H3N2) live (attenuated) antigen', 'Influenza B virus B/Wisconsin/1/2010 live (attenuated) antigen', 'Influenza B virus B/Massachusetts/2/2012 BX-51B hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Brisbane/10/2010 (H1N1) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Texas/50/2012 X-223A (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Massachusetts/2/2012 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Massachusetts/2/2012 BX-51B hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Influenza B virus B/Phuket/3073/2013 hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/California/7/2009 X-181 (H1N1) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza B virus B/Brisbane/9/2014 hemagglutinin antigen (formaldehyde inactivated)', 'Influenza B virus B/Phuket/3073/2013 hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/South Australia/55/2014 IVR-175 (H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Human adenovirus e serotype 4 strain cl-68578 antigen', 'Human adenovirus b serotype 7', 'Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/California/7/2009 X-179A (H1N1) hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Phuket/3073/2013 hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 (H3N2) recombinant hemagglutinin antigen', 'Influenza B virus B/Brisbane/60/2008 recombinant hemagglutinin antigen', 'Influenza A virus A/Christchurch/16/2010 NIB-74XP (H1N1) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/South Australia/55/2014 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Utah/9/2014 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Michigan/45/2015 X-275 (H1N1) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Michigan/45/2015 (H1N1) recombinant hemagglutinin antigen', 'Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Slovenia/2903/2015 (H1N1) live (attenuated) antigen', 'Influenza A virus A/New caledonia/71/2014 (H3N2) live (attenuated) antigen', 'Neisseria meningitidis group a capsular oligosaccharide diphtheria crm197 protein conjugate antigen', 'Neisseria meningitidis group c capsular oligosaccharide diphtheria crm197 protein conjugate antigen', 'Neisseria meningitidis group w-135 capsular oligosaccharide diphtheria crm197 protein conjugate antigen', 'Neisseria meningitidis group y capsular oligosaccharide diphtheria crm197 protein conjugate antigen', 'Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (propiolactone inactivated)', 'Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Singapore/gp2050/2015 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Hong Kong/259/2010 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Brisbane/60/2008 hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Singapore/GP1908/2015 IVR-180A (H1N1) hemagglutinin antigen (propiolactone inactivated)', 'Influenza B virus B/Brisbane/46/2015 hemagglutinin antigen (propiolactone inactivated)', 'Influenza B virus B/Phuket/3073/2013 BVR-1B hemagglutinin antigen (propiolactone inactivated)', 'Vibrio cholerae CVD 103-HgR strain live antigen', 'Influenza A virus A/California/7/2009 (H1N1)-like antigen (propiolactone inactivated)', 'Influenza A virus A/Victoria/210/2009 X-187 (H3N2) antigen (formaldehyde inactivated)', 'Influenza A virus A/South Dakota/6/2007 (H1N1) live (attenuated) antigen', 'Influenza A virus A/Uruguay/716/2007(H3N2) live (attenuated) antigen', 'Influenza A virus A/Perth/16/2009 (H3N2) live (attenuated) antigen', 'Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (formaldehyde inactivated)', 'Influenza B virus B/Texas/6/2011 antigen (formaldehyde inactivated)', 'Influenza A virus A/Texas/50/2012 X-223A (H3N2) antigen (formaldehyde inactivated)', 'Influenza B virus B/Massachusetts/2/2012 antigen (formaldehyde inactivated)', 'Influenza B virus B/Massachusetts/2/2012 BX-51B antigen (formaldehyde inactivated)', 'Influenza A virus A/California/7/2009 X-181 (H1N1) antigen (formaldehyde inactivated)', 'Influenza B virus B/Brisbane/9/2014 antigen (formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) antigen (formaldehyde inactivated)', 'Influenza A virus A/Michigan/45/2015 X-275 (H1N1) antigen (formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) antigen (propiolactone inactivated)', 'Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) antigen (propiolactone inactivated)', 'Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) antigen (formaldehyde inactivated)', 'Influenza A virus A/Singapore/GP1908/2015 IVR-180 (H1N1) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Singapore/GP2050/2015 (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Hong Kong/259/2010 antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Singapore/GP1908/2015 IVR-180 (H1N1) antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Singapore/GP1908/2015 IVR-180A (H1N1) antigen (propiolactone inactivated)', 'Influenza B virus B/Brisbane/46/2015 antigen (propiolactone inactivated)', 'Influenza B virus B/Phuket/3073/2013 BVR-1B antigen (propiolactone inactivated)', 'Beroctocog alfa', 'L-Lactic acid', 'DL-alpha-Tocopherol', 'DL-alpha tocopheryl acetate', 'Povidone K30', 'Calcium phosphate dihydrate', 'Sodium ascorbate', 'Calcium ascorbate', 'Magnesium ascorbate', 'Zinc ascorbate', 'Niacinamide ascorbate', 'Zinc acetate', 'Ferrous gluconate', 'Ferrous succinate', 'Ferrous ascorbate', 'Ferrous fumarate', 'Sodium molybdate', 'Potassium acetate', 'Potassium sulfate', 'Potassium', 'Ferrous glycine sulfate', 'Sodium phosphate, dibasic', 'Sodium phosphate, monobasic, unspecified form', 'Sodium phosphate, dibasic, unspecified form', 'Sodium borate', 'Lithium hydroxide', 'Lithium citrate', 'Lithium carbonate', 'Mometasone furoate', 'Magnesium', 'Aluminium phosphate', 'Aluminum acetate', 'Tetraferric tricitrate decahydrate', 'Chromic nitrate', 'Chromium gluconate', 'Chromium nicotinate', 'Chromous sulfate', 'Zinc chloride', 'Hydrocortisone aceponate', 'Hydrocortisone acetate', 'Hydrocortisone butyrate', 'Hydrocortisone cypionate', 'Hydrocortisone phosphate', 'Hydrocortisone probutate', 'Hydrocortisone valerate', 'Hydrocortisone succinate', 'Zinc sulfate, unspecified form', 'Gallium chloride Ga-67', 'Influenza A virus A/Singapore/INFIMH-16-0019/2016 NIB-104 (H3N2) antigen (formaldehyde inactivated)', 'Influenza B virus B/Maryland/15/2016 BX-69A antigen (formaldehyde inactivated)', 'Influenza A virus A/Singapore/INFIMH-16-0019/2016 IVR-186 (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Maryland/15/2016 BX-69A antigen (UV, formaldehyde inactivated)', 'Andexanet alfa', 'Lavender oil', 'Brown iron oxide', 'Ivosidenib', 'Cobalt', 'Patisiran', 'Segesterone acetate', 'Plantago ovata seed', 'Citrus bioflavonoids', 'Influenza A virus A/Singapore/INFIMH-16-0019/2016 IVR-186 (H3N2) antigen (formaldehyde inactivated)', 'Influenza B virus B/Maryland/15/2016 antigen (formaldehyde inactivated)', 'Influenza A virus A/Singapore/INFIMH-16-0019/2016 IVR-186 (H3N2) antigen (propiolactone inactivated)', 'Influenza B virus B/Maryland/15/2016 antigen (propiolactone inactivated)', 'Influenza A virus A/Singapore/INFIMH-16-0019/2016 (H3N2) recombinant hemagglutinin antigen', 'Influenza B virus B/Maryland/15/2016 recombinant hemagglutinin antigen', 'Loteprednol', 'Lanadelumab', 'Edetate calcium disodium anhydrous', 'Edetate disodium anhydrous', 'Barley malt syrup', 'Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Texas/50/2012 X-223A (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Massachusetts/2/2012 BX-51B antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Texas/50/2012 (H3N2) live (attenuated) antigen', 'Influenza B virus B/Massachusetts/2/2012 live (attenuated) antigen', 'Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Wisconsin/1/2010) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 antigen (formaldehyde inactivated)', 'Influenza A virus A/Victoria/210/2009 X-187 (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/California/7/2009 X-181 (H1N1) antigen (UV, formaldehyde inactivated)', 'Serum, horse', 'Influenza A virus A/Victoria/210/2009 X-187 (H3N2) hemagglutinin antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Uruguay/716/2007(H3N2) antigen (formaldehyde inactivated)', 'Influenza A virus A/Brisbane/59/2007 (H1N1) antigen (formaldehyde inactivated)', 'Influenza A virus A/Uruguay/716/2007(H3N2) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Brisbane/59/2007 (H1N1) hemagglutinin antigen (formaldehyde inactivated)', 'Influenza A virus A/Uruguay/716/2007(H3N2) antigen (propiolactone inactivated)', 'Influenza A virus A/Brisbane/59/2007(H1N1) antigen (propiolactone inactivated)', 'Oxyphenisatin acetate', 'Prednisolone phosphate', 'Prednisolone tebutate', 'Diloxanide furoate', 'Boldenone undecylenate', 'Isoflupredone acetate', 'Lypressin', 'Methylprednisolone aceponate', 'Methylprednisolone hemisuccinate', 'Prednisone acetate', 'Dexamethasone acetate', 'Menadiol diphosphate', 'Drostanolone propionate', 'Chlorphenesin carbamate', 'Paramethasone acetate', 'Chloramphenicol palmitate', 'Trenbolone acetate', 'Betamethasone phosphate', 'Estramustine phosphate', 'Gestonorone caproate', 'Calcium polycarbophil', 'Adenovirus type 7 vaccine live', 'Rosemary oil', 'Ginger oil', 'Willow bark', 'Turmeric oil', 'Atractylodes lancea root', 'Beef liver', 'Influenza A virus A/Singapore/INFIMH-16-0019/2016 (H3N2) live (attenuated) antigen', 'Influenza B virus B/Colorado/06/2017 live (attenuated) antigen', 'Influenza A virus A/Brisbane/59/2007(H1N1) hemagglutinin antigen (UV,formaldehyde inactivated)', 'Influenza A virus A/Uruguay/716/2007(H3N2) hemagglutinin antigen (UV,formaldehyde inactivated)', 'Damoctocog alfa pegol', 'Sodium metavanadate', 'Dexamethasone metasulfobenzoate', 'Cemiplimab', 'Bean', 'Camponotus pennsylvanicus', 'Smallpox (Vaccinia) Vaccine, Live', 'Elapegademase', 'Inotersen', 'Larotrectinib', 'Emapalumab', 'Cefamandole nafate', 'Calaspargase pegol', 'Tagraxofusp', 'Turoctocog alfa pegol', 'Hyaluronidase', 'Polymyxin B', 'Polygala senega root', 'Mecasermin rinfabate', 'Racephedrine', 'Hydroxystilbamidine', 'Solriamfetol', 'Agrostis gigantea top', 'Catfish', 'Red pepper', 'Remdesivir', 'Risankizumab', 'Cycloguanil', 'Dengue tetravalent vaccine, live', 'Diroximel fumarate', 'Isatuximab', 'Ripretinib', 'Filgotinib', 'Brolucizumab', 'Oliceridine', 'Vibegron', 'Inclisiran', 'Flortaucipir F-18', 'Trastuzumab deruxtecan', 'Voxelotor', 'Human interferon beta', 'Viltolarsen', 'Zanubrutinib', 'Tafasitamab', 'Givosiran', 'Volanesorsen', 'Upadacitinib', 'Somapacitan', 'Pemigatinib', 'Ropeginterferon alfa-2b', 'Avapritinib', 'Bulevirtide', 'Imlifidase', 'Crizanlizumab', 'Pertussis vaccine', 'Risdiplam', 'Ubrogepant', 'Elexacaftor', 'Benzhydrocodone', 'Cinoxate', 'Haemagglutinin-strain B (Victoria)', 'Haemagglutinin-strain B (Yamagata)', 'Haemagglutinin-strain A(H3N2)', 'Haemagglutinin-strain A(H1N1)', 'Alloin', 'Frangula purshiana bark', 'Zirconium chloride hydroxide', 'Resorcinol monoacetate', 'Fish liver oil', 'Haemagglutinin-strain B', 'Modified vaccinia ankara', 'VIBRIO CHOLERAE INABA 6973 E1 TOR BIOTYPE, FORMALIN INACTIVATED', 'VIBRIO CHOLERAE INABA 48 CLASSICAL BIOTYPE, HEAT INACTIVATED', 'VIBRIO CHOLERAE OGAWA 50 CLASSICAL BIOTYPE, HEAT INACTIVATED', 'VIBRIO CHOLERAE OGAWA 50 CLASSICAL BIOTYPE, FORMALIN INACTIVATED', 'Edotreotide gallium Ga-68', 'Smallpox and Monkeypox Vaccine, Live, Non-replicating', 'Influenza A virus A/Brisbane/02/2018 IVR-190 (H1N1) antigen (propiolactone inactivated)', 'Influenza A virus A/Kansas/14/2017 X-327 (H3N2) antigen (propiolactone inactivated)', 'Baccharis sarothroides whole', 'Amaranthus hybridus pollen', 'Acacia dealbata pollen', 'Influenza A virus A/Idaho/07/2018 (H1N1) antigen (MDCK cell derived, propiolactone inactivated', 'Influenza A virus A/Indiana/08/2018 (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Singapore/INFTT-16-0610/2016 antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Iowa/06/2017 antigen (MDCK cell derived, propiolactone inactivated)', 'Alnus serrulata pollen', 'Prosopis velutina seed', 'Gibberella fujikuroi', 'Mucor circinelloides f. circinelloides', 'Influenza A virus A/Switzerland/3330/2017 (H1N1) live (attenuated) antigen', 'Influenza A virus A/Kansas/14/2017 (H3N2) live (attenuated) antigen', 'Influenza A virus A/Brisbane/02/2018 IVR-190 (H1N1) antigen (formaldehyde inactivated)', 'Influenza A virus A/Kansas/14/2017 X-327 (H3N2) antigen (formaldehyde inactivated)', 'Influenza A virus A/Brisbane/02/2018 (H1N1) recombinant hemagglutinin antigen', 'Influenza A virus A/Kansas/14/2017 (H3N2) recombinant hemagglutinin antigen', 'Influenza A virus A/Brisbane/02/2018 IVR-190 (H1N1) antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Kansas/14/2017 X-327 (H3N2) antigen (UV, formaldehyde inactivated)', 'Cyd dengue virus serotype 1 live (attenuated) antigen', 'Cyd dengue virus serotype 2 live (attenuated) antigen', 'Cyd dengue virus serotype 3 live (attenuated) antigen', 'Cyd dengue virus serotype 4 live (attenuated) antigen', 'Mucor circinelloides f. lusitanicus', 'Eurotium amstelodami', 'Onasemnogene abeparvovec', 'Leucanthemum vulgare pollen', 'Trifolium pratense pollen', 'Colchiceine', 'Eucalyptus gum', 'Sea scallop', 'Northern brown shrimp', 'Yellowfin tuna', 'Atlantic mackerel', 'Pacific ocean perch', 'Myristica fragrans fruit', 'Pisum sativum whole', 'Skim milk', 'Black sea bass', 'Channel catfish', 'Atlantic halibut', 'Influenza A virus A/North carolina/04/2016 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Iowa/06/2017 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Singapore/INFTT-16-0610/2016 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated)', 'Prednisolone acetate', 'Padeliporfin', 'Ethyl salicylate', 'Cucurbita pepo subsp. ovifera whole', 'Golodirsen', 'Brilliant blue G', 'Ebola Zaire vaccine (live, attenuated)', 'Aminopromazine', 'Aminopentamide', 'Ferric maltol', 'Cashew fruit', 'Coffea arabica seed, roasted', 'Lactuca sativa whole', 'Field mushroom', 'White mustard seed', 'Theobroma cacao whole', 'Muskmelon', 'American hazelnut', 'Musa acuminata fruit', 'Fennel', 'Ferric derisomaltose', 'Influenza B virus B/Washington/02/2019 antigen (formaldehyde inactivated)', 'Influenza A virus A/South Australia/34/2019 IVR-197 (H3N2) antigen (formaldehyde inactivated)', 'Calcium undecylenate', 'Aluminum subacetate', 'Neisseria meningitidis group A capsular polysaccharide tetanus toxoid conjugate antigen', 'Neisseria meningitidis group C capsular polysaccharide tetanus toxoid conjugate antigen', 'Neisseria meningitidis group Y capsular polysaccharide tetanus toxoid conjugate antigen', 'Neisseria meningitidis group W-135 capsular polysaccharide tetanus toxoid conjugate antigen', 'Selpercatinib', 'Fluoroestradiol F-18', 'Cedazuridine', 'Pfizer-BioNTech Covid-19 Vaccine', 'Brexucabtagene autoleucel', 'Influenza A virus A/Hawaii/70/2019 (H1N1) recombinant hemagglutinin antigen', 'Influenza A virus A/Minnesota/41/2019 (H3N2) recombinant hemagglutinin antigen', 'Influenza B virus B/Washington/02/2019 recombinant hemagglutinin antigen', 'Influenza A virus A/Guangdong-Maonan/SWL1536/2019 CNIC-1909 (H1N1) antigen (formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/2671/2019 IVR-208 (H3N2) antigen (formaldehyde inactivated)', 'Influenza A virus A/Victoria/2454/2019 IVR-207 (H1N1) antigen (formaldehyde inactivated)', 'Influenza B virus B/Victoria/705/2018 BVR-11 antigen (formaldehyde inactivated)', 'Influenza A virus A/Nebraska/14/2019 (H1N1) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Delaware/39/2019 (H3N2) antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza B virus B/Darwin/7/2019 antigen (MDCK cell derived, propiolactone inactivated)', 'Influenza A virus A/Hong Kong/2671/2019 NIB-121 (H3N2) antigen (formaldehyde inactivated)', 'Influenza A virus A/Victoria/2454/2019 IVR-207 (H1N1) antigen (propiolactone inactivated)', 'Influenza A virus A/Hong Kong/2671/2019 IVR-208 (H3N2) antigen (propiolactone inactivated)', 'Influenza B virus B/Victoria/705/2018 BVR-11 antigen (propiolactone inactivated)', 'Influenza A virus A/Guangdong-Maonan/SWL1536/2019 CNIC-1909 (H1N1) antigen (UV, formaldehyde inactivated)', 'Influenza A virus A/Hong Kong/2671/2019 NIB-121 (H3N2) antigen (UV, formaldehyde inactivated)', 'Influenza B virus B/Washington/02/2019 antigen (UV, formaldehyde inactivated)', 'Bamlanivimab', 'Belantamab mafodotin', 'Influenza A virus A/Hawaii/66/2019 (H1N1) live (attenuated) antigen', 'Influenza A virus A/Hong Kong/2671/2019 (H3N2) live (attenuated) antigen', 'Influenza B virus B/Washington/02/2019 live (attenuated) antigen', 'Satralizumab', 'Influenza B virus B/Phuket/3073/2013 BVR-1B antigen (formaldehyde inactivated)', 'Zinc phenolsulfonate', 'Boldo leaf extract', 'Pralsetinib', 'Pine tar', 'Copper oxodotreotide Cu-64', 'Agalsidase alfa', 'Haemophilus influenzae b polysaccharide-tetanus conjugate', 'Atoltivimab', 'Maftivimab', 'Odesivimab', 'Acetylene', 'Ammonium carbonate', 'Ferric glycerophosphate', 'Lumasiran', 'Influenza virus type A (H1N1)', 'Influenza virus type A (H3N2)', 'Influenza virus type B (Victoria)', 'Influenza virus type B (Yamagata)', 'Sodium camphorsulfonate', 'Naxitamab', 'Influenza virus type B', 'Berotralstat', 'Cinnamon oil', 'Rurioctocog alfa pegol', 'Gallium Ga-68 gozetotide', 'Thenyldiamine', 'Ansuvimab'])\n"
     ]
    }
   ],
   "source": [
    "# input data 결정\n",
    "print(len(drugbank_approved_df))\n",
    "print(len(documents))\n",
    "\n",
    "doc_list = documents.keys()\n",
    "print(doc_list)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* 전처리 파트 ========================================================================================="
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 소문자 통일\n",
    "for key in documents.keys():\n",
    "    documents[key] = documents[key].lower()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 172,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 의약품 용어 통일 - 단어내 띄어쓰기 처리\n",
    "# 결과에 크게 영향을 미치지 않을 것으로 예상하고 일단 SKip\n",
    "# documents['Dornase alfa'] = documents['Dornase alfa'].replace('dornase alfa', 'dornase-alfa')\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\paten\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 최초 1회 수행\n",
    "# import nltk\n",
    "# nltk.download('stopwords')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Remove stopwords\n",
    "from nltk.corpus import stopwords\n",
    "\n",
    "# stopwords for english\n",
    "listStopWords = stopwords.words('english') \n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# Tokenize (소문자로 통합)\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.tokenize import WordPunctTokenizer  \n",
    "from nltk.tokenize import TreebankWordTokenizer\n",
    "# token_Dornasealf = word_tokenize(doc_Dornasealf)\n",
    "# token_Lepirudin = word_tokenize(doc_Lepirudin)\n",
    "# toekn_Ximelagatr = word_tokenize(doc_Ximelagatr)\n",
    "\n",
    "# token_Dornasealf = WordPunctTokenizer().tokenize(doc_Dornasealf)\n",
    "# token_Lepirudin = WordPunctTokenizer().tokenize(doc_Lepirudin)\n",
    "# toekn_Ximelagatr = WordPunctTokenizer().tokenize(doc_Ximelagatr)\n",
    "\n",
    "\n",
    "tokenizer = TreebankWordTokenizer()\n",
    "token_doc = dict()\n",
    "for key in documents.keys():\n",
    "     token_doc[key] = tokenizer.tokenize(documents[key])\n",
    "\n",
    "# token_Dornasealf = tokenizer.tokenize(doc_Dornasealf)\n",
    "# token_Lepirudin = tokenizer.tokenize(doc_Lepirudin)\n",
    "# toekn_Ximelagatr = tokenizer.tokenize(doc_Ximelagatr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [],
   "source": [
    "# token_doc =  [token_Dornasealf, token_Lepirudin, toekn_Ximelagatr]\n",
    "#Stopwords 제거\n",
    "token_wo_stopword = dict()\n",
    "token_wo_stopword_list = [[word for word in token_doc[key] if not word in listStopWords] for key in token_doc.keys()]\n",
    "\n",
    "drug_keys = token_doc.keys()\n",
    "for i, key in enumerate(drug_keys):\n",
    "    token_wo_stopword[key] = token_wo_stopword_list[i]\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Stemming(어간 추출)\n",
    "from nltk.stem import PorterStemmer\n",
    "\n",
    "pst = PorterStemmer()\n",
    "token_stemmed = [[pst.stem(word) for word in token_wo_stopword[key]] for key in token_wo_stopword.keys()]\n",
    "\n",
    "drug_keys = token_wo_stopword.keys()\n",
    "token_result = dict()\n",
    "for i, key in enumerate(drug_keys):\n",
    "    token_result[key] = token_stemmed[i]\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* 전처리 파트 ========================================================================================="
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.cluster import KMeans\n",
    "from gensim.models.doc2vec import Doc2Vec, TaggedDocument\n",
    "\n",
    "texts_and_tags = [\n",
    "    (documents[key], [key]) for key in documents.keys()\n",
    "]\n",
    "\n",
    "TRAIN_documents = [TaggedDocument(words=text, tags=tags) for text, tags in texts_and_tags]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "running time:  267.9928960800171\n"
     ]
    }
   ],
   "source": [
    "import time\n",
    "\n",
    "start_time = time.time()\n",
    "\n",
    "# Doc2Vec Training\n",
    "model = Doc2Vec(TRAIN_documents, vector_size=5, window=3, epochs=40, min_count=0, workers=4)\n",
    "\n",
    "print(\"running time: \", time.time() - start_time)\n",
    "model.save('doc2vec_approved.model')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('Eteplirsen', 0.9998279809951782),\n",
       " ('Everolimus', 0.999125063419342),\n",
       " ('Felis catus skin', 0.9987452626228333),\n",
       " ('Icosapent ethyl', 0.9983333945274353),\n",
       " ('Citicoline', 0.997514545917511),\n",
       " ('Dactinomycin', 0.9970820546150208),\n",
       " ('Levobunolol', 0.997021496295929),\n",
       " ('Avelumab', 0.9969208240509033),\n",
       " ('Netupitant', 0.9966803789138794),\n",
       " ('Tobramycin', 0.9965447187423706)]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.docvecs.most_similar('Lepirudin')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dornase_Lepirudin:  0.03801298141479492\n",
      "Dornase_Ximelagatran:  1.1024495959281921\n",
      "Lepirudin_Ximelagatran:  0.9245570600032806\n"
     ]
    }
   ],
   "source": [
    "from scipy import spatial\n",
    "\n",
    "dor_lepi = spatial.distance.cosine(model.docvecs['Dornase alfa'], model.docvecs['Lepirudin'])\n",
    "dor_xime = spatial.distance.cosine(model.docvecs['Dornase alfa'], model.docvecs['Ximelagatran'])\n",
    "lepi_xime = spatial.distance.cosine(model.docvecs['Lepirudin'], model.docvecs['Ximelagatran'])\n",
    "\n",
    "print('Dornase_Lepirudin: ', dor_lepi)\n",
    "print('Dornase_Ximelagatran: ', dor_xime)\n",
    "print('Lepirudin_Ximelagatran: ', lepi_xime)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "== word similarity\n",
      "[('﹤', 0.8980783224105835), ('･', 0.8068166971206665), ('⟶', 0.8016313910484314), ('̂', 0.7972032427787781), ('ⅹ', 0.7858506441116333), ('Ρ', 0.7693530321121216), ('͘', 0.7562497854232788), ('ű', 0.7532801032066345), ('·', 0.7370501160621643), ('w', 0.7352657318115234)]\n",
      "== document similarity\n",
      "Document (2S)-2-(1H-indol-3-yl)pentanoic acid - similarity: 0.98621\n",
      "Document Benorilate - similarity: 0.98586\n",
      "Document Maclura tricuspidata whole - similarity: 0.98072\n",
      "Document (5Z)-12-CHLORO-13,15-DIHYDROXY-4,7,8,9-TETRAHYDRO-2-BENZOXACYCLOTRIDECINE-1,10(3H,11H)-DIONE - similarity: 0.98010\n",
      "Document N-((1R,2S)-2-(5-CHLORO-1H-INDOLE-2-CARBOXAMIDO)CYCLOHEXYL)-5-METHYL-4,5,6,7-TETRAHYDROTHIAZOLO[5,4-C]PYRIDINE-2-CARBOXAMIDE - similarity: 0.97496\n",
      "Document 5-(HEXAHYDRO-2-OXO-1H-THIENO[3,4-D]IMIDAZOL-6-YL)PENTANAL - similarity: 0.97391\n",
      "Document Caprylic alcohol - similarity: 0.97388\n",
      "Document 1-[2-DEOXYRIBOFURANOSYL]-2,4-DIFLUORO-5-METHYL-BENZENE-5'MONOPHOSPHATE - similarity: 0.96908\n",
      "Document 6-(3-BROMO-2-NAPHTHYL)-1,3,5-TRIAZINE-2,4-DIAMINE - similarity: 0.96676\n",
      "Document (1R,2R,3R,4S,5R)-4-(BENZYLAMINO)-5-(METHYLTHIO)CYCLOPENTANE-1,2,3-TRIOL - similarity: 0.96562\n",
      "Document Methylbenactyzium - similarity: 0.96447\n",
      "Document CPD 923 - similarity: 0.96394\n",
      "Document (1S)-1-CYCLOPROPYL-2-[(2S)-4-(2,5-DIFLUOROPHENYL)-2-PHENYL-2,5-DIHYDRO-1H-PYRROL-1-YL]-2-OXOETHANAMINE - similarity: 0.95858\n",
      "Document Fluorescin - similarity: 0.95790\n",
      "Document 7-amino-2-tert-butyl-4-(4-pyrimidin-2-ylpiperazin-1-yl)pyrido[2,3-d]pyrimidine-6-carboxamide - similarity: 0.95756\n",
      "Document 8-bromo-4-(2-chlorophenyl)-N-(2-hydroxyethyl)-6-methyl-1,3-dioxo-1,2,3,6-tetrahydropyrrolo[3,4-e]indole-7-carboxamide - similarity: 0.95546\n",
      "Document Neisseria meningitidis group w-135 capsular polysaccharide diphtheria toxoid conjugate antigen - similarity: 0.95022\n",
      "Document Emeramide - similarity: 0.94897\n",
      "Document PF-03635659 - similarity: 0.94892\n",
      "Document Merotocin - similarity: 0.94594\n",
      "Document 2-(4-ETHYLPIPERAZIN-1-YL)-4-(PHENYLAMINO)PYRAZOLO[1,5-A][1,3,5]TRIAZINE-8-CARBONITRILE - similarity: 0.94571\n",
      "Document phenyl ethenesulfonate - similarity: 0.94441\n",
      "Document Salix nigra pollen - similarity: 0.94421\n",
      "Document O-[2-(1,3-dioxo-1,3-dihydro-2H-isoindol-2-yl)ethyl] (4-chlorophenyl)thiocarbamate - similarity: 0.94356\n",
      "Document Phalaris minor pollen - similarity: 0.94293\n",
      "Document Amperozide - similarity: 0.94173\n",
      "Document Quercus agrifolia pollen - similarity: 0.94173\n",
      "Document Somatorelin - similarity: 0.94036\n",
      "Document 1-(PHENYLMETHYL)CYCLOPENTYL[(1S)-1-FORMYLPENTYL]CARBAMATE - similarity: 0.93950\n",
      "Document Ambrosia deltoidea pollen - similarity: 0.93949\n",
      "Document Pinus virginiana pollen - similarity: 0.93756\n",
      "Document Hexapropymate - similarity: 0.93738\n",
      "Document Streptococcus pneumoniae type 20 capsular polysaccharide antigen - similarity: 0.93601\n",
      "Document Streptococcus pneumoniae type 19f capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.93436\n",
      "Document Quercus virginiana pollen - similarity: 0.93401\n",
      "Document N-[(BENZYLOXY)CARBONYL]-L-CYSTEINYLGLYCINE - similarity: 0.93301\n",
      "Document 1-((2-HYDROXYETHOXY)METHYL)-5-(3-(BENZYLOXY)BENZYL)PYRIMIDINE-2,4(1H,3H)-DIONE - similarity: 0.93139\n",
      "Document 6-CHLORO-4-(CYCLOHEXYLOXY)-3-ISOPROPYLQUINOLIN-2(1H)-ONE - similarity: 0.93138\n",
      "Document Oxidronic acid - similarity: 0.92966\n",
      "Document Influenza A virus A/Texas/50/2012 X-223 (H3N2) antigen (propiolactone inactivated) - similarity: 0.92875\n",
      "Document Quinbolone - similarity: 0.92796\n",
      "Document 3-(1-Aminoethyl)Nonanedioic Acid - similarity: 0.92719\n",
      "Document 4-methyl-7,8-dihydro-5H-thiopyrano[4,3-d]pyrimidin-2-amine - similarity: 0.92691\n",
      "Document (5R)-2-[(2-Fluorophenyl)amino]-5-isopropyl-1,3-thiazol-4(5H)-one - similarity: 0.92629\n",
      "Document 4-[(1S,2R,5S)-4,4,8-TRIMETHYL-3-OXABICYCLO[3.3.1]NON-7-EN-2-YL]PHENOL - similarity: 0.92615\n",
      "Document N-Succinyl Phenylglycine - similarity: 0.92539\n",
      "Document Aureobasidium pullulans var. pullutans - similarity: 0.92491\n",
      "Document 1-Naphthylamine-5-sulfonic acid - similarity: 0.92458\n",
      "Document 2-(1,3-benzodioxol-5-yl)-5-[(3-fluoro-4-methoxybenzyl)sulfanyl]-1,3,4-oxadiazole - similarity: 0.92341\n",
      "Document Epomediol - similarity: 0.92274\n",
      "Document Ibuproxam - similarity: 0.92247\n",
      "Document 1-{[(1E)-(3-HYDROXY-2-METHYL-5-{[(TRIHYDROXY-LAMBDA^5^-PHOSPHANYL)OXY]METHYL}PYRIDIN-4-YL)METHYLIDENE]AMINO}UNDECAN-2-ONE - similarity: 0.92163\n",
      "Document Pleospora herbarum - similarity: 0.92059\n",
      "Document Hydrocortisone phosphate - similarity: 0.91959\n",
      "Document 1,1,1-TRIFLUORO-3-((N-ACETYL)-L-LEUCYLAMIDO)-4-PHENYL-BUTAN-2-ONE(N-ACETYL-L-LEUCYL-L-PHENYLALANYL TRIFLUOROMETHYL KETONE) - similarity: 0.91827\n",
      "Document 2-((3'-TERTBUTYL-4'-HYDROXYPHENYL)AZO)BENZOIC ACID - similarity: 0.91817\n",
      "Document Dimetacrine - similarity: 0.91722\n",
      "Document 2,6-dimethyl-4-[(E)-2-phenylethenyl]phenol - similarity: 0.91669\n",
      "Document (S)-5-(7-(4-(4-Ethyl-4,5-dihydro-2-oxazolyl)phenoxy)heptyl)-3-methylisoxazole - similarity: 0.91560\n",
      "Document N-ISOBUTYL-N-[4-METHOXYPHENYLSULFONYL]GLYCYL HYDROXAMIC ACID - similarity: 0.91551\n",
      "Document ZW-25 - similarity: 0.91514\n",
      "Document Corylus americana pollen - similarity: 0.91440\n",
      "Document Etafedrine - similarity: 0.91382\n",
      "Document AZD-7295 - similarity: 0.91362\n",
      "Document N-[4-(2-CHLOROPHENYL)-1,3-DIOXO-1,2,3,6-TETRAHYDROPYRROLO[3,4-C]CARBAZOL-9-YL]FORMAMIDE - similarity: 0.91336\n",
      "Document Stibophen - similarity: 0.91180\n",
      "Document Streptococcus pneumoniae type 3 capsular polysaccharide antigen - similarity: 0.91107\n",
      "Document (2R)-2-AMINO-3,3,3-TRIFLUORO-N-HYDROXY-2-{[(4-PHENOXYPHENYL)SULFONYL]METHYL}PROPANAMIDE - similarity: 0.91102\n",
      "Document Burixafor - similarity: 0.91038\n",
      "Document Tenonitrozole - similarity: 0.90974\n",
      "Document Mimopezil - similarity: 0.90954\n",
      "Document Aleplasinin - similarity: 0.90924\n",
      "Document (2R)-3-(phosphonooxy)propane-1,2-diyl diheptanoate - similarity: 0.90861\n",
      "Document 2-(1H-pyrrol-1-ylcarbonyl)benzene-1,3,5-triol - similarity: 0.90800\n",
      "Document ABL-001 - similarity: 0.90782\n",
      "Document N-Acetyl-L-Citrulline - similarity: 0.90688\n",
      "Document 4-[(5-methoxy-2-methylphenoxy)methyl]pyridine - similarity: 0.90667\n",
      "Document Ceforanide - similarity: 0.90666\n",
      "Document Ferrous aspartate - similarity: 0.90565\n",
      "Document {1-[(3-Hydroxy-Methyl-5-Phosphonooxy-Methyl-Pyridin-4-Ylmethyl)-Amino]-Ethyl}-Phosphonic Acid - similarity: 0.90411\n",
      "Document 3'-THIO-THYMIDINE-5'-PHOSPHATE - similarity: 0.90380\n",
      "Document Nalpha-(2-Naphthylsulfonylglycyl)-3-Amidino-D,L-Phenylalanine-Isopropylester - similarity: 0.90348\n",
      "Document Prostalene - similarity: 0.90276\n",
      "Document MK-0777 - similarity: 0.90267\n",
      "Document 2-(methylsulfanyl)-5-(thiophen-2-ylmethyl)-1H-imidazol-4-ol - similarity: 0.90252\n",
      "Document Azidocillin - similarity: 0.90163\n",
      "Document N-Carboxymethionine - similarity: 0.90161\n",
      "Document Influenza B virus B/Utah/9/2014 antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.90140\n",
      "Document MPI-674 - similarity: 0.90096\n",
      "Document Nagrestipen - similarity: 0.90065\n",
      "Document 3-DIPHENOL-6-NITRO-3H-BENZO[DE]ISOCHROMEN-1-ONE - similarity: 0.90018\n",
      "Document BI-K0376 - similarity: 0.90013\n",
      "Document Swertia japonica whole flowering - similarity: 0.89979\n",
      "Document Streptococcus pneumoniae type 19a capsular polysaccharide antigen - similarity: 0.89952\n",
      "Document (2S)-2-AMINO-1-(5-TERT-BUTYL-1,3,4-OXADIAZOL-2-YL)PROPAN-1-ONE - similarity: 0.89942\n",
      "Document LY-3039478 - similarity: 0.89751\n",
      "Document 4-({[(4-METHYLPIPERAZIN-1-YL)AMINO]CARBONOTHIOYL}AMINO)BENZENESULFONAMIDE - similarity: 0.89712\n",
      "Document (1R,3R)-5-[(2E)-3-{(1S,3R)-2,2,3-trimethyl-3-[6,6,6-trifluoro-5-hydroxy-5-(trifluoromethyl)hex-3-yn-1-yl]cyclopentyl}prop-2-en-1-ylidene]cyclohexane-1,3-diol - similarity: 0.89649\n",
      "Document Agatolimod - similarity: 0.89647\n",
      "Document Mucor circinelloides f. lusitanicus - similarity: 0.89554\n",
      "Document Coleneuramide - similarity: 0.89481\n",
      "Document Sp-722 - similarity: 0.89478\n",
      "Document Aceclidine - similarity: 0.89389\n",
      "Document (3R)-3-hydroxy-2,2-dimethyl-4-oxo-4-({3-oxo-3-[(2-sulfanylethyl)amino]propyl}amino)butyl 2,2-dimethylpropanoate - similarity: 0.89387\n",
      "Document RP01 - similarity: 0.89303\n",
      "Document Dexelvucitabine - similarity: 0.89300\n",
      "Document Ruplizumab - similarity: 0.89234\n",
      "Document Aluminum acetate - similarity: 0.89174\n",
      "Document (1r)-4-[(1e,3e,5e,7z,9e,11z,13e,15e)-17-Hydroxy-3,7,12,16-Tetramethylheptadeca-1,3,5,7,9,11,13,15-Octaen-1-Yl]-3,5,5-Trimethylcyclohex-3-En-1-Ol - similarity: 0.89168\n",
      "Document AZD-2066 - similarity: 0.89161\n",
      "Document Cirazoline - similarity: 0.89121\n",
      "Document Benzoctamine - similarity: 0.89114\n",
      "Document Seliforant - similarity: 0.89103\n",
      "Document Fraxinus americana pollen - similarity: 0.89063\n",
      "Document CYCLOHEXYL-NORSTATINE - similarity: 0.88787\n",
      "Document 2-(3'-Methoxyphenyl) Benzimidazole-4-Carboxamide - similarity: 0.88753\n",
      "Document N-cyclohexyltaurine - similarity: 0.88670\n",
      "Document N-[(4-HYDROXY-8-IODOISOQUINOLIN-3-YL)CARBONYL]GLYCINE - similarity: 0.88559\n",
      "Document Medifoxamine - similarity: 0.88452\n",
      "Document 2',5'-DIDEOXY-ADENOSINE 3'-MONOPHOSPHATE - similarity: 0.88244\n",
      "Document Bambermycins - similarity: 0.88200\n",
      "Document Corticorelin ovine triflutate - similarity: 0.88172\n",
      "Document 1-[1-(3-aminophenyl)-3-tert-butyl-1H-pyrazol-5-yl]-3-phenylurea - similarity: 0.88117\n",
      "Document 2-((3'-METHYL-4'-HYDROXYPHENYL)AZO)BENZOIC ACID - similarity: 0.88078\n",
      "Document Foreskin fibroblast (neonatal) - similarity: 0.88064\n",
      "Document Equine Botulinum Neurotoxin C Immune FAB2 - similarity: 0.87977\n",
      "Document R-95845 - similarity: 0.87971\n",
      "Document ethyl 2-amino-4-hydroxypyrimidine-5-carboxylate - similarity: 0.87920\n",
      "Document Alnus rubra pollen - similarity: 0.87899\n",
      "Document Fraxinus pennsylvanica pollen - similarity: 0.87851\n",
      "Document Chlorophetanol - similarity: 0.87837\n",
      "Document Periciazine - similarity: 0.87827\n",
      "Document (5R)-4-HYDROXY-3,5-DIMETHYL-5-[(1E,3E)-2-METHYLPENTA-1,3-DIENYL]THIOPHEN-2(5H)-ONE - similarity: 0.87757\n",
      "Document Bimiralisib - similarity: 0.87707\n",
      "Document GSK-424887 - similarity: 0.87604\n",
      "Document 3-{ISOPROPYL[(TRANS-4-METHYLCYCLOHEXYL)CARBONYL]AMINO}-5-PHENYLTHIOPHENE-2-CARBOXYLIC ACID - similarity: 0.87453\n",
      "Document Peg-100 stearate - similarity: 0.87395\n",
      "Document Epetraborole - similarity: 0.87312\n",
      "Document Carya ovata pollen - similarity: 0.87304\n",
      "Document Alizapride - similarity: 0.87251\n",
      "Document 2,2-Dichloro-1-methanesulfinyl-3-methyl-cyclopropanecarboxylic acid [1-(4-bromo-phenyl)-ethyl]-amide - similarity: 0.87239\n",
      "Document Vanillyl butyl ether - similarity: 0.87180\n",
      "Document 2-(2-Hydroxy-5-Methoxy-Phenyl)-1h-Benzoimidazole-5-Carboxamidine - similarity: 0.87135\n",
      "Document Oxidized coenzyme A - similarity: 0.87117\n",
      "Document Edetate sodium - similarity: 0.87090\n",
      "Document Rocacetrapib - similarity: 0.87058\n",
      "Document N-(4-carbamimidoylbenzyl)-1-(4-methylpentanoyl)-L-prolinamide - similarity: 0.87050\n",
      "Document (1R,2R,3S,4R,6S)-3,4,6-Trihydroxy-5-{[(S)-hydroxy(3-hydroxy-2-oxopropoxy)phosphoryl]oxy}-1,2-cyclohexanediyl bis[dihydrogen (phosphate)] - similarity: 0.87039\n",
      "Document Streptococcus pneumoniae type 4 capsular polysaccharide antigen - similarity: 0.86987\n",
      "Document Nicomorphine - similarity: 0.86974\n",
      "Document Doxefazepam - similarity: 0.86953\n",
      "Document Thienylfentanyl - similarity: 0.86942\n",
      "Document Z-160 - similarity: 0.86936\n",
      "Document Clonixin - similarity: 0.86890\n",
      "Document Pearl (hyriopsis cumingii) - similarity: 0.86873\n",
      "Document (3Z,6S)-6-Chloro-1-(2-{[(5-chloro-1-benzothiophen-3-yl)methyl]amino}ethyl)-3-({2-[(2R)-2-piperidinyl]ethyl}imino)-2-piperazinol - similarity: 0.86840\n",
      "Document [[(3R,4R,5S,6R)-3-(butanoylamino)-4,5-dihydroxy-6-(hydroxymethyl)oxan-2-ylidene]amino] N-phenylcarbamate - similarity: 0.86788\n",
      "Document 2-(4-HYDROXY-3-NITROPHENYL)ACETIC ACID - similarity: 0.86698\n",
      "Document Briakinumab - similarity: 0.86652\n",
      "Document Meriones unguiculatus skin - similarity: 0.86596\n",
      "Document 4-[3-(4-FLUOROPHENYL)-1H-PYRAZOL-4-YL]PYRIDINE - similarity: 0.86573\n",
      "Document Acacia baileyana pollen - similarity: 0.86567\n",
      "Document (3Z)-N,N-DIMETHYL-2-OXO-3-(4,5,6,7-TETRAHYDRO-1H-INDOL-2-YLMETHYLIDENE)-2,3-DIHYDRO-1H-INDOLE-5-SULFONAMIDE - similarity: 0.86557\n",
      "Document SGS-742 - similarity: 0.86535\n",
      "Document (3R,4S)-1-[(4-amino-5H-pyrrolo[3,2-d]pyrimidin-7-yl)methyl]-4-[(butylsulfanyl)methyl]pyrrolidin-3-ol - similarity: 0.86441\n",
      "Document Carbromal - similarity: 0.86433\n",
      "Document Dimethylaminopropionylphenothiazine - similarity: 0.86426\n",
      "Document (4Z)-6-bromo-4-({[4-(pyrrolidin-1-ylmethyl)phenyl]amino}methylidene)isoquinoline-1,3(2H,4H)-dione - similarity: 0.86380\n",
      "Document Alclofenac - similarity: 0.86375\n",
      "Document Serlopitant - similarity: 0.86303\n",
      "Document ginkgolide-J - similarity: 0.86215\n",
      "Document Iodide I-131 - similarity: 0.86193\n",
      "Document 1-[2-(S)-AMINO-3-BIPHENYL-4-YL-PROPIONYL]-PYRROLIDINE-2-(S)-CARBONITRILE - similarity: 0.86174\n",
      "Document Methoserpidine - similarity: 0.86142\n",
      "Document Influenza A virus A/Indiana/08/2018 (H3N2) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.86123\n",
      "Document Sutilain - similarity: 0.86092\n",
      "Document 4-tert-butyl-N'-[(1E)-(3,5-dibromo-2,4-dihydroxyphenyl)methylidene]benzohydrazide - similarity: 0.86016\n",
      "Document (R)-1-(4-(4-(Hydroxymethyl)-1,3,2-dioxaborolan-2-YL)phenyl)guanidine - similarity: 0.85979\n",
      "Document (3R)-3-[(1,2,3,4-tetrahydroisoquinolin-7-yloxy)methyl]-2,3-dihydrothieno[2,3-f][1,4]oxazepin-5-amine - similarity: 0.85918\n",
      "Document 2-(Thiomethylene)-4-Methylpentanoic Acid - similarity: 0.85760\n",
      "Document BENZYL 6-BENZYL-5,7-DIOXO-6,7-DIHYDRO-5H-[1,3]THIAZOLO[3,2-C]PYRIMIDINE-2-CARBOXYLATE - similarity: 0.85742\n",
      "Document Bamet-UD2 - similarity: 0.85714\n",
      "Document Fluocortin - similarity: 0.85569\n",
      "Document 3-(2,6-difluorophenyl)-2-(methylthio)quinazolin-4(3H)-one - similarity: 0.85560\n",
      "Document Beta vulgaris pollen - similarity: 0.85542\n",
      "Document Proquazone - similarity: 0.85455\n",
      "Document 4-[(4-Imidazo[1,2-a]Pyridin-3-Ylpyrimidin-2-Yl)Amino]Benzenesulfonamide - similarity: 0.85324\n",
      "Document Inarigivir soproxil - similarity: 0.85216\n",
      "Document 3-{[(4-methylphenyl)sulfonyl]amino}propyl pyridin-4-ylcarbamate - similarity: 0.85215\n",
      "Document Cidoxepin - similarity: 0.85190\n",
      "Document Helianthus annuus pollen - similarity: 0.85093\n",
      "Document Mardepodect - similarity: 0.85066\n",
      "Document (4R)-4-(3-butoxy-4-methoxybenzyl)imidazolidin-2-one - similarity: 0.85053\n",
      "Document VSF-173 - similarity: 0.85042\n",
      "Document (2S)-1-[4-({4-[(2,5-Dichlorophenyl)amino]-2-pyrimidinyl}amino)phenoxy]-3-(dimethylamino)-2-propanol - similarity: 0.84973\n",
      "Document Aranidipine - similarity: 0.84871\n",
      "Document (3R,4S)-1-[(4-amino-5H-pyrrolo[3,2-d]pyrimidin-7-yl)methyl]-4-[(benzylsulfanyl)methyl]pyrrolidin-3-ol - similarity: 0.84861\n",
      "Document 5-(5-CHLORO-2,4-DIHYDROXYPHENYL)-N-ETHYL-4-(4-METHOXYPHENYL)-1H-PYRAZOLE-3-CARBOXAMIDE - similarity: 0.84855\n",
      "Document Tedatioxetine - similarity: 0.84848\n",
      "Document Bambusa vulgaris top - similarity: 0.84844\n",
      "Document (3,3-dimethylpiperidin-1-yl)(6-(3-fluoro-4-methylphenyl)pyridin-2-yl)methanone - similarity: 0.84803\n",
      "Document Mitazalimab - similarity: 0.84772\n",
      "Document Methyl undecenoyl leucinate - similarity: 0.84749\n",
      "Document Betula pendula tar oil - similarity: 0.84748\n",
      "Document AZD-1236 - similarity: 0.84732\n",
      "Document Ecamsule - similarity: 0.84686\n",
      "Document N-oxo-2-[(4-phenylphenyl)sulfonylamino]ethanamide - similarity: 0.84541\n",
      "Document Falnidamol - similarity: 0.84532\n",
      "Document 3-fluoro-N-[3-(1H-tetrazol-5-yl)phenyl]benzamide - similarity: 0.84518\n",
      "Document Mirfentanil - similarity: 0.84514\n",
      "Document Diphenadione - similarity: 0.84506\n",
      "Document MK-1454 - similarity: 0.84474\n",
      "Document Fluprednidene - similarity: 0.84455\n",
      "Document Decyl formate - similarity: 0.84414\n",
      "Document L-aminocarnityl-succinyl-leucyl-argininal-diethylacetal - similarity: 0.84400\n",
      "Document CP-547632 - similarity: 0.84351\n",
      "Document N'-(6-aminopyridin-3-yl)-N-(2-cyclopentylethyl)-4-methyl-benzene-1,3-dicarboxamide - similarity: 0.84343\n",
      "Document Interferon alfa-2c - similarity: 0.84321\n",
      "Document Melaleuca quinquenervia pollen - similarity: 0.84319\n",
      "Document GW-870086 - similarity: 0.84309\n",
      "Document Etrasimod - similarity: 0.84288\n",
      "Document PMP-hydroxyisoxazole, pyridoxamine-5-phosphate-hydroxyisoxazole - similarity: 0.84249\n",
      "Document AER001 - similarity: 0.84220\n",
      "Document Dexamethasone isonicotinate - similarity: 0.84181\n",
      "Document LX-2931 - similarity: 0.84107\n",
      "Document Pegpleranib - similarity: 0.83981\n",
      "Document N~4~-methyl-N~4~-(3-methyl-1H-indazol-6-yl)-N~2~-(3,4,5-trimethoxyphenyl)pyrimidine-2,4-diamine - similarity: 0.83975\n",
      "Document [2(Formyl-Hydroxy-Amino)-Ethyl]-Phosphonic Acid - similarity: 0.83974\n",
      "Document 1-(2-{[(6-amino-2-methylpyridin-3-yl)methyl]amino}ethyl)-6-chloro-3-[(2,2-difluoro-2-pyridin-2-ylethyl)amino]-1,4-dihydropyrazin-2-ol - similarity: 0.83949\n",
      "Document Elismetrep - similarity: 0.83931\n",
      "Document Streptococcus pneumoniae type 5 capsular polysaccharide antigen - similarity: 0.83912\n",
      "Document (S)-2-METHYL-1-[(4-METHYL-5-ISOQUINOLINE)SULFONYL]-HOMOPIPERAZINE - similarity: 0.83907\n",
      "Document 6-methyl-5-[3-methyl-3-(3,4,5-trimethoxyphenyl)but-1-yn-1-yl]pyrimidine-2,4-diamine - similarity: 0.83883\n",
      "Document Bacillus calmette-guerin substrain connaught live antigen - similarity: 0.83834\n",
      "Document 6-phenyl-4(R)-(7-phenyl-heptanoylamino)-hexanoic acid - similarity: 0.83833\n",
      "Document Larix occidentalis pollen - similarity: 0.83767\n",
      "Document Cefaloridine - similarity: 0.83767\n",
      "Document 4-{[5-(CYCLOHEXYLAMINO)[1,2,4]TRIAZOLO[1,5-A]PYRIMIDIN-7-YL]AMINO}BENZENESULFONAMIDE - similarity: 0.83733\n",
      "Document (2S,3S)-4-cyclopropyl-3-{(3R,5R)-3-[2-fluoro-4-(methylsulfonyl)phenyl]-1,2,4-oxadiazolidin-5-yl}-1-[(3S)-3-fluoropyrrolidin-1-yl]-1-oxobutan-2-amine - similarity: 0.83692\n",
      "Document Diridavumab - similarity: 0.83691\n",
      "Document (2S,3R,4S,5S)-3,4-Dihydroxy-2-[(methylsulfanyl)methyl]-5-(4-oxo-4,5-dihydro-1H-pyrrolo[3,2-d]pyrimidin-7-yl)pyrrolidinium - similarity: 0.83682\n",
      "Document Artemisia californica pollen - similarity: 0.83642\n",
      "Document Cetrelimab - similarity: 0.83628\n",
      "Document [4-({4-[(5-cyclopropyl-1H-pyrazol-3-yl)amino]-6-(methylamino)pyrimidin-2-yl}amino)phenyl]acetonitrile - similarity: 0.83624\n",
      "Document Carbomer homopolymer type C - similarity: 0.83622\n",
      "Document 4-{2,4-Bis[(3-Nitrobenzoyl)Amino]Phenoxy}Phthalic Acid - similarity: 0.83619\n",
      "Document Urea C-14 - similarity: 0.83583\n",
      "Document Rilematovir - similarity: 0.83582\n",
      "Document Influenza A virus A/Victoria/361/2011 (H3N2) live (attenuated) antigen - similarity: 0.83581\n",
      "Document Ibopamine - similarity: 0.83565\n",
      "Document ALPHA-(2,6-DICHLOROPHENYL)-ALPHA-(2-ACETYL-5-METHYLANILINO)ACETAMIDE - similarity: 0.83561\n",
      "Document N-{3-[(E)-(tert-butoxyimino)methyl]-4-chlorophenyl}-2-methylfuran-3-carbimidothioic acid - similarity: 0.83549\n",
      "Document 2-{2-[(3,5-dimethylphenyl)amino]pyrimidin-4-yl}-N-[(1S)-2-hydroxy-1-methylethyl]-4-methyl-1,3-thiazole-5-carboxamide - similarity: 0.83544\n",
      "Document Streptococcus pneumoniae type 19f capsular polysaccharide antigen - similarity: 0.83526\n",
      "Document 1-DECYL-3-TRIFLUORO ETHYL-SN-GLYCERO-2-PHOSPHOMETHANOL - similarity: 0.83490\n",
      "Document DG031 - similarity: 0.83454\n",
      "Document Tyropanoic acid - similarity: 0.83430\n",
      "Document Ferrous cysteine glycinate - similarity: 0.83428\n",
      "Document VLTS-589 - similarity: 0.83380\n",
      "Document Bis(5-Amidino-2-Benzimidazolyl)Methane Ketone Hydrate - similarity: 0.83370\n",
      "Document N-(2-AMINOETHYL)-P-CHLOROBENZAMIDE - similarity: 0.83365\n",
      "Document 5-(3-HYDROXYPHENYL)ISOTHIAZOL-3(2H)-ONE 1,1-DIOXIDE - similarity: 0.83336\n",
      "Document 3-({[(1Z)-(2-methoxyphenyl)methylidene]amino}oxy)propanoic acid - similarity: 0.83336\n",
      "Document Demoxytocin - similarity: 0.83310\n",
      "Document (4-fluorophenyl)(pyridin-4-yl)methanone - similarity: 0.83291\n",
      "Document (4-ETHYLPHENYL)SULFAMIC ACID - similarity: 0.83247\n",
      "Document Chrysonilia sitophila - similarity: 0.83244\n",
      "Document Penicillium glaucum - similarity: 0.83222\n",
      "Document Pancrelipase protease - similarity: 0.83130\n",
      "Document Aurotioprol - similarity: 0.83064\n",
      "Document 5-[(4-AMINOCYCLOHEXYL)AMINO]-7-(PROPAN-2-YLAMINO)PYRAZOLO[1,5-A]PYRIMIDINE-3-CARBONITRILE - similarity: 0.83060\n",
      "Document Demecarium - similarity: 0.83008\n",
      "Document N,N'-DIPHENYLPYRAZOLO[1,5-A][1,3,5]TRIAZINE-2,4-DIAMINE - similarity: 0.82996\n",
      "Document Ponazuril - similarity: 0.82911\n",
      "Document PF-00489791 - similarity: 0.82901\n",
      "Document Estradiol hemisuccinate - similarity: 0.82892\n",
      "Document (3R,4R)-1-{6-[3-(METHYLSULFONYL)PHENYL]PYRIMIDIN-4-YL}-4-(2,4,5-TRIFLUOROPHENYL)PIPERIDIN-3-AMINE - similarity: 0.82866\n",
      "Document (1-Methyl-1h-Imidazol-2-Yl)-(3-Methyl-4-{3-[(Pyridin-3-Ylmethyl)-Amino]-Propoxy}-Benzofuran-2-Yl)-Methanone - similarity: 0.82858\n",
      "Document Reversin 121 - similarity: 0.82848\n",
      "Document Dusigitumab - similarity: 0.82834\n",
      "Document Tymazoline - similarity: 0.82825\n",
      "Document N-[4-(benzyloxy)phenyl]glycinamide - similarity: 0.82825\n",
      "Document Mitoglitazone - similarity: 0.82824\n",
      "Document Methotrimeprazine - similarity: 0.82818\n",
      "Document [4-(1,3,2-Dioxaborolan-2-Yloxy)Methyl]Benzamidine - similarity: 0.82814\n",
      "Document N-{5-[4-(4-METHYLPIPERAZIN-1-YL)PHENYL]-1H-PYRROLO[2,3-B]PYRIDIN-3-YL}NICOTINAMIDE - similarity: 0.82797\n",
      "Document VTP-194204 - similarity: 0.82781\n",
      "Document Penthienate - similarity: 0.82769\n",
      "Document 2-[4-(2H-1,4-BENZOTHIAZINE-3-YL)-PIPERAZINE-1-LY]-1,3-THIAZOLE-4-CARBOXYLIC ACID ETHYLESTER - similarity: 0.82766\n",
      "Document Holcus lanatus pollen - similarity: 0.82763\n",
      "Document 2-{5-[3-(7-PROPYL-3-TRIFLUOROMETHYLBENZO[D]ISOXAZOL-6-YLOXY)PROPOXY]INDOL-1-YL}ETHANOIC ACID - similarity: 0.82754\n",
      "Document CGI-1842 - similarity: 0.82707\n",
      "Document Phanquinone - similarity: 0.82703\n",
      "Document (2-BROMOETHYL)(2-'FORMYL-4'-AMINOPHENYL) ACETATE - similarity: 0.82661\n",
      "Document OC-459 - similarity: 0.82641\n",
      "Document Deoxyepinephrine - similarity: 0.82539\n",
      "Document Sodium phosphate, monobasic - similarity: 0.82532\n",
      "Document ALPHA-HYDROXYFARNESYLPHOSPHONIC ACID - similarity: 0.82507\n",
      "Document SC-236 - similarity: 0.82501\n",
      "Document DL-dimyristoylphosphatidylglycerol - similarity: 0.82500\n",
      "Document Influenza A virus A/Victoria/210/2009 X-187 (H3N2) antigen (propiolactone inactivated) - similarity: 0.82484\n",
      "Document Coblopasvir - similarity: 0.82473\n",
      "Document [N-(3-dibenzylcarbamoyl-oxiranecarbonyl)-hydrazino]-acetic acid - similarity: 0.82469\n",
      "Document 4-[[(1E)-2-(4-CHLOROPHENYL)ETHENYL]SULFONYL]-1-[[1-(4-PYRIDINYL)-4-PIPERIDINYL]METHYL]PIPERAZINONE - similarity: 0.82454\n",
      "Document (1S)-1,2,3,4-TETRAHYDRO-BENZO[C]PHENANTHRENE-2,3,4-TRIOL - similarity: 0.82446\n",
      "Document N-(4-chlorobenzyl)-N-methylbenzene-1,4-disulfonamide - similarity: 0.82414\n",
      "Document Clopenthixol - similarity: 0.82410\n",
      "Document Glepaglutide - similarity: 0.82372\n",
      "Document Procyclidine - similarity: 0.82340\n",
      "Document Lapyrium - similarity: 0.82334\n",
      "Document Aniracetam - similarity: 0.82330\n",
      "Document Equine Botulinum Neurotoxin A Immune FAB2 - similarity: 0.82325\n",
      "Document Emfilermin - similarity: 0.82324\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 (H3N2) live (attenuated) antigen - similarity: 0.82317\n",
      "Document (17beta)-17-(cyanomethyl)-2-methoxyestra-1(10),2,4-trien-3-yl sulfamate - similarity: 0.82279\n",
      "Document SRP 299 - similarity: 0.82266\n",
      "Document Lucatumumab - similarity: 0.82259\n",
      "Document Triaziquone - similarity: 0.82211\n",
      "Document Niaprazine - similarity: 0.82207\n",
      "Document Navicixizumab - similarity: 0.82183\n",
      "Document 2-AMINO-6-(3,5-DIMETHYLPHENYL)SULFONYLBENZONITRILE - similarity: 0.82178\n",
      "Document 2-(3-NITROPHENYL)ACETIC ACID - similarity: 0.82166\n",
      "Document Blosozumab - similarity: 0.82056\n",
      "Document Quercus muehlenbergii pollen - similarity: 0.82051\n",
      "Document Streptococcus pneumoniae type 9n capsular polysaccharide antigen - similarity: 0.82007\n",
      "Document Quercetin-3'-O-phosphate - similarity: 0.81988\n",
      "Document AVAC - similarity: 0.81972\n",
      "Document Bentoquatam - similarity: 0.81967\n",
      "Document HOMOPHENYLALANINYLMETHANE - similarity: 0.81947\n",
      "Document DLO6002 - similarity: 0.81903\n",
      "Document Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 hemagglutinin antigen (propiolactone inactivated) - similarity: 0.81891\n",
      "Document WNT-974 - similarity: 0.81890\n",
      "Document 4-{4-[(5-hydroxy-2-methylphenyl)amino]quinolin-7-yl}-1,3-thiazole-2-carbaldehyde - similarity: 0.81860\n",
      "Document 3-{[(1R)-1-phenylethyl]amino}-4-(pyridin-4-ylamino)cyclobut-3-ene-1,2-dione - similarity: 0.81825\n",
      "Document R-428 - similarity: 0.81797\n",
      "Document Ditazole - similarity: 0.81790\n",
      "Document CGS-27023 - similarity: 0.81747\n",
      "Document Quinonoid 7,8-Tetrahydrobiopterin - similarity: 0.81724\n",
      "Document Avoralstat - similarity: 0.81696\n",
      "Document Retaspimycin - similarity: 0.81639\n",
      "Document Funapide - similarity: 0.81622\n",
      "Document Calcium glubionate anhydrous - similarity: 0.81607\n",
      "Document Futuximab - similarity: 0.81589\n",
      "Document 3-{[4-(but-2-yn-1-yloxy)phenyl]sulfonyl}propane-1-thiol - similarity: 0.81581\n",
      "Document Norgestrienone - similarity: 0.81571\n",
      "Document Quercus kelloggii pollen - similarity: 0.81563\n",
      "Document ILS-920 - similarity: 0.81549\n",
      "Document TAFA-93 - similarity: 0.81510\n",
      "Document Influenza A virus A/California/7/2009 X-181 (H1N1) antigen (UV, formaldehyde inactivated) - similarity: 0.81380\n",
      "Document Brotizolam - similarity: 0.81356\n",
      "Document 4-{5-[(Z)-(2-IMINO-4-OXO-1,3-THIAZOLIDIN-5-YLIDENE)METHYL]FURAN-2-YL}-2-(TRIFLUOROMETHYL)BENZENESULFONAMIDE - similarity: 0.81339\n",
      "Document Nicofetamide - similarity: 0.81299\n",
      "Document 7-(5-DEOXY-BETA-D-RIBOFURANOSYL)-5-IODO-7H-PYRROLO[2,3-D]PYRIMIDIN-4-AMINE - similarity: 0.81292\n",
      "Document K-134 - similarity: 0.81277\n",
      "Document Bibenzonium - similarity: 0.81272\n",
      "Document N-[4-(AMINOSULFONYL)PHENYL]-2-MERCAPTOBENZAMIDE - similarity: 0.81245\n",
      "Document 2',5'-Dideoxyuridine - similarity: 0.81228\n",
      "Document Streptococcus pneumoniae type 14 capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.81218\n",
      "Document Bumadizone - similarity: 0.81218\n",
      "Document Influenza A virus A/California/7/2009 (H1N1) recombinant hemagglutinin antigen - similarity: 0.81215\n",
      "Document JE-2147 - similarity: 0.81213\n",
      "Document Haloxon - similarity: 0.81183\n",
      "Document 6-(N-Phenylcarbamyl)-2-Naphthalenecarboxamidine - similarity: 0.81177\n",
      "Document (METHYLPYRIDAZINE PIPERIDINE BUTYLOXYPHENYL)ETHYLACETATE - similarity: 0.81171\n",
      "Document Salix lasiolepis pollen - similarity: 0.81088\n",
      "Document N,N,N-Trimethyl-2-(phosphonooxy)ethanaminium - similarity: 0.81057\n",
      "Document Ipamorelin - similarity: 0.80980\n",
      "Document AC-430 - similarity: 0.80973\n",
      "Document O-(((1R)-((N-(PHENYL-METHOXY-CARBONYL)-ALANYL)-AMINO)METHYL)HYDROXYPHOSPHINYL)3-L-PHENYLLACTATE - similarity: 0.80961\n",
      "Document Thonzylamine - similarity: 0.80957\n",
      "Document Trimegestone - similarity: 0.80955\n",
      "Document Pleospora tarda - similarity: 0.80955\n",
      "Document 3-(1H-indol-3-yl)-4-(1-{2-[(2S)-1-methylpyrrolidinyl]ethyl}-1H-indol-3-yl)-1H-pyrrole-2,5-dione - similarity: 0.80953\n",
      "Document Ethyl Isocyanide - similarity: 0.80923\n",
      "Document Polysorbate 20 - similarity: 0.80891\n",
      "Document Casopitant - similarity: 0.80891\n",
      "Document THIENO[3,2-B]PYRIDINE-2-SULFONIC ACID [1-(1-AMINO-ISOQUINOLIN-7-YLMETHYL)-2-OXO-PYRROLDIN-3-YL]-AMIDE - similarity: 0.80872\n",
      "Document Propyliodone - similarity: 0.80854\n",
      "Document Venison - similarity: 0.80839\n",
      "Document Metahexamide - similarity: 0.80837\n",
      "Document Black-eyed pea - similarity: 0.80807\n",
      "Document Streptococcus pneumoniae type 15b capsular polysaccharide antigen - similarity: 0.80800\n",
      "Document ATU-027 - similarity: 0.80796\n",
      "Document 4-[4-(4-Fluorophenyl)-2-[4-[(R)-methylsulfinyl]phenyl]-1H-imidazol-5-yl]pyridine - similarity: 0.80763\n",
      "Document Candoxatrilat - similarity: 0.80750\n",
      "Document N-[3-(Dimethylamino)Propyl]-2-({[4-({[4-(Formylamino)-1-Methyl-1h-Pyrrol-2-Yl]Carbonyl}Amino)-1-Methyl-1h-Pyrrol-2-Yl]Carbonyl}Amino)-5-Isopropyl-1,3-Thiazole-4-Carboxamide - similarity: 0.80701\n",
      "Document 3-(heptyloxy)benzoic acid - similarity: 0.80673\n",
      "Document Jaspisamide A - similarity: 0.80649\n",
      "Document Benzododecinium - similarity: 0.80561\n",
      "Document Idarucizumab - similarity: 0.80518\n",
      "Document 3,20-Pregnanedione - similarity: 0.80504\n",
      "Document Fosamprenavir - similarity: 0.80491\n",
      "Document Acetarsol - similarity: 0.80473\n",
      "Document N-[2-Hydroxy-2-(8-Isopropyl-6,9-Dioxo-2-Oxa-7,10-Diaza-Bicyclo[11.2.2]Heptadeca-1(16),13(17),14-Trien-11-Yl)-Ethyl]-N-(3-Methyl-Butyl)-Benzenesulfonamide,Inhibitor 3 - similarity: 0.80436\n",
      "Document 4-[3-(1,4-diazepan-1-ylcarbonyl)-4-fluorobenzyl]phthalazin-1(2H)-one - similarity: 0.80428\n",
      "Document 1-ethyl-N-(phenylmethyl)-4-(tetrahydro-2H-pyran-4-ylamino)-1H-pyrazolo[3,4-b]pyridine-5-carboxamide - similarity: 0.80420\n",
      "Document Tenoxicam - similarity: 0.80398\n",
      "Document Albusomatropin - similarity: 0.80324\n",
      "Document Canakinumab - similarity: 0.80310\n",
      "Document (1R)-1-(2-thienylacetylamino)-1-phenylmethylboronic acid - similarity: 0.80305\n",
      "Document Pinazepam - similarity: 0.80291\n",
      "Document 9-amino-n-[3-(dimethylamino)propyl]acridine-4-carboxamide - similarity: 0.80274\n",
      "Document 1-(1-HYDROXY-2,2,6,6-TETRAMETHYLPIPERIDIN-4-YL)PYRROLIDINE-2,5-DIONE - similarity: 0.80270\n",
      "Document 4-chloro-6-{5-[(2-morpholin-4-ylethyl)amino]-1,2-benzisoxazol-3-yl}benzene-1,3-diol - similarity: 0.80258\n",
      "Document Bevantolol - similarity: 0.80198\n",
      "Document Mirikizumab - similarity: 0.80191\n",
      "Document 3-chloro-5-[2-chloro-5-(1H-indazol-3-ylmethoxy)phenoxy]benzonitrile - similarity: 0.80128\n",
      "Document Oblimersen - similarity: 0.80112\n",
      "Document Populus alba pollen - similarity: 0.80097\n",
      "Document N-Heptylformamide - similarity: 0.80091\n",
      "Document Evolocumab - similarity: 0.80089\n",
      "Document Drisapersen - similarity: 0.80085\n",
      "Document Human papillomavirus type 33 L1 capsid protein antigen - similarity: 0.80077\n",
      "Document GSK-2330672 - similarity: 0.80069\n",
      "Document Calcium sulfide - similarity: 0.80066\n",
      "Document 8,9,10-trihydroxy-7-hydroxymethyl-3-methyl-6-oxa-1,3-diaza-spiro[4.5]decane-2,4-dione - similarity: 0.80032\n",
      "Document 1-biphenyl-2-ylmethanamine - similarity: 0.80022\n",
      "Document Fagus grandifolia pollen - similarity: 0.80010\n",
      "Document Influenza A virus A/California/7/2009 X-179A (H1N1) antigen (UV, formaldehyde inactivated) - similarity: 0.80001\n",
      "Document 4-(2-HYDROXY-4-FLUOROPHENYLTHIO)-BUTYLPHOSPHONIC ACID - similarity: 0.79998\n",
      "Document 5'-S-ethyl-5'-thioadenosine - similarity: 0.79954\n",
      "Document lambda-bis(2,2'-bipyridine)-(5-methyl-2-2'-bipyridine)-C9-adamantane ruthenium (II) - similarity: 0.79939\n",
      "Document (3R,5Z,8S,9S,11E)-8,9,16-TRIHYDROXY-14-METHOXY-3-METHYL-3,4,9,10-TETRAHYDRO-1H-2-BENZOXACYCLOTETRADECINE-1,7(8H)-DIONE - similarity: 0.79909\n",
      "Document Gemeprost - similarity: 0.79904\n",
      "Document Phoenix dactylifera pollen - similarity: 0.79897\n",
      "Document Brilaroxazine - similarity: 0.79825\n",
      "Document Ifetroban - similarity: 0.79816\n",
      "Document Vantictumab - similarity: 0.79813\n",
      "Document Tradipitant - similarity: 0.79805\n",
      "Document 3,5-diiodothyropropionic acid - similarity: 0.79764\n",
      "Document Fosfructose - similarity: 0.79732\n",
      "Document (3R)-3-HYDROXYDODECANOIC ACID - similarity: 0.79727\n",
      "Document 5-bromo-N-(3-chloro-2-(4-(prop-2-ynyl)piperazin-1-yl)phenyl)furan-2-carboxamide - similarity: 0.79717\n",
      "Document Interferon alfa-n3 - similarity: 0.79694\n",
      "Document Sulfaisodimidine - similarity: 0.79692\n",
      "Document Juniperus californica pollen - similarity: 0.79659\n",
      "Document Rhus Glabra Pollen - similarity: 0.79639\n",
      "Document 5-(4-FLUOROPHENYL)-3-{[(4-METHYLPHENYL)SULFONYL]AMINO}THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.79616\n",
      "Document Radavirsen - similarity: 0.79602\n",
      "Document Ketorolac - similarity: 0.79595\n",
      "Document Upadacitinib - similarity: 0.79582\n",
      "Document Hydroxyaminovaline - similarity: 0.79564\n",
      "Document N-{(1S,2R)-1-BENZYL-3-[(CYCLOPROPYLMETHYL)(2-FURYLSULFONYL)AMINO]-2-HYDROXYPROPYL}-N'-METHYLSUCCINAMIDE - similarity: 0.79557\n",
      "Document PARA-NITROPHENYL 1-THIO-BETA-D-GLUCOPYRANOSIDE - similarity: 0.79519\n",
      "Document Sotagliflozin - similarity: 0.79497\n",
      "Document N-(2-AMINOETHYL)-2-{3-CHLORO-4-[(4-ISOPROPYLBENZYL)OXY]PHENYL} ACETAMIDE - similarity: 0.79463\n",
      "Document 1-(2-HYDROXYETHYLOXYMETHYL)-6-PHENYL THIOTHYMINE - similarity: 0.79457\n",
      "Document Golodirsen - similarity: 0.79385\n",
      "Document (2S)-2-(1H-indol-3-yl)hexanoic acid - similarity: 0.79373\n",
      "Document 4-(1,3-BENZODIOXOL-5-YLOXY)-2-[4-(1H-IMIDAZOL-1-YL)PHENOXY]PYRIMIDINE - similarity: 0.79365\n",
      "Document (3S)-1-(2-hydroxyphenyl)-5-oxopyrrolidine-3-carboxylic acid - similarity: 0.79341\n",
      "Document Metipranolol - similarity: 0.79324\n",
      "Document Trefentanil - similarity: 0.79315\n",
      "Document Pyrrobutamine - similarity: 0.79299\n",
      "Document Taprenepag - similarity: 0.79297\n",
      "Document Tezepelumab - similarity: 0.79270\n",
      "Document 2-[4-({[(3,5-DICHLOROPHENYL)AMINO]CARBONYL}AMINO)PHENOXY]-2-METHYLPROPANOIC ACID - similarity: 0.79252\n",
      "Document D-phenylalanyl-N-{4-[amino(iminio)methyl]benzyl}-L-prolinamide - similarity: 0.79227\n",
      "Document Frovatriptan - similarity: 0.79217\n",
      "Document N~2~-[(BENZYLOXY)CARBONYL]-N-[(1S,2S)-2-HYDROXY-1-(4-HYDROXYBENZYL)PROPYL]-L-LEUCINAMIDE - similarity: 0.79202\n",
      "Document Methoxyphenamine - similarity: 0.79150\n",
      "Document 4-Carbamoyl-4-{[6-(Difluoro-Phosphono-Methyl)-Naphthalene-2-Carbonyl]-Amino}-Butyric Acid - similarity: 0.79145\n",
      "Document Beinaglutide - similarity: 0.79134\n",
      "Document Setrusumab - similarity: 0.79103\n",
      "Document Mebanazine - similarity: 0.79103\n",
      "Document Rimazolium - similarity: 0.79087\n",
      "Document Eniluracil - similarity: 0.79081\n",
      "Document Atecegatran metoxil - similarity: 0.79075\n",
      "Document Guaiazulen - similarity: 0.79051\n",
      "Document (S)-oct-1-en-3-ol - similarity: 0.79018\n",
      "Document Vinbarbital - similarity: 0.79015\n",
      "Document Evinacumab - similarity: 0.78999\n",
      "Document (S)-N-(4-carbamimidoylbenzyl)-1-(3-cyclohexylpropanoyl)pyrrolidine-2-carboxamide - similarity: 0.78985\n",
      "Document Triazolam - similarity: 0.78896\n",
      "Document Trigonellamide - similarity: 0.78836\n",
      "Document RPI-78M - similarity: 0.78831\n",
      "Document Delavirdine - similarity: 0.78829\n",
      "Document Guselkumab - similarity: 0.78824\n",
      "Document 8-chlorotheophylline - similarity: 0.78807\n",
      "Document Rezatomidine - similarity: 0.78767\n",
      "Document 5-Chloro-N-{(3S)-1-[(2S)-1-(4-morpholinyl)-1-oxo-2-propanyl]-2-oxo-3-pyrrolidinyl}-1H-indole-2-sulfonamide - similarity: 0.78763\n",
      "Document Celtis occidentalis pollen - similarity: 0.78736\n",
      "Document Podophyllin - similarity: 0.78732\n",
      "Document Dodecyl-Alpha-D-Maltoside - similarity: 0.78716\n",
      "Document N-[(1S)-1-(aminocarbonyl)-4-(ethanimidoylamino)butyl]benzamide - similarity: 0.78710\n",
      "Document Aluminum zirconium trichlorohydrex gly - similarity: 0.78705\n",
      "Document PSN9301 - similarity: 0.78686\n",
      "Document Lenzilumab - similarity: 0.78682\n",
      "Document (5Z)-3-(4-CHLOROPHENYL)-4-HYDROXY-5-(1-NAPHTHYLMETHYLENE)FURAN-2(5H)-ONE - similarity: 0.78682\n",
      "Document Abarelix - similarity: 0.78676\n",
      "Document 2-(2-chloropyridin-4-yl)-4-methyl-1H-isoindole-1,3(2H)-dione - similarity: 0.78675\n",
      "Document Fenprostalene - similarity: 0.78661\n",
      "Document Disperse Blue 106 - similarity: 0.78660\n",
      "Document CE-326597 - similarity: 0.78651\n",
      "Document Ancestim - similarity: 0.78641\n",
      "Document Trichophyton mentagrophytes - similarity: 0.78639\n",
      "Document (2E,4E)-11-METHOXY-3,7,11-TRIMETHYLDODECA-2,4-DIENOIC ACID - similarity: 0.78615\n",
      "Document Neisseria meningitidis group y capsular oligosaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.78611\n",
      "Document Clobetasone - similarity: 0.78605\n",
      "Document Gantacurium - similarity: 0.78582\n",
      "Document Propicillin - similarity: 0.78565\n",
      "Document Stavudine triphosphate - similarity: 0.78560\n",
      "Document Technetium Tc-99m disofenin - similarity: 0.78524\n",
      "Document Enokizumab - similarity: 0.78483\n",
      "Document N-(4-ACETYLPHENYL)-5-(5-CHLORO-2,4-DIHYDROXYPHENYL)-1H-PYRAZOLE-4-CARBOXAMIDE - similarity: 0.78480\n",
      "Document 2-(6-CHLORO-3-{[2,2-DIFLUORO-2-(1-OXIDO-2-PYRIDINYL)ETHYL]AMINO}-2-OXO-1(2H)-PYRAZINYL)-N-[(2-FLUOROPHENYL)METHYL]ACETAMIDE - similarity: 0.78474\n",
      "Document MVA3000 - similarity: 0.78468\n",
      "Document N-(4-{[(3S)-3-(dimethylamino)pyrrolidin-1-yl]carbonyl}phenyl)-5-fluoro-4-[2-methyl-1-(1-methylethyl)-1H-imidazol-5-yl]pyrimidin-2-amine - similarity: 0.78461\n",
      "Document 2-(3-METHYLPHENYL)-1H-INDOLE-5-CARBOXIMIDAMIDE - similarity: 0.78461\n",
      "Document GSK-364735 - similarity: 0.78458\n",
      "Document (2R,4S)-2-[(1R)-1-{[(2Z)-2-(2-Amino-1,3-thiazol-4-yl)-2-(methoxyimino)acetyl]amino}-2-oxoethyl]-5,5-dimethyl-1,3-thiazolidine-4-carboxylic acid - similarity: 0.78445\n",
      "Document 4-Hydroxy-3-Methyl Butyl Diphosphate - similarity: 0.78445\n",
      "Document 2-(4-HYDROXY-5-PHENYL-1H-PYRAZOL-3-YL)-1H-BENZOIMIDAZOLE-5-CARBOXAMIDINE - similarity: 0.78435\n",
      "Document 6,11-DIHYDRO-11-ETHYL-6-METHYL-9-NITRO-5H-PYRIDO[2,3-B][1,5]BENZODIAZEPIN-5-ONE - similarity: 0.78430\n",
      "Document Polysorbate 80 - similarity: 0.78402\n",
      "Document Balaglitazone - similarity: 0.78377\n",
      "Document Promegestone - similarity: 0.78336\n",
      "Document Anileridine - similarity: 0.78315\n",
      "Document Beclamide - similarity: 0.78313\n",
      "Document Stannsoporfin - similarity: 0.78303\n",
      "Document Tetrazepam - similarity: 0.78261\n",
      "Document Epicillin - similarity: 0.78238\n",
      "Document LCB01-0371 - similarity: 0.78210\n",
      "Document (3E)-3-[(phenylamino)methylidene]dihydrofuran-2(3H)-one - similarity: 0.78207\n",
      "Document 5'-O-(L-Prolylsulfamoyl)adenosine - similarity: 0.78202\n",
      "Document N-Methyl-Pyridoxal-5'-Phosphate - similarity: 0.78193\n",
      "Document {4-Amino-2-[(4-methoxyphenyl)amino]-1,3-thiazol-5-yl}(4-methoxyphenyl)methanone - similarity: 0.78162\n",
      "Document Pacritinib - similarity: 0.78160\n",
      "Document RUS 3108 - similarity: 0.78155\n",
      "Document Ponezumab - similarity: 0.78120\n",
      "Document Benzilone - similarity: 0.78117\n",
      "Document Manganese gluconate - similarity: 0.78106\n",
      "Document 4-HYDROXYBENZALDEHYDE O-(CYCLOHEXYLCARBONYL)OXIME - similarity: 0.78091\n",
      "Document Ornipressin - similarity: 0.78085\n",
      "Document Triticum aestivum pollen - similarity: 0.78084\n",
      "Document {3-[3-(3,4-Dimethoxy-Phenyl)-1-(1-{1-[2-(3,4,5-Trimethoxy-Phenyl)-Butyryl]-Piperidin-2yl}-Vinyloxy)-Propyl]-Phenoxy}-Acetic Acid - similarity: 0.78078\n",
      "Document N-(indole-3-acetyl)-L-aspartic acid - similarity: 0.78076\n",
      "Document Triazavirin - similarity: 0.78063\n",
      "Document Patent Blue - similarity: 0.78061\n",
      "Document 2-{[(4-CHLOROPHENOXY)ACETYL]AMINO}BENZOIC ACID - similarity: 0.78058\n",
      "Document Tramazoline - similarity: 0.78055\n",
      "Document N,N-Bis(3-(D-gluconamido)propyl)deoxycholamide - similarity: 0.78053\n",
      "Document GPI-0100 - similarity: 0.78052\n",
      "Document Repifermin - similarity: 0.78032\n",
      "Document Drometrizole trisiloxane - similarity: 0.78011\n",
      "Document Flumethasone - similarity: 0.77996\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Lintuzumab - similarity: 0.77979\n",
      "Document SLV319 - similarity: 0.77975\n",
      "Document Albaconazole - similarity: 0.77946\n",
      "Document +/-METHYL 4-(AMINOIMINOMETHYL)-BETA-[3- INH (AMINOIMINO)PHENYL]BENZENE PENTANOATE - similarity: 0.77932\n",
      "Document Lebrikizumab - similarity: 0.77931\n",
      "Document Amineptine - similarity: 0.77892\n",
      "Document 1,3-bis-([3-[3-[3-(4-{3-[3-nitro-5-(galactopyranosyloxy)-benzoylamino]-propyl}-piperazin-1-yl)-propylamino-3,4-dioxo-cyclobutenyl]-amino-propoxy-ethoxy-ethoxy]-propyl-]amino-carbonyloxy)-2-amino-propane - similarity: 0.77874\n",
      "Document Magnesium levulinate - similarity: 0.77860\n",
      "Document Dicoria canescens pollen - similarity: 0.77836\n",
      "Document 4-{5-[(Z)-(2-IMINO-4-OXO-1,3-THIAZOLIDIN-5-YLIDENE)METHYL]FURAN-2-YL}BENZENESULFONAMIDE - similarity: 0.77834\n",
      "Document Netazepide - similarity: 0.77827\n",
      "Document Belapectin - similarity: 0.77820\n",
      "Document 5-(5-CHLORO-2-THIENYL)-N-{(3S)-1-[(1S)-1-METHYL-2-MORPHOLIN-4-YL-2-OXOETHYL]-2-OXOPYRROLIDIN-3-YL}-1H-1,2,4-TRIAZOLE-3-SULFONAMIDE - similarity: 0.77812\n",
      "Document {4-[2,2-BIS(5-METHYL-1,2,4-OXADIAZOL-3-YL)-3-PHENYLPROPYL]PHENYL}SULFAMIC ACID - similarity: 0.77745\n",
      "Document Taranabant - similarity: 0.77739\n",
      "Document Ozarelix - similarity: 0.77734\n",
      "Document FR117016 - similarity: 0.77717\n",
      "Document Sugammadex - similarity: 0.77689\n",
      "Document Sirukumab - similarity: 0.77660\n",
      "Document Veralipride - similarity: 0.77641\n",
      "Document {(2S)-1-[N-(tert-butoxycarbonyl)glycyl]pyrrolidin-2-yl}methyl (3-chlorophenyl)acetate - similarity: 0.77603\n",
      "Document 11-[(MERCAPTOCARBONYL)OXY]UNDECANOIC ACID - similarity: 0.77546\n",
      "Document Ustekinumab - similarity: 0.77540\n",
      "Document Aminomethylcyclohexane - similarity: 0.77535\n",
      "Document Apilimod - similarity: 0.77487\n",
      "Document 5'-FLUORO-2',5'-DIDEOXYADENOSINE - similarity: 0.77484\n",
      "Document IQP-0528 - similarity: 0.77455\n",
      "Document R673 - similarity: 0.77438\n",
      "Document Leronlimab - similarity: 0.77407\n",
      "Document Valethamate - similarity: 0.77374\n",
      "Document Heptulose-2-Phosphate - similarity: 0.77373\n",
      "Document Tildrakizumab - similarity: 0.77369\n",
      "Document Ozanimod - similarity: 0.77347\n",
      "Document Acetone Cyanohydrin - similarity: 0.77324\n",
      "Document Polyestradiol phosphate - similarity: 0.77318\n",
      "Document GSI-136 - similarity: 0.77310\n",
      "Document Quaternium-15 - similarity: 0.77300\n",
      "Document Tomuzotuximab - similarity: 0.77292\n",
      "Document N-cyclopropyl-2',6-dimethyl-4'-(5-methyl-1,3,4-oxadiazol-2-yl)biphenyl-3-carboxamide - similarity: 0.77268\n",
      "Document Butorphanol - similarity: 0.77249\n",
      "Document Brodalumab - similarity: 0.77209\n",
      "Document Heterotheca inuloides flower - similarity: 0.77206\n",
      "Document PD-166326 - similarity: 0.77205\n",
      "Document Etilamfetamine - similarity: 0.77194\n",
      "Document 7-[(3-CHLOROBENZYL)OXY]-2-OXO-2H-CHROMENE-4-CARBALDEHYDE - similarity: 0.77171\n",
      "Document Fosifloxuridine nafalbenamide - similarity: 0.77168\n",
      "Document Paramethasone acetate - similarity: 0.77154\n",
      "Document Maribavir - similarity: 0.77136\n",
      "Document 17-METHYL-17-ALPHA-DIHYDROEQUILENIN - similarity: 0.77133\n",
      "Document Pexelizumab - similarity: 0.77128\n",
      "Document AT3022 - similarity: 0.77089\n",
      "Document Racemethionine - similarity: 0.77036\n",
      "Document 3-{3-[(DIMETHYLAMINO)METHYL]-1H-INDOL-7-YL}PROPAN-1-OL - similarity: 0.77030\n",
      "Document Ganetespib - similarity: 0.77022\n",
      "Document 7-amino-2-tert-butyl-4-{[2-(1H-imidazol-4-yl)ethyl]amino}pyrido[2,3-d]pyrimidine-6-carboxamide - similarity: 0.77022\n",
      "Document Estradiol acetate - similarity: 0.77015\n",
      "Document Eribaxaban - similarity: 0.76969\n",
      "Document Momelotinib - similarity: 0.76967\n",
      "Document Sepranolone - similarity: 0.76950\n",
      "Document GTI 2040 - similarity: 0.76903\n",
      "Document Prenylamine - similarity: 0.76896\n",
      "Document Penbutolol - similarity: 0.76878\n",
      "Document Stypocaulon scoparium - similarity: 0.76849\n",
      "Document Hydroquinone - similarity: 0.76827\n",
      "Document Oxilofrine - similarity: 0.76817\n",
      "Document PG-701 - similarity: 0.76810\n",
      "Document Chloroprocaine - similarity: 0.76786\n",
      "Document Vunakizumab - similarity: 0.76770\n",
      "Document Hydroxyprogesterone caproate - similarity: 0.76744\n",
      "Document (2Z)-2-cyano-N-(2,2'-dichlorobiphenyl-4-yl)-3-hydroxybut-2-enamide - similarity: 0.76729\n",
      "Document 3,6,9,12,15-PENTAOXATRICOSAN-1-OL - similarity: 0.76720\n",
      "Document Clofenotane - similarity: 0.76691\n",
      "Document Azidamfenicol - similarity: 0.76681\n",
      "Document Levonorgestrel - similarity: 0.76592\n",
      "Document Telbivudine - similarity: 0.76579\n",
      "Document Surinabant - similarity: 0.76577\n",
      "Document 1-((1R,2S)-1-{2-[2-(4-CHLOROPHENYL)-1,3-BENZOXAZOL-7-YL]ETHYL}-2-HYDROXYPROPYL)-1H-IMIDAZOLE-4-CARBOXAMIDE - similarity: 0.76564\n",
      "Document Uprifosbuvir - similarity: 0.76543\n",
      "Document TP-271 - similarity: 0.76521\n",
      "Document 3-(1H-indol-3-yl)-4-{1-[2-(1-methylpyrrolidin-2-yl)ethyl]-1H-indol-3-yl}-1H-pyrrole-2,5-dione - similarity: 0.76515\n",
      "Document Pegzilarginase - similarity: 0.76508\n",
      "Document 2-[(4-fluorophenyl)sulfonylamino]-N-oxo-ethanamide - similarity: 0.76458\n",
      "Document Nickel sulfate - similarity: 0.76444\n",
      "Document Fosdevirine - similarity: 0.76439\n",
      "Document Lumacaftor - similarity: 0.76431\n",
      "Document 2-AMINO-N,N-BIS(PHENYLMETHYL)-1,3-OXAZOLE-5-CARBOXAMIDE - similarity: 0.76427\n",
      "Document Morphine glucuronide - similarity: 0.76415\n",
      "Document Valrocemide - similarity: 0.76415\n",
      "Document Dimetindene - similarity: 0.76412\n",
      "Document ADP-597 - similarity: 0.76395\n",
      "Document Trospium - similarity: 0.76372\n",
      "Document Temsirolimus - similarity: 0.76372\n",
      "Document 2-(cycloheptylmethyl)-1,1-dioxido-1-benzothiophen-6-yl sulfamate - similarity: 0.76352\n",
      "Document Diclazuril - similarity: 0.76342\n",
      "Document Ralinepag - similarity: 0.76330\n",
      "Document MK-0736 - similarity: 0.76301\n",
      "Document 3-{[4-([amino(imino)methyl]aminosulfonyl)anilino]methylene}-2-oxo-2,3-dihydro-1H-indole - similarity: 0.76295\n",
      "Document Aflibercept - similarity: 0.76264\n",
      "Document Buckwheat - similarity: 0.76260\n",
      "Document Human Varicella-Zoster Immune Globulin - similarity: 0.76251\n",
      "Document 3-(2-aminoquinazolin-6-yl)-1-(3,3-dimethylindolin-6-yl)-4-methylpyridin-2(1H)-one - similarity: 0.76219\n",
      "Document MBX-8025 - similarity: 0.76207\n",
      "Document Dithioerythritol - similarity: 0.76201\n",
      "Document Hetacillin - similarity: 0.76186\n",
      "Document Tanomastat - similarity: 0.76166\n",
      "Document Adapalene - similarity: 0.76152\n",
      "Document Pipamperone - similarity: 0.76116\n",
      "Document Indium cation In-111 - similarity: 0.76097\n",
      "Document Manitimus - similarity: 0.76067\n",
      "Document Prethcamide - similarity: 0.76063\n",
      "Document Morniflumate - similarity: 0.76057\n",
      "Document 4-[[(2S)-2-[[(2S)-3-Carboxy-2-hydroxypropanoyl]amino]-4-methylpentanoyl]amino]butyl-(diaminomethylidene)azanium - similarity: 0.76048\n",
      "Document (R)-pyridin-4-yl[4-(2-pyrrolidin-1-ylethoxy)phenyl]methanol - similarity: 0.76047\n",
      "Document Acetaminophen - similarity: 0.76044\n",
      "Document (5R)-5-(4-{[(2R)-6-HYDROXY-2,5,7,8-TETRAMETHYL-3,4-DIHYDRO-2H-CHROMEN-2-YL]METHOXY}BENZYL)-1,3-THIAZOLIDINE-2,4-DIONE - similarity: 0.76042\n",
      "Document PI-88 - similarity: 0.76039\n",
      "Document 3-({4-[(1E)-3-morpholin-4-yl-3-oxoprop-1-en-1-yl]-2,3-bis(trifluoromethyl)phenyl}sulfanyl)aniline - similarity: 0.76005\n",
      "Document 9-Butyl-8-(3-Methoxybenzyl)-9h-Purin-6-Amine - similarity: 0.76000\n",
      "Document Pimethixene - similarity: 0.75999\n",
      "Document Anecortave acetate - similarity: 0.75994\n",
      "Document 2-(4-CARCOXY-5-ISOPROPYLTHIAZOLYL)BENZOPIPERIDINE - similarity: 0.75994\n",
      "Document Mepartricin - similarity: 0.75981\n",
      "Document Clefamide - similarity: 0.75940\n",
      "Document Loteprednol - similarity: 0.75928\n",
      "Document Oxatomide - similarity: 0.75906\n",
      "Document ISOQUINOLINE-5-SULFONIC ACID (2-(2-(4-CHLOROBENZYLOXY)ETHYLAMINO)ETHYL)AMIDE - similarity: 0.75904\n",
      "Document Bimekizumab - similarity: 0.75894\n",
      "Document Acacia longifolia pollen - similarity: 0.75885\n",
      "Document Fluprednisolone - similarity: 0.75885\n",
      "Document Pipecuronium - similarity: 0.75882\n",
      "Document Tralokinumab - similarity: 0.75880\n",
      "Document Losmapimod - similarity: 0.75861\n",
      "Document 9-(4-hydroxybutyl)-N2-phenylguanine - similarity: 0.75838\n",
      "Document Echinacea angustifolia root - similarity: 0.75832\n",
      "Document Difelikefalin - similarity: 0.75813\n",
      "Document Bouteloua gracilis pollen - similarity: 0.75803\n",
      "Document Hydrocortisone probutate - similarity: 0.75800\n",
      "Document MK-0767 - similarity: 0.75752\n",
      "Document Tetracaine - similarity: 0.75730\n",
      "Document [(3R,4S,5S,7R)-4,8-DIHYDROXY-3,5,7-TRIMETHYL-2-OXOOCTYL]PHOSPHONIC ACID - similarity: 0.75717\n",
      "Document Resorcinol monoacetate - similarity: 0.75680\n",
      "Document (2R)-2-(5-CHLORO-2-THIENYL)-N-{(3S)-1-[(1S)-1-METHYL-2-MORPHOLIN-4-YL-2-OXOETHYL]-2-OXOPYRROLIDIN-3-YL}PROPENE-1-SULFONAMIDE - similarity: 0.75679\n",
      "Document Narlaprevir - similarity: 0.75674\n",
      "Document 1-[4-(2-oxo-2-phenylethyl)phenyl]guanidine - similarity: 0.75657\n",
      "Document Hydrocortisone succinate - similarity: 0.75632\n",
      "Document DP-b99 - similarity: 0.75624\n",
      "Document Amatuximab - similarity: 0.75619\n",
      "Document (1S)-1-(1H-INDOL-3-YLMETHYL)-2-(2-PYRIDIN-4-YL-[1,7]NAPHTYRIDIN-5-YLOXY)-EHYLAMINE - similarity: 0.75607\n",
      "Document Crotamiton - similarity: 0.75601\n",
      "Document NAP-226-90 - similarity: 0.75568\n",
      "Document Dapaconazole - similarity: 0.75532\n",
      "Document Alprenolol - similarity: 0.75503\n",
      "Document Dexchlorpheniramine maleate - similarity: 0.75491\n",
      "Document Insulin glargine - similarity: 0.75484\n",
      "Document S-[(1-Hydroxy-2,2,5,5-tetramethyl-4-phenyl-2,5-dihydro-1H-pyrrol-3-yl)methyl] methanesulfonothioate - similarity: 0.75484\n",
      "Document Azeliragon - similarity: 0.75427\n",
      "Document Levallorphan - similarity: 0.75425\n",
      "Document Osocimab - similarity: 0.75421\n",
      "Document 2-Amino-6-Chloropyrazine - similarity: 0.75419\n",
      "Document Efrotomycin - similarity: 0.75410\n",
      "Document Tilletia caries - similarity: 0.75392\n",
      "Document dimethyl (1R,4S)-5,6-bis(4-hydroxyphenyl)-7-oxabicyclo[2.2.1]hepta-2,5-diene-2,3-dicarboxylate - similarity: 0.75392\n",
      "Document CA4P - similarity: 0.75388\n",
      "Document 4-(2-{[4-{[3-(4-Chlorophenyl)Propyl]Sulfanyl}-6-(1-Piperazinyl)-1,3,5-Triazin-2-Yl]Amino}Ethyl)Phenol - similarity: 0.75387\n",
      "Document Trichlormethiazide - similarity: 0.75385\n",
      "Document Ralimetinib - similarity: 0.75350\n",
      "Document 1-[(2S)-4-(5-phenyl-1H-pyrazolo[3,4-b]pyridin-4-yl)morpholin-2-yl]methanamine - similarity: 0.75340\n",
      "Document Pinus monticola pollen - similarity: 0.75328\n",
      "Document 5-methyl-2'-fluoroarauracil F-18 - similarity: 0.75324\n",
      "Document (2S)-1-{[5-(1H-Indazol-5-yl)-3-pyridinyl]oxy}-3-(7aH-indol-3-yl)-2-propanamine - similarity: 0.75316\n",
      "Document 1-CYCLOHEXYL-N-{[1-(4-METHYLPHENYL)-1H-INDOL-3-YL]METHYL}METHANAMINE - similarity: 0.75308\n",
      "Document Ipragliflozin - similarity: 0.75280\n",
      "Document Influenza A virus A/South Australia/55/2014 IVR-175 (H3N2) antigen (propiolactone inactivated) - similarity: 0.75271\n",
      "Document Butamirate - similarity: 0.75263\n",
      "Document Arbutamine - similarity: 0.75260\n",
      "Document Proline - similarity: 0.75254\n",
      "Document Faldaprevir - similarity: 0.75237\n",
      "Document Ethyl loflazepate - similarity: 0.75231\n",
      "Document N-anthracen-2-yl-5-methyl[1,2,4]triazolo[1,5-a]pyrimidin-7-amine - similarity: 0.75182\n",
      "Document Isavuconazonium - similarity: 0.75171\n",
      "Document Implitapide - similarity: 0.75168\n",
      "Document Tilidine - similarity: 0.75166\n",
      "Document Padeliporfin - similarity: 0.75156\n",
      "Document RAF-265 - similarity: 0.75153\n",
      "Document Bitopertin - similarity: 0.75120\n",
      "Document Coenzyme F420 - similarity: 0.75119\n",
      "Document Ilomastat - similarity: 0.75114\n",
      "Document 2,3,5,6-Tetrafluoro-4-Methoxy-Benzamide - similarity: 0.75105\n",
      "Document Omigapil - similarity: 0.75088\n",
      "Document Amcinonide - similarity: 0.75084\n",
      "Document α-Methylacetylfentanyl - similarity: 0.75074\n",
      "Document Gevokizumab - similarity: 0.75065\n",
      "Document Magnesium lactate - similarity: 0.75055\n",
      "Document Tryptophanamide - similarity: 0.75046\n",
      "Document 5-(2-Chlorophenyl)Furan-2-Carboxylic Acid - similarity: 0.75046\n",
      "Document Potassium triiodide - similarity: 0.75041\n",
      "Document Indoleacetamide - similarity: 0.75032\n",
      "Document 4-(3-amino-1H-indazol-5-yl)-N-tert-butylbenzenesulfonamide - similarity: 0.75027\n",
      "Document Virginiamycin M1 - similarity: 0.74991\n",
      "Document Emibetuzumab - similarity: 0.74968\n",
      "Document Labradimil - similarity: 0.74963\n",
      "Document Vespula pensylvanica venom protein - similarity: 0.74956\n",
      "Document Fenspiride - similarity: 0.74956\n",
      "Document Debrisoquine - similarity: 0.74942\n",
      "Document Dioxybenzone - similarity: 0.74936\n",
      "Document Demcizumab - similarity: 0.74923\n",
      "Document 5-(5-CHLORO-2,4-DIHYDROXYPHENYL)-N-ETHYL-4-PIPERAZIN-1-YL-1H-PYRAZOLE-3-CARBOXAMIDE - similarity: 0.74914\n",
      "Document Avena sativa pollen - similarity: 0.74912\n",
      "Document Etorphine - similarity: 0.74892\n",
      "Document (1S)-2-oxo-1-phenyl-2-[(1,3,4-trioxo-1,2,3,4-tetrahydroisoquinolin-5-yl)amino]ethyl acetate - similarity: 0.74818\n",
      "Document TOP-1288 - similarity: 0.74811\n",
      "Document Benzphetamine - similarity: 0.74810\n",
      "Document Reglitazar - similarity: 0.74788\n",
      "Document Opebacan - similarity: 0.74777\n",
      "Document Polyethylene glycol 400 - similarity: 0.74773\n",
      "Document Deanol - similarity: 0.74763\n",
      "Document 3(S)-AMINO-4-PHENYL-BUTAN-2(S)-OL - similarity: 0.74736\n",
      "Document 1-{2-OXO-3-[(1R)-1-(1H-PYRROL-2-YL)ETHYL]-2H-INDOL-5-YL}UREA - similarity: 0.74731\n",
      "Document 1-(2,5-dideoxy-5-pyrrolidin-1-yl-beta-L-erythro-pentofuranosyl)-5-methylpyrimidine-2,4(1H,3H)-dione - similarity: 0.74729\n",
      "Document Rociverine - similarity: 0.74713\n",
      "Document Pemirolast - similarity: 0.74698\n",
      "Document Pranlukast - similarity: 0.74682\n",
      "Document Carboprost tromethamine - similarity: 0.74658\n",
      "Document FLX475 - similarity: 0.74644\n",
      "Document Tozadenant - similarity: 0.74634\n",
      "Document 5-fluorotryptophan - similarity: 0.74629\n",
      "Document Dorzagliatin - similarity: 0.74615\n",
      "Document Lanadelumab - similarity: 0.74612\n",
      "Document Etymemazine - similarity: 0.74608\n",
      "Document 5-(7-(4-(4,5-dihydro-2-oxazolyl)phenoxy)heptyl)-3-methyl isoxazole - similarity: 0.74599\n",
      "Document Bos taurus gallstone - similarity: 0.74581\n",
      "Document Ferrous fumarate - similarity: 0.74568\n",
      "Document Carya laciniosa pollen - similarity: 0.74563\n",
      "Document GLPG-0974 - similarity: 0.74558\n",
      "Document 4-ethyl-5-methyl-2-(1H-tetrazol-5-yl)-1,2-dihydro-3H-pyrazol-3-one - similarity: 0.74554\n",
      "Document 3-Acetylpyridine Adenine Dinucleotide - similarity: 0.74539\n",
      "Document Cabotegravir - similarity: 0.74531\n",
      "Document AZD-1656 - similarity: 0.74494\n",
      "Document Tedizolid phosphate - similarity: 0.74491\n",
      "Document Tetramethrin - similarity: 0.74491\n",
      "Document Bucetin - similarity: 0.74466\n",
      "Document Vernakalant - similarity: 0.74454\n",
      "Document 4-[(7R,7AS)-7-HYDROXY-1,3-DIOXOTETRAHYDRO-1H-PYRROLO[1,2-C]IMIDAZOL-2(3H)-YL]-1-NAPHTHONITRILE - similarity: 0.74447\n",
      "Document Somatrem - similarity: 0.74421\n",
      "Document 7-PYRIDIN-2-YL-N-(3,4,5-TRIMETHOXYPHENYL)-7H-PYRROLO[2,3-D]PYRIMIDIN-2-AMINE - similarity: 0.74412\n",
      "Document N-({[4-(AMINOSULFONYL)PHENYL]AMINO}CARBONYL)-4-METHYLBENZENESULFONAMIDE - similarity: 0.74392\n",
      "Document Crotalus scutulatus antivenin - similarity: 0.74388\n",
      "Document R411 - similarity: 0.74387\n",
      "Document 2-((4'-HYDROXYNAPHTHYL)-AZO)BENZOIC ACID - similarity: 0.74386\n",
      "Document Lampalizumab - similarity: 0.74382\n",
      "Document 2,3-dichloro-5,6-dicyanobenzoquinone - similarity: 0.74379\n",
      "Document Bibrocathol - similarity: 0.74367\n",
      "Document (Z)-3-BENZYL-5-(2-HYDROXY-3-NITROBENZYLIDENE)-2-THIOXOTHIAZOLIDIN-4-ONE - similarity: 0.74363\n",
      "Document Carumonam - similarity: 0.74340\n",
      "Document Prednisone - similarity: 0.74333\n",
      "Document Palomid 529 - similarity: 0.74306\n",
      "Document Cytisus scoparius flowering top - similarity: 0.74297\n",
      "Document Juglans regia shell - similarity: 0.74284\n",
      "Document Neltenexine - similarity: 0.74261\n",
      "Document 1-(2,3-dihydro-1,4-benzodioxin-6-ylsulfonyl)-4-[(4-methoxyphenyl)sulfonyl]piperazine - similarity: 0.74251\n",
      "Document Azithromycin - similarity: 0.74244\n",
      "Document 1-(c14-c18 esteroyl)-2-docosahexanoyl-sn-glycero-3-phosphoethanolamine - similarity: 0.74242\n",
      "Document Ustilago nuda hordei - similarity: 0.74220\n",
      "Document Gadofosveset trisodium - similarity: 0.74180\n",
      "Document Technetium Tc-99m exametazime - similarity: 0.74177\n",
      "Document Coccidioides immitis spherule - similarity: 0.74154\n",
      "Document AMG-232 - similarity: 0.74153\n",
      "Document (S)-2-Amino-4-[(2s,3r)-2,3,5-Trihydroxy-4-Oxo-Pentyl]Mercapto-Butyric Acid - similarity: 0.74152\n",
      "Document (7as,12ar,12bs)-1,2,3,4,7a,12,12a,12b-Octahydroindolo[2,3-a]Quinolizin-7(6h)-One - similarity: 0.74122\n",
      "Document Tesamorelin - similarity: 0.74118\n",
      "Document Muraglitazar - similarity: 0.74112\n",
      "Document Exametazime - similarity: 0.74103\n",
      "Document Ubiquinol - similarity: 0.74087\n",
      "Document Ontamalimab - similarity: 0.74085\n",
      "Document Phoma destructiva - similarity: 0.74075\n",
      "Document Deracoxib - similarity: 0.74072\n",
      "Document AGN-201904 - similarity: 0.74061\n",
      "Document Gefarnate - similarity: 0.74047\n",
      "Document 4-HYDROXYBENZALDEHYDE O-(3,3-DIMETHYLBUTANOYL)OXIME - similarity: 0.74047\n",
      "Document Cintredekin Besudotox - similarity: 0.74043\n",
      "Document Repinotan - similarity: 0.74025\n",
      "Document Thio-Maltohexaose - similarity: 0.74022\n",
      "Document Crotalus adamanteus antivenin - similarity: 0.74006\n",
      "Document 9-(N-methyl-L-isoleucine)-cyclosporin A - similarity: 0.73993\n",
      "Document 1-[[(1E)-2-(4-CHLOROPHENYL)ETHENYL]SULFONYL]-4-[[1-(4-PYRIDINYL)-4-PIPERIDINYL]METHYL]PIPERAZINE - similarity: 0.73978\n",
      "Document 8-(6-BROMO-BENZO[1,3]DIOXOL-5-YLSULFANYL)-9-(3-ISOPROPYLAMINO-PROPYL)-ADENINE - similarity: 0.73965\n",
      "Document D-Levofloxacin - similarity: 0.73962\n",
      "Document Deslorelin - similarity: 0.73961\n",
      "Document Desoximetasone - similarity: 0.73961\n",
      "Document Pramlintide - similarity: 0.73930\n",
      "Document EW-A-401 - similarity: 0.73910\n",
      "Document Spartalizumab - similarity: 0.73910\n",
      "Document Dimethoxanate - similarity: 0.73905\n",
      "Document NB-001 - similarity: 0.73903\n",
      "Document Bornaprine - similarity: 0.73903\n",
      "Document Selicrelumab - similarity: 0.73900\n",
      "Document Poppy seed - similarity: 0.73897\n",
      "Document Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 antigen (UV, formaldehyde inactivated) - similarity: 0.73864\n",
      "Document Peldesine - similarity: 0.73862\n",
      "Document Atracurium besylate - similarity: 0.73858\n",
      "Document (1,2,6,7-3H)Testosterone - similarity: 0.73854\n",
      "Document Arzoxifene - similarity: 0.73853\n",
      "Document Tadekinig alfa - similarity: 0.73843\n",
      "Document N-Pyridoxyl-Threonine-5-Monophosphate - similarity: 0.73839\n",
      "Document (3-Formyl-but-3-Enyl)-Phosphonic Acid - similarity: 0.73837\n",
      "Document 3-(1H-tetrazol-5-ylamino)cyclohex-2-en-1-one - similarity: 0.73830\n",
      "Document 3-AMINO-3-BENZYL-9-CARBOXAMIDE[4.3.0]BICYCLO-1,6-DIAZANONAN-2-ONE - similarity: 0.73825\n",
      "Document N-sulfoglucosamine sulfohydrolase recombinant - similarity: 0.73809\n",
      "Document Alpha-Methylene Adenosine Monophosphate - similarity: 0.73799\n",
      "Document Albiglutide - similarity: 0.73783\n",
      "Document Bradanicline - similarity: 0.73776\n",
      "Document Gisadenafil - similarity: 0.73739\n",
      "Document Diloxanide furoate - similarity: 0.73736\n",
      "Document 6-CHLORO-3-(DICHLOROMETHYL)-3,4-DIHYDRO-2H-1,2,4-BENZOTHIADIAZINE-7-SULFONAMIDE 1,1-DIOXIDE - similarity: 0.73734\n",
      "Document Octyldodecanol - similarity: 0.73731\n",
      "Document Cilofexor - similarity: 0.73725\n",
      "Document Palladium Pd-103 - similarity: 0.73708\n",
      "Document Pomalidomide - similarity: 0.73702\n",
      "Document 2-[(1S)-1-BENZYL-2-SULFANYLETHYL]-1H-IMIDAZO[4,5-C]PYRIDIN-5-IUM - similarity: 0.73698\n",
      "Document Midecamycin - similarity: 0.73686\n",
      "Document Moxonidine - similarity: 0.73664\n",
      "Document Hexocyclium - similarity: 0.73661\n",
      "Document 6-(2,6-dibromophenyl)pyrido[2,3-d]pyrimidine-2,7-diamine - similarity: 0.73658\n",
      "Document Lobeglitazone - similarity: 0.73647\n",
      "Document Halcinonide - similarity: 0.73638\n",
      "Document Rescinnamine - similarity: 0.73637\n",
      "Document Rupatadine - similarity: 0.73619\n",
      "Document 2-(acetylamino)-2-deoxy-4-O-beta-D-galactopyranosyl-alpha-D-glucopyranose - similarity: 0.73614\n",
      "Document Eluxadoline - similarity: 0.73614\n",
      "Document Icaridin - similarity: 0.73614\n",
      "Document Lithium carbonate - similarity: 0.73611\n",
      "Document 1-(2-Amidinophenyl)-3-(Phenoxyphenyl)Urea - similarity: 0.73595\n",
      "Document Influenza B virus B/Brisbane/60/2008 hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.73586\n",
      "Document Tenecteplase - similarity: 0.73572\n",
      "Document Clazakizumab - similarity: 0.73567\n",
      "Document Eleclazine - similarity: 0.73560\n",
      "Document UNDECA-3,7-DIENE-1,3,7,11-TETRACARBALDEHYDE - similarity: 0.73558\n",
      "Document IMO-2055 - similarity: 0.73555\n",
      "Document Chlorambucil - similarity: 0.73550\n",
      "Document Moxifloxacin - similarity: 0.73513\n",
      "Document 5-methyltetrahydrofolic acid - similarity: 0.73493\n",
      "Document Fluorescein lisicol - similarity: 0.73493\n",
      "Document Metamfetamine - similarity: 0.73457\n",
      "Document AZD-1981 - similarity: 0.73434\n",
      "Document Menaquinone 6 - similarity: 0.73424\n",
      "Document Anecortave - similarity: 0.73415\n",
      "Document Lirilumab - similarity: 0.73410\n",
      "Document Clevudine - similarity: 0.73405\n",
      "Document Ioflupane I-123 - similarity: 0.73398\n",
      "Document (2R,3R,4S)-3-(4-HYDROXYPHENYL)-4-METHYL-2-[4-(2-PYRROLIDIN-1-YLETHOXY)PHENYL]CHROMAN-6-OL - similarity: 0.73396\n",
      "Document Vorozole - similarity: 0.73394\n",
      "Document Clomethiazole - similarity: 0.73391\n",
      "Document Influenza B virus B/Hong Kong/259/2010 antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.73365\n",
      "Document Ioglicic acid - similarity: 0.73360\n",
      "Document Ambrosia chamissonis pollen - similarity: 0.73357\n",
      "Document Cinchophen - similarity: 0.73351\n",
      "Document Metoserpate - similarity: 0.73339\n",
      "Document Idasanutlin - similarity: 0.73334\n",
      "Document Avotermin - similarity: 0.73332\n",
      "Document Ferumoxytol - similarity: 0.73327\n",
      "Document TNP-470 - similarity: 0.73310\n",
      "Document Diiodohydroxypropane - similarity: 0.73310\n",
      "Document AMG 108 - similarity: 0.73307\n",
      "Document Heptaethylene glycol - similarity: 0.73304\n",
      "Document Veltuzumab - similarity: 0.73303\n",
      "Document Kitasamycin - similarity: 0.73282\n",
      "Document Cyd dengue virus serotype 3 live (attenuated) antigen - similarity: 0.73273\n",
      "Document Streptococcus pneumoniae type 11a capsular polysaccharide antigen - similarity: 0.73268\n",
      "Document ATX-914 - similarity: 0.73267\n",
      "Document Methylergometrine - similarity: 0.73260\n",
      "Document Methylatropine - similarity: 0.73248\n",
      "Document (Carboxyhydroxyamino)Ethanoic Acid - similarity: 0.73212\n",
      "Document Liothyronine - similarity: 0.73210\n",
      "Document Phalaris arundinacea pollen - similarity: 0.73209\n",
      "Document Luliconazole - similarity: 0.73207\n",
      "Document Human adenovirus e serotype 4 strain cl-68578 antigen - similarity: 0.73177\n",
      "Document Polysorbate 60 - similarity: 0.73168\n",
      "Document SRT-2104 - similarity: 0.73162\n",
      "Document 2-(2-CHLORO-4-FLUOROPHENOXY)-2-METHYL-N-[(1R,2S,3S,5S,7S)-5-(METHYLSULFONYL)-2-ADAMANTYL]PROPANAMIDE - similarity: 0.73160\n",
      "Document Cenersen - similarity: 0.73155\n",
      "Document 1-{3-oxo-3-[(2S)-2-(pyrrolidin-1-ylcarbonyl)pyrrolidin-1-yl]propyl}-3-phenylquinoxalin-2(1H)-one - similarity: 0.73155\n",
      "Document Thiopropazate - similarity: 0.73154\n",
      "Document Chlorotoxin I-131 - similarity: 0.73153\n",
      "Document N-(4-chlorophenyl)-2-[(pyridin-4-ylmethyl)amino]benzamide - similarity: 0.73151\n",
      "Document Regadenoson - similarity: 0.73146\n",
      "Document Ulimorelin - similarity: 0.73144\n",
      "Document (3R)-METHYLCARBAMOYL-7-SULFOAMINO-3,4-DIHYDRO-1H-ISOQUINOLINE-2-CARBOXYLIC ACID BENZYL ESTER - similarity: 0.73139\n",
      "Document Cefalonium - similarity: 0.73137\n",
      "Document Inositol - similarity: 0.73134\n",
      "Document Cimetropium - similarity: 0.73132\n",
      "Document IMG-7289 - similarity: 0.73108\n",
      "Document Carbamide peroxide - similarity: 0.73070\n",
      "Document Difenpiramide - similarity: 0.73059\n",
      "Document METHYL (2Z)-3-METHOXY-2-{2-[(E)-2-PHENYLVINYL]PHENYL}ACRYLATE - similarity: 0.73057\n",
      "Document Dihydroergocornine - similarity: 0.73047\n",
      "Document Sitamaquine - similarity: 0.73033\n",
      "Document Mirtazapine - similarity: 0.73028\n",
      "Document Ferrous succinate - similarity: 0.73025\n",
      "Document N-[1-(5-bromo-2,3-dimethoxybenzyl)piperidin-4-yl]-4-sulfanylbutanamide - similarity: 0.73023\n",
      "Document Penimepicycline - similarity: 0.73013\n",
      "Document Pivampicillin - similarity: 0.73011\n",
      "Document Atipamezole - similarity: 0.73008\n",
      "Document IGN301 - similarity: 0.73007\n",
      "Document Lestaurtinib - similarity: 0.72968\n",
      "Document Bromopride - similarity: 0.72963\n",
      "Document O1-Methyl-4-Deoxy-4-Thio-Alpha-D-Glucose - similarity: 0.72961\n",
      "Document Thiotepa - similarity: 0.72946\n",
      "Document PSI-697 - similarity: 0.72944\n",
      "Document Quinagolide - similarity: 0.72931\n",
      "Document Selexipag - similarity: 0.72906\n",
      "Document Pinus palustris pollen - similarity: 0.72902\n",
      "Document Tislelizumab - similarity: 0.72899\n",
      "Document Canis lupus familiaris skin - similarity: 0.72895\n",
      "Document 6-(2-fluorobenzyl)-2,4-dimethyl-4,6-dihydro-5H-thieno[2',3':4,5]pyrrolo[2,3-d]pyridazin-5-one - similarity: 0.72889\n",
      "Document Isopentyl 2-cyanoacrylate - similarity: 0.72885\n",
      "Document Aprindine - similarity: 0.72866\n",
      "Document Quizartinib - similarity: 0.72864\n",
      "Document MDX-1379 - similarity: 0.72863\n",
      "Document Fluspirilene - similarity: 0.72859\n",
      "Document (2S)-1-(3H-Indol-3-yl)-3-{[5-(6-isoquinolinyl)-3-pyridinyl]oxy}-2-propanamine - similarity: 0.72856\n",
      "Document Simtuzumab - similarity: 0.72851\n",
      "Document Disitertide - similarity: 0.72850\n",
      "Document Pozelimab - similarity: 0.72843\n",
      "Document Cefbuperazone - similarity: 0.72834\n",
      "Document Benzimate - similarity: 0.72833\n",
      "Document Omidenepag isopropyl - similarity: 0.72830\n",
      "Document (1S,3R,4S,5S,7S)-4-{[2-(4-METHOXYPHENOXY)-2-METHYLPROPANOYL]AMINO}ADAMANTANE-1-CARBOXAMIDE - similarity: 0.72829\n",
      "Document Streptococcus pneumoniae type 1 capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.72826\n",
      "Document (2S)-2,3-Diaminobutanoic acid - similarity: 0.72809\n",
      "Document GC-24 - similarity: 0.72802\n",
      "Document Equus caballus skin - similarity: 0.72796\n",
      "Document Levoketoconazole - similarity: 0.72782\n",
      "Document Flupentixol - similarity: 0.72778\n",
      "Document CYC-065 - similarity: 0.72771\n",
      "Document Azatadine - similarity: 0.72746\n",
      "Document Buquinolate - similarity: 0.72746\n",
      "Document Sprifermin - similarity: 0.72735\n",
      "Document Perifosine - similarity: 0.72733\n",
      "Document Eliprodil - similarity: 0.72730\n",
      "Document Verticillium albo-atrum - similarity: 0.72723\n",
      "Document Rimegepant - similarity: 0.72718\n",
      "Document Sitagliptin - similarity: 0.72714\n",
      "Document Medazepam - similarity: 0.72704\n",
      "Document Doxofylline - similarity: 0.72680\n",
      "Document Methscopolamine bromide - similarity: 0.72672\n",
      "Document PPL-100 - similarity: 0.72672\n",
      "Document Cerliponase alfa - similarity: 0.72651\n",
      "Document Sucralfate - similarity: 0.72638\n",
      "Document SLx-4090 - similarity: 0.72616\n",
      "Document Ferric subsulfate - similarity: 0.72609\n",
      "Document Ethiodized oil - similarity: 0.72603\n",
      "Document N-(3-chlorophenyl)-N-methyl-2-oxo-3-[(3,4,5-trimethyl-1H-pyrrol-2-yl)methyl]-2H-indole-5-sulfonamide - similarity: 0.72602\n",
      "Document 3-[5-(2-nitropent-1-en-1-yl)furan-2-yl]benzoic acid - similarity: 0.72594\n",
      "Document Tienilic acid - similarity: 0.72590\n",
      "Document Brensocatib - similarity: 0.72576\n",
      "Document XL765 - similarity: 0.72564\n",
      "Document Apoptone - similarity: 0.72563\n",
      "Document N-(4-CHLOROPHENYL)-1,2,3,4-TETRAHYDROISOQUINOLINE-7-SULFONAMIDE - similarity: 0.72553\n",
      "Document Guacetisal - similarity: 0.72538\n",
      "Document Ketotifen - similarity: 0.72526\n",
      "Document Palifosfamide - similarity: 0.72506\n",
      "Document Cefazedone - similarity: 0.72501\n",
      "Document Caplacizumab - similarity: 0.72496\n",
      "Document Ropivacaine - similarity: 0.72496\n",
      "Document Curcuma xanthorrhiza oil - similarity: 0.72492\n",
      "Document Dexchlorpheniramine - similarity: 0.72478\n",
      "Document Ambroxol acefyllinate - similarity: 0.72469\n",
      "Document Flax seed - similarity: 0.72467\n",
      "Document Insulin glulisine - similarity: 0.72465\n",
      "Document C11-12 isoparaffin - similarity: 0.72463\n",
      "Document GET-73 - similarity: 0.72449\n",
      "Document G207 - similarity: 0.72440\n",
      "Document Methantheline - similarity: 0.72431\n",
      "Document Efaproxiral - similarity: 0.72403\n",
      "Document Calcium pangamate - similarity: 0.72389\n",
      "Document Lixisenatide - similarity: 0.72388\n",
      "Document Lauryl glucoside - similarity: 0.72377\n",
      "Document DL-alpha tocopheryl acetate - similarity: 0.72366\n",
      "Document Levofloxacin - similarity: 0.72366\n",
      "Document PEV3A - similarity: 0.72356\n",
      "Document Shark liver oil - similarity: 0.72355\n",
      "Document Pegbelfermin - similarity: 0.72342\n",
      "Document Isosulfan blue - similarity: 0.72342\n",
      "Document Tretoquinol - similarity: 0.72338\n",
      "Document Prenoxdiazine - similarity: 0.72321\n",
      "Document Q-Derp1 - similarity: 0.72301\n",
      "Document Pantethine - similarity: 0.72296\n",
      "Document IOWH-032 - similarity: 0.72285\n",
      "Document Filorexant - similarity: 0.72284\n",
      "Document N-BENZYLOXYCARBONYL-ALA-PRO-3-AMINO-4-PHENYL-BUTAN-2-OL - similarity: 0.72274\n",
      "Document Methysergide - similarity: 0.72249\n",
      "Document Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.72230\n",
      "Document Chromium nicotinate - similarity: 0.72230\n",
      "Document Lodoxamide - similarity: 0.72216\n",
      "Document Thermomyces lanuginosus - similarity: 0.72211\n",
      "Document Doxylamine - similarity: 0.72205\n",
      "Document Efpeglenatide - similarity: 0.72205\n",
      "Document AZD-3759 - similarity: 0.72198\n",
      "Document R1204 - similarity: 0.72192\n",
      "Document ORTHONITROPHENYL-BETA-D-FUCOPYRANOSIDE - similarity: 0.72189\n",
      "Document Ferrous bisglycinate - similarity: 0.72177\n",
      "Document Technetium Tc-99m sestamibi - similarity: 0.72164\n",
      "Document Oxymetazoline - similarity: 0.72151\n",
      "Document Ergometrine - similarity: 0.72141\n",
      "Document Methscopolamine - similarity: 0.72118\n",
      "Document Ferric pyrophosphate - similarity: 0.72100\n",
      "Document VX-702 - similarity: 0.72083\n",
      "Document Lifibrol - similarity: 0.72074\n",
      "Document Ethyl dihydrogen phosphate - similarity: 0.72073\n",
      "Document Carya illinoinensis pollen - similarity: 0.72060\n",
      "Document Ivacaftor - similarity: 0.72057\n",
      "Document Tazarotene - similarity: 0.72052\n",
      "Document Xenon Xe-127 - similarity: 0.72047\n",
      "Document MF101 - similarity: 0.72043\n",
      "Document Atractylodes japonica root oil - similarity: 0.72018\n",
      "Document Quarfloxin - similarity: 0.72018\n",
      "Document Flotetuzumab - similarity: 0.72008\n",
      "Document Brofaromine - similarity: 0.72007\n",
      "Document Levovirin - similarity: 0.72007\n",
      "Document Olsalazine - similarity: 0.72002\n",
      "Document VTP-27999 - similarity: 0.71995\n",
      "Document Bimagrumab - similarity: 0.71991\n",
      "Document N-[(1S)-5-amino-1-(chloroacetyl)pentyl]-4-methylbenzenesulfonamide - similarity: 0.71985\n",
      "Document Lymecycline - similarity: 0.71978\n",
      "Document Estradiol sulfamate - similarity: 0.71967\n",
      "Document Pyrophosphoric acid - similarity: 0.71960\n",
      "Document Sodium phosphate P 32 - similarity: 0.71948\n",
      "Document Propyl alcohol - similarity: 0.71928\n",
      "Document Testosterone enanthate - similarity: 0.71921\n",
      "Document Aripiprazole lauroxil - similarity: 0.71916\n",
      "Document Concord grape - similarity: 0.71915\n",
      "Document Zilpaterol - similarity: 0.71906\n",
      "Document (3R,4R,5R)-5-(HYDROXYMETHYL)-1-(3-PHENYLPROPYL)PIPERIDINE-3,4-DIOL - similarity: 0.71896\n",
      "Document Threonine - similarity: 0.71890\n",
      "Document Tritoqualine - similarity: 0.71889\n",
      "Document Sodium tartrate - similarity: 0.71886\n",
      "Document Pentamidine - similarity: 0.71859\n",
      "Document Terameprocol - similarity: 0.71836\n",
      "Document Clozapine - similarity: 0.71824\n",
      "Document Cilansetron - similarity: 0.71817\n",
      "Document Meloxicam - similarity: 0.71817\n",
      "Document Dacetuzumab - similarity: 0.71801\n",
      "Document Juniperus virginiana pollen - similarity: 0.71800\n",
      "Document 2-AMINO-4-FLUORO-5-[(1-METHYL-1H-IMIDAZOL-2-YL)SULFANYL]-N-(1,3-THIAZOL-2-YL)BENZAMIDE - similarity: 0.71790\n",
      "Document Ropeginterferon alfa-2b - similarity: 0.71775\n",
      "Document Vicriviroc - similarity: 0.71760\n",
      "Document Soybean - similarity: 0.71749\n",
      "Document Fimasartan - similarity: 0.71748\n",
      "Document Fluorometholone - similarity: 0.71737\n",
      "Document Coproporphyrin I - similarity: 0.71721\n",
      "Document Lenomorelin - similarity: 0.71717\n",
      "Document Nemolizumab - similarity: 0.71709\n",
      "Document Esreboxetine - similarity: 0.71695\n",
      "Document Simenepag isopropyl - similarity: 0.71693\n",
      "Document Benziodarone - similarity: 0.71671\n",
      "Document Razupenem - similarity: 0.71666\n",
      "Document TA-NIC - similarity: 0.71662\n",
      "Document 1,1,1-TRIFLUORO-3-ACETAMIDO-4-PHENYL BUTAN-2-ONE(N-ACETYL-L-PHENYLALANYL TRIFLUOROMETHYL KETONE) - similarity: 0.71660\n",
      "Document JNJ-37822681 - similarity: 0.71659\n",
      "Document 3-(3,5-Dibromo-4-Hydroxy-Benzoyl)-2-Ethyl-Benzofuran-6-Sulfonic Acid (4-Sulfamoyl-Phenyl)-Amide - similarity: 0.71651\n",
      "Document 4-[[2-[[4-chloro-3-(trifluoromethyl)phenyl]amino]-3H-benzimidazol-5-yl]oxy]-N-methyl-pyridine-2-carboxamide - similarity: 0.71644\n",
      "Document Serdemetan - similarity: 0.71643\n",
      "Document NS-3728 - similarity: 0.71632\n",
      "Document PF-4191834 - similarity: 0.71629\n",
      "Document AEOL 10150 - similarity: 0.71628\n",
      "Document Tamibarotene - similarity: 0.71609\n",
      "Document 4-BROMO-3-(CARBOXYMETHOXY)-5-PHENYLTHIOPHENE-2-CARBOXYLIC ACID - similarity: 0.71591\n",
      "Document Atlantic halibut - similarity: 0.71587\n",
      "Document Ligelizumab - similarity: 0.71578\n",
      "Document Fish liver oil - similarity: 0.71567\n",
      "Document Roquinimex - similarity: 0.71553\n",
      "Document Alvelestat - similarity: 0.71553\n",
      "Document 4-(1,3-BENZOXAZOL-2-YL)-2,6-DIMETHYLPHENOL - similarity: 0.71548\n",
      "Document Glial growth factor 2 - similarity: 0.71545\n",
      "Document Amorolfine - similarity: 0.71540\n",
      "Document Tocopherylquinone - similarity: 0.71539\n",
      "Document Clebopride - similarity: 0.71535\n",
      "Document Hydroxyprogesterone - similarity: 0.71527\n",
      "Document 2-{2-hydroxy-[1,1'-biphenyl]-3-yl}-1H-1,3-benzodiazole-5-carboximidamide - similarity: 0.71479\n",
      "Document Vofopitant - similarity: 0.71474\n",
      "Document Rivoglitazone - similarity: 0.71472\n",
      "Document Viltolarsen - similarity: 0.71466\n",
      "Document Betula pendula pollen - similarity: 0.71464\n",
      "Document (S)-N-(4-carbamimidoylbenzyl)-1-(2-(cyclohexylamino)ethanoyl)pyrrolidine-2-carboxamide - similarity: 0.71456\n",
      "Document 4-[(6-phenyl[1,2,4]triazolo[4,3-b]pyridazin-3-yl)methyl]phenol - similarity: 0.71454\n",
      "Document PAC-14028 - similarity: 0.71449\n",
      "Document 1alpha-Hydroxyvitamin D5 - similarity: 0.71446\n",
      "Document Trimebutine - similarity: 0.71445\n",
      "Document Technetium Tc-99m - similarity: 0.71439\n",
      "Document Hexachlorophene - similarity: 0.71436\n",
      "Document Piritramide - similarity: 0.71429\n",
      "Document Chlorothiazide - similarity: 0.71428\n",
      "Document Amikacin - similarity: 0.71426\n",
      "Document PRO 2000 - similarity: 0.71424\n",
      "Document Meticrane - similarity: 0.71416\n",
      "Document O-[2-(1,3-dioxo-1,3-dihydro-2H-isoindol-2-yl)ethyl] (4-bromophenyl)thiocarbamate - similarity: 0.71413\n",
      "Document Myristyl glyceryl ether - similarity: 0.71412\n",
      "Document Hydrocortisone acetate - similarity: 0.71388\n",
      "Document 5-Methoxy-N,N-diisopropyltryptamine - similarity: 0.71383\n",
      "Document 3-Tyrosine - similarity: 0.71373\n",
      "Document Sodium citrate - similarity: 0.71372\n",
      "Document Cycloleucine - similarity: 0.71368\n",
      "Document 2-[(2-methoxyethyl)amino]-4-(4-oxo-1,2,3,4-tetrahydro-9H-carbazol-9-yl)benzamide - similarity: 0.71355\n",
      "Document Equus caballus dander - similarity: 0.71347\n",
      "Document Certolizumab pegol - similarity: 0.71343\n",
      "Document Pimonidazole - similarity: 0.71336\n",
      "Document Rimantadine - similarity: 0.71335\n",
      "Document 1-(5-CHLORO-2,4-DIMETHOXYPHENYL)-3-(5-CYANOPYRAZIN-2-YL)UREA - similarity: 0.71327\n",
      "Document 2,6-Dimethyl-7-Octen-2-Ol - similarity: 0.71294\n",
      "Document Oxiconazole - similarity: 0.71290\n",
      "Document Imidafenacin - similarity: 0.71273\n",
      "Document Cinchocaine - similarity: 0.71255\n",
      "Document XTL-6865 - similarity: 0.71242\n",
      "Document P-Tert-Butylphenol-Formaldehyde Resin (Low Molecular Weight) - similarity: 0.71229\n",
      "Document Cetrorelix - similarity: 0.71223\n",
      "Document Citarinostat - similarity: 0.71207\n",
      "Document Sodium ascorbate - similarity: 0.71206\n",
      "Document Oteseconazole - similarity: 0.71203\n",
      "Document PF-06291874 - similarity: 0.71200\n",
      "Document Benzonatate - similarity: 0.71199\n",
      "Document Bortezomib - similarity: 0.71199\n",
      "Document Ixazomib - similarity: 0.71194\n",
      "Document Dosulepin - similarity: 0.71180\n",
      "Document (2S,3S)-3-AMINO-4-(3,3-DIFLUOROPYRROLIDIN-1-YL)-N,N-DIMETHYL-4-OXO-2-(TRANS-4-[1,2,4]TRIAZOLO[1,5-A]PYRIDIN-6-YLCYCLOHEXYL)BUTANAMIDE - similarity: 0.71170\n",
      "Document Monoxerutin - similarity: 0.71170\n",
      "Document Trafermin - similarity: 0.71166\n",
      "Document Azaperone - similarity: 0.71159\n",
      "Document Narnatumab - similarity: 0.71151\n",
      "Document (3Z,5S,6R,7S,8S,8aR)-3-(octylimino)hexahydro[1,3]oxazolo[3,4-a]pyridine-5,6,7,8-tetrol - similarity: 0.71139\n",
      "Document PRM-151 - similarity: 0.71123\n",
      "Document Modimelanotide - similarity: 0.71120\n",
      "Document Pumosetrag - similarity: 0.71109\n",
      "Document Pegteograstim - similarity: 0.71107\n",
      "Document CF-102 - similarity: 0.71099\n",
      "Document Succimer - similarity: 0.71086\n",
      "Document Epofolate - similarity: 0.71080\n",
      "Document Jojoba oil - similarity: 0.71077\n",
      "Document Rapacuronium - similarity: 0.71076\n",
      "Document 5-[(1S)-1-(3-chlorophenyl)ethoxy]quinazoline-2,4-diamine - similarity: 0.71076\n",
      "Document LY-2456302 - similarity: 0.71074\n",
      "Document Interleukin-10 - similarity: 0.71071\n",
      "Document Ibrutinib - similarity: 0.71067\n",
      "Document Capra hircus skin - similarity: 0.71065\n",
      "Document Teneligliptin - similarity: 0.71057\n",
      "Document Azficel-T - similarity: 0.71054\n",
      "Document 7,8-dihydro-7,7-dimethyl-6-hydroxypterin - similarity: 0.71028\n",
      "Document Haemophilus influenzae type B strain 20752 capsular polysaccharide tetanus toxoid conjugate antigen - similarity: 0.71024\n",
      "Document Lecithin, soybean - similarity: 0.71010\n",
      "Document Cochliobolus spicifer - similarity: 0.71009\n",
      "Document Azacitidine - similarity: 0.71008\n",
      "Document Prazepam - similarity: 0.71000\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.70995\n",
      "Document Lactulose - similarity: 0.70991\n",
      "Document anti-alpha5Beta1-integrin antibody - similarity: 0.70987\n",
      "Document Populus balsamifera subsp. trichocarpa pollen - similarity: 0.70984\n",
      "Document Mogamulizumab - similarity: 0.70967\n",
      "Document (E)-3,4-DIHYDROXY-N'-[(2-METHOXYNAPHTHALEN-1-YL)METHYLENE]BENZOHYDRAZIDE - similarity: 0.70964\n",
      "Document Thymectacin - similarity: 0.70959\n",
      "Document Dimemorfan - similarity: 0.70959\n",
      "Document 3-{[(3-FLUORO-3'-METHOXYBIPHENYL-4-YL)AMINO]CARBONYL}THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.70955\n",
      "Document Iopydol - similarity: 0.70938\n",
      "Document beta-D-glucuronic acid - similarity: 0.70937\n",
      "Document Hopantenic acid - similarity: 0.70924\n",
      "Document Cyclobarbital - similarity: 0.70922\n",
      "Document N,N-DIMETHYL-4-(4-PHENYL-1H-PYRAZOL-3-YL)-1H-PYRROLE-2-CARBOXAMIDE - similarity: 0.70919\n",
      "Document Henatinib - similarity: 0.70908\n",
      "Document Propacetamol - similarity: 0.70901\n",
      "Document Acumapimod - similarity: 0.70883\n",
      "Document Medetomidine - similarity: 0.70882\n",
      "Document Copper usnate - similarity: 0.70876\n",
      "Document Cipargamin - similarity: 0.70875\n",
      "Document Carbenoxolone - similarity: 0.70854\n",
      "Document 5-[1-(4-methoxyphenyl)-1H-benzimidazol-6-yl]-1,3,4-oxadiazole-2(3H)-thione - similarity: 0.70845\n",
      "Document Dolastatin 10 - similarity: 0.70834\n",
      "Document AVE-2865 - similarity: 0.70826\n",
      "Document Iva annua pollen - similarity: 0.70819\n",
      "Document Vadimezan - similarity: 0.70815\n",
      "Document Dienestrol diacetate - similarity: 0.70807\n",
      "Document Benralizumab - similarity: 0.70804\n",
      "Document Fusarium oxysporum vasinfectum - similarity: 0.70796\n",
      "Document Puccinia graminis - similarity: 0.70793\n",
      "Document Tenofovir - similarity: 0.70788\n",
      "Document Pexidartinib - similarity: 0.70775\n",
      "Document Ensulizole - similarity: 0.70772\n",
      "Document AZD-9684 - similarity: 0.70761\n",
      "Document MIV-711 - similarity: 0.70754\n",
      "Document Mericitabine - similarity: 0.70751\n",
      "Document Streptococcus pneumoniae type 5 capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.70745\n",
      "Document TERT-BUTYL 4-({[4-(BUT-2-YN-1-YLAMINO)PHENYL]SULFONYL}METHYL)-4-[(HYDROXYAMINO)CARBONYL]PIPERIDINE-1-CARBOXYLATE - similarity: 0.70736\n",
      "Document 3-{(R)-(Dihydroxyboryl)[(2-thienylacetyl)amino]methyl}benzoic acid - similarity: 0.70732\n",
      "Document Granotapide - similarity: 0.70730\n",
      "Document Chlorbenzoxamine - similarity: 0.70712\n",
      "Document Esmirtazapine - similarity: 0.70708\n",
      "Document Cloranolol - similarity: 0.70700\n",
      "Document DAS-431 IV - similarity: 0.70695\n",
      "Document Metolazone - similarity: 0.70694\n",
      "Document Eutrema japonicum whole - similarity: 0.70689\n",
      "Document Tylvalosin - similarity: 0.70652\n",
      "Document IR502 - similarity: 0.70646\n",
      "Document 1-[4-(4-chlorobenzyl)-1-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)piperidin-4-yl]methanamine - similarity: 0.70642\n",
      "Document 4-methylumbelliferyl beta-D-glucoside - similarity: 0.70639\n",
      "Document Isoquinoline - similarity: 0.70628\n",
      "Document Dapiprazole - similarity: 0.70627\n",
      "Document (3R)-N-(4-CHLOROPHENYL)-3-(HYDROXYMETHYL)-1,2,3,4-TETRAHYDROISOQUINOLINE-7-SULFONAMIDE - similarity: 0.70615\n",
      "Document Dolasetron - similarity: 0.70613\n",
      "Document 2-{4-[2-(2-AMINO-4-OXO-4,7-DIHYDRO-3H-PYRROLO[2,3-D]PYRIMIDIN-5-YL)-ETHYL]-BENZOYLAMINO}-3-METHYL-BUTYRIC ACID - similarity: 0.70598\n",
      "Document (2S)-({(5Z)-5-[(5-Ethyl-2-furyl)methylene]-4-oxo-4,5-dihydro-1,3-thiazol-2-yl}amino)(4-fluorophenyl)acetic acid - similarity: 0.70594\n",
      "Document Synthetic Conjugated Estrogens, B - similarity: 0.70581\n",
      "Document Prifinium - similarity: 0.70576\n",
      "Document Enzastaurin - similarity: 0.70569\n",
      "Document Novobiocin - similarity: 0.70564\n",
      "Document Risankizumab - similarity: 0.70562\n",
      "Document Besifovir - similarity: 0.70558\n",
      "Document Gentamicin C1a - similarity: 0.70551\n",
      "Document Mebutizide - similarity: 0.70549\n",
      "Document Epelsiban - similarity: 0.70543\n",
      "Document Balovaptan - similarity: 0.70536\n",
      "Document Ixekizumab - similarity: 0.70530\n",
      "Document Becocalcidiol - similarity: 0.70519\n",
      "Document Deoxyspergualin - similarity: 0.70516\n",
      "Document Ormeloxifene - similarity: 0.70507\n",
      "Document Fezakinumab - similarity: 0.70504\n",
      "Document Pimodivir - similarity: 0.70503\n",
      "Document Lomustine - similarity: 0.70495\n",
      "Document Alsterpaullone - similarity: 0.70487\n",
      "Document PF-5190457 - similarity: 0.70484\n",
      "Document Light green SF yellowish - similarity: 0.70482\n",
      "Document Bis(5-Amidino-2-Benzimidazolyl)Methane Ketone - similarity: 0.70477\n",
      "Document Ramelteon - similarity: 0.70477\n",
      "Document Influenza A virus A/California/7/2009 (H1N1)-like hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.70471\n",
      "Document Difetarsone - similarity: 0.70464\n",
      "Document 5'-ACETYL-4-{[(2,4-DIMETHYLPHENYL)SULFONYL]AMINO}-2,2'-BITHIOPHENE-5-CARBOXYLIC ACID - similarity: 0.70451\n",
      "Document AZD-6280 - similarity: 0.70440\n",
      "Document Prochlorperazine - similarity: 0.70435\n",
      "Document Levetiracetam - similarity: 0.70432\n",
      "Document Articaine - similarity: 0.70426\n",
      "Document Zolazepam - similarity: 0.70425\n",
      "Document Midazolam - similarity: 0.70421\n",
      "Document Lepirudin - similarity: 0.70415\n",
      "Document AQX-1125 - similarity: 0.70406\n",
      "Document N-(1-BENZYL-3,3,3-TRIFLUORO-2,2-DIHYDROXY-PROPYL)-ACETAMIDE - similarity: 0.70395\n",
      "Document Alvespimycin - similarity: 0.70373\n",
      "Document Elsamitrucin - similarity: 0.70363\n",
      "Document Febuxostat - similarity: 0.70340\n",
      "Document Methyl-2-S-(Alpha-D-Mannopyranosyl)-2-Thio-Alpha-D-Mannopyranoside - similarity: 0.70335\n",
      "Document Artefenomel - similarity: 0.70328\n",
      "Document Von Willebrand Factor Human - similarity: 0.70324\n",
      "Document Tesofensine - similarity: 0.70322\n",
      "Document Peginterferon beta-1a - similarity: 0.70321\n",
      "Document Poliovirus type 3 antigen (formaldehyde inactivated) - similarity: 0.70319\n",
      "Document Rivipansel - similarity: 0.70319\n",
      "Document PM02734 - similarity: 0.70310\n",
      "Document N-ALLYL-5-AMIDINOAMINOOXY-PROPYLOXY-3-CHLORO-N-CYCLOPENTYLBENZAMIDE - similarity: 0.70310\n",
      "Document Biperiden - similarity: 0.70301\n",
      "Document 2-[N'-(4-AMINO-BUTYL)-HYDRAZINOCARBONYL]-PYRROLIDINE-1-CARBOXYLIC ACID BENZYL ESTER - similarity: 0.70299\n",
      "Document Oritavancin - similarity: 0.70286\n",
      "Document Carfecillin - similarity: 0.70284\n",
      "Document Carmegliptin - similarity: 0.70283\n",
      "Document Verubulin - similarity: 0.70282\n",
      "Document Equine Botulinum Neurotoxin B Immune FAB2 - similarity: 0.70278\n",
      "Document Capsicum oleoresin - similarity: 0.70276\n",
      "Document BI-671800 - similarity: 0.70274\n",
      "Document Azlocillin - similarity: 0.70254\n",
      "Document Timcodar - similarity: 0.70250\n",
      "Document Diphenylpyraline - similarity: 0.70249\n",
      "Document Parsatuzumab - similarity: 0.70246\n",
      "Document Difluprednate - similarity: 0.70239\n",
      "Document Bunazosin - similarity: 0.70234\n",
      "Document Poloxalene - similarity: 0.70229\n",
      "Document Hydronidone - similarity: 0.70219\n",
      "Document Vilazodone - similarity: 0.70200\n",
      "Document (3,4,5-Trihydroxy-6-Hydroxymethyl-Tetrahydro-Pyran-2-Yl)-Phosphoramidic Acid Dimethyl Ester - similarity: 0.70189\n",
      "Document Metaxalone - similarity: 0.70187\n",
      "Document Pradigastat - similarity: 0.70177\n",
      "Document Prolintane - similarity: 0.70160\n",
      "Document Ferrous gluconate - similarity: 0.70157\n",
      "Document Triamcinolone - similarity: 0.70151\n",
      "Document MK-0731 - similarity: 0.70148\n",
      "Document Gambir - similarity: 0.70147\n",
      "Document Tizanidine - similarity: 0.70143\n",
      "Document (2R)-N~4~-hydroxy-2-(3-hydroxybenzyl)-N~1~-[(1S,2R)-2-hydroxy-2,3-dihydro-1H-inden-1-yl]butanediamide - similarity: 0.70136\n",
      "Document Taxifolin - similarity: 0.70129\n",
      "Document 2-(Acetylamino)-2-Deoxy-6-O-Methyl-Alpha-D-Allopyranose - similarity: 0.70125\n",
      "Document Talniflumate - similarity: 0.70121\n",
      "Document XL019 - similarity: 0.70115\n",
      "Document AE-941 - similarity: 0.70115\n",
      "Document Topramezone - similarity: 0.70114\n",
      "Document 2'-deoxy-N-(naphthalen-1-ylmethyl)guanosine 5'-(dihydrogen phosphate) - similarity: 0.70096\n",
      "Document NM-3 - similarity: 0.70095\n",
      "Document Olanzapine - similarity: 0.70079\n",
      "Document (5,6-DIPHENYL-FURO[2,3-D]PYRIMIDIN-4-YLAMINO)-ACETIC - similarity: 0.70077\n",
      "Document Telotristat - similarity: 0.70075\n",
      "Document Tetrahydrodeoxyuridine - similarity: 0.70070\n",
      "Document Interferon alfa-2b - similarity: 0.70068\n",
      "Document Ozenoxacin - similarity: 0.70068\n",
      "Document Allobarbital - similarity: 0.70065\n",
      "Document Iron isomaltoside 1000 - similarity: 0.70058\n",
      "Document XMT-1001 - similarity: 0.70057\n",
      "Document Chromous sulfate - similarity: 0.70056\n",
      "Document Beloranib - similarity: 0.70047\n",
      "Document Butoconazole - similarity: 0.70046\n",
      "Document Leukotriene D4 - similarity: 0.70032\n",
      "Document (3S,6S)-3,6-bis(4-hydroxybenzyl)piperazine-2,5-dione - similarity: 0.70017\n",
      "Document 5-(7-(6-chloro-4-(5-hydro-4-methyl-2-oxazolyl)phenoxy)heptyl)-3-methyl isoxazole - similarity: 0.70009\n",
      "Document Inositol-(1,3,4,5,6)-Pentakisphosphate - similarity: 0.70007\n",
      "Document Cefotiam - similarity: 0.70006\n",
      "Document Cambendazole - similarity: 0.70001\n",
      "Document Tocainide - similarity: 0.69998\n",
      "Document Rivastigmine - similarity: 0.69996\n",
      "Document Atracurium - similarity: 0.69992\n",
      "Document Antipain - similarity: 0.69988\n",
      "Document Rifapentine - similarity: 0.69987\n",
      "Document 4-amino-N-[4-(benzyloxy)phenyl]butanamide - similarity: 0.69967\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) antigen (formaldehyde inactivated) - similarity: 0.69966\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Pamiparib - similarity: 0.69966\n",
      "Document 2'-(4-Dimethylaminophenyl)-5-(4-Methyl-1-Piperazinyl)-2,5'-Bi-Benzimidazole - similarity: 0.69963\n",
      "Document Nalorphine - similarity: 0.69941\n",
      "Document N-METHYL(5-(PYRIDIN-3-YL)FURAN-2-YL)METHANAMINE - similarity: 0.69937\n",
      "Document Maxacalcitol - similarity: 0.69925\n",
      "Document Cefpodoxime - similarity: 0.69907\n",
      "Document Tandutinib - similarity: 0.69898\n",
      "Document MBO7133 - similarity: 0.69897\n",
      "Document Arbekacin - similarity: 0.69886\n",
      "Document 5-{[(4-AMINO-3-CHLORO-5-FLUOROPHENYL)SULFONYL]AMINO}-1,3,4-THIADIAZOLE-2-SULFONAMIDE - similarity: 0.69882\n",
      "Document Pinaverium - similarity: 0.69872\n",
      "Document Edetate disodium anhydrous - similarity: 0.69871\n",
      "Document Sorghum bicolor subsp. drummondii pollen - similarity: 0.69866\n",
      "Document 1-(5-TERT-BUTYL-1,3,4-OXADIAZOL-2-YL)-2-(METHYLAMINO)ETHANONE - similarity: 0.69866\n",
      "Document Poloxamer 188 - similarity: 0.69853\n",
      "Document Eritoran - similarity: 0.69850\n",
      "Document Mepolizumab - similarity: 0.69845\n",
      "Document (5-(PYRIDIN-3-YL)FURAN-2-YL)METHANAMINE - similarity: 0.69841\n",
      "Document Rovalpituzumab Tesirine - similarity: 0.69835\n",
      "Document Lorazepam - similarity: 0.69831\n",
      "Document Chlorsulfaquinoxaline - similarity: 0.69830\n",
      "Document Erythroxylum catuaba bark - similarity: 0.69826\n",
      "Document Inebilizumab - similarity: 0.69809\n",
      "Document m-Chlorophenylpiperazine - similarity: 0.69808\n",
      "Document Aluminum chlorohydrex propylene glycol - similarity: 0.69805\n",
      "Document Flutrimazole - similarity: 0.69804\n",
      "Document Ravidasvir - similarity: 0.69796\n",
      "Document Influenza A virus A/South Dakota/6/2007 (H1N1) live (attenuated) antigen - similarity: 0.69793\n",
      "Document SNX-5422 - similarity: 0.69774\n",
      "Document Nordazepam - similarity: 0.69754\n",
      "Document 2-(2-hydroxy-phenyl)-3H-benzoimidazole-5-carboxamidine - similarity: 0.69752\n",
      "Document Zinc sulfate - similarity: 0.69750\n",
      "Document DL-Methylephedrine - similarity: 0.69750\n",
      "Document Estradiol benzoate - similarity: 0.69746\n",
      "Document Tenofovir alafenamide - similarity: 0.69743\n",
      "Document Anagliptin - similarity: 0.69740\n",
      "Document 1,2-Docosahexanoyl-sn-glycero-3-phosphoserine - similarity: 0.69735\n",
      "Document Activated charcoal - similarity: 0.69729\n",
      "Document ISS-1018 - similarity: 0.69728\n",
      "Document Prothrombin - similarity: 0.69719\n",
      "Document Tecarfarin - similarity: 0.69715\n",
      "Document 1-Methylhistidine - similarity: 0.69711\n",
      "Document Fasinumab - similarity: 0.69708\n",
      "Document Zoliflodacin - similarity: 0.69707\n",
      "Document Olopatadine - similarity: 0.69703\n",
      "Document Turkey - similarity: 0.69667\n",
      "Document 2-HYDROXYMETHYL-6-OCTYLSULFANYL-TETRAHYDRO-PYRAN-3,4,5-TRIOL - similarity: 0.69662\n",
      "Document 4-Methylimidazole - similarity: 0.69662\n",
      "Document N-[2-(METHYLAMINO)ETHYL]-5-ISOQUINOLINESULFONAMIDE - similarity: 0.69661\n",
      "Document Difluoromethionine - similarity: 0.69652\n",
      "Document 5-CYANO-FURAN-2-CARBOXYLIC ACID [5-HYDROXYMETHYL-2-(4-METHYL-PIPERIDIN-1-YL)-PHENYL]-AMIDE - similarity: 0.69648\n",
      "Document Zeranol - similarity: 0.69645\n",
      "Document Luseogliflozin - similarity: 0.69610\n",
      "Document Influenza A virus A/South Australia/55/2014 (H3N2) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.69605\n",
      "Document Lacnotuzumab - similarity: 0.69598\n",
      "Document Chicory leaf - similarity: 0.69579\n",
      "Document 4-METHYL-PENTANOIC ACID {1-[4-GUANIDINO-1-(THIAZOLE-2-CARBONYL)-BUTYLCARBAMOYL]-2-METHYL-PROPYL}-AMIDE - similarity: 0.69565\n",
      "Document Degarelix - similarity: 0.69551\n",
      "Document Filgotinib - similarity: 0.69548\n",
      "Document Fasiglifam - similarity: 0.69536\n",
      "Document Decyl(dimethyl)phosphine oxide - similarity: 0.69515\n",
      "Document Ispinesib - similarity: 0.69491\n",
      "Document UC-781 - similarity: 0.69477\n",
      "Document Savolitinib - similarity: 0.69471\n",
      "Document Albinterferon Alfa-2B - similarity: 0.69458\n",
      "Document Clocortolone caproate - similarity: 0.69457\n",
      "Document Leukotriene C4 - similarity: 0.69455\n",
      "Document 1-BENZYL-3-(4-METHOXYPHENYLAMINO)-4-PHENYLPYRROLE-2,5-DIONE - similarity: 0.69444\n",
      "Document Kallidinogenase - similarity: 0.69426\n",
      "Document Butylparaben - similarity: 0.69416\n",
      "Document Insulin lispro - similarity: 0.69403\n",
      "Document (3Z,5S,6R,7S,8R,8aR)-3-(octylimino)hexahydro[1,3]oxazolo[3,4-a]pyridine-5,6,7,8-tetrol - similarity: 0.69398\n",
      "Document Carbomer - similarity: 0.69395\n",
      "Document Mannose 6-phosphate - similarity: 0.69388\n",
      "Document Urea C-13 - similarity: 0.69386\n",
      "Document Sodium borate - similarity: 0.69380\n",
      "Document Pregnenolone - similarity: 0.69378\n",
      "Document Minoxidil - similarity: 0.69377\n",
      "Document Avilamycin - similarity: 0.69373\n",
      "Document Sea scallop - similarity: 0.69361\n",
      "Document bis(4-nitrophenyl) hydrogen phosphate - similarity: 0.69353\n",
      "Document Litronesib - similarity: 0.69344\n",
      "Document Ferrous glycine sulfate - similarity: 0.69341\n",
      "Document Ado-P-Ch2-P-Ps-Ado - similarity: 0.69338\n",
      "Document Metoclopramide - similarity: 0.69329\n",
      "Document 1-(1-phenylcyclopentyl)methylamine - similarity: 0.69328\n",
      "Document Glycine - similarity: 0.69305\n",
      "Document Ticlopidine - similarity: 0.69293\n",
      "Document Dofetilide - similarity: 0.69278\n",
      "Document Canis lupus familiaris dander - similarity: 0.69262\n",
      "Document Avexitide - similarity: 0.69258\n",
      "Document Amodiaquine - similarity: 0.69252\n",
      "Document Magnesium glycinate - similarity: 0.69244\n",
      "Document DECANE-1-THIOL - similarity: 0.69242\n",
      "Document Aldesleukin - similarity: 0.69239\n",
      "Document Dinoprost tromethamine - similarity: 0.69237\n",
      "Document Tamsulosin - similarity: 0.69222\n",
      "Document 4-AMINO-5-(2-METHYLPHENYL)-2,4-DIHYDRO-3H-1,2,4-TRIAZOLE-3-THIONE - similarity: 0.69220\n",
      "Document Ascorbyl phosphate - similarity: 0.69213\n",
      "Document Milatuzumab - similarity: 0.69191\n",
      "Document Empagliflozin - similarity: 0.69176\n",
      "Document Urtica dioica pollen - similarity: 0.69168\n",
      "Document Mephenytoin - similarity: 0.69144\n",
      "Document Wheat - similarity: 0.69129\n",
      "Document NUC-1031 - similarity: 0.69126\n",
      "Document Deutetrabenazine - similarity: 0.69120\n",
      "Document Droperidol - similarity: 0.69117\n",
      "Document Loteprednol etabonate - similarity: 0.69109\n",
      "Document Sapanisertib - similarity: 0.69099\n",
      "Document Pioglitazone - similarity: 0.69091\n",
      "Document 6-bromopurine - similarity: 0.69083\n",
      "Document N'-(Pyrrolidino[2,1-B]Isoindolin-4-On-8-Yl)-N-(Pyridin-2-Yl)Urea - similarity: 0.69081\n",
      "Document Mentha arvensis top - similarity: 0.69078\n",
      "Document Bordetella pertussis toxoid antigen (glutaraldehyde inactivated) - similarity: 0.69077\n",
      "Document N-(CYCLOPROPYLMETHYL)-4-(METHYLOXY)-3-({5-[3-(3-PYRIDINYL)PHENYL]-1,3-OXAZOL-2-YL}AMINO)BENZENESULFONAMIDE - similarity: 0.69076\n",
      "Document Danvatirsen - similarity: 0.69068\n",
      "Document Temefos - similarity: 0.69064\n",
      "Document Patupilone - similarity: 0.69063\n",
      "Document Coal tar - similarity: 0.69061\n",
      "Document RRx-001 - similarity: 0.69061\n",
      "Document 5-Nitroindazole - similarity: 0.69059\n",
      "Document Compound 18 - similarity: 0.69050\n",
      "Document Oxybuprocaine - similarity: 0.69031\n",
      "Document 3-(HYDROXY-PHENYL-PHOSPHINOYLOXY)-8-METHYL-8-AZA-BICYCLO[3.2.1]OCTANE-2-CARBOXYLIC ACID METHYL ESTER - similarity: 0.69029\n",
      "Document D-Mannuronic Acid - similarity: 0.69025\n",
      "Document Influenza B virus B/Brisbane/60/2008 antigen (formaldehyde inactivated) - similarity: 0.69023\n",
      "Document Phthalofyne - similarity: 0.69019\n",
      "Document Monobenzone - similarity: 0.69012\n",
      "Document Gandotinib - similarity: 0.68988\n",
      "Document Bromus secalinus pollen - similarity: 0.68987\n",
      "Document Almotriptan - similarity: 0.68983\n",
      "Document N-Hydroxy-4-Phosphono-Butanamide - similarity: 0.68969\n",
      "Document (4R)-limonene - similarity: 0.68949\n",
      "Document Ravulizumab - similarity: 0.68948\n",
      "Document 2-Aminothiazoline - similarity: 0.68946\n",
      "Document Selinexor - similarity: 0.68943\n",
      "Document Cefapirin - similarity: 0.68941\n",
      "Document Amitraz - similarity: 0.68939\n",
      "Document Caldaret - similarity: 0.68925\n",
      "Document Clobenzorex - similarity: 0.68924\n",
      "Document Streptococcus pneumoniae type 1 capsular polysaccharide antigen - similarity: 0.68923\n",
      "Document 2-({8-[(3R)-3-AMINOPIPERIDIN-1-YL]-1,3-DIMETHYL-2,6-DIOXO-1,2,3,6-TETRAHYDRO-7H-PURIN-7-YL}METHYL)BENZONITRILE - similarity: 0.68919\n",
      "Document Maraviroc - similarity: 0.68913\n",
      "Document Imrecoxib - similarity: 0.68901\n",
      "Document Fradafiban - similarity: 0.68890\n",
      "Document 5-(4-METHYL-BENZOYLAMINO)-BIPHENYL-3,4'-DICARBOXYLIC ACID 3-DIMETHYLAMIDE-4'-HYDROXYAMIDE - similarity: 0.68890\n",
      "Document (2S)-1-[4-({6-[(2,6-Difluorophenyl)amino]-4-pyrimidinyl}amino)phenoxy]-3-(dimethylamino)-2-propanol - similarity: 0.68874\n",
      "Document Semaglutide - similarity: 0.68873\n",
      "Document Reslizumab - similarity: 0.68869\n",
      "Document 2-chloroethyl-3-sarcosinamide-1-nitrosourea - similarity: 0.68854\n",
      "Document Sorghum halepense pollen - similarity: 0.68846\n",
      "Document Gefapixant - similarity: 0.68844\n",
      "Document Saroglitazar - similarity: 0.68837\n",
      "Document Omeprazole - similarity: 0.68828\n",
      "Document VX-148 - similarity: 0.68827\n",
      "Document Decoglurant - similarity: 0.68824\n",
      "Document Ligustrum vulgare pollen - similarity: 0.68815\n",
      "Document Imgatuzumab - similarity: 0.68802\n",
      "Document PF-06700841 - similarity: 0.68797\n",
      "Document Tiamulin - similarity: 0.68793\n",
      "Document Nadroparin - similarity: 0.68788\n",
      "Document Tipapkinogene sovacivec - similarity: 0.68787\n",
      "Document Neostigmine - similarity: 0.68783\n",
      "Document Acetyl hexapeptide-3 - similarity: 0.68782\n",
      "Document Pyridoxal-5'-Phosphate-N-Oxide - similarity: 0.68778\n",
      "Document R,S-Warfarin alcohol - similarity: 0.68773\n",
      "Document Pefloxacin - similarity: 0.68766\n",
      "Document Teriparatide - similarity: 0.68758\n",
      "Document Dimetofrine - similarity: 0.68745\n",
      "Document CP-39,332 - similarity: 0.68742\n",
      "Document Firategrast - similarity: 0.68741\n",
      "Document Tosedostat - similarity: 0.68733\n",
      "Document Axitinib - similarity: 0.68729\n",
      "Document Deferitazole - similarity: 0.68728\n",
      "Document 2-Isobutyl-3-methoxypyrazine - similarity: 0.68723\n",
      "Document Tulobuterol - similarity: 0.68719\n",
      "Document Denileukin diftitox - similarity: 0.68714\n",
      "Document Pyridoxamine-5'-Phosphate - similarity: 0.68689\n",
      "Document Oxprenolol - similarity: 0.68685\n",
      "Document 4'-nitrophenyl-3i-thiolaminaritrioside - similarity: 0.68678\n",
      "Document Losartan - similarity: 0.68676\n",
      "Document Nitazoxanide - similarity: 0.68654\n",
      "Document PR-104 - similarity: 0.68652\n",
      "Document N-phenyl-1H-pyrazole-3-carboxamide - similarity: 0.68626\n",
      "Document liposomal prostaglandin E1 - similarity: 0.68607\n",
      "Document Venglustat - similarity: 0.68592\n",
      "Document Cavia porcellus hair - similarity: 0.68591\n",
      "Document Miocamycin - similarity: 0.68587\n",
      "Document Vinorelbine - similarity: 0.68587\n",
      "Document L-valinol - similarity: 0.68585\n",
      "Document 4-oxo-4,5-dihydro-1,2,5-thiadiazole-3-carboxylic acid - similarity: 0.68575\n",
      "Document Secukinumab - similarity: 0.68573\n",
      "Document Alipogene tiparvovec - similarity: 0.68571\n",
      "Document APR-246 - similarity: 0.68568\n",
      "Document Cortivazol - similarity: 0.68554\n",
      "Document Iopromide - similarity: 0.68552\n",
      "Document Acipimox - similarity: 0.68543\n",
      "Document N-hydroxy-5-[(3-phenyl-5,6-dihydroimidazo[1,2-a]pyrazin-7(8H)-yl)carbonyl]thiophene-2-carboxamide - similarity: 0.68542\n",
      "Document Fenbendazole - similarity: 0.68538\n",
      "Document Dulaglutide - similarity: 0.68530\n",
      "Document Poziotinib - similarity: 0.68528\n",
      "Document Lomefloxacin - similarity: 0.68528\n",
      "Document N-(5-METHYL-1H-PYRAZOL-3-YL)-2-PHENYLQUINAZOLIN-4-AMINE - similarity: 0.68523\n",
      "Document Imeglimin - similarity: 0.68522\n",
      "Document Kojic acid - similarity: 0.68483\n",
      "Document Tirasemtiv - similarity: 0.68478\n",
      "Document Asoprisnil - similarity: 0.68470\n",
      "Document Testosterone undecanoate - similarity: 0.68468\n",
      "Document Naproxcinod - similarity: 0.68467\n",
      "Document Imiglucerase - similarity: 0.68458\n",
      "Document Lifastuzumab vedotin - similarity: 0.68451\n",
      "Document Hydracarbazine - similarity: 0.68441\n",
      "Document Ocriplasmin - similarity: 0.68441\n",
      "Document Lofepramine - similarity: 0.68435\n",
      "Document Fosdagrocorat - similarity: 0.68433\n",
      "Document Malacidin B - similarity: 0.68414\n",
      "Document Asenapine - similarity: 0.68413\n",
      "Document Travoprost - similarity: 0.68410\n",
      "Document Aganirsen - similarity: 0.68407\n",
      "Document Corydalis ambigua tuber - similarity: 0.68391\n",
      "Document Nizatidine - similarity: 0.68391\n",
      "Document N-[(1R,2R,3E)-2-hydroxy-1-(hydroxymethyl)heptadec-3-en-1-yl]acetamide - similarity: 0.68385\n",
      "Document Mafosfamide - similarity: 0.68382\n",
      "Document Rumex crispus top - similarity: 0.68380\n",
      "Document Norgestomet - similarity: 0.68375\n",
      "Document 3-(5-{[4-(AMINOMETHYL)PIPERIDIN-1-YL]METHYL}-1H-INDOL-2-YL)-1H-INDAZOLE-6-CARBONITRILE - similarity: 0.68370\n",
      "Document Sulfate ion - similarity: 0.68342\n",
      "Document Litoxetine - similarity: 0.68340\n",
      "Document Bacillus amyloliquefaciens - similarity: 0.68340\n",
      "Document Balugrastim - similarity: 0.68334\n",
      "Document Tylosin - similarity: 0.68329\n",
      "Document (RP,SP)-O-(2R)-(1-PHENOXYBUT-2-YL)-METHYLPHOSPHONIC ACID CHLORIDE - similarity: 0.68323\n",
      "Document Buflomedil - similarity: 0.68320\n",
      "Document Pyridoxine - similarity: 0.68319\n",
      "Document Pf-04531083 - similarity: 0.68314\n",
      "Document Eprosartan - similarity: 0.68305\n",
      "Document Butamben - similarity: 0.68302\n",
      "Document Adenosine 5'-[gamma-thio]triphosphate - similarity: 0.68298\n",
      "Document Tifuvirtide - similarity: 0.68296\n",
      "Document JNJ-26489112 - similarity: 0.68296\n",
      "Document 2-CHLORO-N-(3-CYANO-5,6-DIHYDRO-4H-CYCLOPENTA[B]THIOPHEN-2-YL)-5-DIETHYLSULFAMOYL-BENZAMIDE - similarity: 0.68295\n",
      "Document (4S)-4-(2-NAPHTHYLMETHYL)-D-GLUTAMIC ACID - similarity: 0.68295\n",
      "Document Ibudilast - similarity: 0.68292\n",
      "Document Wortmannin - similarity: 0.68291\n",
      "Document Flurithromycin - similarity: 0.68291\n",
      "Document Olipudase alfa - similarity: 0.68285\n",
      "Document 4-[(4-METHYL-1-PIPERAZINYL)METHYL]-N-[3-[[4-(3-PYRIDINYL)-2-PYRIMIDINYL]AMINO]PHENYL]-BENZAMIDE - similarity: 0.68283\n",
      "Document Deutivacaftor - similarity: 0.68281\n",
      "Document 4-Methoxy-E-Rhodomycin T - similarity: 0.68281\n",
      "Document Dihydroergocryptine - similarity: 0.68276\n",
      "Document Trichophyton tonsurans - similarity: 0.68269\n",
      "Document N-1,10-phenanthrolin-5-ylacetamide - similarity: 0.68263\n",
      "Document VIBRIO CHOLERAE OGAWA 50 CLASSICAL BIOTYPE, HEAT INACTIVATED - similarity: 0.68262\n",
      "Document Dimethyl Propionate Ester Heme - similarity: 0.68248\n",
      "Document Lexaptepid Pegol - similarity: 0.68244\n",
      "Document Titanium dioxide - similarity: 0.68233\n",
      "Document Influenza A virus A/Perth/16/2009 (H3N2) live (attenuated) antigen - similarity: 0.68232\n",
      "Document Rizatriptan - similarity: 0.68231\n",
      "Document Trimipramine - similarity: 0.68225\n",
      "Document Benzydamine - similarity: 0.68223\n",
      "Document 2,5-Dimethoxy-4-ethylthioamphetamine - similarity: 0.68222\n",
      "Document CERC-301 - similarity: 0.68212\n",
      "Document Interferon gamma-1b - similarity: 0.68199\n",
      "Document MF268 - similarity: 0.68191\n",
      "Document Imiquimod - similarity: 0.68177\n",
      "Document Sphingosine - similarity: 0.68173\n",
      "Document Steviolbioside - similarity: 0.68162\n",
      "Document Trazodone - similarity: 0.68159\n",
      "Document PHENYLALANYLAMINODI(ETHYLOXY)ETHYL BENZENESULFONAMIDEAMINOCARBONYLBENZENESULFONAMIDE - similarity: 0.68151\n",
      "Document Clazosentan - similarity: 0.68150\n",
      "Document Reldesemtiv - similarity: 0.68138\n",
      "Document Glatiramer - similarity: 0.68128\n",
      "Document Pentostatin - similarity: 0.68120\n",
      "Document AZD-7762 - similarity: 0.68109\n",
      "Document Helium - similarity: 0.68104\n",
      "Document Ibalizumab - similarity: 0.68102\n",
      "Document Tolnaftate - similarity: 0.68093\n",
      "Document Largemouth bass - similarity: 0.68091\n",
      "Document Trabodenoson - similarity: 0.68067\n",
      "Document Allylestrenol - similarity: 0.68063\n",
      "Document Ciraparantag - similarity: 0.68059\n",
      "Document Bromazepam - similarity: 0.68058\n",
      "Document (2s)-2-[(5-Benzofuran-2-Yl-Thiophen-2-Ylmethyl)-(2,4-Dichloro-Benzoyl)-Amino]-3-Phenyl-Propionic Acid - similarity: 0.68042\n",
      "Document Progesterone - similarity: 0.68036\n",
      "Document Adriforant - similarity: 0.68035\n",
      "Document Enfuvirtide - similarity: 0.68024\n",
      "Document Lenalidomide - similarity: 0.68021\n",
      "Document Merestinib - similarity: 0.68014\n",
      "Document Paprika - similarity: 0.68011\n",
      "Document Oltipraz - similarity: 0.68010\n",
      "Document GDC-0917 - similarity: 0.68000\n",
      "Document p-Quaterphenyl - similarity: 0.67999\n",
      "Document Dichlorobenzyl alcohol - similarity: 0.67984\n",
      "Document Tropisetron - similarity: 0.67960\n",
      "Document Abrilumab - similarity: 0.67957\n",
      "Document Hedera helix leaf extract - similarity: 0.67955\n",
      "Document 2-(ACETYL-HYDROXY-AMINO)-4-METHYL-PENTANOIC ACID METHYL ESTER - similarity: 0.67953\n",
      "Document 1-[1-(3-aminophenyl)-3-tert-butyl-1H-pyrazol-5-yl]-3-naphthalen-1-ylurea - similarity: 0.67946\n",
      "Document Insulin peglispro - similarity: 0.67936\n",
      "Document Sildenafil - similarity: 0.67934\n",
      "Document Pomegranate - similarity: 0.67923\n",
      "Document Glyceryl Trioleate - similarity: 0.67916\n",
      "Document Flubendazole - similarity: 0.67900\n",
      "Document Embutramide - similarity: 0.67879\n",
      "Document Somatotropin - similarity: 0.67865\n",
      "Document Ertugliflozin - similarity: 0.67865\n",
      "Document Debromohymenialdisine - similarity: 0.67847\n",
      "Document Gleptoferron - similarity: 0.67844\n",
      "Document Sodium cation - similarity: 0.67844\n",
      "Document Gallichrome - similarity: 0.67837\n",
      "Document Iron Dextran - similarity: 0.67833\n",
      "Document Juniperus occidentalis pollen - similarity: 0.67832\n",
      "Document RO-5028442 - similarity: 0.67829\n",
      "Document Benzylsulfonic acid - similarity: 0.67822\n",
      "Document Levocetirizine - similarity: 0.67822\n",
      "Document Mesoridazine - similarity: 0.67821\n",
      "Document Pembrolizumab - similarity: 0.67812\n",
      "Document Morinidazole - similarity: 0.67809\n",
      "Document Bococizumab - similarity: 0.67803\n",
      "Document Aminorex - similarity: 0.67801\n",
      "Document N-(3-chlorobenzyl)-1-(4-methylpentanoyl)-L-prolinamide - similarity: 0.67800\n",
      "Document Elaeagnus angustifolia pollen - similarity: 0.67798\n",
      "Document Tipranavir - similarity: 0.67784\n",
      "Document Pinus ponderosa pollen - similarity: 0.67778\n",
      "Document CTCE-9908 - similarity: 0.67772\n",
      "Document Patidegib - similarity: 0.67756\n",
      "Document Tributyltin - similarity: 0.67755\n",
      "Document Anamorelin - similarity: 0.67747\n",
      "Document Emicerfont - similarity: 0.67744\n",
      "Document Arverapamil - similarity: 0.67742\n",
      "Document BL-1020 - similarity: 0.67740\n",
      "Document (R)-N-(1-Methyl-Hexyl)-Formamide - similarity: 0.67733\n",
      "Document Magnesium acetate tetrahydrate - similarity: 0.67732\n",
      "Document Wheat bran - similarity: 0.67732\n",
      "Document Propranolol - similarity: 0.67718\n",
      "Document Pegilodecakin - similarity: 0.67707\n",
      "Document Turoctocog alfa - similarity: 0.67696\n",
      "Document Eberconazole - similarity: 0.67691\n",
      "Document Vorapaxar - similarity: 0.67682\n",
      "Document 4-(6-CHLORO-2,4-DIOXO-1,2,3,4-TETRAHYDROPYRIMIDIN-5-YL) BUTYL PHOSPHATE - similarity: 0.67679\n",
      "Document Cabergoline - similarity: 0.67676\n",
      "Document N-[1-(2,6-dimethoxybenzyl)piperidin-4-yl]-4-sulfanylbutanamide - similarity: 0.67674\n",
      "Document Talampanel - similarity: 0.67665\n",
      "Document Etoxeridine - similarity: 0.67664\n",
      "Document Metformin - similarity: 0.67661\n",
      "Document Influenza A virus A/Michigan/45/2015 X-275 (H1N1) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.67659\n",
      "Document Nikethamide - similarity: 0.67649\n",
      "Document Diatrizoate - similarity: 0.67642\n",
      "Document Pseudosaccharin chloride - similarity: 0.67636\n",
      "Document Midostaurin - similarity: 0.67633\n",
      "Document Cefoxitin - similarity: 0.67630\n",
      "Document Meprobamate - similarity: 0.67629\n",
      "Document Narasin - similarity: 0.67621\n",
      "Document Bemiparin - similarity: 0.67620\n",
      "Document Sobetirome - similarity: 0.67620\n",
      "Document Simeprevir - similarity: 0.67619\n",
      "Document 3,4-dihydroxy-9,10-secoandrosta-1(10),2,4-triene-9,17-dione - similarity: 0.67618\n",
      "Document 5-[4-(2-Hydroxyethyl)-1-piperidinyl]-5-phenyl-2,4,6(1H,3H,5H)-pyrimidinetrione - similarity: 0.67615\n",
      "Document Setmelanotide - similarity: 0.67601\n",
      "Document Pentetrazol - similarity: 0.67599\n",
      "Document Stavudine - similarity: 0.67597\n",
      "Document Sorghum bicolor subsp. bicolor pollen - similarity: 0.67595\n",
      "Document Peficitinib - similarity: 0.67594\n",
      "Document SAR-125844 - similarity: 0.67590\n",
      "Document 2-Iodobenzylthio Group - similarity: 0.67583\n",
      "Document Tagraxofusp - similarity: 0.67581\n",
      "Document 7-Hydroxystaurosporine - similarity: 0.67575\n",
      "Document Dextofisopam - similarity: 0.67575\n",
      "Document N-({4-[4-(2-Methyl-1H-imidazol-1-yl)butyl]phenyl}acetyl)-L-seryl-N-(2-cyclohexylethyl)-L-lysinamide - similarity: 0.67572\n",
      "Document Methyl aminolevulinate - similarity: 0.67569\n",
      "Document Alprazolam - similarity: 0.67560\n",
      "Document Pergolide - similarity: 0.67557\n",
      "Document FP-1096 - similarity: 0.67556\n",
      "Document Elotuzumab - similarity: 0.67555\n",
      "Document Phosphoenolpyruvate - similarity: 0.67551\n",
      "Document Cefetamet - similarity: 0.67549\n",
      "Document 1-[(2-NITROPHENYL)SULFONYL]-1H-PYRROLO[3,2-B]PYRIDINE-6-CARBOXAMIDE - similarity: 0.67541\n",
      "Document Pidilizumab - similarity: 0.67538\n",
      "Document LGD-6972 - similarity: 0.67537\n",
      "Document PUROMYCIN AMINONUCLEOSIDE-5'-MONOPHOSPHATE - similarity: 0.67535\n",
      "Document INGN 225 - similarity: 0.67520\n",
      "Document Bleselumab - similarity: 0.67502\n",
      "Document Sodium phosphate, dibasic, unspecified form - similarity: 0.67488\n",
      "Document Mirococept - similarity: 0.67479\n",
      "Document Hydrochloric acid - similarity: 0.67473\n",
      "Document Gatifloxacin - similarity: 0.67468\n",
      "Document Artemisia annua pollen - similarity: 0.67460\n",
      "Document PF-04991532 - similarity: 0.67455\n",
      "Document AMG-208 - similarity: 0.67441\n",
      "Document (5S)-1-benzyl-3-(1,1-dioxido-1,2-benzisothiazol-3-yl)-4-hydroxy-5-(1-methylethyl)-1,5-dihydro-2H-pyrrol-2-one - similarity: 0.67441\n",
      "Document Eprotirome - similarity: 0.67430\n",
      "Document Posaconazole - similarity: 0.67423\n",
      "Document PF-00610355 - similarity: 0.67415\n",
      "Document Lawsone - similarity: 0.67414\n",
      "Document Deramciclane - similarity: 0.67410\n",
      "Document Ambrosia confertiflora pollen - similarity: 0.67409\n",
      "Document Calcium glucoheptonate - similarity: 0.67381\n",
      "Document Penicillium digitatum - similarity: 0.67354\n",
      "Document Epitestosterone - similarity: 0.67346\n",
      "Document N(5)-[(hydroxyamino)(imino)methyl]-L-ornithine - similarity: 0.67339\n",
      "Document Tangerine peel - similarity: 0.67339\n",
      "Document O6-CYCLOHEXYLMETHOXY-2-(4'-SULPHAMOYLANILINO) PURINE - similarity: 0.67325\n",
      "Document Triflusal - similarity: 0.67324\n",
      "Document Coumarin 120 - similarity: 0.67308\n",
      "Document CRA_10972 - similarity: 0.67285\n",
      "Document Gaxilose - similarity: 0.67284\n",
      "Document Trametinib - similarity: 0.67280\n",
      "Document Alvocidib - similarity: 0.67278\n",
      "Document Hydroquinidine - similarity: 0.67275\n",
      "Document Tanezumab - similarity: 0.67275\n",
      "Document Mebendazole - similarity: 0.67273\n",
      "Document Meningococcal polysaccharide vaccine group C - similarity: 0.67268\n",
      "Document Enzacamene - similarity: 0.67266\n",
      "Document Trifolium pratense pollen - similarity: 0.67263\n",
      "Document Estrone sulfate - similarity: 0.67261\n",
      "Document Naphthalene-2,6-disulfonic acid - similarity: 0.67251\n",
      "Document (2R)-4,4-dihydroxy-5-nitro-2-(phenylmethyl)pentanoic acid - similarity: 0.67244\n",
      "Document Hydroxychloroquine - similarity: 0.67241\n",
      "Document (2R,3R,4S,5R)-2-[6-amino-8-[(3,4-dichlorophenyl)methylamino]purin-9-yl]-5-(hydroxymethyl)oxolane-3,4-diol - similarity: 0.67235\n",
      "Document Sulfaphenazole - similarity: 0.67229\n",
      "Document Cobalt Co-60 - similarity: 0.67222\n",
      "Document Cucumber - similarity: 0.67213\n",
      "Document Somavaratan - similarity: 0.67205\n",
      "Document rhThrombin - similarity: 0.67197\n",
      "Document 2-(4-hydroxyphenyl)benzo[b]thiophen-6-ol - similarity: 0.67197\n",
      "Document 8-(2,5-Dimethoxy-Benzyl)-2-Fluoro-9-Pent-9h-Purin-6-Ylamine - similarity: 0.67194\n",
      "Document Hemoglobin raffimer - similarity: 0.67179\n",
      "Document Tilmicosin - similarity: 0.67177\n",
      "Document Di-Stearoyl-3-Sn-Phosphatidylethanolamine - similarity: 0.67168\n",
      "Document Cyclopropyl-{4-[5-(3,4-Dichlorophenyl)-2-[(1-Methyl)-Piperidin]-4-Yl-3-Propyl-3h-Imidazol-4-Yl]-Pyrimidin-2-Yl}Amine - similarity: 0.67167\n",
      "Document Moxaverine - similarity: 0.67163\n",
      "Document Nialamide - similarity: 0.67160\n",
      "Document Ketobemidone - similarity: 0.67159\n",
      "Document Oftasceine - similarity: 0.67153\n",
      "Document Rolapitant - similarity: 0.67146\n",
      "Document Barley malt - similarity: 0.67142\n",
      "Document Influenza A virus A/Christchurch/16/2010 NIB-74XP (H1N1) antigen (formaldehyde inactivated) - similarity: 0.67134\n",
      "Document Common carp - similarity: 0.67133\n",
      "Document TMC-310911 - similarity: 0.67112\n",
      "Document Torasemide - similarity: 0.67108\n",
      "Document Populus fremontii pollen - similarity: 0.67096\n",
      "Document Aprinocarsen - similarity: 0.67095\n",
      "Document Titanium - similarity: 0.67095\n",
      "Document Bazedoxifene - similarity: 0.67083\n",
      "Document Propamidine - similarity: 0.67080\n",
      "Document Duck - similarity: 0.67079\n",
      "Document Ximelagatran - similarity: 0.67078\n",
      "Document Pidotimod - similarity: 0.67070\n",
      "Document 3-Aza-2,3-Dihydrogeranyl Diphosphate - similarity: 0.67067\n",
      "Document Mavoglurant - similarity: 0.67061\n",
      "Document Glucagon-like peptide II - similarity: 0.67054\n",
      "Document Trapidil - similarity: 0.67049\n",
      "Document 5-hydroxymethyl-5,6-dihydrofolic acid - similarity: 0.67036\n",
      "Document Zinc citrate - similarity: 0.67031\n",
      "Document Silodosin - similarity: 0.67029\n",
      "Document Abciximab - similarity: 0.67024\n",
      "Document Zonisamide - similarity: 0.67001\n",
      "Document Diltiazem - similarity: 0.66999\n",
      "Document Mavatrep - similarity: 0.66999\n",
      "Document Mitomycin - similarity: 0.66986\n",
      "Document Zalutumumab - similarity: 0.66980\n",
      "Document Evacetrapib - similarity: 0.66976\n",
      "Document N-({6-[(4-CYANOBENZYL)OXY]NAPHTHALEN-2-YL}SULFONYL)-D-GLUTAMIC ACID - similarity: 0.66973\n",
      "Document ETHYL 3-[4-(AMINOSULFONYL)PHENYL]PROPANOATE - similarity: 0.66954\n",
      "Document N-[(1R)-3-(4-HYDROXYPHENYL)-1-METHYLPROPYL]-2-(2-PHENYL-1H-INDOL-3-YL)ACETAMIDE - similarity: 0.66946\n",
      "Document Muskmelon - similarity: 0.66940\n",
      "Document Felypressin - similarity: 0.66933\n",
      "Document Alimemazine - similarity: 0.66920\n",
      "Document Povidone K30 - similarity: 0.66912\n",
      "Document Mus musculus skin - similarity: 0.66909\n",
      "Document Gpi-1046 - similarity: 0.66892\n",
      "Document Josamycin - similarity: 0.66866\n",
      "Document Paritaprevir - similarity: 0.66861\n",
      "Document Olaptesed Pegol - similarity: 0.66850\n",
      "Document (S,R)-fidarestat - similarity: 0.66849\n",
      "Document Raxibacumab - similarity: 0.66848\n",
      "Document Dilmapimod - similarity: 0.66844\n",
      "Document 1-({2-[2-(4-CHLOROPHENYL)ETHYL]-1,3-DIOXOLAN-2-YL}METHYL)-1H-IMIDAZOLE - similarity: 0.66843\n",
      "Document Aluminum sesquichlorohydrate - similarity: 0.66843\n",
      "Document Suprofen - similarity: 0.66823\n",
      "Document Zopiclone - similarity: 0.66822\n",
      "Document Ceruletide - similarity: 0.66818\n",
      "Document Hydroxyzine - similarity: 0.66816\n",
      "Document Labetuzumab govitecan - similarity: 0.66814\n",
      "Document 4-{[(2S)-3-(tert-butylamino)-2-hydroxypropyl]oxy}-3H-indole-2-carbonitrile - similarity: 0.66807\n",
      "Document 7-Aminodesacetoxycephalosporanic acid - similarity: 0.66794\n",
      "Document Lisavanbulin - similarity: 0.66780\n",
      "Document 2-{5-[3-(6-BENZOYL-1-PROPYLNAPHTHALEN-2-YLOXY)PROPOXY]INDOL-1-YL}ETHANOIC ACID - similarity: 0.66774\n",
      "Document Acalabrutinib - similarity: 0.66769\n",
      "Document Nalmefene - similarity: 0.66768\n",
      "Document Terconazole - similarity: 0.66766\n",
      "Document Sarocladium strictum - similarity: 0.66766\n",
      "Document MK-8776 - similarity: 0.66760\n",
      "Document 1-thio-beta-D-glucopyranose - similarity: 0.66757\n",
      "Document Porfimer sodium - similarity: 0.66754\n",
      "Document Sufugolix - similarity: 0.66752\n",
      "Document Sonepcizumab - similarity: 0.66745\n",
      "Document Droxypropine - similarity: 0.66736\n",
      "Document Nifuroxazide - similarity: 0.66735\n",
      "Document Dimethicone - similarity: 0.66705\n",
      "Document Tenofovir disoproxil - similarity: 0.66698\n",
      "Document Rifaximin - similarity: 0.66689\n",
      "Document Alemtuzumab - similarity: 0.66673\n",
      "Document Alprostadil - similarity: 0.66662\n",
      "Document Influenza B virus B/Massachusetts/2/2012 antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.66659\n",
      "Document Levamlodipine - similarity: 0.66657\n",
      "Document Etizolam - similarity: 0.66651\n",
      "Document Tolvaptan - similarity: 0.66650\n",
      "Document Rivoceranib - similarity: 0.66642\n",
      "Document Levoverbenone - similarity: 0.66641\n",
      "Document Hydrochlorothiazide - similarity: 0.66640\n",
      "Document SB-649868 - similarity: 0.66638\n",
      "Document Robatumumab - similarity: 0.66628\n",
      "Document Odalasvir - similarity: 0.66623\n",
      "Document Doxepin - similarity: 0.66621\n",
      "Document Vedroprevir - similarity: 0.66619\n",
      "Document Manidipine - similarity: 0.66616\n",
      "Document Calcifediol - similarity: 0.66612\n",
      "Document Etravirine - similarity: 0.66610\n",
      "Document Tolazoline - similarity: 0.66604\n",
      "Document AP5280 - similarity: 0.66586\n",
      "Document Zinc Substituted Heme C - similarity: 0.66574\n",
      "Document Aplaviroc - similarity: 0.66573\n",
      "Document UK-500001 - similarity: 0.66573\n",
      "Document Palifermin - similarity: 0.66566\n",
      "Document Montelukast - similarity: 0.66563\n",
      "Document 2,4-thiazolidinedione - similarity: 0.66556\n",
      "Document Salsola kali pollen - similarity: 0.66549\n",
      "Document Tyramine O-sulfate - similarity: 0.66546\n",
      "Document Benzocaine - similarity: 0.66542\n",
      "Document Tiazofurine - similarity: 0.66537\n",
      "Document Mosapride - similarity: 0.66536\n",
      "Document Phenoperidine - similarity: 0.66535\n",
      "Document Lumefantrine - similarity: 0.66526\n",
      "Document [5-AMINO-1-(4-FLUOROPHENYL)-1H-PYRAZOL-4-YL][3-(PIPERIDIN-4-YLOXY)PHENYL]METHANONE - similarity: 0.66518\n",
      "Document Lamivudine - similarity: 0.66511\n",
      "Document Palivizumab - similarity: 0.66506\n",
      "Document Canagliflozin - similarity: 0.66498\n",
      "Document Pateclizumab - similarity: 0.66498\n",
      "Document Fosbretabulin - similarity: 0.66493\n",
      "Document Balapiravir - similarity: 0.66490\n",
      "Document Docetaxel - similarity: 0.66487\n",
      "Document Clenoliximab - similarity: 0.66470\n",
      "Document BQ-123 - similarity: 0.66459\n",
      "Document MEM 1414 - similarity: 0.66453\n",
      "Document Doxazosin - similarity: 0.66443\n",
      "Document Ranolazine - similarity: 0.66424\n",
      "Document [5-AMINO-1-(4-FLUOROPHENYL)-1H-PYRAZOL-4-YL](3-{[(2R)-2,3-DIHYDROXYPROPYL]OXY}PHENYL)METHANONE - similarity: 0.66422\n",
      "Document Hydroxyethylpromethazine - similarity: 0.66421\n",
      "Document Cavia porcellus skin - similarity: 0.66415\n",
      "Document Guanoxabenz - similarity: 0.66415\n",
      "Document Tretinoin - similarity: 0.66402\n",
      "Document PH-797804 - similarity: 0.66376\n",
      "Document Troxipide - similarity: 0.66367\n",
      "Document Cyclobutyrol - similarity: 0.66358\n",
      "Document Efavirenz - similarity: 0.66356\n",
      "Document 4-bromo-2-{[(3R,5S)-3,5-dimethylpiperidin-1-yl]carbonyl}aniline - similarity: 0.66354\n",
      "Document Meperidine - similarity: 0.66352\n",
      "Document Levobupivacaine - similarity: 0.66344\n",
      "Document Atevirdine - similarity: 0.66341\n",
      "Document 5-BROMO-2-{[(4-CHLOROPHENYL)SULFONYL]AMINO}BENZOIC ACID - similarity: 0.66338\n",
      "Document Brontictuzumab - similarity: 0.66336\n",
      "Document L-Myo-Inositol-1-Phosphate - similarity: 0.66333\n",
      "Document 1-METHYL ETHYL 1-CHLORO-5-[[(5,6DIHYDRO-2-METHYL-1,4-OXATHIIN-3-YL)CARBONYL]AMINO]BENZOATE - similarity: 0.66327\n",
      "Document Dimercaptosuccinic acid - similarity: 0.66316\n",
      "Document 5-Phenylsulfanyl-2,4-Quinazolinediamine - similarity: 0.66308\n",
      "Document Fluvoxamine - similarity: 0.66305\n",
      "Document PD-0325901 - similarity: 0.66305\n",
      "Document Basiliximab - similarity: 0.66300\n",
      "Document MK-2748 - similarity: 0.66297\n",
      "Document Omacetaxine mepesuccinate - similarity: 0.66295\n",
      "Document Trastuzumab - similarity: 0.66292\n",
      "Document 4-(Acetylamino)-3-[(Hydroxyacetyl)Amino]Benzoic Acid - similarity: 0.66290\n",
      "Document Bisphenol Z - similarity: 0.66266\n",
      "Document Methyl nicotinate - similarity: 0.66260\n",
      "Document Celery - similarity: 0.66257\n",
      "Document Cytochlor - similarity: 0.66244\n",
      "Document Gimatecan - similarity: 0.66229\n",
      "Document Asparagus - similarity: 0.66228\n",
      "Document Flaviolin - similarity: 0.66227\n",
      "Document Bupivacaine - similarity: 0.66222\n",
      "Document Allenrolfea occidentalis pollen - similarity: 0.66222\n",
      "Document Selatogrel - similarity: 0.66209\n",
      "Document Nilotinib - similarity: 0.66207\n",
      "Document 5-bromo-3-(pyrrolidin-1-ylsulfonyl)-1H-indole-2-carboxamide - similarity: 0.66201\n",
      "Document Chlorproguanil - similarity: 0.66199\n",
      "Document Intetumumab - similarity: 0.66191\n",
      "Document Nevirapine - similarity: 0.66189\n",
      "Document Tandospirone - similarity: 0.66188\n",
      "Document Tyrothricin - similarity: 0.66187\n",
      "Document Pegaptanib - similarity: 0.66182\n",
      "Document 3,7-DIHYDROXYNAPHTHALENE-2-CARBOXYLIC ACID - similarity: 0.66175\n",
      "Document Plerixafor - similarity: 0.66171\n",
      "Document ATP - similarity: 0.66169\n",
      "Document Methocarbamol - similarity: 0.66169\n",
      "Document Morrhuic acid - similarity: 0.66158\n",
      "Document AZD-7687 - similarity: 0.66155\n",
      "Document Influenza B virus B/Phuket/3073/2013 BVR-1B antigen (propiolactone inactivated) - similarity: 0.66142\n",
      "Document Benzoxonium - similarity: 0.66134\n",
      "Document Pertussis vaccine - similarity: 0.66133\n",
      "Document Rilpivirine - similarity: 0.66132\n",
      "Document Lomitapide - similarity: 0.66126\n",
      "Document Thieno[2,3-B]Pyridine-2-Carboxamidine - similarity: 0.66118\n",
      "Document Zuclopenthixol - similarity: 0.66114\n",
      "Document Brimapitide - similarity: 0.66101\n",
      "Document Xylometazoline - similarity: 0.66093\n",
      "Document Trivanadate - similarity: 0.66091\n",
      "Document IC261 - similarity: 0.66071\n",
      "Document Icrucumab - similarity: 0.66071\n",
      "Document Vatreptacog alfa - similarity: 0.66070\n",
      "Document Gestrinone - similarity: 0.66068\n",
      "Document Emtricitabine - similarity: 0.66064\n",
      "Document Obiltoxaximab - similarity: 0.66056\n",
      "Document 3-PYRIDIN-4-YL-2,4-DIHYDRO-INDENO[1,2-.C.] PYRAZOLE - similarity: 0.66053\n",
      "Document BPH-608 - similarity: 0.66049\n",
      "Document Lorcainide - similarity: 0.66048\n",
      "Document Alamifovir - similarity: 0.66048\n",
      "Document KOS-1584 - similarity: 0.66042\n",
      "Document Meningococcal polysaccharide vaccine group A - similarity: 0.66024\n",
      "Document Nebivolol - similarity: 0.66015\n",
      "Document Flucloxacillin - similarity: 0.66000\n",
      "Document Artemisia vulgaris root - similarity: 0.65998\n",
      "Document 2,4-Diamino-5-Methyl-6-[(3,4,5-Trimethoxy-N-Methylanilino)Methyl]Pyrido[2,3-D]Pyrimidine - similarity: 0.65998\n",
      "Document Bexagliflozin - similarity: 0.65992\n",
      "Document Lolium perenne subsp. multiflorum pollen - similarity: 0.65992\n",
      "Document Urofollitropin - similarity: 0.65990\n",
      "Document (2S)-2-hydroxy-2-methyl-N-[4-nitro-3-(trifluoromethyl)phenyl]-3-(pentafluorophenoxy)propanamide - similarity: 0.65990\n",
      "Document Seletalisib - similarity: 0.65961\n",
      "Document Mycocladus corymbiferus - similarity: 0.65960\n",
      "Document Inosinic Acid - similarity: 0.65953\n",
      "Document Prednisolone - similarity: 0.65951\n",
      "Document Bromocriptine - similarity: 0.65951\n",
      "Document Riluzole - similarity: 0.65948\n",
      "Document Thallous Chloride - similarity: 0.65947\n",
      "Document Clopamide - similarity: 0.65944\n",
      "Document LC16M8 - similarity: 0.65943\n",
      "Document Glycyrrhiza glabra - similarity: 0.65940\n",
      "Document Apis mellifera venom - similarity: 0.65940\n",
      "Document Desmoteplase - similarity: 0.65940\n",
      "Document Semorinemab - similarity: 0.65939\n",
      "Document 2',4'-Dinitrophenyl-2deoxy-2-Fluro-B-D-Cellobioside - similarity: 0.65933\n",
      "Document Pralatrexate - similarity: 0.65915\n",
      "Document Rasburicase - similarity: 0.65915\n",
      "Document Codrituzumab - similarity: 0.65914\n",
      "Document Imalumab - similarity: 0.65912\n",
      "Document Etrolizumab - similarity: 0.65911\n",
      "Document Vatalanib - similarity: 0.65903\n",
      "Document Temelimab - similarity: 0.65901\n",
      "Document [5-(5-Amino-1H-pyrrolo[3,2-b]pyridin-2-yl)-6-hydroxy-3'-nitro-3-biphenylyl]acetic acid - similarity: 0.65894\n",
      "Document 3-Hydroxyimino Quinic Acid - similarity: 0.65886\n",
      "Document Pimozide - similarity: 0.65880\n",
      "Document Pinus sylvestris pollen - similarity: 0.65858\n",
      "Document O-Decyl Hydrogen Thiocarbonate - similarity: 0.65857\n",
      "Document Fabomotizole - similarity: 0.65855\n",
      "Document Tolcapone - similarity: 0.65855\n",
      "Document Moxisylyte - similarity: 0.65839\n",
      "Document Dalteparin - similarity: 0.65838\n",
      "Document Levomefolic acid - similarity: 0.65838\n",
      "Document Ritonavir - similarity: 0.65835\n",
      "Document Valbenazine - similarity: 0.65818\n",
      "Document GS-441524 - similarity: 0.65816\n",
      "Document Diacetyl benzoyl lathyrol - similarity: 0.65813\n",
      "Document Acyclovir - similarity: 0.65811\n",
      "Document Geldanamycin - similarity: 0.65811\n",
      "Document Olokizumab - similarity: 0.65795\n",
      "Document (4-sulfamoyl-phenyl)-thiocarbamic acid O-(2-thiophen-3-yl-ethyl) ester - similarity: 0.65791\n",
      "Document Esomeprazole - similarity: 0.65790\n",
      "Document Ketamine - similarity: 0.65785\n",
      "Document BIIB021 - similarity: 0.65783\n",
      "Document Bopindolol - similarity: 0.65779\n",
      "Document 6-Nitroindazole - similarity: 0.65775\n",
      "Document Eprinomectin - similarity: 0.65775\n",
      "Document Tezacaftor - similarity: 0.65772\n",
      "Document AMG-510 - similarity: 0.65754\n",
      "Document 2,3-Dihydroxy-Benzoic Acid - similarity: 0.65751\n",
      "Document Frovocimab - similarity: 0.65746\n",
      "Document Propafenone - similarity: 0.65745\n",
      "Document Inclacumab - similarity: 0.65736\n",
      "Document Motexafin gadolinium - similarity: 0.65736\n",
      "Document Naxifylline - similarity: 0.65735\n",
      "Document Bordetella pertussis toxoid antigen (formaldehyde, glutaraldehyde inactivated) - similarity: 0.65728\n",
      "Document Oxazepam - similarity: 0.65725\n",
      "Document JB991 - similarity: 0.65712\n",
      "Document Lonafarnib - similarity: 0.65710\n",
      "Document AL6528 - similarity: 0.65705\n",
      "Document MK-0752 - similarity: 0.65704\n",
      "Document Vorolanib - similarity: 0.65704\n",
      "Document Mango - similarity: 0.65682\n",
      "Document CMC 001 - similarity: 0.65679\n",
      "Document Perphenazine - similarity: 0.65675\n",
      "Document Peginterferon alfa-2a - similarity: 0.65670\n",
      "Document CARBOBENZYLOXY-(L)-LEUCINYL-(L)LEUCINYL METHOXYMETHYLKETONE - similarity: 0.65665\n",
      "Document Human papillomavirus type 31 L1 capsid protein antigen - similarity: 0.65662\n",
      "Document Omarigliptin - similarity: 0.65655\n",
      "Document Fenoldopam - similarity: 0.65647\n",
      "Document Cephalosporin analog - similarity: 0.65630\n",
      "Document 3'-Deazo-Thiamin Diphosphate - similarity: 0.65625\n",
      "Document Timolol - similarity: 0.65619\n",
      "Document PF-04620110 - similarity: 0.65612\n",
      "Document Argatroban - similarity: 0.65602\n",
      "Document Rocuronium - similarity: 0.65598\n",
      "Document Trichostatin A - similarity: 0.65590\n",
      "Document Idazoxan - similarity: 0.65579\n",
      "Document Eucalyptus globulus leaf - similarity: 0.65578\n",
      "Document (S)-methoprene - similarity: 0.65575\n",
      "Document Imexon - similarity: 0.65575\n",
      "Document 1,2,5-THIADIAZOLIDIN-3-ONE-1,1-DIOXIDE - similarity: 0.65574\n",
      "Document Zebularine - similarity: 0.65572\n",
      "Document Rigosertib - similarity: 0.65567\n",
      "Document ARC-19499 - similarity: 0.65559\n",
      "Document Etilefrine - similarity: 0.65557\n",
      "Document Vorinostat - similarity: 0.65547\n",
      "Document Fotemustine - similarity: 0.65546\n",
      "Document Fosphenytoin - similarity: 0.65545\n",
      "Document Eflapegrastim - similarity: 0.65542\n",
      "Document Chickpea - similarity: 0.65532\n",
      "Document Technetium Tc-99m arcitumomab - similarity: 0.65528\n",
      "Document CYT007-TNFQb - similarity: 0.65523\n",
      "Document Ketoconazole - similarity: 0.65517\n",
      "Document 3-Phosphono-D-alanine - similarity: 0.65515\n",
      "Document Ublituximab - similarity: 0.65507\n",
      "Document Acitretin - similarity: 0.65506\n",
      "Document Ipratropium - similarity: 0.65497\n",
      "Document Barium sulfate - similarity: 0.65497\n",
      "Document Sucrosofate - similarity: 0.65481\n",
      "Document 1-ACETYL-2-LYSO-SN-GLYCERO-3-PHOSPHOETHANOLAMINE - similarity: 0.65465\n",
      "Document 4-{2,6,8-Trioxo-9-[(2S,3R,4R)-2,3,4,5-Tetrahydroxypentyl]-1,2,3,6,8,9-Hexahydro-7h-Purin-7-Yl}Butyl Dihydrogen Phosphate - similarity: 0.65459\n",
      "Document Umbralisib - similarity: 0.65452\n",
      "Document Sarizotan - similarity: 0.65424\n",
      "Document Ilorasertib - similarity: 0.65423\n",
      "Document Exenatide - similarity: 0.65419\n",
      "Document Fluorodopa (18F) - similarity: 0.65399\n",
      "Document 3-(7-DIAMINOMETHYL-NAPHTHALEN-2-YL)-PROPIONIC ACID ETHYL ESTER - similarity: 0.65390\n",
      "Document 5-[(3R)-3-(5-methoxy-4'-methylbiphenyl-3-yl)but-1-yn-1-yl]-6-methylpyrimidine-2,4-diamine - similarity: 0.65387\n",
      "Document N-ethyl-N-[3-(propylamino)propyl]propane-1,3-diamine - similarity: 0.65386\n",
      "Document Romiplostim - similarity: 0.65381\n",
      "Document Simoctocog alfa - similarity: 0.65365\n",
      "Document Pevonedistat - similarity: 0.65365\n",
      "Document Metamizole - similarity: 0.65364\n",
      "Document Lesinidase alfa - similarity: 0.65358\n",
      "Document Tinzaparin - similarity: 0.65339\n",
      "Document GS-5745 - similarity: 0.65333\n",
      "Document Semaxanib - similarity: 0.65327\n",
      "Document Barbital - similarity: 0.65326\n",
      "Document Arotinoid acid - similarity: 0.65321\n",
      "Document Camylofin - similarity: 0.65314\n",
      "Document Lopinavir - similarity: 0.65288\n",
      "Document Segesterone acetate - similarity: 0.65282\n",
      "Document JNJ-40411813 - similarity: 0.65281\n",
      "Document BMS-582949 - similarity: 0.65281\n",
      "Document Agrostis stolonifera pollen - similarity: 0.65279\n",
      "Document Ibutilide - similarity: 0.65276\n",
      "Document Pyridoxamine - similarity: 0.65271\n",
      "Document Onapristone - similarity: 0.65266\n",
      "Document Phenylmercuric nitrate - similarity: 0.65256\n",
      "Document (4R,5S,6S,7R)-1,3-dibenzyl-4,7-bis(phenoxymethyl)-5,6-dihydroxy-1,3 diazepan-2-one - similarity: 0.65250\n",
      "Document Lixivaptan - similarity: 0.65249\n",
      "Document Dimenhydrinate - similarity: 0.65242\n",
      "Document Magnesium - similarity: 0.65238\n",
      "Document Emedastine - similarity: 0.65235\n",
      "Document 4-(2-aminoethyl)-2-ethylphenol - similarity: 0.65234\n",
      "Document Benzoyl peroxide - similarity: 0.65231\n",
      "Document Octacosanol - similarity: 0.65226\n",
      "Document Mesocricetus auratus skin - similarity: 0.65221\n",
      "Document (3aS)-3a-hydroxy-1-phenyl-1,2,3,3a-tetrahydro-4H-pyrrolo[2,3-b]quinolin-4-one - similarity: 0.65209\n",
      "Document Lincomycin - similarity: 0.65205\n",
      "Document N-HYDROXY-2-[4-(4-PHENOXY-BENZENESULFONYL)-TETRAHYDRO-PYRAN-4-YL]-ACETAMIDE - similarity: 0.65204\n",
      "Document Pipendoxifene - similarity: 0.65200\n",
      "Document Adefovir - similarity: 0.65196\n",
      "Document Pegnivacogin - similarity: 0.65194\n",
      "Document Barbexaclone - similarity: 0.65192\n",
      "Document Volixibat - similarity: 0.65189\n",
      "Document Zolbetuximab - similarity: 0.65188\n",
      "Document DL-dimyristoylphosphatidylcholine - similarity: 0.65185\n",
      "Document Triclofos - similarity: 0.65177\n",
      "Document Sertaconazole - similarity: 0.65171\n",
      "Document Perzinfotel - similarity: 0.65168\n",
      "Document Denufosol - similarity: 0.65168\n",
      "Document P54 - similarity: 0.65165\n",
      "Document Tapinarof - similarity: 0.65164\n",
      "Document Ticarcillin - similarity: 0.65159\n",
      "Document Omiganan - similarity: 0.65158\n",
      "Document Olodaterol - similarity: 0.65148\n",
      "Document Lithium citrate - similarity: 0.65141\n",
      "Document Casein - similarity: 0.65135\n",
      "Document Carbaspirin calcium - similarity: 0.65128\n",
      "Document Pegfilgrastim - similarity: 0.65113\n",
      "Document Parathyroid hormone - similarity: 0.65104\n",
      "Document Odanacatib - similarity: 0.65100\n",
      "Document Esketamine - similarity: 0.65095\n",
      "Document BMS-863233 - similarity: 0.65094\n",
      "Document [4-({5-(AMINOCARBONYL)-4-[(3-METHYLPHENYL)AMINO]PYRIMIDIN-2-YL}AMINO)PHENYL]ACETIC ACID - similarity: 0.65093\n",
      "Document Tositumomab - similarity: 0.65092\n",
      "Document Torcetrapib - similarity: 0.65087\n",
      "Document Chloroxylenol - similarity: 0.65086\n",
      "Document Atopaxar - similarity: 0.65084\n",
      "Document Irosustat - similarity: 0.65084\n",
      "Document CAM-2029 - similarity: 0.65080\n",
      "Document Temocapril - similarity: 0.65080\n",
      "Document Theophylline - similarity: 0.65075\n",
      "Document Flurbiprofen Methyl Ester - similarity: 0.65069\n",
      "Document [1-(6-{6-[(1-methylethyl)amino]-1H-indazol-1-yl}pyrazin-2-yl)-1H-pyrrol-3-yl]acetic acid - similarity: 0.65063\n",
      "Document Barnidipine - similarity: 0.65057\n",
      "Document Elosulfase alfa - similarity: 0.65054\n",
      "Document Pirenzepine - similarity: 0.65046\n",
      "Document Platanus racemosa pollen - similarity: 0.65045\n",
      "Document 3-isobutyl-1-methyl-7H-xanthine - similarity: 0.65038\n",
      "Document Oryctolagus cuniculus hair - similarity: 0.65029\n",
      "Document Tesidolumab - similarity: 0.65026\n",
      "Document Avanafil - similarity: 0.65021\n",
      "Document PMI-001 - similarity: 0.65009\n",
      "Document Pancrelipase amylase - similarity: 0.65001\n",
      "Document Amylmetacresol - similarity: 0.64998\n",
      "Document Cefcapene - similarity: 0.64994\n",
      "Document rPSGL-Ig - similarity: 0.64993\n",
      "Document Salix alba bark - similarity: 0.64989\n",
      "Document Suaeda moquinii pollen - similarity: 0.64988\n",
      "Document Sodium chloride - similarity: 0.64984\n",
      "Document Floctafenine - similarity: 0.64980\n",
      "Document Geneticin - similarity: 0.64979\n",
      "Document M-2698 - similarity: 0.64979\n",
      "Document 1,2-Hexanediol - similarity: 0.64975\n",
      "Document Eflornithine - similarity: 0.64974\n",
      "Document Revefenacin - similarity: 0.64973\n",
      "Document 1-[5-Methyl-2-(trifluoromethyl)furan-3-yl]-3-[5-[2-[[6-(1H-1,2,4-triazol-5-ylamino)pyrimidin-4-yl]amino]ethyl]-1,3-thiazol-2-yl]urea - similarity: 0.64971\n",
      "Document THIENO[3,2-B]PYRIDINE-2-SULFONIC ACID [2-OXO-1-(1H-PYRROLO[2,3-C]PYRIDIN-2-YLMETHYL)-PYRROLIDIN-3-YL]-AMIDE - similarity: 0.64966\n",
      "Document 2-AMINO-4-(2,4-DICHLOROPHENYL)-N-ETHYLTHIENO[2,3-D]PYRIMIDINE-6-CARBOXAMIDE - similarity: 0.64964\n",
      "Document Resiquimod - similarity: 0.64959\n",
      "Document Procaterol - similarity: 0.64950\n",
      "Document Dulanermin - similarity: 0.64950\n",
      "Document NGX267 - similarity: 0.64950\n",
      "Document Tetrastearoyl cardiolipin - similarity: 0.64946\n",
      "Document Desvenlafaxine - similarity: 0.64941\n",
      "Document Cranberry - similarity: 0.64940\n",
      "Document 17Alpha-methyl-3beta,17beta-dihydroxy-5alpha-androstane - similarity: 0.64924\n",
      "Document Emactuzumab - similarity: 0.64917\n",
      "Document Vortioxetine - similarity: 0.64914\n",
      "Document Ceftiofur - similarity: 0.64911\n",
      "Document Troglitazone - similarity: 0.64894\n",
      "Document Darglitazone - similarity: 0.64890\n",
      "Document Proparacaine - similarity: 0.64886\n",
      "Document Darapladib - similarity: 0.64884\n",
      "Document Influenza A virus A/California/7/2009 X-179A (H1N1) antigen (formaldehyde inactivated) - similarity: 0.64884\n",
      "Document Naratuximab emtansine - similarity: 0.64877\n",
      "Document Gimeracil - similarity: 0.64876\n",
      "Document Dextroamphetamine - similarity: 0.64874\n",
      "Document Levobetaxolol - similarity: 0.64872\n",
      "Document Pelitrexol - similarity: 0.64872\n",
      "Document Trimetazidine - similarity: 0.64869\n",
      "Document Vactosertib - similarity: 0.64863\n",
      "Document Orris - similarity: 0.64856\n",
      "Document (9aS)-4-bromo-9a-butyl-7-hydroxy-1,2,9,9a-tetrahydro-3H-fluoren-3-one - similarity: 0.64843\n",
      "Document Segesterone - similarity: 0.64831\n",
      "Document Fluanisone - similarity: 0.64824\n",
      "Document Ammonium molybdate - similarity: 0.64822\n",
      "Document Haemagglutinin-strain B - similarity: 0.64818\n",
      "Document Pterin Cytosine Dinucleotide - similarity: 0.64816\n",
      "Document Arbaclofen - similarity: 0.64800\n",
      "Document 5-(3-carbamoylbenzyl)-5,6,7,8,9,10-hexahydrocyclohepta[b]indole-4-carboxylic acid - similarity: 0.64799\n",
      "Document Thymidine - similarity: 0.64799\n",
      "Document Celiprolol - similarity: 0.64797\n",
      "Document Phenoxybenzamine - similarity: 0.64796\n",
      "Document Etoglucid - similarity: 0.64793\n",
      "Document Ebastine - similarity: 0.64792\n",
      "Document Influenza A virus A/Brisbane/10/2010 (H1N1) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.64789\n",
      "Document Linopirdine - similarity: 0.64788\n",
      "Document Carrot - similarity: 0.64736\n",
      "Document Methyltestosterone - similarity: 0.64730\n",
      "Document Cotton - similarity: 0.64724\n",
      "Document Triptorelin - similarity: 0.64717\n",
      "Document Vespula germanica venom protein - similarity: 0.64715\n",
      "Document Dixanthogen - similarity: 0.64702\n",
      "Document TG100-115 - similarity: 0.64699\n",
      "Document Hydrogen peroxide - similarity: 0.64696\n",
      "Document Delanzomib - similarity: 0.64695\n",
      "Document Ginger oil - similarity: 0.64691\n",
      "Document Thiothixene - similarity: 0.64690\n",
      "Document Populus deltoides pollen - similarity: 0.64690\n",
      "Document SUVN-502 - similarity: 0.64687\n",
      "Document Phenmetrazine - similarity: 0.64683\n",
      "Document Lidocaine - similarity: 0.64675\n",
      "Document VM4-037 - similarity: 0.64673\n",
      "Document Metreleptin - similarity: 0.64673\n",
      "Document Influenza B virus B/Brisbane/9/2014 antigen (formaldehyde inactivated) - similarity: 0.64671\n",
      "Document Iloperidone - similarity: 0.64664\n",
      "Document 7-(1-Methyl-1,2,3-Triazol-4-Yl)-6-Formyl-2,7-Dihydro-[1,4]Thiazepine-3-Carboxylic Acid, Brl42715, C6-(N1-Methyl-1,2,3-Triazolylmethylene)Penem - similarity: 0.64663\n",
      "Document Adenosine-5'-[Beta, Gamma-Methylene]Tetraphosphate - similarity: 0.64661\n",
      "Document Navoximod - similarity: 0.64660\n",
      "Document Mumps virus strain B level jeryl lynn live antigen - similarity: 0.64656\n",
      "Document 4-hydroxybutan-2-one - similarity: 0.64654\n",
      "Document KW-7158 - similarity: 0.64647\n",
      "Document Quercus rubra pollen - similarity: 0.64643\n",
      "Document Tertatolol - similarity: 0.64641\n",
      "Document CD564 - similarity: 0.64616\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Mirabegron - similarity: 0.64612\n",
      "Document Influenza B virus B/Massachusetts/2/2012 recombinant hemagglutinin antigen - similarity: 0.64608\n",
      "Document Triglyme - similarity: 0.64606\n",
      "Document Calaspargase pegol - similarity: 0.64603\n",
      "Document PR1 leukemia peptide vaccine - similarity: 0.64599\n",
      "Document 2-Allylphenol - similarity: 0.64598\n",
      "Document Epinastine - similarity: 0.64597\n",
      "Document Cynodon dactylon pollen - similarity: 0.64594\n",
      "Document Tepoxalin - similarity: 0.64593\n",
      "Document Micronomicin - similarity: 0.64584\n",
      "Document Fluvastatin - similarity: 0.64583\n",
      "Document Annamycin - similarity: 0.64583\n",
      "Document 3,6-DIAMINO-5-CYANO-4-(4-ETHOXYPHENYL)THIENO[2,3-B]PYRIDINE-2-CARBOXAMIDE - similarity: 0.64579\n",
      "Document Tiletamine - similarity: 0.64579\n",
      "Document Tigatuzumab - similarity: 0.64570\n",
      "Document Streptococcus pneumoniae type 9v capsular polysaccharide antigen - similarity: 0.64556\n",
      "Document Dopexamine - similarity: 0.64548\n",
      "Document White mustard seed - similarity: 0.64547\n",
      "Document 4,5-Dehydro-D-Glucuronic Acid - similarity: 0.64538\n",
      "Document Dexloxiglumide - similarity: 0.64535\n",
      "Document Sodium molybdate - similarity: 0.64518\n",
      "Document Prexasertib - similarity: 0.64506\n",
      "Document OTX-008 - similarity: 0.64501\n",
      "Document Ranibizumab - similarity: 0.64499\n",
      "Document Phenylbutazone - similarity: 0.64498\n",
      "Document Lagerstroemia indica whole - similarity: 0.64495\n",
      "Document Enoxaparin - similarity: 0.64493\n",
      "Document Aloe ferox leaf - similarity: 0.64485\n",
      "Document Potassium Guaiacolsulfonate - similarity: 0.64484\n",
      "Document (2S)-3-[(9H-fluoren-9-ylideneamino)oxy]-2-methylpropanoic acid - similarity: 0.64465\n",
      "Document ATX-101 - similarity: 0.64463\n",
      "Document Potassium sulfate - similarity: 0.64453\n",
      "Document Ferric maltol - similarity: 0.64446\n",
      "Document (7R,12R,13R)-13-formyl-12,14-dihydroxy-3,5,7-trimethyltetradeca-2,4-dienoic acid - similarity: 0.64438\n",
      "Document 7-Aza-L-tryptophan - similarity: 0.64435\n",
      "Document Fluprednisolone acetate - similarity: 0.64425\n",
      "Document Butalbital - similarity: 0.64424\n",
      "Document L-Baclofen - similarity: 0.64412\n",
      "Document Brexpiprazole - similarity: 0.64399\n",
      "Document AZD 3355 - similarity: 0.64379\n",
      "Document Erythropoietin - similarity: 0.64371\n",
      "Document Monothioglycerol - similarity: 0.64367\n",
      "Document Perphenazine enanthate - similarity: 0.64365\n",
      "Document Estramustine phosphate - similarity: 0.64359\n",
      "Document N-Hexylphosphonate Ethyl Ester - similarity: 0.64355\n",
      "Document Phenytoin - similarity: 0.64347\n",
      "Document 6-PHENYL[5H]PYRROLO[2,3-B]PYRAZINE - similarity: 0.64332\n",
      "Document AT-001 - similarity: 0.64326\n",
      "Document 2-Iminobiotin - similarity: 0.64324\n",
      "Document Panobacumab - similarity: 0.64323\n",
      "Document Pamidronic acid - similarity: 0.64319\n",
      "Document Cobicistat - similarity: 0.64315\n",
      "Document Phenazopyridine - similarity: 0.64314\n",
      "Document Treprostinil - similarity: 0.64314\n",
      "Document Potassium gluconate - similarity: 0.64308\n",
      "Document Talopram - similarity: 0.64308\n",
      "Document Perospirone - similarity: 0.64307\n",
      "Document Difluocortolone - similarity: 0.64297\n",
      "Document Brentuximab vedotin - similarity: 0.64290\n",
      "Document PI-0824 - similarity: 0.64285\n",
      "Document Bevasiranib - similarity: 0.64284\n",
      "Document Trenbolone acetate - similarity: 0.64284\n",
      "Document Adamantanone - similarity: 0.64278\n",
      "Document Cyclobenzaprine - similarity: 0.64272\n",
      "Document Roluperidone - similarity: 0.64271\n",
      "Document Apraclonidine - similarity: 0.64252\n",
      "Document Clinofibrate - similarity: 0.64249\n",
      "Document Xylitol - similarity: 0.64244\n",
      "Document 18-methyl-19-nortestosterone - similarity: 0.64244\n",
      "Document Phosphatidyl ethanol - similarity: 0.64235\n",
      "Document Hemi-Babim - similarity: 0.64225\n",
      "Document Quinalizarin - similarity: 0.64215\n",
      "Document Fluocortolone - similarity: 0.64212\n",
      "Document Enzalutamide - similarity: 0.64210\n",
      "Document Isoxsuprine - similarity: 0.64204\n",
      "Document Chloropyramine - similarity: 0.64198\n",
      "Document Follitropin - similarity: 0.64193\n",
      "Document Virginiamycin - similarity: 0.64189\n",
      "Document Shea butter - similarity: 0.64181\n",
      "Document Lersivirine - similarity: 0.64180\n",
      "Document Light Mineral Oil - similarity: 0.64178\n",
      "Document Mibefradil - similarity: 0.64174\n",
      "Document Ziritaxestat - similarity: 0.64171\n",
      "Document Lynestrenol - similarity: 0.64162\n",
      "Document Starch, tapioca - similarity: 0.64155\n",
      "Document Naphazoline - similarity: 0.64153\n",
      "Document Isradipine - similarity: 0.64152\n",
      "Document (4R)-7-Aza-7,8-dihydrolimonene - similarity: 0.64149\n",
      "Document 3-Cyclohexyl-L-alanine - similarity: 0.64148\n",
      "Document Roxithromycin - similarity: 0.64147\n",
      "Document ABP-700 - similarity: 0.64142\n",
      "Document Cusatuzumab - similarity: 0.64136\n",
      "Document Iodopropynyl butylcarbamate - similarity: 0.64134\n",
      "Document Barasertib - similarity: 0.64131\n",
      "Document Saruplase - similarity: 0.64118\n",
      "Document Dersalazine - similarity: 0.64118\n",
      "Document Fexinidazole - similarity: 0.64112\n",
      "Document Cyclomethicone - similarity: 0.64104\n",
      "Document N-(3-(8-CYANO-4-(PHENYLAMINO)PYRAZOLO[1,5-A][1,3,5]TRIAZIN-2-YLAMINO)PHENYL)ACETAMIDE - similarity: 0.64095\n",
      "Document 1-O-[O-Nitrophenyl]-Beta-D-Galactopyranose - similarity: 0.64092\n",
      "Document Adenosine-5'-[Beta, Gamma-Methylene]Triphosphate - similarity: 0.64091\n",
      "Document Nortriptyline - similarity: 0.64077\n",
      "Document Gluconic Acid - similarity: 0.64066\n",
      "Document Conbercept - similarity: 0.64046\n",
      "Document Quercus palustris pollen - similarity: 0.64037\n",
      "Document Salmon calcitonin - similarity: 0.64030\n",
      "Document Verubecestat - similarity: 0.64029\n",
      "Document Neisseria meningitidis serogroup b fhbp fusion protein antigen - similarity: 0.64016\n",
      "Document Amisulpride - similarity: 0.63998\n",
      "Document Proglumetacin - similarity: 0.63993\n",
      "Document 2-({4-[(5-CHLORO-1H-INDOL-2-YL)SULFONYL]PIPERAZIN-1-YL}CARBONYL)THIENO[3,2-B]PYRIDINE 4-OXIDE - similarity: 0.63988\n",
      "Document Methyl(6s)-1-Thio-L-Manno-Hexodialdo-6,2-Pyranoside - similarity: 0.63977\n",
      "Document 4-Methyl-5-Hydroxyethylthiazole Phosphate - similarity: 0.63974\n",
      "Document 3-(2-Aminoethyl)-4-(Aminomethyl)Heptanedioic Acid - similarity: 0.63973\n",
      "Document Interferon alfa - similarity: 0.63969\n",
      "Document 3-(4-HYDROXYPHENYL)-4,5-DIHYDRO-5-ISOXAZOLE-ACETIC ACID METHYL ESTER - similarity: 0.63969\n",
      "Document N1-(2-AMINO-4-METHYLPENTYL)OCTAHYDRO-PYRROLO[1,2-A] PYRIMIDINE - similarity: 0.63968\n",
      "Document Prasugrel - similarity: 0.63962\n",
      "Document Hordeum vulgare pollen - similarity: 0.63953\n",
      "Document Vipadenant - similarity: 0.63947\n",
      "Document S,S-(2-Hydroxyethyl)Thiocysteine - similarity: 0.63946\n",
      "Document Imetelstat - similarity: 0.63933\n",
      "Document Zileuton - similarity: 0.63931\n",
      "Document Rasagiline - similarity: 0.63926\n",
      "Document Quinine - similarity: 0.63925\n",
      "Document Sunflower oil - similarity: 0.63922\n",
      "Document Ramiprilat - similarity: 0.63916\n",
      "Document N-(4-AMINO-5-CYANO-6-ETHOXYPYRIDIN-2-YL)-2-(4-BROMO-2,5-DIMETHOXYPHENYL)ACETAMIDE - similarity: 0.63900\n",
      "Document Melanoma antigen recognized by T-cells 1 - similarity: 0.63894\n",
      "Document (2S,3S)-3-{3-[2-chloro-4-(methylsulfonyl)phenyl]-1,2,4-oxadiazol-5-yl}-1-cyclopentylidene-4-cyclopropyl-1-fluorobutan-2-amine - similarity: 0.63880\n",
      "Document S-(N-hydroxy-N-iodophenylcarbamoyl)glutathione - similarity: 0.63872\n",
      "Document Caroverine - similarity: 0.63871\n",
      "Document Tulathromycin A - similarity: 0.63869\n",
      "Document Bacillus calmette-guerin substrain tice live antigen - similarity: 0.63862\n",
      "Document 3-bromo-5-phenyl-N-(pyridin-4-ylmethyl)pyrazolo[1,5-a]pyrimidin-7-amine - similarity: 0.63856\n",
      "Document Compound 5, 2-(Naphthalen-1-Yl-Oxalyl-Amino)-Benzoicacid - similarity: 0.63849\n",
      "Document Foscarnet - similarity: 0.63848\n",
      "Document Prucalopride - similarity: 0.63844\n",
      "Document Chymopapain - similarity: 0.63840\n",
      "Document Nicofuranose - similarity: 0.63838\n",
      "Document Phenelzine - similarity: 0.63836\n",
      "Document 1-(3-bromophenyl)-7-chloro-6-methoxy-3,4-dihydroisoquinoline - similarity: 0.63835\n",
      "Document Thyrotropin alfa - similarity: 0.63832\n",
      "Document Trifluorofurnesyl Diphosphate - similarity: 0.63832\n",
      "Document Pancrelipase - similarity: 0.63831\n",
      "Document MIP-1095 I-131 - similarity: 0.63826\n",
      "Document Itopride - similarity: 0.63824\n",
      "Document Levothyroxine - similarity: 0.63814\n",
      "Document Gadoxetic acid - similarity: 0.63807\n",
      "Document Mephentermine - similarity: 0.63801\n",
      "Document Tofogliflozin - similarity: 0.63798\n",
      "Document Tacedinaline - similarity: 0.63763\n",
      "Document Flumedroxone - similarity: 0.63756\n",
      "Document Bis(6-aminohexyl)amine - similarity: 0.63746\n",
      "Document Ingenol disoxate - similarity: 0.63735\n",
      "Document 5,6-Dimethylbenzimidazole - similarity: 0.63729\n",
      "Document MK-1775 - similarity: 0.63729\n",
      "Document Zolpidem - similarity: 0.63712\n",
      "Document Apitolisib - similarity: 0.63705\n",
      "Document Fosravuconazole - similarity: 0.63704\n",
      "Document Lysozyme - similarity: 0.63698\n",
      "Document AN-9 - similarity: 0.63697\n",
      "Document JNJ-27390467 - similarity: 0.63681\n",
      "Document Diphemanil - similarity: 0.63675\n",
      "Document Potassium carbonate - similarity: 0.63674\n",
      "Document Perazine - similarity: 0.63672\n",
      "Document {4-[(CARBOXYMETHOXY)CARBONYL]-3,3-DIOXIDO-1-OXONAPHTHO[1,2-D]ISOTHIAZOL-2(1H)-YL}ACETIC ACID - similarity: 0.63648\n",
      "Document 4-O-methyl-beta-D-glucuronic acid - similarity: 0.63648\n",
      "Document Talacotuzumab - similarity: 0.63639\n",
      "Document Trypsin - similarity: 0.63636\n",
      "Document Seliciclib - similarity: 0.63617\n",
      "Document N-cyclohexyl-3-[3-(trifluoromethyl)phenyl][1,2,4]triazolo[4,3-b]pyridazin-6-amine - similarity: 0.63617\n",
      "Document Hexylphosphonic acid (R)-2-methyl-3-phenylpropyl ester - similarity: 0.63606\n",
      "Document Mivacurium - similarity: 0.63601\n",
      "Document CH-5132799 - similarity: 0.63600\n",
      "Document Bedaquiline - similarity: 0.63596\n",
      "Document 6-(2-HYDROXY-CYCLOPENTYL)-7-OXO-HEPTANAMIDINE - similarity: 0.63583\n",
      "Document Zilucoplan - similarity: 0.63577\n",
      "Document Metildigoxin - similarity: 0.63572\n",
      "Document Duligotuzumab - similarity: 0.63570\n",
      "Document 3,3-Dichloro-2-Phosphonomethyl-Acrylic Acid - similarity: 0.63566\n",
      "Document Dixyrazine - similarity: 0.63565\n",
      "Document Afimoxifene - similarity: 0.63564\n",
      "Document Dydrogesterone - similarity: 0.63535\n",
      "Document Pinatuzumab vedotin - similarity: 0.63525\n",
      "Document ORG-25435 - similarity: 0.63514\n",
      "Document Varespladib methyl - similarity: 0.63512\n",
      "Document 5-(DIMETHYLAMINO)-1-NAPHTHALENESULFONIC ACID(DANSYL ACID) - similarity: 0.63500\n",
      "Document D-Malic acid - similarity: 0.63489\n",
      "Document Famciclovir - similarity: 0.63482\n",
      "Document Volasertib - similarity: 0.63481\n",
      "Document Butafenacil - similarity: 0.63481\n",
      "Document 3-[(Acetyl-Methyl-Amino)-Methyl]-4-Amino-N-Methyl-N-(1-Methyl-1h-Indol-2-Ylmethyl)-Benzamide - similarity: 0.63470\n",
      "Document Buprenorphine - similarity: 0.63469\n",
      "Document 7-thionicotinamide-adenine-dinucleotide phosphate - similarity: 0.63466\n",
      "Document Demplatin pegraglumer - similarity: 0.63455\n",
      "Document Cladosporium sphaerospermum - similarity: 0.63445\n",
      "Document Clodronic acid - similarity: 0.63444\n",
      "Document Levoleucovorin - similarity: 0.63441\n",
      "Document GS 0573 - similarity: 0.63430\n",
      "Document Rosuvastatin - similarity: 0.63418\n",
      "Document Utomilumab - similarity: 0.63417\n",
      "Document Cediranib - similarity: 0.63412\n",
      "Document Rebastinib - similarity: 0.63406\n",
      "Document N2-[(Benzyloxy)carbonyl]-N-[(3R)-1-{N-[(benzyloxy)carbonyl]-L-leucyl}-4-oxo-3-pyrrolidinyl]-L-leucinamide - similarity: 0.63404\n",
      "Document Sarcobatus vermiculatus pollen - similarity: 0.63403\n",
      "Document Juniperus pinchotii pollen - similarity: 0.63387\n",
      "Document Aramchol - similarity: 0.63386\n",
      "Document Histidine - similarity: 0.63373\n",
      "Document Antipyrine - similarity: 0.63372\n",
      "Document Bepridil - similarity: 0.63369\n",
      "Document Ambrosia trifida pollen - similarity: 0.63368\n",
      "Document 7-[2-METHOXY-1-(METHOXYMETHYL)ETHYL]-7H-PYRROLO[3,2-F] QUINAZOLINE-1,3-DIAMINE - similarity: 0.63367\n",
      "Document Apatorsen - similarity: 0.63366\n",
      "Document Magnesium aspartate - similarity: 0.63363\n",
      "Document 4-Vinylguaiacol - similarity: 0.63359\n",
      "Document Cefpirome - similarity: 0.63359\n",
      "Document Influenza A virus A/Uruguay/716/2007(H3N2) hemagglutinin antigen (UV,formaldehyde inactivated) - similarity: 0.63359\n",
      "Document Hydroxy(oxo)(2-{(1S)-2,2,2-trifluoro-1-[2-(trimethylarsonio)ethoxy]ethyl}phenyl)ammonium - similarity: 0.63356\n",
      "Document Cardarine - similarity: 0.63349\n",
      "Document Lactobacillus reuteri - similarity: 0.63345\n",
      "Document Doxapram - similarity: 0.63338\n",
      "Document Thebacon - similarity: 0.63337\n",
      "Document [2-(5-Mercapto-[1,3,4]thiadiazol-2-ylcarbamoyl)-1-phenyl-ethyl]-carbamic acid benzyl ester - similarity: 0.63336\n",
      "Document Egg yolk - similarity: 0.63331\n",
      "Document Perboric acid - similarity: 0.63327\n",
      "Document Sodium tripolyphosphate - similarity: 0.63310\n",
      "Document Elbasvir - similarity: 0.63308\n",
      "Document Crenolanib - similarity: 0.63299\n",
      "Document LY2140023 - similarity: 0.63298\n",
      "Document Decoquinate - similarity: 0.63298\n",
      "Document Buspirone - similarity: 0.63295\n",
      "Document Betula nigra pollen - similarity: 0.63285\n",
      "Document Kabiramide C - similarity: 0.63282\n",
      "Document GSK-1059615 - similarity: 0.63280\n",
      "Document Terfenadine - similarity: 0.63276\n",
      "Document 1,2,3-TRIHYDROXY-1,2,3,4-TETRAHYDROBENZO[A]PYRENE - similarity: 0.63262\n",
      "Document TG-100801 - similarity: 0.63255\n",
      "Document 5-androstenedione - similarity: 0.63247\n",
      "Document 4-Amino Hexanoic Acid - similarity: 0.63242\n",
      "Document Thiamine - similarity: 0.63242\n",
      "Document Tabalumab - similarity: 0.63241\n",
      "Document (9ALPHA,13BETA,17BETA)-2-[(1Z)-BUT-1-EN-1-YL]ESTRA-1,3,5(10)-TRIENE-3,17-DIOL - similarity: 0.63240\n",
      "Document FR-234938 - similarity: 0.63239\n",
      "Document Almond - similarity: 0.63239\n",
      "Document Mitiglinide - similarity: 0.63239\n",
      "Document Blackberry - similarity: 0.63231\n",
      "Document Lemborexant - similarity: 0.63230\n",
      "Document Alglucosidase alfa - similarity: 0.63230\n",
      "Document METHYL N-[(2',4'-DIFLUORO-4-HYDROXY-5-IODOBIPHENYL-3-YL)CARBONYL]-BETA-ALANINATE - similarity: 0.63225\n",
      "Document E-6201 - similarity: 0.63223\n",
      "Document Influenza B virus B/Singapore/INFTT-16-0610/2016 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.63216\n",
      "Document LY-2090314 - similarity: 0.63216\n",
      "Document Acacia dealbata pollen - similarity: 0.63199\n",
      "Document Humicola grisea - similarity: 0.63196\n",
      "Document N-[O-Phosphono-Pyridoxyl]-Isoleucine - similarity: 0.63194\n",
      "Document Sacubitrilat - similarity: 0.63189\n",
      "Document S-(2-{[N-(2-HYDROXY-4-{[HYDROXY(OXIDO)PHOSPHINO]OXY}-3,3-DIMETHYLBUTANOYL)-BETA-ALANYL]AMINO}ETHYL) HEPTANETHIOATE - similarity: 0.63186\n",
      "Document Psilocybin - similarity: 0.63184\n",
      "Document Gemtuzumab ozogamicin - similarity: 0.63183\n",
      "Document Indenolol - similarity: 0.63182\n",
      "Document Gemifloxacin - similarity: 0.63172\n",
      "Document Lysergic acid diethylamide - similarity: 0.63171\n",
      "Document Ciprofloxacin - similarity: 0.63153\n",
      "Document Sifalimumab - similarity: 0.63139\n",
      "Document S-[3-(3,4-DICHLOROPHENYL)-3-OXOPROPYL]-L-CYSTEINE - similarity: 0.63139\n",
      "Document 2'-cyano-2'-deoxy-1-(beta-D-arabinofuranosyl)cytosine - similarity: 0.63133\n",
      "Document 4-amino-5-aminomethyl-2-methylpyrimidine - similarity: 0.63132\n",
      "Document Alfuzosin - similarity: 0.63130\n",
      "Document 2-deoxy-2-fluoro-alpha-D-mannosyl fluoride - similarity: 0.63121\n",
      "Document Gilteritinib - similarity: 0.63117\n",
      "Document Artesunate - similarity: 0.63110\n",
      "Document Namitecan - similarity: 0.63106\n",
      "Document Carba-nicotinamide-adenine-dinucleotide - similarity: 0.63103\n",
      "Document EHT899 - similarity: 0.63088\n",
      "Document Cefroxadine - similarity: 0.63085\n",
      "Document Tramiprosate - similarity: 0.63083\n",
      "Document Mebicar - similarity: 0.63081\n",
      "Document Chicken - similarity: 0.63073\n",
      "Document (2S)-2-(3-bromophenyl)-3-(5-chloro-2-hydroxyphenyl)-1,3-thiazolidin-4-one - similarity: 0.63063\n",
      "Document Trandolapril - similarity: 0.63041\n",
      "Document Cabozantinib - similarity: 0.63036\n",
      "Document Altrenogest - similarity: 0.63031\n",
      "Document Pumactant - similarity: 0.63023\n",
      "Document Flumazenil - similarity: 0.63020\n",
      "Document Parsaclisib - similarity: 0.63018\n",
      "Document Imagabalin - similarity: 0.63016\n",
      "Document 4-Methylaminorex - similarity: 0.62997\n",
      "Document Dirlotapide - similarity: 0.62994\n",
      "Document Lucinactant - similarity: 0.62988\n",
      "Document Equilenin - similarity: 0.62981\n",
      "Document Nadolol - similarity: 0.62977\n",
      "Document Magnesium sulfate - similarity: 0.62971\n",
      "Document Methylthioninium - similarity: 0.62967\n",
      "Document Serotonin - similarity: 0.62958\n",
      "Document Adenosine-5'-Monophosphate Glucopyranosyl-Monophosphate Ester - similarity: 0.62955\n",
      "Document Ciprofibrate - similarity: 0.62940\n",
      "Document Monteplase - similarity: 0.62930\n",
      "Document Balstilimab - similarity: 0.62926\n",
      "Document Nitisinone - similarity: 0.62919\n",
      "Document Triglu-5-formyl-tetrahydrofolate - similarity: 0.62909\n",
      "Document GTI-2040 - similarity: 0.62909\n",
      "Document Figitumumab - similarity: 0.62906\n",
      "Document Ozanezumab - similarity: 0.62904\n",
      "Document Secobarbital - similarity: 0.62903\n",
      "Document Terodiline - similarity: 0.62896\n",
      "Document Anakinra - similarity: 0.62889\n",
      "Document Stannous fluoride - similarity: 0.62889\n",
      "Document Droxicam - similarity: 0.62886\n",
      "Document TRX1 - similarity: 0.62876\n",
      "Document 3-[(4-CHLOROANILINO)SULFONYL]THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.62868\n",
      "Document Oxitropium - similarity: 0.62868\n",
      "Document SCYX-7158 - similarity: 0.62865\n",
      "Document Green olive - similarity: 0.62855\n",
      "Document PF-04634817 - similarity: 0.62849\n",
      "Document S-(3-Iodobenzyl)glutathione - similarity: 0.62848\n",
      "Document Terbinafine - similarity: 0.62841\n",
      "Document Amenamevir - similarity: 0.62836\n",
      "Document Sus scrofa skin - similarity: 0.62834\n",
      "Document 3-CHLORO-2,2-DIMETHYL-N-[4-(TRIFLUOROMETHYL)PHENYL]PROPANAMIDE - similarity: 0.62834\n",
      "Document Atlantic mackerel - similarity: 0.62826\n",
      "Document Burosumab - similarity: 0.62812\n",
      "Document 4-[3-Methylsulfanylanilino]-6,7-Dimethoxyquinazoline - similarity: 0.62811\n",
      "Document Mafenide - similarity: 0.62804\n",
      "Document Nifurzide - similarity: 0.62789\n",
      "Document Anrukinzumab - similarity: 0.62789\n",
      "Document 3'-Phosphate-Adenosine-5'-Diphosphate - similarity: 0.62788\n",
      "Document COVID-19 convalescent plasma - similarity: 0.62787\n",
      "Document Rumex obtusifolius pollen - similarity: 0.62781\n",
      "Document Brown iron oxide - similarity: 0.62780\n",
      "Document 5-(2,3-dichlorophenyl)-N-(pyridin-4-ylmethyl)-3-thiocyanatopyrazolo[1,5-a]pyrimidin-7-amine - similarity: 0.62775\n",
      "Document Promestriene - similarity: 0.62774\n",
      "Document Oxametacin - similarity: 0.62774\n",
      "Document Gallopamil - similarity: 0.62763\n",
      "Document Oxycodone - similarity: 0.62761\n",
      "Document (1Z)-4-(4-FLUOROPHENYL)-2-METHYLIDENEBUTAN-1-IMINE - similarity: 0.62753\n",
      "Document Nocodazole - similarity: 0.62750\n",
      "Document Ademetionine - similarity: 0.62749\n",
      "Document Temocillin - similarity: 0.62749\n",
      "Document Calcium ascorbate - similarity: 0.62747\n",
      "Document Idelalisib - similarity: 0.62746\n",
      "Document Flumatinib - similarity: 0.62712\n",
      "Document N-(6-Aminohexyl)-5-Chloro-1-Naphthalenesulfonamide - similarity: 0.62707\n",
      "Document Tricaine - similarity: 0.62683\n",
      "Document Betaxolol - similarity: 0.62680\n",
      "Document Diisopromine - similarity: 0.62678\n",
      "Document Imidacloprid - similarity: 0.62661\n",
      "Document Cefpiramide - similarity: 0.62661\n",
      "Document Rimexolone - similarity: 0.62654\n",
      "Document Imidapril - similarity: 0.62650\n",
      "Document Influenza A virus A/Texas/50/2012 X-223 (H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.62649\n",
      "Document Cefditoren - similarity: 0.62643\n",
      "Document Canertinib - similarity: 0.62641\n",
      "Document Tetomilast - similarity: 0.62641\n",
      "Document 5-{4-[(3,5-DIFLUOROBENZYL)AMINO]PHENYL}-6-ETHYLPYRIMIDINE-2,4-DIAMINE - similarity: 0.62641\n",
      "Document Trichoderma viride - similarity: 0.62641\n",
      "Document Tetrakis(2-methoxyisobutylisocyanide)copper(I) tetrafluoroborate - similarity: 0.62637\n",
      "Document Abicipar Pegol - similarity: 0.62637\n",
      "Document Hazelnut - similarity: 0.62624\n",
      "Document Panobinostat - similarity: 0.62615\n",
      "Document 6-HYDROXY-1,3-BENZOTHIAZOLE-2-SULFONAMIDE - similarity: 0.62614\n",
      "Document Miltefosine - similarity: 0.62612\n",
      "Document Indapamide - similarity: 0.62609\n",
      "Document Turmeric - similarity: 0.62604\n",
      "Document Oxibendazole - similarity: 0.62602\n",
      "Document Rumex salicifolius var. mexicanus pollen - similarity: 0.62600\n",
      "Document 4-(3-ethylthiophen-2-yl)benzene-1,2-diol - similarity: 0.62600\n",
      "Document Kanamycin - similarity: 0.62593\n",
      "Document Desoxycorticosterone pivalate - similarity: 0.62591\n",
      "Document octyl 3-amino-3-deoxy-2-O-(2,6-dideoxy-alpha-L-lyxo-hexopyranosyl)-beta-D-galactopyranoside - similarity: 0.62581\n",
      "Document Voglibose - similarity: 0.62579\n",
      "Document Dichlororibofuranosylbenzimidazole - similarity: 0.62579\n",
      "Document Gedocarnil - similarity: 0.62576\n",
      "Document 6-[3-(4-Morpholinyl)Propyl]-2-(3-Nitrophenyl)-5-Thioxo-5,6,-Dihydro-7h-Thienol[2',3':4,5]Pyrrolo[1,2-C]Imidazol-7-One - similarity: 0.62572\n",
      "Document Isoeugenol - similarity: 0.62526\n",
      "Document 123I-iodometomidate - similarity: 0.62519\n",
      "Document Iscalimab - similarity: 0.62512\n",
      "Document Influenza B virus B/Phuket/3073/2013 live (attenuated) antigen - similarity: 0.62504\n",
      "Document Tesevatinib - similarity: 0.62500\n",
      "Document Mesalazine - similarity: 0.62478\n",
      "Document Lurtotecan - similarity: 0.62474\n",
      "Document Sodium chromate Cr-51 - similarity: 0.62472\n",
      "Document PF-06650833 - similarity: 0.62461\n",
      "Document Neosaxitoxin - similarity: 0.62453\n",
      "Document Terazosin - similarity: 0.62452\n",
      "Document 6-(4-chloro-2-fluoro-3-phenoxybenzyl)pyridazin-3(2H)-one - similarity: 0.62451\n",
      "Document Aminolevulinic acid - similarity: 0.62448\n",
      "Document Insulin tregopil - similarity: 0.62448\n",
      "Document Semuloparin - similarity: 0.62443\n",
      "Document Isavuconazole - similarity: 0.62434\n",
      "Document Amprenavir - similarity: 0.62432\n",
      "Document Zibotentan - similarity: 0.62420\n",
      "Document Primidone - similarity: 0.62419\n",
      "Document Conivaptan - similarity: 0.62414\n",
      "Document Belladonna - similarity: 0.62413\n",
      "Document Phenoxymethylpenicillin - similarity: 0.62412\n",
      "Document Oregano - similarity: 0.62384\n",
      "Document Schizochytrium DHA oil - similarity: 0.62382\n",
      "Document 2-[(CYCLOPROPYLCARBONYL)AMINO]-4,5,6,7-TETRAHYDRO-1-BENZOTHIOPHENE-3-CARBOXAMIDE - similarity: 0.62381\n",
      "Document ABT-288 - similarity: 0.62379\n",
      "Document Trehalose-6-Phosphate - similarity: 0.62377\n",
      "Document Milk thistle - similarity: 0.62377\n",
      "Document Imatinib - similarity: 0.62373\n",
      "Document Skim milk - similarity: 0.62369\n",
      "Document Influenza A virus A/Kansas/14/2017 X-327 (H3N2) antigen (formaldehyde inactivated) - similarity: 0.62368\n",
      "Document Urethane - similarity: 0.62346\n",
      "Document BMS-690514 - similarity: 0.62334\n",
      "Document Cloprostenol - similarity: 0.62325\n",
      "Document Bromothalonil - similarity: 0.62308\n",
      "Document Mexiletine - similarity: 0.62307\n",
      "Document Macelignan - similarity: 0.62292\n",
      "Document Chlorzoxazone - similarity: 0.62284\n",
      "Document Droxidopa - similarity: 0.62263\n",
      "Document Pantoprazole - similarity: 0.62258\n",
      "Document Telmisartan - similarity: 0.62256\n",
      "Document Penfluridol - similarity: 0.62254\n",
      "Document Opicinumab - similarity: 0.62238\n",
      "Document Pregabalin - similarity: 0.62237\n",
      "Document Ichthammol - similarity: 0.62237\n",
      "Document 3-FLUORO-4-HYDROXYBENZALDEHYDE O-(CYCLOHEXYLCARBONYL)OXIME - similarity: 0.62232\n",
      "Document Omaveloxolone - similarity: 0.62231\n",
      "Document Histidyl-Adenosine Monophosphate - similarity: 0.62228\n",
      "Document Melitracen - similarity: 0.62228\n",
      "Document Influenza B virus B/Maryland/15/2016 BX-69A antigen (formaldehyde inactivated) - similarity: 0.62223\n",
      "Document Clopidogrel - similarity: 0.62222\n",
      "Document Piclozotan - similarity: 0.62215\n",
      "Document Saw palmetto - similarity: 0.62210\n",
      "Document Delamanid - similarity: 0.62204\n",
      "Document Gefitinib - similarity: 0.62203\n",
      "Document Atlantic cod - similarity: 0.62203\n",
      "Document 1-Hydroxy-1-Thio-Glycerol - similarity: 0.62198\n",
      "Document Mercaptoethanol - similarity: 0.62188\n",
      "Document Zinc chloride - similarity: 0.62187\n",
      "Document Dasiglucagon - similarity: 0.62182\n",
      "Document Inosine 5'-sulfate - similarity: 0.62180\n",
      "Document Phentolamine - similarity: 0.62175\n",
      "Document Dichloroisoproterenol - similarity: 0.62166\n",
      "Document Dihydrostreptomycin - similarity: 0.62154\n",
      "Document Plantago lanceolata pollen - similarity: 0.62154\n",
      "Document N,N'-DIMETHYL-N-(ACETYL)-N'-(7-NITROBENZ-2-OXA-1,3-DIAZOL-4-YL)ETHYLENEDIAMINE - similarity: 0.62146\n",
      "Document Ponatinib - similarity: 0.62143\n",
      "Document Brecanavir - similarity: 0.62141\n",
      "Document Influenza A virus A/Singapore/INFIMH-16-0019/2016 NIB-104 (H3N2) antigen (formaldehyde inactivated) - similarity: 0.62137\n",
      "Document 18-methoxycoronaridine - similarity: 0.62134\n",
      "Document Yellowfin tuna - similarity: 0.62130\n",
      "Document QGC-001 - similarity: 0.62121\n",
      "Document MYO-029 - similarity: 0.62110\n",
      "Document GDC-0623 - similarity: 0.62100\n",
      "Document Naveglitazar - similarity: 0.62092\n",
      "Document N-cycloheptylglycyl-N-(4-carbamimidoylbenzyl)-L-prolinamide - similarity: 0.62091\n",
      "Document Fingolimod - similarity: 0.62089\n",
      "Document Mifepristone - similarity: 0.62088\n",
      "Document Hydrogenated coconut oil - similarity: 0.62087\n",
      "Document Nesbuvir - similarity: 0.62075\n",
      "Document Seproxetine - similarity: 0.62067\n",
      "Document PAC-1 - similarity: 0.62066\n",
      "Document Adalimumab - similarity: 0.62065\n",
      "Document Piperonyl butoxide - similarity: 0.62063\n",
      "Document Peginesatide - similarity: 0.62056\n",
      "Document Piperaquine - similarity: 0.62056\n",
      "Document Triclocarban - similarity: 0.62055\n",
      "Document Orilanolimab - similarity: 0.62051\n",
      "Document Cinepazide - similarity: 0.62050\n",
      "Document [(2R)-1-[(2S)-2-[[(2S,3S)-1-Chloro-6-(diaminomethylideneamino)-2-hydroxyhexan-3-yl]carbamoyl]pyrrolidin-1-yl]-1-oxo-3-phenylpropan-2-yl]azanium - similarity: 0.62049\n",
      "Document Influenza B virus B/Massachusetts/2/2012 BX-51B hemagglutinin antigen (propiolactone inactivated) - similarity: 0.62040\n",
      "Document Pork - similarity: 0.62031\n",
      "Document FR230513 - similarity: 0.62031\n",
      "Document Capric acid - similarity: 0.62027\n",
      "Document (E)-(4S,6S)-8-METHYL-6-((S)-3-METHYL-2-{(S)-2-[(5-METHYL-ISOXAZOLE-3-CARBONYL)-AMINO]-PROPIONYLAMINO}-BUTYRYLAMINO)-5-OXO-4-((R)-2-OXO-PYRROLIDIN-3-YLMETHYL)-NON-2-ENOIC ACID BENZYL ESTER - similarity: 0.62015\n",
      "Document Potassium alum - similarity: 0.62007\n",
      "Document Rabusertib - similarity: 0.62007\n",
      "Document N-acetylsulfanilyl chloride - similarity: 0.62007\n",
      "Document 3-FLUORO-N-[1-(4-FLUOROPHENYL)-3-(2-THIENYL)-1H-PYRAZOL-5-YL]BENZENESULFONAMIDE - similarity: 0.62006\n",
      "Document Ganciclovir - similarity: 0.61998\n",
      "Document Mycophenolic acid - similarity: 0.61996\n",
      "Document Thymopentin - similarity: 0.61990\n",
      "Document Vedolizumab - similarity: 0.61990\n",
      "Document Broxuridine - similarity: 0.61989\n",
      "Document Prostaglandin D2 - similarity: 0.61989\n",
      "Document N-(2-AMINOETHYL)-N~2~-{(1S)-1-[4'-(AMINOSULFONYL)BIPHENYL-4-YL]-2,2,2-TRIFLUOROETHYL}-L-LEUCINAMIDE - similarity: 0.61986\n",
      "Document Latanoprostene bunod - similarity: 0.61985\n",
      "Document keyhole limpet hemocyanin - similarity: 0.61975\n",
      "Document Tomato - similarity: 0.61973\n",
      "Document Laquinimod - similarity: 0.61969\n",
      "Document Rhodotorula rubra - similarity: 0.61968\n",
      "Document Trimethyldiphenylpropylamine - similarity: 0.61964\n",
      "Document Tetrathiomolybdate - similarity: 0.61951\n",
      "Document Artemisia dracunculus pollen - similarity: 0.61941\n",
      "Document Desmethylprodine - similarity: 0.61938\n",
      "Document 4-{[4-{[(1R,2R)-2-(dimethylamino)cyclopentyl]amino}-5-(trifluoromethyl)pyrimidin-2-yl]amino}-N-methylbenzenesulfonamide - similarity: 0.61936\n",
      "Document Magnesium malate - similarity: 0.61935\n",
      "Document Pipenzolate - similarity: 0.61932\n",
      "Document Gemcitabine elaidate - similarity: 0.61917\n",
      "Document IT-101 - similarity: 0.61917\n",
      "Document Brompheniramine - similarity: 0.61910\n",
      "Document GDC-0425 - similarity: 0.61906\n",
      "Document Etheno-Nadp - similarity: 0.61904\n",
      "Document Ilaprazole - similarity: 0.61898\n",
      "Document Serine - similarity: 0.61896\n",
      "Document Acer rubrum pollen - similarity: 0.61893\n",
      "Document 4-Methylidene-5-One - similarity: 0.61891\n",
      "Document ONT-093 - similarity: 0.61885\n",
      "Document (3AR,6R,6AS)-6-((S)-((S)-CYCLOHEX-2-ENYL)(HYDROXY)METHYL)-6A-METHYL-4-OXO-HEXAHYDRO-2H-FURO[3,2-C]PYRROLE-6-CARBALDEHYDE - similarity: 0.61885\n",
      "Document (5R)-5-{[(2-Amino-3H-purin-6-yl)oxy]methyl}-2-pyrrolidinone - similarity: 0.61884\n",
      "Document Tenalisib - similarity: 0.61875\n",
      "Document Agave sisalana fiber - similarity: 0.61871\n",
      "Document Grazoprevir - similarity: 0.61871\n",
      "Document Sotalol - similarity: 0.61867\n",
      "Document PF-00446687 - similarity: 0.61857\n",
      "Document Bempegaldesleukin - similarity: 0.61854\n",
      "Document Moricizine - similarity: 0.61853\n",
      "Document MK-0557 - similarity: 0.61852\n",
      "Document Rituximab - similarity: 0.61847\n",
      "Document Domiodol - similarity: 0.61843\n",
      "Document Onercept - similarity: 0.61842\n",
      "Document Pradefovir - similarity: 0.61839\n",
      "Document A-620223 - similarity: 0.61836\n",
      "Document N'-[(1E)-(3,5-dibromo-2,4-dihydroxyphenyl)methylidene]naphthalene-2-carbohydrazide - similarity: 0.61830\n",
      "Document Tarextumab - similarity: 0.61830\n",
      "Document Pindolol - similarity: 0.61825\n",
      "Document Ginkgo biloba - similarity: 0.61821\n",
      "Document 5-chloro-7-[(1-methylethyl)amino]pyrazolo[1,5-a]pyrimidine-3-carbonitrile - similarity: 0.61815\n",
      "Document Peramivir - similarity: 0.61813\n",
      "Document Bevacizumab - similarity: 0.61811\n",
      "Document Hypophosphite - similarity: 0.61809\n",
      "Document Col-118 - similarity: 0.61801\n",
      "Document Liraglutide - similarity: 0.61800\n",
      "Document Fulranumab - similarity: 0.61792\n",
      "Document Calcium threonate - similarity: 0.61786\n",
      "Document Channel catfish - similarity: 0.61774\n",
      "Document Mefloquine - similarity: 0.61773\n",
      "Document Salbutamol - similarity: 0.61772\n",
      "Document Rivaroxaban - similarity: 0.61771\n",
      "Document Methimazole - similarity: 0.61766\n",
      "Document ITI-214 - similarity: 0.61753\n",
      "Document Cobitolimod - similarity: 0.61751\n",
      "Document Cisatracurium - similarity: 0.61745\n",
      "Document (5E,14E)-11-oxoprosta-5,9,12,14-tetraen-1-oic acid - similarity: 0.61738\n",
      "Document Napabucasin - similarity: 0.61728\n",
      "Document (S)-Fluoxetine - similarity: 0.61718\n",
      "Document N-METHYL-N-[(1R)-1-METHYL-2-PHENYLETHYL]PROP-2-EN-1-AMINE - similarity: 0.61713\n",
      "Document Nicardipine - similarity: 0.61713\n",
      "Document 24-Homo-1,25-dihydroxyvitamin D3 - similarity: 0.61712\n",
      "Document (S)-famoxadone - similarity: 0.61709\n",
      "Document Onasemnogene abeparvovec - similarity: 0.61704\n",
      "Document Iomazenil - similarity: 0.61698\n",
      "Document Vemurafenib - similarity: 0.61697\n",
      "Document Baloxavir - similarity: 0.61696\n",
      "Document Tranylcypromine - similarity: 0.61696\n",
      "Document Prednisolone phosphate - similarity: 0.61683\n",
      "Document Quisinostat - similarity: 0.61679\n",
      "Document Fosinoprilat - similarity: 0.61673\n",
      "Document N-(4-CARBAMIMIDOYL-3-CHORO-PHENYL)-2-HYDROXY-3-IODO-5-METHYL-BENZAMIDE - similarity: 0.61672\n",
      "Document Cochliobolus sativus - similarity: 0.61671\n",
      "Document Bradykinin - similarity: 0.61653\n",
      "Document Mefenamic acid - similarity: 0.61650\n",
      "Document SP1049C - similarity: 0.61648\n",
      "Document Tonapofylline - similarity: 0.61645\n",
      "Document Icosapent ethyl - similarity: 0.61641\n",
      "Document Samalizumab - similarity: 0.61632\n",
      "Document 3-(2-aminoquinazolin-6-yl)-4-methyl-1-[3-(trifluoromethyl)phenyl]pyridin-2(1H)-one - similarity: 0.61622\n",
      "Document Zafirlukast - similarity: 0.61619\n",
      "Document BV2 - similarity: 0.61612\n",
      "Document 4-[(3AS,4R,7R,8AS,8BR)-2-(1,3-BENZODIOXOL-5-YLMETHYL)-7-HYDROXY-1,3-DIOXODECAHYDROPYRROLO[3,4-A]PYRROLIZIN-4-YL]BENZENECARBOXIMIDAMIDE - similarity: 0.61611\n",
      "Document Tirzepatide - similarity: 0.61605\n",
      "Document Artichoke - similarity: 0.61605\n",
      "Document SEP-363856 - similarity: 0.61602\n",
      "Document 3-(4-Phenylamino-Phenylamino)-2-(1h-Tetrazol-5-Yl)-Acrylonitrile - similarity: 0.61600\n",
      "Document Cenerimod - similarity: 0.61598\n",
      "Document Toreforant - similarity: 0.61596\n",
      "Document Fulacimstat - similarity: 0.61591\n",
      "Document Hexylcaine - similarity: 0.61589\n",
      "Document Matuzumab - similarity: 0.61583\n",
      "Document 2-(1,3-thiazol-4-yl)-1H-benzimidazole-5-sulfonamide - similarity: 0.61581\n",
      "Document (9Z,11E,13S)-13-hydroxyoctadeca-9,11-dienoic acid - similarity: 0.61578\n",
      "Document Sage oil - similarity: 0.61575\n",
      "Document 4,5-Dehydro-L-Iduronic Acid - similarity: 0.61574\n",
      "Document Briciclib - similarity: 0.61563\n",
      "Document Coumaphos - similarity: 0.61562\n",
      "Document Juglans nigra pollen - similarity: 0.61557\n",
      "Document Chamomile - similarity: 0.61557\n",
      "Document Nonoxynol-9 - similarity: 0.61555\n",
      "Document Tacalcitol - similarity: 0.61552\n",
      "Document SGI-1776 - similarity: 0.61551\n",
      "Document LTX-315 - similarity: 0.61547\n",
      "Document 6-(cyclohexylamino)-9-[2-(4-methylpiperazin-1-yl)-ethyl]-9H-purine-2-carbonitrile - similarity: 0.61546\n",
      "Document Zoledronic acid - similarity: 0.61540\n",
      "Document Poultry - similarity: 0.61540\n",
      "Document Efatutazone - similarity: 0.61537\n",
      "Document Fluoroacetic acid F-18 - similarity: 0.61531\n",
      "Document Stiripentol - similarity: 0.61522\n",
      "Document Bromperidol - similarity: 0.61520\n",
      "Document Acotiamide - similarity: 0.61518\n",
      "Document Rabbit - similarity: 0.61515\n",
      "Document Pipobroman - similarity: 0.61512\n",
      "Document Neridronic Acid - similarity: 0.61509\n",
      "Document Methohexital - similarity: 0.61503\n",
      "Document Selenious acid - similarity: 0.61502\n",
      "Document Pantothenoylaminoethenethiol - similarity: 0.61498\n",
      "Document Willow bark - similarity: 0.61497\n",
      "Document Aliskiren - similarity: 0.61489\n",
      "Document (4-BROMOPHENYL)[4-({(2E)-4-[CYCLOPROPYL(METHYL)AMINO]BUT-2-ENYL}OXY)PHENYL]METHANONE - similarity: 0.61480\n",
      "Document Lacidipine - similarity: 0.61471\n",
      "Document Eggplant - similarity: 0.61467\n",
      "Document Buclizine - similarity: 0.61452\n",
      "Document Colchiceine - similarity: 0.61448\n",
      "Document Turnera diffusa leaf - similarity: 0.61448\n",
      "Document Ivermectin - similarity: 0.61446\n",
      "Document 5-methyl-N-[4-(trifluoromethyl)phenyl][1,2,4]triazolo[1,5-a]pyrimidin-7-amine - similarity: 0.61430\n",
      "Document 3-Methyl-5-(7-{4-[(4R)-4-methyl-4,5-dihydro-1,3-oxazol-2-yl]phenoxy}heptyl)-1,2-oxazole - similarity: 0.61429\n",
      "Document Vanutide cridificar - similarity: 0.61425\n",
      "Document Propolis wax - similarity: 0.61408\n",
      "Document 2-(4-HYDROXY-PHENYL)BENZOFURAN-5-OL - similarity: 0.61408\n",
      "Document 3-methyl-N-(pyridin-4-ylmethyl)imidazo[1,2-a]pyrazin-8-amine - similarity: 0.61407\n",
      "Document Ceftaroline fosamil - similarity: 0.61406\n",
      "Document Radicicol - similarity: 0.61403\n",
      "Document Cambinol - similarity: 0.61402\n",
      "Document Nitrate - similarity: 0.61392\n",
      "Document Flurothyl - similarity: 0.61385\n",
      "Document Dexpramipexole - similarity: 0.61377\n",
      "Document Crypthecodinium cohnii DHA oil - similarity: 0.61372\n",
      "Document Oxandrolone - similarity: 0.61369\n",
      "Document Lime (citrus) - similarity: 0.61365\n",
      "Document Polydioxanone - similarity: 0.61365\n",
      "Document Elcatonin - similarity: 0.61352\n",
      "Document Amphiachyris dracunculoides pollen - similarity: 0.61351\n",
      "Document Pramocaine - similarity: 0.61350\n",
      "Document KD3010 - similarity: 0.61347\n",
      "Document Aviptadil - similarity: 0.61346\n",
      "Document Influenza A virus A/California/7/2009 X-179A (H1N1) hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.61345\n",
      "Document Paquinimod - similarity: 0.61340\n",
      "Document Romosozumab - similarity: 0.61332\n",
      "Document Cyclopenthiazide - similarity: 0.61332\n",
      "Document Dolutegravir - similarity: 0.61327\n",
      "Document Solamargine - similarity: 0.61315\n",
      "Document N-(3,5-dibromo-4-hydroxyphenyl)-4-hydroxy-3,5-dimethylbenzamide - similarity: 0.61314\n",
      "Document Pea - similarity: 0.61314\n",
      "Document S-Methyl Phosphocysteine - similarity: 0.61309\n",
      "Document Tanzisertib - similarity: 0.61307\n",
      "Document Isoniazid - similarity: 0.61307\n",
      "Document Alpha-1-proteinase inhibitor - similarity: 0.61293\n",
      "Document Phleum pratense pollen - similarity: 0.61290\n",
      "Document Tolperisone - similarity: 0.61290\n",
      "Document Perampanel - similarity: 0.61289\n",
      "Document 5-ethoxy-4-(1-methyl-7-oxo-3-propyl-6,7-dihydro-1H-pyrazolo[4,3-d]pyrimidin-5-yl)thiophene-2-sulfonamide - similarity: 0.61285\n",
      "Document Semapimod - similarity: 0.61275\n",
      "Document Tinoridine - similarity: 0.61270\n",
      "Document Biguanide - similarity: 0.61268\n",
      "Document ABT-341 - similarity: 0.61263\n",
      "Document Entospletinib - similarity: 0.61262\n",
      "Document ASP-4058 - similarity: 0.61258\n",
      "Document Dipyrromethane Cofactor - similarity: 0.61256\n",
      "Document 4,5-Dihydroxy-Tetrahydro-Pyran-2-Carboxylic Acid - similarity: 0.61255\n",
      "Document Fluticasone furoate - similarity: 0.61250\n",
      "Document Willardiine - similarity: 0.61246\n",
      "Document Diguanosine-5'-Triphosphate - similarity: 0.61245\n",
      "Document Agalsidase beta - similarity: 0.61244\n",
      "Document Alphameprodine - similarity: 0.61235\n",
      "Document (1R, 2S)-cis 1,2 dihydroxy-1,2-dihydronaphthalene - similarity: 0.61227\n",
      "Document JNJ-41443532 - similarity: 0.61226\n",
      "Document Amsacrine - similarity: 0.61217\n",
      "Document Eptinezumab - similarity: 0.61215\n",
      "Document Talaporfin - similarity: 0.61215\n",
      "Document CCT-018159 - similarity: 0.61205\n",
      "Document Picrotoxin - similarity: 0.61203\n",
      "Document Guanoxan - similarity: 0.61200\n",
      "Document Rolitetracycline - similarity: 0.61199\n",
      "Document Agrostis gigantea pollen - similarity: 0.61193\n",
      "Document Remogliflozin etabonate - similarity: 0.61189\n",
      "Document Tacrolimus - similarity: 0.61188\n",
      "Document Sulfametomidine - similarity: 0.61183\n",
      "Document Nitrazolam - similarity: 0.61179\n",
      "Document Okra - similarity: 0.61173\n",
      "Document Usnea barbata - similarity: 0.61170\n",
      "Document 9-HYDROXY-4-PHENYL-6H-PYRROLO[3,4-C]CARBAZOLE-1,3-DIONE - similarity: 0.61169\n",
      "Document Mercuric iodide - similarity: 0.61162\n",
      "Document Decamethoxine - similarity: 0.61158\n",
      "Document Snail, unspecified - similarity: 0.61157\n",
      "Document PF-04937319 - similarity: 0.61155\n",
      "Document Clorindione - similarity: 0.61148\n",
      "Document Chlormidazole - similarity: 0.61142\n",
      "Document Difluocortolone pivalate - similarity: 0.61141\n",
      "Document Clonazolam - similarity: 0.61141\n",
      "Document LY-295501 - similarity: 0.61139\n",
      "Document Trimethyllysine - similarity: 0.61136\n",
      "Document Picotamide - similarity: 0.61127\n",
      "Document Eurotium herbariorum - similarity: 0.61114\n",
      "Document Arabica coffee bean - similarity: 0.61112\n",
      "Document Hydralazine - similarity: 0.61099\n",
      "Document Pentoxifylline - similarity: 0.61098\n",
      "Document Betula alleghaniensis pollen - similarity: 0.61091\n",
      "Document 1,4-Dideoxy-5-Dehydro-O2-Sulfo-Glucuronic Acid - similarity: 0.61089\n",
      "Document Cefotetan - similarity: 0.61088\n",
      "Document BETA-METHYLLACTOSIDE - similarity: 0.61068\n",
      "Document Alogliptin - similarity: 0.61068\n",
      "Document Talsaclidine - similarity: 0.61067\n",
      "Document Dutogliptin - similarity: 0.61064\n",
      "Document Tributyrin - similarity: 0.61058\n",
      "Document Amantadine - similarity: 0.61055\n",
      "Document NOV-002 - similarity: 0.61055\n",
      "Document Adecatumumab - similarity: 0.61053\n",
      "Document Dexrabeprazole - similarity: 0.61051\n",
      "Document (1Z)-2-HYDROXY-3-OXOHEX-1-EN-1-YL DIHYDROGEN PHOSPHATE - similarity: 0.61049\n",
      "Document Sarilumab - similarity: 0.61046\n",
      "Document Pyritinol - similarity: 0.61045\n",
      "Document Vaccinia virus strain new york city board of health live antigen - similarity: 0.61041\n",
      "Document Nesvacumab - similarity: 0.61040\n",
      "Document Fremanezumab - similarity: 0.61035\n",
      "Document Flumequine - similarity: 0.61030\n",
      "Document RTA 744 - similarity: 0.61025\n",
      "Document Paratoulene phosphate - similarity: 0.61022\n",
      "Document N-[4-(5-fluoro-6-methylpyridin-2-yl)-5-quinoxalin-6-yl-1H-imidazol-2-yl]acetamide - similarity: 0.61019\n",
      "Document Magnesium peroxide - similarity: 0.61015\n",
      "Document Milrinone - similarity: 0.61015\n",
      "Document Patiromer - similarity: 0.61009\n",
      "Document rhMBL - similarity: 0.61008\n",
      "Document Drospirenone - similarity: 0.60995\n",
      "Document Quinaldic Acid - similarity: 0.60990\n",
      "Document Sulfadiazine - similarity: 0.60986\n",
      "Document Pradofloxacin - similarity: 0.60971\n",
      "Document Bombyx mori fiber - similarity: 0.60970\n",
      "Document Sarpogrelate - similarity: 0.60963\n",
      "Document 2-(3-BROMOPHENYL)-6-[(2-HYDROXYETHYL)AMINO]-1H-BENZO[DE]ISOQUINOLINE-1,3(2H)-DIONE - similarity: 0.60958\n",
      "Document Lactic acid - similarity: 0.60957\n",
      "Document Sutezolid - similarity: 0.60956\n",
      "Document Human C1-esterase inhibitor - similarity: 0.60953\n",
      "Document Raltegravir - similarity: 0.60951\n",
      "Document XEN2174 - similarity: 0.60950\n",
      "Document PRX-07034 - similarity: 0.60943\n",
      "Document Resatorvid - similarity: 0.60940\n",
      "Document Influenza B virus B/Maryland/15/2016 recombinant hemagglutinin antigen - similarity: 0.60934\n",
      "Document Zaleplon - similarity: 0.60915\n",
      "Document Latanoprost - similarity: 0.60915\n",
      "Document Daunorubicin - similarity: 0.60914\n",
      "Document Sulprostone - similarity: 0.60912\n",
      "Document Evening primrose oil - similarity: 0.60890\n",
      "Document OSI-930 - similarity: 0.60885\n",
      "Document Etonogestrel - similarity: 0.60883\n",
      "Document Isoleucine - similarity: 0.60876\n",
      "Document Etamiphylline - similarity: 0.60874\n",
      "Document LCP-AtorFen - similarity: 0.60874\n",
      "Document Cephalosporin C - similarity: 0.60866\n",
      "Document (8R,9Z,12Z)-8-hydroxy-6-oxooctadeca-9,12-dienoic acid - similarity: 0.60861\n",
      "Document Paclitaxel poliglumex - similarity: 0.60856\n",
      "Document Dalcetrapib - similarity: 0.60856\n",
      "Document 2-[4-(DIMETHYLAMINO)PHENYL]-6-HYDROXY-3-METHYL-1,3-BENZOTHIAZOL-3-IUM - similarity: 0.60855\n",
      "Document Relugolix - similarity: 0.60851\n",
      "Document Methoxsalen - similarity: 0.60850\n",
      "Document N-DODECANOYL-L-TYROSINE - similarity: 0.60828\n",
      "Document Human albumin microspheres - similarity: 0.60827\n",
      "Document Capra hircus hair - similarity: 0.60827\n",
      "Document Pinometostat - similarity: 0.60826\n",
      "Document Xaliproden - similarity: 0.60824\n",
      "Document Methylchloroisothiazolinone - similarity: 0.60813\n",
      "Document Basifungin - similarity: 0.60796\n",
      "Document Berberine - similarity: 0.60794\n",
      "Document Formoterol - similarity: 0.60794\n",
      "Document Pipazethate - similarity: 0.60789\n",
      "Document Mangafodipir - similarity: 0.60787\n",
      "Document JNJ-38877605 - similarity: 0.60787\n",
      "Document Ractopamine - similarity: 0.60787\n",
      "Document Lecozotan - similarity: 0.60785\n",
      "Document Lamotrigine - similarity: 0.60782\n",
      "Document Loratadine - similarity: 0.60781\n",
      "Document Flosequinan - similarity: 0.60771\n",
      "Document Palm oil - similarity: 0.60768\n",
      "Document Hydroxypropyl tetrahydropyrantriol - similarity: 0.60767\n",
      "Document MK-3207 - similarity: 0.60765\n",
      "Document Etofenamate - similarity: 0.60763\n",
      "Document Clobazam - similarity: 0.60741\n",
      "Document Edivoxetine - similarity: 0.60740\n",
      "Document Trelagliptin - similarity: 0.60734\n",
      "Document Capsicum annuum whole - similarity: 0.60712\n",
      "Document Polaprezinc - similarity: 0.60692\n",
      "Document Cefprozil - similarity: 0.60689\n",
      "Document Denibulin - similarity: 0.60688\n",
      "Document CPI-613 - similarity: 0.60681\n",
      "Document ORG-34517 - similarity: 0.60677\n",
      "Document Hydroxyurea - similarity: 0.60675\n",
      "Document Benzalkonium - similarity: 0.60660\n",
      "Document Protein S human - similarity: 0.60659\n",
      "Document Anidulafungin - similarity: 0.60651\n",
      "Document Procarbazine - similarity: 0.60648\n",
      "Document Silicon dioxide - similarity: 0.60632\n",
      "Document Efizonerimod alfa - similarity: 0.60632\n",
      "Document Buserelin - similarity: 0.60631\n",
      "Document Lobucavir - similarity: 0.60627\n",
      "Document Pinus thunbergii pollen - similarity: 0.60616\n",
      "Document Efalizumab - similarity: 0.60601\n",
      "Document Midodrine - similarity: 0.60600\n",
      "Document Artemether - similarity: 0.60591\n",
      "Document Mineral oil - similarity: 0.60582\n",
      "Document 2-(3,4-Dihydroxyphenyl)Acetic Acid - similarity: 0.60579\n",
      "Document Cilazaprilat - similarity: 0.60577\n",
      "Document Misoprostol - similarity: 0.60576\n",
      "Document Polidocanol - similarity: 0.60571\n",
      "Document Water - similarity: 0.60568\n",
      "Document Iron sucrose - similarity: 0.60566\n",
      "Document Anthrax vaccine - similarity: 0.60559\n",
      "Document Nickel cation - similarity: 0.60558\n",
      "Document Fipamezole - similarity: 0.60558\n",
      "Document BIA - similarity: 0.60555\n",
      "Document Foretinib - similarity: 0.60552\n",
      "Document HONH-BENZYLMALONYL-L-ALANYLGLYCINE-P-NITROANILIDE - similarity: 0.60551\n",
      "Document Phloretin - similarity: 0.60544\n",
      "Document Duloxetine - similarity: 0.60543\n",
      "Document Marizomib - similarity: 0.60529\n",
      "Document Tasimelteon - similarity: 0.60523\n",
      "Document Etamsylate - similarity: 0.60522\n",
      "Document Human thrombin - similarity: 0.60522\n",
      "Document Indium - similarity: 0.60520\n",
      "Document House dust - similarity: 0.60511\n",
      "Document Osimertinib - similarity: 0.60511\n",
      "Document (6S)-1-chloro-3-[(4-fluorobenzyl)oxy]-6-(pyrrolidin-1-ylcarbonyl)pyrrolo[1,2-a]pyrazin-4(6H)-one - similarity: 0.60509\n",
      "Document Obexelimab - similarity: 0.60495\n",
      "Document Leukotriene B4 - similarity: 0.60494\n",
      "Document Viloxazine - similarity: 0.60493\n",
      "Document Propiopromazine - similarity: 0.60489\n",
      "Document Ferrous ascorbate - similarity: 0.60481\n",
      "Document N-BENZYL-4-[4-(3-CHLOROPHENYL)-1H-PYRAZOL-3-YL]-1H-PYRROLE-2-CARBOXAMIDE - similarity: 0.60480\n",
      "Document Tixocortol - similarity: 0.60475\n",
      "Document Insulin degludec - similarity: 0.60466\n",
      "Document Benzophenone - similarity: 0.60465\n",
      "Document (R)-methylmalonyl-CoA - similarity: 0.60462\n",
      "Document Balixafortide - similarity: 0.60443\n",
      "Document (7R,8R)-8-(2,4,5-trifluorophenyl)-6,7,8,9-tetrahydroimidazo[1,2-a:4,5-c']dipyridin-7-amine - similarity: 0.60428\n",
      "Document Progabide - similarity: 0.60410\n",
      "Document Apadamtase alfa - similarity: 0.60409\n",
      "Document Taribavirin - similarity: 0.60403\n",
      "Document Orvepitant - similarity: 0.60399\n",
      "Document Selonsertib - similarity: 0.60396\n",
      "Document 2-Thioethenamine - similarity: 0.60391\n",
      "Document Temazepam - similarity: 0.60378\n",
      "Document Estetrol - similarity: 0.60377\n",
      "Document Glibornuride - similarity: 0.60376\n",
      "Document Iodine - similarity: 0.60373\n",
      "Document 2-({2-[(3R)-3-AMINOPIPERIDIN-1-YL]-4-OXOQUINAZOLIN-3(4H)-YL}METHYL)BENZONITRILE - similarity: 0.60373\n",
      "Document Undecanal - similarity: 0.60363\n",
      "Document Risperidone - similarity: 0.60361\n",
      "Document Sodium apolate - similarity: 0.60358\n",
      "Document Guanethidine - similarity: 0.60353\n",
      "Document Linoleic acid - similarity: 0.60351\n",
      "Document Horse chestnut - similarity: 0.60350\n",
      "Document 4-Amino-N-{4-[2-(2,6-Dimethyl-Phenoxy)-Acetylamino]-3-Hydroxy-1-Isobutyl-5-Phenyl-Pentyl}-Benzamide - similarity: 0.60349\n",
      "Document Ochromycinone - similarity: 0.60347\n",
      "Document Mecobalamin - similarity: 0.60345\n",
      "Document Alteplase - similarity: 0.60342\n",
      "Document N-[3-(1H-BENZIMIDAZOL-2-YL)-1H-PYRAZOL-4-YL]BENZAMIDE - similarity: 0.60335\n",
      "Document Nefazodone - similarity: 0.60323\n",
      "Document Carbimazole - similarity: 0.60313\n",
      "Document Peridinin - similarity: 0.60307\n",
      "Document Cabbage - similarity: 0.60297\n",
      "Document Estramustine - similarity: 0.60288\n",
      "Document Fospropofol - similarity: 0.60284\n",
      "Document AC-262536 - similarity: 0.60284\n",
      "Document Perch - similarity: 0.60280\n",
      "Document Urokinase - similarity: 0.60279\n",
      "Document Palbociclib - similarity: 0.60278\n",
      "Document Iva axillaris pollen - similarity: 0.60274\n",
      "Document Millet seed - similarity: 0.60267\n",
      "Document (2R)-1-[(5,6-DIPHENYL-7H-PYRROLO[2,3-D]PYRIMIDIN-4-YL)AMINO]PROPAN-2-OL - similarity: 0.60266\n",
      "Document Papaverine - similarity: 0.60252\n",
      "Document Choline alfoscerate - similarity: 0.60251\n",
      "Document Sumatriptan - similarity: 0.60248\n",
      "Document Capsicum - similarity: 0.60248\n",
      "Document 4-PIPERIDIN-4-YLBUTANAL - similarity: 0.60242\n",
      "Document 1-(6-CYANO-3-PYRIDYLCARBONYL)-5',8'-DIFLUOROSPIRO[PIPERIDINE-4,2'(1'H)-QUINAZOLINE]-4'-AMINE - similarity: 0.60237\n",
      "Document Methylphosphinic Acid - similarity: 0.60235\n",
      "Document Influenza A virus A/Uruguay/716/2007(H3N2) live (attenuated) antigen - similarity: 0.60225\n",
      "Document Velaglucerase alfa - similarity: 0.60219\n",
      "Document Lobaplatin - similarity: 0.60216\n",
      "Document 3-Fluoro-4-{[(2R)-2-hydroxy-2-(5,5,8,8-tetramethyl-5,6,7,8-tetrahydro-2-naphthalenyl)acetyl]amino}benzoic acid - similarity: 0.60215\n",
      "Document Rusalatide acetate - similarity: 0.60213\n",
      "Document 5-beta-D-ribofuranosylnicotinamide adenine dinucleotide - similarity: 0.60201\n",
      "Document [(4R)-4-(3-HYDROXYPHENYL)-1,6-DIMETHYL-2-THIOXO-1,2,3,4-TETRAHYDROPYRIMIDIN-5-YL](PHENYL)METHANONE - similarity: 0.60201\n",
      "Document Cladribine - similarity: 0.60199\n",
      "Document Uridine diphosphate glucose - similarity: 0.60195\n",
      "Document Tibolone - similarity: 0.60174\n",
      "Document Hyaluronic acid - similarity: 0.60173\n",
      "Document Etretinate - similarity: 0.60170\n",
      "Document Belatacept - similarity: 0.60167\n",
      "Document Butenafine - similarity: 0.60165\n",
      "Document 2-Aminoprop-2-Enamide - similarity: 0.60163\n",
      "Document Linagliptin - similarity: 0.60148\n",
      "Document SCH-486757 - similarity: 0.60146\n",
      "Document Azathioprine - similarity: 0.60135\n",
      "Document MK-5108 - similarity: 0.60131\n",
      "Document Condoliase - similarity: 0.60130\n",
      "Document Escitalopram - similarity: 0.60120\n",
      "Document Doramapimod - similarity: 0.60119\n",
      "Document 3-phospho-D-glyceric acid - similarity: 0.60118\n",
      "Document Biricodar - similarity: 0.60115\n",
      "Document Coelenteramide - similarity: 0.60109\n",
      "Document Fusarium compactum - similarity: 0.60103\n",
      "Document Vandetanib - similarity: 0.60096\n",
      "Document Febantel - similarity: 0.60094\n",
      "Document Spectinomycin - similarity: 0.60084\n",
      "Document Becaplermin - similarity: 0.60082\n",
      "Document Brimonidine - similarity: 0.60079\n",
      "Document Varespladib - similarity: 0.60077\n",
      "Document Tefinostat - similarity: 0.60077\n",
      "Document Abaloparatide - similarity: 0.60073\n",
      "Document 6-CHLORO-3-(3-METHYLISOXAZOL-5-YL)-4-PHENYLQUINOLIN-2(1H)-ONE - similarity: 0.60055\n",
      "Document Ubidecarenone - similarity: 0.60054\n",
      "Document Eltrombopag - similarity: 0.60052\n",
      "Document Streptolydigin - similarity: 0.60037\n",
      "Document Etanercept - similarity: 0.60035\n",
      "Document Binetrakin - similarity: 0.60025\n",
      "Document Indium In-111 pentetreotide - similarity: 0.60016\n",
      "Document Thiostrepton - similarity: 0.60015\n",
      "Document (2S,3S)-3-FORMYL-2-({[(4-METHYLPHENYL)SULFONYL]AMINO}METHYL)PENTANOIC ACID - similarity: 0.60013\n",
      "Document Tyrosine - similarity: 0.60002\n",
      "Document Broccoli - similarity: 0.59968\n",
      "Document Licogliflozin - similarity: 0.59961\n",
      "Document Phenformin - similarity: 0.59961\n",
      "Document Piribedil - similarity: 0.59960\n",
      "Document Montmorillonite - similarity: 0.59958\n",
      "Document PD-0299685 - similarity: 0.59954\n",
      "Document Levamisole - similarity: 0.59949\n",
      "Document Tanaproget - similarity: 0.59948\n",
      "Document Sucrose - similarity: 0.59948\n",
      "Document 5-[1-(Acetylamino)-3-Methylbutyl]-2,5-Anhydro-3,4-Dideoxy-4-(Methoxycarbonyl)Pentonic Acid - similarity: 0.59945\n",
      "Document Perhexiline - similarity: 0.59943\n",
      "Document Darexaban - similarity: 0.59934\n",
      "Document Tosylchloramide - similarity: 0.59930\n",
      "Document Pimecrolimus - similarity: 0.59929\n",
      "Document Azoximer bromide - similarity: 0.59917\n",
      "Document Aloe vera flower - similarity: 0.59914\n",
      "Document Benazeprilat - similarity: 0.59913\n",
      "Document Influenza A virus A/New caledonia/71/2014 (H3N2) live (attenuated) antigen - similarity: 0.59912\n",
      "Document Benzethonium - similarity: 0.59909\n",
      "Document Glyburide - similarity: 0.59905\n",
      "Document Levomethadone - similarity: 0.59888\n",
      "Document CCR5 mAb - similarity: 0.59883\n",
      "Document Valine - similarity: 0.59882\n",
      "Document Influenza A virus A/Michigan/45/2015 (H1N1) recombinant hemagglutinin antigen - similarity: 0.59875\n",
      "Document Davalintide - similarity: 0.59873\n",
      "Document Faropenem - similarity: 0.59872\n",
      "Document Galcanezumab - similarity: 0.59865\n",
      "Document Catumaxomab - similarity: 0.59864\n",
      "Document Landiolol - similarity: 0.59861\n",
      "Document Galeterone - similarity: 0.59856\n",
      "Document Betamethasone phosphate - similarity: 0.59854\n",
      "Document Rotigaptide - similarity: 0.59853\n",
      "Document Colesevelam - similarity: 0.59842\n",
      "Document Spiramycin - similarity: 0.59839\n",
      "Document Homosalate - similarity: 0.59811\n",
      "Document English sole - similarity: 0.59809\n",
      "Document Rubidium - similarity: 0.59806\n",
      "Document Carindacillin - similarity: 0.59798\n",
      "Document Cytarabine - similarity: 0.59798\n",
      "Document Gadodiamide - similarity: 0.59789\n",
      "Document Butylscopolamine - similarity: 0.59782\n",
      "Document N-[(1R)-1-(4-Bromophenyl)ethyl]-5-fluoro-2-hydroxybenzamide - similarity: 0.59777\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Brazil nut - similarity: 0.59776\n",
      "Document Doxercalciferol - similarity: 0.59775\n",
      "Document Aminophenazone - similarity: 0.59771\n",
      "Document Lomerizine - similarity: 0.59771\n",
      "Document Gabapentin - similarity: 0.59770\n",
      "Document TTP889 - similarity: 0.59764\n",
      "Document Pentastarch - similarity: 0.59763\n",
      "Document Luspatercept - similarity: 0.59759\n",
      "Document Quetiapine - similarity: 0.59757\n",
      "Document Mavacamten - similarity: 0.59753\n",
      "Document Morus rubra pollen - similarity: 0.59747\n",
      "Document Ampreloxetine - similarity: 0.59747\n",
      "Document Proguanil - similarity: 0.59743\n",
      "Document Ezetimibe - similarity: 0.59737\n",
      "Document Calcium oxide - similarity: 0.59734\n",
      "Document 2-ANILINO-6-CYCLOHEXYLMETHOXYPURINE - similarity: 0.59729\n",
      "Document Diazepinomicin - similarity: 0.59726\n",
      "Document Menthyl salicylate - similarity: 0.59723\n",
      "Document Lornoxicam - similarity: 0.59720\n",
      "Document LX1031 - similarity: 0.59720\n",
      "Document Cinitapride - similarity: 0.59717\n",
      "Document Lanosterol - similarity: 0.59716\n",
      "Document Mitoxantrone - similarity: 0.59713\n",
      "Document ORG-25935 - similarity: 0.59713\n",
      "Document p-Nitrophenylacetic acid - similarity: 0.59706\n",
      "Document Cod liver oil - similarity: 0.59704\n",
      "Document Dexmecamylamine - similarity: 0.59702\n",
      "Document Urelumab - similarity: 0.59698\n",
      "Document E-2012 - similarity: 0.59696\n",
      "Document Melilotus officinalis pollen - similarity: 0.59679\n",
      "Document Irbesartan - similarity: 0.59676\n",
      "Document 6-(CYCLOHEXYLMETHOXY)-8-ISOPROPYL-9H-PURIN-2-AMINE - similarity: 0.59675\n",
      "Document Tiropramide - similarity: 0.59672\n",
      "Document Bisoprolol - similarity: 0.59666\n",
      "Document N-cyclopropyl-4-pyrazolo[1,5-b]pyridazin-3-ylpyrimidin-2-amine - similarity: 0.59662\n",
      "Document TRW3-(2-AMINO-3-HYDROXY-PROPYL)-6-(N'-CYCLOHEXYL-HYDRAZINO)OCTAHYDRO-INDOL-7-OL - similarity: 0.59661\n",
      "Document Acetyldigoxin - similarity: 0.59649\n",
      "Document Tetra(Imidazole)Diaquacopper (I) - similarity: 0.59645\n",
      "Document Northern bluefin tuna - similarity: 0.59641\n",
      "Document (2S)-2-Methyl-3-phenylpropanoic acid - similarity: 0.59632\n",
      "Document N-[(6-butoxynaphthalen-2-yl)sulfonyl]-L-glutamic acid - similarity: 0.59622\n",
      "Document Phenobarbital - similarity: 0.59620\n",
      "Document Puromycin - similarity: 0.59620\n",
      "Document Thallium Tl-201 - similarity: 0.59618\n",
      "Document Thymalfasin - similarity: 0.59617\n",
      "Document Propantheline - similarity: 0.59607\n",
      "Document Indole-3-Glycerol Phosphate - similarity: 0.59595\n",
      "Document Guaifenesin - similarity: 0.59592\n",
      "Document Isopropyl alcohol - similarity: 0.59579\n",
      "Document Zidovudine diphosphate - similarity: 0.59579\n",
      "Document Calcium undecylenate - similarity: 0.59577\n",
      "Document 3-(4-Fluorophenyl)-2-(6-Methylpyridin-2-Yl)-5,6-Dihydro-4h-Pyrrolo[1,2-B]Pyrazole - similarity: 0.59576\n",
      "Document Sultamicillin - similarity: 0.59575\n",
      "Document Darusentan - similarity: 0.59575\n",
      "Document Gadoteridol - similarity: 0.59570\n",
      "Document Ruxolitinib - similarity: 0.59566\n",
      "Document Phenylpropanolamine - similarity: 0.59558\n",
      "Document Ferric sodium citrate - similarity: 0.59557\n",
      "Document lambda-bis(2,2'-bipyridine)imidazole osmium (II) - similarity: 0.59553\n",
      "Document Pyrazinamide - similarity: 0.59552\n",
      "Document MK-0249 - similarity: 0.59551\n",
      "Document Purvalanol A - similarity: 0.59550\n",
      "Document Bromfenac - similarity: 0.59543\n",
      "Document Solidago canadensis pollen - similarity: 0.59542\n",
      "Document Taurocholic acid - similarity: 0.59538\n",
      "Document 4-[4-AMINO-6-(2,6-DICHLORO-PHENOXY)-[1,3,5]TRIAZIN-2-YLAMINO]-BENZONITRILE - similarity: 0.59537\n",
      "Document Inula helenium root - similarity: 0.59525\n",
      "Document Lentinan - similarity: 0.59520\n",
      "Document (2S) N-acetyl-L-alanyl-αL-phenylalanyl-chloroethylketone - similarity: 0.59518\n",
      "Document Patulin - similarity: 0.59509\n",
      "Document Ganirelix - similarity: 0.59507\n",
      "Document Ammonium lactate - similarity: 0.59504\n",
      "Document Rabeprazole - similarity: 0.59504\n",
      "Document Perfosfamide - similarity: 0.59500\n",
      "Document 1-{2-[2-(2-Methoxyethoxy)Ethoxy]Ethoxy}-4-(1,1,3,3-Tetramethylbutyl)Benzene - similarity: 0.59498\n",
      "Document Ivosidenib - similarity: 0.59495\n",
      "Document Cobamamide - similarity: 0.59494\n",
      "Document 2-(6-methylpyridin-2-yl)-N-pyridin-4-ylquinazolin-4-amine - similarity: 0.59486\n",
      "Document Benzyl benzoate - similarity: 0.59485\n",
      "Document Ramosetron - similarity: 0.59485\n",
      "Document Medroxyprogesterone acetate - similarity: 0.59484\n",
      "Document Cyprenorphine - similarity: 0.59475\n",
      "Document Talimogene laherparepvec - similarity: 0.59475\n",
      "Document Tariquidar - similarity: 0.59469\n",
      "Document Silver oxide - similarity: 0.59451\n",
      "Document Pyridostigmine - similarity: 0.59450\n",
      "Document Ciglitazone - similarity: 0.59449\n",
      "Document 4-{5-[(Z)-(2-IMINO-4-OXO-1,3-THIAZOLIDIN-5-YLIDENE)METHYL]-2-FURYL}-N-METHYLBENZENESULFONAMIDE - similarity: 0.59445\n",
      "Document Oxidized cellulose - similarity: 0.59444\n",
      "Document Basic Fibroblast Growth Factor - similarity: 0.59444\n",
      "Document Oseltamivir - similarity: 0.59439\n",
      "Document 3-CYCLOHEXYL-1-(2-MORPHOLIN-4-YL-2-OXOETHYL)-2-PHENYL-1H-INDOLE-6-CARBOXYLIC ACID - similarity: 0.59436\n",
      "Document Niperotidine - similarity: 0.59435\n",
      "Document MKC-1 - similarity: 0.59432\n",
      "Document Nimotuzumab - similarity: 0.59425\n",
      "Document Trifarotene - similarity: 0.59408\n",
      "Document Cilostazol - similarity: 0.59399\n",
      "Document Tubercidin - similarity: 0.59398\n",
      "Document Tromantadine - similarity: 0.59386\n",
      "Document Tiludronic acid - similarity: 0.59376\n",
      "Document {[5-(5-nitro-2-furyl)-1,3,4-oxadiazol-2-yl]thio}acetic acid - similarity: 0.59375\n",
      "Document Piperazine - similarity: 0.59371\n",
      "Document Sparsentan - similarity: 0.59366\n",
      "Document Lovastatin - similarity: 0.59363\n",
      "Document Nelarabine - similarity: 0.59358\n",
      "Document Obidoxime - similarity: 0.59356\n",
      "Document 3-Nitrophenylboronic Acid - similarity: 0.59355\n",
      "Document TU-100 - similarity: 0.59352\n",
      "Document Potassium perchlorate - similarity: 0.59343\n",
      "Document Hydrogenated soybean lecithin - similarity: 0.59335\n",
      "Document 2-{4-[(4-imidazo[1,2-a]pyridin-3-ylpyrimidin-2-yl)amino]piperidin-1-yl}-N-methylacetamide - similarity: 0.59330\n",
      "Document Tetracosactide - similarity: 0.59327\n",
      "Document 7,8-Dihydroneopterin - similarity: 0.59327\n",
      "Document Pork Collagen - similarity: 0.59309\n",
      "Document Bendamustine - similarity: 0.59308\n",
      "Document Thrombin - similarity: 0.59304\n",
      "Document Buparlisib - similarity: 0.59297\n",
      "Document Metyrosine - similarity: 0.59297\n",
      "Document (1S,4R,7AR)-4-BUTOXY-1-[(1R)-1-FORMYLPROPYL]-2,4,5,6,7,7A-HEXAHYDRO-1H-ISOINDOLE-3-CARBOXYLIC ACID - similarity: 0.59295\n",
      "Document Pipotiazine - similarity: 0.59294\n",
      "Document N7-BUTYL-N2-(5-CHLORO-2-METHYLPHENYL)-5-METHYL[1,2,4]TRIAZOLO[1,5-A]PYRIMIDINE-2,7-DIAMINE - similarity: 0.59288\n",
      "Document Remetinostat - similarity: 0.59275\n",
      "Document PF-04995274 - similarity: 0.59267\n",
      "Document Dicyclomine - similarity: 0.59266\n",
      "Document Icosabutate - similarity: 0.59263\n",
      "Document 4-[3-Hydroxyanilino]-6,7-Dimethoxyquinazoline - similarity: 0.59257\n",
      "Document Saredutant - similarity: 0.59256\n",
      "Document Bendazac - similarity: 0.59255\n",
      "Document Drotaverine - similarity: 0.59247\n",
      "Document V1003 - similarity: 0.59241\n",
      "Document Prasterone - similarity: 0.59239\n",
      "Document Povidone-iodine - similarity: 0.59236\n",
      "Document Natalizumab - similarity: 0.59235\n",
      "Document Udenafil - similarity: 0.59225\n",
      "Document Oryctolagus cuniculus skin - similarity: 0.59223\n",
      "Document Chaetomium globosum - similarity: 0.59213\n",
      "Document Trichosporon cutaneum - similarity: 0.59206\n",
      "Document Vinblastine - similarity: 0.59189\n",
      "Document 2',3'-O-{4-[Hydroxy(oxido)-λ5-azanylidene]-2,6-dinitro-2,5-cyclohexadiene-1,1-diyl}adenosine 5'-(tetrahydrogen triphosphate) - similarity: 0.59188\n",
      "Document Acenocoumarol - similarity: 0.59186\n",
      "Document CODA-001 - similarity: 0.59184\n",
      "Document Lasofoxifene - similarity: 0.59184\n",
      "Document Dexibuprofen - similarity: 0.59178\n",
      "Document Cysteine - similarity: 0.59176\n",
      "Document Tegaserod - similarity: 0.59174\n",
      "Document Fenpiprane - similarity: 0.59161\n",
      "Document Iproniazid - similarity: 0.59159\n",
      "Document 5-CHLORO-N-(2-(4-(2-OXOPYRIDIN-1(2H)-YL)BENZAMIDO)ETHYL)THIOPHENE-2-CARBOXAMIDE - similarity: 0.59157\n",
      "Document Luprostiol - similarity: 0.59155\n",
      "Document Trichophyton verrucosum - similarity: 0.59154\n",
      "Document AC-100 - similarity: 0.59154\n",
      "Document Phentermine - similarity: 0.59151\n",
      "Document 4'-[(1R)-1-amino-2-(2,5-difluorophenyl)ethyl]biphenyl-3-carboxamide - similarity: 0.59151\n",
      "Document Propiolactone - similarity: 0.59146\n",
      "Document Pinus elliottii pollen - similarity: 0.59144\n",
      "Document LB-100 - similarity: 0.59139\n",
      "Document Camponotus pennsylvanicus - similarity: 0.59138\n",
      "Document Mazindol - similarity: 0.59133\n",
      "Document Zolmitriptan - similarity: 0.59128\n",
      "Document Eszopiclone - similarity: 0.59123\n",
      "Document Red king crab - similarity: 0.59121\n",
      "Document Doxycycline - similarity: 0.59117\n",
      "Document Acetylcysteine amide - similarity: 0.59092\n",
      "Document Sunitinib - similarity: 0.59088\n",
      "Document Vanucizumab - similarity: 0.59087\n",
      "Document 4-(2-AMINOETHOXY)-3,5-DICHLORO-N-[3-(1-METHYLETHOXY)PHENYL]BENZAMIDE - similarity: 0.59086\n",
      "Document Faricimab - similarity: 0.59082\n",
      "Document Candesartan cilexetil - similarity: 0.59079\n",
      "Document Green tea leaf - similarity: 0.59077\n",
      "Document 5-iodotubercidin - similarity: 0.59072\n",
      "Document Ridaforolimus - similarity: 0.59067\n",
      "Document Dihydrogenphosphate - similarity: 0.59066\n",
      "Document 2'-HYDROXY-1,1'-BIPHENYL-2-SULFINIC ACID - similarity: 0.59065\n",
      "Document 2-Nonyl-4-quinolinol 1-oxide - similarity: 0.59064\n",
      "Document Normethadone - similarity: 0.59054\n",
      "Document Soneclosan - similarity: 0.59053\n",
      "Document Carbamazepine - similarity: 0.59051\n",
      "Document (2S)-2-Ammonio-3-[5-(2-methyl-2-propanyl)-3-oxido-1,2-oxazol-4-yl]propanoate - similarity: 0.59046\n",
      "Document PU-H71 - similarity: 0.59033\n",
      "Document Calcium alginate - similarity: 0.59022\n",
      "Document Sus scrofa hair - similarity: 0.59004\n",
      "Document (2E,4R,5S)-2,3,4,5-TETRAHYDROXY-6-(PALMITOYLOXY)HEX-2-ENOIC ACID - similarity: 0.58990\n",
      "Document Pumpkin - similarity: 0.58983\n",
      "Document Rimonabant - similarity: 0.58983\n",
      "Document Tiplimotide - similarity: 0.58980\n",
      "Document Perflenapent - similarity: 0.58971\n",
      "Document (5R)-N,N-DIETHYL-5-METHYL-2-[(THIOPHEN-2-YLCARBONYL)AMINO]-4,5,6,7-TETRAHYDRO-1-BENZOTHIOPHENE-3-CARBOXAMIDE - similarity: 0.58967\n",
      "Document CX157 - similarity: 0.58950\n",
      "Document (2S)-2-AMINO-4-(METHYLSULFANYL)-1-(1,3-THIAZOL-2-YL)BUTANE-1,1-DIOL - similarity: 0.58947\n",
      "Document Permethrin - similarity: 0.58945\n",
      "Document Pancreatic Polypeptide - similarity: 0.58945\n",
      "Document TERT-BUTYL 2-CYANO-2-METHYLHYDRAZINECARBOXYLATE - similarity: 0.58936\n",
      "Document Cycloguanil - similarity: 0.58933\n",
      "Document 19-norandrostenedione - similarity: 0.58915\n",
      "Document Butyl chloride - similarity: 0.58906\n",
      "Document Gallium citrate Ga-67 - similarity: 0.58905\n",
      "Document Padimate A - similarity: 0.58896\n",
      "Document Umeclidinium - similarity: 0.58886\n",
      "Document Levorphanol - similarity: 0.58880\n",
      "Document Dezocine - similarity: 0.58880\n",
      "Document (3R)-3-ethyl-N-[(4-methylphenyl)sulfonyl]-L-aspartic acid - similarity: 0.58871\n",
      "Document Bassia hyssopifolia pollen - similarity: 0.58869\n",
      "Document Aceclofenac - similarity: 0.58869\n",
      "Document 9-(3-IODOBENZYLAMINO)-1,2,3,4-TETRAHYDROACRIDINE - similarity: 0.58866\n",
      "Document (1S,2R,3S,4R,5R)-2,3,4-trihydroxy-N-octyl-6-oxa-8-azabicyclo[3.2.1]octane-8-carbothioamide - similarity: 0.58866\n",
      "Document Mycophenolate mofetil - similarity: 0.58856\n",
      "Document Ipidacrine - similarity: 0.58850\n",
      "Document (2R)-2-{[(4-FLUORO-3-METHYLPHENYL)SULFONYL]AMINO}-N-HYDROXY-2-TETRAHYDRO-2H-PYRAN-4-YLACETAMIDE - similarity: 0.58845\n",
      "Document Piracetam - similarity: 0.58836\n",
      "Document Hymecromone - similarity: 0.58833\n",
      "Document Streptokinase - similarity: 0.58833\n",
      "Document Ethylaminobenzylmethylcarbonyl Group - similarity: 0.58826\n",
      "Document Casuarina equisetifolia pollen - similarity: 0.58825\n",
      "Document Calcium gluconate - similarity: 0.58819\n",
      "Document Tramadol - similarity: 0.58814\n",
      "Document Thiamine(1+) monophosphate - similarity: 0.58813\n",
      "Document 3-((3-bromo-5-o-tolylpyrazolo[1,5-a]pyrimidin-7-ylamino)methyl)pyridine 1-oxide - similarity: 0.58812\n",
      "Document Ambrisentan - similarity: 0.58809\n",
      "Document Miglustat - similarity: 0.58794\n",
      "Document N-[3-[[4-[(5-CHLORO-1,3-BENZODIOXOL-4-YL)AMINO]PYRIMIDIN-2-YL]AMINO]PHENYL]METHANESULFONAMIDE - similarity: 0.58793\n",
      "Document Miransertib - similarity: 0.58792\n",
      "Document Methanesulfonyl Fluoride - similarity: 0.58787\n",
      "Document Sulfur hexafluoride - similarity: 0.58779\n",
      "Document Cangrelor - similarity: 0.58768\n",
      "Document IRL-1620 - similarity: 0.58756\n",
      "Document 1-[(2R)-2-aminobutanoyl]-N-(3-chlorobenzyl)-L-prolinamide - similarity: 0.58754\n",
      "Document Fluciclovine (18F) - similarity: 0.58752\n",
      "Document Bimatoprost - similarity: 0.58751\n",
      "Document Oxetorone - similarity: 0.58748\n",
      "Document Mebeverine - similarity: 0.58748\n",
      "Document Pegaspargase - similarity: 0.58741\n",
      "Document Pentagastrin - similarity: 0.58740\n",
      "Document Prednisolone acetate - similarity: 0.58738\n",
      "Document Enbucrilate - similarity: 0.58737\n",
      "Document Trehalose - similarity: 0.58734\n",
      "Document Tegafur - similarity: 0.58725\n",
      "Document Dirithromycin - similarity: 0.58722\n",
      "Document Dalantercept - similarity: 0.58721\n",
      "Document Thiethylperazine - similarity: 0.58720\n",
      "Document Lawsonia inermis leaf - similarity: 0.58717\n",
      "Document Adenosine-5'-ditungstate - similarity: 0.58714\n",
      "Document 4-(2-Amino-Ethoxy)-2-[(3-Hydroxy-2-Methyl-5-Phosphonooxymethyl-Pyridin-4-Ylmethyl)-Amino]-but-3-Enoic Acid - similarity: 0.58713\n",
      "Document Naloxegol - similarity: 0.58709\n",
      "Document Lithium hydroxide - similarity: 0.58708\n",
      "Document Picosulfuric acid - similarity: 0.58705\n",
      "Document Sulfathiourea - similarity: 0.58697\n",
      "Document 3-[(4-fluorophenyl)sulfanyl]-N-(4-methyl-1,3-thiazol-2-yl)-6-[(4-methyl-4H-1,2,4-triazol-3-yl)sulfanyl]pyridine-2-carboxamide - similarity: 0.58697\n",
      "Document Zanamivir - similarity: 0.58695\n",
      "Document Cannabinor - similarity: 0.58692\n",
      "Document Calcitonin eel - similarity: 0.58687\n",
      "Document Gemcitabine - similarity: 0.58686\n",
      "Document Pumice - similarity: 0.58678\n",
      "Document 3-Fluoro-L-tyrosine - similarity: 0.58668\n",
      "Document Bapineuzumab - similarity: 0.58656\n",
      "Document Dihydroquinidine barbiturate - similarity: 0.58656\n",
      "Document BPI-3016 - similarity: 0.58640\n",
      "Document Allosamizoline - similarity: 0.58636\n",
      "Document Danusertib - similarity: 0.58635\n",
      "Document Oxiracetam - similarity: 0.58634\n",
      "Document Uridine-5'-diphosphate-2-deoxy-2-fluorogalactose - similarity: 0.58627\n",
      "Document Trifluoro-thiamin phosphate - similarity: 0.58626\n",
      "Document N'-(5-chloro-1,3-benzodioxol-4-yl)-N-(3-methylsulfonylphenyl)pyrimidine-2,4-diamine - similarity: 0.58620\n",
      "Document 5-Bromo-N[2-(Dimethylamino)Ethyl]-9-Aminoacridine-4-Carboxamide - similarity: 0.58614\n",
      "Document S-Selanyl Cysteine - similarity: 0.58604\n",
      "Document Tadalafil - similarity: 0.58604\n",
      "Document L-gamma-glutamyl-S-(naphthalen-1-ylmethyl)-L-cysteinylglycine - similarity: 0.58599\n",
      "Document Benzimidazole - similarity: 0.58596\n",
      "Document Astemizole - similarity: 0.58588\n",
      "Document Ilepatril - similarity: 0.58584\n",
      "Document Ethoxzolamide - similarity: 0.58583\n",
      "Document Trimethobenzamide - similarity: 0.58562\n",
      "Document Glucose-6-Phosphate - similarity: 0.58560\n",
      "Document Captopril - similarity: 0.58552\n",
      "Document Rotavirus vaccine - similarity: 0.58546\n",
      "Document (1R,2R)-N-(2-Aminoethyl)-2-{[(4-methoxyphenyl)sulfonyl]methyl}cyclohexanecarboxamide - similarity: 0.58540\n",
      "Document Dengue tetravalent vaccine, live - similarity: 0.58533\n",
      "Document Phenothiazine - similarity: 0.58530\n",
      "Document Paricalcitol - similarity: 0.58529\n",
      "Document Alglucerase - similarity: 0.58528\n",
      "Document Finafloxacin - similarity: 0.58528\n",
      "Document Nitrazepam - similarity: 0.58523\n",
      "Document Dapoxetine - similarity: 0.58519\n",
      "Document TT-232 - similarity: 0.58516\n",
      "Document Etofamide - similarity: 0.58510\n",
      "Document Sulfamethazine - similarity: 0.58508\n",
      "Document 4-bromo-N'-[(1E)-(3,5-dibromo-2,4-dihydroxyphenyl)methylidene]benzohydrazide - similarity: 0.58502\n",
      "Document Perfluoro tert-butylcyclohexane - similarity: 0.58502\n",
      "Document Efinaconazole - similarity: 0.58502\n",
      "Document Lafutidine - similarity: 0.58494\n",
      "Document Loxapine - similarity: 0.58491\n",
      "Document Bicine - similarity: 0.58486\n",
      "Document Zinc dibutyldithiocarbamate - similarity: 0.58486\n",
      "Document Vinylbital - similarity: 0.58486\n",
      "Document Cupressus arizonica pollen - similarity: 0.58481\n",
      "Document Basil - similarity: 0.58480\n",
      "Document 6-ethyl-5-[(2S)-1-(3-methoxypropyl)-2-phenyl-1,2,3,4-tetrahydroquinolin-7-yl]pyrimidine-2,4-diamine - similarity: 0.58476\n",
      "Document Influenza B virus B/Washington/02/2019 antigen (formaldehyde inactivated) - similarity: 0.58468\n",
      "Document Caffeine - similarity: 0.58468\n",
      "Document Alfaxalone - similarity: 0.58467\n",
      "Document Tebentafusp - similarity: 0.58461\n",
      "Document S-(2-{[N-(2-HYDROXY-4-{[HYDROXY(OXIDO)PHOSPHINO]OXY}-3,3-DIMETHYLBUTANOYL)-BETA-ALANYL]AMINO}ETHYL) HEXANETHIOATE - similarity: 0.58447\n",
      "Document Meprednisone - similarity: 0.58445\n",
      "Document Turofexorate isopropyl - similarity: 0.58445\n",
      "Document Technetium Tc-99m sulfur colloid - similarity: 0.58445\n",
      "Document Colestilan chloride - similarity: 0.58443\n",
      "Document AZX-100 - similarity: 0.58442\n",
      "Document N-(3-carboxypropanoyl)-L-norvaline - similarity: 0.58427\n",
      "Document Nifurpirinol - similarity: 0.58415\n",
      "Document Pancuronium - similarity: 0.58410\n",
      "Document Bos taurus hair - similarity: 0.58409\n",
      "Document 2,4-DINITROPHENYL 2-DEOXY-2-FLUORO-BETA-D-MANNOPYRANOSIDE - similarity: 0.58408\n",
      "Document Polistes metricus venom protein - similarity: 0.58403\n",
      "Document AMG-222 - similarity: 0.58392\n",
      "Document Iodine povacrylex - similarity: 0.58386\n",
      "Document Amiloxate - similarity: 0.58383\n",
      "Document Intepirdine - similarity: 0.58380\n",
      "Document Moxetumomab Pasudotox - similarity: 0.58373\n",
      "Document Chlormadinone - similarity: 0.58370\n",
      "Document Squalene - similarity: 0.58369\n",
      "Document N-[2-Chloro-5-(trifluoromethyl)phenyl]imidodicarbonimidic diamide - similarity: 0.58367\n",
      "Document Relacorilant - similarity: 0.58366\n",
      "Document Cefalotin - similarity: 0.58363\n",
      "Document Niacinamide ascorbate - similarity: 0.58362\n",
      "Document Rose bengal - similarity: 0.58358\n",
      "Document Somatostatin - similarity: 0.58357\n",
      "Document Avelumab - similarity: 0.58351\n",
      "Document Spinach - similarity: 0.58350\n",
      "Document Azelastine - similarity: 0.58347\n",
      "Document Albendazole oxide - similarity: 0.58347\n",
      "Document Lauroyl chloride - similarity: 0.58337\n",
      "Document Hexetidine - similarity: 0.58336\n",
      "Document Isoflupredone acetate - similarity: 0.58331\n",
      "Document Saxagliptin - similarity: 0.58320\n",
      "Document Benzodiazepine - similarity: 0.58314\n",
      "Document Ferric chloride - similarity: 0.58311\n",
      "Document Moexipril - similarity: 0.58305\n",
      "Document Juniperus monosperma pollen - similarity: 0.58303\n",
      "Document Ribulose-5-Phosphate - similarity: 0.58302\n",
      "Document Oxaprozin - similarity: 0.58299\n",
      "Document Chenopodium botrys pollen - similarity: 0.58299\n",
      "Document 1D-myo-inositol 1,4,5-trisphosphate - similarity: 0.58295\n",
      "Document Camrelizumab - similarity: 0.58285\n",
      "Document Lonidamine - similarity: 0.58271\n",
      "Document Serum, horse - similarity: 0.58268\n",
      "Document Benzbromarone - similarity: 0.58268\n",
      "Document Phosphoaspartate - similarity: 0.58259\n",
      "Document Clidinium - similarity: 0.58256\n",
      "Document Trepibutone - similarity: 0.58255\n",
      "Document Domagrozumab - similarity: 0.58250\n",
      "Document Potassium permanganate - similarity: 0.58247\n",
      "Document Padsevonil - similarity: 0.58247\n",
      "Document Vincamine - similarity: 0.58247\n",
      "Document BMS-488043 - similarity: 0.58241\n",
      "Document Amlexanox - similarity: 0.58237\n",
      "Document Hexaminolevulinate - similarity: 0.58236\n",
      "Document Bisoctrizole - similarity: 0.58236\n",
      "Document Pralidoxime - similarity: 0.58231\n",
      "Document SSR-126517E - similarity: 0.58227\n",
      "Document Fluoroestradiol F-18 - similarity: 0.58222\n",
      "Document Enterococcus faecium - similarity: 0.58222\n",
      "Document Influenza A virus A/Texas/50/2012 X-223A (H3N2) antigen (UV, formaldehyde inactivated) - similarity: 0.58219\n",
      "Document ginkgolide-M - similarity: 0.58215\n",
      "Document Plum - similarity: 0.58213\n",
      "Document Deleobuvir - similarity: 0.58209\n",
      "Document Tetramethylammonium - similarity: 0.58202\n",
      "Document Thrombopoietin - similarity: 0.58185\n",
      "Document 4-AMINO-2-OCTYLOXY-6-HYDROXYMETHYL-TETRAHYDRO-PYRAN-3,5-DIOL - similarity: 0.58185\n",
      "Document Crizotinib - similarity: 0.58185\n",
      "Document Alirocumab - similarity: 0.58179\n",
      "Document Solenopsis invicta - similarity: 0.58174\n",
      "Document Hexylresorcinol - similarity: 0.58172\n",
      "Document Xipamide - similarity: 0.58171\n",
      "Document Phenylethyl resorcinol - similarity: 0.58166\n",
      "Document Antazoline - similarity: 0.58140\n",
      "Document Ciclopirox - similarity: 0.58131\n",
      "Document Peracetic acid - similarity: 0.58128\n",
      "Document Insulin detemir - similarity: 0.58126\n",
      "Document Delapril - similarity: 0.58123\n",
      "Document Ceiba pentandra fiber - similarity: 0.58123\n",
      "Document Glymidine - similarity: 0.58118\n",
      "Document Nitrogen trichloride - similarity: 0.58118\n",
      "Document Sulfabenzamide - similarity: 0.58107\n",
      "Document Cutamesine - similarity: 0.58098\n",
      "Document Ferric carboxymaltose - similarity: 0.58087\n",
      "Document N,N-DIMETHYL(5-(PYRIDIN-3-YL)FURAN-2-YL)METHANAMINE - similarity: 0.58085\n",
      "Document 6-BROMO-13-THIA-2,4,8,12,19-PENTAAZATRICYCLO[12.3.1.1~3,7~]NONADECA-1(18),3(19),4,6,14,16-HEXAENE 13,13-DIOXIDE - similarity: 0.58081\n",
      "Document Milnacipran - similarity: 0.58081\n",
      "Document Ambrosia bidentata pollen - similarity: 0.58076\n",
      "Document Chromium - similarity: 0.58071\n",
      "Document Lonazolac - similarity: 0.58067\n",
      "Document Borate ion - similarity: 0.58064\n",
      "Document Patisiran - similarity: 0.58056\n",
      "Document KOSN-1724 - similarity: 0.58054\n",
      "Document UK-432097 - similarity: 0.58054\n",
      "Document Brigatinib - similarity: 0.58053\n",
      "Document Apaziquone - similarity: 0.58053\n",
      "Document Bismuth subgallate - similarity: 0.58043\n",
      "Document 2',6'-DIFLUOROBIPHENYL-4-CARBOXYLIC ACID - similarity: 0.58038\n",
      "Document Talastine - similarity: 0.58033\n",
      "Document Influenza A virus A/Singapore/INFIMH-16-0019/2016 (H3N2) live (attenuated) antigen - similarity: 0.58031\n",
      "Document Cicletanine - similarity: 0.58028\n",
      "Document Oxitriptan - similarity: 0.58013\n",
      "Document Laromustine - similarity: 0.58000\n",
      "Document Melengestrol acetate - similarity: 0.57996\n",
      "Document Ipatasertib - similarity: 0.57995\n",
      "Document N-METHYLALANYL-3-METHYLVALYL-4-PHENOXY-N-(1,2,3,4-TETRAHYDRONAPHTHALEN-1-YL)PROLINAMIDE - similarity: 0.57995\n",
      "Document Ethanolamine oleate - similarity: 0.57992\n",
      "Document Vincristine - similarity: 0.57986\n",
      "Document Cefazolin - similarity: 0.57985\n",
      "Document Molindone - similarity: 0.57981\n",
      "Document Desloratadine - similarity: 0.57981\n",
      "Document Glucarpidase - similarity: 0.57970\n",
      "Document Bronopol - similarity: 0.57967\n",
      "Document Argifin - similarity: 0.57966\n",
      "Document Marimastat - similarity: 0.57965\n",
      "Document Aminacrine - similarity: 0.57965\n",
      "Document Isopentyl Pyrophosphate - similarity: 0.57964\n",
      "Document Iotrolan - similarity: 0.57956\n",
      "Document Magnesium oxide - similarity: 0.57955\n",
      "Document Ethyl Dimethyl Ammonio Propane Sulfonate - similarity: 0.57953\n",
      "Document Alginic acid - similarity: 0.57948\n",
      "Document Sulodexide - similarity: 0.57946\n",
      "Document 6-Hydroxymethyl-7,8-Dihydropterin - similarity: 0.57941\n",
      "Document Enclomiphene - similarity: 0.57933\n",
      "Document Lentil - similarity: 0.57931\n",
      "Document Carbazochrome - similarity: 0.57928\n",
      "Document Voxelotor - similarity: 0.57909\n",
      "Document Pasireotide - similarity: 0.57906\n",
      "Document Nintedanib - similarity: 0.57896\n",
      "Document Ethylmercurithiosalicylic acid - similarity: 0.57895\n",
      "Document Artemisinin - similarity: 0.57890\n",
      "Document Preladenant - similarity: 0.57889\n",
      "Document IMIDAZOPYRIDAZIN 1 - similarity: 0.57885\n",
      "Document 4-[(3R)-3-{[2-(4-FLUOROPHENYL)-2-OXOETHYL]AMINO}BUTYL]BENZAMIDE - similarity: 0.57882\n",
      "Document Pridopidine - similarity: 0.57881\n",
      "Document Diisopropyl methylphosphonate - similarity: 0.57878\n",
      "Document Helium He-3 - similarity: 0.57872\n",
      "Document Nomifensine - similarity: 0.57871\n",
      "Document Terpinen-4-ol - similarity: 0.57858\n",
      "Document Calcitriol - similarity: 0.57849\n",
      "Document Zoptarelin doxorubicin - similarity: 0.57846\n",
      "Document Lorlatinib - similarity: 0.57845\n",
      "Document Baclofen - similarity: 0.57843\n",
      "Document Leek - similarity: 0.57836\n",
      "Document [(1S)-1-acetamido-2-(4-chlorophenyl)ethyl]-[(2S)-2-amino-3-hydroxy-3-oxo-propoxy]-dihydroxy-boron - similarity: 0.57835\n",
      "Document Retagliptin - similarity: 0.57830\n",
      "Document Norethisterone - similarity: 0.57824\n",
      "Document NVS antibody - similarity: 0.57823\n",
      "Document Eperisone - similarity: 0.57809\n",
      "Document Calfactant - similarity: 0.57795\n",
      "Document Streptomycin - similarity: 0.57788\n",
      "Document Maduramicin - similarity: 0.57782\n",
      "Document Cantuzumab ravtansine - similarity: 0.57782\n",
      "Document Pitavastatin - similarity: 0.57781\n",
      "Document Pentamycin - similarity: 0.57776\n",
      "Document AG-702 - similarity: 0.57770\n",
      "Document Gonadorelin - similarity: 0.57769\n",
      "Document Spearmint - similarity: 0.57766\n",
      "Document 3-({2-[(2-AMINO-6-METHYLPYRIMIDIN-4-YL)ETHYNYL]BENZYL}AMINO)-1,3-OXAZOL-2(3H)-ONE - similarity: 0.57764\n",
      "Document (S)-3-(4-(2-Carbazol-9-Yl-Ethoxy)-Phenyl)-2-Ethoxy-Propionic Acid - similarity: 0.57758\n",
      "Document GDC-0152 - similarity: 0.57756\n",
      "Document G17DT - similarity: 0.57755\n",
      "Document Tricarballylic acid - similarity: 0.57754\n",
      "Document Florasulam - similarity: 0.57751\n",
      "Document (METHYLPYRIDAZINE PIPERIDINE PROPYLOXYPHENYL)ETHYLACETATE - similarity: 0.57750\n",
      "Document Navitoclax - similarity: 0.57742\n",
      "Document CZEN 002 - similarity: 0.57725\n",
      "Document VS-5584 - similarity: 0.57720\n",
      "Document Lenacapavir - similarity: 0.57715\n",
      "Document ONO-8539 - similarity: 0.57713\n",
      "Document Pteroic acid - similarity: 0.57708\n",
      "Document Acetylcholine - similarity: 0.57707\n",
      "Document Desirudin - similarity: 0.57704\n",
      "Document Calcium lactate - similarity: 0.57694\n",
      "Document Furafylline - similarity: 0.57693\n",
      "Document 2-Allyl-6-Methyl-Phenol - similarity: 0.57692\n",
      "Document Merimepodib - similarity: 0.57685\n",
      "Document Benzoyl-tyrosine-alanine-fluoro-methyl ketone - similarity: 0.57683\n",
      "Document (3R,4S)-1-[(4-AMINO-5H-PYRROLO[3,2-D]PYRIMIDIN-7-YL)METHYL]-4-[(METHYLSULFANYL)METHYL]PYRROLIDIN-3-OL - similarity: 0.57672\n",
      "Document Mequinol - similarity: 0.57661\n",
      "Document Entolimod - similarity: 0.57660\n",
      "Document N-methyl-{4-[2-(7-oxo-6,7-dihydro-8H-[1,3]thiazolo[5,4-e]indol-8-ylidene)hydrazino]phenyl}methanesulfonamide - similarity: 0.57658\n",
      "Document Dronabinol - similarity: 0.57655\n",
      "Document Musa acuminata fruit - similarity: 0.57651\n",
      "Document N-Bromoacetyl-Aminoethyl Phosphate - similarity: 0.57648\n",
      "Document Streptococcus pneumoniae type 7f capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.57638\n",
      "Document Tanacetum cinerariifolium flower - similarity: 0.57632\n",
      "Document N1-CYCLOPENTYL-N2-(THIAZOL-2-YL)OXALAMIDE - similarity: 0.57625\n",
      "Document Watercress - similarity: 0.57623\n",
      "Document Pimasertib - similarity: 0.57622\n",
      "Document Dexamethasone - similarity: 0.57608\n",
      "Document Pranoprofen - similarity: 0.57606\n",
      "Document Cemiplimab - similarity: 0.57600\n",
      "Document Disaccharide tripeptide glycerol dipalmitoyl - similarity: 0.57597\n",
      "Document Basimglurant - similarity: 0.57595\n",
      "Document 4-(BENZHYDRYLOXY)-1-[3-(1H-TETRAAZOL-5-YL)PROPYL]PIPERIDINE - similarity: 0.57595\n",
      "Document Artemisia tridentata pollen - similarity: 0.57591\n",
      "Document METHYL-PHOSPHONIC ACID MONO-(4-NITRO-PHENYL) ESTER - similarity: 0.57587\n",
      "Document Denosumab - similarity: 0.57585\n",
      "Document 3-Methylglutamic acid - similarity: 0.57571\n",
      "Document N'-((2S,3R)-3-AMINO-2-HYDROXY-5-(ISOPROPYLSULFANYL)PENTANOYL)-N-3-CHLOROBENZOYL HYDRAZIDE - similarity: 0.57571\n",
      "Document Tetrahydrouridine - similarity: 0.57567\n",
      "Document Propylhexedrine - similarity: 0.57564\n",
      "Document Asparaginase Escherichia coli - similarity: 0.57556\n",
      "Document Clofibride - similarity: 0.57553\n",
      "Document Bacillus calmette-guerin substrain danish 1331 live antigen - similarity: 0.57552\n",
      "Document Nadifloxacin - similarity: 0.57551\n",
      "Document Pinus taeda pollen - similarity: 0.57550\n",
      "Document Citalopram - similarity: 0.57541\n",
      "Document Sabarubicin - similarity: 0.57541\n",
      "Document 3-(2-AMINO-6-BENZOYLQUINAZOLIN-3(4H)-YL)-N-CYCLOHEXYL-N-METHYLPROPANAMIDE - similarity: 0.57539\n",
      "Document String bean - similarity: 0.57536\n",
      "Document Aminohippuric acid - similarity: 0.57522\n",
      "Document Berkelium - similarity: 0.57520\n",
      "Document Mephenesin - similarity: 0.57520\n",
      "Document Axicabtagene ciloleucel - similarity: 0.57517\n",
      "Document Arsenous acid - similarity: 0.57517\n",
      "Document Metampicillin - similarity: 0.57507\n",
      "Document LX6171 - similarity: 0.57503\n",
      "Document Tecovirimat - similarity: 0.57493\n",
      "Document Carbarsone - similarity: 0.57491\n",
      "Document Mephenoxalone - similarity: 0.57485\n",
      "Document DCFBC - similarity: 0.57480\n",
      "Document Ticagrelor - similarity: 0.57457\n",
      "Document Carbopol 974P - similarity: 0.57456\n",
      "Document 3-[3-(3-methyl-6-{[(1S)-1-phenylethyl]amino}-1H-pyrazolo[4,3-c]pyridin-1-yl)phenyl]propanamide - similarity: 0.57446\n",
      "Document ABT-510 - similarity: 0.57442\n",
      "Document Ribavirin - similarity: 0.57428\n",
      "Document Balicatib - similarity: 0.57427\n",
      "Document Ombrabulin - similarity: 0.57425\n",
      "Document Carbetocin - similarity: 0.57424\n",
      "Document p-Fluorofentanyl - similarity: 0.57422\n",
      "Document Parthenolide - similarity: 0.57422\n",
      "Document Tiratricol - similarity: 0.57421\n",
      "Document N-(TERT-BUTYL)-3,5-DIMETHYL-N'-[(5-METHYL-2,3-DIHYDRO-1,4-BENZODIOXIN-6-YL)CARBONYL]BENZOHYDRAZIDE - similarity: 0.57407\n",
      "Document Rilonacept - similarity: 0.57406\n",
      "Document Flavoxate - similarity: 0.57405\n",
      "Document Bafetinib - similarity: 0.57391\n",
      "Document Testosterone - similarity: 0.57389\n",
      "Document L-Lysine - similarity: 0.57388\n",
      "Document (1S)-2-(1H-INDOL-3-YL)-1-[({5-[(E)-2-PYRIDIN-4-YLVINYL]PYRIDIN-3-YL}OXY)METHYL]ETHYLAMINE - similarity: 0.57386\n",
      "Document N~3~-[3-(5-METHOXYPYRIDIN-3-YL)BENZYL]PYRIDINE-2,3-DIAMINE - similarity: 0.57367\n",
      "Document SVV-001 - similarity: 0.57367\n",
      "Document Idebenone - similarity: 0.57364\n",
      "Document Pseudotropine - similarity: 0.57363\n",
      "Document AKN-028 - similarity: 0.57350\n",
      "Document Adamantane - similarity: 0.57346\n",
      "Document 5alpha-androstane-3beta,17alpha-diol - similarity: 0.57345\n",
      "Document Dasolampanel etibutil - similarity: 0.57341\n",
      "Document Ozagrel - similarity: 0.57341\n",
      "Document Mustard seed - similarity: 0.57340\n",
      "Document Streptococcus pneumoniae type 7f capsular polysaccharide antigen - similarity: 0.57339\n",
      "Document Siltuximab - similarity: 0.57336\n",
      "Document 3-(4-{2-[2-(2-bromo-acetylamino)-ethyldisulfanyl]-ethylcarbamoyl}-cyclohexylcarbamoyl)-pyrazine-2-carboxylic acid - similarity: 0.57336\n",
      "Document Barusiban - similarity: 0.57335\n",
      "Document 5-(2-chlorophenyl)furan-2-carbohydrazide - similarity: 0.57332\n",
      "Document Oleandrin - similarity: 0.57329\n",
      "Document Cotton seed - similarity: 0.57319\n",
      "Document Technetium Tc-99m pyrophosphate - similarity: 0.57312\n",
      "Document Abametapir - similarity: 0.57311\n",
      "Document N-phenylthiourea - similarity: 0.57310\n",
      "Document (3S,5R,7R,8S,9S,10R)-7-(hydroxymethyl)-3-(2-naphthyl)-1,6-dioxa-2-azaspiro[4.5]decane-8,9,10-triol - similarity: 0.57304\n",
      "Document GE-2270A - similarity: 0.57298\n",
      "Document Medium-chain triglycerides - similarity: 0.57294\n",
      "Document PQ-10 - similarity: 0.57275\n",
      "Document (3AS,4R,9BR)-4-(4-HYDROXYPHENYL)-1,2,3,3A,4,9B-HEXAHYDROCYCLOPENTA[C]CHROMEN-9-OL - similarity: 0.57255\n",
      "Document Nivocasan - similarity: 0.57253\n",
      "Document BMS-929075 - similarity: 0.57243\n",
      "Document Pectin - similarity: 0.57239\n",
      "Document Trofinetide - similarity: 0.57234\n",
      "Document Cromoglicic acid - similarity: 0.57233\n",
      "Document Ombitasvir - similarity: 0.57232\n",
      "Document Panax quinquefolius whole - similarity: 0.57229\n",
      "Document Piroxicam - similarity: 0.57225\n",
      "Document Sesame seed - similarity: 0.57223\n",
      "Document Vanilla - similarity: 0.57222\n",
      "Document Oxycinchophen - similarity: 0.57213\n",
      "Document 5-(5-chloro-2,4-dihydroxyphenyl)-N-ethyl-4-[4-(morpholin-4-ylmethyl)phenyl]isoxazole-3-carboxamide - similarity: 0.57213\n",
      "Document SUVN-G3031 - similarity: 0.57208\n",
      "Document Arbaclofen Placarbil - similarity: 0.57208\n",
      "Document Ulipristal - similarity: 0.57208\n",
      "Document Benperidol - similarity: 0.57208\n",
      "Document 5-fluoro-2'-deoxyuridine-5'-monophosphate - similarity: 0.57207\n",
      "Document Camostat - similarity: 0.57200\n",
      "Document Beractant - similarity: 0.57199\n",
      "Document Lometrexol - similarity: 0.57193\n",
      "Document Varenicline - similarity: 0.57192\n",
      "Document Nifedipine - similarity: 0.57185\n",
      "Document Latiglutenase - similarity: 0.57175\n",
      "Document Digitoxin - similarity: 0.57175\n",
      "Document Felis catus skin - similarity: 0.57172\n",
      "Document Ammonio methacrylate copolymer type A - similarity: 0.57170\n",
      "Document Tetrabutylammonium Ion - similarity: 0.57165\n",
      "Document Ibacitabine - similarity: 0.57162\n",
      "Document Isoflurane - similarity: 0.57162\n",
      "Document 1,4-Dideoxy-O2-Sulfo-Glucuronic Acid - similarity: 0.57161\n",
      "Document Cow milk - similarity: 0.57148\n",
      "Document Raltitrexed - similarity: 0.57144\n",
      "Document Cefuroxime - similarity: 0.57144\n",
      "Document (5-CHLORO-2-{[(3-NITROBENZYL)AMINO]CARBONYL}PHENOXY)ACETIC ACID - similarity: 0.57137\n",
      "Document Brincidofovir - similarity: 0.57133\n",
      "Document CTT-1057 - similarity: 0.57127\n",
      "Document Troxacitabine - similarity: 0.57124\n",
      "Document (2S)-2-ETHOXY-3-{4-[2-(10H-PHENOXAZIN-10-YL)ETHOXY]PHENYL}PROPANOIC ACID - similarity: 0.57123\n",
      "Document Flibanserin - similarity: 0.57122\n",
      "Document Dexketoprofen - similarity: 0.57114\n",
      "Document (3S)-1-CYCLOHEXYL-5-OXO-N-PHENYLPYRROLIDINE-3-CARBOXAMIDE - similarity: 0.57109\n",
      "Document Isosorbide dinitrate - similarity: 0.57105\n",
      "Document Patritumab - similarity: 0.57101\n",
      "Document Lapatinib - similarity: 0.57090\n",
      "Document Formebolone - similarity: 0.57084\n",
      "Document 2-(ETHOXYMETHYL)-4-(4-FLUOROPHENYL)-3-[2-(2-HYDROXYPHENOXY)PYRIMIDIN-4-YL]ISOXAZOL-5(2H)-ONE - similarity: 0.57065\n",
      "Document Mesoxalic acid - similarity: 0.57065\n",
      "Document Diazepam - similarity: 0.57062\n",
      "Document Metoprine - similarity: 0.57060\n",
      "Document NCX 1015 - similarity: 0.57057\n",
      "Document Eculizumab - similarity: 0.57057\n",
      "Document CP1-1189 - similarity: 0.57056\n",
      "Document D-[(N-Hydroxyamino)Carbonyl]Phenylalanine - similarity: 0.57052\n",
      "Document Alpha-Amino-2-Indanacetic Acid - similarity: 0.57051\n",
      "Document Tazemetostat - similarity: 0.57049\n",
      "Document Naphthyridine Inhibitor - similarity: 0.57048\n",
      "Document Oleoyl chloride - similarity: 0.57043\n",
      "Document Elvitegravir - similarity: 0.57041\n",
      "Document Pancrelipase lipase - similarity: 0.57037\n",
      "Document Anastrozole - similarity: 0.57036\n",
      "Document Satavaptan - similarity: 0.57030\n",
      "Document Silicon - similarity: 0.57025\n",
      "Document Phenyl aminosalicylate - similarity: 0.57023\n",
      "Document Zofenopril - similarity: 0.57011\n",
      "Document Rasdegafusp alfa - similarity: 0.56998\n",
      "Document Nateglinide - similarity: 0.56994\n",
      "Document Zeaxanthin - similarity: 0.56981\n",
      "Document Tropatepine - similarity: 0.56977\n",
      "Document Trebananib - similarity: 0.56973\n",
      "Document Tempol - similarity: 0.56971\n",
      "Document YSIL6 - similarity: 0.56967\n",
      "Document Methazolamide - similarity: 0.56967\n",
      "Document Neodymium - similarity: 0.56967\n",
      "Document Lithium succinate - similarity: 0.56966\n",
      "Document Mitolactol - similarity: 0.56965\n",
      "Document Masitinib - similarity: 0.56965\n",
      "Document 3,4-Epoxybutyl-Alpha-D-Glucopyranoside - similarity: 0.56965\n",
      "Document Zorubicin - similarity: 0.56960\n",
      "Document Flunixin - similarity: 0.56959\n",
      "Document Tetrachlorodecaoxide - similarity: 0.56949\n",
      "Document OREGON GREEN 488 CARBOXYLATE - similarity: 0.56948\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 (H3N2) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.56940\n",
      "Document 3-Oxo-Pentadecanoic Acid - similarity: 0.56935\n",
      "Document Budiodarone - similarity: 0.56935\n",
      "Document Bergapten - similarity: 0.56929\n",
      "Document Biphenylalanine - similarity: 0.56925\n",
      "Document ATL1102 - similarity: 0.56924\n",
      "Document Latamoxef - similarity: 0.56921\n",
      "Document Carfilzomib - similarity: 0.56917\n",
      "Document Gadobenic acid - similarity: 0.56904\n",
      "Document Samidorphan - similarity: 0.56902\n",
      "Document Human papillomavirus type 58 L1 capsid protein antigen - similarity: 0.56895\n",
      "Document Azelaic acid - similarity: 0.56893\n",
      "Document Mepivacaine - similarity: 0.56886\n",
      "Document Amrubicin - similarity: 0.56867\n",
      "Document Toripalimab - similarity: 0.56865\n",
      "Document WIN 55212-2 - similarity: 0.56858\n",
      "Document Kiwi fruit - similarity: 0.56853\n",
      "Document Emepronium - similarity: 0.56851\n",
      "Document Potassium nitrate - similarity: 0.56845\n",
      "Document N-(1-PHENYL-PROPYL)-FORMAMIDE - similarity: 0.56844\n",
      "Document 8-(2-Chloro-3,4,5-Trimethoxy-Benzyl)-9-Pent-4-Ylnyl-9h-Purin-6-Ylamine - similarity: 0.56828\n",
      "Document Valproic acid - similarity: 0.56826\n",
      "Document Nalbuphine - similarity: 0.56825\n",
      "Document Triprolidine - similarity: 0.56816\n",
      "Document Carbon Black - similarity: 0.56815\n",
      "Document Boldione - similarity: 0.56804\n",
      "Document MK-6325 - similarity: 0.56795\n",
      "Document Netupitant - similarity: 0.56790\n",
      "Document Davunetide - similarity: 0.56785\n",
      "Document (+)-menthol - similarity: 0.56784\n",
      "Document QS-21 - similarity: 0.56783\n",
      "Document Zinc acetate - similarity: 0.56775\n",
      "Document Tafluprost - similarity: 0.56775\n",
      "Document Valproate bismuth - similarity: 0.56774\n",
      "Document Promazine - similarity: 0.56758\n",
      "Document Miglitol - similarity: 0.56753\n",
      "Document Racephedrine - similarity: 0.56749\n",
      "Document Diflorasone - similarity: 0.56746\n",
      "Document Cabazitaxel - similarity: 0.56746\n",
      "Document (S)-2-[4-(Aminomethyl)-1h-1,2,3-Triazol-1-Yl]-4-Methylpentanoic Acid - similarity: 0.56727\n",
      "Document Indobufen - similarity: 0.56722\n",
      "Document Potato - similarity: 0.56721\n",
      "Document PHA-793887 - similarity: 0.56720\n",
      "Document Isosorbide mononitrate - similarity: 0.56717\n",
      "Document Ajmaline - similarity: 0.56713\n",
      "Document Tribenuron Methyl - similarity: 0.56712\n",
      "Document Haematonectria haematococca - similarity: 0.56705\n",
      "Document CR002 - similarity: 0.56704\n",
      "Document Paroxetine - similarity: 0.56703\n",
      "Document Atreleuton - similarity: 0.56698\n",
      "Document Vespula squamosa venom protein - similarity: 0.56693\n",
      "Document Demeclocycline - similarity: 0.56687\n",
      "Document Brilliant blue G - similarity: 0.56684\n",
      "Document Neurospora intermedia - similarity: 0.56678\n",
      "Document Cibenzoline - similarity: 0.56676\n",
      "Document Bumetanide - similarity: 0.56674\n",
      "Document 2-(1-Hexyloxyethyl)-2-devinyl pyropheophorbide-a - similarity: 0.56665\n",
      "Document Copanlisib - similarity: 0.56662\n",
      "Document Ubiquinone Q1 - similarity: 0.56659\n",
      "Document Vatelizumab - similarity: 0.56658\n",
      "Document Selenomethionine - similarity: 0.56654\n",
      "Document Crab leg - similarity: 0.56650\n",
      "Document GP-100 antigen - similarity: 0.56649\n",
      "Document Nitroprusside - similarity: 0.56639\n",
      "Document Bivalirudin - similarity: 0.56638\n",
      "Document 2-(4-fluorophenyl)-N-{[3-fluoro-4-(1H-pyrrolo[2,3-b]pyridin-4-yloxy)phenyl]carbamoyl}acetamide - similarity: 0.56634\n",
      "Document Selisistat - similarity: 0.56628\n",
      "Document Girentuximab - similarity: 0.56627\n",
      "Document Myristica fragrans fruit - similarity: 0.56625\n",
      "Document {[7-(Difluoro-Phosphono-Methyl)-Naphthalen-2-Yl]-Difluoro-Methyl}-Phosphonic Acid - similarity: 0.56624\n",
      "Document Nicoboxil - similarity: 0.56622\n",
      "Document Leridistim - similarity: 0.56617\n",
      "Document MGL-3196 - similarity: 0.56610\n",
      "Document 2,3,6A,7,8,9-HEXAHYDRO-11H-[1,4]DIOXINO[2,3-G]PYRROLO[2,1-B][1,3]BENZOXAZIN-11-ONE - similarity: 0.56606\n",
      "Document Teniposide - similarity: 0.56606\n",
      "Document Clenbuterol - similarity: 0.56599\n",
      "Document TG4010 - similarity: 0.56597\n",
      "Document Perchlorate - similarity: 0.56597\n",
      "Document Cefixime - similarity: 0.56595\n",
      "Document Turnip - similarity: 0.56589\n",
      "Document Deoxyuridine monophosphate - similarity: 0.56586\n",
      "Document 6-CHLORO-N-{(3S)-1-[(1S)-1-METHYL-2-(4-MORPHOLINYL)-2-OXO ETHYL]-2-OXO-3-PYRROLIDINYL}-2-NAPHTHALENESULFONAMIDE - similarity: 0.56584\n",
      "Document Dapagliflozin - similarity: 0.56580\n",
      "Document Grn163l - similarity: 0.56579\n",
      "Document Cetirizine - similarity: 0.56572\n",
      "Document Rifabutin - similarity: 0.56572\n",
      "Document Detomidine - similarity: 0.56572\n",
      "Document Otilonium - similarity: 0.56562\n",
      "Document N(2)-carbamimidoyl-N-{2-[4-(3-{4-[(5-carboxyfuran-2-yl)methoxy]-2,3-dichlorophenyl}-1-methyl-1H-pyrazol-5-yl)piperidin-1-yl]-2-oxoethyl}-D-leucinamide - similarity: 0.56560\n",
      "Document Tartronate - similarity: 0.56549\n",
      "Document Iva angustifolia pollen - similarity: 0.56549\n",
      "Document Troleandomycin - similarity: 0.56538\n",
      "Document Palmitoyl oligopeptide - similarity: 0.56532\n",
      "Document Teplizumab - similarity: 0.56525\n",
      "Document Mongersen - similarity: 0.56520\n",
      "Document Macadamia oil - similarity: 0.56517\n",
      "Document 2-[(1R)-1-CARBOXY-2-(4-HYDROXYPHENYL)ETHYL]-1,3-DIOXOISOINDOLINE-5-CARBOXYLIC ACID - similarity: 0.56512\n",
      "Document 1-Amino-6-Cyclohex-3-Enylmethyloxypurine - similarity: 0.56507\n",
      "Document MM-093 - similarity: 0.56503\n",
      "Document Vatiquinone - similarity: 0.56501\n",
      "Document Tribulus terrestris fruit - similarity: 0.56496\n",
      "Document Naloxone - similarity: 0.56496\n",
      "Document Dapivirine - similarity: 0.56474\n",
      "Document Glycopyrronium - similarity: 0.56474\n",
      "Document 4-{[4-(4-fluoro-3-methylphenyl)-1,3-thiazol-2-yl]amino}-2-hydroxybenzoic acid - similarity: 0.56459\n",
      "Document Iscotrizinol - similarity: 0.56456\n",
      "Document Nedaplatin - similarity: 0.56448\n",
      "Document Butacaine - similarity: 0.56439\n",
      "Document Ozone - similarity: 0.56431\n",
      "Document Sporotrichum pruinosum - similarity: 0.56429\n",
      "Document Inositol 1,3,4,5-Tetrakisphosphate - similarity: 0.56427\n",
      "Document SGN-15 - similarity: 0.56423\n",
      "Document 1-Hydroxy-2-S-glutathionyl-3-para-nitrophenoxy-propane - similarity: 0.56422\n",
      "Document Daclatasvir - similarity: 0.56420\n",
      "Document Oxypurinol - similarity: 0.56414\n",
      "Document (3AR,5R,6S,7R,7AR)-5-(HYDROXYMETHYL)-2-PROPYL-5,6,7,7A-TETRAHYDRO-3AH-PYRANO[3,2-D][1,3]THIAZOLE-6,7-DIOL - similarity: 0.56402\n",
      "Document N-Pyridoxyl-1-Amino-Cyclopropanecarboxylic Acid-5-Monophosphate - similarity: 0.56398\n",
      "Document Diethyl 4-Methylbenzylphosphonate - similarity: 0.56397\n",
      "Document Ginseng - similarity: 0.56392\n",
      "Document Yeast - similarity: 0.56391\n",
      "Document Eleuthero - similarity: 0.56391\n",
      "Document Sorivudine - similarity: 0.56389\n",
      "Document 2-(2-PHENYL-3-PYRIDIN-2-YL-4,5,6,7-TETRAHYDRO-2H-ISOPHOSPHINDOL-1-YL)PYRIDINE - similarity: 0.56388\n",
      "Document Mevastatin - similarity: 0.56387\n",
      "Document Aluminum chloride - similarity: 0.56376\n",
      "Document 1-[2-(3-ACETYL-2-HYDROXY-6-METHOXY-PHENYL)-CYCLOPROPYL]-3-(5-CYANO-PYRIDIN-2-YL)-THIOUREA - similarity: 0.56364\n",
      "Document Lanoteplase - similarity: 0.56363\n",
      "Document Butyl alcohol - similarity: 0.56353\n",
      "Document Methacholine - similarity: 0.56352\n",
      "Document Cyfluthrin - similarity: 0.56351\n",
      "Document Amifampridine - similarity: 0.56347\n",
      "Document 8-oxo-dGMP - similarity: 0.56346\n",
      "Document Chlorotrianisene - similarity: 0.56331\n",
      "Document AT1391 - similarity: 0.56328\n",
      "Document PF-04691502 - similarity: 0.56309\n",
      "Document alpha-L-fucose - similarity: 0.56307\n",
      "Document Tolrestat - similarity: 0.56300\n",
      "Document Saquinavir - similarity: 0.56298\n",
      "Document Typhoid Vaccine Live - similarity: 0.56296\n",
      "Document Elamipretide - similarity: 0.56294\n",
      "Document Pirlimycin - similarity: 0.56293\n",
      "Document Magnesium carbonate - similarity: 0.56292\n",
      "Document Molgramostim - similarity: 0.56290\n",
      "Document Oxaceprol - similarity: 0.56284\n",
      "Document 4-Sulfonamide-[1-(4-Aminobutane)]Benzamide - similarity: 0.56278\n",
      "Document Sulpiride - similarity: 0.56276\n",
      "Document Chlortetracycline - similarity: 0.56274\n",
      "Document Edetate copper disodium - similarity: 0.56268\n",
      "Document Cyclophosphamide - similarity: 0.56265\n",
      "Document Norethindrone enanthate - similarity: 0.56261\n",
      "Document Dexmedetomidine - similarity: 0.56256\n",
      "Document Navarixin - similarity: 0.56255\n",
      "Document Indibulin - similarity: 0.56251\n",
      "Document Meladrazine - similarity: 0.56251\n",
      "Document Arginine glutamate - similarity: 0.56250\n",
      "Document Pyruvic acid - similarity: 0.56246\n",
      "Document Ibuprofen - similarity: 0.56240\n",
      "Document Carprofen - similarity: 0.56236\n",
      "Document Sunflower seed - similarity: 0.56233\n",
      "Document Labetalol - similarity: 0.56221\n",
      "Document Pramipexole - similarity: 0.56220\n",
      "Document Fluasterone - similarity: 0.56218\n",
      "Document Zotepine - similarity: 0.56212\n",
      "Document Linezolid - similarity: 0.56208\n",
      "Document Arnica montana - similarity: 0.56207\n",
      "Document Sparsomycin - similarity: 0.56197\n",
      "Document Crenezumab - similarity: 0.56186\n",
      "Document 3-{6-[(8-HYDROXY-QUINOLINE-2-CARBONYL)-AMINO]-2-THIOPHEN-2-YL-HEXANOYLAMINO}-4-OXO-BUTYRI ACID - similarity: 0.56171\n",
      "Document Benzquinamide - similarity: 0.56166\n",
      "Document Cocarboxylase - similarity: 0.56166\n",
      "Document 4,5,6,7-Tetrachloro-3h-Isobenzofuran-1-One - similarity: 0.56166\n",
      "Document N-Methyl-Alpha-Beta-Dehydroalanine - similarity: 0.56164\n",
      "Document Isoconazole - similarity: 0.56163\n",
      "Document Everolimus - similarity: 0.56162\n",
      "Document Dopamine - similarity: 0.56161\n",
      "Document Aminopromazine - similarity: 0.56161\n",
      "Document S-(4-nitrobenzyl)glutathione - similarity: 0.56156\n",
      "Document Atropine - similarity: 0.56151\n",
      "Document Technetium Tc-99m oxidronate - similarity: 0.56150\n",
      "Document Glycerin - similarity: 0.56147\n",
      "Document Tipifarnib - similarity: 0.56144\n",
      "Document Voxtalisib - similarity: 0.56138\n",
      "Document Gabexate - similarity: 0.56135\n",
      "Document Onalespib - similarity: 0.56118\n",
      "Document Glycerol phenylbutyrate - similarity: 0.56117\n",
      "Document Oxcarbazepine - similarity: 0.56112\n",
      "Document Fomivirsen - similarity: 0.56109\n",
      "Document Clonazepam - similarity: 0.56102\n",
      "Document Papaya - similarity: 0.56092\n",
      "Document Poliovirus type 2 antigen (formaldehyde inactivated) - similarity: 0.56091\n",
      "Document Avacincaptad pegol - similarity: 0.56089\n",
      "Document rhIGFBP-3 - similarity: 0.56073\n",
      "Document CPG 10101 - similarity: 0.56071\n",
      "Document Botulinum toxin type A - similarity: 0.56071\n",
      "Document Methylprednisolone - similarity: 0.56063\n",
      "Document Sebelipase alfa - similarity: 0.56054\n",
      "Document Methapyrilene - similarity: 0.56049\n",
      "Document Dinoprostone - similarity: 0.56048\n",
      "Document FKB-001 - similarity: 0.56044\n",
      "Document Ramoplanin - similarity: 0.56030\n",
      "Document Thalidomide - similarity: 0.56029\n",
      "Document Famotidine - similarity: 0.56026\n",
      "Document Nedocromil - similarity: 0.56025\n",
      "Document Tranilast - similarity: 0.56016\n",
      "Document Dipyridamole - similarity: 0.56008\n",
      "Document Suramin - similarity: 0.56008\n",
      "Document 5-CHLORO-6-METHYL-N-(2-PHENYLETHYL)-2-PYRIDIN-2-YLPYRIMIDIN-4-AMINE - similarity: 0.56002\n",
      "Document Streptococcus pneumoniae type 2 capsular polysaccharide antigen - similarity: 0.55999\n",
      "Document Niacin - similarity: 0.55998\n",
      "Document Combretastatin A4 - similarity: 0.55997\n",
      "Document Etybenzatropine - similarity: 0.55997\n",
      "Document Morpholine - similarity: 0.55992\n",
      "Document Norbixin - similarity: 0.55992\n",
      "Document Naftazone - similarity: 0.55991\n",
      "Document GW842166 - similarity: 0.55984\n",
      "Document Ethionamide - similarity: 0.55970\n",
      "Document Tmr - similarity: 0.55968\n",
      "Document RO-4987655 - similarity: 0.55951\n",
      "Document CNS-5161 - similarity: 0.55944\n",
      "Document Streptococcus pneumoniae type 23f capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.55943\n",
      "Document Filgrastim - similarity: 0.55940\n",
      "Document Diphenidol - similarity: 0.55932\n",
      "Document Picropodophyllin - similarity: 0.55932\n",
      "Document alpha-Fluoro-carboxymethyldethia coenzyme A complex - similarity: 0.55917\n",
      "Document Ceftazidime - similarity: 0.55914\n",
      "Document 3-fluoro-N-1H-indol-5-yl-5-morpholin-4-ylbenzamide - similarity: 0.55911\n",
      "Document Polysilicone-15 - similarity: 0.55910\n",
      "Document Stanozolol - similarity: 0.55909\n",
      "Document 6-amino-1-methyl-7H-purin-1-ium - similarity: 0.55907\n",
      "Document Zosuquidar - similarity: 0.55905\n",
      "Document Estradiol - similarity: 0.55904\n",
      "Document Uridine 5'-triphosphate - similarity: 0.55889\n",
      "Document Muromonab - similarity: 0.55887\n",
      "Document Ulmus americana pollen - similarity: 0.55886\n",
      "Document Myrcludex B - similarity: 0.55885\n",
      "Document Methylumbelliferyl sialic acid - similarity: 0.55882\n",
      "Document Harmaline - similarity: 0.55879\n",
      "Document Moroctocog alfa - similarity: 0.55878\n",
      "Document Sodium bicarbonate - similarity: 0.55876\n",
      "Document Calcium - similarity: 0.55874\n",
      "Document Insulin beef - similarity: 0.55874\n",
      "Document Cyd dengue virus serotype 2 live (attenuated) antigen - similarity: 0.55867\n",
      "Document Potassium citrate - similarity: 0.55839\n",
      "Document Trifluoperazine - similarity: 0.55839\n",
      "Document Hydroxocobalamin - similarity: 0.55837\n",
      "Document Bremelanotide - similarity: 0.55833\n",
      "Document Vabicaserin - similarity: 0.55827\n",
      "Document DTS-201 - similarity: 0.55824\n",
      "Document 2,3-di-O-phytanyl-sn-glycerol - similarity: 0.55821\n",
      "Document Diloxanide - similarity: 0.55810\n",
      "Document (6S)-5,6,7,8-tetrahydrofolic acid - similarity: 0.55808\n",
      "Document Grapiprant - similarity: 0.55797\n",
      "Document Corticosterone - similarity: 0.55796\n",
      "Document Plantago ovata seed - similarity: 0.55796\n",
      "Document Zimelidine - similarity: 0.55796\n",
      "Document Aprepitant - similarity: 0.55795\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document (E)-(4S,6S)-6-((S)-2-{(S)-2-[(FURAN-2-CARBONYL)-AMINO]-3-METHYL-BUTYRYLAMINO}-3-METHYL-BUTYRYLAMINO)-8-METHYL-5-OXO-4-((R)-2-OXO-PYRROLIDIN-3-YLMETHYL)-NON-2-ENOIC ACID ETHYL ESTER - similarity: 0.55792\n",
      "Document R667 - similarity: 0.55785\n",
      "Document Margetuximab - similarity: 0.55785\n",
      "Document Cephalothin Group - similarity: 0.55784\n",
      "Document METHYL 3-CHLORO-2-{3-[(2,5-DIHYDROXY-4-METHOXYPHENYL)AMINO]-3-OXOPROPYL}-4,6-DIHYDROXYBENZOATE - similarity: 0.55779\n",
      "Document 4-demethyl-4-cholesteryloxycarbonylpenclomedine - similarity: 0.55777\n",
      "Document Influenza A virus A/Uruguay/716/2007(H3N2) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.55775\n",
      "Document Cibacron Blue - similarity: 0.55772\n",
      "Document beta-2-Thienyl-L-alanine - similarity: 0.55767\n",
      "Document 1-tert-butyl-3-(3-methylbenzyl)-1H-pyrazolo[3,4-d]pyrimidin-4-amine - similarity: 0.55767\n",
      "Document 2,6-DIMETHYL-1-(3-[3-METHYL-5-ISOXAZOLYL]-PROPANYL)-4-[2-METHYL-4-ISOXAZOLYL]-PHENOL - similarity: 0.55766\n",
      "Document Lycopene - similarity: 0.55766\n",
      "Document Dibunate - similarity: 0.55763\n",
      "Document Phosphocreatine - similarity: 0.55757\n",
      "Document 3-[[(METHYLAMINO)SULFONYL]AMINO]-2-OXO-6-PHENYL-N-[3,3,3-TRIFLUORO-1-(1-METHYLETHYL)-2-OXOPHENYL]-1(2H)-PYRIDINE ACETAMIDE - similarity: 0.55752\n",
      "Document Pradefovir mesylate - similarity: 0.55751\n",
      "Document Ketoprofen - similarity: 0.55745\n",
      "Document Calendula officinalis flower - similarity: 0.55740\n",
      "Document (1S)-1-(3-chlorophenyl)-2-oxo-2-[(1,3,4-trioxo-1,2,3,4-tetrahydroisoquinolin-5-yl)amino]ethyl acetate - similarity: 0.55739\n",
      "Document N-{2-[(4'-CYANO-1,1'-BIPHENYL-4-YL)OXY]ETHYL}-N'-HYDROXY-N-METHYLUREA - similarity: 0.55735\n",
      "Document Digoxin - similarity: 0.55733\n",
      "Document ISIS 113715 - similarity: 0.55725\n",
      "Document Serrapeptase - similarity: 0.55720\n",
      "Document 4-({5-[(4-AMINOCYCLOHEXYL)AMINO][1,2,4]TRIAZOLO[1,5-A]PYRIMIDIN-7-YL}AMINO)BENZENESULFONAMIDE - similarity: 0.55716\n",
      "Document Crotonaldehyde - similarity: 0.55712\n",
      "Document Dehydroemetine - similarity: 0.55700\n",
      "Document LY-2886721 - similarity: 0.55688\n",
      "Document Vilanterol - similarity: 0.55683\n",
      "Document Disopyramide - similarity: 0.55681\n",
      "Document Mirogabalin - similarity: 0.55677\n",
      "Document Radish - similarity: 0.55675\n",
      "Document Nalidixic acid - similarity: 0.55673\n",
      "Document Phthalylsulfathiazole - similarity: 0.55669\n",
      "Document 2,3-Anhydro-quinic acid - similarity: 0.55657\n",
      "Document Ferric (59Fe) citrate - similarity: 0.55656\n",
      "Document Methyl red - similarity: 0.55656\n",
      "Document 3-[6-bromo-2-fluoro-3-(1H-pyrazolo[3,4-c]pyridazin-3-ylmethyl)phenoxy]-5-chlorobenzonitrile - similarity: 0.55651\n",
      "Document Pinus echinata pollen - similarity: 0.55650\n",
      "Document Metoprolol - similarity: 0.55648\n",
      "Document Guanoclor - similarity: 0.55647\n",
      "Document Pargyline - similarity: 0.55646\n",
      "Document Idalopirdine - similarity: 0.55646\n",
      "Document Green bell pepper - similarity: 0.55636\n",
      "Document S,S'-(1,3-Phenylene-Bis(1,2-Ethanediyl))Bis-Isothiourea - similarity: 0.55632\n",
      "Document Roxarsone - similarity: 0.55621\n",
      "Document Piperacillin - similarity: 0.55620\n",
      "Document Physostigmine - similarity: 0.55600\n",
      "Document Hops - similarity: 0.55598\n",
      "Document Iprazochrome - similarity: 0.55596\n",
      "Document Fibrinolysin - similarity: 0.55591\n",
      "Document (3R)-4-(p-toluenesulfonyl)-1,4-thiazane-3-carboxylicacid-L-phenylalanine ethyl ester - similarity: 0.55591\n",
      "Document Clevidipine - similarity: 0.55589\n",
      "Document Aloxiprin - similarity: 0.55586\n",
      "Document Clobetasol propionate - similarity: 0.55582\n",
      "Document gamma-Hydroxybutyric acid - similarity: 0.55580\n",
      "Document 2-Amino-3-Ketobutyric Acid - similarity: 0.55575\n",
      "Document Dexfenfluramine - similarity: 0.55564\n",
      "Document Hydrocortisone aceponate - similarity: 0.55560\n",
      "Document Paraldehyde - similarity: 0.55543\n",
      "Document 3-deoxy-D-lyxo-hexonic acid - similarity: 0.55543\n",
      "Document Eplerenone - similarity: 0.55529\n",
      "Document Morus alba root bark - similarity: 0.55518\n",
      "Document Dabigatran etexilate - similarity: 0.55515\n",
      "Document Brodimoprim - similarity: 0.55498\n",
      "Document Benazepril - similarity: 0.55497\n",
      "Document Fructose -6-Phosphate - similarity: 0.55495\n",
      "Document Antineoplaston A10 - similarity: 0.55494\n",
      "Document Cobalt - similarity: 0.55493\n",
      "Document IRX-2 - similarity: 0.55490\n",
      "Document Synthetic camphor - similarity: 0.55488\n",
      "Document Teduglutide - similarity: 0.55486\n",
      "Document Quercus lobata pollen - similarity: 0.55485\n",
      "Document Glycyrrhizic acid - similarity: 0.55485\n",
      "Document Adinazolam - similarity: 0.55482\n",
      "Document Ziprasidone - similarity: 0.55482\n",
      "Document Zidovudine - similarity: 0.55481\n",
      "Document Vesatolimod - similarity: 0.55472\n",
      "Document Kava - similarity: 0.55471\n",
      "Document Thiram - similarity: 0.55462\n",
      "Document Fondaparinux - similarity: 0.55460\n",
      "Document Firtecan pegol - similarity: 0.55452\n",
      "Document Levobunolol - similarity: 0.55444\n",
      "Document Zinc sulfate, unspecified form - similarity: 0.55439\n",
      "Document 2-(2-{2-[2-(2-Methoxy-Ethoxy)-Ethoxy]-Ethoxy}-Ethoxy)-Ethanol - similarity: 0.55436\n",
      "Document (2R)-2-{[4-(benzylamino)-8-(1-methylethyl)pyrazolo[1,5-a][1,3,5]triazin-2-yl]amino}butan-1-ol - similarity: 0.55435\n",
      "Document Iobitridol - similarity: 0.55430\n",
      "Document Zn(Ii)-(20-Oxo-Protoporphyrin Ix) - similarity: 0.55428\n",
      "Document Cashew - similarity: 0.55427\n",
      "Document Sotirimod - similarity: 0.55417\n",
      "Document Nivolumab - similarity: 0.55411\n",
      "Document Squalane - similarity: 0.55411\n",
      "Document Propoxycaine - similarity: 0.55400\n",
      "Document p-Heptylphenol - similarity: 0.55398\n",
      "Document PARA-NITROPHENYLPHOSPHONOBUTANOYL-GLYCINE - similarity: 0.55395\n",
      "Document Ubenimex - similarity: 0.55395\n",
      "Document Captodiame - similarity: 0.55389\n",
      "Document Entacapone - similarity: 0.55384\n",
      "Document Sugarcane - similarity: 0.55384\n",
      "Document N-hexanoyl-L-homoserine - similarity: 0.55371\n",
      "Document Nicotine - similarity: 0.55349\n",
      "Document anle138b - similarity: 0.55347\n",
      "Document Tideglusib - similarity: 0.55342\n",
      "Document Kidney bean - similarity: 0.55340\n",
      "Document Atazanavir - similarity: 0.55339\n",
      "Document Potassium acetate - similarity: 0.55338\n",
      "Document Tildipirosin - similarity: 0.55333\n",
      "Document Linaclotide - similarity: 0.55332\n",
      "Document Tolbutamide - similarity: 0.55328\n",
      "Document Amitriptylinoxide - similarity: 0.55327\n",
      "Document Neocartilage - similarity: 0.55327\n",
      "Document N-[4-(4-nitrophenylphospho)butanoyl]-L-alanine - similarity: 0.55325\n",
      "Document Thiocamphor - similarity: 0.55322\n",
      "Document Dalotuzumab - similarity: 0.55317\n",
      "Document Monensin - similarity: 0.55315\n",
      "Document Ascorbic acid - similarity: 0.55315\n",
      "Document Streptococcus pneumoniae type 3 capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.55314\n",
      "Document Glimepiride - similarity: 0.55314\n",
      "Document Relamorelin - similarity: 0.55294\n",
      "Document Salmon, unspecified - similarity: 0.55290\n",
      "Document Flutamide - similarity: 0.55274\n",
      "Document 5-Chloryl-2,4,6-Quinazolinetriamine - similarity: 0.55267\n",
      "Document Difloxacin - similarity: 0.55263\n",
      "Document Molsidomine - similarity: 0.55255\n",
      "Document Toremifene - similarity: 0.55250\n",
      "Document Iophendylate - similarity: 0.55240\n",
      "Document Taladegib - similarity: 0.55240\n",
      "Document Human papillomavirus type 18 L1 capsid protein antigen - similarity: 0.55225\n",
      "Document Ioxilan - similarity: 0.55223\n",
      "Document Squalamine - similarity: 0.55222\n",
      "Document Aminomethylbenzoic acid - similarity: 0.55218\n",
      "Document Ridinilazole - similarity: 0.55201\n",
      "Document CHROMOPHORE (ASP-TYR-GLY) - similarity: 0.55198\n",
      "Document 7,8-Diamino-Nonanoic Acid - similarity: 0.55198\n",
      "Document Diiodohydroxyquinoline - similarity: 0.55192\n",
      "Document Glecaprevir - similarity: 0.55183\n",
      "Document 2,6-Diamino-(S)-9-[2-(Phosphonomethoxy)Propyl]Purine - similarity: 0.55182\n",
      "Document Paraoxon - similarity: 0.55178\n",
      "Document Terbutaline - similarity: 0.55178\n",
      "Document Mannosulfan - similarity: 0.55176\n",
      "Document Cresol - similarity: 0.55170\n",
      "Document Tetryzoline - similarity: 0.55165\n",
      "Document Carisoprodol - similarity: 0.55159\n",
      "Document Olaparib - similarity: 0.55159\n",
      "Document Iohexol - similarity: 0.55155\n",
      "Document Musca domestica - similarity: 0.55155\n",
      "Document Zinc ascorbate - similarity: 0.55146\n",
      "Document Cmp-2-Keto-3-Deoxy-Octulosonic Acid - similarity: 0.55129\n",
      "Document Dorsomorphin - similarity: 0.55121\n",
      "Document Ethyl 8-hydroxy-alpha-(2-propynylamino)-5-quinolinepropanoate - similarity: 0.55119\n",
      "Document Nigericin - similarity: 0.55119\n",
      "Document Potassium Iodide - similarity: 0.55117\n",
      "Document Anacetrapib - similarity: 0.55106\n",
      "Document Remestemcel-L - similarity: 0.55106\n",
      "Document Sulfacytine - similarity: 0.55104\n",
      "Document Halometasone - similarity: 0.55103\n",
      "Document ABT-560 - similarity: 0.55103\n",
      "Document ABT-751 - similarity: 0.55102\n",
      "Document ADX10059 - similarity: 0.55101\n",
      "Document Smallpox and Monkeypox Vaccine, Live, Non-replicating - similarity: 0.55095\n",
      "Document tert-butyl [(2-amino-4-oxo-4,7-dihydro-3H-pyrrolo[2,3-d]pyrimidin-5-yl)methyl]carbamate - similarity: 0.55095\n",
      "Document 3-{2,6,8-trioxo-9-[(2S,3R,4R)-2,3,4,5-tetrahydroxypentyl]-1,2,3,6,8,9-hexahydro-7H-purin-7-Yl}propyl dihydrogen phosphate - similarity: 0.55093\n",
      "Document Edratide - similarity: 0.55083\n",
      "Document Amifostine - similarity: 0.55082\n",
      "Document Etiocholanedione - similarity: 0.55081\n",
      "Document Bifonazole - similarity: 0.55069\n",
      "Document Epacadostat - similarity: 0.55061\n",
      "Document PD150606 - similarity: 0.55059\n",
      "Document Inonotus obliquus fruiting body - similarity: 0.55059\n",
      "Document Adenosine-5'-diphosphate-2',3'-vanadate - similarity: 0.55058\n",
      "Document Apricoxib - similarity: 0.55055\n",
      "Document Ambenonium - similarity: 0.55052\n",
      "Document Dipyrithione - similarity: 0.55046\n",
      "Document Vardenafil - similarity: 0.55041\n",
      "Document Spironolactone - similarity: 0.55040\n",
      "Document N~3~-(3-PYRIDIN-3-YLBENZYL)PYRIDINE-2,3-DIAMINE - similarity: 0.55040\n",
      "Document Dalbavancin - similarity: 0.55032\n",
      "Document Danoprevir - similarity: 0.55029\n",
      "Document Aspergillus terreus - similarity: 0.55020\n",
      "Document Terguride - similarity: 0.55015\n",
      "Document Juglans regia pollen - similarity: 0.55012\n",
      "Document Venetoclax - similarity: 0.55012\n",
      "Document Quisqualic acid - similarity: 0.55006\n",
      "Document Fig - similarity: 0.55000\n",
      "Document KP-1461 - similarity: 0.54998\n",
      "Document Norfenefrine - similarity: 0.54990\n",
      "Document 12-Phenylheme - similarity: 0.54988\n",
      "Document Sodium oxybate - similarity: 0.54987\n",
      "Document Cyanocobalamin - similarity: 0.54985\n",
      "Document Clostridium tetani toxoid antigen (formaldehyde inactivated) - similarity: 0.54980\n",
      "Document Ganitumab - similarity: 0.54980\n",
      "Document Heptaminol - similarity: 0.54972\n",
      "Document Tepotinib - similarity: 0.54971\n",
      "Document Hematin - similarity: 0.54968\n",
      "Document Reboxetine - similarity: 0.54967\n",
      "Document Ursadiol - similarity: 0.54967\n",
      "Document N-(2-hydroxy-1,1-dimethylethyl)-1-methyl-3-(1H-pyrrolo[2,3-b]pyridin-2-yl)-1H-indole-5-carboxamide - similarity: 0.54961\n",
      "Document 1-[(2S)-4-(5-BROMO-1H-PYRAZOLO[3,4-B]PYRIDIN-4-YL)MORPHOLIN-2-YL]METHANAMINE - similarity: 0.54957\n",
      "Document Onvansertib - similarity: 0.54957\n",
      "Document ABT-869 - similarity: 0.54957\n",
      "Document Amylose - similarity: 0.54956\n",
      "Document Asparagine - similarity: 0.54948\n",
      "Document Glycyl-L-alpha-amino-epsilon-pimelyl-D-alanine - similarity: 0.54935\n",
      "Document Uridine monophosphate - similarity: 0.54931\n",
      "Document 6-amino-2-methyl-1,7-dihydro-8H-imidazo[4,5-g]quinazolin-8-one - similarity: 0.54922\n",
      "Document Henagliflozin - similarity: 0.54915\n",
      "Document Anthrax immune globulin human - similarity: 0.54912\n",
      "Document Methsuximide - similarity: 0.54899\n",
      "Document TAK-475 - similarity: 0.54898\n",
      "Document Olmesartan - similarity: 0.54897\n",
      "Document Evogliptin - similarity: 0.54894\n",
      "Document Geotrichum candidum - similarity: 0.54885\n",
      "Document Dabigatran - similarity: 0.54882\n",
      "Document 1-Aminocyclopropylphosphonate - similarity: 0.54876\n",
      "Document Iron - similarity: 0.54875\n",
      "Document NM-702 - similarity: 0.54870\n",
      "Document Pegloticase - similarity: 0.54870\n",
      "Document Insulin aspart - similarity: 0.54865\n",
      "Document Ergocalciferol - similarity: 0.54857\n",
      "Document Cevipabulin - similarity: 0.54853\n",
      "Document Tidiacic arginine - similarity: 0.54840\n",
      "Document 1-(1,4-dimethyl-1,2,3,4-tetrahydroquinoxalin-6-yl)methanamine - similarity: 0.54833\n",
      "Document 5-[5,6-BIS(METHYLOXY)-1H-BENZIMIDAZOL-1-YL]-3-{[1-(2-CHLOROPHENYL)ETHYL]OXY}-2-THIOPHENECARBOXAMIDE - similarity: 0.54832\n",
      "Document LTX-109 - similarity: 0.54828\n",
      "Document Modified vaccinia ankara - similarity: 0.54827\n",
      "Document Tiadenol - similarity: 0.54819\n",
      "Document N-Hydroxy-4-(Methyl{[5-(2-Pyridinyl)-2-Thienyl]Sulfonyl}Amino)Benzamide - similarity: 0.54817\n",
      "Document N-{3-[5-(1H-1,2,4-triazol-3-yl)-1H-indazol-3-yl]phenyl}furan-2-carboxamide - similarity: 0.54798\n",
      "Document SU9516 - similarity: 0.54796\n",
      "Document N-(3-AMINOPROPYL)-2-NITROBENZENAMINE - similarity: 0.54794\n",
      "Document Cyclopentamine - similarity: 0.54792\n",
      "Document Bupropion - similarity: 0.54792\n",
      "Document Pidolic acid - similarity: 0.54788\n",
      "Document Quinapril - similarity: 0.54786\n",
      "Document 5,7,2′-trihydroxy-6,8-dimethoxyflavone - similarity: 0.54776\n",
      "Document Ustilago cynodontis - similarity: 0.54775\n",
      "Document Phenindione - similarity: 0.54766\n",
      "Document Iopamidol - similarity: 0.54757\n",
      "Document Griseofulvin - similarity: 0.54751\n",
      "Document Trovafloxacin - similarity: 0.54741\n",
      "Document 4-(2-methoxyethoxy)-6-methylpyrimidin-2-amine - similarity: 0.54737\n",
      "Document Zolimidine - similarity: 0.54734\n",
      "Document Alendronic acid - similarity: 0.54732\n",
      "Document Cyclohexanone - similarity: 0.54732\n",
      "Document Canfosfamide - similarity: 0.54725\n",
      "Document Flupirtine - similarity: 0.54724\n",
      "Document Oxfendazole - similarity: 0.54723\n",
      "Document MK-8666 - similarity: 0.54717\n",
      "Document Bimosiamose - similarity: 0.54713\n",
      "Document Calcium lactate gluconate - similarity: 0.54711\n",
      "Document Typhoid vaccine - similarity: 0.54696\n",
      "Document Dinoprost - similarity: 0.54691\n",
      "Document Pyrazole - similarity: 0.54690\n",
      "Document Etilevodopa - similarity: 0.54688\n",
      "Document 4-Methyl-Pyrroline-5-Carboxylic Acid - similarity: 0.54687\n",
      "Document Egg white - similarity: 0.54678\n",
      "Document RH-1 - similarity: 0.54676\n",
      "Document Influenza A virus A/Singapore/GP1908/2015 IVR-180 (H1N1) antigen (UV, formaldehyde inactivated) - similarity: 0.54675\n",
      "Document Policresulen - similarity: 0.54673\n",
      "Document Trolamine - similarity: 0.54669\n",
      "Document Cilazapril - similarity: 0.54659\n",
      "Document Tonabersat - similarity: 0.54658\n",
      "Document Amitriptyline - similarity: 0.54655\n",
      "Document 2-(1H-imidazol-1-yl)-9-methoxy-8-(2-methoxyethoxy)benzo[c][2,7]naphthyridin-4-amine - similarity: 0.54655\n",
      "Document Triphendiol - similarity: 0.54651\n",
      "Document Azadirachta indica leaf - similarity: 0.54644\n",
      "Document Dasabuvir - similarity: 0.54628\n",
      "Document Lactuca sativa whole - similarity: 0.54627\n",
      "Document Ertapenem - similarity: 0.54618\n",
      "Document Terlipressin - similarity: 0.54615\n",
      "Document Ustilago tritici - similarity: 0.54614\n",
      "Document N-({4-[(1R)-4-[(2R,4S,5S)-2,4-diamino-6-oxohexahydropyrimidin-5-yl]-1-(2,2,2-trifluoro-1,1-dihydroxyethyl)butyl]phenyl}carbonyl)-L-glutamic acid - similarity: 0.54609\n",
      "Document 6-O-benzylguanine - similarity: 0.54605\n",
      "Document 4-{5-[(1Z)-1-(2-IMINO-4-OXO-1,3-THIAZOLIDIN-5-YLIDENE)ETHYL]-2-FURYL}BENZENESULFONAMIDE - similarity: 0.54597\n",
      "Document Triethylene glycol - similarity: 0.54594\n",
      "Document 9-(4-Hydroxyphenyl)-2,7-Phenanthroline - similarity: 0.54579\n",
      "Document Selumetinib - similarity: 0.54578\n",
      "Document Birinapant - similarity: 0.54576\n",
      "Document Atosiban - similarity: 0.54574\n",
      "Document Tricaprylin - similarity: 0.54574\n",
      "Document Indigofera tinctoria leaf - similarity: 0.54573\n",
      "Document Elymus repens pollen - similarity: 0.54573\n",
      "Document Yttrium Y-90 - similarity: 0.54573\n",
      "Document Telotristat ethyl - similarity: 0.54570\n",
      "Document Ethyl chloride - similarity: 0.54566\n",
      "Document (S)-camphor - similarity: 0.54563\n",
      "Document (R)-1-Para-Nitro-Phenyl-2-Azido-Ethanol - similarity: 0.54560\n",
      "Document Nepafenac - similarity: 0.54549\n",
      "Document Eplivanserin - similarity: 0.54548\n",
      "Document Ingenol mebutate - similarity: 0.54547\n",
      "Document (2R)-3-Methyl-1-phenyl-2-butanyl [(2S)-1-oxo-2-hexanyl]carbamate - similarity: 0.54545\n",
      "Document 3-(BUTYLSULPHONYL)-PROPANOIC ACID - similarity: 0.54540\n",
      "Document Histamine - similarity: 0.54538\n",
      "Document Hemoglobin - similarity: 0.54537\n",
      "Document Bb-3497 - similarity: 0.54526\n",
      "Document Galsulfase - similarity: 0.54522\n",
      "Document Seletracetam - similarity: 0.54516\n",
      "Document Cefdinir - similarity: 0.54513\n",
      "Document Loracarbef - similarity: 0.54507\n",
      "Document Mexazolam - similarity: 0.54503\n",
      "Document 5-[(3R)-3-(5-methoxybiphenyl-3-yl)but-1-yn-1-yl]-6-methylpyrimidine-2,4-diamine - similarity: 0.54495\n",
      "Document ALT-801 - similarity: 0.54495\n",
      "Document Uproleselan - similarity: 0.54493\n",
      "Document Vofatamab - similarity: 0.54491\n",
      "Document Oteracil - similarity: 0.54489\n",
      "Document Ipafricept - similarity: 0.54487\n",
      "Document 2-[(2,4-DICHLOROBENZOYL)AMINO]-5-(PYRIMIDIN-2-YLOXY)BENZOIC ACID - similarity: 0.54483\n",
      "Document 1-(5-{2-[(1-methyl-1H-pyrazolo[4,3-d]pyrimidin-7-yl)amino]ethyl}-1,3-thiazol-2-yl)-3-[3-(trifluoromethyl)phenyl]urea - similarity: 0.54483\n",
      "Document Velneperit - similarity: 0.54482\n",
      "Document 4-Phospho-D-erythronohydroxamic acid - similarity: 0.54458\n",
      "Document Volociximab - similarity: 0.54456\n",
      "Document Solanezumab - similarity: 0.54453\n",
      "Document 6-CYCLOHEXYLMETHOXY-2-(3'-CHLOROANILINO) PURINE - similarity: 0.54452\n",
      "Document Ethanol - similarity: 0.54451\n",
      "Document Ambrosia ambrosioides pollen - similarity: 0.54443\n",
      "Document Pimavanserin - similarity: 0.54440\n",
      "Document Tocladesine - similarity: 0.54438\n",
      "Document Artenimol - similarity: 0.54431\n",
      "Document Malathion - similarity: 0.54425\n",
      "Document AZD-8848 - similarity: 0.54424\n",
      "Document BMS-986020 - similarity: 0.54423\n",
      "Document Cetalkonium - similarity: 0.54415\n",
      "Document Alanine - similarity: 0.54415\n",
      "Document N-Allyl-6-{[3-(4-bromophenyl)-1-methyl-1H-indazol-6-yl]oxy}-N-methyl-1-hexanamine - similarity: 0.54412\n",
      "Document Cisplatin - similarity: 0.54409\n",
      "Document Vanoxerine - similarity: 0.54408\n",
      "Document Indoximod - similarity: 0.54407\n",
      "Document Acorus calamus root - similarity: 0.54398\n",
      "Document 2'-Monophosphoadenosine-5'-Diphosphate - similarity: 0.54397\n",
      "Document Anti-inhibitor coagulant complex - similarity: 0.54395\n",
      "Document Propentofylline - similarity: 0.54394\n",
      "Document Trichloroethylene - similarity: 0.54391\n",
      "Document N-(6-{[3-(4-Bromophenyl)-1,2-Benzisothiazol-6-Yl]Oxy}Hexyl)-N-Methylprop-2-En-1-Amine - similarity: 0.54391\n",
      "Document CCX915 - similarity: 0.54389\n",
      "Document N-[[3-FLUORO-4-ETHOXY-PYRID-2-YL]ETHYL]-N'-[5-NITRILOMETHYL-PYRIDYL]-THIOUREA - similarity: 0.54389\n",
      "Document Date - similarity: 0.54381\n",
      "Document Ammonia N-13 - similarity: 0.54378\n",
      "Document Indomethacin - similarity: 0.54361\n",
      "Document Sofosbuvir - similarity: 0.54360\n",
      "Document Interferon beta-1b - similarity: 0.54354\n",
      "Document Desvancosaminyl vancomycin - similarity: 0.54346\n",
      "Document Octreotide - similarity: 0.54341\n",
      "Document Xenon - similarity: 0.54338\n",
      "Document Ginsenosides - similarity: 0.54336\n",
      "Document Daprodustat - similarity: 0.54333\n",
      "Document Human immunoglobulin G - similarity: 0.54332\n",
      "Document Methicone (20 CST) - similarity: 0.54329\n",
      "Document Dermatophagoides pteronyssinus - similarity: 0.54324\n",
      "Document Deferiprone - similarity: 0.54322\n",
      "Document N-(2,6-dimethylphenyl)-5-phenylimidazo[1,5-a]pyrazin-8-amine - similarity: 0.54320\n",
      "Document Black sea bass - similarity: 0.54310\n",
      "Document Moxestrol - similarity: 0.54308\n",
      "Document 2-ACETYLAMINO-4-METHYL-PENTANOIC ACID (1-FORMYL-2-PHENYL-ETHYL)-AMIDE - similarity: 0.54302\n",
      "Document Bleomycin A6 - similarity: 0.54299\n",
      "Document Dendryphiella vinosa - similarity: 0.54298\n",
      "Document Epirizole - similarity: 0.54297\n",
      "Document Carfentanil, C-11 - similarity: 0.54293\n",
      "Document Radalbuvir - similarity: 0.54289\n",
      "Document 6-amino-2-[(2-morpholin-4-ylethyl)amino]-3,7-dihydro-8H-imidazo[4,5-g]quinazolin-8-one - similarity: 0.54274\n",
      "Document [(4S)-2,2-dimethyl-1,3-dioxolan-4-yl]methyl hydrogen hex-5-enylphosphonate - similarity: 0.54271\n",
      "Document Altretamine - similarity: 0.54270\n",
      "Document Pirodavir - similarity: 0.54265\n",
      "Document Tranexamic acid - similarity: 0.54257\n",
      "Document ELX-02 - similarity: 0.54253\n",
      "Document Secale cereale pollen - similarity: 0.54252\n",
      "Document Methyl (2S)-2-[[2-[(3R)-1-carbamimidoylpiperidin-3-yl]acetyl]amino]-3-[4-(2-phenylethynyl)phenyl]propanoate - similarity: 0.54246\n",
      "Document Econazole - similarity: 0.54243\n",
      "Document Panax ginseng root oil - similarity: 0.54239\n",
      "Document (1S,4R,9S)-5-(trifluoromethyl)-1,2,3,4-tetrahydro-1,4-methanonaphthalen-9-amine - similarity: 0.54237\n",
      "Document Metisazone - similarity: 0.54234\n",
      "Document Selenazole-4-carboxyamide-adenine dinucleotide - similarity: 0.54217\n",
      "Document Tesetaxel - similarity: 0.54216\n",
      "Document Ginger - similarity: 0.54216\n",
      "Document TRANS-(1S,2S)-2-AMINO-1,2,3,4-TETRAHYDRONAPHTHALEN-1-OL - similarity: 0.54215\n",
      "Document CHIR-124 - similarity: 0.54214\n",
      "Document 4-Aminohydrocinnamic Acid - similarity: 0.54214\n",
      "Document LY-2811376 - similarity: 0.54213\n",
      "Document Vercirnon - similarity: 0.54209\n",
      "Document Resiniferatoxin - similarity: 0.54208\n",
      "Document Phenyltoloxamine - similarity: 0.54205\n",
      "Document Omalizumab - similarity: 0.54203\n",
      "Document Technetium Tc-99m nofetumomab merpentan - similarity: 0.54200\n",
      "Document Isatuximab - similarity: 0.54199\n",
      "Document Cisapride - similarity: 0.54199\n",
      "Document Dexniguldipine - similarity: 0.54199\n",
      "Document Coffea arabica seed, roasted - similarity: 0.54199\n",
      "Document Remoxipride - similarity: 0.54187\n",
      "Document mRNA-1273 - similarity: 0.54177\n",
      "Document IDD-3 - similarity: 0.54176\n",
      "Document Platensimycin - similarity: 0.54173\n",
      "Document Alcloxa - similarity: 0.54172\n",
      "Document Eprazinone - similarity: 0.54170\n",
      "Document 9-Beta-D-Xylofuranosyl-Adenine - similarity: 0.54165\n",
      "Document 5'-O-(N-Ethyl-Sulfamoyl)Adenosine - similarity: 0.54160\n",
      "Document tert-butyl 4-(3-thiophen-2-yl-1,2,4-oxadiazol-5-yl)piperidine-1-carboxylate - similarity: 0.54155\n",
      "Document Zinc cation - similarity: 0.54147\n",
      "Document Emodepside - similarity: 0.54144\n",
      "Document Ulocuplumab - similarity: 0.54144\n",
      "Document Fluciclovine - similarity: 0.54133\n",
      "Document Capmatinib - similarity: 0.54132\n",
      "Document Talsupram - similarity: 0.54121\n",
      "Document Trenonacog alfa - similarity: 0.54120\n",
      "Document GDP-alpha-D-mannuronic acid - similarity: 0.54110\n",
      "Document Fesoterodine - similarity: 0.54109\n",
      "Document Strontium chloride - similarity: 0.54106\n",
      "Document beta-Isosparteine - similarity: 0.54098\n",
      "Document (20S)-19,20,22,23-TETRAHYDRO-19-OXO-5H,21H-18,20-ETHANO-12,14-ETHENO-6,10-METHENOBENZ[D]IMIDAZO[4,3-L][1,6,9,13]OXATRIAZACYCLONOADECOSINE-9-CARBONITRILE - similarity: 0.54089\n",
      "Document Dacarbazine - similarity: 0.54089\n",
      "Document Mitotane - similarity: 0.54074\n",
      "Document Gemigliptin - similarity: 0.54066\n",
      "Document Antroquinonol - similarity: 0.54063\n",
      "Document Sodium stibogluconate - similarity: 0.54061\n",
      "Document Hydrogen fluoride - similarity: 0.54052\n",
      "Document MN-305 - similarity: 0.54050\n",
      "Document Tetraiodothyroacetic acid - similarity: 0.54045\n",
      "Document Larotrectinib - similarity: 0.54044\n",
      "Document Morpholine-4-Carboxylic Acid [1s-(2-Benzyloxy-1r-Cyano-Ethylcarbamoyl)-3-Methyl-Butyl]Amide - similarity: 0.54043\n",
      "Document (5E)-5-[(2,2-DIFLUORO-1,3-BENZODIOXOL-5-YL)METHYLENE]-1,3-THIAZOLIDINE-2,4-DIONE - similarity: 0.54041\n",
      "Document Nadide - similarity: 0.54040\n",
      "Document Tenivastatin - similarity: 0.54032\n",
      "Document Enalapril - similarity: 0.54031\n",
      "Document Triflupromazine - similarity: 0.54029\n",
      "Document Tucatinib - similarity: 0.54025\n",
      "Document Chlorine Dioxide - similarity: 0.54022\n",
      "Document Sapropterin - similarity: 0.54018\n",
      "Document Norgestrel - similarity: 0.54017\n",
      "Document Mercuric chloride - similarity: 0.54013\n",
      "Document GDC-0084 - similarity: 0.53995\n",
      "Document Carboxin - similarity: 0.53989\n",
      "Document Topiramate - similarity: 0.53987\n",
      "Document Apadenoson - similarity: 0.53982\n",
      "Document Ezutromid - similarity: 0.53976\n",
      "Document ABT-639 - similarity: 0.53971\n",
      "Document 2-(4-(2-((3-(5-(PYRIDIN-2-YLTHIO)THIAZOL-2-YL)UREIDO)METHYL)-1H-IMIDAZOL-4-YL)PHENOXY)ACETIC ACID - similarity: 0.53969\n",
      "Document Pyridoxal - similarity: 0.53959\n",
      "Document 5-Mercapto-2-Nitro-Benzoic Acid - similarity: 0.53959\n",
      "Document KB001 - similarity: 0.53952\n",
      "Document Veliparib - similarity: 0.53950\n",
      "Document Influenza B virus B/Brisbane/60/2008 antigen (propiolactone inactivated) - similarity: 0.53949\n",
      "Document Fenebrutinib - similarity: 0.53949\n",
      "Document 1-(4-thiophen-2-ylphenyl)methanamine - similarity: 0.53944\n",
      "Document Red snapper - similarity: 0.53944\n",
      "Document Aclidinium - similarity: 0.53942\n",
      "Document Parachlorophenol - similarity: 0.53932\n",
      "Document Ethylenediamine - similarity: 0.53930\n",
      "Document 2-Methyl-2-propanyl [(1S)-2-methyl-1-(1,3,4-oxadiazol-2-yl)propyl]carbamate - similarity: 0.53926\n",
      "Document S-Acetonylcysteine - similarity: 0.53920\n",
      "Document Bictegravir - similarity: 0.53918\n",
      "Document Enarodustat - similarity: 0.53910\n",
      "Document N-[3-[4-(3-aminopropyl)piperazin-1-yl]propyl]-3-[(2-thiophen-2-ylacetyl)amino]-5-[(2R,3R,4S,5R,6R)-3,4,5-trihydroxy-6-(hydroxymethyl)oxan-2-yl]oxybenzamide - similarity: 0.53908\n",
      "Document Zucapsaicin - similarity: 0.53904\n",
      "Document Selitrectinib - similarity: 0.53900\n",
      "Document Tapentadol - similarity: 0.53895\n",
      "Document Nizofenone - similarity: 0.53888\n",
      "Document Coral snake (micrurus fulvius) immune globulin antivenin (equine) - similarity: 0.53879\n",
      "Document Insulin pork - similarity: 0.53866\n",
      "Document Fluphenazine - similarity: 0.53862\n",
      "Document (5Z)-5-[(5-ethylfuran-2-yl)methylidene]-2-[[(S)-(4-fluorophenyl)-(2H-tetrazol-5-yl)methyl]amino]-1,3-thiazol-4-one - similarity: 0.53857\n",
      "Document Atezolizumab - similarity: 0.53856\n",
      "Document Cellotetraose - similarity: 0.53854\n",
      "Document (3S,5R,7R,8S,9S,10R)-7-(hydroxymethyl)-3-(4-methylphenyl)-1,6-dioxa-2-azaspiro[4.5]decane-8,9,10-triol - similarity: 0.53854\n",
      "Document Quazepam - similarity: 0.53852\n",
      "Document Semagacestat - similarity: 0.53851\n",
      "Document Valganciclovir - similarity: 0.53850\n",
      "Document Vosaroxin - similarity: 0.53848\n",
      "Document Strontium ranelate - similarity: 0.53844\n",
      "Document Cabiralizumab - similarity: 0.53843\n",
      "Document Curvularia inaequalis - similarity: 0.53839\n",
      "Document AV-101 - similarity: 0.53834\n",
      "Document Zalypsis - similarity: 0.53833\n",
      "Document SC45647 - similarity: 0.53828\n",
      "Document Tocilizumab - similarity: 0.53816\n",
      "Document Topotecan - similarity: 0.53814\n",
      "Document Botulinum Toxin Type B - similarity: 0.53812\n",
      "Document 7-Nitroindazole - similarity: 0.53802\n",
      "Document Ixabepilone - similarity: 0.53783\n",
      "Document N-Methylphenylalanyl-N-[(trans-4-aminocyclohexyl)methyl]-L-prolinamide - similarity: 0.53776\n",
      "Document HX-1171 - similarity: 0.53773\n",
      "Document Ondansetron - similarity: 0.53770\n",
      "Document 3-OXO-OCTANOIC ACID (2-OXO-TETRAHYDRO-FURAN-3-YL)-AMIDE - similarity: 0.53758\n",
      "Document 6-AMINO-4-[2-(4-METHOXYPHENYL)ETHYL]-1,7-DIHYDRO-8H-IMIDAZO[4,5-G]QUINAZOLIN-8-ONE - similarity: 0.53743\n",
      "Document Raluridine - similarity: 0.53737\n",
      "Document N-acetylserotonin - similarity: 0.53736\n",
      "Document CDP323 - similarity: 0.53734\n",
      "Document Pyrithione - similarity: 0.53734\n",
      "Document N-[1-Hydroxycarboxyethyl-Carbonyl]Leucylamino-2-Methyl-Butane - similarity: 0.53731\n",
      "Document Influenza A virus A/Brisbane/59/2007 (H1N1) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.53728\n",
      "Document Glucagon - similarity: 0.53726\n",
      "Document Asiatic acid - similarity: 0.53720\n",
      "Document SR12813 - similarity: 0.53717\n",
      "Document Sus scrofa placenta - similarity: 0.53716\n",
      "Document Volanesorsen - similarity: 0.53713\n",
      "Document Pinus banksiana pollen - similarity: 0.53708\n",
      "Document Perillyl alcohol - similarity: 0.53707\n",
      "Document Inotuzumab ozogamicin - similarity: 0.53706\n",
      "Document Emetine - similarity: 0.53706\n",
      "Document Terpin hydrate - similarity: 0.53703\n",
      "Document Omega-3-carboxylic acids - similarity: 0.53702\n",
      "Document Clonidine - similarity: 0.53699\n",
      "Document Danazol - similarity: 0.53698\n",
      "Document Histrelin - similarity: 0.53690\n",
      "Document Rilmenidine - similarity: 0.53687\n",
      "Document Evobrutinib - similarity: 0.53686\n",
      "Document Alcuronium - similarity: 0.53674\n",
      "Document Human papillomavirus type 11 L1 capsid protein antigen - similarity: 0.53671\n",
      "Document Difamilast - similarity: 0.53670\n",
      "Document Human papillomavirus type 6 L1 capsid protein antigen - similarity: 0.53668\n",
      "Document Somatropin pegol - similarity: 0.53667\n",
      "Document Ornidazole - similarity: 0.53656\n",
      "Document Elobixibat - similarity: 0.53651\n",
      "Document Benzoylformic Acid - similarity: 0.53649\n",
      "Document Lodenafil carbonate - similarity: 0.53646\n",
      "Document Pisum sativum whole - similarity: 0.53638\n",
      "Document TAK-733 - similarity: 0.53637\n",
      "Document M40403 - similarity: 0.53636\n",
      "Document Aloe vera leaf polysaccharides - similarity: 0.53633\n",
      "Document (5S)-5-Iododihydro-2,4(1H,3H)-pyrimidinedione - similarity: 0.53631\n",
      "Document 1,3,5-trichlorobenzene - similarity: 0.53627\n",
      "Document Phosphate ion - similarity: 0.53617\n",
      "Document Tetrahydrooxazine - similarity: 0.53616\n",
      "Document Fenretinide - similarity: 0.53614\n",
      "Document Human papillomavirus type 16 L1 capsid protein antigen - similarity: 0.53603\n",
      "Document Ubrogepant - similarity: 0.53597\n",
      "Document Oprelvekin - similarity: 0.53590\n",
      "Document Atabecestat - similarity: 0.53586\n",
      "Document Abivertinib - similarity: 0.53575\n",
      "Document Tofisopam - similarity: 0.53568\n",
      "Document Lumiracoxib - similarity: 0.53567\n",
      "Document Haddock - similarity: 0.53566\n",
      "Document Platanus occidentalis pollen - similarity: 0.53561\n",
      "Document Idarubicin - similarity: 0.53556\n",
      "Document Antihemophilic factor human - similarity: 0.53552\n",
      "Document 11-TRANS-13-TRANS-15-CIS-OCTADECATRIENOIC ACID - similarity: 0.53550\n",
      "Document S-(Dimethylarsenic)Cysteine - similarity: 0.53549\n",
      "Document N-benzyl-4-[(2R)-pyrrolidin-2-ylmethoxy]aniline - similarity: 0.53543\n",
      "Document Indoprofen - similarity: 0.53542\n",
      "Document 4-Iodo-D-phenylalanine - similarity: 0.53538\n",
      "Document Abiraterone - similarity: 0.53536\n",
      "Document Sulfaperin - similarity: 0.53529\n",
      "Document Haloprogin - similarity: 0.53529\n",
      "Document Perflutren - similarity: 0.53527\n",
      "Document Sarafloxacin - similarity: 0.53515\n",
      "Document 1,1'-HEXANE-1,6-DIYLBIS(1H-IMIDAZOLE) - similarity: 0.53506\n",
      "Document Azintamide - similarity: 0.53505\n",
      "Document Atorvastatin - similarity: 0.53502\n",
      "Document GI-5005 - similarity: 0.53476\n",
      "Document Tezampanel - similarity: 0.53476\n",
      "Document Tetracycline - similarity: 0.53475\n",
      "Document Isunakinra - similarity: 0.53473\n",
      "Document 5beta-dihydrotestosterone - similarity: 0.53473\n",
      "Document Vonapanitase - similarity: 0.53467\n",
      "Document 2,4-Diamino-5-phenyl-6-ethylpyrimidine - similarity: 0.53459\n",
      "Document Samatasvir - similarity: 0.53449\n",
      "Document 4-Nitrophenyl Phosphate - similarity: 0.53448\n",
      "Document Carvedilol - similarity: 0.53447\n",
      "Document Asunaprevir - similarity: 0.53446\n",
      "Document SJG-136 - similarity: 0.53431\n",
      "Document 4-PHOSPHONOOXY-PHENYL-METHYL-[4-PHOSPHONOOXY]BENZEN - similarity: 0.53430\n",
      "Document Methyl-D9-choline - similarity: 0.53424\n",
      "Document Albumin Aggregated - similarity: 0.53423\n",
      "Document Ilex paraguariensis leaf - similarity: 0.53419\n",
      "Document Fisetin - similarity: 0.53418\n",
      "Document indane-5-sulfonamide - similarity: 0.53418\n",
      "Document Givosiran - similarity: 0.53418\n",
      "Document Beef liver - similarity: 0.53408\n",
      "Document Argan oil - similarity: 0.53402\n",
      "Document Ferrous asparto glycinate - similarity: 0.53401\n",
      "Document AI-128 - similarity: 0.53398\n",
      "Document S-3-(4-FLUOROPHENOXY)-2-HYDROXY-2-METHYL-N-[4-NITRO-3-(TRIFLUOROMETHYL)PHENYL]PROPANAMIDE - similarity: 0.53396\n",
      "Document Curcumin - similarity: 0.53396\n",
      "Document Apricot - similarity: 0.53395\n",
      "Document Methoxamine - similarity: 0.53389\n",
      "Document Sonidegib - similarity: 0.53386\n",
      "Document Nilvadipine - similarity: 0.53383\n",
      "Document Meprotixol - similarity: 0.53377\n",
      "Document Trimethadione - similarity: 0.53361\n",
      "Document Tantalum - similarity: 0.53356\n",
      "Document Enobosarm - similarity: 0.53354\n",
      "Document Amaranthus tuberculatus pollen - similarity: 0.53351\n",
      "Document N-Pyridoxyl-2-Methylalanine-5-Phosphate - similarity: 0.53350\n",
      "Document Visilizumab - similarity: 0.53346\n",
      "Document Ginsenoside Rb1 - similarity: 0.53345\n",
      "Document Nabumetone - similarity: 0.53341\n",
      "Document CI-1040 - similarity: 0.53340\n",
      "Document Autologous cultured chondrocytes - similarity: 0.53339\n",
      "Document Testosterone cypionate - similarity: 0.53335\n",
      "Document Benidipine - similarity: 0.53333\n",
      "Document Moxidectin - similarity: 0.53332\n",
      "Document Caprylyl glycol - similarity: 0.53329\n",
      "Document Succinylcholine - similarity: 0.53322\n",
      "Document Butyrfentanyl - similarity: 0.53322\n",
      "Document Fostemsavir - similarity: 0.53318\n",
      "Document Alosetron - similarity: 0.53316\n",
      "Document Azelnidipine - similarity: 0.53311\n",
      "Document Aspartame - similarity: 0.53308\n",
      "Document Aspergillus fumigatus - similarity: 0.53307\n",
      "Document Diethylcarbamazine - similarity: 0.53302\n",
      "Document Tozuleristide - similarity: 0.53289\n",
      "Document Gaboxadol - similarity: 0.53272\n",
      "Document RO-4584820 - similarity: 0.53272\n",
      "Document Letosteine - similarity: 0.53266\n",
      "Document 3-Hydroxypropyl 3-[(7-carbamimidoyl-1-naphthyl)carbamoyl]benzenesulfonate - similarity: 0.53266\n",
      "Document 7-methyl-GpppA - similarity: 0.53265\n",
      "Document Lobster - similarity: 0.53258\n",
      "Document Atriplex polycarpa pollen - similarity: 0.53255\n",
      "Document Aldosterone - similarity: 0.53254\n",
      "Document Nonacog beta pegol - similarity: 0.53246\n",
      "Document Blisibimod - similarity: 0.53243\n",
      "Document Cronidipine - similarity: 0.53243\n",
      "Document GS-9256 - similarity: 0.53239\n",
      "Document Lofexidine - similarity: 0.53237\n",
      "Document Quingestanol - similarity: 0.53236\n",
      "Document Inositol nicotinate - similarity: 0.53233\n",
      "Document Leflunomide - similarity: 0.53227\n",
      "Document Tolazamide - similarity: 0.53211\n",
      "Document 2-Anhydro-3-Fluoro-Quinic Acid - similarity: 0.53210\n",
      "Document 6'-Methyl-Thiamin Diphosphate - similarity: 0.53205\n",
      "Document Silibinin - similarity: 0.53195\n",
      "Document Anthralin - similarity: 0.53189\n",
      "Document EHT 0202 - similarity: 0.53188\n",
      "Document Arabinose-5-phosphate - similarity: 0.53180\n",
      "Document Dextranomer - similarity: 0.53175\n",
      "Document Ivabradine - similarity: 0.53171\n",
      "Document Ritiometan - similarity: 0.53168\n",
      "Document Maltose - similarity: 0.53168\n",
      "Document Glycocyamine - similarity: 0.53152\n",
      "Document Amlodipine - similarity: 0.53148\n",
      "Document [4-({4-[(5-cyclopropyl-1H-pyrazol-3-yl)amino]quinazolin-2-yl}amino)phenyl]acetonitrile - similarity: 0.53144\n",
      "Document Squash - similarity: 0.53141\n",
      "Document ICA-105665 - similarity: 0.53140\n",
      "Document Estradiol valerate - similarity: 0.53126\n",
      "Document Iobenzamic acid - similarity: 0.53126\n",
      "Document Synthetic Conjugated Estrogens, A - similarity: 0.53125\n",
      "Document alpha-D-Fucopyranose - similarity: 0.53123\n",
      "Document Acetylphosphate - similarity: 0.53116\n",
      "Document Cryptotanshinone - similarity: 0.53115\n",
      "Document Schinus molle pollen - similarity: 0.53115\n",
      "Document Pittsburgh Compound B - similarity: 0.53110\n",
      "Document Cilomilast - similarity: 0.53107\n",
      "Document White catfish - similarity: 0.53097\n",
      "Document Metaraminol - similarity: 0.53094\n",
      "Document Metandienone - similarity: 0.53085\n",
      "Document Fentanyl - similarity: 0.53078\n",
      "Document Profoxydim - similarity: 0.53073\n",
      "Document IR208 - similarity: 0.53072\n",
      "Document N1,N2-ETHYLENE-2-METHYLAMINO-4,5,6,7-TETRABROMO-BENZIMIDAZOLE - similarity: 0.53057\n",
      "Document Clotrimazole - similarity: 0.53056\n",
      "Document Baloxavir marboxil - similarity: 0.53048\n",
      "Document Vidupiprant - similarity: 0.53042\n",
      "Document Clonostachys rosea f. rosea - similarity: 0.53032\n",
      "Document Rotigotine - similarity: 0.53031\n",
      "Document Sirolimus - similarity: 0.53031\n",
      "Document Streptococcus pneumoniae type 4 capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.53019\n",
      "Document Valdecoxib - similarity: 0.53019\n",
      "Document Tropinone - similarity: 0.53014\n",
      "Document 2-tert-butyl-9-fluoro-1,6-dihydrobenzo[h]imidazo[4,5-f]isoquinolin-7-one - similarity: 0.53011\n",
      "Document PZ-128 - similarity: 0.52993\n",
      "Document Ceftezole - similarity: 0.52992\n",
      "Document MX6 - similarity: 0.52976\n",
      "Document beta-D-Galactopyranuronic acid - similarity: 0.52963\n",
      "Document Ferrous tartrate - similarity: 0.52952\n",
      "Document Fostamatinib - similarity: 0.52950\n",
      "Document Etidocaine - similarity: 0.52946\n",
      "Document Recoflavone - similarity: 0.52942\n",
      "Document Megestrol acetate - similarity: 0.52939\n",
      "Document Pretomanid - similarity: 0.52938\n",
      "Document Levosalbutamol - similarity: 0.52935\n",
      "Document Purvalanol - similarity: 0.52925\n",
      "Document methyl beta-D-glucopyranoside - similarity: 0.52920\n",
      "Document Entinostat - similarity: 0.52915\n",
      "Document Irsogladine - similarity: 0.52914\n",
      "Document Tioconazole - similarity: 0.52911\n",
      "Document Metergoline - similarity: 0.52909\n",
      "Document Pivagabine - similarity: 0.52902\n",
      "Document Diazoxide - similarity: 0.52899\n",
      "Document Oprozomib - similarity: 0.52891\n",
      "Document Veal - similarity: 0.52890\n",
      "Document CX717 - similarity: 0.52880\n",
      "Document (1E)-5-(1-piperidin-4-yl-3-pyridin-4-yl-1H-pyrazol-4-yl)-2,3-dihydro-1H-inden-1-one oxime - similarity: 0.52876\n",
      "Document Salix alba bark volatile oil - similarity: 0.52873\n",
      "Document Tasquinimod - similarity: 0.52870\n",
      "Document Aracytidine 5'-monophosphate - similarity: 0.52869\n",
      "Document WRR-204 - similarity: 0.52863\n",
      "Document TT-301 - similarity: 0.52861\n",
      "Document Vismodegib - similarity: 0.52855\n",
      "Document S-Adenosyl-L-Homoselenocysteine - similarity: 0.52855\n",
      "Document O6-(R)-ROSCOVITINE, R-2-(6-BENZYLOXY-9-ISOPROPYL-9H-PURIN-2-YLAMINO)-BUTAN-1-OL - similarity: 0.52852\n",
      "Document Povidone - similarity: 0.52848\n",
      "Document Sennosides - similarity: 0.52847\n",
      "Document Creatinolfosfate - similarity: 0.52839\n",
      "Document threo-3-methyl-L-aspartic acid - similarity: 0.52835\n",
      "Document Imazaquin - similarity: 0.52832\n",
      "Document Aluminum hydroxide - similarity: 0.52828\n",
      "Document Shikimate-3-Phosphate - similarity: 0.52822\n",
      "Document Pseudotsuga menziesii pollen - similarity: 0.52821\n",
      "Document Hesperidin - similarity: 0.52819\n",
      "Document (4aS)-5-[(2,4-diaminopteridin-6-yl)methyl]-4a,5-dihydro-2H-dibenzo[b,f]azepin-8-ol - similarity: 0.52815\n",
      "Document Rocaglamide - similarity: 0.52811\n",
      "Document N-cyclopropyl-3-{[1-(2,4-difluorophenyl)-7-methyl-6-oxo-6,7-dihydro-1H-pyrazolo[3,4-b]pyridin-4-yl]amino}-4-methylbenzamide - similarity: 0.52803\n",
      "Document Concizumab - similarity: 0.52799\n",
      "Document Tezosentan - similarity: 0.52786\n",
      "Document Relenopride - similarity: 0.52780\n",
      "Document Passalora fulva - similarity: 0.52779\n",
      "Document N-[1-(AMINOMETHYL)CYCLOPROPYL]-3-(BENZYLSULFONYL)-N~2~-[(1S)-2,2,2-TRIFLUORO-1-(4-HYDROXYPHENYL)ETHYL]-L-ALANINAMIDE - similarity: 0.52779\n",
      "Document Ipriflavone - similarity: 0.52777\n",
      "Document Ficlatuzumab - similarity: 0.52777\n",
      "Document Arhalofenate - similarity: 0.52772\n",
      "Document Rofecoxib - similarity: 0.52770\n",
      "Document 4-(quinolin-3-ylmethyl)piperidine-1-carboxylic acid - similarity: 0.52770\n",
      "Document Peginterferon alfa-2b - similarity: 0.52766\n",
      "Document Zicronapine - similarity: 0.52762\n",
      "Document 4-Carboxy-5-(1-Pentyl)Hexylsulfanyl-1,2,3-Triazole - similarity: 0.52761\n",
      "Document Marbofloxacin - similarity: 0.52754\n",
      "Document Pristinamycin - similarity: 0.52750\n",
      "Document Doravirine - similarity: 0.52738\n",
      "Document Tetrasul - similarity: 0.52735\n",
      "Document 4-hydroxybenzyl coenzyme A - similarity: 0.52733\n",
      "Document Clorsulon - similarity: 0.52733\n",
      "Document MBT-0312 - similarity: 0.52719\n",
      "Document 3-Amino-3-Oxopropanoic Acid - similarity: 0.52718\n",
      "Document Aminoanthracene - similarity: 0.52712\n",
      "Document Mitobronitol - similarity: 0.52705\n",
      "Document Figopitant - similarity: 0.52701\n",
      "Document Influenza A virus A/Victoria/210/2009 X-187 (H3N2) antigen (UV, formaldehyde inactivated) - similarity: 0.52699\n",
      "Document Etofibrate - similarity: 0.52698\n",
      "Document Regrelor - similarity: 0.52697\n",
      "Document Aluminum oxide - similarity: 0.52697\n",
      "Document Aluminium glycinate - similarity: 0.52695\n",
      "Document Acetylleucine - similarity: 0.52694\n",
      "Document Ocrelizumab - similarity: 0.52690\n",
      "Document Influenza A virus A/Christchurch/16/2010 NIB-74 (H1N1) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.52689\n",
      "Document Hachimycin - similarity: 0.52684\n",
      "Document 1-Hexadecanosulfonic Acid - similarity: 0.52671\n",
      "Document Albendazole - similarity: 0.52669\n",
      "Document Filanesib - similarity: 0.52662\n",
      "Document Amibegron - similarity: 0.52657\n",
      "Document Levomenthol - similarity: 0.52657\n",
      "Document Fenoverine - similarity: 0.52655\n",
      "Document Ripasudil - similarity: 0.52653\n",
      "Document Itacitinib - similarity: 0.52652\n",
      "Document Red bell pepper - similarity: 0.52651\n",
      "Document Bafilomycin A1 - similarity: 0.52651\n",
      "Document Bretylium - similarity: 0.52648\n",
      "Document Flutemetamol (18F) - similarity: 0.52644\n",
      "Document Niceritrol - similarity: 0.52642\n",
      "Document Axelopran - similarity: 0.52639\n",
      "Document Meclofenoxate - similarity: 0.52635\n",
      "Document Antithrombin III human - similarity: 0.52634\n",
      "Document Afuresertib - similarity: 0.52633\n",
      "Document Demexiptiline - similarity: 0.52630\n",
      "Document Succinylsulfathiazole - similarity: 0.52617\n",
      "Document Recombinant CD40-ligand - similarity: 0.52601\n",
      "Document BMS-919373 - similarity: 0.52596\n",
      "Document MP4 - similarity: 0.52586\n",
      "Document Black widow spider antivenin (equine) - similarity: 0.52585\n",
      "Document (E)-(S)-4-[(S)-4-METHYL-2-((S)-3-METHYL-2{(S)-2-[(5-METHYL-ISOXAZOLE-3- CARBONYL)-AMINO]-PROPIONYLAMINO}-BUTYRYLAMINO)-PENTANOYLAMINO]-5-((S)-2- OXO-PYRROLIDIN-3-YL)-PENT-2-ENOIC ACID ETHYL ESTER - similarity: 0.52585\n",
      "Document Benfluorex - similarity: 0.52578\n",
      "Document 3-[3-chloro-5-(5-{[(1S)-1-phenylethyl]amino}isoxazolo[5,4-c]pyridin-3-yl)phenyl]propan-1-ol - similarity: 0.52576\n",
      "Document Aluminum zirconium pentachlorohydrex gly - similarity: 0.52572\n",
      "Document Fenozolone - similarity: 0.52568\n",
      "Document OSI-461 - similarity: 0.52560\n",
      "Document Selurampanel - similarity: 0.52559\n",
      "Document Avibactam - similarity: 0.52558\n",
      "Document Guanosine-2',3'-cyclophosphorothioate - similarity: 0.52546\n",
      "Document Benzoin resin - similarity: 0.52542\n",
      "Document Certoparin - similarity: 0.52539\n",
      "Document Somatrogon - similarity: 0.52534\n",
      "Document Armodafinil - similarity: 0.52531\n",
      "Document 4-[(5-bromopyridin-2-yl)amino]-4-oxobutanoic acid - similarity: 0.52524\n",
      "Document Desmopressin - similarity: 0.52522\n",
      "Document (2S)-2-(4-chlorophenyl)-2-[4-(1H-pyrazol-4-yl)phenyl]ethanamine - similarity: 0.52513\n",
      "Document Reteplase - similarity: 0.52511\n",
      "Document Cerivastatin - similarity: 0.52510\n",
      "Document Lisinopril - similarity: 0.52504\n",
      "Document Acamprosate - similarity: 0.52492\n",
      "Document Apremilast - similarity: 0.52489\n",
      "Document Solithromycin - similarity: 0.52486\n",
      "Document Methionine - similarity: 0.52481\n",
      "Document Diethylhomospermine - similarity: 0.52477\n",
      "Document Ofranergene obadenovec - similarity: 0.52476\n",
      "Document N-sulfo-flavin mononucleotide - similarity: 0.52461\n",
      "Document Deslanoside - similarity: 0.52460\n",
      "Document Enrasentan - similarity: 0.52453\n",
      "Document (3r)-4-(P-Toluenesulfonyl)-1,4-Thiazane-3-Carboxylicacid-L-Leucine - similarity: 0.52450\n",
      "Document Naftidrofuryl - similarity: 0.52439\n",
      "Document Tivozanib - similarity: 0.52439\n",
      "Document MDX-1303 - similarity: 0.52435\n",
      "Document Gamolenic acid - similarity: 0.52434\n",
      "Document PTI-801 - similarity: 0.52423\n",
      "Document Vespula vulgaris venom protein - similarity: 0.52419\n",
      "Document Benzatropine - similarity: 0.52416\n",
      "Document Latrunculin B - similarity: 0.52410\n",
      "Document Equisetum arvense top - similarity: 0.52401\n",
      "Document 5-Methylpyrrole - similarity: 0.52401\n",
      "Document Descarboxy-nor-N(Omega)-Hydroxy-L-Arginine - similarity: 0.52400\n",
      "Document IDD594 - similarity: 0.52396\n",
      "Document Prothipendyl - similarity: 0.52395\n",
      "Document Tobramycin - similarity: 0.52394\n",
      "Document Zinc gluconate - similarity: 0.52391\n",
      "Document (4R)-7-chloro-9-methyl-1-oxo-1,2,4,9-tetrahydrospiro[beta-carboline-3,4'-piperidine]-4-carbonitrile - similarity: 0.52385\n",
      "Document Influenza B virus B/Wisconsin/1/2010 recombinant hemagglutinin antigen - similarity: 0.52385\n",
      "Document Maropitant - similarity: 0.52382\n",
      "Document Sitimagene ceradenovec - similarity: 0.52381\n",
      "Document Pyronaridine - similarity: 0.52374\n",
      "Document Batimastat - similarity: 0.52372\n",
      "Document Acer saccharinum pollen - similarity: 0.52367\n",
      "Document 16,17-Androstene-3-Ol - similarity: 0.52362\n",
      "Document Nepicastat - similarity: 0.52359\n",
      "Document Metanitrophenyl-Alpha-D-Galactoside - similarity: 0.52347\n",
      "Document 2,5-DICHLORO-N-(5-CHLORO-1,3-BENZOXAZOL-2-YL)BENZENESULFONAMIDE - similarity: 0.52343\n",
      "Document Goserelin - similarity: 0.52343\n",
      "Document Tremella fuciformis whole - similarity: 0.52340\n",
      "Document Dihydralazine - similarity: 0.52335\n",
      "Document Olorofim - similarity: 0.52323\n",
      "Document (2S)-4-(2,5-DIFLUOROPHENYL)-N-METHYL-2-PHENYL-N-PIPERIDIN-4-YL-2,5-DIHYDRO-1H-PYRROLE-1-CARBOXAMIDE - similarity: 0.52323\n",
      "Document Oxymorphone - similarity: 0.52317\n",
      "Document Demecolcine - similarity: 0.52315\n",
      "Document Dimethyl fumarate - similarity: 0.52311\n",
      "Document Mitoguazone - similarity: 0.52304\n",
      "Document Thioridazine - similarity: 0.52300\n",
      "Document Etoposide - similarity: 0.52300\n",
      "Document Givinostat - similarity: 0.52280\n",
      "Document Prosopis juliflora pollen - similarity: 0.52270\n",
      "Document Cianidanol - similarity: 0.52269\n",
      "Document Bleomycin - similarity: 0.52266\n",
      "Document Sulthiame - similarity: 0.52263\n",
      "Document Muzolimine - similarity: 0.52263\n",
      "Document Aluminum zirconium pentachlorohydrate - similarity: 0.52263\n",
      "Document Alnus incana subsp. rugosa pollen - similarity: 0.52262\n",
      "Document Methylsamidorphan - similarity: 0.52259\n",
      "Document Nitrocefin - similarity: 0.52259\n",
      "Document Latrunculin A - similarity: 0.52256\n",
      "Document 5-(2-CHLORO-4-NITROPHENYL)-2-FUROIC ACID - similarity: 0.52250\n",
      "Document N-{2-[6-(2,4-DIAMINO-6-ETHYLPYRIMIDIN-5-YL)-2,2-DIMETHYL-3-OXO-2,3-DIHYDRO-4H-1,4-BENZOTHIAZIN-4-YL]ETHYL}ACETAMIDE - similarity: 0.52246\n",
      "Document Rye - similarity: 0.52246\n",
      "Document 4-phospho-D-erythronic acid - similarity: 0.52244\n",
      "Document ORM-12741 - similarity: 0.52236\n",
      "Document Sulbutiamine - similarity: 0.52231\n",
      "Document Bismuth subnitrate - similarity: 0.52228\n",
      "Document Posiphen - similarity: 0.52226\n",
      "Document 5-QUINOXALIN-6-YLMETHYLENE-THIAZOLIDINE-2,4-DIONE - similarity: 0.52224\n",
      "Document S-(Methylmercury)-L-Cysteine - similarity: 0.52211\n",
      "Document Methylprednisolone hemisuccinate - similarity: 0.52204\n",
      "Document Amyl Nitrite - similarity: 0.52192\n",
      "Document Ammonium chloride - similarity: 0.52190\n",
      "Document Quinfamide - similarity: 0.52189\n",
      "Document Defibrotide - similarity: 0.52184\n",
      "Document Perindopril - similarity: 0.52182\n",
      "Document Sipuleucel-T - similarity: 0.52173\n",
      "Document Pemoline - similarity: 0.52169\n",
      "Document Fluocinolone acetonide - similarity: 0.52168\n",
      "Document Donepezil - similarity: 0.52163\n",
      "Document 4-{2-[4-(2-Aminoethyl)Piperazin-1-Yl]Pyridin-4-Yl}-N-(3-Chloro-4-Methylphenyl)Pyrimidin-2-Amine - similarity: 0.52162\n",
      "Document 2-mercaptobenzothiazole - similarity: 0.52161\n",
      "Document 1-(3-(2,4-DIMETHYLTHIAZOL-5-YL)-4-OXO-2,4-DIHYDROINDENO[1,2-C]PYRAZOL-5-YL)-3-(4-METHYLPIPERAZIN-1-YL)UREA - similarity: 0.52161\n",
      "Document Eltoprazine - similarity: 0.52157\n",
      "Document Flurbiprofen - similarity: 0.52152\n",
      "Document CP-724714 - similarity: 0.52150\n",
      "Document 4-Hydroxy-Aconitate Ion - similarity: 0.52146\n",
      "Document Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 antigen (propiolactone inactivated) - similarity: 0.52141\n",
      "Document BPN-14770 - similarity: 0.52133\n",
      "Document Benserazide - similarity: 0.52133\n",
      "Document Cucurbita pepo subsp. ovifera whole - similarity: 0.52127\n",
      "Document Calendula officinalis whole - similarity: 0.52123\n",
      "Document Rice bran - similarity: 0.52120\n",
      "Document Fosinopril - similarity: 0.52107\n",
      "Document Technetium Tc-99m polyphosphate - similarity: 0.52106\n",
      "Document Cholecystokinin - similarity: 0.52104\n",
      "Document 2-(aminomethyl)phenol - similarity: 0.52096\n",
      "Document N-(Chlorophenyl)-N'-hydroxyguanidine - similarity: 0.52080\n",
      "Document Mocetinostat - similarity: 0.52077\n",
      "Document Sargramostim - similarity: 0.52072\n",
      "Document Kifunensine - similarity: 0.52062\n",
      "Document Nitrous oxide - similarity: 0.52060\n",
      "Document Meticillin - similarity: 0.52056\n",
      "Document Netoglitazone - similarity: 0.52052\n",
      "Document 2-amino-3-(4-amino-1H-indol-3-yl)propanoic acid - similarity: 0.52051\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Motolimod - similarity: 0.52049\n",
      "Document Otamixaban - similarity: 0.52045\n",
      "Document Angiotensin II - similarity: 0.52041\n",
      "Document Dostarlimab - similarity: 0.52035\n",
      "Document 2-(N-morpholino)ethanesulfonic acid - similarity: 0.52033\n",
      "Document MK-0686 - similarity: 0.52027\n",
      "Document Lanolin alcohols - similarity: 0.52026\n",
      "Document Ambrosia acanthicarpa pollen - similarity: 0.52021\n",
      "Document Polycarbophil - similarity: 0.52015\n",
      "Document 1-Hydroxyamine-2-Isobutylmalonic Acid - similarity: 0.52013\n",
      "Document Magnesium cation - similarity: 0.52012\n",
      "Document 5-[(3S)-3-(2-methoxybiphenyl-4-yl)but-1-yn-1-yl]-6-methylpyrimidine-2,4-diamine - similarity: 0.52007\n",
      "Document Phenolphthalein - similarity: 0.52006\n",
      "Document Melarsoprol - similarity: 0.52005\n",
      "Document Fructose-6-phosphate - similarity: 0.52003\n",
      "Document Alvimopan - similarity: 0.52003\n",
      "Document Nomacopan - similarity: 0.51992\n",
      "Document Staurosporine - similarity: 0.51986\n",
      "Document Nitroxoline - similarity: 0.51981\n",
      "Document Balsalazide - similarity: 0.51976\n",
      "Document Sulindac - similarity: 0.51975\n",
      "Document (3Z)-1-[(6-fluoro-4H-1,3-benzodioxin-8-yl)methyl]-4-[(E)-2-phenylethenyl]-1H-indole-2,3-dione 3-oxime - similarity: 0.51959\n",
      "Document Silver - similarity: 0.51955\n",
      "Document Acepromazine - similarity: 0.51947\n",
      "Document Timepidium - similarity: 0.51944\n",
      "Document Aurothioglucose - similarity: 0.51943\n",
      "Document Acetylcarnitine - similarity: 0.51943\n",
      "Document Methyldopa - similarity: 0.51940\n",
      "Document Procaine - similarity: 0.51938\n",
      "Document 4-Deoxy-D-Glucuronic Acid - similarity: 0.51935\n",
      "Document Epanolol - similarity: 0.51928\n",
      "Document Golimumab - similarity: 0.51927\n",
      "Document Corifollitropin alfa - similarity: 0.51920\n",
      "Document Potassium bitartrate - similarity: 0.51916\n",
      "Document Nomegestrol acetate - similarity: 0.51913\n",
      "Document MDL72527 - similarity: 0.51911\n",
      "Document Nefiracetam - similarity: 0.51907\n",
      "Document Calcium glycerophosphate - similarity: 0.51907\n",
      "Document Orphenadrine - similarity: 0.51905\n",
      "Document Panitumumab - similarity: 0.51901\n",
      "Document 7-(aminomethyl)-6-(2-chlorophenyl)-1-methyl-1H-benzimidazole-5-carbonitrile - similarity: 0.51901\n",
      "Document Linseed oil - similarity: 0.51900\n",
      "Document N-(7-CARBAMIMIDOYL-NAPHTHALEN-1-YL)-3-HYDROXY-2-METHYL-BENZAMIDE - similarity: 0.51899\n",
      "Document Equilin - similarity: 0.51892\n",
      "Document 7-BENZYL-1,3-DIMETHYL-8-PIPERAZIN-1-YL-3,7-DIHYDRO-PURINE-2,6-DIONE - similarity: 0.51876\n",
      "Document Pegvaliase - similarity: 0.51875\n",
      "Document Ronacaleret - similarity: 0.51874\n",
      "Document [4-(6-Chloro-Naphthalene-2-Sulfonyl)-Piperazin-1-Yl]-(3,4,5,6-Tetrahydro-2h-[1,4']Bipyridinyl-4-Yl)-Methanone - similarity: 0.51866\n",
      "Document H-89 - similarity: 0.51864\n",
      "Document 5-Amino-3-Methyl-Pyrrolidine-2-Carboxylic Acid - similarity: 0.51863\n",
      "Document Bectumomab - similarity: 0.51863\n",
      "Document Batroxobin - similarity: 0.51860\n",
      "Document (R)-warfarin - similarity: 0.51857\n",
      "Document 6-methyl-formycin A - similarity: 0.51851\n",
      "Document Motesanib - similarity: 0.51850\n",
      "Document Spermidine - similarity: 0.51843\n",
      "Document N-Dimethyl-Lysine - similarity: 0.51843\n",
      "Document Sage - similarity: 0.51842\n",
      "Document Samarium Sm-153 - similarity: 0.51832\n",
      "Document Grepafloxacin - similarity: 0.51830\n",
      "Document (9S,12S)-9-(1-methylethyl)-7,10-dioxo-2-oxa-8,11-diazabicyclo[12.2.2]octadeca-1(16),14,17-triene-12-carboxylic acid - similarity: 0.51828\n",
      "Document (3R)-3-(aminomethyl)-9-methoxy-1,2,3,4-tetrahydro-5H-[1]benzothieno[3,2-e][1,4]diazepin-5-one - similarity: 0.51822\n",
      "Document 1-METHYL ETHYL 2-CHLORO-5-[[[(1-METHYLETHOXY)THIOOXO]METHYL]AMINO]-BENZOATE - similarity: 0.51818\n",
      "Document Roxatidine acetate - similarity: 0.51809\n",
      "Document Quilizumab - similarity: 0.51801\n",
      "Document Blue crab - similarity: 0.51797\n",
      "Document Cefonicid - similarity: 0.51789\n",
      "Document Anemarrhena asphodeloides root - similarity: 0.51784\n",
      "Document AZD-4877 - similarity: 0.51784\n",
      "Document (3s)-2,3,4,5-Tetrahydropyridin-3-Amine - similarity: 0.51781\n",
      "Document Acetylcysteine magnesium - similarity: 0.51778\n",
      "Document 3-(4-CHLOROPHENYL)-5-(METHYLTHIO)-4H-1,2,4-TRIAZOLE - similarity: 0.51777\n",
      "Document Egg phospholipids - similarity: 0.51776\n",
      "Document Naproxen - similarity: 0.51766\n",
      "Document Mavrilimumab - similarity: 0.51764\n",
      "Document N~2~,N~2~-DIMETHYL-N~1~-(6-OXO-5,6-DIHYDROPHENANTHRIDIN-2-YL)GLYCINAMIDE - similarity: 0.51764\n",
      "Document Crizanlizumab - similarity: 0.51764\n",
      "Document Para-Mercury-Benzenesulfonic Acid - similarity: 0.51763\n",
      "Document Tazobactam - similarity: 0.51761\n",
      "Document Cinnamyl alcohol - similarity: 0.51760\n",
      "Document Apomab - similarity: 0.51760\n",
      "Document Pimagedine - similarity: 0.51755\n",
      "Document 2-Butyl-5,6-Dihydro-1h-Imidazo[4,5-D]Pyridazine-4,7-Dione - similarity: 0.51745\n",
      "Document 6-fluoro-2-(2-hydroxy-3-isobutoxy-phenyl)-1H-benzoimidazole-5-carboxamidine - similarity: 0.51741\n",
      "Document Trenbolone - similarity: 0.51740\n",
      "Document 3,5-dibromobiphenyl-4-ol - similarity: 0.51736\n",
      "Document Oxytocin - similarity: 0.51721\n",
      "Document Blonanserin - similarity: 0.51717\n",
      "Document Bicifadine - similarity: 0.51714\n",
      "Document Polythiazide - similarity: 0.51704\n",
      "Document Granisetron - similarity: 0.51704\n",
      "Document Black olive - similarity: 0.51698\n",
      "Document Borage oil - similarity: 0.51696\n",
      "Document Cetylpyridinium - similarity: 0.51694\n",
      "Document Vadastuximab Talirine - similarity: 0.51693\n",
      "Document Sodium zirconium cyclosilicate - similarity: 0.51692\n",
      "Document Phenolsulfonphthalein - similarity: 0.51691\n",
      "Document Molybdenum - similarity: 0.51689\n",
      "Document Chloramphenicol succinate - similarity: 0.51661\n",
      "Document Pizotifen - similarity: 0.51658\n",
      "Document Altropane - similarity: 0.51653\n",
      "Document Uprosertib - similarity: 0.51652\n",
      "Document Rolipram - similarity: 0.51651\n",
      "Document Talc - similarity: 0.51648\n",
      "Document 5-PENTYL-2-PHENOXYPHENOL - similarity: 0.51647\n",
      "Document Ciclonicate - similarity: 0.51646\n",
      "Document BMS-275183 - similarity: 0.51645\n",
      "Document Labetuzumab - similarity: 0.51644\n",
      "Document AEE-788 - similarity: 0.51640\n",
      "Document N-BENZYLOXYCARBONYL-L-SERINE-BETALACTONE - similarity: 0.51638\n",
      "Document Beef - similarity: 0.51636\n",
      "Document Technetium Tc-99m medronate - similarity: 0.51635\n",
      "Document Tiagabine - similarity: 0.51630\n",
      "Document Dihydroorotic Acid - similarity: 0.51625\n",
      "Document Indium In-111 imciromab pentetate - similarity: 0.51622\n",
      "Document Epimedium grandiflorum top - similarity: 0.51614\n",
      "Document Codeine - similarity: 0.51612\n",
      "Document Sodium feredetate - similarity: 0.51610\n",
      "Document 2-(6-CHLORO-3-{[2,2-DIFLUORO-2-(2-PYRIDINYL)ETHYL]AMINO}-2-OXO-1(2H)-PYRAZINYL)-N-[(2-FLUORO-6-PYRIDINYL)METHYL]ACETAMIDE - similarity: 0.51606\n",
      "Document Oregovomab - similarity: 0.51606\n",
      "Document Influenza A virus A/Victoria/361/2011 (H3N2) recombinant hemagglutinin antigen - similarity: 0.51605\n",
      "Document Desonide - similarity: 0.51605\n",
      "Document Chlorthalidone - similarity: 0.51602\n",
      "Document Rose bengal lactone - similarity: 0.51601\n",
      "Document 2,3-Bis-Benzo[1,3]Dioxol-5-Ylmethyl-Succinic Acid - similarity: 0.51599\n",
      "Document Equine Botulinum Neurotoxin G Immune FAB2 - similarity: 0.51596\n",
      "Document Tiopronin - similarity: 0.51593\n",
      "Document Tolevamer - similarity: 0.51592\n",
      "Document Nelfinavir - similarity: 0.51590\n",
      "Document Lorvotuzumab mertansine - similarity: 0.51587\n",
      "Document Leptazoline B - similarity: 0.51585\n",
      "Document Delcasertib - similarity: 0.51584\n",
      "Document Banana - similarity: 0.51571\n",
      "Document 2-(3,4-DIHYDROXYPHENYL)-8-(1,1-DIOXIDOISOTHIAZOLIDIN-2-YL)-3-HYDROXY-6-METHYL-4H-CHROMEN-4-ONE - similarity: 0.51566\n",
      "Document Avobenzone - similarity: 0.51564\n",
      "Document Pear - similarity: 0.51531\n",
      "Document Hydrolyzed soy protein - similarity: 0.51530\n",
      "Document Influenza A virus A/North carolina/04/2016 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.51528\n",
      "Document Amoxapine - similarity: 0.51527\n",
      "Document Bicalutamide - similarity: 0.51521\n",
      "Document 1,2-Dihydroxybenzene - similarity: 0.51519\n",
      "Document Ketanserin - similarity: 0.51508\n",
      "Document Blarcamesine - similarity: 0.51504\n",
      "Document Toloxatone - similarity: 0.51496\n",
      "Document Sitaxentan - similarity: 0.51496\n",
      "Document Magnesium chloride - similarity: 0.51495\n",
      "Document Poa annua pollen - similarity: 0.51484\n",
      "Document Thrombomodulin Alfa - similarity: 0.51475\n",
      "Document Influenza B virus B/Massachusetts/2/2012 antigen (formaldehyde inactivated) - similarity: 0.51470\n",
      "Document Belimumab - similarity: 0.51467\n",
      "Document 2-PHENYLAMINO-4-METHYL-5-ACETYL THIAZOLE - similarity: 0.51457\n",
      "Document Dermatophagoides farinae - similarity: 0.51442\n",
      "Document Vaniprevir - similarity: 0.51440\n",
      "Document Quinaprilat - similarity: 0.51438\n",
      "Document Nimodipine - similarity: 0.51436\n",
      "Document Pilocarpine - similarity: 0.51434\n",
      "Document Technetium Tc-99m red blood cells - similarity: 0.51427\n",
      "Document 1-(2-cyclopropylethyl)-3-(1,1-dioxo-2H-1,2,4-benzothiadiazin-3-yl)-6-fluoro-4-hydroxy-2(1H)-quinolinone - similarity: 0.51423\n",
      "Document 6-(2-phenoxyethoxy)-1,3,5-triazine-2,4-diamine - similarity: 0.51420\n",
      "Document Atomoxetine - similarity: 0.51411\n",
      "Document Influenza B virus B/Phuket/3073/2013 antigen (propiolactone inactivated) - similarity: 0.51408\n",
      "Document Lerisetron - similarity: 0.51397\n",
      "Document beta-Alanine - similarity: 0.51394\n",
      "Document Lumiliximab - similarity: 0.51392\n",
      "Document Nojirimycine Tetrazole - similarity: 0.51391\n",
      "Document Dexlansoprazole - similarity: 0.51391\n",
      "Document 2-AMINO-4-CHLORO-3-HYDROXYBENZOIC ACID - similarity: 0.51387\n",
      "Document 1h-Benoximidazole-2-Carboxylic Acid - similarity: 0.51385\n",
      "Document EMZ702 - similarity: 0.51383\n",
      "Document 6-(Octahydro-1h-Indol-1-Ylmethyl)Decahydroquinazoline-2,4-Diamine - similarity: 0.51380\n",
      "Document N-acetyl-N'-beta-D-glucopyranosyl urea - similarity: 0.51379\n",
      "Document Ensartinib - similarity: 0.51377\n",
      "Document Larotaxel - similarity: 0.51372\n",
      "Document Heptanamide - similarity: 0.51368\n",
      "Document ZEN-012 - similarity: 0.51365\n",
      "Document N-(1-adamantyl)-N'-(4-guanidinobenzyl)urea - similarity: 0.51362\n",
      "Document (1S,4S,5S)-1,4,5-TRIHYDROXY-3-[3-(PHENYLTHIO)PHENYL]CYCLOHEX-2-ENE-1-CARBOXYLIC ACID - similarity: 0.51357\n",
      "Document Diphencyprone - similarity: 0.51347\n",
      "Document Gemfibrozil - similarity: 0.51346\n",
      "Document Salix nigra bark - similarity: 0.51342\n",
      "Document Betula lenta whole - similarity: 0.51319\n",
      "Document Ranitidine - similarity: 0.51307\n",
      "Document Camptothecin - similarity: 0.51303\n",
      "Document 4-Deoxy-D-Mannuronic Acid - similarity: 0.51297\n",
      "Document Veledimex - similarity: 0.51278\n",
      "Document Acarbose - similarity: 0.51277\n",
      "Document Mosapramine - similarity: 0.51267\n",
      "Document Carnosine - similarity: 0.51261\n",
      "Document Lanreotide - similarity: 0.51252\n",
      "Document 4-(3-{[5-(trifluoromethyl)pyridin-2-yl]oxy}benzyl)piperidine-1-carboxylic acid - similarity: 0.51244\n",
      "Document 1-METHYL-3-PHENYL-1H-PYRAZOL-5-YLSULFAMIC ACID - similarity: 0.51241\n",
      "Document Phenazocine - similarity: 0.51219\n",
      "Document Melperone - similarity: 0.51217\n",
      "Document Nicorandil - similarity: 0.51213\n",
      "Document EGS21 - similarity: 0.51207\n",
      "Document Lusutrombopag - similarity: 0.51206\n",
      "Document Tolpropamine - similarity: 0.51200\n",
      "Document Cloricromen - similarity: 0.51199\n",
      "Document Rosemary - similarity: 0.51196\n",
      "Document Trichloroacetate - similarity: 0.51195\n",
      "Document Butylene glycol - similarity: 0.51181\n",
      "Document Aripiprazole - similarity: 0.51181\n",
      "Document PIK3CA - similarity: 0.51179\n",
      "Document Kale - similarity: 0.51164\n",
      "Document Ezogabine - similarity: 0.51158\n",
      "Document Elexacaftor - similarity: 0.51153\n",
      "Document Lanraplenib - similarity: 0.51151\n",
      "Document (2S)-1-(6H-INDOL-3-YL)-3-{[5-(7H-PYRAZOLO[3,4-C]PYRIDIN-5-YL)PYRIDIN-3-YL]OXY}PROPAN-2-AMINE - similarity: 0.51149\n",
      "Document Catfish - similarity: 0.51146\n",
      "Document Sulfasalazine - similarity: 0.51142\n",
      "Document PCI-27483 - similarity: 0.51140\n",
      "Document 2-Chlorophenol - similarity: 0.51135\n",
      "Document Primaquine - similarity: 0.51135\n",
      "Document PF-06282999 - similarity: 0.51134\n",
      "Document Livoletide - similarity: 0.51132\n",
      "Document Lenvatinib - similarity: 0.51119\n",
      "Document 8-Hydroxy-4-(1-Hydroxyethyl)Quinoline-2-Carboxylic Acid - similarity: 0.51103\n",
      "Document Mucuna pruriens seed - similarity: 0.51096\n",
      "Document Levocarnitine - similarity: 0.51095\n",
      "Document Prednisolone tebutate - similarity: 0.51092\n",
      "Document Eletriptan - similarity: 0.51091\n",
      "Document L-homoarginine - similarity: 0.51089\n",
      "Document Penicillium camemberti - similarity: 0.51085\n",
      "Document 2-[(1R)-2-carboxy-1-(naphthalen-1-ylmethyl)ethyl]-1,3-dioxo-2,3-dihydro-1H-isoindole-5-carboxylic acid - similarity: 0.51077\n",
      "Document Mequitazine - similarity: 0.51075\n",
      "Document Agkistrodon piscivorus antivenin - similarity: 0.51069\n",
      "Document Gavilimomab - similarity: 0.51068\n",
      "Document Sequoia sempervirens pollen - similarity: 0.51062\n",
      "Document Tripotassium (1R)-4-biphenyl-4-yl-1-phosphonatobutane-1-sulfonate - similarity: 0.51059\n",
      "Document Aluminium clofibrate - similarity: 0.51045\n",
      "Document 4-acetamido-2,4-didexoy-D-glycero-beta-D-galacto-octopyranosylphosphonic acid - similarity: 0.51045\n",
      "Document Aclerastide - similarity: 0.51044\n",
      "Document Thiocitrulline - similarity: 0.51041\n",
      "Document Meradimate - similarity: 0.51027\n",
      "Document 2-Amino-3-Oxo-4-Sulfo-Butyric Acid - similarity: 0.51012\n",
      "Document 3-[({(1E)-[2-(trifluoromethyl)phenyl]methylidene}amino)oxy]propanoic acid - similarity: 0.51008\n",
      "Document Ginkgolide A - similarity: 0.51004\n",
      "Document Hepatitis B Immune Globulin - similarity: 0.50997\n",
      "Document Sodium fluorophosphate - similarity: 0.50986\n",
      "Document TAS-108 - similarity: 0.50986\n",
      "Document 5-(2-METHOXYPHENYL)-2-FUROIC ACID - similarity: 0.50982\n",
      "Document Fenobam - similarity: 0.50981\n",
      "Document Estriol - similarity: 0.50978\n",
      "Document ginkgolide-B - similarity: 0.50972\n",
      "Document Sarracenia Purpurea - similarity: 0.50971\n",
      "Document 3-[(2,4-Dichlorobenzoyl)(Isopropyl)Amino]-5-Phenylthiophene-2-Carboxylic Acid - similarity: 0.50954\n",
      "Document Polyethylene glycol - similarity: 0.50949\n",
      "Document Northern brown shrimp - similarity: 0.50949\n",
      "Document Streptococcus pneumoniae type 18c capsular polysaccharide antigen - similarity: 0.50935\n",
      "Document Red currant - similarity: 0.50935\n",
      "Document Peginterferon lambda-1a - similarity: 0.50931\n",
      "Document Blueberry - similarity: 0.50929\n",
      "Document SR 121463 - similarity: 0.50924\n",
      "Document Gadobutrol - similarity: 0.50917\n",
      "Document Halofuginone - similarity: 0.50914\n",
      "Document Butyric Acid - similarity: 0.50906\n",
      "Document Sevoflurane - similarity: 0.50906\n",
      "Document 2,6-DIMETHYL-1-(3-[3-METHYL-5-ISOXAZOLYL]-PROPANYL)-4-[4-METHYL-2H-TETRAZOL-2-YL]-PHENOL - similarity: 0.50899\n",
      "Document Climbazole - similarity: 0.50889\n",
      "Document Carbinoxamine - similarity: 0.50880\n",
      "Document GMA-161 - similarity: 0.50874\n",
      "Document Mimosine - similarity: 0.50873\n",
      "Document Thiamylal - similarity: 0.50873\n",
      "Document Hydrocodone - similarity: 0.50872\n",
      "Document Cefadroxil - similarity: 0.50871\n",
      "Document Chlorpheniramine - similarity: 0.50868\n",
      "Document Retosiban - similarity: 0.50864\n",
      "Document AZD-9056 - similarity: 0.50862\n",
      "Document Oxabolone - similarity: 0.50861\n",
      "Document Dyclonine - similarity: 0.50854\n",
      "Document Vanadium - similarity: 0.50844\n",
      "Document Calcium chloride - similarity: 0.50844\n",
      "Document Refametinib - similarity: 0.50841\n",
      "Document Dronedarone - similarity: 0.50837\n",
      "Document Royal jelly - similarity: 0.50837\n",
      "Document Zinc carbonate - similarity: 0.50836\n",
      "Document N-[(2R)-5-(aminosulfonyl)-2,3-dihydro-1H-inden-2-yl]-2-propylpentanamide - similarity: 0.50824\n",
      "Document S-Ethylisothiourea - similarity: 0.50817\n",
      "Document Pravastatin - similarity: 0.50817\n",
      "Document Perflexane - similarity: 0.50815\n",
      "Document (1S,3aS,5aR,8aS)-1,7,7-trimethyl-1,2,3,3a,5a,6,7,8-octahydrocyclopenta[c]pentalene-4-carboxylic acid - similarity: 0.50812\n",
      "Document Nisoldipine - similarity: 0.50811\n",
      "Document Dimethylallyl Diphosphate - similarity: 0.50801\n",
      "Document Ecromeximab - similarity: 0.50798\n",
      "Document N-cyclopropyl-N-(trans-4-pyridin-3-ylcyclohexyl)-4-[(1S)-2,2,2-trifluoro-1-hydroxy-1-methylethyl]benzamide - similarity: 0.50796\n",
      "Document Naphthalene Trisulfonate - similarity: 0.50795\n",
      "Document Neurospora crassa - similarity: 0.50794\n",
      "Document Trichophyton schoenleinii - similarity: 0.50793\n",
      "Document 2-deoxyglucose - similarity: 0.50791\n",
      "Document Theobroma cacao whole - similarity: 0.50786\n",
      "Document N2-[(benzyloxy)carbonyl]-n1-[(3S)-1-cyanopyrrolidin-3-yl]-l-leucinamide - similarity: 0.50785\n",
      "Document Calcium saccharate - similarity: 0.50779\n",
      "Document Indirubin - similarity: 0.50778\n",
      "Document N-Acetyl-D-Galactosamine 6-Sulfate - similarity: 0.50775\n",
      "Document Butonate - similarity: 0.50772\n",
      "Document (R)-2-Hydroxy-3-Sulfopropanoic Acid - similarity: 0.50766\n",
      "Document Atacicept - similarity: 0.50766\n",
      "Document Haemophilus influenzae type B capsular polysaccharide meningococcal outer membrane protein conjugate antigen - similarity: 0.50753\n",
      "Document Nelivaptan - similarity: 0.50750\n",
      "Document Tiocarlide - similarity: 0.50747\n",
      "Document Zea mays pollen - similarity: 0.50746\n",
      "Document Colfosceril palmitate - similarity: 0.50745\n",
      "Document Xylazine - similarity: 0.50744\n",
      "Document Cefotaxime - similarity: 0.50743\n",
      "Document Garenoxacin - similarity: 0.50742\n",
      "Document Indecainide - similarity: 0.50740\n",
      "Document Betadex - similarity: 0.50733\n",
      "Document S-(4-BROMOBENZYL)CYSTEINE - similarity: 0.50732\n",
      "Document Fluoride ion F-18 - similarity: 0.50730\n",
      "Document Bufuralol - similarity: 0.50718\n",
      "Document 1-[(2-AMINO-4-CHLORO-5-METHYLPHENYL)SULFONYL]-L-PROLINE - similarity: 0.50700\n",
      "Document Fish oil - similarity: 0.50697\n",
      "Document Resorcinol - similarity: 0.50691\n",
      "Document Pracinostat - similarity: 0.50689\n",
      "Document 2,3-DIPHENYL-N-(2-PIPERAZIN-1-YLETHYL)FURO[2,3-B]PYRIDIN-4-AMINE - similarity: 0.50682\n",
      "Document Sovaprevir - similarity: 0.50671\n",
      "Document (5S,6S)-6-[(R)ACETOXYETH-2-YL]-PENEM-3-CARBOXYLATEPROPANE - similarity: 0.50670\n",
      "Document Dobutamine - similarity: 0.50669\n",
      "Document Viminol - similarity: 0.50667\n",
      "Document N-Formylmethionine - similarity: 0.50655\n",
      "Document Methionine sulfoximine - similarity: 0.50653\n",
      "Document Lesinurad - similarity: 0.50650\n",
      "Document Pheneturide - similarity: 0.50647\n",
      "Document White fish - similarity: 0.50645\n",
      "Document Propylene glycol - similarity: 0.50640\n",
      "Document Capromab pendetide - similarity: 0.50639\n",
      "Document LY2275796 - similarity: 0.50617\n",
      "Document Silmitasertib - similarity: 0.50610\n",
      "Document D-fluoromethyltyrosine F-18 - similarity: 0.50610\n",
      "Document ISOPENTENYL PYROPHOSPHATE - similarity: 0.50609\n",
      "Document Fanapanel - similarity: 0.50606\n",
      "Document 2-hydroxy-6-oxo-6-phenyl-2,4-hexadienoic acid - similarity: 0.50605\n",
      "Document 3-(9-HYDROXY-1,3-DIOXO-4-PHENYL-2,3-DIHYDROPYRROLO[3,4-C]CARBAZOL-6(1H)-YL)PROPANOIC ACID - similarity: 0.50600\n",
      "Document Pecan - similarity: 0.50592\n",
      "Document Duroquinone - similarity: 0.50589\n",
      "Document Cilengitide - similarity: 0.50584\n",
      "Document Bovine type I collagen - similarity: 0.50583\n",
      "Document Pemetrexed - similarity: 0.50573\n",
      "Document Furazidin - similarity: 0.50572\n",
      "Document AZD-8418 - similarity: 0.50569\n",
      "Document Sacrosidase - similarity: 0.50560\n",
      "Document Nonivamide - similarity: 0.50558\n",
      "Document Corn - similarity: 0.50538\n",
      "Document Carboxymethylcellulose - similarity: 0.50535\n",
      "Document Iofolastat I-123 - similarity: 0.50522\n",
      "Document Molybdenum cofactor - similarity: 0.50515\n",
      "Document Herring - similarity: 0.50506\n",
      "Document Melaleuca alternifolia leaf - similarity: 0.50491\n",
      "Document Silver nitrate - similarity: 0.50491\n",
      "Document Lettuce - similarity: 0.50483\n",
      "Document Fagrocorat - similarity: 0.50480\n",
      "Document Fica - similarity: 0.50474\n",
      "Document Cobalt chloride - similarity: 0.50458\n",
      "Document Ambazone - similarity: 0.50457\n",
      "Document Magnesium phosphate - similarity: 0.50455\n",
      "Document Quercus alba pollen - similarity: 0.50450\n",
      "Document Cholesteryl chloride - similarity: 0.50441\n",
      "Document Dibenzepin - similarity: 0.50423\n",
      "Document Laronidase - similarity: 0.50419\n",
      "Document Norgestimate - similarity: 0.50418\n",
      "Document Phenethylamine - similarity: 0.50417\n",
      "Document 5-FLUOROINDOLE PROPANOL PHOSPHATE - similarity: 0.50415\n",
      "Document Trifluoromethionine - similarity: 0.50415\n",
      "Document Methylprednisolone aceponate - similarity: 0.50405\n",
      "Document Sermorelin - similarity: 0.50402\n",
      "Document Oxyphenbutazone - similarity: 0.50392\n",
      "Document Aprocitentan - similarity: 0.50380\n",
      "Document Selepressin - similarity: 0.50366\n",
      "Document N-[(2S)-3-Phenyl-2-sulfanylpropanoyl]-L-phenylalanyl-L-tyrosine - similarity: 0.50361\n",
      "Document (4S,5E,7Z,10Z,13Z,16Z,19Z)-4-hydroxydocosa-5,7,10,13,16,19-hexaenoic acid - similarity: 0.50355\n",
      "Document Argininosuccinate - similarity: 0.50355\n",
      "Document AT9283 - similarity: 0.50351\n",
      "Document Dipicolinic acid - similarity: 0.50342\n",
      "Document Black walnut - similarity: 0.50337\n",
      "Document Doxacurium - similarity: 0.50326\n",
      "Document Ephedrine - similarity: 0.50320\n",
      "Document Gadoteric acid - similarity: 0.50320\n",
      "Document AC-(D)PHE-PRO-BOROHOMOLYS-OH - similarity: 0.50318\n",
      "Document PL-3994 - similarity: 0.50318\n",
      "Document Levosimendan - similarity: 0.50312\n",
      "Document Pheniramine - similarity: 0.50310\n",
      "Document Salmeterol - similarity: 0.50304\n",
      "Document 4-bromo-2-{[(2R)-2-(2-chlorobenzyl)pyrrolidin-1-yl]carbonyl}aniline - similarity: 0.50296\n",
      "Document Solifenacin - similarity: 0.50293\n",
      "Document Isoflupredone - similarity: 0.50292\n",
      "Document ({[(3E)-2'-Oxo-2',7'-dihydro-2,3'-biindol-3(7H)-ylidene]amino}oxy)acetic acid - similarity: 0.50285\n",
      "Document Trichloroacetaldehyde - similarity: 0.50284\n",
      "Document Medical air - similarity: 0.50279\n",
      "Document Dioctyldimonium - similarity: 0.50278\n",
      "Document Riamet - similarity: 0.50277\n",
      "Document Silver cation - similarity: 0.50276\n",
      "Document Protionamide - similarity: 0.50274\n",
      "Document Methyl salicylate 2-ethylbutyrate - similarity: 0.50273\n",
      "Document Avacopan - similarity: 0.50269\n",
      "Document Chlorphenoxamine - similarity: 0.50261\n",
      "Document (R)-Fluoxetine - similarity: 0.50261\n",
      "Document Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.50257\n",
      "Document Fluoxymesterone - similarity: 0.50253\n",
      "Document Chlordiazepoxide - similarity: 0.50237\n",
      "Document Vildagliptin - similarity: 0.50235\n",
      "Document Neisseria meningitidis group A capsular polysaccharide tetanus toxoid conjugate antigen - similarity: 0.50234\n",
      "Document Polyethylene glycol 300 - similarity: 0.50225\n",
      "Document Cholic Acid - similarity: 0.50221\n",
      "Document 2,2'-{[9-(HYDROXYIMINO)-9H-FLUORENE-2,7-DIYL]BIS(OXY)}DIACETIC ACID - similarity: 0.50215\n",
      "Document (1R,4S,7AS)-1-(1-FORMYLPROP-1-EN-1-YL)-4-METHOXY-2,4,5,6,7,7A-HEXAHYDRO-1H-ISOINDOLE-3-CARBOXYLIC ACID - similarity: 0.50213\n",
      "Document Fenfluramine - similarity: 0.50199\n",
      "Document Lanabecestat - similarity: 0.50198\n",
      "Document Petesicatib - similarity: 0.50197\n",
      "Document 4-Nitro-Inden-1-One - similarity: 0.50182\n",
      "Document Diethyl phosphonate - similarity: 0.50176\n",
      "Document ACT-451840 - similarity: 0.50173\n",
      "Document Galantamine - similarity: 0.50170\n",
      "Document 2'-cytidylic acid - similarity: 0.50166\n",
      "Document Olmutinib - similarity: 0.50155\n",
      "Document Isothipendyl - similarity: 0.50149\n",
      "Document Cyclopentolate - similarity: 0.50133\n",
      "Document Pritelivir - similarity: 0.50131\n",
      "Document N-[(benzyloxy)carbonyl]-L-leucyl-N-[(1S)-3-fluoro-1-(4-hydroxybenzyl)-2-oxopropyl]-L-leucinamide - similarity: 0.50122\n",
      "Document LM-609 - similarity: 0.50117\n",
      "Document Northern pike - similarity: 0.50117\n",
      "Document Brequinar - similarity: 0.50116\n",
      "Document (6AR,11AS,11BR)-10-ACETYL-9-HYDROXY-7,7-DIMETHYL-2,6,6A,7,11A,11B-HEXAHYDRO-11H-PYRROLO[1',2':2,3]ISOINDOLO[4,5,6-CD]INDOL-11-ONE - similarity: 0.50112\n",
      "Document Inolimomab - similarity: 0.50110\n",
      "Document Vestronidase alfa - similarity: 0.50095\n",
      "Document Telisotuzumab vedotin - similarity: 0.50085\n",
      "Document Dexrazoxane - similarity: 0.50082\n",
      "Document Vorsetuzumab mafodotin - similarity: 0.50076\n",
      "Document Temazepam acetate - similarity: 0.50069\n",
      "Document Reloxaliase - similarity: 0.50059\n",
      "Document (R)-(+) 5(9BH)-OXO-9B-PHENYL-2,3-DIHYDROTHIAZOLO[2,3-A]ISOINDOL-3-CARBOXYLIC ACID METHYL ESTER - similarity: 0.50055\n",
      "Document ATB-346 - similarity: 0.50051\n",
      "Document BMS-911543 - similarity: 0.50048\n",
      "Document Capromorelin - similarity: 0.50041\n",
      "Document Meldonium - similarity: 0.50031\n",
      "Document Naquotinib - similarity: 0.50013\n",
      "Document Mannose - similarity: 0.50006\n",
      "Document Glyceryl stearate SE - similarity: 0.50003\n",
      "Document Acetic acid - similarity: 0.50002\n",
      "Document SCH-900271 - similarity: 0.49993\n",
      "Document CEP-1347 - similarity: 0.49992\n",
      "Document 2-METHYLTHIO-N6-ISOPENTENYL-ADENOSINE-5'-MONOPHOSPHATE - similarity: 0.49991\n",
      "Document Carlumab - similarity: 0.49982\n",
      "Document Panax ginseng whole - similarity: 0.49970\n",
      "Document Fluzoparib - similarity: 0.49968\n",
      "Document 99mTc-14 F7 Mab - similarity: 0.49958\n",
      "Document Selenium - similarity: 0.49952\n",
      "Document Phenylacetic acid - similarity: 0.49951\n",
      "Document Aminophylline - similarity: 0.49947\n",
      "Document Dimethyl sulfoxide - similarity: 0.49946\n",
      "Document Curcumin sulfate - similarity: 0.49940\n",
      "Document Protamine sulfate - similarity: 0.49933\n",
      "Document Misonidazole - similarity: 0.49930\n",
      "Document Eucalyptus globulus pollen - similarity: 0.49921\n",
      "Document Grapefruit oil - similarity: 0.49916\n",
      "Document Amaranthus spinosus pollen - similarity: 0.49916\n",
      "Document Simendan - similarity: 0.49915\n",
      "Document 2',3'-Dideoxyadenosine triphosphate - similarity: 0.49907\n",
      "Document Arginine - similarity: 0.49904\n",
      "Document Atlantic salmon - similarity: 0.49897\n",
      "Document Radiprodil - similarity: 0.49897\n",
      "Document Ulmus pumila pollen - similarity: 0.49889\n",
      "Document Oxyquinoline - similarity: 0.49888\n",
      "Document NCX 701 - similarity: 0.49885\n",
      "Document 5-Imino-4-(2-trifluoromethyl-phenylazo)-5H-pyrazol-3-ylamine - similarity: 0.49879\n",
      "Document Thiocoumarin - similarity: 0.49867\n",
      "Document TD-139 - similarity: 0.49865\n",
      "Document Morphine - similarity: 0.49861\n",
      "Document Fenpropidin - similarity: 0.49853\n",
      "Document Propiverine - similarity: 0.49840\n",
      "Document 1-TERT-BUTYL-3-(2,5-DIMETHYLBENZYL)-1H-PYRAZOLO[3,4-D]PYRIMIDIN-4-AMINE - similarity: 0.49835\n",
      "Document (2S)-2,3-bis{[(3R,7R)-3,7,11,15-tetramethylhexadecyl]oxy}propyl (2R)-2-hydroxy-3-[(hydroxyphosphinato)oxy]propyl phosphate - similarity: 0.49832\n",
      "Document Fosmanogepix - similarity: 0.49831\n",
      "Document Stearic acid - similarity: 0.49827\n",
      "Document Gitoformate - similarity: 0.49820\n",
      "Document 4-(4-chlorobenzyl)-1-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)piperidin-4-aminium - similarity: 0.49810\n",
      "Document [[CYCLOHEXANESULFONYL-GLYCYL]-3[PYRIDIN-4-YL-AMINOMETHYL]ALANYL]PIPERIDINE - similarity: 0.49807\n",
      "Document Propylthiouracil - similarity: 0.49804\n",
      "Document Levodopa - similarity: 0.49801\n",
      "Document Elsiglutide - similarity: 0.49799\n",
      "Document Dicloxacillin - similarity: 0.49799\n",
      "Document Cadmium - similarity: 0.49797\n",
      "Document Emapunil - similarity: 0.49794\n",
      "Document Letermovir - similarity: 0.49792\n",
      "Document Incadronic acid - similarity: 0.49785\n",
      "Document Triciribine - similarity: 0.49784\n",
      "Document Vespula maculifrons venom protein - similarity: 0.49779\n",
      "Document Populus nigra pollen - similarity: 0.49777\n",
      "Document Cyamemazine - similarity: 0.49775\n",
      "Document 3-[5-(1H-IMIDAZOL-1-YL)-7-METHYL-1H-BENZIMIDAZOL-2-YL]-4-[(PYRIDIN-2-YLMETHYL)AMINO]PYRIDIN-2(1H)-ONE - similarity: 0.49774\n",
      "Document Alpha-Benzyl-Aminobenzyl-Phosphonic Acid - similarity: 0.49773\n",
      "Document N-{2,2-DIFLUORO-2-[(2R)-PIPERIDIN-2-YL]ETHYL}-2-[2-(1H-1,2,4-TRIAZOL-1-YL)BENZYL][1,3]OXAZOLO[4,5-C]PYRIDIN-4-AMINE - similarity: 0.49769\n",
      "Document Hexafluoroacetone Hydrate - similarity: 0.49765\n",
      "Document Picoplatin - similarity: 0.49760\n",
      "Document CR665 - similarity: 0.49759\n",
      "Document Enalaprilat - similarity: 0.49756\n",
      "Document D-gluconhydroximo-1,5-lactam - similarity: 0.49751\n",
      "Document 2'-Deoxycytidine 5'-triphosphate - similarity: 0.49748\n",
      "Document Tiapride - similarity: 0.49747\n",
      "Document Arecoline - similarity: 0.49746\n",
      "Document Larix sibirica wood - similarity: 0.49738\n",
      "Document Apple - similarity: 0.49732\n",
      "Document Hexamidine - similarity: 0.49728\n",
      "Document Deoxycholic acid - similarity: 0.49714\n",
      "Document Fenoxazoline - similarity: 0.49709\n",
      "Document Ansofaxine - similarity: 0.49705\n",
      "Document Lactitol - similarity: 0.49700\n",
      "Document Levomenol - similarity: 0.49695\n",
      "Document Hydroxyethyl Starch - similarity: 0.49695\n",
      "Document S-[(2E)-3,7-DIMETHYLOCTA-2,6-DIENYL] TRIHYDROGENTHIODIPHOSPHATE - similarity: 0.49694\n",
      "Document TD-8954 - similarity: 0.49686\n",
      "Document Bioallethrin - similarity: 0.49679\n",
      "Document Parnaparin - similarity: 0.49677\n",
      "Document Ecallantide - similarity: 0.49664\n",
      "Document Haloperidol - similarity: 0.49654\n",
      "Document Brovanexine - similarity: 0.49653\n",
      "Document Alcaftadine - similarity: 0.49651\n",
      "Document {4-[(2R)-pyrrolidin-2-ylmethoxy]phenyl}(4-thiophen-3-ylphenyl)methanone - similarity: 0.49647\n",
      "Document Alicaforsen - similarity: 0.49641\n",
      "Document Chlorhexidine - similarity: 0.49639\n",
      "Document Iopanoic acid - similarity: 0.49617\n",
      "Document Triptolide - similarity: 0.49616\n",
      "Document Parsley - similarity: 0.49613\n",
      "Document Lupeol - similarity: 0.49607\n",
      "Document 1-Amino-2,3-Dihydroxy-5-Hydroxymethyl Cyclohex-5-Ene - similarity: 0.49606\n",
      "Document Pineapple - similarity: 0.49604\n",
      "Document Mustard greens - similarity: 0.49597\n",
      "Document Iothalamic acid - similarity: 0.49594\n",
      "Document Nicotinamide riboside - similarity: 0.49593\n",
      "Document Hexamethonium - similarity: 0.49593\n",
      "Document Spebrutinib - similarity: 0.49585\n",
      "Document Etifoxine - similarity: 0.49585\n",
      "Document Betamethasone - similarity: 0.49582\n",
      "Document Iloprost - similarity: 0.49578\n",
      "Document Xibornol - similarity: 0.49572\n",
      "Document Quinolinic Acid - similarity: 0.49569\n",
      "Document REV131 - similarity: 0.49567\n",
      "Document [(3R,4S)-4-HYDROXY-3-METHYL-2-OXOHEXYL]PHOSPHONIC ACID - similarity: 0.49563\n",
      "Document Paspalum notatum pollen - similarity: 0.49563\n",
      "Document Setiptiline - similarity: 0.49563\n",
      "Document Flufenoxuron - similarity: 0.49561\n",
      "Document Evernia prunastri - similarity: 0.49550\n",
      "Document Apararenone - similarity: 0.49547\n",
      "Document Gamithromycin - similarity: 0.49537\n",
      "Document Ramucirumab - similarity: 0.49532\n",
      "Document Trimetrexate - similarity: 0.49529\n",
      "Document Serabelisib - similarity: 0.49526\n",
      "Document O-[2-(1,3-dioxo-1,3-dihydro-2H-isoindol-2-yl)ethyl] (4-iodophenyl)thiocarbamate - similarity: 0.49525\n",
      "Document Phytosphingosine - similarity: 0.49520\n",
      "Document Depatuxizumab - similarity: 0.49515\n",
      "Document 2-HYDROXY-5-{[(1E)-2-PHENYLETHYLIDENE]AMINO}-L-TYROSINE - similarity: 0.49508\n",
      "Document Esatenolol - similarity: 0.49505\n",
      "Document PKI-179 - similarity: 0.49502\n",
      "Document Colloidal oatmeal - similarity: 0.49501\n",
      "Document Tavapadon - similarity: 0.49495\n",
      "Document Reynoutria multiflora root - similarity: 0.49492\n",
      "Document Tuaminoheptane - similarity: 0.49491\n",
      "Document Lonicera confusa flower - similarity: 0.49482\n",
      "Document Elocalcitol - similarity: 0.49479\n",
      "Document Anisindione - similarity: 0.49479\n",
      "Document Blatella germanica - similarity: 0.49475\n",
      "Document Influenza A virus A/Texas/50/2012 X-223A (H3N2) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.49473\n",
      "Document Furazolidone - similarity: 0.49472\n",
      "Document Isoformononetin - similarity: 0.49471\n",
      "Document Lavender oil - similarity: 0.49467\n",
      "Document Uridine triacetate - similarity: 0.49467\n",
      "Document 3-ETHYL-2-(4-HYDROXYPHENYL)-2H-INDAZOL-5-OL - similarity: 0.49466\n",
      "Document Elezanumab - similarity: 0.49463\n",
      "Document Renzapride - similarity: 0.49455\n",
      "Document Decitabine - similarity: 0.49449\n",
      "Document Umifenovir - similarity: 0.49447\n",
      "Document Nusinersen - similarity: 0.49441\n",
      "Document BMS-986094 - similarity: 0.49440\n",
      "Document VX-765 - similarity: 0.49440\n",
      "Document Tegafur-uracil - similarity: 0.49440\n",
      "Document Ofatumumab - similarity: 0.49435\n",
      "Document (2R)-2-({6-[benzyl(methyl)amino]-9-isopropyl-9H-purin-2-yl}amino)butan-1-ol - similarity: 0.49419\n",
      "Document gamma-Aminobutyric acid - similarity: 0.49418\n",
      "Document GSK-3117391 - similarity: 0.49414\n",
      "Document AV-412 - similarity: 0.49411\n",
      "Document Mometasone furoate - similarity: 0.49404\n",
      "Document Bentazepam - similarity: 0.49404\n",
      "Document Dianicline - similarity: 0.49397\n",
      "Document Chloramphenicol - similarity: 0.49385\n",
      "Document Bavituximab - similarity: 0.49381\n",
      "Document Lofentanil - similarity: 0.49377\n",
      "Document Sodium hydroxide - similarity: 0.49373\n",
      "Document Myrothecium verrucaria - similarity: 0.49373\n",
      "Document Lemuteporfin - similarity: 0.49373\n",
      "Document MBX-2982 - similarity: 0.49373\n",
      "Document Platanus hybrida pollen - similarity: 0.49371\n",
      "Document Angelica dahurica root - similarity: 0.49366\n",
      "Document Metopimazine - similarity: 0.49366\n",
      "Document Melatonin - similarity: 0.49364\n",
      "Document NS-018 - similarity: 0.49356\n",
      "Document 2,3-Dicarboxy-4-(2-Chloro-Phenyl)-1-Ethyl-5-Isopropoxycarbonyl-6-Methyl-Pyridinium - similarity: 0.49352\n",
      "Document {3-[(3-Hydroxy-2-Methyl-5-Phosphonooxymethyl-Pyridin-4-Ylmethyl)-Amino]-2-Methyl-Propyl}-Phosphonic Acid - similarity: 0.49342\n",
      "Document 4-Thio-beta-D-glucopyranose - similarity: 0.49333\n",
      "Document Alminoprofen - similarity: 0.49324\n",
      "Document Octasulfur - similarity: 0.49320\n",
      "Document Flunarizine - similarity: 0.49319\n",
      "Document Cergutuzumab amunaleukin - similarity: 0.49313\n",
      "Document Tris(Hydroxyethyl)Aminomethane - similarity: 0.49306\n",
      "Document Niobium - similarity: 0.49297\n",
      "Document 1D09C3 - similarity: 0.49290\n",
      "Document (1S,7S,8S,8AR)-1,2,3,7,8,8A-HEXAHYDRO-7-METHYL-8-[2-[(2R,4R)-TETRAHYDRO-4-HY DROXY-6-OXO-2H-PYRAN-2-YL]ETHYL]-1-NAPHTHALENOL - similarity: 0.49289\n",
      "Document 5-Iodo-2'-Deoxyuridine-5'-Monophosphate - similarity: 0.49280\n",
      "Document Myristalkonium - similarity: 0.49276\n",
      "Document Microascus brevicaulis - similarity: 0.49273\n",
      "Document Pentosan polysulfate - similarity: 0.49273\n",
      "Document Blinatumomab - similarity: 0.49272\n",
      "Document Mosunetuzumab - similarity: 0.49267\n",
      "Document 3-(5-{[4-(AMINOMETHYL)PIPERIDIN-1-YL]METHYL}-1H-INDOL-2-YL)QUINOLIN-2(1H)-ONE - similarity: 0.49266\n",
      "Document Theobromine - similarity: 0.49265\n",
      "Document Influenza A virus A/Texas/50/2012 X-223A (H3N2) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.49261\n",
      "Document Dextromethadone - similarity: 0.49255\n",
      "Document GSK-1004723 - similarity: 0.49244\n",
      "Document Monoisopropylphosphorylserine - similarity: 0.49242\n",
      "Document Coconut oil - similarity: 0.49231\n",
      "Document Cerium oxalate - similarity: 0.49231\n",
      "Document Racivir - similarity: 0.49227\n",
      "Document 2-(3-GUANIDINOPHENYL)-3-MERCAPTOPROPANOIC ACID - similarity: 0.49222\n",
      "Document beta-D-Galactosamine - similarity: 0.49221\n",
      "Document LCL-161 - similarity: 0.49219\n",
      "Document Encainide - similarity: 0.49212\n",
      "Document 6-chloro-N-pyrimidin-5-yl-3-{[3-(trifluoromethyl)phenyl]amino}-1,2-benzisoxazole-7-carboxamide - similarity: 0.49203\n",
      "Document Protamine - similarity: 0.49183\n",
      "Document Corticorelin - similarity: 0.49181\n",
      "Document ISOTHIAZOLIDINONE ANALOG - similarity: 0.49173\n",
      "Document tert-butanol - similarity: 0.49160\n",
      "Document Cyd dengue virus serotype 1 live (attenuated) antigen - similarity: 0.49150\n",
      "Document Lanolin - similarity: 0.49142\n",
      "Document (5S)-3-ANILINO-5-(2,4-DIFLUOROPHENYL)-5-METHYL-1,3-OXAZOLIDINE-2,4-DIONE - similarity: 0.49141\n",
      "Document Isoliquiritigenin - similarity: 0.49140\n",
      "Document Belinostat - similarity: 0.49136\n",
      "Document Chloroquine - similarity: 0.49133\n",
      "Document Collagenase clostridium histolyticum - similarity: 0.49132\n",
      "Document S-23 - similarity: 0.49130\n",
      "Document Carboxyatractyloside - similarity: 0.49130\n",
      "Document Mannitol busulfan - similarity: 0.49121\n",
      "Document Technetium Tc-99m epratuzumab - similarity: 0.49113\n",
      "Document Influenza B virus B/Phuket/3073/2013 hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.49112\n",
      "Document Ganaxolone - similarity: 0.49107\n",
      "Document 5-Bromo-2'-deoxyuridine 5'-(dihydrogen phosphate) - similarity: 0.49104\n",
      "Document PF-05089771 - similarity: 0.49103\n",
      "Document Suvorexant - similarity: 0.49094\n",
      "Document Clocortolone acetate - similarity: 0.49083\n",
      "Document Sodium lauryl sulfoacetate - similarity: 0.49073\n",
      "Document Pamrevlumab - similarity: 0.49071\n",
      "Document Halofantrine - similarity: 0.49067\n",
      "Document Tipiracil - similarity: 0.49065\n",
      "Document Penicillium expansum - similarity: 0.49055\n",
      "Document Gestodene - similarity: 0.49053\n",
      "Document Sodium glycerophosphate - similarity: 0.49053\n",
      "Document L-2-amino-3-butynoic acid - similarity: 0.49049\n",
      "Document Sulfatide - similarity: 0.49036\n",
      "Document Methoxyflurane - similarity: 0.49029\n",
      "Document Zalcitabine - similarity: 0.49025\n",
      "Document Ethinylestradiol - similarity: 0.49021\n",
      "Document Trifluoroethyl acetate - similarity: 0.49019\n",
      "Document Sodium carbonate - similarity: 0.49015\n",
      "Document Ribose - similarity: 0.49014\n",
      "Document 2,6-diamino-1,7-dihydro-8H-imidazo[4,5-g]quinazolin-8-one - similarity: 0.49008\n",
      "Document Liarozole - similarity: 0.49002\n",
      "Document Leuprolide - similarity: 0.49000\n",
      "Document BMS-777607 - similarity: 0.48992\n",
      "Document Arformoterol - similarity: 0.48985\n",
      "Document Chlorprothixene - similarity: 0.48984\n",
      "Document Duvelisib - similarity: 0.48980\n",
      "Document Lolium perenne pollen - similarity: 0.48977\n",
      "Document Talinolol - similarity: 0.48975\n",
      "Document Telisotuzumab - similarity: 0.48968\n",
      "Document Ethyl (2E)-4-({(2S)-2-[(N-{[(2-methyl-2-propanyl)oxy]carbonyl}-L-valyl)amino]-2-phenylacetyl}amino)-5-[(3S)-2-oxo-3-pyrrolidinyl]-2-pentenoate - similarity: 0.48953\n",
      "Document N-[1-(AMINOMETHYL)CYCLOPROPYL]-3-(MORPHOLIN-4-YLSULFONYL)-N~2~-[(1S)-2,2,2-TRIFLUORO-1-(4-FLUOROPHENYL)ETHYL]-L-ALANINAMIDE - similarity: 0.48948\n",
      "Document Hypericin - similarity: 0.48936\n",
      "Document Aminoglutethimide - similarity: 0.48936\n",
      "Document (2E,4E)-2-Hydroxy-6-oxo-6-phenyl-2,4-hexadienoic acid - similarity: 0.48929\n",
      "Document Flunisolide acetate - similarity: 0.48921\n",
      "Document Nicotinamide - similarity: 0.48920\n",
      "Document Miconazole - similarity: 0.48919\n",
      "Document Dichromate - similarity: 0.48915\n",
      "Document Fe-Mesopone - similarity: 0.48910\n",
      "Document Incyclinide - similarity: 0.48907\n",
      "Document Biomed 101 - similarity: 0.48907\n",
      "Document Methylnaltrexone - similarity: 0.48907\n",
      "Document Fumagillin - similarity: 0.48904\n",
      "Document Propofol - similarity: 0.48903\n",
      "Document Macitentan - similarity: 0.48899\n",
      "Document Decamethonium - similarity: 0.48897\n",
      "Document Amiodarone - similarity: 0.48888\n",
      "Document Fomepizole - similarity: 0.48877\n",
      "Document Pemafibrate - similarity: 0.48871\n",
      "Document GSK-239512 - similarity: 0.48870\n",
      "Document SB-269970 - similarity: 0.48863\n",
      "Document RU84687 - similarity: 0.48860\n",
      "Document 5-(hydroxymethyl)-2-furaldehyde - similarity: 0.48852\n",
      "Document (R)-1-(4-(4-(Hydroxymethyl)-1,3,2-dioxaborolan-2-YL)phenethyl)guanidine - similarity: 0.48850\n",
      "Document Sulisobenzone - similarity: 0.48844\n",
      "Document Bintrafusp alfa - similarity: 0.48842\n",
      "Document Firocoxib - similarity: 0.48839\n",
      "Document Lyme disease vaccine (recombinant OspA) - similarity: 0.48838\n",
      "Document (3Z,5S,6R,7S,8R,8aS)-3-(octylimino)hexahydro[1,3]thiazolo[3,4-a]pyridine-5,6,7,8-tetrol - similarity: 0.48838\n",
      "Document Choline magnesium trisalicylate - similarity: 0.48827\n",
      "Document Dabrafenib - similarity: 0.48826\n",
      "Document Aluminum sulfate - similarity: 0.48824\n",
      "Document Laninamivir octanoate - similarity: 0.48817\n",
      "Document SR-9009 - similarity: 0.48814\n",
      "Document Pegvisomant - similarity: 0.48811\n",
      "Document 5-[(5-fluoro-3-methyl-1H-indazol-4-yl)oxy]benzene-1,3-dicarbonitrile - similarity: 0.48803\n",
      "Document Piceatannol - similarity: 0.48803\n",
      "Document Avadomide - similarity: 0.48802\n",
      "Document Perflubron - similarity: 0.48795\n",
      "Document CC-223 - similarity: 0.48790\n",
      "Document Cyclohexyl-pentyl-maltoside - similarity: 0.48786\n",
      "Document Karaya gum - similarity: 0.48773\n",
      "Document 4-[(6-Amino-4-Pyrimidinyl)Amino]Benzenesulfonamide - similarity: 0.48772\n",
      "Document Acetylcysteine zinc - similarity: 0.48770\n",
      "Document PS386113 - similarity: 0.48768\n",
      "Document LS11 - similarity: 0.48763\n",
      "Document Quinoline Yellow WS - similarity: 0.48758\n",
      "Document Indolepropionic acid - similarity: 0.48756\n",
      "Document Dichlorobenzene - similarity: 0.48751\n",
      "Document 5-{[1-(2-fluorobenzyl)piperidin-4-yl]methoxy}quinazoline-2,4-diamine - similarity: 0.48747\n",
      "Document PLX-4720 - similarity: 0.48745\n",
      "Document Pirlindole - similarity: 0.48742\n",
      "Document Coffee bean - similarity: 0.48735\n",
      "Document Phenylalanylmethylchloride - similarity: 0.48724\n",
      "Document Indanazoline - similarity: 0.48723\n",
      "Document Ferric derisomaltose - similarity: 0.48720\n",
      "Document Nz-(Dicarboxymethyl)Lysine - similarity: 0.48716\n",
      "Document Amaranthus palmeri pollen - similarity: 0.48716\n",
      "Document Benzthiazide - similarity: 0.48710\n",
      "Document Lintitript - similarity: 0.48703\n",
      "Document Meclizine - similarity: 0.48701\n",
      "Document Isoxaflutole - similarity: 0.48695\n",
      "Document Thymidine monophosphate - similarity: 0.48691\n",
      "Document Seribantumab - similarity: 0.48688\n",
      "Document Hypoxanthine - similarity: 0.48684\n",
      "Document Rozrolimupab - similarity: 0.48683\n",
      "Document Olaflur - similarity: 0.48680\n",
      "Document Picric acid - similarity: 0.48660\n",
      "Document Ciliary neurotrophic factor - similarity: 0.48658\n",
      "Document Stemphylium sarciniforme - similarity: 0.48652\n",
      "Document (1S,2R,3S,4R,5S)-8-AZABICYCLO[3.2.1]OCTANE-1,2,3,4-TETROL - similarity: 0.48649\n",
      "Document Pyrazolam - similarity: 0.48648\n",
      "Document 6-amino-2-[(thiophen-2-ylmethyl)amino]-1,7-dihydro-8H-imidazo[4,5-g]quinazolin-8-one - similarity: 0.48642\n",
      "Document Elacridar - similarity: 0.48637\n",
      "Document Cladosporium cladosporioides - similarity: 0.48629\n",
      "Document Otenabant - similarity: 0.48629\n",
      "Document Methylcellulose - similarity: 0.48627\n",
      "Document Neisseria meningitidis serogroup b nada protein antigen - similarity: 0.48627\n",
      "Document 4-[(1S)-1-(3-fluoro-4-methoxyphenyl)-2-(2-methoxy-5-nitrophenyl)ethyl]-1H-imidazol-2-amine - similarity: 0.48624\n",
      "Document N-(biphenyl-4-ylsulfonyl)-D-leucine - similarity: 0.48623\n",
      "Document Grapefruit - similarity: 0.48620\n",
      "Document Diaminopropanol tetraacetic acid - similarity: 0.48612\n",
      "Document Bezafibrate - similarity: 0.48584\n",
      "Document Influenza B virus B/Phuket/3073/2013 hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.48583\n",
      "Document Riferminogene pecaplasmid - similarity: 0.48579\n",
      "Document Capravirine - similarity: 0.48575\n",
      "Document Cyclomethicone 5 - similarity: 0.48573\n",
      "Document Pirfenidone - similarity: 0.48569\n",
      "Document Influenza A virus A/Brisbane/59/2007 (H1N1) antigen (formaldehyde inactivated) - similarity: 0.48564\n",
      "Document Phenyl-uridine-5'-diphosphate - similarity: 0.48564\n",
      "Document Dihydroxyphenylglycine - similarity: 0.48561\n",
      "Document Namilumab - similarity: 0.48553\n",
      "Document Valaciclovir - similarity: 0.48550\n",
      "Document Selenomethionine Se-75 - similarity: 0.48540\n",
      "Document Trans-O-Hydroxy-Alpha-Methyl Cinnamate - similarity: 0.48538\n",
      "Document Adenosine Phosphonoacetic Acid - similarity: 0.48532\n",
      "Document 4-Nitrophenyl-Ara - similarity: 0.48527\n",
      "Document 2,6-Diamino-8-Propylsulfanylmethyl-3h-Quinazoline-4-One - similarity: 0.48526\n",
      "Document Cidofovir - similarity: 0.48520\n",
      "Document Bosutinib - similarity: 0.48501\n",
      "Document Clocapramine - similarity: 0.48500\n",
      "Document Ajulemic acid - similarity: 0.48493\n",
      "Document Cesium - similarity: 0.48492\n",
      "Document (S)-econazole - similarity: 0.48485\n",
      "Document Acriflavinium chloride - similarity: 0.48483\n",
      "Document Asparaginase Erwinia chrysanthemi - similarity: 0.48477\n",
      "Document Cilnidipine - similarity: 0.48475\n",
      "Document Cobalt hexammine ion - similarity: 0.48475\n",
      "Document Dimethicone 410 - similarity: 0.48466\n",
      "Document Hydroxypropyl cellulose - similarity: 0.48461\n",
      "Document Ifenprodil - similarity: 0.48461\n",
      "Document Taspoglutide - similarity: 0.48456\n",
      "Document 13-deoxydoxorubicin - similarity: 0.48441\n",
      "Document Tofacitinib - similarity: 0.48441\n",
      "Document Influenza B virus B/Massachusetts/2/2012 BX-51B hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.48440\n",
      "Document Tinidazole - similarity: 0.48438\n",
      "Document 3-({[3,5-DIFLUORO-3'-(TRIFLUOROMETHOXY)BIPHENYL-4-YL]AMINO}CARBONYL)THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.48437\n",
      "Document Mito-4509 - similarity: 0.48434\n",
      "Document N-[(4-methoxyphenyl)sulfonyl]-D-alanine - similarity: 0.48433\n",
      "Document Naratriptan - similarity: 0.48410\n",
      "Document Quercetin - similarity: 0.48397\n",
      "Document N-[2-({[amino(imino)methyl]amino}oxy)ethyl]-2-{6-chloro-3-[(2,2-difluoro-2-phenylethyl)amino]-2-fluorophenyl}acetamide - similarity: 0.48395\n",
      "Document Indole-3-Propanol Phosphate - similarity: 0.48394\n",
      "Document N6-ISOPENTENYL-ADENOSINE-5'-MONOPHOSPHATE - similarity: 0.48389\n",
      "Document Lanicemine - similarity: 0.48382\n",
      "Document Quinidine - similarity: 0.48382\n",
      "Document Quercus nigra pollen - similarity: 0.48380\n",
      "Document (1-HYDROXYHEPTANE-1,1-DIYL)BIS(PHOSPHONIC ACID) - similarity: 0.48376\n",
      "Document Imlifidase - similarity: 0.48374\n",
      "Document 9-(2-carboxyethyl)-10-methylanthracene endoperoxide - similarity: 0.48370\n",
      "Document Levoglucose - similarity: 0.48367\n",
      "Document Pabinafusp alfa - similarity: 0.48366\n",
      "Document Carpinus caroliniana pollen - similarity: 0.48364\n",
      "Document Neratinib - similarity: 0.48360\n",
      "Document Oxolinic acid - similarity: 0.48354\n",
      "Document Hyodeoxycholic Acid - similarity: 0.48353\n",
      "Document Lutein - similarity: 0.48352\n",
      "Document Iniparib - similarity: 0.48348\n",
      "Document Allantoin - similarity: 0.48346\n",
      "Document Citric acid - similarity: 0.48344\n",
      "Document Lefitolimod - similarity: 0.48336\n",
      "Document Nifuratel - similarity: 0.48335\n",
      "Document Cavosonstat - similarity: 0.48330\n",
      "Document Ozoralizumab - similarity: 0.48322\n",
      "Document Indinavir - similarity: 0.48321\n",
      "Document MK-7622 - similarity: 0.48288\n",
      "Document Acetyl simvastatin - similarity: 0.48287\n",
      "Document Idanpramine - similarity: 0.48284\n",
      "Document Dexetimide - similarity: 0.48279\n",
      "Document Influenza A virus A/South australia/34/2019 IVR-197 (H3N2) antigen (formaldehyde inactivated) - similarity: 0.48278\n",
      "Document Influenza A virus A/Brisbane/10/2010 (H1N1) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.48274\n",
      "Document 2,6-Diamino-Hexanoic Acid Amide - similarity: 0.48269\n",
      "Document Mechlorethamine - similarity: 0.48268\n",
      "Document [3,5-Dibromo-4-(4-Hydroxy-3-Phenethylcarbamoyl-Phenoxy)-Phenyl]-Acetic Acid - similarity: 0.48267\n",
      "Document Eucalyptus oil - similarity: 0.48255\n",
      "Document Pacific halibut - similarity: 0.48251\n",
      "Document Triciribine phosphate - similarity: 0.48249\n",
      "Document Ronopterin - similarity: 0.48248\n",
      "Document Poliglusam - similarity: 0.48239\n",
      "Document 1-(N-Imidazolyl)-2-Hydroxy-2-(2,3-Dichlorophenyl)Octane - similarity: 0.48234\n",
      "Document (R)-3-hydroxydecanoyl-CoA - similarity: 0.48231\n",
      "Document Melengestrol - similarity: 0.48231\n",
      "Document NBI-6024 - similarity: 0.48229\n",
      "Document V24343 - similarity: 0.48226\n",
      "Document Phenothrin - similarity: 0.48226\n",
      "Document Invert sugar - similarity: 0.48214\n",
      "Document Influenza A virus A/California/7/2009 X-181 (H1N1) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.48198\n",
      "Document Anfibatide - similarity: 0.48195\n",
      "Document Polihexanide - similarity: 0.48192\n",
      "Document Brostallicin - similarity: 0.48183\n",
      "Document Phosphonoacetohydroxamic Acid - similarity: 0.48178\n",
      "Document Avapritinib - similarity: 0.48177\n",
      "Document Vosoritide - similarity: 0.48174\n",
      "Document Tisagenlecleucel - similarity: 0.48168\n",
      "Document Fenbufen - similarity: 0.48149\n",
      "Document Netilmicin - similarity: 0.48148\n",
      "Document Paliperidone - similarity: 0.48142\n",
      "Document (20S)-19,20,21,22-TETRAHYDRO-19-OXO-5H-18,20-ETHANO-12,14-ETHENO-6,10-METHENO-18H-BENZ[D]IMIDAZO[4,3-K][1,6,9,12]OXATRIAZA-CYCLOOCTADECOSINE-9-CARBONITRILE - similarity: 0.48141\n",
      "Document PF-05105679 - similarity: 0.48141\n",
      "Document Fenoterol - similarity: 0.48140\n",
      "Document Doxifluridine - similarity: 0.48139\n",
      "Document N-[(2-AMINO-6-METHYLPYRIMIDIN-4-YL)METHYL]-3-{[(E)-(2-OXODIHYDROFURAN-3(2H)-YLIDENE)METHYL]AMINO}BENZENESULFONAMIDE - similarity: 0.48129\n",
      "Document Eptifibatide - similarity: 0.48118\n",
      "Document Etirinotecan pegol - similarity: 0.48117\n",
      "Document Sodium sulfide - similarity: 0.48103\n",
      "Document Cefradine - similarity: 0.48095\n",
      "Document Boscalid - similarity: 0.48089\n",
      "Document Salicylic acid - similarity: 0.48087\n",
      "Document Florantyrone - similarity: 0.48086\n",
      "Document Mebenil - similarity: 0.48084\n",
      "Document Leucanthemum vulgare pollen - similarity: 0.48079\n",
      "Document Ritodrine - similarity: 0.48078\n",
      "Document RG2417 - similarity: 0.48077\n",
      "Document Emetonium iodide - similarity: 0.48062\n",
      "Document 1-butanoyl-N-(4-carbamimidoylbenzyl)-L-prolinamide - similarity: 0.48062\n",
      "Document Mercaptomethyl Phosphonate - similarity: 0.48059\n",
      "Document Flavin-adenine dinucleotide-N5-isobutyl ketone - similarity: 0.48056\n",
      "Document Bezlotoxumab - similarity: 0.48053\n",
      "Document HYDROXY(OXO)(3-{[(2Z)-4-[3-(1H-1,2,4-TRIAZOL-1-YLMETHYL)PHENYL]PYRIMIDIN-2(5H)-YLIDENE]AMINO}PHENYL)AMMONIUM - similarity: 0.48045\n",
      "Document TAS-106 - similarity: 0.48042\n",
      "Document Etodolac - similarity: 0.48027\n",
      "Document Huperzine A - similarity: 0.48026\n",
      "Document Strawberry - similarity: 0.48025\n",
      "Document Salinomycin - similarity: 0.48024\n",
      "Document Trihexyphenidyl - similarity: 0.48020\n",
      "Document Insulin human - similarity: 0.48019\n",
      "Document Lisofylline - similarity: 0.48010\n",
      "Document 3-Methyladenine - similarity: 0.48002\n",
      "Document BOS172722 - similarity: 0.47997\n",
      "Document Dibutylsuccinate - similarity: 0.47993\n",
      "Document Phloxine B - similarity: 0.47991\n",
      "Document Pistachio - similarity: 0.47986\n",
      "Document Clarithromycin - similarity: 0.47982\n",
      "Document Pyrroloquinoline Quinone - similarity: 0.47976\n",
      "Document Rosiglitazone - similarity: 0.47976\n",
      "Document Erythritol - similarity: 0.47973\n",
      "Document Bisacodyl - similarity: 0.47971\n",
      "Document Symmetric dimethylarginine - similarity: 0.47969\n",
      "Document Omburtamab - similarity: 0.47966\n",
      "Document 1-Dodecanol - similarity: 0.47959\n",
      "Document Ipilimumab - similarity: 0.47958\n",
      "Document Cevimeline - similarity: 0.47958\n",
      "Document Methyl isocyanate - similarity: 0.47953\n",
      "Document D-Mannose 1-Phosphate - similarity: 0.47951\n",
      "Document 2-Naphthalenesulfonic acid - similarity: 0.47948\n",
      "Document Lactose - similarity: 0.47944\n",
      "Document Atrazine - similarity: 0.47943\n",
      "Document Alfacalcidol - similarity: 0.47943\n",
      "Document Clometocillin - similarity: 0.47932\n",
      "Document Tuna - similarity: 0.47926\n",
      "Document Deoxyuridine-5'-Diphosphate - similarity: 0.47925\n",
      "Document 2-acetylamino-2-deoxy-b-D-allopyranose - similarity: 0.47924\n",
      "Document Trilaciclib - similarity: 0.47922\n",
      "Document Tolmetin - similarity: 0.47920\n",
      "Document Phosphate ion P-32 - similarity: 0.47911\n",
      "Document 2-aminoethyl naphthalen-1-ylacetate - similarity: 0.47905\n",
      "Document Alisertib - similarity: 0.47902\n",
      "Document Mitometh - similarity: 0.47901\n",
      "Document Hypochlorite - similarity: 0.47899\n",
      "Document Tricyclazole - similarity: 0.47889\n",
      "Document 4-(4-HYDROXYPHENYL)-1-NAPHTHALDEHYDE OXIME - similarity: 0.47882\n",
      "Document Triheptanoin - similarity: 0.47878\n",
      "Document Dorzolamide - similarity: 0.47876\n",
      "Document 3-Deoxy-D-Glucosamine - similarity: 0.47873\n",
      "Document Thiocellobiose - similarity: 0.47865\n",
      "Document Acefylline - similarity: 0.47858\n",
      "Document Ethylene Dichloride - similarity: 0.47856\n",
      "Document Ecgonine - similarity: 0.47854\n",
      "Document Tegobuvir - similarity: 0.47853\n",
      "Document Fexofenadine - similarity: 0.47846\n",
      "Document Nelumbo nucifera flower - similarity: 0.47840\n",
      "Document N-(3-Phenyl-2-Sulfanylpropanoyl)Phenylalanylalanine - similarity: 0.47837\n",
      "Document (R)-Bicalutamide - similarity: 0.47834\n",
      "Document MUC1 Dendritic Cell Vaccine - similarity: 0.47834\n",
      "Document Bromhexine - similarity: 0.47822\n",
      "Document Mangifera indica pollen - similarity: 0.47821\n",
      "Document Carfentanil - similarity: 0.47820\n",
      "Document Tioguanine - similarity: 0.47816\n",
      "Document Sulfadicramide - similarity: 0.47807\n",
      "Document PF-06815345 - similarity: 0.47805\n",
      "Document Caramiphen - similarity: 0.47792\n",
      "Document Polydatin - similarity: 0.47792\n",
      "Document 5-Formyl-2'-deoxy-cytidine-5'-monophosphate - similarity: 0.47780\n",
      "Document Revamilast - similarity: 0.47769\n",
      "Document Abexinostat - similarity: 0.47754\n",
      "Document Obicetrapib - similarity: 0.47750\n",
      "Document 1-[2-AMINO-2-CYCLOHEXYL-ACETYL]-PYRROLIDINE-3-CARBOXYLIC ACID 5-CHLORO-2-(2-ETHYLCARBAMOYL-ETHOXY)-BENZYLAMIDE - similarity: 0.47733\n",
      "Document Equus caballus hair - similarity: 0.47729\n",
      "Document 2-(5-HYDROXY-NAPHTHALEN-1-YL)-1,3-BENZOOXAZOL-6-OL - similarity: 0.47727\n",
      "Document Triamterene - similarity: 0.47725\n",
      "Document Selegiline - similarity: 0.47723\n",
      "Document Nesiritide - similarity: 0.47723\n",
      "Document RU83876 - similarity: 0.47718\n",
      "Document Fludarabine - similarity: 0.47715\n",
      "Document Dipraglurant - similarity: 0.47715\n",
      "Document Homogentisic acid - similarity: 0.47713\n",
      "Document Estazolam - similarity: 0.47712\n",
      "Document Interleukin-7 - similarity: 0.47708\n",
      "Document Puccinia striiformis var. striiformis - similarity: 0.47705\n",
      "Document Tunicamycin - similarity: 0.47703\n",
      "Document D-galactohydroximo-1,5-lactam - similarity: 0.47699\n",
      "Document Remifentanil - similarity: 0.47695\n",
      "Document Alkaline Phosphatase - similarity: 0.47691\n",
      "Document N(2)-(2-carboxyethyl)-L-arginine - similarity: 0.47680\n",
      "Document Macadamia seed oil glycereth-8 esters - similarity: 0.47679\n",
      "Document 2-(TOLUENE-4-SULFONYL)-2H-BENZO[D][1,2,3]DIAZABORININ-1-OL - similarity: 0.47679\n",
      "Document Diglyme - similarity: 0.47675\n",
      "Document Nifurtimox - similarity: 0.47674\n",
      "Document Gliclazide - similarity: 0.47673\n",
      "Document Sulfatolamide - similarity: 0.47669\n",
      "Document Modified Ribosylated Glutamyl Ester - similarity: 0.47665\n",
      "Document PCL-016 - similarity: 0.47660\n",
      "Document Gantenerumab - similarity: 0.47655\n",
      "Document (S)-Warfarin - similarity: 0.47652\n",
      "Document 7,8-dihydroinosine - similarity: 0.47650\n",
      "Document Typhonium giganteum whole - similarity: 0.47649\n",
      "Document Tianeptine - similarity: 0.47643\n",
      "Document INO-1001 - similarity: 0.47638\n",
      "Document Tea leaf - similarity: 0.47637\n",
      "Document Safinamide - similarity: 0.47636\n",
      "Document TB-402 - similarity: 0.47632\n",
      "Document Dodecyl sulfate - similarity: 0.47629\n",
      "Document 1,2-Di-N-Pentanoyl-Sn-Glycero-3-Dithiophosphocholine - similarity: 0.47625\n",
      "Document Atractylodes lancea root oil - similarity: 0.47625\n",
      "Document (7S)-2-(2-aminopyrimidin-4-yl)-7-(2-fluoroethyl)-1,5,6,7-tetrahydro-4H-pyrrolo[3,2-c]pyridin-4-one - similarity: 0.47625\n",
      "Document Icosapent - similarity: 0.47625\n",
      "Document Mepyramine - similarity: 0.47625\n",
      "Document BMS-908662 - similarity: 0.47613\n",
      "Document L-2-Amino-6-Methylene-Pimelic Acid - similarity: 0.47601\n",
      "Document PD-168393 - similarity: 0.47594\n",
      "Document Imino-Tryptophan - similarity: 0.47587\n",
      "Document Iodixanol - similarity: 0.47584\n",
      "Document Cathine - similarity: 0.47572\n",
      "Document Sodium tetradecyl sulfate - similarity: 0.47569\n",
      "Document Cimicoxib - similarity: 0.47569\n",
      "Document Esmolol - similarity: 0.47558\n",
      "Document 4-Methyl-N-[5-(5-methyl-furan-2-ylmethylene)-4-oxo-thiazolidin-2-ylidene]-benzenesulfonamide - similarity: 0.47557\n",
      "Document PF-06273340 - similarity: 0.47556\n",
      "Document Beroctocog alfa - similarity: 0.47556\n",
      "Document Broussonetia papyrifera pollen - similarity: 0.47551\n",
      "Document Resveratrol - similarity: 0.47541\n",
      "Document Eucalyptus gum - similarity: 0.47539\n",
      "Document Sesame oil - similarity: 0.47531\n",
      "Document Candicidin - similarity: 0.47526\n",
      "Document MRTX849 - similarity: 0.47524\n",
      "Document Fozivudine Tidoxil - similarity: 0.47521\n",
      "Document Etoperidone - similarity: 0.47519\n",
      "Document Baccharis halimifolia pollen - similarity: 0.47507\n",
      "Document Pentazocine - similarity: 0.47498\n",
      "Document MLN0415 - similarity: 0.47495\n",
      "Document Tetrandrine - similarity: 0.47491\n",
      "Document 4-(2-aminoethoxy)-N-(3-chloro-5-piperidin-1-ylphenyl)-3,5-dimethylbenzamide - similarity: 0.47480\n",
      "Document 2-(3-OXO-PROPYLSULFANYLCARBONYL)-ETHANETHIOLATE - similarity: 0.47479\n",
      "Document Glucose Oxidase - similarity: 0.47478\n",
      "Document Creatine - similarity: 0.47455\n",
      "Document Batefenterol - similarity: 0.47444\n",
      "Document Acacia pollen - similarity: 0.47443\n",
      "Document Buformin - similarity: 0.47438\n",
      "Document Lorecivivint - similarity: 0.47420\n",
      "Document Azilsartan medoxomil - similarity: 0.47418\n",
      "Document Inclisiran - similarity: 0.47417\n",
      "Document Didesmethylrocaglamide - similarity: 0.47406\n",
      "Document 4-(3,5-DIMETHYLPHENOXY)-5-(FURAN-2-YLMETHYLSULFANYLMETHYL)-3-IODO-6-METHYLPYRIDIN-2(1H)-ONE - similarity: 0.47403\n",
      "Document Propyphenazone - similarity: 0.47401\n",
      "Document 4-dimethylaminophenol - similarity: 0.47397\n",
      "Document 2-octyl cyanoacrylate - similarity: 0.47396\n",
      "Document Prasterone sulfate - similarity: 0.47389\n",
      "Document Pyrantel - similarity: 0.47388\n",
      "Document Manganese - similarity: 0.47384\n",
      "Document Naproxen etemesil - similarity: 0.47378\n",
      "Document Turmeric oil - similarity: 0.47376\n",
      "Document (3R)-4-[(3R)-3-AMINO-4-(2,4,5-TRIFLUOROPHENYL)BUTANOYL]-3-METHYL-1,4-DIAZEPAN-2-ONE - similarity: 0.47375\n",
      "Document AX200 - similarity: 0.47364\n",
      "Document Influenza B virus B/Brisbane/60/2008 recombinant hemagglutinin antigen - similarity: 0.47356\n",
      "Document Nicotinamide adenine dinucleotide 3-pentanone adduct - similarity: 0.47352\n",
      "Document Fenofibric acid - similarity: 0.47350\n",
      "Document 2-[(PHENYLSULFONYL)AMINO]-5,6,7,8-TETRAHYDRONAPHTHALENE-1-CARBOXYLIC ACID - similarity: 0.47348\n",
      "Document Fusafungine - similarity: 0.47346\n",
      "Document Moclobemide - similarity: 0.47335\n",
      "Document BTA798 - similarity: 0.47334\n",
      "Document Platinum - similarity: 0.47329\n",
      "Document Cholecalciferol - similarity: 0.47325\n",
      "Document Recombinant human lecithin:cholesterol acyltransferase - similarity: 0.47318\n",
      "Document Tandamine - similarity: 0.47314\n",
      "Document AZD3409 - similarity: 0.47314\n",
      "Document N-acetyl-beta-neuraminic acid - similarity: 0.47313\n",
      "Document Emicizumab - similarity: 0.47310\n",
      "Document Cetyl alcohol - similarity: 0.47306\n",
      "Document A-674563 - similarity: 0.47303\n",
      "Document Procaine benzylpenicillin - similarity: 0.47296\n",
      "Document Porphobilinogen - similarity: 0.47296\n",
      "Document (2R,3R)-N^1^-[(1S)-2,2-DIMETHYL-1-(METHYLCARBAMOYL)PROPYL]-N^4^-HYDROXY-2-(2-METHYLPROPYL)-3-{[(1,3-THIAZOL-2-YLCARBONYL)AMINO]METHYL}BUTANEDIAMIDE - similarity: 0.47292\n",
      "Document 6-(3-AMINOPHENYL)-N-(TERT-BUTYL)-2-(TRIFLUOROMETHYL)QUINAZOLIN-4-AMINE - similarity: 0.47288\n",
      "Document MAX-40279 - similarity: 0.47286\n",
      "Document 5-Phosphoarabinonic Acid - similarity: 0.47286\n",
      "Document 2-butoxy-9-(2,6-difluorobenzyl)-N-(2-morpholin-4-ylethyl)-9H-purin-6-amine - similarity: 0.47281\n",
      "Document Esculin - similarity: 0.47280\n",
      "Document Peanut oil - similarity: 0.47279\n",
      "Document 5-(5-(2,6-dichloro-4-(4,5-dihydro-2-oxazolyl)phenoxy)pentyl)-3-(hydroxyethyl oxymethyleneoxymethyl) isoxazole - similarity: 0.47267\n",
      "Document OCTANE-1,3,5,7-TETRACARBOXYLIC ACID - similarity: 0.47263\n",
      "Document Fluprednidene acetate - similarity: 0.47251\n",
      "Document Mersalyl - similarity: 0.47247\n",
      "Document Betahistine - similarity: 0.47246\n",
      "Document 6-Aminohexyl-uridine-C1,5'-diphosphate - similarity: 0.47243\n",
      "Document Clofoctol - similarity: 0.47238\n",
      "Document Bufexamac - similarity: 0.47234\n",
      "Document Saracatinib - similarity: 0.47233\n",
      "Document Acetylamino-Acetic Acid - similarity: 0.47230\n",
      "Document Cefaclor - similarity: 0.47229\n",
      "Document Festuca pratensis pollen - similarity: 0.47228\n",
      "Document (2R)-2-(3-Biphenylyl)-N-{(2R)-2-hydroxy-3-[(2-pyridinylsulfonyl)amino]propyl}-4-methylpentanamide - similarity: 0.47217\n",
      "Document 5,6,7,8-TETRAHYDRO[1]BENZOTHIENO[2,3-D]PYRIMIDIN-4(3H)-ONE - similarity: 0.47211\n",
      "Document Lacutamab - similarity: 0.47208\n",
      "Document Rimacalib - similarity: 0.47199\n",
      "Document Nitrogen - similarity: 0.47198\n",
      "Document Bafilomycin B1 - similarity: 0.47186\n",
      "Document Thrombin alfa - similarity: 0.47185\n",
      "Document 2-(CARBOXYMETHYL)-1-OXO-1,2-DIHYDRONAPHTHO[1,2-D]ISOTHIAZOLE-4-CARBOXYLIC ACID 3,3-DIOXIDE - similarity: 0.47177\n",
      "Document Magnesium trisilicate - similarity: 0.47171\n",
      "Document Morpholinylmercaptobenzothiazole - similarity: 0.47157\n",
      "Document Bromoform - similarity: 0.47135\n",
      "Document 3,4-Dihydroxybenzoic Acid - similarity: 0.47134\n",
      "Document Oglemilast - similarity: 0.47131\n",
      "Document Isosorbide - similarity: 0.47128\n",
      "Document L-Alpha-Phosphatidyl-Beta-Oleoyl-Gamma-Palmitoyl-Phosphatidylethanolamine - similarity: 0.47127\n",
      "Document Vindesine - similarity: 0.47123\n",
      "Document Medical Cannabis - similarity: 0.47122\n",
      "Document Hydroxycitronellal - similarity: 0.47116\n",
      "Document Quinacrine - similarity: 0.47113\n",
      "Document Azosemide - similarity: 0.47113\n",
      "Document Magnesium silicate - similarity: 0.47112\n",
      "Document N-[2-(6-AMINO-4-METHYLPYRIDIN-2-YL)ETHYL]-4-CYANOBENZAMIDE - similarity: 0.47109\n",
      "Document Isoquercetin - similarity: 0.47109\n",
      "Document 5-HYDROXY-3-[(1R)-1-(1H-PYRROL-2-YL)ETHYL]-2H-INDOL-2-ONE - similarity: 0.47107\n",
      "Document Tirofiban - similarity: 0.47106\n",
      "Document Meropenem - similarity: 0.47101\n",
      "Document Thymidine-3',5'-Diphosphate - similarity: 0.47092\n",
      "Document Methionine Phosphonate - similarity: 0.47088\n",
      "Document Coagulation Factor IX Human - similarity: 0.47082\n",
      "Document Farnesyl diphosphate - similarity: 0.47081\n",
      "Document INCB7839 - similarity: 0.47075\n",
      "Document Oxogluric acid - similarity: 0.47066\n",
      "Document Black cohosh - similarity: 0.47062\n",
      "Document Disulfiram - similarity: 0.47058\n",
      "Document (4R,2S)-5'-(4-(4-CHLOROBENZYLOXY)PYRROLIDIN-2-YLMETHANESULFONYL)ISOQUINOLINE - similarity: 0.47054\n",
      "Document (3S)-3-methyl-D-aspartic acid - similarity: 0.47046\n",
      "Document Altropane I-123 - similarity: 0.47046\n",
      "Document Uteroglobin - similarity: 0.47041\n",
      "Document 2,6-dicarboxynaphthalene - similarity: 0.47040\n",
      "Document Indian frankincense - similarity: 0.47039\n",
      "Document 5-[3-(2-METHOXYPHENYL)-1H-PYRROLO[2,3-B]PYRIDIN-5-YL]-N,N-DIMETHYLPYRIDINE-3-CARBOXAMIDE - similarity: 0.47035\n",
      "Document Alphamethadol - similarity: 0.47018\n",
      "Document Ferric sulfate - similarity: 0.47015\n",
      "Document Spirapril - similarity: 0.47015\n",
      "Document Magnesium gluconate - similarity: 0.47009\n",
      "Document Dextrorphan - similarity: 0.47004\n",
      "Document Riociguat - similarity: 0.46998\n",
      "Document Asunercept - similarity: 0.46997\n",
      "Document Gallamine - similarity: 0.46997\n",
      "Document LErafAON - similarity: 0.46997\n",
      "Document (3S)-3-hydroxy-1-methyl-2,3-dihydro-1H-indole-5,6-dione - similarity: 0.46991\n",
      "Document 4-{4-[4-(3-AMINOPROPOXY)PHENYL]-1H-PYRAZOL-5-YL}-6-CHLOROBENZENE-1,3-DIOL - similarity: 0.46976\n",
      "Document Carboxymethylthio-3-(3-Chlorophenyl)-1,2,4-Oxadiazol - similarity: 0.46976\n",
      "Document 5-(5-chloro-7H-pyrrolo[2,3-d]pyrimidin-4-yl)-4,5,6,7-tetrahydro-1H-imidazo[4,5-c]pyridine - similarity: 0.46975\n",
      "Document PF-114 - similarity: 0.46966\n",
      "Document Cholesterol - similarity: 0.46964\n",
      "Document Clomipramine - similarity: 0.46963\n",
      "Document NOP-1A - similarity: 0.46954\n",
      "Document (5R)-2-sulfanyl-5-[4-(trifluoromethyl)benzyl]-1,3-thiazol-4(5H)-one - similarity: 0.46953\n",
      "Document Loxoprofen - similarity: 0.46953\n",
      "Document Acetoxolone - similarity: 0.46952\n",
      "Document Triethylenetetramine - similarity: 0.46950\n",
      "Document Mezlocillin - similarity: 0.46944\n",
      "Document Carbocisteine - similarity: 0.46942\n",
      "Document 4-(N-ACETYLAMINO)-3-[N-(2-ETHYLBUTANOYLAMINO)]BENZOIC ACID - similarity: 0.46942\n",
      "Document Bacitracin - similarity: 0.46941\n",
      "Document Nerispirdine - similarity: 0.46940\n",
      "Document Cariprazine - similarity: 0.46938\n",
      "Document Drotrecogin alfa - similarity: 0.46938\n",
      "Document Bambuterol - similarity: 0.46929\n",
      "Document Radotinib - similarity: 0.46914\n",
      "Document Ethambutol - similarity: 0.46913\n",
      "Document 5-Iodouracil - similarity: 0.46906\n",
      "Document 5'-O-[(L-methionyl)-sulphamoyl]adenosine - similarity: 0.46897\n",
      "Document AS1409 - similarity: 0.46896\n",
      "Document N-(4-METHYLBENZOYL)-4-BENZYLPIPERIDINE - similarity: 0.46896\n",
      "Document 3-[(4'-cyanobiphenyl-4-yl)oxy]-N-hydroxypropanamide - similarity: 0.46895\n",
      "Document Oxaliplatin - similarity: 0.46895\n",
      "Document BMS-214662 - similarity: 0.46888\n",
      "Document Zuranolone - similarity: 0.46881\n",
      "Document Sulfopyruvate - similarity: 0.46876\n",
      "Document Cadaverine - similarity: 0.46865\n",
      "Document Heme D - similarity: 0.46859\n",
      "Document TMC-647055 - similarity: 0.46854\n",
      "Document Polmacoxib - similarity: 0.46851\n",
      "Document Histapyrrodine - similarity: 0.46841\n",
      "Document Aluminium phosphate - similarity: 0.46839\n",
      "Document (R)-tacrine(10)-hupyridone - similarity: 0.46839\n",
      "Document CoA-s-acetyl tryptamine - similarity: 0.46828\n",
      "Document Dichloroacetic acid - similarity: 0.46826\n",
      "Document methyl (3S)-3-[(tert-butoxycarbonyl)amino]-4-oxopentanoate - similarity: 0.46809\n",
      "Document Mizolastine - similarity: 0.46805\n",
      "Document [5-HYDROXY-2-(4-HYDROXYPHENYL)-1-BENZOFURAN-7-YL]ACETONITRILE - similarity: 0.46799\n",
      "Document Prilocaine - similarity: 0.46799\n",
      "Document 6-[N-(4-Ethyl-1,2,3,4-Tetrahydro-6-Isoquinolinyl)Carbamyl]-2-Naphthalenecarboxamidine - similarity: 0.46797\n",
      "Document (1S)-2-{[{[(2S)-2,3-DIHYDROXYPROPYL]OXY}(HYDROXY)PHOSPHORYL]OXY}-1-[(PENTANOYLOXY)METHYL]ETHYL OCTANOATE - similarity: 0.46794\n",
      "Document Sorbitol - similarity: 0.46793\n",
      "Document Di-2-pyridylketone 4-cyclohexyl-4-methyl-3-thiosemicarbazone - similarity: 0.46790\n",
      "Document Sinefungin - similarity: 0.46789\n",
      "Document N-[(E)-3-[(2R,3S,4R,5R)-5-(6-Aminopurin-9-yl)-3,4-dihydroxyoxolan-2-yl]prop-2-enyl]-2,3-dihydroxy-5-nitrobenzamide - similarity: 0.46787\n",
      "Document Ecopipam - similarity: 0.46779\n",
      "Document 105AD7 - similarity: 0.46775\n",
      "Document Avatrombopag - similarity: 0.46773\n",
      "Document Fluocinolone - similarity: 0.46771\n",
      "Document (2E)-1-[2-hydroxy-4-methoxy-5-(3-methylbut-2-en-1-yl)phenyl]-3-(4-hydroxyphenyl)prop-2-en-1-one - similarity: 0.46770\n",
      "Document Etoricoxib - similarity: 0.46766\n",
      "Document S-palmitoyl-L-cysteine - similarity: 0.46764\n",
      "Document Lacosamide - similarity: 0.46764\n",
      "Document Pimobendan - similarity: 0.46760\n",
      "Document Modufolin - similarity: 0.46758\n",
      "Document 2-Methoxyethanol - similarity: 0.46757\n",
      "Document Nomegestrol - similarity: 0.46756\n",
      "Document Indocyanine green acid form - similarity: 0.46751\n",
      "Document Ascrinvacumab - similarity: 0.46744\n",
      "Document Ethenzamide - similarity: 0.46742\n",
      "Document Poa compressa pollen - similarity: 0.46724\n",
      "Document 7,8-dihydroxy-4-phenyl-2H-chromen-2-one - similarity: 0.46720\n",
      "Document Metocurine - similarity: 0.46719\n",
      "Document Phthalocyanine - similarity: 0.46714\n",
      "Document Panthenol - similarity: 0.46705\n",
      "Document Cyclohexylammonium Ion - similarity: 0.46701\n",
      "Document Influenza A virus A/California/7/2009 X-181 (H1N1) antigen (formaldehyde inactivated) - similarity: 0.46695\n",
      "Document Human interleukin-2 - similarity: 0.46669\n",
      "Document Naldemedine - similarity: 0.46669\n",
      "Document Soy isoflavones - similarity: 0.46666\n",
      "Document 4-Methyl-5-[(2Z)-2-{[4-(4-morpholinyl)phenyl]imino}-2,5-dihydro-4-pyrimidinyl]-1,3-thiazol-2-amine - similarity: 0.46662\n",
      "Document DiaPep 277 - similarity: 0.46645\n",
      "Document 8-epi-Cyanocobalamin - similarity: 0.46632\n",
      "Document Tifacogin - similarity: 0.46629\n",
      "Document OBP-801 - similarity: 0.46628\n",
      "Document Nafamostat - similarity: 0.46628\n",
      "Document Tremelimumab - similarity: 0.46621\n",
      "Document TMP - similarity: 0.46619\n",
      "Document Dimiracetam - similarity: 0.46614\n",
      "Document Trimesic acid - similarity: 0.46614\n",
      "Document Bixalomer - similarity: 0.46605\n",
      "Document Influenza B virus B/Hubei-Wujiagang/158/2009 BX-39 antigen (formaldehyde inactivated) - similarity: 0.46598\n",
      "Document AUY922 - similarity: 0.46594\n",
      "Document Bendroflumethiazide - similarity: 0.46593\n",
      "Document Pepsin - similarity: 0.46590\n",
      "Document TC-6987 - similarity: 0.46574\n",
      "Document Adenosine-5'-Pentaphosphate - similarity: 0.46573\n",
      "Document Trypan blue free acid - similarity: 0.46572\n",
      "Document Iomeprol - similarity: 0.46566\n",
      "Document D-4-hydroxyphenylglycine - similarity: 0.46562\n",
      "Document [2-Aminomethyl-5-Oxo-4-(4-Oxo-Cyclohexa-2,5-Dienylmethyl)-4,5-Dihydro-Imidazol-1-Yl] -Acetaldehyde - similarity: 0.46560\n",
      "Document IRX-5183 - similarity: 0.46557\n",
      "Document Clomifene - similarity: 0.46548\n",
      "Document Phycomyces blakesleeanus - similarity: 0.46543\n",
      "Document 6-[5-(2-Oxo-Hexahydro-Thieno[3,4-D]Imidazol-4-Yl)-Pentanoylamino]-Hexanoic Acid - similarity: 0.46540\n",
      "Document XmAb 2513 - similarity: 0.46536\n",
      "Document Ceftriaxone - similarity: 0.46535\n",
      "Document Levmetamfetamine - similarity: 0.46534\n",
      "Document Adenine - similarity: 0.46529\n",
      "Document Limaprost - similarity: 0.46522\n",
      "Document Radezolid - similarity: 0.46518\n",
      "Document Carbidopa - similarity: 0.46517\n",
      "Document 1,3-Dedimethyl-1,3-Divinyl Heme - similarity: 0.46512\n",
      "Document (2R)-2-benzyl-3-nitropropanoic acid - similarity: 0.46506\n",
      "Document Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (formaldehyde inactivated) - similarity: 0.46505\n",
      "Document Ibipinabant - similarity: 0.46505\n",
      "Document AZD-8186 - similarity: 0.46496\n",
      "Document Talmapimod - similarity: 0.46487\n",
      "Document Cysteamine - similarity: 0.46477\n",
      "Document 2,3,4,5,6-Pentafluorobenzyl Alcohol - similarity: 0.46477\n",
      "Document Minocycline - similarity: 0.46472\n",
      "Document Succinimide - similarity: 0.46469\n",
      "Document Edifoligide - similarity: 0.46467\n",
      "Document Afabicin - similarity: 0.46462\n",
      "Document Pentoxyverine - similarity: 0.46461\n",
      "Document Thiopental - similarity: 0.46459\n",
      "Document Camphor - similarity: 0.46455\n",
      "Document Ribose-1-Phosphate - similarity: 0.46455\n",
      "Document Celgosivir - similarity: 0.46451\n",
      "Document Dactolisib - similarity: 0.46450\n",
      "Document Menaquinone 7 - similarity: 0.46444\n",
      "Document AZD1222 - similarity: 0.46434\n",
      "Document Atriplex wrightii pollen - similarity: 0.46434\n",
      "Document Canrenone - similarity: 0.46430\n",
      "Document Fennel - similarity: 0.46409\n",
      "Document Variolin B - similarity: 0.46407\n",
      "Document 9-DEAZAINOSINE-2',3'-O-ETHYLIDENEPHOSPHONATE - similarity: 0.46400\n",
      "Document Ascorbyl glucoside - similarity: 0.46395\n",
      "Document Diflunisal - similarity: 0.46386\n",
      "Document 2,5-Dimethylpyrimidin-4-Amine - similarity: 0.46375\n",
      "Document Echothiophate - similarity: 0.46364\n",
      "Document Lorcaserin - similarity: 0.46362\n",
      "Document Aconitate Ion - similarity: 0.46350\n",
      "Document 5-ACETAMIDO-5,6-DIHYDRO-4-HYDROXY-6-ISOBUTOXY-4H-PYRAN-2-CARBOXYLIC ACID - similarity: 0.46347\n",
      "Document Equine Botulinum Neurotoxin D Immune FAB2 - similarity: 0.46338\n",
      "Document Leptin - similarity: 0.46336\n",
      "Document Trolamine polypeptide oleate condensate - similarity: 0.46334\n",
      "Document 12-Bromo-1-dodecanol - similarity: 0.46332\n",
      "Document Colchicine - similarity: 0.46331\n",
      "Document 3-[(3-sec-butyl-4-hydroxybenzoyl)amino]azepan-4-yl 4-(2-hydroxy-5-methoxybenzoyl)benzoate - similarity: 0.46329\n",
      "Document Conjugated estrogens - similarity: 0.46325\n",
      "Document 3-(1-NAPHTHYLMETHOXY)PYRIDIN-2-AMINE - similarity: 0.46324\n",
      "Document Dexmethylphenidate - similarity: 0.46311\n",
      "Document 1-(3-HYDROXYPROPYL)-2-[(3-NITROBENZOYL)AMINO]-1H-BENZIMIDAZOL-5-YL PIVALATE - similarity: 0.46310\n",
      "Document 5-O-phosphono-alpha-D-ribofuranosyl diphosphate - similarity: 0.46304\n",
      "Document Bekanamycin - similarity: 0.46302\n",
      "Document 5-[2-(4-hydroxyphenyl)ethyl]benzene-1,3-diol - similarity: 0.46299\n",
      "Document L-alloisoleucine - similarity: 0.46299\n",
      "Document Besifovir dipivoxil - similarity: 0.46290\n",
      "Document Carbon dioxide - similarity: 0.46283\n",
      "Document BUT-3-ENYL-[5-(4-CHLORO-PHENYL)-3,6-DIHYDRO-[1,3,4]THIADIAZIN-2-YLIDENE]-AMINE - similarity: 0.46279\n",
      "Document Neurospora sitophila - similarity: 0.46272\n",
      "Document Chlorphenesin - similarity: 0.46271\n",
      "Document Diethyl ether - similarity: 0.46269\n",
      "Document N-[(2R,3S)-1-((2S)-2-{[(CYCLOPENTYLAMINO)CARBONYL]AMINO}-3-METHYLBUTANOYL)-2-(1-FORMYL-1-CYCLOBUTYL)PYRROLIDINYL]CYCLOPROPANECARBOXAMIDE - similarity: 0.46267\n",
      "Document (S)-Hmg-Coa - similarity: 0.46261\n",
      "Document Cotadutide - similarity: 0.46259\n",
      "Document Avagacestat - similarity: 0.46257\n",
      "Document 4-Fluorophenethyl Alcohol - similarity: 0.46254\n",
      "Document Salicylamide - similarity: 0.46248\n",
      "Document Andarine - similarity: 0.46244\n",
      "Document p-Chlorobenzoic acid - similarity: 0.46239\n",
      "Document Digoxin Immune Fab (Ovine) - similarity: 0.46231\n",
      "Document Tirabrutinib - similarity: 0.46231\n",
      "Document TRANS-2-(DIMETHYLPHENYLSILYL)-PIPERIDINE-N-OXIDE - similarity: 0.46228\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Vancomycin - similarity: 0.46221\n",
      "Document Salsalate - similarity: 0.46221\n",
      "Document (+)-Rutamarin alcohol - similarity: 0.46210\n",
      "Document Asfotase alfa - similarity: 0.46210\n",
      "Document CEP-9722 - similarity: 0.46202\n",
      "Document Orbifloxacin - similarity: 0.46199\n",
      "Document Calcium citrate - similarity: 0.46198\n",
      "Document Lima bean - similarity: 0.46196\n",
      "Document Neisseria meningitidis group y capsular polysaccharide diphtheria toxoid conjugate antigen - similarity: 0.46185\n",
      "Document Edetic acid - similarity: 0.46180\n",
      "Document Cypermethrin - similarity: 0.46178\n",
      "Document Fenofibrate - similarity: 0.46177\n",
      "Document Inosine pranobex - similarity: 0.46176\n",
      "Document 5-[(2-AMINOETHYL)AMINO]-6-FLUORO-3-(1H-PYRROL-2-YL)BENZO[CD]INDOL-2(1H)-ONE - similarity: 0.46167\n",
      "Document Riboprine - similarity: 0.46164\n",
      "Document (2R)-3-{[(BENZYLAMINO)CARBONYL]AMINO}-2-HYDROXYPROPANOIC ACID - similarity: 0.46156\n",
      "Document 1-BENZYL-4-[(5,6-DIMETHOXY-1-INDANON-2-YL)METHYL]PIPERIDINE - similarity: 0.46154\n",
      "Document OPC-28326 - similarity: 0.46151\n",
      "Document Amiloride - similarity: 0.46141\n",
      "Document Fluindione - similarity: 0.46140\n",
      "Document AC-(D)Phe-pro-borolys-OH - similarity: 0.46126\n",
      "Document Flurbiprofen axetil - similarity: 0.46123\n",
      "Document Rebamipide - similarity: 0.46123\n",
      "Document Dusquetide - similarity: 0.46110\n",
      "Document Quingestanol acetate - similarity: 0.46107\n",
      "Document Pibrentasvir - similarity: 0.46107\n",
      "Document NV-07a - similarity: 0.46106\n",
      "Document 5-Methyl-5-(4-Phenoxy-Phenyl)-Pyrimidine-2,4,6-Trione - similarity: 0.46103\n",
      "Document Dapsone - similarity: 0.46103\n",
      "Document Chromanol - similarity: 0.46102\n",
      "Document Danofloxacin - similarity: 0.46092\n",
      "Document Dimefline - similarity: 0.46080\n",
      "Document Ethyl Oxo(Piperidin-1-Yl)Acetate - similarity: 0.46080\n",
      "Document Taurochenodeoxycholic acid - similarity: 0.46077\n",
      "Document 3-(1,1-dioxido-4H-1,2,4-benzothiadiazin-3-yl)-4-hydroxy-1-(3-methylbutyl)quinolin-2(1H)-one - similarity: 0.46077\n",
      "Document 8-azaxanthine - similarity: 0.46073\n",
      "Document Caspofungin - similarity: 0.46066\n",
      "Document Osanetant - similarity: 0.46066\n",
      "Document Voretigene neparvovec - similarity: 0.46060\n",
      "Document Bromodiphenhydramine - similarity: 0.46058\n",
      "Document 1-[4-(PYRIDIN-4-YLOXY)PHENYL]-3-[3-(TRIFLUOROMETHYL)PHENYL]UREA - similarity: 0.46051\n",
      "Document MGB-BP-3 - similarity: 0.46049\n",
      "Document Ferric ammonium citrate - similarity: 0.46048\n",
      "Document Streptococcus pneumoniae type 8 capsular polysaccharide antigen - similarity: 0.46045\n",
      "Document Pazopanib - similarity: 0.46039\n",
      "Document 2,6,8-Trimethyl-3-Amino-9-Benzyl-9-Methoxynonanoic Acid - similarity: 0.46034\n",
      "Document Dibotermin alfa - similarity: 0.46030\n",
      "Document Pafuramidine - similarity: 0.46023\n",
      "Document 1-Androstenediol - similarity: 0.46022\n",
      "Document D-Tryptophan - similarity: 0.46018\n",
      "Document Allopurinol - similarity: 0.46017\n",
      "Document Etanautine - similarity: 0.46003\n",
      "Document Methylphenidate - similarity: 0.46000\n",
      "Document Phenoxypropazine - similarity: 0.45996\n",
      "Document Lurbinectedin - similarity: 0.45990\n",
      "Document 2-[(2,4-DICHLORO-5-METHYLPHENYL)SULFONYL]-1,3-DINITRO-5-(TRIFLUOROMETHYL)BENZENE - similarity: 0.45975\n",
      "Document Aptazapine - similarity: 0.45975\n",
      "Document Influenza B virus B/Singapore/INFTT-16-0610/2016 antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.45973\n",
      "Document Flunisolide - similarity: 0.45969\n",
      "Document Succinobucol - similarity: 0.45968\n",
      "Document Apolizumab - similarity: 0.45963\n",
      "Document Thyroglobulin - similarity: 0.45961\n",
      "Document ROX-888 - similarity: 0.45954\n",
      "Document Siponimod - similarity: 0.45951\n",
      "Document Hydroxide ion - similarity: 0.45921\n",
      "Document ABX-PTH - similarity: 0.45918\n",
      "Document (S)-Mandelic acid - similarity: 0.45915\n",
      "Document Lexatumumab - similarity: 0.45911\n",
      "Document (4aS,6R,8aS)-11-[8-(1,3-Dioxo-1,3-dihydro-2H-isoindol-2-yl)octyl]-6-hydroxy-3-methoxy-5,6,9,10-tetrahydro-4aH-[1]benzofuro[3a,3,2-ef][2]benzazepin-11-ium - similarity: 0.45911\n",
      "Document 1,3-Thiazole-4-Carboxylic Acid - similarity: 0.45895\n",
      "Document Eldecalcitol - similarity: 0.45892\n",
      "Document Cyproterone acetate - similarity: 0.45869\n",
      "Document Trametes versicolor fruiting body - similarity: 0.45867\n",
      "Document Spermine - similarity: 0.45867\n",
      "Document American hazelnut - similarity: 0.45860\n",
      "Document Budesonide - similarity: 0.45860\n",
      "Document Ziconotide - similarity: 0.45859\n",
      "Document Chrysin - similarity: 0.45846\n",
      "Document Vitamin D - similarity: 0.45845\n",
      "Document Amphetamine - similarity: 0.45842\n",
      "Document 2-Phenylheme - similarity: 0.45814\n",
      "Document 2-Thiomethyl-3-Phenylpropanoic Acid - similarity: 0.45809\n",
      "Document ACA 125 - similarity: 0.45809\n",
      "Document Ifosfamide - similarity: 0.45809\n",
      "Document Rheinanthrone - similarity: 0.45797\n",
      "Document Tolterodine - similarity: 0.45781\n",
      "Document Luteolin - similarity: 0.45781\n",
      "Document Chenopodium ambrosioides pollen - similarity: 0.45778\n",
      "Document MLN8054 - similarity: 0.45775\n",
      "Document Ruboxistaurin - similarity: 0.45774\n",
      "Document [(3S)-3-(Methylcarbamoyl)-2-{[(2-methyl-2-propanyl)oxy]carbonyl}-1,2,3,4-tetrahydro-7-isoquinolinyl]sulfamic acid - similarity: 0.45774\n",
      "Document TBC-3711 - similarity: 0.45768\n",
      "Document Hydromorphone - similarity: 0.45762\n",
      "Document Mestranol - similarity: 0.45761\n",
      "Document Lanopepden - similarity: 0.45761\n",
      "Document WL-1002 - similarity: 0.45760\n",
      "Document 2-Methoxy-6-{(E)-[(4-methylphenyl)imino]methyl}phenol - similarity: 0.45749\n",
      "Document Maleic Acid - similarity: 0.45724\n",
      "Document 5-[(3R)-3-(5-methoxy-2',6'-dimethylbiphenyl-3-yl)but-1-yn-1-yl]-6-methylpyrimidine-2,4-diamine - similarity: 0.45724\n",
      "Document INO-4800 - similarity: 0.45722\n",
      "Document AdPEDF - similarity: 0.45721\n",
      "Document Indium In-111 - similarity: 0.45720\n",
      "Document 1-{[1-(2-AMINO-3-PHENYL-PROPIONYL)-PYRROLIDINE-2-CARBONYL]-AMINO}-2-(3-CYANO-PHENYL)-ETHANEBORONIC ACID - similarity: 0.45714\n",
      "Document Garadacimab - similarity: 0.45696\n",
      "Document 5-{[1-(2,3-dichlorobenzyl)piperidin-4-yl]methoxy}quinazoline-2,4-diamine - similarity: 0.45684\n",
      "Document IC41 - similarity: 0.45676\n",
      "Document N-(2-OXOTETRAHYDROFURAN-3-YL)OCTANAMIDE - similarity: 0.45674\n",
      "Document 3-CARBOXY-4-METHYL-5-PROPYL-2-FURANPROPIONIC - similarity: 0.45672\n",
      "Document N-(3,5-dibromo-4-hydroxyphenyl)-2,6-dimethylbenzamide - similarity: 0.45671\n",
      "Document Decernotinib - similarity: 0.45666\n",
      "Document Delorazepam - similarity: 0.45655\n",
      "Document Parecoxib - similarity: 0.45652\n",
      "Document Racementhol - similarity: 0.45644\n",
      "Document Iferanserin - similarity: 0.45636\n",
      "Document D-Treitol - similarity: 0.45621\n",
      "Document Clodantoin - similarity: 0.45620\n",
      "Document Calcium silicate - similarity: 0.45618\n",
      "Document CUDC-101 - similarity: 0.45617\n",
      "Document N4-HYDROXY-2-ISOBUTYL-N1-(9-OXO-1,8-DIAZA-TRICYCLO[10.6.1.013,18]NONADECA-12(19),13,15,17-TETRAEN-10-YL)-SUCCINAMIDE - similarity: 0.45609\n",
      "Document Elagolix - similarity: 0.45607\n",
      "Document Dihydro-alpha-ergocryptine - similarity: 0.45600\n",
      "Document Chlorpromazine - similarity: 0.45595\n",
      "Document Senicapoc - similarity: 0.45594\n",
      "Document Sitravatinib - similarity: 0.45588\n",
      "Document Forasartan - similarity: 0.45578\n",
      "Document Anthoxanthum odoratum pollen - similarity: 0.45565\n",
      "Document Berzosertib - similarity: 0.45563\n",
      "Document Filociclovir - similarity: 0.45562\n",
      "Document Propenidazole - similarity: 0.45558\n",
      "Document Cannabinol - similarity: 0.45556\n",
      "Document Talactoferrin alfa - similarity: 0.45554\n",
      "Document Sambucus nigra flower - similarity: 0.45545\n",
      "Document 6-CHLORO-9-HYDROXY-1,3-DIMETHYL-1,9-DIHYDRO-4H-PYRAZOLO[3,4-B]QUINOLIN-4-ONE - similarity: 0.45524\n",
      "Document Dimethyltryptamine - similarity: 0.45517\n",
      "Document Aminopterin - similarity: 0.45514\n",
      "Document (2S)-4-(4-fluorobenzyl)-N-(3-sulfanylpropyl)piperazine-2-carboxamide - similarity: 0.45504\n",
      "Document Aceglutamide - similarity: 0.45488\n",
      "Document Ledipasvir - similarity: 0.45486\n",
      "Document Hyoscyamine - similarity: 0.45483\n",
      "Document Tamatinib - similarity: 0.45462\n",
      "Document Biotin - similarity: 0.45452\n",
      "Document MK-1496 - similarity: 0.45446\n",
      "Document Niraxostat - similarity: 0.45440\n",
      "Document Atenolol - similarity: 0.45428\n",
      "Document 3,4-Dichloroisocoumarin - similarity: 0.45424\n",
      "Document Phosphonoacetaldehyde - similarity: 0.45420\n",
      "Document Istaroxime - similarity: 0.45415\n",
      "Document Selpercatinib - similarity: 0.45408\n",
      "Document Influenza B virus B/Massachusetts/2/2012 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.45400\n",
      "Document Hexestrol - similarity: 0.45394\n",
      "Document Tagatose - similarity: 0.45391\n",
      "Document NADH - similarity: 0.45387\n",
      "Document Protokylol - similarity: 0.45383\n",
      "Document (4'-{[Allyl(Methyl)Amino]Methyl}-1,1'-Biphenyl-4-Yl)(4-Bromophenyl)Methanone - similarity: 0.45382\n",
      "Document 2''-O-[N-(3-(aminopropyl)2-aminoethyl]paromomycin - similarity: 0.45358\n",
      "Document Erlotinib - similarity: 0.45356\n",
      "Document Antihemophilic Factor (Recombinant), PEGylated - similarity: 0.45345\n",
      "Document N~2~-(biphenyl-4-ylsulfonyl)-N-hydroxy-N~2~-(2-hydroxyethyl)glycinamide - similarity: 0.45338\n",
      "Document Schisandra chinensis fruit - similarity: 0.45337\n",
      "Document 4-CDP-2-C-methyl-D-erythritol 2-phosphate - similarity: 0.45332\n",
      "Document 6,7-dioxo-5H-8-ribitylaminolumazine - similarity: 0.45329\n",
      "Document Darunavir - similarity: 0.45326\n",
      "Document Lysophosphatidylglycerol - similarity: 0.45325\n",
      "Document N'-[(1E)-(3,5-DIBROMO-2,4-DIHYDROXYPHENYL)METHYLENE]NICOTINOHYDRAZIDE - similarity: 0.45325\n",
      "Document Natamycin - similarity: 0.45321\n",
      "Document Cloridarol - similarity: 0.45312\n",
      "Document Prednicarbate - similarity: 0.45302\n",
      "Document Absidia capillata - similarity: 0.45299\n",
      "Document Venlafaxine - similarity: 0.45294\n",
      "Document N-[(5S,7R,8S,9S,10R)-8,9,10-Trihydroxy-7-(hydroxymethyl)-2,4-dioxo-6-oxa-1,3-diazaspiro[4.5]dec-3-yl]acetamide - similarity: 0.45287\n",
      "Document Vadadustat - similarity: 0.45286\n",
      "Document Methylestrenolone - similarity: 0.45283\n",
      "Document Daglutril - similarity: 0.45281\n",
      "Document Pirarubicin - similarity: 0.45280\n",
      "Document N-Octyl-2-Hydroxyethyl Sulfoxide - similarity: 0.45276\n",
      "Document Telapristone acetate - similarity: 0.45267\n",
      "Document Ladiratuzumab vedotin - similarity: 0.45266\n",
      "Document Miravirsen - similarity: 0.45266\n",
      "Document Cortisone - similarity: 0.45258\n",
      "Document Chymotrypsin - similarity: 0.45254\n",
      "Document Noxytiolin - similarity: 0.45252\n",
      "Document Acridine Carboxamide - similarity: 0.45252\n",
      "Document Hydroquinine - similarity: 0.45249\n",
      "Document Sertindole - similarity: 0.45248\n",
      "Document Butylphthalide - similarity: 0.45247\n",
      "Document Elenbecestat - similarity: 0.45247\n",
      "Document PF-06863135 - similarity: 0.45246\n",
      "Document Afamelanotide - similarity: 0.45238\n",
      "Document Quercus velutina pollen - similarity: 0.45236\n",
      "Document Dill - similarity: 0.45227\n",
      "Document 3-Iodo-Benzyl Alcohol - similarity: 0.45223\n",
      "Document Beeswax - similarity: 0.45221\n",
      "Document N7-(5'-Phospho-alpha-ribosyl)-2-hydroxypurine - similarity: 0.45218\n",
      "Document Undecyl-Phosphinic Acid Butyl Ester - similarity: 0.45215\n",
      "Document 6-CHLORO-N-{(3S)-1-[(1S)-1-METHYL-2-MORPHOLIN-4-YL-2-OXOETHYL]-2-OXOPYRROLIDIN-3-YL}-1-BENZOTHIOPHENE-2-SULFONAMIDE - similarity: 0.45214\n",
      "Document Solabegron - similarity: 0.45210\n",
      "Document Guanosine-5'-Diphosphate-Rhamnose - similarity: 0.45208\n",
      "Document MMI-175 - similarity: 0.45207\n",
      "Document Atrasentan - similarity: 0.45206\n",
      "Document Potassium hydroxide - similarity: 0.45203\n",
      "Document Kaempherol - similarity: 0.45203\n",
      "Document Gedatolisib - similarity: 0.45201\n",
      "Document Telavancin - similarity: 0.45191\n",
      "Document Danegaptide - similarity: 0.45180\n",
      "Document 1,3,2-Dioxaborolan-2-Ol - similarity: 0.45166\n",
      "Document Inhibitor of P38 Kinase - similarity: 0.45157\n",
      "Document Raloxifene - similarity: 0.45156\n",
      "Document 2,3-Dihydroxy-Valerianic Acid - similarity: 0.45151\n",
      "Document Leucovorin - similarity: 0.45145\n",
      "Document Peach - similarity: 0.45143\n",
      "Document Tanespimycin - similarity: 0.45138\n",
      "Document Obatoclax - similarity: 0.45135\n",
      "Document Methylthioinosine - similarity: 0.45135\n",
      "Document Bromus inermis pollen - similarity: 0.45132\n",
      "Document Tetrahydropalmatine - similarity: 0.45125\n",
      "Document CCX-354 - similarity: 0.45123\n",
      "Document Paclitaxel docosahexaenoic acid - similarity: 0.45120\n",
      "Document Menotropins - similarity: 0.45120\n",
      "Document Bisindolylmaleimide I - similarity: 0.45114\n",
      "Document Darbepoetin alfa - similarity: 0.45111\n",
      "Document TAS-102 - similarity: 0.45107\n",
      "Document Cefminox - similarity: 0.45106\n",
      "Document N-({(2S)-1-[(3R)-3-AMINO-4-(2-FLUOROPHENYL)BUTANOYL]PYRROLIDIN-2-YL}METHYL)BENZAMIDE - similarity: 0.45106\n",
      "Document Acetic Acid Salicyloyl-Amino-Ester - similarity: 0.45103\n",
      "Document ANILINOMETHYL GLUCO-PHENYLIMIDAZOLE - similarity: 0.45102\n",
      "Document Meglumine antimoniate - similarity: 0.45099\n",
      "Document Dornase alfa - similarity: 0.45094\n",
      "Document Rolofylline - similarity: 0.45093\n",
      "Document Rhamnose - similarity: 0.45089\n",
      "Document 5-Thio-a/B-D-Mannopyranosylamine - similarity: 0.45089\n",
      "Document Bisphenol A - similarity: 0.45084\n",
      "Document Cyclohexanepropanoic acid - similarity: 0.45078\n",
      "Document Phylloquinone - similarity: 0.45075\n",
      "Document Taselisib - similarity: 0.45060\n",
      "Document Ethylmorphine - similarity: 0.45057\n",
      "Document Zinc;[amino-[2-[[5-[amino(azaniumylidene)methyl]benzimidazol-1-id-2-yl]methyl]benzimidazol-1-id-5-yl]methylidene]azanium - similarity: 0.45048\n",
      "Document Melilotus albus pollen - similarity: 0.45044\n",
      "Document Presatovir - similarity: 0.45042\n",
      "Document Influenza A virus A/Texas/50/2012 (H3N2) live (attenuated) antigen - similarity: 0.45037\n",
      "Document Arnica montana flower - similarity: 0.45036\n",
      "Document Piromelatine - similarity: 0.45026\n",
      "Document Crofelemer - similarity: 0.45026\n",
      "Document Bezitramide - similarity: 0.45024\n",
      "Document Glembatumumab vedotin - similarity: 0.45023\n",
      "Document Castanospermine - similarity: 0.45022\n",
      "Document Pentaerithrityl - similarity: 0.45015\n",
      "Document 6-hydroxy-D-norleucine - similarity: 0.45013\n",
      "Document (R)-Mandelic acid - similarity: 0.45007\n",
      "Document Rehmannia glutinosa root - similarity: 0.44992\n",
      "Document Abrocitinib - similarity: 0.44982\n",
      "Document (2S)-hydroxy(4-hydroxyphenyl)ethanoic acid - similarity: 0.44964\n",
      "Document Ulixertinib - similarity: 0.44962\n",
      "Document Pazufloxacin - similarity: 0.44952\n",
      "Document Mycogone nigra - similarity: 0.44942\n",
      "Document S-Ethyl-N-Phenyl-Isothiourea - similarity: 0.44937\n",
      "Document (1S)-1-{[(4'-methoxy-1,1'-biphenyl-4-yl)sulfonyl]amino}-2-methylpropylphosphonic acid - similarity: 0.44936\n",
      "Document Sinapoyl coenzyme A - similarity: 0.44931\n",
      "Document Rhein - similarity: 0.44930\n",
      "Document Trastuzumab emtansine - similarity: 0.44929\n",
      "Document Dihydrouracil - similarity: 0.44926\n",
      "Document 2-(2-{2-[(BIPHENYL-4-YLMETHYL)-AMINO]-3-MERCAPTO-PENTANOYLAMINO}-ACETYLAMINO)-3-METHYL-BUTYRIC ACID METHYL ESTER - similarity: 0.44924\n",
      "Document Piclidenoson - similarity: 0.44917\n",
      "Document Otilimab - similarity: 0.44915\n",
      "Document Simvastatin - similarity: 0.44914\n",
      "Document Guanabenz - similarity: 0.44911\n",
      "Document Sulfamethizole - similarity: 0.44910\n",
      "Document 4-Bromo-2,5-dimethoxyamphetamine - similarity: 0.44909\n",
      "Document Orotic acid - similarity: 0.44902\n",
      "Document (3R)-3-cyclopentyl-6-methyl-7-[(4-methylpiperazin-1-yl)sulfonyl]-3,4-dihydro-2H-1,2-benzothiazine 1,1-dioxide - similarity: 0.44897\n",
      "Document (2S)-1-{4-[(4-Anilino-5-bromo-2-pyrimidinyl)amino]phenoxy}-3-(dimethylamino)-2-propanol - similarity: 0.44893\n",
      "Document Androstenedione - similarity: 0.44889\n",
      "Document Exatecan - similarity: 0.44886\n",
      "Document alpha-Linolenic acid - similarity: 0.44879\n",
      "Document (3R)-3-(FLUOROMETHYL)-7-(THIOMORPHOLIN-4-YLSULFONYL)-1,2,3,4-TETRAHYDROISOQUINOLINE - similarity: 0.44878\n",
      "Document Dimethyl carbate - similarity: 0.44875\n",
      "Document Monofluorophosphate ion - similarity: 0.44869\n",
      "Document (2S)-1,3-benzothiazol-2-yl{2-[(2-pyridin-3-ylethyl)amino]pyrimidin-4-yl}ethanenitrile - similarity: 0.44865\n",
      "Document Ricinoleic Acid - similarity: 0.44861\n",
      "Document Hydrotalcite - similarity: 0.44830\n",
      "Document Atriplex lentiformis pollen - similarity: 0.44821\n",
      "Document GW-3965 - similarity: 0.44820\n",
      "Document Lisdexamfetamine - similarity: 0.44817\n",
      "Document Picloxydine - similarity: 0.44816\n",
      "Document Deoxyamidinoproclavaminic acid - similarity: 0.44811\n",
      "Document Methyl beta-L-fucopyranoside - similarity: 0.44810\n",
      "Document 1-(4-Cyano-phenyl)-3-[2-(2,6-dichloro-phenyl)-1-imino-ethyl]-thiourea - similarity: 0.44806\n",
      "Document LGD-2226 - similarity: 0.44803\n",
      "Document [(3R)-7-NITRO-1,2,3,4-TETRAHYDROISOQUINOLIN-3-YL]METHANOL - similarity: 0.44798\n",
      "Document Abatacept - similarity: 0.44798\n",
      "Document GLPG-0492 - similarity: 0.44794\n",
      "Document Magnesium orotate - similarity: 0.44789\n",
      "Document Carthamus tinctorius flower bud - similarity: 0.44767\n",
      "Document Dioxaphetyl butyrate - similarity: 0.44755\n",
      "Document Lurasidone - similarity: 0.44723\n",
      "Document Paramethadione - similarity: 0.44722\n",
      "Document Remimazolam - similarity: 0.44717\n",
      "Document 3-phenyl-5-(1H-pyrazol-3-yl)isoxazole - similarity: 0.44713\n",
      "Document Purine Riboside-5'-Monophosphate - similarity: 0.44711\n",
      "Document Bemotrizinol - similarity: 0.44701\n",
      "Document MK-8245 - similarity: 0.44696\n",
      "Document Kebuzone - similarity: 0.44687\n",
      "Document Insulin-like growth factor II - similarity: 0.44682\n",
      "Document Moroxydine - similarity: 0.44673\n",
      "Document Estrone - similarity: 0.44655\n",
      "Document GI-4000 - similarity: 0.44655\n",
      "Document Rindopepimut - similarity: 0.44654\n",
      "Document Indirubin-3'-monoxime - similarity: 0.44653\n",
      "Document Rhizopus arrhizus var. arrhizus - similarity: 0.44637\n",
      "Document Teceleukin - similarity: 0.44636\n",
      "Document KAI-1455 - similarity: 0.44631\n",
      "Document Bicuculline - similarity: 0.44630\n",
      "Document Abediterol - similarity: 0.44626\n",
      "Document Nabiximols - similarity: 0.44624\n",
      "Document Tibezonium iodide - similarity: 0.44621\n",
      "Document N-[(13-CYCLOHEXYL-6,7-DIHYDROINDOLO[1,2-D][1,4]BENZOXAZEPIN-10-YL)CARBONYL]-2-METHYL-L-ALANINE - similarity: 0.44615\n",
      "Document TST10088 - similarity: 0.44613\n",
      "Document Tasisulam - similarity: 0.44613\n",
      "Document Zinc oxide - similarity: 0.44608\n",
      "Document 2-[3-(5-Mercapto-[1,3,4]thiadiazol-2yl)-ureido]-N-methyl-3-pentafluorophenyl-propionamide - similarity: 0.44608\n",
      "Document 1'-Deazo-Thiamin Diphosphate - similarity: 0.44577\n",
      "Document 3,6-dihydroxy-xanthene-9-propionic acid - similarity: 0.44576\n",
      "Document Adefovir dipivoxil - similarity: 0.44575\n",
      "Document PF-03382792 - similarity: 0.44574\n",
      "Document Alpha-Adenosine Monophosphate - similarity: 0.44564\n",
      "Document Avocado - similarity: 0.44561\n",
      "Document 9-CYCLOPENTYL-6-[2-(3-IMIDAZOL-1-YL-PROPOXY)-PHENYLAMINO]-9H-PURINE-2-CARBONITRILE - similarity: 0.44554\n",
      "Document Parkinsonia florida pollen - similarity: 0.44543\n",
      "Document Glycine betaine - similarity: 0.44535\n",
      "Document Dexanabinol - similarity: 0.44534\n",
      "Document Chlorophyll B - similarity: 0.44531\n",
      "Document Indole - similarity: 0.44529\n",
      "Document Etamicastat - similarity: 0.44526\n",
      "Document 1,1'-BIPHENYL-2-SULFINIC ACID - similarity: 0.44520\n",
      "Document D-norleucine - similarity: 0.44509\n",
      "Document TRO19622 - similarity: 0.44509\n",
      "Document Tivantinib - similarity: 0.44502\n",
      "Document Telinavir - similarity: 0.44501\n",
      "Document Halicin - similarity: 0.44501\n",
      "Document Oxaloacetate Ion - similarity: 0.44501\n",
      "Document PZM21 - similarity: 0.44494\n",
      "Document Aducanumab - similarity: 0.44493\n",
      "Document Ulmus parvifolia pollen - similarity: 0.44492\n",
      "Document Inositol 1,3-Bisphosphate - similarity: 0.44492\n",
      "Document Benzofuran - similarity: 0.44491\n",
      "Document Tarloxotinib - similarity: 0.44473\n",
      "Document Tyrosyladenylate - similarity: 0.44466\n",
      "Document Eribulin - similarity: 0.44465\n",
      "Document Quinestrol - similarity: 0.44464\n",
      "Document Darifenacin - similarity: 0.44461\n",
      "Document Sulbentine - similarity: 0.44459\n",
      "Document Nitenpyram - similarity: 0.44442\n",
      "Document Tarazepide - similarity: 0.44438\n",
      "Document Meclofenamic acid - similarity: 0.44434\n",
      "Document Calcium Phosphate - similarity: 0.44431\n",
      "Document Allergenic extract- beef liver - similarity: 0.44429\n",
      "Document Letaxaban - similarity: 0.44427\n",
      "Document Artemisia absinthium pollen - similarity: 0.44426\n",
      "Document Tucaresol - similarity: 0.44414\n",
      "Document alpha-D-glucopyranosyl-2-carboxylic acid amide - similarity: 0.44403\n",
      "Document Bexarotene - similarity: 0.44393\n",
      "Document PF-04217903 - similarity: 0.44384\n",
      "Document Hycanthone - similarity: 0.44383\n",
      "Document 10-hydroxycamptothecin - similarity: 0.44376\n",
      "Document Arrhenatherum elatius pollen - similarity: 0.44372\n",
      "Document Miridesap - similarity: 0.44371\n",
      "Document Nona-arginine - similarity: 0.44365\n",
      "Document Ramatroban - similarity: 0.44364\n",
      "Document Menadione - similarity: 0.44359\n",
      "Document 4-(4-STYRYL-PHENYLCARBAMOYL)-BUTYRIC ACID - similarity: 0.44342\n",
      "Document Juniperus scopulorum pollen - similarity: 0.44342\n",
      "Document S,R-Warfarin alcohol - similarity: 0.44337\n",
      "Document JNJ-40346527 - similarity: 0.44335\n",
      "Document Passiflora incarnata flower - similarity: 0.44328\n",
      "Document Damoctocog alfa pegol - similarity: 0.44328\n",
      "Document 1,4-Dioxane - similarity: 0.44325\n",
      "Document Mirvetuximab Soravtansine - similarity: 0.44314\n",
      "Document Glufosfamide - similarity: 0.44309\n",
      "Document LY-2334737 - similarity: 0.44296\n",
      "Document Opilio crab - similarity: 0.44285\n",
      "Document 4-methyl-umbelliferyl-N-acetyl-chitobiose - similarity: 0.44277\n",
      "Document Trifluridine - similarity: 0.44266\n",
      "Document Thyrotropin - similarity: 0.44264\n",
      "Document Cycrimine - similarity: 0.44264\n",
      "Document 2-Aminopropanedioic Acid - similarity: 0.44259\n",
      "Document 6-MORPHOLIN-4-YL-9H-PURINE - similarity: 0.44258\n",
      "Document Neon - similarity: 0.44258\n",
      "Document O-acetyl-L-serine - similarity: 0.44239\n",
      "Document Sivelestat - similarity: 0.44229\n",
      "Document CEP-37440 - similarity: 0.44217\n",
      "Document Licofelone - similarity: 0.44208\n",
      "Document Esaxerenone - similarity: 0.44206\n",
      "Document Dextropropoxyphene - similarity: 0.44202\n",
      "Document 6-bromoindirubin-3'-oxime - similarity: 0.44196\n",
      "Document Tenidap - similarity: 0.44195\n",
      "Document Dienogest - similarity: 0.44189\n",
      "Document Indoramin - similarity: 0.44187\n",
      "Document 13-cis-12-(3'-Carboxyphenyl)retinoic acid - similarity: 0.44186\n",
      "Document Acadesine - similarity: 0.44178\n",
      "Document Carteolol - similarity: 0.44178\n",
      "Document Fluoxetine - similarity: 0.44170\n",
      "Document Ornithine - similarity: 0.44169\n",
      "Document Tripelennamine - similarity: 0.44169\n",
      "Document (4R,5R)-1,2-dithiane-4,5-diol - similarity: 0.44167\n",
      "Document Benproperine - similarity: 0.44157\n",
      "Document BMS-791826 - similarity: 0.44147\n",
      "Document Phenoxyethanol - similarity: 0.44136\n",
      "Document Acebilustat - similarity: 0.44134\n",
      "Document Rozanolixizumab - similarity: 0.44126\n",
      "Document Wood creosote - similarity: 0.44125\n",
      "Document 3'-Uridinemonophosphate - similarity: 0.44123\n",
      "Document Diosmin - similarity: 0.44108\n",
      "Document Phosmet - similarity: 0.44108\n",
      "Document Thiodigalactoside - similarity: 0.44104\n",
      "Document Influenza B virus B/Iowa/06/2017 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.44103\n",
      "Document PF-03814735 - similarity: 0.44095\n",
      "Document Secretin human - similarity: 0.44086\n",
      "Document Betrixaban - similarity: 0.44080\n",
      "Document BMS-817399 - similarity: 0.44076\n",
      "Document (3AS,4R,9BR)-2,2-DIFLUORO-4-(4-HYDROXYPHENYL)-1,2,3,3A,4,9B-HEXAHYDROCYCLOPENTA[C]CHROMEN-8-OL - similarity: 0.44075\n",
      "Document N-(3,5-dimethoxyphenyl)imidodicarbonimidic diamide - similarity: 0.44065\n",
      "Document Simmondsia chinensis seed - similarity: 0.44064\n",
      "Document Echinomycin - similarity: 0.44061\n",
      "Document Salvia miltiorrhiza root - similarity: 0.44029\n",
      "Document 4-(Dimethylamino)benzoic acid - similarity: 0.44028\n",
      "Document 2-(l-menthoxy)ethanol - similarity: 0.44023\n",
      "Document 6-Hydroxymethylpterin - similarity: 0.44022\n",
      "Document A Disubstituted Succinyl Caprolactam Hydroxymate Mmp3inhibitor - similarity: 0.44019\n",
      "Document (2R)-2-(aminomethyl)-2,4-dihydroxy-5-oxo-3-(2-oxoethyl)-2,5-dihydro-1H-imidazol-3-ium - similarity: 0.44009\n",
      "Document Daptomycin - similarity: 0.44006\n",
      "Document Purslane - similarity: 0.44006\n",
      "Document Actinium - similarity: 0.44003\n",
      "Document Onion - similarity: 0.43996\n",
      "Document Glyceryl monostearate - similarity: 0.43992\n",
      "Document (3E)-5-fluoro-1-[(6-fluoro-4H-1,3-benzodioxin-8-yl)methyl]-1H-indole-2,3-dione 3-oxime - similarity: 0.43991\n",
      "Document Homatropine - similarity: 0.43987\n",
      "Document Regorafenib - similarity: 0.43974\n",
      "Document Glucaric acid - similarity: 0.43974\n",
      "Document Telcagepant - similarity: 0.43951\n",
      "Document 5-Aminoisoquinoline - similarity: 0.43950\n",
      "Document Etanidazole - similarity: 0.43949\n",
      "Document N5-Methylglutamine - similarity: 0.43949\n",
      "Document 4-(4-hydroxy-3-methylphenyl)-6-phenylpyrimidin-2(5H)-one - similarity: 0.43944\n",
      "Document [4-(2-Amino-4-Methyl-Thiazol-5-Yl)-Pyrimidin-2-Yl]-(3-Nitro-Phenyl)-Amine - similarity: 0.43942\n",
      "Document Benzoylecgonine - similarity: 0.43934\n",
      "Document LY-2608204 - similarity: 0.43923\n",
      "Document Secoisolariciresinol - similarity: 0.43923\n",
      "Document Deserpidine - similarity: 0.43910\n",
      "Document Ticlatone - similarity: 0.43910\n",
      "Document Hepatitis B Vaccine (Recombinant) - similarity: 0.43892\n",
      "Document 2-chloro-5-nitro-N-phenylbenzamide - similarity: 0.43891\n",
      "Document 2-{4-[4-(4-Chloro-Phenoxy)-Benzenesulfonyl]-Tetrahydro-Pyran-4-Yl}-N-Hydroxy-Acetamide - similarity: 0.43889\n",
      "Document Marine collagen, soluble - similarity: 0.43882\n",
      "Document 1-(biphenyl-4-ylmethyl)-1H-imidazole - similarity: 0.43874\n",
      "Document Methylpropylpropanediol dinitrate - similarity: 0.43858\n",
      "Document (TERT-BUTYLOXYCARBONYL)-ALANYL-AMINO ETHYL-FORMAMIDE - similarity: 0.43858\n",
      "Document METHYLAMINO-PHENYLALANYL-LEUCYL-HYDROXAMIC ACID - similarity: 0.43855\n",
      "Document Ecabet - similarity: 0.43849\n",
      "Document Anagrelide - similarity: 0.43847\n",
      "Document Fenthion - similarity: 0.43847\n",
      "Document N-{2-methyl-5-[(6-phenylpyrimidin-4-yl)amino]phenyl}methanesulfonamide - similarity: 0.43847\n",
      "Document ISO-1 F-18 - similarity: 0.43845\n",
      "Document (2R)-2-(4-CHLOROPHENYL)-2-PHENYLETHANAMINE - similarity: 0.43844\n",
      "Document Calcium carbimide - similarity: 0.43839\n",
      "Document Oxyphenonium - similarity: 0.43837\n",
      "Document Probucol - similarity: 0.43835\n",
      "Document (S)-AMPA - similarity: 0.43824\n",
      "Document 6-O-Cyclohexylmethyl Guanine - similarity: 0.43823\n",
      "Document Tiaprofenic acid - similarity: 0.43812\n",
      "Document Folic acid - similarity: 0.43811\n",
      "Document Guar gum - similarity: 0.43808\n",
      "Document AZD-3043 - similarity: 0.43808\n",
      "Document Chloride ion - similarity: 0.43800\n",
      "Document THYMIDINE-5'-THIOPHOSPHATE - similarity: 0.43797\n",
      "Document Chromic chloride - similarity: 0.43777\n",
      "Document Coagulation factor XIa - similarity: 0.43775\n",
      "Document Pascolizumab - similarity: 0.43774\n",
      "Document (2S)-2-HYDROXY-2H-CHROMENE-2-CARBOXYLIC ACID - similarity: 0.43770\n",
      "Document Zardaverine - similarity: 0.43770\n",
      "Document GTX-758 - similarity: 0.43766\n",
      "Document 5-(4-Methoxyphenoxy)-2,4-Quinazolinediamine - similarity: 0.43759\n",
      "Document (1S,3S,5S)-2-{(2S)-2-amino-2-[(1R,3S,5R,7S)-3-hydroxytricyclo[3.3.1.1~3,7~]dec-1-yl]acetyl}-2-azabicyclo[3.1.0]hexane-3-carbonitrile - similarity: 0.43746\n",
      "Document Zeatin - similarity: 0.43735\n",
      "Document Chlorotoxin - similarity: 0.43728\n",
      "Document Zaltoprofen - similarity: 0.43721\n",
      "Document E-7820 - similarity: 0.43705\n",
      "Document Omecamtiv Mecarbil - similarity: 0.43704\n",
      "Document Merbromin - similarity: 0.43699\n",
      "Document Anthranilic acid - similarity: 0.43697\n",
      "Document Satraplatin - similarity: 0.43697\n",
      "Document Platycodon grandiflorus leaf - similarity: 0.43695\n",
      "Document Dropropizine - similarity: 0.43685\n",
      "Document Ipecac - similarity: 0.43685\n",
      "Document Glycochenodeoxycholic Acid - similarity: 0.43684\n",
      "Document Myristoyl-Coa - similarity: 0.43683\n",
      "Document Lucitanib - similarity: 0.43677\n",
      "Document Lexacalcitol - similarity: 0.43667\n",
      "Document Fitusiran - similarity: 0.43664\n",
      "Document Tropicamide - similarity: 0.43659\n",
      "Document Vidarabine - similarity: 0.43655\n",
      "Document Turoctocog alfa pegol - similarity: 0.43654\n",
      "Document Oyster - similarity: 0.43654\n",
      "Document Iproclozide - similarity: 0.43645\n",
      "Document Safflower - similarity: 0.43634\n",
      "Document PRLX 93936 - similarity: 0.43628\n",
      "Document Chloral hydrate - similarity: 0.43628\n",
      "Document Alatrofloxacin - similarity: 0.43622\n",
      "Document (2S,3R)-2-[[4-(Tert-butylcarbamoyl)piperazine-1-carbonyl]amino]-6-(diaminomethylideneamino)-3-formylhexanoic acid - similarity: 0.43619\n",
      "Document Butalamine - similarity: 0.43613\n",
      "Document Tyramine - similarity: 0.43601\n",
      "Document Phosphonotyrosine - similarity: 0.43587\n",
      "Document Morclofone - similarity: 0.43584\n",
      "Document 7-methyl-5'-guanylic acid - similarity: 0.43583\n",
      "Document 3-[4-(AMINOSULFONYL)PHENYL]PROPANOIC ACID - similarity: 0.43573\n",
      "Document Rubidium Rb-82 - similarity: 0.43562\n",
      "Document N,N'-[biphenyl-4,4'-diyldi(2R)propane-2,1-diyl]dimethanesulfonamide - similarity: 0.43552\n",
      "Document (3R,4S)-1-[6-(6-METHOXYPYRIDIN-3-YL)PYRIMIDIN-4-YL]-4-(2,4,5-TRIFLUOROPHENYL)PYRROLIDIN-3-AMINE - similarity: 0.43536\n",
      "Document 2-Amino-3-Hydroxy-3-Phosphonooxy-Propionic Acid - similarity: 0.43533\n",
      "Document Dihydrocapsiate - similarity: 0.43532\n",
      "Document INS 316 - similarity: 0.43531\n",
      "Document Terizidone - similarity: 0.43530\n",
      "Document Madecassic acid - similarity: 0.43510\n",
      "Document Epicoccum nigrum - similarity: 0.43501\n",
      "Document Methylphenobarbital - similarity: 0.43493\n",
      "Document Desflurane - similarity: 0.43488\n",
      "Document Magnesium citrate - similarity: 0.43487\n",
      "Document Sea salt - similarity: 0.43485\n",
      "Document Chromium picolinate - similarity: 0.43474\n",
      "Document Carboquone - similarity: 0.43470\n",
      "Document Camicinal - similarity: 0.43465\n",
      "Document Fidarestat - similarity: 0.43465\n",
      "Document (4R,5R)-5-AMINO-1-[2-(1,3-BENZODIOXOL-5-YL)ETHYL]-4-(2,4,5-TRIFLUOROPHENYL)PIPERIDIN-2-ONE - similarity: 0.43463\n",
      "Document AM103 - similarity: 0.43458\n",
      "Document 1-(2-DEOXY-5-O-PHOSPHONO-BETA-D-ERYTHRO-PENTOFURANOSYL)-5-NITRO-1H-INDOLE - similarity: 0.43452\n",
      "Document Castor oil - similarity: 0.43449\n",
      "Document N-(dibenzo[b,d]thiophen-3-ylsulfonyl)-L-valine - similarity: 0.43446\n",
      "Document Orange - similarity: 0.43426\n",
      "Document J147 - similarity: 0.43422\n",
      "Document (5S)-2-(Cyclooctylamino)-5-methyl-5-propyl-1,3-thiazol-4(5H)-one - similarity: 0.43412\n",
      "Document Aluminium tristearate - similarity: 0.43405\n",
      "Document Fenoprofen - similarity: 0.43402\n",
      "Document Amino acids - similarity: 0.43397\n",
      "Document Mipomersen - similarity: 0.43391\n",
      "Document Streptococcus pneumoniae type 10a capsular polysaccharide antigen - similarity: 0.43390\n",
      "Document Alfentanil - similarity: 0.43380\n",
      "Document Imidurea - similarity: 0.43374\n",
      "Document Dihydromyricetin - similarity: 0.43371\n",
      "Document Agmatine - similarity: 0.43371\n",
      "Document 2-{[4-(TRIFLUOROMETHOXY)BENZOYL]AMINO}ETHYL DIHYDROGEN PHOSPHATE - similarity: 0.43358\n",
      "Document L-proline betaine - similarity: 0.43348\n",
      "Document Oseltamivir acid - similarity: 0.43343\n",
      "Document Propylparaben - similarity: 0.43340\n",
      "Document Smelt - similarity: 0.43336\n",
      "Document Besifloxacin - similarity: 0.43330\n",
      "Document Florbetaben (18F) - similarity: 0.43329\n",
      "Document Omega-3-acid ethyl esters - similarity: 0.43321\n",
      "Document (S)-Rolipram - similarity: 0.43321\n",
      "Document 5-[(2-methyl-5-{[3-(trifluoromethyl)phenyl]carbamoyl}phenyl)amino]pyridine-3-carboxamide - similarity: 0.43314\n",
      "Document Polygala senega root - similarity: 0.43313\n",
      "Document Telaglenastat - similarity: 0.43301\n",
      "Document Sulfanilamide - similarity: 0.43297\n",
      "Document Cupric sulfate - similarity: 0.43294\n",
      "Document N-(Bromoacetyl)-beta-alanyl-N-(2-{4-[(carboxycarbonyl)amino]phenyl}ethyl)-L-serinamide - similarity: 0.43290\n",
      "Document Clindamycin - similarity: 0.43288\n",
      "Document (4R)-4-(3-HYDROXYPHENYL)-N,N,7,8-TETRAMETHYL-3,4-DIHYDROISOQUINOLINE-2(1H)-CARBOXAMIDE - similarity: 0.43284\n",
      "Document 2-Methoxyestradiol - similarity: 0.43272\n",
      "Document Tragacanth - similarity: 0.43263\n",
      "Document Plevitrexed - similarity: 0.43259\n",
      "Document Inhibitor Idd 384 - similarity: 0.43255\n",
      "Document Docusate - similarity: 0.43252\n",
      "Document Florbenazine F-18 - similarity: 0.43251\n",
      "Document Ondelopran - similarity: 0.43246\n",
      "Document Agerafenib - similarity: 0.43245\n",
      "Document 1,2,4-Triazole - similarity: 0.43241\n",
      "Document Nitrofurantoin - similarity: 0.43237\n",
      "Document beta-Hydroxyasparagine - similarity: 0.43229\n",
      "Document Copper Cu-64 - similarity: 0.43208\n",
      "Document N-(carboxycarbonyl)-D-phenylalanine - similarity: 0.43206\n",
      "Document Cyclosporine - similarity: 0.43201\n",
      "Document UK-396,082 - similarity: 0.43199\n",
      "Document Roflumilast - similarity: 0.43198\n",
      "Document N'-(5-CHLORO-1,3-BENZODIOXOL-4-YL)-N-(3-MORPHOLIN-4-YLPHENYL)PYRIMIDINE-2,4-DIAMINE - similarity: 0.43193\n",
      "Document trifluoro-[hydroxy-[hydroxy-[2-(N-methyl-2-nitro-anilino)ethoxy]phosphoryl]oxy-phosphoryl]oxy-beryllium(1-) - similarity: 0.43192\n",
      "Document Floxuridine - similarity: 0.43191\n",
      "Document Bevacizumab zirconium Zr-89 - similarity: 0.43189\n",
      "Document Gosogliptin - similarity: 0.43189\n",
      "Document Pregnenolone acetate - similarity: 0.43186\n",
      "Document (S)-4-bromo-3-hydroxy-3-methylbutyl diphosphate - similarity: 0.43185\n",
      "Document Minodronic acid - similarity: 0.43180\n",
      "Document (2S,3S,8S,9S)-3-amino-9-methoxy-2,6,8-trimethyl-10-phenyldeca-4,6-dienoic acid - similarity: 0.43171\n",
      "Document SU-11652 - similarity: 0.43167\n",
      "Document Human papillomavirus type 52 L1 capsid protein antigen - similarity: 0.43166\n",
      "Document Pivmecillinam - similarity: 0.43161\n",
      "Document Senrebotase - similarity: 0.43153\n",
      "Document Sorbinil - similarity: 0.43133\n",
      "Document Sulfisoxazole - similarity: 0.43123\n",
      "Document GO-203-2C - similarity: 0.43115\n",
      "Document Prussian blue - similarity: 0.43111\n",
      "Document QAV-680 - similarity: 0.43109\n",
      "Document Grape - similarity: 0.43087\n",
      "Document Becatecarin - similarity: 0.43086\n",
      "Document 5,10-Methylene-6-Hydrofolic Acid - similarity: 0.43085\n",
      "Document Rifampicin - similarity: 0.43083\n",
      "Document 4-(4-CHLOROPHENYL)-4-[4-(1H-PYRAZOL-4-YL)PHENYL]PIPERIDINE - similarity: 0.43071\n",
      "Document Rhodamine 6G - similarity: 0.43071\n",
      "Document Eliglustat - similarity: 0.43049\n",
      "Document N~3~-BENZYLPYRIDINE-2,3-DIAMINE - similarity: 0.43046\n",
      "Document JNJ-39220675 - similarity: 0.43036\n",
      "Document BMS-184476 - similarity: 0.43031\n",
      "Document Potassium - similarity: 0.43030\n",
      "Document 3-Fluoro-2-(Phosphonooxy)Propanoic Acid - similarity: 0.43027\n",
      "Document Metixene - similarity: 0.43020\n",
      "Document Pipemidic acid - similarity: 0.43019\n",
      "Document Eteplirsen - similarity: 0.43016\n",
      "Document PF-04958242 - similarity: 0.43015\n",
      "Document Tauroursodeoxycholic acid - similarity: 0.43011\n",
      "Document Arotinolol - similarity: 0.42996\n",
      "Document N-(QUINOLIN-8-YL)METHANESULFONAMIDE - similarity: 0.42978\n",
      "Document CRx-401 - similarity: 0.42977\n",
      "Document (2Z,3E)-2,3'-biindole-2',3(1H,1'H)-dione 3-{O-[(3R)-3,4-dihydroxybutyl]oxime} - similarity: 0.42971\n",
      "Document 2-{[(1R,2S)-2-aminocyclohexyl]amino}-4-[(3-methylphenyl)amino]pyrimidine-5-carboxamide - similarity: 0.42965\n",
      "Document Ambrosia tenuifolia pollen - similarity: 0.42958\n",
      "Document N-(3-cyclopropyl-1H-pyrazol-5-yl)-2-(2-naphthyl)acetamide - similarity: 0.42947\n",
      "Document 2-[2-(1H-tetrazol-5-yl)ethyl]-1H-isoindole-1,3(2H)-dione - similarity: 0.42938\n",
      "Document S-[2-({N-[(2S)-2-hydroxy-3,3-dimethyl-4-(phosphonooxy)butanoyl]-beta-alanyl}amino)ethyl] octanethioate - similarity: 0.42930\n",
      "Document Eprodisate - similarity: 0.42929\n",
      "Document APD791 - similarity: 0.42918\n",
      "Document N-[7-(3-AMINOPHENYL)-5-METHOXY-1,3-BENZOXAZOL-2-YL]-2,5-DICHLOROBENZENESULFONAMIDE - similarity: 0.42916\n",
      "Document MK-0812 - similarity: 0.42912\n",
      "Document BMS-754807 - similarity: 0.42911\n",
      "Document Benzyl alcohol - similarity: 0.42907\n",
      "Document Linsitinib - similarity: 0.42894\n",
      "Document Iron protein succinylate - similarity: 0.42893\n",
      "Document 4-(4-fluoro-phenylazo)-5-imino-5H-pyrazol-3-ylamine - similarity: 0.42893\n",
      "Document Epinephrine - similarity: 0.42890\n",
      "Document 2-amino-4-[2,4-dichloro-5-(2-pyrrolidin-1-ylethoxy)phenyl]-N-ethylthieno[2,3-d]pyrimidine-6-carboxamide - similarity: 0.42887\n",
      "Document (2S)-2-(BUTYRYLOXY)-3-HYDROXYPROPYL NONANOATE - similarity: 0.42877\n",
      "Document Deferoxamine - similarity: 0.42866\n",
      "Document Edonerpic - similarity: 0.42861\n",
      "Document Iobenguane sulfate I-123 - similarity: 0.42860\n",
      "Document Inecalcitol - similarity: 0.42857\n",
      "Document Lumichrome - similarity: 0.42855\n",
      "Document Pridinol - similarity: 0.42854\n",
      "Document PTC299 - similarity: 0.42835\n",
      "Document Chloral betaine - similarity: 0.42827\n",
      "Document AR-42 - similarity: 0.42823\n",
      "Document Tribenoside - similarity: 0.42822\n",
      "Document Sulesomab - similarity: 0.42820\n",
      "Document Ricolinostat - similarity: 0.42816\n",
      "Document Ospemifene - similarity: 0.42814\n",
      "Document APD668 - similarity: 0.42813\n",
      "Document Icariin - similarity: 0.42807\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) antigen (propiolactone inactivated) - similarity: 0.42805\n",
      "Document XL784 - similarity: 0.42793\n",
      "Document Thyme - similarity: 0.42789\n",
      "Document Snubh-nm-333 F-18 - similarity: 0.42787\n",
      "Document Edotreotide gallium Ga-68 - similarity: 0.42787\n",
      "Document Influenza B virus B/Colorado/06/2017 live (attenuated) antigen - similarity: 0.42780\n",
      "Document Xenon-133 - similarity: 0.42779\n",
      "Document Ammonia - similarity: 0.42778\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.42773\n",
      "Document Boric acid - similarity: 0.42757\n",
      "Document N-[(1S)-1-{1-[(1R,3E)-1-ACETYLPENT-3-EN-1-YL]-1H-1,2,3-TRIAZOL-4-YL}-1,2-DIMETHYLPROPYL]BENZAMIDE - similarity: 0.42746\n",
      "Document Recanaclotide - similarity: 0.42746\n",
      "Document Isocarboxazid - similarity: 0.42744\n",
      "Document Mesterolone - similarity: 0.42738\n",
      "Document 2-amino-8-methyl-4(1H)-quinazolinone - similarity: 0.42732\n",
      "Document (2R)-2-[(1R)-1-[[(2Z)-2-(2-Amino-1,3-thiazol-4-yl)-2-methoxyiminoacetyl]amino]-2-oxoethyl]-5-methylidene-2H-1,3-thiazine-4-carboxylic acid - similarity: 0.42728\n",
      "Document 2-[(5,6-DIPHENYLFURO[2,3-D]PYRIMIDIN-4-YL)AMINO]ETHANOL - similarity: 0.42719\n",
      "Document ADL 10-0101 - similarity: 0.42717\n",
      "Document Norelgestromin - similarity: 0.42714\n",
      "Document (2S,3R)-3-(6-amino-9H-purin-9-yl)nonan-2-ol - similarity: 0.42711\n",
      "Document 2,3,-Dihydroxybenzoylserine - similarity: 0.42708\n",
      "Document Peanut - similarity: 0.42706\n",
      "Document TRV-120027 - similarity: 0.42703\n",
      "Document Horseradish - similarity: 0.42698\n",
      "Document 5-Carbamoyl-1,1':4',1''-terphenyl-3-carboxylic acid - similarity: 0.42688\n",
      "Document Prednisolone hemisuccinate - similarity: 0.42679\n",
      "Document LU-31130 - similarity: 0.42676\n",
      "Document Amobarbital - similarity: 0.42658\n",
      "Document Indoxyl sulfate - similarity: 0.42639\n",
      "Document Microcrystalline cellulose - similarity: 0.42632\n",
      "Document 5-(dodecylthio)-1H-1,2,3-triazole-4-carboxylic acid - similarity: 0.42631\n",
      "Document Terbutryn - similarity: 0.42630\n",
      "Document Galunisertib - similarity: 0.42626\n",
      "Document (S)-Propafenone - similarity: 0.42622\n",
      "Document Clofibrate - similarity: 0.42618\n",
      "Document Lormetazepam - similarity: 0.42616\n",
      "Document N-butyl-3-{[6-(9H-purin-6-ylamino)hexanoyl]amino}benzamide - similarity: 0.42615\n",
      "Document RAD-140 - similarity: 0.42614\n",
      "Document Phenylacetaldehyde - similarity: 0.42609\n",
      "Document (S)-Indapamide - similarity: 0.42608\n",
      "Document Calanolide A - similarity: 0.42605\n",
      "Document OXI-4503 - similarity: 0.42590\n",
      "Document Thiabendazole - similarity: 0.42580\n",
      "Document AVI-7537 - similarity: 0.42578\n",
      "Document 1-hydroperoxy-L-tryptophan - similarity: 0.42570\n",
      "Document Idronoxil - similarity: 0.42558\n",
      "Document Bfpet F-18 - similarity: 0.42556\n",
      "Document N-chlorotaurine - similarity: 0.42550\n",
      "Document Glyphosate - similarity: 0.42534\n",
      "Document LY2181308 - similarity: 0.42532\n",
      "Document Bermekimab - similarity: 0.42530\n",
      "Document Raphanus sativus - similarity: 0.42529\n",
      "Document Ibandronate - similarity: 0.42524\n",
      "Document Corynebacterium diphtheriae toxoid antigen (formaldehyde inactivated) - similarity: 0.42512\n",
      "Document Seltorexant - similarity: 0.42500\n",
      "Document MIP-1095 I-123 - similarity: 0.42497\n",
      "Document Avasimibe - similarity: 0.42496\n",
      "Document Yttrium Y-90 clivatuzumab tetraxetan - similarity: 0.42496\n",
      "Document Apomine - similarity: 0.42495\n",
      "Document Bismuth subsalicylate - similarity: 0.42494\n",
      "Document Bordetella pertussis fimbriae 2/3 antigen - similarity: 0.42494\n",
      "Document NAM NAPTHYLAMINOALANINE - similarity: 0.42493\n",
      "Document R1295 - similarity: 0.42492\n",
      "Document DCFPyL F-18 - similarity: 0.42470\n",
      "Document [4-(3-AMINOMETHYL-PHENYL)-PIPERIDIN-1-YL]-(5-PHENETHYL- PYRIDIN-3-YL)-METHANONE - similarity: 0.42469\n",
      "Document Flortaucipir F-18 - similarity: 0.42460\n",
      "Document Bisphenol A diglycidyl ether - similarity: 0.42460\n",
      "Document Monomethyl fumarate - similarity: 0.42458\n",
      "Document Dipivefrin - similarity: 0.42454\n",
      "Document NTx-265 - similarity: 0.42452\n",
      "Document Citicoline - similarity: 0.42452\n",
      "Document CRA_11092 - similarity: 0.42449\n",
      "Document Telatinib - similarity: 0.42437\n",
      "Document Garcinia gummi-gutta fruit - similarity: 0.42432\n",
      "Document Verteporfin - similarity: 0.42430\n",
      "Document Nitroglycerin - similarity: 0.42422\n",
      "Document Anisodamine - similarity: 0.42417\n",
      "Document Metribolone - similarity: 0.42416\n",
      "Document Liotrix - similarity: 0.42407\n",
      "Document 5-chloro-N-[(3R)-1-(2-{[2-fluoro-4-(2-oxopyridin-1(2H)-yl)phenyl]amino}-2-oxoethyl)pyrrolidin-3-yl]thiophene-2-carboxamide - similarity: 0.42403\n",
      "Document Lenograstim - similarity: 0.42395\n",
      "Document Tea tree oil - similarity: 0.42393\n",
      "Document Silicic acid - similarity: 0.42375\n",
      "Document Mono-[3,4-Dihydroxy-5-(5-Methyl-Benzoimidazol-1-Yl)-Tetrahydor-Furan-2-Ylmethyl] Ester - similarity: 0.42372\n",
      "Document Uric acid - similarity: 0.42366\n",
      "Document ORE-1001 - similarity: 0.42356\n",
      "Document Bean - similarity: 0.42349\n",
      "Document Fructose - similarity: 0.42349\n",
      "Document Setogepram - similarity: 0.42345\n",
      "Document Scutellaria baicalensis root - similarity: 0.42339\n",
      "Document Juniperus ashei pollen - similarity: 0.42339\n",
      "Document Rhodotorula mucilaginosa - similarity: 0.42338\n",
      "Document Phosphorylcolamine - similarity: 0.42336\n",
      "Document Adomiparin - similarity: 0.42331\n",
      "Document Varlilumab - similarity: 0.42319\n",
      "Document PTI-428 - similarity: 0.42316\n",
      "Document Syagrus romanzoffiana pollen - similarity: 0.42311\n",
      "Document 8-demethyl-8-dimethylamino-flavin-adenine-dinucleotide - similarity: 0.42293\n",
      "Document ABT-384 - similarity: 0.42292\n",
      "Document 4-{[4-AMINO-6-(CYCLOHEXYLMETHOXY)-5-NITROSOPYRIMIDIN-2-YL]AMINO}BENZAMIDE - similarity: 0.42287\n",
      "Document 5-HYDROXY-2-(4-HYDROXYPHENYL)-1-BENZOFURAN-7-CARBONITRILE - similarity: 0.42283\n",
      "Document Factor XIII (human) - similarity: 0.42281\n",
      "Document Nitric Oxide - similarity: 0.42270\n",
      "Document Amylopectin - similarity: 0.42269\n",
      "Document Piclamilast - similarity: 0.42265\n",
      "Document Guanidine - similarity: 0.42256\n",
      "Document Tarenflurbil - similarity: 0.42255\n",
      "Document D-Allopyranose - similarity: 0.42243\n",
      "Document Dexamethasone acetate - similarity: 0.42239\n",
      "Document Lucanthone - similarity: 0.42226\n",
      "Document AS-8112 - similarity: 0.42219\n",
      "Document Ioversol - similarity: 0.42218\n",
      "Document Lindane - similarity: 0.42208\n",
      "Document Hydrocortisone valerate - similarity: 0.42207\n",
      "Document Selamectin - similarity: 0.42196\n",
      "Document Phosphorus P-32 - similarity: 0.42194\n",
      "Document Favipiravir - similarity: 0.42191\n",
      "Document PRX-08066 - similarity: 0.42184\n",
      "Document Prezatide - similarity: 0.42184\n",
      "Document Sinecatechins - similarity: 0.42174\n",
      "Document Azanidazole - similarity: 0.42171\n",
      "Document Phenylethyl alcohol - similarity: 0.42166\n",
      "Document Verdiperstat - similarity: 0.42165\n",
      "Document Fraxinus latifolia pollen - similarity: 0.42164\n",
      "Document Influenza A virus A/Brisbane/02/2018 (H1N1) recombinant hemagglutinin antigen - similarity: 0.42162\n",
      "Document Pertuzumab - similarity: 0.42153\n",
      "Document Hydroxyproline - similarity: 0.42147\n",
      "Document Adenovirus type 7 vaccine live - similarity: 0.42147\n",
      "Document Gossypol - similarity: 0.42141\n",
      "Document 4-[3-(4-CHLOROPHENYL)-1H-PYRAZOL-5-YL]PIPERIDINE - similarity: 0.42136\n",
      "Document Isoluminol - similarity: 0.42123\n",
      "Document Metyrapone - similarity: 0.42113\n",
      "Document Resolvin E1 - similarity: 0.42111\n",
      "Document Isotretinoin - similarity: 0.42104\n",
      "Document [[(3R,4R,5S,6R)-4,5-dihydroxy-6-(hydroxymethyl)-3-(pentanoylamino)oxan-2-ylidene]amino] N-phenylcarbamate - similarity: 0.42096\n",
      "Document Glutamic acid - similarity: 0.42086\n",
      "Document Prajmaline - similarity: 0.42077\n",
      "Document Bemegride - similarity: 0.42077\n",
      "Document Amonafide - similarity: 0.42068\n",
      "Document CXL-1020 - similarity: 0.42062\n",
      "Document Omega-3 fatty acids - similarity: 0.42055\n",
      "Document Triethoxycaprylylsilane - similarity: 0.42054\n",
      "Document Influenza B virus B/Massachusetts/2/2012 live (attenuated) antigen - similarity: 0.42053\n",
      "Document Ezatiostat - similarity: 0.42053\n",
      "Document Fluorouracil - similarity: 0.42045\n",
      "Document Safflower oil - similarity: 0.42044\n",
      "Document Mdl-29951 - similarity: 0.42036\n",
      "Document Derazantinib - similarity: 0.42032\n",
      "Document Nystatin - similarity: 0.42027\n",
      "Document (2E)-N-{4-[(3-bromophenyl)amino]quinazolin-6-yl}-4-(dimethylamino)but-2-enamide - similarity: 0.42025\n",
      "Document Tubocurarine - similarity: 0.42025\n",
      "Document N-acetyl-alpha-D-glucosamine - similarity: 0.42022\n",
      "Document Tetracoccosporium paxianum - similarity: 0.42019\n",
      "Document (2S)-1-methyl-2-[(2S,4R)-2-methyl-4-phenylpentyl]piperidine - similarity: 0.41993\n",
      "Document Cetuximab - similarity: 0.41985\n",
      "Document Estradiol cypionate - similarity: 0.41982\n",
      "Document Pegargiminase - similarity: 0.41981\n",
      "Document Dimethyl sulfone - similarity: 0.41980\n",
      "Document Tecemotide - similarity: 0.41975\n",
      "Document Krypton Kr 81m - similarity: 0.41964\n",
      "Document 1D-myo-inositol 1,3,4-trisphosphate - similarity: 0.41962\n",
      "Document 4-CHLORO-3',3''-DIBROMOPHENOL-1,8-NAPHTHALEIN - similarity: 0.41961\n",
      "Document CRA_9678 - similarity: 0.41960\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) antigen (UV, formaldehyde inactivated) - similarity: 0.41952\n",
      "Document N~4~-(3-methyl-1H-indazol-6-yl)-N~2~-(3,4,5-trimethoxyphenyl)pyrimidine-2,4-diamine - similarity: 0.41952\n",
      "Document Setrobuvir - similarity: 0.41945\n",
      "Document Entecavir - similarity: 0.41942\n",
      "Document Vistusertib - similarity: 0.41941\n",
      "Document Cytidine-5'-Diphosphate - similarity: 0.41939\n",
      "Document Daniquidone - similarity: 0.41933\n",
      "Document Acetophenazine - similarity: 0.41932\n",
      "Document Istiratumab - similarity: 0.41930\n",
      "Document Ofloxacin - similarity: 0.41927\n",
      "Document 2,4-Dihydroxybenzoic Acid - similarity: 0.41908\n",
      "Document Gramicidin D - similarity: 0.41907\n",
      "Document Oxalic Acid - similarity: 0.41897\n",
      "Document 2-Amino-6-Aminomethyl-8-Phenylsulfanylmethyl-3h-Quinazolin-4-One - similarity: 0.41892\n",
      "Document (3R,4R)-4-(pyrrolidin-1-ylcarbonyl)-1-(quinoxalin-2-ylcarbonyl)pyrrolidin-3-amine - similarity: 0.41887\n",
      "Document 3-bromo-6-phenyl-N-(pyrimidin-5-ylmethyl)imidazo[1,2-a]pyridin-8-amine - similarity: 0.41885\n",
      "Document Mandelic acid - similarity: 0.41881\n",
      "Document Mecamylamine - similarity: 0.41876\n",
      "Document Risdiplam - similarity: 0.41872\n",
      "Document Hyperforin - similarity: 0.41842\n",
      "Document Framycetin - similarity: 0.41841\n",
      "Document Imipramine oxide - similarity: 0.41838\n",
      "Document Iodohippurate sodium I-131 - similarity: 0.41829\n",
      "Document 3-Pyridin-4-Yl-2,4-Dihydro-Indeno[1,2-.C.]Pyrazole - similarity: 0.41821\n",
      "Document FX06 - similarity: 0.41806\n",
      "Document Bufylline - similarity: 0.41799\n",
      "Document 2-OXOHEPTYLPHOSPHONIC ACID - similarity: 0.41797\n",
      "Document Deglucobalhimycin - similarity: 0.41795\n",
      "Document (4e)-4-Aminohex-4-Enoic Acid - similarity: 0.41786\n",
      "Document Cetyl ethylhexanoate - similarity: 0.41781\n",
      "Document Tannic acid - similarity: 0.41778\n",
      "Document Acetylsalicylic acid - similarity: 0.41776\n",
      "Document ADU-S100 - similarity: 0.41771\n",
      "Document Streptococcus pneumoniae type 9v capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.41770\n",
      "Document N-[(2R,3S)-3-AMINO-2-HYDROXY-4-PHENYLBUTYL]NAPHTHALENE-2-SULFONAMIDE - similarity: 0.41756\n",
      "Document (R)-Atenolol - similarity: 0.41749\n",
      "Document 1-Naphthyl-L-alanine - similarity: 0.41747\n",
      "Document (2R)-1-(2,6-dimethylphenoxy)propan-2-amine - similarity: 0.41746\n",
      "Document Riboflavin - similarity: 0.41746\n",
      "Document (2S)-2-[3-(AMINOMETHYL)PHENYL]-3-[(R)-[(1R)-1-{[(BENZYLOXY)CARBONYL]AMINO}-2-METHYLPROPYL](HYDROXY)PHOSPHORYL]PROPANOIC ACID - similarity: 0.41745\n",
      "Document Selenium Sulfide - similarity: 0.41743\n",
      "Document 2-[(2-methoxy-5-methylphenoxy)methyl]pyridine - similarity: 0.41732\n",
      "Document Temoporfin - similarity: 0.41723\n",
      "Document Valopicitabine - similarity: 0.41719\n",
      "Document Malachite Green - similarity: 0.41719\n",
      "Document 4-{[(2-Oxo-1,2-dihydro-3H-indol-3-ylidene)methyl]amino}-N-(1,3-thiazol-2-yl)benzenesulfonamide - similarity: 0.41698\n",
      "Document N-(5-CHLORO-BENZO[B]THIOPHEN-3-YLMETHYL)-2-[6-CHLORO-OXO-3-(2-PYRIDIN-2-YL-ETHYLAMINO)-2H-PYRAZIN-1-YL]-ACETAMIDE - similarity: 0.41692\n",
      "Document L-[(N-Hydroxyamino)Carbonyl]Phenylalanine - similarity: 0.41690\n",
      "Document Bis-Benzamidine - similarity: 0.41687\n",
      "Document 2-Phenylamino-Ethanesulfonic Acid - similarity: 0.41682\n",
      "Document ME-344 - similarity: 0.41682\n",
      "Document TAK-243 - similarity: 0.41680\n",
      "Document 4-{[(E)-2-(5-CHLOROTHIEN-2-YL)VINYL]SULFONYL}-1-(1H-PYRROLO[3,2-C]PYRIDIN-2-YLMETHYL)PIPERAZIN-2-ONE - similarity: 0.41679\n",
      "Document Indoleacetic acid - similarity: 0.41679\n",
      "Document Influenza B virus B/Phuket/3073/2013 antigen (formaldehyde inactivated) - similarity: 0.41677\n",
      "Document Tilia cordata pollen - similarity: 0.41675\n",
      "Document Elm - similarity: 0.41666\n",
      "Document N-methylnicotinamide - similarity: 0.41664\n",
      "Document AT1022 - similarity: 0.41657\n",
      "Document N-(3-MERCAPTOPROPANOYL)-D-ALANINE - similarity: 0.41653\n",
      "Document Allylprodine - similarity: 0.41648\n",
      "Document PF-03882845 - similarity: 0.41642\n",
      "Document Thiocolchicoside - similarity: 0.41624\n",
      "Document Ellagic acid - similarity: 0.41623\n",
      "Document Beet - similarity: 0.41604\n",
      "Document Etomidate - similarity: 0.41592\n",
      "Document Dinutuximab - similarity: 0.41591\n",
      "Document Homocysteine - similarity: 0.41575\n",
      "Document Vasopressin - similarity: 0.41574\n",
      "Document Mepiprazole - similarity: 0.41566\n",
      "Document Dipropyl-4-hydroxytryptamine - similarity: 0.41565\n",
      "Document Cytidine-5'-Monophosphate-5-N-Acetylneuraminic Acid - similarity: 0.41565\n",
      "Document N-{3-[(4-{[3-(TRIFLUOROMETHYL)PHENYL]AMINO}PYRIMIDIN-2-YL)AMINO]PHENYL}CYCLOPROPANECARBOXAMIDE - similarity: 0.41564\n",
      "Document Sulfamethoxypyridazine - similarity: 0.41562\n",
      "Document Quercus dumosa pollen - similarity: 0.41557\n",
      "Document Ranpirnase - similarity: 0.41557\n",
      "Document Lipegfilgrastim - similarity: 0.41550\n",
      "Document Deflazacort - similarity: 0.41547\n",
      "Document 2'-Chloro-Biphenyl-2,3-Diol - similarity: 0.41527\n",
      "Document SF1126 - similarity: 0.41525\n",
      "Document alpha-Methylleucine - similarity: 0.41525\n",
      "Document Fosaprepitant - similarity: 0.41524\n",
      "Document Pegvorhyaluronidase alfa - similarity: 0.41506\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Ranimustine - similarity: 0.41489\n",
      "Document ADX-10061 - similarity: 0.41488\n",
      "Document Tioxolone - similarity: 0.41481\n",
      "Document Potassium chloride - similarity: 0.41476\n",
      "Document Carbonate ion - similarity: 0.41456\n",
      "Document 11-MERCAPTOUNDECANOIC ACID - similarity: 0.41454\n",
      "Document Unoprostone - similarity: 0.41445\n",
      "Document Tirapazamine - similarity: 0.41443\n",
      "Document PN0621 - similarity: 0.41436\n",
      "Document N-(4-bromo-2-{[(3R,5S)-3,5-dimethylpiperidin-1-yl]carbonyl}phenyl)-4-morpholin-4-yl-4-oxobutanamide - similarity: 0.41425\n",
      "Document 1,2-Dihydroxynaphthalene - similarity: 0.41401\n",
      "Document Icatibant - similarity: 0.41401\n",
      "Document Magnesium acetate - similarity: 0.41397\n",
      "Document Cortisone acetate - similarity: 0.41382\n",
      "Document Starch, corn - similarity: 0.41381\n",
      "Document Mifamurtide - similarity: 0.41381\n",
      "Document Warfarin - similarity: 0.41374\n",
      "Document Influenza A virus A/Singapore/GP2050/2015 (H3N2) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.41364\n",
      "Document N(6)-(pyridoxal phosphate)-L-lysine - similarity: 0.41362\n",
      "Document 1,4-dithio-alpha-D-glucopyranose - similarity: 0.41356\n",
      "Document Hypromellose - similarity: 0.41347\n",
      "Document DG041 - similarity: 0.41346\n",
      "Document 5-phenyl-2-keto-valeric acid - similarity: 0.41343\n",
      "Document Dienestrol - similarity: 0.41341\n",
      "Document Triethylhexanoin - similarity: 0.41337\n",
      "Document Felbamate - similarity: 0.41330\n",
      "Document 1,3-Dinitrobenzene - similarity: 0.41329\n",
      "Document Afatinib - similarity: 0.41325\n",
      "Document Cinoxacin - similarity: 0.41304\n",
      "Document Flutriciclamide F-18 - similarity: 0.41303\n",
      "Document Methylprednisone - similarity: 0.41301\n",
      "Document 4-[(6-chloropyrazin-2-yl)amino]benzenesulfonamide - similarity: 0.41301\n",
      "Document Xanomeline - similarity: 0.41298\n",
      "Document Verdinexor - similarity: 0.41290\n",
      "Document N1-(1-Dimethylcarbamoyl-2-Phenyl-Ethyl)-2-Oxo-N4-(2-Pyridin-2-Yl-Ethyl)-Succinamide - similarity: 0.41287\n",
      "Document 4-Carboxycinnamic Acid - similarity: 0.41268\n",
      "Document 3-AMINO-AZACYCLOTRIDECAN-2-ONE - similarity: 0.41260\n",
      "Document Lanthanum carbonate - similarity: 0.41259\n",
      "Document Iotroxic acid - similarity: 0.41249\n",
      "Document Necitumumab - similarity: 0.41249\n",
      "Document Erythromycin - similarity: 0.41245\n",
      "Document Isopropyl myristate - similarity: 0.41244\n",
      "Document Chlorpropamide - similarity: 0.41237\n",
      "Document Poldine - similarity: 0.41232\n",
      "Document N-Methyl-4-{[(2-oxo-1,2-dihydro-3H-indol-3-ylidene)methyl]amino}benzenesulfonamide - similarity: 0.41227\n",
      "Document Etiracetam - similarity: 0.41220\n",
      "Document Ethyl pyruvate - similarity: 0.41220\n",
      "Document (3S,5E)-3-propyl-3,4-dihydrothieno[2,3-f][1,4]oxazepin-5(2H)-imine - similarity: 0.41216\n",
      "Document 5-methyl-6-phenylquinazoline-2,4-diamine - similarity: 0.41205\n",
      "Document Quercus stellata pollen - similarity: 0.41196\n",
      "Document N-[5-(1,1-DIOXIDOISOTHIAZOLIDIN-2-YL)-1H-INDAZOL-3-YL]-2-(4-PIPERIDIN-1-YLPHENYL)ACETAMIDE - similarity: 0.41189\n",
      "Document 6-oxouridine 5'-phosphate - similarity: 0.41180\n",
      "Document Dicoumarol - similarity: 0.41179\n",
      "Document Salirasib - similarity: 0.41175\n",
      "Document Monastrol - similarity: 0.41167\n",
      "Document Calcium carbonate - similarity: 0.41165\n",
      "Document Amediplase - similarity: 0.41164\n",
      "Document Genistein - similarity: 0.41144\n",
      "Document Palmitic Acid - similarity: 0.41138\n",
      "Document 3-hydroxyanthranilic acid - similarity: 0.41136\n",
      "Document Coproporphyrinogen III - similarity: 0.41135\n",
      "Document Citrus sinensis pollen - similarity: 0.41129\n",
      "Document Robenacoxib - similarity: 0.41129\n",
      "Document 2-[(5-hex-1-yn-1-ylfuran-2-yl)carbonyl]-N-methylhydrazinecarbothioamide - similarity: 0.41122\n",
      "Document Etaracizumab - similarity: 0.41121\n",
      "Document ALGRX 1207 - similarity: 0.41117\n",
      "Document 2-({[3,5-DIFLUORO-3'-(TRIFLUOROMETHOXY)BIPHENYL-4-YL]AMINO}CARBONYL)CYCLOPENT-1-ENE-1-CARBOXYLIC ACID - similarity: 0.41115\n",
      "Document Tetraglyme - similarity: 0.41111\n",
      "Document Dodecyltrimethylammonium - similarity: 0.41109\n",
      "Document Tregalizumab - similarity: 0.41099\n",
      "Document Melagatran - similarity: 0.41097\n",
      "Document Inulin - similarity: 0.41097\n",
      "Document p-Benzoyl-L-phenylalanine - similarity: 0.41097\n",
      "Document 4-Methyl-N-[(2S)-1-oxo-3-phenyl-1-[[(3S)-1-phenyl-5-(phenylmethoxysulfamoyl)pentan-3-yl]amino]propan-2-yl]piperazine-1-carboxamide - similarity: 0.41096\n",
      "Document (4R)-limonene 1α,2α-epoxide - similarity: 0.41094\n",
      "Document Toyocamycin - similarity: 0.41087\n",
      "Document Artemisia vulgaris pollen - similarity: 0.41086\n",
      "Document D-Glutamine - similarity: 0.41081\n",
      "Document Haemagglutinin-strain B (Victoria) - similarity: 0.41079\n",
      "Document Idursulfase - similarity: 0.41065\n",
      "Document Liatermin - similarity: 0.41063\n",
      "Document Heptyl 1-Thiohexopyranoside - similarity: 0.41059\n",
      "Document CP-744809 - similarity: 0.41056\n",
      "Document PF-04457845 - similarity: 0.41052\n",
      "Document Neopterin - similarity: 0.41043\n",
      "Document Pilsicainide - similarity: 0.41040\n",
      "Document 2-hydroxy-7-methoxy-5-methyl-naphthalene-1-carboxylic acid meso-2,5-dihydroxy-cyclopent-3-enyl ester - similarity: 0.41024\n",
      "Document EPC2407 - similarity: 0.41022\n",
      "Document Baricitinib - similarity: 0.41009\n",
      "Document Penicillium chrysogenum var. chrysogenum - similarity: 0.41001\n",
      "Document Naltrexone - similarity: 0.40993\n",
      "Document Neladenoson bialanate - similarity: 0.40993\n",
      "Document Nattokinase - similarity: 0.40990\n",
      "Document ABT-072 - similarity: 0.40985\n",
      "Document (2R)-4-[(8R)-8-METHYL-2-(TRIFLUOROMETHYL)-5,6-DIHYDRO[1,2,4]TRIAZOLO[1,5-A]PYRAZIN-7(8H)-YL]-4-OXO-1-(2,4,5-TRIFLUOROPHENYL)BUTAN-2-AMINE - similarity: 0.40984\n",
      "Document Imidocarb - similarity: 0.40965\n",
      "Document Edaravone - similarity: 0.40962\n",
      "Document Sodium bisulfite - similarity: 0.40959\n",
      "Document Levodropropizine - similarity: 0.40957\n",
      "Document (2S)-4-(2,5-DIFLUOROPHENYL)-N,N-DIMETHYL-2-PHENYL-2,5-DIHYDRO-1H-PYRROLE-1-CARBOXAMIDE - similarity: 0.40956\n",
      "Document Angiotensinamide - similarity: 0.40946\n",
      "Document 4-[(1S,2S,5S,9R)-5-(HYDROXYMETHYL)-8,9-DIMETHYL-3-OXABICYCLO[3.3.1]NON-7-EN-2-YL]PHENOL - similarity: 0.40946\n",
      "Document [(3S)-9-hydroxy-1-methyl-10-oxo-4,10-dihydro-3H-benzo[g]isochromen-3-yl]acetic acid - similarity: 0.40944\n",
      "Document Phosphorus - similarity: 0.40939\n",
      "Document Fontolizumab - similarity: 0.40937\n",
      "Document Combretastatin - similarity: 0.40913\n",
      "Document Deltamethrin - similarity: 0.40908\n",
      "Document Lemon - similarity: 0.40907\n",
      "Document 4-butyl-5-propyl-1,3-selenazol-2-amine - similarity: 0.40897\n",
      "Document Tilarginine - similarity: 0.40897\n",
      "Document Sorbitan - similarity: 0.40896\n",
      "Document Methamidophos - similarity: 0.40893\n",
      "Document 2,5-Dimethoxyamphetamine - similarity: 0.40887\n",
      "Document Haloxyfop-P - similarity: 0.40885\n",
      "Document GS-9191 - similarity: 0.40879\n",
      "Document Endralazine - similarity: 0.40878\n",
      "Document Kynurenic Acid - similarity: 0.40862\n",
      "Document 5-(PARA-NITROPHENYL PHOSPHONATE)-PENTANOIC ACID - similarity: 0.40861\n",
      "Document Aminoethyl nitrate - similarity: 0.40858\n",
      "Document Glucosamine 6-Phosphate - similarity: 0.40855\n",
      "Document 2-Aminoquinazolin-4(3h)-One - similarity: 0.40851\n",
      "Document 3-cyclohexyl-D-alanyl-N-(3-chlorobenzyl)-L-prolinamide - similarity: 0.40848\n",
      "Document Salicylhydroxamic Acid - similarity: 0.40838\n",
      "Document Pirbuterol - similarity: 0.40836\n",
      "Document Istradefylline - similarity: 0.40832\n",
      "Document Ibodutant - similarity: 0.40826\n",
      "Document 2-[3-(2-Hydroxy-1,1-Dihydroxymethyl-Ethylamino)-Propylamino]-2-Hydroxymethyl-Propane-1,3-Diol - similarity: 0.40823\n",
      "Document Laropiprant - similarity: 0.40821\n",
      "Document Amezinium metilsulfate - similarity: 0.40809\n",
      "Document N-Hydroxy-N-Isopropyloxamic Acid - similarity: 0.40806\n",
      "Document Indalpine - similarity: 0.40805\n",
      "Document Brinzolamide - similarity: 0.40803\n",
      "Document NUC B1000 - similarity: 0.40802\n",
      "Document AGS-005 - similarity: 0.40794\n",
      "Document Diphenoxylate - similarity: 0.40793\n",
      "Document 1-[4-(hydroxymethyl)phenyl]guanidine - similarity: 0.40783\n",
      "Document Danirixin - similarity: 0.40774\n",
      "Document CXA-10 - similarity: 0.40769\n",
      "Document Foliglurax - similarity: 0.40763\n",
      "Document Compound 4-D - similarity: 0.40762\n",
      "Document Lutetium Lu 177 dotatate - similarity: 0.40755\n",
      "Document Dantrolene - similarity: 0.40752\n",
      "Document 5-Bromo-N-[(2S)-2,3-dihydroxypropoxy]-3,4-difluoro-2-[(2-fluoro-4-iodophenyl)amino]benzamide - similarity: 0.40751\n",
      "Document Sour cherry - similarity: 0.40743\n",
      "Document BLX-883 - similarity: 0.40742\n",
      "Document (1R,2R,3S,4R,6S)-4,6-diamino-2-{[3-O-(2,6-diamino-2,6-dideoxy-beta-L-idopyranosyl)-2-O-{2-[(piperidin-3-ylmethyl)amino]ethyl}-beta-D-ribofuranosyl]oxy}-3-hydroxycyclohexyl 2-amino-2-deoxy-alpha-D-glucopyranoside - similarity: 0.40730\n",
      "Document Conestat alfa - similarity: 0.40722\n",
      "Document 6-CHLORO-4-(CYCLOHEXYLOXY)-3-PROPYLQUINOLIN-2(1H)-ONE - similarity: 0.40705\n",
      "Document 6-Chloropurine Riboside, 5'-Monophosphate - similarity: 0.40699\n",
      "Document 7-[(3-CHLOROBENZYL)OXY]-4-[(METHYLAMINO)METHYL]-2H-CHROMEN-2-ONE - similarity: 0.40698\n",
      "Document ATG003 - similarity: 0.40689\n",
      "Document Cellobiose - similarity: 0.40659\n",
      "Document Motavizumab - similarity: 0.40658\n",
      "Document Edrecolomab - similarity: 0.40651\n",
      "Document Adenosine-2'-5'-Diphosphate - similarity: 0.40643\n",
      "Document Tiotropium - similarity: 0.40642\n",
      "Document Sarcosine - similarity: 0.40641\n",
      "Document 2-(6-CHLORO-3-{[2,2-DIFLUORO-2-(2-PYRIDINYL)ETHYL]AMINO}-2-OXO-1(2H)-PYRAZINYL)-N-[(2-FLUORO-3-METHYL-6-PYRIDINYL)METHYL]ACETAMIDE - similarity: 0.40640\n",
      "Document 2-Fluoro-2-Deoxy-Beta-D-Galactopyranosyl-Beta-D-Glucopyranose - similarity: 0.40637\n",
      "Document Calcium phosphate dihydrate - similarity: 0.40633\n",
      "Document 2'-Deoxycytidine-5'-Monophosphate - similarity: 0.40628\n",
      "Document Boron - similarity: 0.40624\n",
      "Document Pentetic acid - similarity: 0.40609\n",
      "Document Cholestyramine - similarity: 0.40608\n",
      "Document N-(1,4-dihydro-5H-tetrazol-5-ylidene)-9-oxo-9H-xanthene-2-sulfonamide - similarity: 0.40605\n",
      "Document Tomivosertib - similarity: 0.40592\n",
      "Document SNS-032 - similarity: 0.40589\n",
      "Document (S)-3-phenyllactic acid - similarity: 0.40589\n",
      "Document Maltodextrin - similarity: 0.40588\n",
      "Document Niclosamide - similarity: 0.40580\n",
      "Document 4-[5-[2-(1-phenyl-ethylamino)-pyrimidin-4-yl]-1-methyl-4-(3-trifluoromethylphenyl)-1H-imidazol-2-yl]-piperidine - similarity: 0.40570\n",
      "Document Metrifonate - similarity: 0.40561\n",
      "Document Lintuzumab satetraxetan AC-225 - similarity: 0.40556\n",
      "Document Tigapotide - similarity: 0.40555\n",
      "Document 5-AMINO-6-CYCLOHEXYL-4-HYDROXY-2-ISOBUTYL-HEXANOIC ACID - similarity: 0.40553\n",
      "Document Vayarin - similarity: 0.40553\n",
      "Document Dinaciclib - similarity: 0.40552\n",
      "Document T-2000 - similarity: 0.40552\n",
      "Document S-2-(Boronoethyl)-L-Cysteine - similarity: 0.40550\n",
      "Document Coagulation factor VIIa Recombinant Human - similarity: 0.40547\n",
      "Document diethyl [(1R)-1,5-diaminopentyl]boronate - similarity: 0.40544\n",
      "Document Corticorelin acetate - similarity: 0.40521\n",
      "Document Promethazine - similarity: 0.40513\n",
      "Document Potassium bicarbonate - similarity: 0.40510\n",
      "Document Grape seed oil - similarity: 0.40509\n",
      "Document 2-carboxypropyl-coenzyme A - similarity: 0.40505\n",
      "Document Apocynin - similarity: 0.40501\n",
      "Document Acrylic Acid - similarity: 0.40489\n",
      "Document N-{3-[4-Hydroxy-1-(3-methylbutyl)-2-oxo-1,2-dihydropyrrolo[1,2-b]pyridazin-3-yl]-1,1-dioxido-2H-1,2,4-benzothiadiazin-7-yl}methanesulfonamide - similarity: 0.40472\n",
      "Document Dipropylene glycol - similarity: 0.40472\n",
      "Document Duvortuxizumab - similarity: 0.40467\n",
      "Document 10-(4-Dimethylamino-5-Hydroxy-6-Methyl-Tetrahydro-Pyran-2-Yloxy)-8-Ethyl-1,8,11-Trihydroxy-7,8,9,10-Tetrahydro-Naphthacene-5,12-Dione - similarity: 0.40467\n",
      "Document L-Saccharopine - similarity: 0.40465\n",
      "Document Vinyl ether - similarity: 0.40462\n",
      "Document Halothane - similarity: 0.40461\n",
      "Document Mucor circinelloides f. circinelloides - similarity: 0.40429\n",
      "Document Isometheptene - similarity: 0.40413\n",
      "Document LY374571 - similarity: 0.40412\n",
      "Document Fletikumab - similarity: 0.40411\n",
      "Document SB220025 - similarity: 0.40411\n",
      "Document Muscimol - similarity: 0.40410\n",
      "Document 3-hydroxyglutaric acid - similarity: 0.40409\n",
      "Document Lumateperone - similarity: 0.40400\n",
      "Document Aluminum zirconium tetrachlorohydrex gly - similarity: 0.40399\n",
      "Document Nikkomycin Z - similarity: 0.40390\n",
      "Document L-methionine (R)-S-oxide - similarity: 0.40390\n",
      "Document 5-Bromonicotinamide - similarity: 0.40390\n",
      "Document Nafarelin - similarity: 0.40385\n",
      "Document Fluticasone - similarity: 0.40381\n",
      "Document Alitretinoin - similarity: 0.40376\n",
      "Document Indium In-111 chloride - similarity: 0.40362\n",
      "Document Vanillyl alcohol - similarity: 0.40360\n",
      "Document Betula occidentalis pollen - similarity: 0.40356\n",
      "Document 1-((2-HYDROXYETHOXY)METHYL)-5-(3-(BENZYLOXY)BENZYL)-6-HYDROXYPYRIMIDINE-2,4(1H,3H)-DIONE - similarity: 0.40348\n",
      "Document Bird pepper - similarity: 0.40344\n",
      "Document Macadamia nut - similarity: 0.40340\n",
      "Document Cathinone - similarity: 0.40339\n",
      "Document Nylidrin - similarity: 0.40338\n",
      "Document Tetrodotoxin - similarity: 0.40337\n",
      "Document N-[(2R)-2-{[(2S)-2-(1,3-benzoxazol-2-yl)pyrrolidin-1-yl]carbonyl}hexyl]-N-hydroxyformamide - similarity: 0.40333\n",
      "Document Manganese citrate - similarity: 0.40333\n",
      "Document Bindarit - similarity: 0.40328\n",
      "Document Prostaglandin B2 - similarity: 0.40325\n",
      "Document Cobimetinib - similarity: 0.40323\n",
      "Document Tretazicar - similarity: 0.40316\n",
      "Document Phosphonopyruvate - similarity: 0.40312\n",
      "Document Meningococcal polysaccharide vaccine group Y - similarity: 0.40310\n",
      "Document Abituzumab - similarity: 0.40298\n",
      "Document GSK-256066 - similarity: 0.40296\n",
      "Document Sodium lauryl sulfate - similarity: 0.40288\n",
      "Document (S)-carazolol - similarity: 0.40271\n",
      "Document Typha latifolia pollen - similarity: 0.40270\n",
      "Document Dihydroxyacetone - similarity: 0.40266\n",
      "Document 2,6-dibromo-4-[(E)-2-phenylethenyl]phenol - similarity: 0.40262\n",
      "Document 4-(Acetylamino)-3-[(Aminoacetyl)Amino]Benzoic Acid - similarity: 0.40256\n",
      "Document Lumiflavin - similarity: 0.40255\n",
      "Document Phleum pratense top - similarity: 0.40254\n",
      "Document (2S)-2-HYDROXYOCTANOIC ACID - similarity: 0.40252\n",
      "Document Tryptophan - similarity: 0.40249\n",
      "Document Naftopidil - similarity: 0.40243\n",
      "Document Myristic acid - similarity: 0.40240\n",
      "Document (S)-wiskostatin - similarity: 0.40236\n",
      "Document 4-phospho-L-threonic acid - similarity: 0.40233\n",
      "Document 1,3,4-TRIHYDROXY-5-(3-PHENOXYPROPYL)-CYCLOHEXANE-1-CARBOXYLIC A CID - similarity: 0.40231\n",
      "Document 8-(2,5-Dimethoxy-Benzyl)-2-Fluoro-9h-Purin-6-Ylamine - similarity: 0.40230\n",
      "Document Demegestone - similarity: 0.40222\n",
      "Document Stanolone - similarity: 0.40214\n",
      "Document 1,2-Distearoyllecithin - similarity: 0.40214\n",
      "Document Penicillamine - similarity: 0.40213\n",
      "Document ASP-3026 - similarity: 0.40203\n",
      "Document HMR4011 - similarity: 0.40195\n",
      "Document Phosphoramidon - similarity: 0.40193\n",
      "Document Calcium acetate - similarity: 0.40188\n",
      "Document Talotrexin - similarity: 0.40188\n",
      "Document Motexafin lutetium - similarity: 0.40182\n",
      "Document Human papillomavirus type 16 L1 capsid protein residues 2-471 antigen - similarity: 0.40180\n",
      "Document 2-[(2-chloro-4-iodophenyl)amino]-N-{[(2R)-2,3-dihydroxypropyl]oxy}-3,4-difluorobenzamide - similarity: 0.40165\n",
      "Document 8-anilinonaphthalene-1-sulfonic acid - similarity: 0.40152\n",
      "Document Cenicriviroc - similarity: 0.40151\n",
      "Document Elinogrel - similarity: 0.40147\n",
      "Document (3S)-1-CYCLOHEXYL-N-(3,5-DICHLOROPHENYL)-5-OXOPYRROLIDINE-3-CARBOXAMIDE - similarity: 0.40140\n",
      "Document Thymosin beta-4 - similarity: 0.40137\n",
      "Document (3R,4S)-1-{6-[3-(METHYLSULFONYL)PHENYL]PYRIMIDIN-4-YL}-4-(2,4,5-TRIFLUOROPHENYL)PYRROLIDIN-3-AMINE - similarity: 0.40136\n",
      "Document Boceprevir - similarity: 0.40128\n",
      "Document (1R)-menthyl hexyl phosphonate group - similarity: 0.40124\n",
      "Document Brasofensine - similarity: 0.40122\n",
      "Document β-Methylfentanyl - similarity: 0.40114\n",
      "Document Brolucizumab - similarity: 0.40103\n",
      "Document Cherry - similarity: 0.40096\n",
      "Document S-40503 - similarity: 0.40090\n",
      "Document LY-2452473 - similarity: 0.40076\n",
      "Document Iodocholesterol (131I) - similarity: 0.40060\n",
      "Document 2-(4-CHLOROBENZYLAMINO)-4-(PHENYLAMINO)PYRAZOLO[1,5-A][1,3,5]TRIAZINE-8-CARBONITRILE - similarity: 0.40056\n",
      "Document Lutetium Lu-177 - similarity: 0.40032\n",
      "Document Niraparib - similarity: 0.40020\n",
      "Document 5-deoxyflavanone - similarity: 0.40020\n",
      "Document Mevalonic acid - similarity: 0.40020\n",
      "Document Ovalicin - similarity: 0.40020\n",
      "Document Erteberel - similarity: 0.40017\n",
      "Document 4-[(5-ISOPROPYL-1,3-THIAZOL-2-YL)AMINO]BENZENESULFONAMIDE - similarity: 0.40015\n",
      "Document Buthionine Sulfoximine - similarity: 0.40008\n",
      "Document Guanidinoethylmercaptosuccinic acid - similarity: 0.40007\n",
      "Document 5-amino-1-(4-chlorophenyl)-1H-pyrazole-4-carbonitrile - similarity: 0.40002\n",
      "Document Icodextrin - similarity: 0.40000\n",
      "Document Pheneticillin - similarity: 0.39989\n",
      "Document Bilberry - similarity: 0.39984\n",
      "Document Nelipepimut-S - similarity: 0.39979\n",
      "Document Cytidine-5'-diphospho-beta-D-xylose - similarity: 0.39977\n",
      "Document 1-[Glycerolylphosphonyl]-2-[8-(2-Hexyl-Cyclopropyl)-Octanal-1-Yl]-3-[Hexadecanal-1-Yl]-Glycerol - similarity: 0.39961\n",
      "Document Influenza A virus A/California/7/2009 X-179A (H1N1) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.39958\n",
      "Document 1-naphthaleneacetic acid - similarity: 0.39952\n",
      "Document Polygala tenuifolia root - similarity: 0.39942\n",
      "Document Neisseria meningitidis group a capsular polysaccharide diphtheria toxoid conjugate antigen - similarity: 0.39938\n",
      "Document 7-beta-Hydroxyepiandrosterone - similarity: 0.39937\n",
      "Document Siramesine - similarity: 0.39934\n",
      "Document 3-[3-(4-methylpiperazin-1-yl)-7-(trifluoromethyl)quinoxalin-5-yl]phenol - similarity: 0.39921\n",
      "Document 2-Amino-N,3,3-Trimethylbutanamide - similarity: 0.39916\n",
      "Document ABC-294640 - similarity: 0.39915\n",
      "Document 2'-Deoxycytidine - similarity: 0.39906\n",
      "Document Sizofiran - similarity: 0.39906\n",
      "Document Norepinephrine - similarity: 0.39894\n",
      "Document Capivasertib - similarity: 0.39873\n",
      "Document Tetrahydrofolic acid - similarity: 0.39871\n",
      "Document Stem bromelain - similarity: 0.39860\n",
      "Document L-Lactic acid - similarity: 0.39856\n",
      "Document Influenza A virus A/Brisbane/59/2007(H1N1) antigen (propiolactone inactivated) - similarity: 0.39856\n",
      "Document Lumicitabine - similarity: 0.39856\n",
      "Document Triethyl phosphate - similarity: 0.39853\n",
      "Document Sulfaquinoxaline - similarity: 0.39851\n",
      "Document Isoamyl alcohol - similarity: 0.39851\n",
      "Document Velusetrag - similarity: 0.39851\n",
      "Document Alagebrium - similarity: 0.39830\n",
      "Document Macozinone - similarity: 0.39828\n",
      "Document Olcegepant - similarity: 0.39822\n",
      "Document Nitrofural - similarity: 0.39820\n",
      "Document Silanol - similarity: 0.39804\n",
      "Document 2-(4-(AMINOMETHYL)PIPERIDIN-1-YL)-N-(3_CYCLOHEXYL-4-OXO-2,4-DIHYDROINDENO[1,2-C]PYRAZOL-5-YL)ACETAMIDE - similarity: 0.39781\n",
      "Document Nitromethyldethia coenzyme A - similarity: 0.39776\n",
      "Document Rufloxacin - similarity: 0.39757\n",
      "Document Indopan - similarity: 0.39744\n",
      "Document Didox - similarity: 0.39739\n",
      "Document Protriptyline - similarity: 0.39737\n",
      "Document 3-FLUORO-5-MORPHOLIN-4-YL-N-[1-(2-PYRIDIN-4-YLETHYL)-1H-INDOL-6-YL]BENZAMIDE - similarity: 0.39721\n",
      "Document CRA_10655 - similarity: 0.39716\n",
      "Document Cloxazolam - similarity: 0.39707\n",
      "Document Sepetaprost - similarity: 0.39702\n",
      "Document Fluprostenol - similarity: 0.39685\n",
      "Document Diacerein - similarity: 0.39683\n",
      "Document Tovetumab - similarity: 0.39677\n",
      "Document Bilastine - similarity: 0.39669\n",
      "Document Cythioate - similarity: 0.39667\n",
      "Document Stanolone acetate - similarity: 0.39665\n",
      "Document α-Methylfentanyl - similarity: 0.39660\n",
      "Document GSK-356278 - similarity: 0.39646\n",
      "Document Mackerel - similarity: 0.39632\n",
      "Document Cupric oxide - similarity: 0.39630\n",
      "Document Sodium metavanadate - similarity: 0.39628\n",
      "Document 4-[(7-OXO-7H-THIAZOLO[5,4-E]INDOL-8-YLMETHYL)-AMINO]-N-PYRIDIN-2-YL-BENZENESULFONAMIDE - similarity: 0.39620\n",
      "Document Dimethylcurcumin - similarity: 0.39617\n",
      "Document Mannitol - similarity: 0.39610\n",
      "Document ALTU-135 - similarity: 0.39610\n",
      "Document 5,6-DIPHENYL-N-(2-PIPERAZIN-1-YLETHYL)FURO[2,3-D]PYRIMIDIN-4-AMINE - similarity: 0.39608\n",
      "Document Attapulgite - similarity: 0.39593\n",
      "Document SD-0006 - similarity: 0.39581\n",
      "Document Carbazole - similarity: 0.39575\n",
      "Document Ingavirin - similarity: 0.39573\n",
      "Document Inotersen - similarity: 0.39568\n",
      "Document Polatuzumab vedotin - similarity: 0.39555\n",
      "Document trans-urocanic acid - similarity: 0.39547\n",
      "Document Ancrod - similarity: 0.39545\n",
      "Document PT2977 - similarity: 0.39539\n",
      "Document 4-nitrophenyl-beta-D-galactoside - similarity: 0.39538\n",
      "Document Carbutamide - similarity: 0.39534\n",
      "Document Delafloxacin - similarity: 0.39530\n",
      "Document Amaranthus retroflexus whole - similarity: 0.39522\n",
      "Document Fosnetupitant - similarity: 0.39519\n",
      "Document Barium - similarity: 0.39510\n",
      "Document Gallium cation Ga-67 - similarity: 0.39500\n",
      "Document Desogestrel - similarity: 0.39498\n",
      "Document DCFBC F-18 - similarity: 0.39497\n",
      "Document Torapsel - similarity: 0.39491\n",
      "Document Pinus resinosa pollen - similarity: 0.39485\n",
      "Document {[(2,6-difluorophenyl)carbonyl]amino}-N-(4-fluorophenyl)-1H-pyrazole-3-carboxamide - similarity: 0.39479\n",
      "Document 1-Methylimidazole - similarity: 0.39475\n",
      "Document TAK-428 - similarity: 0.39474\n",
      "Document Rolicyclidine - similarity: 0.39471\n",
      "Document Colforsin - similarity: 0.39463\n",
      "Document BNC105 - similarity: 0.39459\n",
      "Document CTS-1027 - similarity: 0.39451\n",
      "Document Velpatasvir - similarity: 0.39447\n",
      "Document 7-Deazaguanine - similarity: 0.39443\n",
      "Document Iodoxamic acid - similarity: 0.39437\n",
      "Document Hesperetin - similarity: 0.39436\n",
      "Document Lancovutide - similarity: 0.39433\n",
      "Document PANTOTHENYL-AMINOETHANOL-11-PIVALIC ACID - similarity: 0.39422\n",
      "Document Upamostat - similarity: 0.39408\n",
      "Document Rosin - similarity: 0.39401\n",
      "Document Nolatrexed - similarity: 0.39394\n",
      "Document Ctenocephalides canis - similarity: 0.39394\n",
      "Document Cyclohexanol - similarity: 0.39390\n",
      "Document Rimiterol - similarity: 0.39388\n",
      "Document GnRH pharmaccine - similarity: 0.39383\n",
      "Document Mianserin - similarity: 0.39361\n",
      "Document 1,2-dichlorobenzene - similarity: 0.39360\n",
      "Document Diethylstilbestrol - similarity: 0.39354\n",
      "Document Tryptophanyl-5'amp - similarity: 0.39339\n",
      "Document Azapetine - similarity: 0.39338\n",
      "Document EC-17 - similarity: 0.39335\n",
      "Document Cinnarizine - similarity: 0.39325\n",
      "Document 2-(3-BENZOYLPHENOXY)ETHYL(HYDROXY)FORMAMIDE - similarity: 0.39320\n",
      "Document Temsavir - similarity: 0.39318\n",
      "Document CC-115 - similarity: 0.39316\n",
      "Document (2S,3S)-3-{3-[4-(METHYLSULFONYL)PHENYL]-1,2,4-OXADIAZOL-5-YL}-1-OXO-1-PYRROLIDIN-1-YLBUTAN-2-AMINE - similarity: 0.39295\n",
      "Document Antihemophilic factor, human recombinant - similarity: 0.39289\n",
      "Document Aldoxorubicin - similarity: 0.39289\n",
      "Document Liothyronine I-131 - similarity: 0.39285\n",
      "Document Solenopsis richteri - similarity: 0.39284\n",
      "Document Reviparin - similarity: 0.39283\n",
      "Document Chlorfenson - similarity: 0.39274\n",
      "Document Marzeptacog alfa (activated) - similarity: 0.39267\n",
      "Document N-(4-PHENYLAMINO-QUINAZOLIN-6-YL)-ACRYLAMIDE - similarity: 0.39266\n",
      "Document Tribromometacresol - similarity: 0.39264\n",
      "Document Acalisib - similarity: 0.39262\n",
      "Document 4-(Aminosulfonyl)-N-[(3,4,5-Trifluorophenyl)Methyl]-Benzamide - similarity: 0.39253\n",
      "Document Pralnacasan - similarity: 0.39250\n",
      "Document 2,5-Xylidine - similarity: 0.39237\n",
      "Document Nepidermin - similarity: 0.39223\n",
      "Document 2-[1-METHYLHEXYL]-4,6-DINITROPHENOL - similarity: 0.39216\n",
      "Document 4-(4-METHYLPIPERAZIN-1-YL)-N-[5-(2-THIENYLACETYL)-1,5-DIHYDROPYRROLO[3,4-C]PYRAZOL-3-YL]BENZAMIDE - similarity: 0.39213\n",
      "Document O-Trifluoromethylphenyl Anthranilic Acid - similarity: 0.39207\n",
      "Document Human interferon beta - similarity: 0.39202\n",
      "Document Metralindole - similarity: 0.39198\n",
      "Document Propatyl nitrate - similarity: 0.39197\n",
      "Document Trolamine salicylate - similarity: 0.39194\n",
      "Document Orciprenaline - similarity: 0.39181\n",
      "Document Anetumab ravtansine - similarity: 0.39173\n",
      "Document BMS-986142 - similarity: 0.39156\n",
      "Document Icaritin - similarity: 0.39146\n",
      "Document Rubella virus vaccine - similarity: 0.39138\n",
      "Document (S)-N-(4-carbamimidoylbenzyl)-1-(3-cyclopentylpropanoyl)pyrrolidine-2-carboxamide - similarity: 0.39133\n",
      "Document Aluminium acetotartrate - similarity: 0.39112\n",
      "Document Carbaphosphonate - similarity: 0.39105\n",
      "Document 2-{[(6-OXO-1,6-DIHYDROPYRIDIN-3-YL)METHYL]AMINO}-N-[4-PROPYL-3-(TRIFLUOROMETHYL)PHENYL]BENZAMIDE - similarity: 0.39103\n",
      "Document Benzylamine - similarity: 0.39102\n",
      "Document Fucitol - similarity: 0.39093\n",
      "Document N-[2-(1,3-BENZODIOXOL-5-YL)ETHYL]-1-[2-(1H-IMIDAZOL-1-YL)-6-METHYLPYRIMIDIN-4-YL]-D-PROLINAMIDE - similarity: 0.39089\n",
      "Document Benzoin - similarity: 0.39086\n",
      "Document (3s)-3,4-Di-N-Hexanoyloxybutyl-1-Phosphocholine - similarity: 0.39084\n",
      "Document Bithionol - similarity: 0.39081\n",
      "Document 1-[4-(AMINOSULFONYL)PHENYL]-1,6-DIHYDROPYRAZOLO[3,4-E]INDAZOLE-3-CARBOXAMIDE - similarity: 0.39077\n",
      "Document (S)-2-Amino-3-(4h-Selenolo[3,2-B]-Pyrrol-6-Yl)-Propionic Acid - similarity: 0.39077\n",
      "Document Choline C-11 - similarity: 0.39077\n",
      "Document BMS-830216 - similarity: 0.39063\n",
      "Document IMIDAZO[2,1-A]ISOQUINOLINE-2-CARBOHYDRAZIDE - similarity: 0.39063\n",
      "Document Beta-D-Fructopyranose - similarity: 0.39055\n",
      "Document Anas platyrhynchos feather - similarity: 0.39053\n",
      "Document Chitosan low molecular weight (20-200 mpa.s) - similarity: 0.39039\n",
      "Document MK-3118 - similarity: 0.39036\n",
      "Document Racotumomab - similarity: 0.39031\n",
      "Document Enasidenib - similarity: 0.39028\n",
      "Document Bismuth cation - similarity: 0.39026\n",
      "Document Lecanemab - similarity: 0.39019\n",
      "Document (5S)-5-(3-AMINOPROPYL)-3-(2,5-DIFLUOROPHENYL)-N-ETHYL-5-PHENYL-4,5-DIHYDRO-1H-PYRAZOLE-1-CARBOXAMIDE - similarity: 0.39009\n",
      "Document Bamethan - similarity: 0.39001\n",
      "Document Bifenthrin - similarity: 0.38996\n",
      "Document Diphenylguanidine - similarity: 0.38987\n",
      "Document Zanubrutinib - similarity: 0.38978\n",
      "Document OT-551 - similarity: 0.38975\n",
      "Document Influenza B virus B/Phuket/3073/2013 BVR-1B hemagglutinin antigen (propiolactone inactivated) - similarity: 0.38957\n",
      "Document Capecitabine - similarity: 0.38948\n",
      "Document ETHYL 4-[(4-CHLOROPYRIDIN-2-YL)AMINO]PIPERIDINE-1-CARBOXYLATE - similarity: 0.38947\n",
      "Document Bardoxolone methyl - similarity: 0.38946\n",
      "Document Sibutramine - similarity: 0.38945\n",
      "Document AL7099A - similarity: 0.38942\n",
      "Document Valerian - similarity: 0.38940\n",
      "Document Clorazepic acid - similarity: 0.38940\n",
      "Document Evodenoson - similarity: 0.38936\n",
      "Document Equol - similarity: 0.38934\n",
      "Document BMS-986158 - similarity: 0.38933\n",
      "Document Emixustat - similarity: 0.38928\n",
      "Document Trestolone - similarity: 0.38908\n",
      "Document Oxiranpseudoglucose - similarity: 0.38899\n",
      "Document Poa pratensis pollen - similarity: 0.38893\n",
      "Document Rifamycin - similarity: 0.38891\n",
      "Document Butylamine - similarity: 0.38887\n",
      "Document Potassium lactate - similarity: 0.38880\n",
      "Document SOBI-003 - similarity: 0.38878\n",
      "Document Fusicoccin - similarity: 0.38876\n",
      "Document Gastric inhibitory polypeptide - similarity: 0.38865\n",
      "Document Cobiprostone - similarity: 0.38859\n",
      "Document AR-67 - similarity: 0.38855\n",
      "Document (2S)-2-methyl-2,3-dihydrothieno[2,3-f][1,4]oxazepin-5-amine - similarity: 0.38850\n",
      "Document Alnus rhombifolia pollen - similarity: 0.38846\n",
      "Document Distigmine - similarity: 0.38838\n",
      "Document Gadolinium - similarity: 0.38827\n",
      "Document R1626 - similarity: 0.38824\n",
      "Document Heparin, bovine - similarity: 0.38810\n",
      "Document Vericiguat - similarity: 0.38806\n",
      "Document E5531 - similarity: 0.38805\n",
      "Document Quercus garryana pollen - similarity: 0.38805\n",
      "Document Roniciclib - similarity: 0.38800\n",
      "Document Thionicotinamide-Adenine-Dinucleotide - similarity: 0.38792\n",
      "Document Gosuranemab - similarity: 0.38791\n",
      "Document Hydrocortisone - similarity: 0.38788\n",
      "Document Acrivastine - similarity: 0.38780\n",
      "Document 3'-1-carboxy-1-phosphonooxy-ethoxy-uridine-diphosphate-N-acetylglucosamine - similarity: 0.38776\n",
      "Document 1,2,4-Triazole-Carboxamidine - similarity: 0.38776\n",
      "Document Malic acid - similarity: 0.38775\n",
      "Document Nepinalone - similarity: 0.38773\n",
      "Document Clorobiocin - similarity: 0.38771\n",
      "Document 5,5-dimethyl-2-morpholin-4-yl-5,6-dihydro-1,3-benzothiazol-7(4H)-one - similarity: 0.38770\n",
      "Document 10-Propargyl-5,8-Dideazafolic Acid - similarity: 0.38769\n",
      "Document Hyaluronidase - similarity: 0.38761\n",
      "Document Monopotassium phosphate - similarity: 0.38752\n",
      "Document Aminocaproic acid - similarity: 0.38742\n",
      "Document GMX1777 - similarity: 0.38741\n",
      "Document Pentanal - similarity: 0.38740\n",
      "Document Etelcalcetide - similarity: 0.38736\n",
      "Document 3-phenylpropionic acid - similarity: 0.38712\n",
      "Document 4-(4-FLUOROBENZYL)PIPERIDINE - similarity: 0.38711\n",
      "Document Aviscumine - similarity: 0.38706\n",
      "Document Cadazolid - similarity: 0.38704\n",
      "Document 5-fluorouridine - similarity: 0.38701\n",
      "Document Netivudine - similarity: 0.38696\n",
      "Document Calyculin A - similarity: 0.38689\n",
      "Document 5-Hydroxyamino-3-Methyl-Pyrrolidine-2-Carboxylic Acid - similarity: 0.38680\n",
      "Document Flecainide - similarity: 0.38667\n",
      "Document Nebularine - similarity: 0.38646\n",
      "Document GSK-2018682 - similarity: 0.38634\n",
      "Document Zofenoprilat - similarity: 0.38622\n",
      "Document Isochorismic Acid - similarity: 0.38620\n",
      "Document CMX-2043 - similarity: 0.38617\n",
      "Document Cantharidin - similarity: 0.38606\n",
      "Document Serelaxin - similarity: 0.38598\n",
      "Document 6-phospho-D-gluconic acid - similarity: 0.38591\n",
      "Document Nicergoline - similarity: 0.38591\n",
      "Document Fleroxacin - similarity: 0.38579\n",
      "Document 2-phenyl-1H-imidazole-4-carboxylic acid - similarity: 0.38579\n",
      "Document Agomelatine - similarity: 0.38570\n",
      "Document Gabapentin enacarbil - similarity: 0.38567\n",
      "Document DP-VPA - similarity: 0.38562\n",
      "Document 1-Aminocyclopropanecarboxylic Acid - similarity: 0.38548\n",
      "Document Streptococcus pneumoniae type 17f capsular polysaccharide antigen - similarity: 0.38532\n",
      "Document Phosphonoacetic Acid - similarity: 0.38530\n",
      "Document Iopentol - similarity: 0.38526\n",
      "Document 5-(2-NITROPHENYL)-2-FUROIC ACID - similarity: 0.38520\n",
      "Document Ribostamycin - similarity: 0.38517\n",
      "Document Japanese encephalitis virus strain sa 14-14-2 antigen (formaldehyde inactivated) - similarity: 0.38513\n",
      "Document Ethanolamine - similarity: 0.38502\n",
      "Document Diosmetin - similarity: 0.38499\n",
      "Document Biapenem - similarity: 0.38491\n",
      "Document Selenic acid - similarity: 0.38491\n",
      "Document Dotatate gallium Ga-68 - similarity: 0.38480\n",
      "Document Bepotastine - similarity: 0.38473\n",
      "Document Daledalin - similarity: 0.38467\n",
      "Document INCB-9471 - similarity: 0.38463\n",
      "Document Apalutamide - similarity: 0.38452\n",
      "Document 4-[1-(2,6-dichlorobenzyl)-2-methyl-1H-imidazol-4-yl]pyrimidin-2-amine - similarity: 0.38448\n",
      "Document Semustine - similarity: 0.38443\n",
      "Document Leptazoline A - similarity: 0.38441\n",
      "Document Verinurad - similarity: 0.38417\n",
      "Document HZT-501 - similarity: 0.38413\n",
      "Document Brodimoprim-4,6-Dicarboxylate - similarity: 0.38409\n",
      "Document 4-[3-(3-NITROPHENYL)-1,2,4-OXADIAZOL-5-YL]BUTANOIC ACID - similarity: 0.38409\n",
      "Document 3-{2,6,8-trioxo-9-[(2S,3S,4R)-2,3,4,5-tetrahydroxypentyl]-1,2,3,6,8,9-hexahydro-7H-purin-7-Yl}propyl dihydrogen phosphate - similarity: 0.38398\n",
      "Document Cupric glycinate - similarity: 0.38397\n",
      "Document Hemoglobin crosfumaril - similarity: 0.38396\n",
      "Document Radafaxine - similarity: 0.38387\n",
      "Document Arachidonic Acid - similarity: 0.38382\n",
      "Document Yohimbine - similarity: 0.38366\n",
      "Document CB-1158 - similarity: 0.38357\n",
      "Document Putrescine - similarity: 0.38348\n",
      "Document 4-(2-(1H-IMIDAZOL-4-YL)ETHYLAMINO)-2-(PHENYLAMINO)PYRAZOLO[1,5-A][1,3,5]TRIAZINE-8-CARBONITRILE - similarity: 0.38347\n",
      "Document Loperamide oxide - similarity: 0.38341\n",
      "Document GLPG-1205 - similarity: 0.38341\n",
      "Document Tafenoquine - similarity: 0.38335\n",
      "Document Epalrestat - similarity: 0.38322\n",
      "Document Cryptenamine - similarity: 0.38315\n",
      "Document Bevifimod - similarity: 0.38313\n",
      "Document Milataxel - similarity: 0.38312\n",
      "Document Interferon beta-1a - similarity: 0.38306\n",
      "Document Influenza A virus A/California/7/2009 X-181 (H1N1) antigen (propiolactone inactivated) - similarity: 0.38300\n",
      "Document Transfluthrin - similarity: 0.38292\n",
      "Document (R)-N-(3-Indol-1-Yl-2-Methyl-Propyl)-4-Sulfamoyl-Benzamide - similarity: 0.38291\n",
      "Document Endostatin - similarity: 0.38290\n",
      "Document Protein C - similarity: 0.38289\n",
      "Document Velmanase alfa - similarity: 0.38278\n",
      "Document 4,5-Dimethyl-1,2-phenylenediamine - similarity: 0.38275\n",
      "Document Neisseria meningitidis group C capsular polysaccharide tetanus toxoid conjugate antigen - similarity: 0.38269\n",
      "Document Egaptivon pegol - similarity: 0.38261\n",
      "Document NV1020 - similarity: 0.38246\n",
      "Document ST-101 - similarity: 0.38232\n",
      "Document Cacodylic acid - similarity: 0.38224\n",
      "Document Morus alba pollen - similarity: 0.38221\n",
      "Document Succinamide-CoA - similarity: 0.38217\n",
      "Document Ciclesonide - similarity: 0.38217\n",
      "Document Benfotiamine - similarity: 0.38215\n",
      "Document Loprazolam - similarity: 0.38212\n",
      "Document Rufinamide - similarity: 0.38212\n",
      "Document Corynebacterium diphtheriae - similarity: 0.38208\n",
      "Document Tolfenamic acid - similarity: 0.38197\n",
      "Document Ocinaplon - similarity: 0.38191\n",
      "Document Hydrogenobyrinic acid - similarity: 0.38183\n",
      "Document Carbendazim - similarity: 0.38181\n",
      "Document Rosemary oil - similarity: 0.38177\n",
      "Document N4-Hydroxycytidine - similarity: 0.38164\n",
      "Document 2-Bromohypoxanthine - similarity: 0.38162\n",
      "Document 1-(c14-c18 esteroyl)-2-docosahexanoyl-sn-glycero-3-phosphocholine - similarity: 0.38162\n",
      "Document Beta-Hydroxytryptophane - similarity: 0.38162\n",
      "Document Pholcodine - similarity: 0.38161\n",
      "Document Cariporide - similarity: 0.38152\n",
      "Document Podofilox - similarity: 0.38143\n",
      "Document 2-(2-Hydroxy-Phenyl)-1h-Indole-5-Carboxamidine - similarity: 0.38131\n",
      "Document Valtorcitabine - similarity: 0.38119\n",
      "Document Nicaraven - similarity: 0.38115\n",
      "Document Pyrazoloacridine - similarity: 0.38088\n",
      "Document Momordica charantia fruit - similarity: 0.38076\n",
      "Document 2-(methylamino)-N-(4-methyl-1,3-thiazol-2-yl)-5-[(4-methyl-4H-1,2,4-triazol-3-yl)sulfanyl]benzamide - similarity: 0.38075\n",
      "Document AMY-101 - similarity: 0.38073\n",
      "Document Gusperimus - similarity: 0.38070\n",
      "Document Licorice - similarity: 0.38066\n",
      "Document Piperidione - similarity: 0.38063\n",
      "Document N-[2-(5-methyl-4H-1,2,4-triazol-3-yl)phenyl]-7H-pyrrolo[2,3-d]pyrimidin-4-amine - similarity: 0.38047\n",
      "Document Undecylenic acid - similarity: 0.38045\n",
      "Document PHENYLALANINE BORONIC ACID - similarity: 0.38043\n",
      "Document N,N-Dimethyl-L-Alanine - similarity: 0.38039\n",
      "Document 4-TERT-BUTYLBENZENESULFONIC ACID - similarity: 0.38037\n",
      "Document Aurodox - similarity: 0.38026\n",
      "Document Coumarin - similarity: 0.38024\n",
      "Document (2S,3S)-3-FORMYL-2-({[(4-NITROPHENYL)SULFONYL]AMINO}METHYL)PENTANOIC ACID - similarity: 0.38020\n",
      "Document S-Methyl Thiocysteine Group - similarity: 0.38019\n",
      "Document (South)-Methanocarba-Thymidine - similarity: 0.38018\n",
      "Document 3-pyridin-4-yl-1H-indazole - similarity: 0.38016\n",
      "Document Guadecitabine - similarity: 0.38001\n",
      "Document Juglans californica pollen - similarity: 0.37999\n",
      "Document ALK-4290 - similarity: 0.37998\n",
      "Document Ethylhexylglycerin - similarity: 0.37988\n",
      "Document (1-HYDROXYNONANE-1,1-DIYL)BIS(PHOSPHONIC ACID) - similarity: 0.37969\n",
      "Document 3-CHLORO-2-(4-HYDROXYPHENYL)-2H-INDAZOL-5-OL - similarity: 0.37957\n",
      "Document Ceftolozane - similarity: 0.37949\n",
      "Document Phenyl trimethicone - similarity: 0.37948\n",
      "Document 5-(4-Morpholin-4-Yl-Phenylsulfanyl)-2,4-Quinazolinediamine - similarity: 0.37935\n",
      "Document Rivanicline - similarity: 0.37930\n",
      "Document (2S)-N-[(3E)-5-Cyclopropyl-3H-pyrazol-3-ylidene]-2-[4-(2-oxo-1-imidazolidinyl)phenyl]propanamide - similarity: 0.37925\n",
      "Document (1s,3r,4r,6s)-1,3,4,6-Tetrapkisphosphate - similarity: 0.37924\n",
      "Document Althea root - similarity: 0.37923\n",
      "Document Ligandrol - similarity: 0.37920\n",
      "Document 2-Bromo-2-Propene-1-Ol - similarity: 0.37918\n",
      "Document N-(1-CYANOCYCLOPROPYL)-3-({[(2S)-5-OXOPYRROLIDIN-2-YL]METHYL}SULFONYL)-N~2~-[(1S)-2,2,2-TRIFLUORO-1-(4-FLUOROPHENYL)ETHYL]-L-ALANINAMIDE - similarity: 0.37917\n",
      "Document Pegapamodutide - similarity: 0.37914\n",
      "Document Clove oil - similarity: 0.37914\n",
      "Document Sulfotep - similarity: 0.37904\n",
      "Document Ocimum tenuiflorum top - similarity: 0.37900\n",
      "Document N-(3-cyano-4,5,6,7-tetrahydro-1-benzothien-2-yl)-2-fluorobenzamide - similarity: 0.37897\n",
      "Document Rifalazil - similarity: 0.37896\n",
      "Document Ribavirin monophosphate - similarity: 0.37891\n",
      "Document Ferrous sulfate anhydrous - similarity: 0.37876\n",
      "Document PH94B - similarity: 0.37860\n",
      "Document Phloroglucinol - similarity: 0.37843\n",
      "Document Sevelamer - similarity: 0.37839\n",
      "Document N-1H-indazol-5-yl-2-(6-methylpyridin-2-yl)quinazolin-4-amine - similarity: 0.37833\n",
      "Document DIETHYL PROPANE-1,3-DIYLBISCARBAMATE - similarity: 0.37829\n",
      "Document Selenocysteine - similarity: 0.37820\n",
      "Document N-(3,5-dibromo-4-hydroxyphenyl)benzamide - similarity: 0.37819\n",
      "Document Facinicline - similarity: 0.37804\n",
      "Document Cashew fruit - similarity: 0.37796\n",
      "Document 4-{[5-(CYCLOHEXYLOXY)[1,2,4]TRIAZOLO[1,5-A]PYRIMIDIN-7-YL]AMINO}BENZENESULFONAMIDE - similarity: 0.37790\n",
      "Document Cytidine - similarity: 0.37770\n",
      "Document Digitoxigenin - similarity: 0.37764\n",
      "Document Edotecarin - similarity: 0.37753\n",
      "Document 3-({4-[(6-CHLORO-1-BENZOTHIEN-2-YL)SULFONYL]-2-OXOPIPERAZIN-1-YL}METHYL)BENZENECARBOXIMIDAMIDE - similarity: 0.37739\n",
      "Document Nimesulide - similarity: 0.37725\n",
      "Document Oportuzumab monatox - similarity: 0.37717\n",
      "Document Amuvatinib - similarity: 0.37712\n",
      "Document CRA_17693 - similarity: 0.37711\n",
      "Document [(1S)-4-(1-Aminobutylideneamino)-1-carboxybutyl]azanium - similarity: 0.37707\n",
      "Document Konjac mannan - similarity: 0.37705\n",
      "Document Cyclizine - similarity: 0.37705\n",
      "Document Ferumoxsil - similarity: 0.37703\n",
      "Document Sodium acetate - similarity: 0.37702\n",
      "Document 3-(5-Amino-7-oxo-3,7-dihydro-2H-[1,2,3]triazolo[4,5-d]pyrimidin-2-yl)-N-(2-{[2-(hydroxymethyl)phenyl]sulfanyl}benzyl)benzamide - similarity: 0.37696\n",
      "Document Aeruginosin 98-B - similarity: 0.37694\n",
      "Document Perindoprilat - similarity: 0.37694\n",
      "Document Reparixin - similarity: 0.37692\n",
      "Document Feprazone - similarity: 0.37684\n",
      "Document Gelatin - similarity: 0.37680\n",
      "Document Phenolsulfonic acid - similarity: 0.37675\n",
      "Document Decylsulfonic acid - similarity: 0.37671\n",
      "Document Sebacic acid - similarity: 0.37670\n",
      "Document Epothilone D - similarity: 0.37665\n",
      "Document RPR131247 - similarity: 0.37662\n",
      "Document N-alkyl ethylbenzyl dimethyl ammonium (C12-C14) - similarity: 0.37660\n",
      "Document Bifeprunox - similarity: 0.37650\n",
      "Document Opicapone - similarity: 0.37650\n",
      "Document Lamb - similarity: 0.37631\n",
      "Document RO-6870810 - similarity: 0.37620\n",
      "Document Soybean oil - similarity: 0.37604\n",
      "Document Bicisate - similarity: 0.37599\n",
      "Document Acetyl Dithranol - similarity: 0.37599\n",
      "Document Filibuvir - similarity: 0.37594\n",
      "Document Iodoform - similarity: 0.37590\n",
      "Document MK-3577 - similarity: 0.37590\n",
      "Document Ramipril - similarity: 0.37578\n",
      "Document Atractylodes japonica root - similarity: 0.37573\n",
      "Document BLXA4 - similarity: 0.37573\n",
      "Document Atiprimod - similarity: 0.37568\n",
      "Document Ponesimod - similarity: 0.37560\n",
      "Document Dichlorophen - similarity: 0.37535\n",
      "Document Cefozopran - similarity: 0.37531\n",
      "Document Edasalonexent - similarity: 0.37519\n",
      "Document Squaric acid dibutyl ester - similarity: 0.37518\n",
      "Document Vocimagene amiretrorepvec - similarity: 0.37507\n",
      "Document Oxypertine - similarity: 0.37502\n",
      "Document Obinutuzumab - similarity: 0.37498\n",
      "Document Carbomycin - similarity: 0.37498\n",
      "Document Viper antivenom - similarity: 0.37494\n",
      "Document Dithiazanine - similarity: 0.37492\n",
      "Document Anisomycin - similarity: 0.37486\n",
      "Document Artemotil - similarity: 0.37482\n",
      "Document 4'-FLUORO-1,1'-BIPHENYL-4-CARBOXYLIC ACID - similarity: 0.37482\n",
      "Document (3,4-DIHYDROXY-2-NITROPHENYL)(PHENYL)METHANONE - similarity: 0.37458\n",
      "Document Sulfamoxole - similarity: 0.37455\n",
      "Document Gentamicin - similarity: 0.37449\n",
      "Document Human botulinum neurotoxin A/B immune globulin - similarity: 0.37447\n",
      "Document Clotiazepam - similarity: 0.37447\n",
      "Document Capsaicin - similarity: 0.37435\n",
      "Document Gallium nitrate - similarity: 0.37420\n",
      "Document Guanine - similarity: 0.37415\n",
      "Document Phellodendron amurense bark - similarity: 0.37412\n",
      "Document Taurine - similarity: 0.37411\n",
      "Document N-[2-(2-iodo-5-methoxy-1H-indol-3-yl)ethyl]acetamide - similarity: 0.37403\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.37402\n",
      "Document Polyaminopropyl biguanide - similarity: 0.37402\n",
      "Document N-ACETYL-L-PHENYLALANYL-4-[DIFLUORO(PHOSPHONO)METHYL]-L-PHENYLALANINAMIDE - similarity: 0.37388\n",
      "Document Gacyclidine - similarity: 0.37376\n",
      "Document Formycin-5'-Monophosphate - similarity: 0.37373\n",
      "Document Ditiocarb zinc - similarity: 0.37360\n",
      "Document 7-carboxy-5-hydroxy-12,13-dihydro-6H-indolo[2,3-a]pyrrolo[3,4-c]carbazole - similarity: 0.37358\n",
      "Document Culex pipiens - similarity: 0.37348\n",
      "Document D-Aspartic Acid - similarity: 0.37337\n",
      "Document Proxibarbal - similarity: 0.37333\n",
      "Document Lysine Nz-Carboxylic Acid - similarity: 0.37327\n",
      "Document Diminazene - similarity: 0.37323\n",
      "Document Cerlapirdine - similarity: 0.37313\n",
      "Document (2S)-2-(biphenyl-4-yloxy)-3-phenylpropanoic acid - similarity: 0.37305\n",
      "Document Isoflavone - similarity: 0.37298\n",
      "Document 2'-Deoxyguanosine-5'-Monophosphate - similarity: 0.37286\n",
      "Document DADLE - similarity: 0.37270\n",
      "Document Pefcalcitol - similarity: 0.37261\n",
      "Document Carya tomentosa pollen - similarity: 0.37253\n",
      "Document beta-Apopicropodophyllin - similarity: 0.37250\n",
      "Document Prednylidene - similarity: 0.37247\n",
      "Document Exisulind - similarity: 0.37245\n",
      "Document NALPHA-[(BENZYLOXY)CARBONYL]-N-[(1R)-4-HYDROXY-1-METHYL-2-OXOBUTYL]-L-PHENYLALANINAMIDE - similarity: 0.37241\n",
      "Document Indisulam - similarity: 0.37232\n",
      "Document Pavinetant - similarity: 0.37222\n",
      "Document Formocortal - similarity: 0.37211\n",
      "Document Boldenone undecylenate - similarity: 0.37196\n",
      "Document Thenoyltrifluoroacetone - similarity: 0.37194\n",
      "Document [(1R)-1-acetamido-2-(4-chlorophenyl)ethyl]-[(2S)-2-amino-3-hydroxy-3-oxo-propoxy]-dihydroxy-boron - similarity: 0.37178\n",
      "Document Ropinirole - similarity: 0.37177\n",
      "Document Influenza A virus A/Brisbane/59/2007(H1N1) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.37175\n",
      "Document 3-(N-morpholino)propanesulfonic acid - similarity: 0.37173\n",
      "Document Ranirestat - similarity: 0.37172\n",
      "Document Voclosporin - similarity: 0.37159\n",
      "Document Dextrose, unspecified form - similarity: 0.37159\n",
      "Document N-Methylleucine - similarity: 0.37159\n",
      "Document SP-876 - similarity: 0.37156\n",
      "Document Erenumab - similarity: 0.37149\n",
      "Document Dupilumab - similarity: 0.37148\n",
      "Document Ciclobendazole - similarity: 0.37147\n",
      "Document Red pepper - similarity: 0.37142\n",
      "Document Cyclic adenosine monophosphate - similarity: 0.37133\n",
      "Document Amber - similarity: 0.37131\n",
      "Document (2S)-6-(2,4-DIAMINO-6-ETHYLPYRIMIDIN-5-YL)-2-(3,5-DIFLUOROPHENYL)-4-(3-METHOXYPROPYL)-2H-1,4-BENZOXAZIN-3(4H)-ONE - similarity: 0.37122\n",
      "Document Desipramine - similarity: 0.37113\n",
      "Document Cetiedil - similarity: 0.37111\n",
      "Document Vinylglycine - similarity: 0.37107\n",
      "Document Thimerosal - similarity: 0.37102\n",
      "Document Relcovaptan - similarity: 0.37089\n",
      "Document Hymenialdisine - similarity: 0.37088\n",
      "Document 3-Dehydroshikimate - similarity: 0.37088\n",
      "Document Macimorelin - similarity: 0.37084\n",
      "Document (3aS)-3a-hydroxy-5-methyl-1-phenyl-1,2,3,3a-tetrahydro-4H-pyrrolo[2,3-b]quinolin-4-one - similarity: 0.37075\n",
      "Document Oxybutynin - similarity: 0.37062\n",
      "Document Isoprenaline - similarity: 0.37051\n",
      "Document Isobutyric acid - similarity: 0.37040\n",
      "Document Betulinic Acid - similarity: 0.37027\n",
      "Document Polyacrylamide (crosslinked; 2 mole percent bisacrylamide) - similarity: 0.37026\n",
      "Document Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (UV, formaldehyde inactivated) - similarity: 0.37024\n",
      "Document Citraconic acid - similarity: 0.37022\n",
      "Document 2-Tridecanoyloxy-Pentadecanoic Acid - similarity: 0.37021\n",
      "Document 3-(1H-tetrazol-5-ylmethyl)-5,6,7,8-tetrahydro[1]benzothieno[2,3-d]pyrimidin-4(3H)-one - similarity: 0.37010\n",
      "Document Cytisine - similarity: 0.36989\n",
      "Document Methylparaben - similarity: 0.36983\n",
      "Document 5-Fluorolevulinic Acid - similarity: 0.36976\n",
      "Document Racecadotril - similarity: 0.36974\n",
      "Document Phenylephrine - similarity: 0.36974\n",
      "Document N-{[4-(but-2-yn-1-yloxy)phenyl]sulfonyl}-5-methyl-D-tryptophan - similarity: 0.36972\n",
      "Document Phenserine - similarity: 0.36960\n",
      "Document Neisseria meningitidis group a capsular oligosaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.36960\n",
      "Document Azoxystrobin - similarity: 0.36951\n",
      "Document Andexanet alfa - similarity: 0.36948\n",
      "Document 3-(INDOL-3-YL) LACTATE - similarity: 0.36943\n",
      "Document 4-[3-carboxymethyl-3-(4-phosphonooxy-benzyl)-ureido]-4-[(3-cyclohexyl-propyl)-methyl-carbamoyl]butyric acid - similarity: 0.36941\n",
      "Document Imipramine - similarity: 0.36938\n",
      "Document Cocoa butter - similarity: 0.36934\n",
      "Document trans NV-04 - similarity: 0.36924\n",
      "Document TRIAZOLOPYRIMIDINE - similarity: 0.36891\n",
      "Document Acetylglycinamide chloral hydrate - similarity: 0.36889\n",
      "Document Inosine - similarity: 0.36881\n",
      "Document Dexpanthenol - similarity: 0.36876\n",
      "Document Taraxacum officinale pollen - similarity: 0.36872\n",
      "Document Propionamide - similarity: 0.36869\n",
      "Document (3s)-3-Amino-1-(Cyclopropylamino)Heptane-2,2-Diol - similarity: 0.36860\n",
      "Document Eucalyptol - similarity: 0.36848\n",
      "Document Ceritinib - similarity: 0.36830\n",
      "Document Edoxudine - similarity: 0.36824\n",
      "Document Ragaglitazar - similarity: 0.36810\n",
      "Document BMS-599626 - similarity: 0.36805\n",
      "Document Atpenin A5 - similarity: 0.36804\n",
      "Document Omipalisib - similarity: 0.36800\n",
      "Document 4-{[(2R)-2-(2-methylphenyl)pyrrolidin-1-yl]carbonyl}benzene-1,3-diol - similarity: 0.36798\n",
      "Document Octopamine - similarity: 0.36791\n",
      "Document Diethylpropion - similarity: 0.36789\n",
      "Document CHROMOPHORE (HIS-TYR-GLY) - similarity: 0.36779\n",
      "Document GSK-923295 - similarity: 0.36772\n",
      "Document Para-Iodo-D-Phenylalanine Hydroxamic Acid - similarity: 0.36772\n",
      "Document ANZ-100 - similarity: 0.36769\n",
      "Document Somapacitan - similarity: 0.36757\n",
      "Document Vibegron - similarity: 0.36755\n",
      "Document R-851 - similarity: 0.36754\n",
      "Document PF-03187207 - similarity: 0.36745\n",
      "Document (2S)-2-(4-chlorophenoxy)-3-phenylpropanoic acid - similarity: 0.36738\n",
      "Document Ethyl (4R)-4-{[(2R,5S)-2-(4-fluorobenzyl)-6-methyl-5-{[(5-methyl-1,2-oxazol-3-yl)carbonyl]amino}-4-oxoheptanoyl]amino}-5-[(3S)-2-oxo-3-pyrrolidinyl]pentanoate - similarity: 0.36724\n",
      "Document Monalizumab - similarity: 0.36723\n",
      "Document 2-(5-CHLORO-2-THIENYL)-N-{(3S)-1-[(1S)-1-METHYL-2-MORPHOLIN-4-YL-2-OXOETHYL]-2-OXOPYRROLIDIN-3-YL}ETHANESULFONAMIDE - similarity: 0.36720\n",
      "Document Dactinomycin - similarity: 0.36718\n",
      "Document N-(2-(((5-CHLORO-2-PYRIDINYL)AMINO)SULFONYL)PHENYL)-4-(2-OXO-1(2H)-PYRIDINYL)BENZAMIDE - similarity: 0.36716\n",
      "Document Evocalcet - similarity: 0.36713\n",
      "Document Acetohydroxamic acid - similarity: 0.36705\n",
      "Document Imipenem - similarity: 0.36696\n",
      "Document 2-(Oxalyl-Amino)-4,5,6,7-Tetrahydro-Thieno[2,3-C]Pyridine-3-Carboxylic Acid - similarity: 0.36694\n",
      "Document 10,11-dimethoxy-4-methyldibenzo[c,f]-2,7-naphthyridine-3,6-diamine - similarity: 0.36689\n",
      "Document 4-(4-Hydroxy-3-Isopropylphenylthio)-2-Isopropylphenol - similarity: 0.36688\n",
      "Document OSI-027 - similarity: 0.36680\n",
      "Document Vitex agnus-castus fruit extract - similarity: 0.36678\n",
      "Document Benzylmorphine - similarity: 0.36678\n",
      "Document METHOXYUNDECYLPHOSPHINIC ACID - similarity: 0.36667\n",
      "Document Abemaciclib - similarity: 0.36661\n",
      "Document N-Coeleneterazine - similarity: 0.36650\n",
      "Document Gluconolactone - similarity: 0.36645\n",
      "Document RU82209 - similarity: 0.36641\n",
      "Document N-omega-propyl-L-arginine - similarity: 0.36639\n",
      "Document 4-BROMO-3-(CARBOXYMETHOXY)-5-(4-HYDROXYPHENYL)THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.36638\n",
      "Document Phenacetin - similarity: 0.36630\n",
      "Document Streptococcus pneumoniae type 6b capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.36606\n",
      "Document Desaspidin - similarity: 0.36605\n",
      "Document 1-deoxy-1-methoxycarbamido-beta-D-glucopyranose - similarity: 0.36602\n",
      "Document PX-102 - similarity: 0.36591\n",
      "Document 3-Methylfentanyl - similarity: 0.36588\n",
      "Document 1-(2-Chlorophenyl)-3,5-Dimethyl-1h-Pyrazole-4-Carboxylic Acid Ethyl Ester - similarity: 0.36577\n",
      "Document Streptococcus pneumoniae type 6a capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.36574\n",
      "Document Danaparoid - similarity: 0.36569\n",
      "Document O2-Sulfo-Glucuronic Acid - similarity: 0.36546\n",
      "Document (2R)-1-[4-({4-[(2,5-Dichlorophenyl)amino]-2-pyrimidinyl}amino)phenoxy]-3-(dimethylamino)-2-propanol - similarity: 0.36534\n",
      "Document N-{(4S)-4-Amino-5-[(2-aminoethyl)amino]pentyl}-N'-nitroguanidine - similarity: 0.36533\n",
      "Document Theanine - similarity: 0.36532\n",
      "Document 3-Carbamimidamido-1,1-diphenylurea - similarity: 0.36529\n",
      "Document Iodohippuric acid - similarity: 0.36528\n",
      "Document Enibomal - similarity: 0.36527\n",
      "Document METHYL N-({(2S,3S)-3-[(PROPYLAMINO)CARBONYL]OXIRAN-2-YL}CARBONYL)-L-ISOLEUCYL-L-PROLINATE - similarity: 0.36525\n",
      "Document Lisuride - similarity: 0.36522\n",
      "Document 4-isothioureidobutyronitrile - similarity: 0.36522\n",
      "Document Glasdegib - similarity: 0.36503\n",
      "Document Bismuth subcitrate potassium - similarity: 0.36483\n",
      "Document Vinpocetine - similarity: 0.36481\n",
      "Document (E)-3-(5((5-(4-CHLOROPHENYL)FURAN-2-YL)METHYLENE)-4-OXO-2-THIOXOTHIAZOLIDIN-3-YL)PROPANOIC ACID - similarity: 0.36476\n",
      "Document Octylphenoxy polyethoxyethanol - similarity: 0.36468\n",
      "Document (5S)-4,5-difluoro-6-[(2-fluoro-4-iodophenyl)imino]-N-(2-hydroxyethoxy)cyclohexa-1,3-diene-1-carboxamide - similarity: 0.36468\n",
      "Document Hydroflumethiazide - similarity: 0.36453\n",
      "Document Hepatitis A Vaccine - similarity: 0.36453\n",
      "Document 2,4-deoxy-4-guanidino-5-N-acetyl-neuraminic acid - similarity: 0.36444\n",
      "Document CHS-828 - similarity: 0.36443\n",
      "Document N(2)-succinyl-L-arginine - similarity: 0.36442\n",
      "Document 4-Hydroxyphenylglycine - similarity: 0.36411\n",
      "Document 1,2-diacyl-sn-glycero-3-phosphoinositol - similarity: 0.36404\n",
      "Document Bis(Adenosine)-5'-Triphosphate - similarity: 0.36394\n",
      "Document (2R)-2-PHENYL-N-PYRIDIN-4-YLBUTANAMIDE - similarity: 0.36391\n",
      "Document EC-0652 - similarity: 0.36381\n",
      "Document D-phenylalanyl-N-(3-methylbenzyl)-L-prolinamide - similarity: 0.36380\n",
      "Document Pleospora betae - similarity: 0.36374\n",
      "Document Acyline - similarity: 0.36364\n",
      "Document (1-Benzyl-5-methoxy-2-methyl-1H-indol-3-yl)acetic acid - similarity: 0.36360\n",
      "Document GALACTOSE GREASE - similarity: 0.36358\n",
      "Document Hydrogenated palm kernel oil - similarity: 0.36349\n",
      "Document Brivudine - similarity: 0.36340\n",
      "Document Timonacic - similarity: 0.36332\n",
      "Document Zoalene - similarity: 0.36322\n",
      "Document Ceftobiprole - similarity: 0.36315\n",
      "Document Pruvanserin - similarity: 0.36296\n",
      "Document Basmisanil - similarity: 0.36293\n",
      "Document 1-(CYCLOHEXYLAMINO)-3-(6-METHYL-3,4-DIHYDRO-1H-CARBAZOL-9(2H)-YL)PROPAN-2-OL - similarity: 0.36289\n",
      "Document Sofituzumab vedotin - similarity: 0.36285\n",
      "Document C-1027 Aromatized chromophore - similarity: 0.36284\n",
      "Document Minaprine - similarity: 0.36282\n",
      "Document Flortaucipir - similarity: 0.36276\n",
      "Document Efonidipine - similarity: 0.36273\n",
      "Document Citrus bioflavonoids - similarity: 0.36262\n",
      "Document Drostanolone propionate - similarity: 0.36261\n",
      "Document 2-Ammoniobut-3-Enoate, 2-Amino-3-Butenoate - similarity: 0.36257\n",
      "Document Palonosetron - similarity: 0.36252\n",
      "Document Crisaborole - similarity: 0.36241\n",
      "Document (2R)-2-({9-(1-methylethyl)-6-[(4-pyridin-2-ylbenzyl)amino]-9H-purin-2-yl}amino)butan-1-ol - similarity: 0.36238\n",
      "Document Sotrastaurin - similarity: 0.36238\n",
      "Document Biochanin A - similarity: 0.36237\n",
      "Document (3ASR,4RS,8ASR,8BRS)-4-(2-(4-FLUOROBENZYL)-1,3-DIOXODEACAHYDROPYRROLO[3,4-A] PYRROLIZIN-4-YL)BENZAMIDINE - similarity: 0.36235\n",
      "Document Enoxacin - similarity: 0.36232\n",
      "Document 3-BROMO-6-HYDROXY-2-(4-HYDROXYPHENYL)-1H-INDEN-1-ONE - similarity: 0.36231\n",
      "Document Fluazuron - similarity: 0.36230\n",
      "Document Rhubarb - similarity: 0.36229\n",
      "Document Uridine-5'-diphosphate-4-deoxy-4-fluoro-alpha-D-galactose - similarity: 0.36224\n",
      "Document alpha-Tocopherol acetate - similarity: 0.36212\n",
      "Document DIETHYL (1R,2S,3R,4S)-5,6-BIS(4-HYDROXYPHENYL)-7-OXABICYCLO[2.2.1]HEPT-5-ENE-2,3-DICARBOXYLATE - similarity: 0.36206\n",
      "Document 3-BENZYLOXYCARBONYLAMINO-2-HYDROXY-4-PHENYL-BUTYRIC ACID - similarity: 0.36200\n",
      "Document NADPH - similarity: 0.36195\n",
      "Document PF-06651600 - similarity: 0.36187\n",
      "Document Fluticasone propionate - similarity: 0.36181\n",
      "Document Anisotropine methylbromide - similarity: 0.36170\n",
      "Document Triazolopyridine - similarity: 0.36165\n",
      "Document NLX-P101 - similarity: 0.36145\n",
      "Document Oxamic Acid - similarity: 0.36135\n",
      "Document Aminosalicylic acid - similarity: 0.36130\n",
      "Document 2-Aminophenol - similarity: 0.36121\n",
      "Document Palovarotene - similarity: 0.36111\n",
      "Document Oat - similarity: 0.36110\n",
      "Document Turpentine - similarity: 0.36098\n",
      "Document N-Methylmesoporphyrin Containing Copper - similarity: 0.36090\n",
      "Document Influenza A virus A/Uruguay/716/2007(H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.36089\n",
      "Document 3-(cyclohexylamino)propanesulfonic acid - similarity: 0.36085\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document Bucillamine - similarity: 0.36084\n",
      "Document 3-Methyl-5-Sulfo-Pyrrolidine-2-Carboxylic Acid - similarity: 0.36079\n",
      "Document Matairesinol - similarity: 0.36077\n",
      "Document Rutin - similarity: 0.36062\n",
      "Document Abacavir hydroxyacetate - similarity: 0.36032\n",
      "Document Choline C 11 - similarity: 0.36019\n",
      "Document Fosmidomycin - similarity: 0.36018\n",
      "Document Brivanib alaninate - similarity: 0.36016\n",
      "Document Emricasan - similarity: 0.36014\n",
      "Document Bucladesine - similarity: 0.36012\n",
      "Document Proxyphylline - similarity: 0.36006\n",
      "Document Valomaciclovir - similarity: 0.35996\n",
      "Document ENMD-1198 - similarity: 0.35982\n",
      "Document Beta-hydroxyfentanyl - similarity: 0.35981\n",
      "Document N(G)-Iminoethylornithine - similarity: 0.35950\n",
      "Document Adafosbuvir - similarity: 0.35948\n",
      "Document Kinetin - similarity: 0.35938\n",
      "Document INGN 201 - similarity: 0.35936\n",
      "Document 5-monophosphate-9-beta-D-ribofuranosyl xanthine - similarity: 0.35924\n",
      "Document L-threonic acid - similarity: 0.35917\n",
      "Document Darinaparsin - similarity: 0.35910\n",
      "Document THIO-ATPA - similarity: 0.35908\n",
      "Document D-Methionine - similarity: 0.35905\n",
      "Document 5-AMINO-6-CYCLOHEXYL-4-HYDROXY-2-ISOPROPYL-HEXANOIC ACID - similarity: 0.35895\n",
      "Document Methylphosphinate - similarity: 0.35888\n",
      "Document Protocatechualdehyde - similarity: 0.35881\n",
      "Document 5-(5-(6-CHLORO-4-(4,5-DIHYDRO-2-OXAZOLYL)PHENOXY)PENTYL)-3-METHYL ISOXAZOLE - similarity: 0.35879\n",
      "Document Rucaparib - similarity: 0.35876\n",
      "Document GT 389-255 - similarity: 0.35865\n",
      "Document L-Glutamine - similarity: 0.35863\n",
      "Document Nicotinamide-adenine-dinucleotide-5-hydroxy-4-oxonorvaline - similarity: 0.35863\n",
      "Document Sparfloxacin - similarity: 0.35861\n",
      "Document 12-Bromododecanoic Acid - similarity: 0.35854\n",
      "Document 2,6-Anhydro-3-Deoxy-D-Erythro-Hex-2-Enonic Acid - similarity: 0.35843\n",
      "Document N-(TRANS-4-{(1S,2S)-2-AMINO-3-[(3S)-3-FLUOROPYRROLIDIN-1-YL]-1-METHYL-3-OXOPROPYL}CYCLOHEXYL)-N-METHYLACETAMIDE - similarity: 0.35836\n",
      "Document Vinflunine - similarity: 0.35820\n",
      "Document Diethylamino hydroxybenzoyl hexyl benzoate - similarity: 0.35819\n",
      "Document TAS-116 - similarity: 0.35818\n",
      "Document N-(3-TERT-BUTYL-1H-PYRAZOL-5-YL)-N'-{4-CHLORO-3-[(PYRIDIN-3-YLOXY)METHYL]PHENYL}UREA - similarity: 0.35809\n",
      "Document 5-[(3R)-3-(5-methoxy-3',5'-dimethylbiphenyl-3-yl)but-1-yn-1-yl]-6-methylpyrimidine-2,4-diamine - similarity: 0.35805\n",
      "Document Cochliobolus lunatus - similarity: 0.35802\n",
      "Document Glutathione disulfide - similarity: 0.35795\n",
      "Document Cephalexin - similarity: 0.35790\n",
      "Document Equine Botulinum Neurotoxin F Immune FAB2 - similarity: 0.35789\n",
      "Document Deamido-Nad+ - similarity: 0.35777\n",
      "Document Penicillium roqueforti - similarity: 0.35765\n",
      "Document 3,5-Diiodotyrosine - similarity: 0.35752\n",
      "Document 4-(Aminosulfonyl)-N-[(2,3,4-Trifluorophenyl)Methyl]-Benzamide - similarity: 0.35733\n",
      "Document PRI-724 - similarity: 0.35727\n",
      "Document 4-Fluorotryptophane - similarity: 0.35721\n",
      "Document Seviteronel - similarity: 0.35691\n",
      "Document N-{2-[4-(AMINOSULFONYL)PHENYL]ETHYL}ACETAMIDE - similarity: 0.35691\n",
      "Document (1s,2s)-1-Amino-1-(1,3-Thiazol-2-Yl)Propan-2-Ol - similarity: 0.35689\n",
      "Document Malonic acid - similarity: 0.35687\n",
      "Document Cinacalcet - similarity: 0.35683\n",
      "Document Distichlis spicata pollen - similarity: 0.35679\n",
      "Document 5-[Bis(2-bromoethyl)amino]-N-(2,3-dioxopropyl)-2,4-dinitrobenzamide - similarity: 0.35675\n",
      "Document [1-(1-Benzyl-3-Hydroxy-2-Oxo-Propylcarbamoyl)-2-Phenyl-Ethyl]-Carbamic Acid Benzyl Ester - similarity: 0.35664\n",
      "Document ARX201 - similarity: 0.35663\n",
      "Document Mepixanox - similarity: 0.35659\n",
      "Document 2-(hydrazinocarbonyl)-3-phenyl-1H-indole-5-sulfonamide - similarity: 0.35643\n",
      "Document Camphane - similarity: 0.35642\n",
      "Document Ursodeoxycholic acid - similarity: 0.35642\n",
      "Document Pinus contorta pollen - similarity: 0.35627\n",
      "Document Tridecactide - similarity: 0.35625\n",
      "Document 5-(2-CHLOROBENZYL)-2-FUROIC ACID - similarity: 0.35609\n",
      "Document Dihydro-2-thioxo-5-((5-(2-(trifluoromethyl)phenyl)-2-furanyl)methyl)-4,6(1H,5H)-pyrimidinedione - similarity: 0.35601\n",
      "Document 5-Nitroso-6-ribityl-amino-2,4(1H,3H)-pyrimidinedione - similarity: 0.35584\n",
      "Document SP-8203 - similarity: 0.35583\n",
      "Document 5-hydroxyvaleric acid - similarity: 0.35578\n",
      "Document Simethicone - similarity: 0.35578\n",
      "Document Recombinant Cholera Toxin B Subunit - similarity: 0.35577\n",
      "Document Thiosalicylic acid - similarity: 0.35575\n",
      "Document N-{2-[6-(2,4-DIAMINO-6-ETHYLPYRIMIDIN-5-YL)-2,2-DIMETHYL-3-OXO-2,3-DIHYDRO-4H-1,4-BENZOXAZIN-4-YL]ETHYL}ACETAMIDE - similarity: 0.35574\n",
      "Document Arundic acid - similarity: 0.35570\n",
      "Document Clobetasol - similarity: 0.35568\n",
      "Document Beta-Hydroxyleucine - similarity: 0.35566\n",
      "Document Adenosine - similarity: 0.35562\n",
      "Document MB-07803 - similarity: 0.35560\n",
      "Document scyllo-inositol - similarity: 0.35551\n",
      "Document p-Toluenesulfonic acid - similarity: 0.35536\n",
      "Document 1,3-bis-([[3-(4-{3-[3-nitro-5-(galactopyranosyloxy)-benzoylamino]-propyl}-piperazin-1-yl)-propylamino-3,4-dioxo-cyclobutenyl]-amino-ethyl]-amino-carbonyloxy)-2-amino-propane - similarity: 0.35531\n",
      "Document Didecyldimethylammonium - similarity: 0.35515\n",
      "Document Plitidepsin - similarity: 0.35510\n",
      "Document Reversine - similarity: 0.35486\n",
      "Document Pyrotinib - similarity: 0.35484\n",
      "Document Aspartate Semialdehyde - similarity: 0.35482\n",
      "Document Zingerone - similarity: 0.35476\n",
      "Document 2,6-DIMETHYL-1-(3-[3-METHYL-5-ISOXAZOLYL]-PROPANYL)-4-[2N-METHYL-2H-TETRAZOL-5-YL]-PHENOL - similarity: 0.35474\n",
      "Document Geraniol - similarity: 0.35473\n",
      "Document Alverine - similarity: 0.35471\n",
      "Document 3-methyl-benzene-1,2-diol - similarity: 0.35464\n",
      "Document 2-[1-(4-chlorobenzoyl)-5-methoxy-2-methyl-1H-indol-3-yl]-n-[(1S)-1-(hydroxymethyl)propyl]acetamide - similarity: 0.35454\n",
      "Document Deferasirox - similarity: 0.35438\n",
      "Document Pinacidil - similarity: 0.35435\n",
      "Document Factor IIIm - similarity: 0.35420\n",
      "Document Spesolimab - similarity: 0.35407\n",
      "Document Glycocholic acid - similarity: 0.35402\n",
      "Document Olaratumab - similarity: 0.35402\n",
      "Document Rilapladib - similarity: 0.35390\n",
      "Document Dimethylthiambutene - similarity: 0.35387\n",
      "Document NOX-100 - similarity: 0.35383\n",
      "Document Edrophonium - similarity: 0.35358\n",
      "Document Ioxitalamic acid - similarity: 0.35351\n",
      "Document Apabetalone - similarity: 0.35335\n",
      "Document Ataciguat - similarity: 0.35334\n",
      "Document Phosphoric acid - similarity: 0.35324\n",
      "Document Valinomycin - similarity: 0.35321\n",
      "Document Atl146e - similarity: 0.35319\n",
      "Document Danicopan - similarity: 0.35302\n",
      "Document Guanosine-5'-Triphosphate - similarity: 0.35295\n",
      "Document PF-04447943 - similarity: 0.35277\n",
      "Document PF-06751979 - similarity: 0.35276\n",
      "Document Pixantrone - similarity: 0.35263\n",
      "Document Neflamapimod - similarity: 0.35260\n",
      "Document Carglumic acid - similarity: 0.35253\n",
      "Document CT-011 - similarity: 0.35253\n",
      "Document Nitrendipine - similarity: 0.35244\n",
      "Document DL-alpha-Tocopherol - similarity: 0.35234\n",
      "Document Mupirocin - similarity: 0.35229\n",
      "Document Desomorphine - similarity: 0.35227\n",
      "Document PF-03654746 - similarity: 0.35226\n",
      "Document (4R)-N-[4-({[2-(DIMETHYLAMINO)ETHYL]AMINO}CARBONYL)-1,3-THIAZOL-2-YL]-4-METHYL-1-OXO-2,3,4,9-TETRAHYDRO-1H-BETA-CARBOLINE-6-CARBOXAMIDE - similarity: 0.35220\n",
      "Document Gadopentetic acid - similarity: 0.35203\n",
      "Document Branebrutinib - similarity: 0.35200\n",
      "Document Tesmilifene - similarity: 0.35198\n",
      "Document Cilastatin - similarity: 0.35192\n",
      "Document 5-phospho-D-arabinohydroxamic acid - similarity: 0.35173\n",
      "Document Fibroblast growth factor-1 - similarity: 0.35170\n",
      "Document RU79072 - similarity: 0.35163\n",
      "Document Corticotropin - similarity: 0.35158\n",
      "Document Methoxyamine - similarity: 0.35137\n",
      "Document Efgartigimod alfa - similarity: 0.35133\n",
      "Document Temafloxacin - similarity: 0.35126\n",
      "Document 2-propenyl-N-acetyl-neuramic acid - similarity: 0.35120\n",
      "Document Uracil mustard - similarity: 0.35114\n",
      "Document Mecasermin - similarity: 0.35110\n",
      "Document 2-(2-METHYLPHENYL)-1H-INDOLE-6-CARBOXIMIDAMIDE - similarity: 0.35102\n",
      "Document Carbuterol - similarity: 0.35096\n",
      "Document Norleucine - similarity: 0.35090\n",
      "Document 25-Hydroxycholesterol - similarity: 0.35057\n",
      "Document Murepavadin - similarity: 0.35052\n",
      "Document Fluoromisonidazole F-18 - similarity: 0.35048\n",
      "Document White pepper - similarity: 0.35048\n",
      "Document Dinalbuphine sebacate - similarity: 0.35040\n",
      "Document Ferric cation - similarity: 0.35031\n",
      "Document KRN-7000 - similarity: 0.35017\n",
      "Document Pemigatinib - similarity: 0.35008\n",
      "Document Sincalide - similarity: 0.35004\n",
      "Document Phosphatidylethanolamine - similarity: 0.34995\n",
      "Document 1-(5-CHLORO-2-METHOXYPHENYL)-3-{6-[2-(DIMETHYLAMINO)-1-METHYLETHOXY]PYRAZIN-2-YL}UREA - similarity: 0.34994\n",
      "Document Orlistat - similarity: 0.34982\n",
      "Document Stemphylium solani - similarity: 0.34959\n",
      "Document Geranyl Diphosphate - similarity: 0.34951\n",
      "Document Geranylgeranyl diphosphate - similarity: 0.34947\n",
      "Document CAN-508 - similarity: 0.34946\n",
      "Document Enoxolone - similarity: 0.34936\n",
      "Document 4-{3-CHLORO-4-[3-(2,4-DICHLORO-BENZOYL)-UREIDO]-PHENOXY}-BUTYRIC ACID - similarity: 0.34929\n",
      "Document Pleconaril - similarity: 0.34925\n",
      "Document Tertomotide - similarity: 0.34917\n",
      "Document 4-Imidazolmethylene-5-Imidazolone Chromophore - similarity: 0.34912\n",
      "Document Teicoplanin - similarity: 0.34911\n",
      "Document Lanatoside C - similarity: 0.34911\n",
      "Document Lauryldimethylbetaine - similarity: 0.34908\n",
      "Document (1R)-1,2,2-trimethylpropyl (R)-methylphosphinate - similarity: 0.34901\n",
      "Document N-(5-{4-Chloro-3-[(2-hydroxyethyl)sulfamoyl]phenyl}-4-methyl-1,3-thiazol-2-yl)acetamide - similarity: 0.34895\n",
      "Document Orteronel - similarity: 0.34894\n",
      "Document Oxymetholone - similarity: 0.34887\n",
      "Document Begacestat - similarity: 0.34861\n",
      "Document Ravoxertinib - similarity: 0.34860\n",
      "Document Brilliant green cation - similarity: 0.34856\n",
      "Document Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.34856\n",
      "Document Centanafadine - similarity: 0.34844\n",
      "Document (3S)-3-(dioxidosulfanyl)-N-[(1E)-3-oxoprop-1-en-1-yl]-4-(1H-1,2,3-triazol-1-yl)-D-valine - similarity: 0.34836\n",
      "Document Wormwood - similarity: 0.34821\n",
      "Document METHYL N-[(2S,3R)-3-AMINO-2-HYDROXY-3-(4-METHYLPHENYL)PROPANOYL]-D-ALANYL-D-LEUCINATE - similarity: 0.34814\n",
      "Document MMDA - similarity: 0.34807\n",
      "Document 2-[4-(4-Hydroxy-3-Isopropyl-Phenoxy)-3,5-Dimethyl-Phenyl]-2h-[1,2,4]Triazine-3,5-Dione - similarity: 0.34796\n",
      "Document Acetyl sulfisoxazole - similarity: 0.34794\n",
      "Document Clocortolone - similarity: 0.34791\n",
      "Document N-(5-chloro-1,3-benzodioxol-4-yl)-6-methoxy-7-(3-piperidin-1-ylpropoxy)quinazolin-4-amine - similarity: 0.34789\n",
      "Document Boldenone - similarity: 0.34785\n",
      "Document (5E)-14-CHLORO-15,17-DIHYDROXY-4,7,8,9,10,11-HEXAHYDRO-2-BENZOXACYCLOPENTADECINE-1,12(3H,13H)-DIONE - similarity: 0.34782\n",
      "Document Sar9, Met (O2)11-Substance P - similarity: 0.34779\n",
      "Document Atrial natriuretic peptide - similarity: 0.34773\n",
      "Document MLN2222 - similarity: 0.34771\n",
      "Document Polistes exclamans venom protein - similarity: 0.34770\n",
      "Document Phosphomethylphosphonic acid guanosyl ester - similarity: 0.34764\n",
      "Document Trifluoroethanol - similarity: 0.34763\n",
      "Document (1n)-4-N-Butoxyphenylsulfonyl-(2r)-N-Hydroxycarboxamido-(4s)-Methanesulfonylamino-Pyrrolidine - similarity: 0.34761\n",
      "Document Canavanine - similarity: 0.34761\n",
      "Document Pactimibe - similarity: 0.34759\n",
      "Document Influenza A virus A/South Australia/55/2014 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.34757\n",
      "Document Alethine - similarity: 0.34755\n",
      "Document Populus tremuloides pollen - similarity: 0.34751\n",
      "Document Ritanserin - similarity: 0.34751\n",
      "Document Voacamine - similarity: 0.34744\n",
      "Document Oleandomycin - similarity: 0.34742\n",
      "Document Maltosyl-Alpha (1,4)-D-Gluconhydroximo-1,5-Lactam - similarity: 0.34731\n",
      "Document Beclomethasone dipropionate - similarity: 0.34711\n",
      "Document Vobarilizumab - similarity: 0.34708\n",
      "Document Caraway seed - similarity: 0.34701\n",
      "Document Crocin - similarity: 0.34698\n",
      "Document 2-Dehydropantoate - similarity: 0.34697\n",
      "Document Tryptamine - similarity: 0.34695\n",
      "Document N-[amino(imino)methyl]-2-[2-(2-chlorophenyl)-4-(4-propoxyphenyl)-3-thienyl]acetamide - similarity: 0.34688\n",
      "Document (5E,7E,9)-decatrien-2-one - similarity: 0.34659\n",
      "Document DONU - similarity: 0.34657\n",
      "Document N,N-[2,5-O-Dibenzyl-glucaryl]-DI-[1-amino-indan-2-OL] - similarity: 0.34651\n",
      "Document L-Pentahomoserine - similarity: 0.34651\n",
      "Document 2-MERCAPTO-N-[1,2,3,10-TETRAMETHOXY-9-OXO-5,6,7,9-TETRAHYDRO-BENZO[A]HEPTALEN-7-YL]ACETAMIDE - similarity: 0.34633\n",
      "Document Carya glabra pollen - similarity: 0.34631\n",
      "Document 2-[(2e,6e,10e,14e,18e,22e,26e)-3,7,11,15,19,23,27,31-Octamethyldotriaconta-2,6,10,14,18,22,26,30-Octaenyl]Phenol - similarity: 0.34626\n",
      "Document Trinitrotoluene - similarity: 0.34624\n",
      "Document P-Nitrophenol - similarity: 0.34606\n",
      "Document Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (propiolactone inactivated) - similarity: 0.34603\n",
      "Document Cositecan - similarity: 0.34599\n",
      "Document BMS-955176 - similarity: 0.34597\n",
      "Document 2,5-Dideoxy-2,5-Imino-D-Glucitol - similarity: 0.34578\n",
      "Document Drostanolone - similarity: 0.34567\n",
      "Document Magnesium stearate - similarity: 0.34560\n",
      "Document Methylecgonine - similarity: 0.34559\n",
      "Document [(4R)-2,2-dimethyl-1,3-dioxolan-4-yl]methyl hydrogen hex-5-enylphosphonate - similarity: 0.34555\n",
      "Document ORM-13070 C-11 - similarity: 0.34546\n",
      "Document Guaiacol - similarity: 0.34541\n",
      "Document Acacia - similarity: 0.34538\n",
      "Document 1-(5-Carboxypentyl)-5-[(2,6-Dichlorobenzyl)Oxy]-1 H-Indole-2-Carboxylic Acid - similarity: 0.34530\n",
      "Document (2-AMINO-3-PHENYL-BICYCLO[2.2.1]HEPT-2-YL)-PHENYL-METHANONE - similarity: 0.34524\n",
      "Document Docosanol - similarity: 0.34506\n",
      "Document Estriol tripropionate - similarity: 0.34504\n",
      "Document Sulconazole - similarity: 0.34499\n",
      "Document Gemcabene - similarity: 0.34498\n",
      "Document Chenopodium album pollen - similarity: 0.34492\n",
      "Document Lubazodone - similarity: 0.34492\n",
      "Document Trastuzumab deruxtecan - similarity: 0.34484\n",
      "Document Dantron - similarity: 0.34460\n",
      "Document Osbond acid - similarity: 0.34458\n",
      "Document Ortataxel - similarity: 0.34451\n",
      "Document Xanthan gum - similarity: 0.34450\n",
      "Document Ethchlorvynol - similarity: 0.34448\n",
      "Document (R)-Rolipram - similarity: 0.34447\n",
      "Document Azintuxizumab vedotin - similarity: 0.34445\n",
      "Document 4-(2,4-Dimethyl-Thiazol-5-Yl)-Pyrimidin-2-Ylamine - similarity: 0.34434\n",
      "Document 4SC-202 - similarity: 0.34431\n",
      "Document Reidispongiolide A - similarity: 0.34420\n",
      "Document [[4-(Aminomethyl)Phenyl]Amino]Oxo-Acetic Acid, - similarity: 0.34417\n",
      "Document Chondroitin sulfate - similarity: 0.34403\n",
      "Document Phoma glomerata - similarity: 0.34401\n",
      "Document ethyl 3-[(E)-2-amino-1-cyanoethenyl]-6,7-dichloro-1-methyl-1H-indole-2-carboxylate - similarity: 0.34399\n",
      "Document 4-AMINO-N-[(2-SULFANYLETHYL)CARBAMOYL]BENZENESULFONAMIDE - similarity: 0.34398\n",
      "Document Polyvinyl alcohol - similarity: 0.34393\n",
      "Document Histidinol - similarity: 0.34392\n",
      "Document ETC-588 - similarity: 0.34392\n",
      "Document Dubermatinib - similarity: 0.34385\n",
      "Document Thioglycolic acid - similarity: 0.34374\n",
      "Document S-(N-hydroxy-N-bromophenylcarbamoyl)glutathione - similarity: 0.34358\n",
      "Document 7-Methyl-7,8-dihydroguanosine 5'-(tetrahydrogen triphosphate) - similarity: 0.34355\n",
      "Document N-{2'-[(4-FLUOROPHENYL)AMINO]-4,4'-BIPYRIDIN-2-YL}-4-METHOXYCYCLOHEXANECARBOXAMIDE - similarity: 0.34347\n",
      "Document methyl 4-bromo-N-[8-(hydroxyamino)-8-oxooctanoyl]-L-phenylalaninate - similarity: 0.34335\n",
      "Document Loxicodegol - similarity: 0.34333\n",
      "Document Aluminium - similarity: 0.34330\n",
      "Document Calcium polycarbophil - similarity: 0.34298\n",
      "Document Reactive Red 1 Dye - similarity: 0.34296\n",
      "Document Factor IX Complex (Human) - similarity: 0.34289\n",
      "Document (1S,5S,7R)-N~7~-(BIPHENYL-4-YLMETHYL)-N~3~-HYDROXY-6,8-DIOXA-3-AZABICYCLO[3.2.1]OCTANE-3,7-DICARBOXAMIDE - similarity: 0.34280\n",
      "Document N-(Phosphonoacetyl)-L-Ornithine - similarity: 0.34275\n",
      "Document Sorafenib - similarity: 0.34263\n",
      "Document Methyl salicylate - similarity: 0.34245\n",
      "Document DIMETHYL THIOPHOSPHATE - similarity: 0.34245\n",
      "Document Indium In-111 pentetate - similarity: 0.34232\n",
      "Document Human calcitonin - similarity: 0.34220\n",
      "Document Hidrosmin - similarity: 0.34217\n",
      "Document Obeticholic acid - similarity: 0.34214\n",
      "Document Pirprofen - similarity: 0.34204\n",
      "Document IDD-1 - similarity: 0.34194\n",
      "Document Malonaldehyde - similarity: 0.34192\n",
      "Document MK-0533 - similarity: 0.34187\n",
      "Document Clorexolone - similarity: 0.34187\n",
      "Document (C8-S)-Hydantocidin 5'-phosphate - similarity: 0.34185\n",
      "Document Artemisia douglasiana pollen - similarity: 0.34177\n",
      "Document Fluocinonide - similarity: 0.34172\n",
      "Document Colistimethate - similarity: 0.34163\n",
      "Document (4s)-5-Fluoro-L-Leucine - similarity: 0.34163\n",
      "Document 2-{HYDROXY[2-NITRO-4-(TRIFLUOROMETHYL)PHENYL]METHYLENE}CYCLOHEXANE-1,3-DIONE - similarity: 0.34157\n",
      "Document Isocaproic acid - similarity: 0.34156\n",
      "Document S-NONYL-CYSTEINE - similarity: 0.34139\n",
      "Document Encorafenib - similarity: 0.34135\n",
      "Document Human papillomavirus type 18 l1 capsid protein residues 2-472 antigen - similarity: 0.34123\n",
      "Document 4-{[(2,6-difluorophenyl)carbonyl]amino}-N-[(3S)-piperidin-3-yl]-1H-pyrazole-3-carboxamide - similarity: 0.34116\n",
      "Document Acifluorfen - similarity: 0.34113\n",
      "Document Calcipotriol - similarity: 0.34112\n",
      "Document beta-D-glucose 6-phosphate - similarity: 0.34108\n",
      "Document Albutrepenonacog alfa - similarity: 0.34102\n",
      "Document Netarsudil - similarity: 0.34102\n",
      "Document (5r)-5-Amino-6-Hydroxyhexylcarbamic Acid - similarity: 0.34096\n",
      "Document Vecuronium - similarity: 0.34093\n",
      "Document Fenamole - similarity: 0.34092\n",
      "Document Tetraferric tricitrate decahydrate - similarity: 0.34088\n",
      "Document 4-Methylcoumarin - similarity: 0.34086\n",
      "Document N-[(1S)-2-{[(1R)-2-(benzyloxy)-1-cyano-1-methylethyl]amino}-1-(cyclohexylmethyl)-2-oxoethyl]morpholine-4-carboxamide - similarity: 0.34077\n",
      "Document Dibutylphthalate - similarity: 0.34076\n",
      "Document 1-[N-4'-NITROBENZYL-N-4'-CARBOXYBUTYLAMINO]METHYLPHOSPHONIC ACID - similarity: 0.34072\n",
      "Document Tucidinostat - similarity: 0.34068\n",
      "Document Amylamine - similarity: 0.34066\n",
      "Document Plicamycin - similarity: 0.34063\n",
      "Document Proscillaridin - similarity: 0.34034\n",
      "Document Dansylamide - similarity: 0.34028\n",
      "Document N-({6-[(4-CYANO-2-FLUOROBENZYL)OXY]NAPHTHALEN-2-YL}SULFONYL)-D-GLUTAMIC ACID - similarity: 0.34016\n",
      "Document Treosulfan - similarity: 0.34009\n",
      "Document LG-100268 - similarity: 0.34007\n",
      "Document Adamantane-1-Carboxylic Acid-5-Dimethylamino-Naphthalene-1-Sulfonylamino-Octyl-Amide - similarity: 0.34007\n",
      "Document alpha-D-Xylopyranose - similarity: 0.34005\n",
      "Document Pteric acid - similarity: 0.34002\n",
      "Document Malonyl-CoA - similarity: 0.34002\n",
      "Document Beta carotene - similarity: 0.34001\n",
      "Document Calcium sulfate - similarity: 0.34001\n",
      "Document Dequalinium - similarity: 0.33991\n",
      "Document Atriplex confertifolia pollen - similarity: 0.33989\n",
      "Document Ginsenoside Rg1 - similarity: 0.33986\n",
      "Document S-Phosphocysteine - similarity: 0.33984\n",
      "Document Chloric acid - similarity: 0.33976\n",
      "Document Androstenediol - similarity: 0.33962\n",
      "Document 1,8-Di-Hydroxy-4-Nitro-Xanthen-9-One - similarity: 0.33958\n",
      "Document Sodium chlorite - similarity: 0.33956\n",
      "Document Sulfathiazole - similarity: 0.33956\n",
      "Document Carisbamate - similarity: 0.33946\n",
      "Document Norethandrolone - similarity: 0.33946\n",
      "Document 2-amino-N-(4-methyl-1,3-thiazol-2-yl)-5-[(4-methyl-4H-1,2,4-triazol-3-yl)sulfanyl]benzamide - similarity: 0.33943\n",
      "Document Pilaralisib - similarity: 0.33939\n",
      "Document FK-960 - similarity: 0.33938\n",
      "Document 2'-fluoro-5-ethylarabinosyluracil - similarity: 0.33936\n",
      "Document AZD-4017 - similarity: 0.33923\n",
      "Document Dioxo(sulfanyl)molybdenum - similarity: 0.33914\n",
      "Document Hydroxybutyloxide - similarity: 0.33906\n",
      "Document NVP-LEQ-506 - similarity: 0.33890\n",
      "Document 2-aminooxyethyl-[[(2R,3S,4R,5R)-5-(6-aminopurin-9-yl)-3,4-dihydroxy-tetrahydrofuran-2-yl]methyl]-methyl-sulfonium - similarity: 0.33878\n",
      "Document Neocitrullamon - similarity: 0.33874\n",
      "Document Cystine - similarity: 0.33873\n",
      "Document Selenoinosine - similarity: 0.33873\n",
      "Document 1-(9-ethyl-9H-carbazol-3-yl)-N-methylmethanamine - similarity: 0.33863\n",
      "Document Tafamidis - similarity: 0.33863\n",
      "Document Dilazep - similarity: 0.33854\n",
      "Document Isoflurophate - similarity: 0.33852\n",
      "Document Baminercept - similarity: 0.33850\n",
      "Document N-[2-(1h-Indol-5-Yl)-Butyl]-4-Sulfamoyl-Benzamide - similarity: 0.33847\n",
      "Document 3,5-Difluorobenzenesulfonamide - similarity: 0.33842\n",
      "Document Dihomo-gamma-linolenic acid - similarity: 0.33834\n",
      "Document Dihydrolipoic Acid - similarity: 0.33823\n",
      "Document 5-(2-ETHOXYETHYL)-5-[4-(4-FLUOROPHENOXY)PHENOXY]PYRIMIDINE-2,4,6(1H,3H,5H)-TRIONE - similarity: 0.33816\n",
      "Document 2,2-Dimethylthiazolidine-4-Carboxylic Acid;(Dmt)Thiazolidine - similarity: 0.33816\n",
      "Document Imidazole-Derived Cellobiose - similarity: 0.33812\n",
      "Document Uridine-2',3'-vanadate - similarity: 0.33812\n",
      "Document Amylocaine - similarity: 0.33807\n",
      "Document XL820 - similarity: 0.33804\n",
      "Document 8-Benzo[1,3]Dioxol-,5-Ylmethyl-9-Butyl-2-Fluoro-9h-Purin-6-Ylamine - similarity: 0.33786\n",
      "Document Tiracizine - similarity: 0.33781\n",
      "Document Sertraline - similarity: 0.33778\n",
      "Document Mometasone - similarity: 0.33774\n",
      "Document American chestnut - similarity: 0.33768\n",
      "Document AZD-4547 - similarity: 0.33744\n",
      "Document N-[12-(1H-imidazol-1-yl)dodecanoyl]-L-leucine - similarity: 0.33743\n",
      "Document Gibberella fujikuroi - similarity: 0.33728\n",
      "Document Teprasiran - similarity: 0.33722\n",
      "Document Cavia porcellus dander - similarity: 0.33722\n",
      "Document (2S)-1-(2,5-dimethylphenoxy)-3-morpholin-4-ylpropan-2-ol - similarity: 0.33720\n",
      "Document Benzhydrocodone - similarity: 0.33713\n",
      "Document Sutimlimab - similarity: 0.33713\n",
      "Document CM-4307 - similarity: 0.33684\n",
      "Document A-98881 - similarity: 0.33682\n",
      "Document Threonine-Aspartic Ester - similarity: 0.33675\n",
      "Document Streptococcus pneumoniae type 23f capsular polysaccharide antigen - similarity: 0.33664\n",
      "Document 5-(6-Amino-9h-Purin-9-Yl)-4-Hydroxytetrahydrofuran-3-Yl Dihydrogen Phosphate - similarity: 0.33662\n",
      "Document Thymol - similarity: 0.33643\n",
      "Document Influenza B virus B/Phuket/3073/2013 antigen (UV, formaldehyde inactivated) - similarity: 0.33634\n",
      "Document E7107 - similarity: 0.33633\n",
      "Document Fulvestrant - similarity: 0.33632\n",
      "Document Rotenone - similarity: 0.33629\n",
      "Document Alisporivir - similarity: 0.33623\n",
      "Document {(4Z)-2-[(1R,2R)-1-Amino-2-hydroxypropyl]-4-[(4-amino-1H-indol-3-yl)methylene]-5-oxo-4,5-dihydro-1H-imidazol-1-yl}acetic acid - similarity: 0.33617\n",
      "Document 4SC-203 - similarity: 0.33603\n",
      "Document 4'-Hydroxyflavanone - similarity: 0.33599\n",
      "Document 3-Trimethylsilylsuccinic Acid - similarity: 0.33587\n",
      "Document N-({(3R,4R)-4-[(benzyloxy)methyl]pyrrolidin-3-yl}methyl)-N-(2-methylpropyl)benzenesulfonamide - similarity: 0.33576\n",
      "Document {4-[3-(6,7-Diethoxy-Quinazolin-4-Ylamino)-Phenyl]-Thiazol-2-Yl}-Methanol - similarity: 0.33571\n",
      "Document Ala-geninthiocin - similarity: 0.33561\n",
      "Document Lufenuron - similarity: 0.33551\n",
      "Document Chloroxine - similarity: 0.33551\n",
      "Document 4-HYDROXY-N-PROPARGYL-1(R)-AMINOINDAN - similarity: 0.33549\n",
      "Document Undecoylium chloride iodine complex - similarity: 0.33525\n",
      "Document Acer macrophyllum pollen - similarity: 0.33517\n",
      "Document Lipoic acid - similarity: 0.33515\n",
      "Document Farnesyl thiopyrophosphate - similarity: 0.33514\n",
      "Document Carbamic Acid - similarity: 0.33507\n",
      "Document 4-Oxo-2-Phenylmethanesulfonyl-Octahydro-Pyrrolo[1,2-a]Pyrazine-6-Carboxylic Acid [1-(N-Hydroxycarbamimidoyl)-Piperidin-4-Ylmethyl]-Amide - similarity: 0.33506\n",
      "Document 2-Amino-1-methyl-6-phenylimidazo(4,5-b)pyridine - similarity: 0.33503\n",
      "Document PF-03654764 - similarity: 0.33495\n",
      "Document Davanat - similarity: 0.33495\n",
      "Document Pagoclone - similarity: 0.33494\n",
      "Document N-(5-Isopropyl-thiazol-2-YL)-2-pyridin-3-YL-acetamide - similarity: 0.33490\n",
      "Document 2-[CARBOXY-(2-THIOPHEN-2-YL-ACETYLAMINO)-METHYL]-5-METHYLENE-5,6-DIHYDRO-2H-[1,3]THIAZINE-4-CARBOXYLIC ACID - similarity: 0.33488\n",
      "Document 6-(adenosine tetraphosphate-methyl)-7,8-dihydropterin - similarity: 0.33484\n",
      "Document (1R,2S)-2-(5-thioxo-4,5-dihydro-1H-1,2,4-triazol-3-yl)cyclohexanecarboxylic acid - similarity: 0.33482\n",
      "Document Tocofersolan - similarity: 0.33480\n",
      "Document Idraparinux - similarity: 0.33472\n",
      "Document Niridazole - similarity: 0.33471\n",
      "Document Hemin - similarity: 0.33468\n",
      "Document Adrabetadex - similarity: 0.33468\n",
      "Document (2R)-2-{(1R)-2-oxo-1-[(2-thienylacetyl)amino]ethyl}-5,6-dihydro-2h-1,3-thiazine-4-carboxylic acid - similarity: 0.33459\n",
      "Document Raclopride - similarity: 0.33456\n",
      "Document Sulfaethoxypyridazine - similarity: 0.33454\n",
      "Document Ronifibrate - similarity: 0.33453\n",
      "Document Bethanechol - similarity: 0.33443\n",
      "Document Terbogrel - similarity: 0.33442\n",
      "Document Troriluzole - similarity: 0.33432\n",
      "Document TETRAHEDRAL INTERMEDIATE OF BLASTICIDIN S - similarity: 0.33428\n",
      "Document (2R)-2-ETHYL-1-HEXANESULFONIC ACID - similarity: 0.33417\n",
      "Document Tafasitamab - similarity: 0.33400\n",
      "Document 4-({[4-(3-METHYLBENZOYL)PYRIDIN-2-YL]AMINO}METHYL)BENZENECARBOXIMIDAMIDE - similarity: 0.33373\n",
      "Document Bromochlorosalicylanilide - similarity: 0.33372\n",
      "Document Caroxazone - similarity: 0.33353\n",
      "Document Bupranolol - similarity: 0.33350\n",
      "Document IDM-1 - similarity: 0.33349\n",
      "Document Gastric intrinsic factor - similarity: 0.33345\n",
      "Document Curcuma aromatica root oil - similarity: 0.33330\n",
      "Document OTL-38 - similarity: 0.33329\n",
      "Document Osilodrostat - similarity: 0.33321\n",
      "Document (1R)-2-PHENYLACETAMIDO-2-(3-CARBOXYPHENYL)ETHYL BORONIC ACID - similarity: 0.33320\n",
      "Document Madecassoside - similarity: 0.33313\n",
      "Document Na-1 - similarity: 0.33307\n",
      "Document Amcasertib - similarity: 0.33302\n",
      "Document Dihexyverine - similarity: 0.33299\n",
      "Document Hydrocortisone butyrate - similarity: 0.33298\n",
      "Document Urochloa mutica pollen - similarity: 0.33292\n",
      "Document 5-METHYL-2-[(PHENYLSULFONYL)AMINO]BENZOIC ACID - similarity: 0.33291\n",
      "Document L-Xylopyranose - similarity: 0.33277\n",
      "Document GDC-0927 - similarity: 0.33275\n",
      "Document Leucine Phosphonic Acid - similarity: 0.33262\n",
      "Document Echinacoside - similarity: 0.33254\n",
      "Document (S)-acetoin - similarity: 0.33248\n",
      "Document 5,6-Cyclic-Tetrahydropteridine - similarity: 0.33236\n",
      "Document Vaborbactam - similarity: 0.33235\n",
      "Document MK-4965 - similarity: 0.33233\n",
      "Document D-phenylalanyl-N-benzyl-L-prolinamide - similarity: 0.33232\n",
      "Document PAC-113 - similarity: 0.33229\n",
      "Document Tetraethylammonium - similarity: 0.33228\n",
      "Document 1,3-Propanediol - similarity: 0.33220\n",
      "Document Proglumide - similarity: 0.33220\n",
      "Document 1-Beta-Ribofuranosyl-1,3-Diazepinone - similarity: 0.33218\n",
      "Document Lithium cation - similarity: 0.33212\n",
      "Document PHENYL[1-(N-SUCCINYLAMINO)PENTYL]PHOSPHONATE - similarity: 0.33199\n",
      "Document N-[4-CHLORO-3-(PYRIDIN-3-YLOXYMETHYL)-PHENYL]-3-FLUORO- - similarity: 0.33195\n",
      "Document Choline salicylate - similarity: 0.33193\n",
      "Document CP-195543 - similarity: 0.33192\n",
      "Document Cyd dengue virus serotype 4 live (attenuated) antigen - similarity: 0.33189\n",
      "Document Formaldehyde - similarity: 0.33188\n",
      "Document Gallium maltolate - similarity: 0.33186\n",
      "Document Niflumic acid - similarity: 0.33183\n",
      "Document Influenza A virus A/Victoria/210/2009 X-187 (H3N2) antigen (formaldehyde inactivated) - similarity: 0.33180\n",
      "Document Avidin - similarity: 0.33178\n",
      "Document 2-[3-chloro-6-[2,2-difluoro-2-(1-oxidopyridin-1-ium-2-yl)ethyl]imino-1-hydroxypyridin-2-yl]-N-[(1R)-1-(3-chlorophenyl)ethyl]acetamide - similarity: 0.33176\n",
      "Document Methiodal - similarity: 0.33174\n",
      "Document Hard fat - similarity: 0.33171\n",
      "Document Phenprocoumon - similarity: 0.33161\n",
      "Document Brexanolone - similarity: 0.33158\n",
      "Document 2-amino-5-[3-(1-ethyl-1H-pyrazol-5-yl)-1H-pyrrolo[2,3-b]pyridin-5-yl]-N,N-dimethylbenzamide - similarity: 0.33149\n",
      "Document Carboxyamidotriazole - similarity: 0.33139\n",
      "Document Tyvelose - similarity: 0.33138\n",
      "Document Nitrous acid - similarity: 0.33125\n",
      "Document Tetulomab tetraxetan Lu-177 - similarity: 0.33116\n",
      "Document Egg - similarity: 0.33112\n",
      "Document Bis-1,2-{[(Z)-2-carboxy-2-methyl-1,3-dioxane]-5-yloxycarbamoyl}-ethane - similarity: 0.33112\n",
      "Document Norfloxacin - similarity: 0.33111\n",
      "Document 4-(dimethylamino)-N-[7-(hydroxyamino)-7-oxoheptyl]benzamide - similarity: 0.33108\n",
      "Document Phenprobamate - similarity: 0.33097\n",
      "Document VB-201 - similarity: 0.33095\n",
      "Document N-3-OXO-DODECANOYL-L-HOMOSERINE LACTONE - similarity: 0.33094\n",
      "Document 2-Aminoethanimidic Acid - similarity: 0.33089\n",
      "Document 6,7,12,13-tetrahydro-5H-indolo[2,3-a]pyrrolo[3,4-c]carbazol-5-one - similarity: 0.33085\n",
      "Document Allicin - similarity: 0.33082\n",
      "Document Estrane-3,17-dione - similarity: 0.33082\n",
      "Document Cloperastine - similarity: 0.33064\n",
      "Document alpha-Amyl cinnamaldehyde - similarity: 0.33041\n",
      "Document Heptanoic acid - similarity: 0.33040\n",
      "Document Gestonorone caproate - similarity: 0.33031\n",
      "Document Fenpiverinium - similarity: 0.33014\n",
      "Document Formestane - similarity: 0.33010\n",
      "Document EVT-101 - similarity: 0.33002\n",
      "Document Plinabulin - similarity: 0.32993\n",
      "Document (21S)-1AZA-4,4-DIMETHYL-6,19-DIOXA-2,3,7,20-TETRAOXOBICYCLO[19.4.0] PENTACOSANE - similarity: 0.32968\n",
      "Document 4-hydroxycoumarin - similarity: 0.32967\n",
      "Document 4-(2,5-DIAMINO-5-HYDROXY-PENTYL)-PHENOL - similarity: 0.32964\n",
      "Document Peppermint oil - similarity: 0.32961\n",
      "Document 2-[4-[(Z)-2-Acetamido-3-oxo-3-[[(3S)-2-oxo-1-[(4-phenylphenyl)methyl]azepan-3-yl]amino]prop-1-enyl]-2-formylphenyl]acetic acid - similarity: 0.32955\n",
      "Document Sotatercept - similarity: 0.32950\n",
      "Document (9BETA,11ALPHA,13ALPHA,14BETA,17ALPHA)-11-(METHOXYMETHYL)ESTRA-1(10),2,4-TRIENE-3,17-DIOL - similarity: 0.32943\n",
      "Document CHF-6001 - similarity: 0.32927\n",
      "Document 4-({(2R,5S)-2,5-DIMETHYL-4-[(2R)-3,3,3-TRIFLUORO-2-HYDROXY-2-METHYLPROPANOYL]PIPERAZIN-1-YL}CARBONYL)BENZONITRILE - similarity: 0.32918\n",
      "Document N-[4-(AMINOSULFONYL)BENZYL]-5-(5-CHLORO-2,4-DIHYDROXYPHENYL)-1H-PYRAZOLE-4-CARBOXAMIDE - similarity: 0.32910\n",
      "Document Formononetin - similarity: 0.32902\n",
      "Document Diethylnorspermine - similarity: 0.32890\n",
      "Document N-{4-METHYL-3-[(3-PYRIMIDIN-4-YLPYRIDIN-2-YL)AMINO]PHENYL}-3-(TRIFLUOROMETHYL)BENZAMIDE - similarity: 0.32882\n",
      "Document Nafcillin - similarity: 0.32880\n",
      "Document Famitinib - similarity: 0.32879\n",
      "Document Zinc - similarity: 0.32866\n",
      "Document Cytidine-5'-Triphosphate - similarity: 0.32848\n",
      "Document [(1S)-1-(5-CHLORO-1-BENZOTHIEN-3-YL)-2-(2-NAPHTHYLAMINO)-2-OXOETHYL]PHOSPHONIC ACID - similarity: 0.32839\n",
      "Document Pine needle oil (pinus mugo) - similarity: 0.32815\n",
      "Document Chorionic Gonadotropin (Human) - similarity: 0.32813\n",
      "Document Laniquidar - similarity: 0.32812\n",
      "Document Ceramide NP - similarity: 0.32804\n",
      "Document Cycloserine - similarity: 0.32803\n",
      "Document Methyl alpha-galactoside - similarity: 0.32797\n",
      "Document H3B-8800 - similarity: 0.32794\n",
      "Document Amoxicillin - similarity: 0.32785\n",
      "Document Trilostane - similarity: 0.32782\n",
      "Document Influenza B virus B/Massachusetts/2/2012 hemagglutinin antigen (propiolactone inactivated) - similarity: 0.32782\n",
      "Document Samarium (153Sm) lexidronam - similarity: 0.32776\n",
      "Document 4-{[4-(1-CYCLOPROPYL-2-METHYL-1H-IMIDAZOL-5-YL)PYRIMIDIN-2-YL]AMINO}-N-METHYLBENZENESULFONAMIDE - similarity: 0.32775\n",
      "Document Taliglucerase alfa - similarity: 0.32775\n",
      "Document Lortalamine - similarity: 0.32768\n",
      "Document Cerdulatinib - similarity: 0.32763\n",
      "Document Mitoquinone - similarity: 0.32747\n",
      "Document Nutmeg - similarity: 0.32747\n",
      "Document 3,5-DIMETHYL-1-PHENYL-1H-PYRAZOLE-4-CARBOXYLIC ACID ETHYL ESTER - similarity: 0.32738\n",
      "Document TD-2749 - similarity: 0.32731\n",
      "Document Alpha-Methylisocitric Acid - similarity: 0.32730\n",
      "Document Uridine-5'-Diphosphate - similarity: 0.32725\n",
      "Document (S)-tacrine(10)-hupyridone - similarity: 0.32704\n",
      "Document (1R)-4-(3-phenoxyphenyl)-1-phosphonobutane-1-sulfonic acid - similarity: 0.32685\n",
      "Document Glycolic acid - similarity: 0.32682\n",
      "Document Butriptyline - similarity: 0.32675\n",
      "Document Fludrocortisone - similarity: 0.32667\n",
      "Document TRU-015 - similarity: 0.32656\n",
      "Document FM-VP4 - similarity: 0.32654\n",
      "Document Cocoa - similarity: 0.32630\n",
      "Document LGH-447 - similarity: 0.32626\n",
      "Document L-Homoserine - similarity: 0.32624\n",
      "Document AMG-337 - similarity: 0.32619\n",
      "Document Orantinib - similarity: 0.32614\n",
      "Document Heptabarbital - similarity: 0.32611\n",
      "Document Sulfacetamide - similarity: 0.32605\n",
      "Document Topiroxostat - similarity: 0.32604\n",
      "Document Polymyxin B - similarity: 0.32598\n",
      "Document Felbinac - similarity: 0.32585\n",
      "Document Cyanocobalamin Co-57 - similarity: 0.32585\n",
      "Document Imino[2-(2-oxo-1,2-dihydro-3-pyridinyl)-1H-benzimidazol-5-yl]methanaminium - similarity: 0.32581\n",
      "Document 3-Phenylpropylamine - similarity: 0.32580\n",
      "Document PRX-03140 - similarity: 0.32577\n",
      "Document Arylomycin A2 - similarity: 0.32574\n",
      "Document Lesogaberan - similarity: 0.32567\n",
      "Document N-{3-[4-(3-aminopropyl)piperazin-1-yl]propyl}-3-(alpha-D-galactopyranosyloxy)-5-nitrobenzamide - similarity: 0.32550\n",
      "Document Aminobenzoic acid - similarity: 0.32548\n",
      "Document Flunitrazepam - similarity: 0.32542\n",
      "Document 3-Dehydroquinic Acid - similarity: 0.32537\n",
      "Document Benzamidine - similarity: 0.32533\n",
      "Document DPDPE - similarity: 0.32528\n",
      "Document NX-1207 - similarity: 0.32522\n",
      "Document Sparteine - similarity: 0.32515\n",
      "Document Bietaserpine - similarity: 0.32514\n",
      "Document Phenaridine - similarity: 0.32512\n",
      "Document AL5927 - similarity: 0.32504\n",
      "Document Oxamniquine - similarity: 0.32501\n",
      "Document Salix discolor pollen - similarity: 0.32500\n",
      "Document 5-(1-Carboxy-1-Phosphonooxy-Ethoxyl)-Shikimate-3-Phosphate - similarity: 0.32491\n",
      "Document TM30339 - similarity: 0.32485\n",
      "Document (2R)-2-[(1R)-1-{[(2S)-2-Carboxy-2-(4-hydroxyphenyl)acetyl]amino}-1-methoxy-2-oxoethyl]-5-methylene-5,6-dihydro-2H-1,3-oxazine-4-carboxylic acid - similarity: 0.32473\n",
      "Document {4-[(2S)-2-({[(1S)-1-Carboxy-2-phenylethyl]carbamoyl}amino)-3-oxo-3-(pentylamino)propyl]phenoxy}malonic acid - similarity: 0.32446\n",
      "Document Molidustat - similarity: 0.32438\n",
      "Document Nicotinamide adenine dinucleotide cyclohexanone - similarity: 0.32431\n",
      "Document Actelion-1 - similarity: 0.32421\n",
      "Document 1-deoxy-1-{2,6,8-trioxo-7-[4-(phosphonooxy)butyl]-1,2,3,6,7,8-hexahydro-9H-purin-9-yl}-D-arabinitol - similarity: 0.32399\n",
      "Document 4-[(3-CHLORO-4-{[(2R)-3,3,3-TRIFLUORO-2-HYDROXY-2-METHYLPROPANOYL]AMINO}PHENYL)SULFONYL]-N,N-DIMETHYLBENZAMIDE - similarity: 0.32399\n",
      "Document Flavin mononucleotide - similarity: 0.32396\n",
      "Document Icotinib - similarity: 0.32394\n",
      "Document Rontalizumab - similarity: 0.32391\n",
      "Document Copper gluconate - similarity: 0.32368\n",
      "Document Formic acid - similarity: 0.32365\n",
      "Document Trioxsalen - similarity: 0.32349\n",
      "Document CHROMOPHORE (GLU-TYR-GLY) - similarity: 0.32346\n",
      "Document Zinc glycinate - similarity: 0.32342\n",
      "Document Glutathione sulfonic acid - similarity: 0.32341\n",
      "Document Plecanatide - similarity: 0.32325\n",
      "Document 1-Testosterone - similarity: 0.32312\n",
      "Document (R)-2-(FORMYLOXY)-3-(PHOSPHONOOXY)PROPYL PENTANOATE - similarity: 0.32307\n",
      "Document Pentavalent Antimony - similarity: 0.32299\n",
      "Document Carboxylic PRPP - similarity: 0.32297\n",
      "Document Neisseria meningitidis serogroup B recombinant LP2086 B01 protein variant antigen - similarity: 0.32279\n",
      "Document Pentabromophenol - similarity: 0.32273\n",
      "Document 2-(11-{2-[Benzenesulfonyl-(3-Methyl-Butyl)-Amino]-1-Hydroxy-Ethyl}-6,9-Dioxo-2-Oxa-7,10-Diaza-Bicyclo[11.2.2]Heptadeca-1(16),13(17),14-Trien-8-Yl)-Acetamide, Inhibitor 2 - similarity: 0.32272\n",
      "Document Imiglitazar - similarity: 0.32272\n",
      "Document Aedes taeniorhynchus - similarity: 0.32264\n",
      "Document Desmethylsertraline - similarity: 0.32257\n",
      "Document Forodesine - similarity: 0.32256\n",
      "Document Arimoclomol - similarity: 0.32245\n",
      "Document AZD-3839 - similarity: 0.32227\n",
      "Document 4-[5-(Trans-4-Aminocyclohexylamino)-3-Isopropylpyrazolo[1,5-a]Pyrimidin-7-Ylamino]-N,N-Dimethylbenzenesulfonamide - similarity: 0.32215\n",
      "Document Oxtriphylline - similarity: 0.32213\n",
      "Document N-[(1S)-2-amino-1-phenylethyl]-5-(1H-pyrrolo[2,3-b]pyridin-4-yl)thiophene-2-carboxamide - similarity: 0.32209\n",
      "Document Intoplicine - similarity: 0.32205\n",
      "Document 3,9-Dimethyladenine - similarity: 0.32146\n",
      "Document Procaine merethoxylline - similarity: 0.32145\n",
      "Document Juniperus virginiana - similarity: 0.32142\n",
      "Document 7,8-dihydro-6-hydroxymethyl-7-methyl-7-[2-phenylethyl]-pterin - similarity: 0.32139\n",
      "Document AZD0947 - similarity: 0.32128\n",
      "Document N-PYRIDOXYL-D-GLUTAMIC ACID-5'-MONOPHOSPHATE - similarity: 0.32123\n",
      "Document 4-(4-methyl-1,3-dioxo-1,3-dihydro-2H-isoindol-2-yl)benzonitrile - similarity: 0.32109\n",
      "Document 1-(1'-{[3-(methylsulfanyl)-2-benzothiophen-1-yl]carbonyl}spiro[1-benzofuran-3,4'-piperidin]-5-yl)methanamine - similarity: 0.32106\n",
      "Document Cefodizime - similarity: 0.32103\n",
      "Document MDX-018 - similarity: 0.32097\n",
      "Document Pseudoephedrine - similarity: 0.32090\n",
      "Document L-Eflornithine - similarity: 0.32089\n",
      "Document 4-Amino-5-Hydroxymethyl-2-Methylpyrimidine - similarity: 0.32078\n",
      "Document Mofebutazone - similarity: 0.32075\n",
      "Document 1-Ter-Butyl-3-P-Tolyl-1h-Pyrazolo[3,4-D]Pyrimidin-4-Ylamine - similarity: 0.32075\n",
      "Document Nabilone - similarity: 0.32073\n",
      "Document Dihydrocodeine - similarity: 0.32071\n",
      "Document Allspice - similarity: 0.32068\n",
      "Document Barley malt syrup - similarity: 0.32066\n",
      "Document Acetoacetyl-CoA - similarity: 0.32064\n",
      "Document GTS-21 - similarity: 0.32056\n",
      "Document Cocaine - similarity: 0.32049\n",
      "Document [4,6-O-(1-CARBOXYETHYLIDENE)-BETA-D-MANNOSE] - similarity: 0.32039\n",
      "Document Coenzyme A - similarity: 0.32030\n",
      "Document Methotrexate - similarity: 0.32026\n",
      "Document Valspodar - similarity: 0.32014\n",
      "Document S-3304 - similarity: 0.31995\n",
      "Document PYM50018 - similarity: 0.31992\n",
      "Document Paramethasone - similarity: 0.31991\n",
      "Document Omega-6 fatty acids - similarity: 0.31985\n",
      "Document Sapacitabine - similarity: 0.31980\n",
      "Document Citrulline - similarity: 0.31966\n",
      "Document Fibronectin - similarity: 0.31966\n",
      "Document Clavulanic acid - similarity: 0.31963\n",
      "Document Quassia - similarity: 0.31958\n",
      "Document N-Butyl-Benzenesulfonamide - similarity: 0.31958\n",
      "Document Glucoraphanin - similarity: 0.31953\n",
      "Document BIOTINOL-5-AMP - similarity: 0.31941\n",
      "Document Phosphatidyl serine - similarity: 0.31937\n",
      "Document TG1024 - similarity: 0.31933\n",
      "Document Oxyfedrine - similarity: 0.31926\n",
      "Document Octenidine - similarity: 0.31919\n",
      "Document Influenza A virus A/Victoria/210/2009 X-187 (H3N2) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.31914\n",
      "Document Polistes annularis venom protein - similarity: 0.31903\n",
      "Document Clofarabine - similarity: 0.31891\n",
      "Document Oxygen - similarity: 0.31882\n",
      "Document Hydracrylic acid - similarity: 0.31879\n",
      "Document Suptavumab - similarity: 0.31878\n",
      "Document Oxacillin - similarity: 0.31874\n",
      "Document LY-2300559 - similarity: 0.31870\n",
      "Document L-phenylalaninol - similarity: 0.31857\n",
      "Document CRS-3123 - similarity: 0.31852\n",
      "Document DSM-265 - similarity: 0.31843\n",
      "Document TAK-448 - similarity: 0.31839\n",
      "Document CC-401 - similarity: 0.31829\n",
      "Document methyl (1R,2S)-2-(hydroxycarbamoyl)-1-{4-[(2-methylquinolin-4-yl)methoxy]benzyl}cyclopropanecarboxylate - similarity: 0.31800\n",
      "Document N-(3-Propylcarbamoyloxirane-2-Carbonyl)-Isoleucyl-Proline - similarity: 0.31797\n",
      "Document CP-609754 - similarity: 0.31783\n",
      "Document Beclabuvir - similarity: 0.31773\n",
      "Document Urolithin A - similarity: 0.31747\n",
      "Document TRAZEOLIDE - similarity: 0.31743\n",
      "Document Epoprostenol - similarity: 0.31743\n",
      "Document 3-Oxiran-2ylalanine - similarity: 0.31731\n",
      "Document Astodrimer - similarity: 0.31727\n",
      "Document Butobarbital - similarity: 0.31721\n",
      "Document Tris-Hydroxymethyl-Methyl-Ammonium - similarity: 0.31704\n",
      "Document R-306465 - similarity: 0.31695\n",
      "Document 5-ETHYL-3-METHYL-1,5-DIHYDRO-4H-PYRAZOLO[4,3-C]QUINOLIN-4-ONE - similarity: 0.31673\n",
      "Document Chondrus crispus - similarity: 0.31665\n",
      "Document LY249543 - similarity: 0.31663\n",
      "Document 2,4-Dinitrophenol - similarity: 0.31662\n",
      "Document Betaine aldehyde - similarity: 0.31662\n",
      "Document Anhydrovinblastine - similarity: 0.31658\n",
      "Document PPI-1019 - similarity: 0.31656\n",
      "Document 5-fluoro-2'-deoxycytidine - similarity: 0.31647\n",
      "Document (Z)-2-[2-(4-methylpiperazin-1-yl)benzyl]diazenecarbothioamide - similarity: 0.31633\n",
      "Document Ensituximab - similarity: 0.31623\n",
      "Document Metenkefalin - similarity: 0.31610\n",
      "Document Lidoflazine - similarity: 0.31608\n",
      "Document Trimethaphan - similarity: 0.31604\n",
      "Document Fluoroazomycin arabinoside F-18 - similarity: 0.31595\n",
      "Document 4-hydroxybenzoyl-CoA - similarity: 0.31592\n",
      "Document Idoxuridine - similarity: 0.31587\n",
      "Document 2-Chlorodideoxyadenosine - similarity: 0.31583\n",
      "Document Bis(N-maleimidomethyl)ether - similarity: 0.31579\n",
      "Document Prosopis velutina seed - similarity: 0.31577\n",
      "Document P113D - similarity: 0.31551\n",
      "Document Oxothiazolidinecarboxylic acid - similarity: 0.31541\n",
      "Document Clobutinol - similarity: 0.31531\n",
      "Document 2-Ethylhexyl 4-phenylbenzophenone-2'-carboxylate - similarity: 0.31531\n",
      "Document CG7870 - similarity: 0.31526\n",
      "Document Decyloxy-Methanol - similarity: 0.31518\n",
      "Document Evernimicin - similarity: 0.31515\n",
      "Document 7-Iodo-1,2,3,4-Tetrahydro-Isoquinoline - similarity: 0.31513\n",
      "Document Alpha-Methyl-N-Acetyl-D-Glucosamine - similarity: 0.31511\n",
      "Document 4-(2-aminoethoxy)-N-(3-chloro-2-ethoxy-5-piperidin-1-ylphenyl)-3,5-dimethylbenzamide - similarity: 0.31497\n",
      "Document Violaxanthin - similarity: 0.31496\n",
      "Document Varicella Zoster Vaccine (Live/attenuated) - similarity: 0.31489\n",
      "Document 3-bromo-N'-[(1E)-(3,5-dibromo-2,4-dihydroxyphenyl)methylidene]benzohydrazide - similarity: 0.31479\n",
      "Document LY-2584702 - similarity: 0.31467\n",
      "Document 9(S)-HODE - similarity: 0.31459\n",
      "Document Cobomarsen - similarity: 0.31457\n",
      "Document Methyl vanillate - similarity: 0.31453\n",
      "Document Hymenoclea salsola pollen - similarity: 0.31450\n",
      "Document Ginsenoside B2 - similarity: 0.31435\n",
      "Document MCC - similarity: 0.31426\n",
      "Document N-(4-{2-[(3-chlorobenzyl)amino]ethyl}phenyl)thiophene-2-carboximidamide - similarity: 0.31405\n",
      "Document Radium Ra-223 - similarity: 0.31396\n",
      "Document Aceneuramic acid - similarity: 0.31394\n",
      "Document Levocabastine - similarity: 0.31394\n",
      "Document Pyruvaldehyde - similarity: 0.31390\n",
      "Document 2-Deoxy-Glucitol-6-Phosphate - similarity: 0.31390\n",
      "Document PD-217014 - similarity: 0.31384\n",
      "Document 2-(2,6-DIFLUOROPHENOXY)-N-(2-FLUOROPHENYL)-9-ISOPROPYL-9H-PURIN-8-AMINE - similarity: 0.31369\n",
      "Document N-Tridecanoic Acid - similarity: 0.31342\n",
      "Document Saccharin - similarity: 0.31341\n",
      "Document VB2-011 - similarity: 0.31328\n",
      "Document Bergamot oil - similarity: 0.31325\n",
      "Document (2S)-2-({4-[(2S)-1-(2-Amino-4-oxo-1,4-dihydro-6-quinazolinyl)-3-{[2-({(2R,3R,4S,5R)-3,4-dihydroxy-5-[(phosphonooxy)methyl]tetrahydro-2-furanyl}amino)-2-oxoethyl]amino}-2-hydroxy-2-propanyl]benzoyl}ami no)pentanedioic acid - similarity: 0.31297\n",
      "Document Calcium cation - similarity: 0.31282\n",
      "Document (1S)-1-C-{(2R)-6-[(2R)-2-Butanyl]-5,10-dihydroxy-4-oxo-1,2,3,4-tetrahydro-2-anthracenyl}-5-deoxy-1-O-methyl-D-xylulose - similarity: 0.31281\n",
      "Document 3,4-bis(7-chloro-1H-indol-3-yl)-1H-pyrrole-2,5-dicarboxylic acid - similarity: 0.31279\n",
      "Document (S)-4-hydroxymandelonitrile - similarity: 0.31276\n",
      "Document PNU-142721 - similarity: 0.31268\n",
      "Document Telbermin - similarity: 0.31261\n",
      "Document Ostrea edulis shell - similarity: 0.31257\n",
      "Document AC3056 - similarity: 0.31236\n",
      "Document Etofylline nicotinate - similarity: 0.31236\n",
      "Document UNDECYLAMINE-N,N-DIMETHYL-N-OXIDE - similarity: 0.31221\n",
      "Document Tuberculin purified protein derivative - similarity: 0.31213\n",
      "Document AEG35156 - similarity: 0.31210\n",
      "Document NS-398 - similarity: 0.31210\n",
      "Document 3-(alpha-D-Galactopyranosyloxy)-5-nitrobenzamide - similarity: 0.31207\n",
      "Document Valoctocogene roxaparvovec - similarity: 0.31206\n",
      "Document (2S)-1-[(2S,4R)-4-Benzyl-2-hydroxy-5-{[(1S,2R,5S)-2-hydroxy-5-methylcyclopentyl]amino}-5-oxopentyl]-4-{[6-chloro-5-(4-methyl-1-piperazinyl)-2-pyrazinyl]carbonyl}-N-(2-methyl-2-propanyl)-2-piperazineca rboxamide - similarity: 0.31199\n",
      "Document Naringenin - similarity: 0.31169\n",
      "Document Florilglutamic acid F-18 - similarity: 0.31168\n",
      "Document Pipradrol - similarity: 0.31167\n",
      "Document 1-(2-Phenylethyl)-4-phenyl-4-acetoxypiperidine - similarity: 0.31148\n",
      "Document Raspberry - similarity: 0.31124\n",
      "Document Remdesivir - similarity: 0.31109\n",
      "Document Saccharide isomerate - similarity: 0.31105\n",
      "Document PSN357 - similarity: 0.31096\n",
      "Document 2-Benzyl-3-Iodopropanoic Acid - similarity: 0.31095\n",
      "Document Parsnip - similarity: 0.31086\n",
      "Document Naluzotan - similarity: 0.31085\n",
      "Document Diundecyl Phosphatidyl Choline - similarity: 0.31082\n",
      "Document MONOGALACTOSYL-DIACYLGLYCEROL - similarity: 0.31079\n",
      "Document 4-{[(CYCLOHEXYLAMINO)CARBONYL]AMINO}BUTANOIC ACID - similarity: 0.31077\n",
      "Document Levonordefrin - similarity: 0.31070\n",
      "Document LGD-3303 - similarity: 0.31067\n",
      "Document Oxolamine - similarity: 0.31065\n",
      "Document 7A-[(4-cyanophenyl)methyl]-6-(3,5-dichlorophenyl)-5-oxo-2,3,5,7A-tetrahydro-1H-pyrrolo[1,2-A]pyrrole-7-carbonitrile - similarity: 0.31061\n",
      "Document NS-2359 - similarity: 0.31057\n",
      "Document Nicotinyl alcohol - similarity: 0.31054\n",
      "Document Odiparcil - similarity: 0.31047\n",
      "Document Nitrilotriacetic acid - similarity: 0.31046\n",
      "Document Choriogonadotropin alfa - similarity: 0.31046\n",
      "Document 4-[4-(4-methoxyphenyl)-5-methyl-1H-pyrazol-3-yl]benzene-1,3-diol - similarity: 0.31043\n",
      "Document Anti-SARS-CoV-2 REGN-COV2 - similarity: 0.31039\n",
      "Document Cholesterol sulfate - similarity: 0.31035\n",
      "Document Fidaxomicin - similarity: 0.31032\n",
      "Document AZD-1305 - similarity: 0.31030\n",
      "Document Pozanicline - similarity: 0.31019\n",
      "Document Testolactone - similarity: 0.31015\n",
      "Document Betiatide - similarity: 0.31012\n",
      "Document Furosemide - similarity: 0.31009\n",
      "Document Methyl pyrrolidone - similarity: 0.31007\n",
      "Document Acteoside - similarity: 0.31000\n",
      "Document 2-fluoro-6-{[2-({2-methoxy-4-[(methylsulfonyl)methyl]phenyl}amino)-7H-pyrrolo[2,3-d]pyrimidin-4-yl]amino}benzamide - similarity: 0.31000\n",
      "Document Serinus canaria feather - similarity: 0.30993\n",
      "Document 2-Deoxy-D-Glucitol 6-(E)-Vinylhomophosphonate - similarity: 0.30990\n",
      "Document 4,4'-Biphenyldiboronic Acid - similarity: 0.30972\n",
      "Document Tacrine - similarity: 0.30965\n",
      "Document 3-(HYDROXYMETHYL)-1-METHYL-5-(2-METHYLAZIRIDIN-1-YL)-2-PHENYL-1H-INDOLE-4,7-DIONE - similarity: 0.30948\n",
      "Document Al-6619, [2h-Thieno[3,2-E]-1,2-Thiazine-6-Sulfonamide,2-(3-Hydroxyphenyl)-3-(4-Morpholinyl)-, 1,1-Dioxide] - similarity: 0.30940\n",
      "Document PX-12 - similarity: 0.30937\n",
      "Document Linrodostat - similarity: 0.30935\n",
      "Document PF-00217830 - similarity: 0.30934\n",
      "Document 4-(2-AMINOETHYL)BENZENESULFONAMIDE - similarity: 0.30926\n",
      "Document Pyrethrum extract - similarity: 0.30922\n",
      "Document Chlorobutanol - similarity: 0.30921\n",
      "Document Lubiprostone - similarity: 0.30909\n",
      "Document Wine grape - similarity: 0.30895\n",
      "Document GPX-150 - similarity: 0.30893\n",
      "Document N-(2-Aminoethyl)-1-aziridineethanamine - similarity: 0.30883\n",
      "Document 2-Acetamido-2-Deoxy-D-Glucono-1,5-Lactone - similarity: 0.30865\n",
      "Document Tribulus terrestris root - similarity: 0.30864\n",
      "Document Vitamin B6 Complexed with 2-Amino-Pentanoic Acid - similarity: 0.30863\n",
      "Document Fiacitabine - similarity: 0.30863\n",
      "Document FK-614 - similarity: 0.30862\n",
      "Document Garlic - similarity: 0.30857\n",
      "Document Sitotroga cerealella - similarity: 0.30849\n",
      "Document Gold - similarity: 0.30828\n",
      "Document 2-Pyridinethiol - similarity: 0.30827\n",
      "Document Namn - similarity: 0.30822\n",
      "Document 5-(Hydroxy-Methyl-Amino)-3-Methyl-Pyrrolidine-2-Carboxylic Acid - similarity: 0.30821\n",
      "Document Tibulizumab - similarity: 0.30821\n",
      "Document 2-Hydroxyestradiol - similarity: 0.30816\n",
      "Document Pitolisant - similarity: 0.30812\n",
      "Document Magnolia kobus flowering top - similarity: 0.30810\n",
      "Document Defactinib - similarity: 0.30803\n",
      "Document Dihydro-Acarbose - similarity: 0.30799\n",
      "Document Cannabidivarin - similarity: 0.30796\n",
      "Document Technetium Tc-99m tetrofosmin - similarity: 0.30785\n",
      "Document BMS-986115 - similarity: 0.30775\n",
      "Document OLOMOUCINE II - similarity: 0.30769\n",
      "Document PF-03758309 - similarity: 0.30769\n",
      "Document (2S)-2-amino-5-oxo-5-[(4-phenylmethoxyphenyl)amino]pentanoic acid - similarity: 0.30763\n",
      "Document Myxopyronin B - similarity: 0.30759\n",
      "Document Desoxycorticosterone acetate - similarity: 0.30748\n",
      "Document D-chiro-Inositol - similarity: 0.30726\n",
      "Document ACY001 - similarity: 0.30720\n",
      "Document Sporisorium cruentum - similarity: 0.30719\n",
      "Document Letrazuril - similarity: 0.30717\n",
      "Document Hypomyces perniciosus - similarity: 0.30712\n",
      "Document 2-amino-3-[4-hydroxy-6-oxo-3-(2-phenyl-cyclopropylimino)-cyclohexa-1,4-dienyl]-propionic acid - similarity: 0.30700\n",
      "Document N-{3-METHYL-5-[2-(PYRIDIN-4-YLAMINO)-ETHOXY]-PHENYL}-BENZENESULFONAMIDE - similarity: 0.30675\n",
      "Document Heparin - similarity: 0.30670\n",
      "Document 6-Chloro-4-[(R)-cyclohexylsulfinyl]-3-propyl-2(1H)-quinolinone - similarity: 0.30661\n",
      "Document TMC-649128 - similarity: 0.30661\n",
      "Document Hypochlorous acid - similarity: 0.30659\n",
      "Document Iodide I-123 - similarity: 0.30655\n",
      "Document 1-Deoxy-1-methoxycarbamido-beta-D-gluco-2-heptulopyranosonamide - similarity: 0.30642\n",
      "Document L-tyrosinamide - similarity: 0.30641\n",
      "Document Golvatinib - similarity: 0.30635\n",
      "Document Latrepirdine - similarity: 0.30629\n",
      "Document Mucor plumbeus - similarity: 0.30625\n",
      "Document Nitrite - similarity: 0.30618\n",
      "Document SB-1578 - similarity: 0.30613\n",
      "Document 3-amino-5-phenylpentane - similarity: 0.30606\n",
      "Document Efinopegdutide - similarity: 0.30598\n",
      "Document Formycin B - similarity: 0.30590\n",
      "Document Influenza B virus B/Brisbane/60/2008 antigen (UV, formaldehyde inactivated) - similarity: 0.30587\n",
      "Document 3-[2-bromo-4-(1H-pyrazolo[3,4-c]pyridazin-3-ylmethyl)phenoxy]-5-methylbenzonitrile - similarity: 0.30583\n",
      "Document Alferminogene tadenovec - similarity: 0.30578\n",
      "Document Velafermin - similarity: 0.30577\n",
      "Document Iodide - similarity: 0.30573\n",
      "Document D-Lactic acid - similarity: 0.30573\n",
      "Document 6-O-Phosphoryl Inosine Monophosphate - similarity: 0.30566\n",
      "Document Dasotraline - similarity: 0.30563\n",
      "Document Trevogrumab - similarity: 0.30559\n",
      "Document Alfimeprase - similarity: 0.30558\n",
      "Document Fytic acid - similarity: 0.30557\n",
      "Document Acer negundo pollen - similarity: 0.30555\n",
      "Document Prulifloxacin - similarity: 0.30553\n",
      "Document SYRINGATE - similarity: 0.30545\n",
      "Document 3-Hydroxymethyl-5-Aziridinyl-1methyl-2-[1h-Indole-4,7-Dione]-Propanol - similarity: 0.30537\n",
      "Document Cyproheptadine - similarity: 0.30535\n",
      "Document Glyceryl Trierucate - similarity: 0.30531\n",
      "Document 3-hydroxyisoxazole-4-carboxylic acid - similarity: 0.30509\n",
      "Document 4-O-(4,6-Dideoxy-4-{[4-[(4-O-Hexopyranosylhexopyranosyl)Oxy]-5,6-Dihydroxy-3-(Hydroxymethyl)Cyclohex-2-En-1-Yl]Amino}Hexopyranosyl)Hexopyranose - similarity: 0.30485\n",
      "Document Sonolisib - similarity: 0.30484\n",
      "Document Meningococcal (groups A, C, Y and W-135) oligosaccharide diphtheria CRM197 conjugate vaccine - similarity: 0.30476\n",
      "Document Neamine - similarity: 0.30473\n",
      "Document Sulfamerazine - similarity: 0.30459\n",
      "Document 2,3-Dimethylimidazolium Ion - similarity: 0.30447\n",
      "Document Biphenyl dimethyl dicarboxylate - similarity: 0.30445\n",
      "Document Cytidine-5'-Monophosphate - similarity: 0.30413\n",
      "Document Methdilazine - similarity: 0.30408\n",
      "Document N-[(1S)-2-methyl-1-(pyridin-4-ylcarbamoyl)propyl]cyclohexanecarboxamide - similarity: 0.30405\n",
      "Document meso-Tartaric acid - similarity: 0.30396\n",
      "Document Nandrolone - similarity: 0.30395\n",
      "Document Sulfuric acid - similarity: 0.30394\n",
      "Document SLV-334 - similarity: 0.30388\n",
      "Document Dianhydrogalactitol - similarity: 0.30384\n",
      "Document Xamoterol - similarity: 0.30374\n",
      "Document Fezolinetant - similarity: 0.30374\n",
      "Document Tirilazad - similarity: 0.30358\n",
      "Document Edoxaban - similarity: 0.30357\n",
      "Document 3,3',5,5'-tetrachlorobiphenyl-4,4'-diol - similarity: 0.30353\n",
      "Document Tetrabenazine - similarity: 0.30352\n",
      "Document Iodamide - similarity: 0.30335\n",
      "Document Sirna-027 - similarity: 0.30328\n",
      "Document 2-(Oxalyl-Amino)-4,7-Dihydro-5h-Thieno[2,3-C]Pyran-3-Carboxylic Acid - similarity: 0.30322\n",
      "Document Porfiromycin - similarity: 0.30316\n",
      "Document Mapracorat - similarity: 0.30307\n",
      "Document cis-Diamminechlorocholylglycinateplatinum(II) - similarity: 0.30301\n",
      "Document Entrectinib - similarity: 0.30300\n",
      "Document Bulevirtide - similarity: 0.30275\n",
      "Document Bamipine - similarity: 0.30274\n",
      "Document Influenza B virus B/Massachusetts/2/2012 BX-51B antigen (propiolactone inactivated) - similarity: 0.30263\n",
      "Document NPI 32101 - similarity: 0.30263\n",
      "Document 5-Chloro-1h-Indole-2-Carboxylic Acid{[Cyclopentyl-(2-Hydroxy-Ethyl)-Carbamoyl]-Methyl}-Amide - similarity: 0.30245\n",
      "Document Cytidine 3'-monophosphate - similarity: 0.30237\n",
      "Document Hydrogenated soybean oil - similarity: 0.30228\n",
      "Document ADC4022 - similarity: 0.30212\n",
      "Document 6-Amino-4-Hydroxymethyl-Cyclohex-4-Ene-1,2,3-Triol - similarity: 0.30205\n",
      "Document Ferrous chloride - similarity: 0.30192\n",
      "Document Bitolterol - similarity: 0.30163\n",
      "Document Tolciclate - similarity: 0.30158\n",
      "Document Streptococcus pneumoniae type 18c capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.30157\n",
      "Document Cyanamide - similarity: 0.30139\n",
      "Document Orotidylic acid - similarity: 0.30131\n",
      "Document Dolcanatide - similarity: 0.30128\n",
      "Document DDP733 - similarity: 0.30128\n",
      "Document Polyethylene glycol 3500 - similarity: 0.30126\n",
      "Document Streptomyces griseus - similarity: 0.30123\n",
      "Document Cannabichromene - similarity: 0.30116\n",
      "Document Fludeoxyglucose (18F) - similarity: 0.30108\n",
      "Document LJP 1082 - similarity: 0.30092\n",
      "Document 2-(2-HYDROXY-CYCLOPENTYL)-PENT-4-ENAL - similarity: 0.30073\n",
      "Document Ethosuximide - similarity: 0.30072\n",
      "Document Isovaleric Acid - similarity: 0.30072\n",
      "Document Isepamicin - similarity: 0.30041\n",
      "Document Carboxymycobactin T - similarity: 0.30034\n",
      "Document Rabeximod - similarity: 0.30033\n",
      "Document Mangostin - similarity: 0.30018\n",
      "Document Galiximab - similarity: 0.30011\n",
      "Document Crotalus atrox antivenin - similarity: 0.30010\n",
      "Document Posizolid - similarity: 0.30008\n",
      "Document Elesclomol - similarity: 0.29985\n",
      "Document 2-(Oxalyl-Amino)-Benzoic Acid - similarity: 0.29968\n",
      "Document 2,6-anhydro-3-deoxy-3-fluoronononic acid - similarity: 0.29951\n",
      "Document L-deprenyl-D2 C-11 - similarity: 0.29936\n",
      "Document Cerulenin - similarity: 0.29934\n",
      "Document Fluorofuranylnorprogesterone F-18 - similarity: 0.29923\n",
      "Document Influenza B virus B/Maryland/15/2016 antigen (propiolactone inactivated) - similarity: 0.29916\n",
      "Document Sacubitril - similarity: 0.29914\n",
      "Document dATP - similarity: 0.29906\n",
      "Document (Rp)-cAMPS - similarity: 0.29893\n",
      "Document Typhoid Vi polysaccharide vaccine - similarity: 0.29882\n",
      "Document Haloxazolam - similarity: 0.29876\n",
      "Document Pexacerfont - similarity: 0.29875\n",
      "Document 2-Oxo-3-Pentenoic Acid - similarity: 0.29868\n",
      "Document (3,4,8b-Trimethyl-3-oxido-2,3a-dihydro-1H-pyrrolo[2,3-b]indol-3-ium-7-yl) N-(2-ethylphenyl)carbamate - similarity: 0.29867\n",
      "Document Acarbose Derived Hexasaccharide - similarity: 0.29865\n",
      "Document 1,10-Phenanthroline - similarity: 0.29863\n",
      "Document Alpha-L-Arabinose - similarity: 0.29862\n",
      "Document (6Z)-6-{[(1Z)-1-{[(1R)-1-Carboxy-2-methyl-2-propen-1-yl]imino}-1-hydroxy-3-sulfanyl-2-propanyl]imino}-6-hydroxynorleucine - similarity: 0.29841\n",
      "Document Bevirimat - similarity: 0.29823\n",
      "Document Beraprost - similarity: 0.29815\n",
      "Document 2-Fluoroadenosine - similarity: 0.29806\n",
      "Document Gedivumab - similarity: 0.29806\n",
      "Document 1-(2,2'-bithiophen-5-yl)methanamine - similarity: 0.29796\n",
      "Document [1-(3-hydroxy-2-oxo-1-phenethyl-propylcarbamoyl)2-phenyl-ethyl]-carbamic acid pyridin-4-ylmethyl ester - similarity: 0.29786\n",
      "Document Pyridoxine phosphate - similarity: 0.29786\n",
      "Document Siroheme - similarity: 0.29776\n",
      "Document K201 free base - similarity: 0.29773\n",
      "Document LX201 - similarity: 0.29760\n",
      "Document sn-glycerol 3-phosphate - similarity: 0.29760\n",
      "Document Sus scrofa stomach - similarity: 0.29731\n",
      "Document Formycin - similarity: 0.29719\n",
      "Document Vitespen - similarity: 0.29714\n",
      "Document (5-Hydroxy-6-methyl-4-{[(E)-(3-oxo-1,2-oxazolidin-4-ylidene)amino]methyl}-3-pyridinyl)methyl dihydrogen phosphate - similarity: 0.29712\n",
      "Document Cetilistat - similarity: 0.29701\n",
      "Document Stachybotrys chartarum - similarity: 0.29691\n",
      "Document FG-9041 - similarity: 0.29684\n",
      "Document Queuine - similarity: 0.29679\n",
      "Document AI-850 - similarity: 0.29662\n",
      "Document Brefeldin A - similarity: 0.29649\n",
      "Document Beclomethasone 17-monopropionate - similarity: 0.29645\n",
      "Document Diphenhydramine - similarity: 0.29644\n",
      "Document 4-{5-[(Z)-(2,4-DIOXO-1,3-THIAZOLIDIN-5-YLIDENE)METHYL]FURAN-2-YL}BENZENESULFONAMIDE - similarity: 0.29634\n",
      "Document 1-Monohexanoyl-2-Hydroxy-Sn-Glycero-3-Phosphate - similarity: 0.29625\n",
      "Document N-Carbamyl-D-Valine - similarity: 0.29617\n",
      "Document 4-(6-CYCLOHEXYLMETHOXY-9H-PURIN-2-YLAMINO)--BENZAMIDE - similarity: 0.29602\n",
      "Document 19-(cyclopropylamino)-4,6,7,15-tetrahydro-5H-16,1-(azenometheno)-10,14-(metheno)pyrazolo[4,3-o][1,3,9]triazacyclohexadecin-8(9H)-one - similarity: 0.29599\n",
      "Document ONO-2952 - similarity: 0.29582\n",
      "Document Mibampator - similarity: 0.29575\n",
      "Document Cultivated mushroom - similarity: 0.29572\n",
      "Document 3-Amino-6-Hydroxy-Tyrosine - similarity: 0.29570\n",
      "Document 3,4-Dihydroxycinnamic Acid - similarity: 0.29563\n",
      "Document Styramate - similarity: 0.29560\n",
      "Document Medicago sativa pollen - similarity: 0.29555\n",
      "Document OPC-51803 - similarity: 0.29551\n",
      "Document Bivatuzumab - similarity: 0.29543\n",
      "Document Malacidin A - similarity: 0.29541\n",
      "Document Prinomastat - similarity: 0.29541\n",
      "Document Albumin iodinated I-125 serum - similarity: 0.29523\n",
      "Document ACR325 - similarity: 0.29520\n",
      "Document 2-amino-7-fluoro-5-oxo-5H-chromeno[2,3-b]pyridine-3-carboxamide - similarity: 0.29514\n",
      "Document (6,7-Difluoro-Quinazolin-4-Yl)-(1-Methyl-2,2-Diphenyl-Ethyl)-Amine - similarity: 0.29512\n",
      "Document ZW-49 - similarity: 0.29509\n",
      "Document alpha-Conotoxin VC 1.1 - similarity: 0.29507\n",
      "Document Meptazinol - similarity: 0.29505\n",
      "Document Albumin human - similarity: 0.29497\n",
      "Document (5E)-2-Amino-5-(2-pyridinylmethylene)-1,3-thiazol-4(5H)-one - similarity: 0.29445\n",
      "Document Ferric pyrophosphate citrate - similarity: 0.29441\n",
      "Document METHYL N-[(2S,3R)-3-AMINO-2-HYDROXY-3-(4-ISOPROPYLPHENYL)PROPANOYL]-D-ALANYL-D-LEUCINATE - similarity: 0.29436\n",
      "Document N-Alpha-(2-Naphthylsulfonyl)-N(3-Amidino-L-Phenylalaninyl)Isopipecolinic Acid Methyl Ester - similarity: 0.29408\n",
      "Document Tetramethylthiuram monosulfide - similarity: 0.29405\n",
      "Document Carotuximab - similarity: 0.29388\n",
      "Document 2-Amino-3-Methyl-1-Pyrrolidin-1-Yl-Butan-1-One - similarity: 0.29366\n",
      "Document N-({(2S,3S)-3-[(BENZYLAMINO)CARBONYL]OXIRAN-2-YL}CARBONYL)-L-ISOLEUCYL-L-PROLINE - similarity: 0.29336\n",
      "Document Epratuzumab - similarity: 0.29322\n",
      "Document Aniline - similarity: 0.29303\n",
      "Document MAHDL01 - similarity: 0.29302\n",
      "Document Brequinar Analog - similarity: 0.29294\n",
      "Document (1R)-2-amino-1-[3-(trifluoromethyl)phenyl]ethanol - similarity: 0.29293\n",
      "Document Enadenotucirev - similarity: 0.29277\n",
      "Document Elsulfavirine - similarity: 0.29275\n",
      "Document Apomorphine - similarity: 0.29266\n",
      "Document 2'-O-butyl-5-methyluridine - similarity: 0.29265\n",
      "Document 1-(3,5-DICHLOROPHENYL)-5-METHYL-1H-1,2,4-TRIAZOLE-3-CARBOXYLIC ACID - similarity: 0.29262\n",
      "Document Adenosine 5'-methylenediphosphate - similarity: 0.29248\n",
      "Document Sulforaphane - similarity: 0.29240\n",
      "Document (3AALPHA,4ALPHA,7ALPHA,7AALPHA)- 3A,4,7,7A-TETRAHYDRO-2-(4-NITRO-1-NAPHTHALENYL)-4,7-ETHANO-1H-ISOINDOLE-1,3(2H)-DIONE - similarity: 0.29237\n",
      "Document Arsenic trioxide - similarity: 0.29229\n",
      "Document Methylmethionine chloride - similarity: 0.29224\n",
      "Document BMS-903452 - similarity: 0.29218\n",
      "Document Elubrixin - similarity: 0.29203\n",
      "Document N-[Isoleucinyl]-N'-[adenosyl]-diaminosufone - similarity: 0.29203\n",
      "Document Bizelesin - similarity: 0.29198\n",
      "Document Influenza B virus B/Brisbane/60/2008 hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.29197\n",
      "Document Ouabain - similarity: 0.29194\n",
      "Document Mazaticol - similarity: 0.29181\n",
      "Document Cinepazet - similarity: 0.29176\n",
      "Document (S)-atrolactic acid - similarity: 0.29173\n",
      "Document 2,4-Diamino-5-(3,4,5-Trimethoxy-Benzyl)-Pyrimidin-1-Ium - similarity: 0.29172\n",
      "Document Levomilnacipran - similarity: 0.29156\n",
      "Document Clostridium tetani - similarity: 0.29149\n",
      "Document Glucosaminyl-(Alpha-6)-D-Myo-Inositol - similarity: 0.29148\n",
      "Document 4-(2,4-dichlorophenyl)-5-phenyldiazenyl-pyrimidin-2-amine - similarity: 0.29143\n",
      "Document Bufotenine - similarity: 0.29140\n",
      "Document AT2220 - similarity: 0.29139\n",
      "Document Anistreplase - similarity: 0.29139\n",
      "Document Maltotetraose - similarity: 0.29138\n",
      "Document S-Methylcysteine - similarity: 0.29125\n",
      "Document Sweet potato - similarity: 0.29120\n",
      "Document 8-(2-CHLOROPHENYLAMINO)-2-(2,6-DIFLUOROPHENYLAMINO)-9-ETHYL-9H-PURINE-1,7-DIIUM - similarity: 0.29116\n",
      "Document EIDD-2801 - similarity: 0.29109\n",
      "Document N-(2-Aminoethyl)-5-Chloroisoquinoline-8-Sulfonamide - similarity: 0.29100\n",
      "Document (+)-1-bromo-2-propanol - similarity: 0.29095\n",
      "Document Influenza A virus A/Michigan/45/2015 X-275 (H1N1) antigen (formaldehyde inactivated) - similarity: 0.29093\n",
      "Document (2s)-2,8-Diaminooctanoic Acid - similarity: 0.29093\n",
      "Document [{2-bromo-4-[(2R)-3-oxo-2,3-diphenylpropyl]phenyl}(difluoro)methyl]phosphonic acid - similarity: 0.29089\n",
      "Document Vinylsulfonic acid - similarity: 0.29069\n",
      "Document ORG-31710 - similarity: 0.29066\n",
      "Document Eticlopride - similarity: 0.29060\n",
      "Document Methoxy polyethylene glycol-epoetin beta - similarity: 0.29045\n",
      "Document N-[1H-INDOL-3-YL-ACETYL]VALINE ACID - similarity: 0.29035\n",
      "Document (2S,3S)-3-(4-fluorophenyl)-2,3-dihydroxypropanoic acid - similarity: 0.29035\n",
      "Document Phenol - similarity: 0.29028\n",
      "Document Polydextrose - similarity: 0.29027\n",
      "Document rGLP-1 - similarity: 0.29012\n",
      "Document 3-Chloroalaninate - similarity: 0.28994\n",
      "Document Cyclohexylformamide - similarity: 0.28957\n",
      "Document 5-Nitro-6-ribityl-amino-2,4(1H,3H)-pyrimidinedione - similarity: 0.28935\n",
      "Document PF-05180999 - similarity: 0.28915\n",
      "Document Cumin - similarity: 0.28902\n",
      "Document 20-hydroxycholesterol - similarity: 0.28889\n",
      "Document Indole Naphthyridinone - similarity: 0.28867\n",
      "Document Phoma exigua var. exigua - similarity: 0.28863\n",
      "Document 4-Iodopyrazole - similarity: 0.28845\n",
      "Document Methylinositol - similarity: 0.28836\n",
      "Document 4-[Hydroxy-[Methyl-Phosphinoyl]]-3-Oxo-Butanoic Acid - similarity: 0.28823\n",
      "Document 2-oxopropyl-CoM - similarity: 0.28780\n",
      "Document Melevodopa - similarity: 0.28779\n",
      "Document Fentonium - similarity: 0.28776\n",
      "Document Naptumomab Estafenatox - similarity: 0.28776\n",
      "Document Clothiapine - similarity: 0.28773\n",
      "Document Tofersen - similarity: 0.28768\n",
      "Document 5-Heptyl-6-hydroxy-1,3-benzothiazole-4,7-dione - similarity: 0.28766\n",
      "Document SQ-109 - similarity: 0.28762\n",
      "Document CYT997 - similarity: 0.28756\n",
      "Document Prednimustine - similarity: 0.28753\n",
      "Document BMS-906024 - similarity: 0.28753\n",
      "Document Sobrerol - similarity: 0.28751\n",
      "Document Mibenratide - similarity: 0.28750\n",
      "Document R-82913 - similarity: 0.28747\n",
      "Document Chlorine - similarity: 0.28731\n",
      "Document (S)-mandelonitrile - similarity: 0.28724\n",
      "Document Cenobamate - similarity: 0.28721\n",
      "Document Methyl Methylsulfinylmethyl Sulfide - similarity: 0.28697\n",
      "Document 5-Bromoindirubin - similarity: 0.28685\n",
      "Document Acremonium strictum - similarity: 0.28679\n",
      "Document Fusidic acid - similarity: 0.28671\n",
      "Document Enalkiren - similarity: 0.28646\n",
      "Document Gondoic acid - similarity: 0.28644\n",
      "Document Estradiol dienanthate - similarity: 0.28629\n",
      "Document Ceftobiprole medocaril - similarity: 0.28629\n",
      "Document (Hydroxyethyloxy)Tri(Ethyloxy)Octane - similarity: 0.28625\n",
      "Document Eosin - similarity: 0.28617\n",
      "Document Trifluoroalanine - similarity: 0.28612\n",
      "Document Ethoheptazine - similarity: 0.28606\n",
      "Document Talabostat - similarity: 0.28600\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document L-Gluconic Acid - similarity: 0.28595\n",
      "Document N,N-dimethylarginine - similarity: 0.28592\n",
      "Document 2-Fluoroaniline - similarity: 0.28588\n",
      "Document Lifitegrast - similarity: 0.28568\n",
      "Document Phendimetrazine - similarity: 0.28542\n",
      "Document CUDC-907 - similarity: 0.28540\n",
      "Document Carob - similarity: 0.28540\n",
      "Document (2Z)-N-biphenyl-4-yl-2-cyano-3-hydroxybut-2-enamide - similarity: 0.28537\n",
      "Document Bifemelane - similarity: 0.28531\n",
      "Document Epicriptine - similarity: 0.28508\n",
      "Document Pelitinib - similarity: 0.28507\n",
      "Document Ataluren - similarity: 0.28491\n",
      "Document Cauliflower - similarity: 0.28475\n",
      "Document Amdoxovir - similarity: 0.28475\n",
      "Document Censavudine - similarity: 0.28457\n",
      "Document Sinapultide - similarity: 0.28456\n",
      "Document Hexoprenaline - similarity: 0.28453\n",
      "Document Sophora flavescens root - similarity: 0.28451\n",
      "Document Coltuximab ravtansine - similarity: 0.28444\n",
      "Document 5-Mercaptoethanol-2-decenoyl-coenzyme A - similarity: 0.28435\n",
      "Document 6-amino-4-(2-phenylethyl)-1,7-dihydro-8H-imidazo[4,5-g]quinazolin-8-one - similarity: 0.28425\n",
      "Document Clascoterone - similarity: 0.28418\n",
      "Document N-(5-{[(2S)-4-amino-2-(3-chlorophenyl)butanoyl]amino}-1H-indazol-3-yl)benzamide - similarity: 0.28398\n",
      "Document Phosphorylisopropane - similarity: 0.28395\n",
      "Document Reidispongiolide C - similarity: 0.28388\n",
      "Document N-Cyclohexyl-N'-(4-Iodophenyl)Urea - similarity: 0.28387\n",
      "Document Desfesoterodine - similarity: 0.28381\n",
      "Document Xentuzumab - similarity: 0.28381\n",
      "Document Sultopride - similarity: 0.28369\n",
      "Document 5-Oxo-L-Norleucine - similarity: 0.28368\n",
      "Document Immucillin-G - similarity: 0.28355\n",
      "Document 1-Deaza-Adenosine - similarity: 0.28345\n",
      "Document 3'-deoxyguanosine - similarity: 0.28334\n",
      "Document Indium In-111 satumomab pendetide - similarity: 0.28327\n",
      "Document Peginterferon alfacon-1 - similarity: 0.28300\n",
      "Document Vibriolysin - similarity: 0.28297\n",
      "Document (3R,4S)-1-(3,4-DIMETHOXYPHENYL)-3-(3-METHYLPHENYL)PIPERIDIN-4-AMINE - similarity: 0.28290\n",
      "Document 1-((2-HYDROXYETHOXY)METHYL)-5-(PHENYLTHIO)PYRIMIDINE-2,4(1H,3H)-DIONE - similarity: 0.28289\n",
      "Document Cystein-S-Yl Cacodylate - similarity: 0.28280\n",
      "Document Dipyrocetyl - similarity: 0.28276\n",
      "Document 3'-Oxo-Adenosine - similarity: 0.28271\n",
      "Document Ethaselen - similarity: 0.28246\n",
      "Document Indigotindisulfonic acid - similarity: 0.28232\n",
      "Document D-Isovaline - similarity: 0.28206\n",
      "Document Alloin - similarity: 0.28196\n",
      "Document L-thioproline - similarity: 0.28178\n",
      "Document Chlorproethazine - similarity: 0.28171\n",
      "Document 5,10,15,20-Tetrakis(4-Sulpfonatophenyl)-21h,23h-Porphine - similarity: 0.28160\n",
      "Document Phenylboronic acid - similarity: 0.28156\n",
      "Document 1-(2-DEOXY-5-O-PHOSPHONO-BETA-D-ERYTHRO-PENTOFURANOSYL)-4-METHYL-1H-INDOLE - similarity: 0.28154\n",
      "Document 4-Nitrocatechol - similarity: 0.28146\n",
      "Document Technetium Tc-99m albumin colloid - similarity: 0.28141\n",
      "Document Plantago seed - similarity: 0.28134\n",
      "Document SP2456 - similarity: 0.28124\n",
      "Document Metabutethamine - similarity: 0.28105\n",
      "Document Azapropazone - similarity: 0.28087\n",
      "Document GSK-690693 - similarity: 0.28080\n",
      "Document Rumex acetosella pollen - similarity: 0.28071\n",
      "Document Viomycin - similarity: 0.28049\n",
      "Document Olea europaea pollen - similarity: 0.28044\n",
      "Document Nandrolone hydrogen sulfate - similarity: 0.28043\n",
      "Document Adrenalone - similarity: 0.28042\n",
      "Document SPP 301 - similarity: 0.28029\n",
      "Document Sulfapyridine - similarity: 0.28025\n",
      "Document 9-Deazahypoxanthine - similarity: 0.28025\n",
      "Document Glyceraldehyde-3-Phosphate - similarity: 0.28017\n",
      "Document 3-(1h-Indol-3-Yl)-2-[4-(4-Phenyl-Piperidin-1-Yl)-Benzenesulfonylamino]-Propionic Acid - similarity: 0.28016\n",
      "Document Infliximab - similarity: 0.28006\n",
      "Document Influenza A virus A/Singapore/gp2050/2015 (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.28003\n",
      "Document 3-Bromo-7-Nitroindazole - similarity: 0.28002\n",
      "Document GI-181771X - similarity: 0.27997\n",
      "Document 3-chloro-5-[2-chloro-5-(1H-pyrazolo[3,4-b]pyridin-3-ylmethoxy)phenoxy]benzonitrile - similarity: 0.27995\n",
      "Document Xanthine - similarity: 0.27989\n",
      "Document LY-2623091 - similarity: 0.27976\n",
      "Document Triapine - similarity: 0.27973\n",
      "Document Technetium Tc-99m etifenin - similarity: 0.27955\n",
      "Document Ripretinib - similarity: 0.27951\n",
      "Document 2-chloro-4,5-dimethylphenol - similarity: 0.27948\n",
      "Document Clofenamide - similarity: 0.27947\n",
      "Document Rhenium Re-186 sulfide - similarity: 0.27945\n",
      "Document Streptococcus pneumoniae type 14 capsular polysaccharide antigen - similarity: 0.27943\n",
      "Document BIO-11006 - similarity: 0.27939\n",
      "Document Ertiprotafib - similarity: 0.27925\n",
      "Document N-Cyclopentyl-N-Cyclobutylformamide - similarity: 0.27917\n",
      "Document Fasudil - similarity: 0.27910\n",
      "Document [4-(4-PHENYL-PIPERIDIN-1-YL)-BENZENESULFONYLAMINO]-ACETIC ACID - similarity: 0.27906\n",
      "Document GSK159797 - similarity: 0.27905\n",
      "Document N(6)-dimethylallyladenine - similarity: 0.27902\n",
      "Document Iratumumab - similarity: 0.27898\n",
      "Document iota-Carrageenan - similarity: 0.27886\n",
      "Document (2R)-N-HYDROXY-2-[(3S)-3-METHYL-3-{4-[(2-METHYLQUINOLIN-4-YL)METHOXY]PHENYL}-2-OXOPYRROLIDIN-1-YL]PROPANAMIDE - similarity: 0.27866\n",
      "Document 2,4-Difluorobenzyl Alcohol 2,4-Difluoro-1-(Hydroxymethyl)Benzene - similarity: 0.27861\n",
      "Document Bucindolol - similarity: 0.27860\n",
      "Document 3,3'-diindolylmethane - similarity: 0.27843\n",
      "Document Pramiconazole - similarity: 0.27835\n",
      "Document Ergoloid mesylate - similarity: 0.27816\n",
      "Document MC-1 - similarity: 0.27815\n",
      "Document Platelet Activating Factor - similarity: 0.27814\n",
      "Document Astaxanthin - similarity: 0.27811\n",
      "Document Neisseria meningitidis group c capsular polysaccharide diphtheria toxoid conjugate antigen - similarity: 0.27803\n",
      "Document Phthalic Acid - similarity: 0.27779\n",
      "Document N-{[(2S,3S)-3-(ETHOXYCARBONYL)OXIRAN-2-YL]CARBONYL}-L-ISOLEUCINE - similarity: 0.27777\n",
      "Document Glutamine t-butyl ester - similarity: 0.27773\n",
      "Document Tetragalacturonic acid hydroxymethylester - similarity: 0.27772\n",
      "Document 2-Decenoyl N-Acetyl Cysteamine - similarity: 0.27767\n",
      "Document PF-03463275 - similarity: 0.27752\n",
      "Document 3-O-Methylfructose in Linear Form - similarity: 0.27750\n",
      "Document Nitrocefin Acyl-Serine - similarity: 0.27749\n",
      "Document Fumaric Acid - similarity: 0.27747\n",
      "Document Safingol - similarity: 0.27745\n",
      "Document Asimadoline - similarity: 0.27745\n",
      "Document Fluoro-Willardiine - similarity: 0.27740\n",
      "Document N'-(3-CHLORO-4-METHOXY-PHENYL)-N-(3,4,5-TRIMETHOXYPHENYL)-1,3,5-TRIAZINE-2,4-DIAMINE - similarity: 0.27732\n",
      "Document Olinciguat - similarity: 0.27731\n",
      "Document 2-deoxy-2-fluoro-α-D-glucose - similarity: 0.27725\n",
      "Document Soy sterol - similarity: 0.27717\n",
      "Document ICI-164384 - similarity: 0.27717\n",
      "Document 2-Ribofuranosyl-3-Iodo-2,3-Dihydro-1h-Pyrazolo[3,4-D]Pyrimidin-4-Ylamine - similarity: 0.27699\n",
      "Document Crolibulin - similarity: 0.27691\n",
      "Document Abetimus - similarity: 0.27685\n",
      "Document N-(tert-butyl)-4-[5-(pyridin-2-ylamino)quinolin-3-yl]benzenesulfonamide - similarity: 0.27681\n",
      "Document (R)-styrene oxide - similarity: 0.27678\n",
      "Document Vamorolone - similarity: 0.27677\n",
      "Document Ulinastatin - similarity: 0.27670\n",
      "Document Chlorcyclizine - similarity: 0.27659\n",
      "Document 2',4'-DICHLORO-4-HYDROXY-1,1'-BIPHENYL-3-CARBOXYLIC ACID - similarity: 0.27653\n",
      "Document Influenza A virus A/Singapore/GP1908/2015 IVR-180A (H1N1) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.27641\n",
      "Document Propanidid - similarity: 0.27637\n",
      "Document 4-Hydroxytestosterone - similarity: 0.27629\n",
      "Document Teprotumumab - similarity: 0.27621\n",
      "Document X-396 - similarity: 0.27606\n",
      "Document Tipepidine - similarity: 0.27592\n",
      "Document (3R,4S,5S,7R,9E,11R,12R)-12-ETHYL-4-HYDROXY-3,5,7,11-TETRAMETHYLOXACYCLODODEC-9-ENE-2,8-DIONE - similarity: 0.27590\n",
      "Document ZSTK-474 - similarity: 0.27554\n",
      "Document alpha-Lactalbumin - similarity: 0.27548\n",
      "Document 4-[4-(2,5-DIOXO-PYRROLIDIN-1-YL)-PHENYLAMINO]-4-HYDROXY-BUTYRIC ACID - similarity: 0.27541\n",
      "Document N(4)-Adenosyl-N(4)-methyl-2,4-diaminobutanoic acid - similarity: 0.27535\n",
      "Document LY-377604 - similarity: 0.27514\n",
      "Document Butylphosphonate - similarity: 0.27508\n",
      "Document Eupatorium capillifolium pollen - similarity: 0.27494\n",
      "Document Soraphen A - similarity: 0.27493\n",
      "Document Depatuxizumab mafodotin - similarity: 0.27492\n",
      "Document Befunolol - similarity: 0.27492\n",
      "Document Antimony pentasulfide - similarity: 0.27492\n",
      "Document Dipotassium phosphate - similarity: 0.27482\n",
      "Document 4-Fluorobenzylamine - similarity: 0.27481\n",
      "Document 7-Methyl-7,8-dihydroguanosine 5'-(dihydrogen phosphate) - similarity: 0.27446\n",
      "Document Alvircept Sudotox - similarity: 0.27432\n",
      "Document Haemagglutinin-strain A(H1N1) - similarity: 0.27414\n",
      "Document 3,4-Xylenol - similarity: 0.27410\n",
      "Document (5-{3-[5-(PIPERIDIN-1-YLMETHYL)-1H-INDOL-2-YL]-1H-INDAZOL-6-YL}-2H-1,2,3-TRIAZOL-4-YL)METHANOL - similarity: 0.27406\n",
      "Document RO-5045337 - similarity: 0.27403\n",
      "Document 6-(3,4-DIHYDROXYBENZYL)-3-ETHYL-1-(2,4,6-TRICHLOROPHENYL)-1H-PYRAZOLO[3,4-D]PYRIMIDIN-4(5H)-ONE - similarity: 0.27389\n",
      "Document 4-{5-[(Z)-(2-IMINO-4-OXO-1,3-THIAZOLIDIN-5-YLIDENE)METHYL]FURAN-2-YL}BENZOIC ACID - similarity: 0.27383\n",
      "Document (2E)-3-(2-OCT-1-YN-1-YLPHENYL)ACRYLIC ACID - similarity: 0.27380\n",
      "Document Ethyl salicylate - similarity: 0.27373\n",
      "Document Bombykol - similarity: 0.27364\n",
      "Document 4,7-Dimethyl-[1,10]Phenanthroline - similarity: 0.27361\n",
      "Document Olendalizumab - similarity: 0.27360\n",
      "Document AVE9633 - similarity: 0.27357\n",
      "Document Ethyl macadamiate - similarity: 0.27355\n",
      "Document Nebacumab - similarity: 0.27353\n",
      "Document Surotomycin - similarity: 0.27347\n",
      "Document Vidofludimus - similarity: 0.27339\n",
      "Document 2-[4-(2,4-Dichlorophenoxy)Phenoxy]Propanoic Acid - similarity: 0.27331\n",
      "Document Andrographolide - similarity: 0.27324\n",
      "Document Duvoglustat - similarity: 0.27318\n",
      "Document Heme iron polypeptide - similarity: 0.27303\n",
      "Document Acetohexamide - similarity: 0.27297\n",
      "Document (2R,3R,4R,5S)-2-(HYDROXYMETHYL)-1-NONYLPIPERIDINE-3,4,5-TRIOL - similarity: 0.27284\n",
      "Document Ursolic acid - similarity: 0.27282\n",
      "Document Phosporic Acid Mono-[3,4-Dihydroxy-5-(5-Methoxy-Benzoimidazol-1-Yl)-Tetrahydro-Furan-2-Ylmethyl] Ester - similarity: 0.27274\n",
      "Document Heparin Disaccharide Iii-S - similarity: 0.27261\n",
      "Document BCG vaccine - similarity: 0.27254\n",
      "Document Rabies virus inactivated antigen, A - similarity: 0.27240\n",
      "Document Bifunctional epoxide hydrolase 2 - similarity: 0.27237\n",
      "Document 2,5-Diaminopyrimidin-4,6-diol - similarity: 0.27229\n",
      "Document Flufenamic acid - similarity: 0.27220\n",
      "Document Isoferulic acid - similarity: 0.27217\n",
      "Document Benzylparaben - similarity: 0.27197\n",
      "Document Ferric hydroxide - similarity: 0.27179\n",
      "Document Abafungin - similarity: 0.27171\n",
      "Document Rauwolfia serpentina root - similarity: 0.27163\n",
      "Document Influenza A virus A/Uruguay/716/2007(H3N2) antigen (propiolactone inactivated) - similarity: 0.27146\n",
      "Document Pyrithyldione - similarity: 0.27133\n",
      "Document 2,4,6-Triaminoquinazoline - similarity: 0.27120\n",
      "Document Etohexadiol - similarity: 0.27096\n",
      "Document Bentiromide - similarity: 0.27093\n",
      "Document Phosphonothreonine - similarity: 0.27086\n",
      "Document 4-(2-METHOXYPHENYL)-2-OXOBUT-3-ENOIC ACID - similarity: 0.27086\n",
      "Document Chlormezanone - similarity: 0.27050\n",
      "Document Interferon alfa-2a, Recombinant - similarity: 0.27039\n",
      "Document Bevenopran - similarity: 0.27029\n",
      "Document (2S)-1-AMINO-3-[(5-NITROQUINOLIN-8-YL)AMINO]PROPAN-2-OL - similarity: 0.27023\n",
      "Document Anthoxanthum odoratum - similarity: 0.27018\n",
      "Document N-[(1S)-2-AMINO-1-(2,4-DICHLOROBENZYL)ETHYL]-5-[2-(METHYLAMINO)PYRIMIDIN-4-YL]THIOPHENE-2-CARBOXAMIDE - similarity: 0.27016\n",
      "Document Ropidoxuridine - similarity: 0.27001\n",
      "Document Dinor-N(Omega)-Hydroxy-L-Arginine - similarity: 0.26986\n",
      "Document Vitamin A - similarity: 0.26985\n",
      "Document N-(pyridin-3-ylmethyl)aniline - similarity: 0.26984\n",
      "Document JNJ-28330835 - similarity: 0.26981\n",
      "Document Mercury Acetate Ion - similarity: 0.26964\n",
      "Document 2,6-Diamino-8-(2-Dimethylaminoethylsulfanylmethyl)-3h-Quinazolin-4-One - similarity: 0.26944\n",
      "Document Sacituzumab govitecan - similarity: 0.26940\n",
      "Document Dirucotide - similarity: 0.26916\n",
      "Document Erdosteine - similarity: 0.26907\n",
      "Document [(3,7,11-TRIMETHYL-DODECA-2,6,10-TRIENYLOXYCARBAMOYL)-METHYL]-PHOSPHONIC ACID - similarity: 0.26901\n",
      "Document Almasilate - similarity: 0.26884\n",
      "Document Cenisertib - similarity: 0.26884\n",
      "Document Hydroxyethyl ethylcellulose - similarity: 0.26862\n",
      "Document Nisoxetine - similarity: 0.26836\n",
      "Document Nandrolone decanoate - similarity: 0.26836\n",
      "Document Dehydrocholic acid - similarity: 0.26834\n",
      "Document L-histidinol phosphate - similarity: 0.26820\n",
      "Document Zinc picolinate - similarity: 0.26798\n",
      "Document Dactylis glomerata pollen - similarity: 0.26787\n",
      "Document Mebhydrolin - similarity: 0.26784\n",
      "Document Phenylpyruvic acid - similarity: 0.26781\n",
      "Document Alanosine - similarity: 0.26755\n",
      "Document Octocrylene - similarity: 0.26747\n",
      "Document Phenylalanylamide - similarity: 0.26742\n",
      "Document Bordetella pertussis pertactin antigen - similarity: 0.26737\n",
      "Document N-2-Thiophen-2-Yl-Acetamide Boronic Acid - similarity: 0.26730\n",
      "Document alpha-Arbutin - similarity: 0.26708\n",
      "Document N-[2-(4-AMINO-5,8-DIFLUORO-1,2-DIHYDROQUINAZOLIN-2-YL)ETHYL]-3-FURAMIDE - similarity: 0.26692\n",
      "Document 4-(2-AMINOPHENYL)-4-OXOBUTANOIC ACID - similarity: 0.26684\n",
      "Document Custirsen - similarity: 0.26680\n",
      "Document {3-[(5-CHLORO-1,3-BENZOTHIAZOL-2-YL)METHYL]-2,4-DIOXO-3,4-DIHYDROPYRIMIDIN-1(2H)-YL}ACETIC ACID - similarity: 0.26679\n",
      "Document Ambrosia artemisiifolia pollen - similarity: 0.26669\n",
      "Document XL418 - similarity: 0.26663\n",
      "Document Cyclo(his-pro) - similarity: 0.26660\n",
      "Document Reproterol - similarity: 0.26656\n",
      "Document Clove - similarity: 0.26652\n",
      "Document 2-ETHOXYETHYL (2S,3S)-4-((S)-2-BENZYL-3-OXO-4-((3AR,8R,8AS)-2-OXO-3,3A,8,8A-TETRAHYDRO-2H-INDENO[1,2-D]OXAZOL-8-YL)-2,3-DIHYDRO-1H-PYRROL-2-YL)-3-HYDROXY-1-PHENYLBUTAN-2-YLCARBAMATE - similarity: 0.26640\n",
      "Document CYC116 - similarity: 0.26636\n",
      "Document 3-(5-amino-7-hydroxy-(1,2,3)triazolo(4,5-d)pyrimidin-2-yl)benzoic acid - similarity: 0.26608\n",
      "Document ginkgolide-C - similarity: 0.26605\n",
      "Document Mephedrone - similarity: 0.26592\n",
      "Document 3-Methylpyridine - similarity: 0.26585\n",
      "Document N-Methylnaloxonium - similarity: 0.26561\n",
      "Document AR-9281 - similarity: 0.26555\n",
      "Document (4s)-4-{[(2s)-2-Amino-3-Oxopropyl]Sulfanyl}-L-Homoserinate - similarity: 0.26552\n",
      "Document Cetrimonium - similarity: 0.26512\n",
      "Document DEHYDRO-2(S)-AMINO-6-BORONOHEXANOIC ACID - similarity: 0.26507\n",
      "Document Aspartic acid - similarity: 0.26502\n",
      "Document N-Propyl Isocyanide - similarity: 0.26500\n",
      "Document Amithiozone - similarity: 0.26491\n",
      "Document Dihydroetorphine - similarity: 0.26477\n",
      "Document Ferroheme C - similarity: 0.26477\n",
      "Document 6-[(Z)-AMINO(IMINO)METHYL]-N-[3-(CYCLOPENTYLOXY)PHENYL]-2-NAPHTHAMIDE - similarity: 0.26465\n",
      "Document 4-CHLORO-6-(4-{4-[4-(METHYLSULFONYL)BENZYL]PIPERAZIN-1-YL}-1H-PYRAZOL-5-YL)BENZENE-1,3-DIOL - similarity: 0.26459\n",
      "Document 3'-deoxy-3'-aminothymidine monophosphate - similarity: 0.26459\n",
      "Document 2',3'-Dideoxycytidine-5'-Monophosphate - similarity: 0.26436\n",
      "Document Propyl Gallate - similarity: 0.26429\n",
      "Document Interleukin-1 alpha, human recombinant - similarity: 0.26424\n",
      "Document Chenodeoxycholic acid - similarity: 0.26404\n",
      "Document Alpelisib - similarity: 0.26402\n",
      "Document Anlotinib - similarity: 0.26402\n",
      "Document Piritrexim - similarity: 0.26391\n",
      "Document 9CUAB30 - similarity: 0.26376\n",
      "Document Garden snail mucin - similarity: 0.26370\n",
      "Document Phenibut - similarity: 0.26367\n",
      "Document (P-Iodophenylacetylamino)Methylphosphinic Acid - similarity: 0.26367\n",
      "Document 4-(Aminosulfonyl)-N-[(2,4-Difluorophenyl)Methyl]-Benzamide - similarity: 0.26360\n",
      "Document Calcitonin gene-related peptide - similarity: 0.26355\n",
      "Document Oxabolone cipionate - similarity: 0.26340\n",
      "Document Ferricrocin-iron - similarity: 0.26337\n",
      "Document Elacestrant - similarity: 0.26332\n",
      "Document 4-(4-Chlorophenyl)Imidazole - similarity: 0.26298\n",
      "Document Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) antigen (propiolactone inactivated) - similarity: 0.26293\n",
      "Document beta-phenyl-D-phenylalanyl-N-propyl-L-prolinamide - similarity: 0.26287\n",
      "Document Methallenestril - similarity: 0.26264\n",
      "Document Alrestatin - similarity: 0.26260\n",
      "Document Avenanthramides - similarity: 0.26257\n",
      "Document Epigallocatechin gallate - similarity: 0.26247\n",
      "Document Vitamin E - similarity: 0.26239\n",
      "Document Aloglutamol - similarity: 0.26234\n",
      "Document 3-Carboxy-N,N,N-Trimethyl-2-(Octanoyloxy)Propan-1-Aminium - similarity: 0.26234\n",
      "Document SC-74020 - similarity: 0.26230\n",
      "Document Nipecotic acid - similarity: 0.26224\n",
      "Document 3-cyclopropyl-5-phenyl-N-(pyridin-3-ylmethyl)pyrazolo[1,5-a]pyrimidin-7-amine - similarity: 0.26223\n",
      "Document REC-994 - similarity: 0.26220\n",
      "Document gamma-Glutamylcysteine - similarity: 0.26187\n",
      "Document 2,6-Diamino-8-(1h-Imidazol-2-Ylsulfanylmethyl)-3h-Quinazoline-4-One - similarity: 0.26181\n",
      "Document Threonic acid - similarity: 0.26172\n",
      "Document Aspartyl-alanyl-diketopiperazine - similarity: 0.26172\n",
      "Document Rabacfosadine - similarity: 0.26172\n",
      "Document Nitroarginine - similarity: 0.26167\n",
      "Document Dimercaprol - similarity: 0.26155\n",
      "Document Pivhydrazine - similarity: 0.26149\n",
      "Document 2-Amino-3-(Cystein-S-Yl)-Isoxazolidin-5-Yl-Acetic Acid - similarity: 0.26133\n",
      "Document Influenza A virus A/Brisbane/02/2018 IVR-190 (H1N1) antigen (UV, formaldehyde inactivated) - similarity: 0.26128\n",
      "Document Cibinetide - similarity: 0.26120\n",
      "Document Brussels sprout - similarity: 0.26116\n",
      "Document Dobesilic acid - similarity: 0.26112\n",
      "Document C31G - similarity: 0.26108\n",
      "Document CRA_10656 - similarity: 0.26104\n",
      "Document Tolonidine - similarity: 0.26104\n",
      "Document BENZYL N-({(2S,3S)-3-[(PROPYLAMINO)CARBONYL]OXIRAN-2-YL}CARBONYL)-L-ISOLEUCYL-L-PROLINATE - similarity: 0.26101\n",
      "Document Urea - similarity: 0.26089\n",
      "Document (4R)-7,8-dichloro-1',9-dimethyl-1-oxo-1,2,4,9-tetrahydrospiro[beta-carboline-3,4'-piperidine]-4-carbonitrile - similarity: 0.26076\n",
      "Document Depreotide - similarity: 0.26066\n",
      "Document D-Myo-Inositol-1,4-Bisphosphate - similarity: 0.26057\n",
      "Document (1S,3R,6S)-4-oxo-6-{4-[(2-phenylquinolin-4-yl)methoxy]phenyl}-5-azaspiro[2.4]heptane-1-carboxylic acid - similarity: 0.26051\n",
      "Document Alaproclate - similarity: 0.26030\n",
      "Document 1,3,3-trimethyl-2-[(1E,3E)-3-methylpenta-1,3-dien-1-yl]cyclohexene - similarity: 0.26026\n",
      "Document Diamorphine - similarity: 0.26026\n",
      "Document alpha-Naphthoflavone - similarity: 0.26008\n",
      "Document Apixaban - similarity: 0.26006\n",
      "Document 5-AMINO-2-{4-[(4-AMINOPHENYL)SULFANYL]PHENYL}-1H-ISOINDOLE-1,3(2H)-DIONE - similarity: 0.26003\n",
      "Document N-Alpha-(2-Naphthylsulfonyl)-N(3-Amidino-L-Phenylalaninyl)-4-Acetyl-Piperazine - similarity: 0.25997\n",
      "Document 8-azaguanine - similarity: 0.25983\n",
      "Document 2-ethylhexyl benzoate - similarity: 0.25975\n",
      "Document Romifidine - similarity: 0.25975\n",
      "Document AIT-034 - similarity: 0.25969\n",
      "Document 3,5-Diaminophthalhydrazide - similarity: 0.25968\n",
      "Document Carmofur - similarity: 0.25950\n",
      "Document Ailanthus altissima pollen - similarity: 0.25947\n",
      "Document Antilymphocyte immunoglobulin (horse) - similarity: 0.25946\n",
      "Document 5-methylcytidine 5'-monophosphate - similarity: 0.25916\n",
      "Document Grain mill dust - similarity: 0.25913\n",
      "Document Glyoxylic acid - similarity: 0.25873\n",
      "Document Camellia japonica seed oil - similarity: 0.25870\n",
      "Document 1-{[(3R)-3-methyl-4-({4-[(1S)-2,2,2-trifluoro-1-hydroxy-1-methylethyl]phenyl}sulfonyl)piperazin-1-yl]methyl}cyclopropanecarboxamide - similarity: 0.25866\n",
      "Document Cyclofenil - similarity: 0.25858\n",
      "Document Taxodium distichum pollen - similarity: 0.25855\n",
      "Document Benzyl formate - similarity: 0.25849\n",
      "Document 7-{[(CYCLOHEXYLAMINO)CARBONYL]AMINO}HEPTANOIC ACID - similarity: 0.25845\n",
      "Document Lasmiditan - similarity: 0.25816\n",
      "Document 5-(5-CHLORO-2,4-DIHYDROXYPHENYL)-N-ETHYL-4-(4-METHOXYPHENYL)ISOXAZOLE-3-CARBOXAMIDE - similarity: 0.25810\n",
      "Document 6-hydroxydopa quinone - similarity: 0.25793\n",
      "Document N-{2,4-difluoro-3-[(5-pyridin-3-yl-1H-pyrrolo[2,3-b]pyridin-3-yl)carbonyl]phenyl}ethanesulfonamide - similarity: 0.25791\n",
      "Document TAK-901 - similarity: 0.25788\n",
      "Document 2-decyl-5,6-dimethoxy-3-methylcyclohexa-2,5-diene-1,4-dione - similarity: 0.25780\n",
      "Document Guanosine-5'-Monophosphate - similarity: 0.25778\n",
      "Document Mizoribine - similarity: 0.25762\n",
      "Document 5-[3-(2,5-dimethoxyphenyl)prop-1-yn-1-yl]-6-ethylpyrimidine-2,4-diamine - similarity: 0.25757\n",
      "Document Potassium cation - similarity: 0.25756\n",
      "Document 2s,4r-4-Methylglutamate - similarity: 0.25755\n",
      "Document O-Succinylbenzoate - similarity: 0.25745\n",
      "Document Sagopilone - similarity: 0.25715\n",
      "Document Nectarine - similarity: 0.25715\n",
      "Document (2R)-1-[(4-tert-butylphenyl)sulfonyl]-2-methyl-4-(4-nitrophenyl)piperazine - similarity: 0.25713\n",
      "Document Ribose-5-phosphate - similarity: 0.25711\n",
      "Document CAP-1 - similarity: 0.25706\n",
      "Document Cannabivarin - similarity: 0.25697\n",
      "Document Tenapanor - similarity: 0.25694\n",
      "Document PBT-1033 - similarity: 0.25692\n",
      "Document Tedisamil - similarity: 0.25675\n",
      "Document Polistes fuscatus venom protein - similarity: 0.25661\n",
      "Document Talbutal - similarity: 0.25658\n",
      "Document 5-Hydroxy-N-Propargyl-1(R)-Aminoindan - similarity: 0.25644\n",
      "Document Ethyl biscoumacetate - similarity: 0.25639\n",
      "Document 6-{4-[HYDROXY-(4-NITRO-PHENOXY)-PHOSPHORYL]-BUTYRYLAMINO}-HEXANOIC ACID - similarity: 0.25633\n",
      "Document Bluefish - similarity: 0.25629\n",
      "Document Flavin adenine dinucleotide - similarity: 0.25617\n",
      "Document 1,8-Di-Hydroxy-4-Nitro-Anthraquinone - similarity: 0.25606\n",
      "Document H3B-6527 - similarity: 0.25601\n",
      "Document Mipsagargin - similarity: 0.25592\n",
      "Document Ginkgo biloba leaf oil - similarity: 0.25591\n",
      "Document Mebutamate - similarity: 0.25581\n",
      "Document Diphthamide - similarity: 0.25581\n",
      "Document Tioclomarol - similarity: 0.25577\n",
      "Document Dolichovespula maculata venom protein - similarity: 0.25565\n",
      "Document 2-Hydroxyquinoline - similarity: 0.25544\n",
      "Document TAS-117 - similarity: 0.25536\n",
      "Document Vibrio cholerae CVD 103-HgR strain live antigen - similarity: 0.25534\n",
      "Document Emapalumab - similarity: 0.25528\n",
      "Document Cenegermin - similarity: 0.25517\n",
      "Document Varlitinib - similarity: 0.25506\n",
      "Document Nitarsone - similarity: 0.25494\n",
      "Document Etidronic acid - similarity: 0.25492\n",
      "Document HT-0712 - similarity: 0.25470\n",
      "Document Silk sericin - similarity: 0.25453\n",
      "Document Glutamine hydroxamate - similarity: 0.25437\n",
      "Document Dexbrompheniramine - similarity: 0.25435\n",
      "Document Sweet cherry - similarity: 0.25434\n",
      "Document CRA_9785 - similarity: 0.25425\n",
      "Document beta-D-Ribose-5-phosphate - similarity: 0.25409\n",
      "Document Lucerastat - similarity: 0.25408\n",
      "Document Neisseria meningitidis group Y capsular polysaccharide tetanus toxoid conjugate antigen - similarity: 0.25407\n",
      "Document Rosa centifolia flower oil - similarity: 0.25400\n",
      "Document Sinapic acid - similarity: 0.25400\n",
      "Document 1,2-Dimethoxyethane - similarity: 0.25398\n",
      "Document DEBIO-1347 - similarity: 0.25380\n",
      "Document Valrubicin - similarity: 0.25376\n",
      "Document 2-((3',5'-DIMETHYL-4'-HYDROXYPHENYL)AZO)BENZOIC ACID - similarity: 0.25364\n",
      "Document Fluoride ion - similarity: 0.25350\n",
      "Document AZD-1386 - similarity: 0.25348\n",
      "Document VX-659 - similarity: 0.25345\n",
      "Document Amdinocillin - similarity: 0.25334\n",
      "Document Calcitonin porcine - similarity: 0.25312\n",
      "Document Insulin argine - similarity: 0.25312\n",
      "Document Usistapide - similarity: 0.25289\n",
      "Document 3-Chlorophenol - similarity: 0.25285\n",
      "Document Foreskin keratinocyte (neonatal) - similarity: 0.25245\n",
      "Document Bilh 434 - similarity: 0.25239\n",
      "Document 5-methoxy-N,N-dimethyltryptamine - similarity: 0.25234\n",
      "Document Myricetin - similarity: 0.25231\n",
      "Document (2S,4S,5R)-2-ISOBUTYL-5-(2-THIENYL)-1-[4-(TRIFLUOROMETHYL)BENZOYL]PYRROLIDINE-2,4-DICARBOXYLIC ACID - similarity: 0.25205\n",
      "Document 1-methyl-8-(phenylamino)-4,5-dihydro-1H-pyrazolo[4,3-h]quinazoline-3-carboxylic acid - similarity: 0.25194\n",
      "Document S-(2,4-dinitrophenyl)glutathione - similarity: 0.25161\n",
      "Document 4-methoxybenzenesulfinate - similarity: 0.25121\n",
      "Document 6S-5,6,7,8-Tetrahydrobiopterin - similarity: 0.25115\n",
      "Document Colletotrichum coccodes - similarity: 0.25110\n",
      "Document N-{1-[(1-carbamoylcyclopropyl)methyl]piperidin-4-yl}-N-cyclopropyl-4-[(1S)-2,2,2-trifluoro-1-hydroxy-1-methylethyl]benzamide - similarity: 0.25083\n",
      "Document Zk-806450 - similarity: 0.25079\n",
      "Document Guanosine - similarity: 0.25069\n",
      "Document D-Threonine - similarity: 0.25059\n",
      "Document 5-(2-hydroxyethyl)-4-methylthiazole - similarity: 0.25053\n",
      "Document Benoxaprofen - similarity: 0.25033\n",
      "Document Cod, unspecified - similarity: 0.25031\n",
      "Document N-BENZOYL-D-ALANINE - similarity: 0.25018\n",
      "Document 4-[4-(1-Amino-1-Methylethyl)Phenyl]-5-Chloro-N-[4-(2-Morpholin-4-Ylethyl)Phenyl]Pyrimidin-2-Amine - similarity: 0.25008\n",
      "Document 2'-deoxycytidine-2'-deoxyadenosine-3',5'-monophosphate - similarity: 0.25005\n",
      "Document Lobeline - similarity: 0.25003\n",
      "Document 4-NITROPHENYL-(6-S-ALPHA-D-XYLOPYRANOSYL)-BETA-D-GLUCOPYRANOSIDE - similarity: 0.24998\n",
      "Document Albumin iodinated I-131 serum - similarity: 0.24994\n",
      "Document Bis(5-Amidino-Benzimidazolyl)Methane - similarity: 0.24985\n",
      "Document L-BENZYLSUCCINIC ACID - similarity: 0.24968\n",
      "Document 7-(alpha-D-Glucopyranosyloxy)-4-methyl-2H-1-benzopyran-2-one - similarity: 0.24965\n",
      "Document PF-06821497 - similarity: 0.24960\n",
      "Document Fludiazepam - similarity: 0.24930\n",
      "Document Anti-GD2 monoclonal antibody 3F8 - similarity: 0.24925\n",
      "Document 4-Phenylbutylamine - similarity: 0.24919\n",
      "Document S-propylamine-L-cysteine - similarity: 0.24915\n",
      "Document 5-(2-hydroxyethyl)nonane-1,9-diol - similarity: 0.24900\n",
      "Document Chiniofon - similarity: 0.24900\n",
      "Document Coenzyme M - similarity: 0.24895\n",
      "Document 4-PHENOXY-N-(PYRIDIN-2-YLMETHYL)BENZAMIDE - similarity: 0.24891\n",
      "Document 2-Butanol - similarity: 0.24882\n",
      "Document 2-[3-({Methyl[1-(2-Naphthoyl)Piperidin-4-Yl]Amino}Carbonyl)-2-Naphthyl]-1-(1-Naphthyl)-2-Oxoethylphosphonic Acid - similarity: 0.24867\n",
      "Document 5-[Bis-2(Chloro-Ethyl)-Amino]-2,4-Dintro-Benzamide - similarity: 0.24866\n",
      "Document Surufatinib - similarity: 0.24863\n",
      "Document Erdafitinib - similarity: 0.24857\n",
      "Document Pantothenic acid - similarity: 0.24854\n",
      "Document 6-amino-2-[(1-naphthylmethyl)amino]-3,7-dihydro-8H-imidazo[4,5-g]quinazolin-8-one - similarity: 0.24848\n",
      "Document N-[(2S,3S)-4-[2-[(5S)-3a,4,5,6,7,7a-Hexahydro-1,3-benzodioxol-5-yl]ethyl-[3-(1,3-dioxoisoindol-2-yl)propanoyl]amino]-3-hydroxy-1-phenylbutan-2-yl]-3,5-dimethoxy-4-phenylmethoxybenzamide - similarity: 0.24835\n",
      "Document 1,2,3,4-Tetrahydro-Isoquinoline-7-Sulfonic Acid Amide - similarity: 0.24832\n",
      "Document Sulfanitran - similarity: 0.24820\n",
      "Document Phosphoaminophosphonic Acid-Adenylate Ester - similarity: 0.24813\n",
      "Document (2Z,4E)-3-chloro-2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoic acid - similarity: 0.24803\n",
      "Document Dibromothymoquinone - similarity: 0.24794\n",
      "Document (R)-Propylene glycol - similarity: 0.24792\n",
      "Document Valsartan - similarity: 0.24789\n",
      "Document Imidazole salicylate - similarity: 0.24783\n",
      "Document Tasosartan - similarity: 0.24766\n",
      "Document Fibrinogen human - similarity: 0.24764\n",
      "Document Biphenylol - similarity: 0.24757\n",
      "Document Chloramphenicol palmitate - similarity: 0.24756\n",
      "Document Peppermint - similarity: 0.24739\n",
      "Document Feruloyl Coenzyme A - similarity: 0.24725\n",
      "Document (2,2-DIPHOSPHONOETHYL)(DODECYL)DIMETHYLPHOSPHONIUM - similarity: 0.24725\n",
      "Document Dodecyl-Coa - similarity: 0.24722\n",
      "Document Glycovir - similarity: 0.24720\n",
      "Document N-[4-Methyl-3-[[4-(3-Pyridinyl)-2-Pyrimidinyl]Amino]Phenyl]-3-Pyridinecarboxamide - similarity: 0.24719\n",
      "Document Zanapezil - similarity: 0.24707\n",
      "Document 5-CHLORO-N-{(3S)-1-[(1S)-1-METHYL-2-MORPHOLIN-4-YL-2-OXOETHYL]-2-OXOPYRROLIDIN-3-YL}-1-BENZOTHIOPHENE-2-SULFONAMIDE - similarity: 0.24705\n",
      "Document Bolasterone - similarity: 0.24700\n",
      "Document Neisseria meningitidis serogroup B strain NZ98/254 outer membrane vesicle - similarity: 0.24691\n",
      "Document Ribociclib - similarity: 0.24682\n",
      "Document 1-GUANIDINO-4-(N-PHENYLMETHANESULFONYL-L-LEUCYL-L-PROLYLAMINO)BUTANE - similarity: 0.24674\n",
      "Document Chlormerodrin - similarity: 0.24671\n",
      "Document Lanimostim - similarity: 0.24666\n",
      "Document Tocotrienol - similarity: 0.24650\n",
      "Document Auriclosene - similarity: 0.24628\n",
      "Document Phencyclidine - similarity: 0.24627\n",
      "Document L-Rhamnitol - similarity: 0.24626\n",
      "Document 2-Hydroxy-5-({1-[(2-Naphthyloxy)Methyl]-3-Oxoprop-1-Enyl}Amino)Tyrosine - similarity: 0.24603\n",
      "Document 3-(CARBOXYAMIDE(2-CARBOXYAMIDE-2-TERTBUTYLETHYL))PENTAN - similarity: 0.24587\n",
      "Document SNS-314 - similarity: 0.24575\n",
      "Document Cinazepam - similarity: 0.24574\n",
      "Document Lerdelimumab - similarity: 0.24569\n",
      "Document Bazlitoran - similarity: 0.24565\n",
      "Document Fenproporex - similarity: 0.24560\n",
      "Document N-(4-MORPHOLINE)CARBONYL-B-(1-NAPHTHYL)-L-ALANINE-L-LEUCINE BORONIC ACID - similarity: 0.24553\n",
      "Document Neisseria meningitidis serogroup b nhba fusion protein antigen - similarity: 0.24546\n",
      "Document Cobaltous cation - similarity: 0.24542\n",
      "Document (6S)-N-[(2S,3R,6R,7R)-3-(Acetyloxymethyl)-2-carboxy-8-oxo-5-thia-1-azabicyclo[4.2.0]octan-7-yl]-6-[(2-aminoacetyl)amino]-7-hydroxy-7-oxoheptanimidate - similarity: 0.24539\n",
      "Document Silica dimethyl silylate - similarity: 0.24530\n",
      "Document Pelubiprofen - similarity: 0.24526\n",
      "Document Methadyl acetate - similarity: 0.24516\n",
      "Document Chlorphenesin carbamate - similarity: 0.24516\n",
      "Document AN-2718 - similarity: 0.24513\n",
      "Document Epicatechin - similarity: 0.24507\n",
      "Document (R)-(+)9B-(3-METHYL)PHENYL-2,3-DIHYDROTHIAZOLO[2,3-A]ISOINDOL-5(9BH)-ONE - similarity: 0.24500\n",
      "Document 6-{[(CYCLOHEXYLAMINO)CARBONYL]AMINO}HEXANOIC ACID - similarity: 0.24484\n",
      "Document Mirodenafil - similarity: 0.24482\n",
      "Document Birabresib - similarity: 0.24479\n",
      "Document 2-heptyl-4-hydroxyquinoline N-oxide - similarity: 0.24475\n",
      "Document (2R)-2-(4-chlorophenyl)-2-[4-(1H-pyrazol-4-yl)phenyl]ethanamine - similarity: 0.24472\n",
      "Document Pinus rigida pollen - similarity: 0.24452\n",
      "Document P-Hydroxybenzaldehyde - similarity: 0.24451\n",
      "Document Glucosamine 4-Phosphate - similarity: 0.24435\n",
      "Document Vestipitant - similarity: 0.24434\n",
      "Document Endothelin-1 - similarity: 0.24431\n",
      "Document Rhizopus stolonifer - similarity: 0.24427\n",
      "Document methyl L-phenylalaninate - similarity: 0.24416\n",
      "Document Emopamil - similarity: 0.24397\n",
      "Document S-sulphocysteine - similarity: 0.24387\n",
      "Document 3-sulfino-L-alanine - similarity: 0.24377\n",
      "Document Homatropine methylbromide - similarity: 0.24374\n",
      "Document Y-27632 - similarity: 0.24348\n",
      "Document USL-311 - similarity: 0.24334\n",
      "Document Psicose - similarity: 0.24312\n",
      "Document Anatibant - similarity: 0.24306\n",
      "Document (4S)-5-Fluoro-4-hydroxy-3,4-dihydro-2(1H)-pyrimidinone - similarity: 0.24294\n",
      "Document Solriamfetol - similarity: 0.24293\n",
      "Document Cordycepin - similarity: 0.24285\n",
      "Document 9-benzyl-2,3,4,9-tetrahydro-1H-carbazole-8-carboxylic acid - similarity: 0.24281\n",
      "Document DG051 - similarity: 0.24263\n",
      "Document Salvianolic acid A - similarity: 0.24247\n",
      "Document 4-(Cytidine 5'-diphospho)-2-C-methyl-D-erythritol - similarity: 0.24228\n",
      "Document Vilaprisan - similarity: 0.24215\n",
      "Document Opipramol - similarity: 0.24211\n",
      "Document 2-DEOXY-3,4-BIS-O-[3-(4-HYDROXYPHENYL)PROPANOYL]-L-THREO-PENTARIC ACID - similarity: 0.24203\n",
      "Document Hexylphosphonic acid (S)-2-methyl-3-phenylpropyl ester - similarity: 0.24203\n",
      "Document Thio-Maltopentaose - similarity: 0.24201\n",
      "Document Flovagatran - similarity: 0.24195\n",
      "Document 8-cyclopentyl-1,3-dipropylxanthine - similarity: 0.24193\n",
      "Document Esterified estrogens - similarity: 0.24187\n",
      "Document Girentuximab I-124 - similarity: 0.24182\n",
      "Document (1r,4s)-2-Azabornane - similarity: 0.24180\n",
      "Document Lifirafenib - similarity: 0.24166\n",
      "Document 2,4,6-Tribromophenol - similarity: 0.24162\n",
      "Document Pegadricase - similarity: 0.24155\n",
      "Document Resminostat - similarity: 0.24140\n",
      "Document acetylleucyl-leucyl-norleucinal - similarity: 0.24135\n",
      "Document Chymostatin - similarity: 0.24134\n",
      "Document [3-(1-Benzyl-3-Carbamoylmethyl-2-Methyl-1h-Indol-5-Yloxy)-Propyl-]-Phosphonic Acid - similarity: 0.24125\n",
      "Document AFN-1252 - similarity: 0.24112\n",
      "Document Menadiol diphosphate - similarity: 0.24089\n",
      "Document alpha-L-methyl-fucose - similarity: 0.24082\n",
      "Document 5-CYANO-N-(2,5-DIMETHOXYBENZYL)-6-ETHOXYPYRIDINE-2-CARBOXAMIDE - similarity: 0.24075\n",
      "Document 2-Ethoxyethanol - similarity: 0.24071\n",
      "Document Nefopam - similarity: 0.24065\n",
      "Document GW-274150 - similarity: 0.24051\n",
      "Document Sofinicline - similarity: 0.24051\n",
      "Document D-Xylulose - similarity: 0.24050\n",
      "Document LLL-3348 - similarity: 0.24041\n",
      "Document (4-{4-[(TERT-BUTOXYCARBONYL)AMINO]-2,2-BIS(ETHOXYCARBONYL)BUTYL}PHENYL)SULFAMIC ACID - similarity: 0.24040\n",
      "Document 2,4-Diamino-6-[N-(3',5'-Dimethoxybenzyl)-N-Methylamino]Pyrido[2,3-D]Pyrimidine - similarity: 0.24039\n",
      "Document 6-(2,4-DIAMINO-6-ETHYLPYRIMIDIN-5-YL)-4-(3-METHOXYPROPYL)-2,2-DIMETHYL-2H-1,4-BENZOXAZIN-3(4H)-ONE - similarity: 0.24037\n",
      "Document Aleglitazar - similarity: 0.24032\n",
      "Document Eurotium amstelodami - similarity: 0.24014\n",
      "Document 5,10-Dimethylene Tetrahydromethanopterin - similarity: 0.23998\n",
      "Document Hydroxytyrosol - similarity: 0.23995\n",
      "Document Octinoxate - similarity: 0.23989\n",
      "Document Diethyltoluamide - similarity: 0.23982\n",
      "Document Naronapride - similarity: 0.23982\n",
      "Document (S)-DES-ME-AMPA - similarity: 0.23978\n",
      "Document TRC093 - similarity: 0.23963\n",
      "Document AUROVERTIN B - similarity: 0.23958\n",
      "Document Halazepam - similarity: 0.23957\n",
      "Document 4'-Methylene-5,8,10-trideazaaminopterin - similarity: 0.23954\n",
      "Document Undecyl-Beta-D-Maltopyranoside - similarity: 0.23935\n",
      "Document Niludipine - similarity: 0.23932\n",
      "Document Epigallocatechin - similarity: 0.23925\n",
      "Document Verucerfont - similarity: 0.23912\n",
      "Document Diroximel fumarate - similarity: 0.23902\n",
      "Document N-Ethylretinamide - similarity: 0.23897\n",
      "Document Paliroden - similarity: 0.23874\n",
      "Document T487 - similarity: 0.23859\n",
      "Document Pepinemab - similarity: 0.23848\n",
      "Document Daratumumab - similarity: 0.23837\n",
      "Document 4-Piperidino-Piperidine - similarity: 0.23834\n",
      "Document Bedoradrine - similarity: 0.23817\n",
      "Document Asiaticoside - similarity: 0.23810\n",
      "Document Quifenadine - similarity: 0.23786\n",
      "Document AG-24322 - similarity: 0.23766\n",
      "Document 4-[[(2S)-1-[[6-[(1-Amino-1-oxo-3-sulfanylpropan-2-yl)amino]-6-oxohexyl]amino]-3-[4-[difluoro(phosphono)methyl]phenyl]-1-oxopropan-2-yl]amino]-3-[[2-[4-[difluoro(phosphono)methyl]phenyl]acetyl]amino]-4-oxobutanoic acid - similarity: 0.23761\n",
      "Document HY10275 - similarity: 0.23743\n",
      "Document NAV 1800 - similarity: 0.23739\n",
      "Document Bardoxolone - similarity: 0.23721\n",
      "Document CTCE-0214 - similarity: 0.23702\n",
      "Document Xylose - similarity: 0.23700\n",
      "Document Pentasulfide-Sulfur - similarity: 0.23692\n",
      "Document Farletuzumab - similarity: 0.23680\n",
      "Document Kynurenine - similarity: 0.23680\n",
      "Document Acetic acid C-11 - similarity: 0.23675\n",
      "Document 8-hydroxy-2'-deoxyguanosine - similarity: 0.23667\n",
      "Document 2-{[Formyl(Hydroxy)Amino]Methyl}-4-Methylpentanoic Acid - similarity: 0.23664\n",
      "Document Palmitoyl tetrapeptide-7 - similarity: 0.23655\n",
      "Document Cannabidiol - similarity: 0.23646\n",
      "Document Adenosine phosphate - similarity: 0.23614\n",
      "Document AZD-6482 - similarity: 0.23612\n",
      "Document Pentifylline - similarity: 0.23602\n",
      "Document Medronic acid - similarity: 0.23601\n",
      "Document 2,4-Dihydroxy-Trans Cinnamic Acid - similarity: 0.23596\n",
      "Document Mant-Adp - similarity: 0.23593\n",
      "Document 4-(2-HYDROXYPHENYLSULFINYL)-BUTYLPHOSPHONIC ACID - similarity: 0.23580\n",
      "Document (2E,3S)-3-hydroxy-5'-[(4-hydroxypiperidin-1-yl)sulfonyl]-3-methyl-1,3-dihydro-2,3'-biindol-2'(1'H)-one - similarity: 0.23579\n",
      "Document 6-Deoxyerythronolide B - similarity: 0.23565\n",
      "Document Coumermycin A1 - similarity: 0.23565\n",
      "Document D-Gluco-2,5-Anhydro-1-Deoxy-1-Phosphonohexitol-6-Phosphate - similarity: 0.23551\n",
      "Document Edaglitazone - similarity: 0.23545\n",
      "Document Umirolimus - similarity: 0.23544\n",
      "Document Cefsulodin - similarity: 0.23537\n",
      "Document 3-AMINO-3-BENZYL-[4.3.0]BICYCLO-1,6-DIAZANONAN-2-ONE - similarity: 0.23534\n",
      "Document Pardoprunox - similarity: 0.23525\n",
      "Document 4-Nitrophenyl (3S)-3-({N-[(benzyloxy)carbonyl]-L-phenylalanyl}amino)-5-phenyl-1-pentanesulfonate - similarity: 0.23524\n",
      "Document Sibrotuzumab - similarity: 0.23522\n",
      "Document 9-HYDROXY-6-(3-HYDROXYPROPYL)-4-(2-METHOXYPHENYL)PYRROLO[3,4-C]CARBAZOLE-1,3(2H,6H)-DIONE - similarity: 0.23521\n",
      "Document 1-Acetyl-4-(4-{4-[(2-Ethoxyphenyl)Thio]-3-Nitrophenyl}Pyridin-2-Yl)Piperazine - similarity: 0.23520\n",
      "Document 2,2'-Dibenzothiazyl disulfide - similarity: 0.23509\n",
      "Document gamma-Tocopherol - similarity: 0.23504\n",
      "Document Rumex crispus pollen - similarity: 0.23502\n",
      "Document 2-[({2-[(1Z)-3-(DIMETHYLAMINO)PROP-1-ENYL]-4-FLUOROPHENYL}SULFONYL)AMINO]-5,6,7,8-TETRAHYDRONAPHTHALENE-1-CARBOXYLIC ACID - similarity: 0.23500\n",
      "Document Mirostipen - similarity: 0.23429\n",
      "Document Ndelta-(N'-Sulphodiaminophosphinyl)-L-Ornithine - similarity: 0.23427\n",
      "Document 7-ethyl-10-hydroxycamptothecin - similarity: 0.23425\n",
      "Document Yersinia pestis 195/p antigen (formaldehyde inactivated) - similarity: 0.23425\n",
      "Document Edetate calcium disodium anhydrous - similarity: 0.23413\n",
      "Document (2S)-3-(1-{[2-(2-CHLOROPHENYL)-5-METHYL-1,3-OXAZOL-4-YL]METHYL}-1H-INDOL-5-YL)-2-ETHOXYPROPANOIC ACID - similarity: 0.23412\n",
      "Document Eptacog alfa pegol (activated) - similarity: 0.23409\n",
      "Document 4-[(1S,2S,5S)-5-(HYDROXYMETHYL)-6,8,9-TRIMETHYL-3-OXABICYCLO[3.3.1]NON-7-EN-2-YL]PHENOL - similarity: 0.23401\n",
      "Document 4-(2-aminoethyl)-2-cyclohexylphenol - similarity: 0.23376\n",
      "Document 4-(4-propoxy-1H-pyrrolo[2,3-b]pyridin-3-yl)pyrimidin-2-amine - similarity: 0.23338\n",
      "Document BMS-986141 - similarity: 0.23337\n",
      "Document Deptropine - similarity: 0.23321\n",
      "Document (2e)-N-Allyl-4-{[3-(4-Bromophenyl)-5-Fluoro-1-Methyl-1h-Indazol-6-Yl]Oxy}-N-Methyl-2-Buten-1-Amine - similarity: 0.23314\n",
      "Document BIIB015 - similarity: 0.23312\n",
      "Document Methyl [(1E,5R)-5-{3-[(2E,4E)-2,5-dimethyl-2,4-octadienoyl]-2,4-dioxo-3,4-dihydro-2H-pyran-6-yl}hexylidene]carbamate - similarity: 0.23294\n",
      "Document XTL-001 - similarity: 0.23277\n",
      "Document Nimustine - similarity: 0.23263\n",
      "Document Iguratimod - similarity: 0.23260\n",
      "Document European pilchard - similarity: 0.23232\n",
      "Document Adenosine Diphosphate 5-(Beta-Ethyl)-4-Methyl-Thiazole-2-Carboxylic Acid - similarity: 0.23216\n",
      "Document (1,10 Phenanthroline)-(Tri-Carbon Monoxide) Rhenium (I) - similarity: 0.23209\n",
      "Document Influenza B virus B/Iowa/06/2017 antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.23206\n",
      "Document N7-Methyl-Formycin A - similarity: 0.23161\n",
      "Document Phenylalanine-N-Sulfonamide - similarity: 0.23157\n",
      "Document Pexastimogene devacirepvec - similarity: 0.23157\n",
      "Document 5,8-dimethoxy-1,4-dimethylquinolin-2(1H)-one - similarity: 0.23143\n",
      "Document 2-(6-HYDROXY-1,3-BENZOTHIAZOL-2-YL)-1,3-THIAZOL-4(5H)-ONE - similarity: 0.23129\n",
      "Document Sodium ferric gluconate complex - similarity: 0.23121\n",
      "Document CoA-S-trimethylene-acetyl-tryptamine - similarity: 0.23099\n",
      "Document OSI-7904L - similarity: 0.23092\n",
      "Document Phenylferricrocin-iron - similarity: 0.23077\n",
      "Document Human blood group H type 1 trisaccharide - similarity: 0.23068\n",
      "Document Androstanedione - similarity: 0.23067\n",
      "Document N-alpha-Acetyl-3,5-diiodotyrosyl-D-threonine - similarity: 0.23064\n",
      "Document N-acetylmannosamine - similarity: 0.23064\n",
      "Document Pentyl Trihydrogen Diphosphate - similarity: 0.23060\n",
      "Document Ronnel - similarity: 0.23060\n",
      "Document 4-(6-{[(1R)-1-(hydroxymethyl)propyl]amino}imidazo[1,2-b]pyridazin-3-yl)benzoic acid - similarity: 0.23046\n",
      "Document Cis-[4,5-Bis-(4-Chlorophenyl)-2-(2-Isopropoxy-4-Methoxyphenyl)-4,5-Dihyd Roimidazol-1-Yl]-Piperazin-1-Yl-Methanone - similarity: 0.23037\n",
      "Document Methyl nonanoate - similarity: 0.23022\n",
      "Document Cefamandole - similarity: 0.23010\n",
      "Document 2-(Trimethylammonium)Ethyl Thiol - similarity: 0.22992\n",
      "Document PP-121 - similarity: 0.22974\n",
      "Document Tisopurine - similarity: 0.22973\n",
      "Document D-[3-hydroxy-2-methyl-5-phosphonooxymethyl-pyridin-4-ylmethyl]-N,O-cycloserylamide - similarity: 0.22953\n",
      "Document Fluclorolone - similarity: 0.22934\n",
      "Document 1-Deoxy-D-xylulose 5-phosphate - similarity: 0.22923\n",
      "Document Aminoquinuride - similarity: 0.22908\n",
      "Document Propanoic acid - similarity: 0.22903\n",
      "Document GSK-376501 - similarity: 0.22899\n",
      "Document Tagetitoxin - similarity: 0.22884\n",
      "Document Dextrothyroxine - similarity: 0.22880\n",
      "Document 4-({4-[(4-AMINOBUT-2-YNYL)OXY]PHENYL}SULFONYL)-N-HYDROXY-2,2-DIMETHYLTHIOMORPHOLINE-3-CARBOXAMIDE - similarity: 0.22879\n",
      "Document 4-S-Glutathionyl-5-pentyl-tetrahydro-furan-2-ol - similarity: 0.22876\n",
      "Document Tecastemizole - similarity: 0.22872\n",
      "Document 1-Methyl-2-quinolone - similarity: 0.22820\n",
      "Document P-Aminophenyl-Alpha-D-Galactopyranoside - similarity: 0.22820\n",
      "Document [(3R)-3-(Methylcarbamoyl)-2-{[(2-methyl-2-propanyl)oxy]carbonyl}-1,2,3,4-tetrahydro-7-isoquinolinyl]sulfamic acid - similarity: 0.22818\n",
      "Document Martinostat - similarity: 0.22817\n",
      "Document Pantothenyl-Aminoethanol-Acetate Pivalic Acid - similarity: 0.22815\n",
      "Document Olumacostat glasaretil - similarity: 0.22802\n",
      "Document Vintafolide - similarity: 0.22801\n",
      "Document Leptazoline D - similarity: 0.22799\n",
      "Document SOR-C13 - similarity: 0.22792\n",
      "Document 2,5-O,O-BIS-{4',4''-AMIDINOPHENYL}-1,4:3,6-DIANHYDRO-D-SORBITOL - similarity: 0.22769\n",
      "Document Galidesivir - similarity: 0.22762\n",
      "Document 2-Carboxyethylphosphonic Acid - similarity: 0.22755\n",
      "Document Eravacycline - similarity: 0.22749\n",
      "Document Atagabalin - similarity: 0.22748\n",
      "Document Tavolixizumab - similarity: 0.22746\n",
      "Document N-[(1S)-2-[(4-cyano-1-methylpiperidin-4-yl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]morpholine-4-carboxamide - similarity: 0.22738\n",
      "Document Influenza B virus B/Brisbane/60/2008 live (attenuated) antigen - similarity: 0.22738\n",
      "Document p-Aminophenol - similarity: 0.22733\n",
      "Document 5-(4-METHOXYBIPHENYL-3-YL)-1,2,5-THIADIAZOLIDIN-3-ONE 1,1-DIOXIDE - similarity: 0.22731\n",
      "Document D-Myo-Inositol-Hexasulphate - similarity: 0.22729\n",
      "Document (3S)-1-(1,3-BENZODIOXOL-5-YLMETHYL)-3-[4-(1H-IMIDAZOL-1-YL)PHENOXY]PIPERIDINE - similarity: 0.22715\n",
      "Document Coenzyme A persulfide - similarity: 0.22712\n",
      "Document 1,1,1,3,3,3-HEXAFLUORO-2-{4-[(2,2,2-TRIFLUOROETHYL)AMINO]PHENYL}PROPAN-2-OL - similarity: 0.22710\n",
      "Document 2,3-diphenyl-1H-indole-7-carboxylic acid - similarity: 0.22707\n",
      "Document 2'-Deoxyuridine - similarity: 0.22707\n",
      "Document Auranofin - similarity: 0.22700\n",
      "Document ATN-161 - similarity: 0.22687\n",
      "Document HYDROXY[3-(6-METHYLPYRIDIN-2-YL)PROPYL]FORMAMIDE - similarity: 0.22680\n",
      "Document Adenosine-5'-Rp-Alpha-Thio-Triphosphate - similarity: 0.22664\n",
      "Document Influenza B virus B/Utah/9/2014 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.22661\n",
      "Document Ferric oxide - similarity: 0.22659\n",
      "Document 6-(Dihydroxy-Isobutyl)-Thymine - similarity: 0.22655\n",
      "Document Clostebol - similarity: 0.22650\n",
      "Document Alpha-D-Glucose-1-Phosphate-6-Vanadate - similarity: 0.22645\n",
      "Document S-[2-({N-[(2S)-2-hydroxy-3,3-dimethyl-4-(phosphonooxy)butanoyl]-beta-alanyl}amino)ethyl] hexanethioate - similarity: 0.22633\n",
      "Document Influenza A virus A/Victoria/210/2009 X-187 (H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.22630\n",
      "Document Contulakin-G - similarity: 0.22611\n",
      "Document (2s,3r)-1-Amino-2-Methylbutane-2,3-Diol - similarity: 0.22600\n",
      "Document GS-6620 - similarity: 0.22597\n",
      "Document Trodusquemine - similarity: 0.22576\n",
      "Document 2',3'-Dideoxythymidine-5'-Monophosphate - similarity: 0.22573\n",
      "Document 1,N6-Ethenoadenine - similarity: 0.22567\n",
      "Document trans-2-hydroxycinnamic acid - similarity: 0.22565\n",
      "Document Amitifadine - similarity: 0.22562\n",
      "Document Bornyl acetate - similarity: 0.22553\n",
      "Document Gentian violet cation - similarity: 0.22550\n",
      "Document EC-18 - similarity: 0.22512\n",
      "Document CG0070 - similarity: 0.22508\n",
      "Document (2-AMINO-1,3-OXAZOL-5-YL)-(3-BROMOPHENYL)METHANONE - similarity: 0.22507\n",
      "Document 4-(2-HYDROXYPHENYLTHIO)-1-BUTENYLPHOSPHONIC ACID - similarity: 0.22506\n",
      "Document ACP-104 - similarity: 0.22503\n",
      "Document N2-({[(4-Bromophenyl)Methyl]Oxy}Carbonyl)-N1-[(1s)-1-Formylpentyl]-L-Leucinamide - similarity: 0.22489\n",
      "Document Colistin - similarity: 0.22465\n",
      "Document Trolnitrate - similarity: 0.22439\n",
      "Document Influenza A virus A/Brisbane/59/2007(H1N1) hemagglutinin antigen (UV,formaldehyde inactivated) - similarity: 0.22434\n",
      "Document 5R-(3,4-DICHLOROPHENYLMETHYL)-3-(2-THIOPHENESULFONYLAMINO)-4-OXO-2-THIONOTHIAZOLIDINE - similarity: 0.22427\n",
      "Document Dectrekumab - similarity: 0.22416\n",
      "Document Octisalate - similarity: 0.22407\n",
      "Document Methionine Phosphinate - similarity: 0.22406\n",
      "Document Domoic Acid - similarity: 0.22388\n",
      "Document 4-NITRO-BENZYLPHOSPHONOBUTANOYL-GLYCINE - similarity: 0.22377\n",
      "Document FavId - similarity: 0.22374\n",
      "Document Isoetharine - similarity: 0.22353\n",
      "Document 3-Amino-4-{3-[2-(2-Propoxy-Ethoxy)-Ethoxy]-Propylamino}-Cyclobut-3-Ene-1,2-Dione - similarity: 0.22352\n",
      "Document Meclocycline - similarity: 0.22347\n",
      "Document 2-deazo-6-thiophosphate guanosine-5'-monophosphate - similarity: 0.22342\n",
      "Document Iodine (131I) norcholesterol - similarity: 0.22341\n",
      "Document Nepadutant - similarity: 0.22340\n",
      "Document (2R)-N-[(2R)-2-(DIHYDROXYBORYL)-1-L-PROLYLPYRROLIDIN-2-YL]-N-[(5R)-5-(DIHYDROXYBORYL)-1-L-PROLYLPYRROLIDIN-2-YL]-L-PROLINAMIDE - similarity: 0.22335\n",
      "Document Fominoben - similarity: 0.22292\n",
      "Document Sulfachlorpyridazine - similarity: 0.22279\n",
      "Document Citiolone - similarity: 0.22261\n",
      "Document Dotarizine - similarity: 0.22247\n",
      "Document 4-[(CYCLOPROPYLETHYNYL)OXY]-6-FLUORO-3-ISOPROPYLQUINOLIN-2(1H)-ONE - similarity: 0.22245\n",
      "Document (2R)-2-{[4-chloro-6-(ethylamino)-1,3,5-triazin-2-yl]amino}-2-methylbutanenitrile - similarity: 0.22244\n",
      "Document LY-3023414 - similarity: 0.22228\n",
      "Document Gold Au-198 - similarity: 0.22218\n",
      "Document N-hydroxy-4-({4-[4-(trifluoromethyl)phenoxy]phenyl}sulfonyl)tetrahydro-2H-pyran-4-carboxamide - similarity: 0.22214\n",
      "Document Nemorexant - similarity: 0.22199\n",
      "Document 1-(thiophen-2-ylacetyl)-4-(3-thiophen-2-yl-1,2,4-oxadiazol-5-yl)piperidine - similarity: 0.22197\n",
      "Document AZD-4818 - similarity: 0.22193\n",
      "Document (S)-2-(MERCAPTOMETHYL)-5-PHENYLPENTANOIC ACID - similarity: 0.22185\n",
      "Document Flomoxef - similarity: 0.22171\n",
      "Document 3-Isopropylmalic Acid - similarity: 0.22138\n",
      "Document 3,8-DIBROMO-7-HYDROXY-4-METHYL-2H-CHROMEN-2-ONE - similarity: 0.22135\n",
      "Document Telaprevir - similarity: 0.22119\n",
      "Document Khuskia oryzae - similarity: 0.22111\n",
      "Document Etarfolatide - similarity: 0.22111\n",
      "Document Flunoxaprofen - similarity: 0.22068\n",
      "Document 3-fluoro-6-(4-fluorophenyl)-2-hydroxy-6-oxohexa-2,4-dienoic acid - similarity: 0.22066\n",
      "Document 5-Amino 6-Nitro Uracil - similarity: 0.22062\n",
      "Document Dibekacin - similarity: 0.22054\n",
      "Document 6-(2-fluorophenyl)-N-(pyridin-3-ylmethyl)imidazo[1,2-a]pyrazin-8-amine - similarity: 0.22029\n",
      "Document ND7001 - similarity: 0.22027\n",
      "Document Vesencumab - similarity: 0.22003\n",
      "Document 5-(acetylamino)-3,5-dideoxy-3-fluoro-D-erythro-alpha-L-manno-non-2-ulopyranosonic acid - similarity: 0.22001\n",
      "Document 4-Androstenediol - similarity: 0.21993\n",
      "Document 3-(2-Benzothiazolylthio)-1-Propanesulfonic Acid - similarity: 0.21989\n",
      "Document UDP-alpha-D-glucuronic acid - similarity: 0.21985\n",
      "Document Tetrahydropyran - similarity: 0.21971\n",
      "Document Belotecan - similarity: 0.21969\n",
      "Document Ularitide - similarity: 0.21968\n",
      "Document SGN-30 - similarity: 0.21966\n",
      "Document [(5R)-5-(2,3-dibromo-5-ethoxy-4-hydroxybenzyl)-4-oxo-2-thioxo-1,3-thiazolidin-3-yl]acetic acid - similarity: 0.21955\n",
      "Document Moexiprilat - similarity: 0.21950\n",
      "Document 3,4-dimethylaniline - similarity: 0.21949\n",
      "Document (4S)-5-[[(2S)-1-[[(2S)-1-Amino-3-[4-[difluoro(phosphono)methyl]phenyl]-1-oxopropan-2-yl]amino]-3-[4-[difluoro(phosphono)methyl]phenyl]-1-oxopropan-2-yl]amino]-4-benzamido-5-oxopentanoic acid - similarity: 0.21946\n",
      "Document Procalcitonin - similarity: 0.21932\n",
      "Document Golotimod - similarity: 0.21903\n",
      "Document Ebselen - similarity: 0.21897\n",
      "Document MK-0893 - similarity: 0.21894\n",
      "Document Influenza A virus A/Kansas/14/2017 X-327 (H3N2) antigen (propiolactone inactivated) - similarity: 0.21885\n",
      "Document ALKS 27 - similarity: 0.21880\n",
      "Document 4-(6-HYDROXY-1H-INDAZOL-3-YL)BENZENE-1,3-DIOL - similarity: 0.21878\n",
      "Document IPH 2101 - similarity: 0.21872\n",
      "Document Eptotermin Alfa - similarity: 0.21868\n",
      "Document Maclura pomifera pollen - similarity: 0.21867\n",
      "Document Cannabigerol - similarity: 0.21851\n",
      "Document Almitrine - similarity: 0.21839\n",
      "Document Gusacitinib - similarity: 0.21839\n",
      "Document Rogaratinib - similarity: 0.21836\n",
      "Document Pegunigalsidase alfa - similarity: 0.21805\n",
      "Document Higenamine - similarity: 0.21804\n",
      "Document Human rabies virus immune globulin - similarity: 0.21788\n",
      "Document Roxadustat - similarity: 0.21784\n",
      "Document L-Dilinoleoyllecithin - similarity: 0.21780\n",
      "Document (6R)-Folinic acid - similarity: 0.21777\n",
      "Document Olomoucine - similarity: 0.21773\n",
      "Document 8-Aminotheophylline - similarity: 0.21770\n",
      "Document Enviomycin - similarity: 0.21765\n",
      "Document Vonicog Alfa - similarity: 0.21751\n",
      "Document N-[3-(N'-HYDROXYCARBOXAMIDO)-2-(2-METHYLPROPYL)-PROPANOYL]-O-TYROSINE-N-METHYLAMIDE - similarity: 0.21750\n",
      "Document Prednisone acetate - similarity: 0.21734\n",
      "Document (1R)-N,6-DIHYDROXY-7-METHOXY-2-[(4-METHOXYPHENYL)SULFONYL]-1,2,3,4-TETRAHYDROISOQUINOLINE-1-CARBOXAMIDE - similarity: 0.21729\n",
      "Document Pirisudanol - similarity: 0.21718\n",
      "Document O3-Sulfonylgalactose - similarity: 0.21709\n",
      "Document GSK-1521498 - similarity: 0.21709\n",
      "Document 1,4-dithio-beta-D-glucopyranose - similarity: 0.21708\n",
      "Document 2-(6-{[(3-chloro-2-methylphenyl)sulfonyl]amino}pyridin-2-yl)-N,N-diethylacetamide - similarity: 0.21689\n",
      "Document Visnadine - similarity: 0.21681\n",
      "Document N-(2,3-DIFLUORO-BENZYL)-4-SULFAMOYL-BENZAMIDE - similarity: 0.21669\n",
      "Document alpha-L-iduronic acid - similarity: 0.21662\n",
      "Document Inhibitor BEA388 - similarity: 0.21656\n",
      "Document S-{3-[(4-ANILINOQUINAZOLIN-6-YL)AMINO]-3-OXOPROPYL}-L-CYSTEINE - similarity: 0.21656\n",
      "Document 2-({[3-(3,4-dihydroisoquinolin-2(1H)-ylsulfonyl)phenyl]carbonyl}amino)benzoic acid - similarity: 0.21652\n",
      "Document 6-(4-{(1S,2S)-2-AMINO-1-[(DIMETHYLAMINO)CARBONYL]-3-[(3S)-3-FLUOROPYRROLIDIN-1-YL]-3-OXOPROPYL}PHENYL)-1H-[1,2,4]TRIAZOLO[1,5-A]PYRIDIN-4-IUM - similarity: 0.21644\n",
      "Document 7-Keto-8-Aminopelargonic Acid - similarity: 0.21644\n",
      "Document Pegsunercept - similarity: 0.21637\n",
      "Document NPS-2143 - similarity: 0.21629\n",
      "Document Erythrityl tetranitrate - similarity: 0.21626\n",
      "Document Furo[2,3d]Pyrimidine Antifolate - similarity: 0.21610\n",
      "Document [N-(2,4-DIAMINOPTERIDIN-6-YL)-METHYL]-DIBENZ[B,F]AZEPINE - similarity: 0.21607\n",
      "Document Cyclothiazide - similarity: 0.21602\n",
      "Document L-alanyl-N-[(1S,2R)-1-benzyl-2-hydroxypropyl]-L-alaninamide - similarity: 0.21601\n",
      "Document Nirogacestat - similarity: 0.21594\n",
      "Document Ulmus crassifolia pollen - similarity: 0.21591\n",
      "Document Diethylthiambutene - similarity: 0.21578\n",
      "Document Azimilide - similarity: 0.21566\n",
      "Document Proflavine - similarity: 0.21564\n",
      "Document EP-2104R - similarity: 0.21561\n",
      "Document D-phenylalanyl-N-(3-chlorobenzyl)-L-prolinamide - similarity: 0.21547\n",
      "Document Binimetinib - similarity: 0.21547\n",
      "Document Fruquintinib - similarity: 0.21545\n",
      "Document 2,2-bis(4-hydroxy-3-tert-butylphenyl)propane - similarity: 0.21541\n",
      "Document Sanglifehrin A - similarity: 0.21536\n",
      "Document Evatanepag - similarity: 0.21526\n",
      "Document Xanthium strumarium var. canadense pollen - similarity: 0.21509\n",
      "Document Pegdinetanib - similarity: 0.21509\n",
      "Document Picea pungens pollen - similarity: 0.21506\n",
      "Document N-omega-nitro-L-arginine methyl ester - similarity: 0.21505\n",
      "Document GUANOSINE-2',3'-O-ETHYLIDENEPHOSPHONATE - similarity: 0.21505\n",
      "Document Octamethylenediamine - similarity: 0.21499\n",
      "Document 1-Methyl-3-Oxo-1,3-Dihydro-Benzo[C]Isothiazole-5-Sulfonic Acid Amide - similarity: 0.21498\n",
      "Document Ferrous carbonate - similarity: 0.21491\n",
      "Document Enoximone - similarity: 0.21474\n",
      "Document Anifrolumab - similarity: 0.21472\n",
      "Document Quinupramine - similarity: 0.21465\n",
      "Document Avena sativa flowering top - similarity: 0.21459\n",
      "Document N-PYRIDOXYL-2-METHYL-L-GLUTAMIC ACID-5'-MONOPHOSPHATE - similarity: 0.21451\n",
      "Document BPC-157 - similarity: 0.21422\n",
      "Document 4-(2-Aminoethyl)Benzenesulfonyl Fluoride - similarity: 0.21397\n",
      "Document (6S)-2-amino-6-(3'-methoxybiphenyl-3-yl)-3,6-dimethyl-5,6-dihydropyrimidin-4(3H)-one - similarity: 0.21397\n",
      "Document Encenicline - similarity: 0.21397\n",
      "Document Nalfurafine - similarity: 0.21396\n",
      "Document Clofedanol - similarity: 0.21351\n",
      "Document SB-705498 - similarity: 0.21338\n",
      "Document Atractylodes lancea root - similarity: 0.21332\n",
      "Document 4-[(10s,14s,18s)-18-(2-Amino-2-Oxoethyl)-14-(1-Naphthylmethyl)-8,17,20-Trioxo-7,16,19-Triazaspiro[5.14]Icos-11-En-10-Yl]Benzylphosphonic Acid - similarity: 0.21330\n",
      "Document Colestipol - similarity: 0.21324\n",
      "Document N-[(CYCLOHEXYLAMINO)CARBONYL]GLYCINE - similarity: 0.21317\n",
      "Document AMG-131 - similarity: 0.21311\n",
      "Document Huperzine B - similarity: 0.21310\n",
      "Document Aclarubicin - similarity: 0.21306\n",
      "Document Chlorophyll A - similarity: 0.21294\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) antigen (UV, formaldehyde inactivated) - similarity: 0.21253\n",
      "Document SGS518 - similarity: 0.21238\n",
      "Document PHOSPHONIC ACID 2-DODECANOYLAMINO-HEXYL ESTER PROPYL ESTER - similarity: 0.21203\n",
      "Document Rupintrivir - similarity: 0.21196\n",
      "Document 6-Methylpurine - similarity: 0.21194\n",
      "Document Scutellaria lateriflora - similarity: 0.21191\n",
      "Document (2-CARBAMOYLMETHYL-5-PROPYL-OCTAHYDRO-INDOL-7-YL)ACETIC ACID - similarity: 0.21158\n",
      "Document Cefamandole nafate - similarity: 0.21154\n",
      "Document AZD-0424 - similarity: 0.21147\n",
      "Document Carphenazine - similarity: 0.21129\n",
      "Document Relebactam - similarity: 0.21118\n",
      "Document Human Rho(D) immune globulin - similarity: 0.21118\n",
      "Document (5R)-5-[(1E)-BUTA-1,3-DIENYL]-4-HYDROXY-3,5-DIMETHYLTHIOPHEN-2(5H)-ONE - similarity: 0.21109\n",
      "Document Hydroxyethyl cellulose - similarity: 0.21091\n",
      "Document CG-400549 - similarity: 0.21079\n",
      "Document Cat dander extract - similarity: 0.21069\n",
      "Document Helminthosporium solani - similarity: 0.21061\n",
      "Document Tricetin - similarity: 0.21056\n",
      "Document Cepeginterferon alfa-2B - similarity: 0.21051\n",
      "Document 3-(4-fluorophenyl)-5-phenyl-4H-1,2,4-triazole - similarity: 0.21044\n",
      "Document DOTMP HO-166 - similarity: 0.21026\n",
      "Document AZD-9977 - similarity: 0.21025\n",
      "Document 5-[2-(TRIFLUOROMETHOXY)PHENYL]-2-FUROIC ACID - similarity: 0.21024\n",
      "Document VS-4718 - similarity: 0.21009\n",
      "Document Dephospho Coenzyme A - similarity: 0.21009\n",
      "Document G4544 - similarity: 0.21004\n",
      "Document 3-Aminomethyl-Pyridinium-Adenine-Dinucleotide - similarity: 0.21000\n",
      "Document Gabaculine - similarity: 0.20984\n",
      "Document 2,2,2-TRIFLUORO-1-{5-[(3-PHENYL-5,6-DIHYDROIMIDAZO[1,2-A]PYRAZIN-7(8H)-YL)CARBONYL]THIOPHEN-2-YL}ETHANE-1,1-DIOL - similarity: 0.20981\n",
      "Document Tetrahydrogestrinone - similarity: 0.20971\n",
      "Document Mureletecan - similarity: 0.20969\n",
      "Document 4-{2-[(3-Nitrobenzoyl)Amino]Phenoxy}Phthalic Acid - similarity: 0.20944\n",
      "Document 19-Nor-5-androstenedione - similarity: 0.20929\n",
      "Document Alanyl glutamine - similarity: 0.20915\n",
      "Document Koeleria macrantha pollen - similarity: 0.20900\n",
      "Document Phenylmercuric borate - similarity: 0.20889\n",
      "Document Afegostat - similarity: 0.20865\n",
      "Document N-[[2-METHYL-4-HYDROXYCARBAMOYL]BUT-4-YL-N]-BENZYL-P-[PHENYL]-P-[METHYL]PHOSPHINAMID - similarity: 0.20854\n",
      "Document Brilacidin - similarity: 0.20836\n",
      "Document Dofequidar - similarity: 0.20832\n",
      "Document 6-aza uridine 5'-monophosphate - similarity: 0.20830\n",
      "Document Nifurtoinol - similarity: 0.20826\n",
      "Document (2S)-2-({6-[(3-Amino-5-chlorophenyl)amino]-9-isopropyl-9H-purin-2-yl}amino)-3-methyl-1-butanol - similarity: 0.20794\n",
      "Document 1-Benzyl-(R)-Propylamine - similarity: 0.20787\n",
      "Document JNJ-42396302 - similarity: 0.20777\n",
      "Document 4-Hydroxy-L-Threonine-5-Monophosphate - similarity: 0.20765\n",
      "Document Benactyzine - similarity: 0.20749\n",
      "Document Oxytetracycline - similarity: 0.20748\n",
      "Document (S)-blebbistatin - similarity: 0.20727\n",
      "Document Methyl alpha-D-mannoside - similarity: 0.20725\n",
      "Document 6-(Oxalyl-Amino)-1h-Indole-5-Carboxylic Acid - similarity: 0.20702\n",
      "Document Arsthinol - similarity: 0.20694\n",
      "Document Tridolgosir - similarity: 0.20690\n",
      "Document 1-(adamantan-1-yl)-2-(1H-imidazol-1-yl)ethanone - similarity: 0.20686\n",
      "Document Balanol - similarity: 0.20671\n",
      "Document Plasma protein fraction (human) - similarity: 0.20665\n",
      "Document N-6022 - similarity: 0.20629\n",
      "Document Medrogestone - similarity: 0.20617\n",
      "Document Indeglitazar - similarity: 0.20608\n",
      "Document Amedalin - similarity: 0.20594\n",
      "Document 4-{[1-METHYL-2,4-DIOXO-6-(3-PHENYLPROP-1-YN-1-YL)-1,4-DIHYDROQUINAZOLIN-3(2H)-YL]METHYL}BENZOIC ACID - similarity: 0.20586\n",
      "Document XL281 - similarity: 0.20580\n",
      "Document VP-14637 - similarity: 0.20573\n",
      "Document WRR-112 - similarity: 0.20552\n",
      "Document Influenza A virus A/Kansas/14/2017 (H3N2) recombinant hemagglutinin antigen - similarity: 0.20546\n",
      "Document Fucoxanthin - similarity: 0.20544\n",
      "Document CP-271485 - similarity: 0.20535\n",
      "Document 2-Oxy-4-Hydroxy-5-(2-Hydrazinopyridine)Phenylalanine - similarity: 0.20534\n",
      "Document (3-EXO)-3-(10,11-DIHYDRO-5H-DIBENZO[A,D][7]ANNULEN-5-YLOXY)-8,8-DIMETHYL-8-AZONIABICYCLO[3.2.1]OCTANE - similarity: 0.20521\n",
      "Document Brivaracetam - similarity: 0.20497\n",
      "Document RAV12 - similarity: 0.20491\n",
      "Document Mibolerone - similarity: 0.20481\n",
      "Document (2R)-({4-[AMINO(IMINO)METHYL]PHENYL}AMINO){5-ETHOXY-2-FLUORO-3-[(3R)-TETRAHYDROFURAN-3-YLOXY]PHENYL}ACETICACID - similarity: 0.20474\n",
      "Document Dehydrochloromethyltestosterone - similarity: 0.20473\n",
      "Document (E)-[4-(3,5-Difluorophenyl)-3H-pyrrolo[2,3-b]pyridin-3-ylidene](3-methoxyphenyl)methanol - similarity: 0.20463\n",
      "Document Lowbush blueberry - similarity: 0.20446\n",
      "Document AMD-070 - similarity: 0.20431\n",
      "Document Alnus serrulata pollen - similarity: 0.20425\n",
      "Document Nebicapone - similarity: 0.20408\n",
      "Document 4-amino-7,7-dimethyl-7,8-dihydroquinazolin-5(6H)-one - similarity: 0.20395\n",
      "Document HSD-016 - similarity: 0.20393\n",
      "Document RI 624 - similarity: 0.20379\n",
      "Document Sevuparin - similarity: 0.20361\n",
      "Document RU90395 - similarity: 0.20352\n",
      "Document Islatravir - similarity: 0.20338\n",
      "Document 5-(2-PHENYLPYRAZOLO[1,5-A]PYRIDIN-3-YL)-1H-PYRAZOLO[3,4-C]PYRIDAZIN-3-AMINE - similarity: 0.20335\n",
      "Document 9-Deazaguanine - similarity: 0.20303\n",
      "Document Ethinamate - similarity: 0.20298\n",
      "Document Gadoversetamide - similarity: 0.20284\n",
      "Document Ganstigmine - similarity: 0.20280\n",
      "Document alpha-Pinene - similarity: 0.20277\n",
      "Document 5-Chloro-thiophene-2-carboxylic acid ((3S,4S)-1-{[2-fluoro-4-(2-oxo-2H-pyridin-1-yl)-phenylcarbamoyl]-methyl}-4-hydroxy-pyrrolidin-3-yl)-amide - similarity: 0.20276\n",
      "Document Z-Pro-Prolinal - similarity: 0.20252\n",
      "Document Glutaric Acid - similarity: 0.20232\n",
      "Document CRA_1144 - similarity: 0.20232\n",
      "Document Influenza A virus A/Christchurch/16/2010 NIB-74XP (H1N1) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.20230\n",
      "Document P-nitrobiphenyl - similarity: 0.20199\n",
      "Document Oxazepam acetate - similarity: 0.20187\n",
      "Document Obinepitide - similarity: 0.20173\n",
      "Document Lonicera caprifolium flower - similarity: 0.20172\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document 6-[(Z)-Amino(Imino)Methyl]-N-[4-(Aminomethyl)Phenyl]-4-(Pyrimidin-2-Ylamino)-2-Naphthamide - similarity: 0.20162\n",
      "Document Tl-3-093 - similarity: 0.20149\n",
      "Document Albuvirtide - similarity: 0.20141\n",
      "Document (2Z)-3-{[Oxido(oxo)phosphoranyl]oxy}-2-phenylacrylate - similarity: 0.20117\n",
      "Document Methylselenocysteine - similarity: 0.20115\n",
      "Document Varicella Zoster Vaccine (Recombinant) - similarity: 0.20104\n",
      "Document Ulodesine - similarity: 0.20092\n",
      "Document (S)-2-[(R)-3-amino-4-(2-fluorophenyl)butyryl]-1,2,3,4-tetrahydroisoquinoline-3-carboxamide - similarity: 0.20085\n",
      "Document Dihydroxymethoxychalcone - similarity: 0.20073\n",
      "Document 4-(2-Thienyl)Butyric Acid - similarity: 0.20037\n",
      "Document Risedronic acid - similarity: 0.20036\n",
      "Document 1,3-diphenylurea - similarity: 0.20035\n",
      "Document S,S-Propylthiocysteine - similarity: 0.20014\n",
      "Document alpha,alpha-Dibromo-D-camphor - similarity: 0.19993\n",
      "Document Biropepimut-S - similarity: 0.19992\n",
      "Document DAS-181 - similarity: 0.19990\n",
      "Document Epoetin delta - similarity: 0.19981\n",
      "Document Xanthinol - similarity: 0.19962\n",
      "Document Angelica archangelica root - similarity: 0.19957\n",
      "Document PRO-542 - similarity: 0.19953\n",
      "Document 1-[4-(4-chlorophenyl)-1-(7H-pyrrolo[2,3-d]pyrimidin-4-yl)piperidin-4-yl]methanamine - similarity: 0.19950\n",
      "Document Apricot kernel oil - similarity: 0.19942\n",
      "Document Canaline - similarity: 0.19924\n",
      "Document Spiradoline - similarity: 0.19922\n",
      "Document Omapatrilat - similarity: 0.19915\n",
      "Document Aspergillus nidulans - similarity: 0.19913\n",
      "Document Cimlanod - similarity: 0.19899\n",
      "Document (R)-1-(4-(4-(hydroxymethyl)-1,3,2-dioxaborolan-2-yl)benzyl)guanidine - similarity: 0.19897\n",
      "Document Metocurine iodide - similarity: 0.19891\n",
      "Document Syncephalastrum racemosum - similarity: 0.19881\n",
      "Document alpha-D-glucose 6-phosphate - similarity: 0.19876\n",
      "Document Dimazole - similarity: 0.19843\n",
      "Document Hydroxyamphetamine - similarity: 0.19780\n",
      "Document 1-(5-phospho-D-ribosyl)-ATP - similarity: 0.19771\n",
      "Document Genz-10850 - similarity: 0.19770\n",
      "Document (2R,4S,5R,6R)-5-Acetamido-4-amino-6-(diethylcarbamoyl)oxane-2-carboxylic acid - similarity: 0.19767\n",
      "Document beta-D-Ribopyranose - similarity: 0.19767\n",
      "Document 4-((3r,4s,5r)-4-Amino-3,5-Dihydroxy-Hex-1-Ynyl)-5-Fluoro-3-[1-(3-Methoxy-1h-Pyrrol-2-Yl)-Meth-(Z)-Ylidene]-1,3-Dihydro-Indol-2-One - similarity: 0.19763\n",
      "Document Theodrenaline - similarity: 0.19753\n",
      "Document GW-590735 - similarity: 0.19751\n",
      "Document (2S)-2-[(2,1,3-BENZOTHIADIAZOL-4-YLSULFONYL)AMINO]-2-PHENYL-N-PYRIDIN-4-YLACETAMIDE - similarity: 0.19732\n",
      "Document CLX-0921 - similarity: 0.19729\n",
      "Document 3-[(4-AMINO-1-TERT-BUTYL-1H-PYRAZOLO[3,4-D]PYRIMIDIN-3-YL)METHYL]PHENOL - similarity: 0.19720\n",
      "Document (2R,3R,4R,5R)-3,4-Dihydroxy-N,N'-bis[(1S,2R)-2-hydroxy-2,3-dihydro-1H-inden-1-yl]-2,5-bis(2-phenylethyl)hexanediamide - similarity: 0.19707\n",
      "Document Maprotiline - similarity: 0.19648\n",
      "Document Darodipine - similarity: 0.19645\n",
      "Document Aminooxyacetic acid - similarity: 0.19643\n",
      "Document Cebranopadol - similarity: 0.19642\n",
      "Document 1-PHENYL-1H-PYRAZOLE-4-CARBOXYLIC ACID - similarity: 0.19625\n",
      "Document 3-(Mercaptomethylene)Pyridine - similarity: 0.19596\n",
      "Document Xylarohydroxamate - similarity: 0.19595\n",
      "Document Gepefrine - similarity: 0.19573\n",
      "Document Nandrolone phenpropionate - similarity: 0.19568\n",
      "Document N-(2,6-Diflouro-Benzyl)-4-Sulfamoyl-Benzamide - similarity: 0.19560\n",
      "Document (1R)-3-chloro-1-phenylpropan-1-ol - similarity: 0.19552\n",
      "Document Gallamine triethiodide - similarity: 0.19545\n",
      "Document Tetra(Imidazole)Diaquacopper (Ii) - similarity: 0.19539\n",
      "Document (S)-Aspartimide - similarity: 0.19522\n",
      "Document Epicaptopril - similarity: 0.19512\n",
      "Document Dexverapamil - similarity: 0.19499\n",
      "Document Etalocib - similarity: 0.19494\n",
      "Document (S)-2-Amino-3-(1,3,5,7-Pentahydro-2,4-Dioxo-Cyclopenta[E]Pyrimidin-1-Yl) Proionic Acid - similarity: 0.19488\n",
      "Document 9-ACETYL-2,3,4,9-TETRAHYDRO-1H-CARBAZOL-1-ONE - similarity: 0.19471\n",
      "Document AGI-1096 - similarity: 0.19467\n",
      "Document Apricitabine - similarity: 0.19463\n",
      "Document Eldelumab - similarity: 0.19458\n",
      "Document Chromate ion Cr-51 - similarity: 0.19450\n",
      "Document (2S)-2-{3-[({[2-fluoro-4-(trifluoromethyl)phenyl]carbonyl}amino)methyl]-4-methoxybenzyl}butanoic acid - similarity: 0.19446\n",
      "Document N-[2-methyl-5-(methylcarbamoyl)phenyl]-2-{[(1R)-1-methylpropyl]amino}-1,3-thiazole-5-carboxamide - similarity: 0.19444\n",
      "Document Perflubutane - similarity: 0.19442\n",
      "Document 1-Naphthoxyacetic acid - similarity: 0.19437\n",
      "Document Alpha-N-Dichloroacetyl-P-Aminophenylserinol - similarity: 0.19422\n",
      "Document (2S)-2-{[3-(3-aminophenyl)imidazo[1,2-b]pyridazin-6-yl]amino}-3-methylbutan-1-ol - similarity: 0.19414\n",
      "Document Adenosine-5-Diphosphoribose - similarity: 0.19412\n",
      "Document (2R)-3-{[{[(2S)-2,3-DIHYDROXYPROPYL]OXY}(HYDROXY)PHOSPHORYL]OXY}-2-[(9E)-HEXADEC-9-ENOYLOXY]PROPYL (9E)-OCTADEC-9-ENOATE - similarity: 0.19405\n",
      "Document Lypressin - similarity: 0.19397\n",
      "Document 17alpha-methyl-3beta,17beta-dihydroxyandrost-4-ene - similarity: 0.19395\n",
      "Document Hispidulin - similarity: 0.19385\n",
      "Document 5-[(Z)-(5-Chloro-2-oxo-1,2-dihydro-3H-indol-3-ylidene)methyl]-N,2,4-trimethyl-1H-pyrrole-3-carboxamide - similarity: 0.19381\n",
      "Document Antimony - similarity: 0.19376\n",
      "Document N-Acetyl-L-Glutamate - similarity: 0.19376\n",
      "Document 4-[(4-O-SULFAMOYLBENZYL)(4-CYANOPHENYL)AMINO]-4H-[1,2,4]-TRIAZOLE - similarity: 0.19367\n",
      "Document Yellow Fever Vaccine - similarity: 0.19350\n",
      "Document Glumetinib - similarity: 0.19336\n",
      "Document Ocrylate - similarity: 0.19336\n",
      "Document Delta-Amino Valeric Acid - similarity: 0.19319\n",
      "Document Suvratoxumab - similarity: 0.19306\n",
      "Document Uridylyl-2'-5'-phospho-adenosine - similarity: 0.19304\n",
      "Document Otelixizumab - similarity: 0.19295\n",
      "Document Uracil - similarity: 0.19273\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.19270\n",
      "Document Diacetyldeuteroheme - similarity: 0.19265\n",
      "Document Ferulic acid - similarity: 0.19261\n",
      "Document Ribitol - similarity: 0.19256\n",
      "Document Balhimycin - similarity: 0.19217\n",
      "Document 4-Morpholin-4-Yl-Piperidine-1-Carboxylic Acid [1-(3-Benzenesulfonyl-1-Propyl-Allylcarbamoyl)-2-Phenylethyl]-Amide - similarity: 0.19208\n",
      "Document cis-Vaccenic acid - similarity: 0.19204\n",
      "Document JNJ-42165279 - similarity: 0.19202\n",
      "Document alpha-Ketoisovalerate - similarity: 0.19193\n",
      "Document Hydroxypropyl betadex - similarity: 0.19177\n",
      "Document (3S)-N-(3-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE - similarity: 0.19170\n",
      "Document [PHENYLALANINYL-PROLINYL]-[2-(PYRIDIN-4-YLAMINO)-ETHYL]-AMINE - similarity: 0.19144\n",
      "Document 5-chloro-N-{4-[(1R)-1,2-dihydroxyethyl]phenyl}-1H-indole-2-carboxamide - similarity: 0.19123\n",
      "Document Bos taurus dander - similarity: 0.19120\n",
      "Document AVN-944 - similarity: 0.19118\n",
      "Document TAK-593 - similarity: 0.19099\n",
      "Document Nequinate - similarity: 0.19097\n",
      "Document Fexaramine - similarity: 0.19093\n",
      "Document octyl alpha-L-altropyranoside - similarity: 0.19076\n",
      "Document ADL5859 - similarity: 0.19071\n",
      "Document Iobenguane - similarity: 0.19060\n",
      "Document Phenylethane Boronic Acid - similarity: 0.19042\n",
      "Document (3S)-1-(4-acetylphenyl)-5-oxopyrrolidine-3-carboxylic acid - similarity: 0.19032\n",
      "Document D-naphthyl-1-acetamido boronic acid alanine - similarity: 0.19017\n",
      "Document PPI-2458 - similarity: 0.19014\n",
      "Document Furaprofen - similarity: 0.19005\n",
      "Document Fialuridine - similarity: 0.19003\n",
      "Document 2-Oxobutanoic Acid - similarity: 0.18979\n",
      "Document 5-(4-CYANOPHENYL)-3-{[(2-METHYLPHENYL)SULFONYL]AMINO}THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.18977\n",
      "Document 4-iodo-acetamido phenylboronic acid - similarity: 0.18960\n",
      "Document Glycerol 2-phosphate - similarity: 0.18959\n",
      "Document (2E)-3-(3,4-DIHYDROXYPHENYL)-2-IMINOPROPANOIC ACID - similarity: 0.18940\n",
      "Document Zanthoxylum clava-herculis bark - similarity: 0.18940\n",
      "Document PX-478 - similarity: 0.18933\n",
      "Document 7-methyl-guanosine-5'-triphosphate - similarity: 0.18897\n",
      "Document L-1-naphthyl-2-acetamido-ethane boronic acid - similarity: 0.18894\n",
      "Document PD173955 - similarity: 0.18878\n",
      "Document Acetophenone - similarity: 0.18851\n",
      "Document Fenethylline - similarity: 0.18841\n",
      "Document Prasinezumab - similarity: 0.18799\n",
      "Document Piperidolate - similarity: 0.18797\n",
      "Document Triphosphoric acid - similarity: 0.18782\n",
      "Document BMS-181156 - similarity: 0.18780\n",
      "Document 6-BENZYL-1-BENZYLOXYMETHYL-5-ISOPROPYL URACIL - similarity: 0.18762\n",
      "Document Rice bran oil - similarity: 0.18760\n",
      "Document Tocopherol - similarity: 0.18759\n",
      "Document Suxibuzone - similarity: 0.18750\n",
      "Document 9-aminocamptothecin - similarity: 0.18739\n",
      "Document 3-[(1E,7E)-8-(2,6-dioxo-1,2,3,6-tetrahydropyrimidin-4-yl)-3,6-dioxa-2,7-diazaocta-1,7-dien-1-yl]benzoic acid - similarity: 0.18737\n",
      "Document 3-({[(3S)-3,4-dihydroxybutyl]oxy}amino)-1H,2'H-2,3'-biindol-2'-one - similarity: 0.18733\n",
      "Document 3-(3,5-Dibromo-4-Hydroxy-Benzoyl)-2-Ethyl-Benzofuran-6-Sulfonic Acid Dimethylamide - similarity: 0.18731\n",
      "Document Neisseria meningitidis serogroup B recombinant LP2086 A05 protein variant antigen - similarity: 0.18713\n",
      "Document 2-(4-CHLORO-PHENYLAMINO)-NICOTINIC ACID - similarity: 0.18701\n",
      "Document 2-(3-AMINO-2,5,6-TRIMETHOXYPHENYL)ETHYL 5-CHLORO-2,4-DIHYDROXYBENZOATE - similarity: 0.18698\n",
      "Document 6-(3',5'-DIMETHYLBENZYL)-1-ETHOXYMETHYL-5-ISOPROPYLURACIL - similarity: 0.18689\n",
      "Document TC-5231 - similarity: 0.18687\n",
      "Document Clorgiline - similarity: 0.18680\n",
      "Document 4,5-bis(4-methoxyphenyl)-2-thiophen-2-yl-1H-imidazole - similarity: 0.18674\n",
      "Document 3-(Benzyloxy)Pyridin-2-Amine - similarity: 0.18672\n",
      "Document FP0011 - similarity: 0.18669\n",
      "Document 3-({4-[(5-chloro-1,3-benzodioxol-4-yl)amino]pyrimidin-2-yl}amino)benzenesulfonamide - similarity: 0.18646\n",
      "Document Digoxigenin - similarity: 0.18621\n",
      "Document Cinalukast - similarity: 0.18614\n",
      "Document PF-00356231 - similarity: 0.18609\n",
      "Document PI-166 - similarity: 0.18597\n",
      "Document 2-{[N-(2-ACETYL-5-CHLORO-4-FLUOROPHENYL)GLYCYL]AMINO}BENZOIC ACID - similarity: 0.18574\n",
      "Document Modafinil - similarity: 0.18554\n",
      "Document Flurazepam - similarity: 0.18551\n",
      "Document Oleic Acid - similarity: 0.18543\n",
      "Document 2-Hexyloxy-6-Hydroxymethyl-Tetrahydro-Pyran-3,4,5-Triol - similarity: 0.18536\n",
      "Document S-9-(2-hydroxypropyl)adenine - similarity: 0.18532\n",
      "Document Menatetrenone - similarity: 0.18527\n",
      "Document Sisomicin - similarity: 0.18504\n",
      "Document 3,5-Dihydro-5-Methylidene-4h-Imidazol-4-On - similarity: 0.18493\n",
      "Document VIBRIO CHOLERAE OGAWA 50 CLASSICAL BIOTYPE, FORMALIN INACTIVATED - similarity: 0.18483\n",
      "Document 4-Bromo-2,5-dimethoxyphenethylamine - similarity: 0.18480\n",
      "Document 5-ALPHA-PREGNANE-3-BETA-OL-HEMISUCCINATE - similarity: 0.18458\n",
      "Document Apstatin - similarity: 0.18456\n",
      "Document Isatoic anhydride - similarity: 0.18428\n",
      "Document Octamoxin - similarity: 0.18422\n",
      "Document CIS-(1R,2S)-2-AMINO-1,2,3,4-TETRAHYDRONAPHTHALEN-1-OL - similarity: 0.18419\n",
      "Document MK-0873 - similarity: 0.18399\n",
      "Document Xanthohumol - similarity: 0.18393\n",
      "Document Vopratelimab - similarity: 0.18389\n",
      "Document 2'-Monophosphoadenosine-5'-Diphosphoribose - similarity: 0.18383\n",
      "Document PBI-1402 - similarity: 0.18377\n",
      "Document 5-Oxoprolinal - similarity: 0.18371\n",
      "Document Tridihexethyl - similarity: 0.18351\n",
      "Document GSK-945237 - similarity: 0.18340\n",
      "Document Carbenicillin - similarity: 0.18329\n",
      "Document Frangula purshiana bark - similarity: 0.18323\n",
      "Document 1-[(3S)-5-PHENYL-3-THIOPHEN-2-YL-3H-1,4-BENZODIAZEPIN-2-YL]AZETIDIN-3-OL - similarity: 0.18318\n",
      "Document Oxetacaine - similarity: 0.18309\n",
      "Document Thymine - similarity: 0.18304\n",
      "Document N-({(2S)-1-[(3R)-3-amino-4-(3-chlorophenyl)butanoyl]pyrrolidin-2-yl}methyl)-3-(methylsulfonyl)benzamide - similarity: 0.18284\n",
      "Document 1-Deoxy-1-Thio-Heptaethylene Glycol - similarity: 0.18272\n",
      "Document Human papillomavirus type 45 L1 capsid protein antigen - similarity: 0.18261\n",
      "Document Siagoside - similarity: 0.18251\n",
      "Document 7beta-hydroxycholesterol - similarity: 0.18232\n",
      "Document (1S,2S,3R,4S,5S)-2,3,4-TRIHYDROXY-5-(HYDROXYMETHYL)CYCLOHEXYL (1E)-2-PHENYL-N-(SULFOOXY)ETHANIMIDOTHIOATE - similarity: 0.18231\n",
      "Document 2-(Oxalyl-Amino)-4,7-Dihydro-5h-Thieno[2,3-C]Thiopyran-3-Carboxylic Acid - similarity: 0.18226\n",
      "Document humanized SMART Anti-IL-12 Antibody - similarity: 0.18225\n",
      "Document GDC-0077 - similarity: 0.18221\n",
      "Document Iocetamic acid - similarity: 0.18213\n",
      "Document GPI-1485 - similarity: 0.18179\n",
      "Document Tretamine - similarity: 0.18173\n",
      "Document Diclofop-methyl - similarity: 0.18167\n",
      "Document Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.18146\n",
      "Document 7,8-Dichloro-1,2,3,4-tetrahydroisoquinoline - similarity: 0.18123\n",
      "Document Cafedrine - similarity: 0.18121\n",
      "Document Enecadin - similarity: 0.18117\n",
      "Document PROPYL-1-PHOSPHATE - similarity: 0.18108\n",
      "Document 3-Mercuri-4-Aminobenzenesulfonamide - similarity: 0.18104\n",
      "Document 2-ACETAMIDO-2-DEOXY-BETA-D-GLUCOPYRANOSE(BETA1-4)-2-ACETAMIDO-1,6-ANHYDRO-3-O-[(R)-1-CARBOXYETHYL]-2-DEOXY-BETA-D-GLUCOPYRANOSE-L-ALANYL-GAMMA-D-GLUTAMYL-MESO-DIAMINOPIMELYL-D-ALANINE - similarity: 0.18104\n",
      "Document L-cysteic acid - similarity: 0.18101\n",
      "Document Hydroxydione - similarity: 0.18095\n",
      "Document Rimeporide - similarity: 0.18095\n",
      "Document Influenza A virus A/California/7/2009 (H1N1) live (attenuated) antigen - similarity: 0.18095\n",
      "Document Gancotamab - similarity: 0.18086\n",
      "Document 2-[4-(Hydroxy-Methoxy-Methyl)-Benzyl]-7-(4-Hydroxymethyl-Benzyl)-1,1-Dioxo-3,6-Bis-Phenoxymethyl-1lambda6-[1,2,7]Thiadiazepane-4,5-Diol - similarity: 0.18063\n",
      "Document Aluminium nicotinate - similarity: 0.18048\n",
      "Document Efmoroctocog alfa - similarity: 0.18022\n",
      "Document AME-527 - similarity: 0.17984\n",
      "Document Ara-Alpha(1,3)-Xyl - similarity: 0.17946\n",
      "Document Rebaudioside A - similarity: 0.17878\n",
      "Document CAD106 - similarity: 0.17877\n",
      "Document D-Serine - similarity: 0.17869\n",
      "Document Oxaflozane - similarity: 0.17862\n",
      "Document Oxomemazine - similarity: 0.17856\n",
      "Document Dehydroascorbic acid - similarity: 0.17828\n",
      "Document 5-hydroxy-4-(7-methoxy-1,1-dioxido-2H-1,2,4-benzothiadiazin-3-yl)-2-(3-methylbutyl)-6-phenylpyridazin-3(2H)-one - similarity: 0.17824\n",
      "Document Alovudine - similarity: 0.17802\n",
      "Document 1-Methyl-2-Oxy-5,5-Dimethyl Pyrrolidine - similarity: 0.17801\n",
      "Document NV-18 - similarity: 0.17790\n",
      "Document UK-390957 - similarity: 0.17775\n",
      "Document 9-(2-Deoxy-Beta-D-Ribofuranosyl)-6-Methylpurine - similarity: 0.17753\n",
      "Document Apiospora montagnei - similarity: 0.17749\n",
      "Document Oxfenicine - similarity: 0.17745\n",
      "Document Influenza B virus B/Phuket/3073/2013 recombinant hemagglutinin antigen - similarity: 0.17704\n",
      "Document 1,4-Dithiothreitol - similarity: 0.17696\n",
      "Document {4-[(2S)-2-Acetamido-3-({(1S)-1-[3-carbamoyl-4-(cyclohexylmethoxy)phenyl]ethyl}amino)-3-oxopropyl]-2-phosphonophenoxy}acetic acid - similarity: 0.17691\n",
      "Document Neramexane - similarity: 0.17669\n",
      "Document IPI-493 - similarity: 0.17667\n",
      "Document Apratastat - similarity: 0.17650\n",
      "Document Cyclo(prolylglycyl) - similarity: 0.17641\n",
      "Document Creatinine - similarity: 0.17639\n",
      "Document octyl beta-D-galactopyranoside - similarity: 0.17621\n",
      "Document Etafenone - similarity: 0.17605\n",
      "Document 1-[(2R)-2-aminobutanoyl]-N-(4-carbamimidoylbenzyl)-L-prolinamide - similarity: 0.17604\n",
      "Document Mitumomab - similarity: 0.17602\n",
      "Document (3Z)-1-[(6-fluoro-4H-1,3-benzodioxin-8-yl)methyl]-4-phenyl-1H-indole-2,3-dione 3-oxime - similarity: 0.17596\n",
      "Document TNX-901 - similarity: 0.17588\n",
      "Document Solnatide - similarity: 0.17581\n",
      "Document Voruciclib - similarity: 0.17566\n",
      "Document N-[4-(2-Methylimidazo[1,2-a]Pyridin-3-Yl)-2-Pyrimidinyl]Acetamide - similarity: 0.17559\n",
      "Document GW-559090 - similarity: 0.17546\n",
      "Document 1-(2,6-Dichlorophenyl)-5-(2,4-Difluorophenyl)-7-Piperazin-1-Yl-3,4-Dihydroquinazolin-2(1h)-One - similarity: 0.17540\n",
      "Document Dacomitinib - similarity: 0.17528\n",
      "Document (2S)-3-[(R)-[(1S)-1-amino-3-phenylpropyl](hydroxy)phosphoryl]-2-benzylpropanoic acid - similarity: 0.17499\n",
      "Document 2-Hydroxybenzalpyruvate - similarity: 0.17499\n",
      "Document 4-(1-Benzyl-3-Carbamoylmethyl-2-Methyl-1h-Indol-5-Yloxy)-Butyric Acid - similarity: 0.17496\n",
      "Document CP-601927 - similarity: 0.17487\n",
      "Document AL7182 - similarity: 0.17486\n",
      "Document RWJ-56423 - similarity: 0.17486\n",
      "Document N-Acetylmannosaminitol - similarity: 0.17461\n",
      "Document Archexin - similarity: 0.17453\n",
      "Document Carbamoylcholine - similarity: 0.17444\n",
      "Document t-Butylhydroquinone - similarity: 0.17442\n",
      "Document SPP1148 - similarity: 0.17429\n",
      "Document 6-Methylamino-5-Nitroisocytosine - similarity: 0.17422\n",
      "Document 6-CARBAMIMIDOYL-2-[2-HYDROXY-6-(4-HYDROXY-PHENYL)-INDAN-1-YL]-HEXANOIC ACID - similarity: 0.17412\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.17412\n",
      "Document Influenza A virus A/Victoria/210/2009 X-187 (H3N2) hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.17408\n",
      "Document Sulfadimethoxine - similarity: 0.17406\n",
      "Document Testosterone succinate - similarity: 0.17398\n",
      "Document 1-GUANIDINO-4-(N-NITRO-BENZOYLAMINO-L-LEUCYL-L-PROLYLAMINO)BUTANE - similarity: 0.17396\n",
      "Document (4S)-5-[[(1S)-1-Carboxy-2-cyclohexylethyl]amino]-4-[[(2S)-2-[[(3R)-4-cyclohexyl-2-oxo-3-(phenylmethoxycarbonylamino)butanoyl]amino]-5-(diaminomethylideneamino)pentanoyl]amino]-5-oxopentanoic acid - similarity: 0.17393\n",
      "Document [4-amino-2-(tert-butylamino)-1,3-thiazol-5-yl](phenyl)methanone - similarity: 0.17358\n",
      "Document 4-{4-[2-(1A,7A-DIMETHYL-4-OXY-OCTAHYDRO-1-OXA-4-AZA-CYCLOPROPA[A]NAPHTHALEN-4-YL) -ACETYLAMINO]-PHENYLCARBAMOYL}-BUTYRIC ACID - similarity: 0.17351\n",
      "Document 2,6-dibromo-4-phenoxyphenol - similarity: 0.17315\n",
      "Document Peptide YY (3-36) - similarity: 0.17311\n",
      "Document PSI-352938 - similarity: 0.17308\n",
      "Document Phosphomethylphosphonic acid guanylate ester - similarity: 0.17302\n",
      "Document Refanezumab - similarity: 0.17297\n",
      "Document Dipipanone - similarity: 0.17287\n",
      "Document Thiosulfuric acid - similarity: 0.17284\n",
      "Document (5-phenyl-7-(pyridin-3-ylmethylamino)pyrazolo[1,5-a]pyrimidin-3-yl)methanol - similarity: 0.17252\n",
      "Document AZD-8055 - similarity: 0.17245\n",
      "Document HM-43239 - similarity: 0.17243\n",
      "Document Deoxyinosine - similarity: 0.17242\n",
      "Document (2S,3S)-2-azanyl-3-methyl-pentanedioic acid - similarity: 0.17236\n",
      "Document Tamarix gallica pollen - similarity: 0.17230\n",
      "Document Penicillin G Acyl-Serine - similarity: 0.17222\n",
      "Document 5-METHYL-3-(9-OXO-1,8-DIAZA-TRICYCLO[10.6.1.013,18]NONADECA-12(19),13,15,17-TETRAEN-10-YLCARBAMOYL)-HEXANOIC ACID - similarity: 0.17220\n",
      "Document N-(5'-Phosphopyridoxyl)-D-Alanine - similarity: 0.17214\n",
      "Document Bromelains - similarity: 0.17187\n",
      "Document AT-7519 - similarity: 0.17182\n",
      "Document GLPG-0259 - similarity: 0.17171\n",
      "Document 2-[(4-ETHYNYL-2-FLUOROPHENYL)AMINO]-3,4-DIFLUORO-N-(2-HYDROXYETHOXY)BENZAMIDE - similarity: 0.17168\n",
      "Document PF-232798 - similarity: 0.17161\n",
      "Document Elgemtumab - similarity: 0.17150\n",
      "Document 6-Chloro-2-(2-Hydroxy-Biphenyl-3-Yl)-1h-Indole-5-Carboxamidine - similarity: 0.17141\n",
      "Document Artemin - similarity: 0.17138\n",
      "Document 5-methyl-2'-deoxypseudouridine - similarity: 0.17136\n",
      "Document 6-Hydro-1-Methyladenosine-5'-Monophosphate - similarity: 0.17117\n",
      "Document Krascheninnikovia lanata pollen - similarity: 0.17095\n",
      "Document D-Tyrosine - similarity: 0.17088\n",
      "Document Phenyl salicylate - similarity: 0.17085\n",
      "Document 3-[(3-Nitrophenyl)sulfamoyl]-2-thiophenecarboxylic acid - similarity: 0.17081\n",
      "Document 2-Deoxy-Glucose-6-Phosphate - similarity: 0.17073\n",
      "Document 3-Hydroxypyruvic Acid - similarity: 0.17060\n",
      "Document Chromium gluconate - similarity: 0.17055\n",
      "Document N-(4-{[amino(imino)methyl]amino}butyl)-2,4'-bi-1,3-thiazole-4-carboxamide - similarity: 0.17053\n",
      "Document Lesopitron - similarity: 0.17047\n",
      "Document 3,5-Dimethyl-1h-Pyrazole-4-Carboxylic Acid Ethyl Ester - similarity: 0.17043\n",
      "Document Foxy-5 - similarity: 0.17032\n",
      "Document S-4-Nitrobutyryl-CoA - similarity: 0.17024\n",
      "Document Styphnolobium japonicum flower bud - similarity: 0.17021\n",
      "Document (2,6-Dimethylphenoxy)acetic acid - similarity: 0.17020\n",
      "Document 4-HYDROXY-5-IODO-3-NITROPHENYLACETYL-EPSILON-AMINOCAPROIC ACID ANION - similarity: 0.17019\n",
      "Document Ethynodiol diacetate - similarity: 0.17006\n",
      "Document p-Iodophenol - similarity: 0.17002\n",
      "Document ALT-2074 - similarity: 0.16991\n",
      "Document 4-Carboxy-4-Aminobutanal - similarity: 0.16987\n",
      "Document Protoporphyrin Ix Containing Zn - similarity: 0.16977\n",
      "Document 5-(4-CHLORO-5-PHENYL-3-THIENYL)-1,2,5-THIADIAZOLIDIN-3-ONE 1,1-DIOXIDE - similarity: 0.16960\n",
      "Document Cephalosporium roseum - similarity: 0.16951\n",
      "Document Callistemon citrinus pollen - similarity: 0.16947\n",
      "Document 1-Azepan-1-Yl-2-Phenyl-2-(4-Thioxo-1,4-Dihydro-Pyrazolo[3,4-D]Pyrimidin-5-Yl)Ethanone Adduct - similarity: 0.16937\n",
      "Document Octyl triazone - similarity: 0.16926\n",
      "Document CG-200745 - similarity: 0.16924\n",
      "Document Hydroxystilbamidine - similarity: 0.16910\n",
      "Document Silver citrate - similarity: 0.16902\n",
      "Document Calcium magnesium potassium carbonate chloride hydroxide - similarity: 0.16898\n",
      "Document (R)-N-methyl-N-2-propynyl-1-indanamine - similarity: 0.16890\n",
      "Document Carboxyethyllumazine - similarity: 0.16886\n",
      "Document 4-[(METHYLSULFONYL)AMINO]BENZOIC ACID - similarity: 0.16873\n",
      "Document L-naphthyl-1-acetamido boronic acid alanine - similarity: 0.16858\n",
      "Document Rovatirelin - similarity: 0.16857\n",
      "Document METHYL 4-{[({[(2R,5S)-5-{[(2S)-2-(AMINOMETHYL)PYRROLIDIN-1-YL]CARBONYL}PYRROLIDIN-2-YL]METHYL}AMINO)CARBONYL]AMINO}BENZOATE - similarity: 0.16851\n",
      "Document 2-Methylbutanoic Acid - similarity: 0.16842\n",
      "Document 7-Deaza-7-Aminomethyl-Guanine - similarity: 0.16828\n",
      "Document mu4-sulfido-quadro-tetracopper - similarity: 0.16827\n",
      "Document N~2~-1,3-BENZOXAZOL-2-YL-3-CYCLOHEXYL-N-{2-[(4-METHOXYPHENYL)AMINO]ETHYL}-L-ALANINAMIDE - similarity: 0.16772\n",
      "Document Ciluprevir - similarity: 0.16763\n",
      "Document N-(5-Cyclopropyl-1h-Pyrazol-3-Yl)Benzamide - similarity: 0.16752\n",
      "Document PF-06260414 - similarity: 0.16739\n",
      "Document Malonate Ion - similarity: 0.16710\n",
      "Document 2-(4-Chlorophenyl)-5-Quinoxalinecarboxamide - similarity: 0.16697\n",
      "Document Sulfaguanidine - similarity: 0.16689\n",
      "Document 12-(2-hydroxyethyl)-2-(1-methylethoxy)-13,14-dihydronaphtho[2,1-a]pyrrolo[3,4-c]carbazol-5(12H)-one - similarity: 0.16677\n",
      "Document NXN-188 - similarity: 0.16677\n",
      "Document N-Propargyl-1(S)-Aminoindan - similarity: 0.16662\n",
      "Document Amaranthus retroflexus pollen - similarity: 0.16659\n",
      "Document Diampromide - similarity: 0.16656\n",
      "Document Dethiobiotin - similarity: 0.16651\n",
      "Document SB-409513 - similarity: 0.16639\n",
      "Document Iodo-Willardiine - similarity: 0.16638\n",
      "Document Lidorestat - similarity: 0.16637\n",
      "Document Bempedoic acid - similarity: 0.16634\n",
      "Document 5-N-Allyl-arginine - similarity: 0.16608\n",
      "Document Florbetapir (18F) - similarity: 0.16604\n",
      "Document 6-Hydroxypropylthymine - similarity: 0.16603\n",
      "Document Rokitamycin - similarity: 0.16603\n",
      "Document (3s,8ar)-3-(4-Hydroxybenzyl)Hexahydropyrrolo[1,2-a]Pyrazine-1,4-Dione - similarity: 0.16590\n",
      "Document 5-[[(2R)-2-cyclopropyl-7,8-dimethoxy-2H-chromen-5-yl]methyl]pyrimidine-2,4-diamine - similarity: 0.16585\n",
      "Document 3'-Fluoro-3'-deoxythymidine 5'-monophosphate - similarity: 0.16584\n",
      "Document P-Anisic Acid - similarity: 0.16534\n",
      "Document Phenglutarimide - similarity: 0.16525\n",
      "Document N-[2-(carbamimidamidooxy)ethyl]-2-{6-cyano-3-[(2,2-difluoro-2-pyridin-2-ylethyl)amino]-2-fluorophenyl}acetamide - similarity: 0.16500\n",
      "Document Dipentamethylenethiuram disulfide - similarity: 0.16491\n",
      "Document Leriglitazone - similarity: 0.16481\n",
      "Document Quaternium-18 - similarity: 0.16470\n",
      "Document 2-{3-[4-(4-Fluorophenyl)-3,6-Dihydro-1(2h)-Pyridinyl]Propyl}-8-Methyl-4(3h)-Quinazolinone - similarity: 0.16460\n",
      "Document N,N'-diphenyl-1,4-phenylenediamine - similarity: 0.16459\n",
      "Document Oglufanide - similarity: 0.16459\n",
      "Document N-(2-hydroxybenzyl)-2,5-dimethoxy-4-cyanophenylethylamine - similarity: 0.16454\n",
      "Document m-Hydroxyhippuric acid - similarity: 0.16452\n",
      "Document D-tartaric acid - similarity: 0.16423\n",
      "Document N-(3-Cyclopropyl(5,6,7,8,9,10-Hexahydro-2-Oxo-2h-Cycloocta[B]Pyran-3-Yl)Methyl)Phenylbenzensulfonamide - similarity: 0.16414\n",
      "Document Influenza B virus B/Brisbane/60/2008 hemagglutinin antigen (propiolactone inactivated) - similarity: 0.16408\n",
      "Document Ethylhexyl methoxycrylene - similarity: 0.16402\n",
      "Document 5-phenylpentanoic acid - similarity: 0.16380\n",
      "Document Technetium Tc-99m tilmanocept - similarity: 0.16376\n",
      "Document 2',3'-dideoxy-3'-fluoro-urididine-5'-diphosphate - similarity: 0.16366\n",
      "Document N-acetyl-alpha-neuraminic acid - similarity: 0.16358\n",
      "Document L-1-(4-chlorophenyl)-2-(acetamido)ethane boronic acid - similarity: 0.16326\n",
      "Document Relacatib - similarity: 0.16304\n",
      "Document (R)-3-hydroxytetradecanoic acid - similarity: 0.16301\n",
      "Document (4-Hydroxymaltosephenyl)Glycine - similarity: 0.16273\n",
      "Document Cysteine-S-acetamide - similarity: 0.16264\n",
      "Document Gibberellin A4 - similarity: 0.16237\n",
      "Document Quaternium-24 - similarity: 0.16208\n",
      "Document [Methyltelluro]Acetate - similarity: 0.16193\n",
      "Document D-1-naphthyl-2-acetamido-ethane boronic acid - similarity: 0.16180\n",
      "Document (4r)-2-Methylpentane-2,4-Diol - similarity: 0.16170\n",
      "Document Melopsittacus undulatus feather - similarity: 0.16156\n",
      "Document Vigabatrin - similarity: 0.16080\n",
      "Document Tofimilast - similarity: 0.16071\n",
      "Document Aspartic Acid-4-Carboxymethyl Ester - similarity: 0.16067\n",
      "Document Uridine-Diphosphate-N-Acetylglucosamine - similarity: 0.16055\n",
      "Document Benzhydroxamic Acid - similarity: 0.16055\n",
      "Document MK-212 - similarity: 0.16014\n",
      "Document Tetrahydrocannabivarin - similarity: 0.15998\n",
      "Document MK-6186 - similarity: 0.15938\n",
      "Document Iroxanadine - similarity: 0.15911\n",
      "Document Furazabol - similarity: 0.15911\n",
      "Document BMS-986104 - similarity: 0.15910\n",
      "Document 3-FORMYL-2-HYDROXY-5-METHYL-HEXANOIC ACID HYDROXYAMIDE - similarity: 0.15907\n",
      "Document (11R)-10-acetyl-11-(2,4-dichlorophenyl)-6-hydroxy-3,3-dimethyl-2,3,4,5,10,11-hexahydro-1H-dibenzo[b,e][1,4]diazepin-1-one - similarity: 0.15907\n",
      "Document Borneol - similarity: 0.15906\n",
      "Document Quercus gambelii pollen - similarity: 0.15904\n",
      "Document C-(1-Azido-Alpha-D-Glucopyranosyl) Formamide - similarity: 0.15898\n",
      "Document Vesnarinone - similarity: 0.15881\n",
      "Document LY-2874455 - similarity: 0.15847\n",
      "Document N-Formylpiperidine - similarity: 0.15841\n",
      "Document gamma-butyrolactone - similarity: 0.15840\n",
      "Document Benzylthiouracil - similarity: 0.15829\n",
      "Document Dithiopyr - similarity: 0.15818\n",
      "Document 3-fluoro-4-[2-hydroxy-2-(5,5,8,8-tetramethyl-5,6,7,8,-tetrahydro-naphtalen-2-yl)-acetylamino]-benzoic acid - similarity: 0.15812\n",
      "Document Cupric Chloride - similarity: 0.15807\n",
      "Document Pantoyl Adenylate - similarity: 0.15802\n",
      "Document Liquidambar styraciflua resin - similarity: 0.15795\n",
      "Document E-6005 - similarity: 0.15786\n",
      "Document 5-CHLORO-THIOPHENE-2-CARBOXYLIC ACID ((3S,4S)-4-FLUORO- 1-{[2-FLUORO-4-(2-OXO-2H-PYRIDIN-1-YL)-PHENYLCARBAMOYL]-METHYL}-PYRROLIDIN-3-YL)-AMIDE - similarity: 0.15779\n",
      "Document Benzylfentanyl - similarity: 0.15764\n",
      "Document 6-ETHYL-5-[9-(3-METHOXYPROPYL)-9H-CARBAZOL-2-YL]PYRIMIDINE-2,4-DIAMINE - similarity: 0.15731\n",
      "Document Aglatimagene besadenovec - similarity: 0.15723\n",
      "Document 5alpha-androstane-3alpha,17beta-diol - similarity: 0.15716\n",
      "Document Flurandrenolide - similarity: 0.15687\n",
      "Document OMS-103HP - similarity: 0.15681\n",
      "Document 2-Oxo-4-Methylpentanoic Acid - similarity: 0.15669\n",
      "Document ANX-510 - similarity: 0.15649\n",
      "Document GlycoPEG-GCSF - similarity: 0.15623\n",
      "Document RU78300 - similarity: 0.15615\n",
      "Document ETHYL (1E)-2-PHENYL-N-(SULFOOXY)ETHANIMIDOTHIOATE - similarity: 0.15614\n",
      "Document 1,4-Deoxy-4-((5-Hydroxymethyl-2,3,4-Trihydroxycyclohex-5-Enyl)Amino)Fructose - similarity: 0.15603\n",
      "Document Hyaluronidase (human recombinant) - similarity: 0.15593\n",
      "Document 3-[4-AMINO-1-(1-METHYLETHYL)-1H-PYRAZOLO[3,4-D]PYRIMIDIN-3-YL]PHENOL - similarity: 0.15580\n",
      "Document Trofosfamide - similarity: 0.15521\n",
      "Document (5Z)-13-CHLORO-14,16-DIHYDROXY-3,4,7,8,9,10-HEXAHYDRO-1H-2-BENZOXACYCLOTETRADECINE-1,11(12H)-DIONE - similarity: 0.15509\n",
      "Document Tenocyclidine - similarity: 0.15506\n",
      "Document Influenza A virus A/Brisbane/02/2018 IVR-190 (H1N1) antigen (formaldehyde inactivated) - similarity: 0.15493\n",
      "Document DLO6001 - similarity: 0.15478\n",
      "Document 19-Nor-5-androstenediol - similarity: 0.15466\n",
      "Document 4-O-Demethylpenclomedine - similarity: 0.15425\n",
      "Document D-Phenylalanine - similarity: 0.15414\n",
      "Document 1-3 Sugar Ring of Pentamannosyl 6-Phosphate - similarity: 0.15412\n",
      "Document 2-[5,6-BIS-(4-METHOXY-PHENYL)-FURO[2,3-D]PYRIMIDIN-4-YLAMINO]-ETHANOL - similarity: 0.15406\n",
      "Document Influenza A virus A/South Australia/55/2014 IVR-175 (H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: 0.15401\n",
      "Document 2-O-Methyl-beta-L-fucopyranose - similarity: 0.15393\n",
      "Document 2-chloro-4-{[(1R,3Z,7S,7aS)-7-hydroxy-1-(trifluoromethyl)tetrahydro-1H-pyrrolo[1,2-c][1,3]oxazol-3-ylidene]amino}-3-methylbenzonitrile - similarity: 0.15384\n",
      "Document Chlorphentermine - similarity: 0.15375\n",
      "Document Arsanilic acid - similarity: 0.15359\n",
      "Document (1R)-2-[(CYANOMETHYL)AMINO]-1-({[2-(DIFLUOROMETHOXY)BENZYL]SULFONYL}METHYL)-2-OXOETHYL MORPHOLINE-4-CARBOXYLATE - similarity: 0.15358\n",
      "Document (2S)-2-[3-(AMINOMETHYL)PHENYL]-3-{(S)-HYDROXY[(1R)-2-METHYL-1-{[(2-PHENYLETHYL)SULFONYL]AMINO}PROPYL]PHOSPHORYL}PROPANOIC ACID - similarity: 0.15353\n",
      "Document 2-Amino-3-(5-Tert-Butyl-3-(Phosphonomethoxy)-4-Isoxazolyl)Propionic Acid - similarity: 0.15337\n",
      "Document 8-Bromo-Adenosine-5'-Monophosphate - similarity: 0.15335\n",
      "Document L-Phospholactate - similarity: 0.15328\n",
      "Document D-Cysteine - similarity: 0.15314\n",
      "Document Alovudine F-18 - similarity: 0.15300\n",
      "Document VGV-1 - similarity: 0.15296\n",
      "Document Microsporum audouinii - similarity: 0.15291\n",
      "Document N-(4-CHLOROPHENYL)-3-(PHOSPHONOOXY)NAPHTHALENE-2-CARBOXAMIDE - similarity: 0.15247\n",
      "Document 5-phenyl-1H-indazol-3-amine - similarity: 0.15243\n",
      "Document (9R,10R)-9-(S-glutathionyl)-10-hydroxy-9,10-dihydrophenanthrene - similarity: 0.15238\n",
      "Document Elafibranor - similarity: 0.15236\n",
      "Document N-ETHYL-N-ISOPROPYL-3-METHYL-5-{[(2S)-2-(PYRIDIN-4-YLAMINO)PROPYL]OXY}BENZAMIDE - similarity: 0.15233\n",
      "Document 5-Phosphoribosyl-1-(Beta-Methylene) Pyrophosphate - similarity: 0.15232\n",
      "Document Quercus macrocarpa pollen - similarity: 0.15200\n",
      "Document 5,7-Dichlorokynurenic acid - similarity: 0.15198\n",
      "Document Dalfopristin - similarity: 0.15146\n",
      "Document Zamicastat - similarity: 0.15128\n",
      "Document 4-[4-(4-Methyl-2-Methylamino-Thiazol-5-Yl)-Pyrimidin-2-Ylamino]-Phenol - similarity: 0.15124\n",
      "Document Bis(5-Amidino-2-Benzimidazolyl)Methanone - similarity: 0.15123\n",
      "Document CP-526423 - similarity: 0.15098\n",
      "Document Rebeccamycin - similarity: 0.15086\n",
      "Document Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.15073\n",
      "Document (1S,2R,5S)-5-[3-(TRIFLUOROMETHYL)-5,6-DIHYDRO[1,2,4]TRIAZOLO[4,3-A]PYRAZIN-7(8H)-YL]-2-(2,4,5-TRIFLUOROPHENYL)CYCLOHEXANAMINE - similarity: 0.15064\n",
      "Document Eprozinol - similarity: 0.15063\n",
      "Document Iocarmic acid - similarity: 0.15039\n",
      "Document Isopropyl beta-D-thiogalactopyranoside - similarity: 0.15030\n",
      "Document (3S)-4-{[4-(BUT-2-YNYLOXY)PHENYL]SULFONYL}-N-HYDROXY-2,2-DIMETHYLTHIOMORPHOLINE-3-CARBOXAMIDE - similarity: 0.15027\n",
      "Document 3,5-Difluoroaniline - similarity: 0.15025\n",
      "Document Isoxicam - similarity: 0.15019\n",
      "Document DA-6886 - similarity: 0.15010\n",
      "Document Tenamfetamine - similarity: 0.15007\n",
      "Document 4-(2-HYDROXYBENZYLAMINO)-N-(3-(4-FLUOROPHENOXY)PHENYL)PIPERIDINE-1-SULFONAMIDE - similarity: 0.14998\n",
      "Document N-Methylmesoporphyrin - similarity: 0.14988\n",
      "Document Ioxaglic acid - similarity: 0.14966\n",
      "Document 3-({3-[(1S,4aS,6S,7S,9S,9aR)-1,6-dimethyl-2-oxodecahydro-6,9-epoxy-4a,7-methanobenzo[7]annulen-1-yl]propanoyl}amino)-2,4-dihydroxybenzoic acid - similarity: 0.14942\n",
      "Document RDEA806 - similarity: 0.14928\n",
      "Document Allolactose - similarity: 0.14895\n",
      "Document (5Z,7E,9)-decatrien-2-one - similarity: 0.14879\n",
      "Document Begelomab - similarity: 0.14865\n",
      "Document (C8-R)-hydantocidin 5'-phosphate - similarity: 0.14861\n",
      "Document Caprospinol - similarity: 0.14853\n",
      "Document 7,8-Dihydroxy-1-Methoxy-3-Methyl-10-Oxo-4,10-Dihydro-1h,3h-Pyrano[4,3-B]Chromene-9-Carboxylic Acid - similarity: 0.14821\n",
      "Document Iodipamide - similarity: 0.14818\n",
      "Document 4-Flourobenzenesulfonamide - similarity: 0.14810\n",
      "Document 4-Amino-1-[(1s,3r,4r,7s)-7-Hydroxy-1-(Hydroxymethyl)-2,5-Dioxabicyclo[2.2.1]Hept-3-Yl]-5-Methylpyrimidin-2(1h)-One - similarity: 0.14806\n",
      "Document Atamestane - similarity: 0.14804\n",
      "Document 2-Fluoro-2'-Deoxyadenosine - similarity: 0.14790\n",
      "Document Bolandiol - similarity: 0.14769\n",
      "Document Cumidine - similarity: 0.14768\n",
      "Document Lutropin alfa - similarity: 0.14764\n",
      "Document Radium Ra-223 cation - similarity: 0.14741\n",
      "Document Sulfamazone - similarity: 0.14692\n",
      "Document 5-(3-(2,6-dichloro-4-(4,5-dihydro-2-oxazolyl)phenoxy)propyl)-3-methyl isoxazole - similarity: 0.14688\n",
      "Document Infigratinib - similarity: 0.14684\n",
      "Document S-Ethyl-N-[4-(Trifluoromethyl)Phenyl]Isothiourea - similarity: 0.14645\n",
      "Document 2'deoxy-Thymidine-5'-Diphospho-Alpha-D-Glucose - similarity: 0.14618\n",
      "Document [2-(3-DIBENZOFURAN-4-YL-PHENYL)-1-HYDROXY-1-PHOSPHONO-ETHYL]-PHOSPHONIC ACID - similarity: 0.14614\n",
      "Document Metrizoic acid - similarity: 0.14613\n",
      "Document 4-(5-phenyl-1H-pyrrolo[2,3-b]pyridin-3-yl)benzoic acid - similarity: 0.14588\n",
      "Document alpha,beta-Methyleneadenosine 5'-triphosphate - similarity: 0.14544\n",
      "Document Sulbenicillin - similarity: 0.14539\n",
      "Document Hexamethylene diamine - similarity: 0.14536\n",
      "Document Diodone - similarity: 0.14522\n",
      "Document Brinase - similarity: 0.14514\n",
      "Document CE-224535 - similarity: 0.14501\n",
      "Document Fasitibant - similarity: 0.14488\n",
      "Document Samarium - similarity: 0.14485\n",
      "Document Lonoctocog alfa - similarity: 0.14474\n",
      "Document Prenalterol - similarity: 0.14462\n",
      "Document GSK-2636771 - similarity: 0.14453\n",
      "Document Nogalaviketone - similarity: 0.14452\n",
      "Document Phosphoglycolohydroxamic Acid - similarity: 0.14447\n",
      "Document NP-G2-044 - similarity: 0.14438\n",
      "Document 4-{[1-Methyl-5-(2-Methyl-Benzoimidazol-1-Ylmethyl)-1h-Benzoimidazol-2-Ylmethyl]-Amino}-Benzamidine - similarity: 0.14434\n",
      "Document 3,6-Anhydro-2-(hydrogen sulfate)-alpha-D-galactopyranose - similarity: 0.14407\n",
      "Document Fraxinus velutina pollen - similarity: 0.14407\n",
      "Document Gimsilumab - similarity: 0.14400\n",
      "Document GLPG-0187 - similarity: 0.14382\n",
      "Document 6,4'-Dihydroxy-3-Methyl-3',5'-Dibromoflavone - similarity: 0.14373\n",
      "Document Cp-Coeleneterazine - similarity: 0.14331\n",
      "Document Influenza A virus A/California/7/2009(H1N1)-like antigen (formaldehyde inactivated) - similarity: 0.14322\n",
      "Document Dazoxiben - similarity: 0.14314\n",
      "Document Streptococcus pneumoniae type 6b capsular polysaccharide antigen - similarity: 0.14296\n",
      "Document [(2-AMINO-ALPHA-METHOXYIMINO-4-THIAZOLYLACETYL)AMINO]METHYLBORONIC ACID - similarity: 0.14278\n",
      "Document 4-AMINO-2-TRIFLUOROMETHYL-5-HYDROXYMETHYLPYRIMIDINE PYROPHOSPHATE - similarity: 0.14276\n",
      "Document 4-[3-(4-chlorophenyl)-2,1-benzisoxazol-5-yl]pyrimidin-2-amine - similarity: 0.14274\n",
      "Document Selenomethionine Selenoxide - similarity: 0.14262\n",
      "Document Inhibitor Bea428 - similarity: 0.14259\n",
      "Document Rilotumumab - similarity: 0.14248\n",
      "Document MK-7145 - similarity: 0.14240\n",
      "Document Futibatinib - similarity: 0.14232\n",
      "Document 3-(7-hydroxy-8-ribityllumazine-6-yl) propionic acid - similarity: 0.14226\n",
      "Document 2-[(7-HYDROXY-NAPHTHALEN-1-YL)-OXALYL-AMINO]-BENZOIC ACID - similarity: 0.14218\n",
      "Document Adenosine 5'-phosphosulfate - similarity: 0.14203\n",
      "Document AGS-004 - similarity: 0.14202\n",
      "Document 4-Iodo-L-phenylalanine - similarity: 0.14198\n",
      "Document (2S)-2-{[4-chloro-6-(ethylamino)-1,3,5-triazin-2-yl]amino}-2-methylbutanenitrile - similarity: 0.14198\n",
      "Document Rebimastat - similarity: 0.14178\n",
      "Document Daxalipram - similarity: 0.14177\n",
      "Document Benzoic acid - similarity: 0.14175\n",
      "Document Bropirimine - similarity: 0.14165\n",
      "Document 2-chloro-5-[(1S)-1-hydroxy-3-oxo-2H-isoindol-1-yl]benzenesulfonamide - similarity: 0.14154\n",
      "Document Dihydroxy(oxo)molybdenum(6+) - similarity: 0.14149\n",
      "Document Trans-6-(2-Phenylcyclopropyl)-Naphthalene-2-Carboxamidine - similarity: 0.14146\n",
      "Document Camphor leaf oil - similarity: 0.14135\n",
      "Document MEDI-6469 - similarity: 0.14098\n",
      "Document N-Acetylproline - similarity: 0.14084\n",
      "Document IDD552 - similarity: 0.14072\n",
      "Document Budipine - similarity: 0.14069\n",
      "Document (13R,15S)-13-METHYL-16-OXA-8,9,12,22,24-PENTAAZAHEXACYCLO[15.6.2.16,9.1,12,15.0,2,7.0,21,25]HEPTACOSA-1(24),2,4,6,17(25),18,20-HEPTAENE-23,26-DIONE - similarity: 0.14057\n",
      "Document Influenza A virus A/Texas/50/2012 X-223A (H3N2) hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.14048\n",
      "Document Rostafuroxin - similarity: 0.14039\n",
      "Document BMS-564929 - similarity: 0.14031\n",
      "Document 4-(Fluorophenyl)-1-Cyclopropylmethyl-5-(2-Amino-4-Pyrimidinyl)Imidazole - similarity: 0.14026\n",
      "Document Adenosine 3',5'-diphosphate - similarity: 0.14005\n",
      "Document (4e,8e,12z,16z)-N,N,4,8,13,17,21-Heptamethyldocosa-4,8,12,16,20-Pentaen-1-Amine - similarity: 0.14003\n",
      "Document RU78262 - similarity: 0.13950\n",
      "Document 2-Bromoacetyl Group - similarity: 0.13950\n",
      "Document JNJ-39393406 - similarity: 0.13947\n",
      "Document Acetyl chloride - similarity: 0.13934\n",
      "Document Lanifibranor - similarity: 0.13934\n",
      "Document Ferric oxyhydroxide - similarity: 0.13913\n",
      "Document Brivanib - similarity: 0.13904\n",
      "Document Managlinat dialanetil - similarity: 0.13904\n",
      "Document (4s-Trans)-4-(Methylamino)-5,6-Dihydro-6-Methyl-4h-Thieno(2,3-B)Thiopyran-2-Sulfonamide-7,7-Dioxide - similarity: 0.13891\n",
      "Document 4,6-Dideoxy-4-{[4,5,6-trihydroxy-3-(hydroxymethyl)-2-cyclohexen-1-yl]amino}hexopyranose - similarity: 0.13867\n",
      "Document 2-(Beta-D-Glucopyranosyl)-5-Methyl-1,2,3-Benzimidazole - similarity: 0.13863\n",
      "Document Sofalcone - similarity: 0.13863\n",
      "Document Tasonermin - similarity: 0.13859\n",
      "Document BMS184394 - similarity: 0.13846\n",
      "Document (4s)-2-[(1e)-1-Aminoprop-1-Enyl]-4,5-Dihydro-1,3-Thiazole-4-Carboxylic Acid - similarity: 0.13843\n",
      "Document 1-(2,6-Dichlorophenyl)-5-(2,4-Difluorophenyl)-7-Piperidin-4-Yl-3,4-Dihydroquinolin-2(1h)-One - similarity: 0.13822\n",
      "Document D-Glycero-D-Mannopyranose-7-Phosphate - similarity: 0.13814\n",
      "Document Uridine-5'-Diphosphate-N-Acetylmuramoyl-L-Alanine - similarity: 0.13814\n",
      "Document JTK-853 - similarity: 0.13812\n",
      "Document ONC-201 - similarity: 0.13808\n",
      "Document Risuteganib - similarity: 0.13783\n",
      "Document Methyclothiazide - similarity: 0.13765\n",
      "Document (3S)-1-{[4-(but-2-yn-1-yloxy)phenyl]sulfonyl}pyrrolidine-3-thiol - similarity: 0.13760\n",
      "Document 6-(cyclohexylsulfanyl)-1-(ethoxymethyl)-5-(1-methylethyl)pyrimidine-2,4(1H,3H)-dione - similarity: 0.13757\n",
      "Document AR-AO-14418 - similarity: 0.13756\n",
      "Document Denenicokin - similarity: 0.13750\n",
      "Document Angelica sinensis root - similarity: 0.13726\n",
      "Document Beclanorsen - similarity: 0.13721\n",
      "Document Motixafortide - similarity: 0.13720\n",
      "Document Gavestinel - similarity: 0.13711\n",
      "Document naphthalene-1,2,4,5,7-pentol - similarity: 0.13701\n",
      "Document 5,6,7,8,9,10-HEXAHYDRO-4-HYDROXY-3-(1-PHENYLPROPYL)CYCLOOCTA[B]PYRAN-2-ONE - similarity: 0.13699\n",
      "Document Butaperazine - similarity: 0.13694\n",
      "Document 4-(5,11-DIOXO-5H-INDENO[1,2-C]ISOQUINOLIN-6(11H)-YL)BUTANOATE - similarity: 0.13690\n",
      "Document 4,6-dideoxy-4-amino-beta-D-glucopyranoside - similarity: 0.13659\n",
      "Document Aluminum chlorohydrate - similarity: 0.13657\n",
      "Document Para-Bromobenzyl Alcohol - similarity: 0.13631\n",
      "Document ATL-2502 - similarity: 0.13624\n",
      "Document 6-hydroxynorleucine - similarity: 0.13617\n",
      "Document Tilactase - similarity: 0.13612\n",
      "Document 2'-C-methylcytidine - similarity: 0.13605\n",
      "Document 6-AMINO-4-[2-(4-METHYLPHENYL)ETHYL]-1,7-DIHYDRO-8H-IMIDAZO[4,5-G]QUINAZOLIN-8-ONE - similarity: 0.13605\n",
      "Document Phloretic acid - similarity: 0.13604\n",
      "Document 4,4'-DIPYRIDYL DISULFIDE - similarity: 0.13593\n",
      "Document PROGESTERONE-11-ALPHA-OL-HEMISUCCINATE - similarity: 0.13589\n",
      "Document Influenza A virus A/Texas/50/2012 X-223A (H3N2) antigen (formaldehyde inactivated) - similarity: 0.13553\n",
      "Document Reposal - similarity: 0.13516\n",
      "Document Sri-9662 - similarity: 0.13516\n",
      "Document Acetyleneurea - similarity: 0.13498\n",
      "Document (4Z)-2,8:7,12:11,15:14,18:17,22-PENTAANHYDRO-4,5,6,9,10,13,19,20,21-NONADEOXY-D-ARABINO-D-ALLO-D-ALLO-DOCOSA-4,9,20-TRIENITOL - similarity: 0.13471\n",
      "Document Padimate O - similarity: 0.13469\n",
      "Document Heptamolybdate - similarity: 0.13466\n",
      "Document (5R,6S,8S)-8-[3-(AMINOMETHYL)PHENYL]-6-HYDROXY-5-ISOPROPYL-3-OXO-1-PHENYL-2,7-DIOXA-4-AZA-6-PHOSPHANONAN-9-OIC ACID 6-OXIDE - similarity: 0.13444\n",
      "Document Alsactide - similarity: 0.13441\n",
      "Document 4-[(1E,7E)-8-(2,6-DIOXO-1,2,3,6-TETRAHYDROPYRIMIDIN-4-YL)-3,6-DIOXA-2,7-DIAZAOCTA-1,7-DIEN-1-YL]BENZOIC ACID - similarity: 0.13439\n",
      "Document Clomocycline - similarity: 0.13421\n",
      "Document N-phenyl-1H-pyrrolo[2,3-b]pyridin-3-amine - similarity: 0.13420\n",
      "Document 13-Acetylphorbol - similarity: 0.13409\n",
      "Document PF-04418948 - similarity: 0.13380\n",
      "Document PHENYL-5-(1H-PYRAZOL-3-YL)-1,3-THIAZOLE - similarity: 0.13369\n",
      "Document N-{4-[(Carboxymethyl)carbamoyl]benzoyl}-L-valyl-N-[(3S)-1,1,1-trifluoro-4-methyl-2-oxo-3-pentanyl]-L-prolinamide - similarity: 0.13331\n",
      "Document Nitrosoethane - similarity: 0.13326\n",
      "Document 5-(2-methylpiperazine-1-sulfonyl)isoquinoline - similarity: 0.13314\n",
      "Document Elacytarabine - similarity: 0.13313\n",
      "Document [(2R,3S,4R,5R)-5-(4-acetonyl-3-carbamoyl-pyridin-1-ium-1-yl)-3,4-dihydroxy-tetrahydrofuran-2-yl]methyl [[(2R,3S,4R,5R)-5-(6-aminopurin-9-yl)-3,4-dihydroxy-tetrahydrofuran-2-yl]methoxy-hydroxy-phosphoryl] phosphate - similarity: 0.13313\n",
      "Document N-[1H-INDOL-3-YL-ACETYL]GLYCINE ACID - similarity: 0.13308\n",
      "Document (6S)-6-CYCLOPENTYL-6-[2-(3-FLUORO-4-ISOPROPOXYPHENYL)ETHYL]-4-HYDROXY-5,6-DIHYDRO-2H-PYRAN-2-ONE - similarity: 0.13305\n",
      "Document GDC-0349 - similarity: 0.13303\n",
      "Document Ephedra sinica stem - similarity: 0.13288\n",
      "Document Enfortumab vedotin - similarity: 0.13288\n",
      "Document LFF-571 - similarity: 0.13283\n",
      "Document Betula lenta pollen - similarity: 0.13281\n",
      "Document N~2~-1H-benzimidazol-5-yl-N~4~-(3-cyclopropyl-1H-pyrazol-5-yl)pyrimidine-2,4-diamine - similarity: 0.13275\n",
      "Document Neuropeptide Y - similarity: 0.13270\n",
      "Document LGD2941 - similarity: 0.13265\n",
      "Document 2-(4-METHOXYPHENYL)ACETAMIDE - similarity: 0.13257\n",
      "Document Guanfacine - similarity: 0.13254\n",
      "Document (S)-N-(4-carbamimidoylbenzyl)-1-(2-(cyclopentyloxy)ethanoyl)pyrrolidine-2-carboxamide - similarity: 0.13252\n",
      "Document Butenoic Acid - similarity: 0.13210\n",
      "Document Heptane-1,2,3-Triol - similarity: 0.13204\n",
      "Document Nonan-1-Ol - similarity: 0.13197\n",
      "Document Aplyronine A - similarity: 0.13196\n",
      "Document Digalactosyl Diacyl Glycerol (Dgdg) - similarity: 0.13187\n",
      "Document 4-Methylthio-Alpha-D-Mannose - similarity: 0.13160\n",
      "Document 4-[5-(2-CARBOXY-1-FORMYL-ETHYLCARBAMOYL)-PYRIDIN-3-YL]-BENZOIC ACID - similarity: 0.13160\n",
      "Document N'-[(1E)-(3,5-dibromo-2,4-dihydroxyphenyl)methylidene]-4-methoxybenzohydrazide - similarity: 0.13110\n",
      "Document 3-[(4-Carboxy-2-hydroxyphenyl)sulfamoyl]-2-thiophenecarboxylic acid - similarity: 0.13082\n",
      "Document N-(2-METHOXYETHYL)-4-({4-[2-METHYL-1-(1-METHYLETHYL)-1H-IMIDAZOL-5-YL]PYRIMIDIN-2-YL}AMINO)BENZENESULFONAMIDE - similarity: 0.13082\n",
      "Document (5E)-12-CHLORO-13,15-DIHYDROXY-4,7,8,9-TETRAHYDRO-2-BENZOXACYCLOTRIDECINE-1,10(3H,11H)-DIONE - similarity: 0.13081\n",
      "Document benzyl (2-oxopropyl)carbamate - similarity: 0.13065\n",
      "Document Acemetacin - similarity: 0.13064\n",
      "Document Gallium cation - similarity: 0.13054\n",
      "Document LI-301 - similarity: 0.13019\n",
      "Document CPI-1205 - similarity: 0.13019\n",
      "Document (2R)-4-(2-BENZOYL-1,2-DIAZEPAN-1-YL)-4-OXO-1-(2,4,5-TRIFLUOROPHENYL)BUTAN-2-AMINE - similarity: 0.12975\n",
      "Document Influenza A virus A/Switzerland/3330/2017 (H1N1) live (attenuated) antigen - similarity: 0.12963\n",
      "Document 4-[3-(2-Chloro-4,5-difluoro-benzoyl)ureido]-3-trifluoromethoxybenzoic acid - similarity: 0.12943\n",
      "Document Vapreotide - similarity: 0.12939\n",
      "Document 5-[(phenylsulfonyl)amino]-1,3,4-thiadiazole-2-sulfonamide - similarity: 0.12913\n",
      "Document Bifidobacterium longum infantis - similarity: 0.12911\n",
      "Document Morpholine-4-Carboxylic Acid (1-(3-Benzenesulfonyl-1-Phenethylallylcarbamoyl)-3-Methylbutyl)-Amide - similarity: 0.12905\n",
      "Document Acetyldihydrocodeine - similarity: 0.12905\n",
      "Document Ritobegron - similarity: 0.12881\n",
      "Document 2-[2-ETHANESULFONYLAMINO-3-(5-PROPOXY-1H-INDOL-3-YL)-PROPIONYLAMINO]-PENTANEDIOIC ACID 5-AMIDE 1-(4-CARBAMIMIDOYL-BENZYLAMIDE) - similarity: 0.12873\n",
      "Document ATG-Fresenius S - similarity: 0.12863\n",
      "Document Chromic nitrate - similarity: 0.12862\n",
      "Document Ceramide AP - similarity: 0.12858\n",
      "Document PH-284 - similarity: 0.12847\n",
      "Document D-Leucine - similarity: 0.12845\n",
      "Document 3-(alpha-D-galactopyranosyloxy)-N-(3-{4-[3-({2-[(3-{4-[3-({[3-(hexopyranosyloxy)-5-nitrophenyl]carbonyl}amino)propyl]piperazin-1-yl}propyl)amino]-3,4-dioxocyclobut-1-en-1-yl}amino)propyl]piperazin-1-yl}propyl)-5-nitrobenzamide - similarity: 0.12837\n",
      "Document D-Lysine - similarity: 0.12826\n",
      "Document Casimersen - similarity: 0.12825\n",
      "Document (3,4-Dihydroxy-Phenyl)-Triphenyl-Arsonium - similarity: 0.12815\n",
      "Document 3-Bromo-3-buten-1-ol - similarity: 0.12802\n",
      "Document Metenolone - similarity: 0.12789\n",
      "Document 1-Deoxy-6-O-Phosphono-1-[(Phosphonomethyl)Amino]-L-Threo-Hexitol - similarity: 0.12778\n",
      "Document N,4-dimethyl-3-[(1-phenyl-1H-pyrazolo[3,4-d]pyrimidin-4-yl)amino]benzamide - similarity: 0.12774\n",
      "Document NM-404 I-131 - similarity: 0.12761\n",
      "Document 2-fluoro-4-[4-(4-fluorophenyl)-1H-pyrazol-3-yl]pyridine - similarity: 0.12743\n",
      "Document Vinburnine - similarity: 0.12734\n",
      "Document [[1-[N-HYDROXY-ACETAMIDYL]-3-METHYL-BUTYL]-CARBONYL-LEUCINYL]-ALANINE ETHYL ESTER - similarity: 0.12713\n",
      "Document Dimethylphthalate - similarity: 0.12700\n",
      "Document Ty800 - similarity: 0.12693\n",
      "Document (2S,4S,5R)-1-(4-TERT-BUTYLBENZOYL)-2-ISOBUTYL-5-(1,3-THIAZOL-2-YL)PYRROLIDINE-2,4-DICARBOXYLIC ACID - similarity: 0.12687\n",
      "Document Glucosamine 1-Phosphate - similarity: 0.12665\n",
      "Document Betula populifolia pollen - similarity: 0.12641\n",
      "Document PASBN - similarity: 0.12640\n",
      "Document Aminophosphonic acid-guanylate ester - similarity: 0.12634\n",
      "Document (2r,4s)-2-Methyl-2,3,3,4-Tetrahydroxytetrahydrofuran - similarity: 0.12633\n",
      "Document AZD-9164 - similarity: 0.12630\n",
      "Document 3-(4-Benzenesulfonyl-Thiophene-2-Sulfonylamino)-Phenylboronic Acid - similarity: 0.12610\n",
      "Document Antithrombin Alfa - similarity: 0.12586\n",
      "Document Glesatinib - similarity: 0.12584\n",
      "Document 1-deoxymannojirimycin - similarity: 0.12573\n",
      "Document SPK-843 - similarity: 0.12556\n",
      "Document Influenza A virus A/Bolivia/559/2013 (H1N1) live (attenuated) antigen - similarity: 0.12544\n",
      "Document 5'-S-[2-(decylamino)ethyl]-5'-thioadenosine - similarity: 0.12540\n",
      "Document Lomibuvir - similarity: 0.12487\n",
      "Document Neisseria meningitidis group w-135 capsular oligosaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.12485\n",
      "Document REL-(9R,12S)-9,10,11,12-TETRAHYDRO-9,12-EPOXY-1H-DIINDOLO[1,2,3-FG:3',2',1'-KL]PYRROLO[3,4-I][1,6]BENZODIAZOCINE-1,3(2H)-DIONE - similarity: 0.12466\n",
      "Document Pimelic Acid - similarity: 0.12466\n",
      "Document Bunamidine - similarity: 0.12451\n",
      "Document TAK-580 - similarity: 0.12448\n",
      "Document Trencam-3,2-Hopo - similarity: 0.12447\n",
      "Document 3,6,9,12,15,18-HEXAOXAICOSANE - similarity: 0.12425\n",
      "Document Acer saccharum pollen - similarity: 0.12419\n",
      "Document SL017 - similarity: 0.12409\n",
      "Document D-Alanine - similarity: 0.12392\n",
      "Document L-xylitol 5-phosphate - similarity: 0.12388\n",
      "Document Balipodect - similarity: 0.12387\n",
      "Document N-(4-sulfamoylphenyl)-1H-indazole-3-carboxamide - similarity: 0.12372\n",
      "Document (2s)-Pyrrolidin-2-Ylmethylamine - similarity: 0.12349\n",
      "Document Antimony potassium tartrate - similarity: 0.12339\n",
      "Document Thonzonium - similarity: 0.12336\n",
      "Document 3,4-Methylenedioxy-N-isopropylamphetamine - similarity: 0.12315\n",
      "Document Indirubin-5-sulphonate - similarity: 0.12314\n",
      "Document Nifenazone - similarity: 0.12309\n",
      "Document Emivirine - similarity: 0.12298\n",
      "Document Smilagenin - similarity: 0.12289\n",
      "Document (2E)-3-(4-CHLOROPHENYL)-N-HYDROXYACRYLAMIDE - similarity: 0.12279\n",
      "Document Sarecycline - similarity: 0.12267\n",
      "Document Tripterygium wilfordii whole - similarity: 0.12264\n",
      "Document 4-HYDROXY-3-NITROPHENYLACETYL-EPSILON-AMINOCAPROIC ACID ANION - similarity: 0.12263\n",
      "Document Tetrafluoroaluminate Ion - similarity: 0.12260\n",
      "Document Dolichovespula arenaria venom protein - similarity: 0.12254\n",
      "Document 5-amino-1,3,4-thiadiazole-2-thiol - similarity: 0.12251\n",
      "Document 3-(3-aminophenyl)-N-(3-chlorophenyl)pyrazolo[1,5-a]pyrimidin-5-amine - similarity: 0.12240\n",
      "Document (S)-butane-1,3-diol - similarity: 0.12238\n",
      "Document 4-Hydroxy-1,2,5-oxadiazole-3-carboxylic acid - similarity: 0.12224\n",
      "Document Ethanesulfonic acid - similarity: 0.12185\n",
      "Document 6-[N-(1-Isopropyl-1,2,3,4-Tetrahydro-7-Isoquinolinyl)Carbamyl]-2-Naphthalenecarboxamidine - similarity: 0.12185\n",
      "Document Fendiline - similarity: 0.12179\n",
      "Document N-(R-Carboxy-Ethyl)-Alpha-(S)-(2-Phenylethyl) - similarity: 0.12175\n",
      "Document N-cyclopropyl-4-methyl-3-[1-(2-methylphenyl)phthalazin-6-yl]benzamide - similarity: 0.12154\n",
      "Document Sambucus nigra flowering top - similarity: 0.12151\n",
      "Document 2-({2-[(3-HYDROXYPHENYL)AMINO]PYRIMIDIN-4-YL}AMINO)BENZAMIDE - similarity: 0.12150\n",
      "Document (1S,2S,3R,6R)-4-(hydroxymethyl)-6-(octylamino)cyclohex-4-ene-1,2,3-triol - similarity: 0.12147\n",
      "Document Ioglycamic acid - similarity: 0.12145\n",
      "Document Rumex acetosella whole - similarity: 0.12143\n",
      "Document Pyroquilon - similarity: 0.12131\n",
      "Document Cephaloglycin - similarity: 0.12129\n",
      "Document Norvaline - similarity: 0.12128\n",
      "Document N-[4-(4-nitrophenylphospho)butanoyl]-D-alanine - similarity: 0.12120\n",
      "Document Remtolumab - similarity: 0.12108\n",
      "Document 4-(Aminosulfonyl)-N-[(2,4,6-Trifluorophenyl)Methyl]-Benzamide - similarity: 0.12107\n",
      "Document Cefatrizine - similarity: 0.12089\n",
      "Document L-serine O-sulfate - similarity: 0.12067\n",
      "Document 7-HYDROXY-4-METHYL-3-(2-HYDROXY-ETHYL)COUMARIN - similarity: 0.12059\n",
      "Document DDP-200 - similarity: 0.12040\n",
      "Document 8-iodo-guanine - similarity: 0.12034\n",
      "Document (10E,12Z)-octadecadienoic acid - similarity: 0.12029\n",
      "Document D-arginine - similarity: 0.12013\n",
      "Document AB192 - similarity: 0.12003\n",
      "Document S-METHYL-4,5,6,7-TETRABROMO-BENZIMIDAZOLE - similarity: 0.12000\n",
      "Document 3-(4-NITRO-PHENOXY)-PROPAN-1-OL - similarity: 0.11996\n",
      "Document alpha-D-mannose 6-phosphate - similarity: 0.11986\n",
      "Document 4,6-O-(1-Carboxyethylidene)-Beta-D-Glucose - similarity: 0.11985\n",
      "Document S-P-Nitrobenzyloxycarbonylglutathione - similarity: 0.11936\n",
      "Document 2-[3-(5-Mercapto-[1,3,4]thiadiazol-2-yl)-ureido]-N-methyl-3-phenyl-propionamide - similarity: 0.11935\n",
      "Document Anethole trithione - similarity: 0.11933\n",
      "Document 1-Amino-1-Carbonyl Pentane - similarity: 0.11931\n",
      "Document Strontium malonate - similarity: 0.11878\n",
      "Document Gold cation (1+) - similarity: 0.11877\n",
      "Document bis(molybdopterin)tungsten cofactor - similarity: 0.11865\n",
      "Document Elvucitabine - similarity: 0.11855\n",
      "Document REN-850 - similarity: 0.11829\n",
      "Document KB002 - similarity: 0.11828\n",
      "Document Glutathionylspermidine - similarity: 0.11811\n",
      "Document Tinostamustine - similarity: 0.11798\n",
      "Document N-(5'-phosphopyridoxyl)-L-alanine - similarity: 0.11785\n",
      "Document (1R)-1-{[(4'-methoxy-1,1'-biphenyl-4-yl)sulfonyl]amino}-2-methylpropylphosphonic acid - similarity: 0.11761\n",
      "Document Proxazole - similarity: 0.11750\n",
      "Document Bisegliptin - similarity: 0.11735\n",
      "Document Hexane-1,6-Diol - similarity: 0.11731\n",
      "Document M-Aminophenylboronic Acid - similarity: 0.11709\n",
      "Document Mercaptocarboxylate Inhibitor - similarity: 0.11702\n",
      "Document 6-AMINO-BENZO[DE]ISOQUINOLINE-1,3-DIONE - similarity: 0.11699\n",
      "Document Alobresib - similarity: 0.11688\n",
      "Document (S)-4-Nitrostyrene oxide - similarity: 0.11687\n",
      "Document Peruvoside - similarity: 0.11678\n",
      "Document 7-cyano-7-deazaguanine - similarity: 0.11677\n",
      "Document Secretin porcine - similarity: 0.11639\n",
      "Document Sulfoxone - similarity: 0.11617\n",
      "Document CRA_1802 - similarity: 0.11611\n",
      "Document Racepinephrine - similarity: 0.11586\n",
      "Document 4-chloro-N-(3-methoxypropyl)-N-[(3S)-1-(2-phenylethyl)piperidin-3-yl]benzamide - similarity: 0.11583\n",
      "Document 4-Deoxy-4-Amino-Beta-D-Glucose - similarity: 0.11570\n",
      "Document INCB3284 - similarity: 0.11570\n",
      "Document 1-Butane Boronic Acid - similarity: 0.11548\n",
      "Document 2-amino-6-[2-(1H-indol-6-yl)ethyl]pyrimidin-4(3H)-one - similarity: 0.11544\n",
      "Document 19-Nor-4-androstenedione - similarity: 0.11543\n",
      "Document 3',5'-DIBROMO-2',4,4',6'-TETRAHYDROXY AURONE - similarity: 0.11541\n",
      "Document 5-Bromothienyldeoxyuridine - similarity: 0.11540\n",
      "Document Zenocutuzumab - similarity: 0.11536\n",
      "Document Canrenoic acid - similarity: 0.11522\n",
      "Document ACV tripeptide - similarity: 0.11492\n",
      "Document Euflavine - similarity: 0.11490\n",
      "Document alpha-Pyrrolidinononanophenone - similarity: 0.11468\n",
      "Document 2-Benzo[1,3]Dioxol-5-Ylmethyl-3-Benzyl-Succinic Acid - similarity: 0.11465\n",
      "Document Imolamine - similarity: 0.11449\n",
      "Document 5-{3-[3-(2,4-DICHLORO-BENZOYL)-UREIDO]-2-METHYL-PHENOXY}-PENTANOIC ACID - similarity: 0.11414\n",
      "Document ERB-196 - similarity: 0.11411\n",
      "Document Oxeladin - similarity: 0.11399\n",
      "Document Juniperus osteosperma pollen - similarity: 0.11382\n",
      "Document Salvinorin A - similarity: 0.11376\n",
      "Document 12-Hydroxydodecanoic Acid - similarity: 0.11376\n",
      "Document Influenza B virus B/Brisbane/46/2015 hemagglutinin antigen (propiolactone inactivated) - similarity: 0.11372\n",
      "Document {4-[(2s,4e)-2-(1,3-Benzothiazol-2-Yl)-2-(1h-1,2,3-Benzotriazol-1-Yl)-5-Phenylpent-4-Enyl]Phenyl}(Difluoro)Methylphosphonic Acid - similarity: 0.11371\n",
      "Document D-leucyl-N-(4-carbamimidoylbenzyl)-L-prolinamide - similarity: 0.11371\n",
      "Document Al-6629, [2h-Thieno[3,2-E]-1,2-Thiazine-6-Sulfonamide,2-(3-Methoxyphenyl)-3-(4-Morpholinyl)-, 1,1-Dioxide] - similarity: 0.11357\n",
      "Document 3-[(2,2-DIMETHYLPROPANOYL)AMINO]-N-1,3-THIAZOL-2-YLPYRIDINE-2-CARBOXAMIDE - similarity: 0.11342\n",
      "Document 2-[(Dioxidophosphoranyl)oxy]benzoate - similarity: 0.11341\n",
      "Document CC-220 - similarity: 0.11341\n",
      "Document N-[(5R,14R)-5-AMINO-5,14-DIMETHYL-4-OXO-3-OXA-18-AZATRICYCLO[15.3.1.1~7,11~]DOCOSA-1(21),7(22),8,10,17,19-HEXAEN-19-YL]-N-METHYLMETHANESULFONAMIDE - similarity: 0.11335\n",
      "Document 2-Undecanone - similarity: 0.11330\n",
      "Document Aluminum subacetate - similarity: 0.11330\n",
      "Document 7-Hydroxy-2-Oxo-Chromene-3-Carboxylic Acid Ethyl Ester - similarity: 0.11326\n",
      "Document (E)-2-Fluoro-P-Hydroxycinnamate - similarity: 0.11319\n",
      "Document D-JNKI-1 - similarity: 0.11291\n",
      "Document Uridine-5'-diphosphate-mannose - similarity: 0.11262\n",
      "Document 2-({6-[(3-Chlorophenyl)amino]-9-isopropyl-9H-purin-2-yl}amino)ethanol - similarity: 0.11247\n",
      "Document 2-(1H-pyrazol-3-yl)-1H-benzimidazole - similarity: 0.11243\n",
      "Document SGX-523 - similarity: 0.11213\n",
      "Document Human cytomegalovirus immune globulin - similarity: 0.11204\n",
      "Document Palmitoyl-Linoleoyl Phosphatidylcholine - similarity: 0.11194\n",
      "Document Sulfobromophthalein - similarity: 0.11185\n",
      "Document Balsam of Peru - similarity: 0.11176\n",
      "Document 3-bromo-5-phenyl-N-(pyridin-3-ylmethyl)pyrazolo[1,5-a]pyrimidin-7-amine - similarity: 0.11163\n",
      "Document Guanosine-2'-monophosphate - similarity: 0.11156\n",
      "Document N,N,N-trimethylglycinium - similarity: 0.11153\n",
      "Document N-Methyl-N-(Methylbenzyl)Formamide - similarity: 0.11133\n",
      "Document Methylthiouracil - similarity: 0.11128\n",
      "Document (5-Chloropyrazolo[1,5-a]Pyrimidin-7-Yl)-(4-Methanesulfonylphenyl)Amine - similarity: 0.11125\n",
      "Document (3R,4R,5S,6R)-6-Fluoro-3,4,5-trihydroxytetrahydro-2H-pyran-2-olate - similarity: 0.11099\n",
      "Document Diphenylacetic acid - similarity: 0.11097\n",
      "Document Iprindole - similarity: 0.11076\n",
      "Document Octanoyl-Coenzyme A - similarity: 0.11069\n",
      "Document Deoxyuridine-5'-Triphosphate - similarity: 0.11052\n",
      "Document Crotedumab - similarity: 0.11048\n",
      "Document Mubritinib - similarity: 0.11030\n",
      "Document 4-(dihydroxyboranyl)-2-({[4-(phenylsulfonyl)thiophen-2-yl]sulfonyl}amino)benzoic acid - similarity: 0.11000\n",
      "Document Cridanimod - similarity: 0.10991\n",
      "Document Acid yellow 54 free acid - similarity: 0.10980\n",
      "Document Alphaprodine - similarity: 0.10974\n",
      "Document L-Xylulose 5-Phosphate - similarity: 0.10971\n",
      "Document 3,8,9,10-tetrahydroxy-7-hydroxymethyl-6-oxa-1,3-diaza-spiro[4.5]decane-2,4-dione - similarity: 0.10941\n",
      "Document (3aS)-3a-hydroxy-7-methyl-1-phenyl-1,2,3,3a-tetrahydro-4H-pyrrolo[2,3-b]quinolin-4-one - similarity: 0.10941\n",
      "Document Banoxantrone - similarity: 0.10935\n",
      "Document 5-[1-(3,4-Dimethoxy-Benzoyl)-1,2,3,4-Tetrahydro-Quinolin-6-Yl]-6-Methyl-3,6-Dihydro-[1,3,4]Thiadiazin-2-One - similarity: 0.10927\n",
      "Document Lauryl alcohol diphosphonic acid - similarity: 0.10903\n",
      "Document N-[3-(2-fluoroethoxy)phenyl]-N'-(1,3,4-trioxo-1,2,3,4-tetrahydroisoquinolin-6-yl)butanediamide - similarity: 0.10895\n",
      "Document 2-[4-[4-[(5-cyclopropyl-1H-pyrazol-3-yl)amino]quinazolin-2-yl]iminocyclohexa-2,5-dien-1-yl]acetonitrile - similarity: 0.10876\n",
      "Document 1-[(2-Amino-6,9-Dihydro-1h-Purin-6-Yl)Oxy]-3-Methyl-2-Butanol - similarity: 0.10849\n",
      "Document Dezamizumab - similarity: 0.10822\n",
      "Document Quinupristin - similarity: 0.10814\n",
      "Document (2S)-7-hydroxyflavanone - similarity: 0.10808\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document 3-thiaoctanoyl-CoA - similarity: 0.10803\n",
      "Document S-Isopropyl-Isothiourea - similarity: 0.10795\n",
      "Document Zalifrelimab - similarity: 0.10773\n",
      "Document 3-FLUORO-5-MORPHOLIN-4-YL-N-[3-(2-PYRIDIN-4-YLETHYL)-1H-INDOL-5-YL]BENZAMIDE - similarity: 0.10760\n",
      "Document Furaprevir - similarity: 0.10734\n",
      "Document N-(M-Trifluoromethylphenyl) Phenoxazine-4,6-Dicarboxylic Acid - similarity: 0.10732\n",
      "Document Englitazone - similarity: 0.10725\n",
      "Document Practolol - similarity: 0.10719\n",
      "Document Polyquaternium-10 (400 cps at 2%) - similarity: 0.10712\n",
      "Document N-acetyl-L-arginine - similarity: 0.10692\n",
      "Document [2,4,6-Triisopropyl-Phenylsulfonyl-L-[3-Amidino-Phenylalanine]]-Piperazine-N'-Beta-Alanine - similarity: 0.10687\n",
      "Document (1S)-1-AMINO-2-(1H-INDOL-3-YL)ETHANOL - similarity: 0.10683\n",
      "Document beta-(1->4)-galactotriose - similarity: 0.10666\n",
      "Document 3-nitro-L-tyrosine - similarity: 0.10665\n",
      "Document Irdabisant - similarity: 0.10658\n",
      "Document H TYPE II TRISACCHARIDE - similarity: 0.10653\n",
      "Document UDP-6-sulfoquinovose - similarity: 0.10632\n",
      "Document 4-Oxoretinol - similarity: 0.10615\n",
      "Document (9BETA,13ALPHA,14BETA,17ALPHA)-2-METHOXYESTRA-1,3,5(10)-TRIENE-3,17-DIYL DISULFAMATE - similarity: 0.10566\n",
      "Document Influenza A virus A/Singapore/gp1908/2015 IVR-180 (H1N1) antigen (formaldehyde inactivated) - similarity: 0.10551\n",
      "Document OBE101 - similarity: 0.10520\n",
      "Document Reactive Red 6 hapten - similarity: 0.10496\n",
      "Document 2-O-(4'-AMIDINOPHENYL)-5-O-(3''-AMIDINOPHENYL)-1,4:3,6-DIANHYDRO-D-SORBITOL - similarity: 0.10490\n",
      "Document Dimenoxadol - similarity: 0.10478\n",
      "Document 3-(2-AMINOQUINAZOLIN-6-YL)-4-METHYL-N-[3-(TRIFLUOROMETHYL)PHENYL]BENZAMIDE - similarity: 0.10459\n",
      "Document 4-Amino-2-[(3-chlorophenyl)amino]-5-(4-fluorobenzoyl)-1,3-thiazol-3-ium - similarity: 0.10455\n",
      "Document (5E,7S)-2-amino-7-(4-fluoro-2-pyridin-3-ylphenyl)-4-methyl-7,8-dihydroquinazolin-5(6H)-one oxime - similarity: 0.10453\n",
      "Document NF-kappaB Decoy - similarity: 0.10442\n",
      "Document (2R,3R,4S,5R)-2-acetamido-3,4-dihydroxy-5-hydroxymethyl-piperidine - similarity: 0.10428\n",
      "Document AZD-1480 - similarity: 0.10419\n",
      "Document Bis-Napthyl Beta-Ketophosphonic Acid - similarity: 0.10417\n",
      "Document Robalzotan - similarity: 0.10414\n",
      "Document L-Histidine Beta Naphthylamide - similarity: 0.10406\n",
      "Document Fedovapagon - similarity: 0.10401\n",
      "Document Isobromindione - similarity: 0.10398\n",
      "Document beta-D-fucose - similarity: 0.10391\n",
      "Document Ocaperidone - similarity: 0.10353\n",
      "Document 3,5-Dimethyl-1-(3-Nitrophenyl)-1h-Pyrazole-4-Carboxylic Acid Ethyl Ester - similarity: 0.10300\n",
      "Document S-[5-(TRIFLUOROMETHYL)-4H-1,2,4-TRIAZOL-3-YL] 5-(PHENYLETHYNYL)FURAN-2-CARBOTHIOATE - similarity: 0.10277\n",
      "Document Quahog, unspecified - similarity: 0.10271\n",
      "Document Ulmus rubra pollen - similarity: 0.10259\n",
      "Document Sonedenoson - similarity: 0.10243\n",
      "Document 2,5-Dimethoxy-4-ethylamphetamine - similarity: 0.10227\n",
      "Document 1,2-Di-1-(3,7,11,15-Tetramethyl-Hexadecane)-Sn-Glycero-3-Phosphate - similarity: 0.10217\n",
      "Document N-(2-chlorophenyl)-5-phenylimidazo[1,5-a]pyrazin-8-amine - similarity: 0.10211\n",
      "Document L-methionine sulfone - similarity: 0.10201\n",
      "Document Lorpiprazole - similarity: 0.10186\n",
      "Document Alphacetylmethadol - similarity: 0.10186\n",
      "Document (5-BROMO-4-CHLORO-3-INDOLYL)-A-D-MANNOSE - similarity: 0.10166\n",
      "Document (6-[4-(AMINOMETHYL)-2,6-DIMETHYLPHENOXY]-2-{[4-(AMINOMETHYL)PHENYL]AMINO}-5-BROMOPYRIMIDIN-4-YL)METHANOL - similarity: 0.10166\n",
      "Document Serine-3'-aminoadenosine - similarity: 0.10136\n",
      "Document alpha-Hydroxy glycineamide - similarity: 0.10132\n",
      "Document Iodine I-125 - similarity: 0.10115\n",
      "Document VIBRIO CHOLERAE INABA 48 CLASSICAL BIOTYPE, HEAT INACTIVATED - similarity: 0.10104\n",
      "Document (4S,5S)-1,2-dithiane-4,5-diol - similarity: 0.10089\n",
      "Document Influenza B virus B/Texas/6/2011 antigen (formaldehyde inactivated) - similarity: 0.10085\n",
      "Document Ferrous cation - similarity: 0.10074\n",
      "Document Dichloralphenazone - similarity: 0.10055\n",
      "Document 2-(2,4-DICHLOROPHENOXY)-5-(PYRIDIN-2-YLMETHYL)PHENOL - similarity: 0.10049\n",
      "Document DAS869 - similarity: 0.10010\n",
      "Document VIT-100 - similarity: 0.09977\n",
      "Document 2-[(Formyl-Hydroxy-Amino)-Methyl]-Heptanoic Acid [1-(2-Hydroxymethyl-Pyrrolidine-1-Carbonyl)-2-Methyl-Propyl]-Amide - similarity: 0.09975\n",
      "Document 4-Carboxyphenylboronic Acid - similarity: 0.09949\n",
      "Document SR-123781A - similarity: 0.09946\n",
      "Document INCB13739 - similarity: 0.09941\n",
      "Document Bethanidine - similarity: 0.09887\n",
      "Document 5-[(3AS,4R,6AR)-2-OXOHEXAHYDRO-1H-THIENO[3,4-D]IMIDAZOL-4-YL]PENTANOIC ACID - similarity: 0.09880\n",
      "Document 7-((Carboxy(4-Hydroxyphenyl)Acetyl)Amino)-7-Methoxy-(3-((1-Methyl-1h-Tetrazol-5-Yl)Thio)Methyl)-8-Oxo-5-Oxa-1-Azabicyclo[4.2.0]Oct-2-Ene-2-Carboxylic Acid - similarity: 0.09862\n",
      "Document (S)-propane-1,2-diol - similarity: 0.09851\n",
      "Document Ticalopride - similarity: 0.09846\n",
      "Document Irofulven - similarity: 0.09819\n",
      "Document Ethacizine - similarity: 0.09811\n",
      "Document Phenacemide - similarity: 0.09792\n",
      "Document Ethotoin - similarity: 0.09769\n",
      "Document Chamaecyparis obtusa leaf - similarity: 0.09754\n",
      "Document Droloxifene - similarity: 0.09745\n",
      "Document 6-METHYL-2(PROPANE-1-SULFONYL)-2H-THIENO[3,2-D][1,2,3]DIAZABORININ-1-OL - similarity: 0.09663\n",
      "Document T-62 - similarity: 0.09663\n",
      "Document (1'R,2'S)-9-(2-Hydroxy-3'-Keto-Cyclopenten-1-yl)Adenine - similarity: 0.09645\n",
      "Document 5'-deoxy-5'-piperidin-1-ylthymidine - similarity: 0.09637\n",
      "Document 2'-Deoxyguanosine-5'-Diphosphate - similarity: 0.09625\n",
      "Document Selenium (75Se) norcholesterol - similarity: 0.09619\n",
      "Document ING-1 - similarity: 0.09615\n",
      "Document NCX 950 - similarity: 0.09574\n",
      "Document Opaviraline - similarity: 0.09574\n",
      "Document D-alpha-Tocopherol acetate - similarity: 0.09570\n",
      "Document 5'-O-(L-Cysteinylsulfamoyl)adenosine - similarity: 0.09556\n",
      "Document Dihydrexidine - similarity: 0.09554\n",
      "Document Dihydroxyacetone phosphate - similarity: 0.09539\n",
      "Document Dimetotiazine - similarity: 0.09537\n",
      "Document N-{[(2S,3S)-3-(Ethoxycarbonyl)-2-oxiranyl]carbonyl}-L-threonyl-L-isoleucine - similarity: 0.09504\n",
      "Document 4-Hydroxybutan-1-Aminium - similarity: 0.09500\n",
      "Document HE-2200 - similarity: 0.09470\n",
      "Document Benzenesulfinic acid - similarity: 0.09462\n",
      "Document (1S)-2-[(2S,5R)-2-(AMINOMETHYL)-5-ETHYNYLPYRROLIDIN-1-YL]-1-CYCLOPENTYL-2-OXOETHANAMINE - similarity: 0.09390\n",
      "Document CRA_9076 - similarity: 0.09387\n",
      "Document 5,6-Dihydro-Benzo[H]Cinnolin-3-Ylamine - similarity: 0.09369\n",
      "Document ILY101 - similarity: 0.09366\n",
      "Document Tezacitabine - similarity: 0.09365\n",
      "Document XL228 - similarity: 0.09356\n",
      "Document ACR-16 - similarity: 0.09356\n",
      "Document DIETHYL 4-METHOXYPHENYL PHOSPHATE - similarity: 0.09346\n",
      "Document N-Acetylalanine - similarity: 0.09329\n",
      "Document Amido Phenyl Pyruvic Acid - similarity: 0.09321\n",
      "Document Mercuribenzoic Acid - similarity: 0.09318\n",
      "Document [(1S)-4-[[Amino-[(2R)-2-amino-2-carboxyethyl]sulfanylmethylidene]amino]-1-carboxybutyl]azanium - similarity: 0.09315\n",
      "Document Thenalidine - similarity: 0.09299\n",
      "Document 6,7-DIMETHOXY-4-[(3R)-3-(2-NAPHTHYLOXY)PYRROLIDIN-1-YL]QUINAZOLINE - similarity: 0.09286\n",
      "Document L-2-amino-4-methoxy-cis-but-3-enoic acid - similarity: 0.09276\n",
      "Document PAS219 - similarity: 0.09248\n",
      "Document Bromo-Willardiine - similarity: 0.09247\n",
      "Document Piprozolin - similarity: 0.09213\n",
      "Document Casimiroin - similarity: 0.09207\n",
      "Document Pentafluoropropane - similarity: 0.09164\n",
      "Document Modified Acarbose Pentasaccharide - similarity: 0.09164\n",
      "Document 2-[1-(4-CHLORO-PHENYL)-ETHYL]-4,6-DINITRO-PHENOL - similarity: 0.09154\n",
      "Document Vanillic acid - similarity: 0.09153\n",
      "Document alpha,beta-Dehydroaminobutyric acid - similarity: 0.09129\n",
      "Document Pyrrole-2-Carboxylate - similarity: 0.09115\n",
      "Document BMSC-0013 - similarity: 0.09110\n",
      "Document 2H-Benzimidazol-2-amine - similarity: 0.09108\n",
      "Document Olorinab - similarity: 0.09108\n",
      "Document 4-Deoxylactose - similarity: 0.09043\n",
      "Document RB106 - similarity: 0.09022\n",
      "Document Teverelix - similarity: 0.09009\n",
      "Document Beta-3-Serine - similarity: 0.08998\n",
      "Document Bunaftine - similarity: 0.08955\n",
      "Document ADH-1 - similarity: 0.08934\n",
      "Document Haemagglutinin-strain B (Yamagata) - similarity: 0.08918\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 (H3N2) recombinant hemagglutinin antigen - similarity: 0.08916\n",
      "Document beta-cellotriose - similarity: 0.08883\n",
      "Document Coprogen - similarity: 0.08877\n",
      "Document Iodobenzene - similarity: 0.08859\n",
      "Document Skf 107457 - similarity: 0.08844\n",
      "Document 9-Methylguanine - similarity: 0.08828\n",
      "Document Cyclohexyl-{4-[5-(3,4-Dichlorophenyl)-2-Piperidin-4-Yl-3-Propyl-3h-Imidazol-4-Yl]-Pyrimidin-2-Yl}Amine - similarity: 0.08822\n",
      "Document Protirelin - similarity: 0.08820\n",
      "Document Meadowfoam seed oil - similarity: 0.08781\n",
      "Document Mozenavir - similarity: 0.08770\n",
      "Document Flurandrenolide acetate - similarity: 0.08743\n",
      "Document D-Asparagine - similarity: 0.08734\n",
      "Document Dianexin - similarity: 0.08717\n",
      "Document Nardostachys chinensis whole - similarity: 0.08716\n",
      "Document SC12267 - similarity: 0.08709\n",
      "Document BG-777 - similarity: 0.08683\n",
      "Document Phosphorylated Dihydropteroate - similarity: 0.08663\n",
      "Document Kaempferia galanga root - similarity: 0.08660\n",
      "Document AC-(D)PHE-PRO-BOROHOMOORNITHINE-OH - similarity: 0.08647\n",
      "Document (2s)-2-Amino-4-(Methylsulfanyl)-1-Pyridin-2-Ylbutane-1,1-Diol - similarity: 0.08641\n",
      "Document Rapastinel - similarity: 0.08633\n",
      "Document Epibatidine - similarity: 0.08603\n",
      "Document Acetoacetic acid - similarity: 0.08588\n",
      "Document CRA_10991 - similarity: 0.08570\n",
      "Document Nα-[(2S)-2-{[(S)-[(1S)-1-{[(Benzyloxy)carbonyl]amino}-2-phenylethyl](hydroxy)phosphoryl]methyl}-5-phenylpentanoyl]-L-tryptophanamide - similarity: 0.08565\n",
      "Document 2-Amino-3-(3-Hydroxy-7,8-Dihydro-6h-Cyclohepta[D]-4-Isoxazolyl)Propionic Acid - similarity: 0.08560\n",
      "Document Dansyllysine - similarity: 0.08521\n",
      "Document Idrocilamide - similarity: 0.08502\n",
      "Document N-(2-chloro-6-methylphenyl)-8-[(3S)-3-methylpiperazin-1-yl]imidazo[1,5-a]quinoxalin-4-amine - similarity: 0.08495\n",
      "Document VIBRIO CHOLERAE INABA 6973 E1 TOR BIOTYPE, FORMALIN INACTIVATED - similarity: 0.08474\n",
      "Document alpha-Tocopherol succinate - similarity: 0.08472\n",
      "Document Pyromellitic Acid - similarity: 0.08466\n",
      "Document 4,7-Dioxosebacic Acid - similarity: 0.08446\n",
      "Document 6-[2-(1H-INDOL-6-YL)ETHYL]PYRIDIN-2-AMINE - similarity: 0.08427\n",
      "Document Midomafetamine - similarity: 0.08419\n",
      "Document Guanadrel - similarity: 0.08418\n",
      "Document Human vaccinia virus immune globulin - similarity: 0.08417\n",
      "Document 5-[2-(TRIFLUOROMETHYL)PHENYL]-2-FUROIC ACID - similarity: 0.08396\n",
      "Document (2Z)-1-(5-Chloro-1H-indol-3-yl)-3-hydroxy-3-(1H-tetrazol-5-yl)-2-propen-1-one - similarity: 0.08379\n",
      "Document Chromium Cr-51 - similarity: 0.08366\n",
      "Document Darolutamide - similarity: 0.08348\n",
      "Document (2S,2'S)-2,2'-(1,2-Hydrazinediylbis{methylene[(2S)-1-oxo-2,1-hexanediyl]imino})bis(6-amino-N-phenylhexanamide) - similarity: 0.08344\n",
      "Document 3-(1H-BENZIMIDAZOL-2-YL)-1H-INDAZOLE - similarity: 0.08330\n",
      "Document [Methylthio]Acetate - similarity: 0.08305\n",
      "Document CRx-119 - similarity: 0.08288\n",
      "Document 4-(1,3,2-Dioxaborolan-2-Yloxy)Butan-1-Aminium - similarity: 0.08285\n",
      "Document ANA971 - similarity: 0.08228\n",
      "Document DDP-225 - similarity: 0.08227\n",
      "Document N,N-dimethylglycine - similarity: 0.08216\n",
      "Document 2-[(1R)-1-carboxy-2-naphthalen-1-ylethyl]-1,3-dioxo-2,3-dihydro-1H-isoindole-5-carboxylic acid - similarity: 0.08195\n",
      "Document Methaqualone - similarity: 0.08188\n",
      "Document (5-methyl-6-oxo-1,6-dihydro-pyridin-3-yl)-1,2-dideoxy-ribofuranose-5-monophosphate - similarity: 0.08182\n",
      "Document 3-(3-methoxybenzyl)-1H-pyrrolo[2,3-b]pyridine - similarity: 0.08180\n",
      "Document Porphyrin Fe(III) - similarity: 0.08172\n",
      "Document 2-(2-{2-[2-(2-{2-[2-(2-Ethoxy-Ethoxy)-Ethoxy]-Ethoxy}-Ethoxy)-Ethoxy]-Ethoxy}-Ethoxy)-Ethanol, Polyethyleneglycol Peg400 - similarity: 0.08153\n",
      "Document Nicotinamide Mononucleotide - similarity: 0.08145\n",
      "Document Simotaxel - similarity: 0.08121\n",
      "Document 3-(4-carbamoyl-1-carboxy-2-methylsulfonyl-buta-1,3-dienylamino)-indolizine-2-carboxylic acid - similarity: 0.08087\n",
      "Document 1,4-Butanediol - similarity: 0.08058\n",
      "Document Magaldrate - similarity: 0.08055\n",
      "Document N-[amino(imino)methyl]-2-(2,5-diphenyl-1H-pyrrol-1-yl)acetamide - similarity: 0.08051\n",
      "Document Aminopentamide - similarity: 0.08035\n",
      "Document AP5346 - similarity: 0.08034\n",
      "Document Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.08017\n",
      "Document Cpad - similarity: 0.08015\n",
      "Document Clinafloxacin - similarity: 0.08011\n",
      "Document Salophen-10-carboxylate iron chelate - similarity: 0.08010\n",
      "Document Triacetylchitotriose - similarity: 0.08002\n",
      "Document ALS-08 - similarity: 0.07994\n",
      "Document Allantoate - similarity: 0.07988\n",
      "Document 2,4-Diamino-6-[N-(2',5'-Dimethoxybenzyl)-N-Methylamino]Quinazoline - similarity: 0.07979\n",
      "Document Apple fruit oil - similarity: 0.07957\n",
      "Document 1-(5-OXO-2,3,5,9B-TETRAHYDRO-1H-PYRROLO[2,1-A]ISOINDOL-9-YL)-3-(5-PYRROLIDIN-2-YL-1H-PYRAZOL-3-YL)-UREA - similarity: 0.07952\n",
      "Document Forsythia suspensa fruit - similarity: 0.07946\n",
      "Document Ferumoxides - similarity: 0.07946\n",
      "Document (2R)-3-[(tetrahydroxyphosphoranyl)oxy]-1,2-propanediyl dibutanoate - similarity: 0.07942\n",
      "Document D-Glutamic Acid - similarity: 0.07931\n",
      "Document Betazole - similarity: 0.07919\n",
      "Document (5S)-2-{[(1S)-1-(4-Fluorophenyl)ethyl]amino}-5-(2-hydroxy-2-propanyl)-5-methyl-1,3-thiazol-4(5H)-one - similarity: 0.07884\n",
      "Document 4-HYDROXY-N'-(4-ISOPROPYLBENZYL)BENZOHYDRAZIDE - similarity: 0.07874\n",
      "Document Zinc glycinate citrate - similarity: 0.07836\n",
      "Document BMS-986012 - similarity: 0.07817\n",
      "Document Mepenzolate - similarity: 0.07814\n",
      "Document 5-Aminouracil - similarity: 0.07807\n",
      "Document Bromamphenicol - similarity: 0.07799\n",
      "Document Monoctanoin - similarity: 0.07796\n",
      "Document Fedrilate - similarity: 0.07789\n",
      "Document Coagulation factor X human - similarity: 0.07775\n",
      "Document YKP-1358 - similarity: 0.07763\n",
      "Document 1-Hexadecanosulfonyl-O-L-Serine - similarity: 0.07747\n",
      "Document Siplizumab - similarity: 0.07747\n",
      "Document 4-Hydroxyphenylpyruvic acid - similarity: 0.07741\n",
      "Document Dihydromorphine - similarity: 0.07695\n",
      "Document Priralfinamide - similarity: 0.07687\n",
      "Document 6-[4-(2-fluorophenyl)-1,3-oxazol-5-yl]-N-(1-methylethyl)-1,3-benzothiazol-2-amine - similarity: 0.07672\n",
      "Document 2,5-DIPHENYLFURAN-3,4-DICARBOXYLIC ACID - similarity: 0.07670\n",
      "Document Setipiprant - similarity: 0.07662\n",
      "Document [(2-Ethoxy-1-Naphthoyl)Amino]Methylboronic Acid - similarity: 0.07658\n",
      "Document Ceramide NG - similarity: 0.07637\n",
      "Document Cp403700, (S)-1-{2-[(5-Chloro-1h-Indole-2-Carbonyl)-Amino]-3-Phenyl-Propionyl}-Azetidine-3-Carboxylate - similarity: 0.07637\n",
      "Document Sodium phosphate tribasic dodecahydrate - similarity: 0.07634\n",
      "Document SALOPHEN-10-PROPIONATE IRON CHELATE - similarity: 0.07630\n",
      "Document (3R)-4-{[(3,4-dihydroxyphenyl)acetyl]oxy}-N-(2-formylindolizin-3-yl)-3-sulfino-D-valine - similarity: 0.07600\n",
      "Document Myxothiazol - similarity: 0.07569\n",
      "Document Leymus condensatus pollen - similarity: 0.07534\n",
      "Document Etripamil - similarity: 0.07489\n",
      "Document TA-CIN - similarity: 0.07459\n",
      "Document Medrysone - similarity: 0.07456\n",
      "Document Phenyl hydrogen sulfate - similarity: 0.07453\n",
      "Document Domiphen - similarity: 0.07440\n",
      "Document 1(R)-1-Acetamido-2-(3-Carboxyphenyl)Ethyl Boronic Acid - similarity: 0.07439\n",
      "Document IDM-2 - similarity: 0.07414\n",
      "Document Hyaluronidase (ovine) - similarity: 0.07394\n",
      "Document 2-Hydroxyethyl Disulfide - similarity: 0.07362\n",
      "Document 2-(Pyrido[1,2-E]Purin-4-Yl)Amino-Ethanol - similarity: 0.07359\n",
      "Document Virginiamycin S1 - similarity: 0.07359\n",
      "Document BZL101 - similarity: 0.07344\n",
      "Document Butyramide - similarity: 0.07335\n",
      "Document R-2-{[4'-Methoxy-(1,1'-Biphenyl)-4-Yl]-Sulfonyl}-Amino-6-Methoxy-Hex-4-Ynoic Acid - similarity: 0.07323\n",
      "Document Calusterone - similarity: 0.07292\n",
      "Document Biphenyl-2,3-Diol - similarity: 0.07270\n",
      "Document Fursultiamine - similarity: 0.07261\n",
      "Document Filaminast - similarity: 0.07252\n",
      "Document PV903 - similarity: 0.07249\n",
      "Document (6E,11E)-HEPTADECA-6,11-DIENE-9,9-DIYLBIS(PHOSPHONIC ACID) - similarity: 0.07242\n",
      "Document Adrafinil - similarity: 0.07230\n",
      "Document N-Butyl Isocyanide - similarity: 0.07190\n",
      "Document Daporinad - similarity: 0.07188\n",
      "Document CNV-2197944 - similarity: 0.07179\n",
      "Document 3-{[(2,2-dioxido-1,3-dihydro-2-benzothien-5-yl)amino]methylene}-5-(1,3-oxazol-5-yl)-1,3-dihydro-2H-indol-2-one - similarity: 0.07177\n",
      "Document 2'-Deoxy-Thymidine-Beta-L-Rhamnose - similarity: 0.07165\n",
      "Document octyl 3-deoxy-2-O-(6-deoxy-alpha-L-galactopyranosyl)-beta-D-xylo-hexopyranoside - similarity: 0.07165\n",
      "Document 3'-Deoxy 3'-Amino Adenosine-5'-Diphosphate - similarity: 0.07163\n",
      "Document 3-Deazaadenosine - similarity: 0.07159\n",
      "Document Eucommia ulmoides bark - similarity: 0.07136\n",
      "Document (2R,3R)-7-(methylsulfonyl)-3-(2,4,5-trifluorophenyl)-1,2,3,4-tetrahydropyrido[1,2-a]benzimidazol-2-amine - similarity: 0.07133\n",
      "Document Benzene Hexacarboxylic Acid - similarity: 0.07091\n",
      "Document AL-4623 - similarity: 0.07058\n",
      "Document 2,3-Difluorobenzyl Alcohol - similarity: 0.07045\n",
      "Document Sparfosic acid - similarity: 0.07035\n",
      "Document P1-(adenosine-5'-P5-(uridine-5')pentaphosphate - similarity: 0.07011\n",
      "Document Pentaglyme - similarity: 0.06996\n",
      "Document N,N-[2,5-O-dibenzyl-glucaryl]-DI-[valinyl-aminomethanyl-pyridine] - similarity: 0.06990\n",
      "Document Dibromotyrosine - similarity: 0.06972\n",
      "Document 2-{3-[(2S)-4,4-difluoro-2-(pyrrolidin-1-ylcarbonyl)pyrrolidin-1-yl]-3-oxopropyl}-isoindole-1,3(2H)-dione - similarity: 0.06960\n",
      "Document 3,4-Dihydro-2h-Pyrrolium-5-Carboxylate - similarity: 0.06957\n",
      "Document Influenza A virus A/Singapore/INFIMH-16-0019/2016 (H3N2) recombinant hemagglutinin antigen - similarity: 0.06951\n",
      "Document Chromic cation - similarity: 0.06933\n",
      "Document Gepirone - similarity: 0.06917\n",
      "Document Tipranavir C-14 - similarity: 0.06896\n",
      "Document Haemagglutinin-strain A(H3N2) - similarity: 0.06886\n",
      "Document 6-(2,3,4,5,6,7-HEXAHYDRO-2,4,4-TRIMETHYL-1-METYLENEINDEN-2-YL)-3-METHYLHEXA-2,4-DIENOIC ACID - similarity: 0.06856\n",
      "Document SCY-635 - similarity: 0.06836\n",
      "Document Pacific ocean perch - similarity: 0.06804\n",
      "Document Methoxy PEG-12 retinamide - similarity: 0.06792\n",
      "Document Inarigivir - similarity: 0.06775\n",
      "Document Lulizumab pegol - similarity: 0.06768\n",
      "Document 2'-Deoxyguanosine-5'-Triphosphate - similarity: 0.06726\n",
      "Document Cyclachaena xanthifolia pollen - similarity: 0.06717\n",
      "Document 4-(ACETYLAMINO)-5-AMINO-3-HYDROXYBENZOIC ACID - similarity: 0.06697\n",
      "Document beta-L-fucose - similarity: 0.06691\n",
      "Document bis(5-amidino-benzimidazolyl)methanone zinc - similarity: 0.06685\n",
      "Document 3-({4-[(5-CHLORO-1,3-BENZODIOXOL-4-YL)AMINO]PYRIMIDIN-2-YL}AMINO)BENZAMIDE - similarity: 0.06681\n",
      "Document Hexobarbital - similarity: 0.06664\n",
      "Document N-{(1S,2R)-2-hydroxy-1-[(hydroxyamino)carbonyl]propyl}-4-{[4-(morpholin-4-ylmethyl)phenyl]ethynyl}benzamide - similarity: 0.06644\n",
      "Document D-phenylalanyl-N-(3-fluorobenzyl)-L-prolinamide - similarity: 0.06642\n",
      "Document 4-O-(4,6-Dideoxy-4-{[4,5,6-Trihydroxy-3-(Hydroxymethyl)Cyclohex-2-En-1-Yl]Amino}-Beta-D-Lyxo-Hexopyranosyl)-Alpha-D-Erythro-Hexopyranose - similarity: 0.06624\n",
      "Document Catridecacog - similarity: 0.06607\n",
      "Document (S)-N-(4-carbamimidoylbenzyl)-1-(2-(cyclohexyloxy)ethanoyl)pyrrolidine-2-carboxamide - similarity: 0.06606\n",
      "Document N-cyclopropyl-6-[(6,7-dimethoxyquinolin-4-yl)oxy]naphthalene-1-carboxamide - similarity: 0.06585\n",
      "Document BK-MDA - similarity: 0.06571\n",
      "Document Norcamphor - similarity: 0.06569\n",
      "Document (R)-1-phenylethanol - similarity: 0.06563\n",
      "Document Hydroxyfasudil - similarity: 0.06548\n",
      "Document Pegamotecan - similarity: 0.06507\n",
      "Document Hexafluronium - similarity: 0.06494\n",
      "Document Epimestrol - similarity: 0.06485\n",
      "Document 11-Deoxy-Beta-Rhodomycin - similarity: 0.06471\n",
      "Document 6-hydroxy-1,6-dihydro purine nucleoside - similarity: 0.06463\n",
      "Document 3-METHOXY-6-[4-(3-METHYLPHENYL)-1-PIPERAZINYL]PYRIDAZINE - similarity: 0.06425\n",
      "Document 2-(Biphenyl-4-Sulfonyl)-1,2,3,4-Tetrahydro-Isoquinoline-3-Carboxylic Acid - similarity: 0.06416\n",
      "Document 2,5-bis-O-{3-[amino(imino)methyl]phenyl}-1,4:3,6-dianhydro-D-glucitol - similarity: 0.06404\n",
      "Document Cuprous chloride - similarity: 0.06394\n",
      "Document Zenarestat - similarity: 0.06393\n",
      "Document 4-guanidinobenzoic acid - similarity: 0.06391\n",
      "Document Morinamide - similarity: 0.06369\n",
      "Document Thyroid, porcine - similarity: 0.06360\n",
      "Document 9-Hydroxy-8-Methoxy-6-Nitro-Phenanthrol[3,4-D][1,3]Dioxole-5-Carboxylic Acid - similarity: 0.06352\n",
      "Document Honeydew melon - similarity: 0.06348\n",
      "Document 4-[3-(1H-BENZIMIDAZOL-2-YL)-1H-INDAZOL-6-YL]-2-METHOXYPHENOL - similarity: 0.06316\n",
      "Document 3-Deazacytidine - similarity: 0.06313\n",
      "Document 3-[2,6,8-Trioxo-9-[(2R,3S,4R)-2,3,4,5-tetrahydroxypentyl]-3H-purin-7-yl]propyl dihydrogen phosphate - similarity: 0.06290\n",
      "Document SAR-405838 - similarity: 0.06280\n",
      "Document (2S)-2-amino-3-phenylpropane-1,1-diol - similarity: 0.06274\n",
      "Document Transcrocetinate - similarity: 0.06265\n",
      "Document Cyclohexyl-Hexyl-Beta-D-Maltoside - similarity: 0.06258\n",
      "Document Alpha-Cyclodextrin (Cyclohexa-Amylose) - similarity: 0.06245\n",
      "Document 5-(2-fluorophenyl)-N-(pyridin-4-ylmethyl)pyrazolo[1,5-a]pyrimidin-7-amine - similarity: 0.06222\n",
      "Document Tert-butyl N-[(2S,5R)-6-[[(2S)-5-amino-1-[[(2S)-1-amino-1-oxo-3-phenylpropan-2-yl]amino]-1,5-dioxopentan-2-yl]amino]-5-benzyl-3,6-dioxo-1-phenylhexan-2-yl]carbamate - similarity: 0.06208\n",
      "Document Bisoxatin acetate - similarity: 0.06207\n",
      "Document Recombinant alpha 1-antitrypsin - similarity: 0.06202\n",
      "Document 4-[3-(dibenzylamino)phenyl]-2,4-dioxobutanoic acid - similarity: 0.06190\n",
      "Document 3-OXO-N-[(3S)-2-OXOPYRROLIDIN-3-YL]DODECANAMIDE - similarity: 0.06179\n",
      "Document 9-Amino-2-deoxy-2,3-dehydro-n-acetyl-neuraminic acid - similarity: 0.06165\n",
      "Document TO-901317 - similarity: 0.06131\n",
      "Document N-Hydroxy-4-[(4-Methoxylphenyl)Sulfonyl]-2,2-Dimethyl-Hexahydro-1,4-Thiazepine-3(S)-Carboxamide - similarity: 0.06129\n",
      "Document D-Eritadenine - similarity: 0.06121\n",
      "Document N-{[6-(PENTYLOXY)NAPHTHALEN-2-YL]SULFONYL}-D-GLUTAMIC ACID - similarity: 0.06110\n",
      "Document [2'-HYDROXY-3'-(1H-PYRROLO[3,2-C]PYRIDIN-2-YL)-BIPHENYL-3-YLMETHYL]-UREA - similarity: 0.06056\n",
      "Document 2-(2,6-DICHLOROPHENYL)-1,3-BENZOXAZOLE-6-CARBOXYLIC ACID - similarity: 0.06050\n",
      "Document D-erythrose 4-phosphate - similarity: 0.06031\n",
      "Document (5Z)-5-(3-BROMOCYCLOHEXA-2,5-DIEN-1-YLIDENE)-N-(PYRIDIN-4-YLMETHYL)-1,5-DIHYDROPYRAZOLO[1,5-A]PYRIMIDIN-7-AMINE - similarity: 0.05989\n",
      "Document N'-(5-CHLORO-1,3-BENZODIOXOL-4-YL)-N-(3,4,5- TRIMETHOXYPHENYL)PYRIMIDINE-2,4-DIAMINE - similarity: 0.05981\n",
      "Document N-Pyridoxyl-7-Keto-8-Aminopelargonic Acid-5'-Monophosphate - similarity: 0.05979\n",
      "Document 1-{[N-(1-Imino-guanidino-methyl)]sulfanylmethyl}-3-trifluoromethyl-benzene - similarity: 0.05964\n",
      "Document 2-KETO-6-PHOSPHATE-D-GLUCONIC ACID, ALPHA-FURANOSE FORM - similarity: 0.05953\n",
      "Document Aspartyl-Adenosine-5'-Monophosphate - similarity: 0.05948\n",
      "Document Baccharis sarothroides whole - similarity: 0.05931\n",
      "Document Dodecyldimethylamine N-oxide - similarity: 0.05931\n",
      "Document PG-530742 - similarity: 0.05930\n",
      "Document L-2,4-diaminobutyric acid - similarity: 0.05921\n",
      "Document S-Benzylcysteine - similarity: 0.05888\n",
      "Document Bavisant - similarity: 0.05868\n",
      "Document Guanazodine - similarity: 0.05839\n",
      "Document Pramiracetam - similarity: 0.05825\n",
      "Document 6-Hydroxy-6-Methyl-Heptan-3-One - similarity: 0.05806\n",
      "Document 2-({5-CHLORO-2-[(2-METHOXY-4-MORPHOLIN-4-YLPHENYL)AMINO]PYRIMIDIN-4-YL}AMINO)-N-METHYLBENZAMIDE - similarity: 0.05804\n",
      "Document Ambrosia dumosa pollen - similarity: 0.05786\n",
      "Document 1,2-Dichloro-Propane - similarity: 0.05734\n",
      "Document AN0128 - similarity: 0.05722\n",
      "Document N-Carbamyl-D-Methionine - similarity: 0.05712\n",
      "Document 4-(2,2,2-TRIFLUOROETHYL)-L-PHENYLALANINE - similarity: 0.05660\n",
      "Document 1-{3-[(4-pyridin-2-ylpiperazin-1-yl)sulfonyl]phenyl}-3-(1,3-thiazol-2-yl)urea - similarity: 0.05649\n",
      "Document 2,4-Dihydroxy-3,3-Dimethyl-Butyrate - similarity: 0.05631\n",
      "Document Tiemonium iodide - similarity: 0.05575\n",
      "Document Rovafovir etalafenamide - similarity: 0.05532\n",
      "Document Rumex crispus whole - similarity: 0.05522\n",
      "Document (2Z)-2-(Benzoylamino)-3-[4-(2-bromophenoxy)phenyl]acrylic acid - similarity: 0.05482\n",
      "Document 4-hydroxyphenacyl coenzyme A - similarity: 0.05478\n",
      "Document 4-(3-Pyridin-2-Yl-1h-Pyrazol-4-Yl)Quinoline - similarity: 0.05455\n",
      "Document 1-(2-Fluorobenzyl)-3-Butyl-8-(N-Acetyl-4-Aminobenzyl)-Xanthine - similarity: 0.05450\n",
      "Document XL844 - similarity: 0.05439\n",
      "Document Nicotinamide 8-bromo-adenine dinucleotide phosphate - similarity: 0.05433\n",
      "Document Cis-[4,5-Bis-(4-Bromophenyl)-2-(2-Ethoxy-4-Methoxyphenyl)-4,5-Dihydroimidazol-1-Yl]-[4-(2-Hydroxyethyl)Piperazin-1-Yl]Methanone - similarity: 0.05418\n",
      "Document RSV-604 - similarity: 0.05415\n",
      "Document Dpb-T - similarity: 0.05409\n",
      "Document N-[2-(1-Maleimidyl)Ethyl]-7-Diethylaminocoumarin-3-Carboxamide - similarity: 0.05407\n",
      "Document Flavin-N7 protonated-adenine dinucleotide - similarity: 0.05405\n",
      "Document Sapitinib - similarity: 0.05404\n",
      "Document Mucor racemosus - similarity: 0.05373\n",
      "Document N-Carbamoylaspartic acid - similarity: 0.05370\n",
      "Document Adenylosuccinic acid - similarity: 0.05360\n",
      "Document {[(2,2-Dihydroxy-Ethyl)-(2,3,4,5-Tetrahydroxy-6-Phosphonooxy-Hexyl)-Amino]-Methyl}-Phosphonic Acid - similarity: 0.05342\n",
      "Document Ditiocarb - similarity: 0.05323\n",
      "Document WRR-99 - similarity: 0.05317\n",
      "Document Beta-D-Glucose - similarity: 0.05313\n",
      "Document Benzo[B]Thiophene-2-Boronic Acid - similarity: 0.05296\n",
      "Document 4-Oxosebacic Acid - similarity: 0.05254\n",
      "Document Radium Ra 223 dichloride - similarity: 0.05245\n",
      "Document HKI-357 - similarity: 0.05218\n",
      "Document Pegademase - similarity: 0.05204\n",
      "Document RU85493 - similarity: 0.05199\n",
      "Document N-{(3S,4S)-4-[(6-AMINO-4-METHYLPYRIDIN-2-YL)METHYL]PYRROLIDIN-3-YL}-N'-(4-CHLOROBENZYL)ETHANE-1,2-DIAMINE - similarity: 0.05185\n",
      "Document CPG-52852 - similarity: 0.05180\n",
      "Document Influenza B virus B/Maryland/15/2016 antigen (formaldehyde inactivated) - similarity: 0.05164\n",
      "Document (4aS,4bR,10bS,12aS)-12a-methyl-1,3-dioxo-2-(pyridin-3-ylmethyl)-1,2,3,4,4a,4b,5,6,10b,11,12,12a-dodecahydronaphtho[2,1-f]isoquinolin-8-yl sulfamate - similarity: 0.05139\n",
      "Document S-(D-Carboxybutyl)-L-Homocysteine - similarity: 0.05110\n",
      "Document (2E)-3-(2,4-DICHLOROPHENYL)-N-HYDROXYACRYLAMIDE - similarity: 0.05097\n",
      "Document HSV-2 theracine - similarity: 0.05094\n",
      "Document [2-(Methyleneamine)-4-(4-Hydroxy-Benzylidine)-5-Oxo-4,5-Dihydro-Imidazol-1-Yl]-Acetaldehyde - similarity: 0.05092\n",
      "Document 4-[(Isopropylamino)Methyl]Phenylalanine - similarity: 0.05089\n",
      "Document TP-508 - similarity: 0.05088\n",
      "Document Quinidine barbiturate - similarity: 0.05062\n",
      "Document 18-CHLORO-11,12,13,14-TETRAHYDRO-1H,10H-8,4-(AZENO)-9,15,1,3,6-BENZODIOXATRIAZACYCLOHEPTADECIN-2-ONE - similarity: 0.05034\n",
      "Document Carbazole Butanoic Acid - similarity: 0.05009\n",
      "Document Coproporphyrin I containing CO(III) - similarity: 0.04997\n",
      "Document Betula papyrifera pollen - similarity: 0.04996\n",
      "Document Haemophilus influenzae type B strain 1482 capsular polysaccharide tetanus toxoid conjugate antigen - similarity: 0.04987\n",
      "Document Strontium chloride Sr-89 - similarity: 0.04957\n",
      "Document 4-(trifluoromethyl)phenol - similarity: 0.04951\n",
      "Document (Diphosphono)Aminophosphonic Acid - similarity: 0.04949\n",
      "Document 4-Methyl-2,5-dimethoxyamphetamine - similarity: 0.04930\n",
      "Document 4-{[(1R,2S)-1,2-dihydroxy-2-methyl-3-(4-nitrophenoxy)propyl]amino}-2-(trifluoromethyl)benzonitrile - similarity: 0.04918\n",
      "Document Isopenicillin N - similarity: 0.04907\n",
      "Document [{(5-Chloro-2-Pyridinyl)Amino} Methylene]-1,1-Bisphosphonate - similarity: 0.04882\n",
      "Document 1-Palmitoyl-2-oleoyl-sn-glycero-3-(phospho-rac-(1-glycerol)) - similarity: 0.04844\n",
      "Document Etacrynic acid - similarity: 0.04822\n",
      "Document Methylenedioxyethamphetamine - similarity: 0.04786\n",
      "Document 4-Phenyl-1h-Imidazole - similarity: 0.04783\n",
      "Document N-methyl-1-(2-thiophen-2-ylphenyl)methanamine - similarity: 0.04773\n",
      "Document CVT-6883 - similarity: 0.04768\n",
      "Document Thymidine 5'-triphosphate - similarity: 0.04755\n",
      "Document Neisseria meningitidis group c capsular oligosaccharide diphtheria crm197 protein conjugate antigen - similarity: 0.04703\n",
      "Document Tecadenoson - similarity: 0.04663\n",
      "Document 3,4-Biphenyldiol - similarity: 0.04642\n",
      "Document N-(2-CHLORO-4-FLUOROBENZOYL)-N'-(5-HYDROXY-2-METHOXYPHENYL)UREA - similarity: 0.04572\n",
      "Document 3-(Phosphonomethyl)Pyridine-2-Carboxylic Acid - similarity: 0.04565\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) hemagglutinin antigen (UV, formaldehyde inactivated) - similarity: 0.04559\n",
      "Document RU79073 - similarity: 0.04557\n",
      "Document Isatoribine - similarity: 0.04548\n",
      "Document 3-Chloro-4-Hydroxyphenylglycine - similarity: 0.04505\n",
      "Document 3-(10-methyl-9-anthryl)propanoic acid - similarity: 0.04482\n",
      "Document N-Benzyl-4-Sulfamoyl-Benzamide - similarity: 0.04474\n",
      "Document Etozoline - similarity: 0.04467\n",
      "Document nor-NOHA - similarity: 0.04435\n",
      "Document Ramatercept - similarity: 0.04432\n",
      "Document Deacetylbisacodyl - similarity: 0.04423\n",
      "Document 3-Indolebutyric Acid - similarity: 0.04414\n",
      "Document 5-[3-(BENZYLAMINO)PHENYL]-4-BROMO-3-(CARBOXYMETHOXY)THIOPHENE-2-CARBOXYLIC ACID - similarity: 0.04407\n",
      "Document [[N-(Benzyloxycarbonyl)Amino]Methyl]Phosphate - similarity: 0.04389\n",
      "Document Beta-1,2,3,4,6-Penta-O-Galloyl-D-Glucopyranose - similarity: 0.04380\n",
      "Document English walnut - similarity: 0.04354\n",
      "Document Sulfameter - similarity: 0.04354\n",
      "Document Methyldiazene - similarity: 0.04352\n",
      "Document 2-Hydroxymethyl-Pyrrolidine-3,4-Diol - similarity: 0.04344\n",
      "Document 4-{(1E)-3-OXO-3-[(2-PHENYLETHYL)AMINO]PROP-1-EN-1-YL}-1,2-PHENYLENE DIACETATE - similarity: 0.04322\n",
      "Document (R)-4-Nitrostyrene oxide - similarity: 0.04311\n",
      "Document IPH 1101 - similarity: 0.04307\n",
      "Document Liquidambar styraciflua pollen - similarity: 0.04299\n",
      "Document Sodium cellulose phosphate - similarity: 0.04244\n",
      "Document PF-03715455 - similarity: 0.04223\n",
      "Document D-Proline - similarity: 0.04221\n",
      "Document Thiazinam - similarity: 0.04220\n",
      "Document 2,4-Diamino-6-[N-(3',4',5'-Trimethoxybenzyl)-N-Methylamino]Pyrido[2,3-D]Pyrimidine - similarity: 0.04220\n",
      "Document ABX-464 - similarity: 0.04212\n",
      "Document 2-Iminiopropanoate - similarity: 0.04178\n",
      "Document N-(1-benzylpiperidin-4-yl)-4-sulfanylbutanamide - similarity: 0.04177\n",
      "Document 5-Benzylacyclouridine - similarity: 0.04125\n",
      "Document 1,6-DI-O-PHOSPHONO-D-MANNITOL - similarity: 0.04123\n",
      "Document Indiplon - similarity: 0.04120\n",
      "Document (S)-2-CHLORO-N-(1-(2-(2-HYDROXYETHYLAMINO)-2-OXOETHYL)-2-OXO-1,2,3,4-TETRAHYDROQUINOLIN-3-YL)-6H-THIENO[2,3-B]PYRROLE-5-CARBOXAMIDE - similarity: 0.04120\n",
      "Document N'-(2s,3r)-3-Amino-4-Cyclohexyl-2-Hydroxy-Butano-N-(4-Methylphenyl)Hydrazide - similarity: 0.04110\n",
      "Document Aluminium monostearate - similarity: 0.04085\n",
      "Document Etheno-NAD - similarity: 0.04084\n",
      "Document (3R)-3-(FLUOROMETHYL)-N-(3,3,3-TRIFLUOROPROPYL)-1,2,3,4-TETRAHYDROISOQUINOLINE-7-SULFONAMIDE - similarity: 0.04076\n",
      "Document 1-Deoxy-1-acetylamino-beta-D-gluco-2-heptulopyranosonamide - similarity: 0.04069\n",
      "Document Field mushroom - similarity: 0.04066\n",
      "Document 1-(1-methylethyl)-3-quinolin-6-yl-1H-pyrazolo[3,4-d]pyrimidin-4-amine - similarity: 0.04051\n",
      "Document N-[3-(aminomethyl)benzyl]acetamidine - similarity: 0.04051\n",
      "Document 2-[2-(2-FLUOROPHENYL)PYRIDIN-4-YL]-1,5,6,7-TETRAHYDRO-4H-PYRROLO[3,2-C]PYRIDIN-4-ONE - similarity: 0.04003\n",
      "Document Ambrosia psilostachya pollen - similarity: 0.03967\n",
      "Document Emylcamate - similarity: 0.03966\n",
      "Document OSI-7836 - similarity: 0.03917\n",
      "Document 4-(Acetylamino)-3-Amino Benzoic Acid - similarity: 0.03904\n",
      "Document 1-[4-Carboxy-2-(3-Pentylamino)Phenyl]-5,5'-Di(Hydroxymethyl)Pyrrolidin-2-One - similarity: 0.03901\n",
      "Document DNB-001 - similarity: 0.03900\n",
      "Document Thiorphan - similarity: 0.03893\n",
      "Document (2S)-2-amino-6-[[(3R,4R)-1-[(1S)-1-carboxy-2-[(S)-methylsulfinyl]ethyl]-2-oxo-4-sulfanylazetidin-3-yl]amino]-6-oxohexanoic acid - similarity: 0.03850\n",
      "Document 4,6-Dideoxy-4-{[4,5,6-Trihydroxy-3-(Hydroxymethyl)Cyclohex-2-En-1-Yl]Amino}-Alpha-D-Lyxo-Hexopyranosyl-(1->4)-Alpha-D-Threo-Hexopyranosyl-(1->6)-Alpha-L-Threo-Hexopyranose - similarity: 0.03832\n",
      "Document 2,8-bis[oxido(oxo)vanadio]-1,1,1,3,5,5,7,7,9,9,9-undecaoxopentavanadoxane-2,8-diium - similarity: 0.03816\n",
      "Document N1,N14-Bis((S-Methyl)Isothioureido)Tetradecane - similarity: 0.03796\n",
      "Document Silver sulfadiazine - similarity: 0.03781\n",
      "Document HE3286 - similarity: 0.03778\n",
      "Document NOX-700 - similarity: 0.03772\n",
      "Document 2-deoxy-2-fluoro-β-D-galactose - similarity: 0.03759\n",
      "Document 1-[3-({[(4-AMINO-5-FLUORO-2-METHYLQUINOLIN-3-YL)METHYL]THIO}METHYL)PHENYL]-2,2,2-TRIFLUOROETHANE-1,1-DIOL - similarity: 0.03722\n",
      "Document AVI-4557 - similarity: 0.03710\n",
      "Document 5-[4-(DIMETHYLAMINO)PHENYL]-6-[(6-MORPHOLIN-4-YLPYRIDIN-3-YL)ETHYNYL]PYRIMIDIN-4-AMINE - similarity: 0.03668\n",
      "Document 3-[1-(3-AMINOPROPYL)-1H-INDOL-3-YL]-4-(1H-INDOL-3-YL)-1H-PYRROLE-2,5-DIONE - similarity: 0.03647\n",
      "Document Diazolidinylurea - similarity: 0.03633\n",
      "Document ATG002 - similarity: 0.03598\n",
      "Document Diisopropylphosphono Group - similarity: 0.03587\n",
      "Document Methoxy arachidonyl fluorophosphonate - similarity: 0.03584\n",
      "Document Bephenium - similarity: 0.03579\n",
      "Document 6-hydroxymethylpterin diphosphate - similarity: 0.03561\n",
      "Document Calcium levulinate - similarity: 0.03527\n",
      "Document Metacycline - similarity: 0.03526\n",
      "Document gamma-Glutamyl[S-(2-iodobenzyl)cysteinyl]glycine - similarity: 0.03524\n",
      "Document 1,1,1-TRIFLUORO-3-(OCTYLTHIO)ACETONE - similarity: 0.03519\n",
      "Document PF-05241328 - similarity: 0.03492\n",
      "Document Poly(lactic acid) - similarity: 0.03486\n",
      "Document LY-2624803 - similarity: 0.03482\n",
      "Document 1-[4-(AMINOMETHYL)BENZOYL]-5'-FLUORO-1'H-SPIRO[PIPERIDINE-4,2'-QUINAZOLIN]-4'-AMINE - similarity: 0.03470\n",
      "Document CP-866087 - similarity: 0.03468\n",
      "Document 4-Nitrocatechol sulfate - similarity: 0.03439\n",
      "Document 1-Carboxyethylaminomethyl-4-Aminomethylbenzene - similarity: 0.03433\n",
      "Document N-(cyclopropylmethyl)-2'-methyl-5'-(5-methyl-1,3,4-oxadiazol-2-yl)biphenyl-4-carboxamide - similarity: 0.03428\n",
      "Document Butabarbital - similarity: 0.03424\n",
      "Document Influenza A virus A/Singapore/GP1908/2015 IVR-180A (H1N1) antigen (propiolactone inactivated) - similarity: 0.03407\n",
      "Document SD118 - similarity: 0.03379\n",
      "Document Coagulation Factor IX (Recombinant) - similarity: 0.03376\n",
      "Document 2-deoxy-2-fluoro-β-D-mannose - similarity: 0.03369\n",
      "Document Influenza A virus A/Switzerland/9715293/2013 NIB-88 (H3N2) antigen (propiolactone inactivated) - similarity: 0.03369\n",
      "Document GFT14 - similarity: 0.03355\n",
      "Document Cloprednol - similarity: 0.03344\n",
      "Document 5-hydroxynaphthalene-1-sulfonamide - similarity: 0.03326\n",
      "Document M-(N,N,N-Trimethylammonio)-2,2,2-Trifluoro-1,1-Dihydroxyethylbenzene - similarity: 0.03308\n",
      "Document Desoxyribonuclease - similarity: 0.03301\n",
      "Document 2c-Methyl-D-Erythritol 2,4-Cyclodiphosphate - similarity: 0.03295\n",
      "Document R-348 - similarity: 0.03293\n",
      "Document (3e)-3-[(4-Hydroxyphenyl)Imino]-1h-Indol-2(3h)-One - similarity: 0.03274\n",
      "Document N,O6-Disulfo-Glucosamine - similarity: 0.03262\n",
      "Document Rabies virus inactivated antigen, B - similarity: 0.03259\n",
      "Document Paclitaxel trevatide - similarity: 0.03249\n",
      "Document N-(4-carbamimidoylbenzyl)-1-(3-phenylpropanoyl)-L-prolinamide - similarity: 0.03226\n",
      "Document N-Pyridoxyl-Glycine-5-Monophosphate - similarity: 0.03198\n",
      "Document Cyclacillin - similarity: 0.03196\n",
      "Document Talnetant - similarity: 0.03190\n",
      "Document D-glyceraldehyde - similarity: 0.03150\n",
      "Document Diprenorphine - similarity: 0.03119\n",
      "Document 3-(3-Methyl-2-buten-1-yl)-3H-purin-6-amine - similarity: 0.03110\n",
      "Document AVI-4020 - similarity: 0.03100\n",
      "Document ARC183 - similarity: 0.03095\n",
      "Document CTA018 - similarity: 0.03093\n",
      "Document N-{(2S)-3-[(1R)-1-aminoethyl](hydroxy)phosphoryl-2-benzylpropanoyl}-L-phenylalanine - similarity: 0.03065\n",
      "Document (3R)-4,4-DIFLUORO-3-[(4-METHOXYPHENYL)SULFONYL]BUTANOIC ACID - similarity: 0.03047\n",
      "Document [(2S)-4-methyl-3-oxo-2,3,4,5-tetrahydro-1H-1,4-benzodiazepin-2-yl]acetic acid - similarity: 0.03025\n",
      "Document Benzethidine - similarity: 0.03022\n",
      "Document 2-Acetyl-3-[(4-Amino-2-Methyl-5-Pyrimidinyl)Methyl]-4-Methyl-5-(4,6,6-Trihydroxy-3,5-Dioxa-4,6-Diphosphahex-1-Yl)Thiazolium Inner Salt P,P'-Dioxide - similarity: 0.02984\n",
      "Document Pyridoxyl-Glutamic Acid-5'-Monophosphate - similarity: 0.02956\n",
      "Document 3-{[(9-CYANO-9,10-DIHYDRO-10-METHYLACRIDIN-9-YL)CARBONYL]AMINO}PROPANOIC ACID - similarity: 0.02944\n",
      "Document 7-methyl-guanosine-5'-triphosphate-5'-guanosine - similarity: 0.02915\n",
      "Document Succinyl-Coenzyme A - similarity: 0.02899\n",
      "Document Ceftazidime BATSI - similarity: 0.02887\n",
      "Document CRx-139 - similarity: 0.02884\n",
      "Document 1-[1'-(3-phenylacryloyl)spiro[1-benzofuran-3,4'-piperidin]-5-yl]methanamine - similarity: 0.02869\n",
      "Document LY-294002 - similarity: 0.02867\n",
      "Document Ioforminol - similarity: 0.02864\n",
      "Document Aluminum zirconium octachlorohydrex gly - similarity: 0.02849\n",
      "Document WP 1066 - similarity: 0.02845\n",
      "Document Elapegademase - similarity: 0.02806\n",
      "Document JHU-75528 C-11 - similarity: 0.02806\n",
      "Document Enflurane - similarity: 0.02803\n",
      "Document Moperone - similarity: 0.02784\n",
      "Document N-acetyltyrosine - similarity: 0.02779\n",
      "Document (7R)-7-(6,7-Dihydro-5H-cyclopenta[d]imidazo[2,1-b][1,3]thiazol-2-yl)-2,7-dihydro-1,4-thiazepine-3,6-dicarboxylic acid - similarity: 0.02768\n",
      "Document Fispemifene - similarity: 0.02736\n",
      "Document Gestonorone - similarity: 0.02700\n",
      "Document Bordetella pertussis filamentous hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.02697\n",
      "Document (R,R)-2,3-Butanediol - similarity: 0.02697\n",
      "Document Myrrh - similarity: 0.02646\n",
      "Document (Z,Z)-4-Hydroxy-N,N,N-Trimethyl-10-Oxo-7-[(1-Oxo-9-Octadecenyl)Oxy]-3,5,9-Trioxa-4-Phosphaheptacos-18-En-1-Aminium-4-Oxide - similarity: 0.02585\n",
      "Document N-{(3R,4S)-4-[(6-amino-4-methylpyridin-2-yl)methyl]pyrrolidin-3-yl}-N'-(3-chlorobenzyl)ethane-1,2-diamine - similarity: 0.02580\n",
      "Document Penclomedine - similarity: 0.02573\n",
      "Document 2-Aminobenzyl alcohol - similarity: 0.02571\n",
      "Document Bromisoval - similarity: 0.02563\n",
      "Document GW-813893 - similarity: 0.02555\n",
      "Document 4-(2-amino-1,3-thiazol-4-yl)phenol - similarity: 0.02529\n",
      "Document 3-Methylcytosine - similarity: 0.02520\n",
      "Document PA-1050040 - similarity: 0.02518\n",
      "Document Equine Botulinum Neurotoxin E Immune FAB2 - similarity: 0.02508\n",
      "Document MK-1064 - similarity: 0.02475\n",
      "Document OXIMINOARYLSULFONAMIDE - similarity: 0.02464\n",
      "Document beta-Hydroxyisovaleric acid - similarity: 0.02462\n",
      "Document 3-(4-DIETHYLAMINO-2-HYDROXY-PHENYL)-2-METHYL-PROPIONIC ACID - similarity: 0.02456\n",
      "Document FR239087 - similarity: 0.02453\n",
      "Document Indacaterol - similarity: 0.02440\n",
      "Document Giripladib - similarity: 0.02436\n",
      "Document 4-{[(Z)-(5-oxo-2-phenyl-1,3-oxazol-4(5H)-ylidene)methyl]amino}butanoic acid - similarity: 0.02405\n",
      "Document Aluminium acetoacetate - similarity: 0.02403\n",
      "Document [HYDROXY(3-PHENYLPROPYL)AMINO]METHANOL - similarity: 0.02371\n",
      "Document [(1-{2[(4-Carbamimidoyl-Phenylamino)-Methyl]-1-Methyl-1h-Benzoimidazol-5-Yl}-Cyclopropyl)-Pyridin-2-Yl-Methyleneaminooxy]-Acetic Acid Ethyl Ester - similarity: 0.02358\n",
      "Document Metizoline - similarity: 0.02346\n",
      "Document Piromidic acid - similarity: 0.02337\n",
      "Document Allosamidin - similarity: 0.02327\n",
      "Document Sulfamic Acid 2,3-O-(1-Methylethylidene)-4,5-O-Sulfonyl-Beta-Fructopyranose Ester - similarity: 0.02308\n",
      "Document 2-deoxy-2,3-dehydro-N-acetylneuraminic acid - similarity: 0.02304\n",
      "Document 2-HYDROXY-5-(2-MERCAPTO-ETHYLSULFAMOYL)-BENZOIC ACID - similarity: 0.02298\n",
      "Document Co(III)-(Deuteroporphyrin IX) - similarity: 0.02297\n",
      "Document FET F-18 - similarity: 0.02290\n",
      "Document I-Coeleneterazine - similarity: 0.02287\n",
      "Document 1-O-Octyl-2-Heptylphosphonyl-Sn-Glycero-3-Phosphoethanolamine - similarity: 0.02260\n",
      "Document 3-(4-nitrophenyl)-1H-pyrazole - similarity: 0.02227\n",
      "Document 6-AMINO-3,7-DIHYDRO-IMIDAZO[4,5-G]QUINAZOLIN-8-ONE - similarity: 0.02219\n",
      "Document Thiarsa Dihydroxy Cysteine - similarity: 0.02216\n",
      "Document Technetium Tc-99m mebrofenin - similarity: 0.02211\n",
      "Document 5,6-Diaminouracil - similarity: 0.02192\n",
      "Document β-Hydroxythiofentanyl - similarity: 0.02185\n",
      "Document 4'-Deoxy-4'-Acetylyamino-Pyridoxal-5'-Phosphate - similarity: 0.02161\n",
      "Document Phenyldehydroalanine - similarity: 0.02151\n",
      "Document CIS-4-METHYL-N-[(1S)-3-(METHYLSULFANYL)-1-(PYRIDIN-4-YLCARBAMOYL)PROPYL]CYCLOHEXANECARBOXAMIDE - similarity: 0.02131\n",
      "Document Dibenzyl (carbonylbis{2,1-hydrazinediyl[(2S)-4-methyl-1-oxo-1,2-pentanediyl]})biscarbamate - similarity: 0.02119\n",
      "Document Indantadol - similarity: 0.02111\n",
      "Document 3-[(5s)-1-Acetyl-3-(2-Chlorophenyl)-4,5-Dihydro-1h-Pyrazol-5-Yl]Phenol - similarity: 0.02104\n",
      "Document 8-BENZO[1,3]DIOXOL-,5-YLMETHYL-9-BUTYL-9H- - similarity: 0.02084\n",
      "Document Trimazosin - similarity: 0.02047\n",
      "Document Enoblituzumab - similarity: 0.02031\n",
      "Document Fluperolone - similarity: 0.02025\n",
      "Document beta-D-galactopyranosyl-(1->4)-beta-D-galactopyranose - similarity: 0.02003\n",
      "Document (4S)-4-hydroxy-3,4-dihydro-2(1H)-pyrimidinone - similarity: 0.01986\n",
      "Document Concanamycin A - similarity: 0.01980\n",
      "Document Sodium phosphate, monobasic, unspecified form - similarity: 0.01951\n",
      "Document Fenoxaprop-ethyl - similarity: 0.01904\n",
      "Document 2-METHYLCARBAMOYL-3-(4-PHOSPHONOOXY-PHENYL)-CYCLOPROPANECARBOXYLIC ACID - similarity: 0.01900\n",
      "Document Oxaprotiline - similarity: 0.01900\n",
      "Document Tioxidazole - similarity: 0.01893\n",
      "Document Oliceridine - similarity: 0.01870\n",
      "Document 7-Methylguanosine - similarity: 0.01865\n",
      "Document Furoyl-Leucine - similarity: 0.01856\n",
      "Document S-Hydroxymethyl Glutathione - similarity: 0.01848\n",
      "Document AZD-0328 - similarity: 0.01813\n",
      "Document Susoctocog alfa - similarity: 0.01811\n",
      "Document Oxyphenisatin acetate - similarity: 0.01803\n",
      "Document 6-CARBAMIMIDOYL-4-(3-HYDROXY-2-METHYL-BENZOYLAMINO)-NAPHTHALENE-2-CARBOXYLIC ACID METHYL ESTER - similarity: 0.01796\n",
      "Document 3-Butylthiolane 1-Oxide - similarity: 0.01790\n",
      "Document (2R)-N-{4-[Ethyl(phenyl)sulfamoyl]-2-methylphenyl}-3,3,3-trifluoro-2-hydroxy-2-methylpropanamide - similarity: 0.01773\n",
      "Document (5S,7R,8S,9S,10R)-3-Amino-8,9,10-trihydroxy-7-(hydroxymethyl)-6-oxa-1,3-diazaspiro[4.5]decane-2,4-dione - similarity: 0.01728\n",
      "Document Dexfosfoserine - similarity: 0.01722\n",
      "Document VIR201 - similarity: 0.01713\n",
      "Document (2S)-2-(4-ethylphenoxy)-3-phenylpropanoic acid - similarity: 0.01713\n",
      "Document rNAPc2 - similarity: 0.01684\n",
      "Document 4-(1H-IMIDAZOL-1-YL)PHENOL - similarity: 0.01652\n",
      "Document 4-({4-[(6-CHLORO-1-BENZOTHIEN-2-YL)SULFONYL]-2-OXOPIPERAZIN-1-YL}METHYL)BENZENECARBOXIMIDAMIDE - similarity: 0.01594\n",
      "Document (3E)-2,6-DIOXO-6-PHENYLHEX-3-ENOATE - similarity: 0.01592\n",
      "Document Influenza A virus A/California/7/2009 X-181 (H1N1) hemagglutinin antigen (formaldehyde inactivated) - similarity: 0.01546\n",
      "Document N-1-methylheptylformamide - similarity: 0.01544\n",
      "Document 2-Keto-3-Deoxygluconate - similarity: 0.01531\n",
      "Document Cyclandelate - similarity: 0.01517\n",
      "Document Influenza A virus A/Singapore/GP1908/2015 IVR-180 (H1N1) antigen (MDCK cell derived, propiolactone inactivated) - similarity: 0.01502\n",
      "Document 1,6-Fructose Diphosphate (Linear Form) - similarity: 0.01497\n",
      "Document 2-({[4-(TRIFLUOROMETHOXY)PHENYL]SULFONYL}AMINO)ETHYL DIHYDROGEN PHOSPHATE - similarity: 0.01439\n",
      "Document 3,4-DI-1H-INDOL-3-YL-1H-PYRROLE-2,5-DICARBOXYLIC ACID - similarity: 0.01435\n",
      "Document N-(2,3,4,5,6-Pentaflouro-Benzyl)-4-Sulfamoyl-Benzamide - similarity: 0.01420\n",
      "Document Branaplam - similarity: 0.01418\n",
      "Document Lanthanum III cation - similarity: 0.01417\n",
      "Document L-Octahydroindole-2-carboxylic acid - similarity: 0.01398\n",
      "Document (5Z)-7-{(1R,4S,5R,6R)-6-[(1E)-1-Octen-1-yl]-2,3-diazabicyclo[2.2.1]hept-2-en-5-yl}-5-heptenoic acid - similarity: 0.01378\n",
      "Document 1-ACETYL-2-CARBOXYPIPERIDINE - similarity: 0.01377\n",
      "Document 5-Alpha-Androstane-3-Beta,17beta-Diol - similarity: 0.01364\n",
      "Document Famphur - similarity: 0.01342\n",
      "Document Methyl 4,6-O-[(1R)-1-carboxyethylidene]-beta-D-galactopyranoside - similarity: 0.01335\n",
      "Document Glypromate - similarity: 0.01280\n",
      "Document 1-methyl-3-naphthalen-2-yl-1H-pyrazolo[3,4-d]pyrimidin-4-amine - similarity: 0.01267\n",
      "Document Guanosine-5'-Diphosphate - similarity: 0.01262\n",
      "Document Aspergillus repens - similarity: 0.01257\n",
      "Document AVE-1625 - similarity: 0.01240\n",
      "Document Bromotheophylline - similarity: 0.01233\n",
      "Document Nimacimab - similarity: 0.01219\n",
      "Document Remacemide - similarity: 0.01148\n",
      "Document N-[3-[(1-Aminoethyl)(Hydroxy)Phosphoryl]-2-(1,1'-Biphenyl-4-Ylmethyl)Propanoyl]Alanine - similarity: 0.01140\n",
      "Document AICA ribonucleotide - similarity: 0.01104\n",
      "Document 2,3-Didehydroalanine - similarity: 0.01060\n",
      "Document 2-Propyl-Aniline - similarity: 0.01057\n",
      "Document Deacetoxycephalosporin C - similarity: 0.01048\n",
      "Document Lefradafiban - similarity: 0.01047\n",
      "Document Aminocandin - similarity: 0.01012\n",
      "Document 2,6-Difluorobenzenesulfonamide - similarity: 0.01008\n",
      "Document Methyl 4-thio-beta-D-glucopyranoside - similarity: 0.00982\n",
      "Document Gluco-Phenylimidazole - similarity: 0.00979\n",
      "Document (2Z)-2-cyano-N-(3'-ethoxybiphenyl-4-yl)-3-hydroxybut-2-enamide - similarity: 0.00945\n",
      "Document 2'-Deoxyuridine 5'-alpha,beta-imido-triphosphate - similarity: 0.00942\n",
      "Document Diperodon - similarity: 0.00926\n",
      "Document 3-Hydroxybutyryl-Coenzyme A - similarity: 0.00924\n",
      "Document (2R,3R)-2,3-Dihydroxy-4-oxo-4-(propylamino)butanoic acid - similarity: 0.00918\n",
      "Document 1-(4-Amidinophenyl)-3-(4-Chlorophenyl)Urea - similarity: 0.00914\n",
      "Document Methylpentynol - similarity: 0.00905\n",
      "Document 1-(2-Ethanone)-2-Hydroxy-2-(1-Amino-2-Methyl-2-Ethanol)-4-(2-Dimethyl)Ethane-Imidazoline-5-One;Chromophore (Thr-Leu-Gly) - similarity: 0.00879\n",
      "Document Balanol Analog 2 - similarity: 0.00876\n",
      "Document 3-carboxamido-1,3,5(10)-estratrien-17(R)-spiro-2'(5',5'-dimethyl-6'oxo)tetrahydropyran - similarity: 0.00876\n",
      "Document Fencamfamin - similarity: 0.00852\n",
      "Document (1R)-2-[(Diaminomethylene)amino]-1-{4-[(4R)-4-(hydroxymethyl)-1,3,2-dioxaborolan-2-yl]phenyl}ethyl nicotinate - similarity: 0.00847\n",
      "Document PARA-NITROBENZYL GLUTARYL GLYCINIC ACID - similarity: 0.00829\n",
      "Document Cymarin - similarity: 0.00828\n",
      "Document AZD-5423 - similarity: 0.00828\n",
      "Document Aspartate Beryllium Trifluoride - similarity: 0.00802\n",
      "Document AZD-5672 - similarity: 0.00788\n",
      "Document 2-(2-Hydroxy-1,1-Dihydroxymethyl-Ethylamino)-Ethanesulfonic Acid - similarity: 0.00767\n",
      "Document 2,3-Dideoxyfucose - similarity: 0.00721\n",
      "Document Derenofylline - similarity: 0.00678\n",
      "Document BTA9881 - similarity: 0.00671\n",
      "Document FR236913 - similarity: 0.00669\n",
      "Document L-Glycero-D-Manno-Heptopyranose - similarity: 0.00631\n",
      "Document Quinisocaine - similarity: 0.00606\n",
      "Document Methyl-[4-(4-Piperidine-1-Ylmethyl-Phenyl)-Cyclohexyl]-Carbaminic Acid-(4-Chlorophenyl)-Ester - similarity: 0.00604\n",
      "Document N-(4-phenoxyphenyl)-2-[(pyridin-4-ylmethyl)amino]nicotinamide - similarity: 0.00590\n",
      "Document difluoro(5-{2-[(5-octyl-1H-pyrrol-2-yl-kappaN)methylidene]-2H-pyrrol-5-yl-kappaN}pentanoato)boron - similarity: 0.00574\n",
      "Document Tetrabromo-2-Benzotriazole - similarity: 0.00571\n",
      "Document (5R,6E,8Z,11Z,14Z,17Z)-5-hydroxyicosa-6,8,11,14,17-pentaenoic acid - similarity: 0.00559\n",
      "Document 2-Amino-1H-benzimidazol-5-ol - similarity: 0.00539\n",
      "Document 3-ETHYL-6-{[(4-FLUOROPHENYL)SULFONYL]AMINO}-2-METHYLBENZOIC ACID - similarity: 0.00497\n",
      "Document Tetrazolyl Histidine - similarity: 0.00438\n",
      "Document Tivirapine - similarity: 0.00436\n",
      "Document [(4-{4-[4-(Difluoro-Phosphono-Methyl)-Phenyl]-Butyl}-Phenyl)-Difluoro-Methyl]-Phosphonic Acid - similarity: 0.00432\n",
      "Document Vazegepant - similarity: 0.00411\n",
      "Document GEM-231 - similarity: 0.00410\n",
      "Document Cyclic 3',5'-thymidine monophosphate - similarity: 0.00401\n",
      "Document N-[(3Z)-5-Tert-butyl-2-phenyl-1,2-dihydro-3H-pyrazol-3-ylidene]-N'-(4-chlorophenyl)urea - similarity: 0.00372\n",
      "Document ND1251 - similarity: 0.00367\n",
      "Document 5-[4-(1-Carboxymethyl-2-Oxo-Propylcarbamoyl)-Benzylsulfamoyl]-2-Hydroxy-Benzoic Acid - similarity: 0.00331\n",
      "Document omega interferon - similarity: 0.00323\n",
      "Document (2s,4s)-Alpha-Campholinic Acid - similarity: 0.00306\n",
      "Document (1S,2S,5S)2-(4-GLUTARIDYLBENZYL)-5-PHENYL-1-CYCLOHEXANOL - similarity: 0.00299\n",
      "Document Se-Ethyl-Isoselenourea - similarity: 0.00289\n",
      "Document 6-[N-(1-Isopropyl-3,4-Dihydro-7-Isoquinolinyl)Carbamyl]-2-Naphthalenecarboxamidine - similarity: 0.00274\n",
      "Document 3-Methylthiofentanyl - similarity: 0.00262\n",
      "Document Bisoxatin - similarity: 0.00262\n",
      "Document Glutathionylspermidine disulfide - similarity: 0.00237\n",
      "Document S-azabisabolene - similarity: 0.00233\n",
      "Document Saralasin - similarity: 0.00215\n",
      "Document HGS-TR2J - similarity: 0.00153\n",
      "Document Neisseria meningitidis group W-135 capsular polysaccharide tetanus toxoid conjugate antigen - similarity: 0.00151\n",
      "Document 2'-deoxyuridine 5'-alpha,beta-imido-diphosphate - similarity: 0.00040\n",
      "Document Trandolaprilat - similarity: 0.00008\n",
      "Document Iron azide hydrate - similarity: 0.00007\n",
      "Document JNJ-27548547 - similarity: 0.00004\n",
      "Document Amphomycin - similarity: 0.00003\n",
      "Document (3R,4R)-3-Hydroxy-4-(hydroxymethyl)-1-[(4-oxo-4,4a,5,7a-tetrahydro-3H-pyrrolo[3,2-d]pyrimidin-7-yl)methyl]pyrrolidinium - similarity: 0.00001\n",
      "Document [4-R-(4-ALPHA,6-BETA,7-BETA]-HEXAHYDRO-5,6-DI(HYDROXY)-1,3-DI(ALLYL)-4,7-BISPHENYLMETHYL)-2H-1,3-DIAZEPINONE - similarity: -0.00008\n",
      "Document Phosphoaminophosphonic acid guanylate ester - similarity: -0.00021\n",
      "Document Beta-D-Glucopyranose Spirohydantoin - similarity: -0.00035\n",
      "Document Cu-Cyclam - similarity: -0.00052\n",
      "Document Biliverdin Ix Gamma Chromophore - similarity: -0.00075\n",
      "Document 1-{4-[4-Amino-6-(4-methoxyphenyl)furo[2,3-d]pyrimidin-5-yl]phenyl}-3-[2-fluoro-5-(trifluoromethyl)phenyl]urea - similarity: -0.00109\n",
      "Document N-(3-cyanophenyl)-2'-methyl-5'-(5-methyl-1,3,4-oxadiazol-2-yl)-4-biphenylcarboxamide - similarity: -0.00145\n",
      "Document 3-[(3-(2-CARBOXYETHYL)-4-METHYLPYRROL-2-YL)METHYLENE]-2-INDOLINONE - similarity: -0.00151\n",
      "Document PF-05175157 - similarity: -0.00158\n",
      "Document N-hydroxyguanidine - similarity: -0.00165\n",
      "Document cis-tetracosenoyl sulfatide - similarity: -0.00168\n",
      "Document Forigerimod - similarity: -0.00187\n",
      "Document 2-Cyclopropylmethylenepropanal - similarity: -0.00232\n",
      "Document 2-({4-[4-(pyridin-4-ylmethyl)-1H-pyrazol-3-yl]phenoxy}methyl)quinoline - similarity: -0.00275\n",
      "Document 2-aminoisobutyric acid - similarity: -0.00280\n",
      "Document Sulfur chloride - similarity: -0.00320\n",
      "Document N-[4-({[5-(DIMETHYLAMINO)-1-NAPHTHYL]SULFONYL}AMINO)BUTYL]-3-SULFANYLPROPANAMIDE - similarity: -0.00349\n",
      "Document TRANS-4-(GUANIDINOMETHYL)-CYCLOHEXANE-L-YL-D-3-CYCLOHEXYLALANYL-L-AZETIDINE-2-YL-D-TYROSINYL-L-HOMOARGININAMIDE - similarity: -0.00371\n",
      "Document Adomeglivant - similarity: -0.00381\n",
      "Document Ceralasertib - similarity: -0.00393\n",
      "Document [2-(1-Amino-2-Hydroxy-Propyl)-4-(4-Fluoro-1h-Indol-3-Ylmethyl)-5-Hydroxy-Imidazol-1-Yl]-Acetic Acid - similarity: -0.00401\n",
      "Document Atriplex canescens pollen - similarity: -0.00412\n",
      "Document 5-Hydroxymethyl-Chonduritol - similarity: -0.00424\n",
      "Document 2-Phosphoglycolic Acid - similarity: -0.00446\n",
      "Document N-[[3-FLUORO-4-ETHOXY-PYRID-2-YL]ETHYL]-N'-[5-CHLORO-PYRIDYL]-THIOUREA - similarity: -0.00447\n",
      "Document 2'-Deoxyuridine 3'-Monophosphate - similarity: -0.00461\n",
      "Document 3-Amino-Alanine - similarity: -0.00464\n",
      "Document Camazepam - similarity: -0.00473\n",
      "Document Ragweed pollen extract - similarity: -0.00480\n",
      "Document Glycyl-L-a-Aminopimelyl-E-(D-2-Aminoethyl)Phosphonate - similarity: -0.00538\n",
      "Document Ubiquinone Q2 - similarity: -0.00540\n",
      "Document KX-01 - similarity: -0.00563\n",
      "Document Uridine-Diphosphate-N-Acetylgalactosamine - similarity: -0.00604\n",
      "Document 2-Methyl-3-(2-Aminothiazolo)Propanal - similarity: -0.00623\n",
      "Document TH0318 - similarity: -0.00626\n",
      "Document (S)-2-Tetrahydrofuroic acid - similarity: -0.00659\n",
      "Document Methyl-1-testosterone - similarity: -0.00669\n",
      "Document Acedoben - similarity: -0.00686\n",
      "Document N-(4-Aminobutanoyl)-S-(4-methoxybenzyl)-L-cysteinylglycine - similarity: -0.00694\n",
      "Document Arginineamide - similarity: -0.00698\n",
      "Document Benzylpenicilloyl polylysine - similarity: -0.00704\n",
      "Document Artemisia frigida pollen - similarity: -0.00714\n",
      "Document 1-benzylimidazole - similarity: -0.00734\n",
      "Document Ephedra sinica root - similarity: -0.00788\n",
      "Document 1-Methylcytosine - similarity: -0.00790\n",
      "Document LY-518674 - similarity: -0.00790\n",
      "Document Serine vanadate - similarity: -0.00847\n",
      "Document Zopolrestat - similarity: -0.00859\n",
      "Document 5'-{[4-(aminooxy)butyl](methyl)amino}-5'-deoxy-8-ethenyladenosine - similarity: -0.00880\n",
      "Document p1-(5'-adenosyl)p5-(5'-thymidyl)pentaphosphate - similarity: -0.00988\n",
      "Document Seratrodast - similarity: -0.00991\n",
      "Document Xanthium strumarium pollen - similarity: -0.00998\n",
      "Document 4,6-Dideoxyglucose - similarity: -0.01022\n",
      "Document N-(2-ferrocenylethyl)maleimide - similarity: -0.01037\n",
      "Document Fipexide - similarity: -0.01041\n",
      "Document Chromophore (Met-Tyr-Gly) - similarity: -0.01055\n",
      "Document Sulfametopyrazine - similarity: -0.01062\n",
      "Document (1S,2R)-2-[(2,5-difluorophenyl)carbamoyl]cyclopropanecarboxylic acid - similarity: -0.01067\n",
      "Document SKF-91488 - similarity: -0.01073\n",
      "Document Masoprocol - similarity: -0.01113\n",
      "Document Laevulinic Acid - similarity: -0.01137\n",
      "Document Compound 19 - similarity: -0.01139\n",
      "Document 3-(2-CHLOROPHENYL)-1-(2-{[(1S)-2-HYDROXY-1,2-DIMETHYLPROPYL]AMINO}PYRIMIDIN-4-YL)-1-(4-METHOXYPHENYL)UREA - similarity: -0.01198\n",
      "Document {1-[2-(1-FORMYL-PROPYL)-3-METHANESULFONYLAMINO-PYRROLIDINE-1-CARBONYL]-2-METHYL-PROPYL}-CARBAMIC ACID TERT-BUTYL ESTER - similarity: -0.01199\n",
      "Document N-(Benzoylcarbamoyl)-beta-D-glucopyranosylamine - similarity: -0.01201\n",
      "Document Pirnabine - similarity: -0.01207\n",
      "Document N-Acetyl-Serine - similarity: -0.01214\n",
      "Document Adenylyl-(3'-5')-uridine 3'-monophosphate - similarity: -0.01227\n",
      "Document ES-936 - similarity: -0.01239\n",
      "Document Mixed Carbamic Phosphoric Acid Anhydride of 7,8-Diaminononanic Acid - similarity: -0.01242\n",
      "Document (3E)-4-(1-METHYL-1H-INDOL-3-YL)BUT-3-EN-2-ONE - similarity: -0.01285\n",
      "Document Valnoctamide - similarity: -0.01306\n",
      "Document 3-bromo-5-phenyl-N-(pyrimidin-5-ylmethyl)pyrazolo[1,5-a]pyridin-7-amine - similarity: -0.01355\n",
      "Document Somatoprim - similarity: -0.01378\n",
      "Document 9-(6-deoxy-beta-D-allofuranosyl)-6-methylpurine - similarity: -0.01395\n",
      "Document 2-Amino-7-[2-(2-Hydroxy-1-Hydroxymethyl-Ethylamino)-Ethyl]-1,7-Dihydro-Purin-6-One - similarity: -0.01398\n",
      "Document Beta-L-Arabinose - similarity: -0.01432\n",
      "Document Lavoltidine - similarity: -0.01442\n",
      "Document Lorukafusp alfa - similarity: -0.01465\n",
      "Document 9-Deazaadenine - similarity: -0.01472\n",
      "Document Modotuximab - similarity: -0.01488\n",
      "Document 2-(4-METHYLPHENOXY)ETHYLPHOSPHINATE - similarity: -0.01492\n",
      "Document AMGN-0007 - similarity: -0.01501\n",
      "Document 5-(5-(4-(5-hydro-4-methyl-2-oxazolyl)phenoxy)pentyl)-3-methyl isoxazole - similarity: -0.01501\n",
      "Document LY341770 - similarity: -0.01530\n",
      "Document alpha-D-quinovopyranose - similarity: -0.01534\n",
      "Document 4-(2-Thienyl)-1-(4-Methylbenzyl)-1h-Imidazole - similarity: -0.01543\n",
      "Document (S)-1-PHENYL-1-[4-(9H-PURIN-6-YL)PHENYL]METHANAMINE - similarity: -0.01563\n",
      "Document PYRIMIDINE-4,6-DICARBOXYLIC ACID BIS-[(PYRIDIN-3-YLMETHYL)-AMIDE] - similarity: -0.01583\n",
      "Document PF-06459988 - similarity: -0.01588\n",
      "Document Chlorhexadol - similarity: -0.01589\n",
      "Document Niguldipine - similarity: -0.01628\n",
      "Document 3-[Isopropyl(4-Methylbenzoyl)Amino]-5-Phenylthiophene-2-Carboxylic Acid - similarity: -0.01652\n",
      "Document N-Acetyl-glucosamine thiazoline - similarity: -0.01754\n",
      "Document N-[Tosyl-D-Prolinyl]Amino-Ethanethiol - similarity: -0.01868\n",
      "Document Chamaecyparis obtusa wood oil - similarity: -0.01890\n",
      "Document N-[(1R,2R)-1,3-Dihydroxy-1-(4-nitrophenyl)-2-propanyl]acetamide - similarity: -0.01892\n",
      "Document 2-[4-(3-METHYL-1H-PYRAZOL-4-YL)PHENYL]ETHANAMINE - similarity: -0.01933\n",
      "Document Mitemcinal - similarity: -0.01957\n",
      "Document 2-Methyl-2-propanyl [(1R)-2-methyl-1-(1,3,4-oxadiazol-2-yl)propyl]carbamate - similarity: -0.01971\n",
      "Document Des(carbamimidoyl) zanamivir - similarity: -0.01983\n",
      "Document Eftrenonacog alfa - similarity: -0.02000\n",
      "Document Allyl-{4-[3-(4-Bromo-Phenyl)-Benzofuran-6-Yloxy]-but-2-Enyl}-Methyl-Amine - similarity: -0.02059\n",
      "Document 4-(1,3-Benzodioxol-5-Yl)-5-(5-Ethyl-2,4-Dihydroxyphenyl)-2h-Pyrazole-3-Carboxylic Acid - similarity: -0.02076\n",
      "Document Bassia scoparia pollen - similarity: -0.02099\n",
      "Document N-Alpha-(2-Naphthylsulfonyl)-N-(3-Amidino-L-Phenylalaninyl)-D-Pipecolinic Acid - similarity: -0.02106\n",
      "Document Mdl 101,146 - similarity: -0.02117\n",
      "Document N-[2-(1-Formyl-2-Methyl-Propyl)-1-(4-Piperidin-1-Yl-but-2-Enoyl)-Pyrrolidin-3-Yl]-Methanesulfonamide - similarity: -0.02119\n",
      "Document 7-Nitroindazole-2-Carboxamidine - similarity: -0.02139\n",
      "Document Doconexent - similarity: -0.02153\n",
      "Document 3-[4-(1-formylpiperazin-4-yl)-benzylidenyl]-2-indolinone - similarity: -0.02167\n",
      "Document 2(S)-Amino-6-Boronohexanoic Acid - similarity: -0.02211\n",
      "Document Bismuth subcarbonate - similarity: -0.02227\n",
      "Document Bambusa arundinacea stem - similarity: -0.02248\n",
      "Document Tilbroquinol - similarity: -0.02252\n",
      "Document 5-amino-2,4,6-tribromobenzene-1,3-dicarboxylic acid - similarity: -0.02279\n",
      "Document Faropenem medoxomil - similarity: -0.02313\n",
      "Document 4-O-{4,6-O-[(1S)-1-Carboxyethylidene]-β-D-mannopyranosyl}-D-glucopyranuronic acid - similarity: -0.02314\n",
      "Document (1S)-2-[(2S,5R)-2-(AMINOMETHYL)-5-PROP-1-YN-1-YLPYRROLIDIN-1-YL]-1-CYCLOPENTYL-2-OXOETHANAMINE - similarity: -0.02315\n",
      "Document Ulapualide A - similarity: -0.02341\n",
      "Document Leucine - Reduced Carbonyl - similarity: -0.02399\n",
      "Document 2,2':6',2''-Terpyridine Platinum(Ii) - similarity: -0.02420\n",
      "Document AG-858 - similarity: -0.02437\n",
      "Document O,O-DIETHYL HYDROGEN THIOPHOSPHATE - similarity: -0.02465\n",
      "Document N-Caffeoyltyramine - similarity: -0.02465\n",
      "Document 4-O-methyl-alpha-D-glucuronic acid - similarity: -0.02498\n",
      "Document Beta-(2-Naphthyl)-Alanine - similarity: -0.02515\n",
      "Document Ad5-nCoV - similarity: -0.02517\n",
      "Document IGN311 - similarity: -0.02521\n",
      "Document Streptococcus pneumoniae type 12f capsular polysaccharide antigen - similarity: -0.02525\n",
      "Document Pyrithiamine Pyrophosphate - similarity: -0.02552\n",
      "Document REP8839 - similarity: -0.02554\n",
      "Document Agrostis gigantea top - similarity: -0.02576\n",
      "Document N,N-[2,5-O-di-2-fluoro-benzyl-glucaryl]-di-[1-amino-indan-2-ol] - similarity: -0.02580\n",
      "Document 4-(2-amino-1,3-thiazol-4-yl)pyrimidin-2-amine - similarity: -0.02583\n",
      "Document Cyclouridine - similarity: -0.02600\n",
      "Document ({3-[1-(4-Hydroxy-2-oxo-2H-chromen-3-yl)-propyl]-phenylcarbamoyl}-methyl)-carbamic acid tert-butyl ester - similarity: -0.02639\n",
      "Document Oxyphenisatin - similarity: -0.02652\n",
      "Document Nelotanserin - similarity: -0.02657\n",
      "Document (2S)-1-(1H-INDOL-3-YL)-3-{[5-(3-METHYL-1H-INDAZOL-5-YL)PYRIDIN-3-YL]OXY}PROPAN-2-AMINE - similarity: -0.02727\n",
      "Document S-Arsonocysteine - similarity: -0.02734\n",
      "Document Seocalcitol - similarity: -0.02747\n",
      "Document 6-[N-(4-(Aminomethyl)Phenyl)Carbamyl]-2-Naphthalenecarboxamidine - similarity: -0.02751\n",
      "Document N-Cyclohexyl-N'-(Propyl)Phenyl Urea - similarity: -0.02753\n",
      "Document Scorpion (centruroides) immune Fab2 antivenin (equine) - similarity: -0.02773\n",
      "Document CAT-213 - similarity: -0.02774\n",
      "Document cryopreserved human liver cells - similarity: -0.02814\n",
      "Document L-gulonic acid 6-phosphate - similarity: -0.02818\n",
      "Document 1-(HYDROXYMETHYLENEAMINO)-8-HYDROXY-OCTANE - similarity: -0.02860\n",
      "Document Ketodarolutamide - similarity: -0.02882\n",
      "Document 2-[(3-Hydroxy-2-Methyl-5-Phosphonooxymethyl-Pyridin-4-Ylmethyl)-Imino]-5-Phosphono-Pent-3-Enoic Acid - similarity: -0.02914\n",
      "Document 4-(2,5-Dichloro-Thiophen-3-Yl)-Pyrimidin-2-Ylamine - similarity: -0.02922\n",
      "Document S-Methyl glutathione - similarity: -0.02922\n",
      "Document 9-(6-deoxy-alpha-L-talofuranosyl)-6-methylpurine - similarity: -0.02941\n",
      "Document Diethylene glycol diethyl ether - similarity: -0.02953\n",
      "Document Meglutol - similarity: -0.02970\n",
      "Document Motretinide - similarity: -0.02973\n",
      "Document N-(1-carboxy-3-phenylpropyl)phenylalanyl-alpha-asparagine - similarity: -0.02984\n",
      "Document CRA_8696 - similarity: -0.03005\n",
      "Document Magnesium Aluminum Silicate - similarity: -0.03046\n",
      "Document ReN001 - similarity: -0.03060\n",
      "Document S-octylglutathione - similarity: -0.03082\n",
      "Document 4,6-DIDEOXY-4-{[4-[(4-O-HEXOPYRANOSYLHEXOPYRANOSYL)OXY]-5,6-DIHYDROXY-3-(HYDROXYMETHYL)CYCLOHEX-2-EN-1-YL]AMINO}HEXOPYRANOSYL-(1->4)HEXOPYRANOSYL-(1->4)HEXOPYRANOSE - similarity: -0.03107\n",
      "Document 5'-O-(N-(Alanyl)sulfamoyl)adenosine - similarity: -0.03114\n",
      "Document 6-(DIFLUORO-PHOSPHONO-METHYL)-NAPHTHALENE-2-CARBOXYLIC ACID - similarity: -0.03117\n",
      "Document D-1-(4-chlorophenyl)-2-(acetamido)ethane boronic acid - similarity: -0.03123\n",
      "Document 2,6-Diaminopimelic Acid - similarity: -0.03146\n",
      "Document Verdoheme - similarity: -0.03152\n",
      "Document DIMETHYL-(4,5,6,7-TETRABROMO-1H-BENZOIMIDAZOL-2-YL)-AMINE - similarity: -0.03175\n",
      "Document L-N(omega)-Nitroarginine-2,4-L-diaminobutyric amide - similarity: -0.03187\n",
      "Document Placulumab - similarity: -0.03217\n",
      "Document CDC-801 - similarity: -0.03251\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document [N-(BENZYLOXYCARBONYL)AMINO](4-AMIDINOPHENYL)METHANE-PHOSPHONATE - similarity: -0.03256\n",
      "Document Eltanolone - similarity: -0.03262\n",
      "Document 3-{[(5R,6R)-5-Benzyl-6-hydroxy-2,4-bis(4-hydroxybenzyl)-3-oxo-1,2,4-triazepan-1-yl]sulfonyl}benzonitril - similarity: -0.03270\n",
      "Document 1(R)-1-ACETAMIDO-2-(3-CARBOXY-2-HYDROXYPHENYL)ETHYL BORONIC ACID - similarity: -0.03286\n",
      "Document Inhibitor BEA409 - similarity: -0.03293\n",
      "Document N-Carbamoylphenylalanine - similarity: -0.03299\n",
      "Document 1-(3-METHYLPHENYL)-1H-BENZIMIDAZOL-5-AMINE - similarity: -0.03315\n",
      "Document R-30490 - similarity: -0.03333\n",
      "Document 17alpha-Estriol - similarity: -0.03335\n",
      "Document 2-({[2,3,5,6-TETRAFLUORO-3'-(TRIFLUOROMETHOXY)BIPHENYL-4-YL]AMINO}CARBONYL)CYCLOPENTA-1,3-DIENE-1-CARBOXYLIC ACID - similarity: -0.03361\n",
      "Document CHF 4227 - similarity: -0.03386\n",
      "Document Omoconazole - similarity: -0.03446\n",
      "Document Odocoileus virginianus hair - similarity: -0.03452\n",
      "Document L-erythro-7,8-dihydrobiopterin - similarity: -0.03456\n",
      "Document 2-Amino-4-(Hydroxymethyl-Phosphinyl)Butanoic Acid - similarity: -0.03466\n",
      "Document beta-D-arabinofuranose 5-phosphate - similarity: -0.03468\n",
      "Document Ethadione - similarity: -0.03481\n",
      "Document Guanosine tetraphosphate - similarity: -0.03488\n",
      "Document PEG-uricase - similarity: -0.03525\n",
      "Document Z-Ala Prolinal - similarity: -0.03526\n",
      "Document DODECANESULFONATE ION - similarity: -0.03545\n",
      "Document Carpropamide - similarity: -0.03546\n",
      "Document Mepindolol - similarity: -0.03560\n",
      "Document 3-(5-amino-7-hydroxy-[1,2,3]triazolo[4,5-d]pyrimidin-2-yl)-N-(3,5-dichlorobenzyl)-benzamide - similarity: -0.03620\n",
      "Document S-[(1-Hydroxy-2,2,5,5-tetramethyl-2,5-dihydro-1H-pyrrol-3-yl)methyl] methanesulfonothioate - similarity: -0.03623\n",
      "Document (3AS,4R,9BR)-4-(4-HYDROXYPHENYL)-6-(METHOXYMETHYL)-1,2,3,3A,4,9B-HEXAHYDROCYCLOPENTA[C]CHROMEN-8-OL - similarity: -0.03631\n",
      "Document 5-(6-D-ribitylamino-2,4-dihydroxypyrimidin-5-yl)-1-pentyl-phosphonic acid - similarity: -0.03650\n",
      "Document CP-320626 - similarity: -0.03655\n",
      "Document N-{3-[(7ar,12as,12bs)-7-Oxo-1,3,4,6,7,7a,12a,12b-Octahydroindolo[2,3-a]Quinolizin-12(2h)-Yl]Propyl}Propane-2-Sulfonamide - similarity: -0.03660\n",
      "Document 2-{4-[4-({4-[2-methyl-1-(1-methylethyl)-1H-imidazol-5-yl]pyrimidin-2-yl}amino)phenyl]piperazin-1-yl}-2-oxoethanol - similarity: -0.03698\n",
      "Document Fiboflapon - similarity: -0.03699\n",
      "Document (5E,13E)-11-HYDROXY-9,15-DIOXOPROSTA-5,13-DIEN-1-OIC ACID - similarity: -0.03704\n",
      "Document Mefenorex - similarity: -0.03711\n",
      "Document Asvasiran - similarity: -0.03719\n",
      "Document 3,4-Dihydro-2'-deoxyuridine-5'-monophosphate - similarity: -0.03727\n",
      "Document N-beta-D-glucopyranosylacetamide - similarity: -0.03774\n",
      "Document (1aR,8S,13S,14S,15aR)-5,13,14-trihydroxy-3-methoxy-8-methyl-8,9,13,14,15,15a-hexahydro-6H-oxireno[k][2]benzoxacyclotetradecine-6,12(1aH)-dione - similarity: -0.03795\n",
      "Document Chromophore (Gly-Tyr-Gly) - similarity: -0.03860\n",
      "Document Quinacrine mustard - similarity: -0.03870\n",
      "Document Lorajmine - similarity: -0.03877\n",
      "Document Isocitric Acid - similarity: -0.03885\n",
      "Document Cp-166572, 2-Hydroxymethyl-4-(4-N,N-Dimethylaminosulfonyl-1-Piperazino)-Pyrimidine - similarity: -0.03886\n",
      "Document (2R)-N-hydroxy-3-naphthalen-2-yl-2-[(naphthalen-2-ylsulfonyl)amino]propanamide - similarity: -0.03935\n",
      "Document 2,3-DIMETHOXY-12H-[1,3]DIOXOLO[5,6]INDENO[1,2-C]ISOQUINOLIN-6-IUM - similarity: -0.03953\n",
      "Document 8,9,10-Trihydroxy-7-hydroxymethyl-2-thioxo-6-oxa-1,3-diaza-spiro[4.5]decan-4-one - similarity: -0.03954\n",
      "Document (11S)-8-CHLORO-11-[1-(METHYLSULFONYL)PIPERIDIN-4-YL]-6-PIPERAZIN-1-YL-11H-BENZO[5,6]CYCLOHEPTA[1,2-B]PYRIDINE - similarity: -0.04003\n",
      "Document T-900607 - similarity: -0.04012\n",
      "Document B-nonylglucoside - similarity: -0.04025\n",
      "Document Tert-Butyloxycarbonyl Group - similarity: -0.04039\n",
      "Document Pyrazolanthrone - similarity: -0.04065\n",
      "Document CRA_23653 - similarity: -0.04065\n",
      "Document 1-Phenylcyclohexylamine - similarity: -0.04067\n",
      "Document 2-[({4-[2-(trifluoromethyl)phenyl]piperidin-1-yl}carbonyl)amino]benzoic acid - similarity: -0.04087\n",
      "Document 2-(Phosphonooxy)Butanoic Acid - similarity: -0.04098\n",
      "Document Tolu balsam - similarity: -0.04108\n",
      "Document 1-[2-HYDROXY-3-(4-CYCLOHEXYL-PHENOXY)-PROPYL]-4-(2-PYRIDYL)-PIPERAZINE - similarity: -0.04115\n",
      "Document S-Hexylglutathione - similarity: -0.04135\n",
      "Document Streptococcus pneumoniae type 33f capsular polysaccharide antigen - similarity: -0.04138\n",
      "Document FG-2216 - similarity: -0.04203\n",
      "Document Carboxymycobactin S - similarity: -0.04205\n",
      "Document 4-Iodobenzo[B]Thiophene-2-Carboxamidine - similarity: -0.04212\n",
      "Document R048-8071 - similarity: -0.04222\n",
      "Document Artemisia ludoviciana pollen - similarity: -0.04222\n",
      "Document 1-[6-(2-CHLORO-4-METHYXYPHENOXY)-HEXYL]-IMIDAZOLE - similarity: -0.04227\n",
      "Document Freselestat - similarity: -0.04238\n",
      "Document CT-2584 - similarity: -0.04247\n",
      "Document N-[1,3-Di(4-morpholinyl)-2-propanyl]-3-(α-D-galactopyranosyloxy)-5-nitrobenzamide - similarity: -0.04249\n",
      "Document N-acetyl-beta-D-glucosaminyl-(1->4)-N-acetyl-beta-D-glucosamine - similarity: -0.04319\n",
      "Document N-Acetylmethionine - similarity: -0.04335\n",
      "Document 8-Benzyl-2-Hydroxy-2-(4-Hydroxy-Benzyl)-6-(4-Hydroxy-Phenyl)-2h-Imidazo[1,2-a]Pyrazin-3-One - similarity: -0.04362\n",
      "Document Gremubamab - similarity: -0.04417\n",
      "Document sn-glycero-3-phosphoethanolamine - similarity: -0.04422\n",
      "Document SEphB4-HSA - similarity: -0.04423\n",
      "Document 4-(acetylamino)-N-(4-fluorophenyl)-1H-pyrazole-3-carboxamide - similarity: -0.04426\n",
      "Document 1-(O-Carboxy-Phenylamino)-1-Deoxy-D-Ribulose-5-Phosphate - similarity: -0.04445\n",
      "Document Parasorbic acid - similarity: -0.04494\n",
      "Document 3'-beta-Sialyl-beta-lactose - similarity: -0.04542\n",
      "Document 5-imino-4-(3-trifluoromethyl-phenylazo)-5H-pyrazol-3-ylamine - similarity: -0.04568\n",
      "Document Elaidamide - similarity: -0.04576\n",
      "Document 2-[2-(4-Chloro-Phenylsulfanyl)-Acetylamino]-3-(4-Guanidino-Phenyl)-Propionamide - similarity: -0.04576\n",
      "Document R450 - similarity: -0.04586\n",
      "Document Hexanoyl-CoA - similarity: -0.04617\n",
      "Document ISIS 14803 - similarity: -0.04640\n",
      "Document Sivifene - similarity: -0.04648\n",
      "Document Argadin - similarity: -0.04664\n",
      "Document Amrinone - similarity: -0.04667\n",
      "Document Methylumbelliferyl chitotriose - similarity: -0.04671\n",
      "Document Prostaglandin G2 - similarity: -0.04678\n",
      "Document N-cyclooctylglycyl-N-(4-carbamimidoylbenzyl)-L-prolinamide - similarity: -0.04687\n",
      "Document (2R,4S)-2-[(1R)-1-{[(2R)-2-Amino-2-(4-hydroxyphenyl)acetyl]amino}-2-oxoethyl]-5,5-dimethyl-1,3-thiazolidine-4-carboxylic acid - similarity: -0.04715\n",
      "Document Afasevikumab - similarity: -0.04715\n",
      "Document 9-Butyl-8-(2-Chloro-3,4,5-Trimethoxy-Benzyl)-9h-Purin-6-Ylamine - similarity: -0.04728\n",
      "Document N-{[(2-Methyl-2-propanyl)oxy]carbonyl}-L-alanyl-L-alaninamide - similarity: -0.04745\n",
      "Document Alpha-Aminobutyric Acid - similarity: -0.04754\n",
      "Document N-((2-aminopyrimidin-5-yl)methyl)-5-(2,6-difluorophenyl)-3-ethylpyrazolo[1,5-a]pyrimidin-7-amine - similarity: -0.04815\n",
      "Document 2-(3-CARBOXYPROPIONYL)-6-HYDROXY-CYCLOHEXA-2,4-DIENE CARBOXYLIC ACID - similarity: -0.04831\n",
      "Document EHC18 - similarity: -0.04958\n",
      "Document 2-Amino-4-(2-Amino-Ethoxy)-Butyric Acid - similarity: -0.04958\n",
      "Document Cyanocinnoline - similarity: -0.04961\n",
      "Document Sulfabromomethazine - similarity: -0.05001\n",
      "Document Mesobiliverdin IV alpha - similarity: -0.05005\n",
      "Document 2-fluorofucose - similarity: -0.05029\n",
      "Document 7-{2,6-DICHLORO-4-[3-(2-CHLORO-BENZOYL)-UREIDO]-PHENOXY}-HEPTANOIC ACID - similarity: -0.05171\n",
      "Document 4-(2,4-Dimethyl-1,3-thiazol-5-yl)-N-[4-(trifluoromethyl)phenyl]-2-pyrimidinamine - similarity: -0.05185\n",
      "Document 2,3-Dimethyl-1,4-naphthoquinone - similarity: -0.05213\n",
      "Document 2-{[4-(4-pyridin-4-yl-1H-pyrazol-3-yl)phenoxy]methyl}quinoline - similarity: -0.05213\n",
      "Document Sri-9439 - similarity: -0.05258\n",
      "Document Cysteinamide - similarity: -0.05266\n",
      "Document Deoxycytidylyl-3',5'-guanosine - similarity: -0.05327\n",
      "Document Gadolinium cation (3+) - similarity: -0.05344\n",
      "Document R-9-(2-hydroxypropyl)adenine - similarity: -0.05349\n",
      "Document SR 140333 - similarity: -0.05363\n",
      "Document Sumanirole - similarity: -0.05413\n",
      "Document Fx-1006A - similarity: -0.05465\n",
      "Document N,N-[2,5-O-[Dibenzyl]-glucaryl]-DI-[isoleucyl-amido-methane] - similarity: -0.05506\n",
      "Document THIOPHENE-2,5-DISULFONIC ACID 2-AMIDE-5-(4-METHYL-BENZYLAMIDE) - similarity: -0.05510\n",
      "Document MIV-701 - similarity: -0.05539\n",
      "Document Nz-(1-Carboxyethyl)-Lysine - similarity: -0.05561\n",
      "Document Guvacine - similarity: -0.05562\n",
      "Document M0002 - similarity: -0.05573\n",
      "Document 4-Methylpiperazin-1-Yl Carbonyl Group - similarity: -0.05595\n",
      "Document Norethynodrel - similarity: -0.05615\n",
      "Document 4-[(5-CHLOROINDOL-2-YL)SULFONYL]-2-(2-METHYLPROPYL)-1-[[5-(PYRIDIN-4-YL)PYRIMIDIN-2-YL]CARBONYL]PIPERAZINE - similarity: -0.05653\n",
      "Document [N-[N-(4-Methoxy-2,3,6-trimethylphenylsulfonyl)-L-aspartyl]-D-(4-amidino-phenylalanyl)]-piperidine - similarity: -0.05676\n",
      "Document QR-334 - similarity: -0.05699\n",
      "Document Influenza A virus A/Texas/50/2012 (H3N2) recombinant hemagglutinin antigen - similarity: -0.05703\n",
      "Document Benzyl N-[(2S)-5-(diaminomethylamino)-1-[[(2S)-4-fluoro-3-oxobutan-2-yl]amino]-1-oxopentan-2-yl]carbamate - similarity: -0.05765\n",
      "Document MSX-122 - similarity: -0.05776\n",
      "Document 2'-O-Methyl-3'-Methyl-3'-Deoxy-Arabinofuranosyl-Thymine-5'-Phosphate - similarity: -0.05791\n",
      "Document 3-(5-methoxy-1H-indol-3-yl)propanoic acid - similarity: -0.05819\n",
      "Document ACE393 - similarity: -0.05852\n",
      "Document 4-[1-allyl-7-(trifluoromethyl)-1H-indazol-3-yl]benzene-1,3-diol - similarity: -0.05865\n",
      "Document 1-Deoxy-Ribofuranose-5'-Phosphate - similarity: -0.05883\n",
      "Document 5-ETHYL-3-[(2-METHOXYETHYL)METHYLAMINO]-6-METHYL-4-(3-METHYLBENZYL)PYRIDIN-2(1H)-ONE - similarity: -0.05929\n",
      "Document 7-Alpha-D-Ribofuranosyl-Purine-5'-Phosphate - similarity: -0.05931\n",
      "Document 17alpha-methyl-4-hydroxynandrolone - similarity: -0.05975\n",
      "Document Etiprednol dicloacetate - similarity: -0.05979\n",
      "Document Technetium Tc-99m ciprofloxacin - similarity: -0.05994\n",
      "Document Adenylate-3'-phosphate-[[2'-deoxy-uridine-5'-phosphate]-3'-phosphate] - similarity: -0.06001\n",
      "Document Influenza A virus A/Uruguay/716/2007(H3N2) antigen (formaldehyde inactivated) - similarity: -0.06026\n",
      "Document alpha-maltotriose - similarity: -0.06033\n",
      "Document Galactose-uridine-5'-diphosphate - similarity: -0.06044\n",
      "Document Plozalizumab - similarity: -0.06100\n",
      "Document BX-528 - similarity: -0.06149\n",
      "Document 5-(5-(4-(4,5-dihydro-2-oxazoly)phenoxy)pentyl)-3-methyl osoxazole - similarity: -0.06160\n",
      "Document (9S)-9-[(8-AMMONIOOCTYL)AMINO]-1,2,3,4,9,10-HEXAHYDROACRIDINIUM - similarity: -0.06217\n",
      "Document S-Adenosyl-1,8-Diamino-3-Thiooctane - similarity: -0.06248\n",
      "Document 5-benzyl-1,3-thiazol-2-amine - similarity: -0.06251\n",
      "Document Penamecillin - similarity: -0.06270\n",
      "Document Fentiazac - similarity: -0.06354\n",
      "Document (2R,4S)-2-[(R)-BENZYLCARBAMOYL-PHENYLACETYL-METHYL]-5,5-DIMETHYL-THIAZOLIDINE-4-CARBOXYLIC ACID - similarity: -0.06383\n",
      "Document (S)-N-(3-Indol-1-Yl-2-Methyl-Propyl)-4-Sulfamoyl-Benzamide - similarity: -0.06401\n",
      "Document N-Methyl-N-[3-(6-Phenyl[1,2,4]Triazolo[4,3-B]Pyridazin-3-Yl)Phenyl]Acetamide - similarity: -0.06408\n",
      "Document 2,4-DIAMINO-1,5-DIPHENYL-3-HYDROXYPENTANE - similarity: -0.06414\n",
      "Document gamma-carboxy-L-glutamic acid - similarity: -0.06456\n",
      "Document Uridine-5'-diphosphate-2-deoxy-2-fluoro-alpha-D-galactose - similarity: -0.06468\n",
      "Document Minretumomab - similarity: -0.06471\n",
      "Document (10R)-10-methyl-3-(6-methylpyridin-3-yl)-9,10,11,12-tetrahydro-8H-[1,4]diazepino[5',6':4,5]thieno[3,2-f]quinolin-8-one - similarity: -0.06472\n",
      "Document 4-(2-amino-1-methyl-1H-imidazo[4,5-b]pyridin-6-yl)phenol - similarity: -0.06487\n",
      "Document Ethyl N-methylcarbamate - similarity: -0.06516\n",
      "Document PF-06305591 - similarity: -0.06522\n",
      "Document Interferon alfa-n1 - similarity: -0.06527\n",
      "Document N~3~-cyclopropyl-N~4~'-(cyclopropylmethyl)-6-methylbiphenyl-3,4'-dicarboxamide - similarity: -0.06535\n",
      "Document Delparantag - similarity: -0.06537\n",
      "Document Iboctadekin - similarity: -0.06561\n",
      "Document Lpc-Ether - similarity: -0.06575\n",
      "Document (R)-minalrestat - similarity: -0.06582\n",
      "Document 6-(3-AMINOPROPYL)-4,9-DIMETHYLPYRROLO[3,4-C]CARBAZOLE-1,3(2H,6H)-DIONE - similarity: -0.06582\n",
      "Document Clonitazene - similarity: -0.06601\n",
      "Document (2R)-1-(DIMETHYLAMINO)-3-{4-[(6-{[2-FLUORO-5-(TRIFLUOROMETHYL)PHENYL]AMINO}PYRIMIDIN-4-YL)AMINO]PHENOXY}PROPAN-2-OL - similarity: -0.06631\n",
      "Document Propyl acetate - similarity: -0.06658\n",
      "Document Diamino-N-[(4S)-5-anilino-4-{[(2S)-2-{[(1R)-1-carboxyethyl]amino}-4-phenylbutanoyl]amino}-5-oxopentyl]methaniminium - similarity: -0.06666\n",
      "Document 2-{(9as)-9a-[(1s)-1-Hydroxyethyl]-2,7-Dimethyl-9a,10-Dihydro-5h-Pyrimido[4,5-D][1,3]Thiazolo[3,2-a]Pyrimidin-8-Yl}Ethyl Trihydrogen Diphosphate - similarity: -0.06668\n",
      "Document 3-(P-Tolyl)Propionic Acid - similarity: -0.06689\n",
      "Document 6-CYCLOHEXYLMETHYLOXY-2-(4'-HYDROXYANILINO)PURINE - similarity: -0.06801\n",
      "Document 5'-S-methyl-5'-thioadenosine - similarity: -0.06806\n",
      "Document Ethylestrenol - similarity: -0.06818\n",
      "Document 2'-deoxy-5-(hydroxymethyl)uridine 5'-(dihydrogen phosphate) - similarity: -0.06867\n",
      "Document 1,3-DI(N-Propyloxy-A-mannopyranosyl)-carbomyl 5-methyazido-benzene - similarity: -0.06872\n",
      "Document Hexadecanal - similarity: -0.06892\n",
      "Document L-Leucyl-Hydroxylamine - similarity: -0.06892\n",
      "Document O-Benzylsulfonyl-Serine - similarity: -0.06916\n",
      "Document Maxy-G34 - similarity: -0.06920\n",
      "Document 1-((1R)-1-(HYDROXYMETHYL)-3-{6-[(3-PHENYLPROPANOYL)AMINO]-1H-INDOL-1-YL}PROPYL)-1H-IMIDAZOLE-4-CARBOXAMIDE - similarity: -0.06920\n",
      "Document 2-Deoxy-2-Amino Glucitol-6-Phosphate - similarity: -0.06955\n",
      "Document 4,4'-Dihydroxybenzophenone - similarity: -0.07029\n",
      "Document Hydroxy-Phenyl-Acetic Acid 8-Methyl-8-Aza-Bicyclo[3.2.1]Oct-3-Yl Ester - similarity: -0.07036\n",
      "Document Influenza A virus A/Idaho/07/2018 (H1N1) antigen (MDCK cell derived, propiolactone inactivated - similarity: -0.07042\n",
      "Document (4E)-N-(4-fluorophenyl)-4-[(phenylcarbonyl)imino]-4H-pyrazole-3-carboxamide - similarity: -0.07051\n",
      "Document N-(1-ISOPROPYLPIPERIDIN-4-YL)-1-(3-METHOXYBENZYL)-1H-INDOLE-2-CARBOXAMIDE - similarity: -0.07052\n",
      "Document D-pantetheine 4'-phosphate - similarity: -0.07070\n",
      "Document MLN-977 - similarity: -0.07081\n",
      "Document Testosterone enantate benzilic acid hydrazone - similarity: -0.07110\n",
      "Document 4,4'-BIS([H]METHYLSULFONYL)-2,2',5,5'-TETRACHLOROBIPHENYL - similarity: -0.07150\n",
      "Document Cytidyl-2'-5'-phospho-guanosine - similarity: -0.07184\n",
      "Document iCo-007 - similarity: -0.07207\n",
      "Document Human adenovirus b serotype 7 - similarity: -0.07209\n",
      "Document 2-{2-[2-2-(Methoxy-Ethoxy)-Ethoxy]-Ethoxy}-Ethanol - similarity: -0.07219\n",
      "Document Valpromide - similarity: -0.07244\n",
      "Document QR-333 - similarity: -0.07291\n",
      "Document (10R)-10-formyl-5,8,10-trideazafolic acid - similarity: -0.07295\n",
      "Document AVR118 - similarity: -0.07313\n",
      "Document 5-(2,5-DICHLOROPHENYL)-2-FUROIC ACID - similarity: -0.07327\n",
      "Document 4-methylthio-2-oxobutanoic acid - similarity: -0.07355\n",
      "Document O-(N-acetyl-alpha-D-galactosaminyl)-L-serine - similarity: -0.07466\n",
      "Document 1,2-dihexadecanoyl-sn-glycero-3-phosphoethanolamine - similarity: -0.07481\n",
      "Document NCX 1022 - similarity: -0.07485\n",
      "Document 6-CARBAMIMIDOYL-2-[2-HYDROXY-5-(3-METHOXY-PHENYL)-INDAN-1-YL]-HEXANOIC ACID - similarity: -0.07502\n",
      "Document Regramostim - similarity: -0.07563\n",
      "Document Oleoyl-estrone - similarity: -0.07591\n",
      "Document Inositol 2,4,5-trisphosphate - similarity: -0.07594\n",
      "Document N-ethyl-4-{[5-(methoxycarbamoyl)-2-methylphenyl]amino}-5-methylpyrrolo[2,1-f][1,2,4]triazine-6-carboxamide - similarity: -0.07605\n",
      "Document Etonitazene - similarity: -0.07606\n",
      "Document 7-(Carboxyamino)-8-Amino-Nonanoic Acid - similarity: -0.07712\n",
      "Document Phenylphosphate - similarity: -0.07714\n",
      "Document (12R)-12-hydroxyeicosapentaenoic acid - similarity: -0.07720\n",
      "Document 2-Hydroxy-Tryptophan - similarity: -0.07726\n",
      "Document S-{2-[Amino(Dihydroxy)-Lambda~4~-Sulfanyl]Ethyl}-D-Cysteine - similarity: -0.07757\n",
      "Document Adipiplon - similarity: -0.07769\n",
      "Document (R)-acetoin - similarity: -0.07802\n",
      "Document Burlulipase - similarity: -0.07806\n",
      "Document Pascopyrum smithii pollen - similarity: -0.07812\n",
      "Document 2-Dimethylamino-Ethyl-Diphosphate - similarity: -0.07829\n",
      "Document Tipelukast - similarity: -0.07847\n",
      "Document 5-FLUORO-1-[4-(4-PHENYL-3,6-DIHYDROPYRIDIN-1(2H)-YL)BUTYL]QUINAZOLINE-2,4(1H,3H)-DIONE - similarity: -0.07879\n",
      "Document 3-{[(1r)-1-Benzyl-2-Sulfanylethyl]Amino}-3-Oxopropanoic Acid - similarity: -0.07885\n",
      "Document 4-(2-Oxo-Hexahydro-Thieno[3,4-D]Imidazol-4-Yl)-Butyricacid - similarity: -0.07886\n",
      "Document Norleucine Phosphonate - similarity: -0.07907\n",
      "Document Fazadinium bromide - similarity: -0.07924\n",
      "Document P1-(5'-Adenosyl)P5-(5'-(3'azido-3'-Deoxythymidyl))Pentaphosphate - similarity: -0.07941\n",
      "Document Protiofate - similarity: -0.07974\n",
      "Document BF-37 - similarity: -0.08012\n",
      "Document N-cyclohexyl-4-imidazo[1,2-a]pyridin-3-yl-N-methylpyrimidin-2-amine - similarity: -0.08018\n",
      "Document 5-(DIMETHYLAMINO)-2-NAPHTHALENESULFONIC ACID - similarity: -0.08020\n",
      "Document 9-Butyl-8-(2,5-Dimethoxy-Benzyl)-9h-Purin-6-Ylamine - similarity: -0.08038\n",
      "Document S-8510 - similarity: -0.08128\n",
      "Document Pinus nigra pollen - similarity: -0.08154\n",
      "Document 1-{5-[2-(thieno[3,2-d]pyrimidin-4-ylamino)ethyl]-1,3-thiazol-2-yl}-3-[3-(trifluoromethyl)phenyl]urea - similarity: -0.08180\n",
      "Document XR5944 - similarity: -0.08180\n",
      "Document (3Z)-6-(4-HYDROXY-3-METHOXYPHENYL)-3-(1H-PYRROL-2-YLMETHYLENE)-1,3-DIHYDRO-2H-INDOL-2-ONE - similarity: -0.08182\n",
      "Document N-acetylhistamine - similarity: -0.08189\n",
      "Document 6-Bromo-1-hexanol - similarity: -0.08223\n",
      "Document Antithymocyte immunoglobulin (rabbit) - similarity: -0.08228\n",
      "Document 5'-Guanosine-Diphosphate-Monothiophosphate - similarity: -0.08232\n",
      "Document Motuporin - similarity: -0.08271\n",
      "Document 4-chloro-N'-[(1E)-(3,5-dibromo-2,4-dihydroxyphenyl)methylidene]benzohydrazide - similarity: -0.08312\n",
      "Document Valanafusp alfa - similarity: -0.08428\n",
      "Document Aminaphthone - similarity: -0.08444\n",
      "Document 4-Hydroperoxy-2-Methoxy-Phenol - similarity: -0.08463\n",
      "Document Oleclumab - similarity: -0.08467\n",
      "Document 1-Tert-Butyl-3-(4-Chloro-Phenyl)-1h-Pyrazolo[3,4-D]Pyrimidin-4-Ylamine - similarity: -0.08537\n",
      "Document 5-[(4-Methylphenyl)Sulfanyl]-2,4-Quinazolinediamine - similarity: -0.08551\n",
      "Document (S)-N-(1-(3-CHLORO-4-FLUOROPHENYL)-2-HYDROXYETHYL)-4-(4-(3-CHLOROPHENYL)-1H-PYRAZOL-3-YL)-1H-PYRROLE-2-CARBOXAMIDE - similarity: -0.08650\n",
      "Document METHYL(2-ACETOXY-2-(2-CARBOXY-4-AMINO-PHENYL))ACETATE - similarity: -0.08669\n",
      "Document N-[(2R)-4-diazonio-3-oxoniumylidene-1-phenylbutan-2-yl]-1-phenylmethoxymethanimidate - similarity: -0.08713\n",
      "Document N-(2-Flouro-Benzyl)-4-Sulfamoyl-Benzamide - similarity: -0.08739\n",
      "Document 4-Imino-5-Methidyl-2-Trifluoromethylpyrimidine - similarity: -0.08742\n",
      "Document RG-4733 - similarity: -0.08751\n",
      "Document RU85053 - similarity: -0.08789\n",
      "Document L-Threoninol - similarity: -0.08819\n",
      "Document LIC-477 - similarity: -0.08841\n",
      "Document 2'-Deoxymaltose - similarity: -0.08852\n",
      "Document UDP-alpha-D-xylose - similarity: -0.08899\n",
      "Document 4-Sulfonamide-[4-(Thiomethylaminobutane)]Benzamide - similarity: -0.08909\n",
      "Document Influenza B virus B/Massachusetts/2/2012 hemagglutinin antigen (formaldehyde inactivated) - similarity: -0.08942\n",
      "Document Guanosine-3'-monophosphate-5'-diphosphate - similarity: -0.08948\n",
      "Document Strontium cation Sr-89 - similarity: -0.08972\n",
      "Document 6-o-Capryloylsucrose - similarity: -0.09075\n",
      "Document 4-(2-aminoethoxy)-N-(2,5-diethoxyphenyl)-3,5-dimethylbenzamide - similarity: -0.09092\n",
      "Document VTP-201227 - similarity: -0.09125\n",
      "Document 1-ETHOXYCARBONYL-D-PHE-PRO-2(4-AMINOBUTYL)HYDRAZINE - similarity: -0.09155\n",
      "Document Edodekin alfa - similarity: -0.09171\n",
      "Document (2R,3R,4R,5R)-4,5-Dihydroxy-2-(hydroxymethyl)-6-oxo-3-piperidinyl beta-D-glucopyranoside - similarity: -0.09248\n",
      "Document 8-(2-Chloro-3,4,5-Trimethoxy-Benzyl)-2-Fluoro-9-Pent-4-Ylnyl-9h-Purin-6-Ylamine - similarity: -0.09294\n",
      "Document lambda-bis(2,2'-bipyridine)imidazole ruthenium (II) - similarity: -0.09322\n",
      "Document N,N-Bis(4-Chlorobenzyl)-1h-1,2,3,4-Tetraazol-5-Amine - similarity: -0.09356\n",
      "Document 4-Deoxyglucarate - similarity: -0.09384\n",
      "Document 3-[[(3R,4S,5S,6R)-7-Benzyl-4,5-dihydroxy-1,1-dioxo-3,6-bis(phenoxymethyl)-1,2,7-thiadiazepan-2-yl]methyl]-N-methylbenzamide - similarity: -0.09385\n",
      "Document 9r,13r-Opda - similarity: -0.09387\n",
      "Document Rimiducid - similarity: -0.09387\n",
      "Document 25-desacetylrifapentine - similarity: -0.09406\n",
      "Document 8-Methyl-9-Oxoguanine - similarity: -0.09413\n",
      "Document Muplestim - similarity: -0.09430\n",
      "Document Carbocromen - similarity: -0.09453\n",
      "Document 4-(ACETYLAMINO)-3-HYDROXY-5-NITROBENZOIC ACID - similarity: -0.09483\n",
      "Document L-Alpha-Glycerophosphorylserine - similarity: -0.09545\n",
      "Document N-cyclopropyl-4-methyl-3-{2-[(2-morpholin-4-ylethyl)amino]quinazolin-6-yl}benzamide - similarity: -0.09553\n",
      "Document (1-HYDROXY-1-PHOSPHONO-2-[1,1';4',1'']TERPHENYL-3-YL-ETHYL)-PHOSPHONIC ACID - similarity: -0.09632\n",
      "Document 3-[1-(4-BROMO-PHENYL)-2-METHYL-PROPYL]-4-HYDROXY-CHROMEN-2-ONE - similarity: -0.09645\n",
      "Document Saprisartan - similarity: -0.09667\n",
      "Document 6-[2-(3'-METHOXYBIPHENYL-3-YL)ETHYL]PYRIDIN-2-AMINE - similarity: -0.09702\n",
      "Document (3S)-N-(5-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE - similarity: -0.09718\n",
      "Document N~3~-[3-(1H-INDOL-6-YL)BENZYL]PYRIDINE-2,3-DIAMINE - similarity: -0.09762\n",
      "Document AGG-523 - similarity: -0.09765\n",
      "Document 4-Acetyl-4-guanidino-6-methyl(propyl)carboxamide-4,5-dihydro-2H-pyran-2-carboxylic acid - similarity: -0.09864\n",
      "Document Hexobendine - similarity: -0.09880\n",
      "Document Amelubant - similarity: -0.09889\n",
      "Document P-(2'-Iodo-5'-Thenoyl)Hydrotropic Acid - similarity: -0.09897\n",
      "Document 1-(4-IODOBENZOYL)-5-METHOXY-2-METHYL INDOLE-3-ACETIC ACID - similarity: -0.09910\n",
      "Document 2,5-dichloro-N-(3,5-dibromo-4-hydroxyphenyl)benzamide - similarity: -0.09957\n",
      "Document Tetradecyl hydrogen sulfate (ester) - similarity: -0.09959\n",
      "Document Kappadione - similarity: -0.09971\n",
      "Document 2-(Beta-D-Glucopyranosyl)-5-Methyl-1,3,4-Oxadiazole - similarity: -0.10023\n",
      "Document [3-(Dodecanoylamino)Propyl](Hydroxy)Dimethylammonium - similarity: -0.10046\n",
      "Document Bacillus calmette-guerin substrain tokyo 172-1 live antigen - similarity: -0.10067\n",
      "Document Gallium chloride Ga-67 - similarity: -0.10069\n",
      "Document CP-94707 - similarity: -0.10069\n",
      "Document Influenza B virus B/Massachusetts/2/2012 BX-51B antigen (UV, formaldehyde inactivated) - similarity: -0.10086\n",
      "Document (8E,10S,12Z)-10-hydroxy-6-oxooctadeca-8,12-dienoic acid - similarity: -0.10094\n",
      "Document 2-[METHYL-(5-GERANYL-4-METHYL-PENT-3-ENYL)-AMINO]-ETHYL-DIPHOSPHATE - similarity: -0.10111\n",
      "Document 6-(5-BROMO-2-HYDROXYPHENYL)-2-OXO-4-PHENYL-1,2-DIHYDROPYRIDINE-3-CARBONITRILE - similarity: -0.10117\n",
      "Document INHIBITOR Q8467 OF DUPONT MERCK - similarity: -0.10125\n",
      "Document LFA703 - similarity: -0.10132\n",
      "Document Veglin - similarity: -0.10140\n",
      "Document 1-[4-(Octahydro-Pyrido[1,2-a]Pyrazin-2-Yl)-Phenyl]-2-Phenyl-1,2,3,4-Tetrahydro-Isoquinolin-6-Ol - similarity: -0.10146\n",
      "Document Influenza A virus A/Brisbane/02/2018 IVR-190 (H1N1) antigen (propiolactone inactivated) - similarity: -0.10225\n",
      "Document 3-{5-[AMINO(IMINIO)METHYL]-1H-INDOL-2-YL}-5-METHOXY-1,1'-BIPHENYL-2-OLATE - similarity: -0.10252\n",
      "Document B-2-Octylglucoside - similarity: -0.10343\n",
      "Document Glyceryl 1-oleate - similarity: -0.10370\n",
      "Document N-[(Furan-2-Yl)Carbonyl]-(S)-Leucyl-(R)-[1-Amino-2(1h-Indol-3-Yl)Ethyl]-Phosphonic Acid - similarity: -0.10396\n",
      "Document Mevinolinic acid - similarity: -0.10415\n",
      "Document 1-METHYL-3-TRIFLUOROMETHYL-1H-THIENO[2,3-C]PYRAZOLE-5-CARBOXYLIC ACID (2-MERCAPTO-ETHYL)-AMIDE - similarity: -0.10421\n",
      "Document WX-UK1 - similarity: -0.10434\n",
      "Document 3',5'-Dinitro-N-Acetyl-L-Thyronine - similarity: -0.10468\n",
      "Document P-57AS3 - similarity: -0.10484\n",
      "Document 2,5-di-tert-butylhydroquinone - similarity: -0.10485\n",
      "Document Galicaftor - similarity: -0.10489\n",
      "Document S-oxy-L-cysteine - similarity: -0.10511\n",
      "Document Iopodic acid - similarity: -0.10535\n",
      "Document 1-CHLORO-6-(4-HYDROXYPHENYL)-2-NAPHTHOL - similarity: -0.10576\n",
      "Document Cenderitide - similarity: -0.10626\n",
      "Document ZD-6126 - similarity: -0.10708\n",
      "Document 4-Methoxyamphetamine - similarity: -0.10730\n",
      "Document N-[(2S,4S,6R)-2-(dihydroxymethyl)-4-hydroxy-3,3-dimethyl-7-oxo-4lambda~4~-thia-1-azabicyclo[3.2.0]hept-6-yl]-2-phenylacetamide - similarity: -0.10740\n",
      "Document N'-(5-CHLOROBENZOFURAN-2-CARBONYL)-2-(TRIFLUOROMETHYL)BENZENESULFONOHYDRAZIDE - similarity: -0.10779\n",
      "Document Influenza B virus B/Massachusetts/2/2012 BX-51B antigen (formaldehyde inactivated) - similarity: -0.10786\n",
      "Document 2-Amino-3-(1h-Indol-3-Yl)-Propan-1-Ol - similarity: -0.10821\n",
      "Document 2-Morpholinoethylamine - similarity: -0.10849\n",
      "Document 2-(3,4-Dihydro-3-Oxo-2h-Benzo[B][1,4]Thiazin-2-Yl)-N-Hydroxyacetamide - similarity: -0.10889\n",
      "Document 5-Aminoimidazole Ribonucleoside - similarity: -0.10892\n",
      "Document Apronalide - similarity: -0.10926\n",
      "Document DTP-348 - similarity: -0.10967\n",
      "Document Robinia pseudoacacia pollen - similarity: -0.10974\n",
      "Document Influenza A virus A/Singapore/INFIMH-16-0019/2016 IVR-186 (H3N2) antigen (propiolactone inactivated) - similarity: -0.10984\n",
      "Document (2S)-2-{[HYDROXY(4-IODOBENZYL)PHOSPHORYL]METHYL}PENTANEDIOIC ACID - similarity: -0.11014\n",
      "Document 4-[8-(3-nitrophenyl)-1,7-naphthyridin-6-yl]benzoic acid - similarity: -0.11016\n",
      "Document Tricosanoic acid - similarity: -0.11083\n",
      "Document 9-deazainosine - similarity: -0.11118\n",
      "Document 1-(5-Tert-Butyl-2-Methyl-2h-Pyrazol-3-Yl)-3-(4-Chloro-Phenyl)-Urea - similarity: -0.11121\n",
      "Document Magnesium salicylate - similarity: -0.11122\n",
      "Document 4-{[5-(CYCLOHEXYLMETHOXY)[1,2,4]TRIAZOLO[1,5-A]PYRIMIDIN-7-YL]AMINO}BENZENESULFONAMIDE - similarity: -0.11138\n",
      "Document Vapitadine - similarity: -0.11146\n",
      "Document 6((S)-3-Benzylpiperazin-1-Yl)-3-(Naphthalen-2-Yl)-4-(Pyridin-4-Yl)Pyrazine - similarity: -0.11155\n",
      "Document Cysteine-Methylene-Carbamoyl-1,10-Phenanthroline - similarity: -0.11157\n",
      "Document Degraded Cephaloridine - similarity: -0.11166\n",
      "Document Moschus berezovskii musk sac resin - similarity: -0.11176\n",
      "Document ETHYL 4-[(4-METHYLPYRIDIN-2-YL)AMINO]PIPERIDINE-1-CARBOXYLATE - similarity: -0.11184\n",
      "Document Moxalactam (hydrolyzed) - similarity: -0.11220\n",
      "Document GSK-1292263 - similarity: -0.11228\n",
      "Document Spaglumic Acid - similarity: -0.11243\n",
      "Document (1S)-1-Phenylethyl (4-acetamidobenzyl)phosphonate - similarity: -0.11248\n",
      "Document 3-(3,5-Dibromo-4-Hydroxy-Benzoyl)-2-Ethyl-Benzofuran-6-Sulfonic Acid [4-(Thiazol-2-Ylsulfamoyl)-Phenyl]-Amide - similarity: -0.11263\n",
      "Document RU78191 - similarity: -0.11264\n",
      "Document alpha-D-galacturonic acid - similarity: -0.11266\n",
      "Document Betacetylmethadol - similarity: -0.11279\n",
      "Document S-{2-[(2-chloro-4-sulfamoylphenyl)amino]-2-oxoethyl} 6-methyl-3,4-dihydroquinoline-1(2H)-carbothioate - similarity: -0.11292\n",
      "Document F-Loop of Vitamin B12 - similarity: -0.11396\n",
      "Document Bisindolylmaleimide VIII - similarity: -0.11428\n",
      "Document 5-Amidino-Benzimidazole - similarity: -0.11516\n",
      "Document 3-Iodo-Tyrosine - similarity: -0.11534\n",
      "Document Linsidomine - similarity: -0.11538\n",
      "Document Mesoheme - similarity: -0.11544\n",
      "Document 2-[2-(1,3-Dioxo-1,3-Dihydro-2h-Isoindol-2-Yl)Ethyl]-4-(4'-Ethoxy-1,1'-Biphenyl-4-Yl)-4-Oxobutanoic Acid - similarity: -0.11602\n",
      "Document Capric dimethyl amine oxide - similarity: -0.11733\n",
      "Document Acetrizoic acid - similarity: -0.11743\n",
      "Document Topsalysin - similarity: -0.11744\n",
      "Document N6-(2,5-Dimethoxy-Benzyl)-N6-Methyl-Pyrido[2,3-D]Pyrimidine-2,4,6-Triamine - similarity: -0.11767\n",
      "Document 4-[5-(3-IODO-PHENYL)-2-(4-METHANESULFINYL-PHENYL)-1H-IMIDAZOL-4-YL]-PYRIDINE - similarity: -0.11768\n",
      "Document Xylose-Derived Imidazole - similarity: -0.11784\n",
      "Document Ginsenoside C - similarity: -0.11798\n",
      "Document N-[2-hydroxy-1-indanyl]-5-[(2-tertiarybutylaminocarbonyl)-4(benzo[1,3]dioxol-5-ylmethyl)-piperazino]-4-hydroxy-2-(1-phenylethyl)-pentanamide - similarity: -0.11800\n",
      "Document Cupric cation - similarity: -0.11800\n",
      "Document L-Threonohydroxamate 4-Phosphate - similarity: -0.11813\n",
      "Document RWJ-51084 - similarity: -0.11836\n",
      "Document Metiamide - similarity: -0.11848\n",
      "Document 9-Aminophenanthrene - similarity: -0.11910\n",
      "Document INKP-102 - similarity: -0.11911\n",
      "Document 3,8-Diamino-6-Phenyl-5-[6-[1-[2-[(1,2,3,4-Tetrahydro-9-Acridinyl)Amino]Ethyl]-1h-1,2,3-Triazol-4-Yl]Hexyl]-Phenanthridinium - similarity: -0.11923\n",
      "Document Zuretinol acetate - similarity: -0.11936\n",
      "Document AL5424 - similarity: -0.11942\n",
      "Document Astuprotimut-R - similarity: -0.11954\n",
      "Document 2-deoxy-2-acetamido-beta-D-galactose-4-sulfate - similarity: -0.11964\n",
      "Document Lactaldehyde - similarity: -0.11992\n",
      "Document 9,9,9-TRIFLUORO-8-OXO-N-PHENYLNONANAMIDE - similarity: -0.12116\n",
      "Document 3'-O-N-Octanoyl-a-D-Glucopyranosyl-B-D-Fructofuranoside - similarity: -0.12124\n",
      "Document Ethylmethylthiambutene - similarity: -0.12147\n",
      "Document Beta-Hydroxyaspartic Acid - similarity: -0.12180\n",
      "Document N-Benzylformamide - similarity: -0.12252\n",
      "Document Glutamyl Group - similarity: -0.12279\n",
      "Document VPM4001 - similarity: -0.12289\n",
      "Document BIS-1,2-{[(Z)-2CARBOXY-2-METHYL-1,3-DIOXANE]-5-YLOXYCARBONYL}-PIPERAZINE - similarity: -0.12292\n",
      "Document Z-Val-Ala-Asp fluoromethyl ketone - similarity: -0.12379\n",
      "Document Dexpropranolol - similarity: -0.12405\n",
      "Document 4-(1R,3AS,4R,8AS,8BR)-[1-DIFLUOROMETHYL-2-(4-FLUOROBENZYL)-3-OXODECAHYDROPYRROLO[3,4-A]PYRROLIZIN-4-YL]BENZAMIDINE - similarity: -0.12501\n",
      "Document IDM-4 - similarity: -0.12581\n",
      "Document Hydantocidin-5'-phosphate - similarity: -0.12583\n",
      "Document Nanatinostat - similarity: -0.12594\n",
      "Document AeroLEF - similarity: -0.12603\n",
      "Document MS-1819 - similarity: -0.12629\n",
      "Document Cordycepin Triphosphate - similarity: -0.12693\n",
      "Document beta-D-fructofuranose 1,6-bisphosphate - similarity: -0.12714\n",
      "Document 2-(2-QUINOLIN-3-YLPYRIDIN-4-YL)-1,5,6,7-TETRAHYDRO-4H-PYRROLO[3,2-C]PYRIDIN-4-ONE - similarity: -0.12733\n",
      "Document (2S)-2-[(3aR,4R,7S,7aS)-1,3-dioxooctahydro-2H-4,7-methanoisoindol-2-yl]propanoic acid - similarity: -0.12815\n",
      "Document Quinethazone - similarity: -0.12838\n",
      "Document (2E)-2-({(2S)-2-CARBOXY-2-[(PHENOXYACETYL)AMINO]ETHOXY}IMINO)PENTANEDIOIC ACID - similarity: -0.12924\n",
      "Document (S)-2-Amino-3-(6h-Selenolo[2,3-B]-Pyrrol-4-Yl)-Propionic Acid - similarity: -0.12941\n",
      "Document (2s)-4-(Beta-Alanylamino)-2-Aminobutanoic Acid - similarity: -0.12996\n",
      "Document 2-[2-(2-Cyclohexyl-2-guanidino-acetylamino)-acetylamino]-N-(3-mercapto-propyl)-propionamide - similarity: -0.13017\n",
      "Document N-METHYL-1-[4-(9H-PURIN-6-YL)PHENYL]METHANAMINE - similarity: -0.13059\n",
      "Document 1-Piperidinocyclohexanecarbonitrile - similarity: -0.13061\n",
      "Document Ketazolam - similarity: -0.13063\n",
      "Document 3-(6-HYDROXY-NAPHTHALEN-2-YL)-BENZO[D]ISOOXAZOL-6-OL - similarity: -0.13095\n",
      "Document Meningococcal polysaccharide vaccine group W-135 - similarity: -0.13121\n",
      "Document Beta-3-Cysteine - similarity: -0.13147\n",
      "Document N,4-Dimethyl-N-(1-phenyl-1H-pyrazol-5-yl)benzenesulfonamide - similarity: -0.13148\n",
      "Document RU79256 - similarity: -0.13170\n",
      "Document Abequose - similarity: -0.13190\n",
      "Document 5-{[Ethyl(Methyl)Amino]Methyl}-2-Methyl-5,6-Dihydropyrimidin-4-Amine - similarity: -0.13261\n",
      "Document CGC-11047 - similarity: -0.13298\n",
      "Document Soblidotin - similarity: -0.13350\n",
      "Document ZYC300 - similarity: -0.13383\n",
      "Document R-1487 - similarity: -0.13417\n",
      "Document 3-[4-(2,4-Dimethyl-Thiazol-5-Yl)-Pyrimidin-2-Ylamino]-Phenol - similarity: -0.13449\n",
      "Document Hadacidin - similarity: -0.13463\n",
      "Document N-(5-{3,4-difluoro-2-[(2-fluoro-4-iodophenyl)amino]phenyl}-1,3,4-oxadiazol-2-yl)ethane-1,2-diamine - similarity: -0.13548\n",
      "Document NRP409 - similarity: -0.13569\n",
      "Document K-00546 - similarity: -0.13576\n",
      "Document Arylacenamide - similarity: -0.13640\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) hemagglutinin antigen (formaldehyde inactivated) - similarity: -0.13655\n",
      "Document Ebola Zaire vaccine (live, attenuated) - similarity: -0.13703\n",
      "Document 2,4-Dihydroxy-7-(Methyloxy)-2h-1,4-Benzoxazin-3(4h)-One - similarity: -0.13723\n",
      "Document 3-[3-(2,3-Dihydroxy-Propylamino)-Phenyl]-4-(5-Fluoro-1-Methyl-1h-Indol-3-Yl)-Pyrrole-2,5-Dione - similarity: -0.13724\n",
      "Document Palmidrol - similarity: -0.13739\n",
      "Document ETC-1001 - similarity: -0.13748\n",
      "Document Amolimogene - similarity: -0.13806\n",
      "Document Piboserod - similarity: -0.13825\n",
      "Document Iodofiltic acid I-123 - similarity: -0.13874\n",
      "Document (5S)-2-{[(1S)-1-(2-fluorophenyl)ethyl]amino}-5-methyl-5-(trifluoromethyl)-1,3-thiazol-4(5H)-one - similarity: -0.13899\n",
      "Document ATL-104 - similarity: -0.13931\n",
      "Document 2-Chloro-6-Methyl-Aniline - similarity: -0.13937\n",
      "Document Deacetoxyvinzolidine - similarity: -0.13955\n",
      "Document (3S)-3-(dioxidosulfanyl)-N-[(1Z)-3-oxoprop-1-en-1-yl]-4-(1H-1,2,3-triazol-1-yl)-D-valine - similarity: -0.14008\n",
      "Document Isoglutamine - similarity: -0.14032\n",
      "Document CYCLOHEXYLMETHYL-2,3-DIHYDROXY-5-METHYL-HEXYLAMIDE - similarity: -0.14119\n",
      "Document 2-(acetylamino)-2-deoxy-4-O-sulfo-alpha-D-galactopyranose - similarity: -0.14168\n",
      "Document Influenza B virus B/Maryland/15/2016 BX-69A antigen (UV, formaldehyde inactivated) - similarity: -0.14177\n",
      "Document (S)-TETRAHYDROFURAN-3-YL (2S,3S)-4-((S)-4-((1R,3R)-3-(2-AMINO-2-OXOETHYL)-2,3-DIHYDRO-1H-INDEN-1-YL)-2-BENZYL-3-OXO-2,3-DIHYDRO-1H-PYRROL-2-YL)-3-HYDROXY-1-PHENYLBUTAN-2-YLCARBAMATE - similarity: -0.14314\n",
      "Document Ziresovir - similarity: -0.14350\n",
      "Document (3aS,4R,9bR)-2,2-difluoro-4-(4-hydroxyphenyl)-6-(methoxymethyl)-1,2,3,3a,4,9b-hexahydrocyclopenta[c]chromen-8-ol - similarity: -0.14487\n",
      "Document (2Z)-N-(3-chloro-2'-methoxybiphenyl-4-yl)-2-cyano-3-hydroxybut-2-enamide - similarity: -0.14515\n",
      "Document 3-[(9H-fluoren-9-ylideneamino)oxy]propanoic acid - similarity: -0.14520\n",
      "Document Friulimicin B - similarity: -0.14534\n",
      "Document Rumex altissimus pollen - similarity: -0.14536\n",
      "Document Influenza B virus B/Phuket/3073/2013 hemagglutinin antigen (propiolactone inactivated) - similarity: -0.14545\n",
      "Document Antrafenine - similarity: -0.14578\n",
      "Document Valomaciclovir stearate - similarity: -0.14699\n",
      "Document Difenoxin - similarity: -0.14706\n",
      "Document 6-Carboxymethyluracil - similarity: -0.14708\n",
      "Document 3-(4-Amino-2-Tert-Butyl-5-Methyl-Phenylsulfanyl)-6-Cyclopentyl-4-Hydroxy-6-[2-(4-Hydroxy-Phenyl)-Ethyl]-5,6-Dihydro-Pyran-2-One - similarity: -0.14728\n",
      "Document 9-Butyl-8-(3,4,5-Trimethoxybenzyl)-9h-Purin-6-Amine - similarity: -0.14743\n",
      "Document GLYCYLALANYL-N-2-NAPHTHYL-L-PROLINEAMIDE - similarity: -0.14746\n",
      "Document C-(1-hydrogyl-beta-D-glucopyranosyl) formamide - similarity: -0.14772\n",
      "Document ZK-806711 - similarity: -0.14791\n",
      "Document Propanoyl-CoA - similarity: -0.14835\n",
      "Document 2-[4-(4-Chlorophenyl)Cyclohexylidene]-3,4-Dihydroxy-1(2h)-Naphthalenone - similarity: -0.14850\n",
      "Document D-leucyl-N-(3-chlorobenzyl)-L-prolinamide - similarity: -0.14885\n",
      "Document Rosoxacin - similarity: -0.14899\n",
      "Document 2-HYDROXY-3,5-DIIODOBENZOIC ACID - similarity: -0.14908\n",
      "Document ATL1101 - similarity: -0.14927\n",
      "Document 1,3-CYCLOHEXANEDIOL, 4-METHYLENE-5-[(2E)-[(1S,3AS,7AS)-OCTAHYDRO-1-(5-HYDROXY-5-METHYL-1,3-HEXADIYNYL)-7A-METHYL-4H-INDEN-4-YLIDENE]ETHYLIDENE]-, (1R,3S,5Z) - similarity: -0.15026\n",
      "Document 4-Deoxy-Alpha-D-Glucose - similarity: -0.15038\n",
      "Document Dexamethasone metasulfobenzoate - similarity: -0.15080\n",
      "Document PCS-499 - similarity: -0.15114\n",
      "Document Epitizide - similarity: -0.15159\n",
      "Document alpha-N-Acetyl-D-galactosamine - similarity: -0.15161\n",
      "Document Influenza B virus B/Hong Kong/259/2010 hemagglutinin antigen (MDCK cell derived, propiolactone inactivated) - similarity: -0.15192\n",
      "Document Methyl-O3-(Alpha-D-Mannose)-Alpha-D-Mannose - similarity: -0.15197\n",
      "Document Bencyclane - similarity: -0.15210\n",
      "Document 4-{[5-chloro-4-(1H-indol-3-yl)pyrimidin-2-yl]amino}-N-ethylpiperidine-1-carboxamide - similarity: -0.15215\n",
      "Document 4-[(2-{4-[(CYCLOPROPYLCARBAMOYL)AMINO]-1H-PYRAZOL-3-YL}-1H-BENZIMIDAZOL-6-YL)METHYL]MORPHOLIN-4-IUM - similarity: -0.15319\n",
      "Document 9-N-Phenylmethylamino-Tacrine - similarity: -0.15345\n",
      "Document Allyl-{6-[3-(4-Bromo-Phenyl)-Benzofuran-6-Yloxy]-Hexyl-}-Methyl-Amin - similarity: -0.15413\n",
      "Document AZD-9496 - similarity: -0.15435\n",
      "Document N-(3-Carboxypropanoyl)-L-phenylalanyl-3-carboxy-O-(carboxymethyl)-N-pentyl-L-tyrosinamide - similarity: -0.15441\n",
      "Document 4-{2-[(7-amino-2-furan-2-yl[1,2,4]triazolo[1,5-a][1,3,5]triazin-5-yl)amino]ethyl}phenol - similarity: -0.15479\n",
      "Document Amfecloral - similarity: -0.15527\n",
      "Document Fenquizone - similarity: -0.15533\n",
      "Document Pheniprazine - similarity: -0.15534\n",
      "Document [2(R,S)-2-Sulfanylheptanoyl]-Phe-Ala - similarity: -0.15549\n",
      "Document 6-CYCLOHEXYLMETHYLOXY-5-NITROSO-PYRIMIDINE-2,4-DIAMINE - similarity: -0.15571\n",
      "Document AL5300 - similarity: -0.15591\n",
      "Document N-{3-[4-(3-amino-propyl)-piperazin-1-yl]-propyl}-3-nitro-5-(galactopyranosyl)-beta-benzamide - similarity: -0.15610\n",
      "Document Influenza A virus A/Slovenia/2903/2015 (H1N1) live (attenuated) antigen - similarity: -0.15627\n",
      "Document 2'-Deoxy-2'-[(3,5-dimethoxybenzoyl)amino]-N-[(1R)-1,2,3,4-tetrahydro-1-naphthalenyl]adenosine - similarity: -0.15692\n",
      "Document Indusatumab vedotin - similarity: -0.15695\n",
      "Document RPR128515 - similarity: -0.15733\n",
      "Document Sodium aurotiosulfate - similarity: -0.15737\n",
      "Document 1-{7-cyclohexyl-6-[4-(4-methylpiperazin-1-yl)benzyl]-7H-pyrrolo[2,3-d]pyrimidin-2-yl}methanamine - similarity: -0.15775\n",
      "Document PHENYLAMINOIMIDAZO(1,2-ALPHA)PYRIDINE - similarity: -0.15804\n",
      "Document [2-(1-amino-2-hydroxy-propyl)-4-(1H-indol-3-ylmethylene)-5-oxo-4,5-dihydro-imidazol-1-yl]-acetaldehyde - similarity: -0.15816\n",
      "Document Silodrate - similarity: -0.15832\n",
      "Document Hexadecyl Octanoate - similarity: -0.15845\n",
      "Document Metrizamide - similarity: -0.15854\n",
      "Document N-{[(2S,3S)-3-(ETHOXYCARBONYL)OXIRAN-2-YL]CARBONYL}-L-ISOLEUCYL-L-ISOLEUCINE - similarity: -0.15916\n",
      "Document 3-(Benzoylamino)-L-Alanine - similarity: -0.15964\n",
      "Document Balanol Analog 1 - similarity: -0.15988\n",
      "Document 5-(4-PHENOXYPHENYL)-5-(4-PYRIMIDIN-2-YLPIPERAZIN-1-YL)PYRIMIDINE-2,4,6(2H,3H)-TRIONE - similarity: -0.16001\n",
      "Document Sodium aurothiomalate - similarity: -0.16007\n",
      "Document Influenza B virus B/Brisbane/9/2014 hemagglutinin antigen (formaldehyde inactivated) - similarity: -0.16024\n",
      "Document Meprednisone acetate - similarity: -0.16034\n",
      "Document KB-141 - similarity: -0.16054\n",
      "Document 4-(Hydroxymethyl)Benzamidine - similarity: -0.16056\n",
      "Document Fenyramidol - similarity: -0.16062\n",
      "Document 8,9-Dichloro-2,3,4,5-Tetrahydro-1h-Benzo[C]Azepine - similarity: -0.16071\n",
      "Document DERIVATIVE OF AKLANONIC ACID METHYL ESTER (AAME) - similarity: -0.16086\n",
      "Document Tilia americana pollen - similarity: -0.16193\n",
      "Document (8alpha,10alpha,13alpha,17beta)-17-[(4-hydroxyphenyl)carbonyl]androsta-3,5-diene-3-carboxylic acid - similarity: -0.16215\n",
      "Document S-Hydroxycysteine - similarity: -0.16241\n",
      "Document (1S)-1-(PHENOXYMETHYL)PROPYL METHYLPHOSPHONOCHLORIDOATE - similarity: -0.16263\n",
      "Document (1H-indol-3-yl)-(2-mercapto-ethoxyimino)-acetic acid - similarity: -0.16345\n",
      "Document 3-(Prop-2-Ene-1-Sulfinyl)-Propene-1-Thiol - similarity: -0.16396\n",
      "Document 4-Bromo-3-(5'-Carboxy-4'-Chloro-2'-Fluorophenyl)-1-Methyl-5-Trifluoromethyl-Pyrazol - similarity: -0.16464\n",
      "Document Sulfinpyrazone - similarity: -0.16522\n",
      "Document LY-517717 - similarity: -0.16591\n",
      "Document GW-493838 - similarity: -0.16601\n",
      "Document 4-(6-{[(4-METHYLCYCLOHEXYL)AMINO]METHYL}-1,4-DIHYDROINDENO[1,2-C]PYRAZOL-3-YL)BENZOIC ACID - similarity: -0.16608\n",
      "Document 3-[5-(3-nitrophenyl)thiophen-2-yl]propanoic acid - similarity: -0.16637\n",
      "Document Methyl beta-galactoside - similarity: -0.16658\n",
      "Document 2-deoxy-2-fluoro-α-D-mannose - similarity: -0.16697\n",
      "Document 3-Decyl-2,5-Dioxo-4-Hydroxy-3-Pyrroline - similarity: -0.16704\n",
      "Document N'-[4-(2,4-Dimethyl-1,3-thiazol-5-yl)-2-pyrimidinyl]-N-hydroxyimidoformamide - similarity: -0.16734\n",
      "Document (3-Carboxy-2-(R)-Hydroxy-Propyl)-Trimethyl-Ammonium - similarity: -0.16750\n",
      "Document Thymidine-5'-diphospho-beta-D-xylose - similarity: -0.16776\n",
      "Document 5-[2-(1H-pyrrol-1-yl)ethoxy]-1H-indole - similarity: -0.16779\n",
      "Document (2s,3r)-3-Amino-2-Hydroxy-5-(Ethylsulfanyl)Pentanoyl-((S)-(-)-(1-Naphthyl)Ethyl)Amide - similarity: -0.16786\n",
      "Document Meclinertant - similarity: -0.16813\n",
      "Document AL7089A - similarity: -0.16868\n",
      "Document 2-(4-hydroxybiphenyl-3-yl)-4-methyl-1H-isoindole-1,3(2H)-dione - similarity: -0.16911\n",
      "Document Cyclotheonamide A - similarity: -0.16942\n",
      "Document S-Mercaptocysteine - similarity: -0.16957\n",
      "Document Dextrin 2-sulfate - similarity: -0.16980\n",
      "Document Salophen iron chelate - similarity: -0.16985\n",
      "Document Hexatantalum Dodecabromide - similarity: -0.16989\n",
      "Document Landogrozumab - similarity: -0.17057\n",
      "Document Acediasulfone - similarity: -0.17066\n",
      "Document Simfibrate - similarity: -0.17066\n",
      "Document 3-(3,4-dimethoxyphenyl)propanoic acid - similarity: -0.17067\n",
      "Document AMPCPR - similarity: -0.17100\n",
      "Document Zomepirac - similarity: -0.17108\n",
      "Document 2-CHLORO-5-(3-CHLORO-PHENYL)-6-[(4-CYANO-PHENYL)-(3-METHYL-3H-IMIDAZOL-4-YL)- METHOXYMETHYL]-NICOTINONITRILE - similarity: -0.17134\n",
      "Document 3,4-Dihydro-5-Methyl-Isoquinolinone - similarity: -0.17170\n",
      "Document 3-(6-Aminopyridin-3-Yl)-N-Methyl-N-[(1-Methyl-1h-Indol-2-Yl)Methyl]Acrylamide - similarity: -0.17171\n",
      "Document 1-cyclobutyl-3-(3,4-dimethoxyphenyl)-1H-pyrazolo[3,4-d]pyrimidin-4-amine - similarity: -0.17278\n",
      "Document METHYL 1-(4-{[(2,4-DIAMINOPTERIDIN-6-YL)METHYL](METHYL)AMINO}BENZOYL)PIPERIDINE-4-CARBOXYLATE - similarity: -0.17280\n",
      "Document Streptococcus pneumoniae type 19a capsular polysaccharide diphtheria crm197 protein conjugate antigen - similarity: -0.17289\n",
      "Document Glutaral - similarity: -0.17293\n",
      "Document N-[(2S)-4-Methyl-1-oxo-1-{[(4S)-3-oxo-1-(2-pyridinylsulfonyl)-4-azepanyl]amino}-2-pentanyl]-1-benzofuran-2-carboxamide - similarity: -0.17347\n",
      "Document 5-methylbenzimidazole - similarity: -0.17410\n",
      "Document N-[(2',4'-DIFLUORO-4-HYDROXY-5-IODOBIPHENYL-3-YL)CARBONYL]-BETA-ALANINE - similarity: -0.17411\n",
      "Document 4-{4-[3-(2,4-DICHLORO-BENZOYL)-UREIDO]-2,3-DIMETHYL-PHENOXY}-BUTYRIC ACID - similarity: -0.17466\n",
      "Document CX501 - similarity: -0.17492\n",
      "Document Sulglicotide - similarity: -0.17495\n",
      "Document LGD-1550 - similarity: -0.17536\n",
      "Document S-benzylglutathione - similarity: -0.17565\n",
      "Document (3-Chloro-4-Propoxy-Phenyl)-Acetic Acid - similarity: -0.17599\n",
      "Document Interferon alfacon-1 - similarity: -0.17600\n",
      "Document 3,4-Dihydroxy-1-Methylquinolin-2(1h)-One - similarity: -0.17602\n",
      "Document Thallous cation Tl-201 - similarity: -0.17651\n",
      "Document (2S)-1-(Dimethylamino)-3-(4-{[4-(2-methylimidazo[1,2-a]pyridin-3-yl)-2-pyrimidinyl]amino}phenoxy)-2-propanol - similarity: -0.17707\n",
      "Document Influenza A virus A/Victoria/361/2011 IVR-165 (H3N2) hemagglutinin antigen (propiolactone inactivated) - similarity: -0.17731\n",
      "Document N-Ethyl-5'-Carboxamido Adenosine - similarity: -0.17758\n",
      "Document Verpacamide A - similarity: -0.17835\n",
      "Document SR-9011 - similarity: -0.17915\n",
      "Document 2-{[4-(2-Acetylamino-2-pentylcarbamoyl-ethyl)-naphthalen-1-YL]-oxalyl-amino}-benzoic acid - similarity: -0.17958\n",
      "Document Ustilago avenae - similarity: -0.18027\n",
      "Document 7-methyl-7,8-dihydroguanosine-5'-diphosphate - similarity: -0.18062\n",
      "Document o-Bromophenol - similarity: -0.18105\n",
      "Document (3S)-N-(3-BROMOPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE - similarity: -0.18176\n",
      "Document 3'-phospho-5'-adenylyl sulfate - similarity: -0.18182\n",
      "Document Methacrylyl-Coenzyme A - similarity: -0.18184\n",
      "Document Biotinyl P-Nitroaniline - similarity: -0.18188\n",
      "Document [4-(4-Hydroxy-Benzyl)-2-(2-Hydroxy-1-Methyl-Ethyl)-5-Oxo-Imidazolidin-1-Yl]-Acetic Acid - similarity: -0.18192\n",
      "Document Enprofylline - similarity: -0.18219\n",
      "Document Magnesium ascorbate - similarity: -0.18288\n",
      "Document METHYL (3R)-3-{[(3R)-3-{[(3R)-3-HYDROXYBUTANOYL]OXY}BUTANOYL]OXY}BUTANOATE - similarity: -0.18327\n",
      "Document Pecilocin - similarity: -0.18363\n",
      "Document 2-[3,4-Dihydroxy-2-Hydroxymethyl-5-(2-Hydroxy-Nonyl)-Tetrahydro-Furan-2-Yloxy]-6-Hydroxymethyl-Tetra Hydro-Pyran-3,4,5-Triol - similarity: -0.18401\n",
      "Document 3-Methylphenylalanine - similarity: -0.18411\n",
      "Document Influenza B virus B/Wisconsin/1/2010 live (attenuated) antigen - similarity: -0.18464\n",
      "Document 4-(6-HYDROXY-BENZO[D]ISOXAZOL-3-YL)BENZENE-1,3-DIOL - similarity: -0.18476\n",
      "Document Influenza B virus B/Brisbane/46/2015 antigen (propiolactone inactivated) - similarity: -0.18502\n",
      "Document 2-Methoxy-3-Isopropylpyrazine - similarity: -0.18508\n",
      "Document Fluclorolone acetonide - similarity: -0.18531\n",
      "Document D-glucitol 6-phosphate - similarity: -0.18533\n",
      "Document 1-N-(4-SULFAMOYLPHENYL-ETHYL)-2,4,6-TRIMETHYLPYRIDINIUM - similarity: -0.18543\n",
      "Document Pentolinium - similarity: -0.18554\n",
      "Document Ethyl Dihydrogen Diphosphate - similarity: -0.18579\n",
      "Document Ardeparin - similarity: -0.18596\n",
      "Document Kelatorphan - similarity: -0.18605\n",
      "Document N~2~-Succinylornithine - similarity: -0.18676\n",
      "Document Carya cordiformis pollen - similarity: -0.18779\n",
      "Document N-(2-Acetamido)Iminodiacetic Acid - similarity: -0.18822\n",
      "Document Benmoxin - similarity: -0.18873\n",
      "Document ZK-805623 - similarity: -0.18925\n",
      "Document 2,5-Anhydroglucitol-1,6-Biphosphate - similarity: -0.18932\n",
      "Document (+)-2-(4-biphenyl)propionic acid - similarity: -0.18935\n",
      "Document Dimethylallyl S-Thiolodiphosphate - similarity: -0.18948\n",
      "Document AGRO100 - similarity: -0.18967\n",
      "Document Thymidine-5'-(dithio)phosphate - similarity: -0.18988\n",
      "Document Br-Coeleneterazine - similarity: -0.18995\n",
      "Document N-[(6-butoxynaphthalen-2-yl)sulfonyl]-D-glutamic acid - similarity: -0.18997\n",
      "Document N-hexanoyl-L-homocysteine - similarity: -0.19046\n",
      "Document 3-Amino-4,5-Dihydroxy-Cyclohex-1-Enecarboxylate - similarity: -0.19081\n",
      "Document RU78299 - similarity: -0.19095\n",
      "Document Ohmefentanyl - similarity: -0.19122\n",
      "Document N-Hydroxy-1-(4-Methoxyphenyl)Sulfonyl-4-Benzyloxycarbonyl-Piperazine-2-Carboxamide - similarity: -0.19178\n",
      "Document (2S,3S,4E,6E,8S,9S)-3-amino-9-methoxy-2,6,8-trimethyl-10-phenyldeca-4,6-dienoic acid - similarity: -0.19212\n",
      "Document 4-(1,3-BENZODIOXOL-5-YLOXY)-2-[4-(1H-IMIDAZOL-1-YL)PHENOXY]-6-METHYLPYRIMIDINE - similarity: -0.19283\n",
      "Document 2-Deoxy-alpha-D-galactopyranose - similarity: -0.19314\n",
      "Document 2,6-diaminoquinazolin-4-ol - similarity: -0.19314\n",
      "Document Influenza B virus B/Texas/6/2011 hemagglutinin antigen (formaldehyde inactivated) - similarity: -0.19358\n",
      "Document (2R,3R,4S,5R,6E)-3,4,5-Trihydroxy-N-[(3S,6R)-6-hydroxy-2-oxo-3-azepanyl]-2-methoxy-8,8-dimethyl-6-nonenamide - similarity: -0.19377\n",
      "Document 6-Hydroxy-5-undecyl-4,7-benzothiazoledione - similarity: -0.19421\n",
      "Document 2-Methylpentane-1,2,4-Triol - similarity: -0.19450\n",
      "Document N6-Benzyl Adenosine-5'-Diphosphate - similarity: -0.19488\n",
      "Document TERT-BUTYL {2-[(1,3-THIAZOL-2-YLAMINO)CARBONYL]PYRIDIN-3-YL}CARBAMATE - similarity: -0.19552\n",
      "Document Bordetella pertussis toxoid antigen (formaldehyde inactivated) - similarity: -0.19556\n",
      "Document (2S)-N-(4-cyano-3-iodophenyl)-3-(4-cyanophenoxy)-2-hydroxy-2-methylpropanamide - similarity: -0.19564\n",
      "Document KD7040 - similarity: -0.19649\n",
      "Document Hydrocortamate - similarity: -0.19650\n",
      "Document Verpasep caltespen - similarity: -0.19668\n",
      "Document 3-[N-[benzyloxycarbonyl]-phenylalaninyl-amino]-5-phenyl-pentane-1-sulfonylmethylbenzene - similarity: -0.19717\n",
      "Document Bromo-WR99210 - similarity: -0.19743\n",
      "Document (5-hydroxyindolo[1,2-a]quinazolin-7-yl)acetic acid - similarity: -0.19764\n",
      "Document WIN-54954 - similarity: -0.19791\n",
      "Document Uridine-5'-Diphosphate-N-Acetylmuramoyl-L-Alanine-D-Glutamate - similarity: -0.19798\n",
      "Document Di-Stearoyl-3-Sn-Phosphatidylcholine - similarity: -0.19833\n",
      "Document (2S,3S,4R,5S)-2-(4-Amino-4,5-dihydro-1H-pyrrolo[3,2-d]pyrimidin-7-yl)-5-[(methylsulfanyl)methyl]-3,4-pyrrolidinediol - similarity: -0.19950\n",
      "Document 2-[4-chloro-2-(phenylcarbonyl)phenoxy]-N-phenylacetamide - similarity: -0.19956\n",
      "Document Acetorphine - similarity: -0.20033\n",
      "Document (2R)-1-{4-[(4-Anilino-5-bromo-2-pyrimidinyl)amino]phenoxy}-3-(dimethylamino)-2-propanol - similarity: -0.20042\n",
      "Document 4-(Acetylamino)-3-Guanidinobenzoic Acid - similarity: -0.20122\n",
      "Document PM-060184 - similarity: -0.20163\n",
      "Document IMC-1C11 - similarity: -0.20168\n",
      "Document 5-(3,3-Dihydroxypropeny)-3-Methoxy-Benzene-1,2-Diol - similarity: -0.20178\n",
      "Document Mesulfen - similarity: -0.20180\n",
      "Document 6,6-DIMETHYL-1-[3-(2,4,5-TRICHLOROPHENOXY)PROPOXY]-1,6-DIHYDRO-1,3,5-TRIAZINE-2,4-DIAMINE - similarity: -0.20221\n",
      "Document 5,8-Di-Amino-1,4-Dihydroxy-Anthraquinone - similarity: -0.20271\n",
      "Document 3-(CARBOXYMETHOXY)THIENO[2,3-B]PYRIDINE-2-CARBOXYLIC ACID - similarity: -0.20297\n",
      "Document 6,7,8,9-TETRAHYDRO-4-HYDROXY-3-(1-PHENYLPROPYL)CYCLOHEPTA[B]PYRAN-2-ONE - similarity: -0.20323\n",
      "Document 5-fluoro-beta-L-gulosyl fluoride - similarity: -0.20361\n",
      "Document [4-(5-naphthalen-2-yl-1H-pyrrolo[2,3-b]pyridin-3-yl)phenyl]acetic acid - similarity: -0.20396\n",
      "Document Cinoxate - similarity: -0.20417\n",
      "Document L-709,587 - similarity: -0.20435\n",
      "Document 4-(N,N-Dimethylamino)cinnamoyl-CoA - similarity: -0.20449\n",
      "Document Ghavamiol - similarity: -0.20458\n",
      "Document 4-Methyl-1,2-Benzenediol - similarity: -0.20536\n",
      "Document Cefilavancin - similarity: -0.20601\n",
      "Document Poliovirus type 1 antigen (formaldehyde inactivated) - similarity: -0.20631\n",
      "Document Praconase - similarity: -0.20634\n",
      "Document Mercuric amidochloride - similarity: -0.20655\n",
      "Document DPI59 - similarity: -0.20668\n",
      "Document Desulfo-coenzyme A - similarity: -0.20688\n",
      "Document 3-({2-[(4-{[6-(CYCLOHEXYLMETHOXY)-9H-PURIN-2-YL]AMINO}PHENYL)SULFONYL]ETHYL}AMINO)PROPAN-1-OL - similarity: -0.20733\n",
      "Document Benzo[B]Thiophene-2-Carboxamidine - similarity: -0.20774\n",
      "Document Bamosiran - similarity: -0.20859\n",
      "Document Mecasermin rinfabate - similarity: -0.20877\n",
      "Document 1-(5-BROMO-PYRIDIN-2-YL)-3-[2-(6-FLUORO-2-HYDROXY-3-PROPIONYL-PHENYL)-CYCLOPROPYL]-UREA - similarity: -0.20887\n",
      "Document 1-(4-HEXYLPHENYL)PROP-2-EN-1-ONE - similarity: -0.20896\n",
      "Document Epicept NP-1 - similarity: -0.20931\n",
      "Document Brivudine monophosphate - similarity: -0.20952\n",
      "Document N-(5,6-DIPHENYLFURO[2,3-D]PYRIMIDIN-4-YL)GLYCINE - similarity: -0.21022\n",
      "Document Chamaemelum nobile flower - similarity: -0.21026\n",
      "Document (S)-(+)-2-[4-(FLUOROBENZYLOXY-BENZYLAMINO)PROPIONAMIDE] - similarity: -0.21043\n",
      "Document Glutethimide - similarity: -0.21047\n",
      "Document {[2-Amino-4-oxo-6,7-di(sulfanyl-κS)-3,5,5a,8,9a,10-hexahydro-4H-pyrano[3,2-g]pteridin-8-yl]methyl dihydrogenato(2-) phosphate}(dioxo)sulfanylmolybdenum - similarity: -0.21057\n",
      "Document 2-(2-METHYLPHENYL)-1H-INDOLE-5-CARBOXIMIDAMIDE - similarity: -0.21130\n",
      "Document PF-477736 - similarity: -0.21148\n",
      "Document 4-(Isopropylamino)diphenylamine - similarity: -0.21164\n",
      "Document Oxyphencyclimine - similarity: -0.21174\n",
      "Document Thioctic acid tromethamine - similarity: -0.21184\n",
      "Document N-Succinyl Methionine - similarity: -0.21336\n",
      "Document Salix lucida ssp. lasiandra pollen - similarity: -0.21361\n",
      "Document N-[(2R,3S)-3-AMINO-2-HYDROXY-4-PHENYLBUTYL]-4-METHOXY-2,3,6-TRIMETHYLBENZENESULFONAMIDE - similarity: -0.21384\n",
      "Document SB-249553 - similarity: -0.21392\n",
      "Document 1-(4-Aminophenyl)-3,5-Dimethyl-1h-Pyrazole-4-Carboxylic Acid Ethyl Ester - similarity: -0.21443\n",
      "Document SR11254 - similarity: -0.21509\n",
      "Document KRN-5500 - similarity: -0.21512\n",
      "Document Profenamine - similarity: -0.21547\n",
      "Document 2-Phenethyl-2,3-Dihydro-Phthalazine-1,4-Dione - similarity: -0.21557\n",
      "Document Clortermine - similarity: -0.21571\n",
      "Document 5'-Deoxy-5'-(Methylthio)-Tubercidin - similarity: -0.21619\n",
      "Document (S)-2-(Phosphonoxy)Caproyl-L-Leucyl-P-Nitroanilide - similarity: -0.21634\n",
      "Document Hydrolyzed Cephalothin - similarity: -0.21647\n",
      "Document LSZ-102 - similarity: -0.21705\n",
      "Document 2'-fluoro-2'-deoxyuridine 3'-monophosphate - similarity: -0.21710\n",
      "Document Zinc trihydroxide - similarity: -0.21765\n",
      "Document 7,10,13-Tri(Carboxymethyl)-5,15-Dioxo-4,7,10,13,16-Pentaaza-1,19-Dithianonadecane - similarity: -0.21854\n",
      "Document Sphinxolide B - similarity: -0.21895\n",
      "Document Acetyldigitoxin - similarity: -0.21917\n",
      "Document Bishydroxy[2h-1-Benzopyran-2-One,1,2-Benzopyrone] - similarity: -0.21957\n",
      "Document N-Valeric Acid - similarity: -0.22022\n",
      "Document CP-122721 - similarity: -0.22057\n",
      "Document 4-BROMO-3-(CARBOXYMETHOXY)-5-[3-(CYCLOHEXYLAMINO)PHENYL]THIOPHENE-2-CARBOXYLIC ACID - similarity: -0.22099\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 (H3N2) recombinant hemagglutinin antigen - similarity: -0.22102\n",
      "Document 2-((4'-HYDROXYPHENYL)-AZO)BENZOIC ACID - similarity: -0.22185\n",
      "Document Ilodecakin - similarity: -0.22228\n",
      "Document 4-(2-chlorophenyl)-8-(2-hydroxyethyl)-6-methylpyrrolo[3,4-e]indole-1,3(2H,6H)-dione - similarity: -0.22242\n",
      "Document SOT-107 - similarity: -0.22265\n",
      "Document Hg9a-9, Nonanoyl-N-Hydroxyethylglucamide - similarity: -0.22303\n",
      "Document NU1025 - similarity: -0.22313\n",
      "Document Echinacea purpurea flowering top - similarity: -0.22407\n",
      "Document Diethylene glycol ethyl methyl ether - similarity: -0.22408\n",
      "Document [2-Amino-6-(2,6-Difluoro-Benzoyl)-Imidazo[1,2-a]Pyridin-3-Yl]-Phenyl-Methanone - similarity: -0.22426\n",
      "Document 1-Hydroxy-2-Amino-3-Cyclohexylpropane - similarity: -0.22482\n",
      "Document 3,5,6,8-Tetramethyl-N-Methyl Phenanthrolinium - similarity: -0.22491\n",
      "Document SP4160 - similarity: -0.22529\n",
      "Document Levacetylmethadol - similarity: -0.22540\n",
      "Document Binodenoson - similarity: -0.22551\n",
      "Document Phensuximide - similarity: -0.22552\n",
      "Document Tomaralimab - similarity: -0.22572\n",
      "Document 4-Hydroxy-3,5-Dimethyl-5-(2-Methyl-Buta-1,3-Dienyl)-5h-Thiophen-2-One - similarity: -0.22631\n",
      "Document Guanidine-3-propanol - similarity: -0.22680\n",
      "Document 1-(4-Methoxyphenyl)-3,5-Dimethyl-1h-Pyrazole-4-Carboxylic Acid Ethyl Ester - similarity: -0.22681\n",
      "Document 2-(3-((4,5,7-trifluorobenzo[d]thiazol-2-yl)methyl)-1H-pyrrolo[2,3-b]pyridin-1-yl)acetic acid - similarity: -0.22727\n",
      "Document Diethyltryptamine - similarity: -0.22753\n",
      "Document N-(PARA-GLUTARAMIDOPHENYL-ETHYL)-PIPERIDINIUM-N-OXIDE - similarity: -0.22762\n",
      "Document GW 468816 - similarity: -0.22829\n",
      "Document 2-phospho-D-glyceric acid - similarity: -0.22864\n",
      "Document Hydroxyethylcysteine - similarity: -0.22876\n",
      "Document p-Hydroxymercuribenzoic acid - similarity: -0.23001\n",
      "Document delta-(L-alpha-Aminoadipoyl)-L-cysteinyl-D-vinylglycine - similarity: -0.23008\n",
      "Document 6-hydroxyuridine-5'-phosphate - similarity: -0.23026\n",
      "Document METHYL 1-(4-{[(2,4-DIAMINOPTERIDIN-6-YL)METHYL]AMINO}BENZOYL)PIPERIDINE-4-CARBOXYLATE - similarity: -0.23026\n",
      "Document (2S)-N-[(3S)-1-(2-AMINO-2-OXOETHYL)-2-OXO-1,2,3,4-TETRAHYDROQUINOLIN-3-YL]-2-CHLORO-2H-THIENO[2,3-B]PYRROLE-5-CARBOXAMIDE - similarity: -0.23035\n",
      "Document Ulobetasol - similarity: -0.23056\n",
      "Document 10-decarboxymethylaclacinomycin A - similarity: -0.23067\n",
      "Document 5-[4-Tert-Butylphenylsulfanyl]-2,4-Quinazolinediamine - similarity: -0.23070\n",
      "Document MB07811 - similarity: -0.23074\n",
      "Document Pterin-6-Yl-Methyl-Monophosphate - similarity: -0.23087\n",
      "Document Influenza A virus A/Singapore/INFIMH-16-0019/2016 IVR-186 (H3N2) antigen (formaldehyde inactivated) - similarity: -0.23117\n",
      "Document Vitamin B6 Complexed with 2-Amino-Hexanoic Acid - similarity: -0.23265\n",
      "Document beta-Mannobiose - similarity: -0.23348\n",
      "Document 2-(Beta-D-Glucopyranosyl)-5-Methyl-1,3,4-Benzothiazole - similarity: -0.23352\n",
      "Document 2-({[4-bromo-3-(diethylsulfamoyl)phenyl]carbonyl}amino)benzoic acid - similarity: -0.23389\n",
      "Document alpha-D-arabinofuranose - similarity: -0.23407\n",
      "Document 6-hydroxy-FAD - similarity: -0.23423\n",
      "Document Aprobarbital - similarity: -0.23441\n",
      "Document (5R,7R,8S,9S,10R)-7-(hydroxymethyl)-3-phenyl-1,6-dioxa-2-azaspiro[4.5]dec-2-ene-8,9,10-triol - similarity: -0.23456\n",
      "Document Tavilermide - similarity: -0.23512\n",
      "Document S-Acetyl-Cysteine - similarity: -0.23526\n",
      "Document L523S - similarity: -0.23634\n",
      "Document L-778123 - similarity: -0.23750\n",
      "Document Prephenic acid - similarity: -0.23847\n",
      "Document 4-(Aminosulfonyl)-N-[(2,5-Difluorophenyl)Methyl]-Benzamide - similarity: -0.23881\n",
      "Document BMS-394136 - similarity: -0.23911\n",
      "Document Caviar, unspecified - similarity: -0.23930\n",
      "Document Human cord blood hematopoietic progenitor cell - similarity: -0.23998\n",
      "Document Dextromoramide - similarity: -0.24028\n",
      "Document Morpholine-4-Carboxylic Acid [1-(2-Benzylsulfanyl-1-Formyl-Ethylcarbamoyl)-2-Phenyl-Ethyl]-Amide - similarity: -0.24113\n",
      "Document (1S)-menthyl hexyl phosphonate group - similarity: -0.24230\n",
      "Document 2-[1-(4-chlorobenzoyl)-5-methoxy-2-methyl-1H-indol-3-yl]-n-[(1R)-1-(hydroxymethyl)propyl]acetamide - similarity: -0.24248\n",
      "Document (2E)-3-{3-[(5-ETHYL-3-IODO-6-METHYL-2-OXO-1,2-DIHYDROPYRIDIN-4-YL)OXY]PHENYL}ACRYLONITRILE - similarity: -0.24350\n",
      "Document rhGAD65 - similarity: -0.24370\n",
      "Document 2',6'-Dichloro-Biphenyl-2,6-Diol - similarity: -0.24436\n",
      "Document Nerofe - similarity: -0.24448\n",
      "Document 3-[2,6,8-Trioxo-9-[(2R,3R,4R)-2,3,4,5-tetrahydroxypentyl]-3H-purin-7-yl]propyl dihydrogen phosphate - similarity: -0.24472\n",
      "Document GUANOSINE-2',3'-O-METHYLIDENEPHOSPHONATE - similarity: -0.24610\n",
      "Document Farampator - similarity: -0.24616\n",
      "Document 4-(5-BENZENESULFONYLAMINO-1-METHYL-1H-BENZOIMIDAZOL-2-YLMETHYL)-BENZAMIDINE - similarity: -0.24643\n",
      "Document FAV-201 - similarity: -0.24744\n",
      "Document 2-Amino-3-[5-(Amino-Carboxy-Methyl)-2,3-Dihydro-Isoxazol-3-Ylsulfanyl]-Propionic Acid - similarity: -0.24744\n",
      "Document Betameprodine - similarity: -0.24764\n",
      "Document 3-(4-hydroxy-3-imino-6-oxo-cyclohexa-1,4-dienyl)-alanine - similarity: -0.24778\n",
      "Document PCLX-001 - similarity: -0.24850\n",
      "Document AP1081 - similarity: -0.24868\n",
      "Document Ethyl hydroxybenzoate - similarity: -0.24878\n",
      "Document {[2-(1h-1,2,3-Benzotriazol-1-Yl)-2-(3,4-Difluorophenyl)Propane-1,3-Diyl]Bis[4,1-Phenylene(Difluoromethylene)]}Bis(Phosphonic Acid) - similarity: -0.25039\n",
      "Document XV638 - similarity: -0.25040\n",
      "Document Rostaporfin - similarity: -0.25061\n",
      "Document Influenza A virus A/Hong Kong/4801/2014 X-263B (H3N2) antigen (formaldehyde inactivated) - similarity: -0.25106\n",
      "Document Malate Like Intermediate - similarity: -0.25236\n",
      "Document Previtamin D(3) - similarity: -0.25245\n",
      "Document BW-A 58C - similarity: -0.25275\n",
      "Document Prolease-r-hFSH - similarity: -0.25330\n",
      "Document Drotebanol - similarity: -0.25364\n",
      "Document Influenza A virus A/California/7/2009 (H1N1)-like antigen (propiolactone inactivated) - similarity: -0.25404\n",
      "Document 4-(4-FLUOROPHENYL)-1-CYCLOROPROPYLMETHYL-5-(4-PYRIDYL)-IMIDAZOLE - similarity: -0.25555\n",
      "Document N-Hydroxy 1n(4-Methoxyphenyl)Sulfonyl-4-(Z,E-N-Methoxyimino)Pyrrolidine-2r-Carboxamide - similarity: -0.25626\n",
      "Document 2'-O-Acetyl Adenosine-5-Diphosphoribose - similarity: -0.25642\n",
      "Document N-(Sulfanylacetyl)Tyrosylprolylmethioninamide - similarity: -0.25661\n",
      "Document 2-((3',5'-DIMETHOXY-4'-HYDROXYPHENYL)AZO)BENZOIC ACID - similarity: -0.25672\n",
      "Document N-acetyl-D-glucosamine-6-phosphate - similarity: -0.25688\n",
      "Document 7-(1,1-Dioxo-1h-Benzo[D]Isothiazol-3-Yloxymethyl)-2-(Oxalyl-Amino)-4,7-Dihydro-5h-Thieno[2,3-C]Pyran-3-Carboxylic Acid - similarity: -0.25760\n",
      "Document Chromic citrate - similarity: -0.25771\n",
      "Document Thiohexam - similarity: -0.25789\n",
      "Document N-Cyclohexyl-N'-Decylurea - similarity: -0.25791\n",
      "Document 4,6-dideoxy-4-amino-alpha-D-glucose - similarity: -0.25793\n",
      "Document (6-METHYL-3,4-DIHYDRO-2H-CHROMEN-2-YL)METHYLPHOSPHINATE - similarity: -0.25864\n",
      "Document Talarozole - similarity: -0.25906\n",
      "Document PYRIMIDINE-4,6-DICARBOXYLIC ACID BIS-(3-METHYL-BENZYLAMIDE) - similarity: -0.25921\n",
      "Document Aceprometazine - similarity: -0.25926\n",
      "Document 6-(Hydroxyethyldithio)-8-(Aminomethylthio)Octanoic Acid - similarity: -0.25952\n",
      "Document Influenza A virus A/Singapore/INFIMH-16-0019/2016 IVR-186 (H3N2) antigen (UV, formaldehyde inactivated) - similarity: -0.25994\n",
      "Document 3-(6-CYCLOHEXYLMETHOXY-9H-PURIN-2-YLAMINO)-BENZENESULFONAMIDE - similarity: -0.26000\n",
      "Document 4-{(Z)-[2-[3-(Methylsulfanyl)Propanoyl]-5-Oxo-1-(2-Oxoethyl)-1,5-Dihydro-4h-Imidazol-4-Ylidene]Methyl}Benzenolate - similarity: -0.26055\n",
      "Document 3-deoxy-alpha-D-manno-oct-2-ulopyranosonic acid - similarity: -0.26146\n",
      "Document (5S)-5-(2-amino-2-oxoethyl)-4-oxo-N-[(3-oxo-3,4-dihydro-2H-1,4-benzoxazin-6-yl)methyl]-3,4,5,6,7,8-hexahydro[1]benzothieno[2,3-d]pyrimidine-2-carboxamide - similarity: -0.26258\n",
      "Document Bevonium - similarity: -0.26276\n",
      "Document 5-Methoxybenzimidazole - similarity: -0.26290\n",
      "Document (3-ENDO)-8-METHYL-8-AZABICYCLO[3.2.1]OCT-3-YL 1H-PYRROLO[2,3-B]PYRIDINE-3-CARBOXYLATE - similarity: -0.26327\n",
      "Document (2s,3s)-Trans-2,3-Dihydro-3-Hydroxyanthranilic Acid - similarity: -0.26344\n",
      "Document (3R)-8-(dioxidosulfanyl)-3-methyl-1,2,3,4-tetrahydroquinoline - similarity: -0.26387\n",
      "Document AV608 - similarity: -0.26389\n",
      "Document 3-Mercapto-1-(1,3,4,9-Tetrahydro-B-Carbolin-2-Yl)-Propan-1-One - similarity: -0.26404\n",
      "Document Riboflavin tetrabutyrate - similarity: -0.26439\n",
      "Document BMS-188797 - similarity: -0.26486\n",
      "Document L-methionine (S)-S-oxide - similarity: -0.26577\n",
      "Document 4-bromo-3-hydroxy-3-methyl butyl diphosphate - similarity: -0.26578\n",
      "Document Tenitramine - similarity: -0.26582\n",
      "Document 2-Phenyl-1-[4-(2-Piperidin-1-Yl-Ethoxy)-Phenyl]-1,2,3,4-Tetrahydro-Isoquinolin-6-Ol - similarity: -0.26598\n",
      "Document 2-(3,5-DIMETHYLPHENYL)-1,3-BENZOXAZOLE - similarity: -0.26618\n",
      "Document Zipeprol - similarity: -0.26624\n",
      "Document Biliverdine IX Alpha - similarity: -0.26651\n",
      "Document R-azabisabolene - similarity: -0.26703\n",
      "Document 4-(Aminosulfonyl)-N-[(4-Fluorophenyl)Methyl]-Benzamide - similarity: -0.26736\n",
      "Document Tosyl-D-Proline - similarity: -0.26796\n",
      "Document Alpha-D-Glucose 1,6-Bisphosphate - similarity: -0.26849\n",
      "Document Tenofovir exalidex - similarity: -0.26870\n",
      "Document CRA-9334 - similarity: -0.26871\n",
      "Document Anhydrovitamin A - similarity: -0.26881\n",
      "Document N-(6,7,9,10,17,18,20,21-octahydrodibenzo[b,k][1,4,7,10,13,16]hexaoxacyclooctadecin-2-yl)acetamide - similarity: -0.26889\n",
      "Document 5-(Aminomethyl)-6-(2,4-Dichlorophenyl)-2-(3,5-Dimethoxyphenyl)Pyrimidin-4-Amine - similarity: -0.26920\n",
      "Document CB-103 - similarity: -0.26922\n",
      "Document Arabinouridine 3'-phosphate - similarity: -0.26953\n",
      "Document 2,3-Di-O-Sulfo-Alpha-D-Glucopyranose - similarity: -0.26975\n",
      "Document Vitamin K7 - similarity: -0.26976\n",
      "Document HGTV-43 - similarity: -0.26998\n",
      "Document TU-514 - similarity: -0.27078\n",
      "Document GTI-2501 - similarity: -0.27122\n",
      "Document Fluoresceinylthioureido - similarity: -0.27164\n",
      "Document Dihydrotachysterol - similarity: -0.27166\n",
      "Document Enprostil - similarity: -0.27186\n",
      "Document 16-Bromoepiandrosterone - similarity: -0.27205\n",
      "Document (2S)-2-(3-Carbamimidamidophenyl)-3-[hydroxy(3-phenylpropyl)phosphoryl]propanoic acid - similarity: -0.27247\n",
      "Document N-(4-OXO-5,6,7,8-TETRAHYDRO-4H-[1,3]THIAZOLO[5,4-C]AZEPIN-2-YL)ACETAMIDE - similarity: -0.27256\n",
      "Document TACRINE(8)-4-AMINOQUINOLINE - similarity: -0.27279\n",
      "Document Heptyl glucoside - similarity: -0.27314\n",
      "Document Sequifenadine - similarity: -0.27320\n",
      "Document 4-(4-methoxy-1H-pyrrolo[2,3-b]pyridin-3-yl)pyrimidin-2-amine - similarity: -0.27379\n",
      "Document [(2R,3R,4S,5S)-3,4-Dihydroxy-5-(4-oxo-4,5-dihydro-1H-pyrrolo[3,2-d]pyrimidin-7-yl)-2-pyrrolidinyl]methyl dihydrogen phosphate - similarity: -0.27434\n",
      "Document 2,5-Dimethoxy-4-(n)-propylthiophenethylamine - similarity: -0.27454\n",
      "Document [(Z)-octadec-9-enyl] (2R)-2,3-bis(oxidanyl)propanoate - similarity: -0.27457\n",
      "Document Methyl N-{[(1R)-1-({1-[(benzyloxy)carbonyl]-L-prolyl-6-ammonio-L-norleucyl}amino)-2-phenylethyl](hydroxy)phosphoryl}-L-alanyl-L-prolinate - similarity: -0.27469\n",
      "Document NRX-1074 - similarity: -0.27489\n",
      "Document Protoporphyrin Ix Containing Co - similarity: -0.27517\n",
      "Document Zirconium chloride hydroxide - similarity: -0.27556\n",
      "Document ISO24 - similarity: -0.27602\n",
      "Document Liafensine - similarity: -0.27625\n",
      "Document 2-((9H-PURIN-6-YLTHIO)METHYL)-5-CHLORO-3-(2-METHOXYPHENYL)QUINAZOLIN-4(3H)-ONE - similarity: -0.27625\n",
      "Document Hydrocortisone cypionate - similarity: -0.27655\n",
      "Document CRA_10762 - similarity: -0.27797\n",
      "Document 4-[(3R)-3-Amino-4-(2,4,5-trifluorophenyl)butanoyl]-3-(2,2,2-trifluoroethyl)-1,4-diazepan-2-one - similarity: -0.27917\n",
      "Document 6-Chloro-2-fluoropurine - similarity: -0.27984\n",
      "Document (1R,2S)-2-Phenylcyclopropanaminium - similarity: -0.27990\n",
      "Document Phenindamine - similarity: -0.27992\n",
      "Document Influenza A virus A/Kansas/14/2017 (H3N2) live (attenuated) antigen - similarity: -0.28031\n",
      "Document 4-[(1S,2S,5S)-5-(HYDROXYMETHYL)-8-METHYL-3-OXABICYCLO[3.3.1]NON-7-EN-2-YL]PHENOL - similarity: -0.28041\n",
      "Document EVT 201 - similarity: -0.28067\n",
      "Document Antimonous acid - similarity: -0.28119\n",
      "Document Measles virus strain enders' attenuated edmonston live antigen - similarity: -0.28154\n",
      "Document N-[2-(2,4-diaminopyrido[2,3-d]pyrimidin-7-yl)-2-methylpropyl]-4-phenoxybenzamide - similarity: -0.28247\n",
      "Document Rubitecan - similarity: -0.28267\n",
      "Document Thiopyrophosphate - similarity: -0.28273\n",
      "Document Alclometasone - similarity: -0.28319\n",
      "Document 1-[(4S)-4-amino-5-(1,3-benzothiazol-2-yl)-5-oxopentyl]guanidine - similarity: -0.28332\n",
      "Document HM-30181 - similarity: -0.28336\n",
      "Document Codeine methylbromide - similarity: -0.28473\n",
      "Document 6-CHLORO-4-(CYCLOHEXYLSULFANYL)-3-PROPYLQUINOLIN-2(1H)-ONE - similarity: -0.28490\n",
      "Document Novo Nordisk a/S Compound - similarity: -0.28552\n",
      "Document 2-O-methyl-alpha-L-fucopyranose - similarity: -0.28574\n",
      "Document Populus deltoides subsp. monilifera pollen - similarity: -0.28668\n",
      "Document Alpha-D-Galactose-1-Phosphate - similarity: -0.28736\n",
      "Document 3-(4-Fluorophenyl)-1-Hydroxy-2-(Pyridin-4-Yl)-1h-Pyrrolo[3,2-B]Pyridine - similarity: -0.28746\n",
      "Document S-(2-Acetamidoethyl) hexadecanethioate - similarity: -0.28859\n",
      "Document Piretanide - similarity: -0.28877\n",
      "Document D-Erythro-2,3-diaminobutyric acid - similarity: -0.28981\n",
      "Document Aspoxicillin - similarity: -0.28992\n",
      "Document Etamivan - similarity: -0.29089\n",
      "Document Pipequaline - similarity: -0.29106\n",
      "Document (R)-3-BROMO-2-HYDROXY-2-METHYL-N-[4-NITRO-3-(TRIFLUOROMETHYL)PHENYL]PROPANAMIDE - similarity: -0.29109\n",
      "Document Thioproperazine - similarity: -0.29133\n",
      "Document Magnolia kobus bark - similarity: -0.29241\n",
      "Document AZD-7325 - similarity: -0.29375\n",
      "Document Propyl Trihydrogen Diphosphate - similarity: -0.29406\n",
      "Document MRK-409 - similarity: -0.29534\n",
      "Document Methylethylamine - similarity: -0.29591\n",
      "Document Indium In-111 oxyquinoline - similarity: -0.29603\n",
      "Document Amaranthus hybridus pollen - similarity: -0.29623\n",
      "Document Candoxatril - similarity: -0.29633\n",
      "Document Aminodi(Ethyloxy)Ethylaminocarbonylbenzenesulfonamide - similarity: -0.29637\n",
      "Document Talampicillin - similarity: -0.29644\n",
      "Document Febarbamate - similarity: -0.29762\n",
      "Document (6R)-2-amino-6-[2-(3'-methoxybiphenyl-3-yl)ethyl]-3,6-dimethyl-5,6-dihydropyrimidin-4(3H)-one - similarity: -0.29762\n",
      "Document (2R,6S)-6-{[methyl(3,4,5-trimethoxyphenyl)amino]methyl}-1,2,5,6,7,8-hexahydroquinazoline-2,4-diamine - similarity: -0.29794\n",
      "Document Streptococcus pneumoniae type 22f capsular polysaccharide antigen - similarity: -0.29813\n",
      "Document Guanosine 3'-monophosphate - similarity: -0.29838\n",
      "Document [Methylseleno]Acetate - similarity: -0.29941\n",
      "Document 2-Amino-Adenosine - similarity: -0.29971\n",
      "Document Uridylyl-2'-5'-phospho-guanosine - similarity: -0.29985\n",
      "Document N-Carbamoyl-Alanine - similarity: -0.30053\n",
      "Document GW-468816 - similarity: -0.30159\n",
      "Document N-[4-(2,4-DIMETHYL-THIAZOL-5-YL)-PYRIMIDIN-2-YL]-N',N'-DIMETHYL-BENZENE-1,4-DIAMINE - similarity: -0.30165\n",
      "Document Betaprodine - similarity: -0.30168\n",
      "Document 9,10-Deepithio-9,10-Didehydroacanthifolicin - similarity: -0.30196\n",
      "Document L-Alfa-Lysophosphatidylcholine, Lauroyl - similarity: -0.30228\n",
      "Document 1-[3,3-Dimethyl-2-(2-Methylamino-Propionylamino)-Butyryl]-Pyrrolidine-2-Carboxylic Acid(1,2,3,4-Tetrahydro-Naphthalen-1-Yl)-Amide - similarity: -0.30243\n",
      "Document 1alpha,24S-Dihydroxyvitamin D2 - similarity: -0.30271\n",
      "Document 3-(Oxalyl-Amino)-Naphthalene-2-Carboxylic Acid - similarity: -0.30327\n",
      "Document Bacampicillin - similarity: -0.30347\n",
      "Document (S)-alpha-methyl-4-carboxyphenylglycine - similarity: -0.30426\n",
      "Document (9S,10S)-9-(S-glutathionyl)-10-hydroxy-9,10-dihydrophenanthrene - similarity: -0.30446\n",
      "Document Zanthoxylum clava-herculis whole - similarity: -0.30454\n",
      "Document SUCCINIC ACID MONO-(13-METHYL-3-OXO-2,3,6,7,8,9,10,11,12,13,14,15,16,17-TETRADECAHYDRO-1H-CYCLOPENTA[A]PHENANTHREN-17-YL) ESTER - similarity: -0.30476\n",
      "Document 5'-O-(L-Threonylsulfamoyl)adenosine - similarity: -0.30544\n",
      "Document [4-(4-ACETYLAMINO-PHENYL)-3,5-DIOXO-4-AZA-TRICYCLO[5.2.2.0 2,6]UNDEC-1-YLCARBAMOYLOXY]-ACETIC ACID - similarity: -0.30574\n",
      "Document 4-AMINO-2-HEXYLOXY-6-HYDROXYMETHYL-TETRAHYDRO-PYRAN-3,5-DIOL - similarity: -0.30589\n",
      "Document 3-Amino-4-Oxybenzyl-2-Butanone - similarity: -0.30660\n",
      "Document Milademetan - similarity: -0.30682\n",
      "Document AZD-5991 - similarity: -0.30742\n",
      "Document AKI-001 - similarity: -0.30767\n",
      "Document AZD-8165 - similarity: -0.30782\n",
      "Document 5-Methoxy-1,2-Dimethyl-3-(Phenoxymethyl)Indole-4,7-Dione - similarity: -0.30787\n",
      "Document Dioxyselenocysteine - similarity: -0.30843\n",
      "Document Aetiocholanolone - similarity: -0.30985\n",
      "Document Butyrylthiocholine - similarity: -0.31058\n",
      "Document Ro 12-7310 - similarity: -0.31099\n",
      "Document Ox bile extract - similarity: -0.31128\n",
      "Document 3-Nitro-4-(2-Oxo-Pyrrolidin-1-Yl)-Benzenesulfonamide - similarity: -0.31165\n",
      "Document Cytidine 5'-diphosphoglycerol - similarity: -0.31182\n",
      "Document N-[2-(2-methyl-1H-indol-3-yl)ethyl]thiophene-2-carboxamide - similarity: -0.31330\n",
      "Document 4-bromo-6-(6-hydroxy-1,2-benzisoxazol-3-yl)benzene-1,3-diol - similarity: -0.31435\n",
      "Document 3-({1-[3-CARBAMIMIDOYL-1-(4-CARBAMIMIDOYL-BENZYLCARBAMOYL)-PROPYLCARBAMOYL]-2-METHYL-BUTYLSULFAMOYL}-METHYL)-BENZOIC ACID - similarity: -0.31437\n",
      "Document 3,7,11,15-tetramethyl-hexadecan-1-ol - similarity: -0.31539\n",
      "Document Alpha-Ribazole-5'-Phosphate Derivative - similarity: -0.31580\n",
      "Document S-(2-oxo)pentadecylcoa - similarity: -0.31604\n",
      "Document PIPERIDINE-2-CARBOXYLIC ACID TERT-BUTYLAMIDE - similarity: -0.31668\n",
      "Document N,N-DIETHYL-2-[(2-THIENYLCARBONYL)AMINO]-4,5,6,7-TETRAHYDRO-1-BENZOTHIOPHENE-3-CARBOXAMIDE - similarity: -0.31668\n",
      "Document Glutathione S-atrazine - similarity: -0.31750\n",
      "Document 1-[2-(4-ETHOXY-3-FLUOROPYRIDIN-2-YL)ETHYL]-3-(5-METHYLPYRIDIN-2-YL)THIOUREA - similarity: -0.31828\n",
      "Document 4-Carbamoyl-1-Beta-D-Ribofuranosyl-Imidazolium-5-Olate-5'-Phosphate - similarity: -0.31859\n",
      "Document Declopramide - similarity: -0.31922\n",
      "Document 3H-pyrazolo[4,3-d]pyrimidin-7-ol - similarity: -0.31929\n",
      "Document 1-(4-fluorophenyl)-N-[3-fluoro-4-(1H-pyrrolo[2,3-b]pyridin-4-yloxy)phenyl]-2-oxo-1,2-dihydropyridine-3-carboxamide - similarity: -0.31947\n",
      "Document Ncs-Chromophore - similarity: -0.32007\n",
      "Document N-[4-hydroxymethyl-cyclohexan-6-yl-1,2,3-triol]-4,6-dideoxy-4-aminoglucopyranoside - similarity: -0.32015\n",
      "Document (5S,8R,9S,10S,13R,14S,17S)-13-{2-[(3,5-DIFLUOROBENZYL)OXY]ETHYL}-17-HYDROXY-10-METHYLHEXADECAHYDRO-3H-CYCLOPENTA[A]PHENANTHREN-3-ONE - similarity: -0.32020\n",
      "Document N-[4-([(2-Amino-4-oxo-1,4-dihydropyrido[3,2-d]pyrimidin-6-yl)methyl]{(2E)-3-[4-carbamoyl-1-(5-O-phosphono-beta-D-ribofuranosyl)-1H-imidazol-5-yl]-2-propenoyl}amino)benzoyl]-L-glutamic acid - similarity: -0.32035\n",
      "Document Distearyldimonium - similarity: -0.32063\n",
      "Document L-N(omega)-nitroarginine-(4R)-amino-L-proline amide - similarity: -0.32183\n",
      "Document (Diaminomethyl-Methyl-Amino)-Acetic Acid - similarity: -0.32199\n",
      "Document 2-Amino-4h-1,3-Benzoxathiin-4-Ol - similarity: -0.32239\n",
      "Document 6-(3-METHYL-1,4-DIOXO-1,4-DIHYDRONAPHTHALEN-2-YL)HEXANOIC ACID - similarity: -0.32345\n",
      "Document 7-alpha-D-Ribofuranosyl-2-aminopurine-5'-phosphate - similarity: -0.32361\n",
      "Document 2-Amino-3-(Diethoxy-Phosphoryloxy)-Propionic Acid - similarity: -0.32411\n",
      "Document beta-D-galactose 6-phosphate - similarity: -0.32412\n",
      "Document (S)-2-{Methyl-[2-(Naphthalene-2-Sulfonylamino)-5-(Naphthalene-2-Sulfonyloxy)-Benzoyl]-Amino}-Succinicacid - similarity: -0.32442\n",
      "Document N-{[(2S,3S)-3-(ETHOXYCARBONYL)OXIRAN-2-YL]CARBONYL}-L-ISOLEUCYL-L-ALANINE - similarity: -0.32565\n",
      "Document 5'-Fluoro-5'-Deoxyadenosine - similarity: -0.32605\n",
      "Document N-Cyclohexyl-N'-phenyl-1,4-phenylenediamine - similarity: -0.32647\n",
      "Document O-(((1R)-((N-PHENYLMETHOXYCARBONYL-L-ALANYL)AMINO)ETHYL)HYDROXYPHOSPHONO)-L-BENZYLACETIC ACID - similarity: -0.32711\n",
      "Document Prinaberel - similarity: -0.32785\n",
      "Document 4-Phenylfentanyl - similarity: -0.32908\n",
      "Document N-Benzyl-3-(alpha-D-galactopyranosyloxy)benzamide - similarity: -0.32915\n",
      "Document 5-amino-2-methyl-N-[(1R)-1-naphthalen-1-ylethyl]benzamide - similarity: -0.32920\n",
      "Document Pinus strobus pollen - similarity: -0.32922\n",
      "Document Levopropoxyphene - similarity: -0.32965\n",
      "Document [1-(3-CHLORO-2-FORMYL-PHENYLCARBAMOYL)-2-METHYL-PROPYL]-CARBAMIC ACID TERT-BUTYL ESTER - similarity: -0.32985\n",
      "Document Remikiren - similarity: -0.32993\n",
      "Document 1-{2-[(4-CHLOROPHENYL)AMINO]-2-OXOETHYL}-N-(1-ISOPROPYLPIPERIDIN-4-YL)-1H-INDOLE-2-CARBOXAMIDE - similarity: -0.33049\n",
      "Document 2-Prolyl-5-Tert-Butyl-[1,3,4]Oxadiazole - similarity: -0.33170\n",
      "Document 6,8-DIMERCAPTO-OCTANOIC ACID AMIDE - similarity: -0.33305\n",
      "Document 4-(Carboxyvin-2-Yl)Phenylboronic Acid - similarity: -0.33308\n",
      "Document Modified Acarbose Hexasaccharide - similarity: -0.33344\n",
      "Document (S)-2-((S)-3-isobutyl-2,5-dioxo-4-quinolin-3-ylmethyl-[1,4]diazepan-1yl)-N-methyl-3-naphtalen-2-yl-propionamide - similarity: -0.33345\n",
      "Document SJ-733 - similarity: -0.33389\n",
      "Document 2,3-Dihydroxy-5-Oxo-Hexanedioate - similarity: -0.33404\n",
      "Document NNZ-2591 - similarity: -0.33415\n",
      "Document XL-888 - similarity: -0.33452\n",
      "Document CRA_16847 - similarity: -0.33458\n",
      "Document 2-(cyclohexylamino)benzoic acid - similarity: -0.33486\n",
      "Document Diclofenamide - similarity: -0.33522\n",
      "Document Influenza B virus B/Wisconsin/1/2010) antigen (MDCK cell derived, propiolactone inactivated) - similarity: -0.33574\n",
      "Document 5'-Guanylylmethylenebisphosphonate - similarity: -0.33584\n",
      "Document 1-Benzyl-3-(4-Methoxy-Benzenesulfonyl)-6-Oxo-Hexahydro-Pyrimidine-4-Carboxylic Acid Hydroxyamide - similarity: -0.33601\n",
      "Document NP-50301 - similarity: -0.33608\n",
      "Document 5-(2-chlorophenyl)-1,3,4-thiadiazole-2-sulfonamide - similarity: -0.33660\n",
      "Document 2-[(2',3',4'-TRIFLUOROBIPHENYL-2-YL)OXY]ETHANOL - similarity: -0.33705\n",
      "Document 5-(3-{3-[3-HYDROXY-2-(METHOXYCARBONYL)PHENOXY]PROPENYL}PHENYL)-4-(HYDROXYMETHYL)ISOXAZOLE-3-CARBOXYLIC ACID - similarity: -0.33789\n",
      "Document (2S,3S)-3-AMINO-4-[(3S)-3-FLUOROPYRROLIDIN-1-YL]-N,N-DIMETHYL-4-OXO-2-(TRANS-4-[1,2,4]TRIAZOLO[1,5-A]PYRIDIN-5-YLCYCLOHEXYL)BUTANAMIDE - similarity: -0.33999\n",
      "Document Xenysalate - similarity: -0.34133\n",
      "Document Dyphylline - similarity: -0.34214\n",
      "Document 3-Hydroxy-4-(3,4,5-trihydroxy-tetrahydro-pyran-2-yloxy)-piperidin-2-one - similarity: -0.34346\n",
      "Document Relatlimab - similarity: -0.34381\n",
      "Document VP025 - similarity: -0.34421\n",
      "Document Propiomazine - similarity: -0.34484\n",
      "Document 2,4-Diamino-6-Phenyl-5,6,7,8,-Tetrahydropteridine - similarity: -0.34502\n",
      "Document 3-Phenyl-1,2-Propandiol - similarity: -0.34576\n",
      "Document L-756423 - similarity: -0.34616\n",
      "Document p-Cumylphenol - similarity: -0.34688\n",
      "Document 1,6-Di-O-Phosphono-D-Allitol - similarity: -0.34718\n",
      "Document (2E)-N-hydroxy-3-[1-methyl-4-(phenylacetyl)-1H-pyrrol-2-yl]prop-2-enamide - similarity: -0.34783\n",
      "Document 1-Allyl-3-Butyl-8-(N-Acetyl-4-Aminobenzyl)-Xanthine - similarity: -0.34935\n",
      "Document 5-Exo-Hydroxycamphor - similarity: -0.34968\n",
      "Document Taberminogene vadenovec - similarity: -0.35183\n",
      "Document 6-(2,6-DIMETHOXYPHENYL)PYRIDO[2,3-D]PYRIMIDINE-2,7-DIAMINE - similarity: -0.35259\n",
      "Document 6-FLUORO-2-[2-HYDROXY-3-(2-METHYL-CYCLOHEXYLOXY)-PHENYL]-1H-INDOLE-5-CARBOXAMIDINE - similarity: -0.35279\n",
      "Document LAX-101 - similarity: -0.35357\n",
      "Document (2s)-2-[(2,4-Dichloro-Benzoyl)-(3-Trifluoromethyl-Benzyl)-Amino]-3-Phenyl-Propionic Acid - similarity: -0.35434\n",
      "Document N~3~-[5-(1H-INDOL-6-YL)-2-(PYRIDIN-2-YLMETHOXY)BENZYL]PYRIDINE-2,3-DIAMINE - similarity: -0.35450\n",
      "Document SB-559448 - similarity: -0.35453\n",
      "Document GW-406381 - similarity: -0.35502\n",
      "Document N-Alpha-Acetyl-3,5-Diiodotyrosylglycine - similarity: -0.35513\n",
      "Document 4-[4-AMINO-6-(5-CHLORO-1H-INDOL-4-YLMETHYL)-[1,3,5]TRIAZIN-2-YLAMINO]-BENZONITRILE - similarity: -0.35531\n",
      "Document Open Form of 2'-Deoxy-Ribofuranose-5'-Phosphate - similarity: -0.35689\n",
      "Document L-2-aminopimelic acid - similarity: -0.35702\n",
      "Document Brifentanil - similarity: -0.35775\n",
      "Document Dibrompropamidine - similarity: -0.35811\n",
      "Document 1-Ethyl-Pyrrolidine-2,5-Dione - similarity: -0.35820\n",
      "Document 2-[2-ETHANESULFONYLAMINO-3-(1H-INDOL-3-YL)-PROPIONYLAMINO]-PENTANEDIOIC ACID 5-AMIDE 1-(4-CARBAMIM IDOYL-BENZYLAMIDE) - similarity: -0.35853\n",
      "Document (2S)-2-[3-(AMINOMETHYL)PHENYL]-3-[(R)-HYDROXY{(1R)-2-METHYL-1-[(PHENYLSULFONYL)AMINO]PROPYL}PHOSPHORYL]PROPANOIC ACID - similarity: -0.35866\n",
      "Document Difemerine - similarity: -0.35932\n",
      "Document 7-(1-ETHYL-PROPYL)-7H-PYRROLO-[3,2-F]QUINAZOLINE-1,3-DIAMINE - similarity: -0.35970\n",
      "Document 2,5-DICHLORO-N-[5-METHOXY-7-(6-METHOXYPYRIDIN-3-YL)-1,3-BENZOXAZOL-2-YL]BENZENESULFONAMIDE - similarity: -0.35971\n",
      "Document OC-1012 - similarity: -0.35989\n",
      "Document Tallimustine - similarity: -0.36063\n",
      "Document N,O-Didansyl-L-Tyrosine - similarity: -0.36088\n",
      "Document 4-(4-CHLORO-PHENYL)-1-{3-[2-(4-FLUORO-PHENYL)-[1,3]DITHIOLAN-2-YL]-PROPYL}-PIPERIDIN-4-OL - similarity: -0.36174\n",
      "Document XP12B - similarity: -0.36271\n",
      "Document Bis(Adenosine)-5'-Pentaphosphate - similarity: -0.36317\n",
      "Document [(2r,3s,4r,5r)-5-(6-Amino-9h-Purin-9-Yl)-3,4-Dihydroxytetrahydro-2-Furanyl]Methyl Sulfamate - similarity: -0.36334\n",
      "Document Tyrosinal - similarity: -0.36401\n",
      "Document Safrazine - similarity: -0.36514\n",
      "Document 1-Guanidinium-7-Aminoheptane - similarity: -0.36518\n",
      "Document 7-(2-Amino-2-Phenyl-Acetylamino)-3-Chloro-8-Oxo-1-Aza-Bicyclo[4.2.0]Oct-2-Ene-2-Carboxylic Acid - similarity: -0.36534\n",
      "Document CRA_10433 - similarity: -0.36565\n",
      "Document KH064 - similarity: -0.36652\n",
      "Document (R)-carnitinyl-CoA betaine - similarity: -0.36686\n",
      "Document 4-(1,3-BENZOXAZOL-2-YL)-2,6-DIBROMOPHENOL - similarity: -0.36777\n",
      "Document Nd1-Phosphonohistidine - similarity: -0.36967\n",
      "Document N-(ethoxycarbonyl)-L-leucine - similarity: -0.36985\n",
      "Document CoA-S-acetyl 5-bromotryptamine - similarity: -0.37027\n",
      "Document N-oxo-2-(phenylsulfonylamino)ethanamide - similarity: -0.37149\n",
      "Document 6-(4-{[2-(3-iodobenzyl)-3-oxocyclohex-1-en-1-yl]amino}phenyl)-5-methyl-4,5-dihydropyridazin-3(2H)-one - similarity: -0.37165\n",
      "Document 8-Bromoadenosine-5'-Diphosphate - similarity: -0.37275\n",
      "Document Furethidine - similarity: -0.37316\n",
      "Document 7-Methoxy-8-[1-(Methylsulfonyl)-1h-Pyrazol-4-Yl]Naphthalene-2-Carboximidamide - similarity: -0.37322\n",
      "Document Cadralazine - similarity: -0.37356\n",
      "Document 3,6,9,12,15-Pentaoxaheptadecane - similarity: -0.37419\n",
      "Document LY-2881835 - similarity: -0.37481\n",
      "Document (-)-menthol 1-propylene glycol carbonate - similarity: -0.37524\n",
      "Document (2R)-3-{[(4Z)-5,6-DIPHENYL-6,7-DIHYDRO-4H-PYRROLO[2,3-D]PYRIMIDIN-4-YLIDENE]AMINO}PROPANE-1,2-DIOL - similarity: -0.37690\n",
      "Document Cefmenoxime - similarity: -0.37776\n",
      "Document Alpha-Ribazole-5'-Phosphate - similarity: -0.37860\n",
      "Document 3-(1,10-Phenanthrol-2-Yl)-L-Alanine - similarity: -0.37989\n",
      "Document N-hexadecanoylglycine - similarity: -0.38186\n",
      "Document [1-(4-Fluorobenzyl)Cyclobutyl]Methyl (1s)-1-[Oxo(1h-Pyrazol-5-Ylamino)Acetyl]Pentylcarbamate - similarity: -0.38396\n",
      "Document 2-[5-Methanesulfonylamino-2-(4-Aminophenyl)-6-Oxo-1,6-Dihydro-1-Pyrimidinyl]-N-(3,3,3-Trifluoro-1-Isopropyl-2-Oxopropyl)Acetamide - similarity: -0.38452\n",
      "Document Aluminum cation - similarity: -0.38465\n",
      "Document Thermus thermophilus lysate - similarity: -0.38486\n",
      "Document D-Galctopyranosyl-1-On - similarity: -0.38560\n",
      "Document GW 597599 - similarity: -0.38611\n",
      "Document Dihydrofolic Acid - similarity: -0.38653\n",
      "Document 4-HYDROXY-7-METHOXY-3-(1-PHENYL-PROPYL)-CHROMEN-2-ONE - similarity: -0.38669\n",
      "Document Quercus acutissima whole - similarity: -0.38697\n",
      "Document 3-Amino-1-Chloro-4-Phenyl-Butanol-2-Yl - similarity: -0.38707\n",
      "Document N-1H-imidazol-2-yl-N'-[4-(1H-imidazol-2-ylamino)phenyl]benzene-1,4-diamine - similarity: -0.38836\n",
      "Document AVI-4065 - similarity: -0.38883\n",
      "Document [4-({[5-Benzyloxy-1-(3-Carbamimidoyl-Benzyl)-1h-Indole-2-Carbonyl]-Amino}-Methyl)-Phenyl]-Trimethyl-Ammonium - similarity: -0.38891\n",
      "Document CRA_17312 - similarity: -0.38935\n",
      "Document 4-(METHYLSULFONYL)BENZENECARBOXIMIDAMIDE - similarity: -0.38950\n",
      "Document ALKS 29 - similarity: -0.39092\n",
      "Document Technetium Tc-99m pertechnetate - similarity: -0.39214\n",
      "Document 7,9-Dimethylguanine - similarity: -0.39347\n",
      "Document Influenza A virus A/Christchurch/16/2010 NIB-74 (H1N1) antigen (propiolactone inactivated) - similarity: -0.39429\n",
      "Document Stepronin - similarity: -0.39452\n",
      "Document Thymidine-5'- Diphosphate - similarity: -0.39568\n",
      "Document S-Butyryl-Cystein - similarity: -0.39581\n",
      "Document Zidovudine monophosphate - similarity: -0.39921\n",
      "Document 1-{(1R,2S)-2-HYDROXY-1-[2-(2-NAPHTHYLOXY)ETHYL]PROPYL}-1H-IMIDAZONE-4-CARBOXAMIDE - similarity: -0.39951\n",
      "Document Glycol salicylate - similarity: -0.39986\n",
      "Document Deoxy-2-fluoro-beta-D-cellotrioside - similarity: -0.40016\n",
      "Document 9-Butyl-8-(4-Methoxybenzyl)-9h-Purin-6-Amine - similarity: -0.40059\n",
      "Document 2-[(3,5-Dichloro-4-trioxidanylphenyl)amino]benzoic acid - similarity: -0.40215\n",
      "Document K-252a - similarity: -0.40222\n",
      "Document (2-{[(4-BROMO-2-FLUOROBENZYL)AMINO]CARBONYL}-5-CHLOROPHENOXY)ACETIC ACID - similarity: -0.40282\n",
      "Document (2S,5R,6R)-6-({(6S)-6-[(Ammonioacetyl)amino]-6-carboxylatohexanoyl}amino)-3,3-dimethyl-7-oxo-4-thia-1-azabicyclo[3.2.0]heptane-2-carboxylate - similarity: -0.40326\n",
      "Document Tert-Butyl(1s)-1-Cyclohexyl-2-Oxoethylcarbamate - similarity: -0.40525\n",
      "Document {4-[2-BENZYL-3-METHOXY-2-(METHOXYCARBONYL)-3-OXOPROPYL]PHENYL}SULFAMIC ACID - similarity: -0.40547\n",
      "Document CHROMOPHORE (LYS-TYR-GLY) - similarity: -0.40622\n",
      "Document PNU177836 - similarity: -0.40631\n",
      "Document Dibenzofuran-4,6-Dicarboxylic Acid - similarity: -0.40677\n",
      "Document 2-[{4-[(2S)-2-{[(Allyloxy)carbonyl]amino}-3-({4-[3-hydroxy-2-(methoxycarbonyl)phenoxy]butyl}amino)-3-oxopropyl]phenyl}(carboxycarbonyl)amino]benzoic acid - similarity: -0.40687\n",
      "Document 1-hydroxy-2-isopropylguanidine - similarity: -0.40829\n",
      "Document 2-Nitro-4-cresol - similarity: -0.40847\n",
      "Document Isoaminile - similarity: -0.40869\n",
      "Document 4-[(3-BROMO-4-O-SULFAMOYLBENZYL)(4-CYANOPHENYL)AMINO]-4H-[1,2,4]-TRIAZOLE - similarity: -0.40890\n",
      "Document Nanaomycin D - similarity: -0.40922\n",
      "Document 1,4:3,6-Dianhydro-2-O-(3-carbamimidoylphenyl)-5-O-(4-carbamimidoylphenyl)-D-glucitol - similarity: -0.40943\n",
      "Document Nitroaspirin - similarity: -0.41061\n",
      "Document 2-(2f-Benzothiazolyl)-5-Styryl-3-(4f-Phthalhydrazidyl)Tetrazolium Chloride - similarity: -0.41118\n",
      "Document VK-2809 - similarity: -0.41199\n",
      "Document BOC-GAMMA-D-GLU-L-LYS(CBZ)-D-BOROALA - similarity: -0.41227\n",
      "Document 2-Bromo-6-chloro-purine - similarity: -0.41594\n",
      "Document 2-CHLORO-N-[(1R,2R)-1-HYDROXY-2,3-DIHYDRO-1H-INDEN-2-YL]-6H-THIENO[2,3-B]PYRROLE-5-CARBOXAMIDE - similarity: -0.41687\n",
      "Document Influenza A virus A/California/7/2009(H1N1)-like hemagglutinin antigen (propiolactone inactivated) - similarity: -0.41695\n",
      "Document 3-(5-Amino-3-imino-3H-pyrazol-4-ylazo)-benzoic acid - similarity: -0.41730\n",
      "Document (1R,3S,5S,8R)-8-Hydroxy-2-oxabicyclo[3.3.1]non-6-ene-3,5-dicarboxylic acid - similarity: -0.42067\n",
      "Document (3S)-3-cyclopentyl-6-methyl-7-[(4-methylpiperazin-1-yl)sulfonyl]-3,4-dihydro-2H-1,2,4-benzothiadiazine 1,1-dioxide - similarity: -0.42121\n",
      "Document Raxatrigine - similarity: -0.42395\n",
      "Document alpha-methylthiofentanyl - similarity: -0.42453\n",
      "Document Cefacetrile - similarity: -0.42547\n",
      "Document D-threo-neopterin - similarity: -0.42553\n",
      "Document Gamma-Arsono-Beta, Gamma-Methyleneadenosine-5'-Diphosphate - similarity: -0.42612\n",
      "Document Glisoxepide - similarity: -0.43076\n",
      "Document N-[1-(4-CARBAMIMIDOYL-BENZYLCARBAMOYL)-3-METHYLSULFANYL-PROPYL]-3-HYDROXY-2-PROPOXYAMINO-BUTYRAMID - similarity: -0.43376\n",
      "Document Oxazolam - similarity: -0.43592\n",
      "Document (2S)-8-[(tert-butoxycarbonyl)amino]-2-(1H-indol-3-yl)octanoic acid - similarity: -0.43758\n",
      "Document Etynodiol - similarity: -0.43888\n",
      "Document PCO-371 - similarity: -0.43995\n",
      "Document 9-Methyl Uric Acid - similarity: -0.44045\n",
      "Document (2R)-2-Acetamido-3-(octadecyloxy)propyl 2-(methylsulfanyl)ethyl hydrogen phosphate - similarity: -0.44111\n",
      "Document 3'-O-Acetylthymidine-5'-diphosphate - similarity: -0.44269\n",
      "Document Bordetella pertussis pertactin antigen (formaldehyde inactivated) - similarity: -0.44449\n",
      "Document 2-butyl-1-hydroxyguanidine - similarity: -0.44468\n",
      "Document Teclozan - similarity: -0.44515\n",
      "Document Sodium phosphate, dibasic - similarity: -0.44552\n",
      "Document (S)-N-(4-carbamimidoylbenzyl)-1-(2-(cyclopentylamino)ethanoyl)pyrrolidine-2-carboxamide - similarity: -0.44677\n",
      "Document 8-(Pyrimidin-2-Ylamino)Naphthalene-2-Carboximidamide - similarity: -0.44678\n",
      "Document (5R,6S,7S,8S)-5-hydroxymethyl-6,7,8-trihydroxy-tetrazolo[1,5-A]piperidine - similarity: -0.44779\n",
      "Document 5-(2,3-dichlorophenyl)-N-(pyridin-4-ylmethyl)pyrazolo[1,5-a]pyrimidin-7-amine - similarity: -0.44780\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Document (3R)-3-[3,5-Bis(trifluoromethyl)anilino]-2-cyano-3-sulfanylpropanamide - similarity: -0.44860\n",
      "Document Heparin Disaccharide I-S - similarity: -0.44944\n",
      "Document CGP 4832 - similarity: -0.44974\n",
      "Document 1,3,4,9-Tetrahydro-2-(Hydroxybenzoyl)-9-[(4-Hydroxyphenyl)Methyl]-6-Methoxy-2h-Pyrido[3,4-B]Indole - similarity: -0.44983\n",
      "Document 5-(4'-AMINO-1'-ETHYL-5',8'-DIFLUORO-1'H-SPIRO[PIPERIDINE-4,2'-QUINAZOLINE]-1-YLCARBONYL)PICOLINONITRILE - similarity: -0.45022\n",
      "Document Trestolone acetate - similarity: -0.45203\n",
      "Document 3h-Indole-5,6-Diol - similarity: -0.45475\n",
      "Document Bimoclomol - similarity: -0.45507\n",
      "Document (S)-oxalosuccinic acid - similarity: -0.45621\n",
      "Document Guanosine 5'-diphosphate 2':3'-cyclic monophosphate - similarity: -0.45830\n",
      "Document 2-Hexyloxy-6-Hydroxymethyl-Tetrahydro-Pyran-3,5-Diol - similarity: -0.45834\n",
      "Document Tucotuzumab celmoleukin - similarity: -0.45918\n",
      "Document 1-METHYL-5-(2-PHENOXYMETHYL-PYRROLIDINE-1-SULFONYL)-1H-INDOLE-2,3-DIONE - similarity: -0.45943\n",
      "Document 1-Androstenedione - similarity: -0.45966\n",
      "Document O1-Pentyl-Mannose - similarity: -0.45979\n",
      "Document Pyroxamide - similarity: -0.46058\n",
      "Document Oxidized Acetyl Dithranol - similarity: -0.46337\n",
      "Document Zytron - similarity: -0.46512\n",
      "Document 4-({4-[(4-methoxypyridin-2-yl)amino]piperidin-1-yl}carbonyl)benzonitrile - similarity: -0.46516\n",
      "Document Methicillin Acyl-Serine - similarity: -0.46536\n",
      "Document PDPSC18 - similarity: -0.46552\n",
      "Document WAY-151693 - similarity: -0.46663\n",
      "Document methyl 4-(2,3-dihydroxy-5-methylphenoxy)-2-hydroxy-6-methylbenzoate - similarity: -0.46897\n",
      "Document 3-Amino-N-{4-[2-(2,6-Dimethyl-Phenoxy)-Acetylamino]-3-Hydroxy-1-Isobutyl-5-Phenyl-Pentyl}-Benzamide - similarity: -0.47147\n",
      "Document CTX001 - similarity: -0.47155\n",
      "Document Sodium 1,2-Dipalmitoyl-sn-glycero-3-phospho-(1'-rac-glycerol) - similarity: -0.47175\n",
      "Document N-naphthalen-1-ylmethyl-2'-[3,5-dimethoxybenzamido]-2'-deoxy-adenosine - similarity: -0.47277\n",
      "Document 2-(CYCLOHEXYLMETHYLAMINO)-4-(PHENYLAMINO)PYRAZOLO[1,5-A][1,3,5]TRIAZINE-8-CARBONITRILE - similarity: -0.47385\n",
      "Document Adamantane-1-Carboxylic Acid-5-Dimethylamino-Naphthalene-1-Sulfonylamino-Butyl-Amide - similarity: -0.47520\n",
      "Document CMLVAX100 - similarity: -0.47665\n",
      "Document 3,8-Diamino-6-Phenyl-5-[6-[1-[2-[(1,2,3,4-Tetrahydro-9-Acridinyl)Amino]Ethyl]-1h-1,2,3-Triazol-5-Yl]Hexyl]-Phenanthridinium - similarity: -0.47826\n",
      "Document Cinolazepam - similarity: -0.48206\n",
      "Document CYT006-AngQb - similarity: -0.48207\n",
      "Document N-({4-[(2-aminopyridin-4-yl)oxy]-3-fluorophenyl}carbamoyl)-2-(4-fluorophenyl)acetamide - similarity: -0.48217\n",
      "Document 3-Chloro-9-Ethyl-6,7,8,9,10,11-Hexahydro-7,11-Methanocycloocta[B]Quinolin-12-Amine - similarity: -0.48283\n",
      "Document 2'-O-[1-ethyl-1H-imidazol)] thymidine-5'-monophosphate - similarity: -0.48316\n",
      "Document Ethyl-Carbamic Acid Methyl Ester - similarity: -0.48329\n",
      "Document Bunamiodyl - similarity: -0.48342\n",
      "Document RU82197 - similarity: -0.48545\n",
      "Document Metharbital - similarity: -0.48616\n",
      "Document 1,6-Naphthalenediol - similarity: -0.48691\n",
      "Document Talviraline - similarity: -0.48771\n",
      "Document (2Z)-N-biphenyl-4-yl-2-cyano-3-cyclopropyl-3-hydroxyprop-2-enamide - similarity: -0.48906\n",
      "Document Isocyanomethane - similarity: -0.48914\n",
      "Document 1-[(2,6-difluorophenyl)sulfonyl]-4-(2,3-dihydro-1,4-benzodioxin-6-ylsulfonyl)piperazine - similarity: -0.49011\n",
      "Document S,S'-(1,4-Phenylene-Bis(1,2-Ethanediyl))Bis-Isothiourea - similarity: -0.49019\n",
      "Document BA-2664 - similarity: -0.49028\n",
      "Document 4-(1h-Imidazol-4-Yl)-3-(5-Ethyl-2,4-Dihydroxy-Phenyl)-1h-Pyrazole - similarity: -0.49156\n",
      "Document N-[5-(ETHYLSULFONYL)-2-METHOXYPHENYL]-5-[3-(2-PYRIDINYL)PHENYL]-1,3-OXAZOL-2-AMINE - similarity: -0.49303\n",
      "Document Trifluperidol - similarity: -0.49333\n",
      "Document N-Dodecyl-N,N-Dimethyl-3-Ammonio-1-Propanesulfonate - similarity: -0.49574\n",
      "Document 5-aminonaphthalene-2-sulfonic acid - similarity: -0.49655\n",
      "Document Etryptamine - similarity: -0.49852\n",
      "Document Aspergillus niger var. niger - similarity: -0.50155\n",
      "Document Hydromorphinol - similarity: -0.50182\n",
      "Document Ridogrel - similarity: -0.50244\n",
      "Document 5-(3-Amino-4,4-Dihyroxy-Butylsulfanylmethyl)-Tetrahydro-Furan-2,3,4-Triol - similarity: -0.50278\n",
      "Document EM-1745 - similarity: -0.50639\n",
      "Document 1-Phenylsulfonamide-3-Trifluoromethyl-5-Parabromophenylpyrazole - similarity: -0.50652\n",
      "Document (2r)-2-{[Formyl(Hydroxy)Amino]Methyl}Hexanoic Acid - similarity: -0.50698\n",
      "Document 4-(5-Bromo-2-Oxo-2h-Indol-3-Ylazo)-Benzenesulfonamide - similarity: -0.50843\n",
      "Document N-((1R,2R)-2-(5-CHLORO-1H-INDOLE-2-CARBOXAMIDO)CYCLOHEXYL)-5-METHYL-4,5,6,7-TETRAHYDROTHIAZOLO[5,4-C]PYRIDINE-2-CARBOXAMIDE - similarity: -0.50943\n",
      "Document Adenosine-5'-Propylphosphate - similarity: -0.50979\n",
      "Document N-[(6S)-6-Carboxy-6-(glycylamino)hexanoyl]-D-alanyl-D-alanine - similarity: -0.51108\n",
      "Document LY231514 tetra glu - similarity: -0.51263\n",
      "Document Butanilicaine - similarity: -0.51315\n",
      "Document 9-Butyl-8-(2,5-Dimethoxy-Benzyl)-2-Fluoro-9h-Purin-6-Ylamine - similarity: -0.51329\n",
      "Document 3,4,5-Trimethoxyamphetamine - similarity: -0.51486\n",
      "Document Trifluoroacetonyl coenzyme A - similarity: -0.51595\n",
      "Document OPC-14523 - similarity: -0.51733\n",
      "Document 8-Benzyl-2-Hydroperoxy-2-(4-Hydroxy-Benzyl)-6-(4-Hydroxy-Phenyl)-2h-Imidazo[1,2-a]Pyrazin-3-One - similarity: -0.52134\n",
      "Document 10-formyl-5,8,10-trideazafolic acid - similarity: -0.52157\n",
      "Document Lysophosphotidylserine - similarity: -0.52355\n",
      "Document 2,5-dibenzyloxy-3-hydroxy-hexanedioic acid bis-[(2-hydroxy-indan-1-yl)-amide] - similarity: -0.52521\n",
      "Document 2-(Sec-Butyl)Thiazole - similarity: -0.52673\n",
      "Document Isopropamide - similarity: -0.52974\n",
      "Document Pyruvamide - similarity: -0.52992\n",
      "Document {4-[3-(4-acetyl-3-hydroxy-2-propylphenoxy)propoxy]phenoxy}acetic acid - similarity: -0.53028\n",
      "Document 2',3'-Dehydro-2',3'-Deoxy-Thymidine 5'-Diphosphate - similarity: -0.53057\n",
      "Document MK-0354 - similarity: -0.53159\n",
      "Document (2S)-4-METHYL-2-(3-PHENYLTHIOUREIDO)-N-((3S)-TETRAHYDRO-2-HYDROXY-3-FURANYL)PENTANAMIDE - similarity: -0.53163\n",
      "Document ETHYL HYDROGEN DIETHYLAMIDOPHOSPHATE - similarity: -0.53180\n",
      "Document 4R-Fluoro-N6-ethanimidoyl-L-lysine - similarity: -0.53362\n",
      "Document Epsilon-N-methyllysine - similarity: -0.53496\n",
      "Document D-1,4-dithiothreitol - similarity: -0.53544\n",
      "Document [1-(1-Methyl-4,5-Dioxo-Pent-2-Enylcarbamoyl)-2-Phenyl-Ethyl]-Carbamic Acid Benzyl Ester - similarity: -0.53911\n",
      "Document FR221647 - similarity: -0.54006\n",
      "Document Pseudouridine-5'-Monophosphate - similarity: -0.54229\n",
      "Document Adenosine-5'-(Dithio)Phosphate - similarity: -0.54260\n",
      "Document SO-101 - similarity: -0.54362\n",
      "Document 4-[(4-bromo-2-{[(3R,5S)-3,5-dimethylpiperidin-1-yl]carbonyl}phenyl)amino]-4-oxobutanoic acid - similarity: -0.54385\n",
      "Document (2S)-4-(4-fluorobenzyl)-N-(2-sulfanylethyl)piperazine-2-carboxamide - similarity: -0.54441\n",
      "Document (3R)-3-cyclopentyl-7-[(4-methylpiperazin-1-yl)sulfonyl]-3,4-dihydro-2H-1,2-benzothiazine 1,1-dioxide - similarity: -0.54631\n",
      "Document (3S)-Tetrahydro-3-furanyl {(2S,3S)-4-[(2S,4R)-4-{(1S,2R)-2-[(S)-amino(hydroxy)methoxy]-2,3-dihydro-1H-inden-1-yl}-2-benzyl-3-oxo-2-pyrrolidinyl]-3-hydroxy-1-phenyl-2-butanyl}carbamate - similarity: -0.54640\n",
      "Document Adenosine monotungstate - similarity: -0.54660\n",
      "Document CTS-21166 - similarity: -0.54766\n",
      "Document Broxyquinoline - similarity: -0.54808\n",
      "Document Lividomycin A - similarity: -0.54958\n",
      "Document Bamifylline - similarity: -0.55234\n",
      "Document CRA_1801 - similarity: -0.55286\n",
      "Document Sisunatovir - similarity: -0.55681\n",
      "Document Afelimomab - similarity: -0.55838\n",
      "Document Methyprylon - similarity: -0.56174\n",
      "Document 4-CHLORO-6-(4-PIPERAZIN-1-YL-1H-PYRAZOL-5-YL)BENZENE-1,3-DIOL - similarity: -0.56236\n",
      "Document 3-[(1-Amino-2-Carboxy-Ethyl)-Hydroxy-Phosphinoyl]-2-Methyl-Propionic Acid - similarity: -0.56242\n",
      "Document N-{[2-({[1-(4-CARBOXYBUTANOYL)AMINO]-2-PHENYLETHYL}-HYDROXYPHOSPHINYL)OXY]ACETYL}-2-PHENYLETHYLAMINE - similarity: -0.56330\n",
      "Document Efloxate - similarity: -0.56387\n",
      "Document 3-(4-Amino-1-Tert-Butyl-1h-Pyrazolo[3,4-D]Pyrimidin-3-Yl)Phenol - similarity: -0.56543\n",
      "Document 2-Amino-Vinyl-Phosphate - similarity: -0.56592\n",
      "Document SD146 - similarity: -0.56862\n",
      "Document N-[(Aminooxy)Carbonyl]Aniline - similarity: -0.57031\n",
      "Document 3-deoxy-D-arabino-hexonic acid - similarity: -0.57092\n",
      "Document 6-(2-Oxo-Hexahydro-Thieno[3,4-D]Imidazol-4-Yl)-Hexanoic Acid - similarity: -0.57248\n",
      "Document 2,4-Dinitrophenyl 2-Deoxy-2-Fluoro-Beta-D-Allopyranoside - similarity: -0.57275\n",
      "Document 4-(Hydrogen sulfate)-beta-D-galactopyranose - similarity: -0.57636\n",
      "Document AZD-8330 - similarity: -0.57877\n",
      "Document Aplindore - similarity: -0.57987\n",
      "Document 1,2-Dipalmitoyl-Phosphatidyl-Glycerole - similarity: -0.58030\n",
      "Document 1,2-icosapentoyl-sn-glycero-3-phosphoserine - similarity: -0.58192\n",
      "Document Acylated ceftazidime - similarity: -0.58889\n",
      "Document 9-(5,5-Difluoro-5-Phosphonopentyl)Guanine - similarity: -0.59031\n",
      "Document RU78783 - similarity: -0.59057\n",
      "Document Cyclo-Tetrametavanadate - similarity: -0.59391\n",
      "Document 3-Fluoro-2-Methyl-Aniline - similarity: -0.60075\n",
      "Document HCV-086 - similarity: -0.60145\n",
      "Document Thiarsahydroxy-Cysteine - similarity: -0.60241\n",
      "Document CHGN111 - similarity: -0.60352\n",
      "Document XL999 - similarity: -0.60522\n",
      "Document [1-HYDROXY-2-(1,1':3',1''-TERPHENYL-3-YLOXY)ETHANE-1,1-DIYL]BIS(PHOSPHONIC ACID) - similarity: -0.60604\n",
      "Document 2-[Trans-(4-Aminocyclohexyl)Amino]-6-(Benzyl-Amino)-9-Cyclopentylpurine - similarity: -0.60780\n",
      "Document N-(2-Thienylmethyl)-2,5-Thiophenedisulfonamide - similarity: -0.60888\n",
      "Document CERE-110 - similarity: -0.61111\n",
      "Document Tetanus Immune Globulin - similarity: -0.61367\n",
      "Document Hexadecanesulfonyl fluoride - similarity: -0.61405\n",
      "Document Xylose-derived lactam oxime - similarity: -0.62244\n",
      "Document Norflurane - similarity: -0.62518\n",
      "Document (2R)-1-[4-({6-[(2,6-Difluorophenyl)amino]-4-pyrimidinyl}amino)phenoxy]-3-(dimethylamino)-2-propanol - similarity: -0.62545\n",
      "Document 2-Amino-4-(4-Amino-Cyclohexa-2,5-Dienyl)-Butyric Acid - similarity: -0.62616\n",
      "Document 5-CHLORO-N-((1R,2S)-2-(4-(2-OXOPYRIDIN-1(2H)-YL)BENZAMIDO) CYCLOPENTYL)THIOPHENE-2-CARBOXAMIDE - similarity: -0.63145\n",
      "Document 4-Imino-5-Methidyl-2-Methylpyrimidine - similarity: -0.63203\n",
      "Document NN344 - similarity: -0.63488\n",
      "Document N-Methyldehydrobutyrine - similarity: -0.63621\n",
      "Document 2-CHLORO-N-[(3R)-2-OXO-1,2,3,4-TETRAHYDROQUINOLIN-3-YL]-6H-THIENO[2,3-B]PYRROLE-5-CARBOXAMIDE - similarity: -0.63775\n",
      "Document PYRIMIDINE-4,6-DICARBOXYLIC ACID BIS-(4-FLUORO-3-METHYL-BENZYLAMIDE) - similarity: -0.64375\n",
      "Document 2-Amino-4-methylphenol - similarity: -0.64650\n",
      "Document (2s,5s)-5-Carboxymethylproline - similarity: -0.64757\n",
      "Document Codeine-N-oxide - similarity: -0.65558\n",
      "Document 6-{4-[4-(4-CHLOROPHENYL)PIPERIDIN-4-YL]PHENYL}-9H-PURINE - similarity: -0.65572\n",
      "Document 5-(2-Fluoro-5-{(1E)-3-[3-hydroxy-2-(methoxycarbonyl)phenoxy]-1-propen-1-yl}phenyl)-1,2-oxazole-3-carboxylic acid - similarity: -0.65744\n",
      "Document 2-Hydroxy-5-({1-[(4-Methylphenoxy)Methyl]-3-Oxoprop-1-Enyl}Amino)-L-Tyrosine - similarity: -0.66012\n",
      "Document 1-{(2S,5S)-4-FLUORO-5-[(TRITYLOXY)METHYL]TETRAHYDROFURAN-2-YL}PYRIMIDINE-2,4(1H,3H)-DIONE - similarity: -0.66331\n",
      "Document Influenza A virus A/Kansas/14/2017 X-327 (H3N2) antigen (UV, formaldehyde inactivated) - similarity: -0.66344\n",
      "Document 4-epi-vancosaminyl derivative of vancomycin - similarity: -0.66664\n",
      "Document Hybrid Between B and C Type Hemes (Protoporphyrin Ixcontaining Fe) - similarity: -0.66956\n",
      "Document (2R,3S,5S)-2-(Hydroxymethyl)-5-[(2-sulfoethyl)carbamoyl]-3-pyrrolidiniumyl 2-acetamido-2-deoxy-4-O-sulfo-beta-D-glucopyranoside - similarity: -0.67120\n",
      "Document (4-{(2S)-2-[(tert-butoxycarbonyl)amino]-3-methoxy-3-oxopropyl}phenyl)methaneseleninic acid - similarity: -0.67131\n",
      "Document 3,6,9,12,15-PENTAOXAHEPTADECAN-1-OL - similarity: -0.68659\n",
      "Document 2-methyl-3,5,7,8-tetrahydro-4H-thiopyrano[4,3-d]pyrimidin-4-one - similarity: -0.69819\n",
      "Document GLUTATHIONE SULFINATE - similarity: -0.69961\n",
      "Document N-Allylaniline - similarity: -0.70090\n",
      "Document Diamino-N-[3-(1,3,2-dioxaborolan-2-yloxy)propyl]methaniminium - similarity: -0.71722\n",
      "Document 5,6-dihydroxy-NADP - similarity: -0.72024\n",
      "Document 7-(2,5-dihydropyrrol-1-yl)-6-phenyl-pyrido[6,5-d]pyrimidin-2-amine - similarity: -0.72145\n",
      "Document 2-[(2-NAPHTHYLSULFONYL)AMINO]ETHYL DIHYDROGEN PHOSPHATE - similarity: -0.72229\n",
      "Document 2-{[2-Oxo-2-(1-piperidinyl)ethyl]sulfanyl}-6-(trifluoromethyl)-4(1H)-pyrimidinone - similarity: -0.72413\n",
      "Document Iofetamine I-123 - similarity: -0.73193\n",
      "Document 5'-O-(L-Serylsulfamoyl)adenosine - similarity: -0.74171\n",
      "Document IR501 - similarity: -0.74294\n",
      "Document 3-Allylfentanyl - similarity: -0.75881\n",
      "Document (3r)-1-Acetyl-3-Methylpiperidine - similarity: -0.76109\n",
      "Document AP-22408 - similarity: -0.77022\n",
      "Document KC706 - similarity: -0.77275\n",
      "Document Ethyl-Trimethyl-Silane - similarity: -0.78648\n",
      "Document N-(P-CYANOPHENYL)-N'-DIPHENYLMETHYL-GUANIDINE-ACETIC ACID - similarity: -0.78806\n",
      "Document ATX-201 - similarity: -0.80258\n",
      "Document Cholesteryl Linoleate - similarity: -0.80665\n",
      "Document Mefruside - similarity: -0.81436\n",
      "Document Morella cerifera pollen - similarity: -0.85205\n",
      "Document N-(TRANS-4'-NITRO-4-STILBENYL)-N-METHYL-5-AMINO-PENTANOIC ACID - similarity: -0.92366\n",
      "Document Beta-Amino Isobutyrate - similarity: -0.92743\n",
      "== complete\n"
     ]
    }
   ],
   "source": [
    "lepirudin_abstracts = documents['Lepirudin']\n",
    "\n",
    "doc_vec = model.infer_vector(lepirudin_abstracts.lower().split(\" \"))\n",
    "\n",
    "# word2vec\n",
    "print(\"== word similarity\")\n",
    "print(model.wv.similar_by_vector(doc_vec))\n",
    "\n",
    "# doc2vec\n",
    "print(\"== document similarity\")\n",
    "doc_sim_list = model.docvecs.most_similar(positive=[doc_vec], topn=len(model.docvecs))\n",
    "for doc_id, sim in doc_sim_list:\n",
    "    print(f\"Document {doc_id} - similarity: {sim:.5f}\")\n",
    "\n",
    "print(\"== complete\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-50-e2d2d85d7922>:17: DeprecationWarning: Call to deprecated `similarity` (Method will be removed in 4.0.0, use self.wv.similarity() instead).\n",
      "  score = model.similarity('Lepirudin', 'Ximelagatran')\n"
     ]
    },
    {
     "ename": "KeyError",
     "evalue": "\"word 'Lepirudin' not in vocabulary\"",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-50-e2d2d85d7922>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     15\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     16\u001b[0m \u001b[1;31m# score = model.n_similarity(questions1_split[i],questions2_split[i])\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 17\u001b[1;33m \u001b[0mscore\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0msimilarity\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'Lepirudin'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'Ximelagatran'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     18\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     19\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\utils.py\u001b[0m in \u001b[0;36mnew_func1\u001b[1;34m(*args, **kwargs)\u001b[0m\n\u001b[0;32m   1459\u001b[0m                     \u001b[0mstacklevel\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m2\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1460\u001b[0m                 )\n\u001b[1;32m-> 1461\u001b[1;33m                 \u001b[1;32mreturn\u001b[0m \u001b[0mfunc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m*\u001b[0m\u001b[0margs\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;33m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1462\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1463\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0mnew_func1\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\models\\base_any2vec.py\u001b[0m in \u001b[0;36msimilarity\u001b[1;34m(self, w1, w2)\u001b[0m\n\u001b[0;32m   1436\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1437\u001b[0m         \"\"\"\n\u001b[1;32m-> 1438\u001b[1;33m         \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mwv\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0msimilarity\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mw1\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mw2\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1439\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1440\u001b[0m     \u001b[1;33m@\u001b[0m\u001b[0mdeprecated\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"Method will be removed in 4.0.0, use self.wv.n_similarity() instead\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\models\\keyedvectors.py\u001b[0m in \u001b[0;36msimilarity\u001b[1;34m(self, w1, w2)\u001b[0m\n\u001b[0;32m    972\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    973\u001b[0m         \"\"\"\n\u001b[1;32m--> 974\u001b[1;33m         \u001b[1;32mreturn\u001b[0m \u001b[0mdot\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mmatutils\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0munitvec\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mw1\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmatutils\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0munitvec\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mw2\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    975\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    976\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mn_similarity\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mws1\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mws2\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\models\\keyedvectors.py\u001b[0m in \u001b[0;36m__getitem__\u001b[1;34m(self, entities)\u001b[0m\n\u001b[0;32m    351\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mentities\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mstring_types\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    352\u001b[0m             \u001b[1;31m# allow calls like trained_model['office'], as a shorthand for trained_model[['office']]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 353\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mentities\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    354\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    355\u001b[0m         \u001b[1;32mreturn\u001b[0m \u001b[0mvstack\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mentity\u001b[0m\u001b[1;33m)\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0mentity\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mentities\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\models\\keyedvectors.py\u001b[0m in \u001b[0;36mget_vector\u001b[1;34m(self, word)\u001b[0m\n\u001b[0;32m    469\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    470\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mget_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 471\u001b[1;33m         \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mword_vec\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    472\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    473\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mwords_closer_than\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mw1\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mw2\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\models\\keyedvectors.py\u001b[0m in \u001b[0;36mword_vec\u001b[1;34m(self, word, use_norm)\u001b[0m\n\u001b[0;32m    466\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    467\u001b[0m         \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 468\u001b[1;33m             \u001b[1;32mraise\u001b[0m \u001b[0mKeyError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"word '%s' not in vocabulary\"\u001b[0m \u001b[1;33m%\u001b[0m \u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    469\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    470\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mget_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mKeyError\u001b[0m: \"word 'Lepirudin' not in vocabulary\""
     ]
    }
   ],
   "source": [
    "# trained_doc_vec = model.docvecs[tags[0]]\n",
    "# inferred_doc_vec = model.infer_vector(text)\n",
    "# print(texts_and_tags[0].tags)\n",
    "# print(texts_and_tags[0][0])\n",
    "\n",
    "# len(model.docvecs)\n",
    "# model.docvecs.most_similar('Lepirudin')\n",
    "# model.most_similar('and', topn=10)\n",
    "from scipy import spatial\n",
    "\n",
    "# vec1 = d2v_model.infer_vector(fisrt_text.split())\n",
    "# vec2 = d2v_model.infer_vector(second_text.split())\n",
    "\n",
    "# cos_distance = spatial.distance.cosine(vec1, vec2)\n",
    "\n",
    "# score = model.n_similarity(questions1_split[i],questions2_split[i])\n",
    "score = model.docvecs.similarity('Lepirudin', 'Ximelagatran')\n",
    "\n",
    "\n",
    "# lep_xime_similarity = spatial.distance.cosine(model.docvecs['Lepirudin'], model.docvecs['Ximelagatran'])\n",
    "# lep_nita_similarity = spatial.distance.cosine(model.docvecs['Lepirudin'], model.docvecs['Nitazoxanide'])\n",
    "# print(lep_xime_similarity)\n",
    "# print(lep_nita_similarity)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "P01AX11 — Nitazoxanide\n",
    "B01AE02 — Lepirudin\n",
    "> similar 0.9988\n",
    "\n",
    "B01AE02 — Lepirudin\n",
    "B01AE05 — Ximelagatran\n",
    "\n",
    "유사도 API\n",
    "model.n_similarity()  \n",
    ": 단어 리스트에 있는 모든 단어 벡터의 mean을 계산\n",
    "\n",
    " -> unit-normalzation -> cosine-similary 계산하기    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>drugbank_id</th>\n",
       "      <th>name</th>\n",
       "      <th>type</th>\n",
       "      <th>groups</th>\n",
       "      <th>atc_codes</th>\n",
       "      <th>categories</th>\n",
       "      <th>inchikey</th>\n",
       "      <th>inchi</th>\n",
       "      <th>description</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>DB00001</td>\n",
       "      <td>Lepirudin</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved</td>\n",
       "      <td>B01AE02</td>\n",
       "      <td>Amino Acids, Peptides, and Proteins|Anticoagul...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Lepirudin is identical to natural hirudin exce...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DB00002</td>\n",
       "      <td>Cetuximab</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved</td>\n",
       "      <td>L01XC06</td>\n",
       "      <td>Amino Acids, Peptides, and Proteins|Antibodies...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Cetuximab is an epidermal growth factor recept...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DB00003</td>\n",
       "      <td>Dornase alfa</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved</td>\n",
       "      <td>R05CB13</td>\n",
       "      <td>Amino Acids, Peptides, and Proteins|Cough and ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Dornase alfa is a biosynthetic form of human d...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>DB00004</td>\n",
       "      <td>Denileukin diftitox</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved|investigational</td>\n",
       "      <td>L01XX29</td>\n",
       "      <td>ADP Ribose Transferases|Amino Acids, Peptides,...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A recombinant DNA-derived cytotoxic protein co...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>DB00005</td>\n",
       "      <td>Etanercept</td>\n",
       "      <td>biotech</td>\n",
       "      <td>approved|investigational</td>\n",
       "      <td>L04AB01</td>\n",
       "      <td>Agents reducing cytokine levels|Amino Acids, P...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Dimeric fusion protein consisting of the extra...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  drugbank_id                 name     type                    groups  \\\n",
       "0     DB00001            Lepirudin  biotech                  approved   \n",
       "1     DB00002            Cetuximab  biotech                  approved   \n",
       "2     DB00003         Dornase alfa  biotech                  approved   \n",
       "3     DB00004  Denileukin diftitox  biotech  approved|investigational   \n",
       "4     DB00005           Etanercept  biotech  approved|investigational   \n",
       "\n",
       "  atc_codes                                         categories inchikey inchi  \\\n",
       "0   B01AE02  Amino Acids, Peptides, and Proteins|Anticoagul...      NaN   NaN   \n",
       "1   L01XC06  Amino Acids, Peptides, and Proteins|Antibodies...      NaN   NaN   \n",
       "2   R05CB13  Amino Acids, Peptides, and Proteins|Cough and ...      NaN   NaN   \n",
       "3   L01XX29  ADP Ribose Transferases|Amino Acids, Peptides,...      NaN   NaN   \n",
       "4   L04AB01  Agents reducing cytokine levels|Amino Acids, P...      NaN   NaN   \n",
       "\n",
       "                                         description  \n",
       "0  Lepirudin is identical to natural hirudin exce...  \n",
       "1  Cetuximab is an epidermal growth factor recept...  \n",
       "2  Dornase alfa is a biosynthetic form of human d...  \n",
       "3  A recombinant DNA-derived cytotoxic protein co...  \n",
       "4  Dimeric fusion protein consisting of the extra...  "
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_atc.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "Parameter doc_words of infer_vector() must be a list of strings (not a single string).",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-14-282a1fa2ef63>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      5\u001b[0m     \u001b[0mcnt\u001b[0m \u001b[1;33m+=\u001b[0m \u001b[1;36m1\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      6\u001b[0m     \u001b[0mtrained_doc_vec\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mdocvecs\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mtags\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 7\u001b[1;33m     \u001b[0minferred_doc_vec\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0minfer_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mtext\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      8\u001b[0m     \u001b[0mdoc_vec\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mtags\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;33m[\u001b[0m\u001b[0mtrained_doc_vec\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0minferred_doc_vec\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      9\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\anaconda3\\lib\\site-packages\\gensim\\models\\doc2vec.py\u001b[0m in \u001b[0;36minfer_vector\u001b[1;34m(self, doc_words, alpha, min_alpha, epochs, steps)\u001b[0m\n\u001b[0;32m    660\u001b[0m         \"\"\"\n\u001b[0;32m    661\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mdoc_words\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mstring_types\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 662\u001b[1;33m             \u001b[1;32mraise\u001b[0m \u001b[0mTypeError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"Parameter doc_words of infer_vector() must be a list of strings (not a single string).\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    663\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    664\u001b[0m         \u001b[0malpha\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0malpha\u001b[0m \u001b[1;32mor\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0malpha\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mTypeError\u001b[0m: Parameter doc_words of infer_vector() must be a list of strings (not a single string)."
     ]
    }
   ],
   "source": [
    "# Accesing vector\n",
    "doc_vec = dict()\n",
    "cnt=0\n",
    "for text, tags in texts_and_tags:\n",
    "    cnt += 1\n",
    "    trained_doc_vec = model.docvecs[tags[0]]\n",
    "    inferred_doc_vec = model.infer_vector(text)\n",
    "    doc_vec[tags[0]] = [trained_doc_vec, inferred_doc_vec]\n",
    "    \n",
    "#     print(f\"tags: {tags}, text: {text}\")\n",
    "    print(f\"tags: {tags}\")\n",
    "    print(f\"trained_doc_vec: {trained_doc_vec}\")\n",
    "    print(f\"inferred_doc_vec: {inferred_doc_vec}\")\n",
    "    print(\"--\"*20)\n",
    "    if(cnt == 3):\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 181,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "dor_lepi:  0.5919299721717834\n",
      "dor_xime:  0.5868140459060669\n",
      "lepi_xime:  0.6061707735061646\n"
     ]
    }
   ],
   "source": [
    "# return_docs = model.docvecs.most_similar(positive=[inferred_vector],topn=5)\n",
    "from scipy import spatial\n",
    "\n",
    "dor_lepi = spatial.distance.cosine(doc_vec['Dornasealf'][1], doc_vec['Lepirudin'][1])\n",
    "dor_xime = spatial.distance.cosine(doc_vec['Dornasealf'][1], doc_vec['Ximelagatr'][1])\n",
    "lepi_xime = spatial.distance.cosine(doc_vec['Lepirudin'][1], doc_vec['Ximelagatr'][1])\n",
    "\n",
    "print('dor_lepi: ', dor_lepi)\n",
    "print('dor_xime: ', dor_xime)\n",
    "print('lepi_xime: ', lepi_xime)\n",
    "# for i in enumerate(doc_vec.keys()):\n",
    "#     print(i)\n",
    "#     spatial.distance.cosine(doc_vec[i][1], doc_vec[i+1][1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 171,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "############################################################\n",
      "K-Means Clustering\n",
      "############################################################\n",
      "Cluster 0\n",
      "['[', '``', 'dornase-alfa', 'cystic', 'fibrosis.', 'dornase-alfa', 'current', 'use', 'mucolyt', 'treat', 'pulmonari', 'diseas', '(', 'major', 'caus', 'morbid', 'mortal', ')', 'cystic', 'fibrosis.', 'reduc', 'mucu', 'viscos', 'lung', ',', 'promot', 'improv', 'clearanc', 'secretions.', 'updat', 'previous', 'publish', 'review.', 'determin', 'whether', 'use', 'dornase-alfa', 'cystic', 'fibrosi', 'associ', 'improv', 'mortal', 'morbid', 'compar', 'placebo', 'medic', 'improv', 'airway', 'clearanc', ',', 'identifi', 'advers', 'event', 'associ', 'use.', 'search', 'cochran', 'cystic', 'fibrosi', 'genet', 'disord', 'group', 'trial', 'regist', 'compris', 'refer', 'identifi', 'comprehens', 'electron', 'databas', 'search', ',', 'handsearch', 'relev', 'journal', 'abstract', 'conferences.', 'date', 'recent', 'search', 'group', \"'s\", 'cystic', 'fibrosi', 'regist', ':', '23', 'april', '2018.clinicaltrials.gov', 'intern', 'clinic', 'trial', 'registri', 'platform', 'also', 'search', 'identifi', 'unpublish', 'ongo', 'trials.', 'date', 'recent', 'search', ':', '07', 'june', '2018.', 'randomis', 'quasi-randomis', 'control', 'trial', 'compar', 'dornase-alfa', 'placebo', ',', 'standard', 'therapi', 'medic', 'improv', 'airway', 'clearance.', 'author', 'independ', 'assess', 'trial', 'inclus', 'criteria', ';', 'two', 'author', 'carri', 'analysi', 'methodolog', 'qualiti', 'data', 'extraction.', 'grade', 'use', 'assess', 'level', 'evidence.', 'search', 'identifi', '69', 'trial', ',', '19', '(', '2565', 'particip', ')', 'met', 'inclus', 'criteria.', 'fifteen', 'trial', 'compar', 'dornase-alfa', 'placebo', 'dornase-alfa', '(', '2447', 'particip', ')', ';', 'two', 'compar', 'daili', 'dornas', 'hyperton', 'salin', '(', '32', 'particip', ')', ';', 'one', 'compar', 'daili', 'dornase-alfa', 'hyperton', 'salin', 'altern', 'day', 'dornase-alfa', '(', '48', 'particip', ')', ';', 'one', 'compar', 'dornase-alfa', 'mannitol', 'combin', 'drug', '(', '38', 'particip', ')', '.', 'trial', 'durat', 'vari', 'six', 'day', 'three', 'years.dornase-alfa', 'compar', 'placebo', 'treatmentdornase-alfa', 'improv', 'forc', 'expiratori', 'volum', 'one', 'second', 'one', 'month', '(', 'four', 'trial', ',', '248', 'particip', ')', ',', 'three', 'month', '(', 'one', 'trial', ',', '320', 'particip', ';', 'moderate-qu', 'evid', ')', ',', 'six', 'month', '(', 'one', 'trial', ',', '647', 'particip', ';', 'high-qual', 'evid', ')', 'two', 'year', '(', 'one', 'trial', ',', '410', 'particip', ')', '.', 'limit', 'low-qual', 'evid', 'show', 'differ', 'group', 'chang', 'qualiti', 'life.', 'decreas', 'pulmonari', 'exacerb', 'dornase-alfa', 'trial', 'two', 'year', '(', 'moderate-qu', 'evid', ')', '.', 'one', 'trial', 'examin', 'cost', 'care', ',', 'includ', 'cost', 'dornase-alfa', ',', 'found', 'cost', 'save', 'dornase-alfa', 'offset', '18', '%', '38', '%', 'medic', 'costs.dornase-alfa', ':', 'daili', 'versu', 'altern', 'dayon', 'cross-ov', 'trial', '(', '43', 'children', ')', 'found', 'differ', 'treatment', 'regimen', 'lung', 'function', ',', 'qualiti', 'life', 'pulmonari', 'exacerb', '(', 'low-qual', 'evid', ')', '.dornase-alfa', 'compar', 'medic', 'improv', 'airway', 'clearanceresult', 'comparison', 'mixed.', 'one', 'trial', '(', '43', 'children', ')', 'show', 'greater', 'improv', 'forc', 'expiratori', 'volum', 'one', 'second', 'dornase-alfa', 'compar', 'hyperton', 'salin', '(', 'low-qual', 'evid', ')', ',', 'one', 'trial', '(', '23', 'particip', ')', 'report', 'differ', 'lung', 'function', 'dornase-alfa', 'mannitol', 'dornase-alfa', 'dornase-alfa', 'plu', 'mannitol', '(', 'low-qual', 'evid', ')', '.', 'one', 'trial', '(', '23', 'particip', ')', 'found', 'differ', 'qualiti', 'life', 'favour', 'dornase-alfa', 'compar', 'dornase-alfa', 'plu', 'mannitol', '(', 'low-qual', 'evid', ')', ';', 'comparison', 'found', 'differ', 'outcom', '(', 'low-qual', 'evid', ')', '.', 'trial', 'comparison', 'report', 'differ', 'group', 'number', 'pulmonari', 'exacerb', '(', 'low-qual', 'evid', ')', '.when', 'comparison', 'assess', ',', 'dornase-alfa', 'caus', 'significantli', 'advers', 'effect', 'treatment', ',', 'except', 'voic', 'alter', 'rash.', 'evid', 'show', ',', 'compar', 'placebo', ',', 'therapi', 'dornase-alfa', 'improv', 'lung', 'function', 'peopl', 'cystic', 'fibrosi', 'trial', 'last', 'one', 'month', 'two', 'years.', 'decreas', 'pulmonari', 'exacerb', 'trial', 'six', 'month', 'longer.', 'voic', 'alter', 'rash', 'appear', 'advers', 'event', 'report', 'increas', 'frequenc', 'randomis', 'control', 'trials.', 'enough', 'evid', 'firmli', 'conclud', 'dornase-alfa', 'superior', 'hyperosmolar', 'agent', 'improv', 'lung', 'function.', '``', ',', \"'dornase-alfa\", 'dornase-alfa', 'util', 'treatment', 'manag', 'cystic', 'fibrosi', '(', 'cf', ')', 'conjunct', 'standard', 'therapies.', 'dornase-alfa', 'discov', '1992', 'mucolyt', 'agent', 'treatment', 'hyper', 'viscou', 'mucou', 'produc', 'patient', 'cf', ',', 'notic', '1950', 'use', 'bovin', 'deoxyribonucleas', 'decreas', 'thick', 'sputum', 'cf', 'patients.', '[', '1', ']', \"'\", ',', '``', 'dornase-alfa', '(', 'pulmozym', ')', '.', 'dornase-alfa', '(', 'pulmozym', ')', 'one', 'commonli', 'use', 'medic', 'treat', 'lung', 'diseas', 'cystic', 'fibrosi', '(', 'cf', ')', '.', 'respiratori', 'medic', 'enter', 'clinic', 'market', ',', 'understand', 'proper', 'use', 'indic', 'dornase-alfa', 'increasingli', 'important.', 'addit', ',', 'dornase-alfa', 'use', 'treat', 'medic', 'conditions.', 'review', 'cover', 'recent', 'public', 'expand', 'indications.', 'understand', 'dornase-alfa', \"'s\", 'mechan', 'action', 'impact', 'pathophysiolog', 'cf', 'lead', 'improv', 'understand', 'optim', 'therapi', ',', 'way', 'improv', 'adher', 'use', 'medications.', 'importantli', ',', 'routin', 'use', 'dornase-alfa', 'associ', 'improv', 'lung', 'function', 'surviv', 'patient', 'cf.', 'outsid', 'cf', ',', 'potenti', 'use', 'includ', 'treat', 'patient', 'empyema', 'mechan', 'ventilation.', 'dornase-alfa', 'avail', 'clinic', 'use', 'nearli', '20', 'year', 'one', 'commonli', 'use', 'medic', 'patient', 'cf.', 'routin', 'use', 'associ', 'reduc', 'rate', 'pulmonari', 'deterior', 'improv', 'survival.', 'recent', 'clinic', 'report', 'suggest', 'dornase-alfa', 'may', 'clinic', 'valu', 'medic', 'problem', 'complic', 'pneumonia', 'mechan', 'ventil', 'patient', 'atelectasis.', '``', ',', \"'use\", 'dornase-alfa', 'paediatr', 'intens', 'care', 'unit', ':', 'current', 'literatur', 'nation', 'cross-sect', 'survey.', 'airway', 'mucu', 'obstruct', 'major', 'challeng', 'children', 'admit', 'paediatr', 'intens', 'care', 'unit', '(', 'picu', ')', '.', 'aim', 'evalu', 'evid', 'contemporari', 'use', 'mucolyt', 'medic', 'dornase-alfa', 'non-cyst', 'fibrosi', 'condit', 'picu.', '(', '1', ')', 'perform', 'systemat', 'review', 'search', 'pubm', ',', 'embas', ',', 'cochran', 'library.', 'studi', 'select', ':', 'qualiti', 'assess', 'data', 'synthesi', ',', 'includ', 'randomis', 'control', 'trial', '(', 'rct', ')', 'compar', 'dornase-alfa', 'standard', 'care', 'placebo', 'critically-il', 'paediatr', 'patient', '(', '<', '18', 'year', 'age', ')', 'picu.', 'howev', ',', 'non-randomis', 'control', 'studi', 'case', 'seri', 'also', 'discussed.', 'data', 'extract', ':', 'data', 'extract', 'independ', 'multipl', 'review', 'use', 'data', 'extract', 'forms.', 'primari', 'outcom', 'durat', 'mechan', 'ventilation.', 'data', 'synthesi', ':', 'grade', 'approach', 'use', 'qualiti', 'assessment.', 'meta-analysi', 'could', 'performed.', '(', '2', ')', 'nation', 'cross-sect', 'survey', 'among', 'seven', 'picu', 'netherland', 'also', 'performed.', 'systemat', 'review', 'yield', 'one', 'rct', ',', 'compar', 'dornase-alfa', 'normal', 'salin', 'children', 'cardiac', 'surgery.', 'studi', ',', 'dornase-alfa', 'led', 'reduct', 'durat', 'mechan', 'ventil', 'approxim', '1', 'day', '(', '36', '%', 'reduct', ')', '.', 'addit', ',', 'found', 'nine', 'retrospect', 'observ', 'case', 'studies.', 'survey', 'reveal', 'high', 'current', 'use', 'dornase-alfa', 'dutch', 'picu', ':', '42', '%', 'respond', 'report', 'prescrib', 'dornase-alfa', 'least', 'everi', 'week.', '4', '%', 'respond', 'report', 'access', 'local', 'picu', 'dornase-alfa', 'protocol.', 'off-label', 'use', 'dornase-alfa', 'picu', 'frequent', 'without', 'strong', 'evid', 'local', 'protocol', ',', 'highlight', 'need', 'research', 'effect', 'mucolyt', 'agent.', \"'\", ',', \"'nebul\", 'in-lin', 'endotrach', 'dornase-alfa', 'albuterol', 'administ', 'mechan', 'ventil', 'covid-19', 'patient', ':', 'case', 'series.', 'background', 'mechan', 'ventil', 'patient', 'coronaviru', 'diseas', '2019', '(', 'covid-19', ')', 'mortal', '24-53', '%', ',', 'part', 'due', 'distal', 'mucopurul', 'secret', 'interf', 'ventilation.', 'dornase-alfa', 'recombin', 'human', 'dnase', '1', 'digest', 'dna', 'mucoid', 'sputum.', 'nebul', 'dornase-alfa', 'fda-approv', 'cystic', 'fibrosi', 'treatment.', 'dna', 'neutrophil', 'extracellular', 'trap', '(', 'net', ')', 'contribut', 'viscos', 'mucopurul', 'secretions.', 'net', 'found', 'serum', 'patient', 'sever', 'covid-19', ',', 'target', 'net', 'reduc', 'mortal', 'anim', 'model', 'acut', 'respiratori', 'distress', 'syndrom', '(', 'ard', ')', '.', 'thu', ',', 'dornase-alfa', 'may', 'benefici', 'patient', 'sever', 'covid-19', ',', 'act', 'mucolyt', 'target', 'nets.', 'howev', ',', 'deliveri', 'nebul', 'drug', 'aerosol', 'sars-cov-2', ',', 'caus', 'covid-19', ',', 'increas', 'infect', 'risk', 'staff.', ',', 'report', 'singl', 'center', 'case', 'seri', 'dornase-alfa', 'administ', 'in-lin', 'nebul', 'system', 'minim', 'risk', 'viru', 'aerosolization.', 'method', 'demograph', ',', 'clinic', 'data', ',', 'outcom', 'collect', 'electron', 'medic', 'record', 'five', 'mechan', 'ventil', 'patient', 'covid-19', ',', 'includ', 'three', 'requir', 'veno-ven', 'extracorpor', 'membran', 'oxygen', '(', 'vv-ecmo', ')', ',', 'treat', 'nebul', 'in-lin', 'endotrach', 'dornase-alfa', 'co-administ', 'albuterol', '(', 'use', 'increas', 'deliveri', 'alveoli', ')', ',', 'march', '31', 'april', '24', ',', '2020.', 'data', 'toler', 'respons', ',', 'includ', 'longitudin', 'valu', 'captur', 'respiratori', 'function', 'inflammatori', 'statu', ',', 'analyzed.', 'result', 'follow', 'nebul', 'in-lin', 'administr', 'dornase-alfa', 'albuterol', ',', 'fraction', 'inspir', 'oxygen', 'requir', 'reduc', 'five', 'patients.', 'patient', 'remain', 'aliv', 'two', 'patient', 'discharg', 'intens', 'care', 'unit.', 'drug', 'associ', 'toxic', 'identified.', 'conclus', 'result', 'present', 'case', 'seri', 'suggest', 'dornase-alfa', 'well-toler', 'critic', 'ill', 'patient', 'covid-19.', 'clinic', 'trial', 'requir', 'formal', 'test', 'dose', ',', 'safeti', ',', 'efficaci', 'dornase-alfa', 'covid-19', ',', 'two', 'recent', 'regist', '(', 'nct04359654', 'nct04355364', ')', '.', 'case', 'seri', ',', 'hope', 'contribut', 'develop', 'manag', 'approach', 'critic', 'ill', 'patient', 'covid-19.', \"'\", ',', '``', 'dornase-alfa', 'inform', 'avail', 'clinic', 'use', 'dornas', 'alpha', '(', 'hamster', ')', 'breastfeeding.', 'larg', 'protein', 'molecul', 'molecular', 'weight', '37,000', ',', 'amount', 'milk', 'like', 'low', 'absorpt', 'unlik', 'probabl', 'destroy', 'infant', \"'s\", 'gastrointestin', 'tract.', 'task', 'forc', 'respiratori', 'expert', 'europ', ',', 'australia', 'new', 'zealand', 'found', 'dornase-alfa', 'compat', 'breastfeeding.', '[', '1', ']', '``', ',', \"'consider\", 'dornase-alfa', 'treatment', 'sever', 'covid-19', 'acut', 'respiratori', 'distress', 'syndrome.', 'propos', 'like', 'contribut', 'sever', 'covid-19', 'morbid', 'extracellular', 'dna', 'neutrophil', 'extracellular', 'trap', '(', 'net', ')', '.', 'dornase-alfa', 'degrad', 'extracellular', 'dna', 'reduc', 'mucu', 'rigid', 'accumul', ',', 'associ', 'respiratori', 'improv', 'first', 'patient.', 'dornase-alfa', 'consid', 'clinic', 'trial', 'treatment', 'sever', 'covid-19.', \"'\", ',', '``', 'dornase-alfa', 'cystic', 'fibrosis.', 'dornase-alfa', 'current', 'use', 'mucolyt', 'treat', 'pulmonari', 'diseas', '(', 'major', 'caus', 'morbid', 'mortal', ')', 'cystic', 'fibrosis.', 'reduc', 'mucu', 'viscos', 'lung', ',', 'promot', 'improv', 'clearanc', 'secretions.', 'updat', 'previous', 'publish', 'review.', 'determin', 'whether', 'use', 'dornase-alfa', 'cystic', 'fibrosi', 'associ', 'improv', 'mortal', 'morbid', 'compar', 'placebo', 'medic', 'improv', 'airway', 'clearanc', ',', 'identifi', 'advers', 'event', 'associ', 'use.', 'search', 'cochran', 'cystic', 'fibrosi', 'genet', 'disord', 'group', 'trial', 'regist', 'compris', 'refer', 'identifi', 'comprehens', 'electron', 'databas', 'search', ',', 'handsearch', 'relev', 'journal', 'abstract', 'conferences.', 'date', 'recent', 'search', 'group', \"'s\", 'cystic', 'fibrosi', 'regist', ':', '30', 'novemb', '2015.clinicaltrials.gov', 'also', 'search', 'identifi', 'unpublish', 'ongo', 'trials.', 'date', 'recent', 'search', ':', '28', 'novemb', '2015.', 'randomis', 'quasi-randomis', 'control', 'trial', 'compar', 'dornase-alfa', 'placebo', ',', 'standard', 'therapi', 'medic', 'improv', 'airway', 'clearance.', 'author', 'independ', 'assess', 'trial', 'inclus', 'criteria', ';', 'two', 'author', 'carri', 'analysi', 'methodolog', 'qualiti', 'data', 'extraction.', 'search', 'identifi', '54', 'trial', ',', '19', '(', 'includ', 'total', '2565', 'particip', ')', 'met', 'inclus', 'criteria.', 'three', 'addit', 'paper', 'examin', 'healthcar', 'cost', 'one', 'clinic', 'trials.', 'fifteen', 'trial', 'compar', 'dornase-alfa', 'placebo', 'dornase-alfa', 'treatment', '(', '2447', 'particip', ')', ';', 'two', 'compar', 'daili', 'dornas', 'hyperton', 'salin', '(', '32', 'particip', ')', ';', 'one', 'compar', 'daili', 'dornase-alfa', 'hyperton', 'salin', 'altern', 'day', 'dornase-alfa', '(', '48', 'particip', ')', ';', 'one', 'compar', 'dornase-alfa', 'mannitol', 'combin', 'drug', '(', '38', 'particip', ')', '.', 'trial', 'durat', 'vari', 'six', 'day', 'three', 'years.compar', 'placebo', ',', 'forc', 'expiratori', 'volum', 'one', 'second', 'improv', 'intervent', 'group', ',', 'signific', 'differ', 'one', ',', 'three', ',', 'six', 'month', 'two', 'years.', 'also', 'signific', 'improv', 'lung', 'clearanc', 'index', 'one', 'month.', 'decreas', 'pulmonari', 'exacerb', 'compar', 'placebo', 'trial', 'longer', 'duration.', 'qualiti', 'evid', 'placebo-control', 'trial', 'moder', 'high', 'outcom', 'lung', 'function', 'pulmonari', 'exacerbations.', 'limit', ',', 'low', 'qualiti', 'evid', 'avail', 'chang', 'qualiti', 'life', 'baseline.', 'one', 'trial', 'examin', 'cost', 'care', ',', 'includ', 'cost', 'dornase-alfa', ',', 'found', 'cost', 'save', 'dornase-alfa', 'offset', '18', '%', '38', '%', 'medic', 'costs.th', 'result', 'trial', 'compar', 'dornase-alfa', 'medic', 'improv', 'airway', 'clearanc', '(', 'hyperton', 'salin', 'mannitol', ')', 'mix', ',', 'one', 'trial', 'show', 'greater', 'improv', 'forc', 'expiratori', 'volum', 'one', 'second', 'dornase-alfa', 'compar', 'hyperton', 'salin', ',', 'three', 'trial', 'find', 'differ', 'medications.', 'trial', 'assess', 'combin', 'dornase-alfa', 'anoth', 'medic', 'compar', 'dornas', 'alon', ',', 'benefit', 'seen', 'combin', 'dornase-alfa', 'mannitol.', 'evid', 'dornase-alfa', 'compar', 'medic', 'limit', 'open-label', 'design', 'trial', 'may', 'induc', 'bia', ',', 'therefor', 'qualiti', 'evid', 'judg', 'low.dornase-alfa', 'caus', 'significantli', 'advers', 'effect', ',', 'except', 'voic', 'alter', 'rash.', 'evid', 'show', ',', 'compar', 'placebo', ',', 'therapi', 'dornase-alfa', 'improv', 'lung', 'function', 'peopl', 'cystic', 'fibrosi', 'trial', 'last', 'one', 'month', 'two', 'years.', 'decreas', 'pulmonari', 'exacerb', 'trial', 'six', 'month', 'longer.', 'voic', 'alter', 'rash', 'appear', 'advers', 'event', 'report', 'increas', 'frequenc', 'randomis', 'control', 'trials.', 'enough', 'evid', 'firmli', 'conclud', 'dornase-alfa', 'superior', 'hyperosmolar', 'agent', 'improv', 'lung', 'function.', '``', ',', \"'time\", 'dornase-alfa', 'inhal', 'cystic', 'fibrosis.', 'inhal', 'enzym', 'dornase-alfa', 'reduc', 'sputum', 'viscos', 'improv', 'clinic', 'outcom', 'peopl', 'cystic', 'fibrosis.', 'updat', 'previous', 'publish', 'cochran', 'review.', 'determin', 'effect', 'time', 'dornase-alfa', 'inhal', 'measur', 'clinic', 'efficaci', 'peopl', 'cystic', 'fibrosi', '(', 'relat', 'airway', 'clearanc', 'techniqu', 'time', 'day', ')', '.', 'relev', 'randomis', 'quasi-randomis', 'control', 'trial', 'identifi', 'cochran', 'cystic', 'fibrosi', 'trial', 'regist', ',', 'physiotherapi', 'evid', 'databas', '(', 'pedro', ')', ',', 'intern', 'cystic', 'fibrosi', 'confer', 'proceedings.d', 'recent', 'search', ':', '25', 'april', '2016.', 'trial', 'dornase-alfa', 'peopl', 'cystic', 'fibrosi', 'time', 'inhal', 'randomis', 'element', 'studi', 'either', ':', 'inhal', 'compar', 'airway', 'clearanc', 'techniqu', ';', 'morn', 'compar', 'even', 'inhalation.', 'author', 'independ', 'select', 'trial', ',', 'assess', 'risk', 'bia', 'extract', 'data', 'disagr', 'resolv', 'discussion.', 'relev', 'data', 'extract', ',', 'possibl', ',', 'meta-analysed.', 'identifi', '115', 'trial', 'report', 'repres', '55', 'studi', ',', 'five', 'studi', '(', 'provid', 'data', '122', 'particip', ')', 'met', 'inclus', 'criteria.', 'five', 'studi', 'use', 'cross-ov', 'design.', 'intervent', 'period', 'rang', 'two', 'eight', 'weeks.', 'four', 'trial', 'compar', 'dornase-alfa', 'inhal', 'versu', 'airway', 'clearanc', 'techniques.', 'inhal', 'instead', 'airway', 'clearanc', 'significantli', 'chang', 'forc', 'expiratori', 'volum', 'one', 'second.', 'similarli', ',', 'forc', 'vital', 'capac', 'qualiti', 'life', 'significantli', 'affect', ';', 'forc', 'expiratori', 'flow', '25', '%', 'significantli', 'wors', 'dornase-alfa', 'inhal', 'airway', 'clearanc', ',', 'mean', 'differ', '-0.17', 'litr', '(', '95', '%', 'confid', 'interv', '-0.28', '-0.05', ')', ',', 'base', 'pool', 'data', 'two', 'small', 'studi', 'children', '(', 'seven', '19', 'year', ')', 'well-preserv', 'lung', 'function.', 'secondari', 'outcom', 'statist', 'non-significant.in', 'one', 'trial', ',', 'morn', 'versu', 'even', 'inhal', 'impact', 'lung', 'function', 'symptoms.', 'current', 'evid', 'deriv', 'small', 'number', 'particip', 'indic', 'inhal', 'dornase-alfa', 'airway', 'clearanc', 'techniqu', 'less', 'effect', 'tradit', 'recommend', 'inhal', 'nebulis', 'dornase-alfa', '30', 'minut', 'prior', 'airway', 'clearanc', 'techniqu', ',', 'outcomes.', 'children', 'well-preserv', 'lung', 'function', ',', 'inhal', 'airway', 'clearanc', 'may', 'benefici', 'small', 'airway', 'function', 'inhal', 'after.', 'howev', ',', 'result', 'reli', 'measur', 'high', 'variabl', 'studi', 'variabl', 'follow', 'up.', 'absenc', 'strong', 'evid', 'indic', 'one', 'time', 'regimen', 'better', 'anoth', ',', 'time', 'dornas', 'alpha', 'inhal', 'larg', 'base', 'pragmat', 'reason', 'individu', 'prefer', 'respect', 'time', 'airway', 'clearanc', 'time', 'day.\\\\xa0furth', 'research', 'warranted.', \"'\", ',', '``', '[', 'immunoenzym', 'method', 'determin', 'o-antigen', 'sonn', \"'s\", 'shigella', 'use', 'affin', 'isol', 'antibodi', ']', '.', 'sensit', 'elisa', 'system', 'assay', 'shigella', 'sonnei', 'o-antigen', ',', 'develop', 'basi', 'specif', 'rabbit', 'antibodi', 'isol', 'antisera', 'differ', 'chromatographi', 'techniqu', 'compared.', 'highest', 'specif', 'sensit', 'o-antigen', 'determin', '(', '0.5-1', 'ng/ml', ')', 'achiev', 'use', 'affinity-isol', 'antibodi', 'o-antigen.', 'clinic', 'trial', 'elisa', 'system', 'carri', 'out.', '``', ']']\n",
      "------------------------------------------------------------\n",
      "Cluster 1\n",
      "['[', \"'lepirudin\", 'lepirudin', 'longer', 'market', 'unit', 'states.', 'limit', 'inform', 'indic', 'lepirudin', 'dose', '100', 'mg', 'daili', 'produc', 'low', 'level', 'milk.', 'larg', 'molecular', 'weight', ',', 'would', 'expect', 'absorb', 'breastmilk', 'infant.', 'lepirudin', 'would', 'expect', 'caus', 'advers', 'effect', 'breastf', 'infant', ',', 'especi', 'infant', 'older', '2', 'months.', '[', '1', ']', \"'\", ',', \"'comparison\", 'argatroban', 'lepirudin', 'bivalirudin', 'treatment', 'heparin-induc', 'thrombocytopenia', ':', 'systemat', 'review', 'meta-analysis.', 'prevent', 'thromboembol', 'event', 'associ', 'heparin-induc', 'thrombocytopenia', '(', 'hit', ')', ',', 'patient', 'usual', 'treat', 'argatroban', ',', 'lepirudin', ',', 'bivalirudin.', ',', 'conduct', 'meta-analysi', 'studi', 'compar', 'treatment', 'hit', 'follow', 'direct', 'thrombin', 'inhibitor', ':', 'argatroban', 'versu', 'lepirudin', 'argatroban', 'versu', 'bivalirudin.', 'systemat', 'search', 'pubm', ',', 'embas', ',', 'cochran', 'librari', 'databas', 'relev', 'studies.', 'clinic', 'outcom', 'thromboembol', 'complic', 'bleeding.', 'total', '589', 'articl', 'found', '9', 'final', 'includ', 'meta-analysis.', 'significantli', 'differ', 'thromboembol', 'complic', 'argatroban', 'hirudin', 'analogu', '(', 'lepirudin', 'bivalirudin', ')', 'treatment', 'hit', '(', 'lepirudin', ':', 'rr\\\\xa0=\\\\xa00.773', ',', '95', '%', 'ci\\\\xa0=\\\\xa00.449-1.331', ',', 'p\\\\xa0=\\\\xa00.353', ';', 'bivalirudin', ':', 'rr\\\\xa0=\\\\xa00.768', ',', '95', '%', 'ci\\\\xa0=\\\\xa00.386-1.527', ',', 'p\\\\xa0=\\\\xa00.452', ')', '.', 'moreov', ',', 'incid', 'clinic', 'bleed', 'argatroban', 'similar', 'hirudin', 'analogu', '(', 'lepirudin', ':', 'rr\\\\xa0=\\\\xa00.755', ',', '95', '%', 'ci\\\\xa0=\\\\xa00.531-1.073', ',', 'p\\\\xa0=\\\\xa00.117', ';', 'bivalirudin', ':', 'rr\\\\xa0=\\\\xa00.995', ',', '95', '%', 'ci\\\\xa0=\\\\xa00.673-1.472', ',', 'p\\\\xa0=\\\\xa00.981', ')', '.', 'current', 'evid', 'show', 'argatroban', 'similar', 'effect', 'safeti', 'lepirudin', 'bivalirudin', 'defend', 'hit.', \"'\", ',', \"'lepirudin\", 'manag', 'heparin-induc', 'thrombocytopenia.', 'lepirudin', 'recombin', 'hirudin', 'exert', 'anticoagul', 'effect', 'direct', 'inhibit', 'thrombin.', 'lepirudin', 'indic', 'anticoagul', 'patient', 'heparin-induc', 'thrombocytopenia', '(', 'hit', ')', 'associ', 'thromboembol', 'diseas', 'prevent', 'thromboembol', 'complications.', 'review', 'address', 'variou', 'clinic', 'use', 'lepirudin', 'includ', 'limit', 'fda', 'approv', 'indication.', 'object', 'provid', 'updat', 'overview', 'clinic', 'use', 'pharmacolog', 'agent.', 'addit', ',', 'address', 'certain', 'area', 'controversi', 'especi', 'pertain', 'dose', 'lepirudin', 'use', 'differ', 'clinic', 'situations.', 'literatur', 'review', 'use', 'appropri', 'search', 'term', 'pubm', 'ovid', 'medlin', 'databases.', 'lepirudin', 'continu', 'valuabl', 'anticoagul', 'manag', 'hit.', 'lepirudin', 'highest', 'recommend', 'treatment', 'hit', 'base', 'evid', 'clinic', 'trials.', 'minor', 'modif', 'dose', 'standard', 'label', 'recommend', 'may', 'simplifi', 'use', 'enhanc', 'safety.', \"'\", ',', \"'lepirudin\", ':', 'bival', 'direct', 'thrombin', 'inhibitor', 'anticoagul', 'therapy.', 'lepirudin', '(', 'refludan', ')', ',', 'berlex', 'laboratori', ',', 'usa', 'canada', ';', 'pharmion', ',', 'countri', ')', ',', 'recombin', 'deriv', 'natur', 'occur', 'leech', 'anticoagul', 'hirudin', ',', 'first', 'direct', 'thrombin', 'inhibitor', 'approv', 'european', 'agenc', 'evalu', 'medicin', 'product', 'us', 'food', 'drug', 'administr', 'treatment', 'heparin-induc', 'thrombocytopenia.', 'sinc', 'introduct', 'europ', 'usa', ',', 'studi', '7000', 'patient', 'requir', 'anticoagul', 'condit', 'includ', 'acut', 'coronari', 'syndrom', ',', 'percutan', 'coronari', 'intervent', ',', 'cardiopulmonari', 'bypass', 'heparin-induc', 'thrombocytopenia.', 'three', 'european', 'clinic', 'trial', ',', 'design', 'heparin-associ', 'thrombocytopenia', '(', 'hat', ')', '-1', ',', '-2', '-3', ',', 'demonstr', 'efficaci', 'safeti', 'lepirudin', 'prevent', 'treatment', 'thrombosi', 'patient', 'antibody-confirm', 'heparin-induc', 'thrombocytopenia.', 'postmarket', ',', 'observ', 'studi', ',', 'term', 'drug-monitor', 'program', ',', 'evalu', 'lepirudin', '1000', 'patient', 'heparin-induc', 'thrombocytopenia', 'set', 'routin', 'clinic', 'practice.', 'drug-monitor', 'program', ',', 'advers', 'event', 'substanti', 'reduc', 'compar', 'clinic', 'trial', ',', 'clinic', 'efficaci', 'maintain', ';', 'suggest', 'insight', 'gain', 'clinic', 'experi', 'translat', 'improv', 'safety.', ',', 'pharmacotherapi', 'use', 'lepirudin', 'review', ',', 'particular', 'refer', 'clinic', 'studi', 'heparin-induc', 'thrombocytopenia', ',', 'recommend', 'base', 'extens', 'clinic', 'experi', 'lepirudin', 'provided.', 'although', 'approv', 'treatment', 'heparin-induc', 'thrombocytopenia', ',', 'use', 'lepirudin', 'acut', 'coronari', 'syndrom', ',', 'percutan', 'coronari', 'intervent', ',', 'vascular', 'surgeri', 'coronari', 'arteri', 'bypass', 'graft', 'also', 'discussed.', 'review', 'conclud', 'discuss', 'pharmacokinet', 'clinic', 'data', 'support', 'potenti', 'subcutan', 'administr', 'lepirudin.', \"'\", ',', '``', 'remov', 'lepirudin', ',', 'recombin', 'hirudin', ',', 'hemodialysi', ',', 'hemofiltr', ',', 'plasmapheresis.', 'lepirudin', '(', 'recombin', 'hirudin', ')', ',', 'direct', 'thrombin', 'inhibitor', ',', 'effect', 'altern', 'method', 'anticoagul', 'patient', 'heparin-induc', 'thrombocytopenia.', 'howev', ',', 'elimin', 'kidney', ',', 'half-lif', 'lepirudin', 'may', 'substanti', 'prolong', 'patient', 'renal', 'failure.', 'patient', 'undergo', 'hemodialysi', 'must', 'close', 'monitor', ',', 'therapi', 'must', 'individu', 'base', 'patient', \"'s\", 'abil', 'clear', 'drug.', 'current', 'literatur', 'remov', 'lepirudin', 'dialysi', 'plasmapheresi', 'limit', ',', 'avail', 'data', 'suggest', 'lepirudin', 'remov', 'methods.', 'abil', 'filtrat', 'system', 'remov', 'lepirudin', 'blood', 'highli', 'depend', 'membran', 'materi', 'use', 'system.', 'understand', 'effect', 'hemodialysi', ',', 'hemofiltr', ',', 'plasmapheresi', 'lepirudin', 'level', 'import', ',', 'especi', 'sinc', 'antidot', 'avail', 'treat', 'elev', 'serum', 'lepirudin', 'concentrations.', '``', ',', \"'the\", 'direct', 'thrombin', 'inhibitor', 'dabigatran', 'lepirudin', 'inhibit', 'gpibα-medi', 'platelet', 'aggregation.', 'direct', 'thrombin', 'inhibitor', '(', 'dti', ')', 'dabigatran', 'non-vitamin', 'k', 'antagonist', 'oral', 'anticoagul', 'prevent', 'stroke', 'system', 'embol', 'patient', 'non-valvular', 'atrial', 'fibrillation.', 'addit', 'anti-thrombot', 'efficaci', ',', 'dabigatran', 'suggest', 'exert', 'pro-thrombot', 'effect', 'due', 'foster', 'ligat', 'thrombin', 'high', 'affin', 'platelet', 'receptor', 'glycoprotein', '(', 'gp', ')', 'ibα', 'patient', 'atrial', 'fibrillation.', 'hand', ',', 'provid', 'evid', 'member', 'anoth', 'class', 'dti', ',', 'lepirudin', ',', 'stimul', 'inhibitori', 'cyclic', 'guanosin', 'monophosph', '(', 'cgmp', ')', '/solubl', 'guanyl', 'cyclas', 'pathway', 'human', 'platelets.', ',', 'investig', 'effect', 'lepirudin', 'dabigatran', 'spike', 'platelet', 'healthi', 'volunt', 'gpibα-medi', 'platelet', 'aggreg', 'agglutination.', 'ristocetin/von', 'willebrand', 'factor', '(', 'vwf', ')', '-induc', 'aggreg', 'platelet', 'presenc', 'absenc', 'plasma', 'significantli', 'inhibit', 'lepirudin', ',', 'dabigatran', 'd-phenylalanyl-l-prolyl-l-arginin', 'chloromethyl', 'keton', '(', 'ppack', ')', '.', 'howev', ',', 'ristocetin/vwf-medi', 'platelet', 'agglutin', 'bind', 'vwf', 'platelet', 'affect', 'dtis.', 'anti-aggregatori', 'effect', 'confirm', 'use', 'gpibα-specif', 'agonist', 'echicetin', 'bead', 'human', 'murin', 'platelets.', 'dti', 'diminish', 'echicetin', 'beads-induc', 'syk', 'y352', 'phosphoryl', '(', 'use', 'readout', 'earli', 'signal', 'occur', 'echicetin-induc', 'platelet', 'aggreg', ')', ',', 'inhibit', 'adenosin', 'diphosphate-', 'thromboxan', 'a2-induc', 'platelet', 'aggregation.', 'thrombin', 'gener', 'respons', 'ristocetin/vwf', 'echicetin', 'bead', 'therefor', 'explain', 'inhibitori', 'effect', 'dtis.', 'therapeut', 'concentr', 'lepirudin', 'dabigatran', 'affect', 'significantli', 'platelet', 'vasodilator-stimul', 'phosphoprotein', 's239', 'phosphoryl', 'cgmp', 'cyclic', 'adenosin', 'monophosph', 'levels.', 'data', 'suggest', 'dti', ',', 'lepirudin', 'dabigatran', ',', 'impair', 'platelet', 'activ', 'measur', 'platelet', 'aggreg', 'induc', 'ristocetin/vwf', 'echicetin', 'beads.', \"'\", ',', \"'monitor\", 'argatroban', 'lepirudin', ':', 'input', 'laboratori', 'valu', '``', 'real', 'life', \"''\", '?', 'monitor', 'direct', 'thrombin', 'inhibitor', '(', 'dti', ')', 'patient', 'heparin-induc', 'thrombocytopenia', '(', 'hit', ')', 'primarili', 'perform', 'use', 'activ', 'partial', 'thromboplastin', 'time', '(', 'aptt', ')', '.', 'assay', 'poorli', 'standard', ',', 'reagent', 'depend', ',', 'dti', 'specific.', 'compar', 'aptt', ',', 'thrombin', 'time', '(', 'tt', ')', ',', 'prothrombin', 'time', '(', 'pt', ')', 'drug', 'level', 'obtain', 'ecarin', 'chromogen', 'assay', '(', 'eca', ')', '.', 'analyz', '495', 'sampl', 'patient', 'confirm', 'suspect', 'hit', 'treatment', 'either', 'argatroban', '(', 'n', '=', '37', ')', 'lepirudin', '(', 'n', '=', '80', ')', '.', 'mean', 'dti', 'level', '±', 'standard', 'deviat', '(', 'sd', ')', '0.41', '±', '0.36', 'µg/ml', 'argatroban', '0.20', '±', '0.21', 'µg/ml', 'lepirudin.', 'result', 'aptt', 'highli', 'variabl', ':', '67', '±', '22', 'second', 'argatroban', '55', '±', '20', 'second', 'lepirudin.', 'signific', 'correl', '(', 'p', '<', '.01', ')', 'found', 'eca-bas', 'dti', 'level', 'tt', '(', 'argatroban', ',', 'r', '=', '.820', 'lepirudin', ',', 'r', '=', '.830', ')', ',', 'pt', '(', 'argatroban', ',', 'r', '=', '-.544', ')', ',', 'aptt', '(', 'lepirudin', ',', 'r', '=', '.572', ')', '.', 'howev', ',', 'correl', 'aptt', 'argatroban', 'pt', 'lepirudin', 'concentration.', 'multipl', 'regress', 'analys', 'reveal', 'tt', 'predict', '54', '%', 'argatroban', '42', '%', 'lepirudin', 'level', ',', 'signific', 'impact', 'seen', 'pt', 'aptt.', 'aptt-guid', 'monitor', 'dti', 'therapi', 'lead', 'high', 'percentag', 'patient', 'inaccur', 'plasma', 'level', ',', 'henc', 'result', 'either', 'undertreat', 'overtreatment.', 'knowledg', 'baselin', 'valu', 'prior', 'dti', 'therapi', 'inclus', 'clinic', 'set', 'essenti', 'dose', 'dti', 'use', 'aptt.', 'howev', ',', 'due', 'sever', 'limit', 'aptt', ',', 'monitor', 'accord', 'exact', 'plasma', 'concentr', 'obtain', 'specif', 'test', 'eca', 'may', 'appropriate.', \"'\", ',', \"'flush\", 'central', 'venou', 'cathet', ':', 'heparin', 'necessari', '?', 'aim', 'systemat', 'review', 'assess', 'efficaci', 'heparin', 'flush', 'lock', 'central', 'venou', 'catheters.', 'search', 'medlin', 'cinahl', 'databases.', 'elig', 'studi', 'random', 'control', 'trial', 'evalu', 'use', 'heparin', 'versu', 'normal', 'salin', 'solut', 'flush', 'central', 'cathet', 'among', 'adult', 'patients.', 'languag', 'restrict', 'applied.', 'two', 'review', 'independ', 'screen', 'titl', 'abstract', 'order', 'identifi', 'relev', 'publications.', 'two', 'review', 'retriev', 'evalu', 'full', 'texts.', 'paramet', 'estim', 'regard', 'cathet', 'occlus', 'pool', 'use', 'network', 'meta-analysi', 'bayesian', 'hierarch', 'modeling.', 'identifi', '462', 'references.', 'eight', 'studi', 'included.', 'evid', 'heparin', 'effect', 'normal', 'salin', 'reduc', 'occlusions.', 'unclear', 'whether', 'urokinas', 'lepirudin', 'effect', 'heparin', 'reduc', 'occlusions.', 'vitamin', 'c', 'solut', 'appear', 'prolong', 'cathet', 'patency.', 'evid', 'differ', 'effect', 'heparin', 'flush', 'normal', 'salin', 'solut', 'reduc', 'cathet', 'occlusions.', 'due', 'littl', 'inconclus', 'evid', 'avail', 'field', ',', 'studi', 'might', 'necessary.', \"'\", ',', \"'monitor\", 'argatroban', 'lepirudin', 'anticoagul', 'critic', 'ill', 'patient', 'convent', 'laboratori', 'paramet', 'rotat', 'thromboelastometri', '-', 'prospect', 'control', 'random', 'double-blind', 'clinic', 'trial.', 'argatroban', 'lepirudin', 'anticoagul', 'therapi', 'patient', 'heparin', 'induc', 'thrombocytopenia', '(', 'hit', ')', 'hit', 'suspect', 'typic', 'monitor', 'use', 'activ', 'partial', 'thromboplastin', 'time', '(', 'aptt', ')', '.', 'although', 'aptt', 'correl', 'well', 'plasma', 'level', 'argatroban', 'lepirudin', 'healthi', 'volunt', ',', 'might', 'method', 'choic', 'critic', 'ill', 'patients.', 'howev', ',', 'in-vivo', 'data', 'lack', 'patient', 'population.', 'therefor', ',', 'studi', 'vivo', 'whether', 'rotem', 'global', 'clot', 'time', 'would', 'provid', 'altern', 'monitor', 'anticoagul', 'intens', 'effect', 'critic', 'ill', 'patients.', 'studi', 'part', 'double-blind', 'random', 'trial', '``', 'argatroban', 'versu', 'lepirudin', 'critic', 'ill', 'patient', '(', 'alicia', ')', \"''\", ',', 'compar', 'critic', 'ill', 'patient', 'treat', 'argatroban', 'lepirudin.', 'follow', 'institut', 'review', 'board', 'approv', 'written', 'inform', 'consent', ',', 'sub-studi', 'blood', '35', 'critic', 'ill', 'patient', 'analysed.', 'well', '12', ',', '24', ',', '48', '72\\\\xa0h', 'initi', 'argatroban', 'lepirudin', 'infus', ',', 'blood', 'analys', 'aptt', ',', 'aptt', 'ratio', ',', 'thrombin', 'time', '(', 'tt', ')', ',', 'intem', 'ct', ',', 'intem', 'ct', 'ratio', ',', 'extem', 'ct', ',', 'extem', 'ct', 'ratio', 'maximum', 'clot', 'firm', '(', 'mcf', ')', 'correl', 'correspond', 'plasma', 'concentr', 'direct', 'thrombin', 'inhibitor.', 'reach', 'target', 'aptt', '1.5', '2', 'time', 'baselin', ',', 'median', '[', 'iqr', ']', 'plasma', 'concentr', '0.35', '[', '0.01-1.2', ']', 'μg/ml', 'argatroban', '0.17', '[', '0.1-0.32', ']', 'μg/ml', 'lepirudin', 'required.', 'drug', ',', 'signific', 'correl', 'aptt', 'aptt', 'ratio', 'plasma', 'concentrations.', 'intem', 'ct', ',', 'intem', 'ct', 'ratio', ',', 'extem', 'ct', ',', 'extem', 'ct', 'ratio', ',', 'tt', 'tt', 'ratio', 'correl', 'significantli', 'plasma', 'concentr', 'drugs.', 'addit', ',', 'agreement', 'argatroban', 'plasma', 'level', 'extem', 'ct', 'extem', 'ct', 'ratio', 'superior', 'agreement', 'argatroban', 'plasma', 'level', 'aptt', 'bland', 'altman', 'analysis.', 'mcf', 'remain', 'unchang', 'therapi', 'drugs.', 'critic', 'ill', 'patient', ',', 'tt', 'rotem', 'paramet', 'may', 'provid', 'better', 'correl', 'argatroban', 'lepirudin', 'plasma', 'concentr', 'aptt.', 'clinicaltrials.gov', ',', 'nct00798525', ',', 'regist', '25', 'nov', '2008.', \"'\", ',', \"'xxx\", 'wherri', 'memori', 'lecture.', 'subpar', 'hearing.', \"'\", ']']\n",
      "['[', \"'ximelagatran.\", 'despit', 'signific', 'advanc', 'last', '50', 'year', 'regard', 'anticoagul', 'therapi', ',', 'warfarin', 'remain', 'definit', 'standard', 'long-term', 'prevent', 'thromboembol', 'event', 'at-risk', 'patient', ',', 'except', 'acut', 'coronari', 'syndrom', ',', 'antiplatelet', 'preferred.', 'ximelagatran', ',', 'prodrug', 'melagatran', ',', 'oral', 'administ', 'direct', 'thrombin', 'inhibitor', 'whose', 'therapeut', 'potenti', 'investig', 'venou', 'thromboembol', ',', 'acut', 'coronari', 'syndrom', 'prevent', 'stroke', 'atrial', 'fibrillation.', 'clinic', 'studi', 'demonstr', 'ximelagatran', 'compar', 'efficaci', 'oral', 'vitamin', 'k', 'antagonist', 'warfarin', 'low', 'molecular', 'weight', 'heparin', 'prophylaxi', 'venou', 'thromboembol', ',', 'compar', 'warfarin', 'stroke', 'prevent', 'set', 'atrial', 'fibril', ',', ',', 'combin', 'aspirin', ',', 'effect', 'aspirin', 'alon', 'prevent', 'major', 'advers', 'cardiovascular', 'event', 'patient', 'recent', 'myocardi', 'infarction.', 'double-blind', 'trial', 'also', 'reveal', 'efficaci', 'ximelagatran', 'secondari', 'prevent', 'venou', 'thromboembol', 'shown', 'agent', 'effect', 'enoxaparin/warfarin', 'treat', 'patient', 'acut', 'deep', 'vein', 'thrombosis.', 'advers', 'effect', 'ximelagatran', 'includ', 'elev', 'alanin', 'transaminas', '(', 'alt', ')', ',', 'may', 'requir', 'monitor', ',', 'bleed', 'complications.', 'bleed', 'complic', 'appear', 'less', 'least', 'compar', 'occur', 'standard', 'anticoagul', 'treatment', 'like', 'warfarin', 'low', 'molecular', 'weight', 'heparin.', 'addit', 'favor', 'efficaci', 'safeti', 'profil', 'comparison', 'standard', 'anticoagul', 'therapi', ',', 'conveni', 'oral', ',', 'fixed-dos', 'administr', 'without', 'need', 'anticoagul', 'monitor', 'might', 'help', 'encourag', 'wider', 'use', 'appropri', 'anticoagul', 'use', 'ximelagatran', 'across', 'popul', 'risk', ',', 'reduc', 'incid', 'thromboembol', 'events.', \"'\", ',', \"'ximelagatran\", '(', 'astrazeneca', ')', '.', 'astrazeneca', '(', 'formerli', 'astra', ')', 'develop', 'ximelagatran', ',', 'oral', 'activ', 'thrombin', 'inhibitor', 'prodrug', 'melagatran', ',', 'potenti', 'prevent', 'treatment', 'venou', 'thromboembol', '(', 'vte', ')', 'hip', 'knee', 'replac', ',', 'prevent', 'stroke', 'atrialfibril', '(', 'af', ')', 'post', 'acut', 'coronari', 'syndrome.', 'decemb', '1999', ',', 'compound', 'undergo', 'phase', 'iii', 'trial', 'prevent', 'vte', '[', '349551', ']', ';', 'ongo', 'decemb', '2001.', 'decemb', '2000', ',', 'ximelagatran', 'enter', 'phase', 'iii', 'trial', 'prevent', 'stroke', 'atrial', 'fibril', '[', '395212', ']', '.', 'septemb', '2000', ',', 'compani', 'expect', 'global', 'nda', 'file', 'indic', 'take', 'place', 'middl', '2001', '[', '383469', ']', ';', 'howev', ',', 'decemb', '2000', 'revis', 'second', 'half', '2002', '[', '395212', ']', ',', 'march', '2001', ',', 'second', 'quarter', '2003', '[', '402040', ']', ',', '[', '416882', ']', ',', '[', '431673', ']', '.', 'decemb', '2000', ',', 'phase', 'ii', 'trial', 'underway', 'ximelagatran', 'potenti', 'treatment', 'follow', 'myocardi', 'infarct', '[', '395237', ']', '.', 'decemb', '2001', ',', 'earli', 'phase', 'clinic', 'trial', 'ongo', 'post', 'acut', 'coronari', 'syndrom', '[', '431673', ']', '.', 'decemb', '2001', ',', 'eu', 'us', 'file', 'vte', 'prevent', 'anticip', 'third', 'quarter', '2002', 'second', 'quarter', '2003', ',', 'respect', '[', '3144721', ',', '[', '350050', ']', ',', '[', '431673', ']', '.', 'exanta', 'trade', 'name', 'melagatran', 'ximelagatran.', '1998', ',', 'market', 'ximelagatran', 'us', 'assign', 'astra/merek', 'joint', 'market', 'ventur', ',', 'prior', 'merger', '[', '276577', ']', '.', 'april', '1999', ',', 'abn', 'predict', 'sale', '4', 'pound', 'million', '2003', '[', '328676', ']', ',', '[', '394606', ']', '.', 'decemb', '2000', ',', 'lehman', 'brother', 'estim', 'peak', 'sale', '$', '1.4', 'billion', '[', '394606', ']', '.', 'time', ',', 'abn-amro', 'predict', 'peak', 'sale', '$', '1', 'billion', '[', '395237', ']', '.', 'june', '2000', ',', 'deutsch', 'bank', 'predict', 'sale', '$', '70', 'million', '2002', ',', 'rise', '$', '245', 'million', '2003', '[', '374500', ']', '.', 'septemb', '2000', ',', 'merril', 'lynch', 'predict', 'sale', '125', 'pound', 'million', '2002', ',', 'rise', '625', 'pound', 'million', '2004', '[', '383742', ']', '.', 'februari', '2001', ',', 'merril', 'lynch', 'estim', 'peak', 'sale', 'would', 'reach', '$', '924', 'million', '2005', '[', '429697', ']', '.', \"'\", ',', \"'ximelagatran\", 'orthopaed', 'surgery.', 'ximelagatran', 'repres', 'first', 'new', 'oral', 'anticoagul', 'sinc', 'introduct', 'warfarin', 'almost', '60', 'year', 'ago', ',', 'evalu', 'treatment', 'prevent', 'rang', 'venou', 'arteri', 'thromboembol', 'disorders.', 'melagatran', 'thromboprophylaxi', 'orthopaed', 'surgeri', '(', 'methro', ')', 'expand', 'prophylaxi', 'evalu', 'surgeri', 'studi', '(', 'express', ')', 'studi', 'investig', 'efficaci', 'safeti', 'subcutan', '(', 's.c.', ')', 'melagatran', 'follow', 'oral', 'ximelagatran', 'prevent', 'venou', 'thromboembol', '(', 'vte', ')', 'patient', 'undergo', 'total', 'hip', 'replac', 'total', 'knee', 'replacement.', 'methro', 'ii', ',', 'immedi', 'pre-operative-initi', 's.c.', 'melagatran', 'follow', 'post-op', 'ximelagatran', 'dose-depend', 'reduc', 'vte', ',', 'highest', 'dose', '(', 'melagatran', '3', 'mg/ximelagatran', '24', 'mg', 'twice', 'daili', ')', 'associ', 'significantli', 'reduc', 'incid', 'vte', 'compar', 'low-molecular-weight', 'heparin', '(', 'lmwh', ')', 'dalteparin', '(', '15.1', 'vs.', '28.2', '%', ';', 'p', '<', '0.0001', ')', '.', 'methro', 'iii', ',', 'efficaci', 's.c.', 'melagatran', '3', 'mg/ximelagatran', '24', 'mg', 'twice', 'daili', 'initi', 'post-op', '(', '4-12', 'h', 'surgeri', ')', 'compar', 'lmwh', 'enoxaparin', 'initi', '12', 'h', 'surgeri', '(', 'total', 'vte', 'incid', ',', '31.0', '27.3', '%', ',', 'respect', ')', '.', 'rate', 'sever', 'bleed', 'also', 'compar', 'treatment', '(', 'melagatran/ximelagatran', '=', '1.4', '%', ';', 'enoxaparin', '=', '1.7', '%', ')', '.', 'treatment', 'melagatran/ximelagatran', 'significantli', 'effect', 'initi', 'earlier', '(', '4-8', 'h', ')', 'rather', 'later', '(', '8-12', 'h', ')', 'surgeri', '(', 'total', 'vte', 'incid', ',', '27.5', 'vs.', '35.4', '%', ';', 'p', '=', '0.0034', ')', '.', 'base', 'result', 'methro', 'ii', 'iii', ',', 'express', 'studi', 'evalu', 'efficaci', 'bleed', 'profil', 's.c.', 'melagatran', '2', 'mg', 'immedi', 'surgeri', ',', 'follow', 's.c.', 'melagatran', '3', 'mg', 'even', 'day', 'surgeri', 'ximelagatran', '24', 'mg', 'twice', 'daily.', 'regimen', 'significantli', 'effect', 'enoxaparin', '(', 'total', 'vte', 'incid', ',', '20.3', 'vs.', '26.6', '%', ';', 'p', '<', '0.0004', ')', '.', 'excess', 'bleed', '(', 'judg', 'investig', ')', 'frequent', 'melagatran/ximelagatran', ',', 'rate', 'fatal', 'bleed', ',', 'critical-sit', 'bleed', 'bleed', 'requir', 're-oper', 'differ', 'groups.', 'taken', 'togeth', ',', 'methro', 'express', 'studi', 'demonstr', 'melagatran/ximelagatran', 'compar', 'superior', 'efficaci', 'lmwh', 'prevent', 'vte', 'orthopaed', 'surgeri', 'patient', ',', 'time', 'dose', 'melagatran', 'import', 'optim', 'balanc', 'efficaci', 'bleed', 'risk.', \"'\", ',', '``', 'ximelagatran', ':', 'new', 'type', 'oral', 'anticoagulant.', 'assess', 'sought', 'evalu', 'compar', 'benefit', 'advers', 'effect', 'profil', 'ximelagatran', ',', 'well', 'clinic', 'issu', 'surround', 'potenti', 'use.', 'perform', 'dialog', 'onesearch', 'across', 'biosi', 'preview', ',', 'embas', ',', 'medlin', ',', 'pascal', ',', 'toxfil', 'identifi', 'publish', 'literature.', 'pubm', 'cochran', 'librari', 'also', 'searched.', 'gray', 'literatur', 'identifi', 'search', 'varieti', 'web', 'site', 'health', 'technolog', 'assess', 'relat', 'agenc', 'associ', 'databases.', 'manufactur', \"'s\", 'canadian', 'offic', ',', 'astrazeneca', ',', 'invit', 'submit', 'information.', 'ximelagatran', 'first', 'oral', 'agent', 'new', 'class', 'anticoagul', 'call', 'direct', 'thrombin', 'inhibitors.', 'oral', 'anticoagul', 'requir', 'routin', 'blood', 'monitor', ';', 'ximelagatran', 'not.', 'ximelagatran', 'evalu', 'area', 'venou', 'thromboembol', 'manag', ',', 'particularli', 'orthoped', 'surgeri', ',', 'stroke', 'prevent', 'patient', 'atrial', 'fibrillation.', 'overal', ',', 'ximelagatran', \"'s\", 'efficaci', 'appear', 'compar', 'anticoagul', 'clinic', 'settings.', 'also', ',', 'bleed', 'rate', 'gener', 'similar', 'ximelagatran', 'compar', ',', 'warfarin', ',', 'bleed', 'risk', 'increas', 'higher', 'ximelagatran', 'doses.', 'addit', ',', 'specif', 'antidot', 'help', 'manag', 'ximelagatran-induc', 'bleeding.', 'final', ',', 'significantli', 'patient', 'expos', 'long-term', 'ximelagatran', 'develop', 'elev', 'liver', 'enzym', 'three', 'time', 'upper', 'normal', 'limit', ',', 'compar', 'patient', 'compar', 'anticoagulants.', 'given', 'appar', 'simplic', 'use', ',', 'ximelagatran', 'carri', 'potenti', 'replac', ',', 'least', 'part', ',', 'anticoagul', 'current', 'use', 'manag', 'venou', 'thromboembol', 'prevent', 'stroke', 'atrial', 'fibril', 'patients.', 'howev', ',', 'safeti', 'ximelagatran', 'fulli', 'known', 'without', 'evalu', 'surveil', 'potenti', 'liver', 'toxicity.', 'also', 'need', 'evalu', 'use', 'special', 'popul', 'patient', 'renal', 'failur', 'patient', 'use', 'sever', 'concurr', 'medications.', '``', ',', \"'ximelagatran/melagatran\", ':', 'review', 'use', 'prevent', 'venou', 'thromboembol', 'orthopaed', 'surgery.', 'ximelagatran', '(', 'exanta', ')', ',', 'first', 'avail', 'oral', 'direct', 'thrombin', 'inhibitor', ',', 'activ', 'form', ',', 'melagatran', ',', 'evalu', 'prevent', 'venou', 'thromboembol', '(', 'vte', ')', 'patient', 'undergo', 'hip', 'knee', 'replacement.', 'oral', 'administr', 'ximelagatran', 'rapidli', 'bioconvert', 'melagatran.', 'melagatran', 'inactiv', 'circul', 'clot-bound', 'thrombin', 'bind', 'thrombin', 'activ', 'site', ',', 'thu', ',', 'inhibit', 'platelet', 'activ', 'and/or', 'aggreg', 'reduc', 'fibrinolysi', 'time.', 'efficaci', 'subcutan', 'melagatran', 'follow', 'oral', 'ximelagatran', 'investig', 'four', 'european', 'trial', 'efficaci', 'oral', 'ximelagatran', 'regimen', 'investig', 'five', 'us', 'trials.', 'dose-rang', 'european', 'studi', ',', 'preoper', 'initi', 'subcutan', 'melagatran', '3', 'mg', 'twice', 'daili', 'follow', 'oral', 'ximelagatran', '24', 'mg', 'twice', 'daili', 'significantli', 'effect', 'subcutan', 'dalteparin', 'sodium', '5000iu', 'daili', 'prevent', 'occurr', 'vte', ',', 'includ', 'deep', 'vein', 'thrombosi', '(', 'dvt', ')', 'pulmonari', 'embol', '(', 'pe', ')', ',', 'patient', 'undergo', 'hip', 'knee', 'replacement.', 'one', 'studi', ',', 'signific', 'differ', 'vte', 'prevent', 'subcutan', 'melagatran', '3', 'mg', 'administ', 'surgeri', 'follow', 'ximelagatran', '24', 'mg', 'twice', 'daili', 'enoxaparin', 'sodium', '(', 'enoxaparin', ')', '40', 'mg', 'daily.', 'compar', 'enoxaparin', ',', 'significantli', 'lower', 'rate', 'proxim', 'dvt', 'and/or', 'pe', '(', 'major', 'vte', ')', 'total', 'vte', 'observ', 'melagatran', 'initi', 'preoper', '(', '2mg', ')', 'postop', '(', '3mg', ')', 'follow', 'ximelagatran', '24', 'mg', 'twice', 'daily.', 'us', ',', 'four', 'studi', 'show', 'postop', 'initi', 'ximelagatran', '24', 'mg', 'twice', 'daili', 'similar', 'efficaci', 'enoxaparin', 'warfarin', 'prevent', 'vte', 'patient', 'undergo', 'hip', 'knee', 'replacement.', 'howev', ',', 'ximelagatran', '36', 'mg', 'twice', 'daili', 'superior', 'warfarin', '(', 'target', 'intern', 'normalis', 'ratio', '2.5', ')', 'prevent', 'incid', 'vte', 'patient', 'undergo', 'total', 'knee', 'replac', 'two', 'studies.ximelagatran', 'alon', 'melagatran', 'gener', 'well', 'tolerated.', 'overal', ',', 'incid', 'bleed', 'event', 'transfus', 'rate', 'markedli', 'differ', 'document', 'compar', 'anticoagulants.', 'post-hoc', 'analysi', 'one', 'studi', ',', 'transfus', 'rate', 'lower', 'ximelagatran', 'enoxaparin', 'recipients.', 'oral', 'ximelagatran', 'alon', 'conjunct', 'subcutan', 'melagatran', 'shown', 'good', 'efficaci', 'gener', 'well', 'toler', 'prevent', 'vte', 'patient', 'undergo', 'orthopaed', 'surgery.', 'furthermor', ',', 'patient', 'receiv', 'ximelagatran/melagatran', 'requir', 'anticoagul', 'monitoring.', 'drug', 'low', 'potenti', 'drug', 'interact', 'administ', 'either', 'subcutan', 'inject', 'orally.', 'thu', ',', 'basi', 'avail', 'evid', ',', 'ximelagatran/melagatran', 'appear', 'pois', 'play', 'import', 'role', 'prophylaxi', 'vte', 'patient', 'undergo', 'orthopaed', 'surgery.', \"'\", ',', \"'ximelagatran\", ':', 'new', 'oral', 'anticoagulant.', 'although', 'mani', 'signific', 'advanc', 'last', '50', 'year', 'regard', 'anticoagul', 'therapi', ',', 'warfarin', 'remain', 'definit', 'standard', 'long-term', 'prevent', 'thromboembol', 'event', 'mani', 'patient', 'risk', 'complications.', 'although', 'effect', ',', 'warfarin', 'narrow', 'therapeut', 'window', ',', 'necessit', 'frequent', 'laboratori', 'monitor', 'anticoagul', 'effect.', 'ximelagatran', 'investig', 'anticoagul', 'directli', 'inhibit', 'thrombin', ',', 'unlik', 'heparin', 'warfarin', ',', 'indirect', 'inhibitors.', 'although', 'indirect', 'thrombin', 'inhibitor', 'mainli', 'effect', 'inhibit', 'circul', 'thrombin', ',', 'direct', 'thrombin', 'inhibitor', 'abl', 'inhibit', 'free', 'clot-bound', 'thrombin', ',', 'therebi', 'produc', 'effect', 'anticoagulation.', 'ximelagatran', 'first', 'oral', 'avail', 'direct', 'thrombin', 'inhibitor', 'reach', 'phase', '3', 'clinic', 'trials.', 'ximelagatran', 'prodrug', 'activ', 'metabolit', 'melagatran', ',', 'demonstr', 'rel', 'wide', 'therapeut', 'window', 'term', 'bleed', 'antithrombot', 'effect', 'compar', 'warfarin.', 'clinic', 'studi', 'demonstr', 'ximelagatran', 'compar', 'efficaci', 'warfarin', 'low-molecular-weight', 'heparin', '(', 'lmwh', ')', 'prophylaxi', 'venou', 'thromboembol', ',', 'compar', 'warfarin', 'stroke', 'prevent', 'set', 'atrial', 'fibril', ',', ',', 'combin', 'aspirin', ',', 'possibl', 'effect', 'aspirin', 'alon', 'prevent', 'major', 'advers', 'cardiovascular', 'event', 'patient', 'recent', 'myocardi', 'infarction.', 'advers', 'effect', 'ximelagatran', 'primarili', 'involv', 'bleed', 'complic', ',', 'frequent', 'placebo', ',', 'appear', 'compar', 'occur', 'standard', 'anticoagul', 'treatment', '(', 'ie', ',', 'warfarin', 'lmwh', ')', '.', 'ximelagatran', 'also', 'demonstr', 'caus', 'transient', 'increas', 'liver', 'enzym', ',', 'signific', 'need', 'address', 'ongo', 'phase', '3', 'studies.', 'ongo', 'trial', 'prove', 'ximelagatran', 'least', 'similar', 'therapeut', 'efficaci', 'safeti', 'warfarin', ',', 'ximelagatran', 'may', 'becom', 'first-lin', 'anticoagul', 'due', 'eas', 'administr', 'lack', 'need', 'drug', 'monitoring.', 'result', 'trial', 'eagerli', 'await', 'help', 'defin', 'place', 'therapi', 'promis', 'new', 'agent.', \"'\", ',', \"'ximelagatran\", ':', 'direct', 'thrombin', 'inhibitor.', 'warfarin', 'sodium', 'effect', 'oral', 'anticoagul', 'drug.', 'howev', ',', 'warfarin', 'narrow', 'therapeut', 'window', 'signific', 'risk', 'hemorrhag', 'therapeut', 'concentrations.', 'dose', 'difficult', 'requir', 'frequent', 'monitoring.', 'new', 'oral', 'anticoagul', 'agent', 'requir', 'improv', 'current', 'anticoagul', 'therapy.', 'furthermor', ',', 'warfarin', 'effect', 'venou', 'diseas', ',', 'provid', '60', '%', 'risk', 'reduct', 'compar', 'placebo', 'venou', 'thrombosi', 'prophylaxi', 'consider', 'lower', 'risk', 'reduct', 'term', 'arteri', 'thrombosis.', 'ximelagatran', 'oral', 'pro-drug', 'melagatran', ',', 'synthet', 'small', 'peptidomimet', 'direct', 'thrombin', 'inhibitori', 'action', 'anticoagul', 'activity.', 'oral', 'agent', ',', 'ximelagatran', 'number', 'desir', 'properti', 'includ', 'rapid', 'onset', 'action', ',', 'fix', 'dose', ',', 'stabl', 'absorpt', ',', 'appar', 'low', 'potenti', 'medic', 'interact', ',', 'requir', 'monitor', 'drug', 'level', 'dose', 'adjustment.', 'short', 'plasma', 'elimin', 'half-lif', '4', 'hour', 'case', 'unexpect', 'hemorrhag', 'need', 'reversal.', 'main', 'toxic', 'relat', 'develop', 'abnorm', 'liver', 'biochemistri', 'and/or', 'liver', 'dysfunct', '``', 'long-term', \"''\", 'use', 'drug.', 'usual', 'occur', 'within', 'first', '6', 'month', 'commenc', 'therapi', ',', 'small', 'percentag', 'patient', 'develop', 'jaundice.', 'biochem', 'abnorm', 'usual', 'resolv', 'despit', 'continu', 'drug.', 'caus', 'toxic', 'remain', 'unknown.', 'clinic', 'studi', 'date', 'shown', 'ximelagatran', 'noninferior', 'warfarin', 'stroke', 'prevent', 'patient', 'nonvalvular', 'atrial', 'fibril', ',', 'noninferior', 'standard', 'therapi', 'acut', 'extend', 'therapi', 'deep', 'vein', 'thrombosi', '(', 'dvt', ')', ',', 'superior', 'warfarin', 'prevent', 'venou', 'thromboembol', 'post-major', 'orthoped', 'surgery.', 'also', 'shown', 'effect', 'aspirin', 'alon', 'prevent', 'recurr', 'major', 'cardiovascular', 'event', 'patient', 'recent', 'myocardi', 'infarction.', \"'\", ',', \"'ximelagatran\", ',', 'new', 'oral', 'anticoagul', ':', 'would', 'warfarin', 'surviv', 'challeng', '?', 'last', 'decad', 'wit', 'major', 'advanc', 'prevent', 'treatment', 'venou', 'well', 'arteri', 'thrombosis.', 'limit', 'exist', 'anticoagul', 'led', 'develop', 'novel', 'therapeut', 'approaches.', 'ximelagatran', 'new', 'direct', 'thrombin', 'inhibitor', '(', 'dti', ')', 'given', 'oral', ',', 'without', 'need', 'close', 'monitoring.', 'compound', 'tri', 'treatment', 'activ', 'venou', 'thromboembol', ',', 'result', 'encouraging.', 'random', 'trial', 'suggest', 'ximelagatran', 'inferior', 'warfarin', 'prevent', 'stroke', 'patient', 'nonvalvular', 'atrial', 'fibrillation.', 'multipl', 'control', ',', 'prospect', 'trial', 'compar', 'ximelagatran', 'low', 'molecular', 'weight', 'heparin', 'warfarin', 'prevent', 'venou', 'thromboembol', 'patient', 'undergo', 'major', 'orthoped', 'procedures.', 'result', 'clinic', 'trial', 'review', 'article.', 'certain', 'safeti', 'concern', '(', 'increas', 'liver', 'enzym', ',', 'potenti', 'hepatonecrosi', ',', 'increas', 'coronari', 'event', ')', 'ximelagatran', 'yet', 'approv', 'fda.', 'addit', 'studi', 'may', 'requir', 'address', 'concerns.', 'ximelagatran', 'approv', ',', 'howev', ',', 'european', 'regulatori', 'author', 'short-term', 'thromboprophylaxis.', 'success', 'ximelagatran', 'oral', 'antithrombin', 'agent', 'would', 'provid', 'signific', 'proof', 'concept', 'long-term', 'use', 'oral', 'antithrombin', 'prevent', 'treatment', 'arteri', 'venou', 'thrombosis.', \"'\", ',', \"'ximelagatran\", 'treatment', 'prophylaxi', 'recurr', 'event', 'deep', 'vein', 'thrombosis.', 'treatment', 'acut', 'venou', 'thromboembol', 'prophylaxi', 'recurr', 'event', 'heparin/low', 'molecular', 'weight', 'heparin', 'follow', 'vitamin', 'k', 'antagonist', 'limit', 'sever', 'factors.', 'oral', 'direct', 'thrombin', 'inhibitor', '(', 'odti', ')', 'show', 'better', 'pharmacolog', 'activ', 'might', 'altern', 'treatment', 'venou', 'thromboembolism.', 'thrombin', 'inhibit', 'venou', 'thromboembol', '(', 'thrive', ')', 'program', 'perform', 'studi', 'develop', 'odti', 'ximelagatran', 'indic', ',', 'present', 'overview.', 'aim', 'thrive', 'studi', 'dose', 'find', ',', 'thrive', 'iv', 'studi', 'applic', 'hemodynam', 'stabil', 'pulmonari', 'embolism.', 'prospect', ',', 'random', ',', 'doubl', 'blind', 'trial', 'perform', 'compar', 'oral', 'ximelagatran', 'enoxaparin/warfarin', '6-month', 'treatment', 'acut', 'venou', 'thrombosi', '(', 'thrive', 'ii', 'v', ')', '.', 'second', 'doubl', 'blind', 'studi', 'compar', 'ximelagatran', 'placebo', '18', 'month', '6-month', 'anticoagul', 'therapi', 'acut', 'deep', 'vein', 'thrombosis.', 'efficaci', 'safeti', 'treatment', 'patient', 'acut', 'deep', 'venou', 'thrombosi', 'receiv', '2', 'infin', '36', 'mg', 'ximelagatran', 'inferior', 'patient', 'receiv', 'convent', 'anticoagul', 'prophylaxi', 'recurr', 'event', '6', 'months.', 'ximelagatran', '2', 'infin', '24', 'mg', 'significantli', 'reduc', 'recurr', 'thromboembol', 'event', 'compar', 'placebo', 'without', 'increas', 'risk', 'hemorrhage.', 'revers', 'symptomless', 'increas', 'alanin', 'aminotransferas', 'occur', '6', '%', '9.6', '%', 'patient', 'month', '2', '4.', 'result', 'follow-up', 'studi', 'suggest', 'thromboembol', 'event', 'may', 'recur', 'patient', 'acut', 'venou', 'thromboembol', 'termin', 'treatment', 'vitamin', 'k', 'antagonist', 'ximelagatran.', \"'\", ',', \"'nonimmun\", 'hydrop', 'fetali', 'fetal', 'congenit', 'syphilis.', 'case', 'report.', 'nonimmun', 'hydrop', 'fetali', 'occur', 'secondari', 'syphilit', 'infection.', 'ultrasonograph', 'evalu', 'cordocentesi', 'use', 'confirm', 'antenat', 'infection.', 'igm', 'antibodi', 'specif', 'treponema', 'pallidum', 'wall', 'antigen', '(', 'anti-47-kda', ')', 'use', 'document', 'fetal', 'infection.', 'high-dos', 'intraven', 'penicillin', 'administ', 'attempt', 'achiev', 'adequ', 'fetal', 'levels.', \"'\", ']']\n",
      "------------------------------------------------------------\n",
      "########################################\n"
     ]
    }
   ],
   "source": [
    "# Document Clustering\n",
    "print(\"##\"*30)\n",
    "print(\"K-Means Clustering\")\n",
    "print(\"##\"*30)\n",
    "Clustering_Method = KMeans(n_clusters=2, random_state=0)\n",
    "X = model.docvecs.vectors_docs # document vector 전체를 가져옴. \n",
    "Clustering_Method.fit(X)# fitting \n",
    "\n",
    "cluster_dict = {i:[] for i in range(0, 2)}\n",
    "for text_tags, label in zip(texts_and_tags, Clustering_Method.labels_):\n",
    "    text, tags = text_tags\n",
    "    cluster_dict[label].append(text)\n",
    "for label, lst in cluster_dict.items():\n",
    "    print(f\"Cluster {label}\")\n",
    "    for x in lst:\n",
    "        print(x)\n",
    "    print(\"--\"*30)\n",
    "print(\"##\"*20)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
